{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "c04bfa80",
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import numpy as np\n",
    "\n",
    "from warnings import filterwarnings\n",
    "filterwarnings('ignore')\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "01f9cf06",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Unnamed: 0</th>\n",
       "      <th>Keywords</th>\n",
       "      <th>Payors/LBM</th>\n",
       "      <th>Policy url</th>\n",
       "      <th>Policy URL Correct/Incorrect Flag</th>\n",
       "      <th>\"Covered/Yes\" Language</th>\n",
       "      <th>\"Not Covered/No\" Language</th>\n",
       "      <th>Vague / non-No</th>\n",
       "      <th>\"Conditional/Need PA\" language</th>\n",
       "      <th>Notable patterns within document</th>\n",
       "      <th>Unnamed: 9</th>\n",
       "      <th>link_status</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0</td>\n",
       "      <td>81521 OR MammaPrint Breast Cancer Recurrence S...</td>\n",
       "      <td>Aetna</td>\n",
       "      <td>https://www.aetna.com/cpb/medical/data/300_399...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>BrCA testing  is covered,</td>\n",
       "      <td>NaN</td>\n",
       "      <td>not listed in investigational and experimental</td>\n",
       "      <td>NaN</td>\n",
       "      <td>81521 and MammaPrint do not appear in policy, ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>HTML - Working</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1</td>\n",
       "      <td>81552 OR DecisionDx-UM OR Gene expression prof...</td>\n",
       "      <td>Aetna</td>\n",
       "      <td>https://www.aetna.com/cpb/medical/data/300_399...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>?</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "      <td>NaN</td>\n",
       "      <td>81552  and test name do not appear in list of ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>HTML - Working</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   Unnamed: 0                                           Keywords Payors/LBM  \\\n",
       "0           0  81521 OR MammaPrint Breast Cancer Recurrence S...      Aetna   \n",
       "1           1  81552 OR DecisionDx-UM OR Gene expression prof...      Aetna   \n",
       "\n",
       "                                          Policy url  \\\n",
       "0  https://www.aetna.com/cpb/medical/data/300_399...   \n",
       "1  https://www.aetna.com/cpb/medical/data/300_399...   \n",
       "\n",
       "   Policy URL Correct/Incorrect Flag      \"Covered/Yes\" Language  \\\n",
       "0                                NaN  BrCA testing  is covered,    \n",
       "1                                NaN                           ?   \n",
       "\n",
       "  \"Not Covered/No\" Language                                  Vague / non-No  \\\n",
       "0                       NaN  not listed in investigational and experimental   \n",
       "1                                                                             \n",
       "\n",
       "  \"Conditional/Need PA\" language  \\\n",
       "0                            NaN   \n",
       "1                            NaN   \n",
       "\n",
       "                    Notable patterns within document Unnamed: 9  \\\n",
       "0  81521 and MammaPrint do not appear in policy, ...        NaN   \n",
       "1  81552  and test name do not appear in list of ...        NaN   \n",
       "\n",
       "      link_status  \n",
       "0  HTML - Working  \n",
       "1  HTML - Working  "
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data = pd.read_csv(\"Link_status_updated.xlsx - Sheet1.csv\")\n",
    "data.head(2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 96,
   "id": "68ac6569",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['81521 ',\n",
       " ' MammaPrint Breast Cancer Recurrence Score ',\n",
       " ' Gene expression profiling for Breast Cancer Prognosis ',\n",
       " ' Genetic expression for breast cancer prognosis']"
      ]
     },
     "execution_count": 96,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data.Keywords[0].split('OR')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 97,
   "id": "b4f8496c",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "array(['HTML - Working', 'Not Working', 'Invalid Link', 'PDF - Working',\n",
       "       'Other - Working'], dtype=object)"
      ]
     },
     "execution_count": 97,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data['link_status'].unique()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 95,
   "id": "51475244",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Unnamed: 0</th>\n",
       "      <th>Keywords</th>\n",
       "      <th>Payors/LBM</th>\n",
       "      <th>Policy url</th>\n",
       "      <th>Policy URL Correct/Incorrect Flag</th>\n",
       "      <th>\"Covered/Yes\" Language</th>\n",
       "      <th>\"Not Covered/No\" Language</th>\n",
       "      <th>Vague / non-No</th>\n",
       "      <th>\"Conditional/Need PA\" language</th>\n",
       "      <th>Notable patterns within document</th>\n",
       "      <th>Unnamed: 9</th>\n",
       "      <th>link_status</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0</td>\n",
       "      <td>81521 OR MammaPrint Breast Cancer Recurrence S...</td>\n",
       "      <td>Aetna</td>\n",
       "      <td>https://www.aetna.com/cpb/medical/data/300_399...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>BrCA testing  is covered,</td>\n",
       "      <td>NaN</td>\n",
       "      <td>not listed in investigational and experimental</td>\n",
       "      <td>NaN</td>\n",
       "      <td>81521 and MammaPrint do not appear in policy, ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>HTML - Working</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1</td>\n",
       "      <td>81552 OR DecisionDx-UM OR Gene expression prof...</td>\n",
       "      <td>Aetna</td>\n",
       "      <td>https://www.aetna.com/cpb/medical/data/300_399...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>?</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "      <td>NaN</td>\n",
       "      <td>81552  and test name do not appear in list of ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>HTML - Working</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>2</td>\n",
       "      <td>81416 OR Exome sequencing OR WES OR Exome sequ...</td>\n",
       "      <td>Aetna</td>\n",
       "      <td>https://www.aetna.com/cpb/medical/data/100_199...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>\"CPT codes covered if selection critera met\"</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>HTML - Working</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   Unnamed: 0                                           Keywords Payors/LBM  \\\n",
       "0           0  81521 OR MammaPrint Breast Cancer Recurrence S...      Aetna   \n",
       "1           1  81552 OR DecisionDx-UM OR Gene expression prof...      Aetna   \n",
       "2           2  81416 OR Exome sequencing OR WES OR Exome sequ...      Aetna   \n",
       "\n",
       "                                          Policy url  \\\n",
       "0  https://www.aetna.com/cpb/medical/data/300_399...   \n",
       "1  https://www.aetna.com/cpb/medical/data/300_399...   \n",
       "2  https://www.aetna.com/cpb/medical/data/100_199...   \n",
       "\n",
       "   Policy URL Correct/Incorrect Flag  \\\n",
       "0                                NaN   \n",
       "1                                NaN   \n",
       "2                                NaN   \n",
       "\n",
       "                         \"Covered/Yes\" Language \"Not Covered/No\" Language  \\\n",
       "0                    BrCA testing  is covered,                        NaN   \n",
       "1                                             ?                             \n",
       "2  \"CPT codes covered if selection critera met\"                       NaN   \n",
       "\n",
       "                                   Vague / non-No  \\\n",
       "0  not listed in investigational and experimental   \n",
       "1                                                   \n",
       "2                                             NaN   \n",
       "\n",
       "  \"Conditional/Need PA\" language  \\\n",
       "0                            NaN   \n",
       "1                            NaN   \n",
       "2                            NaN   \n",
       "\n",
       "                    Notable patterns within document Unnamed: 9  \\\n",
       "0  81521 and MammaPrint do not appear in policy, ...        NaN   \n",
       "1  81552  and test name do not appear in list of ...        NaN   \n",
       "2                                                NaN        NaN   \n",
       "\n",
       "      link_status  \n",
       "0  HTML - Working  \n",
       "1  HTML - Working  \n",
       "2  HTML - Working  "
      ]
     },
     "execution_count": 95,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data_html=data[data['link_status']=='HTML - Working'].copy()\n",
    "data_html.head(3)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "09ca02f2",
   "metadata": {},
   "source": [
    "### oPTIONAL edA"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "23234634",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "PandasGUI INFO — pandasgui.gui — Opening PandasGUI\n",
      "2023-10-31 16:52:54,264 - INFO     - Opening PandasGUI\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "<pandasgui.gui.PandasGui at 0x1d6c0c9a940>"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import pandasgui as pg\n",
    "pg.show(data)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "97fa78a5",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\Users\\Dhananjay\\anaconda3\\lib\\site-packages\\dtale\\views.py:785: FutureWarning:\n",
      "\n",
      "['Keywords', 'Payors/LBM', 'Policy url', '\"Covered/Yes\" Language', '\"Not Covered/No\" Language', 'Vague / non-No', '\"Conditional/Need PA\" language', 'Notable patterns within document', 'Unnamed: 9'] did not aggregate successfully. If any error is raised this will raise in a future version of pandas. Drop these columns/ops to avoid this warning.\n",
      "\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "\n",
       "        <iframe\n",
       "            width=\"100%\"\n",
       "            height=\"475\"\n",
       "            src=\"http://LAPTOP-0PG7UMRR:40000/dtale/iframe/1\"\n",
       "            frameborder=\"0\"\n",
       "            allowfullscreen\n",
       "            \n",
       "        ></iframe>\n",
       "        "
      ],
      "text/plain": [
       "<IPython.lib.display.IFrame at 0x2552dccff70>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/plain": []
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import dtale as dt\n",
    "dt.show(data)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "id": "c004643c",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "1614"
      ]
     },
     "execution_count": 24,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(data['Policy url'].unique())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 216,
   "id": "78b8eace",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "990"
      ]
     },
     "execution_count": 216,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data_html[\"Payors/LBM\"].value_counts().head(7).values.sum()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "a8c02d48",
   "metadata": {},
   "outputs": [],
   "source": [
    "url=[]\n",
    "for i in data_html['Policy url']:\n",
    "    if i[-4:]==\"html\":\n",
    "        url.append(i)    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "c80d97c3",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'https://www.anthem.com/dam/medpolicies/abc/active/policies/mp_pw_a049877.html'"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "url[148]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "55ba7391",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "id": "35a72eea",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'https://www.aetna.com/cpb/medical/data/300_399/0352.html'"
      ]
     },
     "execution_count": 43,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "url=url[0]\n",
    "url"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "id": "f7f073f5",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      "\n",
      "\n",
      "Tumor Markers - Medical Clinical Policy Bulletins | Aetna\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "-->\n",
      " \n",
      "   \n",
      " \n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Top\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Tumor Markers\n",
      "\n",
      "\n",
      "\n",
      " \n",
      " \n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Clinical Policy Bulletins\n",
      "Medical Clinical Policy Bulletins\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Print\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Share\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Number: 0352\n",
      "Table Of Contents\n",
      "Policy\n",
      "Applicable CPT / HCPCS / ICD-10 Codes\n",
      "Background\n",
      "References\n",
      "\n",
      "Policy\n",
      "Scope of Policy\n",
      "This Clinical Policy Bulletin addresses tumor markers, including somatic (acquired) mutations, in oncology.\n",
      "For criteria related to germline (inherited) mutations, see CPB 0140 - Genetic Testing.\n",
      "\n",
      "\n",
      "Medical Necessity\n",
      "\n",
      "\n",
      "Aetna considers any of the following tumor markers for the stated indication medically necessary:\n",
      "\n",
      "1p19q codeletion molecular cytogenetic analysis for astrocytomas and gliomas;\n",
      "5-hydroxyindoleacetic acid (5-HIAA) for neuroendocrine tumors;\n",
      "Afirma Thyroid FNA analysis for assessing fine needle aspiration samples from thyroid nodules that are indeterminate; experimental for other indications.  Repeat testing is considered experimental and investigational;\n",
      "ALK expression for pancreatic adenocarcinoma, pediatric Hodgkin's lymphoma, inflammatory myofibroblastic tumor (IMT) with ALK translocation, breast implant-associated ALCL, peripheral T-cell lymphoma, and uterine sarcoma;\n",
      "ALK gene fusion as a molecular biomarker in non-small cell lung cancer;\n",
      "ALK gene rearrangement for diffuse large B cell lymphoma, anaplastic thyroid carcinoma, primary cutaneous CD30+ T-cell lymphoproliferative disorders, post-transplant lymphoproliferative disorder, and non-small cell lung cancer;\n",
      "Alpha fetoprotein (AFP) for testing for hepatocellular carcinoma in hepatitis B carriers, or for persons with cirrhosis and one or more of the following risk factors: alcohol use; alpha-1 antitrypsin deficiency; Asian female at least 50 years of age; Asian male at least 40 years of age; family history of HCC; genetic hemochromatosis; hepatitis C; nonalcoholic steatohepatitis; and stage 4 primary biliary cirrhosis;\n",
      "Alpha fetoprotein (AFP) for the following indications: hepatocellular carcinoma; mediastinal mass; ovarian cancer; pelvic mass; testicular cancer; testicular mass; thymic carcinoma; and thymoma;\n",
      "Alpha fetoprotein (AFP): serial measurements to diagnose germ cell tumors in members with adenocarcinoma, or carcinoma not otherwise specified, involving mediastinal nodes; or the diagnosis and monitoring of hepatocellular carcinoma (e.g., before considering liver transplantation);\n",
      "Androgen receptor splice variant 7 (AR-V7) in circulating tumor cells to select therapy in metastatic castrate-resistant prostate cancer after progression on abiraterone or enzalutamide;\n",
      "APC for familial adenomatous polyposis when criteria are met in CPB 0140 - Genetic Testing; and for desmoid fibromatosis; experimental for other indications;\n",
      "BCL2 and BCL6 for the diagnosis of non-Hodgkinâs lymphoma and Castleman's disease;\n",
      "BCR/ABL fluorescent in situ hybridization (FISH) for lymphoblastic lymphoma, acute myeloid leukemia, acute lymphocytic leukemia, chronic myelogenous leukemia, and suspected myeloproliferative neoplasm; experimental for other indications;\n",
      "Beta-2 microglobulin (B2M) for multiple myeloma, non-Hodgkin's lymphoma and WaldenstrÃ¶m's macroglobulinemia/ lymphoplasmacytic lymphoma;\n",
      "BIRC3 and MALT1 for gastric MALT lymphoma, non-gastric MALT lymphoma, nodal marginal zone lymphoma, and splenic marginal zone lymphoma; \n",
      "BRAF V600 mutation for indeterminate thyroid nodules, hairy cell leukemia; gastrointestinal stromal tumors; colorectal cancer, Lynch syndrome; non-small cell lung cancer; thyroid carcinoma; infiltrative glioma, pancreatic adenocarcinoma, and melanoma (see CPB 0715 - Pharmacogenomic and Pharmacodynamic Testing); or Lynch syndrome for persons meeting criteria in CPB 0140 - Genetic Testing; and colorectal cancer if KRAS nonmutated; experimental for other indications;\n",
      "Breast Cancer IndexFootnote2** to assess necessity of adjuvant chemotherapy or adjuvant endocrine therapy in females or males with recently diagnosed breast tumors, where all of the following criteria are met:\n",
      "\n",
      "Breast cancer is nonmetastatic (node negative) or with 1-3 involved ipsilateral axillary lymph nodes; and\n",
      "Breast tumor is estrogen receptor and/or progesterone receptor positive; and\n",
      "Breast tumor is HER2 receptor negative; and\n",
      "Adjuvant therapy is not precluded due to any other factor (e.g., advanced age and/or significant co-morbidities); and\n",
      "Member and physician (prior to testing) have discussed the potential results of the test and agree to use the results to guide therapy;\n",
      "\n",
      "BCI is also considered medically necessary for persons with breast cancer with 0-3 positive nodes who received 5 years of endocrine therapy without recurrence to guide decisions about extended endocrine therapy. \n",
      "\n",
      "BTK (Bruton's tyrosine kinase) for chronic lymphocytic leukemia/small lymphocytic lymphoma;\n",
      "CA 15-3: Serial measurements of CA 15-3 (also known as CA 27-29 or Truquant RIA) in following the course of treatment in women diagnosed with breast cancer, especially advanced metastatic breast cancer (an increasing CA 15-3 level may suggest treatment failure);\n",
      "CA 19-9 for the following indications:\n",
      "\n",
      "To monitor the clinical response to therapy or detect early recurrence of disease in members with known gastric cancer, pancreatic cancer, gallbladder cancer, cholangiocarcinoma, ovarian cancer, small bowel adenocarcinoma, or adenocarcinoma of the ampulla of Vater; or\n",
      "To rule out cholangiocarcinoma in persons with primary sclerosing cholangitis undergoing liver transplantation; or\n",
      "For evaluation of jaundice, abnormal liver function tests (LFTs) or hepatobiliary obstruction/abnormality on abdominal imaging; or\n",
      "As a tumor marker for mucinous appendiceal carcinoma;\n",
      "\n",
      "\n",
      "CALB2 (calretinin) expression for lung cancer and occult primary;\n",
      "CALCA (calcitonin) expression for medullary thyroid cancer or for adenocarcinoma or anaplastic/undifferentiated tumors of the head and neck;\n",
      "CALR (calreticulin) for chronic myeloid leukemia (chronic phase, adult), myelodysplastic syndrome, or myeloproliferative neoplasms;\n",
      "Cancer antigen 125 (CA 125) levels for any of the following: \n",
      "\n",
      "As a preoperative diagnostic aid in women with ovarian masses that are suspected to be malignant, such that arrangements can be made for intraoperative availability of a gynecological oncologist if the CA 125 is increased; or\n",
      "As a screening test for ovarian cancer when there is a family history of hereditary ovarian cancer syndrome (a pattern of clusters of ovarian cancer within two or more generations), where testing is performed concurrently with transvaginal ultrasound and prophylactic salpingo-oophorectomy has not been performed. For this indication, screening is considered medically necessary every six months beginning at 30 years of age or 10 years before the earliest age of the first diagnosis of ovarian cancer in the family; or\n",
      "Diagnosis of ovarian cancer in women with new symptoms (bloating, pelvic or abdominal pain, difficulty eating or feeling full quickly, or urinary frequency and urgency) that have persisted for three or more weeks, where the clinician has performed a pelvic and rectal examination and suspects ovarian cancer; or\n",
      "In members with adenocarcinoma of unknown primary, to rule out ovarian cancer; or\n",
      "In members with known ovarian cancer, as an aid in the monitoring of disease, response to treatment, detection of recurrent disease, or assessing value of performing second-look surgery;\n",
      "\n",
      "\n",
      "\n",
      "Carcinoembryonic antigen (CEA) for any of the following:\n",
      "\n",
      "As a preoperative prognostic indicator in members with known colorectal carcinoma or mucinous appendiceal carcinoma when it will assist in staging and surgical treatment planning; or \n",
      "Pancreatic cyst fluid CEA for distinguishing mucinous from non-mucinous malignant pancreatic cysts; or\n",
      "To detect asymptomatic recurrence of colorectal cancer after surgical and/or medical treatment for the diagnosis of colorectal cancer (not as a screening test for colorectal cancer); or\n",
      "To monitor response to treatment for metastatic colorectal cancer; or\n",
      "For cholangiocarcinoma, gallbladder cancer, lung cancer, medullary thyroid cancer, metastatic breast cancer, mucinous ovarian cancer, and occult primary; or\n",
      "For evaluation of jaundice, abnormal liver function tests (LFTs) or for obstruction/abnormality of the bile duct on liver imaging;\n",
      "\n",
      "\n",
      "CBFB for acute myeloid leukemia;\n",
      "CCND1 (cyclin D1) for B-cell lymphomas, primary cutaneous B-cell lymphomas, chronic lymphocytic leukemia/small lymphocytic lymphoma, and hairy cell leukemia;\n",
      "CD 20, for determining eligibility for anti-CD20 treatment (rituximab) (see CPB 0314 - Rituximab);\n",
      "CD 25, for determining eligibility for denileukin diftitox (Ontak) treatment;\n",
      "CD 31 immunostaining, for diagnosis of angiosarcoma;\n",
      "CD 33, for lymphoblastic lymphoma and for determining eligibility for anti-CD33 (gemtuzumab, Mylotarg) treatment;\n",
      "CD 52, for post-transplant lymphoproliferative disorder, T-cell prolymphocytic leukemia, and for determining eligibility for anti-CD52 (alemtuzumab, Campath) treatment;\n",
      "CD117 (c-kit), for acute myeloid leukemia, cutaneous melanoma, gastrointestinal stromal tumors and systemic mastocytosis;\n",
      "CHGA (Chromogranin A) expression for neuroendocrine tumors, non-small cell lung cancer, small cell lung cancer, Merkel cell carcinoma and occult primary;\n",
      "DecipherFootnote3*** for the following indications:\n",
      "\n",
      "Post biopsy in men with NCCN very-low-risk, low-risk, and favorable intermediate-risk prostate cancer who have a greater than 10 year life expectancy who have not received treatment for prostate cancer and are candidates for active surveillance or definitive therapy; or\n",
      "Post biopsy in men with intermediate-risk prostate cancer when deciding whether to add androgen-deprivation therapy to radiation; or\n",
      "Men with an undetectable PSA after prostatectomy for prostate cancer, to determine adjuvant versus salvage radiation therapy or to determine whether to initiate systemic therapies;\n",
      "\n",
      "\n",
      "DecisionDx-UM (Castle Biosciences, Phoenix, AZ) for risk stratification of persons with localized uveal melanoma;\n",
      "EndoPredict (also known as 12-gene score)Footnote2** to assess necessity of adjuvant chemotherapy in females or males with recently diagnosed breast tumors, where all of the following criteria are met:\n",
      "\n",
      "Breast cancer is nonmetastatic (node negative) or with 1-3 involved ipsilateral axillary lymph nodes; and\n",
      "Breast tumor is estrogen receptor positive; and\n",
      "Breast tumor is HER2 receptor negative; and\n",
      "Adjuvant chemotherapy is not precluded due to any other factor (e.g., advanced age and/or significant co-morbidities); and\n",
      "Member and physician (prior to testing) have discussed the potential results of the test and agree to use the results to guide therapy;\n",
      "\n",
      "\n",
      "Epidermal growth factor receptor (EGFR) mutation testing for predicting response to EGFR-targeting tyrosine kinase inhibitors (erlotinib (Tarceva), gefitinib (Iressa), afatinib (Gilotrif), osimertinib (Tagrisso)) in non-small cell lung cancer;\n",
      "EZH2 (enhancer of zeste 2 polycomb repressive complex 2 subunit) for the workup of the following: \n",
      "\n",
      "myelodysplastic syndrome (MDS), and\n",
      "myeloproliferative neoplasms (MPN) to evaluate for higher-risk mutations associated with disease progression in members with primary myelofibrosis (PMF);\n",
      "\n",
      "Aetna considers EZH2 experimental and investigational for all other indications including diffuse large B-cell lymphomas;\n",
      "\n",
      "FIP1L1-PDGFRA fusion oncogene for systemic mastocytosis with peripheral blood eosinophilia;\n",
      "FIP1L1-PDGFRA gene rearrangements for myeloid/lymphoid neoplasms with peripheral blood eosinophilia and tyrosine kinase fusion genes;\n",
      "FLT3 gene mutation testing for acute lymphoblastic leukemia, acute myeloid leukemia (AML), myelodysplastic syndromes, myeloproliferative neoplasms, and myeloid/lymphoid neoplasms with eosinophilia and tyrosine kinase fusion genes;\n",
      "Human chorionic gonadotropin (HCG), serial measurement to diagnose germ cell tumors in members with adenocarcinoma, or carcinoma not otherwise specified, involving mediastinal nodes, or to monitor treatment in members with known trophoblastic tumors (invasive hydatidiform moles and choriocarcinomas) and germinal cell tumors (teratocarcinoma and embryonal cell carcinoma) of the ovaries or testes, or to monitor for relapse after remission is achieved;\n",
      "Human chorionic gonadotropin, beta (beta-HCG) for mediastinal mass; ovarian cancer; pelvic mass; testicular mass; testicular cancer; thymoma; or thymic carcinoma;\n",
      "Human epidermal growth factor receptor 2 (HER2) (ERBB2) evaluation in breast, gastric, colorectal, esophageal, esophageal gastric junction, salivary gland tumors, and non-small cell lung cancer - see CPB 0313 - Trastuzumab (Herceptin and biosimilars), Trastuzumab and Hyaluronidase-oysk (Herceptin Hylecta);\n",
      "Human papillomavirus (HPV) tumor testing (p16) for the workup of head and neck cancer (including oropharynx cancer) or occult primary cancers;\n",
      "IGH@ (Immunoglobulin heavy chain locus), gene rearrangement analysis to detect abnormal clonal population(s) in non-Hodgkinâs lymphomas, chronic lymphocytic leukemia, hairy cell leukemia, and post-transplant lymphoproliferative disorder;\n",
      "IGK@ (Immunoglobulin kappa light chain locus), gene rearrangement analysis, evaluation to detect abnormal clonal population(s) for non-Hodgkinâs lymphoma, systemic light chain amyloidosis;\n",
      "INHA (inhibin) expression for ovarian cancer or pelvic mass;\n",
      "Isocitrate dehydrogenase 1 and 2 (IDH1 and IDH2) gene mutation for AML, chondrosarcomas, myelodysplastic syndromes, myeloproliferative neoplasms, or gliomas and glioblastomas;\n",
      "K-ras (KRAS) mutation analysis, with BRAF reflex testing, to predict non-response to cetuximab (Erbitux) and panitumumab (Vectibix) in the treatment of anal adenocarcinoma, metastatic colorectal cancer and small bowel adenocarcinoma; K-ras (KRAS) mutation analysis to predict non-response to erlotinib (Tarceva) in the treatment of non-small cell lung cancer; experimental for all other indications;\n",
      "KRAS for metastatic colorectal cancer, myelodysplastic syndromes, non-small cell lung cancer, pancreatic adenocarcinoma, and uterine sarcoma;\n",
      "Lactate dehydrogenase (LDH) for acute lymphoblastic leukemia (ALL), bone cancer, kidney cancer, kidney mass, lung cancer, multiple myeloma, non-Hodgkin's lymphoma, pelvic mass, ovarian cancer, testicular cancer, or testicular mass;\n",
      "Liquid biopsy (up to 50 genes) (e.g., Resolution ctDx Lung, InVisionFirst-Lung) for persons with non-small cell lung cancer who are not medically fit for invasive sampling, or there is insufficient tissue for molecular analysis and follow-up tissue-based analysis will be done if an oncogenic driver is not identified; large liquid biopsy panels (greater than 50 genes) are considered experimental and investigational for non-small cell lung cancer; for Guardant360CDx non-small cell lung cancer and FoundationOne Liquid CDx for non-small cell lung cancer and prostate cancer (see CPB 0715 - Pharmacogenetic and Pharmacodynamic Testing). Note: for additional information on liquid biopsy, see Experimental and Investigational section;\n",
      "MammaprintFootnote2** to assess necessity of adjuvant chemotherapy in females or males with recently diagnosed breast tumors, where all of the following criteria are met:\n",
      "\n",
      "Breast cancer is nonmetastatic (node negativeFootnote1*) or with 1-3 involved ipsilateral axillary lymph nodes; and\n",
      "Breast tumor is estrogen receptor positive or progesterone receptor positive; and\n",
      "Breast tumor is HER2 receptor negative (Rationale: adjuvant chemotherapy with trastuzumab (Herceptin) is considered to be medically necessary regardless of Mammaprint score for HER2 receptor positive lesions); and\n",
      "Member is determined to be at \"high clinical risk\" of recurrence using Adjuvant! Online (see page 20 of MINDACT study supplement for definitions of high clinical risk); and\n",
      "Adjuvant chemotherapy is not precluded due to any other factor (e.g., advanced age and/or significant co-morbidities); and\n",
      "Member and physician (prior to testing) have discussed the potential results of the test and agree to use the results to guide therapy;\n",
      "\n",
      "\n",
      "Measurement of estrogen receptors (ESR1) for breast cancer, endometrial carcinoma, non-small cell lung cancer, occult primary, ovarian cancer, or uterine sarcoma;\n",
      "Measurement of progesterone receptors (PGR) for breast cancer, non-small cell lung cancer, occult primary, or uterine sarcoma;\n",
      "Mismatch repair (MSI/dMMR) (MLH1, MSH2, MSH6, PMS2) tumor testing (somatic mutations) for breast cancer, ovarian cancer, colorectal cancer, small bowel adenocarcinoma, esophageal cancer, esophagogastric junction cancer, gastric cancer, pancreatic cancer, cholangiocarcinoma, gallbladder cancer, pancreatic adenocarcinoma, cervical cancer, uterine cancer, prostate cancer, testicular cancer, penile cancer, myelodysplastic syndromes, Ewing sarcoma, and occult primary; for medical necessity of screening of germline mutations for HNPCC/Lynch Syndrome with MLH1, MSH2, MSH6, see CPB 0140 - Genetic Testing;\n",
      "MLH1 tumor promoter hypermethylation for endometrial cancer;\n",
      "MPL (myeloproliferative leukemia protein) for chronic myeloid leukemia (chronic phase, adult), myelodysplastic syndromes, or myeloproliferative neoplasms; \n",
      "Murine double minute 2 (MDM2) for uterine sarcoma and soft tissue sarcoma;\n",
      "Mycosis fungoides, diagnosis: polymerase chain reaction (PCR) for T-cell receptor gamma chain gene rearrangement as an adjunct to the histopathologic diagnosis of mycosis fungoides;\n",
      "MYD88 (myeloid differentiation primary response 88) to differentiate Waldenstrom's macroglobinemia (WM) versus marginal zone lymphoma (MZL) if plasmacytic differentiation present for gastric MALT lymphoma, nodal marginal zone lymphoma, nongastric MALT lymphoma, and splenic marginal zone lymphoma; and for multiple myeloma;\n",
      "Myeloperoxidase (MPO) immunostaining, CEBPA mutation, and KIT mutation for diagnosis of acute myeloid leukemia;\n",
      "MyMRD NGS Panel for comprehensive prognostic assessment in individuals with acute myeloid leukemia (AML) or myelodysplastic syndromes (MDS);\n",
      "Next generation sequencing of tumor DNA (e.g., ClonoSeq) to detect or quantify minimal residual disease in persons with multiple myeloma or acute lymphocytic leukemia;\n",
      "NPM1 in acute myeloid leukemia (AML), chronic myeloid leukemia (chronic phase, adult), myelodysplastic syndromes, or myeloproliferative neoplasms; experimental for other indications;\n",
      "NRAS for colorectal cancer, myelodysplastic syndrome, or blastic plasmacytoid dendritic cell neoplasm (BPDCN);\n",
      "NTRK for all solid tumors;\n",
      "Oncotype Dx Breast (also known as 21 gene RT-PCR test) to assess necessity of adjuvant chemotherapy in females or males with recently diagnosed breast tumors, where all of the following criteria are met:\n",
      "\n",
      "Breast cancer is nonmetastatic (node negativeFootnote1*) or with 1-3 involved ipsilateral axillary lymph nodes; and\n",
      "Breast tumor is estrogen receptor positive; and\n",
      "Breast tumor is HER2 receptor negative or breast tumor is HER2 receptor positive and less than 1 cm in diameter. (Rationale: adjuvant chemotherapy with trastuzumab (Herceptin) is considered to be medically necessary regardless of an Oncotype Dx Breast score for HER2 receptor positive lesions 1 cm or more in diameter); and\n",
      "Adjuvant chemotherapy is not precluded due to any other factor (e.g., advanced age and/or significant co-morbidities); and\n",
      "Member and physician (prior to testing) have discussed the potential results of the test and agree to use the results to guide therapy (i.e., member will forgo adjuvant chemotherapy if Oncotype Dx Breast score is low);\n",
      "\n",
      "\n",
      "\n",
      "Oncotype DX ProstateFootnote3*** for the following indications post biopsy:\n",
      "\n",
      "Men with NCCN very-low-risk, low-risk, and favorable intermediate-risk prostate cancer who have greater than 10 year life expectancy and who have not received treatment for prostate cancer and are candidates for active surveillance or definitive therapy; or\n",
      "Men with intermediate-risk prostate cancer when deciding whether to add androgen-deprivation therapy to radiation;\n",
      "\n",
      "\n",
      "\n",
      "PAM50 Risk of Recurrence (ROR) Score (also known as Prosigna Breast Cancer Prognostic Gene Signature Assay)Footnote2** to assess necessity of adjuvant chemotherapy in females or males with recently diagnosed breast tumors, where all of the following criteria are met:\n",
      "\n",
      "Breast cancer is nonmetastatic (node negative); and\n",
      "Breast tumor is estrogen receptor positive; and\n",
      "Breast tumor is HER2 receptor negative; and\n",
      "Adjuvant chemotherapy is not precluded due to any other factor (e.g., advanced age and/or significant co-morbidities); and\n",
      "Member and physician (prior to testing) have discussed the potential results of the test and agree to use the results to guide therapy;\n",
      "\n",
      "\n",
      "PDGFRA for gastrointestinal stromal tumors (GIST) and for pediatric acute lymphoblastic leukemia (see also entry above for FIP1L1-PDGFRA gene rearrangements and fusions);\n",
      "PDGFRB testing for myelodysplastic syndromes (MDS), dermatofibrosarcoma protuberans, acute lymphoblastic leukemia, and for myeloid/lymphoid neoplasms with peripheral blood eosinophilia and tyrosine kinase fusion genes;\n",
      "Phosphatidylinositol-4,5-bisphosphonate 3-kinase, catalytic subunit alpha polypeptide gene (PIK3CA) for breast cancer and uterine sarcoma;\n",
      "Placental alkaline phosphatase (PLAP), to diagnose germ cell seminoma and non-seminoma germ cell tumors in unknown primary cancers;\n",
      "PLCG2 (phospholipase C gamma 2) for chronic lymphocytic leukemia (CLL)/small lymphocytic lymphoma (SLL);\n",
      "PML/RARA for acute promyelocytic leukemia; experimental for all other indications;\n",
      "ProlarisFootnote3*** for the following indications post-biopsy:\n",
      "\n",
      "Men with NCCN very-low-risk, low-risk, and favorable intermediate-risk prostate cancer who have greater than 10 year life expectancy and who have not received treatment for prostate cancer and are candidates for active surveillance or definitive therapy; or\n",
      "Men with intermediate-risk prostate cancer when deciding whether to add androgen-deprivation therapy to radiation;\n",
      "\n",
      "\n",
      "\n",
      "ProMarkFootnote3*** for the following indications post-biopsy:\n",
      "\n",
      "Men with NCCN very-low-risk, low-risk men, and favorable intermediate risk prostate cancer who have greater than 10 year life expectancy and who have not received treatment for prostate cancer and are candidates for active surveillance or definitive therapy; or\n",
      "Men with intermediate-risk prostate cancer when deciding whether to add androgen-deprivation therapy to radiation;\n",
      "\n",
      "\n",
      "Prostate-specific antigen (PSA) for prostate cancer screening (see CPB 0521 - Prostate Cancer Screening), staging, monitoring response to therapy, and detecting disease recurrence;\n",
      "PTEN for uterine sarcoma and for persons meeting Cowden syndrome testing criteria in CPB 0140 - Genetic Testing; experimental for all other indications;\n",
      "Quest Diagnostics Thyroid Cancer Mutation Panel for assessing fine needle aspiration samples from thyroid nodules that are indeterminate; experimental for other indications. Repeat testing is considered experimental and investigational;\n",
      "ROS-1 to predict response to crizotinib (Xalkori) for the treatment of non-small cell lung cancer (NSCLC);\n",
      "RUNX1 for acute myeloid leukemia, myelodysplastic syndrome, and systemic mastocytosis;\n",
      "SF3B1 (splicing factor 3b subunit 1) for chronic myeloid leukemia (chronic phase, adult), myelodysplastic syndromes, myeloproliferative neoplasms, or uveal melanoma;\n",
      "SRSF2 (serine and arginine rich splicing factor 2) for chronic myeloid leukemia (chronic phase, adult), myelodysplastic syndromes, myeloproliferative neoplasms, or systemic mastocytosis; \n",
      "Steroid hormone receptor status in both pre-menopausal and post-menopausal members to identify individuals most likely to benefit from endocrine forms of adjuvant therapy and therapy for recurrent or metastatic breast cancer;\n",
      "Targeted hematologic genomic sequencing panel (5-50 genes) for acute lymphocytic leukemia, acute myeloid leukemia, chronic myelogenous leukemia, myelodysplastic syndromes (MDS) and myeloproliferative neoplasms (MPN) (e.g., MedFusion myeloid malignancy analysis panel). Repeating a hematologic malignancy genomic sequencing panel within 60 days of prior panel testing for the same indication is considered not medically necessary;\n",
      "Targeted solid organ genomic sequencing panel (5-50 genes) for colorectal cancer, cutaneous melanoma, pancreatic cancer, prostate cancer and non-small cell lung cancer (including Oncomine Dx Target Test (Thermo Fisher Scientific, Carlsbad, CA)). Repeating a solid organ malignancy genomic sequencing panel within 60 days of prior panel testing for the same indication is considered not medically necessary;\n",
      "T-cell receptor gene rearrangements (TRA@, TRB@, TRD@, TRG@) for T-cell prolymphocytic leukemia, T-cell large granular lymphocytic leukemia, nasal type extranodal NK/T-cell lymphoma, hepatosplenic gamma-delta T-cell lymphoma, peripheral T-cell lymphoma, primary cutaneous CD30+ T-cell lymphoproliferative disorders, myelodysplastic syndromes, Castleman's disease, mycosis fungoides/Sezary syndrome and myeloid/lymphoid neoplasms with eosinophilia and tyrosine kinase fusion genes;\n",
      "TERT (telomerase reverse transcriptase) medically necessary for the workup of:\n",
      "\n",
      "Gliomas (i.e., infiltrative supratentorial astrocytoma/oligodendroglioma, anaplastic gliomas/glioblastoma), and\n",
      "Myelodysplastic syndrome (MDS).\n",
      "\n",
      "Aetna considers TERT experimental and investigational for all other indications including thyroid carcinoma.\n",
      "\n",
      "ThyGeNEXT Thyroid Oncogene Panel (formerly e.g., ThyGenX, miRInform thyroid test) and ThyraMIR microRNA Classifier for assessing fine needle aspiration samples from thyroid nodules that are indeterminate; experimental for other indications; repeat testing is considered experimental and investigational;\n",
      "Thymidine kinase for chronic lymphocytic leukemia (CLL)/small lymphocytic lymphoma (SLL);\n",
      "Thyroglobulin antibodies for thyroid cancer;\n",
      "Thyroglobulin (TG) expression for thyroid cancer, occult primary, and adenocarcinoma or anaplastic/undifferentiated tumors of the head and neck\n",
      "Thyroid transcription factor-1 (TTF-1) for lung cancer or neuroendocrine tumors;\n",
      "Thyroseq for assessing fine needle aspiration samples from thyroid nodules that are indeterminate; experimental for other indications. Repeat testing is considered experimental and investigational;\n",
      "TP53 for acute myeloid leukemia; adult medulloblastoma; chronic lymphocytic leukemia/small lymphocytic lymphoma; chronic myeloid leukemia (chronic phase, adult); endometrial carcinoma; malignant peritoneal or pleural mesothelioma; mantle cell lymphoma; myelodysplastic syndromes; myeloproliferative neoplasms; occult primary; pediatric acute lymphoblastic leukemia; peripheral T-cell lymphomas; splenic marginal zone lymphoma; uterine sarcoma; or well-differentiated, grade 3 neuroendocrine tumors;\n",
      "U2AF1 (U2 small nuclear RNA auxiliary factor 1) for blastic plasmacytoid dendritic cell neoplasm (BPDCN), chronic myeloid leukemia (chronic phase, adult), myelodysplastic syndromes, or myeloproliferative neoplasms; \n",
      "Urokinase plasminogen activator (uPA) and plasminogen activator inhibitor 1 (PAI-1)Footnote2** to assess necessity of adjuvant chemotherapy in females or males with recently diagnosed breast tumors, where all of the following criteria are met:\n",
      "\n",
      "Breast cancer is nonmetastatic (node negative); and\n",
      "Breast tumor is estrogen receptor positive; and\n",
      "Breast tumor is HER2 receptor negative; and\n",
      "Adjuvant chemotherapy is not precluded due to any other factor (e.g., advanced age and/or significant co-morbidities); and\n",
      "Member and physician (prior to testing) have discussed the potential results of the test and agree to use the results to guide therapy;\n",
      "\n",
      "In addition, urokinase plasminogen activator (uPA) and plasminogen activator inhibitor 1 (PAI-1) is considered medically necessary for the determination of prognosis in persons with newly diagnosed, node negative breast cancer;\n",
      "\n",
      "Vascular endothelial growth factor (VEGF) expression for Castleman's disease;\n",
      "Veristrat proteomic testing for patients with advanced NSCLC, whose tumors were without EGFR and anaplastic lymphoma kinase (ALK) mutations, who had progressed after at least one chemotherapy regimen), and for whom erlotinib was considered an appropriate treatment;\n",
      "WT-1 gene expression for desmoplastic round cell tumors, ovarian clear cell carcinomas, non-small cell lung cancer and occult primary;\n",
      "ZAP-70, for assessing prognosis and need for aggressive therapy in persons with chronic lymphocytic leukemia (CLL)/small lymphocytic lymphoma (SLL);\n",
      "ZRSR2 (zinc finger CCCH-type, RNA binding motif and serine/arginine rich 2) for chronic myeloid leukemia (chronic phase, adult) or myelodysplastic syndromes. \n",
      "\n",
      "\n",
      "\n",
      "Aetna considers somatic genomic testing for Janus Kinase 2 (JAK2) mutations in persons with chronic myeloproliferative disorders (CMPDs) medically necessary for the following indications:\n",
      "\n",
      "Qualitative assessment of JAK2-V617F sequence variant using methods with detection thresholds of up to 5% for initial diagnostic assessment of adult patients presenting with symptoms of CMPD;\n",
      "Diagnostic assessment of polycythemia vera in adults; and\n",
      "Differential diagnosis of essential thrombocytosis and primary myelofibrosis from reactive conditions in adults. \n",
      "\n",
      "Aetna considers somatic genomic testing for Janus Kinase 2 (JAK2) mutations in persons with chronic myeloproliferative disorders (CMPDs) experimental and investigational for any other indication including:\n",
      "\n",
      "Diagnostic assessment of myeloproliferative disorders in children;\n",
      "Quantitative assessment of JAK2-V617F allele burden subsequent to qualitative detection of JAK2-V617F.\n",
      "\n",
      "\n",
      "\n",
      "Aetna considers the use of fluorescence immunocytology (e.g., ImmunoCyt/uCyt) medically necessary as an adjunct to cystoscopy or cytology in the monitoring of persons with bladder cancer.\n",
      "Aetna considers the ImmunoCyte/uCyt immunohistochemistry test experimental and investigational in the evaluation of hematuria, diagnosing bladder cancer, or for screening for bladder cancer in asymptomatic persons.\n",
      "\n",
      "\n",
      "Aetna considers urinary biomarkers (e.g., bladder tumor antigen (BTA) (e.g., BTA Stat and BTA TRAK), nuclear matrix protein (NMP22) test, the fibrin/fibrinogen degradation products (Aura-Tek FDfP) test, or fluorescence in situ hybridization (FISH) (e.g., Pathnostics Bladder FISH test, UroVysion Bladder Cancer test medically necessary in any of the following conditions:\n",
      "\n",
      "Follow-up of treatment for bladder cancer; or \n",
      "Monitoring for eradication of bladder cancer; or \n",
      "Recurrences after eradication.\n",
      "\n",
      "Aetna considers the BTA Stat test, the NMP22 test, the Aura-Tek FDP test, or the UroVysion fluorescent in situ hybridization (FISH) test experimental and investigational for screening of bladder cancer, evaluation of hematuria, and diagnosing bladder cancer in symptomatic individuals, and all other indications.  \n",
      "\n",
      "\n",
      "Footnote1* Either standard node dissection negative by hematoxylin and eosin (H&E) staining or sentinel node negative by H&E staining (if sentinel node is negative by H&E, but immunoassay is positive, then still considered node negative for this purpose). In addition, women with isolated tumor cells in lymph nodes (micrometastases) are considered node negative.\n",
      "More than one Oncotype Dx test may be medically necessary for persons with breast cancer who have two or more histologically distinct tumors that meet medical necessity criteria. Repeat Oncotype Dx testing or testing of multiple tumor sites in the same person has no proven value for other indications. Oncotype Dx is considered experimental and investigational for ductal carcinoma in situ (OncotypeDx DCIS), colon cancer (OncotypeDx Colon), and all other indications other than breast cancer and prostate cancer. \n",
      "Footnote2** Aetna considers use of more than one type of test to determine necessity of adjuvant therapy in breast cancer (Oncotype Dx Breast, Breast Cancer Index, EndoPredict, PAM50, Mammaprint, or uPA and PAI-1) experimental and investigational.\n",
      "Footnote3*** Aetna considers repeat testing or use of more than one type of test to assess risk of prostate cancer progression (Oncotype Dx Prostate, Decipher, Prolaris, or ProMark) experimental and investigational.\n",
      "\n",
      "\n",
      "Experimental and Investigational\n",
      "\n",
      "\n",
      "Aetna considers each of the following experimental and investigational. The peer-reviewed medical literature does not support these tests as having sufficient sensitivity or specificity necessary to define their clinical role:\n",
      "\n",
      "3D Predict Ovarian Doublet Panel\n",
      "3D Predict Ovarian PARP Panel\n",
      "4Kscore\n",
      "Afirma Xpression Atlas\n",
      "AFP for the diagnosis of trophoblastic tumors and oncologic indications other than those listed in Section I\n",
      "AMBLor Melanoma Prognostic Test\n",
      "ArteraAI Prostate Test\n",
      "Assaying for loss of heterozygosity (LOH) on the long arm of chromosome 18 (18q) or deleted in colon cancer (DCC) protein (18q-LOH/DCC) for colorectal cancer\n",
      "Augusta Hematology Optical Genome Mapping\n",
      "BBDRisk Dx\n",
      "Biodesix BDX-XL2, Nodify CDT, Nodify Lung, or Nodify XL2 test for distinguishing benign from malignant lung nodules\n",
      "Biomarker Translation (BT) test for breast cancer and other indications\n",
      "BioSpeciFx, including Comprehensive Tumor Profiling for any indication\n",
      "BostonGene Tumor Portrait Test\n",
      "BRAF and EGFR for esophageal carcinoma\n",
      "Breast Cancer Gene Expression Ratio (HOXB13:IL17BR)\n",
      "BreastSentry\n",
      "CA 125 for all other indications including use as a screening test for colorectal cancer or ovarian cancer (other than as indicated in Section I) or for differential diagnosis of members with symptoms of colonic disease\n",
      "CA 19-9 for all other indications not listed in Section I. including breast, colorectal, esophageal, gastro-esophageal, liver, or uterine cancer; ovarian cyst, NUT midline carcinoma of the nasal cavity, prediction of prognosis or treatment effect in persons with bladder (urothelial) cancer, screening persons with primary sclerosing cholangitis without signs or symptoms of cholangiocarcinoma; or screening persons with primary sclerosing cholangitis for development of cholangiocarcinoma\n",
      "Carcinoembryonic antigen cell adhesion molecule 6 (CEACAM6) (e.g., Benign Diagnostics Risk Test) for breast atypical hyperplasia and for predicting the risk of breast cancer\n",
      "Carcinoembryonic antigen cellular adhesion molecule-7 (CEACAM-7) expression as a predictive marker for rectal cancer recurrence\n",
      "Caris Molecular Intelligence/Caris Target Now Molecular Profiling Test\n",
      "Castle Biosciences myPath Melanoma (formerly Myriad myPath Melanoma)\n",
      "CDH1 for ovarian cancer\n",
      "CDX2 as a prognostic biomarker for colon cancer\n",
      "CEA used for all other indications not noted in Section I including any of the following:\n",
      "\n",
      "As a screening test for colorectal cancer; or \n",
      "As a sole determinant to treat a colorectal cancer member with adjuvant therapy or systemic therapy for presumed metastatic disease; or\n",
      "For diagnosis of esophageal carcinoma; or\n",
      "For screening, diagnosis, staging or routine surveillance of gastric cancer\n",
      "\n",
      "\n",
      "Circulating cell-free nucleic acids in colorectal cancer\n",
      "Circulating tumor cell (CTC) assays (e.g., CELLSEARCH Circulating Multiple Myeloma Cell (CMMC) and CELLSEARCH HER2 Circulating Tumor Cell (CTC-HER2) tests) for all indications, including, but not limited to metastatic breast, colorectal, melanoma, and prostate cancers\n",
      "CK5, CK14, p63, and Racemase P504S testing for prostate cancer\n",
      "c-Met expression for predicting prognosis in persons with advanced NSCLC and colorectal cancer, and other indications\n",
      "Cyfra21-1 (a cytokeratin 19 fragment), p53, squamous cell carcinoma antigen (SCC-Ag) and vascular endothelial growth factor C (VEGF-C) for diagnosis of esophageal carcinoma\n",
      "Cofilin (CFL1) as a prognostic and drug resistance marker in non-small cell lung cancer\n",
      "ColonSentry test for screening of colorectal cancer\n",
      "ColoPrint, CIMP, LINE-1 hypomethylation, and Immune cells for colon cancer\n",
      "Colorectal Cancer DSA (Almac Diagnostics, Craigavon, UK)\n",
      "ColoScape Test\n",
      "ConfirmMDx for prostate cancer\n",
      "Cxbladder tests (e.g., Cxbladder Triage) for bladder cancer\n",
      "Cyclin D1 and FADD (Fas-associated protein with death domain) for head and neck squamous cell carcinoma\n",
      "DAWN IO Melanoma;\n",
      "DCIS Recurrence Score\n",
      "DCISionRT\n",
      "Decipher Bladder\n",
      "DecisionDx DiffDx-Melanoma (Castle Biosciences, Phoenix, AZ)\n",
      "DecisionDx-Melanoma (Castle Biosciences, Phoenix, AZ)\n",
      "DecisionDx-SCC (Castle Biosciences, Phoenix, AZ)\n",
      "Des-gamma-carboxy prothrombin (DCP) (also known as \"prothrombin produced by vitamin K absence or antagonism II\" [PIVKA II]) for diagnosing and monitoring hepatocellular carcinoma (HCC) and other indications\n",
      "DetermaRx\n",
      "EarlyCDT-Lung test\n",
      "EGFR gene expression analysis for transitional (urothelial) cell cancer\n",
      "EGFRVIII for glioblastoma multiforme\n",
      "EML4-ALK as a diagnostic tool for stage IV non-small-cell lung cancer\n",
      "Envisia Genomic Classifier\n",
      "EpiSwitch CiRT (Checkpoint-inhibitor Response Test)\n",
      "Excision repair cross-complementation group 1 protein (ERCC1) for persons with NSCLC, colon or with gastric cancer who are being considered for treatment with platinum-based chemotherapy, and other indications\n",
      "ExoDx Prostate/ExosomeDx Prostate (IntelliScore)\n",
      "Fibrin/fibrinogen degradation products (FDP) test (e.g., DR-70 or Onko-Sure) for colorectal cancer\n",
      "FoundationOne, FoundationOne CDx and FoundationOne Heme (except where FoundationOne CDx is used as a companion diagnostic test for somatic/tumor BRCA testing, see CPB 0227 - BRCA Testing, Prophylactic Mastectomy, and Prophylactic Oophorectomy and CPB 0715 - Pharmacogenetic and Pharmacodynamic Testing)\n",
      "Galectin-3 for breast cancer, myelodysplastic syndrome, osteosarcoma, ovarian cancer, pancreatic cancer, and prostate cancer\n",
      "Gene hypermethylation for prostate cancer\n",
      "GeneKey (GeneKey Corp., Boston, MA)\n",
      "GeneSearch Breast Lymph Node (BLN) assay\n",
      "Glutathione-S-transferase P1 (GSTP1) for screening, detection and management of prostate cancer\n",
      "Grail Galleri Test\n",
      "Guanylyl cyclase c (GCC or GUCY2C) (e.g., Previstage GCC Colorectal Cancer State Test) for colorectal cancer\n",
      "Guardant360 TissueNext\n",
      "HelioLiver Test\n",
      "HeproDx\n",
      "HER2 testing of appendiceal cancer\n",
      "HERmark testing for breast cancer and other indications\n",
      "HMGB1 and RAGE in cutaneous malignancy (e.g., basal cell carcinoma, melanoma, and squamous cell carcinoma)\n",
      "Human epididymis protein 4 (HE4) (e.g., Elecsys HE4 assay) for endometrial cancer, ovarian cancer, or evaluation of pelvic mass, including to assist in the determination of referral for surgery to a gynecologic oncologist or general surgery, and for other indications\n",
      "IHC4 (e.g., NexCourse IHC4 by AQUA Technology) for breast cancer\n",
      "IMMray PanCan-d for detecting pancreatic ductal adenocarcinoma\n",
      "Immunoassay using magnetic nanosensor for diagnosis of lung cancer\n",
      "Immunoscore for estimating risk of recurrence or determining adjuvant therapy in persons with colon cancer\n",
      "Insight DX Breast Cancer Profile\n",
      "Insight TNBCtype\n",
      "IsoPSA\n",
      "Ki67 for breast cancer\n",
      "Ki-67 in upper tract urinary carcinoma\n",
      "Lectin-reactive alpha-fetoprotein (AFP-L3) for liver cancer\n",
      "Liquid biopsy (e.g., CancerIntercept, Colvera, GeneStrat, FoundationACT, FoundationOne Liquid, Guardant360, Neolab Prostate) for any indication (other than small panels (less than 50 genes) for non-small cell lung cancer), including, but not limited to, breast cancer, colorectal cancer, melanoma, ovarian cancer or prostate cancer (For EGFR liquid biopsy for non-small cell lung cancer (e.g., cobas EGFR Mutation Test v2), PIK3CA testing (therascreen PIK3CA RGQ PCR Kit) for breast cancer, and ESR1 gene mutations (e.g., Guardant360 CDx assay) for breast cancer, see CPB 0715 - Pharmacogenetic and Pharmacodynamic Testing)\n",
      "Long non-coding RNA in gallbladder cancer\n",
      "LungLB and LungLife AI\n",
      "Lymph2CX and Lymph3Cx Lymphoma Molecular Classification Assay to distinguish between primary mediastinal B-cell lymphoma (PMBCL) and diffuse large B-cell lymphoma (DLBCL)\n",
      "Mammostrat\n",
      "Mass spectrometry-based proteomic profiling for indeterminate pulmonary nodules\n",
      "MatePair targeted rearrangements (whole genome next-generation sequencing) for hematolymphoid neoplasia and solid organ neoplasia\n",
      "Mayo Clinic Laboratories Urinary Steroid Profile for the management of adrenal malignancies\n",
      "Measurement of circulating tumor cells (e.g., CellMax Life and FirstSightCRC) for screening of colorectal cancer\n",
      "Merkel SmT Oncoprotein Antibody Titer\n",
      "Merkel Virus VP1 Capsid Antibody\n",
      "MI Cancer Seek\n",
      "Microarray-based gene expression profile testing using the MyPRS test for multiple myeloma\n",
      "Micro-RNAs (miRNAs) miRview mets and miRview mets2 (Rosetta Genomics Laboratories, Philadelphia, PA; Rosetta Genomics Ltd., Rehovot, Israel)\n",
      "Mi-Prostate Score (MiPS), an assay of TMPRSS2:ERG (T2:ERG) gene fusion, post-DRE urine expression of PCA3, and serum PSA (KLK3)\n",
      "miR-31now\n",
      "miR Sentinel Prostate Cancer Test\n",
      "Molecular Intelligence Services, including MI Profile and MI Profile X (formerly Target Now Molecualr Profiling Test, including Target Now Select and Target Now Comprehensive)\n",
      "Molecular subtyping profile (e.g., BluePrint) for breast cancer\n",
      "mRNA gene expression profiling for cutaneous melanoma\n",
      "MSK-IMPACT\n",
      "MUC1 in gastric cancer\n",
      "Mucin 4 expression as a predictor of survival in colorectal cancer\n",
      "Mucin 5AC (MUC5AC) as serum marker for biliary tract cancer\n",
      "My Prognostic Risk Signature (MyPRS) (Signal Genetics LLC, New York, NY)\n",
      "MyAML Next Generation Sequencing Panel\n",
      "NantHealth GPS Cancer Panels\n",
      "Natera Signatera Molecular Monitoring (MRD) for breast cancer\n",
      "NavDx for surveillance of cancer recurrence in HPV-associated oropharyngeal cancer\n",
      "NeoLAB Prostate Liquid Biopsy\n",
      "NETest\n",
      "NF1, RET, and SDHB for ovarian cancer\n",
      "NRAS mutation for selecting persons with metastatic colorectal cancer who may benefit from anti-VEGF antibody bevacizumab; to predict disease prognosis and select persons with melanoma who may benefit from tyrosine kinase inhibitor therapies, and other indications\n",
      "OmniSeq Advance DNA and RNA sequencing (OmniSeq and LabCorp)\n",
      "OncInsights (Intervention Insights, Grand Rapids, MI)\n",
      "OncobiotaLUNG\n",
      "Oncomap ExTra (formerly known as Oncotype MAP)\n",
      "OncoOmicDx Targeted Proteomic Assay\n",
      "OncoSignal test for analysis of solid tumors\n",
      "OncoTarget/OncoTreat\n",
      "Oncotype MAP PanCancer Tissue Test\n",
      "OncoVantage\n",
      "Oncuria Detect, Oncuria Monitor and Oncuria Predict for bladder cancer and all other indications\n",
      "OVA1/Overa test\n",
      "OvaCheck test\n",
      "OvaSure\n",
      "OvaWatch\n",
      "PancreaSeq Genomic Classifier\n",
      "PanGIA Prostate for determining if an individual should undergo a prostate biopsy\n",
      "Pathwork Tissue of Origin Test/ResponseDx Tissue of Origin Test\n",
      "Percepta Bronchial Genomic Classifier\n",
      "PGDx elio tissue complete (Personal Genome Diagnostics, Inc.) for tumor mutation profiling\n",
      "Pharmaco-oncologic AlgorithmicTreatment Ranking Service\n",
      "Phosphatidylinositol-4,5-bisphosphonate 3-kinase, catalytic subunit alpha polypeptide gene (PIK3CA) for predicting disease prognosis and selecting individuals with metastatic colorectal cancer who are being considered for treatment with EGFR antagonists cetuximab and panitumumab, and indications other than breast cancer and uterine sarcoma\n",
      "PLCG2 (phospholipase C gamma 2) for all indications other than chronic lymphocytic leukemia (CLL)\n",
      "Praxis Somatic Combined Whole Genome Sequencing and Optical Genome Mapping\n",
      "Praxis Somatic Optical Genome Mapping\n",
      "Praxis Somatic Transcriptome\n",
      "Praxis Somatic Whole Genome Sequencing\n",
      "PreciseDx Breast Cancer Test\n",
      "PreOvar test for the KRAS-variant to determine ovarian cancer risk\n",
      "ProOnc TumorSourceDx test (Prometheus Laboratories, San Diego, CA) to identify tissue or origin for metastatic tumor\n",
      "Prostate core mitotic test\n",
      "Prostate Px and Post-Op Px for predicting recurence of prostate cancer\n",
      "Prostate Cancer Risk Panel (FISH analysis by Mayo Clinic)\n",
      "Proveri prostate cancer assay (PPCA)\n",
      "PSA for screening women with breast cancer or for differentiating benign from malignant breast masses\n",
      "PTEN gene expression for non-small cell lung cancer\n",
      "RadTox cfDNA test\n",
      "Ras oncogenes (except KRAS, NRAS and BRAF)\n",
      "ResponseDx Colon\n",
      "Ribonucleotide reductase subunit M1 (RRM1) for persons with NSCLC who are being considered for treatment with gemcitabine-based chemotherapy, and other indications\n",
      "RNA gene expression profiling for hematolymphoid disorder or neoplasm\n",
      "RNA gene expression for solid organ neoplasm\n",
      "ROMA (Risk of Ovarian Malignancy Algorithm) for ovarian cancer\n",
      "Rotterdam Signature 76-gene panel\n",
      "Salivary metatranscriptome analysis for oral cancers (i.e., mRNA CancerDetect)\n",
      "SelectMDx for prostate cancer\n",
      "Sentinel Prostate Test for prostate cancer screening and determining the risk level of the disease\n",
      "Serum amyloid A as a biomarker for endometrial endometrioid carcinoma to monitor disease recurrence and target response to therapy\n",
      "Signatera minimal residual disease test for alveolar soft tissue sarcoma, cutaneous melanoma, gastric adenocarcinoma, prostate cancer, renal cell carcinoma, and uterine cancer\n",
      "Signatera for individuals with stage II/III colorectal cancer who are considering adjuvant chemotherapy and/or who are being monitored for relapse post-treatment\n",
      "Solid Tumor Expanded Panel (Quest)\n",
      "Strata Select\n",
      "TargetPrint gene expression test for evaluation of estrogen receptor, progesterone receptor, and HER2receptor status in breast cancer\n",
      "Tempus Tumor Origin (TO) testing\n",
      "The 41-gene signature assay\n",
      "Theros CancerType ID (bioTheranostics Inc., San Diego, CA)\n",
      "Thymidylate synthase\n",
      "Thyroid GuidePx\n",
      "TMPRSS fusion genes for prostate cancer\n",
      "Topographic genotyping (Pancragen (formerly PathFinderTG))\n",
      "Total (whole) gene sequencing for cancer\n",
      "TP53 mutation analysis for ovarian cancer\n",
      "UroCor cytology panels (DD23 and P53) for bladder cancer\n",
      "Vascular Endothelial Growth Factor (VEGF) except for Castleman's disease\n",
      "Vascular endothelial growth factor receptor 2 (VEGFR2) expression for identifying persons with colorectal cancer that is likely to respond to VEGF inhibition, and other indications\n",
      "Whole exome sequencing (somatic mutations) (e.g., EXaCT-1 Whole Exome Testing) for cancer.\n",
      "\n",
      "\n",
      "\n",
      "Any of the following circulating tumor markers are also is considered experimental and investigational for screening asymptomatic subjects for cancer, diagnosis, staging, routine surveillance of cancer and monitoring the response to treatment:\n",
      "\n",
      "\n",
      "\n",
      "a2-PAG\n",
      "CA-SCC\n",
      "MAM-6\n",
      "TAG12\n",
      "\n",
      "\n",
      "AMACR\n",
      "Cathepsin-D, Cathepsin-L\n",
      "Motility-related protein (MRP)\n",
      "TAG72\n",
      "\n",
      "\n",
      " \n",
      "Cyclin E (fragments or whole length)\n",
      "Multidrug resistance glycoprotein (Mdr1)\n",
      "TAG72.3\n",
      "\n",
      "\n",
      "BCM\n",
      "DU-PAN-2\n",
      " \n",
      "TAG72.5\n",
      "\n",
      "\n",
      "CA195\n",
      "Early prostate cancer antigen (EPCA)\n",
      "NSE\n",
      "TATI\n",
      "\n",
      "\n",
      "CA242\n",
      "Guanylyl cyclase C (Previstage GCC molecular test)\n",
      " \n",
      "Thrombospondin-1 (THBS-1)\n",
      "\n",
      "\n",
      "CA50\n",
      "Hepsin\n",
      "PCA3 (DD3) / UpM3\n",
      "Thymosin B15\n",
      "\n",
      "\n",
      "CA549\n",
      "Human kallikrein 2 (HK2)\n",
      "PNA/ELLA\n",
      "TNF-a\n",
      "\n",
      "\n",
      "CA72-4\n",
      "LASA\n",
      "Prostate stem cell antigen (PSCA)\n",
      "Topoisomerase II Alpha (TOP2A)\n",
      "\n",
      "\n",
      "CAM17-1\n",
      "LPA\n",
      "SCC\n",
      "TPA\n",
      "\n",
      "\n",
      "CAM26\n",
      "M 26\n",
      "SLEX\n",
      "Thymosin B15\n",
      "\n",
      "\n",
      "CAM29\n",
      "M 29\n",
      "SPAN-1\n",
      "Nuclear Matrix Protein 66 (NMP66)\n",
      "\n",
      "\n",
      "CAR-3\n",
      "MSA\n",
      "SLX\n",
      "Anti-malignin antibody screen (AMAS) test\n",
      "\n",
      "\n",
      "CYFRA21-1\n",
      "MCA\n",
      "ST-439\n",
      "  \n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Related Policies\n",
      "\n",
      "\n",
      "CPB 0140 - Genetic Testing\n",
      "CPB 0227 - BRCA Testing, Prophylactic Mastectomy, and Prophylactic Oophorectomy\n",
      "CPB 0245 - Tumor Chemosensitivity Assays\n",
      "CPB 0313 - Trastuzumab (Herceptin and biosimilars), Trastuzumab and Hyaluronidase-oysk (Herceptin Hylecta)\n",
      "CPB 0314 - Rituximab\n",
      "CPB 0319 - RET Proto-Oncogene Testing\n",
      "CPB 0521 - Prostate Cancer Screening\n",
      "CPB 0715 - Pharmacogenetic and Pharmacodynamic Testing\n",
      "CPB 0758 - Tumor Chemoresistance Assays\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Table: CPT Codes / HCPCS Codes / ICD-10 Codes\n",
      "\n",
      "\n",
      "Code\n",
      "Code Description\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Prostate-specific antigen (PSA):\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "84152\n",
      "Prostate specific antigen (PSA); complexed (direct measurement)\n",
      "\n",
      "84153\n",
      "    total \n",
      "\n",
      "84154\n",
      "    free \n",
      "\n",
      "CPT codes not covered for indications listed in the CPB:\n",
      "\n",
      "81313\n",
      "PCA3/KLK3 (prostate cancer antigen 3 [non-protein coding]/kallikrein-related peptidase 3 [prostate specific antigen]) ratio (eg, prostate cancer) \n",
      "\n",
      "HCPCS codes covered if selection criteria are met:\n",
      "\n",
      "G0103\n",
      "Prostate cancer screening; prostate specific antigen test (PSA)\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C61\n",
      "Malignant neoplasm of prostate\n",
      "\n",
      "D07.5\n",
      "Carcinoma in situ of prostate\n",
      "\n",
      "D40.0\n",
      "Neoplasm of uncertain behavior of prostate\n",
      "\n",
      "R97.20 - R97.21\n",
      "Elevated prostate specific antigen [PSA]\n",
      "\n",
      "Z12.5\n",
      "Encounter for screening for malignant neoplasm of prostate\n",
      "\n",
      "Z85.46\n",
      "Personal history of malignant neoplasm of prostate\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C50.011 - C50.929\n",
      "Malignant neoplasm of breast\n",
      "\n",
      "D05.00 - D05.92\n",
      "Carcinoma in situ of breast\n",
      "\n",
      "D24.1 - D24.9\n",
      "Benign neoplasm of breast\n",
      "\n",
      "D48.60 - D48.62\n",
      "Neoplasm of uncertain behavior of breast\n",
      "\n",
      "D49.3\n",
      "Neoplasm of unspecified behavior of breast\n",
      "\n",
      "Z12.39\n",
      "Encounter for other screening for malignant neoplasm of breast\n",
      "\n",
      "Carcinoembryonic antigen (CEA):\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "82378\n",
      "Carcinoembryonic antigen (CEA) \n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C18.0 - C20\n",
      "Malignant neoplasm of colon, rectosigmoid junction and rectum\n",
      "\n",
      "C22.1\n",
      "Intrahepatic bile duct carcinoma [cholangiocarcinoma]\n",
      "\n",
      "C23 - C24.9 \n",
      "Malignant neoplasm of gallbladder and other and unspecified parts of biliary tract\n",
      "\n",
      "C25.0 - C25.9\n",
      "Malignant neoplasm of pancreas\n",
      "\n",
      "C34.00 - C34.92\n",
      "Malignant neoplasm of bronchus and lung\n",
      "\n",
      "C50.011 - C50.929\n",
      "Malignant neoplasm of breast\n",
      "\n",
      "C56.1 - C56.9 \n",
      "Malignant neoplasm of ovary\n",
      "\n",
      "C73\n",
      "Malignant neoplasm of thyroid gland [medullary thyroid cancer]\n",
      "\n",
      "C80.0 - C80.1\n",
      "Disseminated and other malignant neoplasm, unspecified\n",
      "\n",
      "D01.0\n",
      "Carcinoma in situ of colon\n",
      "\n",
      "D01.5\n",
      "Carcinoma in situ of liver, gallbladder and bile ducts\n",
      "\n",
      "D02.20 - D02.22\n",
      "Carcinoma in situ of bronchus and lung\n",
      "\n",
      "D05.00 - D05.92 \n",
      "Carcinoma in situ of breast\n",
      "\n",
      "D07.39\n",
      "Carcinoma in situ of other female genital organs [ovary]\n",
      "\n",
      "D09.3\n",
      "Carcinoma in situ of thyroid and other endocrine glands\n",
      "\n",
      "D13.4\n",
      "Benign neoplasm of liver [intrahepatic bile ducts]\n",
      "\n",
      "D13.6\n",
      "Benign neoplasm of pancreas\n",
      "\n",
      "D13.7\n",
      "Benign neoplasm of endocrine pancreas [Benign neoplasm of islets of Langerhans]\n",
      "\n",
      "D24.1 0 D24.9\n",
      "Benign neoplasm of breast\n",
      "\n",
      "D27.0 - D27.9 \n",
      "Benign neoplasm of ovary\n",
      "\n",
      "D34\n",
      "Benign neoplasm of thyroid gland\n",
      "\n",
      "K86.2 - K86.3\n",
      "Cyst and pseudocyst of pancreas \n",
      "\n",
      "R17\n",
      "Carcinoma in situ of other female genital organs [ovary]\n",
      "\n",
      "R93.2\n",
      "Abnormal findings on diagnostic imaging of liver and biliary tract\n",
      "\n",
      "R94.5\n",
      "Abnormal results of liver function studies\n",
      "\n",
      "Z85.030 - Z85.048\n",
      "Personal history of malignant neoplasm of large intestine, rectum, rectosigmoid junction, and anus\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C15.3 - C15.9\n",
      "Malignant neoplasm of esophagus\n",
      "\n",
      "D48.60 - D48.62\n",
      "Neoplasm of uncertain behavior of breast\n",
      "\n",
      "D49.3\n",
      "Neoplasm of unspecified behavior of breast\n",
      "\n",
      "Z12.2\n",
      "Encounter for screening for malignant neoplasm of respiratory organs\n",
      "\n",
      "Z12.11 - Z12.12\n",
      "Encounter for screening for malignant neoplasm of colon and rectum\n",
      "\n",
      "Z12.39\n",
      "Encounter for other screening for malignant neoplasm of breast\n",
      "\n",
      "CDH1 and TP53:\n",
      "\n",
      "CPT codes not covered for indications listed in the CPB:\n",
      "\n",
      "CDH1 and TP53 - no specific code:\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C56.1 - C56.9\n",
      "Malignant neoplasm of ovary \n",
      "\n",
      "Adenomatous polyposis coli (APC):\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "81201 - 81203\n",
      "APC (adenomatous polyposis coli) (eg, familial adenomatosis polyposis [FAP], attenuated FAP) gene analysis\n",
      "\n",
      "ICD-10 covered if selection criteria are met:\n",
      "\n",
      "D12.0 - D12.9\n",
      "Benign neoplasm of colon\n",
      "\n",
      "D48.1\n",
      "Neoplasm of uncertain behavior of connective and other soft tissue [desmoid fibromatosis]\n",
      "\n",
      "Z83.71\n",
      "Family history of colonic polyps \n",
      "\n",
      "Afirma Thyroid FNA analysis:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "0208U\n",
      "Oncology (medullary thyroid carcinoma), mRNA, gene expression analysis of 108 genes, utilizing fine needle aspirate, algorithm reported as positive or negative for medullary thyroid carcinoma\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "D44.0 \n",
      "Neoplasm of uncertain behavior of thyroid gland [indeterminate thyroid nodules] [not covered for repeat testing of indeterminate thyroid nodules]\n",
      "\n",
      "E04. 0 - E04.9\n",
      "Other nontoxic goiter\n",
      "\n",
      "Androgen receptor splice variant 7 (AR-V7):\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "Androgen receptor splice variant 7 (AR-V7) - no specific code:\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C61\n",
      "Malignant neoplasm of prostate\n",
      "\n",
      "BCL2 and BCL6 :\n",
      "\n",
      "CPT codes covered when selection criteria are met:\n",
      "\n",
      "BCL6 - no specific code:\n",
      "\n",
      "81278\n",
      "IGH@/BCL2 (t(14;18)) (eg, follicular lymphoma) translocation analysis, major breakpoint region (MBR) and minor cluster region (mcr) breakpoints, qualitative or quantitative\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C82.00 - C88.9\n",
      "Follicular lymphoma, Non-follicular lymphoma, Mature T/NK-cell lymphomas, Other specified and unspecified types of non-follicular lymphoma, Other specified types of T/NK-cell lymphoma, and Malignant immunoproliferative diseases and certain other B-cell lymphomas [non-Hodgkinâs lymphomas]\n",
      "\n",
      "D47.Z2\n",
      "Castleman disease\n",
      "\n",
      "Z12.89\n",
      "Encounter for screening for malignant neoplasm of other sites [for diagnosis of non-Hodgkin's lymphoma and Castleman disease]\n",
      "\n",
      "FISH assay of the BCR/ABL gene:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "0040U\n",
      "BCR/ABL1 (t(9;22)) (eg, chronic myelogenous leukemia) translocation analysis, major breakpoint, quantitative\n",
      "\n",
      "81206 - 81208\n",
      "BCR/ABL1 (t(9;22)) (eg, chronic myelogenous leukemia) translocation analysis\n",
      "\n",
      "88271\n",
      "Molecular cytogenetics; DNA probe, each (eg, FISH) \n",
      "\n",
      "88275\n",
      "    interphase in situ hybridization, analyze 100-300 cells\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C83.50 - C83.59\n",
      "Lymphoblastic (diffuse) lymphoma\n",
      "\n",
      "C91.00 - C91.02\n",
      "Acute myeloblastic leukemia  \n",
      "\n",
      "C91.10 - C91.12\n",
      "Chronic lymphocytic leukemia of B-cell type \n",
      "\n",
      "C92.00 - C92.12\n",
      "Myeloid leukemia\n",
      "\n",
      "C92.20  - C92.62\n",
      "Atypical chronic myeloid leukemia BCR/ABL â negative, myeloid sarcoma, acute promyelocytic leukemia, acute myelomonocytic leukemia, and acute myeloid leukemia with 11q23-abnormality  \n",
      "\n",
      "C92.A0 - C92.A2\n",
      "Acute myeloid leukemia with multilineage dysplasia \n",
      "\n",
      "C92.Z0 - C92.Z2\n",
      "Other myeloid leukemia\n",
      "\n",
      "C94.40 - C94.42\t\n",
      "Acute panmyelosis with myelofibrosis\n",
      "\n",
      "D45\n",
      "Polycythemia vera\n",
      "\n",
      "D47.1\n",
      "Chronic myeloproliferative disease\n",
      "\n",
      "D47.4\n",
      "Osteomyelofibrosis\n",
      "\n",
      "D69.3\n",
      "Immune thrombocytopenic purpura\n",
      "\n",
      "D75.81\n",
      "Myelofibrosis\n",
      "\n",
      "Cancer antigen 125 (CA 125):\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "86304\n",
      "Immunoassay for tumor antigen, quantitative; CA 125\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C56.1 - C56.9\n",
      "Malignant neoplasm of ovary\n",
      "\n",
      "D39.10 - D39.12\n",
      "Neoplasm of uncertain behavior of ovary\n",
      "\n",
      "Z12.73\n",
      "Encounter for screening for malignant neoplasm of ovary\n",
      "\n",
      "Z80.41\n",
      "Family history of malignant neoplasm of ovary\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "Z12.11 - Z12.12\n",
      "Encounter for screening for malignant neoplasm of colon and rectum\n",
      "\n",
      "Z85.43\n",
      "Personal history of malignant neoplasm of ovary\n",
      "\n",
      "Serial measurements of CA 15-3 (also known as CA 27-29 or Truquant RIA):\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "86300\n",
      "Immunoassay for tumor antigen, quantitative; CA 15-3 (27.29)\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C50.011 - C50.019C50.111 - C50.119C50.211 - C50.219C50.311 - C50.319C50.411 - C50.419C50.511 - C50.519C50.611 - C50.619C50.811 - C50.819C50.911 - C50.919\n",
      "Malignant neoplasm of the female breast\n",
      "\n",
      "D05.00 - D05.92\n",
      "Carcinoma in situ of breast\n",
      "\n",
      "Z85.3\n",
      "Personal history of malignant neoplasm of breast\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "Z12.31 - Z12.39\n",
      "Encounter for screening for malignant neoplasm of breast\n",
      "\n",
      "CA 19-9:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "86301\n",
      "Immunoassay for tumor antigen, quantitative; CA 19-9\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C16.0 - C16.9\n",
      "Malignant neoplasm of stomach\n",
      "\n",
      "C17.0 - C17.9\n",
      "Malignant neoplasm of small intestine [small bowel adenocarcinoma]\n",
      "\n",
      "C18.1\n",
      "Malignant neoplasm of appendix [mucinous appendiceal carcinoma]\n",
      "\n",
      "C22.1\n",
      "Intrahepatic bile duct carcinoma [cholangiocarcinoma]\n",
      "\n",
      "C23 - C24.9\n",
      "Malignant neoplasm of gallbladder and other and unspecified parts of biliary tract\n",
      "\n",
      "C25.0 - C25.9\n",
      "Malignant neoplasm of pancreas\n",
      "\n",
      "C30.0\n",
      "Malignant neoplasm of nasal cavity [NUT midline carcinoma]\n",
      "\n",
      "C56.1 - C56.9\t\n",
      "Malignant neoplasm of ovary\n",
      "\n",
      "D00.2\n",
      "Carcinoma in situ of stomach\n",
      "\n",
      "D01.5\n",
      "Carcinoma in situ of liver, gallbladder and bile ducts [covered for gallbladder and bile duct]\n",
      "\n",
      "D01.7 - D01.9\n",
      "Carcinoma in situ of other and unspecified digestive organs\n",
      "\n",
      "K83.1\n",
      "Obstruction of bile duct\n",
      "\n",
      "R17\n",
      "Unspecified jaundice\n",
      "\n",
      "R93.2\n",
      "Abnormal findings on diagnostic imaging of liver and biliary tract\n",
      "\n",
      "R94.5\n",
      "Abnormal results of liver function studies\n",
      "\n",
      "Z76.82\n",
      "Awaiting organ transplant status \n",
      "\n",
      "Z85.028\n",
      "Personal history of other malignant neoplasm of stomach\n",
      "\n",
      "Z85.07 - Z85.09\n",
      "Personal history of malignant neoplasm of pancreas and other digestive organs\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB (not all-inclusive):\n",
      "\n",
      "C15.3 - C15.9\n",
      "Malignant neoplasm of esophagus\n",
      "\n",
      "C18.0 - C20\n",
      "Malignant neoplasm of colon, rectosigmoid junction and rectum\n",
      "\n",
      "C22.0, C22.2 - C22.9\n",
      "Malignant neoplasm of liver\n",
      "\n",
      "C50.011 - C50.929\n",
      "Malignant neoplasm of the breast\n",
      "\n",
      "C53.0 - C55, C58\n",
      "Malignant neoplasm of uterus\n",
      "\n",
      "C67.0 - C67.9\n",
      "Malignant neoplasm of bladder\n",
      "\n",
      "D01.0\n",
      "Carcinoma in situ of colon\n",
      "\n",
      "D01.5\n",
      "Carcinoma in situ of liver, gallbladder and bile ducts [not covered for liver]\n",
      "\n",
      "D05.00 - D05.92\n",
      "Carcinoma in situ of breast\n",
      "\n",
      "N83.00 - N83.299\n",
      "Ovarian cysts\n",
      "\n",
      "Cardioembryonic antigen cellular adhesion molecule-7 (CEACAM-7) - No specific code:\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C19 - C21.8\n",
      "Malignant neoplasm of rectum, rectosigmoid junction and anus\n",
      "\n",
      "D01.1 - D01.2\n",
      "Carcinoma in situ of rectosigmoid junction and rectum\n",
      "\n",
      "Z85.048\n",
      "Personal history of other malignant neoplasm of rectum, rectosigmoid junction, and anus\n",
      "\n",
      "Molecular Intelligence Services, including MI Profile and MI Profile PLUS (formerly Target Now Molecular Profiling Test, including Target Now Select and Target Now Comprehensive) - No specific code:\n",
      "\n",
      "Cyfra21-1 (a cytokeratin 19 fragment,) p53, & Squamous cell carcinoma antigen (SCC-Ag) - No specific code:\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C15.3 - C15.9\n",
      "Malignant neoplasm of esophagus\n",
      "\n",
      "Vascular endothelial growth factor (VEGF) :\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "VEGF - No specific code:\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "D47.Z2\t\n",
      "Castleman's disease\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C15.3 - C15.9\n",
      "Malignant neoplasm of esophagus\n",
      "\n",
      "Human epidermal growth factor receptor 2 (HER2) evaluation:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "83950\n",
      "Oncoprotein;  Her-2/neu\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C15.3 - C15.9\n",
      "Malignant neoplasm of esophagus\n",
      "\n",
      "C16.0 - C16.9\n",
      "Malignant neoplasm of stomach\n",
      "\n",
      "C34.00 - C34.92\n",
      "Malignant neoplasm of bronchus and lung [non-small cell]\n",
      "\n",
      "C50.011 - C50.929\n",
      "Malignant neoplasm of breast [see criteria]\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB (not all-inclusive):\n",
      "\n",
      "C18.1\n",
      "Malignant neoplasm of appendix\n",
      "\n",
      "IGH@ (Immunoglobulin heavy chain locus):\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "81168\n",
      "CCND1/IGH (t(11;14)) (eg, mantle cell lymphoma) translocation analysis, major breakpoint, qualitative and quantitative, if performed\n",
      "\n",
      "81261\n",
      "IGH@ (Immunoglobulin heavy chain locus) (eg, leukemias and lymphomas, B-cell), gene rearrangement analysis to detect abnormal clonal population(s); amplified methodology (eg, polymerase chain reaction)\n",
      "\n",
      "81278\n",
      "IGH@/BCL2 (t(14;18)) (eg, follicular lymphoma) translocation analysis, major breakpoint region (MBR) and minor cluster region (mcr) breakpoints, qualitative or quantitative\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C85.10 - C85.99\n",
      "Other specified and unspecified types of non-Hodgkin lymphoma\n",
      "\n",
      "C91.40 - C91.42\n",
      "Hairy cell leukemia\n",
      "\n",
      "D47.z1  \n",
      "Post-transplant lymphoproliferative disorder (PTLD) \n",
      "\n",
      "E85.9\n",
      "Amyloidosis, unspecified [systemic light chain]\n",
      "\n",
      "IGK@ (Immunoglobulin kappa light chain locus):\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "81264\n",
      "IGK@ (Immunoglobulin kappa light chain locus) (eg, leukemia and lymphoma, B-cell), gene rearrangement analysis, evaluation to detect abnormal clonal population(s)\n",
      "\n",
      "83521\n",
      "Immunoglobulin light chains (ie, kappa, lambda), free, each\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C85.10 - C85.99\n",
      " Other specified and unspecified types of non-Hodgkin lymphoma\n",
      "\n",
      "C91.40 - C91.42\n",
      "Hairy cell leukemia\n",
      "\n",
      "E85.9\n",
      "Amyloidosis, unspecified [systemic light chain]\n",
      "\n",
      "Serial measurements of human chorionic gonadotropin (HCG):\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "84702\n",
      "Gonadotropin, chorionic (hCG); quantitative \n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C56.1 - C56.9\n",
      "Malignant neoplasm of ovary\n",
      "\n",
      "C58\n",
      "Malignant neoplasm of placenta (e.g., choriocarcinoma)\n",
      "\n",
      "C62.00 - C62.92\n",
      "Malignant neoplasm of testis\n",
      "\n",
      "C77.1\n",
      "Secondary malignant neoplasm of intrathoracic lymph nodes [mediastinal nodes]\n",
      "\n",
      "D07.30 - D07.39\n",
      "Carcinoma in situ of other and unspecified female genital organs [germinal cell tumors (teratocarcinoma and embryonal cell carcinoma) of the ovaries] [tumors (teratocarcinoma and embryonal cell carcinoma) of the ovaries]\n",
      "\n",
      "D07.60 - D07.69\n",
      "Carcinoma in situ of other and unspecified male genital organs\n",
      "\n",
      "D39.2\n",
      "Neoplasm of uncertain behavior of placenta \n",
      "\n",
      "O01.9\n",
      "Hydatidiform mole, unspecified\n",
      "\n",
      "Z85.43\n",
      "Personal history of malignant neoplasm of ovary\n",
      "\n",
      "Z85.47\n",
      "Personal history of malignant neoplasm of testis\n",
      "\n",
      "Serial measurements of AFP to diagnose germ cell tumors or the diagnosis and monitoring of hepatocellular carcinoma:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "82105\n",
      "Alpha-fetoprotein (AFP); serum \n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "B17.10 - B17.11\n",
      "Acute hepatitis C without or with hepatic coma\n",
      "\n",
      "B18.2\n",
      "Chronic viral hepatitis C\n",
      "\n",
      "B19.20 - B19.21\n",
      "Unspecified viral hepatitis C without or with hepatic coma\n",
      "\n",
      "C22.0 - C22.9\n",
      "Malignant neoplasm of the liver and intrahepatic bile ducts\n",
      "\n",
      "C37\n",
      "Malignant neoplasm of thymus\n",
      "\n",
      "C56.1 - C56.9\n",
      "Malignant neoplasm of ovary\n",
      "\n",
      "C62.00 - C62.92\n",
      "Malignant neoplasm of testes\n",
      "\n",
      "C77.1\n",
      "Secondary malignant neoplasm of intrathoracic lymph nodes [mediastinal nodes]\n",
      "\n",
      "D01.5\n",
      "Carcinoma in situ of liver, gallbladder and bile ducts\n",
      "\n",
      "D07.30 - D07.39\n",
      "Carcinoma in situ of other and unspecified female genital organs [germ cell tumors]\n",
      "\n",
      "D07.60 - D07.69\n",
      "Carcinoma in situ of other and unspecified male genital organs\n",
      "\n",
      "D15.0\n",
      "Benign neoplasm of thymus\n",
      "\n",
      "E83.110\n",
      "Hereditary hemochromatosis\n",
      "\n",
      "E88.01\n",
      "Alpha-1-antitrypsin deficiency\n",
      "\n",
      "F10.10 - F10.99\n",
      "Alcohol related disorders\n",
      "\n",
      "K70.30 - K70.31\n",
      "Alcoholic cirrhosis of liver without or with ascites\n",
      "\n",
      "K74.3\n",
      "Primary biliary cirrhosis brackets [stage 4 primary biliary cirrhosis]\n",
      "\n",
      "K74.60 - K74.69\n",
      "Unspecified or other cirrhosis of liver\n",
      "\n",
      "K75.81\n",
      "Nonalcoholic steatohepatitis (NASH)\n",
      "\n",
      "N50.8\n",
      "Other specified disorders of male genital organs [testicular mass]\n",
      "\n",
      "R19.00\n",
      "Intra-abdominal and pelvic swelling, mass, lump, unspecified site\n",
      "\n",
      "R19.07 - R19.09\n",
      "Generalized and other intra-abdominal and pelvic swelling, mass and lump\n",
      "\n",
      "R22.2\n",
      "Localized swelling, mass and lump, trunk\n",
      "\n",
      "Z12.89\n",
      "Encounter for screening for malignant neoplasm of other sites\n",
      "\n",
      "Z22.51\n",
      "Carrier of viral Hepatitis B\n",
      "\n",
      "Z80.0\n",
      "Family history of malignant neoplasm of digestive organs [family history of hepatocellular carcinoma]\n",
      "\n",
      "Z85.43\n",
      "Personal history of malignant neoplasm of ovary\n",
      "\n",
      "Z85.47\n",
      "Personal history of malignant neoplasm of testis\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB (not all-inclusive):\n",
      "\n",
      "C58\n",
      "Malignant neoplasm of placenta (e.g., choriocarcinoma)\n",
      "\n",
      "D39.2\n",
      "Neoplasm of uncertain behavior of placenta\n",
      "\n",
      "O01.9\n",
      "Hydatidiform mole, unspecified\n",
      "\n",
      "Serial measurements of AFP and HCG together to diagnose and monitor testicular cancer:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "82105\n",
      "Alpha-fetoprotein (AFP); serum \n",
      "\n",
      "84702\n",
      "Gonadotropin, chorionic (hCG); quantitative\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C62.00 - C62.92\n",
      "Malignant neoplasm of testes\n",
      "\n",
      "D07.60 - D07.69\n",
      "Carcinoma in situ of other and unspecified male genital organs\n",
      "\n",
      "Z12.71\n",
      "Encounter for screening for malignant neoplasm of testis\n",
      "\n",
      "Measurement of estrogen and progesterone receptors and steroid receptor:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "84233\n",
      "Receptor assay; estrogen\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C50.011 - C50.929\n",
      "Malignant neoplasm of breast\n",
      "\n",
      "C53.0 - C55, C58\n",
      "Malignant neoplasm of uterus [sarcoma]\n",
      "\n",
      "C56.1 - C56.9\n",
      "Malignant neoplasm of ovary\n",
      "\n",
      "C80.1\n",
      "Malignant (primary) neoplasm, unspecified [occult primary]\n",
      "\n",
      "D05.00 - D05.92\n",
      "Carcinoma in situ of breast\n",
      "\n",
      "Measurement of progesterone receptors:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "84234\n",
      "Receptor assay; progesterone\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C34.00 - C34.92\n",
      "Malignant neoplasm of bronchus and lung [non-small cell lung cancer]\n",
      "\n",
      "C50.011 - C50.929\n",
      "Malignant neoplasm of breast\n",
      "\n",
      "C53.0 - C55, C58\n",
      "Malignant neoplasm of uterus [sarcoma]\n",
      "\n",
      "C80.1\n",
      "Malignant (primary) neoplasm, unspecified [occult primary]\n",
      "\n",
      "D05.00 - D05.92\n",
      "Carcinoma in situ of breast\n",
      "\n",
      "Targeted hematologic genomic sequencing panel (5-50 genes) for myelodysplastic syndromes (e.g., MedFusion myeloid malignancy analysis panel):\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "81450\n",
      "Targeted genomic sequence analysis panel, hematolymphoid neoplasm or disorder, DNA analysis, and RNA analysis when performed, 5-50 genes (eg, BRAF, CEBPA, DNMT3A, EZH2, FLT3, IDH1, IDH2, JAK2, KRAS, KIT, MLL, NRAS, NPM1, NOTCH1), interrogation for sequence variants, and copy number variants or rearrangements, or isoform expression or mRNA expression levels, if performed  \n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "D45\n",
      "Polycythemia vera\n",
      "\n",
      "D46.0 - D46.9\n",
      "Myelodysplastic syndromes\n",
      "\n",
      "D47.1\n",
      "Chronic myeloproliferative disease\n",
      "\n",
      "D47.3\n",
      "Essential (hemorrhagic) thrombocythemia\n",
      "\n",
      "Targeted solid organ genomic sequencing panel (5-50 genes):\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "81445\n",
      "Targeted genomic sequence analysis panel, solid organ neoplasm, DNA analysis, and RNA analysis when performed, 5-50 genes (eg, ALK, BRAF, CDKN2A, EGFR, ERBB2, KIT, KRAS, NRAS, MET, PDGFRA, PDGFRB, PGR, PIK3CA, PTEN, RET), interrogation for sequence variants and copy number variants or rearrangements, if performed  \n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C18.0 - C20\n",
      "Malignant neoplasm of colon, rectosigmoid junction and rectum\n",
      "\n",
      "C25.0 - C25.9\n",
      "Malignant neoplasm of pancreas\n",
      "\n",
      "C34.00 - C34.92\n",
      "Malignant neoplasm of bronchus and lung [non-small cell]\n",
      "\n",
      "C43.0 - C43.9\n",
      "Melanoma of skin \t\t\t\n",
      "\n",
      "C61\n",
      "Malignant neoplasm of prostate\n",
      "\n",
      "Oncomine™ Dx Target Test:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "0022U\n",
      "Targeted genomic sequence analysis panel, non-small cell lung neoplasia, DNA and RNA analysis, 23 genes, interrogation for sequence variants and rearrangements, reported as presence/absence of variants and associated therapy(ies) to consider [Oncomine™ Dx Target Test]\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C34.00 - C34.92\n",
      "Malignant neoplasm of bronchus and lung [non-small cell]\n",
      "\n",
      "T-cell receptor gene rearrangements:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "81340\n",
      "TRB@ (T cell antigen receptor, beta) (eg, leukemia and lymphoma), gene rearrangement analysis to detect abnormal clonal population(s); using amplification methodology (eg, polymerase chain reaction) \n",
      "\n",
      "81341\n",
      "    using direct probe methodology (eg, Southern blot)\n",
      "\n",
      "81342\n",
      "TRG@ (T cell antigen receptor, gamma) (eg, leukemia and lymphoma), gene rearrangement analysis, evaluation to detect abnormal clonal population(s)\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C84.00 - C84.09\n",
      "Mycosis fungoides\n",
      "\n",
      "C84.10 - C84.19\n",
      "Sezary disease\n",
      "\n",
      "C84.40 - C84.49\n",
      "Peripheral T-cell lymphoma, not classified\n",
      "\n",
      "C86.0\n",
      "Extranodal NK/T-cell lymphoma, nasal type\n",
      "\n",
      "C86.1\n",
      "Hepatosplenic T-cell lymphoma\n",
      "\n",
      "C86.6\n",
      "Primary cutaneous CD30-positive T-cell lymphoproliferations\n",
      "\n",
      "C91.60 - C91.62\n",
      "Prolymphocytic leukemia of T-cell type\n",
      "\n",
      "C91.Z0 - C91.Z2\n",
      "Other lymphoid leukemia with bracketed info [T-cell large granular lymphocytic]\n",
      "\n",
      "D46.0 - D46.9\n",
      "Myelodysplastic syndromes\n",
      "\n",
      "D47.Z2\n",
      "Castleman's disease\n",
      "\n",
      "D47.Z9\n",
      "Other specified neoplasms of uncertain or unknown behavior of lymphoid, hematopoietic, and related tissue\n",
      "\n",
      "ThyGeNEXT Thyroid Oncogene Panel and ThyraMIR:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "0018U\n",
      "Oncology (thyroid), microRNA profiling by RT-PCR of 10 microRNA sequences, utilizing fine needle aspirate, algorithm reported as a positive or negative result for moderate to high risk of malignancy\n",
      "\n",
      "0245U\n",
      "Oncology (thyroid), mutation analysis of 10 genes and 37 RNA fusions and expression of 4 mRNA markers using next-generation sequencing, fine needle aspirate, report includes associated risk of malignancy expressed as a percentage\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "D44.0\n",
      "Neoplasm of uncertain behavior of thyroid gland [indeterminate thyroid nodules]\n",
      "\n",
      "E04.0 - E04.9      \n",
      "Other nontoxic goiter [thyroid nodules]\n",
      "\n",
      "Thyroseq:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "0026U\n",
      "Oncology (thyroid), DNA and mRNA of 112 genes, next-generation sequencing, fine needle aspirate of thyroid nodule, algorithmic analysis reported as a categorical result (\"Positive, high probability of malignancy\" or \"Negative, low probability of malignancy\")\n",
      "\n",
      "0287U\n",
      "Oncology (thyroid), DNA and mRNA, next-generation sequencing analysis of 112 genes, fine needle aspirate or formalin-fixed paraffin-embedded (FFPE) tissue, algorithmic prediction of cancer recurrence, reported as a categorical risk result (low, intermediate, high)\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "D44.0\n",
      "Neoplasm of uncertain behavior of thyroid gland [indeterminate thyroid nodules]  \n",
      "\n",
      "E04.0 - E04.9\n",
      " Other nontoxic goiter [thyroid nodules] [not covered for repeat testing of indeterminate thyroid nodules]\n",
      "\n",
      "TP53:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "81351\n",
      "TP53 (tumor protein 53) (eg, Li-Fraumeni syndrome) gene analysis; full gene sequence\n",
      "\n",
      "81352\n",
      "     targeted sequence analysis (eg, 4 oncology)\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C45.0\n",
      "Mesothelioma of pleura \n",
      "\n",
      "C45.1\n",
      "Mesothelioma of peritoneum\n",
      "\n",
      "C53.0 - C55, C58\n",
      "Malignant neoplasm of uterus [sarcoma]\n",
      "\n",
      "C71.0 - C71.9\n",
      "Malignant neoplasm of brain [medulloblastoma]\n",
      "\n",
      "C80.1\n",
      "Malignant (primary) neoplasm, unspecified [occult primary]\t\n",
      "\n",
      "C7A.00 - C7A.8\n",
      "Malignant neuroendocrine tumors\n",
      "\n",
      "C83.00 - C83.09\n",
      "Small cell B cell lymphoma [splenic marginal zone lymphoma]\t\n",
      "\n",
      "C83.10 - C83.19\n",
      "Mantle cell lymphoma\n",
      "\n",
      "C84.40 - C84.49\t\n",
      "Peripheral T-cell lymphoma, not classified\n",
      "\n",
      "C91.00 â C91.02\n",
      "Acute lymphoblastic leukemia [ALL] [pediatric]\n",
      "\n",
      "C91.10 - C91.12\n",
      "Chronic lymphocytic leukemia of B-cell type\n",
      "\n",
      "C92.20 - C92.22\n",
      "Atypical chronic myeloid leukemia, BCR/ABL-negative\n",
      "\n",
      "C92.60 - C92.62\n",
      "Acute myeloid leukemia with 11q23 abnormality\n",
      "\n",
      "C92.A0 - C92.A2\n",
      "Acute myeloid leukemia with multilineage dysplasia\n",
      "\n",
      "C94.00 - C94.02\n",
      "Acute erythroid leukemia [acute myeloid leukemia]\n",
      "\n",
      "C94.20 - C94.22\n",
      "Acute megakaryoblastic leukemia [acute myeloid leukemia]\n",
      "\n",
      "C94.6\n",
      "Myelodysplastic disease, not elsewhere classified [myeloproliferative neoplasms]\n",
      "\n",
      "D3A.00 - D3A.8\n",
      "Benign neuroendocrine tumors\n",
      "\n",
      "D45\n",
      "Polycythemia vera [myeloproliferative neoplasms]\n",
      "\n",
      "D46.0 - D46.Z\n",
      "Myelodysplastic syndromes\n",
      "\n",
      "D47.3\n",
      "Essential (hemorrhagic) thrombocythemia [myeloproliferative neoplasms]\n",
      "\n",
      "D75.81\n",
      "Myelofibrosis [myeloproliferative neoplasms]\n",
      "\n",
      "U2AF1 test:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "81357\n",
      "U2AF1 (U2 small nuclear RNA auxiliary factor 1) (eg, myelodysplastic syndrome, acute myeloid leukemia) gene analysis, common variants (eg, S34F, S34Y, Q157R, Q157P)\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C86.4\n",
      "Blastic NK-cell lymphoma\n",
      "\n",
      "C92.10 â C92.12\n",
      "Chronic myeloid leukemia, BCR/ABL-positive\n",
      "\n",
      "C92.20 - C92.22\n",
      "Atypical chronic myeloid leukemia, BCR/ABL-negative\n",
      "\n",
      "C94.6\n",
      "Myelodysplastic disease, not elsewhere classified [myeloproliferative neoplasms]\n",
      "\n",
      "D45\n",
      "Polycythemia vera [myeloproliferative neoplasms]\n",
      "\n",
      "D46.0 â D46.Z\n",
      "Myelodysplastic syndromes\n",
      "\n",
      "D47.3\n",
      "Essential (hemorrhagic) thrombocythemia [myeloproliferative neoplasms]\n",
      "\n",
      "D75.81\n",
      "Myelofibrosis [myeloproliferative neoplasms]\n",
      "\n",
      "K-ras (KRAS) with BRAF reflex testing:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "81210\n",
      "BRAF (v-raf murine sarcoma viral oncogene homolog B1) (eg, colon cancer), gene analysis, V600E variant \n",
      "\n",
      "81275\n",
      "KRAS (Kirsten rat sarcoma viral oncogene homolog) (eg, carcinoma) gene analysis; variants in exon 2 (eg, codons 12 and 13)  \n",
      "\n",
      "81276\n",
      "KRAS (Kirsten rat sarcoma viral oncogene homolog) (eg, carcinoma) gene analysis; additional variant(s) (eg, codon 61, codon 146) \n",
      "\n",
      "Other CPT codes related to the CPB:\n",
      "\n",
      "88363\n",
      "Examination and selection of retrieved archival (ie, previously diagnosed) tissue(s) for molecular analysis (eg, KRAS mutational analysis)\n",
      "\n",
      "Other HCPCS codes related to the CPB:\n",
      "\n",
      "J9055\n",
      "Injection, cetuximab, 10 mg [to predict non-response to cetuximab (Erbitux) and panitumumab (Vectibix) in the treatment of anal adenocarcinoma]\n",
      "\n",
      "J9303\n",
      "Injection, panitumumab, 10 mg [to predict non-response to cetuximab (Erbitux) and panitumumab (Vectibix) in the treatment of anal adenocarcinoma]\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C17.0 - C17.9\n",
      "Malignant neoplasm of small intestine [small bowel adenocarcinoma]\n",
      "\n",
      "C18.0 - C20\n",
      "Malignant neoplasm of colon, rectosigmoid junction and rectum [metastatic colorectal cancer]\n",
      "\n",
      "C21.0 - C21.1\n",
      "Malignant neoplasm of anal canal and anus [anal adenocarcinoma]\n",
      "\n",
      "C34.00 - C34.92\n",
      "Malignant neoplasm of bronchus and lung\n",
      "\n",
      "D01.1 - D01.2\n",
      "Carcinoma in situ of rectum [if KRAS nonmutated] [Lynch syndrome (HNPCC)]\n",
      "\n",
      "D12.7 - D12.9\n",
      "Benign neoplasm of rectum and anal canal [if KRAS nonmutated] [Lynch syndrome (HNPCC)]\n",
      "\n",
      "D44.0\n",
      "Neoplasm of uncertain behavior of thyroid gland [indeterminate thyroid nodules]  \n",
      "\n",
      "E04.0 - E04.9\n",
      "Other nontoxic goiter [thyroid nodules] [not covered for repeat testing of indeterminate thyroid nodules]\n",
      "\n",
      "Mismatch repair (MSI/dMMR, MLH1, MSH2, MSH6):\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "81292 - 81294\n",
      "MLH1 gene analysis\n",
      "\n",
      "81295 - 81297\n",
      "MSH2 gene analysis\n",
      "\n",
      "81298 - 81300\n",
      "MSH6 gene analysis\n",
      "\n",
      "81301\n",
      "Microsatellite instability analysis (eg, hereditary non-polyposis colorectal cancer, Lynch syndrome) of markers for mismatch repair deficiency (eg, BAT25, BAT26), includes comparison of neoplastic and normal tissue, if performed\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C15.3 - C15.9\n",
      "Malignant neoplasm of esophagus\n",
      "\n",
      "C16.0 - C16.9\n",
      "Malignant neoplasm of stomach\n",
      "\n",
      "C17.0 - C17.9\n",
      "Malignant neoplasm of small intestine [small bowel adenocarcinoma]\n",
      "\n",
      "C18.0 - C18.9\n",
      "Malignant neoplasm of colon [Lynch syndrome (HNPCC)] [all persons with Stage 2 colon cancer]\n",
      "\n",
      "C19 - C21.8\n",
      "Malignant neoplasm of rectum, rectosigmoid junction, and anus  [Lynch syndrome (HNPCC)] [all persons with Stage 2 colon cancer] [under age 50]\n",
      "\n",
      "C23\n",
      "Malignant neoplasm of gallbladder\n",
      "\n",
      "C25.0 - C25.9\n",
      "Malignant neoplasm of pancreas\n",
      "\n",
      "C41.0 â C41.9\n",
      "Malignant neoplasm of bone and articular cartilage of other and unspecified sites [Ewing sarcoma]\n",
      "\n",
      "C50.011 - C50.929\n",
      "Malignant neoplasm of breast\n",
      "\n",
      "C53.0 - C53.9\n",
      "Malignant neoplasm of cervix uteri\n",
      "\n",
      "C54.0 - C54.9\n",
      "Malignant neoplasm of corpus uteri\n",
      "\n",
      "C60.0 - C60.9\n",
      "Malignant neoplasm of penis\n",
      "\n",
      "C61\n",
      "Malignant neoplasm of prostate\n",
      "\n",
      "C62.0 - C62.92\n",
      "Malignant neoplasm of testis\n",
      "\n",
      "C80.1\n",
      "Malignant (primary) neoplasm, unspecified [occult primary]\n",
      "\n",
      "D01.1 - D01.2\n",
      "Carcinoma in situ of rectum [under age 50]\n",
      "\n",
      "D12.7 - D12.9\n",
      "Benign neoplasm of rectum and anal canal [under age 50]\n",
      "\n",
      "D46.0 - D46.9\t\n",
      "Myelodysplastic syndromes\n",
      "\n",
      "MLH1 tumor promoter hypermethylation:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "81288\n",
      "MLH1 (mutl homolog 1, colon cancer, nonpolyposis type 2) (eg, hereditary non-polyposis colorectal cancer, Lynch syndrome) gene analysis; promoter methylation analysis\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met::\n",
      "\n",
      "C54.1\n",
      "Malignant neoplasm of endometrium\n",
      "\n",
      "MPL (myeloproliferative leukemia protein):\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "81338\n",
      "MPL (MPL proto-oncogene, thrombopoietin receptor) (eg, myeloproliferative disorder) gene analysis; common variants (eg, W515A, W515K, W515L, W515R)\n",
      "\n",
      "81339\n",
      "MPL (MPL proto-oncogene, thrombopoietin receptor) (eg, myeloproliferative disorder) gene analysis; sequence analysis, exon 10\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C92.10 â C92.12\n",
      "Chronic myeloid leukemia, BCR/ABL-positive\n",
      "\n",
      "C92.20 - C92.22\n",
      "Atypical chronic myeloid leukemia, BCR/ABL-negative\n",
      "\n",
      "C94.6\n",
      "Myelodysplastic disease, not elsewhere classified [myeloproliferative neoplasms]\n",
      "\n",
      "D45\n",
      "Polycythemia vera \n",
      "\n",
      "D46.0 â D46.Z\n",
      "Myelodysplastic syndromes\n",
      "\n",
      "D47.3\n",
      "Essential (hemorrhagic) thrombocythemia \n",
      "\n",
      "D75.81\n",
      "Myelofibrosis \n",
      "\n",
      "Murine double minute 2 (MDM2):\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "Murine double minute 2 (MDM2) â No specific code\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C49.0 - C49.9\n",
      "Malignant neoplasm of other connective and soft tissue [sarcoma]\n",
      "\n",
      "C53.0 - C55, C58\n",
      "Malignant neoplasm of uterus [sarcoma]\n",
      "\n",
      "MYD88:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "81305\n",
      "MYD88 (myeloid differentiation primary response 88) (eg, Waldenstrom's macroglobulinemia, lymphoplasmacytic leukemia) gene analysis, p.Leu265Pro (L265P) variant\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C83.00 - C83.09\n",
      "Small cell B-cell lymphoma\n",
      "\n",
      "C88.0\n",
      "Waldenstrom macroglobulinemia\n",
      "\n",
      "C88.4\n",
      "Extranodal marginal zone B-cell lymphoma of mucosa-associated lymphoid tissue [MALT-lymphoma]  \n",
      "\n",
      "C90.00 - C90.02\n",
      "Multiple myeloma\n",
      "\n",
      "MyMRD NGS Panel:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "0171U\n",
      "Targeted genomic sequence analysis panel, acute myeloid leukemia, myelodysplastic syndrome, and myeloproliferative neoplasms, DNA analysis, 23 genes, interrogation for sequence variants, rearrangements and minimal residual disease, reported as presence/absence\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C92.00 - C92.02 C92.40 - C92.A2\n",
      "Acute myeloid leukemia (AML)\n",
      "\n",
      "D46.0 - D46.Z\n",
      "Myelodysplastic syndromes\n",
      "\n",
      "Next generation sequencing of tumor DNA (e.g., ClonoSeq):\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "0364U\n",
      "Oncology (hematolymphoid neoplasm), genomic sequence analysis using multiplex (PCR) and next-generation sequencing with algorithm, quantification of dominant clonal sequence(s), reported as presence or absence of minimal residual disease (MRD) with quantitation of disease burden, when appropriate\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C90.00 - C90.02\n",
      "Multiple myeloma\n",
      "\n",
      "C91.00 - C91.02 \n",
      "Acute lymphoblastic leukemia [ALL]\n",
      "\n",
      "MSK-IMPACT:\n",
      "\n",
      "CPT codes not covered for indications listed in the CPB:\n",
      "\n",
      "0048U\n",
      "Oncology (solid organ neoplasia), DNA, targeted sequencing of protein-coding exons of 468 cancer-associated genes, including interrogation for somatic mutations and microsatellite instability, matched with normal specimens, utilizing formalin-fixed paraffin-embedded tumor tissue, report of clinically significant mutation(s)\n",
      "\n",
      "MUC1 - no specific code:\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C16.0 - C16.9 \n",
      "Malignant neoplasm of stomach \n",
      "\n",
      "ALK Gene Fusion:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "ALK Gene Fusion - no specific code\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C34.00 - C34.92\n",
      "Malignant neoplasm of bronchus and lung [non-small-cell cancer]\n",
      "\n",
      "ALK Gene Rearrangement:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "ALK Gene Rearrangement - no specific code\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C34.00 â C34.92\n",
      "Malignant neoplasm of bronchus and lung [non-small cell]\n",
      "\n",
      "C83.30 - C83.39\n",
      "Diffuse large B-cell lymphoma\n",
      "\n",
      "C84.40 - C84.49\n",
      "Peripheral T-cell lymphoma, not classified\n",
      "\n",
      "D47.z1 \n",
      "Post-transplant lymphoproliferative disorder (PTLD) \n",
      "\n",
      "ALK :\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "ALK Expression - no specific code\n",
      "\n",
      "Other CPT codes related to CPB:\n",
      "\n",
      "81401\n",
      "Molecular pathology procedure, Level 2 (eg, 2-10 SNPs, 1 methylated variant, or 1 somatic variant [typically using nonsequencing target variant analysis], or detection of a dynamic mutation disorder/triplet repeat)  \n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C25.0 - C25.9\n",
      "Malignant neoplasm of pancreas\n",
      "\n",
      "C34.00 - C34.92\n",
      "Malignant of neoplasm of bronchus and lung [non-small-cell lung cancer]\n",
      "\n",
      "C53.0 â C55\n",
      "Malignant neoplasm of cervix uteri, corpus uteri, and uterus, part unspecified\n",
      "\n",
      "C81.00 â C81.99\n",
      "Hodgkin lymphoma [pediatric only]\n",
      "\n",
      "C84.40 - C84.49\t\n",
      "Peripheral T-cell lymphoma, not classified\n",
      "\n",
      "C84.60 â C84.69\n",
      "Anaplastic large cell lymphoma, ALK-positive [breast implant-associated]\n",
      "\n",
      "C84.70 â C84.79\n",
      "Anaplastic large cell lymphoma, ALK-negative [breast implant-associated]\n",
      "\n",
      "Urokinase plasminogen activator (uPA) and plasminogen activator inhibitor 1 (PAI-1):\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "85415\n",
      "Fibrinolytic factors and inhibitors; plasminogen activator\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C50.011 - C50.929\n",
      "Malignant neoplasm of breast [node negative]\n",
      "\n",
      "D05.00 - D05.92\n",
      "Carcinoma in situ of breast     \n",
      "\n",
      "Veristrat:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "81538\n",
      "Oncology (lung), mass spectrometric 8-protein signature, including amyloid A, utilizing serum, prognostic and predictive algorithm reported as good versus poor overall survival \n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C34.00 - C34.92\n",
      "Malignant neoplasm of bronchus and lung [for persons with advanced NSCLC, whose tumors are without EGFR and ALK mutations, who have progressed after at least one chemotherapy regimen, and for whom erlotinib is considered an appropriate treatment]\n",
      "\n",
      "CD 117 (c-kit):\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "81272\n",
      "KIT (v-kit Hardy-Zuckerman 4 feline sarcoma viral oncogene homolog) (eg, gastrointestinal stromal tumor [GIST], acute myeloid leukemia, melanoma), gene analysis, targeted sequence analysis (eg, exons 8, 11, 13, 17, 18) \n",
      "\n",
      "81273\n",
      "KIT (v-kit Hardy-Zuckerman 4 feline sarcoma viral oncogene homolog) (eg, mastocytosis), gene analysis, D816 variant(s) \n",
      "\n",
      "88184\n",
      "Flow cytometry, cell surface, cytoplasmic, or nuclear marker, technical component only; first marker\n",
      "\n",
      "+ 88185\n",
      "    each additional marker (List separately in addition to code for first marker)\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "[for determining eligibility for treatment with Gleevac]\n",
      "\n",
      "C15.3 - C15.9\n",
      "Malignant neoplasm of esophagus\n",
      "\n",
      "C43.0 - C43.9\n",
      "Melanoma of skin\n",
      "\n",
      "C49.4\n",
      "Malignant neoplasm of connective and soft tissue of abdomen [gastrointestinal stromal tumors]\n",
      "\n",
      "C92.00 - C92.12\n",
      "Myeloid leukemia\n",
      "\n",
      "D47.02\n",
      "Systemic mastocytosis\n",
      "\n",
      "CD 20:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "88184\n",
      "Flow cytometry, cell surface, cytoplasmic, or nuclear marker, technical component only; first marker\n",
      "\n",
      "+ 88185\n",
      "    each additional marker (List separately in addition to code for first marker)\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "[for determining eligibility for treatment with Rituxan]\n",
      "\n",
      "C81.00 - C86.6C88.4C91.10 - C91.12C91.40 - C91.42C96.0 - C96.4C96.a - C96.9\n",
      "Malignant neoplasms of lymphoid, hematopoietic and related tissue\n",
      "\n",
      "CD 25:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "88184\n",
      "Flow cytometry, cell surface, cytoplasmic, or nuclear marker, technical component only; first marker\n",
      "\n",
      "+ 88185\n",
      "    each additional marker (List separately in addition to code for first marker)\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "[for determining eligibility for treatment with Ontak]\n",
      "\n",
      "C84.00 - C84.49\n",
      "Mycosis fungoides, Sezary disease and peripheral T-cell lymphoma, not classified\n",
      "\n",
      "CD 31 - no specific code:\n",
      "\n",
      "Other CPT codes related to the CPB:\n",
      "\n",
      "88341 - 88344\n",
      "Immunohistochemistry or immunocytochemistry, per specimen\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C49.0 - C49.9\n",
      "Malignant neoplasm of other connective and soft tissue [angiosarcoma]\n",
      "\n",
      "CD 33:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "88184\n",
      "Flow cytometry, cell surface, cytoplasmic, or nuclear marker, technical component only; first marker\n",
      "\n",
      "+ 88185\n",
      "    each additional marker (List separately in addition to code for first marker)\n",
      "\n",
      "88187\n",
      "Flow cytometry, interpretation; 2 to 8 markers\n",
      "\n",
      "88189\n",
      "    16 or more markers\n",
      "\n",
      "88341\n",
      "Immunohistochemistry or immunocytochemistry, per specimen; each additional single antibody stain procedure (List separately in addition to code for primary procedure)\n",
      "\n",
      "88342\n",
      "Immunohistochemistry or immunocytochemistry, per specimen; initial single antibody stain procedure\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "[for determining eligibility for treatment with Mylotarg]\n",
      "\n",
      "C83.50 - C83.59\n",
      "Lymphoblastic (diffuse) lymphoma\n",
      "\n",
      "C91.00 - C91.02\n",
      "Acute lymphoblastic leukemia [ALL]\n",
      "\n",
      "C92.00 - C92.02C92.40 - C92.a2\n",
      "Acute myeloid leukemia\n",
      "\n",
      "C93.00 - C93.02\n",
      "Acute monoblastic/monocytic leukemia\n",
      "\n",
      "C94.00 - C94.02\n",
      "Acute erythroid leukemia\n",
      "\n",
      "C95.00 - C95.02\n",
      "Acute leukemia of unspecified cell type\n",
      "\n",
      "CD 52:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "88184\n",
      "Flow cytometry, cell surface, cytoplasmic, or nuclear marker, technical component only; first marker\n",
      "\n",
      "+ 88185\n",
      "    each additional marker (List separately in addition to code for first marker)\n",
      "\n",
      "88187\n",
      "Flow cytometry, interpretation; 2 to 8 markers\n",
      "\n",
      "88342 \n",
      "Immunohistochemistry or immunocytochemistry, per specimen; initial single antibody stain procedure\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "[for determining eligibility for treatment with Campath]\n",
      "\n",
      "C82.00 - C82.99C83.10 - C83.89C84.00 - C84.49C84.a0 - C84.99C85.10 - C86.6C91.10 - C91.12C91.40 - C91.42\n",
      "Malignant neoplasms of lymphoid, hematopoietic and related tissue\n",
      "\n",
      "C91.60 - C91.62\n",
      "Prolymphocytic leukemia of T-cell type\n",
      "\n",
      "D47.Z1\n",
      "Post-transplant lymphoproliferative disorder (PTLD)\n",
      "\n",
      "Cyclin D1:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "81168\n",
      "CCND1/IGH (t(11;14)) (eg, mantle cell lymphoma) translocation analysis, major breakpoint, qualitative and quantitative, if performed\n",
      "\n",
      "81401\n",
      " Molecular pathology procedure, Level 2 (eg, 2-10 SNPs, 1 methylated variant, or 1 somatic variant [typically using nonsequencing target variant analysis], or detection of a dynamic mutation disorder/triplet repeat) (EML4/ALK inv(2)) (eg, non-small-cell lung cancer), translocation or inversion analysis\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C83.10 - C83.19\n",
      "Mantle cell lymphoma [diagnosing and predicting disease recurrence]\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C44.02, C44.121 - C44.129, C44.221 - C44.229, C44.320 - C44.329, C44.42\n",
      "Squamous cell carcinoma of lip, eyelid, ear and external canal, face, scalp and neck\n",
      "\n",
      "Decipher test (a RNA biomarkers assay):\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "81542\n",
      "Oncology (prostate), mRNA, microarray gene expression profiling of 22 content genes, utilizing formalin-fixed paraffin-embedded tissue, algorithm reported as metastasis risk score \n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C61\n",
      "Malignant neoplasm of prostate [not covered for repeat testing to assess risk of prostate cancer progression]\n",
      "\n",
      "DecisionDx-UM:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "81552\n",
      "Oncology (uveal melanoma), mRNA, gene expression profiling by real-time RT-PCR of 15 genes (12 content and 3 housekeeping), utilizing fine needle aspirate or formalin-fixed paraffin-embedded tissue, algorithm reported as risk of metastasis\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C69.30 - C69.42\n",
      "Malignant neoplasm of choroid and ciliary body [localized uveal melanoma]\n",
      "\n",
      "Endopredict (12-gene score):\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "81522\n",
      "Oncology (breast), mRNA, gene expression profiling by RT-PCR of 12 genes (8 content and 4 housekeeping), utilizing formalin-fixed paraffin-embedded tissue, algorithm reported as recurrence risk score\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C50.011 - C50.929 \n",
      "Malignant neoplasm of breast \n",
      "\n",
      "FIP1L1-PDGFRA fusion:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "88271\t   \n",
      "Molecular cytogenetics; DNA probe, each (eg, FISH)\n",
      "\n",
      "88275\n",
      "     interphase in situ hybridization, analyze 100-300 cells\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "D47.02\n",
      "Systemic mastocytosis\n",
      "\n",
      "FIP1L1-PDGFRA gene rearrangements:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "88374\n",
      "Morphometric analysis, in situ hybridization (quantitative or semi-quantitative), using computer-assisted technology, per specimen; each multiplex probe stain procedure\n",
      "\n",
      "88377\n",
      "Morphometric analysis, in situ hybridization (quantitative or semi-quantitative), manual, per specimen; each multiplex probe stain procedure\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "D47.Z9\n",
      "Other specified neoplasms of uncertain or unknown behavior of lymphoid, hematopoietic, and related tissue\n",
      "\n",
      "FLT3 gene mutation:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "0046U\t\n",
      "FLT3 (fms-related tyrosine kinase 3) (eg, acute myeloid leukemia) internal tandem duplication (ITD) variants, quantitative\n",
      "\n",
      "81245\n",
      "FLT3 (fms-related tyrosine kinase 3) (eg, acute myeloid leukemia), gene analysis; internal tandem duplication (ITD) variants (ie, exons 14, 15)\n",
      "\n",
      "81246\t\n",
      "FLT3 (fms-related tyrosine kinase 3) (eg, acute myeloid leukemia), gene analysis; tyrosine kinase domain (TKD) variants (eg, D835, I836)\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "D47.Z9\n",
      "Other specified neoplasms of uncertain or unknown behavior of lymphoid, hematopoietic, and related tissue\n",
      "\n",
      "Fas-Associated Protein with Death Domain FADD - No specific code:\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C44.02, C44.121 - C44.129, C44.221 - C44.229, C44.320 - C44.329, C44.42\n",
      "Squamous cell carcinoma of lip, eyelid, ear and external canal, face, scalp and neck\n",
      "\n",
      "Prostate PX, Post-op PX:\n",
      "\n",
      "Other CPT codes related to the CPB:\n",
      "\n",
      "88305\n",
      "Level IV - Surgical pathology, gross and microscopic examination\n",
      "\n",
      "88313\n",
      "Special stain including interpretation and report; Group II, al other (eg, iron trichrome), except stain for microorganisms, stains for enzyme constituents, or immunocytochemistry and immunohistochemistry\n",
      "\n",
      "88323\n",
      "Consultation and report on referred material requiring preparation of slides\n",
      "\n",
      "88341 - 88344\n",
      "Immunohistochemistry or immunocytochemistry, per specimen\n",
      "\n",
      "88350\n",
      "Immunofluorescence, per specimen; each additional single antibody stain procedure (List separately in addition to code for primary procedure) \n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C61\n",
      "Malignant neoplasm of prostate\n",
      "\n",
      "NRAS mutation:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "81311\n",
      "NRAS (neuroblastoma RAS viral [v-ras] oncogene homolog) (eg, colorectal carcinoma), gene analysis, variants in exon 2 (eg, codons 12 and 13) and exon 3 (eg, codon 61) \n",
      "\n",
      "Other CPT codes related to the CPB:\n",
      "\n",
      "81404 \n",
      "Molecular pathology procedure, Level 5 (eg, analysis of 2-5 exons by DNA sequence analysis, mutation scanning or duplication/deletion variants of 6-10 exons, or characterization of a dynamic mutation disorder/triplet repeat by Southern blot analysis) \n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C18.0 - C20\n",
      "Malignant neoplasm of colon, rectosigmoid junction and rectum\n",
      "\n",
      "C86.4\t\n",
      "Blastic NK-cell lymphoma [blastic plasmacytoid dendritic cell neoplasm (BPDCN)]\n",
      "\n",
      "D46.0 - D46.9\n",
      "Myelodysplastic syndromes\n",
      "\n",
      "NTRK:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "81194\n",
      "NTRK (neurotrophic-tropomyosin receptor tyrosine kinase 1, 2, and 3) (eg, solid tumors) translocation analysis\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C11.0 - C88.4\n",
      "Malignant neoplasms - solid tumors\n",
      "\n",
      "Ras oncogenes (except KRAS and BRAF) - No specific code:\n",
      "\n",
      "Epidermal growth factor receptor (EGFR) Testing:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "81235\n",
      "EGFR (epidermal growth factor receptor) (eg, non-small cell lung cancer) gene analysis, common variants (eg, exon 19 LREA deletion, L858R, T790M, G719A, G719S, L861Q) \n",
      "\n",
      "Other CPT codes related to the CPB:\n",
      "\n",
      "88341 - 88344\n",
      "Immunohistochemistry or immunocytochemistry, per specimen\n",
      "\n",
      "88381\n",
      "Microdissection (ie, sample preparation of microscopically identified target); manual\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C34.00 - C34.92\n",
      "Malignant neoplasm of bronchus and lung [non small cell lung cancer]\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB  :\n",
      "\n",
      "C71.0 - C71.9\n",
      "Malignant neoplasm of brain [Glioblastoma multiforme]\n",
      "\n",
      "D09.0\n",
      "Carcinoma in situ of bladder [urothelial carcinoma]\n",
      "\n",
      "D09.10 - D09.19\n",
      "Carcinoma in situ of other and unspecified urinary organs (ureter, renal pelvis) [urothelial carcinoma]\n",
      "\n",
      "ROS-1 - No specific code:\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C34.00 - C34.92\n",
      "Malignant neoplasm of bronchus and lung [non small cell lung cancer]\n",
      "\n",
      "ZAP-70:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "88184\n",
      "Flow cytometry, cell surface, cytoplasmic, or nuclear marker, technical component only; first marker\n",
      "\n",
      "+ 88185\n",
      "    each additional marker (List separately in addition to code for first marker)\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C91.10 - C91.12\n",
      "Chronic lymphocytic leukemia of B-cell type  [assessing prognosis and need for aggressive therapy]\n",
      "\n",
      "ZRSR2 test:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "81360\n",
      "ZRSR2 (zinc finger CCCH-type, RNA binding motif and serine/arginine-rich 2) (eg, myelodysplastic syndrome, acute myeloid leukemia) gene analysis, common variant(s) (eg, E65fs, E122fs, R448fs)\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C92.10 â C92.12\n",
      "Chronic myeloid leukemia, BCR/ABL-positive\n",
      "\n",
      "C92.20 - C92.22\n",
      "Atypical chronic myeloid leukemia, BCR/ABL-negative\n",
      "\n",
      "D46.0 â D46.Z\n",
      "Myelodysplastic syndromes\n",
      "\n",
      "Oncotype Dx:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "0047U\n",
      "Oncology (prostate), mRNA, gene expression profiling by real-time RT-PCR of 17 genes (12 content and 5 housekeeping), utilizing formalin-fixed paraffin-embedded tissue, algorithm reported as a risk score\n",
      "\n",
      "81518\n",
      "Oncology (breast), mRNA, gene expression profiling by real-time RT-PCR of 11 genes (7 content and 4 housekeeping), utilizing formalin-fixed paraffin- embedded tissue, algorithms reported as percentage risk for metastatic recurrence and likelihood of benefit from extended endocrine therapy \n",
      "\n",
      "81519\n",
      "Oncology (breast), mRNA, gene expression profiling by real-time RT-PCR of 21 genes, utilizing formalin-fixed paraffin embedded tissue, algorithm reported as recurrence score \n",
      "\n",
      "CPT codes not covered for indications listed in the CPB:\n",
      "\n",
      "81525\n",
      "Oncology (colon), mRNA, gene expression profiling by real-time RT-PCR of 12 genes (7 content and 5 housekeeping), utilizing formalin-fixed paraffin-embedded tissue, algorithm reported as a recurrence score \n",
      "\n",
      "Other CPT codes related to the CPB:\n",
      "\n",
      "88360\n",
      "Morphometric analysis, tumor immunohistochemistry (eg, Her-2/neu, estrogen receptor/progesterone receptor), quantitative or semiquantitative, per specimen, each single antibody stain procedure; manual  \n",
      "\n",
      "88361\n",
      "    using computer-assisted technology\n",
      "\n",
      "88367 - 88377\n",
      "Morphometric analysis, in situ hybridization, (quantitative or semi-quantitative) \n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C50.011 - C50.019C50.111 - C50.119C50.211 - C50.219C50.311 - C50.319C50.411 - C50.419C50.511 - C50.519C50.611 - C50.619C50.811 - C50.819C50.911 - C50.919\n",
      "Malignant neoplasm of female breast [except node positive] [HER2-negative, estrogen-receptor positive, node-negative breast cancer]\n",
      "\n",
      "C50.021 - C50.029C50.121 - C50.129C50.221 - C50.229C50.321 - C50.329C50.421 - C50.429C50.521 - C50.529C50.621 - C50.629C50.821 - C50.829C50.921 - C50.929\n",
      "Malignant neoplasm of male breast\n",
      "\n",
      "C61\n",
      "Malignant neoplasm of prostate [not covered for repeat testing to assess risk of prostate cancer progression]\n",
      "\n",
      "C77.3\n",
      "Secondary and unspecified malignant neoplasm of axilla and upper limb lymph nodes [1-3 involved ipsilateral axillary lymph nodes]\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C18.0 - C20\n",
      "Malignant neoplasm of colon, rectosigmoid junction and rectum\n",
      "\n",
      "D01.0\n",
      "Carcinoma in situ of colon\n",
      "\n",
      "D05.10 - D05.12\n",
      "Intraductal carcinoma in situ of breast \n",
      "\n",
      "D07.5\n",
      "Carcinoma in situ of prostate\n",
      "\n",
      "Z85.030 - Z85.048\n",
      "Personal history of malignant neoplasm of large intestine, rectum, rectosigmoid junction, and anus\n",
      "\n",
      "Myeloperoxidase (MPO) immunostaining FLT3-ITD, CEBPA mutation, NPM1 mutation and KIT mutation:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "0046U\n",
      "FLT3 (fms-related tyrosine kinase 3) (eg, acute myeloid leukemia) internal tandem duplication (ITD) variants, quantitative\n",
      "\n",
      "0049U\n",
      "NPM1 (nucleophosmin) (eg, acute myeloid leukemia) gene analysis, quantitative\n",
      "\n",
      "81245 - 81246\n",
      "FLT3 (fms-related tyrosine kinase 3) (eg, acute myeloid leukemia), gene analysis\n",
      "\n",
      "83876\n",
      "Myeloperoxidase (MPO)\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C92.00 - C92.02C92.40 - C92.a2\n",
      "Acute myeloid leukemia\n",
      "\n",
      "NPM1:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "0049U\n",
      "NPM1 (nucleophosmin) (eg, acute myeloid leukemia) gene analysis, quantitative\n",
      "\n",
      "81310\n",
      "NPM1 (nucleophosmin) (eg, acute myeloid leukemia) gene analysis, exon 12 variants\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C92.00 - C92.02, C92.40 - C92.A2\n",
      "Acute myeloid leukemia\n",
      "\n",
      "C92.10 â C92.12\n",
      "Chronic myeloid leukemia, BCR/ABL-positive\n",
      "\n",
      "C92.20 - C92.22\n",
      "Atypical chronic myeloid leukemia, BCR/ABL-negative\n",
      "\n",
      "C94.6\n",
      "Myelodysplastic disease, not elsewhere classified [myeloproliferative neoplasms]\n",
      "\n",
      "D45\n",
      "Polycythemia vera \n",
      "\n",
      "D46.0 - D46.Z\n",
      "Myelodysplastic syndromes\n",
      "\n",
      "D47.3\n",
      "Essential (hemorrhagic) thrombocythemia \n",
      "\n",
      "D75.81\n",
      "Myelofibrosis \n",
      "\n",
      "PDGFRA:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "81314\n",
      "PDGFRA (platelet-derived growth factor receptor, alpha polypeptide) (eg, gastrointestinal stromal tumor [GIST]), gene analysis, targeted sequence analysis (eg, exons 12, 18)\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C49.4\n",
      "Malignant neoplasm of connective and soft tissue of abdomen\n",
      "\n",
      "C91.00 - C91.02\n",
      "Acute lymphoblastic leukemia (ALL)  [pediatric]\n",
      "\n",
      "D47.02\n",
      "Systemic mastocytosis\n",
      "\n",
      "D47.Z9\n",
      "Other specified neoplasms of uncertain or unknown behavior of lymphoid, hematopoietic, and related tissue\n",
      "\n",
      "PML/RARA:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "81315 - 81316\n",
      "PML/RARalpha, (t(15;17)), (promyelocytic leukemia/retinoic acid receptor alpha) (eg, promyelocytic leukemia) translocation analysis\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C92.00 - C92.02\n",
      "Acute myeloblastic leukemia\n",
      "\n",
      "Prolaris:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "81541\n",
      "Oncology (prostate), mRNA gene expression profiling by real-time RT-PCR of 46 genes (31 content and 15 housekeeping), utilizing formalin-fixed paraffin-embedded tissue, algorithm reported as a disease-specific mortality risk score\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C61\n",
      "Malignant neoplasm of prostate [not covered for repeat testing to assess risk of prostate cancer progression]\n",
      "\n",
      "ProMark:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "ProMark - no specific code:\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C61\n",
      "Malignant neoplasm of prostate [not covered for repeat testing to assess risk of prostate cancer progression]\n",
      "\n",
      "Placental alkaline phosphatase (PLAP):\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "84080\n",
      "Phosphatase, alkaline; isoenzymes\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C56.1 - C56.9\n",
      "Malignant neoplasm of ovary\n",
      "\n",
      "C62.00 - C62.92\n",
      "Malignant neoplasm of testes\n",
      "\n",
      "D07.30 - D07.39\n",
      "Carcinoma in situ of other and unspecified female genital organs [germ cell tumors]\n",
      "\n",
      "D07.60 - D07.69\n",
      "Carcinoma in situ of other and unspecified male genital organs\n",
      "\n",
      "Z85.43\n",
      "Personal history of malignant neoplasm of ovary\n",
      "\n",
      "Z85.47\n",
      "Personal history of malignant neoplasm of testis\n",
      "\n",
      "Bladder tumor antigen (BTA) Stat Test, the nuclear matrix protein (NMP22) test, the fibrin/fibrinogen degradation products (Aura-Tek FDP) test, Pathnostics Bladder FISH test or the UroVysion fluorescent in situ hybridization (FISH) test, BTA TRAK:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "85362 - 85380\n",
      "Fibrin degradation products\n",
      "\n",
      "86294\n",
      "Immunoassay for tumor antigen, qualitative or semiquantitative (e.g., bladder tumor antigen)\n",
      "\n",
      "86386\n",
      "Nuclear Matrix Protein 22 (NMP22) qualitative \n",
      "\n",
      "88120\n",
      "Cytopathology, in situ hybridization (eg, FISH), urinary tract specimen with morphometric analysis, 3-5 molecular probes, each specimen; manual\n",
      "\n",
      "88121\n",
      "    using computer-assisted technology\n",
      "\n",
      "88364 - 88366\n",
      "In situ hybridization (eg, FISH), each probe\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C67.0 - C67.9\n",
      "Malignant neoplasm of bladder\n",
      "\n",
      "D09.0\n",
      "Carcinoma in situ of bladder\n",
      "\n",
      "Z85.51\n",
      "Personal history of malignant neoplasm of bladder\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB (not all-inclusive):\n",
      "\n",
      "R31.0 - R31.9\n",
      "Hematuria\n",
      "\n",
      "Z12.6\n",
      "Encounter for screening for malignant neoplasm of bladder\n",
      "\n",
      "ImmunoCyte (uCyt) - No specific code:\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C67.0 - C67.9\n",
      "Malignant neoplasm of bladder\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "R31.0 - R31.9\n",
      "Hematuria\n",
      "\n",
      "Z12.6\n",
      "Encounter for screening for malignant neoplasm of bladder [diagnosis or screening in asymptomatic persons]\n",
      "\n",
      "Janus Kinase 2 (JAK2) mutations:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "0027U\n",
      "JAK2 (Janus kinase 2) (eg, myeloproliferative disorder) gene analysis, targeted sequence analysis exons 12-15\n",
      "\n",
      "81270\n",
      "JAK2 (Janus kinase 2) (eg, myeloproliferative disorder) gene analysis, p.Val617Phe (V617F) variant [not covered for diagnostic assessment of myeloproliferative disorders in children; and quantitative assessment of JAK2-V617F allele burden subsequent to qualitative detection of JAK2-V617F]\n",
      "\n",
      "81279\n",
      "JAK2 (Janus kinase 2) (eg, myeloproliferative disorder) targeted sequence analysis (eg, exons 12 and 13)\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C92.10 - C92.12\n",
      "Chronic myeloid leukemia, BCR/ABL-positive\n",
      "\n",
      "D45\n",
      "Polycythemia vera \n",
      "\n",
      "D47.1\n",
      "Chronic myeloproliferative disease \n",
      "\n",
      "D47.3\n",
      "Essential (hemorrhagic) thrombocythemia\n",
      "\n",
      "D47.4\n",
      "Osteomyelofibrosis \n",
      "\n",
      "D75.81\n",
      "Myelofibrosis \n",
      "\n",
      "KRAS:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "81275\n",
      "KRAS (Kirsten rat sarcoma viral oncogene homolog) (eg, carcinoma) gene analysis; variants in exon 2 (eg, codons 12 and 13)\n",
      "\n",
      "81276\n",
      "KRAS (Kirsten rat sarcoma viral oncogene homolog) (eg, carcinoma) gene analysis; additional variant(s) (eg, codon 61, codon 146)\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C18.0 - C20\n",
      "Malignant neoplasm of colon, rectosigmoid junction and rectum\n",
      "\n",
      "C21.8\n",
      "Malignant neoplasm of overlapping sites of rectum, anus and anal canal\n",
      "\n",
      "C25.0 - C25.9\n",
      "Malignant neoplasm of pancreas\n",
      "\n",
      "C34.00 - C34.92\t\n",
      "Malignant neoplasm of bronchus and lung [non-small cell]\n",
      "\n",
      "C53.0 - C55, C58\n",
      "Malignant neoplasm of uterus [sarcoma]\n",
      "\n",
      "D46.0 - D46.9\t\n",
      "Myelodysplastic syndromes\n",
      "\n",
      "BRAF, V600 mutation analysis:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "81210\n",
      "BRAF (v-raf murine sarcoma viral oncogene homolog B1) (eg, colon cancer), gene analysis, V600E variant\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C18.0 - C21.8\n",
      "Malignant neoplasm of colon, rectosigmoid junction, rectum, anus and anal canal\n",
      "\n",
      "C25.0 - C25.9\n",
      "Malignant neoplasm of pancreas\n",
      "\n",
      "C34.00 - C34.92\n",
      "Malignant neoplasm of bronchus and lung\n",
      "\n",
      "C43.0 - C43.9\n",
      "Melanoma of skin [for consideration of Vemurafenib, Dabrafenib and Trametinib]\n",
      "\n",
      "C49.4\n",
      "Malignant neoplasm of connective and soft tissue of abdomen [gastrointestinal stromal tumors ] \n",
      "\n",
      "C71.0 - C71.9\t\n",
      "Malignant neoplasm of brain [infiltrative glioma]\t\n",
      "\n",
      "C73\n",
      "Malignant neoplasm of thyroid gland\n",
      "\n",
      "C91.40 - C91.42\n",
      "Hairy cell leukemia \n",
      "\n",
      "D44.0\n",
      "Neoplasm of uncertain behavior of thyroid gland [indeterminate thyroid nodules]\n",
      "\n",
      "Assaying for loss of heterozygosity (LOH) on the long arm of chromosome 18 (18q) or deleted in colon cancer (DCC) protein (18q-LOH/DCC) for colorectal cancer:\n",
      "\n",
      "No specific code \n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C18.0 - C20\n",
      "Malignant neoplasm of colon, rectum, and rectosigmoid junction \n",
      "\n",
      "Biodesix BDX-XL2, Nodify XL2, Nodify Lung, Nodify CDT:\n",
      "\n",
      "CPT codes not covered for indications listed in the CPB:\n",
      "\n",
      "Nodify Lung & Nodify CDT â no specific code\n",
      "\n",
      "0080U\n",
      "Oncology (lung), mass spectrometric analysis of galectin-3-binding protein and scavenger receptor cysteine-rich type 1 protein M130, with five clinical risk factors (age, smoking status, nodule diameter, nodule-spiculation status and nodule location), utilizing plasma, algorithm reported as a categorical probability of malignancy\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C34.00 - C34.92\n",
      "Malignant neoplasm of bronchus and lung\n",
      "\n",
      "C78.00 - C78.02\n",
      "Secondary malignant neoplasm of lung\n",
      "\n",
      "D14.30 - D14.32\n",
      "Benign neoplasm of bronchus and lung\n",
      "\n",
      "R91.1\n",
      "Solitary pulmonary nodule\n",
      "\n",
      "R91.8\n",
      "Other nonspecific abnormal finding of lung field\n",
      "\n",
      "OvaCheck test:\n",
      "\n",
      "No specific code \n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C56.1 - C56.9\n",
      "Malignant neoplasm of ovary\n",
      "\n",
      "Z12.73\n",
      "Encounter for screening for malignant neoplasm of ovary\n",
      "\n",
      "Ovasure - No specific code:\n",
      "\n",
      "Other CPT codes related to the CPB:\n",
      "\n",
      "82985\n",
      "Glycated protein\n",
      "\n",
      "83520\n",
      "Immunoassay, analyte quantitative; not otherwise specified\n",
      "\n",
      "84146\n",
      "Prolactin\n",
      "\n",
      "84305\n",
      "Somatomedin\n",
      "\n",
      "86304\n",
      "Immunoassay for tumor antigen, quantitative; CA 125\n",
      "\n",
      "Circulating cell-free nucleic acids - No specific code:\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C18.0 - C20\n",
      "Malignant neoplasm of colon, rectosigmoid junction, and rectum\n",
      "\n",
      "Circulating tumor cell (CTC) (e.g., CELLSEARCH tests):\n",
      "\n",
      "CPT codes not covered for indications listed in the CPB:\n",
      "\n",
      "0091U\n",
      "Oncology (colorectal) screening, cell enumeration of circulating tumor cells, utilizing whole blood, algorithm, for the presence of adenoma or cancer, reported as a positive or negative result\n",
      "\n",
      "0337U\n",
      "Oncology (plasma cell disorders and myeloma), circulating plasma cell immunologic selection, identification, morphological characterization, and enumeration of plasma cells based on differential CD138, CD38, CD19, and CD45 protein biomarker expression, peripheral blood\n",
      "\n",
      "0338U\n",
      "Oncology (solid tumor), circulating tumor cell selection, identification, morphological characterization, detection and enumeration based on differential EpCAM, cytokeratins 8, 18, and 19, and CD45 protein biomarkers, and quantification of HER2 protein biomarkerâexpressing cells, peripheral blood\n",
      "\n",
      "86152\n",
      "Cell enumeration using immunologic selection and identification in fluid specimen (eg, circulating tumor cells in blood)\n",
      "\n",
      "86153\n",
      "    physician interpretation and report, when required\n",
      "\n",
      "88346\n",
      "Immunofluorescence, per specimen; initial single antibody stain procedure  \n",
      "\n",
      "88361\n",
      "Morphometric analysis, tumor immunohistochemistry (e.g., Her-2/neu, estrogen receptor/progesterone receptor), quantitative or semiquantitative, each antibody; using computer-assisted technology\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB (not all-inclusive):\n",
      "\n",
      "C18.0 - C20\n",
      "Malignant neoplasm of colon, rectosigmoid junction, and rectum\n",
      "\n",
      "C43.0 - C43.9\n",
      "Malignant melanoma of skin\n",
      "\n",
      "C50.011 - C50.929\n",
      "Malignant neoplasm of breast\n",
      "\n",
      "C61\n",
      "Malignant neoplasm of prostate\n",
      "\n",
      "C79.81\n",
      "Secondary malignant neoplasm of breast\n",
      "\n",
      "Cofilin (CFL1) - no specific code:\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C34.00 - C34.92\n",
      "Malignant neoplasm of bronchus and lung [non-small-cell lung cancer] \n",
      "\n",
      "ColonSentry - no specific code:\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "Z12.11 - Z12.12\n",
      "Encounter for screening for malignant neoplasm of colon and rectum \n",
      "\n",
      "ColoScape Test:\n",
      "\n",
      "CPT codes not covered for indications listed in the CPB:\n",
      "\n",
      "0368U\n",
      "Oncology (colorectal cancer), evaluation for mutations of APC, BRAF, CTNNB1, KRAS, NRAS, PIK3CA, SMAD4, and TP53, and methylation markers (MYO1G, KCNQ5, C9ORF50, FLI1, CLIP4, ZNF132 and TWIST1), multiplex quantitative polymerase chain reaction (qPCR), circulating cell-free DNA (cfDNA), plasma, report of risk score for advanced adenoma or colorectal cancer\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C18.0 - C18.9\n",
      "Malignant neoplasm of colon\n",
      "\n",
      "C19\n",
      "Malignant neoplasm of rectosigmoid junction\n",
      "\n",
      "C20\n",
      "Malignant neoplasm of rectum\n",
      "\n",
      "Oncotype DXÂ® Breast DCIS Score™ Test:\n",
      "\n",
      "CPT codes not covered for indictions listed in the CPB:\n",
      "\n",
      "0045U\n",
      "Oncology (breast ductal carcinoma in situ), mRNA, gene expression profiling by real-time RT-PCR of 12 genes (7 content and 5 housekeeping), utilizing formalin-fixed paraffin-embedded tissue, algorithm reported as recurrence score\n",
      "\n",
      "Early CDT-Lung Test:\n",
      "\n",
      "CPT codes not covered for indications listed in the CPB:\n",
      "\n",
      "83520\n",
      "Immunoassay for analyte other than infectious agent antibody or infectious agent antigen; quantitative, not otherwise specified [as a screening for the early detection of lung cancer]\n",
      "\n",
      "Galectin-3:\n",
      "\n",
      "CPT codes not covered for indications listed in the CPB:\n",
      "\n",
      "82777\n",
      "Galectin-3\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB (not all-inclusive):\n",
      "\n",
      "C25.0 - C25.9\n",
      "Malignant neoplasm of pancreas\n",
      "\n",
      "C40.00 - C41.9\n",
      "Malignant neoplasm of bone and articular cartilage [osteosarcoma]\n",
      "\n",
      "C50.011 - C50.929 \n",
      "Malignant neoplasm of breast\n",
      "\n",
      "C56.1 - C56.9\n",
      "Malignant neoplasm of ovary\n",
      "\n",
      "C61\n",
      "Malignant neoplasm of prostate\n",
      "\n",
      "D46.0 - D46.Z\n",
      "Myelodysplastic syndromes\n",
      "\n",
      "Insight TNBCtype:\n",
      "\n",
      "CPT codes not covered for indications listed in the CPB:\n",
      "\n",
      "0153U\n",
      "Oncology (breast), mRNA, gene expression profiling by next-generation sequencing of 101 genes, utilizing formalin-fixed paraffin-embedded tissue, algorithm reported as a triple negative breast cancer clinical subtype(s) with information on immune cell involvement\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C50.011 - C50.929\n",
      "Malignant neoplasm of breast\n",
      "\n",
      "Ki67 :\n",
      "\n",
      "CPT codes not covered for indications listed in the CPB:\n",
      "\n",
      "88360\n",
      "Morphometric analysis, tumor immunohistochemistry (eg, Her-2/neu, estrogen receptor/progesterone receptor), quantitative or semiquantitative, per specimen, each single antibody stain procedure; manual \n",
      "\n",
      "88361\n",
      "    using computer-assisted technology\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C50.011 - C50.929\n",
      "Malignant neoplasm of breast\n",
      "\n",
      "C64.1 0 C66.9\n",
      "Malignant neoplasm of kidney, renal pelvis, and ureter\n",
      "\n",
      "Breast cancer index:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "81518\n",
      "Oncology (breast), mRNA, gene expression profiling by real-time RT-PCR of 11 genes (7 content and 4 housekeeping), utilizing formalin-fixed paraffin-embedded tissue, algorithms reported as percentage risk for metastatic recurrence and likelihood of benefit from extended endocrine therapy\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met :\n",
      "\n",
      "C50.011 - C50.929\n",
      "Malignant neoplasm of breast\n",
      "\n",
      "BTK:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "81233\n",
      "BTK (Bruton's tyrosine kinase) (eg, chronic lymphocytic leukemia) gene analysis, common variants (eg, C481S, C481R, C481F)\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C83.00 - C83.09\t  \n",
      "Small cell B cell lymphoma\n",
      "\n",
      "C91.10 - C91.12\n",
      "Chronic lymphocytic leukemia of B-cell type\n",
      "\n",
      "Mammaprint:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "81521\n",
      "Oncology (breast), mRNA, microarray gene expression profiling of 70 content genes and 465 housekeeping genes, utilizing fresh frozen or formalin-fixed paraffin-embedded tissue, algorithm reported as index related to risk of distant metastasis\n",
      "\n",
      "81523\n",
      "Oncology (breast), mRNA, next-generation sequencing gene expression profiling of 70 content genes and 31 housekeeping genes, utilizing formalin-fixed paraffin-embedded tissue, algorithm reported as index related to risk to distant metastasis\n",
      "\n",
      "HCPCS codes covered if selection criteria are met:\n",
      "\n",
      "S3854\n",
      "Gene expression profiling panel for use in the management of breast cancer treatment\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C50.011 - C50.929\n",
      "Malignant neoplasm of breast\n",
      "\n",
      "Z17.0\n",
      "Estrogen receptor positive status [ER+]\n",
      "\n",
      "Z17.1\n",
      "Estrogen receptor negative status [ER-]\n",
      "\n",
      "Lymph2CX and Lymph3Cx:\n",
      "\n",
      "CPT codes not covered for indications listed in the CPB:\n",
      "\n",
      "0017U\n",
      "Oncology (hematolymphoid neoplasia), JAK2 mutation, DNA, PCR amplification of exons 12-14 and sequence analysis, blood or bone marrow, report of JAK2 mutation not detected or detected\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB (not all-inclusive):\n",
      "\n",
      "C85.20 - C85.29\t\n",
      "Mediastinal (thymic) large B-cell lymphoma\n",
      "\n",
      "C83.30 - C83.39\n",
      "Diffuse large B-cell lymphoma\n",
      "\n",
      "Mucin 4 expression:\n",
      "\n",
      "CPT codes not covered for indications listed in the CPB:\n",
      "\n",
      "88313\n",
      "Group II, all other (eg, iron, trichrome), except immunocytochemistry and immunoperoxidase stains, including interpretation and report, each\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C18.0 - C20\n",
      "Malignant neoplasm of colon, rectum and rectosigmoid junction\n",
      "\n",
      "Mucin 5AC (MUC5AC) - No specific code:\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C22.1\n",
      "Intrahepatic bile duct carcinoma\n",
      "\n",
      "C24.0 - C24.9\n",
      "Malignant neoplasm of other and unspecified parts of biliary tract\n",
      "\n",
      "NF1, RET, and SDHB:\n",
      "\n",
      "CPT codes not covered for indications listed in the CPB:\n",
      "\n",
      "NF1, RET, and SDHB - no specific code:\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C56.1 - C56.9\n",
      "Malignant neoplasm of ovary\n",
      "\n",
      "Microarray-based gene expression profile testing:\n",
      "\n",
      "Other CPT codes related to the CPB:\n",
      "\n",
      "81406\n",
      "Molecular pathology procedure, Level 7 (eg, analysis of 11-25 exons by DNA sequence analysis, mutation scanning or duplication/deletion variants of 26-50 exons, cytogenomic array analysis for neoplasia)  \n",
      "\n",
      "OVA1:\n",
      "\n",
      "CPT codes not covered for indications listed in the CPB:\n",
      "\n",
      "0003U\n",
      "Oncology (ovarian) biochemical assays of five proteins (apolipoprotein A-1, CA 125 II, follicle stimulating hormone, human epididymis protein 4, transferrin), utilizing serum, algorithm reported as a likelihood score\n",
      "\n",
      "81503\n",
      "Oncology (ovarian), biochemical assays of five proteins (CA-125, apolipoprotein A1, beta-2 microglobulin, transferrin and pre-albumin), utilizing serum, algorithm reported as a risk score \n",
      "\n",
      "p16 protein expression - No specific code:\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C00.0 - C14.8\n",
      "Malignant neoplasms of lip, oral cavity and pharynx [non-oropharyngeal squamous cell carcinoma]\n",
      "\n",
      "Pathwork Tissue of Origin Test:\n",
      "\n",
      "CPT codes not covered for indications listed in the CPB:\n",
      "\n",
      "81504\n",
      "Oncology (tissue of origin), microarray gene expression profiling of > 2000 genes, utilizing formalin-fixed paraffin-embedded tissue, algorithm reported as tissue similarity scores \n",
      "\n",
      "PreOvar Test for the KRAS-variant [to determine ovarian cancer risk]:\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C56.1 - C56.9\n",
      "Malignant neoplasm of ovary\n",
      "\n",
      "Z85.43\n",
      "Personal history of malignant neoplasm of ovary\n",
      "\n",
      "ProOnc Tumor Source Dx Test - No specific code:\n",
      "\n",
      "ROMA:\n",
      "\n",
      "CPT codes not covered for indications listed in the CPB:\n",
      "\n",
      "81500\n",
      "Oncology (ovarian), biochemical assays of two proteins (CA-125 and HE4), utilizing serum, with menopausal status, algorithm reported as a risk score\n",
      "\n",
      "86304\n",
      "Immunoassay for tumor antigen, quantitative; CA 125\n",
      "\n",
      "86305\n",
      "Human epididymis protein 4 (HE4)\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB (not all-inclusive):\n",
      "\n",
      "C56.1 - C56.9\n",
      "Malignant neoplasm of ovary\n",
      "\n",
      "Rotterdam Signature 76-gene Panel:\n",
      "\n",
      "HCPCS codes not covered for indications listed in the CPB:\n",
      "\n",
      "S3854\n",
      "Gene expression profiling panel for use in the management of breast cancer treatment \n",
      "\n",
      "Serum amyloid A:\n",
      "\n",
      "CPT codes not covered for indications listed in the CPB:\n",
      "\n",
      "88342\n",
      "Immunohistochemistry or immunocytochemistry, each separately identifiable antibody per block, cytologic preparation, or hematologic smear; first separately identifiable antibody per slide \n",
      "\n",
      "Other CPT codes related to the CPB :\n",
      "\n",
      "88341 - 88344\n",
      "Immunohistochemistry or immunocytochemistry, per specimen\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C54.0 - C54.8\n",
      "Malignant neoplasm of corpus uteri, isthmus and uterus\n",
      "\n",
      "Z85.42\n",
      "Personal history of malignant neoplasm of uterus\n",
      "\n",
      "Breast Cancer Gene Expression Ratio (HOXB13:IL17BR):\n",
      "\n",
      "HCPCS codes not covered for indications listed in the CPB:\n",
      "\n",
      "S3854\n",
      "Gene expression profiling panel for use in the management of breast cancer treatment \n",
      "\n",
      "PAM50 ROR (Prosigna Breast Cancer Prognostic Gene Signature Assay) :\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "0008M\n",
      "Oncology (breast), MRNA analysis of 58 genes using hybrid capture, on formalin-fixed paraffin-embedded (FFPE) tissue, prognostic algorithm reported as a risk score [Prosigna] \n",
      "\n",
      "81520\n",
      "Oncology (breast), mRNA gene expression profiling by hybrid capture of 58 genes (50 content and 8 housekeeping), utilizing formalin-fixed paraffin-embedded tissue, algorithm reported as a recurrence risk score\n",
      "\n",
      "CPT codes not covered for indications listed in the CPB:\n",
      "\n",
      "81406\n",
      "Molecular pathology procedure, Level 7 (eg, analysis of 11-25 exons by DNA sequence analysis, mutation scanning or duplication/deletion variants of 26-50 exons, cytogenomic array analysis for neoplasia) [when specified as the following]:  PALB2 (partner and localizer of BRCA2) (eg, breast and pancreatic cancer), full gene sequence\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C50.011 - C50.929\n",
      "Malignant neoplasm of breast\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C61\n",
      "Malignant neoplasm of prostate \n",
      "\n",
      "PTEN:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "81321 - 81323\n",
      "PTEN (phosphatase and tensin homolog) (eg, Cowden syndrome, PTEN hamartoma tumor syndrome) gene analysis\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met :\n",
      "\n",
      "C53.0 - C55, C58\n",
      "Malignant neoplasm of uterus\n",
      "\n",
      "Q85.81 - Q85.89\n",
      "Other phakomatoses, not elsewhere classified [Cowden syndrome]\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C34.00 - C34.92\n",
      "Malignant neoplasm of bronchus and lung [non-small cell lung cancer]\n",
      "\n",
      "GeneSearch Breast Lymph Node (BLN) assay - No specific code:\n",
      "\n",
      "Thymidylate synthase - No specific code:\n",
      "\n",
      "No specific code \n",
      "\n",
      "Other CPT codes related to the CPB:\n",
      "\n",
      "88341 - 88344\n",
      "Immunohistochemistry or immunocytochemistry, per specimen\n",
      "\n",
      "88360\n",
      "Morphometric analysis, tumor immunohistochemistry (eg, Her-2/neu, estrogen receptor/progesterone receptor), quantitative or semiquantitative, per specimen, each single antibody stain procedure; manual \n",
      "\n",
      "88361\n",
      "    using computer-assisted technology\n",
      "\n",
      "Topographic genotyping (PathfinderTG) - No specific code:\n",
      "\n",
      "Biomarker Translation (BT) - No specific code:\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C50.011 - C50.929\n",
      "Malignant neoplasm of breast\n",
      "\n",
      "BRAF and EGFR:\n",
      "\n",
      "CPT codes not covered for indications listed in the CPB:\n",
      "\n",
      "81210\n",
      "BRAF (B-Raf proto-oncogene, serine/threonine kinase) (eg, colon cancer, melanoma), gene analysis, V600 variant(s)\n",
      "\n",
      "81235\n",
      "EGFR (epidermal growth factor receptor) (eg, non-small cell lung cancer) gene analysis, common variants (eg, exon 19 LREA deletion, L858R, T790M, G719A, G719S, L861Q)\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C15.3 - C15.9\n",
      "Malignant neoplasm of esophagus\n",
      "\n",
      "HE4:\n",
      "\n",
      "CPT codes not covered for indications listed in the CPB:\n",
      "\n",
      "81500\n",
      "Oncology (ovarian), biochemical assays of two proteins (CA-125 and HE4), utilizing serum, with menopausal status, algorithm reported as a risk score \n",
      "\n",
      "86305\n",
      "Human epididymis protein 4 (HE4)\n",
      "\n",
      "Other CPT codes related to the CPB:\n",
      "\n",
      "86316\n",
      "Immunoassay for tumor antigen; other antigen, quantitative (e.g., CA 50, 72-4, 549), each\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C54.1\n",
      "Malignant neoplasm of endometrium\n",
      "\n",
      "C56.1 - C56.9\n",
      "Malignant neoplasm of ovary\n",
      "\n",
      "R19.00\n",
      "Intra-abdominal and pelvic swelling, mass, lump, unspecified site [not covered for evaluation of pelvic mass, including assistance in the determination of referral for surgery to a gynecologic oncologist or general surgery]\n",
      "\n",
      "R19.07 - R19.09\n",
      "Generalized and other intra-abdominal and pelvic swelling, mass and lump [not covered for evaluation of pelvic mass, including assistance in the determination of referral for surgery to a gynecologic oncologist or general surgery]\n",
      "\n",
      "HERmark - No specific code:\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C50.011 - C50.929\n",
      "Malignant neoplasm of breast\n",
      "\n",
      "D05.00 - D05.92\n",
      "Carcinoma in situ of breast\n",
      "\n",
      "TargetPrint Gene Expression:\n",
      "\n",
      "Other CPT codes related to the CPB:\n",
      "\n",
      "88360\n",
      "Morphometric analysis, tumor immunohistochemistry (eg, Her-2/neu, estrogen receptor/progesterone receptor), quantitative or semiquantitative, per specimen, each single antibody stain procedure; manual \n",
      "\n",
      "88361\n",
      "    using computer-assisted technology\n",
      "\n",
      "88367 - 88377\n",
      "Morphometric analysis,in situ hybridization (quantitative or semiquantitative) \n",
      "\n",
      "HCPCS codes not covered for indications listed in the CPB:\n",
      "\n",
      "S3854\n",
      "Gene expression profiling panel for use in the management of breast cancer treatment\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C50.011 - C50.929\n",
      "Malignant neoplasm of breast \n",
      "\n",
      "TP53 - No specific code:\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C56.1 - C56.9\n",
      "Malignant neoplasm of ovary\n",
      "\n",
      "CK5, CK14, p63 and Racemase P504S:\n",
      "\n",
      "Other CPT codes related to the CPB:\n",
      "\n",
      "88341 - 88344\n",
      "Immunohistochemistry or immunocytochemistry, per specimen \n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C61\n",
      "Malignant neoplasm of prostate\n",
      "\n",
      "EML4-ALK:\n",
      "\n",
      "Other CPT codes related to the CPB:\n",
      "\n",
      "88381\n",
      "Microdissection (ie, sample preparation of microscopically identified target); manual\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C34.0 - C34.92\n",
      "Malignant neoplasm of bronchus and lung [non-small-cell lung cancer]\n",
      "\n",
      "Coloprint, CIMP, Line-1 hypomethylation and immune cells - No specific code:\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C18.0 - C20\n",
      "Malignant neoplasm of colon, rectosigmoid junction and rectum\n",
      "\n",
      "ConfirmMDx:\n",
      "\n",
      "CPT codes not covered for indications listed in the CPB:\n",
      "\n",
      "81551\n",
      "Oncology (prostate), promoter methylation profiling by real-time PCR of 3 genes (GSTP1, APC, RASSF1), utilizing formalin-fixed paraffin-embedded tissue, algorithm reported as a likelihood of prostate cancer detection on repeat biopsy\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C61\n",
      "Malignant neoplasm of prostate\n",
      "\n",
      "Des-gamma-carboxyl prothrombin (DCP):\n",
      "\n",
      "CPT codes not covered for indications listed in the CPB:\n",
      "\n",
      "83951\n",
      "Oncoprotein; des-gamma-carboxy-prothrombin (DCP)\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB (not all-inclusive):\n",
      "\n",
      "C22.0 \n",
      "Liver cell carcinoma\n",
      "\n",
      "D01.5\n",
      "Carcinoma in situ of liver and biliary system\n",
      "\n",
      "5-hydroxyindoleacetic acid (5-HIAA):\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "83497\n",
      "Hydroxyindolacetic acid, 5-(HIAA)\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C7A.00 - C7A.8 \n",
      "Malignant neuroendocrine tumors\n",
      "\n",
      "D3A.00 - D3A.8\n",
      "Benign neuroendocrine tumors\n",
      "\n",
      "Beta-2 microglobulin (B2M):\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "82232\n",
      "Beta-2 microglobulin\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C85.10 - C85.99\n",
      "Other specified and unspecified types of non-Hodgkin lymphoma\n",
      "\n",
      "C88.0\n",
      "Waldenstrom macroglobulinemia \n",
      "\n",
      "C90.00 - C90.02\n",
      "Multiple myeloma\n",
      "\n",
      "CALCA (Calcitonin) expression:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "82308\n",
      "Calcitonin\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C73\n",
      "Malignant neoplasm of thyroid\n",
      "\n",
      "C76.0\n",
      "Malignant neoplasm of head, face and neck\n",
      "\n",
      "CALR (calreticulin) expression:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "81219\n",
      "CALR (calreticulin) (eg, myeloproliferative disorders), gene analysis, common variants in exon 9\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C92.10 â C92.12\n",
      "Chronic myeloid leukemia, BCR/ABL-positive\n",
      "\n",
      "C92.20 - C92.22\n",
      "Atypical chronic myeloid leukemia, BCR/ABL-negative\n",
      "\n",
      "C94.6\n",
      "Myelodysplastic disease, not elsewhere classified [myeloproliferative neoplasms]\n",
      "\n",
      "D45\n",
      "Polycythemia vera \n",
      "\n",
      "D46.0 â D46.Z\n",
      "Myelodysplastic syndromes\n",
      "\n",
      "D47.3\n",
      "Essential (hemorrhagic) thrombocythemia \n",
      "\n",
      "D75.81\n",
      "Myelofibrosis \n",
      "\n",
      "CALB2 (Calretinin) expression:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "88342\n",
      "Immunohistochemistry or immunocytochemistry, each separately identifiable antibody per block, cytologic preparation, or hematologic smear; first separately identifiable antibody per slide\n",
      "\n",
      "88341\n",
      "    each additional single antibody stain procedure (List separately in addition to code for primary procedure)\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C34.00 - C34.92\n",
      "Malignant neoplasm of bronchus and lung\n",
      "\n",
      "C80.0 - C80.1\n",
      "Disseminated and other malignant neoplasm, unspecified\n",
      "\n",
      "CHGA (Chromogranin A) expression:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "86316\n",
      "Immunoassay for tumor antigen; other antigen, quantitative (e.g., CA 50, 72-4, 549), each\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C34.00 - C34.92\n",
      "Malignant neoplasm of bronchus and lung \n",
      "\n",
      "C4A.0 - C4A.9\n",
      "Merkel cell carcinoma\n",
      "\n",
      "C7A.00 - C7A.8\n",
      "Malignant neuroendocrine tumors\n",
      "\n",
      "C80.0 - C80.1\n",
      "Disseminated and other malignant neoplasm, unspecified\n",
      "\n",
      "D3A.00 - D3A.8\n",
      "Benign neuroendocrine tumors\n",
      "\n",
      "Beta human chorionic Gonadotropin (beta-hCG):\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "84704\n",
      "Gonadotropin, chorionic (hCG); free beta chain\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C37\n",
      "Malignant neoplasm of thymus\n",
      "\n",
      "C56.1 - C56.9\n",
      "Malignant neoplasm of ovary\n",
      "\n",
      "C62.00 - C62.92\n",
      "Malignant neoplasm of testis\n",
      "\n",
      "D07.39\n",
      "Carcinoma in situ of other female genital organs\n",
      "\n",
      "D07.69\n",
      "Carcinoma in situ of other male genital organs [testis]\n",
      "\n",
      "D15.0\n",
      "Benign neoplasm of thymus\n",
      "\n",
      "D27.0 - D27.9\n",
      "Benign neoplasm of ovary\n",
      "\n",
      "D29.20 - D29.22\n",
      "Benign neoplasm of testis\n",
      "\n",
      "N50.8\n",
      "Other specified disorders of male genital organs [testicular mass]\n",
      "\n",
      "R19.00\n",
      "Intra-abdominal and pelvic swelling, mass, lump, unspecified site\n",
      "\n",
      "R19.07 - R19.09\n",
      "Generalized and other intra-abdominal and pelvic swelling, mass and lump\n",
      "\n",
      "R22.2\n",
      "Localized swelling, mass and lump, trunk\n",
      "\n",
      "Isocitrate dehydrogenase 1 and 2 (IDH1, IDH2):\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "81120\n",
      "IDH1 (isocitrate dehydrogenase 1 [NADP+], soluble) (eg, glioma), common variants (eg, R132H, R132C)\n",
      "\n",
      "81121\n",
      "IDH2 (isocitrate dehydrogenase 2 [NADP+], mitochondrial) (eg, glioma), common variants (eg, R140W, R172M)\n",
      "\n",
      "83570\n",
      "Isocitric dehydrogenase (IDH)\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C40.00 - C41.9\n",
      "Malignant neoplasm of bone and articular cartilage [chondrosarcoma] \n",
      "\n",
      "C71.0 - C71.9\n",
      "Malignant neoplasm of brain, spinal cord, cranial nerves and other parts of central nervous system [glioma] [glioblastoma]\n",
      "\n",
      "C92.00 - C92.02, C92.40 - C92.a2\n",
      "Acute myeloid leukemia (AML)\n",
      "\n",
      "D46.0 - D46.9\n",
      "Myelodysplastic syndromes (MDS)\n",
      "\n",
      "D47.1\n",
      "Chronic myeloproliferative disease\n",
      "\n",
      "INHA (Inhibin) expression:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "86336\n",
      "Inhibin A\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C56.1 - C56.9\n",
      "Malignant neoplasm of ovary\n",
      "\n",
      "D07.39\n",
      "Carcinoma in situ of other female genital organs\n",
      "\n",
      "D27.0 - D27.9\n",
      "Benign neoplasm of ovary\n",
      "\n",
      "R19.00\n",
      "Intra-abdominal and pelvic swelling, mass, lump, unspecified site\n",
      "\n",
      "R19.07 - R19.09\n",
      "Generalized and other intra-abdominal and pelvic swelling, mass and lump\n",
      "\n",
      "Lactate dehydrogenase (LDH):\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "83615\n",
      "Lactate dehydrogenase (LD), (LDH)\n",
      "\n",
      "83625\n",
      "    isoenzymes, separation and quantitation\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C34.00 - C34.92\n",
      "Malignant neoplasm of bronchus and lung\n",
      "\n",
      "C40.00 - C41.9\n",
      "Malignant neoplasm of bone and articular cartilage\n",
      "\n",
      "C56.1 - C56.9\n",
      "Malignant neoplasm of ovary\n",
      "\n",
      "C62.00 - C62.92\n",
      "Malignant neoplasm of testis\n",
      "\n",
      "C64.1 - C65.9\n",
      "Malignant neoplasm of kidney and renal pelvis\n",
      "\n",
      "C85.10 - C85.99\n",
      "Non-hodgkin's lymphoma\n",
      "\n",
      "C90.00 - C90.02\n",
      "Multiple myeloma\n",
      "\n",
      "C91.00 - C91.02\n",
      "Acute lymphoblastic leukemia (ALL)\n",
      "\n",
      "D02.20 - D02.22\n",
      "Carcinoma in situ of bronchus and lung\n",
      "\n",
      "D07.39\n",
      "Carcinoma in situ of other female genital organs\n",
      "\n",
      "D07.69\n",
      "Carcinoma in situ of other male genital organs [testis]\n",
      "\n",
      "D14.30 - D14.32\n",
      "Benign neoplasm of bronchus and lung\n",
      "\n",
      "D16.0 - D16.9\n",
      "Benign neoplasm of bone and articular cartilage\n",
      "\n",
      "D27.0 - D27.9\n",
      "Benign neoplasm of ovary\n",
      "\n",
      "D29.20 - D29.22\n",
      "Benign neoplasm of testes\n",
      "\n",
      "D30.00 - D30.12\n",
      "Benign neoplasm of kidney and renal pelvis\n",
      "\n",
      "N28.89\n",
      "Other specified disorders of kidney and ureter [kidney mass]\n",
      "\n",
      "N50.8\n",
      "Other specified disorders of male genital organs [testicular mass]\n",
      "\n",
      "R19.00\n",
      "Intra-abdominal and pelvic swelling, mass, lump, unspecified site\n",
      "\n",
      "R19.07 - R19.09\n",
      "Generalized and other intra-abdominal and pelvic swelling, mass and lump\n",
      "\n",
      "PDGFRB testing :\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "PDGFRB testing - No specific code\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C91.00 - C91.02\t\t\n",
      "Acute lymphoblastic leukemia [ALL]\n",
      "\n",
      "D46.0 - D46.9\n",
      "Myelodysplastic syndromes (MDS)\n",
      "\n",
      "D47.Z9\n",
      "Other specified neoplasms of uncertain or unknown behavior of lymphoid, hematopoietic, and related tissue\n",
      "\n",
      "D48.5\n",
      "Neoplasm of uncertain behavior of skin [dermatofibrosarcoma]\n",
      "\n",
      "Phosphatidylinositol-4,5-bisphosphonate 3-kinase, catalytic subunit alpha polypeptide gene (PIK3CA):\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "0155U\n",
      "PIK3CA (phosphatidylinositol-4,5-bisphosphate 3-kinase, catalytic subunit alpha) (eg, breast cancer) gene analysis (ie, p.C420R, p.E542K, p.E545A, p.E545D [g.1635G>T only], p.E545G, p.E545K, p.Q546E, p.Q546R, p.H1047L, p.H1047R, p.H1047Y)\n",
      "\n",
      "81309\t\n",
      "PIK3CA (phosphatidylinositol-4, 5-biphosphate 3-kinase, catalytic subunit alpha) (eg, colorectal and breast cancer) gene analysis, targeted sequence analysis (eg, exons 7, 9, 20)\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C50.011 - C50.929\n",
      "Malignant neoplasm of breast\t\n",
      "\n",
      "C53.0 - C55, C58\n",
      "Malignant neoplasm of uterus\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C18.0 - C20\n",
      "Malignant neoplasm of colon, rectosigmoid junction and rectum\n",
      "\n",
      "PLCG2:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "81320\n",
      "PLCG2 (phospholipase C gamma 2) (eg, chronic lymphocytic leukemia) gene analysis, common variants (eg, R665W, S707F, L845F)\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C83.00 - C83.09\t  \n",
      "Small cell B cell lymphoma\n",
      "\n",
      "C91.10 - C91.12\n",
      "Chronic lymphocytic leukemia of B-cell type\n",
      "\n",
      "Quest Diagnostic Thyroid Cancer Mutation Panel:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "81445\n",
      "Targeted genomic sequence analysis panel, solid organ neoplasm, DNA analysis, and RNA analysis when performed, 5-50 genes (eg, ALK, BRAF, CDKN2A, EGFR, ERBB2, KIT, KRAS, NRAS, MET, PDGFRA, PDGFRB, PGR, PIK3CA, PTEN, RET), interrogation for sequence variants and copy number variants or rearrangements, if performed\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "D44.0 \n",
      "Neoplasm of uncertain behavior of thyroid gland \n",
      "\n",
      "E04.0 - E04.9\n",
      "Other nontoxic goiter [thyroid nodules] [not covered for repeat testing of indeterminate thyroid nodules]\n",
      "\n",
      "RUNX1:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "81334\n",
      "RUNX1 (runt related transcription factor 1) (eg, acute myeloid leukemia, familial platelet disorder with associated myeloid malignancy), gene analysis, targeted sequence analysis (eg, exons 3-8)\n",
      "\n",
      "81401\n",
      "Molecular pathology procedure, Level 2 (eg, 2-10 SNPs, 1 methylated variant, or 1 somatic variant [typically using nonsequencing target variant analysis], or detection of a dynamic mutation disorder/triplet repeat)\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C92.00 - C92.02, C92.40 - C92.A2\n",
      "Acute myeloid leukemia\t\n",
      "\n",
      "D46.0 - D46.9\n",
      "Myelodysplastic syndromes (MDS)\n",
      "\n",
      "D47.02\n",
      "Systemic mastocytosis\n",
      "\n",
      "SF3B1 test:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "81347\n",
      "SF3B1 (splicing factor [3b] subunit B1) (eg, myelodysplastic syndrome/acute myeloid leukemia) gene analysis, common variants (eg, A672T, E622D, L833F, R625C, R625L)\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C69.30 â C69.32\n",
      "Malignant neoplasm of choroid\n",
      "\n",
      "C69.40 â C69.42\n",
      "Malignant neoplasm of ciliary body\n",
      "\n",
      "C92.10 â C92.12\n",
      "Chronic myeloid leukemia, BCR/ABL-positive\n",
      "\n",
      "C92.20 - C92.22\n",
      "Atypical chronic myeloid leukemia, BCR/ABL-negative\n",
      "\n",
      "C94.6\n",
      "Myelodysplastic disease, not elsewhere classified [myeloproliferative neoplasms]\n",
      "\n",
      "D45\n",
      "Polycythemia vera \n",
      "\n",
      "D46.0 â D46.Z\n",
      "Myelodysplastic syndromes\n",
      "\n",
      "D47.3\n",
      "Essential (hemorrhagic) thrombocythemia \n",
      "\n",
      "D75.81\n",
      "Myelofibrosis \n",
      "\n",
      "SRSF2 test:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "81348\n",
      "SRSF2 (serine and arginine-rich splicing factor 2) (eg, myelodysplastic syndrome, acute myeloid leukemia) gene analysis, common variants (eg, P95H, P95L)\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C92.10 â C92.12\n",
      "Chronic myeloid leukemia, BCR/ABL-positive\n",
      "\n",
      "C92.20 - C92.22\n",
      "Atypical chronic myeloid leukemia, BCR/ABL-negative\n",
      "\n",
      "C94.6\n",
      "Myelodysplastic disease, not elsewhere classified [myeloproliferative neoplasms]\n",
      "\n",
      "C96.21\n",
      "Aggressive systemic Mastocytosis \n",
      "\n",
      "D45\n",
      "Polycythemia vera \n",
      "\n",
      "D46.0 â D46.Z\n",
      "Myelodysplastic syndromes\n",
      "\n",
      "D47.3\n",
      "Essential (hemorrhagic) thrombocythemia \n",
      "\n",
      "D75.81\n",
      "Myelofibrosis \n",
      "\n",
      "Thymidine kinase:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "81405\n",
      "Molecular pathology procedure, Level 6 (eg, analysis of 6-10 exons by DNA sequence analysis, mutation scanning or duplication/deletion variants of 11-25 exons, regionally targeted cytogenomic array analysis)\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C91.10 - C91.12, C91.90 - C91.91\n",
      "Chronic lymphocytic leukemia (CLL)/small lymphocytic lymphoma (SLL)\n",
      "\n",
      "Thyroglobulin antibody:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "86800\n",
      "Thyroglobulin antibody\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C73\n",
      "Malignant neoplasm of thyroid gland\n",
      "\n",
      "D09.3\n",
      "Carcinoma in situ of thyroid and other endocrine glands\n",
      "\n",
      "D34\n",
      "Benign neoplasm of thyroid gland\n",
      "\n",
      "Thyroglobulin (TG) expression:\n",
      "\n",
      "CPT codes covered if selection criteria are met :\n",
      "\n",
      "84432\n",
      "Thyroglobulin\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C73\n",
      "Malignant neoplasm of thyroid gland\n",
      "\n",
      "C76\n",
      "Malignant neoplasm of head, face and neck\n",
      "\n",
      "C80.0 - C80.1\n",
      "Disseminated and other malignant neoplasm, unspecified\n",
      "\n",
      "D09.3\n",
      "Carcinoma in situ of thyroid and other endocrine glands\n",
      "\n",
      "D34\n",
      "Benign neoplasm of thyroid gland\n",
      "\n",
      "Thyroid transcription factor-1 (TTF-1):\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "88342\n",
      "Immunohistochemistry or immunocytochemistry, each separately identifiable antibody per block, cytologic preparation, or hematologic smear; first separately identifiable antibody per slide\n",
      "\n",
      "88341\n",
      "    each additional single antibody stain procedure (List separately in addition to code for primary procedure)\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C34.00 - C34.92\n",
      "Malignant neoplasm of bronchus and lung\n",
      "\n",
      "C7A.00 - C7A.8\n",
      "Malignant neuroendocrine tumors\n",
      "\n",
      "D02.20 - D02.22\n",
      "Carcinoma in situ of bronchus and lung\n",
      "\n",
      "D14.30 - D14.32\n",
      "Benign neoplasm of bronchus and lung\n",
      "\n",
      "D3A.00 - D3A.8\n",
      "Benign neuroendocrine tumors\n",
      "\n",
      "WT-1 gene expression - No specific code:\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C34.00 - C34.92\n",
      "Malignant neoplasm of bronchus and lung [non-small-cell lung cancer]\n",
      "\n",
      "C48.2\n",
      "Malignant neoplasm of peritoneum, unspecified [Desmoplastic round cell tumor]\n",
      "\n",
      "C56.1 - C56.9\n",
      "Malignant neoplasm of ovary [ovarian clear cell carcinomas]\n",
      "\n",
      "C80.0 - C80.1\n",
      "Disseminated and other malignant neoplasm, unspecified\n",
      "\n",
      "HPV testing tumor testing (p16):\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "87624\n",
      "Infectious agent detection by nucleic acid (DNA or RNA); Human Papillomavirus (HPV), high-risk types (eg, 16, 18, 31, 33, 35, 39, 45, 51, 52, 56, 58, 59, 68)\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C00.0 - C14.8\n",
      "Malignant neoplasm of lip, oral cavity, and pharynx\n",
      "\n",
      "C76.0\n",
      "Malignant neoplasm of head, face and neck\n",
      "\n",
      "C80.1\n",
      "Malignant (primary) neoplasm, unspecified\n",
      "\n",
      "EZH2:\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "81236\n",
      "EZH2 (enhancer of zeste 2 polycomb repressive complex 2 subunit) (eg, myelodysplastic syndrome, myeloproliferative neoplasms) gene analysis, full gene sequence\n",
      "\n",
      "81237\n",
      "EZH2 (enhancer of zeste 2 polycomb repressive complex 2 subunit) (eg, diffuse large B-cell lymphoma) gene analysis, common variant(s) (eg, codon 646)\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "D46.20 - D46.9 \n",
      "Myelodysplastic syndrome\n",
      "\n",
      "D45\n",
      "Polycythemia vera\n",
      "\n",
      "D69.3\n",
      "Immune thrombocytopenic purpura\n",
      "\n",
      "C94.40 - C94.42\n",
      "Acute panmyelosis with myelofibrosis\n",
      "\n",
      "D47.1\n",
      "Chronic myeloproliferative disease\n",
      "\n",
      "D47.4\n",
      "Osteomyelofibrosis\n",
      "\n",
      "D75.81\n",
      "Myelofibrosis\n",
      "\n",
      "C92.10 - C92.12\n",
      "Chronic myeloid leukemia, BCR/ABL-positive\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB (not all-inclusive):\n",
      "\n",
      "C83.30 - C83.39\n",
      "Diffuse large B-cell lymphoma\n",
      "\n",
      "TERT (telomerase reverse transcriptase):\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "81345\n",
      "TERT (telomerase reverse transcriptase) (eg, thyroid carcinoma, glioblastoma multiforme) gene analysis, targeted sequence analysis (eg, promoter region)\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C71.0 - C71.9\n",
      "Malignant neoplasm of brain\n",
      "\n",
      "D46.20 - D46.9\n",
      "Myelodysplastic syndrome\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB (not all-inclusive):\n",
      "\n",
      "C73\n",
      "Malignant neoplasm of thyroid gland\n",
      "\n",
      "Carcinoembryonic antigen cell adhesion molecule 6 (CEACAM6) (e.g., Benign Diagnostics Risk Test) - No specific code:\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB (not all-inclusive):\n",
      "\n",
      "N62\n",
      "Hypertrophy of breast [breast atypical hyperplasia]\n",
      "\n",
      "CDX2:\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB (not all-inclusive):\n",
      "\n",
      "C18.0 - C18.9\n",
      "Malignant neoplasm of colon\n",
      "\n",
      "D01.0\n",
      "Carcinoma in situ of colon\n",
      "\n",
      "D12.0 - D12.9\n",
      "Benign neoplasm of colon\n",
      "\n",
      "CxBladder test:\n",
      "\n",
      "CPT codes not covered for indications listed in the CPB:\n",
      "\n",
      "0012M\n",
      "Oncology (urothelial), mRNA, gene expression profiling by real-time quantitative PCR of five genes (MDK, HOXA13, CDC2 [CDK1], IGFBP5, and XCR2), utilizing urine, algorithm reported as a risk score for having urothelial carcinoma\n",
      "\n",
      "0013M\n",
      "Oncology (urothelial), mRNA, gene expression profiling by real-time quantitative PCR of five genes (MDK, HOXA13, CDC2 [CDK1], IGFBP5, and CXCR2), utilizing urine, algorithm reported as a risk score for having recurrent urothelial carcinoma\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB (not all-inclusive):\n",
      "\n",
      "C67.0 - C67.9\n",
      "Malignant neoplasm of bladder\n",
      "\n",
      "4Kscore:\n",
      "\n",
      "CPT codes not covered for indications listed in the CPB:\n",
      "\n",
      "81539 \n",
      "Oncology (high-grade prostate cancer), biochemical assay of four proteins (Total PSA, Free PSA, Intact PSA, and human kallikrein-2 [hK2]), utilizing plasma or serum, prognostic algorithm reported as a probability score\n",
      "\n",
      "Artera AI Prostate Test:\n",
      "\n",
      "CPT codes not covered for indications listed in the CPB:\n",
      "\n",
      "0376U\n",
      "Oncology (prostate cancer), image analysis of at least 128 histologic features and clinical factors, prognostic algorithm determining the risk of distant metastases, and prostate cancer- specific mortality, includes predictive algorithm to androgen deprivation- therapy response, if appropriate\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C61\n",
      "Malignant neoplasm of prostate\n",
      "\n",
      "D07.5\n",
      "Carcinoma in situ of prostate\n",
      "\n",
      "Fibrinogen degradation products (FDP) test (e.g., DR-70 or Onko-Sure) - No specific code:\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB (not all-inclusive):\n",
      "\n",
      "C18.0 - C20\n",
      "Malignant neoplasm of colon, rectosigmoid junction, and rectum\n",
      "\n",
      "HMGB1 and RAGE - No specific code:\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C43.0 - C44.99\n",
      "Melanoma and other malignant neoplasms of skin\n",
      "\n",
      "IHC4 (e.g., NexCourse IHC4) - No specific code:\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB (not all-inclusive):\n",
      "\n",
      "C50.011 - C50.929\n",
      "Malignant neoplasm of breast\n",
      "\n",
      "Lectin-reactive alpha-fetoprotein (AFP-L3):\n",
      "\n",
      "CPT codes not covered for indications listed in the CPB:\n",
      "\n",
      "82107\n",
      "Alpha-fetoprotein (AFP); AFP-L3 fraction isoform and total AFP (including ratio)\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB (not all-inclusive):\n",
      "\n",
      "C22.0, C22.2 - C22.9\n",
      "Malignant neoplasm of liver\n",
      "\n",
      "Liquid biopsy (e.g., CancerIntercept, GeneStrat, Colvera, Neolab Prostate, FoundationACT, FoudationOne Liquid CDx, Guardant360CDx, InVisionFirst-Lung test):\n",
      "\n",
      "CPT codes covered if selection criteria are met:\n",
      "\n",
      "0179U\n",
      "Oncology (non-small cell lung cancer), cell-free DNA, targeted sequence analysis of 23 genes (single nucleotide variations, insertions and deletions, fusions without prior knowledge of partner/breakpoint, copy number variations), with report of significant mutation(s) [covered up to 50 genes]\n",
      "\n",
      "0388U\n",
      "Oncology (non-small cell lung cancer), next-generation sequencing with identification of single nucleotide variants, copy number variants, insertions and deletions, and structural variants in 37 cancer-related genes, plasma, with report for alteration detection\n",
      "\n",
      "CPT codes not covered for indications listed in the CPB:\n",
      "\n",
      "Neolab Prostate- no specific code\n",
      "\n",
      "0011M\n",
      "Oncology, prostate cancer, mRNA expression assay of 12 genes (10 content and 2 housekeeping), RT-PCR test utilizing blood plasma and/or urine, algorithms to predict high-grade prostate cancer risk\n",
      "\n",
      "0326U\n",
      "Targeted genomic sequence analysis panel, solid organ neoplasm, cell-free circulating DNA analysis of 83 or more genes, interrogation for sequence variants, gene copy number amplifications, gene rearrangements, microsatellite instability and tumor mutational burden\n",
      "\n",
      "86152\n",
      "Cell enumeration using immunologic selection and identification in fluid specimen (eg, circulating tumor cells in blood)\n",
      "\n",
      "86153\n",
      "    physician interpretation and report, when required\n",
      "\n",
      "ICD-10 codes covered if selection criteria are met:\n",
      "\n",
      "C34.00 - C34.92\n",
      "Malignant neoplasm of bronchus and lung\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB (not all-inclusive):\n",
      "\n",
      "C18.0 - C20\n",
      "Malignant neoplasm of colon, rectosigmoid junction, and rectum\n",
      "\n",
      "C43.0 - C43.9\n",
      "Malignant melanoma of skin\n",
      "\n",
      "C50.011 - C50.929\n",
      "Malignant neoplasm of breast\n",
      "\n",
      "C56.1 - C56.9\n",
      "Malignant neoplasm of ovary\n",
      "\n",
      "C61\n",
      "Malignant neoplasm of prostate\n",
      "\n",
      "Long non-coding RNA - No specific code:\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C23\n",
      "Malignant neoplasm of gallbladder\n",
      "\n",
      "Mass spectrometry-based proteomic profiling (e.g., Xpresys Lung):\n",
      "\n",
      "CPT codes not covered for indications listed in the CPB:\n",
      "\n",
      "0174U \n",
      "Oncology (solid tumor), mass spectrometric 30 protein targets, formalin-fixed paraffin-embedded tissue, prognostic and predictive algorithm reported as likely, unlikely, or uncertain benefit of 39 chemotherapy and targeted therapeutic oncology agents \n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB (not all-inclusive):\n",
      "\n",
      "R91.8\n",
      "Other nonspecific abnormal finding of lung field [indeterminate pulmonary nodules]\n",
      "\n",
      "OncoVantage:\n",
      "\n",
      "CPT codes not covered for indications listed in the CPB:\n",
      "\n",
      "81445\n",
      "Targeted genomic sequence analysis panel, solid organ neoplasm, DNA analysis, and RNA analysis when performed, 5-50 genes (eg, ALK, BRAF, CDKN2A, EGFR, ERBB2, KIT, KRAS, NRAS, MET, PDGFRA, PDGFRB, PGR, PIK3CA, PTEN, RET), interrogation for sequence variants and copy number variants or rearrangements, if performed\n",
      "\n",
      "Select MDX - No specific code:\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C61\n",
      "Malignant neoplasm of prostate\n",
      "\n",
      "Oncuria Detect, Monitor and Predict Tests:\n",
      "\n",
      "CPT codes not covered for indications listed in the CPB:\n",
      "\n",
      "0365U\n",
      "Oncology (bladder), analysis of 10 protein biomarkers (A1AT, ANG, APOE, CA9, IL8, MMP9, MMP10, PAI1, SDC1 and VEGFA) by immunoassays, urine, algorithm reported as a probability of bladder cancer\n",
      "\n",
      "0366U\n",
      "Oncology (bladder), analysis of 10 protein biomarkers (A1AT, ANG, APOE, CA9, IL8, MMP9, MMP10, PAI1, SDC1 and VEGFA) by immunoassays, urine, algorithm reported as a probability of recurrent bladder cancer\n",
      "\n",
      "0367U\n",
      "Oncology (bladder), analysis of 10 protein biomarkers (A1AT, ANG, APOE, CA9, IL8, MMP9, MMP10, PAI1, SDC1 and VEGFA) by immunoassays, urine, diagnostic algorithm reported as a risk score for probability of rapid recurrence of recurrent or persistent cancer following transurethral resection\n",
      "\n",
      "Other CPT codes related to CPB:\n",
      "\n",
      "90586\n",
      "Bacillus Calmette-Guerin vaccine (BCG) for bladder cancer, live, for intravesical use\n",
      "\n",
      "Other HCPCS codes related to CPB:\n",
      "\n",
      "J9030\n",
      "BCG live intravesical instillation, 1 mg\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C67.0 - C67.9\n",
      "Malignant neoplasm of bladder\n",
      "\n",
      "D09.0\n",
      "Carcinoma in situ of bladder\n",
      "\n",
      "OvaWatch Test:\n",
      "\n",
      "CPT codes not covered for indications listed in the CPB:\n",
      "\n",
      "0375U\n",
      "Oncology (ovarian), biochemical assays of 7 proteins (follicle stimulating hormone, human epididymis protein 4, apolipoprotein A-1, transferrin, beta-2 macroglobulin, prealbumin [ie, transthyretin], and cancer antigen 125), algorithm reported as ovarian cancer risk score\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C56.1 - C56.9\n",
      "Malignant neoplasm of ovary\n",
      "\n",
      "D07.39\n",
      "Carcinoma in situ of other female genital organs [ovary]\n",
      "\n",
      "D39.10 â D39.12\n",
      "Neoplasm of uncertain behavior of ovary\n",
      "\n",
      "Matepair targeted rearrangements (whole genome next-generation sequencing):\n",
      "\n",
      "CPT codes not covered for indications listed in the CPB:\n",
      "\n",
      "0013U\n",
      "Oncology (solid organ neoplasia), gene rearrangement detection by whole genome next-generation sequencing, DNA, fresh or frozen tissue or cells, report of specific gene rearrangement(s)\n",
      "\n",
      "0014U\n",
      "Hematology (hematolymphoid neoplasia), gene rearrangement detection by whole genome next-generation sequencing, DNA, whole blood or bone marrow, report of specific gene rearrangement(s)\n",
      "\n",
      "0056U\n",
      "Hematology (acute myelogenous leukemia), DNA, whole genome next-generation sequencing to detect gene rearrangement(s), blood or bone marrow, report of specific gene rearrangement(s)\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C81.00 - C96.9\n",
      "Hematolymphoid neoplasia\n",
      "\n",
      "C00.0 - C43.9, C44.00 - C80.2\n",
      "Solid organ neoplasia\n",
      "\n",
      "Signatera:\n",
      "\n",
      "CPT codes not covered for indications listed in the CPB:\n",
      "\n",
      "0340U\n",
      "Oncology (pan-cancer), analysis of minimal residual disease (MRD) from plasma, with assays personalized to each patient based on prior next-generation sequencing of the patient's tumor and germline DNA, reported as absence or presence of MRD, with disease-burden correlation, if appropriate\n",
      "\n",
      "ICD-10 codes not covered for indications listed in the CPB:\n",
      "\n",
      "C16.0 â C16.9\n",
      "Malignant neoplasm of stomach\n",
      "\n",
      "C18.0 â C21.8 \n",
      "Malignant neoplasm of colon, rectosigmoid junction, rectum, anus and anal canal\n",
      "\n",
      "C43.0 â C43.9\n",
      "Malignant melanoma of skin\n",
      "\n",
      "C49.0 - C49.9\n",
      "Malignant neoplasm of other connective and soft tissue [alveolar soft tissue sarcoma]\n",
      "\n",
      "C50.011 â C50.929\n",
      "Malignant neoplasm of breast\n",
      "\n",
      "C53.0 â C53.9\n",
      "Malignant neoplasm of cervix uteri\n",
      "\n",
      "C54.0 â C54.9\n",
      "Malignant neoplasm of corpus uteri\n",
      "\n",
      "C55\n",
      "Malignant neoplasm of uterus, part unspecified\n",
      "\n",
      "C61\n",
      "Malignant neoplasm of prostate\n",
      "\n",
      "C64.1 - C64.9\n",
      "Malignant neoplasm of kidney, except renal pelvis\n",
      "\n",
      "Experimental and investigational tumor markers:\n",
      "\n",
      "CPT codes not covered for indications listed in the CPB:\n",
      "\n",
      "0006M\n",
      "Oncology (hepatic), MRNA expression levels of 161 genes, utilizing fresh hepatocellular carcinoma tumor tissue, with alpha-fetoprotein level, algorithm reported as a risk classifier [Heprodx] \n",
      "\n",
      "0007M\n",
      "Oncology (gastrointestinal neuroendocrine tumors), real-time PCR expression analysis of 51 genes, utilizing whole peripheral blood, algorithm reported as a nomogram of tumor disease index [Netest]\n",
      "\n",
      "0015M\n",
      "Adrenal cortical tumor, biochemical assay of 25 steroid markers, utilizing 24-hour urine specimen and clinical parameters, prognostic algorithm reported as a clinical risk and integrated clinical steroid risk for adrenal cortical carcinoma, adenoma, or other adrenal malignancy\n",
      "\n",
      "0016M\n",
      "Oncology (bladder), mRNA, microarray gene expression profiling of 209 genes, utilizing formalin-fixed paraffin-embedded tissue, algorithm reported as molecular subtype (luminal, luminal infiltrated, basal, basal claudin-low, neuroendocrine-like)\n",
      "\n",
      "0005U\n",
      "Oncology (prostate) gene expression profile by real-time RT-PCR of 3 genes (ERG, PCA3, and SPDEF), urine, algorithm reported as risk score\n",
      "\n",
      "0013U\n",
      "Oncology (solid organ neoplasia), gene rearrangement detection by whole genome next-generation sequencing, DNA, fresh or frozen tissue or cells, report of specific gene rearrangement(s)\n",
      "\n",
      "0019U\n",
      "Oncology, RNA, gene expression by whole transcriptome sequencing, formalin-fixed paraffin embedded tissue or fresh frozen tissue, predictive algorithm reported as potential targets for therapeutic agents\n",
      "\n",
      "0037U\n",
      "Targeted genomic sequence analysis, solid organ neoplasm, DNA analysis of 324 genes, interrogation for sequence variants, gene copy number amplifications, gene rearrangements, microsatellite instability and tumor mutational burden\n",
      "\n",
      "0050U\n",
      "Targeted genomic sequence analysis panel, acute myelogenous leukemia, DNA analysis, 194 genes, interrogation for sequence variants, copy number variants or rearrangements\n",
      "\n",
      "0053U\n",
      "Oncology (prostate cancer), FISH analysis of 4 genes (ASAP1, HDAC9, CHD1 and PTEN), needle biopsy specimen, algorithm reported as probability of higher tumor grade\n",
      "\n",
      "0057U\n",
      "Oncology (solid organ neoplasia), mRNA, gene expression profiling by massively parallel sequencing for analysis of 51 genes, utilizing formalin-fixed paraffin-embedded tissue, algorithm reported as a normalized percentile rank\n",
      "\n",
      "0058U\n",
      "Oncology (Merkel cell carcinoma), detection of antibodies to the Merkel cell polyoma virus oncoprotein (small T antigen), serum, quantitative\n",
      "\n",
      "0059U\n",
      "Oncology (Merkel cell carcinoma), detection of antibodies to the Merkel cell polyoma virus capsid protein (VP1), serum, reported as positive or negative\n",
      "\n",
      "0067U\n",
      "Oncology (breast), immunohistochemistry, protein expression profiling of 4 biomarkers (matrix metalloproteinase-1 [MMP-1], carcinoembryonic antigen-related cell adhesion molecule 6 [CEACAM6], hyaluronoglucosaminidase [HYAL1], highly expressed in cancer protein [HEC1]), formalin-fixed paraffin-embedded precancerous breast tissue, algorithm reported as carcinoma risk score\n",
      "\n",
      "0069U\n",
      "Oncology (colorectal), microRNA, RT-PCR expression profiling of miR-31-3p, formalin-fixed paraffin-embedded tissue, algorithm reported as an expression score\n",
      "\n",
      "0090U\n",
      "Oncology (cutaneous melanoma), mRNA gene expression profiling by RT-PCR of 23 genes (14 content and 9 housekeeping), utilizing formalin-fixed paraffin-embedded tissue, algorithm reported as a categorical result (ie, benign, indeterminate, malignant)\n",
      "\n",
      "0092U\n",
      "Oncology (lung), three protein biomarkers, immunoassay using magnetic nanosensor technology, plasma, algorithm reported as risk score for likelihood of malignancy\n",
      "\n",
      "0113U\n",
      "Oncology (prostate), measurement of PCA3 and TMPRSS2-ERG in urine and PSA in serum following prostatic massage, by RNA amplification and fluorescence-based detection, algorithm reported as risk score\n",
      "\n",
      "0120U\n",
      "Oncology (B-cell lymphoma classification), mRNA, gene expression profiling by fluorescent probe hybridization of 58 genes (45 content and 13 housekeeping genes), formalin-fixed paraffin-embedded tissue, algorithm reported as likelihood for primary mediastinal B-cell lymphoma (PMBCL) and diffuse large B-cell lymphoma (DLBCL) with cell of origin subtyping in the latter\n",
      "\n",
      "0174U\n",
      "Oncology (solid tumor), mass spectrometric 30 protein targets, formalin- fixed paraffin-embedded tissue, prognostic and predictive algorithm reported as likely, unlikely, or uncertain benefit of 39 chemotherapy and targeted therapeutic oncology agents\n",
      "\n",
      "0204U\n",
      "Oncology (thyroid), mRNA, gene expression analysis of 593 genes (including BRAF, RAS, RET, PAX8, and NTRK) for sequence variants and rearrangements, utilizing fine needle aspirate, reported as detected or not detected\n",
      "\n",
      "0211U\n",
      "Oncology (pan-tumor), DNA and RNA by next-generation sequencing, utilizing formalin-fixed paraffin-embedded tissue, interpretative report for single nucleotide variants, copy number alterations, tumor mutational burden, and microsatellite instability, with therapy association\n",
      "\n",
      "0220U\n",
      "Oncology (breast cancer), image analysis with artificial intelligence assessment of 12 histologic and immunohistochemical features, reported as a recurrence score\n",
      "\n",
      "0228U\n",
      "Oncology (prostate), multianalyte molecular profile by photometric detection of macromolecules adsorbed on nanosponge array slides with machine learning, utilizing first morning voided urine, algorithm reported as likelihood of prostate cancer\n",
      "\n",
      "0229U\n",
      "BCAT1 (Branched chain amino acid transaminase 1) or IKZF1 (IKAROS family zinc finger 1) (eg, colorectal cancer) promoter methylation analysis\n",
      "\n",
      "0242U\n",
      "Targeted genomic sequence analysis panel, solid organ neoplasm, cell-free circulating DNA analysis of 55-74 genes, interrogation for sequence variants, gene copy number amplifications, and gene rearrangements\n",
      "\n",
      "0244U\n",
      "Oncology (solid organ), DNA, comprehensive genomic profiling, 257 genes, interrogation for single-nucleotide variants, insertions/deletions, copy number alterations, gene rearrangements, tumor-mutational burden and microsatellite instability, utilizing formalin-fixed paraffin-embedded tumor tissue\n",
      "\n",
      "0250U\n",
      "Oncology (solid organ neoplasm), targeted genomic sequence DNA analysis of 505 genes, interrogation for somatic alterations (SNVs [single nucleotide variant], small insertions and deletions, one amplification, and four translocations), microsatellite instability and tumor-mutation burden\n",
      "\n",
      "0261U\n",
      "Oncology (colorectal cancer), image analysis with artificial intelligence assessment of 4 histologic and immunohistochemical features (CD3 and CD8 within tumor-stroma border and tumor core), tissue, reported as immune response and recurrence-risk score\n",
      "\n",
      "0262U\n",
      "Oncology (solid tumor), gene expression profiling by real-time RT-PCR of 7 gene pathways (ER, AR, PI3K, MAPK, HH, TGFB, Notch), formalin-fixed paraffin-embedded (FFPE), algorithm reported as gene pathway activity score\n",
      "\n",
      "0285U\n",
      "Oncology, response to radiation, cell-free DNA, quantitative branched chain DNA amplification, plasma, reported as a radiation toxicity score\n",
      "\n",
      "0288U\n",
      "Oncology (lung), mRNA, quantitative PCR analysis of 11 genes (BAG1, BRCA1, CDC6, CDK2AP1, ERBB3, FUT3, IL11, LCK, RND3, SH3BGR, WNT3A) and 3 reference genes (ESD, TBP, YAP1), formalin-fixed paraffin-embedded (FFPE) tumor tissue, algorithmic interpretation reported as a recurrence risk score\n",
      "\n",
      "0295U\n",
      "Oncology (breast ductal carcinoma in situ), protein expression profiling by immunohistochemistry of 7 proteins (COX2, FOXA1, HER2, Ki-67, p16, PR, SIAH2), with 4 clinicopathologic factors (size, age, margin status, palpability), utilizing formalin-fixed paraffin-embedded (FFPE) tissue, algorithm reported as a recurrence risk score\n",
      "\n",
      "0296U\n",
      "Oncology (oral and/or oropharyngeal cancer), gene expression profiling by RNA sequencing at least 20 molecular features (eg, human and/or microbial mRNA), saliva, algorithm reported as positive or negative for signature associated with malignancy\n",
      "\n",
      "0297U\n",
      "Oncology (pan tumor), whole genome sequencing of paired malignant and normal DNA specimens, fresh or formalin-fixed paraffin-embedded (FFPE) tissue, blood or bone marrow, comparative sequence analyses and variant identification\n",
      "\n",
      "0298U\n",
      "Oncology (pan tumor), whole transcriptome sequencing of paired malignant and normal RNA specimens, fresh or formalin-fixed paraffin-embedded (FFPE) tissue, blood or bone marrow, comparative sequence analyses and expression level and chimeric transcript identification\n",
      "\n",
      "0299U\n",
      "Oncology (pan tumor), whole genome optical genome mapping of paired malignant and normal DNA specimens, fresh frozen tissue, blood, or bone marrow, comparative structural variant identification\n",
      "\n",
      "0300U\n",
      "Oncology (pan tumor), whole genome sequencing and optical genome mapping of paired malignant and normal DNA specimens, fresh tissue, blood, or bone marrow, comparative sequence analyses and variant identification\n",
      "\n",
      "0313U\n",
      "Oncology (pancreas), DNA and mRNA next-generation sequencing analysis of 74 genes and analysis of CEA (CEACAM5) gene expression, pancreatic cyst fluid, algorithm reported as a categorical result (ie, negative, low probability of neoplasia or positive, high probability of neoplasia)\n",
      "\n",
      "0314U\n",
      "Oncology (cutaneous melanoma), mRNA gene expression profiling by RT-PCR of 35 genes (32 content and 3 housekeeping), utilizing formalin-fixed paraffin-embedded (FFPE) tissue, algorithm reported as a categorical result (ie, benign, intermediate, malignant)\n",
      "\n",
      "0315U\n",
      "Oncology (cutaneous squamous cell carcinoma), mRNA gene expression profiling by RT-PCR of 40 genes (34 content and 6 housekeeping), utilizing formalin-fixed paraffin-embedded (FFPE) tissue, algorithm reported as a categorical risk result (ie, Class 1, Class 2A, Class 2B)\n",
      "\n",
      "0317U\n",
      "Oncology (lung cancer), four-probe FISH (3q29, 3p22.1, 10q22.3, 10cen) assay, whole blood, predictive algorithm-generated evaluation reported as decreased or increased risk for lung cancer\n",
      "\n",
      "0324U\n",
      "Oncology (ovarian), spheroid cell culture, 4-drug panel (carboplatin, doxorubicin, gemcitabine, paclitaxel), tumor chemotherapy response prediction for each drug\n",
      "\n",
      "0325U\n",
      "Oncology (ovarian), spheroid cell culture, poly (ADP-ribose) polymerase (PARP) inhibitors (niraparib, olaparib, rucaparib, velparib), tumor response prediction for each drug\n",
      "\n",
      "0329U\n",
      "Oncology (neoplasia), exome and transcriptome sequence analysis for sequence variants, gene copy number amplifications and deletions, gene rearrangements, icrosatellite instability and tumor mutational burden utilizing DNA and RNA from tumor with DNA from normal blood or saliva for subtraction, report of clinically significant mutation(s) with therapy associations \n",
      "\n",
      "0331U\n",
      "Oncology (hematolymphoid neoplasia), optical genome mapping for copy number alterations and gene rearrangements utilizing DNA from blood or bone marrow, report of clinically significant alternations\n",
      "\n",
      "0332U\n",
      "Oncology (pan-tumor), genetic profiling of 8 DNA-regulatory (epigenetic) markers by quantitative polymerase chain reaction (qPCR), whole blood, reported as a high or low probability of responding to immune checkpointâinhibitor therapy\n",
      "\n",
      "0333U\n",
      "Oncology (liver), surveillance for hepatocellular carcinoma (HCC) in high- risk patients, analysis of methylation patterns on circulating cell-free DNA (cfDNA) plus measurement of serum of AFP/AFP-L3 and oncoprotein des-gamma- carboxy-prothrombin (DCP), algorithm reported as normal or abnormal result\n",
      "\n",
      "0334U\n",
      "Oncology (solid organ), targeted genomic sequence analysis, formalin-fixed paraffin- embedded (FFPE) tumor tissue, DNA analysis, 84 or more genes, interrogation for sequence variants, gene copy number amplifications, gene rearrangements, microsatellite instability and tumor mutational burden\n",
      "\n",
      "0342U\n",
      "Oncology (pancreatic cancer), multiplex immunoassay of C5, C4, cystatin C, factor B, osteoprotegerin (OPG), gelsolin, IGFBP3, CA125 and multiplex electrochemiluminescent immunoassay (ECLIA) for CA19-9, serum, diagnostic algorithm reported qualitatively as positive, negative, or borderline\n",
      "\n",
      "0343U\n",
      "Oncology (prostate), exosome-based analysis of 442 small noncoding RNAs (sncRNAs) by quantitative reverse transcription polymerase chain reaction (RT-qPCR), urine, reported as molecular evidence of no-, low-, intermediate- or high-risk of prostate cancer\n",
      "\n",
      "0356U\n",
      "Oncology (oropharyngeal), evaluation of 17 DNA biomarkers using droplet digital PCR (ddPCR), cell-free DNA, algorithm reported as a prognostic risk score for cancer recurrence\n",
      "\n",
      "0357U\n",
      "Oncology (melanoma), artificial intelligence (AI)-enabled quantitative mass spectrometry analysis of 142 unique pairs of glycopeptide and product fragments, plasma, prognostic, and predictive algorithm reported as likely, unlikely, or uncertain benefit from immunotherapy agents\n",
      "\n",
      "0359U\n",
      "Oncology (prostate cancer), analysis of all prostate-specific antigen (PSA) structural isoforms by phase separation and immunoassay, plasma, algorithm reports risk of cancer\n",
      "\n",
      "0360U\n",
      "Oncology (lung), enzyme-linked immunosorbent assay (ELISA) of 7 autoantibodies (p53, NY-ESO-1, CAGE, GBU4-5, SOX2, MAGE A4, and HuD), plasma, algorithm reported as a categorical result for risk of malignancy\n",
      "\n",
      "0362U\n",
      "Oncology (papillary thyroid cancer), gene-expression profiling via targeted hybrid captureâenrichment RNA sequencing of 82 content genes and 10 housekeeping genes, formalin-fixed paraffin embedded (FFPE) tissue, algorithm reported as one of three molecular subtypes\n",
      "\n",
      "0363U\n",
      "Oncology (urothelial), mRNA, gene-expression profiling by real-time quantitative PCR of 5 genes (MDK, HOXA13, CDC2 [CDK1], IGFBP5, and CXCR2), utilizing urine, algorithm incorporates age, sex, smoking history, and macrohematuria frequency, reported as a risk score for having urothelial carcinoma \n",
      "\n",
      "0379U\n",
      "Targeted genomic sequence analysis panel, solid organ neoplasm, DNA (523 genes) and RNA (55 genes) by next-generation sequencing, interrogation for sequence variants, gene copy number amplifications, gene rearrangements, microsatellite instability, and tumor mutational burden\n",
      "\n",
      "0387U\n",
      "Oncology (melanoma), autophagy and beclin 1 regulator 1 (AMBRA1) and loricrin (AMLo) by immunohistochemistry, formalin- fixed paraffin-embedded (FFPE) tissue, report for risk of progression\n",
      "\n",
      "0391U\n",
      "Oncology (solid tumor), DNA and RNA by next-generation sequencing, utilizing formalin-fixed paraffin-embedded (FFPE) tissue, 437 genes, interpretive report for single nucleotide variants, splice- site variants, insertions/deletions, copy number alterations, gene fusions, tumor mutational burden, and microsatellite instability, with algorithm quantifying immunotherapy response score\n",
      "\n",
      "0395U\n",
      "Oncology (lung), multi-omics (microbial DNA by shotgun next- generation sequencing and carcinoembryonic antigen and osteopontin by immunoassay), plasma, algorithm reported as malignancy risk for lung nodules in early-stage disease\n",
      "\n",
      "0794T\n",
      "Patient-specific, assistive, rules-based algorithm for ranking pharmaco-oncologic treatment options based on the patient's tumor-specific cancer marker information obtained from prior molecular pathology, immunohistochemical, or other pathology results which have been previously interpreted and reported separately\n",
      "\n",
      "81218\n",
      "CEBPA (CCAAT/enhancer binding protein [C/EBP], alpha) (eg, acute myeloid leukemia), gene analysis, full gene sequence \n",
      "\n",
      "81449\n",
      "Targeted genomic sequence analysis panel, solid organ neoplasm, 5-50 genes (eg, ALK, BRAF, CDKN2A, EGFR, ERBB2, KIT, KRAS, MET, NRAS, PDGFRA, PDGFRB, PGR, PIK3CA, PTEN, RET), interrogation for sequence variants and copy number variants or rearrangements, if performed; RNA analysis\n",
      "\n",
      "81451\n",
      "Targeted genomic sequence analysis panel, hematolymphoid neoplasm or disorder, 5-50 genes (eg, BRAF, CEBPA, DNMT3A, EZH2, FLT3, IDH1, IDH2, JAK2, KIT, KRAS, MLL, NOTCH1, NPM1, NRAS), interrogation for sequence variants, and copy number variants or rearrangements, or isoform expression or mRNA expression levels, if performed; RNA analysis\n",
      "\n",
      "81455\n",
      "Targeted genomic sequence analysis panel, solid organ or hematolymphoid neoplasm, DNA analysis, and RNA analysis when performed, 51 or greater genes (eg, ALK, BRAF, CDKN2A, CEBPA, DNMT3A, EGFR, ERBB2, EZH2, FLT3, IDH1, IDH2, JAK2, KIT, KRAS, MLL, NPM1, NRAS, MET, NOTCH1, PDGFRA, PDGFRB, PGR, PIK3CA, PTEN, RET), interrogation for sequence variants and copy number variants or rearrangements, if performed  \n",
      "\n",
      "81456\n",
      "Targeted genomic sequence analysis panel, solid organ or hematolymphoid neoplasm or disorder, 51 or greater genes (eg, ALK, BRAF, CDKN2A, CEBPA, DNMT3A, EGFR, ERBB2, EZH2, FLT3, IDH1, IDH2, JAK2, KIT, KRAS, MET, MLL, NOTCH1, NPM1, NRAS, PDGFRA, PDGFRB, PGR, PIK3CA, PTEN, RET), interrogation for sequence variants and copy number variants or rearrangements, or isoform expression or mRNA expression levels, if performed; RNA analysis\n",
      "\n",
      "81529\n",
      "Oncology (cutaneous melanoma), mRNA, gene expression profiling by real-time RT-PCR of 31 genes (28 content and 3 housekeeping), utilizing formalin-fixed paraffin-embedded tissue, algorithm reported as recurrence risk, including likelihood of sentinel lymph node metastasis [DecisionDx-Melanoma]\n",
      "\n",
      "81540\n",
      "Oncology (tumor of unknown origin), mRNA, gene expression profiling by real-time RT-PCR of 92 genes (87 content and 5 housekeeping) to classify tumor into main cancer type and subtype, utilizing formalin-fixed paraffin-embedded tissue, algorithm reported as a probability of a predicted main cancer type and subtype \n",
      "\n",
      "82387\n",
      "Cathepsin-D\n",
      "\n",
      "84275\n",
      "Sialic acid\n",
      "\n",
      "86316\n",
      "Immunoassay for tumor antigen; other antigen, quantitative (e.g., CA 50, 72-4, 549), each\n",
      "\n",
      "88342\n",
      "Immunohistochemistry or immunocytochemistry, each separately identifiable antibody per block, cytologic preparation, or hematologic smear; first separately identifiable antibody per slide [Cyclin E (fragments or whole length)] \n",
      "\n",
      "There are no specific codes for the tumor markers listed below:\n",
      "\n",
      "anti-VEGF antibody bevacizumab; BluePrint molecular subtyping profile for breast cancer; BreastSentry; C-Met expression; Glutathione-S-transferase P1 (GSTP1); Mammostrat;  Percepta Bronchial Genomic Classifier; Phosphatidylinositol-4,5-bisphosphonate 3-kinase; Proveri prostate cancer assay (PPCA); Ribonucleotide reductase subunit M1 (RRM1); ROS1 re-arrangements; Previstage GCC; Prostate core mitotic test; UroCor cytology assay (DD23 and P53); BioSpeciFx; Nucleus Detect Assay; Envisia Genomic Classifier, Myriad myPath Melanoma; NantHealth; Sentinel PCa test; Salivary metatranscriptome analysis for oral cancers (i.e., mRNA CancerDetect), BostonGene Tumor Portrait Test; Grail Galleri Test\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Background\n",
      "A tumor marker is a substance such as a protein, antigen or hormone in the body that may indicate the presence of cancer. Generally, these markers are specific to certain types of cancer and can be detected in blood, urine and tissue samples. The body may produce the marker in response to cancer or the tumor itself may produce the marker. The detection of tumor markers may be used to determine a diagnosis or as an indicator of disease (cancer) progression. It can also be used to document clinical response to treatment. Tumor markers include, but may not be limited to, alpha- fetoprotein (AFP), CA 15-3/CA 27.29, CA 19-9, CA-125, carcinoembryonic antigen (CEA) and prostate-specific antigen (PSA).\n",
      "Tumor markers are normally produced in low quantities by cells in the body. Detection of a higher-than-normal serum level by radioimmunoassay or immunohistochemical techniques usually indicates the presence of a certain type of cancer.  Currently, the main use of tumor markers is to assess a cancer's response to treatment and to check for recurrence. In some types of cancer, tumor marker levels may reflect the extent or stage of the disease and can be useful in predicting how well the disease will respond to treatment.  A decrease or return to normal in the level of a tumor marker may indicate that the cancer has responded favorably to therapy.  If the tumor marker level rises, it may indicate that the cancer is spreading.  Finally, measurements of tumor marker levels may be used after treatment has ended as a part of follow-up care to check for recurrence.\n",
      "However, in many cases the literature states that measurements of tumor marker levels alone are insufficient to diagnose cancer for the following reasons:\n",
      "\n",
      "tumor marker levels can be elevated in people with benign conditions;\n",
      "tumor marker levels are not elevated in every person with cancer, especially in the early stages of the disease; and\n",
      "many tumor markers are not specific to a particular type of cancer; and\n",
      "the level of a tumor marker can be elevated by more than one type of cancer.\n",
      "\n",
      "Examples of Tumor Markers Include\n",
      "\n",
      "5-Hydroxyindoleacetic acid (5-HIAA) â the main metabolite of serotonin, used as a marker in the evaluation of carcinoid tumors;\n",
      "Beta-2-Microglobulin (B2M) â A protein found on the surface of many cells. High levels of B2M are an indicator of certain kinds of cancer, including chronic lymphocytic leukemia, non-Hodgkin's lymphoma and multiple myeloma or kidney disease;\n",
      "Beta Human Chorionic Gonadotropin (beta HCG) â A type of tumor marker that may be found in higher than normal amounts in individuals with some types of cancer;\n",
      "Calcitonin â Hormone secreted by the thyroid that lowers blood calcium;\n",
      "Calretinin â A calcium-binding protein that is used as a marker in the evaluation of lung cancer and other diseases.\n",
      "Chromogranin A â A protein found inside neuroendocrine cells, which releases chromogranin A and other hormones into the blood. Chromogranin A may be found in higher than normal amounts in individuals with certain neuroendocrine tumors, small cell lung cancer, prostate cancer and other conditions\n",
      "Guanylyl cyclase c (GCC) â An enzyme that may be expressed only in the cells that line the intestine from the duodenum to the rectum.\n",
      "Inhibin â One of two hormones (designated inhibin-A and inhibin-B) secreted by the gonads (by Sertoli cells in the male and the granulosa cells in the female) and inhibits the production of follicle-stimulating hormone (FSH) by the pituitary gland;\n",
      "Lactate Dehydrogenase (LDH) â Marker used to monitor treatment of testicular cancer;\n",
      "Mucin-1 (MUC-1) â Carbohydrate antigen elevated in individuals with tumors of the breast, ovary, lung and prostate as well as other disorders;\n",
      "Napsin A â Protein used as a marker in the evaluation of lung cancer;\n",
      "Prealbumin â Marker of nutritional status and a sensitive indicator of protein synthesis. Also referred to as transthyretin;\n",
      "Prostate Specific Antigen (PSA) â Substance produced by the prostate gland. Levels of PSA in the blood often increase in men with prostate cancer.\n",
      "Thyroglobulin â Protein found in the thyroid gland. Some thyroglobulin can be found in the blood and this amount may be measured after thyroid surgery to determine whether thyroid cancer has recurred;\n",
      "Thyroid Transcription Factor-1 (TTF-1) â A protein that is used as a tumor marker in the evaluation of lung cancer;\n",
      "Transferrin â A protein in blood plasma that carries iron derived from food intake to the liver, spleen and bone marrow.  \n",
      "\n",
      "Tumors may be evaluated with histology, which involves examination of the structure, especially the microscopic structure, of organic tissues. Methods of detecting tumor markers include, but are not limited to: Fluorescence in Situ Hybridization (FISH) â Laboratory technique used to detect small  deletions or rearrangements in chromosomes. Immunohistochemical (IHC) Analysis â Laboratory process of detecting an organism in tissues with antibodies. \n",
      "Gene mutation testing can purportedly be used to find somatic mutations in cancerous cells that are not inherited. Some examples of genes that may have somatic mutations include: IDH1 and IDH2 genes (associated with acute myeloid leukemia [AML], gliomas and chondrosarcomas); NPM1 and FLT3 genes (associated with AML).\n",
      "Individualized molecular tumor profiling is a laboratory method of testing a panel of tumor markers, which may include genetic as well as biochemical markers, to establish a personalized molecular profile of a tumor to recommend treatment options.\n",
      "Mass spectrometry based proteomic profiling (eg, Veristrat, Xpresys Lung) is a multivariate serum protein test that uses mass spectrometry and proprietary algorithms to analyze proteins in an individualâs serum. The Xpresys is no longer on the market.\n",
      "Next-generation sequence (NGS) tests use select genes to purportedly identify molecular growth drivers for improved risk stratification and targeted therapies. Examples include: FoundationOne and OncoVantage for solid tumor cancers; FoundationOne Heme for hematological cancers and sarcomas; and ThyGenX for indeterminate thyroid nodules.  \n",
      "Liquid biopsy refers to serum testing for DNA fragments that are shed by cancer cells and released into the bloodstream. This method is purportedly used for screening, diagnosis and/or monitoring of cancer cells that may otherwise require a tissue sample.\n",
      "Multianalyte assays with algorithmic analyses (MAAAs) are laboratory measurements that use a mathematical formula to analyze multiple markers that may be associated with a particular disease state and are designed to evaluate disease activity or an individualâs risk for disease. The laboratory performs an algorithmic analysis using the results of the assays and sometimes other information, such as sex and age and converts the information into a numeric score, which is conveyed on a laboratory report. Generally, MAAAs are exclusive to a single laboratory which owns the algorithm. MAAAs have been proposed for the evaluation of pelvic masses, including assisting in the determination of referral for surgery to a gynecologic oncologist or to a general surgeon.\n",
      "Topographic genotyping (eg, PathFinderTG) is a test that examines a panel of 15 to 20 genetic markers in tissue biopsy or other tissue specimens to purportedly aid in the determination of indeterminate or equivocal cancer diagnoses.\n",
      "An ASCO Provisional Opinion on somatic genomic testing in patients with metastatic or advanced cancer (Chakravarty, et al., 2022) states: \"Repeat genomic testing may be performed for patients with acquired resistance on targeted therapies, especially when known acquired resistance mechanisms may affect the choice of next-line therapy. Repeat testing may also assist in identifying new targets in tumors after progression or after prolonged stable disease on targeted therapies.\"\n",
      "Myeloproliferative Neoplasms and Myelodysplastic Syndromes\n",
      "Myeloproliferative neoplasms and myelodysplastic syndromes are both blood cell diseases and both carry an increased risk of transformation into acute myelogenous leukemia (AML). Myelodysplastic syndromes (MDSs) refer to a heterogeneous group of myeloid disorders characterized by varying reductions in the production of red blood cells, platelets, and mature granulocytes that may also exhibit functional (i.e., qualitative) defects.\n",
      "Conversely, Myeloproliferative neoplasms (MPN) refer to a group of heterogenous disorders characterized by overproduction of one or more types of blood cells. MPNs include polycythemia vera, essential thrombocythemia, chronic myeloid leukemia, primary myelofibrosis, chronic neutrophilic leukemia, and other less well defined entities such as chronic eosinophilic leukemia, not otherwise categorized.\n",
      "A third category, Myelodysplastic/myeloproliferative neoplasms (MDS/MPN), include disorders that manifest both dysplastic and proliferative features. These include chronic myelomonocytic leukemia, juvenile myelomonocytic leukemia, atypical CML (aCML, BCR-ABL1 negative), MDS/MPN with ring sideroblasts and thrombocytosis, and unclassifiable MDS/MPN.\n",
      "PSA\n",
      "Prostate Specific Antigen (PSA) is a substance produced by the prostate gland. Levels of PSA in the blood often increase in men with prostate cancer. Elevated levels of Prostate-Specific Antigen (PSA) may also be found in the blood of men with benign prostate conditions, such as prostatitis and benign prostatic hyperplasia (BPH). While PSA does not allow distinction between benign prostate conditions and cancer, an elevated PSA level may indicate that other tests are necessary to determine whether cancer is present. PSA levels have been shown to be useful in monitoring the effectiveness of prostate cancer treatment, and in checking for recurrence after treatment has ended. Use of PSA for screening remains very controversial.  Although researchers are in the process of studying the value of PSA along with digital rectal exams for routine screening of men ages 55 to 74 for prostate cancer; and the literature does not show at this time whether using PSA to screen for prostate cancer actually does reduce the number of deaths caused by this cancer.  The American Cancer Society recommends clinicians and patients consider screening with PSA and digital rectal exam for African American men and men with familial tendency age 40 or older and all men age 50 or older.\n",
      "Cancer Care Ontario guidelines on active surveillance of prostate cancer (Morash, et al., 2015) state that the active surveillance protocol should include the following tests: PSA test every 3 to 6 months; digital rectal examination every year, and a 12- to 14-core confirmatory transrectal ultrasound (TRUS) biopsy (including anterior directed cores) within 6 to 12 months, then serial biopsy a minimum of every 3 to 5 years thereafter. The guidelines state that \"[c]urrent evidence shows that PSA kinetics does not reliably predict disease stability or reclassification to higher risk state. There was conflicting evidence whether PSA is a good predictor of disease progression or reclassification. Differences were also found in the ability of different measures of PSA, such as PSA velocity, PSA density, and PSA doubling time for predicting progression or reclassification. PSA monitoring is considered a necessary component of an AS protocol, but a rising PSA may be best viewed as a trigger for reappraisal (e.g., MRI, repeat biopsy) rather than a trigger for intervention.\"\n",
      "PCA3\n",
      "Prostate cancer antigen 3 (PCA3, also known as DD3) is a gene that has been found to be highly overexpressed in prostate cancer.  This gene has been investigated as a potential diagnostic marker for prostate cancer.  However, there are no published clinical outcome studies of the effectiveness of the PCA3 gene in screening, diagnosis or management of prostate cancer.\n",
      "Prostate cancer antigen 3 (PCA3) (Progensa, Gene-Probe, Inc.) encodes a prostate-specific mRNA. It is one of the most prostate cancer-specific genes identified, with over-expression in about 95% of cancers tested. The PCA3 urine assay is an amplified nucleic acid assay, which uses transcription-mediated amplification (TMA) to quantify PCA3 and PSA mRNA in prostate cells found in urine samples. The PCA3 score is calculated as the ratio between PCA3 and PSA mRNA. The main target population of this non-invasive test is men with raised PSA but a negative prostate biopsy. Other target groups include men with a slightly raised PSA, as well as men with signs and symptoms suggestive of prostate cancer.\n",
      "van Gils and colleagues (2007) stated that PCA3 is a promising prostate cancer marker. These investigators performed a multi-center study to validate the diagnostic performance of the PCA3 urine test established in an earlier single-institution study. The first voided urine after digital rectal examination (DRE) was collected from a total of 583 men with serum PSA levels between 3 and 15 ng/ml who were to undergo prostate biopsies. These researchers determined the PCA3 score in these samples and correlated the results with the results of the prostate biopsies. A total of 534 men (92 %) had an informative sample. The area under the receiver-operating characteristic curve, a measure of the diagnostic accuracy of a test, was 0.66 for the PCA3 urine test and 0.57 for serum PSA. The sensitivity for the PCA3 urine test was 65 %, the specificity was 66 % (versus 47 % for serum PSA), and the negative predictive value was 80 %. The authors concluded that the findings of this multi-center study validated the diagnostic performance of the PCA3 urine test in the largest group studied thus far using a PCA3 gene-based test.\n",
      "Marks and associates (2007) examined the potential utility of the investigational PCA3 urine assay to predict the repeat biopsy outcome. Urine was collected after DRE (3 strokes per lobe) from 233 men with serum PSA levels persistently 2.5 ng/ml or greater and at least one previous negative biopsy. The PCA3 scores were determined using a highly sensitive quantitative assay with TMA. The ability of the PCA3 score to predict the biopsy outcome was assessed and compared with the serum PSA levels. The RNA yield was adequate for analysis in the urine samples from 226 of 233 men (i.e., the informative specimen rate was 97 %). Repeat biopsy revealed prostate cancer in 60 (27 %) of the 226 remaining subjects. Receiver operating characteristic curve analysis yielded an area under the curve of 0.68 for the PCA3 score. In contrast, the area under the curve for serum PSA was 0.52. Using a PCA3 score cutoff of 35, the assay sensitivity was 58 % and specificity 72 %, with an odds ratio of 3.6. At PCA3 scores of less than 5, only 12 % of men had prostate cancer on repeat biopsy; at PCA3 scores of greater than 100, the risk of positive biopsy was 50 %. The authors concluded that in men undergoing repeat prostate biopsy to rule out cancer, the urinary PCA3 score was superior to serum PSA determination for predicting the biopsy outcome. The high specificity and informative rate suggest that the PCA3 assay could have an important role in prostate cancer diagnosis.\n",
      "Groskopf et al (2007) reported that the PCA3 score is independent of prostate volume and was highly correlated with the risk of positive biopsy. The PCA3 test was performed on 529 men scheduled for prostate biopsy. Overall, the PCA3 score had a sensitivity of 54% and a specificity of 74%. A PCA3 score of less than 5 was associated with a 14% risk of positive biopsy, while a PCA3 score of greater than 100 was associated with a 69% risk of positive biopsy.\n",
      "Haese et al (2007) presented preliminary results from a European multicenter study of PCA3. Enrolled patients had a PSA level of less than or equal to 2.5 ng/mL, had 1 or 2 previous negative biopsies, and were scheduled for repeat biopsy. The specificity of the PCA3 score (cutoff 35) was found to be 78%, and the sensitivity was 67%. Patients with a PCA3 score of greater than or equal to 35 had a 33% probability of a positive repeat biopsy, compared to a 6% probability for those with a PCA3 score of less than 35.\n",
      "In a review on biomarkers for prostate cancer detection, Parekh, et al. (2007) stated that prostate stem cell antigen, alpha-methyl coenzyme-A racemase, PCA3, early prostate cancer antigen, hepsin and human kallikrein 2 are promising markers that are currently undergoing validation.\n",
      "An assessment by the BlueCross BlueShield Association Technology Evaluation Center (BCBSA, 2008) found that, in general, PCA3 assay results to date are preliminary; interpretation of results has not been standardized and clinical utility studies of decision-making for initial biopsy, repeat biopsy or treatment have not been reported.\n",
      "Tosoian et al (2010) evaluated the relationship between PCA3 and prostate biopsy results in men in a surveillance program. Urine specimens were obtained from 294 men with prostate cancer enrolled in the Johns Hopkins surveillance program. The follow-up protocol included semi-annual free and total PSA measurements, digital rectal examination and annual surveillance prostate biopsy. Cox proportional hazards regression was used to evaluate the association between PCA3 results and progression on surveillance biopsy (defined as Gleason pattern 4 or 5, more than 2 positive biopsy cores or more than 50% involvement of any core with cancer). Patients with progression on biopsy (12.9%) had a mean PCA3 score similar to that of those without progression (60.0 versus 50.8, p = 0.131). Receiver operating characteristics analysis suggested that PCA3 alone could not be used to identify men with progression on biopsy (area under the curve = 0.589, 95% CI 0.496 to 0.683, p = 0.076). After adjustment for age and date of diagnosis PCA3 was not significantly associated with progression on biopsy (p = 0.15). The authors concluded that in men with low risk prostate cancer who were carefully selected for surveillance the PCA3 score was not significantly associated with short-term biopsy progression. They stated that further analysis is necessary to assess the usefulness of PCA3 in combination with other biomarkers or in selected subsets of patients undergoing surveillance.\n",
      "While there are studies examining the positive and negative predictive values of the PCA3 urine assay, there is currently a lack of evidence of the effect of this test on management of individuals with or suspected of prostate cancer. The PCA3 urine assay shows promise as a prostate cancer diagnostic tool, however, more research is needed to ascertain the clinical value of this assay for screening and diagnostic purposes.\n",
      "An assessment of PCA3 prepared for the Agency for Healthcare Research and Quality (2013) concluded: \"For diagnostic accuracy, there was a low strength of evidence that PCA3 had better diagnostic accuracy for positive biopsy results than tPSA elevations, but insufficient evidence that this led to improved intermediate or long-term health outcomes. For all other settings, comparators, and outcomes, there was insufficient evidence.\"\n",
      "The Evaluation of Genomic Applications in Practice and Prevention (EGAPP) Working Group (2013) found insufficient evidence to recommend prostate cancer antigen 3 (PCA3) testing to inform decisions for when to re-biopsy previously biopsy-negative patients for prostate cancer or to inform decisions to conduct initial biopsies for prostate cancer in at-risk men (e.g., previous elevated prostate-specific antigen test or suspicious digital rectal examination). The EGAPP Working Group found insufficient evidence to recommend PCA3 testing in men with cancer-positive biopsies to determine if the disease is indolent or aggressive in order to develop an optimal treatment plan. The EGAPP Working Group concluded that, based on the available evidence, the overall certainty of clinical validity to predict the diagnosis of prostate cancer using PCA3 is deemed \"low.\" The EGAPP Working Group discouraged clinical use for diagnosis unless further evidence supports improved clinical validity. The EGAPP Working Group also found that, based on the available evidence, the overall certainty of net health benefit is deemed \"low.\" The EGAPP Working Group discourages clinical use unless further evidence supports improved clinical outcomes.\n",
      "Guidelines from the European Association of Urology (2015) state that \"[b]iological markers, include urine markers such as PCA3, the TMPRSS2: ERG fusion gene or PSA isoforms such as the Phi index, appear promising as does genomics on the tissue sample itself. However, further study data will be needed before such markers can be used in standard clinical practice.\"\n",
      "A Cancer Care Ontario Guideline on prostate cancer surveillance (Morash, et al., 2015), which has been endorsed by the American Society for Clinical Oncology (2016), did not include PCA3 level in their recommendation because evidence of PCA3 to predict disease reclassification in prostate cancer was lacking. \n",
      "National Institute for Health and Care Excellence (NICE)âs clinical practice guideline on \"Diagnosing prostate cancer: PROGENSA PCA3 assay and Prostate Health Index\" (2015) stated that \" The PROGENSA PCA3 assay and the Prostate Health Index are not recommended for use in people having investigations for suspected prostate cancer, who have had a negative or inconclusive transrectal ultrasound prostate biopsy\". The assessment cited studies finding that adding the PCA3 score to clinical assessment and MRI had very little effect on the size of the reported area under the curve, with minimal change in derived sensitivity and specificity for clinical assessment with MRI compared with clinical assessment using MRI and the PCA3 assay.\n",
      "In a Lancet review of prostate cancer, Attard, et al. (2016) stated that \"[s]everal studies have so far proven inconclusive as to whether PCA3 is useful to selectively detect aggressive prostate cancers.\" \n",
      "B15\n",
      "Hutchinson et al (2005) stated that in tissue-based assays, thymosin beta15 (B15) has been shown to correlate with prostate cancer and with recurrence of malignancy. To be clinically effective, it must be shown that thymosin B15 is released by the tumor into body fluids in detectable concentrations.  These researchers developed a quantitative assay that can measure clinically relevant levels of thymosin B15 in human urine.  Sixteen antibodies were raised against recombinant thymosin B15 and/or peptide conjugates.  One antibody, having stable characteristics over the wide range of pH and salt concentrations found in urine and minimal cross-reactivity with other beta thymosins, was used to develop a competitive enzyme-linked immunosorbent assay (ELISA).  Urinary thymosin B15 concentration was determined for control groups; normal (n = 52), prostate intraepithelial neoplasia (PIN, n = 36), and patients with prostate cancer; untreated (n = 7) with subsequent biochemical failure, radiation therapy (n = 17) at risk of biochemical recurrence.  The operating range of the competition ELISA fell between 2.5 and 625 ng/ml.  Recoveries exceeded 75%, and the intra- and inter-assay coefficients of variability were 3.3% and 12.9%, respectively.  No cross-reactivity with other urine proteins was observed.  A stable thymosin B15 signal was recovered from urine specimens stored at -20 degrees C for up to 1 year.  At a threshold of 40 (ng/dl)/microg protein/mg creatinine), the assay had a sensitivity of 58% and a specificity of 94%.  Relative to the control groups, thymosin B15 levels were greater than this threshold in a significant fraction of patients with prostate cancer (p < 0.001), including 5 of the 7 patients who later experienced PSA recurrence.  The authors concluded that an ELISA that is able to detect thymosin B15 at clinically relevant concentrations in urine from patients with prostate cancer has been established.  They noted that the assay will provide a tool for future clinical studies to validate urinary thymosin B15 as a predictive marker for recurrent prostate cancer.\n",
      "CEA\n",
      "Carcinoembryonic antigen (CEA) is a normal cell product that is over-expressed by adenocarcinomas, primarily of the colon, rectum, breast, and lung. It is normally found in small amounts in the blood of most healthy people, but may become elevated in people who have cancer or some benign conditions. \n",
      "CEA is an oncofetal glycoprotein present in the gastrointestinal tract and body fluids of the embryo and fetus (Chin, et al., 2006). It is also present in certain adult gastrointestinal cells, including the mucosal cells of the colorectum, and small amounts are present in blood. Blood levels are often elevated in patients with disseminated cancers and in some patients with nonmalignant disease.\n",
      "According to the available literature, the primary use of CEA is in monitoring colorectal cancer, especially when the disease has metastasized.  CEA is also used after treatment to check for recurrence of colorectal cancer.  However, the literature indicates a wide variety of other cancers can produce elevated levels of this tumor marker, including melanoma; lymphoma; and cancers of the breast, lung, pancreas, stomach, cervix, bladder, kidney, thyroid, liver, and ovary.  Elevated CEA levels can also occur in patients with non-cancerous conditions, including inflammatory bowel disease, pancreatitis, and liver disease.\n",
      "The American Society of Clinical Oncology (ASCO)'s update of recommendations for the use of tumor markers in gastrointestinal cancer (Gershon, et al., 2006) stated that post-operative CEA levels should be performed every 3 months for stage II and III disease for at least 3 years if the patient is a potential candidate for surgery or chemotherapy of metastatic disease.\n",
      "CA-125\n",
      "Cancer antigen 125 (CA-125) is a test that evaluates ovarian cancer treatment. CA-125 is a protein that is found more in ovarian cancer cells than in other cells. CA-125 is expressed by >80 percent of non-mucinous ovarian epithelial neoplasms (Chin et al, 2006). Approximately half of women with metastatic ovarian cancer have an elevated CA-125 level.\n",
      "The Gynecologic Cancer Foundation, the Society of Gynecologic Oncologists, and the American Cancer Society have issued a consensus statement to promote early detection of ovarian cancer, which recommends that women who have symptoms, including bloating, pelvic or abdominal pain, difficulty eating or feeling full quickly, and urinary frequency and urgency, are urged to see a gynecologist if symptoms are new and persist for more than three weeks (ACS, 2007; SGO, 2007).  Ovarian cancer is among the deadliest types of cancer because diagnosis usually comes very late, after the cancer has spread. If the cancer is found and surgically removed before it spreads outside the ovary, the five year survival rate is 93%. However, only 19% of cases are detected early enough for that kind of successful intervention. It is estimated that 22,430 new cases and 15,280 deaths will be reported in 2007 (ACS, 2007). The consensus statement recommendations are based on studies that show the above symptoms appeared in women with ovarian cancer more than in other women (Goff, et al., 2004; Daly & Ozols, 2004). The recommendations acknowledge that there is not consensus on what physicians should do when patients present with these symptoms. According to a consensus statement issued by the Gynecologic Cancer Foundation, pelvic and rectal examination in women with the symptoms is one first step. If there is a suspicion of cancer, the next step may be a transvaginal ultrasound to check the ovaries for abnormal growths, enlargement, or telltale pockets of fluid that can indicate cancer. Testing for CA-125 levels should also be considered.\n",
      "There is no evidence available that measurement of CA-125 can be effectively used for widespread screening to reduce mortality from ovarian cancer, nor that the use of this test would result in decreased rather than increased morbidity and mortality.  According to the available literature, not all women with elevated CA 125 levels have ovarian cancer. CA 125 levels may also be elevated by cancers of the uterus, cervix, pancreas, liver, colon, breast, lung, and digestive tract. Non-cancerous conditions that can cause elevated CA 125 levels include endometriosis, pelvic inflammatory disease, peritonitis, pancreatitis, liver disease, and any condition that inflames the pleura.  Menstruation and pregnancy can also cause an increase in CA 125.  However, according to the available literature, changes in CA 125 levels can be effectively used in the management of treatment for ovarian cancer. In women with ovarian cancer being treated with chemotherapy, the literature suggests a falling CA 125 level generally indicates that the cancer is responding to treatment and increased survival is expected. Increasing CA 125 levels during or after treatment, on the other hand, may suggest that the cancer is not responding to therapy or that residual cancer remains.  According to the available literature, failure of the CA 125 level to return to normal after three cycles of chemotherapy indicates residual tumor, early treatment failure and decreased survival. Under accepted guidelines, CA 125 levels can also be used to monitor patients for recurrence of ovarian cancer.  Although an elevated CA 125 level is highly correlated with the presence of ovarian cancer, the literature suggests a normal value does not exclude residual or recurrent disease.\n",
      "Aetna's preventive services guidelines are based on the recommendations of leading primary care medical professional organizations and federal public health agencies.  None of these organizations recommend routine screening of average-risk, asymptomatic women with serum CA-125 levels for ovarian cancer.  These organizations have concluded that serum CA-125 levels are not sufficiently sensitive or specific for use as a screening test for ovarian cancer, and that the harms of such screening outweigh the benefits.\n",
      "The American College of Obstetricians and Gynecologists (2002) has stated that \"[u]nfortunately, there is no screening test for ovarian cancer that has proved effective in screening low-risk asymptomatic women. Measurement of CA 125 levels and completion of pelvic ultrasonography (both abdominal and transvaginal) have been the two tests most thoroughly evaluated.... Data suggest that currently available tests do not appear to be beneficial for screening low-risk, asymptomatic women because their sensitivity, specificity, positive predictive value, and negative predictive value have all been modest at best. Because of the low incidence of disease, reported to be approximately one case per 2,500 women per year, it has been estimated that a test with even 100% sensitivity and 99% specificity would have a positive predictive value of only 4.8%, meaning 20 of 21 women undergoing surgery would not have primary ovarian cancer. Unfortunately, no test available approaches this level of sensitivity or specificity.\"\n",
      "The National Cancer Institute (2004) has stated: \"There is insufficient evidence to establish that screening for ovarian cancer with serum markers such as CA 125 levels, transvaginal ultrasound, or pelvic examinations would result in a decrease in mortality from ovarian cancer.  A serious potential harm is the false-positive test result, which may lead to anxiety and invasive diagnostic procedures. There is good evidence that screening for ovarian cancer with the tests above would result in more diagnostic laparoscopies and laparotomies than new ovarian cancers found. Unnecessary oophorectomies may also result.\"\n",
      "The U.S. Preventive Services Task Force (2004) recommends against routine screening with serum CA-125 level for ovarian cancer.  The Task Force concluded that the potential harms of such screening outweigh the potential benefits.\n",
      "HE4\n",
      "Human Epididymis Protein 4 (HE4) is a secreted glycoprotein that is being studied as a potential marker for ovarian cancer.\n",
      "A variety of other tumor markers have been investigated for early detection of ovarian cancer as well as different combinations of tumor markers complementary to CA 125 that could potentially offer greater sensitivity and specificity than CA 125 alone.  Preliminary studies on HE4 (human epididymis protein 4), a marker for ovarian cancer, reported similar sensitivity to CA 125 when comparing ovarian cancer cases to healthy controls, and a higher sensitivity when comparing ovarian cancer cases to benign gynecologic disease (Hellstrom, et al., 2003 & 2008; Moore, et al., 2008;)  However, an assessment on genomic tests for ovarian cancer prepared by Duke University for the Agency for Healthcare Research and Quality (AHRQ, 2006) stated, \"Although research remains promising, adaptation of genomic tests into clinical practice must await appropriately designed and powered studies in relevant clinical settings.\"  Further studies are needed to determine if HE4 significantly adds to the sensitivity of CA 125 while maintaining a high specificity.\n",
      "National Comprehensive Cancer Network (NCCN) guidelines (2016) state that data show that HE4 and several other markers do not increase early enough to be useful in detecting early-stage ovarian cancer.\n",
      "CA 15-3\n",
      "Cancer antigen 15-3 (CA 15-3) is a serum cancer antigen that has been used in the management of patients with breast cancer.  According to the available literature, its low detection rate in early stage disease indicates that CA 15-3 cannot be used to screen or diagnose patients with breast cancer.  It has been widely used to monitor the effectiveness of treatment for metastatic cancer. Elevated serum CA 15-3 concentrations are found in 5 percent of stage I, 29 percent of stage II, 32 percent of stage III and 95 percent of stage IV carcinoma of the breast (Chin, et al, 2006). Most (96 percent) patients with a CA 15-3 increase of greater than 25 percent have disease progression. Most (nearly 100 percent) patients with a CA 15-3 decrease of greater than 50 percent are responding to treatment.\n",
      "Cancers of the ovary, lung, and prostate may also raise CA 15-3 levels.  The literature indicates elevated levels of CA 15-3 may be associated with non-cancerous conditions, such as benign breast or ovarian disease, endometriosis, pelvic inflammatory disease, and hepatitis.\n",
      "Similar to the CA 15-3 antigen, CA 27-29 is found in the blood of most breast cancer patients. The literature indicates CA 27-29 levels may be used in conjunction with other procedures (such as mammograms and measurements of other tumor marker levels) to check for recurrence in women previously treated for stage II and stage III breast cancer. CA 27-29 levels can also be elevated by cancers of the colon, stomach, kidney, lung, ovary, pancreas, uterus, and liver.  First trimester pregnancy, endometriosis, ovarian cysts, benign breast disease, kidney disease, and liver disease are non-cancerous conditions that can also elevate CA 27-29 levels.\n",
      "Elevated CA 27.29 levels are primarily associated with metastatic breast cancer, where it can be used to monitor the course of disease, response to treatment, and detect disease recurrence (Chin, et al., 2006). Elevated serum CA 27.29 concentrations are found in 95 percent of stage IV breast cancer. In addition, CA 27.29 has been found to be elevated in lung (43 percent), pancreas (47 percent), ovarian (56 percent), and liver (55 percent) cancer.\n",
      "CA 19-9\n",
      "Cancer antigen 19-9 (CA 19-9) is a mucin-glycoprotein first identified from a human colorectal carcinoma cell line and is present in epithelial tissue of the stomach, gall bladder, pancreas and prostate (Chin, et al., 2006). Concentrations are increased in patients with pancreatic, gastric, and colon cancer as well as in some nonmalignant conditions. Increasing levels generally indicate disease progression, whereas decreasing levels suggest therapeutic response.\n",
      "Initially found in colorectal cancer patients, CA 19-9 has also been identified in patients with pancreatic, stomach, hepatocellular cancer, and bile duct cancer. In those who have pancreatic cancer, the literature indicates higher levels of CA 19-9 tend to be associated with more advanced disease.  Although the sensitivity of the CA 19-9 level in patients with pancreatic cancer is relatively high, the specificity is lowered by elevations that occur in patients with benign pancreatic or liver disease.  Non-cancerous conditions that may elevate CA 19-9 levels include gallstones, pancreatitis, cirrhosis of the liver, and cholecystitis.  Although excellent correlations have been reported between CA 19-9 measurements and relapse in patients with pancreatic cancer who are followed after surgical resection, no patient has been salvaged by the earlier diagnosis of relapse, a fact that reflects the lack of effective therapy.\n",
      "Guidelines from the National Comprehensive Cancer Network (NCCN, 2010) state that measurement of CA 19-9 should be considered in evaluating patients with intrahepatic or extrahepatic cholangiocarcinoma and gallbladder cancer. The guidelines note that CA 19-9 is often elevated in persons with cholangiocarcinoma or gallbladder cancer, although this marker is not specific for these cancers.  Nehls, et al. (2004) considered CA19-9 as one of the several new potential tumor markers for the diagnosis of cholangiocarcinoma.  Levy, et al. (2005) aimed to characterize the test properties of CA 19-9 and of a change in CA 19-9 over time in predicting cholangiocarcinoma in patients with primary sclerosing cholangitis. Charts of 208 patients were reviewed. Fourteen patients had cholangiocarcinoma. Median CA 19-9 was higher with cholangiocarcinoma (15 versus 290 U/ml, p < 0.0001). A cutoff of 129 U/ml provided: sensitivity 78.6%, specificity 98.5%, adjusted positive predictive value 56.6% and negative predictive value 99.4%. The median change over time was 664 U/ml in cholangiocarcinoma compared to 6.7 U/ml in primary sclerosing cholangitis alone (p < 0.0001). A cutoff of 63.2 U/ml for change in CA 19-9 provided: sensitivity 90%, specificity 98% and positive predictive value 42%.\n",
      "CA 19-9 is produced by adenocarcinomas of the pancreas, stomach, gall-bladder, colon, ovary, and lung, and it is shed into the circulation.  Although numerous studies have addressed the potential utility of CA 19-9 in adenocarcinoma of the colon and rectum, the sensitivity of CA 19-9 was always less than that of the CEA test for all stages of disease. Its use for screening asymptomatic populations has been hampered by a false-positive rate of 15% to 30% in patients with non-neoplastic diseases of the pancreas, liver, and biliary tract.  Only a few studies have addressed the use of CA 19-9 in monitoring patients' post-primary therapy. Significant postsurgical decreases are observed for CA 19-9, but these decreases have not been correlated with survival or disease-free interval. In monitoring response to treatment, decreases in CEA have been found to more accurately reflect response to therapy than did decreases of CA 19-9. Progressive increases of the marker may signal disease progression in 25% of the patients who express the CA 19-9 marker, but this monitoring provides only a minimal lead time of 1 to 3 months. Monitoring with CA 19-9 has not been shown to improve the management of patients with colorectal cancer. The serum CA 19-9 level does not add significant information to that provided by CEA, which is currently regarded as the marker of choice for this neoplasm.\n",
      "Sinakos and colleagues (2011) evaluated the long-term outcomes in Mayo Clinic patients presenting with primary sclerosing cholangitis (PSC)  between 2000 and 2010 (n= 73) for incidence of cholangiocarcinoma (CCA).    The results showed initial levels of CA 19-9 in patients without CCA were significantly lower than those from patients with CCA (p < 0.0001).  No factors known  to affect CA 19-9 levels were identified in 33% of the patients without CCA; endoscopic treatment and recurrent bacterial cholangitis were associated with levels of CA 19-9 in 26% and 22% of these patients, respectively.\n",
      "Juntermanns (2011) prospectively analyzed a bile duct tumor database and retrieved records of 238 patients who underwent surgery between 1999 and 2008.  Their findings included that pre-operative CA19-9 serum levels did not show a statically reliable differentiation between benign or malignant dignity.  The authors concluded that current diagnostics cannot differentiate malignant from benign tumor masses in the hepatic hilum with required reliability.  The authors further concluded that administration of CIK cells, thymus factor, IL-2 and IFN-alpha after AHSCT could improve the immunologic function of patients, and TH1/TH2 ratio may virtually reflect the immune status of patients, but that more information is required to make prognostic assessments of immune reconstruction and the long-term survival rate.\n",
      "Sarbia et al (1993) investigated 69 adenocarcinomas of the esophagogastric junction and found high rates of antigen expression were found for the \"intestinal\" markers CA 19-9 (between 55.5% and 100%) and BW 494 (between 42.9 and 86.7%).  The authors concluded that these data, in combination with CK-20 expression, PGII, and 2B5 indicate that the distribution of adenocarcinomas with gastric and.or intestinal differentiation at the esophagogastric junction forms a continuum with out clear-cut borders.  This study has not been replicated and NCCN guidelines for Esophageal and Esophagogastric Junction Cancers does not include recommendations for CA 19-9 testing for these indications (NCCN, 2011).\n",
      "The American Society of Clinical Oncology (ASCO)'s update of recommendations for the use of tumor markers in gastrointestinal cancer (Gershon, et al., 2006) stated that for pancreatic cancer, CA 19-9 can be measured every 1 to 3 months for patients with locally advanced or metastatic disease receiving active therapy.\n",
      "Mucinous carcinoma of the appendix is a rare entity most commonly associated with primary tumors of the appendix and colon, and for which spread is generally confined to the abdominal cavity (Andreopoulou et al, 2007).  Imaging assessment of these mucinous lesions is difficult, and recent studies have explored the use of tumor markers as clinical tools in evaluation of mucinous carcinoma of the appendix. \n",
      "Carmignani et al (2004)  evaluated patients with synchronous systemic and intraperitoneal dissemination of appendix cancer treated with cytoreductive surgery and perioperative regional chemotherapy with a mean follow up time of 42.6 months.  Results of this study indicated that patients with elevated CEA and CA 19-9 levels had a shorter median survival time (p=0.0083 and p = 0.0193, respectively).  In a subsequent study, Carmingnani et al (2004) prospectively recorded tumor markers CEA and CA19-9 within 1 week prior to definitive treatment.  The investigators found CEA elevated in 56% of 532 patients and CA19-9 elevated in 67.1% of those patients.  They reported that \"although the absolute level of tumor marker did not correlate with prognosis, a normal value indicated an improved survival.\"  Their findings included an elevated CEA in 35.2% of 110 patients determined to have recurrent disease and an elevated CA 19-9 in 62.9%, while 68.2% of patients had at least one of the tumor markers elevated.\n",
      "Current guidelines indicate that for liver transplantation for primary sclerosing cholangitis, stringent efforts should be made to detect superadded cholangiocarcinoma, including measurement of CA 19-9 (Devlin & O'Grady, 1999).\n",
      "Carmignani et al (2004a) conducted a study to report the role of combined treatments, including cytoreductive surgery and perioperative regional chemotherapy, in patients with synchronous systemic and intraperitoneal dissemination of appendix cancer. Study subjects were treated with cytoreductive surgery and perioperative regional chemotherapy and statistical analysis of variables utilized survival as an end point and included demographic characteristics, prior surgical score (PSS), tumor marker levels, peritoneal cancer index (PCI), and completeness of cytoreduction (CC). With a mean follow-up of 42.6 months, median survival time (MST) for 15 patients was 28 months and 5-year survival rate was 29.4 %. Female patients had a longer MST than male patients (p = 0.0199) and survival was better in patients with PSS 0 and 1 (p = 0.0277). Patients with elevated CEA and CA 19-9 levels had a shorter MST (p = 0.0083 and p = 0.0193, respectively) while PCI and CC comparisons did not show significant differences. The morbidity rate (n = 2) was 13.3 % and the mortality (n = 2) rate was also 13.3 %. The authors concluded that \"acceptable morbidity and mortality and a 29.4 % 5-year survival rate allows cytoreductive surgery and regional chemotherapy to be considered as a treatment option for selected patients with synchronous systemic and intraperitoneal dissemination of appendix cancer.\"\n",
      "Carmignani et al (2004b) in a further publication regarding gastrointestinal cancer, stated that carcinoembryonic antigen (CEA) and carbohydrate antigen 19-9 (CA 19-9) tumor markers have found selected clinical application. The authors remarked that the use of these tumor markers in mucinous epithelial tumors of the appendix has not been previously determined. Thus, the authors conducted a study in which, in patients with peritoneal dissemination of a mucinous epithelial malignancy of the appendix, tumor markers CEA and CA 19-9 were prospectively recorded preoperatively within 1 week prior to definitive treatment and if the appendiceal tumor recurred, the tumor marker was determined. The primary endpoint was the accuracy of these two tumor markers in the management of this disease for these two specific clinical situations. CEA was elevated in 56 % of 532 patients and CA 19-9 was elevated in 67.1 % of these patients. Although the absolute level of tumor marker did not correlate with prognosis, a normal value indicated an improved survival. CEA was elevated in 35.2 % of 110 patients determined to have recurrent disease and CA 19-9 was elevated in 62.9 %. At least one of the tumor markers was elevated in 68.2 % of patients. An elevated CEA tumor marker at the time of recurrence indicated a reduced prognosis and both CEA and CA 19-9 tumor markers were elevated in a majority of these patients. This should be a valuable diagnostic tool previously underutilized in this group of patients. These tumor markers were also of benefit in the assessment of prognosis in that a normal level indicated an improved prognosis. At the time of a reoperative procedure, CEA and CA 19-9 tumor markers gave information regarding the progression of disease and have practical value in the management of epithelial appendiceal malignancy with peritoneal dissemination.\n",
      "Andreopoulou et al (2007) stated that mucinous carcinoma of the appendix is a rare entity with a distinct natural history that poses diagnostic and therapeutic challenges and that mucinous peritoneal carcinomatosis is most commonly associated with primary tumors of the appendix and colon. The authors stated that usually the spread remains confined to the abdominal cavity and that imaging assessment of these mucinous lesions is difficult, while tumor markers (CEA and CA19.9) may be surrogates for extent of disease. \n",
      "Recruitment for large scale studies given the rare nature of mucinous appendiceal carcinoma would be challenging.  However, available evidence does illustrate a benefit to use of CA 19-9 in patients with mucinous appendiceal carcinoma.\n",
      "National Comprehensive Cancer Networkâs clinical practice guideline on âHepatobiliary cancersâ (Version 1.2021) states that CEA and Ca 19-9 are baseline tests, and should not be performed to confirm diagnosis of gallbladder cancer, or cholangiocarcinoma (extra-hepatic or intra-hepatic).\n",
      "An UpToDate review on âTumors of the nasal cavityâ (Dagan et al, 2021) does not mention CA 19-9.\n",
      "Furthermore, National Comprehensive Cancer Networkâs Biomarkers Compendium (2021) does not list NUT midline carcinoma tumor of the nasal cavity to be associated with CA 19-9 expression.\n",
      "Cathepsins\n",
      "This enzyme plays a critical role in protein catabolism and tissue remodeling (Chin, et al., 2006). Over-expression is associated with non-ductal carcinoma and metastasis at the time of breast cancer diagnosis. High levels may have clinical significance in predicting decreased metastasis-free survival and decreased overall survival in women with node-negative breast cancer.\n",
      "Svatek et al (2008) examined the role of urinary cathepsin B and L in the detection of bladder urothelial cell carcinoma.  These investigators concluded that urinary cathepsin L is an independent predictor of bladder cancer presence and invasiveness in patients with a history of urothelial carcinoma of the bladder.  They stated that further evaluation of this marker is necessary before its use as an adjunct to cystoscopy for urothelial carcinoma of the bladder.\n",
      "CD 20\n",
      "CD 20 is used to determine eligibility for rituximab (Rituxan; anti-CD20) treatment in patients with B-cell non-Hodgkin's lymphomas (NHL) (Chin, et al., 2006). Rituximab is a genetically engineered, chimeric murine/human monoclonal antibody directed against the CD20 antigen found on the surface of normal and malignant B-cell lymphocytes. Since non-Hodgkin's Lymphoma (NHL) subtypes may differ in their response to rituximab, determination of drug sensitivity is important for choosing therapy.\n",
      "CD 25\n",
      "CD 25 is used to determine eligibility for denileukin diftitox treatment in patients with persistent or recurrent CTCL (Chin, et al., 2006). Denileukin diftitox (Ontak) is a cutaneous T-cell lymphoma (CTCL) therapy that targets the high-affinity interleukin-2 (IL-2) receptor. The IL-2 receptor may exist in a low-affinity form (CD25), an intermediate-affinity form (CD122/CD132), and a high-affinity form (CD25/CD122/CD132). Patients whose malignant cells express the CD25 component of the IL-2 receptor may respond to Ontak therapy.\n",
      "CD 33\n",
      "CD 33 is used to determine eligibility for gemtuzumab (Mylotarg, anti-CD33) treatment in patients with acute myeloid leukemia (Chen, et al., 2006). Gemtuzumab consists of a recombinant, humanized IgG kappa antibody conjugated to a cytotoxic anti-tumor antibiotic, calicheamicin, which binds specifically to the CD33 antigen. This antigen is found on the surface of leukemic blasts and immature normal cells of myelomonocytic lineage, but not in normal hematopoietic stem cells.\n",
      "CD 52\n",
      "CD 52 is used to determine eligibility for alemtuzumab (Campath, anti-CD52) treatment in patients with chronic lymphocytic leukemia (Chen, et al., 2006). CD52 is an antigen that can be expressed at high density on the surface of malignant CLL cells. Alemtuzumab is a humanized antibody targeted against CD52 and its binding is necessary for cell death and therapeutic response.\n",
      "CD 117, c-kit\n",
      "CD 117 is used to determine eligibility for treatment with imatinib mesylate in patients with c-kit-positive gastrointestinal stromal tumors (GISTs) (Chen, et al., 2006). The glycoprotein c-kit (CD117) is a member of the receptor tyrosine kinase subclass III family and has been implicated in a number of malignancies. Imatinib mesylate, a tyrosine kinase inhibitor, is effective in treating GISTs and other tumors that express c-kit.\n",
      "HCG\n",
      "Human chorionic gonadotropin (HCG) is normally produced in increasing quantities by the placenta during pregnancy. Accepted guidelines provide that HCG levels can be used to screen for choriocarcinoma in women who are at high risk for the disease, and to monitor the treatment of trophoblastic disease. The literature states that elevated HCG levels may also indicate the presence of cancers of the testis, ovary, liver, stomach, pancreas, and lung.\n",
      "Accepted guidelines provide that alpha fetoprotein (AFP) and b-HCG measurements are valuable for determining prognosis and monitoring therapy in patients with non-seminomatous germ cell cancer. Because of the low incidence of elevated AFP and b-HCG levels in early-stage cancer, the literature suggests these markers have no value in screening for testicular cancer. However, the specificity of these markers is such that when determined simultaneously, at least one marker will be positive in 85% of patients with active cancer. The value of AFP and b-HCG as markers is enhanced by a low frequency of false-positive results and by the chemoresponsiveness of testicular cancer. The literature states that only rarely do patients with other types of cancer have elevated levels of AFP. Non-cancerous conditions that can cause elevated AFP levels include benign liver conditions, such as cirrhosis or hepatitis, ataxia telangiectasia, Wiscott-Aldrich syndrome, and pregnancy.\n",
      "AFP\n",
      "Alpha-fetoprotein (AFP) is a protein that is normally elevated in pregnant women since it is produced by the fetus; however, AFP is not usually found in the blood of adults. In men and in women who are not pregnant, an elevated level of AFP may indicate liver, ovarian or testicular cancer. \n",
      "Alpha-fetoprotein is normally produced by a developing fetus.  Alpha fetoprotein levels begin to decrease soon after birth and are usually undetectable in the blood of healthy adults, except during pregnancy. According to accepted guidelines, an elevated level of AFP strongly suggests the presence of either primary liver cancer or germ cell cancer of the ovary or testicle. As AFP is an established test for the diagnosis and monitoring of hepatoma, it is used as a screening tool to rule out the presence of a liver neoplasm before considering liver transplantation. This is especially pertinent in cases (e.g., cirrhosis) where there is an increased risk of developing a primary liver tumor.\n",
      "Elevated serum AFP levels are most closely associated with nonseminomatous testicular cancer and hepatocellular cancer (Chin, 2006). The rate of clearance from serum after treatment is an indicator of the effectiveness of therapy. Conversely, the growth rate of progressive disease can be monitored by serially measuring serum AFP concentrations over time.\n",
      "ER, PR\n",
      "Estrogen receptor (ER) and progesterone receptor (PR) predicts response to hormone therapy for women with advanced breast cancer and those receiving adjuvant treatment, and prognosticates the aggressiveness of a tumor (Chin, 2006).\n",
      "The estrogen receptor and progesterone receptor are intracellular receptors that are measured directly in tumor tissue. These receptors are polypeptides that bind their respective hormones, translocate to the nucleus, and induce specific gene expression. Breast cancers are dependent upon estrogen and/or progesterone for growth and this effect is mediated through ERs and progesterone receptors (ER/PR) (Chin, et al., 2006). Both receptors may be over-expressed in malignant breast tissue. Most oncologists have used the estrogen receptor and also the progesterone receptor not only to predict the probability of response to hormonal therapy at the time of metastatic disease, but also to predict the likelihood of recurrent disease, and to predict the need for adjuvant hormonal therapy or chemotherapy. Although these latter uses for estrogen and progesterone receptors are commonly accepted by most oncologists, the data on which these conclusions are based are controversial.\n",
      "NSE\n",
      "Neuron-specific enolase (NSE) has been detected in patients with neuroblastoma, small cell lung cancer, Wilms' tumor, melanoma, and cancers of the thyroid, kidney, testicle, and pancreas. However, studies of NSE as a tumor marker have concentrated primarily on patients with neuroblastoma and small cell lung cancer. According to the available literature, measurement of NSE level in patients with these diseases cannot be correlated to the extent of the disease, the patient's prognosis, or the patient's response to treatment because of the poor sensitivity of this marker.\n",
      "LASA\n",
      "LASA is a complex marker that measures the amount of sialic acid in serum and can be elevated in serum from patients with any number of different neoplasms. Elevations in blood LASA levels have been reported in patients with mammary (63 percent), gastroenteric (65 percent), pulmonary (79 percent), and ovarian (94 percent) neoplasms as well as those with leukemia (86 percent), lymphoma (87 percent), melanoma (84 percent), sarcoma (97 percent), and Hodgkin disease (91 percent). As a result, this assay may not have high specificity or sensitivity necessary for cancer detection (Chen, et al., 2006). This serum cancer marker has not been widely accepted for use in the detection or prognosis of colorectal carcinoma. There is no practical information concerning outcome and the use of LASA in the medical literature. Although several articles describe the use of LASA in the diagnosis of colorectal cancer and its association with tumor-node-metastasis (TNM) stage, it has been shown that patients with colorectal polyps and colorectal carcinoma both have elevated LASA levels, and that the levels returned to baseline after removal of either polyps or carcinomas.\n",
      "p53\n",
      "p53 is a tumor suppressor gene on the short arm of chromosome 17 that encodes a protein that is important in the regulation of cell division. Although the full role of p53 in the normal and neoplastic cell is unknown, there is evidence that the gene product is important in preventing the division of cells containing damaged DNA. p53 gene deletion or mutation is a frequent event along with other molecular abnormalities in colorectal carcinogenesis. The literature on p53 abnormality and prognosis in colorectal cancer suffers from a paucity of reported data and the use of a variety of techniques in assay and statistical analysis in the small numbers of cases analyzed. For these reasons, the literature generally does not recommend p53 analysis as a routine approach to assisting in the management of patients with colorectal cancer.\n",
      "Guidelines from the American Society for Clinical Oncology (2016) recommend against the use of p53 to guide adjuvant chemotherapy in  breast cancer. This is a moderate-strength recommendation based upon intermediate-quality evidence.\n",
      "Zap-70\n",
      "Zeta-chain-associated protein kinase 70, which is used as a prognostic marker in (CLL).\n",
      "Zap-70 is indicated to assess prognosis and need for aggressive therapy in patients with chronic lymphocytic leukemia (CLL) (Chin, et al., 2006). ZAP-70 is a 70-kD member of the Syk family of protein tyrosine kinases. It is expressed primarily in T-cells and natural killer (NK) cells and is critical for signal transduction following T-cell receptor engagement. In CLL B-cells, elevated ZAP-70 expression appears to predict the need for therapy as effectively as IgVH mutation status. Although ZAP-70 expression is strongly correlated with IgVH mutation status, the combination of the two markers may provide greater prognostic value than either marker alone. Positive ZAP-70 results predict an aggressive disease course.\n",
      "uPA\n",
      "The serine protease urokinase-type plasminogen activator (uPA) and its primary inhibitor, plasminogen activator inhibitor-1 (PAI-1), have shown promise for risk assessment and prediction of therapeutic response in primary breast cancer (Chin, et al., 2006). High levels of uPA or PAI-1 in primary tumor tissue are associated with an aggressive disease course and poor prognosis in both node-positive and node-negative breast cancer.\n",
      "A report by the Belgian Healthcare Knowledge Centre (KCE) (San Miguel, et al., 2015) found no studies reporting on the impact of uPA/PAI-1 on clinical management (clinical utility). \n",
      "Guidelines from the American Society for Clinical Oncology (2016) state: \"If a patient has ER/PgR-positive, HER2-negative (node-negative) breast cancer, the clinician may use urokinase plasminogen activator and plasminogen activator inhibitor type 1 to guide decisions on adjuvant systemic therapy.\" This is a weak recommendation based upon high-quality evidence.  The ASCO guidelines recommend the use of urokinase plasminogen activator and plasminogen activator inhibitor type 1 to guide decisions on adjuvant systemic therapy in patients with HER2-positive breast cancer or TN breast cancer.\n",
      "IgVh Mutation Status\n",
      "Chronic lymphocytic leukemia (CLL) patients can be divided into two basic groups on the basis of the mutational status of the immunoglobulin heavy-chain variable-region (IgVH) gene in leukemic cells (Chin, 2006). Patients with IgVH mutations have longer survival than those without IgVH mutation. Thus, mutation analysis may be useful for planning management strategies.\n",
      "Kappa / Lambda Light Chain\n",
      "Elevated serum levels of monoclonal free light chains are associated with malignant plasma cell proliferation (e.g., multiple myeloma), primary amyloidosis, and light chain deposition disease (Chen et al, 2006). The appearance of higher levels of free light chains in the urine may be indicative of kidney disease or malignant lymphoproliferative disease such as multiple myeloma. These tests have been used for the detection of multiple myeloma.\n",
      "KRAS\n",
      "The ras proto-oncogenes are normal cellular components, which are thought to be important for transduction of signals required for proliferation and differentiation.   The ras oncogene family has 3 members: H-ras, K-ras, and N-ras.  Ras gene mutations can be found in a variety of tumor types, although the incidence varies greatly. The highest incidences are found in adenocarcinomas of the pancreas (90 %), colon (50 %), and lung (30 %); thyroid tumors (50 %), and myeloid leukemia (30 %).\n",
      "Investigators have established an association between some genotypes of K-ras (KRAS) oncogenes and response to treatment with cetuximab or panitumumab (Lievre et al, 2006 and 2008; Di Fiore et al, 2007; GonÃ§alves et al, 2008; De Roock et al, 2008).  Patients whose tumors express specific forms of the KRAS gene exhibit considerably decreased responses to cetuximab and panitumumab.  It has been theorized that cetuximab and panitumumab do not target epidermal growth factor receptor (EGFR) associated with these specific KRAS mutations and thus are unable to block their activation.  It has been suggested that KRAS genotype be considered as a selection factor for cancer patients who are candidates for treatment with cetuximab or panitumumab.\n",
      "Karapetis and colleagues (2008) stated that treatment with cetuximab improves overall survival (OS) and progression-free survival (PFS) and preserves the quality of life in patients with colorectal cancer that has not responded to chemotherapy. The mutation status of the K-ras gene in the tumor may affect the response to cetuximab and have treatment-independent prognostic value.  These investigators analyzed tumor samples, obtained from 394 of 572 patients (68.9 %) with colorectal cancer who were randomly assigned to receive cetuximab plus best supportive care or best supportive care alone, to look for activating mutations in exon 2 of the K-ras gene.  They evaluated if the mutation status of the K-ras gene was associated with survival in the cetuximab and supportive-care groups.  Of the tumors evaluated for K-ras mutations, 42.3 % had at least one mutation in exon 2 of the gene.  The effectiveness of cetuximab was significantly associated with K-ras mutation status (p = 0.01 and p < 0.001 for the interaction of K-ras mutation status with OS and PFS, respectively).  In patients with wild-type K-ras tumors, treatment with cetuximab as compared with supportive care alone significantly improved OS (median of 9.5 versus 4.8 months; hazard ratio for death, 0.55; 95 % confidence interval [CI], 0.41 to 0.74; p < 0.001) and PFS (median of 3.7 months versus 1.9 months; hazard ratio for progression or death, 0.40; 95 % CI, 0.30 to 0.54; p < 0.001).  Among patients with mutated K-ras tumors, there was no significant difference between those who were treated with cetuximab and those who received supportive care alone with respect to OS (hazard ratio, 0.98; p = 0.89) or PFS (hazard ratio, 0.99; p = 0.96).  In the group of patients receiving best supportive care alone, the mutation status of the K-ras gene was not significantly associated with OS (hazard ratio for death, 1.01; p = 0.97).  The authors concluded that patients with a colorectal tumor bearing mutated K-ras did not benefit from cetuximab, whereas patients with a tumor bearing wild-type K-ras did benefit from cetuximab.  The mutation status of the K-ras gene had no influence on survival among patients treated with best supportive care alone.\n",
      "The ASCO's provisional clinical opinion on testing for KRAS gene mutations in patients with metastatic colorectal carcinoma to predict response to anti-EGFR monoclonal antibody therapy (Allegra et al, 2009) stated that based on systematic reviews of the relevant literature, all patients with metastatic colorectal carcinoma who are candidates for anti-EGFR antibody therapy should have their tumor tested for KRAS mutations in a CLIA-accredited laboratory.  If KRAS mutation in codon 12 or 13 is detected, then patients with metastatic colorectal carcinoma should not receive anti-EGFR antibody therapy as part of their treatment.\n",
      "The KRAS oncogene mutation tests are intended to aid in the formulation of treatment decisions for patients who may be candidates for treatment of metastatic epithelial cancers with anti-EGFR therapies such as cetuximab or panitumumab.  Several tests for KRAS mutation are currently available in the United States; however, at this time, no KRAS genotype test kits have been approved by the FDA.\n",
      "At the 2008 Annual Meeting of the American Society of Clinical Oncology (ASCO), data on 540 patients with metastatic colorectal cancer in the randomized, phase III CRYSTAL trial were presented.  Among 192 patients with KRAS mutations, there was no improvement in overall responses or PFS from the addition of cetuximab to standard chemotherapy.  In the patients with normal KRAS, the 1-year PFS rate was 43 % for patients receiving cetuximab versus 25 % for those receiving only standard chemotherapy, and the overall response rate was 59 % versus 43 %, respectively (van Cutsem, 2008).  Also at the 2008 ASCO meeting, data from 233 metastatic colorectal cancer patients were presented that confirmed the correlation of KRAS status with patient response to anti-EGFR therapy.  No benefit was found after addition of cetuximab to standard chemotherapy with FOLFOX (the combination of fluorouracil, leucovorin, and oxaliplatin) in patients with a mutated KRAS; however, addition of cetuximab to FOLFOX increased both response rate and PFS in patients with a wild-type (i.e., un-mutated) KRAS gene (Bokemeyer, 2008).  Response to panitumumab was correlated to KRAS status in a published phase III trial.  A total of 427 patients with metastatic colorectal cancer received either panitumumab or best supportive care.  Panitumumab exhibited a 17% response rate among patients with normal KRAS, but 0% response among patients with KRAS mutations (Amado, 2008).\n",
      "A meta-analysis of results from 8 studies involving 817 patients with colorectal cancer found that the presence of KRAS mutation predicted lack of response to treatment with anti-EGFR monoclonal antibodies (e.g., panitumumab or cetuximab), whether as stand-alone therapy or in combination with chemotherapy (Linardou et al, 2008).  This analysis also provided empirical evidence that k-RAS mutations are highly specific negative predictors of response (de-novo resistance) to single-agent EGFR tyrosine-kinase inhibitors in advanced non-small cell lung cancer; and similarly to anti-EGFR monoclonal antibodies alone or in combination with chemotherapy in patients with metastatic colorectal cancer.\n",
      "The Blue Cross and Blue Shield Association (BCBSA, 2008) Technology Evaluation Center Medical Advisory Panel concluded that use of KRAS mutation analysis meets TEC criteria to predict non-response to anti-EGFR monoclonal antibodies cetuximab and panitumumab to treat metastatic colorectal cancer. The TEC assessment found that the evidence is sufficient to conclude that patients with mutated KRAS tumors in the setting of metastatic colorectal cancer do not respond to anti-EGFR monoclonal antibody therapy. The assessment explained that the data show that the clinical benefit of using EGFR inhibitors in treating metastatic colorectal cancer, either as monotherapy or in combination with other treatment regimens, is not seen in patients with KRAS-mutated tumors.  The assessment found: \"This data supports knowing a patient's tumor mutation status before consideration of use of an EGFR inhibitor in the treatment regimen.  Identifying patients whose tumors express mutated KRAS will avoid exposing patients to ineffective drugs, avoid exposure to unnecessary drug toxicities, and expedite the use of the best available alternative therapy.\"\n",
      "Colorectal cancer guidelines from the National Comprehensive Cancer Network (NCCN, 2010) recommend consideration of reflex BRAF testing in patients with wild type KRAS. The NCCN guidelines explain that several small studies suggest that patients with wild-type KRAS and a BRAF mutation are unlikely to respond to anti-EGFR therapies such as cetuximab and panitumumab. The guidelines explain that patients with a known BRAF mutation are unlikely to respond to anti-EGFR antibodies, although the data are somewhat inconsistent. Studies demonstrate that in patients with metastatic colorectal cancer, about 8 percent have mutations in the BRAF gene. Testing for the BRAF V600E mutation is performed by PCR amplification and direct DNA sequence analysis.\n",
      "Ratner et al (2010) stated that ovarian cancer (OC) is the single most deadly form of women's cancer, typically presenting as an advanced disease at diagnosis in part due to a lack of known risk factors or genetic markers of risk.  The KRAS oncogene and altered levels of the microRNA (miRNA) let-7 are associated with an increased risk of developing solid tumors.  In this study, these researchers investigated a hypothesized association between an increased risk of OC and a variant allele of KRAS at rs61764370, referred to as the KRAS-variant, which disrupts a let-7 miRNA binding site in this oncogene.  Specimens obtained were tested for the presence of the KRAS-variant from non-selected OC patients in 3 independent cohorts, 2 independent ovarian case-control studies, and OC patients with hereditary breast and ovarian cancer syndrome (HBOC) as well as their family members.  The results indicated that the KRAS-variant is associated with more than 25 % of non-selected OC cases.  Furthermore, these researchers found that it is a marker for a significant increased risk of developing OC, as confirmed by 2 independent case-control analyses.  Lastly, they determined that the KRAS-variant was present in 61 % of HBOC patients without BRCA1 or BRCA2 mutations, previously considered uninformative, as well as in their family members with cancer.  These findings supported the hypothesis that the KRAS-variant is a genetic marker for increased risk of developing OC, and they suggested that the KRAS-variant may be a new genetic marker of cancer risk for HBOC families without other known genetic abnormalities.\n",
      "Hollestelle et al (2011) noted that recently, a variant allele in the 3'UTR of the KRAS gene (rs61764370 T>G) was shown to be associated with an increased risk for developing non-small cell lung cancer, as well as OC, and was most enriched in OC patients from HBOC families.  This functional variant has been shown to disrupt a let-7 miRNA binding site leading to increased expression of KRAS in vitro.  In the current study, these investigators genotyped this KRAS-variant in breast cancer index cases from 268 BRCA1 families, 89 BRCA2 families, 685 non-BRCA1/BRCA2 families, and 797 geographically matched controls.  The allele frequency of the KRAS-variant was found to be increased among patients with breast cancer from BRCA1, but not BRCA2 or non-BRCA1/BRCA2 families as compared to controls.  As BRCA1 carriers mostly develop ER-negative breast cancers, these researchers also examined the variant allele frequency among indexes from non-BRCA1/BRCA2 families with ER-negative breast cancer.  The prevalence of the KRAS-variant was, however, not significantly increased as compared to controls, suggesting that the variant allele not just simply associates with ER-negative breast cancer.  Subsequent expansion of the number of BRCA1 carriers with breast cancer by including other family members in addition to the index cases resulted in loss of significance for the association between the variant allele and mutant BRCA1 breast cancer.  In this same cohort, the KRAS-variant did not appear to modify breast cancer risk for BRCA1 carriers.  More importantly, results from the current study suggested that KRAS-variant frequencies might be increased among BRCA1 carriers, but solid proof requires confirmation in a larger cohort of BRCA1 carriers. \n",
      "Therascreen KRAS RGQ PCR Kit (Qiagen) is intended to detect 7 mutations in codons 12 and 13 of the KRAS gene (Raman, et al., 2013). The kit utilizes two technologies â ARMS and Scorpions â for detection of mutations in real-time PCR. The therascreen KRAS RGQ PCR kit is being developed as a companion diagnostic to aid clinicians, through detection of KRAS mutations, in the identification of patients with metastatic colorectal cancer (mCRC) who are more likely to benefit from cetuximab.\n",
      "PreOvarâ¢ tests (Mira Dx) for the KRAS-variant, and will help identify ovarian cancer patients whose female relatives should also be evaluated for the KRAS-variant (Raman, et al., 2013). PreOvarâ¢ may also help assess the relative risk of developing ovarian cancer for women who have a family history of ovarian cancer without a living proband (ancestor with the disease). The KRAS-Variant is present in 6-10% of the general population and 25% of non-selected women with epithelial ovarian cancer. Additionally, the KRAS-variant was identified in over 60% of Hereditary Breast and Ovarian Cancer (HBOC) patients that were previously classified as \"uninformative,\" or negative for other known genetic markers of ovarian cancer risk. The test determines if KRAS-variant may put someone at increased risk for developing ovarian cancer.\n",
      "The Evaluation of Genomic Applications in Practice and Prevention (EGAPP) Working Group (EWG) (2013) found that, for patients with metastatic colorectal cancer (mCRC) who are being considered for treatment with cetuximab or panitumumab, there is convincing evidence to recommend clinical use of KRAS mutation analysis to determine which patients are KRAS mutation positive and therefore unlikely to benefit from these agents before initiation of therapy. The level of certainty of the evidence was deemed high, and the magnitude of net health benefit from avoiding potentially ineffective and harmful treatment, along with promoting more immediate access to what could be the next most effective treatment, is at least moderate.\n",
      "The EWG found insufficient evidence to recommend for or against BRAF V600E testing for the same clinical scenario (EGAPP, 2013). The level of certainty for BRAF V600E testing to guide antiepidermal growth factor receptor (EGFR) therapy was deemed low. The EWG encourages further studies of the potential value of testing in patients with mCRC who were found to have tumors that are wild type (mutation negative) for KRAS to predict responsiveness to therapy.\n",
      "Bladder Cancer: BTA-stat, NMP22, Urovysion, ImmunoCyt\n",
      "In the United States, bladder malignancy is the 4th commonest cancer in men and the 8th commonest in women.  Patients usually present with urinary tract symptoms (e.g., gross or microscopic hematuria or irritative voiding symptoms such as frequency, dysuria, and urgency).  Evaluations of these patients usually entail voided-urine cytology, cystoscopy, and upper urinary tract imaging such as intravenous pyelography, renal sonography, or retrograde pyelography.  Most newly diagnosed bladder cancers are superficial (i.e., not invading beyond the lamina propria on histological examination), and are known as transitional cell carcinoma (TCC).  These superficial bladder cancers are usually managed by transurethral resection.  However, the literature shows that approximately 50 to 75 % of treated TCC recur.  Furthermore, 10 to 15 % of TCC progress to muscle-invasive bladder cancer.  According to the literature, the prevalence of recurrence after initial treatment as well as the natural history of TCC necessitates long-term follow-up.  Following treatment, accepted guidelines provide that patients who have previously been diagnosed with TCC should usually undergo urine cytology/cystoscopy every 3 months in the 1st year, every 6 months in the 2nd year, and once-yearly afterwards.\n",
      "Currently, urine cytology with confirmatory cystoscopy represents the cornerstone for the identification of bladder tumors. However, the subjectivity and low sensitivity of cytology led to the development of several urine-based tests as adjuncts to cytology/cystoscopy for the diagnosis and follow-up of patients with TCC. These tests include the BTA Stat test (Bard Diagnostic, Redmond, WA), the NMP22 test (Matritech, Newton, MA), the Aura-Tek FDP test (PerImmune, Rockville, MD), and the Vysis UroVysion FISH Test (Vysis, Inc., Downers Grove, IL). They are usually objective, qualitative (BTA Stat and Aura-Tek FDP), or quantitative (NMP22, UroVysion), and have higher sensitivity than cytology, but some have lower specificity. So far, no single bladder tumor marker has emerged as the generally accepted test of choice, and none has been established as a screening tool for bladder malignancy.\n",
      "Urine-based markers, such as proteins with increased cancer cell expression or chromosomal abnormalities in the urine, may be detected using a variety of laboratory methods to aid in the management of bladder cancer. The following markers/tests are currently available:\n",
      "\n",
      "Bladder tumor antigen (BTA) (eg, BTA stat and BTA TRAK)\n",
      "Fluorescence immunocytology (eg, ImmunoCyt/uCyt+)\n",
      "Fluorescence in situ hybridization (FISH) (eg, UroVysion)\n",
      "mRNA quantification by RT-qPRC testing (eg, Cxbladder)\n",
      "Nuclear matrix protein 22 (NMP22) (eg, NMP22 BladderChek and Matritech NMP22 Test).\n",
      "\n",
      "Urine-based markers have a role in the detection of bladder cancer recurrence in individuals with a history of bladder cancer and are used adjunctively with urinary cytology and cystoscopy. These tests have also been proposed for bladder cancer screening, diagnosis of bladder cancer in individuals symptomatic of bladder cancer and for the evaluation of hematuria. \n",
      "The UroVysion Bladder Cancer Kit (UroVysion Kit) (Baycare Laboratories) is designed to detect aneuploidy for chromosomes 3, 7, 17, and loss of the 9p21 locus via fluorescence in situ hybridization (FISH) in urine specimens from persons with hematuria suspected of having bladder cancer (Raman, et al., 2013). FISH analysis is used in conjunction with cystoscopy to monitor for recurrence among those with previously diagnosed bladder cancer. FISH analysis is a surveillance tool in established primary and secondary bladder adenocarcinoma.\n",
      "The ImmunoCyt is an immunocytochemistry assay for the detection of tumor cells shed in the urine of patients previously diagnosed with bladder cancer (Chen, et al., 2006). This test is intended to augment the sensitivity of cytology for the detection of tumor cells in the urine of individuals previously diagnosed with bladder cancer. The test has been used for detection of tumor cells in the urine of individuals previously diagnosed with bladder cancer, and for use in conjunction with cytoscopy as an aid in the management of bladder cancer.\n",
      "Although urine cytology has been shown to be less accurate than urinary biomarker tests, familiarity with the method as well as ease of performance justify the continued routine use of the former by primary care physicians, especially in patients who have no history of bladder malignancy. The urine-based biomarker tests have been shown to be accurate in detecting low-grade bladder tumors. In particular, these tests may be of help in deciding the need for further diagnostic assessment of patients with a history of bladder cancer and negative results on urine cytology. For example, elevated levels of urinary bladder tumor markers in patients with a history of TCC may warrant earlier, rather than delayed, cystoscopic examination. On the other hand, consideration may be given to lengthening the intervals between cystoscopic investigations when values of these tumor markers are normal.\n",
      "An assessment by the Adelaide Health Technology Assessment (Mundy & Hiller, 2009) concluded that the NMP BladderCheck and UroVysion FISH assay, designed for the detection of bladder cancer in high risk patients, have poor sensitivity and poor positive predictive values. The assessment recommended that these assays not be used in asymptomatic patients. The assessment suggested, however, that these tests may be useful in the monitoring of patients with transitional cell carcinoma between cytoscopies. The AHTA recommended that this technology not be assessed further.\n",
      "An assessment prepared for the Agency for Healthcare Research and Quality (Meleth, et al., 2014) found: \"Although UroVysion is marketed as a diagnostic rather than a prognostic test, limited evidence from two small studies (total N=168) rated as low or medium risk of bias supported associations between test result and prognosis for risk of recurrence. We found no studies that directly assessed the impact of a test of interest on both physician decision-making and downstream health outcomes to establish clinical utility. We attempted to construct an indirect chain of evidence to answer the overarching question, but we were unable to do so. Even in the cases where the tests seemed to add value in determining prognosis (i.e., evidence of clinical validity), we found no evidence that using the test was related to improved outcomes for patients.\"\n",
      "The American Urologic Associationâs guideline on \"Diagnosis, evaluation and follow-up of asymptomatic microhematuria (AMH) in adults\" (Davis et al, 2012) stated that \"The use of urine cytology and urine markers (Nuclear Matrix Protein 22 [NMP22], bladder tumor antigen [BTA]-stat, and UroVysion fluorescence in situ hybridization assay [FISH]) is not recommended as a part of the routine evaluation of the asymptomatic microhematuria patient\".\n",
      "Chou et al (2015) systematically reviewed the evidence on the accuracy of urinary biomarkers for diagnosis of bladder cancer in adults who have signs or symptoms of the disease or are undergoing surveillance for recurrent disease. Data sources included Ovid MEDLINE (January 1990 through June 2015), Cochrane Central Register of Controlled Trials, Cochrane Database of Systematic Reviews, and reference lists.  A total of 57 studies that evaluated the diagnostic accuracy of quantitative or qualitative nuclear matrix protein 22 (NMP22), qualitative or quantitative bladder tumor antigen (BTA), FISH, fluorescent immunohistochemistry (ImmunoCyt [Scimedx]), and Cxbladder (Pacific Edge Diagnostics USA) using cystoscopy and histopathology as the reference standard met inclusion criteria; case-control studies were excluded.  Dual extraction and quality assessment of individual studies were carried out; overall strength of evidence (SOE) was also assessed.  Across biomarkers, sensitivities ranged from 0.57 to 0.82 and specificities ranged from 0.74 to 0.88.  Positive likelihood ratios ranged from 2.52 to 5.53, and negative likelihood ratios ranged from 0.21 to 0.48 (moderate SOE for quantitative NMP22, qualitative BTA, FISH, and ImmunoCyt; low SOE for others).  For some biomarkers, sensitivity was higher for initial diagnosis of bladder cancer than for diagnosis of recurrence.  Sensitivity increased with higher tumor stage or grade.  Studies that directly compared the accuracy of quantitative NMP22 and qualitative BTA found no differences in diagnostic accuracy (moderate SOE); head-to-head studies of other biomarkers were limited.  Urinary biomarkers plus cytologic evaluation were more sensitive than biomarkers alone but missed about 10 % of bladder cancer cases.  The authors concluded that urinary biomarkers miss a substantial proportion of patients with bladder cancer and are subject to false-positive results in others; accuracy is poor for low-stage and low-grade tumors.  They stated that research is needed to understand how the use of these biomarkers with other diagnostic tests affect the use of cystoscopy and clinical outcomes.\n",
      "In an editorial that accompanied the afore-mentioned study, Abbosh and Plimack (2015) stated that \"Until urinary biomarkers become available that are sufficiently accurate to supplant the current recommended detection algorithms in biomarker-negative patients, they will not be a cost-effective addition to strategies to detect bladder cancer\".\n",
      "In summary, urine-based bladder tumor marker tests have been shown to be useful as an adjunct to urine cytology and cystoscopy in monitoring for recurrences of bladder cancer, but according to the available literature should not be used as a screening tool for bladder malignancy.  The U.S. Preventive Services Task Force (USPSTF, 2004) has concluded that the potential harms of screening for bladder cancer using available tests, such as microscopic urinalysis, urine dipstick, urine cytology, or such new tests as bladder tumor antigen (BTA) or nuclear matrix protein (NMP22) immunoassay, outweigh any potential benefits.\n",
      "Cxbladder\n",
      "O'Sullivan and colleagues (2012) examined if the RNA assay uRNAÂ® and its derivative CxbladderÂ® have greater sensitivity for the detection of bladder cancer than cytology, NMP22â¢ BladderChekâ¢ and NMP22â¢ ELISA, and whether they are useful in risk stratification.  A total of 485 patients presenting with gross hematuria but without a history of urothelial cancer were recruited prospectively from 11 urology clinics in Australasia.  Voided urine samples were obtained before cystoscopy.  The sensitivity and specificity of the RNA tests were compared to cytology and the NMP22 assays using cystoscopy as the reference.  The ability of Cxbladder to distinguish between low grade, stage Ta urothelial carcinoma and more advanced urothelial carcinoma was also determined.  uRNA detected 41 of 66 urothelial carcinoma cases (62.1 % sensitivity, 95 % confidence interval [CI]: 49.3 to 73.8) compared with NMP22 ELISA (50.0 %, 95 % CI: 37.4 to 62.6), BladderChek (37.9 %, 95 % CI: 26.2 to 50.7) and cytology (56.1 %, 95 % CI: 43.8 to 68.3).  Cxbladder, which was developed on the study data, detected 82 %, including 97 % of the high grade tumors and 100 % of tumors stage 1 or greater.  The cut-offs for uRNA and Cxbladder were pre-specified to give a specificity of 85 %.  The specificity of cytology was 94.5 % (95 % CI: 91.9 to 96.5), NMP22 ELISA 88.0 %, (95 % CI: 84.6 to 91.0) and BladderChek 96.4 % (95 % CI: 94.2 to 98.0).  Cxbladder distinguished between low-grade Ta tumors and other detected urothelial carcinoma with a sensitivity of 91 % and a specificity of 90 %.  The authors concluded that uRNA and Cxbladder showed improved sensitivity for the detection of urothelial carcinoma compared to the NMP22 assays.  Stratification with Cxbladder provides a potential method to prioritize patients for the management of waiting lists.\n",
      "An UpToDate review on \"Clinical presentation, diagnosis, and staging of bladder cancer\" (Lotan and Choueiri, 2013) does not mention the use of mRNA biomarkers/PCR testing as a management tool for bladder cancer.  Furthermore, NCCNâs clinical practice guideline on \"Bladder cancer\" (Version 1.2014) does not mention the use of mRNA biomarkers/PCR testing as a management tool for bladder cancer.\n",
      "An assessment of urinary biomarkers for diagnosis of bladder cancer prepared for the Agency for Healthcare Research and Quality (Chou, et al., 2016) identified only one study of Cxbladder meeting inclusion criteria, graded as moderate quality, with an overall strength of evidence of \"low\".\n",
      "Cxbladder Triage is a urine test designed to rule out the presence of bladder cancer in low-risk patients with hematuria. The test uses an algorithm to incorporate clinical risk factor markers (age, sex, smoking history and macrohematuria frequency) and genetic information (mRNA), using gene-expression profiling by real-time quantitative PCR of 5 biomarker genes (MDK, HOXA13, CDC2 [CDK1], IGFBP5, and CXCR2), to determine a risk score for having urothelial carcinoma.\n",
      "Laukhtina et al (2021) conducted a systematic review and network meta-analysis (NMA) on the diagnostic accuracy of novel urinary biomarker tests (UBTs) in non-nuscle-invasive bladder cancer (NMIBC). PubMed, Web of Science, and Scopus were searched up to April 2021 to identify studies addressing the diagnostic values of UBTs: Xpert bladder cancer, Adxbladder, Bladder EpiCheck, Uromonitor and Cxbladder Monitor, and Triage and Detect. The primary endpoint was to assess the pooled diagnostic values for disease recurrence in NMIBC patients using a DTA meta-analysis and to compare them with cytology using an NMA. The secondary endpoints were the diagnostic values for high-grade (HG) recurrence as well as for the initial detection of bladder cancer. Twenty-one studies, comprising 7330 patients, were included in the quantitative synthesis. In most of the studies, there was an unclear risk of bias. For NMIBC surveillance, novel UBTs demonstrated promising pooled diagnostic values with sensitivities up to 93%, specificities up to 84%, positive predictive values up to 67%, and negative predictive value up to 99%. Pooled estimates for the diagnosis of HG recurrence were similar to those for the diagnosis of any-grade recurrence. The analysis of the number of cystoscopies potentially avoided during the follow-up of 1000 patients showed that UBTs might be efficient in reducing the number of avoidable interventions with up to 740 cystoscopies. The NMA revealed that diagnostic values (except specificity) of the novel UBTs were significantly higher than those of cytology for the detection of NMIBC recurrence. There were too little data on UBTs in the primary diagnosis setting to allow a statistical analysis. The authors concluded that their analyses support high diagnostic accuracy of the studied novel UBTs, supporting their utility in the NMIBC surveillance setting. All of these might potentially help prevent unnecessary cystoscopies safely. There are not enough data to reliably assess their use in the primary diagnostic setting. These results have to be confirmed in a larger cohort as well as in head-to-head comparative studies. \n",
      "OncotypeDx Breast\n",
      "Oncotype Dx (Genomic Health, Inc., Redwood City, CA) is a diagnostic laboratory-developed assay that quantifies the likelihood of breast cancer recurrence in women with newly diagnosed, stage I or II, node negative, estrogen receptor positive breast cancer, who will be treated with tamoxifen.  The assay analyzes the expression of a panel of 21 genes, and is intended for use in conjunction with other conventional methods of breast cancer analysis.  Together with staging, grading, and other tumor marker analyses, Oncotype Dx is intended to provide greater insight into the likelihood of systemic disease recurrence.  Clinical studies have evaluated the prognostic significance of the Oncotype Dx multigene assay in breast cancer (Paik et al, 2004; Esteva et al, 2003). \n",
      "Oncotype Dx analyses the patterns of 21 genes is being applied as a quantification tool for likelihood of breast cancer recurrence within 10 years of newly diagnosed, stage I or II, lymph node-negative, hormone receptor-positive breast cancer in women who will be treated with tamoxifen (Raman, et a., 2013). Oncotype is being applied as a quantification tool for likelihood of breast cancer recurrence in 10 years in women with newly diagnosed breast cancer. It is also intended to assist in making decisions regarding adjuvant chemotherapy based on recurrence likelihood.\n",
      "There currently is a lack of evidence from prospective clinical studies of the impact of this test on the management of women with breast cancer demonstrating improvements in clinical outcomes (Lopez, et al., 2010; Romeo, et al., 2010; Tiwana, et al., 2013; IETS, 2013),  Bast and Hortobagyi (2004) commented that \"[b]efore use of the recurrence score [from the Oncotype Dx multigene assay] is applied to general patient care, however, additional studies are needed.\" The National Cancer Institute is sponsoring a prospective, randomized controlled clinical study, the TAILORx study, using the Oncotype Dx assay to help identify a group of patients with a mid-range risk of recurrence to determine whether treating patients with hormonal therapy only is equivalent to treating them with hormonal therapy in combination with adjuvant chemotherapy.\n",
      "However, there is indirect evidence of the clinical utility of the Oncotype Dx. Paik et al (2006) used banked tumor samples from previous clinical studies of tamoxifen and adjuvant chemotherapy in early breast cancer to assess the performance of the Oncotype Dx multigene assay in predicting response to adjuvant chemotherapy. These investigators examined tumor samples from subjects enrolled in the National Surgical Adjuvant Breast and Bowel Project (NSABP) B20 trial to determine whether there is a correlation between the recurrence score (RS) determined by Oncotype Dx in tumor samples and subsequent response to adjuvant chemotherapy. A total of 651 patients were assessable (227 randomly assigned to tamoxifen and 424 randomly assigned to tamoxifen plus chemotherapy). The test for interaction between chemotherapy treatment and RS was statistically significant (p = 0.038). Patients with high-RS (RS greater than or equal to 31) tumors (ie, high risk of recurrence) had a large benefit from chemotherapy (relative risk, 0.26; 95% confidence interval 0.13 to 0.53; absolute decrease in 10-year distant recurrence rate: mean, 27.6%; standard error, 8.0%). Patients with low-RS (less than 18) tumors derived minimal, if any, benefit from chemotherapy treatment (relative risk, 1.31; 95% confidence interval, 0.46 to 3.78; absolute decrease in distant recurrence rate at 10 years: mean, -1.1%; standard error, 2.2%). The investigators found that patients with intermediate-RS tumors did not appear to have a large benefit, but the investigators concluded that the uncertainty in the estimate cannot exclude a clinically important benefit.\n",
      "One limitation of the study by Paik et al (2006) is that the NASBP B20 trial was conducted before the advent of important advances in breast cancer chemotherapy, including the introduction of trastuzumab (Herceptin), which has been demonstrated to improve overall and disease-free survival in breast cancer patients with HER2 positive tumors. Current guidelines recommend the use of trastuzumab adjuvant chemotherapy in women with metastatic HER2 positive breast cancer, and women with HER2 positive nonmetastatic breast cancers 1 cm or more in diameter. Thus, the Oncotype Dx score would not influence the decision to use adjuvant trastuzumab in women with HER2 positive tumors 1 cm or more in diameter.\n",
      "Commenting on an early report of this study by Paik et al, of the Oncotype Dx presented in abstract form, the BlueCross BlueShield Association Technology Evaluation Center assessment stated that \"additional studies in different populations are needed to confirm whether risk prediction is sufficiently accurate for physicians and patients to choose with confidence whether to withhold adjuvant chemotherapy.\"\n",
      "An international consensus group (Azim, et al., 2013) found the available evidence on the analytical and clinical validity of Oncotype Dx Breast to be convincing. However, neither the Oncotype Dx or none of the other genomic tests the evaluated demonstrated robust evidence of clinical utility: they stated that it was not clear from the current evidence that modifying treatment decisions based on the results of a given genomic test could result in improving clinical outcome.\n",
      "The selection criteria for the TailorRx prospective trial of OncotypeDx state that candidates should have negative axillary nodes as determined by a sentinel lymph node biopsy and/or axillary dissection as defined by the American Joint Committee on Cancer 6th Edition Staging System (NCI, 2009). The American Joint Committee on Cancer (AJCC) 6th Edition criteria redefined isolated tumor cells as node negative (the prior version of the criteria, AJCC 5th Edition, classified isolated tumor cells as node positive). \"Isolated tumor cells (single cells or cell deposits) will now be defined as tumor cell deposits no larger than 0.2 mm in diameter that may or may not (but usually do not) show histologic evidence of malignant activity. Pending further information, isolated tumor cells will be classified as node-negative, because it is believed that the unknown benefits of providing treatment for these small lesions would not outweigh the morbidity caused by the treatment itself.\" (Singletary, et al., 2002). However, the banked tumor samples used in the study by Paik, et al. (2006) to validate the OncotypeDx were classified based on AJCC 5th Ed. criteria. In addition, there is new evidence demonstrating that women with isolated tumor cells are at a significantly increased risk of breast cancer. Investigators from the Netherlands found an association between isolated tumor cells and micrometastases in regional lymph nodes and clinical outcome of breast cancer (de Boer, et al., 2009). These investigators identified all patients in the Netherlands who underwent a sentinel-node biopsy for breast cancer before 2006 and had breast cancer with favorable primary-tumor characteristics and isolated tumor cells or micrometastases in the regional lymph nodes. Patients with node-negative disease were randomly selected from the years 2000 and 2001. The primary end point was disease-free survival. The investigators identified 856 patients with node-negative disease who had not received systemic adjuvant therapy (the node-negative, no-adjuvant-therapy cohort), 856 patients with isolated tumor cells or micrometastases who had not received systemic adjuvant therapy (the node-positive, no-adjuvant-therapy cohort), and 995 patients with isolated tumor cells or micrometastases who had received such treatment (the node-positive, adjuvant-therapy cohort). The median follow-up was 5.1 years. The adjusted hazard ratio for disease events among patients with isolated tumor cells who did not receive systemic therapy, as compared with women with node-negative disease, was 1.50 (95% confidence interval [CI], 1.15 to 1.94); among patients with micrometastases, the adjusted hazard ratio was 1.56 (95% CI, 1.15 to 2.13). Among patients with isolated tumor cells or micrometastases, the adjusted hazard ratio was 0.57 (95% CI, 0.45 to 0.73) in the node-positive, adjuvant-therapy cohort, as compared with the node-positive, no-adjuvant-therapy cohort. The investigators concluded that isolated tumor cells or micrometastases in regional lymph nodes were associated with a reduced 5-year rate of disease-free survival among women with favorable early-stage breast cancer who did not receive adjuvant therapy. In patients with isolated tumor cells or micrometastases who received adjuvant therapy, disease-free survival was improved.\n",
      "The Medical Advisory Panel of the BlueCross BlueShield Association Technology Evaluation Center (BCBCA, 2014) concluded that use of Oncotype DX to determine recurrence risk for deciding whether to undergo adjuvant chemotherapy in women with unilateral, nonfixed, hormone receptorâpositive, lymph nodeânegative breast cancer who will receive hormonal therapy meets the Blue Cross and Blue Shield Association Technology Evaluation Center (TEC) criteria. A technology assessment by the BlueCross BlueShield Association (2014) stated: \"Technical performance of the assay is well documented and is unlikely to be a major source of variability; rather, tissue sampling is likely the greatest source of variability. Retrospective epidemiologic analyses indicated strong, independent associations between Oncotype DX recurrence score (RS) result and distant disease recurrence or death from breast cancer. The evidence identified a subset of conventionally classified, high-risk patients who are at sufficiently low risk of recurrence by Oncotype DX that they might reasonably decide that the harms (toxicity) of chemotherapy outweigh the very small absolute benefit. Two studies of the original validation data, in which conventionally classified patients were reclassified by Oncotype DX result, indicated that the test provides significant recurrence risk information in addition to conventional criteria for individual patient risk classification. Additional evidence indicated that Oncotype DX results are significantly associated with breast cancer death in a community-based patient population, and that RS high-risk patients benefit from chemotherapy, whereas benefits for other RS categories were not statistically significant. Thus, the evidence was judged sufficient to permit conclusions regarding probable health outcomes.\"\n",
      "The Oncotype Dx has also been promoted for use in women with node-positive, ER-positive breast cancer. An assessment by the BlueCross BlueShield Association (2010) concluded that it has not yet been demonstrated whether use of the Oncotype Dx for selecting adjuvant chemotherapy in patients with lymph-node-positive breast cancer improves health outcomes. The report explained that the evidence for not selecting chemotherapy for women with low RS values is based on low event rates and wide confidence intervals that include the possibility of benefit from chemotherapy.  Because the data allow for a possible benefit of chemotherapy in patients with low RS results, it is unknown if health outcomes would be improved, the same, or worse, if chemotherapy was withheld in these women. The report stated that, due to the lack of clear and sufficient information, there is a need for a second, confirmatory study.  The report stated that the Fred Hutchinson Cancer Research Center will conduct a nationwide, NCI-sponsored, Phase III clinical trial to determine the predictive ability of the Oncotype Dx to identify which patients with lymph-node-positive breast cancer will benefit from chemotherapy.\n",
      "The clinical evidence base for OncotypeDX is considered to be the most robust. There was some evidence on the impact of the test on decision-making and to support the case that OncotypeDX predicts chemotherapy benefit; however, few studies were UK based and limitations in relation to study design were identified. OncotypeDX has a more robust evidence base, but further evidence on its impact on decision-making in the UK and the predictive ability of the test in an estrogen receptor positive (ER+), lymph node negative (LN-), human epidermal growth factor receptor 2 negative (HER2-) population receiving current drug regimens is needed.\n",
      "Guidelines from the National Comprehensive Cancer Network (NCCN, 2015) state that \"the 21-gene RT-PCR assay recurrence score can be considered in select patients with 1-3 involved ipsilateral axillary lymph nodes to guide the addition of combination chemotherapy to standard hormone therapy. A retrospective analysis of a prospective randomized trial suggests that the test is predictive in this group similar to its performance in node-negative disease.\" The NCCN guidelines (2015) explained: \"Unplanned, retrospective subset analysis from a single randomized clinical trial in post-menopausal, ALN-positive, ER-positive breast cancer found that the 21-gene RT-PCR assay may provide predictive information for chemotherapy in addition to tamoxifen [citing Albain, et al., 2010]. Patients with a high score in the study benefited from chemotherapy, whereas patients with a low score did not appear to benefit from the addition of chemotherapy regardless of the number of positive lymph nodes. Patient selection for assay use remains controversial.\" \"The RxPONDER trial will confirm the SWOG-8814 trial data for women with ER-positive, node-positive disease treated with endocrine therapy with or without chemotherapy based on risk scores.\"\n",
      "Guidance from the National Institute for Health and Care Excellence (2013) stated: \"Oncotype DX is recommended as an option for guiding adjuvant chemotherapy decisions for people with estrogen ER+, LN- and HER2- early breast cancer if: The person is assessed as being at intermediate risk; and information on the biological features of the cancer provided by Oncotype DX is likely to help in predicting the course of the disease and would therefore help when making the decision about prescribing chemotherapy; and the manufacturer provides Oncotype DX to National Health Service (NHS) organisations according to the confidential arrangement agreed with the National Institute for Health and Care Excellence (NICE). NICE encourages further data collection on the use of Oncotype DX in the NHS.\"\n",
      "An assessment by the Belgian Healthcare Knowledge Center (KCE) (San Miguel, et al., 2015) concluded that \"the evidence for Oncotype DX is more robust than the evidence for other tests.\" The KCE Report noted, however, that important evidence gaps are still present. The KCE review mostly identified studies supporting the prognostic ability (clinical validity) of the test. The KCE judged these studies to be of moderate to high quality. The KCE found no prospective studies reporting on the impact of Oncotype DX on long-term outcomes such as overall survival, while four studies indicated that Oncotype DX leads to changes in decision making. The KCE identified two studies on the predictive benefit of the test, one for lymph node patients. The KCE reported also noted that the first evidence relating to improvements in quality of life and reductions in patient anxiety as a result of using the test has been reported, but this is based on small patient numbers and further evidence is required.\n",
      "Guidelines from the American Society for Clinical Oncology (2016) state: \"If a patient has ER/PgR-positive, HER2-negative (node-negative) breast cancer, the clinician may use the 21-gene recurrence score (RS; Oncotype DX; Genomic Health, Redwood City, CA) to guide decisions on adjuvant systemic chemotherapy.\" This is a strong recommendation based upon high quality evidence. The ASCO guidelines recommend against OncotypeDx Breast to guide decisions on adjuvant systemic chemotherapy for patients with ER/PgR-positive, HER2-negative (node-positive) breast cancer.  The guidelines also recommend against the use of OncotypeDx Breast in women with HER2-positive breast cancer or TN breast cancer. The guidelines recommended against the use of OncotypeDx Breast to guide decisions on extended endocrine therapy for patients with ER/PgR-positive, HER-2 negative (node-negative) breast cancer who have had 5 years of endocrine therapy without evidence of recurrence. \n",
      "Acceptance of 21-gene recurrence score assays as tools for clinical decision making in women or men with early stage breast cancer is controversial due to the lack of prospective validation studies, nevertheless, 2007 guidelines from an expert panel convened by ASCO on tumor markers in breast cancer concluded that multiparameter gene expression analysis (i.e., Oncotype Dx assay) can be used to predict the risk of recurrence in women with newly diagnosed, node-negative, ER-positive breast cancer.  Although it is reasonable to consider the use of a 21-gene recurrence score assay in males, none of the data generated to date have been in men with breast cancer (Gradishar, 2010). \n",
      "A 2009 abstract that looked at cases of male breast cancer (BC) with Oncotype Dx, concluded, \"This large genomic study of male BC reveals a heterogeneous biology as measured by the standardized quantitative oncotype Dx breast cancer assay, similar to that observed in female BC.  Some differences, which may reflect the differences in hormone biology between males and females, were noted and deserve further study.\" (Shak et al, 2009).\n",
      "MammaPrint\n",
      "MammaPrint a 70-gene profile that classifies breast cancer into Low Risk or High Risk of recurrence, by measuring genes representative of all the pathways of cancer metastases which were selected for their predictive relationship to 10-year recurrence probability (Raman, et al., 2013). MammaPrint is indicated for women who have stage I or II breast cancer, are lymph node positive or negative, are ER-positive or negative and tumor size of less than five centimeters. MammaPrint determines if the patient is a candidate for chemotherapy.\n",
      "In February 2007, the Food and Drug Administration (FDA) approved Mammaprint (Agendia,  Amsterdam), a DNA microarray-based test used to predict whether women with early breast cancer might face the disease again. The test measures the activity of 70 genes, providing information about the likelihood that cancer will recur. It measures each of these genes in a sample of a woman's breast-cancer tumor and then uses a specific formula to produce a score that determines if the patient is deemed low-risk or high-risk for metastasis. In clinical trials, 1 in 4 women found to be at high risk by Mammaprint had recurrence of their cancer within 5 years. However, there are questions regarding the accuracy of this test. The positive predictive values at 5 and 10 years were 23 % and 29 %, respectively, while the corresponding negative predictive values were 95 % and 90 %, respectively.\n",
      "Mammaprint was tested on 307 patients under the age of 61 years who underwent surgery for stage I or stage II breast cancer, and who have tumor size equal to or less than 5 cm, and lymph node-negative. The study found that Mammaprint more than doubled physicians' ability to predict breast cancer recurrence.\n",
      "Cardoso et al (2016) conducted a study to evaluate the clinical utility of the 70-gene signature test (MammaPrint).  The study was exerpted from a phase III randomized trial.  In this study, of 6693 enrolled women with early stage breast cancer, women with low clinical and genomic risk did not receive chemotherapy whereas those at high risk did receive chemotherapy.  All study subjects had their genomic risk evaluated using MammaPrint. The authors noted that \"the primary goal was to assess whether, among patients with high-risk clinical features and a low-risk-gene-expression profile who did not receive chemotherapy, the lower boundary of the 95% confidence interval for the rate of 5-year survival without distant metastasis would be 92% (i.e. the noninferiority boundary, or higher). The number of women found to be at high clinical risk and low genomic risk was 1550.  In this group, the 5 year survival rate without distant metastases was 94.7% among those not receiving chemotherapy.  The authors concluded that among women with early-stage-breast cancer who were at high clinical risk and low genomic risk for recurrence, the receipt of chemotherapy on the basis of the 60 gene signature led to a 5-year survival rate without distant metastasis that was 1.5 percentage points lower than the rate with chemotherapy.    \n",
      "A comment by Hudis and Dickler (2016) stated that it can be challenging to convince practitioners that chemotherapy is not need in an otherwise healthy younger population.  They further noted that the primary aim of the study on one study of a 70-gene signature test was to \"declare non-inferiority against a predefined benchmark of a 5 year metastasis-free survival rate in just one cohort: patients with a high clinical risk for whom a discordant low genomic risk led to the omission of otherwise standard chemotherapy.\"  They concluded that although for select patients providers may wish to use the MammaPrint, the actions they will take as a result of this testing will be variable and may over time change as a result of further study.\n",
      "The study by Cardoso et al (2016) was a 5-year median follow-up results of the MINDACT trial, which is to follow subjects for 10 years.  The authors noted that follow-up is ongoing to determine whether their findings remain valid for longer-term outcome.  These investigators noted that \"In the critical group of patients at high clinical risk and low genomic risk, the use of adjuvant chemotherapy led to a trend toward a higher rate of the 5-year outcome than that with no chemotherapy, which included a rate of survival without distant metastasis that was 1.5 percentage points higher, a rate of disease-free survival that was 2.8 percentage points higher, and a rate of overall survival that was 1.4 percentage points higher with chemotherapy than with no chemotherapy in the intention-to-treat population and a rate of survival without distant metastasis that was 1.9 percentage points higher, a rate of disease-free survival that was 3 percentage points higher, and a rate of overall survival that was 1.5 percentage points higher with chemotherapy than with no chemotherapy in the per-protocol population.  The study was not powered to assess the statistical significance of these differences.  Some 50 % of the study patients were defined as being at low clinical risk.  In this group, we did not find any meaningful difference in the 5-year rate of survival without distant metastasis between patients at high genomic risk who received chemotherapy and those who did not receive chemotherapy.  On the basis of these data, the results for the 70-gene signature do not provide evidence for making recommendations regarding chemotherapy for patients at low clinical risk\". \n",
      "In an editorial that accompanied the afore-mentioned study, Hudis and Dickler (2016) stated that \"a difference of 1.5 percentage points, if real, might mean more to one patient than to another.  Thus, the stated difference does not precisely exclude a benefit that clinicians and patients might find meaningful.  An adequately powered randomization or a higher threshold for 5-year metastasis-free survival might have provided a more convincing result but would have raised other major challenges for the investigators\".\n",
      "A focused update by the American Society for Clinical Oncology (ASCO) (Kopp, et al., 2017) states that If a patient has hormone receptorâpositive, human epidermal growth factor receptor 2 (HER2)ânegative, node-negative breast cancer, the MammaPrint assay may be used in those with high clinical risk to inform decisions on withholding adjuvant systemic chemotherapy due to its ability to identify a good-prognosis population with potentially limited chemotherapy benefit. The guidelines state that, if a patient has hormone receptorâpositive, HER2-negative, node-positive breast cancer, the MammaPrint assay may be used in patients with one to three positive nodes and a high clinical risk to inform decisions on withholding adjuvant systemic chemotherapy. However, such patients should be informed that a benefit from chemotherapy cannot be excluded, particularly in patients with greater than one involved lymph node. The guideline update was based upon an assessment of data on clinical utility from the MINDACT trial plus other published literature. \n",
      "BluePrint\n",
      "Molecular subtyping profile or BluePrint is proposed for the evaluation of an individualâs prognosis when diagnosed with breast cancer. The multigene profile classifies breast cancer into basal type, luminal type and ERBB type (HER2/neu positive) molecular subclasses to stratify an individualâs risk to purportedly assist with treatment decisions.\n",
      "Agendia BluePrint has an 80-gene profile that classifies breast cancer into molecular subtypes (Raman, et al., 2013). The profile separates tumors into Basal-type, Luminal-type and ERBB2-type subgroups by measuring the functionality of downstream genes for each of these molecular pathways to inform the physician of the potential effect of adjuvant therapy.\n",
      "Krijgsman et al (2012) noted that classification of breast cancer into molecular subtypes maybe important for the proper selection of therapy, as tumors with seemingly similar histopathological features can have strikingly different clinical outcomes.  Herein, these researchers reported the development of a molecular subtyping profile (BluePrint), which enables rationalization in patient selection for either chemotherapy or endocrine therapy prescription.  An 80-Gene Molecular Subtyping Profile (BluePrint) was developed using 200 breast cancer patient specimens and confirmed on 4 independent validation cohorts (n = 784).  Additionally, the profile was tested as a predictor of chemotherapy response in 133 breast cancer patients, treated with T/FAC neoadjuvant chemotherapy.  BluePrint classification of a patient cohort that was treated with neoadjuvant chemotherapy (n = 133) shows improved distribution of pathological Complete Response (pCR), among molecular subgroups compared with local pathology: 56 % of the patients had a pCR in the Basal-type subgroup, 3 % in the MammaPrint low-risk, luminal-type subgroup, 11 % in the MammaPrint high-risk, luminal-type subgroup, and 50 % in the HER2-type subgroup.  The group of genes identifying luminal-type breast cancer is highly enriched for genes having an Estrogen Receptor binding site proximal to the promoter-region, suggesting that these genes are direct targets of the Estrogen Receptor.  Implementation of this profile may improve the clinical management of breast cancer patients, by enabling the selection of patients who are most likely to benefit from either chemotherapy or from endocrine therapy.\n",
      "An assessment by the National Institute for Health Research (Ward, et al., 2013) found the evidence for Blueprint was limited. Because of the limited available data identified for this test, the NIHR was unable to draw firm conclusions about its analytical validity, clinical validity (prognostic ability) and clinical utility. The report stated that further evidence on the prognostic and predictive ability of this test was required.\n",
      "A report by the Belgian Healthcare Knowledge Centre (KCE) (San Miguel, et al., 2015) found that limited evidence for the prognostic ability (clinical validity) of BluePrint. The KCE found insufficient evidence on the impact of BluePrint on clinical management (clinical utility).\n",
      "Furthermore, there is no information regarding BluePrint/molecular subtyping from NCCNâs clinical practice guideline on \"Breast cancer\" (Version 2.2013).\n",
      "TargetPrint\n",
      "TargetPrintÂ®, ER/PR/HER2 Expression Assay (Agendia) is a microarray-based gene expression test which offers a quantitative assessment of the patientâs level of estrogen receptor (ER), progesterone receptor (PR) and HER2/neu overexpression within her breast cancer (Raman, et al., 2013). TargetPrint is offered in conjunction with MammaPrint to provide the physician an even more complete basis for treatment decisions. TargetPrint delivers an added benefit to the diagnostic process. Immunohistochemistry provides a semi-quantitative positive or negative result, whereas the gene expression result provided by TargetPrint allows physicians to integrate the absolute level of ER, PR and HER2 gene expression into treatment planning. TargetPrint determines if the patient is a candidate for hormonal therapy.\n",
      "TargetPrint is a microarray-based gene expression test which offers a quantitative assessment of the patientâs level of estrogen receptor (ER), progesterone receptor (PR) and HER2/neu overexpression in breast cancer. The manufacturer states that TargetPrint is offered in conjunction with MammaPrint gene expression profiling to provide the physician an even more complete basis for treatment decisions. The manufacturer states that, as compared to Immunohistochemistry (IHC), TargetPrint provides additional information. Whereas IHC provides a semi-quantitative positive or negative result, the gene expression result provided by TargetPrint provides data on the absolute level of ER, PR and HER2 gene expression. Published information on the TargetPrint is limited to studies examining its correlation with measurements of ER, PR, and HER2 receptors (Gunven et al, 2011; Gevensleben et al, 2010; Roepman et al, 2009).  There is a lack of evidence from published prospective clinical studies that demonstrates that quantification of ER, PR, and HER2 gene expression by TargetPrint alters management such that clinical outcomes are improved.\n",
      "Symphony\n",
      "Symphony (Agendia) provides complete tumor profiling and is used to support therapeutic choices for breast cancer (Raman, 2013). SYMPHONY includes four assays to support breast cancer treatment decisions: MammaPrintÂ® determines the risk of recurrence. BluePrintâ¢ determines molecular subtypes and TargetPrintÂ® determines estrogen receptor (ER), progesterone receptor (PR), and HER2 status. TheraPrintâ¢ identifies alternative types of therapy for metastatic disease. SYMPHONY provides genomic information assisting with therapeutic decisions even for cases that have been otherwise classified as indeterminate, such as grade 2, small tumors, HER2 and/or lymph node positive. MammaPrintÂ® determines if the patient is a candidate for chemotherapy. TargetPrintÂ® determines if the patient is a candidate for hormonal therapy. BluePrintÂ® provides information on the sub-classification of the tumor which guides the choice of therapies and combinations of therapies. TheraPrintÂ® identifies alternative types of therapy for metastatic disease.\n",
      "Rotterdam Signature 76-Gene Panel\n",
      "The Rotterdam Signature test (Veridex) is a 76-gene expression assay (Raman, 2013). Sixty genes are intended to evaluate estrogen-receptor positive samples and 16 genes to evaluate estrogen-receptor negative samples. In a validation study that tested the signature on samples from 148 women, 50 fell into the low-risk group and 98 into the high-risk group. The test had 88% specificity and 39% sensitivity for the low-risk group, with a hazard ratio for distant relapse within 5 years of 5.74 comparing the high-risk group to the low-risk group. The Rotterdam Signature identifies women at high and low risk of disease recurrence.\n",
      "The Rotterdam Signature 76-gene panel (Veridex, LLC) is a multivariate index assay that is intended to assist in assessing a patientâs risk of systemic recurrence of cancer following successful initial treatment of localized node-negative breast cancer with surgery and tamoxifen alone. This multigene assay is intended for use in lymph-node negative breast cancer patients. The Rotterdam Signature panel uses microarray processing to measure cellular concentrations of mRNA in fresh tissue samples. The Rotterdam Signature panel uses the Human Genome U133a GeneChip (Affymetrix, Inc.) to identify patients that have gene expression signatures associated with either a low or high risk of developing metastatic disease.  A multicenter study investigated the ability of the Rotterdam 76-gene signature to identify patients at risk of distant metastases within 5 and 10 years of first diagnosis, using frozen tissue samples from 180 patients with node-negative breast cancer who had not received systemic chemotherapy (Foekens, et al., 2006). The Rotterdam 76-gene signature correctly identified 27 out of 30 cases of relapse within 5 years (90% sensitivity) and 75 out of 150 patients who did not relapse (50% specificity). An earlier summary of the same study (Foekens, et al., 2005) reported a hazard ratio for distant metastasis-free survival comparing favorable versus unfavorable signature = 7.41 (95% confidence interval 2.63-20.9); p = 8.5 x 10-6). The hazard ratio of overall survival comparing favorable versus unfavorable signature = 5.45 (95% confidence interval 1.62-18.3); p = .002. There are no published studies that have assessed the clinical utility of the Rotterdam 76-gene signature by monitoring the long-term outcomes of the patients selected and not selected for chemotherapy on the basis of assay results.\n",
      "Breast Cancer Gene Expression Ratio / Breast Cancer Index\n",
      "The Breast Cancer Gene Expression Ratio (HOXB13:IL17BR, also known as H/I) (AviaraDx, Inc., Carlsbad, CA) is intended to predict the risk of disease recurrence in women with estrogen receptor (ER)-positive, lymph node-negative breast cancer. The Breast Cancer Gene Expression Ratio is based on the ratio of the expression of two genes: the homeobox gene-B13 (HOXB13) and the interleukin- 17B receptor gene (IL17BR). In breast cancers that are more likely to recur, the HOXB13 gene tends to be over-expressed, while the IL-17BR gene tends to be under-expressed.\n",
      "Ma et al (2004) reported on the early validation of the HOXB13:IL17BR gene expression ratio. The investigators generated gene expression profiles of hormone receptor-positive primary breast cancers in a set of 60 patients treated with adjuvant tamoxifen monotherapy. An expression signature predictive of disease-free survival was reduced to a two-gene ratio, HOXB13 versus IL17BR, which outperformed existing biomarkers. The investigators concluded that ectopic expression of HOXB13 in MCF10A breast epithelial cells enhances motility and invasion in vitro, and its expression is increased in both preinvasive and invasive primary breast cancer. The investigators suggested that HOXB13:IL17BR expression ratio may be useful for identifying patients appropriate for alternative therapeutic regimens in early-stage breast cancer.\n",
      "In an 852-patient retrospective study, Ma, et al (2006) found that the HOXB13:IL17BR ratio (H:I expression ratio) independently predicted breast cancer recurrence in patients with ER-positive, lymph-node negative cancer. The H:I expression ratio was found to be predictive in patients who received tamoxifen therapy as well as in those who did not. Expression of HOXB13, IL17BR, CHDH, estrogen receptor (ER) and progesterone receptor (PR) were quantified by real-time polymerase chain reaction (PCR) in 852 formalin-fixed, paraffin-embedded primary breast cancers from 566 untreated and 286 tamoxifen-treated breast cancer patients. Gene expression and clinical variables were analyzed for association with relapse-free survival (RFS) by Cox proportional hazards regression models. The investigators reported that, in the entire cohort, expression of HOXB13 was associated with shorter RFS (p = .008), and expression of IL17BR and CHDH was associated with longer RFS (p < 0.0001 for IL17BR and p = 0.0002 for CHDH). In ER-positive patients, the HOXB13:IL17BR index predicted clinical outcome independently of treatment, but more strongly in node-negative patients. In multivariate analysis of the ER-positive node-negative subgroup including age, PR status, tumor size, S phase fraction, and tamoxifen treatment, the two-gene index remained a significant predictor of RFS (hazard ratio [HR] = 3.9; 95 % CI:1.5 to 10.3; p = .007).\n",
      "The value of the Breast Cancer Gene Expression Ratio was also evaluated in a study by Goetz et al (2006). That study found that a high H:I expression ratio is associated with an increased rate of relapse and mortality in ER-positive, lymph node-negative cancer patients treated with surgery and tamoxifen. Goetz et al (2006) examined the association between the ratio of the HOXB13 to IL17BR expression and the clinical outcomes of relapse and survival in women with ER-positive breast cancer enrolled onto a North Central Cancer Treatment Group adjuvant tamoxifen trial (NCCTG 89-30-52). Tumor blocks were obtained from 211 of 256 eligible patients, and quantitative reverse transcription-PCR profiles for HOXB13 and IL-17BR were obtained from 206 patients.  In the node-positive cohort (n = 86), the HOXB13/IL-17BR ratio was not associated with relapse or survival. In contrast, in the node-negative cohort (n = 130), a high HOXB13/IL-17BR ratio was associated with significantly worse RFS [HR, 1.98; p = 0.031], disease-free survival (DFS) (HR, 2.03; p = 0.015), and OS (HR, 2.4; p = 0.014), independent of standard prognostic markers.\n",
      "The Blue Cross and Blue Shield Association Technology Evaluation Center (BCBSA, 2007) announced that its Medical Advisory Panel (MAP) concluded that the use of the Breast Cancer Gene Expression Ratio gene expression profiling does not meet the TEC criteria.\n",
      "The Evaluation of Genomic Applications in Practice and Prevention (EGAPP) Working Group (2009) found insufficient evidence to make a recommendation for or against the use of the H:I ratio test to improve outcomes in defined populations of women with breast cancer. EGAPP concluded that the evidence is insufficient to assess the balance of benefits and harms of the proposed uses of this test. The EWG encouraged further development and evaluation of these technologies.\n",
      "In a systematic review on gene expression profiling assays in early-stage breast cancer, Marchionni, et al. (2008) summarized evidence on the validity and utility of 3 gene expression-based prognostic breast cancer tests: Oncotype Dx, MammaPrint, and H/I.  The authors concluded that gene expression technologies show great promise to improve predictions of prognosis and treatment benefit for women with early-stage breast cancer.  However, more information is needed on the extent of improvement in prediction, characteristics of women in whom the tests should be used, and how best to incorporate test results into decision making about breast cancer treatment. \n",
      "Guidelines from the American Society for Clinical Oncology (Harris, et al., 2007) found that, in newly diagnosed patients with node-negative, estrogen-receptor positive breast cancer, the Oncotype Dx assay can be used to predict the risk of recurrence in patients treated with tamoxifen. The ASCO guidelines concluded that Oncotype Dx may be used to identify patients who are predicted to obtain the most therapeutic benefit from adjuvant tamoxifen and may not require adjuvant chemotherapy. The ASCO guidelines found, in addition, that patients with high recurrence scores appear to achieve relatively more benefit from adjuvant chemotherapy than from tamoxifen. ASCO found that there are insufficient data at present to comment on whether these conclusions generalize to hormonal therapies other than tamoxifen, or whether this assay applies to other chemotherapy regimens.  Guidelines from the American Society for Clinical Oncology (Harris, et al., 2007) concluded that the precise clinical utility and appropriate application for other multiparameter assays, such as the MammaPrint assay, the Rotterdam Signature, and the Breast Cancer Gene Expression Ratio are under investigation. ASCO also found insufficient data to recommend use of proteomic patterns for management of patients with breast cancer. \n",
      "Sgori et al (2013) found that, in the absence of extended letrozole therapy, high H/I identifies a subgroup of ER-positive patients disease-free after 5 years of tamoxifen who are at risk for late recurrence. The investigators also found that, when extended endocrine therapy with letrozole is prescribed, high H/I predicts benefit from therapy and a decreased probability of late disease recurrence. Sgori, et al. conducted a prospective-retrospective, nested case-control design of 83 recurrences matched to 166 nonrecurrences from letrozole- and placebo-treated patients within MA.17 trial. Expression of H/I within primary tumors was determined by reverse-transcription polymerase chain reaction with a prespecified cutpoint. The investigators determined the predictive ability of H/I for ascertaining benefit from letrozole using multivariable conditional logistic regression including standard clinicopathological factors as covariates. All statistical tests were two-sided. The investigators reported that high H/I was statistically significantly associated with a decrease in late recurrence in patients receiving extended letrozole therapy (odds ratio [OR] = 0.35; 95% confidence interval [CI] = 0.16 to 0.75; P = .007). In an adjusted model with standard clinicopathological factors, high H/I remained statistically significantly associated with patient benefit from letrozole (OR = 0.33; 95% CI = 0.15 to 0.73; P = .006). Reduction in the absolute risk of recurrence at 5 years was 16.5% for patients with high H/I (P = .007). The interaction between H/I and letrozole treatment was statistically significant (P = .03).\n",
      "BioTheranostics Breast Cancer Index (BCI) is a prognostic biomarker that provides quantitative assessment of the likelihood of distant recurrence in patients diagnosed with estrogen receptor-positive, lymph node-negative breast cancer (Raman, et al., 2013). In development and validation studies, BCI stratified about 50% of tamoxifen treated ER+, node-negative breast cancer patients into a low risk group for 10-year distant recurrence. BCI is a molecular assay developed from the combination of two indices: HOXB13:IL17BR and five cell cycle-associate gene index (BUB1B, CENPA, NEK2, RACGAP1, RRM2) that assesses tumor grade. The test is performed on a formalin-fixed, paraffin-embedded (FFPE) tissue block.\n",
      "Ma et al (2008) reported on the development and early validation of a five-gene reverse transcription PCR assay for molecular grade index (MGI) that has subsequently been incorporated into BCI and is suitable for analyzing routine formalin-fixed paraffin-embedded clinical samples. The investigators found that the combination of MGI and HOXB13:IL17BR outperformed either alone and identifies a subgroup (approximately 30%) of early stage estrogen receptor-positive breast cancer patients with very poor outcome despite endocrine therapy. From their previously published list of genes whose expression correlates with both tumor grade and tumor stage progression, the investigators selected five cell cycle-related genes to build MGI and evaluated MGI in two publicly available microarray data sets totaling 410 patients. Using two additional cohorts (n =323), the investigators developed a real-time reverse transcription PCR assay for MGI, validated its prognostic utility, and examined its interaction with HOXB13:IL17BR. The investigators reported that MGI performed consistently as a strong prognostic factor and was comparable with a more complex 97-gene genomic grade index in multiple data sets. In patients treated with endocrine therapy, MGI and HOXB13:IL17BR modified each other's prognostic performance. High MGI was associated with significantly worse outcome only in combination with high HOXB13:IL17BR, and likewise, high HOXB13:IL17BR was significantly associated with poor outcome only in combination with high MGI.\n",
      "Jerevall et al (2011) reported on the development of the Breast Cancer Index, a dichotomous index combining two gene expression assays, HOXB13:IL17BR (H:I) and molecular grade index (MGI), to assess risk of recurrence in breast cancer patients. The study objective was to demonstrate the prognostic utility of the combined index in early-stage breast cancer. In a blinded retrospective analysis of 588 ER-positive tamoxifen-treated and untreated breast cancer patients from the randomized prospective Stockholm trial which was conducted during 1976 to 1990, H:I and MGI were measured using real-time RT-PCR. Association with patient outcome was evaluated by Kaplan-Meier analysis and Cox proportional hazard regression. A continuous risk index was developed using Cox modelling. The investigators found that the dichotomous H:I+MGI was significantly associated with distant recurrence and breast cancer death. The greater than 50% of tamoxifen-treated patients categorized as low-risk had less than 3% 10-year distant recurrence risk. A continuous risk model (Breast Cancer Index (BCI)) was developed with the tamoxifen-treated group and the prognostic performance tested in the untreated group was 53% of patients categorized as low risk with an 8.3% 10-year distant recurrence risk. \n",
      "Jankowitz et al (2011) reported on a study to validate the prognostic performance of BCI in estrogen-receptor positive, lymph node negative breast cancer patients. The investigators found that, in this characteristically low-risk cohort, BCI classified high versus low-risk groups with about a five-fold difference in 10-year risk of distant recurrence and breast cancer-specific death. The investigators identified tumor samples from 265 estrogen-receptor positive, lymph-node negative tamoxifen-treated patients from a single academic institution's cancer research registry. They performed the BCI assay and assigned scores based on a predetermined risk model. The investigators assessed risk by BCI and Adjuvant Online! (AO) and correlated these to clinical outcomes in the patient cohort. The investigators found that BCI was a significant predictor of outcome in this cohort of estrogen-receptor positive, lymph-node negative patients (median age: 56-y; median follow-up: 10.3-y), treated with adjuvant tamoxifen alone or tamoxifen with chemotherapy (32%). BCI categorized 55%, 21%, and 24% of patients as low, intermediate and high-risk, respectively. The 10-year rates of distant recurrence were 6.6%, 12.1% and 31.9% and of breast cancer-specific mortality were 3.8%, 3.6% and 22.1% in low, intermediate, and high-risk groups, respectively. In a multivariate analysis including clinicopathological factors, BCI was a significant predictor of distant recurrence (HR for 5-unit increase = 5.32 [CI 2.18-13.01; P = 0.0002]) and breast cancer-specific mortality (HR for a 5-unit increase = 9.60 [CI 3.20-28.80; P < 0.0001]). AO was significantly associated with risk of recurrence. In a separate multivariate analysis, both BCI and AO were significantly predictive of outcome. In a time-dependent (10-year) ROC curve accuracy analysis of recurrence risk, the addition of BCI and AO increased predictive accuracy in all patients from 66% (AO only) to 76% (AO+BCI) and in tamoxifen-only treated patients from 65% to 81%. The authors concluded that BCI and AO are independent predictors with BCI having additive utility beyond standard of care parameters that are encompassed in AO. The authors acknowledge that this study is limited by the fact that it was a retrospective, single-institution study and that results may have been biased on the basis of specimen availability and patterns of referral to the tertiary academic center.\n",
      "Mathieu et al (2012) assessed the performance of BCI to predict chemosensitivity based on pathological complete response (pCR) and breast conservation surgery (BCS). The authors performed the BCI assay on tumor samples from 150 breast cancer patients from a single institution treated with neoadjuvant chemotherapy. The authors used logistical regression and c-index to assess predictive strength and additive accuracy of BCI beyond clinicopathologic factors. BCI classified 42% of patients as low, 35% as intermediate and 23% as high risk. Low BCI risk group had 98.4% negative predictive value (NPV) for pCR and 86% NPV for BCS. High versus low BCI group had a 34 and 5.8 greater likelihood of achieving pCR and BCS, respectively (P=0.0055; P=0.0022). BCI increased c-index for pCR (0.875-0.924; p=0.017) and BCS prediction (0.788-0.843; p=0.027) beyond clinicopathologic factors. The authors concluded that BCI significantly predicted pCR and BCS beyond clinicopathologic factors. High NPVs indicate that BCI could be a useful tool to identify breast cancer patients who are not eligible for neoadjuvant chemotherapy. The authors concluded that \"these results suggest that BCI could be used to assess both chemosensitivity and eligibility for BCS.\" The authors stated that an important limitation of this study is that,  in this retrospective analysis, patients were not selected based on ER or HER2 expression for the indications of neoadjuvant chemotherapy. The authors explained that this could have increased the predictive strength of BCI given that this biomarker was initially developed and validated in ER + node-negative patients\n",
      "Zhang et al (2013) examined the prognostic performance of BCI for prediction of early (0-5 years) and late (more than 5 years) risk of distant recurrence in patients with estrogen receptor-positive (ER(+)), lymph node-negative (LN(-)) tumors. The BCI model was validated by retrospective analyses of tumor samples from tamoxifen-treated patients from a randomized prospective trial (Stockholm TAM, n = 317) and a multi-institutional cohort (n = 358). Within the Stockholm TAM cohort, BCI risk groups stratified the majority (approximately 65%) of patients as low risk with less than 3% distant recurrence rate for 0 to 5 years and 5 to 10 years. In the multi-institutional cohort, which had larger tumors, 55% of patients were classified as BCI low risk with less than 5% distant recurrence rate for 0 to 5 years and 5 to 10 years. Zhang and colleagues found that, for both cohorts, continuous BCI was the most significant prognostic factor beyond standard clinicopathologic factors for 0 to 5 years and more than five years. The authors concluded that the prognostic sustainability of BCI to assess early- and late-distant recurrence risk at diagnosis has clinical use for decisions of chemotherapy at diagnosis and for decisions for extended adjuvant endocrine therapy beyond five years.\n",
      "Sgori et al (2013) compared the prognostic ability of the BCI assay, the Oncotype DX Breast, and IHC4 for both early and late recurrence in patients with estrogen-receptor-positive, node-negative (N0) disease who took part in the Arimidex, Tamoxifen, Alone or in Combination (ATAC) clinical trial. In this prospective comparison study, Sgori and colleagues obtained archival tumor blocks from the TransATAC tissue bank from all postmenopausal patients with estrogen-receptor-positive breast cancer from whom the Oncotype DX and IHC4 values had already been derived. The investigators did BCI analysis in matched samples with sufficient residual RNA using two BCI models -- cubic (BCI-C) and linear (BCI-L)-using previously validated cutoffs. The prospectively-defined primary study objective was to evaluate overall (0â10y) prognostic performance of the BCI-C model for DR in ER+ N0 patients. Secondary objectives were: 1) assessment of the prognostic performance of the BCI-L model and its components, H/I and MGI, for overall (0â10y), early (0â5y) and late (5â10y) DR; 2) comparative performance of BCI-L versus the Oncotype DX RS and IHC4. To assess the ability of the biomarkers to predict recurrence beyond standard clinicopathological variables, the investigators calculated the change in the likelihood-ratio from Cox proportional hazards models. Suitable tissue was available from 665 patients with estrogen-receptor-positive, N0 breast cancer for BCI analysis. The primary analysis showed significant differences in risk of distant recurrence over 10 years in the categorical BCI-C risk groups (p<0Â·0001) with 6Â·8% (95% CI 4Â·4-10Â·0) of patients in the low-risk group, 17Â·3% (12Â·0-24Â·7) in the intermediate group, and 22Â·2% (15Â·3-31Â·5) in the high-risk group having distant recurrence. BCI-C analyzed as a continuous variable was not significantly associated with overall (0â10y) risk of DR when adjusted for CTS (inter-quartile HR=1Â·39; 95% CI, 0Â·99 to 3Â·70; LR-ÎÏ2=3Â·70; P=0Â·054). Comparison of the prognostic performance of BCI-L to BCI-C indicated that unlike BCI-C, BCI-L was a significant predictor of risk of recurrence as a continuous variable, and the HR after adjustment with CTS was 2Â·19 versus 4.86 between high- and low-risk groups for BCI-C and BCI-L, respectively. Thus, all subsequent analyses were performed utilizing BCI-L. The secondary analysis showed that BCI-L was a much stronger predictor for overall (0-10 year) distant recurrence compared with BCI-C (interquartile HR 2Â·30 [95% CI 1Â·62-3Â·27]; likelihood ratio (LR)-ÎÏ(2)=22Â·69; p<0Â·0001). When compared with BCI-L, the Oncotype Dx breast score was less predictive (HR 1Â·48 [95% CI 1Â·22-1Â·78]; LR-ÎÏ(2)=13Â·68; p=0Â·0002) and IHC4 was similar (HR 1Â·69 [95% CI 1Â·51-2Â·56]; LR-ÎÏ(2)=22Â·83; p<0Â·0001). All further analyses were done with the BCI-L model. In a multivariable analysis, all assays had significant prognostic ability for early distant recurrence (BCI-L HR 2Â·77 [95% CI 1Â·63-4Â·70], LR-ÎÏ(2)=15Â·42, p<0Â·0001; Oncotype Dx Breast score HR 1Â·80 [1Â·42-2Â·29], LR-ÎÏ(2)=18Â·48, p<0Â·0001; IHC4 HR 2Â·90 [2Â·01-4Â·18], LR-ÎÏ(2)=29Â·14, p<0Â·0001); however, only BCI-L was significant for late distant recurrence (BCI-L HR 1Â·95 [95% CI 1Â·22-3Â·14], LR-ÎÏ(2)=7Â·97, p=0Â·0048; 21-gene recurrence score HR 1Â·13 [0Â·82-1Â·56], LR-ÎÏ(2)=0Â·48, p=0Â·47; IHC4 HR 1Â·30 [0Â·88-1Â·94], LR-ÎÏ(2)=1Â·59, p=0Â·20). The authors concluded that BCI-L was the only significant prognostic test for risk of both early and late distant recurrence and identified two risk populations for each timeframe. BCI-L could help to identify patients at high risk for late distant recurrence who might benefit from extended endocrine or other therapy. An important limitation is that the evaluation of BCI-L was a secondary objective of this study; the primary objective was evaluation of BCI-C.\n",
      "An editorial (Ignatiadis, 2013) accompanying the study by Sgroi, et al. stated that the BCI test is \"ready for prime time\" in treatment decision making for post-menopausal, estrogen-receptor positive women who have undergone 5 years of hormonal therapy. The editorial noted that there are other molecular diagnostic assays that also have been shown to predict late recurrence. For support, the editorial cited a study by Sestak, et al. (2013), which found that, in the last follow-up phase, Clinical Treatment Score (CTS) added most prognostic information for distant recurrence in years 5 to 10 for breast cancer patients in the ATAC trial. Sestak, et al. reported that, in a multivariate model that incorporated CTS, PAM50 provided the strongest additional prognostic factor in the 5 to 10 year followup phase, followed by BCI, and with IHC4 and RS adding the least prognostic information.\n",
      "A manufacturer funded study (Gustavsen, et al., 2014) reported on a model that found BCI to be cost saving from a third-party payer perspective, based upon assumptions about the impact of BCI on adjuvant chemotherapy use, extended endocrine therapy use, and endocrine therapy compliance. The authors developed two economic models to project the cost-effectiveness of BCI in a hypothetical population of patients with estrogen-receptor positive, lymph-node negative breast cancer compared with standard clinicopathologic diagnostic modalities. The authors modeled costs associated with adjuvant chemotherapy, toxicity, followup, endocrine therapy, and recurrence over 10 years. The models examined cost utility compared with standard practice when used at diagnosis and in patients disease-free at 5 years post diagnosis. The authors reported that use of BCI was projected to be cost saving in both models. In the newly diagnosed population, net cost savings were $3803 per patient tested. In the 5 years post diagnosis population, BCI was projected to yield a net cost savings of $1803 per patient tested. The authors reported that sensitivity analyses demonstrated that BCI was cost saving across a wide range of clinically relevant input assumptions.\n",
      "Preliminary data suggest that molecular approaches including gene expression platforms such as BCI may add to classical clinical parameters including tumor size and node status at diagnosis, but further research is needed (Smith, et al., 2014; Bianchini & Gianni, 2013; Ignatiadis and Sotiriou, 2013). The clinical utility of BCI and other molecular diagnostics in predicting late recurrence has yet to be established (Foukakis and Bergh, 2015). It also remains to be established which of several molecular diagnostic tests in development are the most appropriate for detecting late recurrence (Sestak & Kuzick, 2015).\n",
      "An assessment by the National Institute for Health Research (Ward, et al., 2013) found that, based on the limited available data, no firm conclusions can be drawn about the analytical validity, clinical validity (prognostic ability) and clinical utility of the Breast Cancer Index. The assessment stated that further evidence on the prognostic and predictive ability of this test is required. An assessment by IETS (2013) and a consensus statement (Azim, et al., 2013) reached similar conclusions.\n",
      "An assessment by the BlueCross BlueShield Association (2015) concluded that the evidence is insufficient to permit conclusions about the Breast Cancer Index on health outcomes. Although evidence supports the association of risk classes defined by the Breast Cancer Index and recurrence and survival outcomes, it remains to be shown whether the Breast Cancer Index adds incremental prognostic information to standard clinical risk classifiers.\n",
      "An assessment by the Belgian Healthcare Knowledge Centre (KCE) (San Miguel, et al.,2015) found that the evidence for the H/I ratio assay is limited to studies supporting the prognostic ability (clinical validity) of the test. They found insufficient evidence for the impact of the H/I ratio assay on clinical management (clinical utility).\n",
      "A review published in the ASCO Educational Book (Smith, et al., 2014) reviewed the BCI and other currently available molecular diagnostics for selecting and determining the optimal duration of endocrine adjuvant therapy in women with early stage estrogen receptor positive breast cancer: \"Further research into applying molecular features and gene expression scores to standard clinico-pathologic criteria for tailoring extended endocrine therapy is now a high priority.... An important research challenge is now to identify which patients are likely to benefit from this type of long-term therapy. Preliminary data suggest that molecular approaches including gene expression platforms such as ROR may add to classical clinical parameters including tumor size and node status at diagnosis.\"\n",
      "A Palmetto Medicare Local Coverage Determination (LCD) allows coverage of the Breast Cancer Index in certain post-menopausal women with estrogen-receptor positive breast cancer, reasoning that the data defined benefit of the BCI test appears to be when a woman is having significant side effects or has other concerns regarding adjuvant tamoxifen therapy and is opposed to taking more than 5 years of tamoxifen or starting on an AI (letrazole) after tamoxifen (CMS, 2014). The LCD noted, however, that, there is an increase in recurrence risk with increasing BCI score such that, \"at the 95% confidence interval (CI), the risk in some individuals categorized in the BCI-low group could be as high as 20%. Due to the data complexity, there is a significant possibility that a physician might consider all BCI-L patients at negligible risk, and thus not consider extended hormone therapy and consequently lead women from the NCCN recommended interventions. Given the low toxicity and low cost of extended therapy, the false sense of security could deny many women from lifesaving therapy.\"\n",
      "There is a lack of consensus among guidelines regarding the value of molecular assays in determining whether longer durations of adjuvant endocrine therapy beyond 5 years are clinically indicated. Guidelines from the American Society for Clinical Oncology (Burstein, et al., 2014) on adjuvant endocrine therapy for hormone-receptor positive breast cancer state: \"Well-established clinical factors including tumor size; nodal status; ER, PgR, and HER2 biomarkers; and molecular diagnostic assays serve as prognostic factors for breast cancer recurrence. However, there are no robust specific clinical or biomarker measures that selectively predict early versus late recurrence, nor predict whether tamoxifen or AI therapy would be appropriate treatment, nor determine whether longer durations of adjuvant endocrine therapy are clinically indicated.\" The National Comprehensive Cancer Network guidelines for breast cancer version 2, 2015 states: \"Multiple other multi-gene mor multi-gene expression assay systems have been developed. These systems are generally based upon small, retrospective studies, and the Panel believes that none are currently sufficiently validated to warrant inclusion in the guideline.\" The St. Gallen guideline panel (Coates, et al., 2015) found that Oncotype DX, MammaPrint, PAM-50 ROR score, EndoPredict and the Breast Cancer Index were all considered usefully prognostic for years 1-5, but only the Oncotype Dx commanded a majority in favor of its value in predicting the usefulness of chemotherapy. The Panel agreed that the PAM50 ROR score was clearly prognostic beyond five years, and that the Mammaprint was not prognostic beyond 5 years. The Panel was divided about the prognostic value of the Breast Cancer Index, the Oncotype DX, and EndoPredict in this time period. ESMO guidelines (Senkus, et al., 2013) state: \"Molecular signatures for ER-positive breast cancer such as OncotypeDx, EndoPredict, Breast Cancer Index or for all types of breast cancer (pNO-1) such as MammaPrint and Genomic Grade Index are commercially available, but none of them have proven robust clinical utility so far. In some cases of difficult decision, such as grade 2 ER-positive HER-2 negative and node-negative breast cancer, MammaPrint and Oncotype DX may be used in conjunction with all clinicopathological factors, to help in treatment decision-making.\"\n",
      "Guidelines from the American Society for Clinical Oncology (2016) state: \"If a patient has ER/PgR-positive, HER2-negative (node-negative) breast cancer, the clinician may use the Breast Cancer Index to guide decisions on adjuvant systemic therapy.\" This is a moderate strength recommendation based upon intermediate quality evidence. ASCO guidelines recommend use of the Breast Cancer Index to guide decisions on adjuvant systemic therapy in patients with ER/PgR=positive, HER2-negative (node-positive) breast cancer. The guidelines also recommend against the use of the Breast Cancer Index in HER2-positive breast cancer or TN breast cancer. The guidelines also recommended against the use of The Breast Cancer Index to guide decisions on extended endocrine therapy for patients with ER/PgR-positive, HER-2 negative (node-negative) breast cancer who have had 5 years of endocrine therapy without evidence of recurrence.  \n",
      "Mammostrat  \n",
      "Mammostrat (Clarient) is a novel test for estimating the risk for recurrence in hormone-receptor positive, early stage breast cancer that is independent of proliferation and grade (Raman, et al., 2013). Five biomarkers are combined with a defined mathematical algorithm resulting in a risk index. Mammostrat is clinically validated and has been studied on more than 4,500 total patients in numerous independent cohorts that include the NSABP B14 and B20 trials. Clinicians and patients are faced with difficult choices as to whether to add toxic adjuvant chemotherapy in addition to standard endocrine treatment. Mammostrat may help clinicians understand the inherent aggressiveness of the tumor and the likelihood of tumor recurrence.\n",
      "The Mammostrat is a prognostic immunohistochemistry (IHC) test that measures the risk of breast cancer recurrence in post-menopausal, node-negative, estrogen receptor-expressing breast cancer patients who will receive hormonal therapy and are considering adjuvant chemotherapy. The test analyzes five monoclonal antibody biomarkers and applies a diagnostic algorithm to assess whether patients have a high, moderate, or low risk of recurrence after they have had their breast cancer tumor surgically removed and have been treated with tamoxifen.\n",
      "Bartlett et al (2010) tested the efficacy of the Mammostrat in a mixed population of cases treated in a single center with breast-conserving surgery and long-term follow-up.  Tissue microarrays from a consecutive series of 1,812 women managed by wide local excision and post-operative radiotherapy were collected.  Of 1,390 cases stained, 197 received no adjuvant hormonal or chemotherapy, 1,044 received tamoxifen only, and 149 received a combination of hormonal therapy and chemotherapy.  Median age at diagnosis was 57 years, 71% were post-menopausal, 23.9% were node-positive and median tumor size was 1.5 cm.  Samples were stained using triplicate 0.6 mm2 tissue microarray cores, and positivity for p53, HTF9C, CEACAM5, NDRG1 and SLC7A5 was assessed.  Each case was assigned a Mammostrat risk score, and distant recurrence-free survival (DRFS), relapse-free survival (RFS) and overall survival (OS) were analyzed by marker positivity and risk score.  Increased Mammostrat scores were significantly associated with reduced DRFS, RFS and OS in ER-positive breast cancer (p < 0.00001).  In multivariate analyses the risk score was independent of conventional risk factors for DRFS, RFS and OS (p < 0.05).  In node-negative, tamoxifen-treated patients, 10-year recurrence rates were 7.6 +/- 1.5% in the low-risk group versus 20.0 +/- 4.4% in the high-risk group.  Further, exploratory analyses revealed associations with outcome in both ER-negative and un-treated patients.  The authors concluded that the Mammostrat can act as an independent prognostic tool for ER-positive, tamoxifen-treated breast cancer and the results of the study revealed a possible association with outcome regardless of node status and ER-negative tumors.\n",
      "There is insufficient evidence to determine whether the Mammostrat test is better than conventional risk assessment tools in predicting the recurrence of breast cancer.  Furthermore, neither NCCN or ASCO have incorporated the test into their guidelines as a management tool. Guidance from the National Institute for Health and Clinical Excellence (NICE, 2013) states that the Mammostrat is \"only recommended for use in research in people with ER+, LNâ and HER2â early breast cancer, to collect evidence about potentially important clinical outcomes and to determine the ability of the tests to predict the benefit of chemotherapy ... The tests are not recommended for general use in these people because of uncertainty about their overall clinical benefit and consequently their cost effectiveness.\"\n",
      "An assessment by the Belgian Healthcare Knowledge Centre (KCE) (San Miguel, et al., 2015) found that the evidence for Mammostrat is mainly limited to studies supporting the prognostic ability (clinical validity) of the test. The KCE stated that these studies include a large sample size and appear to be of reasonable quality. The KCE cited one study reporting on clinical utility in terms of the predictive ability of the test by risk group. \"However, further evidence is required.\"\n",
      "Guidelines from the American Society for Clinical Oncology (2016) state: \"If a patient has ER/PgR-positive, HER2-negative (node-positive or node-negative) breast cancer, the clinician should not use the five-protein assay (Mammostrat; Clarient, a GE Healthcare company, Aliso Viejo, CA) to guide decisions on adjuvant systemic therapy.\" This is a moderate strength recommendation based upon intermediate-quality evidence. The ASCO guidelines recommend against the use of Mammostrat to guide decisions on adjuvant systemic therapy for patients with HER2-positive or TN breast cancer.\n",
      "OvaChek\n",
      "The OvaCheckâ¢ (Correlogic Systems, Inc.) is a proteomic analysis of blood for the early detection of ovarian cancer.  A similar test, which involved a different molecular pattern, was the subject of a 2002 study of 216 women with ovarian cancer.  That study showed that the proteomic test had a specificity of 100% and a sensitivity of 95%, with a positive predictive value of 94% (Petricoin, et al., 2002).  While this study showed that a proteomic test detected ovarian cancers even where CA-125 levels were normal, this study included only women who had been detected with ovarian cancer by other means.  There is inadequate evidence that this test will be effective for screening women with undetected ovarian cancer. \n",
      "In addition, there is concern, given the low prevalence of ovarian cancer, that this test is not sufficiently specific for use in screening.  The National Cancer Institute explains that even an ovarian cancer test with a specificity of 99% means that 1% of those who did not have cancer would test positive, which is \"far too high a rate for commercial use\" (NCI, 2004). For a rare disease such as ovarian cancer, which has an approximate prevalence of 1 in 2,500 in the general population, a 99% specificity and 100% sensitivity translates into 25 women falsely identified for every one true cancer found.\n",
      "The OvaCheckâ¢ test employs electrospray ionization (ESI) type of mass spectrometry using highly diluted denatured blood samples.  This method differs from a matrix-assisted laser desorption ionization (MALDI) analysis of undiluted native sera samples that was used in the Lancet study and is currently under investigation by the National Cancer Institute and Food and Drug Administration (NCI, 2004).  The NCI notes that \"[t]he class of molecules analyzed by these two approaches, and thus the molecules that constitute the diagnostic patterns, would be expected to be entirely different.\" Neither the NCI nor FDA has been involved in the design or validation of OvaCheckâ¢ methodology. \n",
      "As the Ovacheck test is performed as a \"home-brewed\" test by two national laboratories instead of as a commercially available kit, FDA approval of the OvaCheck test may not be required.  The Society for Gynecologic Oncologists (SGO, 2004) has reviewed the literature regarding OvaCheck and concluded that \"more research is needed to validate the testâs effectiveness before offering it to the public.\"  Similarly, the American College of Obstetricians and Gynecologists (2004) has stated that \"more research is needed to validate the test's effectiveness before recommending it to the public.\"\n",
      "An assessment of the Ovacheck test and other genomic tests for ovarian cancer prepared for the Agency for Healthcare Research and Quality by the Duke Evidence-Based Practice Center (Myers, et al., 2006) reached the following conclusions:  \"Genomic test sensitivity/specificity estimates are limited by small sample sizes, spectrum bias, and unrealistically large prevalence of ovarian cancer; in particular, estimates of positive predictive values derived from most of the studies are substantially higher than would be expected in most screening or diagnostic settings. We found no evidence relevant to the question of the impact of genomic tests on health outcomes in asymptomatic women. Although there is a relatively large literature on the association of test results and various clinical outcomes, the clinical utility of changing management based on these results has not been evaluated.\"  Specifically regarding Ovacheck and other proteomic tests for ovarian cancer, the assessment found that, \"[a] lthough all studies reported good discrimination for the particular protein profile studied, there were several recurrent issues that limit the ability to draw inferences about potential clinical applicability,\" in particular technical issues with the assays themselves, variations in analytic methods used among studies, and an unrealistically high prevalence of ovarian cancer in the datasets compared to what would be expected in a normal screening population.\n",
      "OvaSure\n",
      "OvaSure is an ovarian cancer screening test that entails the use of 6 biomarkers (leptin, prolactin, osteopontin, insulin-like growth factor II, macrophage inhibitory factor and CA-125) to assess the presence of early stage ovarian cancer in high-risk women.  Visintin et al (2008) characterized and validated the OvaSure for discriminating between disease-free and ovarian cancer patients.  These researchers analyzed 362 healthy controls and 156 newly diagnosed ovarian cancer patients.  Concentrations of leptin, prolactin, osteopontin, insulin-like growth factor II, macrophage inhibitory factor, and CA-125 were determined using a multiplex, bead-based, immunoassay system.  All 6 markers were evaluated in a training set (181 samples from the control group and 113 samples from ovarian cancer patients) and a test set (181 sample control group and 43 ovarian cancer).  Multiplex and ELISA exhibited the same pattern of expression for all the biomarkers.  None of the biomarkers by themselves was good enough to differentiate healthy versus cancer cells.  However, the combination of the 6 markers provided a better differentiation than CA-125.  Four models with less than 2% classification error in training sets all had significant improvement (sensitivity 84 % to 98% at specificity 95%) over CA-125 (sensitivity 72% at specificity 95%) in the test set.  The chosen model correctly classified 221 out of 224 specimens in the test set, with a classification accuracy of 98.7%.  The authors noted that the OvaSure is the first blood biomarker test with a sensitivity of 95.3% and a specificity of 99.4% for the detection of ovarian cancer.  Six markers provided a significant improvement over CA-125 alone for ovarian cancer detection.  Validation was performed with a blinded cohort.  They stated that this novel multiplex platform has the potential for efficient screening in patients who are at high risk for ovarian cancer.\n",
      "However, the Society of Gynecologic Oncologists (SGO, 2008) released an opinion regarding OvaSure, which stated that additional research is needed before the test should be offered to women outside the context of a research study.  Moreover, SGO stated that it will \"await the results of further clinical validation of OvaSure with great interest\".\n",
      "Furthermore, according to the FDAâs web site, the FDA sent the Laboratory Corporation of America a warning letter stating that it is illegally marketing OvaSure to detect ovarian cancer.  According to the FDA warning letter, their review indicates that this product is a device under section 201(h) of the Food, Drug, and Cosmetic Act (FDCA or Act), 21 U.S.C. 321(h), because it is intended for use in the diagnosis of disease or other conditions, or in the cure, treatment, prevention, or mitigation of disease. The Act requires that manufacturers of devices that are not exempt obtain marketing approval or clearance for their products from the FDA before they may offer them for sale. This helps protect the public health by ensuring that new devices are shown to be both safe and effective or substantially equivalent to other devices already legally marketed in this country for which approval is not required.  According to the FDA warning letter, no such determination has been made for OvaSure.\n",
      "NCCN Guidelines Panel Members (NCCN, 2016) believe that the OvaSure screening test should not be used to detect ovarian cancer. The NCCN guidelines explain that the OvaSure test uses 6 biomarkers, including leptin, prolactin, osteopontin, insulin-like growth factor II, macrophae inhibitory factor, and CA-125.\n",
      "Thrombospondin-1\n",
      "Thrombospondin-1 (THBS-1), an angiogenesis inhibitor, has been identified as a potential monitoring marker in gynecologic malignancies.  In a randomized phase III study on the co-expression of angiogenic markers and their associations with prognosis in advanced epithelial ovarian cancer, Secord, et al. (2007) reported that high THBS-1 may be an independent predictor of worse progression-free and overall survival in women with advanced-stage EOC.  However, the authors stated, \"A larger prospective study is warranted for validation of these findings.\"\n",
      "Previstage GCC\n",
      "Guanylyl Cyclase C (GCC or GUCY2C) (Diagnocure) a gene coding for a protein found in cells, lining the intestine from the duodenum to the rectum (Raman, et al., 2013). It is involved in water transport, crypt morphology and suppression of tumorigenesis. It is not normally found in tissue in other parts of the body, and therefore, GCC detected outside of the intestine, indicates presence of colorectal cancer metastases. Early studies have indicated that the presence of GCC in the blood may be an early indicator of micrometastases that would otherwise escape detection by the current standard methods of monitoring. Earlier detection provides an opportunity for more immediate treatment or surgical intervention to potentially improve patient outcomes and survival rates. This is a  diagnostic test for recurrence by identification of micrometastasis in the blood.\n",
      "Guanyl cyclase C (GCC) is a receptor protein normally expressed in high concentrations on the luminal surface of the gastrointestinal epithelium.  Expression of GCC persists on mucosal cells that have undergone malignant transformation.  Thus, GCC has potential use as a marker to determine spread of colorectal cancer to lymph nodes.  A retrospective study of 21 patients post surgical resection of colorectal cancer found that all 11 of 21 patients who were free of cancer for 5 years or more were negative for GCC in lymph nodes, whereas all 10 of 21 patients whose cancer returned within 3 years of surgery were positive for GCC.  However, the value of the GCC marker test in the management of colorectal cancer needs to be evaluated in prospective clinical outcome studies.  A large prospective study is currently being conducted to compare standard histological examination of lymph nodes to the GCC marker test.\n",
      "Previstageâ¢ Guanylyl Cyclase C (GCC or GUCY2C) (Diagnocure) is a gene coding for a protein found in cells, lining the intestine from the duodenum to the rectum (Raman, et al., 2013). It is involved in water transport, crypt morphology and suppression of tumorigenesis. It is not normally found in tissue in other parts of the body, and therefore, GCC detected outside of the intestine, indicates presence of colorectal cancer metastases. GCC mRNA has shown to be highly accurate in detecting the spread and recurrence of colorectal cancer, respectively in lymph nodes and blood, thereby representing a significant improvement over traditional detection methods. Previstage is a predictive test for risk stratification of recurrence and prognostic marker for recurrence.\n",
      "Thymidylate Synthase\n",
      "Thymidylate synthase is a DNA synthesis related gene.  According to Compton (2008), the prognostic value of this promising and potentially clinically applicable molecular marker has been studied in colorectal cancer.  Compton found that the independent influence of this marker on prognosis remains unproven.  Compton explained that \"[v]ariability in assay methodology, conflicting results from various studies examining the same factor, and the prevalence of multiple small studies that lack statistically robust, multivariate analyses all contribute to the lack of conclusive data.\"  Compton concluded that before this marker can be incorporated into clinically meaningful prognostic stratification systems, more studies are required using multivariate analysis, well-characterized patient populations, reproducible and current methodology, and standardized reagents.\n",
      "In a special report on pharmacogenomics of cancer, the BlueCross and BlueShield Association's Technology Evaluation Center (TEC) (2007) described the results of a meta-analysis on thymidylate synthase protein expression and survival in colorectal cancer that stated low thymidylate synthase expression was significantly associated with better survival, but heterogeneity and possible bias prevented firm conclusions.\n",
      "Guidelines from the American Society for Colon and Rectal Surgeons (2004) stated: \"In the future, DNA analysis and the intratumoral expression of specific chemical substances\", including thymidylate synthase, \"may be used routinely to further assess prognosis or response to therapy.\"  In addition, Shankaran et al (2008) stated in a review on the role of molecular markers in predicting response to therapy in patients with colorectal cancer, \"Although to date no molecular characteristics have emerged as consistent predictors of response to therapy, retrospective studies have investigated the role of a variety of biomarkers, including microsatellite instability, loss of heterozygosity of 18q, type II transforming growth factor beta receptor, thymidylate synthase, epidermal growth factor receptor, and Kirsten-ras (KRAS).\"\n",
      "VEGF\n",
      "Tumour angiogenesis is associated with invasiveness and the metastatic potential of various cancers.  Vascular endothelial growth factor (VEGF), the most potent and specific angiogenic factor identified to date, regulates normal and pathologic angiogenesis.  An evidence report from Cancer Care Ontario (Welch et al, 2008) on the use of the VEGF inhibitor bevacizumab in colorectal cancer explained that the increased expression of VEGF has been correlated with metastasis, recurrence, and poor prognosis in many cancers, including colorectal cancer. Guidelines from the National Institute for Health and Clinical Excellence (NICE, 2007) explained that bevacizumab (Avastin) is a recombinant humanised monoclonal IgG1 antibody that acts as an angiogenesis inhibitor.  It targets the biological activity of VEGF, which stimulates new blood vessel formation in the tumour.  However, neither the FDA approved labeling of bevacizumab or evidence-based guidelines recommend measurement of VEGF to diagnose colorectal cancer or to select patients for treatment.  In a special report on pharmacogenomics of cancer, the BlueCross and BlueShield Association's Technology Evaluation Center (TEC) (2007) stated that pre-treatment VEGF levels do not appear to be predictive of response to anti-angiogenic therapy.\n",
      "Shin and colleagues (2013) evaluated inhibitory effects of bevacizumab on VEGF signaling and tumor growth in-vitro and in-vivo, and assessed phosphorylation of VEGF receptor 2 (VEGFR2) and downstream signaling in endothelial cells as pharmacodynamic markers using phospho-flow cytometry.  These researchers also validated markers in patients with mCRC treated with bevacizumab-based chemotherapy.  In in-vitro studies, bevacizumab inhibited proliferation of human umbilical vein endothelial cells in association with reduced VEGF signaling.  Notably, bevacizumab inhibited VEGF-induced phosphorylation of VEGFR-2, Akt, and extra-cellular signal-regulated kinase (ERK).  In-vivo, treatment with bevacizumab inhibited growth of xenografted tumors and attenuated VEGF-induced phosphorylation of Akt and ERK.  The median percentages of VEGFR2â+âpAktâ+âand VEGFR2â+âpERKâ+âcells, determined by phospho-flow cytometry, were approximately 3-fold higher in mCRC patients than in healthy controls.  Bevacizumab treatment decreased VEGFR2â+âpAktâ+âcells in 18 of 24 patients on day 3.  The authors concluded that bevacizumab combined with chemotherapy decreased the number of VEGFR2â+âpAktâ+âcells, reflecting impaired VEGFR2 signaling.  Together, these data suggested that changes in the proportion of circulating VEGFR2â+âpAktâ+âcells may be a potential pharmacodynamic marker of the effectiveness of anti-angiogenic agents, and could prove valuable in determining drug dosage and administration schedule.\n",
      "ProstatePx\n",
      "Donovan et al (2008) from Aureon, the manufacturer of Prostate Px, reported on the development and validation of their systems pathology model for predicting prostate cancer recurrence after prostatectomy. The clinical utility of defining high risk for failure after radical prostatectomy is to decide whether patients require closer follow-up than average or whether adjuvant radiotherapy, hormone therapy, or chemotherapy would be of benefit. In this analysis, the concordance index for the systems pathology approach used by Aureon was 0.83, but was only slightly better than a 10-variable model that used only the usual clinical parameters, with a concordance index of 0.80. The corresponding hazard ratios for clinical failure were 6.37 for the 10-variable clinical model, and 9.11 for the systems pathology approach. In an accompanying editorial, Klein, et al. (2009)  questioned the clinical significance of these differences. They noted that \"[a]lthough the difference in concordance indices was statistically significant, the question is whether there is sufficient clinical relevance to justify the extra effort, expense, and clinical expertise needed for the systems approach ... In contemporary clinical practice, a patient with a hazard ratio of 6.37 generated by the model using easily derived, routinely reported clinical and pathological parameters is just as likely to be a candidate for closer monitoring or adjuvant therapy than one with a hazard ratio of 9.11 generated by the systems approach\".\n",
      "Sutcliffe et al (2009) provided an evidence-based perspective on the prognostic value of novel markers in localized prostate cancer and identified the best prognostic model including the 3 classical markers and investigated if models incorporating novel markers are better. Eight electronic bibliographic databases were searched. The reference lists of relevant articles were checked and various health services research-related resources consulted via the internet. The search was restricted to publications from 1970 onwards in the English language. Selected studies were assessed, data extracted using a standard template, and quality assessed using an adaptation of published criteria. Because of the heterogeneity regarding populations, outcomes and study type, meta-analyses were not undertaken and the results are presented in tabulated format with a narrative synthesis of the results. A total of 30 papers met the inclusion criteria, of which 28 reported on prognostic novel markers and 5 on prognostic models. A total of 21 novel markers were identified from the 28 novel marker studies. There was considerable variability in the results reported, the quality of the studies was generally poor and there was a shortage of studies in some categories. The marker with the strongest evidence for its prognostic significance was PSA velocity (or doubling time). There was a particularly strong association between PSA velocity and prostate cancer death in both clinical and pathological models. In the clinical model the hazard ratio for death from prostate cancer was 9.8 (95 % CI 2.8 to 34.3, p < 0.001) in men with an annual PSA velocity of more than 2 ng/ml versus an annual PSA velocity of 2 ng/ml or less; similarly, the hazard ratio was 12.8 (95 % CI 3.7 to 43.7, p < 0.001) in the pathological model. The quality of the prognostic model studies was adequate and overall better than the quality of the prognostic marker studies. Two issues were poorly dealt with in most or all of the prognostic model studies:\n",
      "\n",
      "inclusion of established markers, and\n",
      "consideration of the possible biases from study attrition. Given the heterogeneity of the models, they can not be considered comparable.\n",
      "\n",
      "Only 2 models did not include a novel marker, and 1 of these included several demographical and co-morbidity variables to predict all-cause mortality. Only 2 models reported a measure of model performance, the C-statistic, and for neither was it calculated in an external data set. It was not possible to assess whether the models that included novel markers performed better than those without. This review highlighted the poor quality and heterogeneity of studies, which render much of the results inconclusive. It also pinpointed the small proportion of models reported in the literature that are based on patient cohorts with a mean or median follow-up of at least 5 years, thus making long-term predictions unreliable. Prostate-specific antigen velocity, however, stood out in terms of the strength of the evidence supporting its prognostic value and the relatively high hazard ratios. There is great interest in PSA velocity as a monitoring tool for active surveillance but there is as yet no consensus on how it should be used and, in particular, what threshold should indicate the need for radical treatment.\n",
      "In an editorial on clinically relevant prognostic markers for prostate cancer, Gelmann and Henshall (2009) stated that \"[u]ntil we have sufficiently discriminating markers to inform treatment decisions, the problem of whom to treat will continue to grow exponentially as the number of cases of screening-detected low-risk cancer increases\".\n",
      "Circulating Tumor Cells (e.g., CELLSEARCH)\n",
      "Circulating tumor cell (CTC) test, CELLSEARCH, is a blood test that has been proposed as a method to determine prognosis, evaluate progression and assess treatment response in individuals with metastatic breast, colorectal and prostate cancers. CTC assays were developed to detect cells that break away from tumors and enter the blood stream. \n",
      "The CellSearchâ¢ Epithelial Cell Kit, along with the CellSpotterâ¢ Analyzer (Veridex, LLC, Warren, NJ) is a device designed to automate the detection and enumeration of circulating tumor cells (CTCs) of epithelial origin (CD45-, EpCAM+, and cytokeratins 8, 18+ and/or 19+) in whole blood in patients with advanced breast cancer (Ellery, et al., 2010; Raman, et al., 2011).  It is intended for use in adjunctively monitoring and predicting cancer disease progression and response to therapy. \n",
      "The CellSearch Epithelial Cell Kit received FDA 510(k) clearance on January 21, 2004.  The FDA concluded that the device is substantially equivalent to immunomagnetic circulating cancer cell selection and enumeration systems.  These devices consist of biological probes, fluorochromes and other reagents, preservation and preparation devices and semi-automated analytical instruments to select and count circulating cancer cells in a prepared sample of whole blood. \n",
      "The CellSearch Epithelial Cell Kit quantifies CTCs by marking cancerous cells with tiny, protein-coated magnetic balls in whole blood.  These cells are stained with fluorescent markers for identification and then dispensed into a cartridge for analysis where a strong magnetic field is applied to the mixture causing the magnetically marked cells to move to the cartridge surface.  The cartridge is then analyzed by the CellSpotter Analyzer.  A medical professional rechecks the CTCs and the CellSpotter Analyzer tallies the final CTC count. \n",
      "In a prospective, multicenter study, Cristofanilli et al (2004) used the CellSearch System on 177 patients with measurable metastatic breast cancer for levels of CTCs both before the patients started a new line of therapy and at follow-up.  The progression of the disease or the response to treatment was determined with the use of standard imaging studies at the participating centers every nine to twelve weeks.  Outcomes were assessed according to levels of CTCs at baseline, before the patients started a new therapy.  In the first test, patients with 5 or more CTCs per 7.5 ml of blood compared to a group with fewer than 5 CTCs had a shorter median progression-free survival (2.7 months vs. 7.0 months) and shorter overall survival (10.1 months vs. greater than 18 months).  At the follow-up visit, approximately three to four weeks after the initiation of therapy, the percentage of patients with more than 5 CTC was reduced from 49 percent to 30 percent, suggesting a benefit from therapy.  The difference in progression-free survival between the two groups remained consistent (2.1 months for women with 5 or more CTCs vs. 7 months for women with less than 5 CTCs).  Overall, survival in the women with more than 5 CTCs was 8.2 months compared to greater than 18 months in the cohort with less than 5 CTCs.  Cristofanilli concluded that the number of CTCs before treatment was an independent predictor of progression-free survival and overall survival in patients with metastatic breast cancer.  However, Cristofanilli also concluded that the results may not be valid for patients who do not have measurable disease or for those starting a new regimen of hormone therapy, immunotherapy, or both.  He states, \"The prognostic implications of an elevated level of circulating tumor cells for patients with metastatic disease who are starting a new treatment may be an opportunity to stratify these patients in investigational studies\".  Furthermore, the study did not address whether patients with an elevated number of circulating tumor cells might benefit from other therapies.  Thus, this minimally invasive assay requires further evaluation as a prognostic marker of disease progression and response to therapy. \n",
      "The clinical application of quantifying CTCs in the peripheral blood of breast cancer patients remains unclear.  Published data in the peer-reviewed medical literature are needed to determine how such measurements would guide treatment decisions and whether these decisions would result in beneficial patient outcomes (Kahn, et al., 2004; Abeloff, et al., 2004). An assessment of CellSearch by AETSA (2006) concluded \"In the current stage of development of this technology, there is no evidence that it provides any advantage over existing technology for CTC identification or indeed any additional clinical use.\" Guidelines from the American Society for Clinical Oncology (Harris, et al., 2007) found: \"The measurement of circulating tumor cells (CTCs) should not be used to make the diagnosis of breast cancer or to influence any treatment decisions in patients with breast cancer. Similarly, the use of the recently U.S. Food and Drug Administration (FDA)-cleared test for CTC (CellSearch Assay) in patients with metastatic breast cancer cannot be recommended until further validation confirms the clinical value of this test.\"\n",
      "An assessment by the Canadian Agency for Drugs and Technologies in Health (CADTH, 2012) found that studies indicate that measurement of CTCs using the CellSearch system could be used as prognostic factors for progression of the disease and the potential treatment of patients with ovarian cancer. No economic studies were identified, therefore the cost-effectiveness of the CellSearch system could not be summarized.\n",
      "Although studies relate circulating tumor cells to prognostic indicators (see, e.g., Cohen, et al., 2008; De Giorgi, et al., 2009), there are a lack of published prospective clinical studies demonstrating that measurement of CTCs alters management such that clinical outcomes are improved. Such clinical outcome studies are currently ongoing. Current guidelines from the National Comprehensive Cancer Network (NCCN) make no recommendations for use of circulating tumor cells.\n",
      "Guidelines from the American Society for Clinical Oncology (2016) state: \"The clinician should not use circulating tumor cells to guide decisions on adjuvant systemic therapy.\" This is a strong recommendation based upon intermediate-quality evidence.  \n",
      "Scher et al (2015) noted that clinical trials in castration-resistant prostate cancer (CRPC) need new clinical end-points that are valid surrogates for survival. These researchers evaluated circulating tumor cell (CTC) enumeration as a surrogate outcome measure.  Examining CTCs alone and in combination with other biomarkers as a surrogate for OS was a secondary objective of COU-AA-301, a multi-national, randomized, double-blind phase III trial of abiraterone acetate plus prednisone versus prednisone alone in patients with metastatic CRPC previously treated with docetaxel.  The biomarkers were measured at baseline and 4, 8, and 12 weeks, with 12 weeks being the primary measure of interest.  The Prentice criteria were applied to test candidate biomarkers as surrogates for OS at the individual-patient level.  A biomarker panel using CTC count and lactate dehydrogenase (LDH) level was shown to satisfy the 4 Prentice criteria for individual-level surrogacy; 12-week surrogate biomarker data were available for 711 patients.  The abiraterone acetate plus prednisone and prednisone-alone groups demonstrated a significant survival difference (p = 0.034); surrogate distribution at 12 weeks differed by treatment (p < 0.001); the discriminatory power of the surrogate to predict mortality was high (weighted c-index, 0.81); and adding the surrogate to the model eliminated the treatment effect on survival.  Overall, 2-year survival of patients with CTCs less than 5 (low risk) versus patients with CTCs greater than or equal to 5 cells/7.5 ml of blood and LDH greater than 250 U/L (high risk) at 12 weeks was 46 % and 2 %, respectively.  The authors concluded that a biomarker panel containing CTC number and LDH level was shown to be a surrogate for survival at the individual-patient level in this trial of abiraterone acetate plus prednisone versus prednisone alone for patients with metastatic CRPC.  They stated that independent phase III clinical trials are needed to validate these findings.\n",
      "An assessment from the Institut National dâExcellence en SantÃ© et Services Sociaux (INESSS) (Arsenault & Le Blanc, 2016) concluded: \"Based on the scientific literature identified, the use of CellSearch tests as a predictive and prognostic biomarker in patients with early-stage breast cancer is not justified. The evidence is insufficient for establishing a concrete association between the presence of CTCs pre- and posttreatment and patient survival. In the case of patients with metastatic breast cancer, the examination of the scientific literature suggests that CTC enumeration prior to treatment could be a prognostic biomarker for patient survival. Despite the prognostic value of CTC enumeration, based on studies, its clinical utility has yet to be confirmed. For now, CellSearch tests should not be used outside the context of a clinical study. Further studies are needed to determine if the CellSearch test could play a clinically significant role in managing breast cancer patients.\"\n",
      "The CELLSEARCH Circulating Multiple Myeloma Test (Menarini Silicon Biosystems, Inc) analyzes peripheral blood for circulating multiple myeloma cells (CMMCs). The test consists of  circulating plasma cell immunologic selection, identification, morphological characterization, and enumeration of plasma cells based on differential CD138, CD38, CD19, and CD45 protein biomarker expression.\n",
      "The CELLSEARCH HER2 Circulating Tumor Cell (CTC-HER2) Test (Menarini Silicon Biosystems, Inc) analyzes peripheral blood for HER2 status of the circulating tumor cells. The test consists of circulating tumor cell selection, identification, morphological characterization, detection and enumeration based on differential EpCAM, cytokeratins 8, 18, and 19, and CD45 protein biomarkers, and quantification of HER2 protein biomarkerâexpressing cells.\n",
      "Her-2/neu\n",
      "Estrogen and progestin receptors are important prognostic markers in breast cancer, and the higher the percentage of overall cells positive as well as the greater the intensity, the better the prognosis. Estrogen and progesterone receptor positivity in breast cancer cells is an indication the patient may be a good candidate for hormone therapy. HER-2/neu is an oncogene. Its gene product, a protein, is over-expressed in approximately 20 to 30% of breast cancers. The over-expressed protein is present in unusually high concentration on the surface of some malignant breast cancer cells, causing these cells to rapidly proliferate. It is important because these tumors are susceptible to treatment with Herceptin (trastuzumab), which specifically binds to this over-expressed protein. Herceptin blocks these protein receptors, inhibiting continued replication and tumor growth. HER2/neu may also be expressed in ovarian, gastric, colorectal, endometrial, lung, bladder, prostate, and salivary gland (Chen, et al., 2006).\n",
      "HER-2/neu is an oncogene encoding a growth factor receptor related to epidermal growth factor receptor (EGFR) and is amplified in approximately 25-30 percent of node-positive breast cancers (Chin, et al. 2006). Overexpression of HER-2/neu is associated with decreased disease-free and overall survival. Over-expression of HER-2/neu may be used to identify patients who may be may benefit from trastuzumab (Herceptinâ¢ ) and/or high dose chemotherapy. Trastuzumab is a humanized monoclonal antibody targeting the HER 2/neu (c-erbB-2) oncogene.\n",
      "Her-2 has been used to: assess prognosis of stage II, node positive breast cancer patients; predict disease-free and overall survival in patients with stage II, node positive breast cancer treated with adjuvant cyclophosphamide, doxorubicin, 5-fluorouracil chemotherapy; and determine patient eligibility for Herceptin treatment (Chen, et al., 2006). The College of American Pathologists (CAP) recommends FISH as an optimal method for HER2/neu testing; therefore, positive IHC results are usually confirmed by FISH testing.\n",
      "There are additional tests that may be used in breast cancer cases, such as DNA ploidy, Ki-67 or other proliferation markers. However, most authorities believe that HER-2/neu, estrogen and progesterone receptor status are the most important to evaluate first. The other tests do not have therapeutic implications and, when compared with grade and stage of the disease, are not independently significant with respect to prognosis.\n",
      "Harris et al (2007) updated ASCO's recommendations for the use of tumor marker tests in the prevention, screening, treatment, and surveillance of breast cancer. Thirteen categories of breast tumor markers were considered, 6 of which were new for the guideline. The following categories showed evidence of clinical utility and were recommended for use in practice: CA 15-3, CA 27.29, CEA, estrogen receptor, progesterone receptor, human epidermal growth factor receptor 2, urokinase plasminogen activator, plasminogen activator inhibitor 1, and certain multi-parameter gene expression assays. Not all applications for these markers were supported, however. The following categories demonstrated insufficient evidence to support routine use in clinical practice: DNA/ploidy by flow cytometry, p53, cathepsin D, cyclin E (fragments or whole length), proteomics, certain multi-parameter assays, detection of bone marrow micrometastases, and circulating tumor cells (e.g., CellSearch assay). These guidelines found present data insufficient to recommend measurement of Ki67, cyclin D, cyclin E, p27, p21, thymidine kinase, topoisomerase II, or other markers of proliferation to assign patients to prognostic groups. The guidelines also found insufficient data to recommend assessment of bone marrow micrometastases for management of patients with breast cancer.\n",
      "Guidelines from the American Society for Clinical Oncology (2016) recommend against the use of soluble HER2 levels to guide selection of type of adjuvant therapy in breast cancer. This is a moderate-strength recommendation based upon low-quality evidence. The guidelines also recommend against the use of HER2 gene coamplification to guide adjuvant chemotherapy selection in breast cancer.\n",
      "PancraGen (formerly PathFinderTG - Pancreas)\n",
      "PathFinderTG (RedPath Integrated Pathology, Pittsburgh, PA), also known as topographic genotyping, is described by the manufacturer as a quantitative genetic mutational analysis platform for resolving \"indeterminate, atypical, suspicious, equivocal and non-diagnostic specimen\" diagnoses from pathology specimens (RedPath, 2007). The manufacturer states that PathFinder TG \"focuses on acquired mutational damage rather than inherited genetic predisposition for certain diseases, although there are certain NIH recommended inherited conditions for which we do test.\"  The manufacturer states that the temporal sequence of acquired mutational damage revealed by the PathFinderTG test is an earlier demonstration of tumor biological aggressiveness than current staging systems that rely on the depth of invasion already achieved by the tumor.  Most available published evidence for topographic genotyping focuses on retrospective analyses of pathology specimens examining correlations of test results with tumor characteristics (e.g., Saad et al, 2008; Lin et al, 2008; Finkelstein et al, 2003; Pollack et al, 2001; Riberio et al, 1998; Kounelis et al, 1998; Finkelstein et al, 1998; Holst et al, 1998; Jones et al, 1997; Holst et al, 1997; Pricolo et al, 1997; Przygodzki et al, 1997; Finkelstein et al, 1996; Kanbour-shakir et al, 1996; Ribeiro et al, 1996; Pryzgodzki et al, 1996; Safatle-Ribeiro et al, 1996; Papadaki et al, 1996; Przygodzki et al, 1996; Pricolo et al, 1996; Finkelstein et al, 1994). There are no prospective clinical outcome studies on the use of topographic genotyping in guiding patient management. Current evidence-based guidelines from leading medical professional organizations and public health agencies do not include recommendations for topographic genotyping. In a review on molecular analysis of pancreatic cyst fluid, Shen and colleagues (2009) stated that a large study with validation of PathFinderTG molecular testing of pancreatic fluid will be needed before a firm conclusion can be drawn.\n",
      "A systematic evidence review of the PathFinderTG prepared for the Agency for Healthcare Research and Quality (Trikalinos, et al., 2010) reviewed evidence available at that time, and found that most studies on loss-of-heterozygosity based topographic genotyping with PathfinderTG were excluded because they only described the molecular profile of different tumors, without assessing the ability of the method to help make diagnosis, prognosis or treatment guidance. The review found no studies that directly measured whether using loss-of-heterozygosity based topographic genotyping with PathfinderTG improves patient-relevant clinical outcomes. The review reported that eligible studies on the diagnostic and prognostic ability of loss-of-heterozygosity based topographic genotyping with PathfinderTG were small in sample sizes and had overt methodological limitations. The review reported that important characteristics of their designs were not clearly reported. The report noted that loss-of-heterozygosity based topographic genotyping with PathfinderTG is claimed to be particularly useful in cases where conventional pathology is unable to provide a conclusive diagnosis. However, the included studies were not designed to address this question. Therefore, it is unclear if the findings of the reviewed studies are directly applicable to patients with the same cancers but with inconclusive diagnosis.\n",
      "A subsequent study by Panarelli et al (2012) comparing PathFinderTG to cytological examination, finding concordance in 35 percent of cases. The authors concluded that the PathfinderTG panel may aid the classification of pancreatic lesions, but is often inaccurate and should not replace cytologic evaluation of these lesions.\n",
      "The manufacturer has announced that the PathginderTG - Pancreas has been rebranded Pancragen.\n",
      "Al Haddad et al (2015) reported on a multicenter retrospective chart review study to determine the diagnostic accuracy of integrated molecular pathology (Pancragen) for pancreatic adenocarcinoma, and the utility of IMP testing under current guideline recommendations for managing pancreatic cysts. The authors found that Pancragen more accurately determined the malignant potential of pancreatic cysts than a Sendai 2012 guideline management criteria model. Patients who had undergone previous Pancragen testing as prescribed by their physician and for whom clinical outcomes were available from retrospective record review were included (nâ=â492). Performance was determined by correlation between clinical outcome and previous Pancragen diagnosis (\"benign\"/\"statistically indolent\" vs. \"statistically higher risk [SHR]\"/\"aggressive\") or an International Consensus Guideline (Sendai 2012) criteria model for \"surveillance\" vs. \"surgery.\" The Cox proportional hazards model determined hazard ratios for malignancy. Benign and statistically indolent Pancragen diagnoses had a 97â% probability of benign follow-up for up to 7 years and 8 months from initial Pancragen testing. SHR and aggressive diagnoses had relative hazard ratios for malignancy of 30.8 and 76.3, respectively (both Pâ<â0.0001). Sendai surveillance criteria had a 97â% probability of benign follow-up for up to 7 years and 8 months, but for surgical  criteria the hazard ratio was only 9.0 (Pâ<â0.0001). In patients who met Sendai surgical criteria, benign and statistically indolent Pancragen diagnoses had a >â93â% probability of benign follow-up, with relative hazard ratios for SHR and aggressive IMP diagnoses of 16.1 and 50.2, respectively (both Pâ<â0.0001). The authors concluded that Pancragen may improve patient management by justifying more relaxed observation in patients meeting Sendai surveillance criteria.\n",
      "Loren et al (2016) used registry data to determine if initial adjunctive Pancragen testing influenced future real-world pancreatic cyst management decisions for intervention or surveillance relative to 2012 International Consensus Guideline (ICG) recommendations, and if this benefitted patient outcomes. Using data from the National Pancreatic Cyst Registry, the investigators evaluated associations between real-world decisions (intervention vs. surveillance), ICG model recommendations (surgery vs. surveillance) and Pancragen diagnoses (high-risk vs. low-risk) using 2 Ã 2 tables. The investigators used Kaplan Meier and hazard ratio analyses to assess time to malignancy. Odds ratios (OR) for surgery decision were determined using logistic regression.  Of 491 patients, 206 received clinical intervention at follow-up (183 surgery, 4 chemotherapy, 19 presumed by malignant cytology). Overall, 13 % (66/491) of patients had a malignant outcome and 87 % (425/491) had a benign outcome at 2.9 years' follow-up. When ICG and Pancragen were concordant for surveillance/surgery recommendations, 83 % and 88 % actually underwent surveillance or surgery, respectively. However, when discordant, Pancragen diagnoses were predictive of real-world decisions, with 88 % of patients having an intervention when ICG recommended surveillance but Pancragen indicated high risk, and 55 % undergoing surveillance when ICG recommended surgery but Pancragen indicated low risk. These Pancragen-associated management decisions benefitted patient outcomes in these subgroups, as 57 % had malignant and 99 % had benign outcomes at a median 2.9 years' follow-up. Pancragen was also more predictive of real-world decisions than ICG by multivariate analysis: OR 11.4 (95 % CI 6.0â-â23.7) versus 3.7 (2.4â-â5.8), respectively. \n",
      "Kowalski et al (2016) examined the utility of integrated molecular pathology (IMP) in managing surveillance of pancreatic cysts based on outcomes and analysis of false negatives (FNs) from a previously published cohort (n=492). In endoscopic ultrasound with fine-needle aspiration (EUS-FNA) of cyst fluid lacking malignant cytology, IMP demonstrated better risk stratification for malignancy at approximately 3 yearsâ follow-up than International Consensus Guideline (Fukuoka) 2012 management recommendations in such cases. The investigators reviewed patient outcomes and clinical features of Fukuoka and IMP FN cases. Practical guidance for appropriate surveillance intervals and surgery decisions using IMP were derived from follow-up data, considering EUS-FNA sampling limitations and high-risk clinical circumstances observed. Surveillance intervals for patients based on IMP predictive value were compared with those of Fukuoka. Outcomes at follow-up for IMP low-risk diagnoses supported surveillance every 2 to 3 years, independent of cyst size, when EUS-FNA sampling limitations or high-risk clinical circumstances were absent. In 10 of 11 patients with FN IMP diagnoses (2% of cohort), EUS-FNA sampling limitations existed; Fukuoka identified high risk in 9 of 11 cases. In 4 of 6 FN cases by Fukuoka (1% of cohort), IMP identified high risk. Overall, 55% of cases had possible sampling limitations and 37% had high-risk clinical circumstances. Outcomes support more cautious management in such cases when using IMP.\n",
      "An American Gastroenterological Association Technical Review (Scheiman et al, 2014) stated: \"Testing for molecular alterations in pancreatic cyst fluid is currently available and reimbursed by Medicare under certain circumstances. Case series have confirmed malignant cysts have greater number and quality of molecular alterations, but no study has been properly designed to identify how the test performs in predicting outcome with regard need to surgery, surveillance or predicts interventions leading to improved survival.  This adjunct to fine-needle aspiration (FNA) may provide value in distinct clinical circumstances, such as confirmation of a serous lesion due to a lack of KRAS or GNAS mutation in a macrocystic serous cystadenoma, but its routine use is not supported at the present time.\"\n",
      "A guideline from the American Society for Gastrointestinal Endoscopy (Muthusamy, et al., 2016) stated: \"A more recent study demonstrated that integrated molecular analysis of cyst fluid (ie, combining molecular analysis with results of imaging and clinical features) was able to better characterize the malignant potential of pancreatic cysts compared to consensus guidelines for the management of mucinous cysts [citing Al Haddad, et al., 2015]. ... Molecular analysis (which requires only 200 mL of fluid) may be most useful in small cysts with nondiagnostic cytology, equivocal cyst fluid CEA results, or when insufficient fluid is present for CEA testing [citing Al Haddad, et al., 2014]. However, additional research is needed to determine the precise role molecular analysis of cyst fluid will play in evaluating pancreatic cystic lesions.\"\n",
      "Guidelines published in April 2015 by the American Gastroenterological Association (Vege, et al., 2015) have no recommendations for use of topographic genotyping for evaluating pancreatic cysts. Other guidelines (NCCN, 2015; Vege, et al., 2015; Del Chiaro, et al., 2013; Sahani, et al., 2013; Tanaka, et al., 2012) have no firm recommendations for topographic genotyping for assessing indeterminate pancreatic cysts.\n",
      "The International consensus guidelines for \"The management intraductal papillary mucinous neoplasm (IPMN) and mucinous cystic neoplasm (MCN) of the pancreas\" (Tanaka et al, 2012) stated that endoscopic ultrasound (EUS) is recommended for all cysts with worrisome features or for cysts greater than 3 cm without these features.  Endoscopic US confirmation of a mural nodule, any features of main duct involvement (intraductal mucin or thickened main duct wall), or suspicious or positive cytology for malignancy is an indication for surgical resection.  Cysts with high-risk stigmata should be resected in patients medically fit for surgery, although EUS is optional.  Endoscopic US can be considered in smaller cysts without worrisome features but is not required.  Endoscopic US analysis should include at least cytology, amylase level, and CEA.  The guidelines stated that elevated CEA is a marker that distinguishes mucinous from non-mucinous cysts, but not benign from malignant cysts.\n",
      "Khalid et al (2004) noted that brush cytology of biliary strictures to diagnose pancreaticobiliary malignancy suffers from poor sensitivity.  These researchers attempted to improve the diagnostic yield of pancreaticobiliary brush cytology through analysis of tumor suppressor gene linked microsatellite marker loss of heterozygosity (LOH) and k-ras codon 12 mutation detection.  A total of 26 patients with biliary strictures underwent endoscopic retrograde cholangiography with brush cytology.  A panel of 12 polymorphic microsatellite markers linked to 6 tumor suppressor genes was developed.  Genomic DNA from cell clusters acquired from brush cytology specimens and micro-dissected surgical malignant and normal tissue underwent polymerase chain amplification reaction (PCR); PCR products were compared for LOH and k-ras codon 12 mutations.  A total of 17 patients were confirmed to have pancreaticobiliary adenocarcinoma; 9 patients had benign strictures (8 proven surgically, 1 by follow-up).  Cytomorphological interpretation was positive for malignancy (n = 8), indeterminate (n = 10), and negative for malignancy (n = 8).  Selected malignant appearing cytological cell clusters and micro-dissected histological samples from cancer showed abundant LOH characteristic of malignancy while brushings from 9 cases without cancer carried no LOH (p < 0.001); LOH and k-ras mutations profile of the cytological specimens was almost always concordant with the tissue samples.  Presence of k-ras mutation predicted malignancy of pancreatic origin (p < 0.001).  The authors concluded that LOH and k-ras codon 12 mutation analysis of PCR amplified DNA from biliary brush cytology discriminated reactive from malignant cells, with 100 % sensitivity, specificity, and accuracy.  Minor variations in LOH in brushings and in different sites within the same tumor likely reflect intra-tumoral mutational heterogeneity during clonal expansion of pre- and neoplastic lineages.\n",
      "Nodit et al (2006) noted that the clinical course of pancreatic endocrine tumor (PET) varies depending on tumor aggressiveness, disease extent, and possibly accumulated molecular alterations.  Endoscopic ultrasound-guided fine-needle aspiration (EUS-FNA) results, although accurate in diagnosing PET, correlated poorly with PET outcome.  The role of detecting key molecular abnormalities in predicting PET behavior and clinical outcome from EUS-FNA material remains unknown.  In this study, patients with confirmed PET who underwent EUS-FNA during a 32-month period were included.  Patient demographics and clinical data were recorded and follow-up information was obtained by contacting their physician to evaluate disease progression.  Representative tumor cells were micro-dissected from the FNA material. DNA was harvested and amplified, targeting a panel of 17 polymorphic microsatellite markers on chromosomes 1p, 3p, 5q, 9p, 10q, 11q, 17p, 17q, 21q, and 22q.  The polymerase chain reaction (PCR) products were subjected to fluorescent capillary gel electrophoresis to detect microsatellite loss.  The fractional allelic loss (FAL) was calculated.  A total of 25 patients were studied; 13 were classified histologically as benign PET limited to the pancreas and 12 as malignant PET (invasive or metastatic).  The mean FAL in the benign and malignant PET was 0.03 and 0.37 (p < 0.0001), respectively.  In addition, the mean FAL was significantly greater in those with disease progression as compared with patients with stable disease (0.45 versus 0.09, respectively, p < 0.0001).   The authors concluded that micro-satellite loss analysis of EUS-FNA material from PET can be performed reliably and an FAL value of more than 0.2 is associated with disease progression . These researchers stated that this technique may have value in the pre-operative assessment and risk stratification of patients with PET.\n",
      "The authors stated that the small sample size and limited follow-up period were drawbacks of this study, which needed replication in larger prospective studies with longer follow-up periods.  The impact of individual micro-satellite markers on the PET clinical course also required further study.  In this study certain microsatellites (3p26, 5q23, 17q23, and 21q23) were lost only in malignant PETs, but with the small number of specimens studied the significance of this was unclear.  Interestingly, both malignant PETs with a single allelic loss (5q23 and 17q23) each involved micro-satellites not lost in the benign PET.\n",
      "Finkelstein et al (2012) aimed to supplement microscopic examination of biliary cytobrush specimens to improve sensitivity by mutational profiling (MP) of:\n",
      "\n",
      "selected cells micro-dissected from cytology slides; and\n",
      "corresponding cell-free DNA (cfDNA) in residual supernatant fluid. \n",
      "\n",
      "From 43 patients with brushings of bile or pancreatic duct strictures, DNA was extracted from micro-dissected cells and 1 to 2 ml of cytocentrifugation supernatant fluid.  Mutational analysis targeted 17 genomic sites associated with pancreaticobiliary cancer, including sequencing for KRAS point mutation and LOH analysis of micro-satellites located at 1p, 3p, 5q, 9p, 10q, 17p, 17q, 21q, and 22q.  Mutations were found in 25/28 patients with malignancy, and no mutations were found in 5/5 patients with benign surgical results.  The cell-free supernatant fluid generally contained higher levels and quality of DNA, resulting in increased detection of mutations in most patients.  KRAS mutations only occurred in patients with pancreatic cancer; MP of supernatant fluid specimens resulted in high sensitivity and specificity for malignancy, improving the detection of malignancy over cytology alone.  The authors concluded that In this study they had shown that neoplastic free DNA was present in the extracellular compartment even when a particular cytology sample lacked sufficient cellularity to afford a definitive diagnosis.  Most importantly, the cell-free supernatant, available as a residual specimen after cytocentrifugation, should be regarded as a potentially valuable source of information due to its content of adequate amounts of free DNA for robust mutational analysis with the capacity to address issues related to sampling variation and to detect neoplasia at an early stage of development.\n",
      "The authors stated that this study had several limitations.  First, the total number of test samples was relatively small and the results shown here require confirmation with additional specimens.  In particular, the addition of more confirmed negative specimens would strengthen the findings around sensitivity.  Second, this study was restricted to the use of Saccomannoâs fixation which, though commonly used, was not the only fixative used in cytology practice.  While each fixative merits individual testing with respect to its capacity to deliver adequate levels of representative, intact supernatant DNA for MP, it was reasonable to expect favorable results with other methods of sample preparation as most cytology fixatives were alcohol-based and not expected to induce significant DNA degradation.  Indeed, additional unpublished work in the authorsâ laboratory involving testing of other supernatant fluids indicated that most common cytology fixatives yielded amplifiable DNA (with the notable exception of CytoRich Red).  This was consistent with their prior experience in genotyping micro-dissected cytology slides, in which most slides yielded amplifiable DNA regardless of the cytology fixative used.  While none of the supernatant specimens evaluated in this study failed to provide adequate DNA for MP, it was expected that a small proportion of markedly hypocellular specimens, likely from non-neoplastic states, would fail to meet the lower limit of DNA quantity for analysis.  It should be noted that in this study, when micro-dissection alone was used, 2/18 cases proved to be false negative for mutation detection (1 cholangiocarcinoma and 1 pancreatic adenocarcinoma).  While no false negative malignant stricture cases were seen in cohort 2A where both micro-dissection and supernatant fluid analysis were utilized, in 2 patients, the supernatant fluid manifested a lesser extent of mutational change than that present in the corresponding micro-dissected stained cytology cells.  These findings emphasized that the sampling variation and other limitations may nevertheless be present in individual cases limiting or preventing the detection of cancer.  It remained essential to integrate all of the information including clinical and imaging findings to optimize individual patient diagnosis.\n",
      "Finkelstein et al (2014) noted that diagnosis of fine-needle aspirations of pancreatic solid masses is complicated by many factors that keep its false-negative rate high.  These researchersâ novel approach analyzes cell-free cytocentrifugation supernatant, currently a discarded portion of the specimen.  Supernatant and cytology slides were collected from 25 patients: 11 cases with confirmed outcome [5 positive (adenocarcinoma) and 6 negative (inflammatory states)], plus 14 without confirmed outcomes.  Slides were micro-dissected, DNA was extracted from micro-dissections and corresponding supernatants, and all were analyzed for KRAS point mutation and loss of heterozygosity.  Notably, higher levels of free DNA were found in supernatants than in corresponding micro-dissected cells.  Supernatants contained sufficient DNA for mutational profiling even when samples contained few to no cells.  Mutations were present in 5/5 malignancies and no mutations were present in inflammatory states.  The authors concluded that these findings supported using supernatant for mutational genotyping when diagnostic confirmation is needed for pancreatic solid masses.  These researchers stated that the data presented suggested that supernatant fluid should be regarded as a valuable source of information that may address many diagnostic issues and may serve as a useful, complimentary tool for pathologists when microscopic examination is suboptimal.\n",
      "The authors stated that several limitations of this molecular analysis of cytocentrifugation supernatant were recognized.  The total number of test samples was not large, and the promising results shown here need to be evaluated with a greater number of specimens.  In addition, this study was restricted to the use of Saccomannoâs fixation.  Ideally, each commonly used fixative should be individually tested for its capacity to deliver adequate levels of representative supernatant DNA for mutational profiling.  It is reasonable, however, to expect favorable results with other methods of sample preparation since most cytology fixatives are alcohol based and are not expected to induce significant DNA degradation.  Consistently, prior work has shown that cytology specimens based on micro-dissected stained cytology cells, are especially suitable for mutational analysis.\n",
      "Deftereos et al (2014) stated that FNA of pancreatic solid masses can be significantly impacted by sampling variation.  Molecular analysis of tumor DNA can be an aid for more definitive diagnosis.  These investigators evaluated how molecular analysis of the cell-free cytocentrifugation supernatant DNA can help reduce sampling variability and increase diagnostic yield.   Atotal of 23-FNA smears from pancreatic solid masses were performed.  Remaining aspirates were rinsed for preparation of cytocentrifuged slides or cell blocks.  DNA was extracted from supernatant fluid and assessed for DNA quantity spectrophotometrically and for amplifiability by quantitative PCR (qPCR).  Supernatants with adequate DNA were analyzed for mutations using PCR/capillary electrophoresis for a broad panel of markers (KRAS point mutation by sequencing, micro-satellite fragment analysis for loss of heterozygosity (LOH) of 16 markers at 1p, 3p, 5q, 9p, 10q, 17p, 17q, 21q, and 22q).  In selected cases, micro-dissection of stained cytology smears and/or cytocentrifugation cellular slides were analyzed and compared.  In all, 5/23 samples cytologically confirmed as adenocarcinoma showed detectable mutations both in the micro-dissected slide-based cytology cells and in the cytocentrifugation supernatant.  While most mutations detected were present in both micro-dissected slides and supernatant fluid specimens, the latter showed additional mutations supporting greater sensitivity for detecting relevant DNA damage.  Clonality for individual marker mutations was higher in the supernatant fluid than in micro-dissected cells.  Cytocentrifugation supernatant fluid contains levels of amplifiable DNA suitable for mutation detection and characterization.  The finding of additional detectable mutations at higher clonality indicated that supernatant fluid may be enriched with tumor DNA.  The authors concluded that the findings of this study suggested how the supernatant fluid can be utilized as a source of molecular information and could become a powerful addition to standard cytology evaluation.  Mutational profiling of DNA in normally discarded supernatant fluid may help resolve occasional diagnostic challenges and may serve as a useful, complementary tool for cytopathologists when microscopic examination yielded no conclusive diagnosis or when a specimen is suboptimal.\n",
      "Malhotra et al (2014) aimed to better understand the supporting role that MP of DNA from micro-dissected cytology slides and supernatant specimens may play in the diagnosis of malignancy in FNA and biliary brushing specimens from patients with pancreaticobiliary masses.  Cytology results were examined in a total of 30 patients with associated surgical (n = 10) or clinical (n = 20) outcomes; MP of DNA from micro-dissected cytology slides and from discarded supernatant fluid was analyzed in 26 patients with atypical, negative or indeterminate cytology.  Cytology correctly diagnosed aggressive disease in 4 patients.  Cytological diagnoses for the remaining 26 were as follows: 16 negative (9 false negative), 9 atypical, 1 indeterminate.  MP correctly determined aggressive disease in 1 false negative cytology case and confirmed a negative cytology diagnosis in 7 of 7 cases of non-aggressive disease.  Of the 9 atypical cytology cases, MP correctly diagnosed 7 as positive and 1 as negative for aggressive disease.  One specimen that was indeterminate by cytology was correctly diagnosed as non-aggressive by MP.  When first-line malignant (positive) cytology results were combined with positive 2nd-line MP results, 12/21 cases of aggressive disease were identified, compared to 4/21 cases identified by positive cytology alone.  The authors concluded that when 1st-line cytology results were uncertain (atypical), questionable (negative), or not possible (non-diagnostic/indeterminate), MP provided additional information regarding the presence of aggressive disease.  When used in conjunction with 1st-line cytology, MP increased detection of aggressive disease without compromising specificity in patients that were difficult to diagnose by cytology alone.\n",
      "The authors stated that this study had several drawbacks including a small sample size that limited their ability to calculate the diagnostic performance of MP in pancreatic masses and associated biliary strictures.  Although MP allowed these researchers to detect additional cases of aggressive disease, even when cytology and MP results were combined into one overall diagnosis, 9 cases of malignancy were missed.  These falsely negative results were likely due to a combination of the less than perfect sensitivity of both tests as well as to sampling limitations related to FNA and brushing techniques.  These investigators noted that despite such limitations, these promising findings do provide support for future larger scale studies, with the addition of supernatant analysis providing an opportunity to overcome some of these limitations.\n",
      "Gonda et al (2017) stated that it is a challenge to detect malignancies in biliary strictures.  Various sampling methods are available to increase diagnostic yield, but these require additional procedure time and expertise.  These investigators evaluated the combined accuracy of fluorescence in situ hybridization (FISH) and PCR-based DNA MP of specimens collected using standard brush techniques.  These researchers performed a prospective study of 107 consecutive patients treated for biliary strictures by endoscopic retrograde cholangiopancreatography from June 2012 through June 2014.  They carried out routine cytology and FISH analyses on cells collected by standard brush techniques, and analyzed supernatants for point mutations in KRAS and LOH mutations in tumor-suppressor genes at 10 loci (MP analysis was performed at Interpace Diagnostics).  Strictures were determined to be non-malignant based on repeat image analysis or laboratory test results 12 months after the procedure.  Malignant strictures were identified based on subsequent biopsy or cytology analyses, pathology analyses of samples collected during surgery, or death from biliary malignancy.  These researchers determined the sensitivity and specificity with which FISH and MP analyses detected malignancies using the exact binomial test.  The final analysis included 100 patients; 41 % had biliary malignancies.  Cytology analysis identified patients with malignancies with 32 % sensitivity and 100 % specificity.  Addition of FISH or MP results to cytology results increased the sensitivity of detection to 51 % (p < 0.01) without reducing specificity.  The combination of cytology, MP, and FISH analyses detected malignancies with 73 % sensitivity (p < 0.001); FISH identified an additional 9 of the 28 malignancies not detected by cytology analysis, and MP identified an additional 8 malignancies; FISH and MP together identified 17 of the 28 malignancies not detected by cytology analysis.  The authors concluded that these findings supported the use of both FISH testing and PCR-based MP of tumor-suppressor gene LOH and KRAS in evaluation of cytology-negative or indeterminate biliary strictures; MP allowed for increased diagnostic yield from each individual brush, given that normally discarded, cell-free supernatant material that contains DNA can be analyzed.  Based on these findings, these researchers suggested using either FISH or MP as a 2nd-line diagnostic modality to 1st-line cytology.  They stated that MP may be best prioritized to scenarios of low cellularity.  Any case that is negative or indeterminate by 2 testing modalities should undergo a 3rd to increase the probability of detecting possible malignancy.  To do so, normally discarded supernatant fluid should be retained for MP testing during the standard cytology cytocentrifugation preparation of cells for cytology.  These researchers stated that additional studies may help to better understand the reflex order of sequential testing and the impact of this reflex on health economics.\n",
      "The authors stated that this study had several drawbacks that may have impacted generalized conclusions.  A somewhat higher benign stricture rate was noted in their cases than in other prior series.  There also were relatively few primary sclerosing cholangitis (PSC) patients included in this study.  Prior studies have shown that there is a significant aneuploidy rate associated with pre-malignant lesions seen in PSC.  Because of this, specificity of FISH for malignancy was expected to be lower in a cohort of PSC patients than the authors reported in their cohort.  Less was known about detection of KRAS mutations in the progression of PSC to cholangiocarcinoma.  However, based on this study cohort and prior studies, these findings likely were not generalizable to the PSC population.\n",
      "Khosravi et al (2018) stated that indeterminate cytology occurs in a significant number of patients with solid pancreaticobiliary lesion that undergo EUS-FNA or endoscopic retrograde cholangiopancreatography (ERCP) and can incur further expensive testing and inappropriate surgical intervention.  Mutation profiling improves diagnostic accuracy and yield but the impact on clinical management is uncertain.  These researchers determined the performance of MP in clinical practice and its impact on management in solid pancreaticobiliary patients with indeterminate cytology.  Solid pancreaticobiliary patients with non-diagnostic, benign, atypical or suspicious cytology who had past MP testing were included.  Mutation profiling examined KRAS mutation and a tumor suppressor gene associated loss of heterozygosity mutation panel covering 10 genomic loci.  Two endo-sonographers made management recommendations without and then with MP results, indicating their level of confidence.  Mutation profiling improved diagnostic accuracy in 232 patients with indeterminate cytology.  Among patients with non-diagnostic cytology, low-risk MP provided high specificity and negative predictive value (NPV) for the absence of malignancy while high-risk MP identified malignancies otherwise undetected.  Mutation profiling increased clinician confidence in management recommendations and resulted in more conservative management in 10 % of patients.  Mutation profiling increased the rate of benign disease in patients recommended for conservative management (84 % to 92 %, p < 0.05) and the rate of malignant disease in patients recommended for aggressive treatment (53 % to 71 %, p < 0.05).  The authors concluded that MP improved diagnostic accuracy and significantly impacted management decisions.  Low-risk MP results increased recommendations for conservative management and increased the rate of benign outcomes those patients, helping to avoid unnecessary aggressive interventions and improve patient outcomes.  These researchers stated that their study was limited by its retrospective nature.  Moreover, they noted that although high-risk MP results were able to help confirm the presence of malignancy in cases in which cytology indicated a high suspicion of malignancy, low-risk results could not effectively exclude the possibility of malignancy in such cases.\n",
      "Kushnir et al (2019) noted that routine cytology of biliary stricture brushings obtained during ERCP has suboptimal sensitivity for malignancy.  These investigators compared the individual and combined ability of cytology, FISH analysis and PCR-based MP to detect malignancy in standard biliary brushings.  They performed a prospective study of patients undergoing ERCP using histology or 1 year follow-up to determine patient outcomes; MP was performed on free-DNA from biliary brushing specimens using normally discarded supernatant fluid.  MP examined KRAS point mutations and tumor suppressor gene associated LOH mutations at 10 genomic loci; FISH examined chromosome specific gains or losses.  A total of 101 patients were included in final analysis and 69 % had malignancy.  Cytology had 26 % sensitivity and 100 % specificity for malignancy.  Using either FISH or MP in combination with cytology increased sensitivity to 44 % and 56 %, respectively.  The combination of all 3 tests (cytology, FISH, and MP) had the highest sensitivity for malignancy (66 %).  There was no difference in the specificity of cytology, FISH or MP testing when examined alone or in combination; MP improved diagnostic yield of each procedure from 22 % to 100 %; FISH improved yield to 90 %; MP detected 21 malignancies beyond that identified by cytology; FISH detected an additional 13.  The combination of FISH and MP testing detected an additional 28 malignancies.  The authors concluded that both MP and FISH are complimentary molecular tests that could significantly increase detection of biliary malignancies when used in combination with routine cytology of standard biliary brush specimens.\n",
      "Gene Expression Profiling for Cancer of Unknown Primary\n",
      "Carcinoma of unknown primary (CUP) is a biopsy-proven metastatic solid tumor with no primary tumor identified and represents approximately 2% to 4% of all cancer cases.  The diagnosis of CUP is made following inconclusive results from standard tests (e.g., biopsy, immunochemistry and other blood work, chest x-rays, and occult blood stool test).  The absence of a known primary tumor presents challenges to the selection of appropriate treatment strategies.  As a result, patients have a poor prognosis, and fewer than 25% survive 1 year from the time of diagnosis.  A variety of tissue-biopsy testing techniques currently are used to determine the origin of the CUP, including immunochemistry; histological examination of specimens stained with eosin and hematoxylin, and electron microscopy.  These techniques definitively identify the type of carcinoma in less than 20% to 30% of CUP.\n",
      "Gene expression profiling is a technique used to identify the genetic makeup of a tissue sample by characterizing the patterns of mRNA transcribed, or \"expressed\", by its DNA.  Specific patterns of gene expression, reflected in unique configurations of mRNA, are associated with different tumor types.  By comparing the gene expression profile (GEP) of an unknown tumor to the profiles of known primary cancers (\"referent\" profiles), it may be possible to determine the type of tumor from which the CUP originated.\n",
      "In July 2008, the FDA cleared for marketing the Pathwork Tissue of Origin test (Pathwork Diagnostics, Sunnyvale, CA), a gene expression profiling test that uses microarray processing to determine the type of cancer cells present in a tumor of unknown origin.  The test uses the PathChip (Affymetrix Inc., Santa Clara, CA), a custom-designed gene expression array, to measure the expression from 1,668 probe sets to quantify the similarity of tumor specimens to 15 common malignant tumor types, including: bladder, breast, colorectal, gastric, germ cell, hepatocellular, kidney, non-small cell lung, non-Hodgkin's lymphoma, melanoma, ovarian, pancreatic, prostate, soft tissue sarcoma, and thyroid.  The degree of correspondence between the tissue sample's GEP and a referent profile is quantified and expressed as a probability-based score.\n",
      "A multi-center, clinical validation study reported on comparisons of diagnoses based on GEP from 477 banked tissue samples of undifferentiated and poorly differentiated metastases versus standard of care pathology based diagnoses.  Comparison of the GEP based diagnoses versus pathology based diagnoses yielded an 89 % agreement and the concurrence was greater than 92 % for 8 out of 15 types of primary tumors.  The overall accuracy of the test was approximately 95 % and 98 % for positive and negative determinations, respectively (Monzon et al, 2007).\n",
      "Gene expression profiling is a promising technology in the management of CUP; however, there is insufficient evidence of its clinical utility compared to that achieved by expert pathologists using current standards of practice. A draft clinical guideline on metastatic malignant disease of unknown origin by the National Institute for Clinical Excellence (NICE, 2010) recommends against using gene expression profiling (e.g., Pathwork TOT, CupPrint, Theros CancerTypeID, miRview Mets) to identify primary tumors in patients with CUP. The guideline explained that currently there is no evidence that gene-expression based profiling improves the management or changes the outcomes for patients with CUP. Guidelines on occult primary from the National Comprehensive Cancer Network (NCCN, 2010) state that, while gene expression profiling looks promising, \"prospective clinical trials are necessary to confirm whether this approach can be used in choosing treatment options which would improve the prognosis of patients with occult primary cancers.\" \n",
      "An assessment by the Andalusian Agency for Health Technology Assessment (AETSA, 2012) of microRNAs as a diagnostic tool for lung cancer found only two studies assessing the analytical validity of miRview in patients with non-small cell lung cancer. The sensitivity of the miRNA for the detection of carcinoma was between 96% and 100% and the specificity was between 90% and 100%. The area under the ROC curve was close to unity and the positive and negative probability ratios showed a high diagnostic accuracy (9.6 and 0.04, respectively). The assessment stated that, although the quality of the studies was moderate to high, the sensitivity of the diagnostic test may be overestimated as it is a case-control design.\n",
      "A technology assessment prepared for the Agency for Healthcare Research and Quality (Meleth et al, 2013) found that the clinical accuracy of the PathworkDx, miRview, and CancerTypeID are similar, ranging from 85 percent to 88 percent, and that the evidence that the tests contribute to identifying a tumor of unknown origin was moderate. The assessment concluded that we do not have sufficient evidence to assess the effect of the tests on treatment decisions and outcomes. The assessment noted that most studies of these tests were funded wholly or partially by the manufacturers of these tests, and that the most urgent need in the literature is to have the clinical utility of the tests evaluated by research groups that have no evidence conflict of interest.\n",
      "Monzon et al (2009) stated that malignancies found in unexpected locations or with poorly differentiated morphologies can pose a significant challenge for tissue of origin determination.  Current histologic and imaging techniques fail to yield definitive identification of the tissue of origin in a significant number of cases.  The aim of this study was to validate a predefined 1,550-gene expression profile for this purpose.  Four institutions processed 547 frozen specimens representing 15 tissues of origin using oligonucleotide microarrays were used in this study.  Half of the specimens were metastatic tumors, with the remainder being poorly differentiated and undifferentiated primary cancers chosen to resemble those that present as a clinical challenge.  In this blinded multi-center validation study the 1,550-gene expression profile was highly informative in tissue determination.  The study found overall sensitivity (positive percent agreement with reference diagnosis) of 87.8 % (95 % CI: 84.7 % to 90.4 %) and overall specificity (negative percent agreement with reference diagnosis) of 99.4 % (95 % CI: 98.3 % to 99.9 %).  Performance within the subgroup of metastatic tumors (n = 258) was found to be slightly lower than that of the poorly differentiated and undifferentiated primary tumor subgroup, 84.5 % and 90.7 %, respectively (p = 0.04).  Differences between individual laboratories were not statistically significant.  The authors concluded that this study represents the first adequately sized, multi-center validation of a gene-expression profile for tissue of origin determination restricted to poorly differentiated and undifferentiated primary cancers and metastatic tumors.  These results indicate that this profile should be a valuable addition or alternative to currently available diagnostic methods for the evaluation of uncertain primary cancers.\n",
      "Monzon and Koen (2010) stated that tumors of uncertain or unknown origin are estimated to constitute 3 % to 5 % of all metastatic cancer cases.  Patients with these types of tumors show worse outcomes when compared to patients in which a primary tumor is identified.  New molecular tests that identify molecular signatures of a tissue of origin have become available.  The authors reviewed the literature on existing molecular approaches to the diagnosis of metastatic tumors of uncertain origin and discuss the current status and future developments in this area.  Published peer-reviewed literature, available information from medical organizations (NCCN), and other publicly available information from tissue-of-origin test providers and/or manufacturers were used in this review.  The authors concluded that molecular tests for tissue-of-origin determination in metastatic tumors are available and have the potential to significantly impact patient management.  However, available validation data indicate that not all tests have shown adequate performance characteristics for clinical use.  Pathologists and oncologists should carefully evaluate claims for accuracy and clinical utility for tissue-of-origin tests before using test results in patient management.  The personalized medicine revolution includes the use of molecular tools for identification/confirmation of the site of origin for metastatic tumors, and in the future, this strategy might also be used to determine specific therapeutic approaches.\n",
      "Anderson and Weiss (2010) noted that pathologists use various panels of IHC stains to identify the site of tissue of origin for metastatic tumors, particularly poorly or undifferentiated cancers of unknown or uncertain origin.  Although clinicians believe that immunostains contribute greatly to determining the probable primary site among 3 or more possibilities, objective evidence has not been convincingly presented.  This meta-analysis reviews the objective evidence supporting this practice and summarizes the performance reported in 5 studies published between 1993 and 2007.  A literature search was conducted to identify IHC performance studies published since 1990 that were masked, included more than 3 tissues types, and used more than 50 specimens.  The 5 studies found in this search were separated into 2 subgroups for analysis: those, which included only metastatic tumors (n = 368 specimens) and the blended studies, which combined primary tumors and metastases (n = 289 specimens).  The meta-analysis found that IHCs provided the correct tissue identification for 82.3 % (95 % CI: 77.4 % to 86.3 %) of the blended primary and metastatic samples and 65.6 % (95 % CI: 60.1 % to 70.7 %) of metastatic cancers.  This difference is both clinically and statistically significant.  The authors concluded that this literature review confirms that there is still an unmet medical need in identification of the primary site of metastatic tumors.  It establishes minimum performance requirements for any new diagnostic test intended to aid the pathologist and oncologist in tissue of origin determination.\n",
      "GeneSearch BLN\n",
      "The presence of breast tumor cells in axillary lymph nodes is a key prognostic indicator in breast cancer.  During surgery to remove breast tumors, patients often undergo biopsy of the sentinel (i.e., first) node(s) that receive lymphatic fluid from the breast.  Excised sentinel lymph nodes are currently evaluated post-operatively by formalin-fixed paraffin-embedded Hematoxylin and Eosin (H&E) histology and IHC.  GeneSearchâ¢ Breast Lymph Node (BLN) assay (Veridex, LLC, Warren, NJ) is a novel method to examine the extracted sentinel lymph nodes for metastases and can provide information during surgery within 30 to 40 minutes from the time the sentinel node is removed, potentially avoiding a second operation for some patients.  The GeneSearch BLN assay received FDA pre-market approval on July 16, 2007 as a qualitative in vitro diagnostic test for the rapid detection of metastases larger than 0.2 mm in nodal tissue removed from sentinel lymph node biopsies of breast cancer patients.  GeneSearch BLN assay uses real time reverse transcriptase polymerase chain reaction (RT-PCR) to detect the gene expression markers, mammaglobin (MG) and cytokeratin 10 (CI19), which are abundant in breast tissue but scarce in lymph node cells.  In the clinical trial conducted by Veridex, which was submitted to the FDA, the sensitivity of the GeneSearch BLN Assay was reported to be 87.6 % and the specificity was 94.2 % (Julian et al, 2008).  According to the product labeling, \"The GeneSearchâ¢ Breast Lymph Node (BLN) assay may be used in conjunction with sentinel lymph node biopsy for a patient who has been counseled on use of this test and has been informed of its performance.  False positive results may be associated with increased morbidity.  False negative and inconclusive test results may be associated with delayed axillary node dissection.  Clinical studies so far are inconclusive about a benefit from treatment based on findings of breast cancer micro-metastases in sentinel lymph nodes.\"\n",
      "Blumencranz et al (2007) compared the GeneSearch BLN assay with results from conventional histologic evaluation from 416 patients at 11 clinical sites and reported that the GeneSearch BLN assay detected 98 % of metastases greater than 2 mm in size and 57 % of metastasis less than 0.2 mm.  False positives were reported in 4 % of the cases.  However, there were several limitations of this study, including the lack of a description of patient recruitment, inadequate descriptions of several analyses performed, substantial variations in test performance across sites, and ad hoc comparison of the assay to other intra-operative techniques.\n",
      "Viale et al (2008) analyzed 293 lymph nodes from 293 patients utilizing the GeneSearch BLN assay.  Using histopathology as the reference standard, the authors reported that the BLN assay correctly identified 51 of 52 macro-metastatic and 5 of 20 micro-metastatic sentinel lymph nodes (SLNs), with a sensitivity of 98.1 % to detect metastases larger than 2 mm, 94.7 % for metastases larger than 1 mm, and 77.8 % for metastases larger than 0.2 mm.  The overall concordance with histopathology was 90.8 %, with a specificity of 95.0 %, a positive predictive value of 83.6 %, and a negative predictive value of 92.9 %.  When the results were evaluated according to the occurrence of additional metastases to non-SLN in patients with histologically positive SLNs, the assay was positive in 33 (91.7 %) of the 36 patients with additional metastases and in 22 (66.6 %) of the 33 patients without further echelon involvement.  The authors concluded that the sensitivity of the GeneSearch BLN assay is comparable to that of the histopathologic examination of the entire SLN by serial sectioning at 1.5 to 2 mm.\n",
      "Although treatment for metastases larger than 2.0 mm is widely accepted as beneficial, clinical studies have not yet provided data for a consensus on benefit from treatment based on very small breast cancer metastases (between 0.2 mm and 2.0 mm) in SLNs.  False positive results may be associated with increased morbidity, usually due to effects of axillary node dissection surgery.  Patients who undergo axillary lymph node dissection (ALND) have significantly higher rates of increased swelling in the upper arm and forearm (lymphedema), pain, numbness, and motion restriction about the shoulder when compared with patients who undergo only sentinel lymph node dissection (SLND).  False negative and inconclusive test results may be associated with delayed axillary node dissection.  Clinical studies so far are inconclusive regarding a benefit from treatment based on findings of breast cancer micro-metastases in SLNs.  Preliminary data suggest that the GeneSearch BLN assay has high specificity and moderate sensitivity when only macro-metastases are included in the analysis.  The clinical significance of micro-metastases is still being debated in the literature, thus, the failure of the GeneSearch BLN assay to perform adequately in the detection of micro-metastases is of unknown significance.\n",
      "A systematic evidence review by the BlueCross BlueShield Association Technology Evaluation Center (BCBSA, 2007) determined that the use of the GeneSearch BLN assay to detect sentinel node metastases in early stage breast cancer does not meet the TEC criteria.  The assessment stated, \"There are several operational issues that add difficulty to the use of the GeneSearch assay, including the need for fresh specimens (rather than putting them in formalin for permanent fixation), the learning curve involved in reducing both the percentage of invalid results (from about 15% initially to 4 - 8% for more experienced technicians) and the time to perform the test compared to alternative intra-operative techniques (which take less than 15 minutes).\"  Furthermore, the assessment stated \"The GeneSearch assay also provides less information for staging than other intra-operative procedures, since it cannot distinguish between micro- and macro-metases.  Nor can it indicate the location of the metastasis (inside or outside the node).  Post-operative histology is therefore required in all cases.  It is less crucial when frozen section histology is performed, since pathologists can judge the size of the metastasis and its location from this test, although distortion is possible.  To summarize, the data available is inadequate to assess the clinical utility of the GeneSearch assay compared to either post-operative histology alone or to the alternative intra-operative tests such as imprint cytology and frozen section histology.  In addition, the balance of benefits versus harms may require higher specificity to avoid unnecessary ALNDs and their sequelae, whereas the GeneSearch design emphasizes sensitivity.\"\n",
      "A report by Adelaide Health Technology Assessment stated that, if the GeneSearch BLN Assay is to play a role in reducing the mortality of breast cancer patients, it will be through more accurate diagnosis of breast cancer metastasis during SNB (Ellery, et al., 2010). The report noted, however, that, as yet there are no data to indicate whether SNB itself lowers the mortality rate among breast cancer patients. Hence, it is unclear whether the GeneSearch BLN Assay would have any indirect effect on breast cancer mortality until further investigation into SNB concludes.\n",
      "Thus, there is insufficient evidence to make a conclusion about the effectiveness of the GeneSearch BLN assay.  The FDA is requiring the manufacturer to conduct two post-approval studies.  The primary objective of the first study is to estimate the positive predictive concordance between the GeneSearch BLN assay and histology as routinely practiced and the objectives of the second clinical study are\n",
      "\n",
      "determine the assay turn-around-time from the time of node removal to the report of the assay result to the surgeon and\n",
      "determine whether the assay result was or was not received in time to make an intra-operative decision and\n",
      "collect data in relation to other surgical procedures during the sentinel lymph node dissection/breast surgery to determine if the assay turn-around-time resulted in longer surgery time.\n",
      "\n",
      "BT Test\n",
      "Provista Life Sciences (Phoenix, AZ) has developed a laboratory test called the Biomarker Translation Test, or the BT Test, which is a test score based on a multi-protein biomarker analysis (i.e., IL-2, -6,-8,-12, TNFa, EGF, FGF, HGF, VEGF) and medical profile of an individual's risk factors for breast cancer.  It is intended to be used as an adjunctive test along with other breast cancer detection modalities, however, there are no published studies of the effectiveness of this test.\n",
      "Bcl-2\n",
      "Bcl-2 (B-cell CLL/lymphoma 2; BCL2) is a proto-oncogene whose protein product, bcl-2, suppresses programmed cell death (apoptosis), resulting in prolonged cellular survival without increasing cellular proliferation.  Dysregulation of programmed cell death mechanisms plays a role in the pathogenesis and progression of cancer as well as in the responses of tumors to therapeutic interventions.  Many members of the Bcl-2 family of apoptosis-related genes have been found to be differentially expressed in various malignancies (Reed, 1997).\n",
      "Salgia (2008) reviewed the evidence for detection of Bcl-2 in lung cancer.  The author observed that Bcl-2 over-expression has been reported in 22 to 56% of lung cancers with a higher expression in squamous cell carcinoma as compared to adenocarcinoma histology.  The author concluded, however, that the association of Bcl-2 expression and prognosis in non-small cell lung cancer is unclear.  Multiple reports have demonstrated that Bcl-2-positive lung cancers are associated with a superior prognosis compared to those that are Bcl-2 negative.  However, other studies have failed to demonstrate any survival impact with bcl-2 positivity, while over-expression has also been associated with a poorer outcome.  A meta-analysis that included 28 studies examining the prognostic influence of Bcl-2 in non-small cell lung cancer concluded that over-expression of Bcl-2 was associated with a significantly better prognosis in surgically resected (hazard ratio 0.5, 95% CI 0.39-0.65).\n",
      "Compton (2008) recently reviewed the evidence on the Bcl-2 oncogene and other tumor markers in colon cancer.  Compton explained that Bcl-2 is a gene related to apoptosis/cell suicide.  Bcl-2 over-expression leading to inhibition of cell death signaling has been observed as a relatively early event in colorectal cancer development.  The author concluded that the independent influence of the Bcl-2 oncogene on prognosis remains unproven, and explained that the variability in assay methodology, conflicting results from various studies examining the same factor, and the prevalence of multiple small studies that lack statistically robust, multivariate analyses all contribute to the lack of conclusive data.  Compton concluded that before the Bcl-2 oncogene and certain other tumor markers can be incorporated into clinically meaningful prognostic stratification systems, \"more studies are required using multivariate analysis, well-characterized patient populations, reproducible and current methodology, and standardized reagents.\"\n",
      "CD31\n",
      "Compton (2008) reviewed the evidence for intratumor microvessel density (MVD) and antibodies against CD31 in colorectal cancer.  The author explained that intratumoral MVD is a reflection of tumor-induced angiogenesis.  Microvessel density has been independently associated with shorter survival in some, but not all studies.  A meta-analysis of all studies relating MVD expression to prognosis concluded that at least some of the variability could be explained by the different methods of MVD assessment.  The author noted that there was a significant inverse correlation between immunohistochemical expression and survival when MVD was assessed using antibodies against CD31 or CD34, but not factor VIII.  The author concluded, however, that there is a need for evaluation of MVD in large studies of prognostic factors using multivariate analysis; however, standard guidelines for staining, evaluation, and interpretation of MVD are lacking.\n",
      "In a review, Hayes (2008) reviewed the evidence for assessing angiogenesis factors in breast cancer.  The author noted that, in an early report, MVD count (as indicated by IHC staining for endothelial cells, such as factor VIII-related antigen or CD31) was a statistically significant independent predictor of both disease-free and overall survival in women with both node-negative and node-positive breast cancer.  The author noted, however, that subsequent data are conflicting, with some studies confirming and others refuting the initial findings.  The author stated that, \"As with many of the other tumor marker studies, evaluation of angiogenesis is complicated by technical variation, reader inconsistency, and potential interaction with therapy.\"\n",
      "Burgdorf (2006) reviewed the use of CD31 in acquired progressive lymphangioma.  The author stated that special staining techniques reveal that the cells are variably positive for CD31, but that the staining patterns are too variable to be of diagnostic importance.\n",
      "Some authorities have stated that CD31 staining may be useful for diagnosing angiosarcomas (Schwartz, 2008; Carsi and Sim, 2008; Fernandez and Schwartz, 2007; McMains and Gourin, 2007).  CD31 immunostaining can help confirm that the tumor originates from blood vessels.\n",
      "TOP2A\n",
      "Topoisonmerase II alpha is a protein encoded by the TOP2A gene and is proposed as a predictive and prognostic marker for breast cancer. It is also proposed as an aid in predicting response to anthracycline therapy in breast cancer. Two types of tests are available for topoisonmerase II alpha: topoisomerase II alpha protein expression testing by immunohistochemistry (IHC); and TOP2A gene amplification testing by FISH (eg, TOP2A FISH pharmDx Assay). \n",
      "The topoisomerase II alpha gene (TOP2A) is located adjacent to the HER-2 oncogene at the chromosome location 17q12-q21 and is either amplified or deleted (with equal frequency) in a great majority of HER-2 amplified primary breast tumors and also in tumors without HER-2 amplification.  Recent experimental as well as numerous, large, multi-center trials suggest that amplification (and/or deletion) of TOP2A may account for both sensitivity or resistance to commonly used cytotoxic drugs (e.g., anthracyclines) depending on the specific genetic defect at the TOP2A locus.  An analysis of TOP2A aberrations in the Danish Breast Cancer Cooperative Group trial 89D (Nielsen, et al., 2008) suggested a differential benefit of adjuvant chemotherapy in patients with primary breast cancer, favoring treatment with epirubicin in patients with TOP2A amplifications, and perhaps deletions; however, the authors concluded that, \"Additional studies are needed to clarify the exact importance of TOP2A deletions on outcome, but deletions have proven to be associated with a very poor prognosis.\"\n",
      "The National Comprehensive Cancer Network (NCCN, 2008) guideline on breast cancer does not address the use of TOP2A testing. Guidelines from the American Society for Clinical Oncology (2016) state: \"The clinician should not use TOP2A gene amplification or TOP2A protein expression by IHC to guide adjuvant chemotherapy selection.: This is a moderate-strength recommendation based upon high quality evidence. The guidelines also recommend against the use of TOP2A gene coamplification to guide adjuvant chemotherapy selection.  \n",
      "TSP-1\n",
      "Ghoneim et al (2008) explained that thrombospondin-1 (TSP-1) is a member of a family of five structurally related extracellular glycoproteins that plays a major role in cell-matrix and cell to cell interactions.  Due to its multifunctional nature and its ability to bind to a variety of cell surface receptors and matrix proteins, TSP-1 has been identified as a potential regulator of angiogenesis and tumor progression.  Data collected by Secord, et al. (2007) suggested that high THBS-1 levels may be an independent predictor of worse progression-free and overall survival in women with advanced-stage epithelial ovarian cancer.  However, a phase II clinical trail (Garcia, et al., 2008) of bevacizumab and low-dose metronomic oral cyclophosphamide in recurrent ovarian cancer reported that levels of TSP-1 were not associated with clinical outcome.\n",
      "mdr1\n",
      "In a review on multidrug resistance in acute leukemia, List and Spier (1992) explained that the mdr1 gene or its glycoprotein product, P-glycoprotein, is detected with high frequency in secondary acute myeloid leukemia (AML) and poor-risk subsets of acute lymphoblastic leukemia.  Investigations of mdr1 regulation in normal hematopoietic elements have shown a pattern that corresponds to its regulation in acute leukemia, explaining the linkage of mdr1 to specific cellular phenotypes.  Therapeutic trials are now in progress to test the ability of various MDR-reversal agents to restore chemotherapy sensitivity in high-risk acute leukemias.\n",
      "In a phase III multi-center randomized study to determine whether quinine would improve the survival of adult patients with de novo AML, Soary et al (2003) reported that neither mdr1 gene or P-glycoprotein expression influenced clinical outcome.\n",
      "A phase I/II study of the MDR modulator Valspodar (PSC 833, Novartis Pharma) combined with daunorubicin and cytarabine in patients with relapsed and primary refractory acute myeloid leukemia (Gruber et al, 2003) reported that P-glycoprotein did not give an obvious improvement to the treatment results.\n",
      "MRP-1\n",
      "Motility-related protein (MRP-1) is a glycoprotein with a sequence identical to that of CD9, a white blood cell differentiation antigen.  The level of MRP-1/CD9 expression has been found in investigational studies to inhibit cell motility and low MRP-1/CD9 expression may be associated with the metastatic potential of breast cancer (Miyake et al, 1995).  CD9 immuno-expression is also being investigated as a potential new predictor of tumor behavior in patients with squamous cell carcinoma of the head and neck (Mhawech et al, 2004) as well as other tumors (e.g., urothelial bladder carcinoma, colon cancer, lung cancer); however, prospective studies are needed to determine the clinical role of MRP-1/CD9 expression in tumors.\n",
      "PLAP\n",
      "The National Comprehensive Cancer Network's guideline on occult primary tumors includes placental alkaline phosphatase (PLAP) as a useful marker to assist in identifying germ cell seminoma and non-seminoma germ cell tumors in unknown primary cancer (NCCN, 2009).\n",
      "MPO\n",
      "Myeloperoxidase (MPO), a blood protein, is a major component of azurophilic granules of neutrophils.  Myeloperoxidase analysis has been used to distinguish between the immature cells in acute myeloblastic leukemia (cells stain positive) and those in acute lymphoblastic leukemia (cells stain negative).  The National Comprehensive Cancer Network guidelines on acute myeloid leukemia (AML) include MPO analysis in the classification of AML (NCCN, 2008).\n",
      "Matsuo et al (2003) examined the prognostic factor of the percentage of MPO-positive blast cells for AML.  Cytochemical analysis of MPO was performed in 491 patients who were registered to the Japan Adult Leukemia Study Group (AML92 study).  Patients were divided into two groups using the percentage of MPO-positive blast (high [ > or = 50%] and low [< 50%]).  Complete remission rates were 85.4% in the former and 64.1% in the latter (p = 0.001).  The OS and DFS were significantly better in the high MPO group (48.3 versus 18.7% for OS, and 36.3 versus 20.1% for DFS, p < 0.001, respectively).  Multi-variate analysis showed that both karyotype and the percentage of MPO-positive blast cells were equally important prognostic factors.  The high MPO group still showed a better survival even when restricted to the intermediate chromosomal risk group or the patients with normal karyotype (p < 0.001).  The OS of patients with normal karyotype in the high MPO group was almost equal to that of the favorable chromosomal risk group.  The authors concluded that the percentage of MPO-positive blast cells is a simple and highly significant prognostic factor for AML patients, and especially useful to stratify patients with normal karyotype. \n",
      "DCP\n",
      "The most commonly used marker for hepatocellular carcinoma (HCC) is the AFP level.  Des-gamma-carboxy prothrombin (DCP) (also known as \"prothrombin produced by vitamin K absence or antagonism II\" [PIVKA II]) has also shown promise in the diagnosis of HCC (Toyoda et al, 2006; Ikoma et al, 2002; Nomura et al, 1996; Liebman et al, 1984).  In one series of 76 patients with HCC, this marker was elevated in 69 patients with a mean serum concentration of 900 mcg/L.  Much lower mean values were seen in patients with chronic active hepatitis, metastatic disease to the liver, and normal subjects (10 and 42 mcg/L and undetectable, respectively) (Liebman et al, 1984).  Elevations in serum levels of DCP are less frequent in tumors less than 3 cm in size (Nakamura et al, 2006; Weitz and Liebman, 1993;).  Aoyagi et al (1996) as well as Weitz and Liebman (1993) reported that abnormal prothrombin levels do not correlate well with serum AFP.\n",
      "Toyoda at al (2006) measured AFP, lens culinaris agglutinin A-reactive fraction of AFP (AFP-L3), and DCP for the evaluation of tumor progression and prognosis of patients with HCC (n = 685) at the time of initial diagnosis.  Positivity for AFP > 20 ng/dL, AFP-L3 > 10% of total AFP, and/or DCP > 40 mAU/mL was determined.  In addition, tumor markers were measured after treatment of HCC.  Of the 685 patients, 337 (55.8%) were positive for AFP, 206 (34.1%) were positive for AFP-L3, and 371 (54.2%) were positive for DCP.  In a comparison of patients positive for only 1 tumor marker, patients positive for AFP-L3 alone had a greater number of tumors, whereas patients positive for DCP alone had larger tumors and a higher prevalence of portal vein invasion.  When patients were compared according to the number of tumor markers present, the number of markers present clearly reflected the extent of HCC and patient outcomes.  The number of markers present significantly decreased after treatment.  The authors concluded that tumor markers AFP-L3 and DCP appeared to represent different features of tumor progression in patients with HCC and that the number of tumor markers present could be useful for the evaluation of tumor progression, prediction of patient outcome, and treatment efficacy.\n",
      "The National Comprehensive Cancer Network's guideline on HCC (NCCN, 2008) does not include measurement of DCP among the surveillance test options for HCC.  According to NCCN guidelines, proposed surveillance for the early detection of HCC among high-risk populations (e.g., chronic hepatitis C virus-infected patients) includes liver ultrasonography every 3 to 6 months and evaluation of alkaline phosphatase, albumin, and AFP.  The guidelines stated, \"It is not yet clear if early detection of hepatocellular cancer with routine screening improves the percentage of patients detected with disease at a potentially curative stage, but high-risk chronic hepatitis C virus - infected patients should be considered for ongoing recurrent screening until these issues have been resolved.  The level of des-gamma-carboxy-prothrombin protein induced by vitamin K absence (PIVKA-II) is also increased in many patients with hepatocellular carcinoma.  However, as is true with AFP, PIVKA-II may be elevated in patients with chronic hepatitis.\"  Furthermore, according to Sherman (2008), DCP has not been adequately studied as a screening test for HCC and cannot be recommended at this time.\n",
      "NMP66\n",
      "Researchers at Matritech (Newton, MA) have detected the presence of nuclear matrix protein (NMP) in the blood of women at the early stage of breast cancer, which is absent in the blood of healthy women, as well as those with fibroadenoma.  NMP66 has been selected as a marker for further development and clinical trials of a test for use in the detection and monitoring of women with, or at risk for, breast cancer have been initiated (Wright and McGechan, 2003).  However, there are no published studies on the effectiveness of NMP66 testing at this time.\n",
      "HERmark\n",
      "HERmark Breast Cancer Assay ( biosciences monogram) is used to help determine prognosis and therapeutic choices for metastatic breast cancer (Raman, et al., 2013). Clinical practice guidelines recommend determining HER2 status in patients with all invasive breast cancer, but caution that current HER2 testing methods such as central immunohistochemistry and Fluorescence in situ Hybridization test may be inaccurate in approximately 20% of cases. According to the HERmark Web site, their method precisely quantifies HER2 total protein and HER2 homodimer levels in formalin-fixed, paraffin-embedded tissue sections and outperformed Fluorescence in situ Hybridization at determining patient outcomes in patients with metastatic breast cancer.\n",
      "HERmark testing has been proposed for a number of indications, including use to predict response to trastuzumab in the treatment of metastatic breast cancer.  Monogram, the manufacturer of the HERmark test, claims that the test can provide a more precise and quantitative measurement of the HER2 gene than IHC and fluorescent in-situ hybridization (FISH) tests. The HERmark provides a quantitative measurement of HER2 total protein and HER2 homodimer levels, while conventional methods are an indirect measure of the HER2 gene, the manufacturer claims. The HERmark test will be offered as a CLIA-validated assay through Monogram's CAP-certified clinical laboratory.  Other proposed indications for HERmark include determining the prognosis for breast cancer, and predicting treatment results in cancers other than breast cancer (e.g., ovarian prostate, head and neck, etc.).  There are no current recommendations from leading medical professional organizations for the use of HERmark testing for breast cancer.\n",
      "Yardley et al (2015) compared quantitative HER2 expression by the HERmark Breast Cancer Assay (HERmark) with routine HER2 testing by immunohistochemistry (IHC) and fluorescence in situ hybridization (FISH), and correlated HER2 results with overall survival (OS) of breast cancer patients in a multicenter Collaborative Biomarker Study (CBS). Two hundred and thirty-two formalin-fixed, paraffin-embedded breast cancer tissues and local laboratory HER2 testing results were provided by 11 CBS sites. HERmark assay and central laboratory HER2 IHC retesting were retrospectively performed in a blinded fashion. HER2 results by all testing methods were obtained in 192 cases. HERmark yielded a continuum of total HER2 expression (H2T) ranging from 0.3 to 403 RF/mm2 (approximately 3 logs). The distribution of H2T levels correlated significantly (P<0.0001) with all routine HER2 testing results. The concordance of positive and negative values (equivocal cases excluded) between HERmark and routine HER2 testing was 84% for local IHC, 96% for central IHC, 85% for local FISH, and 84% for local HER2 status. OS analysis revealed a significant correlation of shorter OS with HER2 positivity by local IHC (HR=2.6, P=0.016), central IHC (HR=3.2, P=0.015), and HERmark (HR=5.1, P<0.0001) in this cohort of patients most of whom received no HER2-targeted therapy. The OS curve of discordant low (HER2 positive but H2T low, 10% of all cases) was aligned with concordant negative (HER2 negative and H2T low, HR=1.9, P=0.444), but showed a significantly longer OS than concordant positive (HER2 positive and H2T high, HR=0.31, P=0.024). Conversely, the OS curve of discordant high (HER2 negative but H2T high, 9% of all cases) was aligned with concordant positive (HR=0.41, P=0.105), but showed a significantly shorter OS than concordant negative (HR=41, P<0.0001).\n",
      "MDM2\n",
      "Noon et al (2010) stated that renal cell carcinoma (RCC) is the most common type of kidney cancer and follows an unpredictable disease course.  These researchers reviewed 2 critical genes associated with disease progression -- p53 and murine double minute 2 (MDM2) -- and provided a comprehensive summary and critical analysis of the literature regarding these genes in RCC.  Information was compiled by searching the PubMed database for articles that were published or e-published up to April 1, 2009. Search terms included renal cancer, renal cell carcinoma, p53, and MDM2.  Full articles and any supplementary data were examined; and, when appropriate, references were checked for additional material.  All studies that described assessment of p53 and/or MDM2 in renal cancer were included.  The authors concluded that increased p53 expression, but not p53 mutation, is associated with reduced overall survival/more rapid disease progression in RCC.  There also was evidence that MDM2 up-regulation is associated with decreased disease-specific survival.  Two features of RCC stood out as unusual and will require further investigation:\n",
      "\n",
      "increased p53 expression is tightly linked with increased MDM2 expression; and\n",
      "patients who have tumors that display increased p53 and MDM2 expression may have the poorest overall survival. \n",
      "\n",
      "Because there was no evidence to support the conclusion that p53 mutation is associated with poorer survival, it seemed clear that increased p53 expression in RCC occurs independent of mutation.  The authors stated that further investigation of the mechanisms leading to increased p53/MDM2 expression in RCC may lead to improved prognostication and to the identification of novel therapeutic interventions.\n",
      "OVA1\n",
      "OVA1 is a blood test used to aid in the evaluation of pelvic masses for the likelihood of malignancy before surgery. OVA1 measures five biomarkers: apolipoprotein A1 (Apo A-1), beta-2 microglobulin (B2M), CA-125 prealbumin and transferrin. The results of these measurements are applied to an algorithm, resulting in a numerical score.\n",
      "The OVA1 Test (Vermillion Inc. and Quest Diagnostics) is a serum test that is intended to help physicians determine if a woman is at risk for a malignant pelvic mass prior to biopsy or exploratory surgery, when the physicianâs independent clinical and radiological evaluation does not indicate malignancy (Mundy, et al., 2010). The OVA1 Test employs an in vitro diagnostic multivariate index (IVDMIA) that combines the results of five immunoassays to produce a numerical score indicating a women's likelihood of malignancy. The OVA1 Test is intended to help physicians assess if a pelvic mass is benign or malignant in order to help determine whether to refer a woman to a gynecologic oncologist for surgery. The OVA1 Test was cleared by the FDA for use in women who meet the following criteria: over age 18, ovarian adnexal mass present for which surgery is planned, and not yet referred to an oncologist. The intended use of the OVA1 Test is an aid to further assess the likelihood that malignancy is present when the physicianâs independent clinical and radiological evaluation does not indicate malignancy. According to the product labeling, the OVA1 Test is not intended as a screening or stand-alone diagnostic assay. There is a lack of evidence in the peer-reviewed published medical literature on the OVA1 Test.\n",
      "Ueland et al (2011) sought to compare the effectiveness of physician assessment with the OVA1 multivariate index assay in identifying high-risk ovarian tumors.  The multivariate index assay was evaluated in women scheduled for surgery for an ovarian tumor in a prospective, multi-institutional trial involving 27 primary- care and specialty sites throughout the United States. Preoperative serum was collected, and results for the multivariate index assay, physician assessment, and CA 125 were correlated with surgical pathology. Physician assessment was documented by each physician before surgery. CA 125 cutoffs were chosen in accordance with the referral guidelines of the American College of Obstetricians and Gynecologists.  The study enrolled 590 women, with 524 evaluable for the multivariate index assay and CA 125, and 516 for physician assessment. Fifty-three percent were enrolled by nongynecologic oncologists. There were 161 malignancies and 363 benign ovarian tumors. Physician assessment plus the multivariate index assay correctly identified malignancies missed by physician assessment in 70% of nongynecologic oncologists, and 95% of gynecologic oncologists. The multivariate index assay also detected 76% of malignancies missed by CA 125. Physician assessment plus the multivariate index assay identified 86% of malignancies missed by CA 125, including all advanced cancers. The investigators stated that the performance of the multivariate index assay was consistent in early- and late-stage cancers. \n",
      "Ware Miller et al (2011) sought to estimate the performance of the ACOG referral guidelines for pelvic mass with the OVA1 multivariate index assay. A prospective, multi-institutional trial included 27 primary care and specialty sites throughout the United States. The College guidelines were evaluated in women scheduled for surgery for an ovarian mass. Clinical criteria and blood for biomarkers were collected before surgery. A standard CA 125-II assay was used and the value applied to the multivariate index assay algorithm and the CA 125 analysis. Study results were correlated with surgical pathology. Of the 590 women enrolled with ovarian mass on pelvic imaging, 516 were evaluable. There were 161 malignancies (45 premenopausal and 116 postmenopausal). The College referral criteria had a modest sensitivity in detecting malignancy. Replacing CA 125 with the multivariate index assay increased the sensitivity (77-94%) and negative predictive value (87-93%) while decreasing specificity (68-35%) and positive predictive value (52-40%). Similar trends were noted for premenopausal women and early-stage disease. \n",
      "Bristow et al (2013) sought to validate the effectiveness of a multivariate index assay in identifying ovarian malignancy compared to clinical assessment and CA125-II, among women undergoing surgery for an adnexal mass after enrollment by non-gynecologic oncology providers. A prospective, multi-institutional trial enrolled female patients scheduled to undergo surgery for an adnexal mass from 27 non-gynecologic oncology practices. Pre-operative serum samples and physician assessment of ovarian cancer risk were correlated with final surgical pathology.  A total of 494 subjects were evaluable for multivariate index assay, CA125-II, and clinical impression. Overall, 92 patients (18.6%) had a pelvic malignancy. Primary ovarian cancer was diagnosed in 65 patients (13.2%), with 43.1% having FIGO stage I disease. For all ovarian malignancies, the sensitivity of the multivariate index assay was 95.7% (95%CI=89.3-98.3) when combined with clinical impression. The multivariate index assay correctly predicted ovarian malignancy in 91.4% (95%CI=77.6-97.0) of cases of early-stage disease, compared to 65.7% (95%CI=49.2-79.2) for CA125-II. The multivariate index assay correctly identified 83.3% malignancies missed by clinical impression and 70.8% cases missed by CA125-II. Multivariate index assay was superior in predicting the absence of an ovarian malignancy, with a negative predictive value of 98.1% (95%CI=95.2-99.2). Both clinical impression and CA125-II were more accurate at identifying benign disease. The multivariate index assay correctly predicted benign pathology in 204 patients (50.7%, 95%CI=45.9-55.6) when combined with clinical impression. \n",
      "Longoria et al (2014) sought to analyze the effectiveness of the OVA1 multivariate index assay (MIA) in identifying early-stage ovarian malignancy compared to clinical assessment, CA 125-II, and modified American Congress of Obstetricians and Gynecologists (ACOG) guidelines among women undergoing surgery for an adnexal mass.  Patients were recruited in 2 related prospective, multi-institutional trials involving 44 sites. All women had preoperative imaging and biomarker analysis. Preoperative biomarker values, physician assessment of ovarian cancer risk, and modified ACOG guideline risk stratification were correlated with surgical pathology. A total of 1016 patients were evaluable for MIA, CA 125-II, and clinical assessment. Overall, 86 patients (8.5%) had primary-stage I/II primary ovarian malignancy, with 70.9% having stage I disease and 29.1% having stage II disease. For all early-stage ovarian malignancies, MIA combined with clinical assessment had significantly higher sensitivity (95.3%; 95% confidence interval [CI], 88.6-98.2) compared to clinical assessment alone (68.6%; 95% CI, 58.2-77.4), CA 125-II (62.8%; 95% CI, 52.2-72.3), and modified ACOG guidelines (76.7%; 95% CI, 66.8-84.4) (P < .0001). Among the 515 premenopausal patients, the sensitivity for early-stage ovarian cancer was 89.3% (95% CI, 72.8-96.3) for MIA combined with clinical assessment, 60.7% (95% CI, 42.4-76.4) for clinical assessment alone, 35.7% (95% CI, 20.7-54.2) for CA 125-II, and 78.6% (95% CI, 60.5-89.8) for modified ACOG guidelines. Early-stage ovarian cancer in postmenopausal patients was correctly detected in 98.3% (95% CI, 90.9-99.7) of cases by MIA combined with clinical assessment, compared to 72.4% (95% CI, 59.8-82.2) for clinical assessment alone, 75.9% (95% CI, 63.5-85.0) for CA 125-II, and 75.9% (95% CI, 63.5-85.0) for modified ACOG guidelines. \n",
      "Bristow et al (2013) assessed the impact on referral patterns of using the OVA1 Multivariate Index Assay, CA125, modified-American College of Obstetricians and Gynecologists referral guidelines, and clinical assessment among patients undergoing surgery for an adnexal mass after initial evaluation by nongynecologic oncologists. Overall, 770 patients were enrolled by nongynecologic oncologists from 2 related, multiinstitutional, prospective trials and analyzed retrospectively. All patients had preoperative imaging and biomarker analysis. The subset of patients enrolled by nongynecologic oncologists was analyzed to determine the projected referral patterns and sensitivity for malignancy based on multivariate index assay (MIA), CA125, modified-American College of Obstetricians and Gynecologists (ACOG) guidelines, and clinical assessment compared with actual practice. The prevalence of malignancy was 21.3% (n = 164). In clinical practice, 462/770 patients (60.0%) were referred to a gynecologic oncologist for surgery. Triage based on CA125 predicted referral of 157/770 patients (20.4%) with sensitivity of 68.3% (95% confidence interval [CI], 60.8-74.9). Triage based on modified-ACOG guidelines would have resulted in referral of 256/770 patients (33.2%) with a sensitivity of 79.3% (95% CI, 72.4-84.8). Clinical assessment predicted referral of 184/763 patients (24.1%) with a sensitivity of 73.2% (95% CI, 65.9-79.4). Risk stratification using multivariate index assay would have resulted in referral of 429/770 (55.7%) patients, with sensitivity of 90.2% (95%  CI, 84.7-93.9). MIA demonstrated statistically significant higher sensitivity (P < .0001) and lower specificity (P < .0001) for detecting malignancy compared with clinical assessment, CA125, and modified-ACOG guidelines. \n",
      "Goodrich et al (2014) investigated the relationship between imaging and the multivariate index assay (MIA) in the prediction of the likelihood of ovarian malignancy before surgery. Subjects were recruited in 2 related prospective, multiinstitutional trials that involved 44 sites across the United States. Women had ovarian imaging, biomarker analysis, and surgery for an adnexal mass. Ovarian tumors were classified as high risk for solid or papillary morphologic condition on imaging study. Biomarker and imaging results were correlated with surgical findings. Of the 1110 women who were enrolled with an adnexal mass on imaging, 1024 cases were evaluable. There were 255 malignant and 769 benign tumors. High-risk findings were present in 46% of 1232 imaging tests and 61% of 1024 MIA tests. The risk of malignancy increased with rising MIA scores; similarly, the likelihood of malignancy was higher for high-risk, compared with low-risk, imaging. Sensitivity and specificity for the prediction of malignancy were 98% (95% CI, 92-99) and 31% (95% CI, 27-34) for ultrasound or MIA; 68% (95% CI, 58-77) and 75% (95% CI, 72-78) for ultrasound and MIA, respectively. For computed tomography scan or MIA, sensitivity was 97% (95% CI, 92-99) and specificity was 22% (95% CI, 16-28); the sensitivity and specificity for computed tomography scan and MIA were 71% (95% CI, 62-79) and 70% (95% CI, 63-76). Only 1.6% of ovarian tumors were malignant when both tests indicated low risk. \n",
      "An assessment by the BlueCross BlueShield Association Technology Evaluation Center (BCBSA, 2013) stated: \"The evidence regarding the effect of OVA1 ...on health outcomes is indirect and based on studies of diagnostic performance of the tests in patients undergoing surgery for adnexal masses. Although the studies show improvements in sensitivity and worsening of specificity with the use of the tests in conjunction with clinical assessment, there are problems in concluding that this results in improved health outcomes. The clinical assessment performed in the studies is not well characterized. Although OVA1 improves sensitivity, specificity declines so much that most patients test positive.\"\n",
      "An technology assessment by the ECRI Institute (2015) concluded that the evidence on OVA1 consists of cross-sectional diagnostic accuracy studies. This evidence as reported in article abstracts is unclear as to whether use of OVA1 improves patient-oriented outcomes because none of the studies reported the direct impact of these tests on survival or quality of life. The primary rationale for using these tests is to select the type of surgeon to perform the primary surgery.\n",
      "Stewart et al (2016) reported on a survey of primary care physicians on how often they refer patients diagnosed with ovarian cancer to gynecological oncologists, finding that a total of 84% of primary care physicians (87% of family/general practitioners, 81% of internists and obstetrician/gynecologists) said they always referred patients to gynecologic oncologists for treatment. Common reasons for not always referring were patient preference or lack of gynecologic oncologists in the practice area. A total of 23% of primary care physicians had heard of the OVA1 test, which helps to determine whether gynecologic oncologist referral is needed. The authors noted that, although referral rates reported here are high, it is not clear whether ovarian cancer patients are actually seeing gynecologic oncologists for care. \n",
      "Eskander et al (2016) conducted a retrospective chart review of patients who received the OVA1. Twenty-two obstetricians/gynecologists were recruited from a variety of practices and hospitals throughout the United States. A total of 136 patients with elevated-risk assay results were assessed, of whom 122 underwent surgery to remove an adnexal mass. Prior to surgery, 98 (80%) of the patients were referred to a gynecologic oncologist with an additional 11 (9%) having a gynecologic oncologist available if required by intra-operative findings. Primary ovarian cancer was found in 65 (53%) patients, and gynecologic oncologists performed 61 (94%) of the initial surgeries these patients. Similar results were found in premenopausal and postmenopausal patients.\n",
      "Forde et al (2016) conducted an economic analysis model to evaluate the clinical and cost implications of adopting OVA1 in clinical practice versus the modified ACOG referral guidelines and CA-125 alone, over a lifetime horizon, from the perspective of the public payer. Clinical parameters used to characterize patients' disease status, quality of life, and treatment decisions were estimated using the results of published studies; costs were approximated using reimbursement rates from CMS fee schedules. Model endpoints included overall survival (OS), costs, quality-adjusted life years (QALYs), and incremental cost-effectiveness ratio (ICER). The cost-effectiveness threshold was set to $50,000 per QALY. One-way sensitivity analysis was performed to assess uncertainty of individual parameters included in the analysis. All costs were reported in 2014 US dollars. Use of OVA1 was cost-effective, resulting in fewer re-operations and pre-treatment CT scans. Overall OVA1 resulted in an ICER of $35,094/QALY gained. OVA1 was also cost-saving and QALY-increasing compared to use of CA-125 alone with an ICER of $12,189/QALY gained. One-way sensitivity analysis showed the ICER was most affected by the following parameters:\n",
      "\n",
      "sensitivity of OVA1;\n",
      "sensitivity of mACOG; and\n",
      "percentage of patients, not referred to a gynecologic oncologist, who were correctly diagnosed with advanced epithelial ovarian cancer (EOC).\n",
      "\n",
      "The authors concluded that OVA1 is a more cost-effective triage strategy than mACOG or CA-125. It is expected to increase the percentage of women with ovarian cancer that are referred to gynecologic oncologists, which is shown to improve clinical outcomes. Limitations include the use of assumptions when published data was unavailable, and the use of multiple sources for survival data. \n",
      "Urban et al (2017) reported that the addition of a patient-reported symptom index (SI), which captures subjective symptoms in an objective manner, improved the sensitivity of the OVA1 multivariate index assay (MIA). The investigators conducted a prospective study of patients seen at a tertiary care medical center. Following consent, patients completed an SI and preoperative serum was collected for an OVA1 multivariate index assay. Results for the SI and OVA1 were correlated with operative findings and surgical pathology. Of 218 patients enrolled, 124 (56.9%) had benign disease and 94 (43.1%) had borderline tumors or carcinomas. Sixty-six patients had a primary ovarian or fallopian tube cancer. The median age of patients enrolled in this study was 54 years (interquartile range, 44-63 years), of whom 148 (67.9%) were postmenopausal. More than a third (36.3%) of patients with benign masses was accurately identified as low risk by MIA and SI. The sensitivity, specificity, positive predictive value (PPV) and negative predictive value (NPV) of the SI relative to primary ovarian cancer was 87.9% (95% CI, 77.9%-93.7%), 70.2% (95% CI, 61.6%-77.5%), 61.1% (95% CI 51.0-70.2%) and 91.6% (95% CI, 84.3%-95.7%), respectively. The sensitivity, specificity, PPV and NPV of CA125 was 75.4% (95% CI, 63.7%-84.2%), 85.7% (95% CI 78.3%-90.9%), 74.2% (62.6%-82.3%) and 86.4% (95% CI, 79.1%-91.5%), respectively. The sensitivity, specificity, PPV and NPV of the MIA were 93.9% (95% CI, 85.4%-97.6%), 55.6 (95% CI 46.9%-64.1%), 53.0% (95% CI 44.0%-61.8% and 94.5% (95% CI, 94.5%-100%), respectively. The overall sensitivity for the combination of MIA plus SI was 100% (66/66; 95% CI, 94.5%-100%), and specificity was 36.3% (45/124; 95% CI, 28.4%-45.0%), with a PPV of 45.5% (37.6% to 53.6%) and a NPV of 100% (95% CI, 92.1%-100%).  Limitations of this study noted by the authors include the small sample size and the high prevalence of ovarian malignancies in this population that was largely from a tertiary care center. It should also be noted that the sensitivity and negative predictive value of SI plus CA 125 was 96.9% (95% CI 89.5%-99.2%) and 97.3% (95% CI 90.5%-99.2%), which exceeded that of MIA alone but was somewhat less than MIA plus SI.\n",
      "Ovarian cancer guidelines from the National Comprehensive Cancer Network (2016) note that the Society of Gynecologic Oncology (SGO), the FDA, and the Mayo Clinic have stated that the OVA1 test should not be used as a screening tool to detect ovarian cancer. The NCCN explains that the OVA1 attempts to preoperatively classify adenixal masses as benign or malignant and suggests that patients can be assessed for who should undergo surgery by an experienced gynecologic oncologist and who can have surgery in the community. \"Based upon data documenting an increased survival, NCCN guidelines panel members recommended that all patients should undergo surgery by an experienced gynecologic oncologist (Category 1 recommendation).\"\n",
      "Guidelines on management of adnexal masses from the American College of Obstetricians and Gynecologists (ACOG, 2016) state that the OVA1 multivariate index assay has demonstrated higher sensitivity and negative predictive value compared with clinical impression and CA 125 alone. The guidelines state that serum biomarker panels [OVA1 and ROMA] may be used as an alternative to CA 125 alone in determining the need for referral to or consultation with a gynecological oncologist when an adenexal mass requires surgery. The guidelines state that trials that have evaluated the predictive value of these panels show potential for improved specificity; \"[h]owever, comparative research has not yet defined the best testing approach.\"\n",
      "ACOG guidelines (2016) state that, primarily based upon consensus and expert opinion (Level C), \"[s]erum biomarker panels may be used as an alternative to CA 125 level alone in determining the need for referral to or consultation with a gynecological oncologist when an adnexal mass requires surgery.\" The guidelines state that, based upon \"limited or inconsistent\" evidence (Level B), consultation or referral to a gynecological oncologist is recommended for women with an adnexal mass that meet one or more of the following criteria; \n",
      "\n",
      "postmenopausal with elevated CA 125 level, ultrasound findings suggestive of malignancy, ascites, a nodular or fixed pelvic mass, or evidence of abdominal or distant metastases;\n",
      "premenosausal with very elevated CA 125 level, ultrasound findings suggestive of malignancy, ascites, a nodular or fixed pelvic mass, or evidence of abdominal or distant metastases;\n",
      "premenopausal or postmenopausal with an elevated score on a formal risk assessment test such as the multivariate index assay, risk of malignancy index, or the Risk of Ovarian Malignancy algorithm or one of the ultrasound-based scoring systems from the International Ovarian Tumor Analysis group.\n",
      "\n",
      "The UK National Institute for Health Research Health Technology Assessment Programme has commissioned an assessment (Westwood, et al., 2016) comparing the Risk of Malignancy Index (RMI) to alternative risk scores for ovarian cancer, including Overa/OVA2 (Vermillion), as well as the ROMA score, simple rules ultrasound classification system (IOTA), Assessment of Different NEoplasias in the adnexa (ADNEX) model (IOTA group). The assessment is scheduled to be completed in 2017.\n",
      "Dunton et al (2019) stated that based on evidence that African American (AA) women have lower CA125 values than Caucasian (C) women, these investigators examined if this disparity would have an impact on ovarian cancer detection using CA125 and multi-variate index assay (MIA).  Serum from 2 prospective trials of 1,029 samples (274 malignancies [250 C/24 AA]) were analyzed for CA125 and MIA results.  Clinical performance was calculated.  Sensitivity of MIA in Caucasian women was 93.2 %, 74.4 % for CA125 at the ACOG approved cut-off level of 200 U/ml cut-off, and 80.4 % using the 2007, Dearking 67 U/ml cut-off.  In AA women, MIA sensitivity was 79.2 %, 33.3 % for CA125 at the ACOG approved cut-off levels and 62.5 % at the 2007, Dearking 67 U/ml cut-off.  The authors concluded that these findings supported that CA125 in AA women with adnexal masses had lower sensitivity than MIA no matter what the cut-off value was.  Implementation of MIA in evaluation of adnexal masses should increase sensitivity of detection of malignancy compared with CA125, especially in AA women.\n",
      "These researchers stated that as the number of AA patients in the studies that made up the analysis database for this research was fairly small, they presented data on all malignancies rather than dividing them based on histologic subtype.  During their exploratory analysis, these investigators carried out these calculations, but due to the small sample sizes found the confidence intervals (CIs) too wide to draw solid conclusions from.  These researchers are in the process of developing research opportunities to add further AA women with primary ovarian malignancies to their specimen repositories in order to confirm the results found in this analysis.  Moreover, the authors stated that within the next 10 years, this research could be a stepping-stone toward closing the survivorship gap between Caucasian women and AA women, where ovarian cancer is concerned.  Increased use of a more sensitive test such as MIA in minority women, and clinical awareness of the shortcomings of entrenched medical practices like CA125 could potentially increase early detection, which is key for improved survivorship.\n",
      "In a retrospective study, Dunton et al (2020) examined the use of Multivariate Index Assay (MIA OVA1) by gynecologists and determined referral practices and surgical decision-making for women with adnexal masses and low-risk MIA OVA1 scores.  Information on patients who received an OVA1 test was collected from 22 gynecologic practices via a chart review.  Referral patterns were examined for patients with low-risk OVA1 results before 1st surgical intervention.  Chart reviews were from a variety of practice and hospital settings representing major geographic regions within the U.S.  A total of 282 independent patient charts were reviewed.  Low-risk results were found for 146 patients (52 %).  Surgery was carried out on 82 (56 %) patients with low-risk scores.  The referral rate to specialty care was 21 % (17/82) for low-risk OVA1 patients.  A total of 3 low-malignant potential tumors were identified in the low-risk patients, with no cases of invasive malignancy; 86 % of the surgeries carried out on low-risk OVA1 patients were minimally invasive.  In 44 % of the low-risk OVA1 patients, no surgical intervention was carried out.  The authors concluded that a high proportion of low-risk OVA1 patients were not referred to a gynecologic oncologist before surgery, indicating gynecologists may use MIA OVA1 along with clinical and radiographic findings to appropriately retain patients for their care.  This practice is safe and may be cost-saving, with patient satisfaction implications.  These researchers stated that these findings may help catalyze the development of prospective studies to better examine the role of OVA1 in the clinical decision-making process and to help with long-term collection of data that can be used to examine outcomes of decisions made on both low-and elevated risk OVA1 tests.\n",
      "The authors stated that this study had several drawbacks.  Because this was a retrospective study, it may be of limited use in causal inference because it was not possible to identify all confounding factors impacting referral and surgical planning.  Physicians who chose to participate in the study may have been more likely to have a positive opinion of the test and use it to guide referral.  Furthermore, because physicians were allowed to select patients for review there was a possibility of a selection or recall bias for cases that best fit the physicianâs ideal of the clinical application in both the elevated and low-risk situations.  For instance, in a previous study (Eskander et al, 2016), these investigators showed the malignancy rate of this population to be 64 %, which is much higher than previous studies of intended use populations (Bristow et al, 2013; Ueland et al, 2011).\n",
      "Dunton et al (2021) noted that ovarian cancer is the deadliest gynecologic cancer, with no recommended screening test to aid in early detection.  Cancer antigen 125 (CA125) is a serum biomarker commonly used by clinicians to evaluate pre-operative cancer risk, but it under-performs in pre-menopausal women, early-stage malignancies, and several histologic subtypes.  OVA1 is a multi-variate index assay that combines CA125 and 4 other serum proteins to evaluate the malignant risk of an adnexal mass.  These researchers examined the performance of OVA1 in a cohort of patients with low-risk serum CA125 values.  They analyzed patient data from previous collections (n = 2,305, prevalence = 4.5 %) where CA125 levels were at or below 67 units/milliliter (U/ml) for pre-menopausal women and 35 U/ml for post-menopausal women.  These investigators compared the performance of OVA1 to CA125 in classifying the risk of malignancy in this cohort, including sensitivity, specificity, PPV and NPV.  The overall sensitivity of OVA1 in patients with a low-risk serum CA125 was 59 % with a false-positive rate of 30 %.  OVA1 detected over 50 % of ovarian malignancies in pre-menopausal women despite a low-risk serum CA125.  OVA1 also correctly identified 63 % of early-stage cancers missed by CA125.  The most common epithelial ovarian cancer subtypes in the study population were mucinous (25 %) and serous (23 %) carcinomas.  Despite a low-risk CA125, OVA1 successfully detected 83 % of serous, 58 % of mucinous, and 50 % of clear cell ovarian cancers.  The authors concluded that as a standalone test, CA125 missed a significant number of ovarian malignancies that could be detected by OVA1.  This was particularly important for pre-menopausal women and early-stage cancers, which have a much better long-term survival than late-stage malignancies.  These investigators stated that using OVA1 in the setting of a normal serum CA125 could aid in identifying at-risk ovarian tumors for referral to a gynecologic oncologist, potentially improving OS.\n",
      "The authors stated that a drawback of this study was the retrospective nature of the data analysis, which was carried out after merging several study databases.  Furthermore, the percentage of early-stage ovarian cancer in this study (70 %) was twice that expected in the general population, suggesting a possible sampling bias.  However, this shift toward early-stage cancers allowed for a more robust evaluation of test performance in this cohort.\n",
      "In the largest-of-its-kind study, Reilly et al (2023) examined the use of CA125 and OVA1, commonly employed as ovarian tumor markers for evaluating the risk of malignancy.  This study focused on the ability and utility of these tests to reliably predict patients at low risk for ovarian cancer.  Clinical utility endpoints were 12-month maintenance of benign mass status, reduction in gynecologic oncologist referral, avoidable surgical intervention, and associated cost savings.  This was a retrospective, multi-center review of data from electronic medical records and administrative claims databases.  Patients receiving a CA125 or OVA1 test between October 2018 and September 2020 were identified and followed for 12 months using site-specific electronic medical records (EMRs) to examine tumor status and utilization outcomes.  Propensity score adjustment was used to control for confounding variables.  Payer allowed amounts from Merative MarketScan Research Databases were used to estimate 12-month episode-of-care costs per patient, including surgery and other interventions.  Among 290 low-risk OVA1 patients, 99.0 % remained benign for 12 months compared with 97.2 % of 181 low-risk CA125 patients.  The OVA1 cohort exhibited 75 % lower odds of surgical intervention in the overall sample of patients (adjusted OR: 0.251, p â¤ 0.0001), and 63 % lower odds of gynecologic oncologist utilization among pre-menopausal women (adjusted OR: 0.37, p = 0.0390) versus CA125.  OVA1 showed significant savings in surgical interventions ($2,486, p â¤ 0.0001) and total episode-of-care costs ($2,621, p â¤ 0.0001) versus CA125.  The authors concluded that the findings of this study suggested that OVA1 was associated with significant reduction in avoidable surgeries overall and sub-specialty referrals for pre-menopausal patients, as well as substantial cost savings per patient.  Moreover, these researchers stated that opportunities for future research include using cost-utility modeling to estimate the long-term impact of using biomarker assays on ovarian tumor treatment costs and quality of life.\n",
      "The authors stated that there were drawbacks inherent to retrospective chart reviews.  Because of the reliance on medical records, data quality is dependent upon accurate documentation and the ability of the medical records reviewer to understand the gathered information.  As is true of all âreal worldâ studies, it is not possible to fully control for exogenous factors that could influence the results of the study, such as determination of eligibility to receive an ovarian cancer diagnostic test and which test (OVA1 or CA125) they should receive.  Despite both tests using biomarkers to determine the likelihood of ovarian cancer, there may be other clinical factors physicians rely upon to determine which test patients receive.  Furthermore, OVA1 is not widely covered across all commercial plans, limiting which physicians can order the test and ultimately which patients can receive it.  These investigators examined the impact of the volume imbalance and potential data skewness and determined that a propensity score adjustment methodology was most appropriate to both leverage the considerable treatment cohort volume and maintain the comparator cohort volume.  Moreover, the low prevalence of high-risk cases led these researchers to postpone analysis of this subgroup until adequately powered samples can be obtained.  In addition, the studyâs utilization and economic outcomes were captured from events occurring within the enrolled siteâs EMRs, which may have excluded a nominal number of âout-of-networkâ events.  Finally, the surgery metrics did not capture utilization of laparotomies, which the participating sites confirmed was not the standard-of-care and is rarely deployed based on the presence of the preferred minimally invasive surgical approach.\n",
      "ColonSentry\n",
      "The ColonSentry test (GeneNews, Toronto, Canada) measures the expression of seven genes, which serve as biomarkers to detect colorectal cancer. Interpretation of the status of these seven biomarkers is intended to assist physicians in identifying patients who have an increased current risk. According to the manufacturer, individuals assessed as having an increased current risk of colorectal cancer should consider having a colonoscopy.  Individuals assessed as having a decreased current risk of colorectal cancer should discuss with their doctor further screening, including repeating ColonSentry at regular intervals. There is a lack of evidence in the peer-reviewed published medical literature on the effectiveness of colorectal cancer screening with ColonSentry. No current evidence-based guidelines from medical professional organizations or public health agencies recommend ColonSentry for colorectal cancer screening.\n",
      "Prostate Px\n",
      "Prostate Px (Aureon) uses a prostate cancer patient's biopsy tissue to provide an assessment of disease severity and disease recurrence. Clinical data is integrated with an analysis of each patientâs cancer using tissue histology and molecular biomarkers, such as androgen receptor, associated with disease progression. Although the manufacturer states that the results of the Prostate Px can be used in decision-making, there is a lack of evidence of the clinical utility of this test in altering the management of patients such that clinical outcomes are improved.\n",
      "Post-Op Px\n",
      "Post-Op Px is a prognostic test that utilizes a patented systems pathology approach to analyze prostatectomy tissue by combining cellular, molecular and clinical information to provide a thorough and more accurate picture of each patient's individual risk of prostate cancer recurrence. (Aureon, 2010).   Donovan et al  (2011) evaluated the performance of a systems-based risk assessment tool with standard defined risk groups and the 10 year postoperative normogram for predicting disease progression. The systems model was found to be more accurate than standard risk groups both to predict significant disease progression (p <  0.001) and for predicting prostate-specific antigen recurrence (p < 0.001).  However, this study has not been replicated in the peer-reviewed literature.\n",
      "EGFR\n",
      "An assessment by the BlueCross BlueShield Association Technology Evaluation Center (2010) concluded that tumor-cell epidermal growth factor receptor (EGFR) mutation analysis to predict response to erlotinib (Tarceva) in patients with advanced non-small cell lung cancer (NSCLC) meets the Blue Cross and Blue Shield Association Technology Evaluation Center (TEC) criteria.  Furthermore, guidelines from the National Comprehensive Cancer Network (NCCN, 2010) recommend EGFR testing for the following histologic subtypes of NSCLC:\n",
      "\n",
      "adenocarcinoma,\n",
      "large cell, and\n",
      "NSCLC not otherwise specified. \n",
      "\n",
      "Epidermal growth factor receptor testing is not recommended for squamous cell carcinoma.\n",
      "The Alberta Provincial Thoracic Tumour Teamâs clinical practice guideline on \"Non-small cell lung cancer stage IV\" (2011) stated that \"First-line monotherapy with the epidermal growth factor receptor (EGFR) tyrosine kinase inhibitor gefitinib is recommended for patients with EGFR mutation-positive NSCLC.  Testing for EGFR mutations should take place for all eligible patients with advanced NSCLC and adenocarcinoma histology who are being considered for first-line therapy with gefitinib, irrespective of their gender, ethnicity, and smoking status\".\n",
      "NCCN non-small cell lung cancer guidelines (2015) state that EGFR and ALK testing should be conducted as part of a multiplex/next generation sequencing. The NCCN NSCLC Guidelines Panel \"strongly endorses broader molecular profiling with the goal of identifying rare mutations for which effective drugs may already be available, or to appropriately counsel patients regarding the availability of clinical trials. Broad molecular profiling is a key component of the improvement of care of patients with NSCLC.\"\n",
      "Gao et al (2012) stated that gefiinib and erlotinib are 2 similar small molecules of selective and reversible epidermal growth factor receptor-tyrosine kinase inhibitors (EGFR-TKIs), which have been approved for second-line or third-line indication in previously treated advanced NSCLC patients.  The results of comparing the EGFR-TKI with standard platinum-based doublet chemotherapy as the first-line treatment in advanced NSCLC patients with activated EGFR mutation were still controversial.  A meta-analysis was performed to derive a more precise estimation of these regimens.  Finally, 6 eligible trials involved 1,021 patients were identified.  The patients receiving EGFR-TKI as front-line therapy had a significantly longer PFS than patients treated with chemotherapy [median PFS was 9.5 versus 5.9 months; HR = 0.37; 95 % CI: 0.27 to 0.52; p < 0.001].  The overall response rate (ORR) of EGFR-TKI was 66.60 %, whereas the ORR of chemotherapy regimen was 30.62 %, which was also a statistically significant favor for EGFR-TKI [relative risk (RR) = 5.68; 95 % CI: 3.17 to 10.18; p < 0.001].  The OS was numerically longer in the patients received EGFR-TKI than patients treated by chemotherapy, although the difference did not reach a statistical significance (median OS was 30.5 versus 23.6 months; HR = 0.94; 95 % CI: 0.77 to 1.15; p = 0.57).  Comparing with first-line chemotherapy, treatment of EGFR-TKI achieved a statistical significantly longer PFS, higher ORR and numerically longer OS in the advanced NSCLC patients harboring activated EGFR mutations, thus, it should be the first choice in the previously untreated NSCLC patients with activated EGFR mutation.\n",
      "Guidelines from the American Society for Clinical Oncology (2016) state: \"The clinician should not use HER1/epidermal growth factor receptor expression by IHC to guide adjuvant chemotherapy selection\" in breast cancer.\n",
      "CEACAM-7\n",
      "Messick et al (2010) evaluated carcinoembryonic antigen cellular adhesion molecule-7 (CEACAM-7) expression in rectal cancer as a predictive recurrence factor.  A single-institution colorectal cancer database and a frozen tissue biobank were queried for rectal cancer patients.  CEACAM-7 messenger RNA (mRNA) expression from normal rectal mucosa and rectal cancers was analyzed using quantitative real-time polymerase chain reaction (PCR).  Expression-level differences among normal tissue, disease-free survivors, and those that developed recurrence were analyzed.  A total of 84 patients were included in the study, which consisted of 37 patients with non-recurrent disease (median follow-up of 170 months), 29 patients with recurrent disease, and 18 patients with stage IV disease.  CEACAM-7 expression was decreased 21-fold in rectal cancers compared with normal mucosa (p = 0.002).  The expression levels of CEACAM-7 were relatively decreased in tumors that developed recurrence compared with non-recurrence, significantly for stage II patients (14-fold relative decrease, p = 0.002).  For stages I-III, disease-free survival segregates were based on relative CEACAM-7 expression values (p = 0.036), specifically for stage II (p = 0.018).  The authors concluded that CEACAM-7 expression is significantly decreased in rectal cancer.  Expression differences between long-term survivors and those with recurrent disease introduce a potential tumor marker to define a subset of patients who benefit most from adjuvant therapy.  Moreover, they stated that additional study and validation are needed before CEACAM-7 can be applied in clinical settings.\n",
      "CFL1\n",
      "Castro et al (2010) assessed the potential value of cofilin (CFL1) gene (main member of the invasion/metastasis pathway) as a prognostic and predictive NSCLC biomarker.  Meta-analysis of tumor tissue microarray was applied to examine expression of CFL1 in archival lung cancer samples from 111 patients, and its clinicopathologic significance was investigated.  The robustness of the finding was validated using another independent data set.  Finally, the authors assayed in vitro the role of CFL1 levels in tumor invasiveness and drug resistance using 6 human NSCLC cell lines with different basal degrees of CFL1 gene expression.  Cofilin levels in biopsies discriminated between good and bad prognosis at early tumor stages (IA, IB, and IIA/B), where high CFL1 levels are correlated with lower overall survival rate (p < 0.0001).  Biomarker performance was further analyzed by IHC, hazard ratio (p < 0.001), and receiver-operating characteristic curve (area = 0.787; p < 0.001).  High CFL1 mRNA levels and protein content are positively correlated with cellular invasiveness (determined by Matrigel Invasion Chamber System) and resistance (2-fold increase in drug 50 % growth inhibition dose) against a list of 22 alkylating agents.  Hierarchical clustering analysis of the CFL1 gene network had the same robustness for stratified NSCLC patients.  The authors concluded that these findings indicated that the CFL1 gene and its functional gene network can be used as prognostic biomarkers for NSCLC and could also guide chemotherapeutic interventions.  Moreover, prospective, large-scale, randomized clinical trials are needed to establish the role of CFL1 as a prognostic and drug resistance marker for NSCLC.\n",
      "EarlyCDT-Lung\n",
      "The EarlyCDT-Lung (Oncimmune, De Soto, KS) test measures antibodies to 6 tumor-associated antigens: p53, NY-ESO-1, CAGE, GBU4-5, Annexin 1, and SOX2.  Elevation of any one of the panel of immuno-biomarkers above a predetermined cut-off value suggests that a tumor might be present.  The test is designed to be used in conjunction with diagnostic imaging.  High-risk individuals with a positive EarlyCDT-Lung would have additional testing such as a CT scan or the test would be used as a follow-up test for indeterminate lung nodules identified by CT.\n",
      "Boyle et al (2011) reported the sensitivity and specificity of an autoantibody panel of 6 tumor-related antigens (p53, NY-ESO-1, CAGE, GBU4-5, Annexin 1 and SOX2) in patients with lung cancer.  Three cohorts of patients with newly diagnosed lung cancer were identified: group 1 (n = 145), group 2 (n = 241) and group 3 (n = 269).  Patients were individually matched by gender, age and smoking history to a control individual with no history of malignant disease.  Serum samples were obtained after diagnosis but before any anticancer treatment.  Autoantibody levels were measured against the panel of 6 tumor-related antigens (p53, NY-ESO-1, CAGE, GBU4-5, Annexin 1 and SOX2).  Assay sensitivity was tested in relation to demographic variables and cancer type/stage.  The autoantibody panel demonstrated a sensitivity/specificity of 36 %/91 %, 39 %/89 % and 37 %/90 % in groups 1, 2 and 3, respectively, with good reproducibility.  There was no significant difference between different lung cancer stages, indicating that the antigens included covered the different types of lung cancer well.   The authors concluded that the assay confirms the value of an autoantibody panel as a diagnostic tool and offers a potential system for monitoring patients at high-risk of lung cancer.\n",
      "There is insufficient evidence of the effectiveness of the EarlyCDT-Lung as a screening test for the early detection of lung cancer.  Systematic screening for lung cancer is not unequivocally recommended by any major professional organization.  The USPSTF (2004) concluded that current evidence was insufficient to recommend for, or against, screening for lung cancer.  Whether earlier detection of lung cancer will translate to a mortality benefit remains unclear.\n",
      "E-cad\n",
      "Deeb et al (2004) stated that E-cadherin (E-cad) and epidermal growth factor receptor (EGFR) are important cell adhesion and signaling pathway mediators. Theyr reported the results of a study which aimed to assess their expression in lung adenocarcinoma (AdC) and squamous cell carcinoma (SCC) and their association with clinicopathologic variables.  Two to three cores from 130 resectable lung cancers (stages I-IIIA) were arrayed into three blocks using a Beecher system. Markers expression and coexpression were analyzed against clinicopathologic variables (age, gender, smoking status, performance status, weight loss, histology, grade, stage, and lymph node involvement) and patient survival. For E-cad, 65 cases (55%) were positive (+), 53 (45%) were negative (-); and for EGRF, 43 cases (34%) were (+), and 83 (66%) were (-). There was no significant association between E-cad or EGFR, and any of the clinicopathologic variables except for an association between EGFR(+) and SCC histologic type. Both negative and cytoplasmic staining of E-cad correlated with shorter patient survival with P=0.008 and 0.002, respectively. EGFR expression did not correlate with patient survival, but, patients with E-cad(-)/EGFR(+) phenotype had poorer survival than those with E-cad(+)/EGFR(-) (P=0.026). The authors concluded that lung AdC and SCC may be stratified based on expression of E-cad and EGFR with the E-cad(-)/EGFR(+) expression having a worse disease outcome.\n",
      "EML4-ALK\n",
      "Yoshida et al (2011) report that a subset of lung cancers harbors an EML4-ALK (echinoderm microtubule-associated protein-like 4-anaplastic lymphoma kinase) gene fusion, and they examined 15 lung adenocarcinomas with reverse-transcriptase polymerase chain reaction-proven EML4-ALK fusion transcripts and 30 ALK-negative cases. Positive rearrangement signals (splits or isolated 3' signals) were identified in 13 to 78% (mean Â± SD, 41% Â± 19%) of tumor cells in the ALK-positive cohort and in 0 to 15% (mean Â± SD, 6% Â± 4%) of cells in the ALK-negative cohort. Sensitivity was at 93% and specificity at 100%. The only false-negative tumor having only 13% CISH-positive cells displayed predominantly (76%) isolated 5' signals unaccompanied by 3' signals. FISH showed largely similar signal profiles, and the results were completely concordant with CISH.The authors stated that they have successfully introduced CISH for diagnosing EML4-ALK-positive lung adenocarcinoma. This method allows simultaneous visualization of genetics and tumor cytomorphology and facilitates the molecular evaluation and could be applicable in clinical practice to detect lung cancer that may be responsive to ALK inhibitors.\n",
      "Ellis et al (2011) conducted a systematic review and a consensus meeting of Canadian lung cancer oncologists and pathologists to make recommendations on the use of biomarkers in NSCLC.  The articles were reviewed by pairs of oncologists and pathologists to determine eligibility for inclusion.  Ten oncologists and pathologists reviewed and summarized the literature at a meeting attended by 37 individuals.  The findings included that there is some evidence that histology is prognostic for survival as well as evidence from multiple randomized clinical trials to recommend the following: histologic subtype is predictive of treatment efficacy and for some agents toxicity. Immunohistochemistry testing should be performed on NSCLC specimens that cannot be classified accurately with conventional H&E staining. As EGFR mutations are predictive of benefit from tyrosine kinase inhibitors, diagnostic NSCLC samples should be routinely tested for EGFR-activating mutations. Clinical data on K-RAS mutations are inconsistent, therefore testing is not recommended. There is insufficient evidence to recommend other biomarker testing. No biomarkers to date reliably predict improved efficacy for anti-VEGF therapy. The authors concluded that outine assessment for EML4/ALK mutations is not recommended at present, although emerging data suggest that it may become valuable in the near future. \n",
      "MUC4\n",
      "Shanmugan and co-workers (2010) stated that mucin 4 (MUC4) is aberrantly expressed in colorectal adenocarcinomas (CRCs) but its prognostic value is unknown.  Archival tissue specimens collected from 132 CRC patients who underwent surgical resection without pre-surgery or pos-tsurgery therapy were evaluated for expression of MUC4 by using a mouse monoclonal antibody and horseradish peroxidase.  MUC4 expression levels were correlated with clinicopathologic features and patient survival.  Survival was estimated by both uni-variate Kaplan-Meier and multi-variate Cox regression methods.  In both normal colonic epithelium and CRCs, MUC4 staining was localized primarily in the cytoplasm.  The optimal immunostaining cut-off value (greater than or equal to 75 % positive cells and an immunostaining score greater than or equal to 2.0), which was derived by using the bootstrap method, was used to categorize CRCs into groups of high expression (33 of 132 patients; 25 %) or low expression (99 of 132 patients; 75 %).  Patients who had early stage tumors (stages I and II) with high MUC4 expression had a shorter disease-specific survival (log-rank; p = 0.007) than patients who had with low expression.  Patients who had advanced-stage CRCs (stages III and IV) did not demonstrate such a difference (log-rank; p = 0.108).  Multi-variate regression models that were generated separately for patients with early stage and advanced-stage CRC confirmed that increased expression of MUC4 was an independent indicator of a poor prognosis only for patients who had early stage CRCs (HR 3.77; 95 % CI: 1.46 to 9.73).  The authors stated that aftr validating these findings in larger retrospective and prospective studies, a stage-based anayses could establish the utility of MUC4 as a prognostic molecular marker of early stage CRC.\n",
      "ProOnc TumorSourceDx\n",
      "ProOnc TumorSourceDx test is designed to identify tissue or origin for metastastic tumor.  It identifies 25 possible classes of tissue origin corresponding to 17 distinct tissues and organs.  It requires only 48 microRNAs to identify tissue of origin based on microRNA expression levels.  However, there is insufficient evidence regarding its clinical value as tumor markers.\n",
      "SAA\n",
      "Cocco and associates (2010) examined the expression of serum amyloid A (SAA) in endometrial endometrioid carcinoma and evaluated its potential as a serum biomarker.  SAA gene and protein expression levels were evaluated in endometrial endometrioid carcinoma and normal endometrial tissues, by real-time PCR, IHC, and flow cytometry.  SAA concentration in 194 serum samples from 50 healthy women, 42 women with benign diseases, and 102 patients including 49 grade 1, 38 grade 2, and 15 grade 3 endometrial endometrioid carcinoma was also studied by a sensitive bead-based immunoassay.  SAA gene expression levels were significantly higher in endometrial endometrioid carcinoma when compared with normal endometrial tissues (mean copy number by real-time PCR = 182 versus 1.9; p = 0.001).  IHC revealed diffuse cytoplasmic SAA protein staining in poorly differentiated endometrial endometrioid carcinoma tissues.  High intra-cellular levels of SAA were identified in primary endometrial endometrioid carcinoma cell lines evaluated by flow cytometry, and SAA was found to be actively secreted in vitro.  SAA concentrations (microg/ml) had medians of 6.0 in normal healthy women and 6.0 in patients with benign disease (p = 0.92).  In contrast, SAA values in the serum of endometrial endometrioid carcinoma patients had a median of 23.7, significantly higher than those of the healthy group (p = 0.001) and benign group (p = 0.001).  Patients harboring G3 endometrial endometrioid carcinoma were found to have SAA concentrations significantly higher than those of G1/G2 patients.  The authors concluded that SAA is not only a liver-secreted protein, but is also an endometrial endometrioid carcinoma cell product.  SAA is expressed and actively secreted by G3 endometrial endometrioid carcinoma, and it is present in high concentration in the serum of endometrial endometrioid carcinoma patients.  SAA may represent a novel biomarker for endometrial endometrioid carcinoma to monitor disease recurrence and response to therapy.  They stated that additional studies are needed to validate these findings.\n",
      "Caris Target Now / Caris Molecular Profiling Service\n",
      "Molecular Intelligence Services (formerly Target Now Molecular Profiling Test) uses a multi-platform profiling approach including gene sequencing (NGS and Sanger), protein expression analysis (immunohistochemistry) and gene copy number analysis (chromogenic or fluorescence in situ hybridization [FISH]). The test has been used to examine tumor samples for underlying molecular alterations that may yield insights into potentially overlapping and different therapeutic options for individuals with these tumor types.\n",
      "According to the manufacturer, the Caris Life Sciences molecular profiling test, Caris Target Now, examines the genetic and molecular changes unique to a patient's tumor so that treatment options may be matched to the tumor's molecular profile. The manufacturer states that the Caris Target Now test is performed after a cancer diagnosis has been established and the patient has exhausted standard of care therapies or if questions in therapeutic management exist. Using tumor samples obtained from a biopsy, the tumor is examined to identify biomarkers that may have an influence on therapy. Using this information, Caris Target Now is intended to provide information on the drugs that will be more likely to produce a positive response. The manufacturer states that Caris Target Now can be used with any solid cancer such as lung cancer, breast cancer, and prostate cancer.\n",
      "There is insufficient evidence to support the use of Caris Target Now molecular profiling. A study (Von Hoff et al, 2010) compared the progression-free survival (PFS) of patients with refractory metastatic cancers using a treatment regimen selected by Caris Target Now molecular profiling of the patient's tumor with the PFS for the most recent regimen on which the patient had experienced progression. The investigators prespecified that a molecular profiling approach would be deemed of clinical benefit for the individual patient who had a PFS ratio (defined as a ratio of PFS on molecular profiling-selected therapy to PFS on prior therapy) of greater than or equal to 1.3. In 86 patients who had molecular profiling attempted, there was a molecular target detected in 84 (98 %). Sixty-six of the 84 patients were treated according to molecular profiling results.  Eighteen (27 %) of 66 patients had a PFS ratio of greater than or equal 1.3 (95 % CI:17 % to 38 %). Therefore, the null hypothesis (that less than or equal to 15 % of this patient population would have a PFS ratio of greater than or equal to 1.3) was rejected. The authors concluded that, in 27 % of patients, the molecular profiling approach resulted in a longer PFS on an molecular profiling-suggested regimen than on the regimen on which the patient had just experienced progression. An accompanying editorial (Doroshow, 2010) noted that the trial had a number of significant limitations, including uncertainty surrounding the achievement of time to progression (the studyâs primary endpoint), and a lack of a randomized design for this trial. \n",
      "A report by the National Horizon Scanning Centre (2013) stated that the company stated that the tumor profiling service provided by Caris Life Sciences has been extensively altered with the addition of several new technologies. The new service is named Caris Life Sciences Molecular Intelligence Services. The NHSC stated that randomized controlled trials comparing clinical outcomes for patients using Caris molecular profiling to those receiving standard specialist care are needed to determine whether this testing service is effective and cost-effective.\n",
      "CoA racemase (P504S) and HMWCK (34betaE12)\n",
      "Kumaresan et al (2010) reviewed 1034 cases of morphologically difficult prostate cancer, which were divided into benign (585), malignant (399) and suspicious (50) and evaluated using CoA racemase (P504S) and HMWCK (34betaE12).  Forty nine suspicious cases were resolved by using both markers whereas 1 case was resolved by further support with CD68.  The original diagnosis was changed in 15 of 50 suspicious cases from benign to malignant, one case from benign to high grade PIN, and in one case from malignant to benign.  The authors concluded that a combination of HMWCK and AMACR is of value in combating morphologically suspicious cases and that although the sensitivity and specificity of HMWCK and AMACR in this study were high, \"it should be used with caution, keeping in mind all their pitfalls and limitations.\"\n",
      "P504S\n",
      "Murray et al (2010) studied P504S expressing circulating prostate cells as a marker for prostate cancer.  The authors stated that PSA is the only biomarker routinely used in screening.  This study aimed to develop a system to test the presence of circulating prostate cells in men without a diagnosis of prostate cancer in realation with age, serum PSA levels and prostate biopsy by determining the co-expression of several markers such as CD82, HER-2 and matrix metalloproteinase 2 MMP-20.  The results indicated that among 409 men screened for prostate cancer 16.6% were positive for circulating prostate cells.  The authors concluded that the study of circulating prostate cells with various markers could be a useful complementary screening test for prostate cancer in men with increased PSA level.\n",
      "FLT3\n",
      "FLT3 has been used to predict prognosis in acute myelogenous leukemia (Chin, et al, 2006). Mutations in FLT3 are common in AML and have been associated with poorer survival in children and in younger adults with normal cytogenetics receiving intensive chemotherapy.\n",
      "The NCCN Task Force issued a report in November of 2011 which updated their position regarding molecular markers for diagnosis, prognosis, prediction, and companion diagnostic markers (Febbo et. al., 2011).  As a result of these recommendations, use of MGMT, IDH mutation and 1p/19q codeletion are now established for glioma.  Also, use of ALK gene fusion has been established for non-small cell lung cancer.  The updated NCCN guidelines have not yet established the efficacy of ColoPrint, CIMP, LINE-1 hypomethylation, or Immune cells for colon cancer.  Similarly, the efficacy of FLT3-TKD mutation, WT1 mutation, RUNX1 mutation, MLL-PTD, IDH1 mutation, IDH2 R172, and IDH2 codon 140 mutation has not been established for use in acute myeloid leukemia.\n",
      "ColoPrint\n",
      "ColoPrint (Agendia) is an 18-gene profile that classifies colon cancer into Low Risk or High Risk of relapse, by measuring genes representative of the metastatic pathways of colon cancer metastases which were selected for their predictive relationship to 5-year distant metastases probability (Raman, et al., 2013). ColoPrint is indicated for stage II colon cancer, and provides relapse risk stratification independent of clinical and pathologic factors such as T4-stage and MSI status. ColoPrint determines if the patient is a candidate for chemotherapy. An NCCN Task Force report (NCCN, 2011) concluded that the efficacy of ColoPrint has not been established.\n",
      "DecisionDx-UM\n",
      "The DecisionDx test is a gene expression profile that determines the molecular signature of a patient's melanoma. The results of the test provide knowledge regarding the risk of near term metastasis (5 years). Tumors with a Class 1 signature are associated with a good prognosis and a low potential to spread (or metastasize), while tumors with a Class 2 signature have a high potential to spread. \n",
      "Aaberg et al (2014) conducted a chart review and cross-sectional survey of ophthalmologists who treat uveal melanoma to assess current clinical practices for uveal melanoma (UM) and the impact of molecular prognostic testing on treatment decisions. This study involved a chart review of all Medicare beneficiaries tested by UM gene expression profile in 2012, conducted under an institutional review board-approved protocol. In addition, 109 ophthalmologists specializing in the treatment of UM were invited to participate in 24-question survey in 2012; 72 were invited to participate in a 23-question survey in 2014. The review of Medicare medical records included 191 evaluable patients, 88 (46%) with documented medical treatment actions or institutional policies related to surveillance plans. Of these 88, all gene expression profiling (GEP) Class 1 UM patients were treated with low-intensity surveillance. All GEP Class 2 UM patients were treated with high-intensity surveillance (P<0.0001 versus Class 1). There were 36 (19%) with information concerning referrals after initial diagnosis. Of these 36, all 23 Class 2 patients were referred to medical oncology; however, none of the 13 Class 1 patients were referred (P<0.0001 versus Class 1). Only Class 2 patients were recommended for adjunctive treatment regimens. 2012 survey: 50 respondents with an annual median of 35 new UM patients. The majority of respondents (82%) performed molecular analysis of UM tumors after fine needle biopsy (FNAB); median: 15 FNAB per year; 2014 survey: 35 respondents with an annual median of 30 new UM patients. The majority offered molecular analyses of UM tumor samples to most patients. Patients with lowmetastatic risk (disomy 3 or GEP Class 1) were generally assigned to less frequent (every 6 or 12 months) and less intensive clinical visits. Patients with high metastatic risk (monosomy 3 or GEP Class 2) were assigned to more frequent surveillance with hepatic imaging and liver function testing every 3-6 months. High-risk patients were considered more suitable for adjuvant treatment protocols.\n",
      "Chappell et al (2012) reported on a retrospective case series of uveal melanoma patients gene expression profiles to characterize the clinical spectrum of class 1 and class 2 uveal melanomas and their relationship with intraocular proton radiation response. A total of 197 uveal melanoma patients from a single institution were analyzed for pathology, clinical characteristics, and response to radiation therapy. A total of 126 patients (64%) had class 1 tumors and 71 (36%) had class 2 tumors. Patients with class 2 tumors had more advanced age (mean: 64 years vs 57 years; P = .001), had thicker initial mean ultrasound measurements (7.4 mm vs 5.9 mm; P = .0007), and were more likely to have epithelioid or mixed cells on cytopathology (66% vs 38%; P = .0004). Although mean pretreatment and posttreatment ultrasound thicknesses were significantly different between class 1 and class 2 tumors, there was no difference in the mean change in thickness 24 months after radiation therapy (mean difference: class 1 = -1.64 mm, class 2 = -1.47; P = .47) or in the overall rate of thickness change (slope: P = .64). Class 2 tumors were more likely to metastasize and cause death than class 1 tumors (disease-specific surviva [DSS]: P < .0001).\n",
      "Worley et al (2007) compared a gene expression-based classifier versus the standard genetic prognostic marker, monosomy 3, for predicting metastasis in uveal melanoma. Gene expression profiling, fluorescence in situ hybridization (FISH), and array comparative genomic hybridization (aCGH) were done on 67 primary uveal melanomas. Clinical and pathologic prognostic factors were also assessed. The investigators found that the gene expression-based molecular classifier assigned 27 tumors to class 1 (low risk) and 25 tumors to class 2 (high risk). By Cox univariate proportional hazards, class 2 signature (P = 0.0001), advanced patient age (P =0.01), and scleral invasion (P = 0.007) were the only variables significantly associated with metastasis. Only the class 2 signature was needed to optimize predictive accuracy in a Cox multivariate model. A less significant association with metastasis was observed for monosomy 3 detected by aCGH (P = 0.076) and FISH (P = 0.127). The sensitivity and specificity for the molecular classifier (84.6% and 92.9%, respectively) were superior to monosomy 3 detected by aCGH (58.3% and 85.7%, respectively) and FISH (50.0% and 72.7%, respectively). Positive and negative predictive values (91.7% and 86.7%, respectively) and positive and negative likelihood ratios (11.9 and 0.2, respectively) for the molecular classifier were also superior to those for monosomy 3.\n",
      "In a prospective case series study, CorrÃªa and Augsburger (2016) sought to determine whether any conventional clinical prognostic factors for metastasis from uveal melanoma retain prognostic significance in multivariate models incorporating gene expression profile (GEP) class of the tumor cells. The investigators conducted a single-institution study of GEP testing and other conventional prognostic factors for metastasis and metastatic death in 299 patients with posterior uveal melanoma evaluated by fine-needle aspiration biopsy (FNAB) at the time of or shortly prior to initial treatment. Univariate prognostic significance of all evaluated potential prognostic variables (patient age, largest linear basal diameter of tumor [LBD], tumor thickness, intraocular location of tumor, melanoma cytomorphologic subtype, and GEP class) was performed by comparison of Kaplan-Meier event rate curves and univariate Cox proportional hazards modeling.  Multivariate prognostic significance of combinations of significant prognostic factors identified by univariate analysis was performed using step-up and step-down Cox proportional hazards modeling. GEP class was the strongest prognostic factor for metastatic death in this series. However, tumor LBD, tumor thickness, and intraocular tumor location also proved to be significant individual prognostic factors in this study. On multivariate analysis, a 2-term model that incorporated GEP class and largest basal diameter was associated with strong independent significance of each of the factors.\n",
      "Correa and Augsburger (2014)  sought to determine the relative sufficiency of paired aspirates of posterior uveal melanomas obtained by FNAB for cytopathology and GEP, and their prognostic significance for predicting death from metastasis The investigators conducted a prospective non-randomized single-center study of 159 patients with posterior uveal melanoma sampled by FNAB in at least two tumor sites between September 2007 and December 2010. Cases were analyzed with regard to sufficiency of the obtained aspirates for cytopathologic classification and GEP classification. Statistical strength of associations between variables and GEP class was computed using Chi-square test. Cumulative actuarial survival curves of subgroups of these patients based on their cytopathologic versus GEP-assigned categories were computed by the Kaplan-Meier method. The endpoint for this survival analysis was death from metastatic uveal melanoma. FNAB aspirates were insufficient for cytopathologic classification in 34 of 159 cases (21.9 %). In contrast, FNAB aspirates were insufficient for GEP classification in only one of 159 cases (0.6 %). This difference is statistically significant (Pâ<â0.001). Six of 34 tumors (17.6 %) that yielded an insufficient aspirate for cytopathologic diagnosis were categorized as GEP class 2, while 43 of 125 tumors (34.7 %) that yielded a sufficient aspirate for cytopathologic diagnosis were categorized as GEP class 2. To date, 14 of the 49 patients with a GEP class 2 tumor (28.6 %) but only five of the 109 patients with a GEP class 1 tumor (5.6 %) have developed metastasis. Fifteen of 125 patients (12 %) whose tumors yielded sufficient aspirates for cytopathologic classification but only four of 34 patients (11.8 %) whose tumors yielded insufficient aspirates for cytopathologic classification developed metastasis. The median post-biopsy follow-up time for surviving patients in this series was 32.5 months. Cumulative actuarial 5-year probability of death from metastasis 14.1 % for those with an insufficient aspirate for cytopathologic classification versus 22.4 % for those with a sufficient aspirate for cytopathologic classification (log rank Pâ=â0.68). In contrast, the cumulative actuarial 5-year probability of metastatic death was 8.0 % for those with an insufficient/unsatisfactory aspirate for GEP classification or GEP class 1 tumor, versus 45.0 % for those with a GEP class 2 tumor (log rank Pâ=â0.005).\n",
      "In a prospective study, Oniken et al (2012) evaluated the prognostic performance of the DecisionDx 15 gene expression profiling (GEP) assay that assigns primary posterior uveal melanomas to prognostic subgroups: class 1 (low metastatic risk) and class 2 (high metastatic risk). A total of 459 patients with posterior uveal melanoma were enrolled from 12 independent centers. Tumors were classified by GEP as class 1 or class 2. The first 260 samples were also analyzed for chromosome 3 status using a single nucleotide polymorphism assay. Net reclassification improvement analysis was performed to compare the prognostic accuracy of GEP with the 7th edition clinical Tumor-Node-Metastasis (TNM) classification and chromosome 3 status. The investigators found that the GEP assay successfully classified 446 of 459 cases (97.2%). The GEP  was class 1 in 276 cases (61.9%) and class 2 in 170 cases (38.1%). Median follow-up was 17.4 months (mean, 18.0 months). Metastasis was detected in 3 class 1 cases (1.1%) and 44 class 2 cases (25.9%) (log-rank test, P<10(-14)). Although there was an association between GEP class 2 and monosomy 3 (Fisher exact test, P<0.0001), 54 of 260 tumors (20.8%) were discordant for GEP and chromosome 3 status, among which GEP demonstrated superior prognostic accuracy (log-rank test, P = 0.0001). By using multivariate Cox modeling, GEP class had a stronger independent association with metastasis than any other prognostic factor (P<0.0001). Chromosome 3 status did not contribute additional prognostic information that was independent of GEP (P = 0.2). At 3 years follow-up, the net reclassification improvement of GEP over TNM classification was 0.43 (P = 0.001) and 0.38 (P = 0.004) over chromosome 3 status.\n",
      "Klufas et al (2015) reported their experience with uveal melanoma (UM)-specific GEP testing on a series of choroidal metastatic tumors confirmed by cytopathology so that clinicians may be aware that receiving a class 1 or class 2 test result in non-melanoma is possible. These investigators performed a retrospective review of all cytopathology and DecisionDx-UM GEP reports between January 2012 to December 2014 from intra-operative FNA biopsy of choroidal tumors undergoing brachytherapy.  A total of 4 patients were identified to have cytopathology consistent with a non-melanoma primary.  All 4 patients presented with a unilateral, single choroidal tumor, which was treated with iodine-125 brachytherapy and underwent intra-operative FNA biopsy for cytopathology and UM-specific GEP testing for molecular prognostication.  Gene expression profile testing of the choroidal tumor in each patient revealed class 1A in 3 patients and class 2 in 1 patient.  The authors concluded that DecisionDx-UM GEP may be a helpful test for molecular prognostication in patients with UM; however, class 1 and class 2 test results are indeed possible in the setting of a non-melanoma malignancy.  They recommended that cytopathology and/or other melanoma-specific testing be performed in all cases of suspected choroidal melanoma because GEP with this assay is unable to rule out the diagnosis of a choroidal melanoma.\n",
      "Plasseuard et al (2016) sought to evaluate the clinical validity and utility of DecisionDx-UM. Beginning March 2010, 70 patients were enrolled in a prospective,multicenter, IRB-approved study to document patient management differences and clinical outcomes associated with low-risk Class 1 and high-risk Class 2 results indicated by DecisionDx-UM testing. Thirty-seven patients in the prospective study were Class 1 and 33 were Class 2. Class 1 patients had 100% 3-year metastasis-free survival compared to 63% for Class 2 (log rank test ð = 0.003) with 27.3 median follow-up months in this interim analysis. Class 2 patients received significantly higher-intensity monitoring and more oncology/clinical trial referrals compared to Class 1 patients (Fisherâs exact test ð = 2.1 Ã 10 â13 and ð = 0.04, respectively.). The investigators concluded that the results of this study provide additional, prospective evidence in an independent cohort of patients that Class 1 and Class 2 patients are managed according to the differential metastatic risk indicated by DecisionDx-UM.\n",
      "In a review of the management of ocular melanoma, Blum et al (2016) commented on the potential use of DecisionDx, noting that, although there is no clear survival benefit from earlier detection of metastatic disease, patients could benefit from clinical trial eligibility and palliative therapy with earlier detection.\n",
      "The United Kingdomâs national guidelines on \"Uveal melanoma\" (2015) recommended these molecular diagnostic tests be performed as part of a research protocol.\n",
      "Molecular Diagnostics for Thyroid Cancer\n",
      "Molecular markers associated with thyroid cancer have been proposed to assist in determining malignancy and to guide surgery decisions for individuals with indeterminate fine needle aspiration (FNA) thyroid nodule cytopathology.\n",
      "Thyroid nodules are abnormal growths or lumps that develop in the thyroid gland. While most are benign (not cancerous), a small percent are malignant (cancerous).9 To determine malignancy, fine needle aspiration (FNA) is used to obtain a specimen (aspirate) from the nodule which is evaluated by cytopathology and classified based on the results. Most are classified as benign (70 to 75%) and a small percentage as malignant (5% to 10%). Approximately 25% are classified as indeterminate (unable to determine a diagnosis) and warrant further evaluation, which often includes thyroid  surgery and histopathologic evaluation of thyroid tissue. However, nearly 80% of indeterminate nodules are benign based upon histopathology results.\n",
      "Thyroid gene expression classifier tests and thyroid cancer mutation analysis of fine needle aspirates in thyroid nodules differ from genetic testing. Genetic testing, also known as germline mutation testing, analyzes an individualâs DNA and can identify genetic mutations to determine inherited risk of disease. An individualâs germline DNA is constant and identical in all body tissue types. RNA activity is measured by gene expression analysis. It is dynamic and responds to cellular environmental signals. Mutation analysis of fine needle aspirates or tumor tissue determines DNA mutations that have been acquired over an individualâs lifetime. These DNA changes are present only in the tissue sampled, are not usually representative of an individualâs germline DNA and are not inheritable. For information regarding gene testing (also known as germline mutation testing) for thyroid cancer (eg, multiple endocrine neoplasia [MEN]), see CPB 0319 - RET Proto-Oncogene Testing.\n",
      "Guidelines on thyroid carcinoma from the National Comprehensive Cancer Network (NCCN, 2014) state: \"Molecular diagnostic testing to detect individual mutations (e.g., BRAF, RET/PTC, RAS, PAX8/PPAR [peroxisome proliferator-activated receptors] gamma) or pattern recognition approaches using molecular classifiers may be useful in the evaluation of FNA samples that are indeterminate to assist in management decisions. The choice of the precise molecular test depends on the cytology and the clinical question being asked.\" Indeterminate groups include: 1) follicular or Hurthle cell neoplasms; and 2) AUS/FLUS. The NCCN Panel recommends (category 2B) molecular diagnostic testing for evaluating FNA results that are suspicious for: 1) follicular or Hurthle cell neoplasms; or 2) AUS/FLUS (see Nodule Evaluation in the NCCN Guidelines for Thyroid Carcinoma). For the 2014 update, the NCCN Panel revised the recommendation for molecular diagnostic testing from category 2A to category 2B for indeterminate FNA results based on a series of panel votes. The panel noted that the molecular testing (both the Gene Expression Classifier and the individual mutation analysis) was available in the majority of NCCN Member Institutions (>75%). About 70% of the panelists would recommend using a gene expression classifier in the evaluation of follicular lesions. The gene expression classifier measures the expression of at least 140 genes. BRAF mutation analysis was recommended by 50% of the panelists in the evaluation of thyroid nodules (not restricted to the follicular lesions). Furthermore, about 60% of the panelists would recommend BRAF testing in the evaluation of follicular lesions. A minority of panelists expressed concern regarding observation of follicular lesions because they were perceived as potentially pre-malignant lesions with a very low, but unknown, malignant potential if not surgically resected (leading to a recommendation for either observation or definitive surgical resection in lesions classified as benign by molecular testing). Rather than proceeding to immediate surgical resection to obtain a definitive diagnosis for these intermediate FNA cytology groups (follicular lesions), patients can be followed with observation if the application of a specific molecular diagnostic tests results in a predicted risk of malignancy that is comparable to the rate seen in cytologically benign thyroid FNAs (approximately < 5%). NCCN guidelines state that it is important to note that the predictive value of molecular diagnostics may be significantly influenced by the pre-test probability of disease associated with the various FNA cytology groups. Furthermore, in the cytologically indeterminate groups, the risk of malignancy for FNA can vary widely between institutions. Because the published studies have focused primarily on adult patients with thyroid nodules, the diagnostic utility of molecular diagnostics in pediatric patients remains to be defined. Therefore, proper implementation of molecular diagnostics into clinical care requires an understanding of both the performance characteristics of the specific molecular test and its clinical meaning across a range of pre-test disease probabilities.\n",
      "For support for use of a gene classifier, the NCCN guidelines reference validation studies of the Afirma Thyroid FNA Analysis (Alexander et al, 2012; Chudova et al, 2010; Kloos, et al, 2013; McIver et al, 2014) and Thyroseq (Nikiforov et al, 2009; Ohori et al, 2010; Nikiforov et al, 2011). These studies demonstrate that this molecular diagnostic meets NCCN threshold of predicting malignancy of 5 % or less (i.e., a negative predictive value of 95 %), allowing physicians to observe an indeterminate thyroid nodule in lieu of surgery.\n",
      "Guidelines from the American Thyroid Association (2015) state that, \"if molecular testing is being considered, patients should be counseled regarding the potential benefits and limitations of testing and about the possible uncertainties in the therapeutic and long-term clinical implications of results. This is a strong recommendation, based upon low quality evidence. The guidelines state that the largest studies of preoperative molecular markers in patients with indeterminate FNA cytology have respectively evaluated a seven-gene panel of genetic mutations and rearrangements (BRAF, RAS, RET/PTC, PAX8/PPARc), a gene expression classifier (167 GEC; mRNA expression of 167 genes), and galectin-3 immunohistochemistry (cell blocks). The guidelines note that these respective studies have been subject to various degrees of blinding of outcome assessment. The guidelines state that \"there is currently no single optimal molecular test that can definitively rule in or rule out malignancy in all cases of indeterminate cytology, and long-term outcome data proving clinical utility are needed.\"\n",
      "Guidelines from the American Association of Clinical Endocrinologists (Gharib et al, 2016) state that molecular testing should be considered to complement not replace cytologic evaluation, where the results are expected to influence clinical management. As a general rule, not recommended in nodules with established benign or malignant cytologic characteristics. The guidelines recommend considering the detection of BRAF and RET/PTC and, possibly, PAX8/PPARG and RAS mutations if such detection is available. The guidelines state that, because of the insufficient evidence and the limited follow-up, they do not recommend either in favor of or against the use of gene expression classifiers (GECs) for cytologically indeterminate nodules.\n",
      "Thyroid gene expression classifier (GEC) (e.g., Afirma Gene Expression Classifier) is a messenger ribonucleic acid (mRNA) gene expression assay that analyzes thyroid FNA specimens to classify indeterminate nodules using a proprietary algorithm. The thyroid GEC analyzes the mRNA expression of 167 genets. The thyroid GEC is described as a \"rule-out\" test because a negative (ie, benign) result rules out the presence of cancer.\n",
      "Thyroid malignancy classifier tests (eg, Afirma MTC, Afirma BRAF) purport to analyze thyroid nodules that have been classified as \"malignant\" or \"suspicious for malignancy\" through cytopathology or as \"suspicious for malignancy\" on GEC. Afirma MTC was developed to identify the presence of medullary thyroid cancer (MTC) while Afirma BRAF was designed to determine the presence of BRAF V600E mutation. Both are automatically initiated in the lab, if ordered by a physician, after a thyroid nodule has been classified as having abnormal findings. This process is referred to as reflex testing.\n",
      "Thyroid cancer targeted mutational analysis of thyroid FNA samples has also been proposed to detect individual gene mutations associated with thyroid cancer and include BRAF V600E, RAS (HRAS, KRAS, NRAS), RET/PTC, PAX8/PPARgamma, PIK3CA.\n",
      "On May 4, 2018, the FDA approved Tafinlar (dabrafenib) and Mekinist (trametinib), administered together, for the treatment of anaplastic thyroid cancer (ATC) that cannot be removed by surgery or has spread to other parts of the body (metastatic), and has a type of abnormal gene, BRAF V600E (BRAF V600E mutation-positive).\n",
      "Thyroid cancer mutational panel using next generation sequencing (NGS) (eg, ThyroSeq) analyzes deoxyribonucleic acid (DNA) and RNA in FNA thyroid samples to determine the presence of gene mutations associated with thyroid cancer.\n",
      "Quest Diagnostics offers a molecular test panel designed to help physicians determine if a thyroid gland is cancerous and requires surgical removal. The test includes the seven gene panel of mutations and rearrangements addressed by the American Thyroid Association for the clinical management of indeterminate thyroid biopsies. According to Quest Diagnostics, the Quest Diagnostics Thyroid Cancer Mutation Panel aids in detecting cancer in thyroid biopsies which are found to be indeterminate for cancer by current cytology test methods. Approximately 15% to 20% of these biopsies, which are collected by fine needle aspiration (FNA), produce indeterminate results. An unclear result may increase the risk that a physician, in an abundance of caution, will biopsy additional tissue using a larger needle or surgically remove part or all of a thyroid suspected of having cancer that is later diagnosed as healthy. About 300,000 thyroid FNA biopsy procedures are performed annually in the United States. The panel identifies mutations of the molecular markers BRAF V600E, RAS, RET/PTC, and PAX8/PPAR gamma, which are associated with papillary and follicular thyroid cancer, two common forms of the disease. The manufacturer states that practice guidelines from the American Thyroid Association recommend that physicians consider these markers as aids in clinical management of patients with indeterminate biopsy test results. Results of a Quest Diagnostics study found that 90 of 149 FNA specimens, or about 60%, had mutations of one or more of the four markers tested by the new panel (Reitz, et al., 2014). The authors of the study stated that the presence of the four markers was generally mutually exclusive, suggesting potential value in a hierarchical screening strategy for samples with limited tissue. According to the American Cancer Society, about two tests in every 10 may need to be repeated because the sample does not contain enough cells for testing.\n",
      "Thyroid cancer mutational panel (e.g., ThyGenX [formerly miRInform], Thyroid Cancer Mutational Panel) evaluates thyroid FNA samples to detect gene mutations associated with thyroid cancer (eg, BRAF V600E, RAS [HRAS, KRAS, NRAS], RET/PTC, PAX8/ PPARgamma, PIK3CA). Mutational panels are described as \"rule-in\" tests because a positive result indicates that a nodule is at high risk for malignancy; therefore, identifies or, rules in, cancer. However, because these mutations occur infrequently overall in thyroid cancer, a negative result does not rule out cancer. \n",
      "Thyroid microRNA (miRNA) GEC (e.g., RosettaGX Reveal, ThyraMIR) measures the expression levels of microRNAs to supposedly classify thyroid nodules with indeterminate FNA cytopathology. miRNA GEC may be offered alone or in combination with a thyroid cancer mutational panel (ie, ThyraMIR and ThyGenX ) to purportedly enhance specificity and sensitivity testing results.\n",
      "ThyraMIR thyroid miRNA classifier is a PCR-based microRNA (miRNA) gene expression classifier that examines the expression levels of 10 miRNA genes within FNA biopsy: miR-29-1-5p, miR-31-5p, miR-138-1-3p, miR-139-5p, miR-146b-5p, miR-155. miR-204-5p, miR-222-3p, miR-375, and miR-551b-3p. It is performed following a negative ThyGenX result for all mutations or when mutations detected are not fully indicative of malignancy (i.e., ThyGenX results which favor a benign nodule but cancer could still be present).  The test is used on the same FNA cytology sample.  The ThyraMIR test reports a qualitative positive or negative result based on the gene expression levels. \n",
      "A study combining seven-gene mutational testing (ThyGenX) with expression of a set of 10 miRNA genes (ThyraMIR) on preoperative FNA sampling from 109 patients with indeterminate cytology, showed 89% sensitivity, 85% specificity, with a 73% PPV and 94% NPV on this group with a 32% prevalence of malignancy (Labourier, et al., 2015). Labourier, et al. (2015) reported that testing with ThyGenX and ThyraMIR for DNA, mRNA, and miRNA can accurately classify benign and malignant thyroid nodules, increase the diagnostic yield, and futher improve the preoperative risk-based management of benign thyroid nodules with indeterminate cytology. Labourier, et al. (2015) tested surgical specimens and preoperative FNAs (n = 638) for 17 validated gene alterations using the miRInform Thyroid test (ThyGenX) and with a 10-miRNA gene expression classifier (ThyraMIR) generating positive (malignant) or negative (benign) results. Cross-sectional sampling of thyroid nodules with atypia of undetermined significance/follicular lesion of undetermined significance (AUS/FLUS) or follicular neoplasm/suspicious for a follicular neoplasm (FN/SFN) cytology (n = 109) was conducted at 12 endocrinology centers across the United States. Qualitative molecular results were compared with surgical histopathology to determine diagnostic performance and model clinical effect. Mutations were detected in 69% of nodules with malignant outcome. Among  mutation-negative specimens, miRNA testing correctly identified 64% of malignant  cases and 98% of benign cases. The diagnostic sensitivity and specificity of the combined algorithm was 89% (95% confidence interval [CI], 73-97%) and 85% (95% CI, 75-92%), respectively. At 32% cancer prevalence, 61% of the molecular results were benign with a negative predictive value of 94% (95% CI, 85-98%). Independently of variations in cancer prevalence, the test increased the yield of true benign results by 65% relative to mRNA-based gene expression classification and decreased the rate of avoidable diagnostic surgeries by 69%. \n",
      "In a news article on tests for indeterminate thyroid nodules, Tucker (2015) reviewed the data from Labourier, et al. (20150 plus additional abstracts on ThyGenX/ThyraMIR presented at the American Association of Clinical Endocrinologists' 2015 Annual Scientific and Clinical Congress, The article quoted AACE immediate past president R Mack Harrell, MD, as cautioning that \"more validation is needed in real-world settings with larger numbers for this new platform, as well as for other 'next-generation' molecular tests such as the ThyroSeq â¦. So much of the predictive value of these tests depends on what you start with. If you start with a highly selected tertiary-care-referral cancer community, the efficacy of the test is completely different from starting with a practice that's receiving every thyroid nodule in town.  So, it needs to be tested in a true community-practice-type setting with lots of patients before you can be sure exactly how it's going to perform\".\n",
      "Lithwick-Yanai et al (2017) sought to develop an assay, the RosettaGX Reveal, that could classify indeterminate thyroid nodules as benign or suspicious, using routinely prepared fine needle aspirate (FNA) cytology smears. A training set of 375 FNA smears was used to develop the microRNA-based assay, which was validated using a blinded, multicenter, retrospective cohort of 201 smears. Final diagnosis of the validation samples was determined based on corresponding surgical specimens, reviewed by the contributing institute pathologist and two independent pathologists. Validation samples were from adult patients (â¥18â",
      "years) with nodule size >0.5â",
      "cm, and a final diagnosis confirmed by at least one of the two blinded, independent pathologists. The developed assay differentiates benign from malignant thyroid nodules, using quantitative RT-PCR. Test performance on the 189 samples that passed quality control: negative predictive value: 91% (95% CI 84% to 96%); sensitivity: 85% (CI 74% to 93%); specificity: 72% (CI 63% to 79%). Performance for cases in which all three reviewing pathologists were in agreement regarding the final diagnosis (n=150): negative predictive value: 99% (CI 94% to 100%); sensitivity: 98% (CI 87% to 100%); specificity: 78% (CI 69% to 85%). The auhors concluded that this assay utilizing microRNA expression in cytology smears distinguishes benign from malignant thyroid nodules using a single FNA stained smear, and does not require fresh tissue or special collection and shipment conditions. The authors stated that this assay offers a valuable tool for the preoperative classification of thyroid samples with indeterminate cytology. Limitations of this study include its small size and large number of post hoc exclusions to create a set in which all three pathologists were in agreement.\n",
      "Benjamin et al (2016) reported on the analytical validation of the RosettaGX Reveal assay. More than 800 FNA slides were tested, including slides stained with Romanowsky-type and Papanicolaou stains. The assay was examined for the following features: intranodule concordance, effect of stain type, minimal acceptable RNA amounts, performance on low numbers of thyroid cells, effect of time since sampling, and analytical sensitivity, specificity, and reproducibility. The authors reported that the assay can be run on FNA slides for which as little as 1% of the cells are thyroid epithelial cells or from which only 5 ng of RNA have been extracted. Samples composed entirely of blood failed quality control and were not classified. Stain type did not affect performance. All slides were stored at room temperature. However, the length of time between FNA sampling and processing did not affect assay performance. There was a high level of concordance between laboratories (96%), and the concordance for slides created from the same FNA pass was 93%. The authors concluded that the microRNA-based assay was robust to various physical processing conditions and to differing sample characteristics. The authors concluded that given the assay's performance, robustness, and use of routinely prepared FNA slides, it has the potential to provide valuable aid for physicians in the diagnosis of thyroid nodules.\n",
      "Bhatia et al (2015) noted that FNA cytology, being the mainstay to diagnose thyroid nodules, does not provide definitive results in a subset of patients. The use of molecular markers testing has been described as a useful aid in differentiation of thyroid nodules that present with an indeterminate cytodiagnosis.  Molecular tests, such as the Afirma gene classifier, mutational assay and immunohistochemical markers have been increasingly used to further increase the accuracy and defer unnecessary surgeries for benign thyroid nodules.  However, in light of the current literature, their emerging roles in clinical practice are limited due to financial and technical limitations.  Nevertheless, their synergistic implementation can predict the risk of malignancy and yield an accurate diagnosis.  This review discussed the clinical utility of various molecular tests done on FNA indeterminate nodules to avoid diagnostic thyroidectomies and warrant the need of future multi-Institutional studies.\n",
      "Cibas and Ali (2017) noted that the Bethesda System for Reporting Thyroid Cytopathology (TBSRTC) established a standardized, category-based reporting system for thyroid fine-needle aspiration (FNA) specimens.  The 2017 revision re-affirmed that every thyroid FNA report should begin with 1 of the 6 diagnostic categories, the names of which remain unchanged since they were first introduced:\n",
      "\n",
      "Non-diagnostic or unsatisfactory\n",
      "Benign\n",
      "Atypia of undetermined significance (AUS) or follicular lesion of undetermined significance (FLUS)\n",
      "Follicular neoplasm or suspicious for a follicular neoplasm;\n",
      "Suspicious for malignancy\n",
      "Malignant.\n",
      "\n",
      "There is a choice of 2 different names for some of the categories.  A laboratory should choose the one it prefers and use it exclusively for that category.  Synonymous terms (e.g., AUS and FLUS) should not be used to denote 2 distinct interpretations.  Each category has an implied cancer risk that ranges from 0 % to 3 % for the \"benign\" category to virtually 100 % for the \"malignant\" category, and, in the 2017 revision, the malignancy risks have been updated based on new (post 2010) data.  As a function of their risk associations, each category is linked to updated, evidence-based clinical management recommendations.  The recent re-classification of some thyroid neoplasms as non-invasive follicular thyroid neoplasm with papillary-like nuclear features (NIFTP) has implications for the risk of malignancy, and this is accounted for with regard to diagnostic criteria and optional notes.  Such notes can be useful in helping guide surgical management.\n",
      "ROMA\n",
      "Risk of Ovarian Malignancy Algorithm (ROMA) is a blood test cleared by the FDA to aid in the evaluation of pelvic masses for the likelihood of malignancy before surgery. ROMA measures human epididymis protein 4 (HE4) and CA-125. These measurements are applied to an algorithm, combined with menopausal status, to calculate a numerical score.\n",
      "The BCBS TECâs assessment on \"Multi-Analyte Testing for the Evaluation of Adnexal Masses\" (2013) concluded that ROMA does not meet TEC criteria.  It noted that \"evidence regarding the effect of ... ROMA and effects on health outcomes is indirect, and based on studies of diagnostic performance of the tests in patients undergoing surgery for adnexal masses.  Although the studies show improvements in sensitivity and worsening of specificity with the use of the tests in conjunction with clinical assessment, there are problems in concluding that this results in improved health outcomes. The clinical assessment performed in the studies is not well characterized... ROMA does not improve the sensitivity of testing to a great extent. Underlying these issues is some uncertainty regarding the benefit of initial treatment by a gynecologic oncologist beyond the need for reoperation is some cases\".\n",
      "Guidelines on ovarian cancer from the National Comprehensive Cancer Network (NCCN, 2016) state that \"it has been suggested that specific biomarkers (serum HE4 and CA-125) along with an algorithm (Risk of Ovarian Malignancy Algorithm [ROMA]) may be useful for determining whether a pelvic mass is malignant or benign. The FDA has approved the use of HE4 and CA-125 for estimating the risk for ovarian cancer in women with a pelvic mass. Currently, the NCCN Panel does not recommend the use of these biomarkers for determining the status of an undiagnosed pelvic mass.\" \n",
      "Guidelines on management of adnexal masses from the American College of Obstetricians and Gynecologists (ACOG, 2017) state that ROMA includes HE4, which has been found to be more sensitive and specific than CA 125 for the evaluation of adnexal masses. The guidelines state that serum biomarker panels [OVA1 and ROMA] may be used as an alternative to CA 125 alone in determining the need for referral to or consultation with a gynecological oncologist when an adenexal mass requires surgery. The guidelines state that trials that have evaluated the predictive value of these panels show potential for improved specificity; \"[h]owever, comparative research has not yet defined the best testing approach.\"\n",
      "The UK National Institute for Health Research Health Technology Assessment Programme has commissioned an assessment (Westwood, et al., 2016) comparing the Risk of Malignancy Index (RMI) to alternative risk scores for ovarian cancer, including the ROMA score, as well as Overa/OVA2 (Vermillion), simple rules ultrasound classification system (IOTA), Assessment of Different NEoplasias in the adnexa (ADNEX) model (IOTA group). The assessment is scheduled to be completed in 2017.\n",
      "ERCC1\n",
      "Yu and colleagues (2012) stated that the excision repair cross-complementation group 1 (ERCC1) plays an essential role in DNA repair and has been linked to resistance to platinum-based anticancer drugs among advanced NSCLC patients.  These investigators examined if ERCC1 Asn118Asn and C8092A genetic variants are associated with treatment response of platinum chemotherapy.  They performed a meta-analysis using 10 eligible cohort studies (including 11 datasets) with a total of 1,252 NSCLC patients to summarize the existing data on the association between the ERCC1 Asn118Asn and C8092A polymorphisms and response to platinum regiments.  Odds ratio or hazard ratio with 95 % CI were calculated to estimate the correlation.  These researchers found that neither ERCC1 C8092A polymorphism nor Asn118Asn variant is associated with different response of platinum-based treatment among advanced NSCLC patients.  Additionally, these 2 genetic variants are not related to treatment response in either Caucasian patients or Asian patients.  The authors concluded that the findings of this meta-analysis indicated that the ERCC1 Asn118Asn and C8092A polymorphisms may not be good prognostic biomarkers for platinum-based chemotherapy in patients with stage III-IV NSCLC.\n",
      "Wang et al (2012) performed a meta-analysis by using 20 eligible studies to examine polymorphisms of ERCC1, GSTs, TS and MTHFR in predicting clinical outcomes (response rate, OS and toxicity) of gastric cancer (GC) patients treated with platinum/5-Fu-based chemotherapy.  The association was measured using random/fixed effect odds ratios (ORs) or hazard ratios (HRs) combined with their 95 % CIs according to the studies' heterogeneity.  Statistical analysis was performed with the software STATA 9.0 package.  No significant association was found between response rate and genetic polymorphism in TS, MTHFR, ERCC1, GSTM1 and GSTP1.  However, response rate was higher in GSTT1 (+) genotype compared with GSTT1 (-) genotype (T-/T+: OR = 0.67, 95 % CI: 0.47 to 0.97).  With regard to long-term outcomes, these researchers observed a significant longer OS in TS 3R/3R [(2R2R+2R3R)/3R3R: HR = 1.29, 95 % CI: 1.02 to 1.64] and GSTP1 GG/GA [(GG+AG)/AA: HR = 0.51, 95 % CI: 0.39 to 0.67] genotypes.  In addition, significant association was demonstrated between toxicity and genetic polymorphism in TS, MTHFR and GSTP1 in included studies.  The authors concluded that polymorphisms of ERCC1, GSTs, TS and MTHFR were closely associated with clinical outcomes of GC patients treated with platinum/5-Fu-based chemotherapy.  Moreover, they state that studies with large sample size using the method of multi-variant analyses may help us to give more persuasive data on the putative association in future.\n",
      "In a meta-analysis, Gong and colleagues (2012) examined if RRM1 expression is associated with the clinical outcome of gemcitabine-containing regimen in advanced NSCLC.  An electronic search was conducted using the databases PubMed, Medline, EMBASE, Cochrane library and CNKI, from inception to May, 2011.  A systemic review of the studies on the association between RRM1 expression in advanced NSCLC and clinical outcome of gemcitabine-containing regimen was performed.  Pooled odds ratios (OR) for the response rate, weighted median survival and time to progression were calculated using the software Revman 5.0.  The search strategy identified 18 eligible studies (n = 1,243).  Response rate to gemcitabine-containing regimen was significantly higher in patients with low/negative RRM1 (OR = 0.31, 95 % CI: 0.21 to 0.45, p < 0.00001).  Non-small cell lung cancer SCLC patients with low/negative RRM1 who were treated with gemicitabine-containing regimen survived 3.94 months longer (95 % CI: 2.15 to 5.73, p < 0.0001) and had longer time to progression for 2.64 months (95 % CI: 0.39 to 4.89, p = 0.02) than those with high/positive RRM1.  The authors concluded that low/negative RRM1 expression in advanced NSCLC was associated with higher response rate to gemcitabine-containing regimen and better prognosis.  Moreover, they stated that large phase III randomized trials are needed to identify whether RRM1 detection is clinically valuable for predicting the prognosis and sensitivity to gemcitabine-containing regimen in advanced NSCLC.\n",
      "Friboulet et al (2013) stated that the ERCC1 protein is a potential prognostic biomarker of the effectiveness of cisplatin-based chemotherapy in NSCLC.  Although several ongoing trials are evaluating the level of expression of ERCC1, no consensus has been reached regarding a method for evaluation.\n",
      "Besse et al (2013) noted that somatic ERCC1 and ribonucleotide reductase M1 (RRM1) expression levels have been extensively explored as markers of DNA repair capacity in tumor cells.  Although low ERCC1 and/or RRM1 expression is generally associated with sensitivity to platinum, the results published in retrospective and prospective studies are not always consistent.  These researchers examined the function of these 2 biomarkers as well as the tools available for their assessment and the associated technical issues.  Their prognostic and predictive values were summarized and considered in terms of customizing systemic therapy according to biomarker (ERCC1 and RRM1) expression levels.  The authors discussed why the use of both markers should at this point be restricted to clinical research.\n",
      "GSTP1\n",
      "Prostate cancer is the most common cancer in men and the second leading cause of cancer-related deaths in the US. Due to the reportedly high false-negative rate of initial biopsy results after elevated PSA level, new approaches for improved detection in prostate cancer are needed. Several studies have shown that hypermethylation of the promoter regions of the GST-P1 and APC genes occurs at a significantly higher frequency in prostate cancer samples than in benign conditions of the prostate gland. Hypermethylation of the promoter regions of the GST-P1 and APC genes can aid in prognosticating for prostate cancer (Raman, et al., 2013).\n",
      "Trock et al (2012) noted that hyper-methylation of genes such as glutathione-S-transferase P1 (GSTP1) and adenomatous polyposis coli (APC) occurs with high frequency in prostate tumor tissue but is much less common in the benign prostate; however, the potential value of gene methylation biomarkers as an adjunct to biopsy histopathology has had little study.  When measured in histologically benign prostate biopsy tissue, APC gene hyper-methylation was found to have high negative-predictive value (NPV) and high sensitivity.  GSTP1 hype-rmethylation was found to have lower performance than APC.  These investigators evaluated the performance of DNA methylation biomarkers in the setting of repeat biopsy in men with an initially negative prostate biopsy but a high index of suspicion for missed prostate cancer.  They prospectively evaluated 86 men with an initial histologically negative prostate biopsy and high-risk features.  All men underwent repeat 12-core ultrasonography-guided biopsy.  DNA methylation of GSTP1 and APC was determined using tissue from the initially negative biopsy and compared with histology of the repeat biopsy.  The primary outcome was the relative NPV of APC compared with GSTP1, and its 95 % CI.  On repeat biopsy, 21/86 (24 %) men had prostate cancer.  APC and GSTP1 methylation ratios below the threshold (predicting no cancer) produced a NPV of 0.96 and 0.80, respectively.  The relative NPV was 1.2 (95 % CI: 1.06 to 1.36), indicating APC has significantly higher NPV.  Methylation ratios above the threshold yielded a sensitivity of 0.95 for APC and 0.43 for GSTP1.  Combining both methylation markers produced a performance similar to that of APC alone.  APC methylation patterns were consistent with a possible field effect or occurrence early in carcinogenesis.  The authors concluded that APC methylation provided a very high NPV with a low percentage of false-negatives, in the first prospective study to evaluate performance of DNA methylation markers in a clinical cohort of men undergoing repeat biopsy.  They stated that the potential of APC methylation to reduce unnecessary repeat biopsies warrants validation in a larger prospective cohort.\n",
      "In a systematic review and meta-analysis, Yu and colleagues (2013) examined the association between GSTP1 Ile105Val polymorphism and prostate cancer (PCa) in different inheritance models.  A total of 13 eligible studies were pooled into this meta-analysis.  There was significant association between the GSTP1 Ile158Val variant genotypes and PCa for Ile/Ile versus Val/Val comparison [odds ratio (OR)â=â0.705; I 2â=â63.7 %; 95 % CI:â0.508 to 0.977], Ile/Val versus Val/Val comparison (ORâ=â0.736; I 2â=â8.0 %; 95 % CI:â0.613 to 0.883), and dominant model (ORâ=â0.712; I 2â=â45.5 %; 95 % CI:â0.555 to 0.913).  However, no associations were detected for other genetic models.  In the stratified analysis by ethnicity, significant associations between GSTP1 Ile105Val polymorphism and PCa risk were also found among Caucasians (Ile/Ile versus Val/Val comparison ORâ=â0.818, I 2â=â0.0 %, 95 % CI: 0.681 to 0.982; Ile/Val versus Val/Val comparison ORâ=â0.779, I 2â=â0.0 %, 95 % CI:â0.651 to 0.933; and dominant model ORâ=â0.794, I 2â=â0.0 %, 95 % CI:â0.670 to 0.941), while there were no associations found for other genetic models.  However, no associations were found in Asians and African-Americans for all genetic models when stratified by ethnicity. The authors concluded that the findings of this meta-analysis indicated that GSTP1 Ile105Val polymorphisms contributed to the PCa susceptibility.  However, they stated that a study with the larger sample size is needed to further evaluate gene-environment interaction on GSTP1 Ile105Val polymorphisms and PCa risk.\n",
      "An assessment by the Swedish Office of Heath Technology Assessment (SBU, 2011) concluded that the scientific evidence is insufficient to determine the diagnostic accuracy of the me-GSTP1 urine test.\n",
      "CEACAM6\n",
      "An UpToDate review on \"Screening for breast cancer\" (Fletcher, 2013) does NOT mention the use of carcinoembryonic antigen cell adhesion molecule 6 (CEACAM6).  Also, the NCCN's clinical practice guideline on \"Breast Cancer\" (Version 2.2013) does NOT mention the use of carcinoembryonic antigen cell adhesion molecule 6 (CEACAM6).\n",
      "DCIS Recurrence Score\n",
      "An UpToDate review on \"Ductal carcinoma in situ: Treatment and prognosis\" (Collins et al, 2013) states that \"A DCIS Recurrence Score utilizing a multigene assay has been developed and a prospective evaluation of this assay was performed using tumors from 327 patients who participated in the aforementioned E5194 trial.  In a preliminary analysis, patients were stratified by recurrence score into three groups that were associated the following risks of an ipsilateral breast event (DCIS or invasive breast cancer) or invasive breast cancer:\n",
      "\n",
      "Low (less than 39) -- 12 and 5 %, respectively\n",
      "Intermediate (39 to 54) -- 25 and 9 %, respectively\n",
      "High (greater than or equal to 55) -- 27 and 19 %, respectively\n",
      "\n",
      "These results suggest that the DCIS score may help select patients who should undergo adjuvant radiation.  However, further validation of these results is required before the multigene assay can become a part of clinical practice.  It is also worth noting that a 12 percent risk of an ipsilateral breast event at 10 years in the lowest risk category may not be low enough to justify the routine omission of post-excision RT\".\n",
      "MyPRS\n",
      "MyPRS Plus (Signal Genetics) analyzes all of the nearly 25,000 genes in a patientâs genome to determine the gene expression profile that is associated with their condition (Raman, et al., 2013). In the case of myeloma, the gene expression profile is made up of the 70 most relevant genes which aid in the prediction of the patientâs outcome. MyPRS helps patients and physicians determine the best treatment for patients with Myeloma.\n",
      "The NCCNâs clinical practice guideline on multiple myeloma (MM) (Version 2.2013) stated that \"Further understanding of the molecular subtypes of MM is emerging from the application of high-throughput genomic tools such as gene expression profiling (GEP).  With the currently available novel treatment approaches, a majority of patients with MM can now anticipate long-term disease control.  However, patients with cytogenetically and molecularly defined high-risk disease do not receive the same benefit from current approaches as low-risk patients.  GEP is a powerful and fast tool with the potential to provide additional prognostic value to further define risk-stratification, help therapeutic decisions, and inform novel drug design and development.  At the present time, standardized testing for GEP is not available and there is inadequate data to determine how this prognostic information should be used to direct patient management\".  The NCCN guideline does not include a specific recommendation for the use of the MyPRS test in risk-stratification or determining prognosis in the clinical management of patients with MM.\n",
      "Oxnard et al (2013) stated that the identification of oncogenic driver mutations underlying sensitivity to EGFR and anaplastic lymphoma kinase tyrosine kinase inhibitors has led to a surge of interest in identifying additional targetable oncogenes in NSCLC.  A number of new potentially oncogenic gene alterations have been characterized in recent years, including BRAF mutations, HER2 insertions, phosphatidylinositol-4,5-bisphosphonate 3-kinase, catalytic subunit alpha polypeptide gene (PIK3CA) mutations, fibroblast growth factor receptor 1 (FGFR1) amplifications, discoidin domain receptor 2 (DDR2) mutations, ROS1 re-arrangements, and RET re-arrangements.  These investigators discussed the techniques used to discover each of these candidate oncogenes, the prevalence of each in NSCLC, the pre-clinical data supporting their role in lung cancer, and data on small molecular inhibitors in development.\n",
      "NRAS\n",
      "Janku et al (2013) noted that despite development of new therapies, metastatic colorectal cancer (mCRC) largely remains an incurable disease.  Approximately 2 to 6 % of colorectal cancers harbor NRAS mutations.  The anti-VEGF antibody bevacizumab is a backbone of most therapeutic regimens; however, biomarkers predicting its activity are not known.  These investigators reported 2 cases of mCRC with a Q61K NRAS mutation that had a favorable response to bevacizumab and the histone deacetylase inhibitor valproic acid.  In contrast, none of 10 patients with wild-type NRAS or unknown NRAS status and mutated KRAS (NRAS and KRAS mutations are mutually exclusive) responded to the same regimen.  The authors concluded that these results suggested that NRAS mutation merits further investigation as a potential biomarker predicting the efficacy of bevacizumab-based treatment.\n",
      "The EGAPP EWG (2013) found insufficient evidence to recommend for or against testing for mutations in NRAS, and/or loss of expression of PTEN or AKT proteins. The level of certainty for this evidence was low. In the absence of supporting evidence, and with consideration of other contextual issues, the EWG discourages the use of these tests in guiding decisions on initiating anti-EGFR therapy with cetuximab or panitumumab unless further evidence supports improved clinical outcomes.\n",
      "Phosphatidylinositol-4,5-bisphosphonate 3-kinase, catalytic subunit alpha polypeptide gene (PIK3CA)\n",
      "The EGAPP EWG (2013) found insufficient evidence to recommend for or against testing for mutations in PIK3CA. The level of certainty for this evidence was low. In the absence of supporting evidence, and with consideration of other contextual issues, the EWG discourages the use of these tests in guiding decisions on initiating anti-EGFR therapy with cetuximab or panitumumab unless further evidence supports improved clinical outcomes.\n",
      "Guidelines from the American Society for Clinical Oncology (Sepulveda, et al., 2017) stated: \"There is insufficient evidence to recommend PIK3CA mutational analysis of colorectal carcinoma tissue for therapy selection outside of a clinical trial (Type: no recommendation; Strength of Evidence: insufficient, benefits/harms balance unknown; Quality of Evidence: insufficient).\"\n",
      "Cyclin D1 and FADD\n",
      "Cyclin D1 is used to diagnose of mantle cell lymphoma and predict recurrence of disease (Chin, et al., 2006). D-type cyclins are predominantly expressed in the G1 phase of the cell cycle. The expression pattern of cyclin D1 has been extensively studied in certain cancer types including lymphoma and non-small cell lung cancer. Approximately 30 percent of breast carcinomas are Cyclin D1 positive. Over expression of Cyclin D1 is now a well established criterion for the diagnosis of Mantle Cell Lymphoma, a malignant, non-Hodgkin's lymphoma which is characterized by a unique chromosomal translocation t(11;14).\n",
      "Rasamny et al (2012) stated that cyclin D1 and FADD (Fas-associated protein with death domain) regulate the cell cycle and apoptosis, respectively, and are located on chromosome 11q13, which is frequently amplified in head and neck squamous cell carcinoma (HNSCC).  This study evaluated these proteins as predictors of clinical outcomes for HNSCC.  A total of 222 patients with upper aero-digestive HNSCC were included in this study.  Patients with tumors that were strongly positive for cyclin D1 and FADD had reduced OS (p = 0.003 and p < 0.001), disease-specific survival (DSS; p = 0.039 and p < 0.001), and DFS (p = 0.026 and p < 0.001) survival, respectively.  Together, the 2 markers effectively stratified OS (p < 0.001), DSS (p < 0.001), and DFS (p = 0.002).  Strong FADD staining correlated with greater alcohol consumption and varied significantly with primary tumor site: 56 % of hypopharynx tumors expressed high levels of FADD but only 7 % of glottis tumors.  Using Cox regression analysis, FADD and N stage were significant independent predictors of DSS and DFS, whereas cyclin D1, FADD, and N stage were independently significant for OS.  The authors concluded that cyclin D1 and FADD may have utility as predictors of long-term outcomes for patients with HNSCC.  Moreover, they stated that further study is needed to determine if these proteins predict response to different treatment approaches or assist in selecting patients for multi-modality therapy.\n",
      "Prolaris\n",
      "Prolaris (Myriad Genetics, Salt Lake City, UT) uses archived tumor specimens as the mRNA source, reverse transcriptase polymerase chain reaction amplification, and a low density RTPCR array platform. Prolaris is used to quantify expression levels of 31 cell cycle progression (CCP) genes and 15 housekeeper genes to generate a CCP score. An assessment by the BlueCross BlueShield Association Technology Evaluation Center (TEC) concluded that direct evidence is insufficient to establish the analytic validity, clinical validity, or clinical utility of the Prolaris test. The BlueCross BlueShield Association assessment (BCBSA, 2015) stated: \"Published evidence is sparse and insufficient to draw conclusions on the analytic validity, clinical validity, or clinical utility of Prolaris ... in patients under active surveillance program.\"\n",
      "An assessment by the Adelaide Health Technology Assessment (Ellery, et al., 2014) found that there is currently uncertainty around the clinical utility of Prolaris. Citing a study by Shore, et al. (2014) showning that only a small percent of urologist would definitely change treatment based on the test results, \"it would appear that there is hesitancy about the use of the technology in clinical practice, and it appears that changes to clinical management based on the prognostic information provided by these genetic tests are unlikely to occur. Therefore HealthPACT recommends that no further research be conducted on their behalf at this point in time.\"\n",
      "NCCN prostate cancer guidelines (2015) state: \"The Prolaris assay produces a cell cycle progression (CCP) score from RNA expression levels of 31 genes involved in CCP. . . .For example, Prolaris has been successful in 93% of radical prostatectomy specimens, and 70% of diagnostic prostate biopsy specimens. The Prolaris CCP score has been demonstrated predictive when applied in prospective-retrospective designs for biochemical recurrence or metastasis after radical prostatectomy, for survival when men were observed after diagnosis on transurethral resection of prostate or diagnostic needle biopsy, and for biochemical recurrence and survival after external beam radiation therapy. .. Prolaris has changed treatment recommendations in 32% to 65% of cases and may enhance adherence to the treatment recommended. .. Both [Prolaris and Oncotype DX Prostate] molecular biomarker tests have been developed with extensive industry support, guidance, and involvement, and have been marketed under the less rigorous FDA regulatory pathway for biomarkers. Their clinical utility awaits evaluation by prospective, randomized clinical trials, which are unlikely to be done. The marketplace and comparative effectiveness research may be the only means for these tests and others like them to gain their proper place for better risk stratification for men with clinically localized prostate cancer.\"\n",
      "A guideline from the American Society for Clinical Oncology on active surveillance of prostate cancer (Chen, et al., 2016) stated that \"[u]se of ancillary tests beyond DRE, PSA, and biopsy to improve patient selection or as part of monitoring in an AS regimen remains investigational. Although there is a potential for genomic tests that use biopsy tissue to predict patients who are more rather than less likely to have disease progression and cancer-specific mortality and for multiparametric magnetic resonance imaging (mpMRI) to guide biopsies to find more clinically aggressive disease, prospective validation of these tests is needed to assess their impact on patient outcomes such as survival. Selective use of these ancillary tests in patients with discordant clinical and/or pathologic findings may be appropriate.\" \n",
      "An assessment by the National Institute for Health and Care Excellence (NICE, 2016) noted that most of the relevant evidence for Prolaris is on clinical validity, and evidence for the prognostic value of Prolaris is based only on the retrospective analyses of archived material. No studies examined the prospective use of Prolaris on patient outcomes. \n",
      "Guidelines on localized prostate cancer from the American Urologic Association (Sanda et al, 2017) state based upon expert opinion that, among most low-risk localized prostate cancer patients, Prolaris and other tissue based genomic biomarkers have not shown a clear role in the selection of candidates for active surveillance. The guidelines also indicate that tissue based genomic biomarkers are not necessary for followup. The guidelines state that the Prolaris test has not been validated as providing substantial benefit in the active surveillance population.\n",
      "National Comprehensive Cancer Networkâs clinical practice guideline on \"Prostate cancer\" (Version 4.2019) recommends coverage of  \"Decipher\", \"Oncotype DX Prostate\", \"Prolaris\", and \"ProMark\" (Category 2A).  Prolaris: For post-biopsy based on NCCN very-low-, low-risk, and favorable intermediate-risk patients with greater than 10 year life expectancy who have not received treatment for prostate cancer and are candidates for active surveillance or definitive therapy.\n",
      "Oncotype Dx Prostate\n",
      "The Oncotype DX test for prostate cancer (Genomic Health) is a genomic test that determines the risk of the cancer before treatment begins (Raman, et al., 2013). The test predicts how likely it is that the cancer is low risk and contained within the prostate, or higher risk and more likely to grow and spread. With this information, the patient and their doctor can choose the most appropriate treatment option. For example, a lower risk prostate cancer with more favorable pathology, one that may not need invasive treatment and can be safely managed through close and careful monitoring â a treatment approach called active surveillance. This genomic test measures biology through the expression of 17 genes across multiple key biological pathways in prostate cancer which can predict the aggressiveness of prostate cancer providing an individualized risk assessment.\n",
      "Oncotype Dx Prostate Cancer Assay (Genomic Health, Redwood City, CA) is used to quantify expression levels of 12 cancer-related and 5 reference genes to generate a Genomic Prostate Score (GPS). In the final analysis, the cell cycle progression (CCP) score (median 1.03, interquartile range 0.41 to 1.74) and GPS (range of 0 to 100) are combined in proprietary algorithms with clinical risk criteria (PSA, Gleason grade, tumor stage) to generate new risk categories (i.e., reclassification) intended to reflect biological indolence or aggressiveness of individual lesions, and thus inform management decisions.\n",
      "Klein et al (2014) sought to identify and validate a biopsy-based gene expression signature that predicts clinical recurrence, prostate cancer (PCa) death, and adverse pathology. Gene expression was quantified by reverse transcription-polymerase chain reaction for three studies-a discovery prostatectomy study (n=441), a biopsy study (n=167), and a prospectively designed, independent clinical validation study (n=395)-testing retrospectively collected needle biopsies from contemporary (1997-2011) patients with low to intermediate clinical risk who were candidates for active surveillance (AS).  The main outcome measures defining aggressive PCa were clinical recurrence, PCa death, and adverse pathology at prostatectomy. Cox proportional hazards regression models were used to evaluate the association between gene expression and time to event end points. Results from the prostatectomy and biopsy studies were used to develop and lock a multigene-expression-based signature, called the Genomic Prostate Score (GPS); in the validation study, logistic regression was used to test the association between the GPS and pathologic stage and grade at prostatectomy. Decision-curve analysis and risk profiles were used together with clinical and pathologic characteristics to evaluate clinical utility. Of the 732 candidate genes analyzed, 288 (39%) were found to predict clinical recurrence despite heterogeneity and multifocality, and 198 (27%) were predictive of aggressive disease after adjustment for prostate-specific antigen, Gleason score, and clinical stage. Further analysis identified 17 genes representing multiple biological pathways that were combined into the GPS algorithm. In the validation study, GPS predicted high-grade (odds ratio [OR] per 20 GPS units: 2.3; 95% confidence interval [CI], 1.5-3.7; p<0.001) and high-stage (OR per 20 GPS units: 1.9; 95% CI, 1.3-3.0; p=0.003) at surgical pathology. GPS predicted high-grade and/or high-stage disease after controlling for established clinical factors (p<0.005) such as an OR of 2.1 (95% CI, 1.4-3.2) when adjusting for Cancer of the Prostate Risk Assessment score. A limitation of the validation study was the inclusion of men with low-volume intermediate-risk PCa (Gleason score 3+4), for whom some providers would not consider AS. \n",
      "Cullen et al (2015) used a racially diverse cohort of men (20% African American [AA]) to evaluate the association of the clinically validated 17-gene Genomic Prostate Score (GPS) with recurrence after radical prostatectomy and adverse pathology (AP) at surgery. Biopsies from 431 men treated for National Comprehensive Cancer Network (NCCN) very low-, low-, or intermediate-risk PCa between 1990 and 2011 at two US military medical centers were tested to validate the association between GPS and biochemical recurrence (BCR) and to confirm the association with AP. Metastatic recurrence (MR) was also evaluated. Cox proportional hazards models were used for BCR and MR, and logistic regression was used for AP. Central pathology review was performed by one uropathologist. AP was defined as primary Gleason pattern 4 or any pattern 5 and/or pT3 disease. GPS results (scale: 0-100) were obtained in 402 cases (93%); 62 men (15%) experienced BCR, 5 developed metastases, and 163 had AP. Median follow-up was 5.2 yr. GPS predicted time to BCR in univariable analysis (hazard ratio per 20 GPS units [HR/20 units]: 2.9; p<0.001) and after adjusting for NCCN risk group (HR/20 units: 2.7; p<0.001). GPS also predicted time to metastases (HR/20 units: 3.8; p=0.032), although the event rate was low (n=5). GPS was strongly associated with AP (odds ratio per 20 GPS units: 3.3; p<0.001),  adjusted for NCCN risk group. In AA and Caucasian men, the median GPS was 30.3 for both, the distributions of GPS results were similar, and GPS was similarly predictive of outcome. \n",
      "Dall'Era et al (2015) performed a retrospective chart review to assess the impact of incorporating the Oncotype DX Genomic Prostate Score on treatment recommendations and decisions for men with newly diagnosed low risk prostate cancer in community urology practices. A total of 24 urologists who ordered the Oncotype DX prostate cancer assay soon after launch (May 2013) were invited to participate in the study. Clinicopathological data, Genomic Prostate Score results and treatment related information were retrieved from medical records. Data also were collected for a pre-Genomic Prostate Score baseline group diagnosed from May 2012 to April 2013. Descriptive analyses were performed to evaluate the proportion of men for whom active surveillance was recommended and used before and after the availability of Genomic Prostate Score. Overall 15 physicians contributing 211 patients (Genomic Prostate Score group 124, baseline group 87) participated in the chart review. Patients in the Genomic Prostate Score and baseline groups had comparable risk based on traditional clinical pathological features, with 82% with NCCNÂ® very low or low risk disease. With Genomic Prostate Score the relative increase in active surveillance recommended was 22% (baseline 50% and Genomic Prostate Score 61%, absolute increase of 11%) and the relative increase in use of active surveillance was 56% (baseline 43% and Genomic Prostate Score 67%, absolute increase of 24%). Treatment recommendations for active surveillance were directionally consistent with assay reported risk.\n",
      "Badani et al (2015) performed a prospective study to assess the impact of incorporating Oncotype Dx GPS into treatment recommendations in 3 high volume urology practices. Men with newly diagnosed prostate cancer meeting specific NCCN criteria were prospectively enrolled in the trial. Biopsy tissue was analyzed. Urologists indicated treatment recommendations on questionnaires administered before and after GPS. The primary study objectives were to assess all changes in treatment modality and/or treatment intensity after GPS. A total of 158 men were included in analysis, including 35, 71 and 52 at NCCN very low, low and low-intermediate risk. Biological risk predicted by GPS differed from NCCN clinical risk alone in 61 men (39%). Overall 18% of recommendations between active surveillance and immediate treatment changed after GPS. The relative increase in recommendations for active surveillance was 24% (absolute change 41% to 51%). In 41 of 158 men (26%) modality and/or intensity recommendations changed after GPS, including 25, 14 and 2 in whom recommendation intensity decreased, increased and were equivocal, respectively. All changes were directionally consistent with GPS. The NCCN low risk group showed the greatest absolute recommendation change after GPS (37%). In 17 of 57 men (30%) the initial recommendation of radical prostatectomy was changed to active surveillance after GPS. Urologists indicated greater confidence and found that incorporating GPS was useful in 85% and 79% of cases, respectively, including when biological risk confirmed the clinical risk category.\n",
      "Brand et al (2016) performed a patient-specific meta-analysis (MA) of two independent clinical validation studies of a 17-gene biopsy-based genomic assay (Oncotype Dx Prostate) as a predictor of favorable pathology at radical prostatectomy. Patient-specific MA was performed on data from 2 studies (732 patients) using the Genomic Prostate Score (GPS; scale 0-100) together with Cancer of the Prostate Risk Assessment (CAPRA) score or National Comprehensive Cancer Network (NCCN) risk group as predictors of the likelihood of favorable pathology (LFP). Risk profile curves associating GPS with LFP by CAPRA score and NCCN risk group were generated. Decision curves and receiver operating characteristic curves were calculated using patient-specific MA risk estimates. Patient-specific MA-generated risk profiles ensure more precise estimates of LFP with narrower confidence intervals than either study alone. The investigators stated that GPS added significant predictive value to each clinical classifier. A model utilizing GPS and CAPRA provided the most risk discrimination. In decision-curve analysis, greater net benefit was shown when combining GPS with each clinical classifier compared with the classifier alone. The area under the receiver operating characteristic curve improved from 0.68 to 0.73 by adding GPS to CAPRA, and 0.64 to 0.70 by adding GPS to NCCN risk group. The proportion of patients with LFP >80% increased from 11% using NCCN risk group alone to 23% using GPS with NCCN. Using GPS with CAPRA identified the highest proportion-31%-of patients with LFP > 80%.\n",
      "In a prospective study, Albala et al (2016) evaluated the clinical utility and economic impact of the Oncotype DX Prostate GPS in patients with low-risk prostate cancer. The study was conducted at a single large urology group practice and enrolled patients with a single insurance carrier. The insurance carrier calculated cost data from the first 180 days after diagnosis (including the cost of the diagnostic biopsy) and provided the average treatment cost per patient from their analysis, and compared management patterns and costs from a baseline, untested population to a similar prospective, GPS-tested population. The primary endpoint of the study was the net percentage difference in prospective treatment decisions with use of Oncotype DX Prostate GPS as compared with the baseline treatment patterns without use of GPS. Of the 71 men in the baseline group who were NCCN very low risk and low risk, 27 (38%) were managed with AS, 25 (35%) had RP, 18 (25%) were managed with IMRT and 1 (1%) had whole-gland cryoablation. In the 51 GPS-tested NCCN very-low-risk and low-risk patients, 30 (59%) were managed with AS, 13 (25%) had an RP, 6 (12%) were managed with IMRT, 1 (2%) was managed with multimodal therapy (IMRT and brachytherapy), and 1 (2%) chose focal cryoablation. AS utilization was 21% higher in the prospective GPS-tested cohort of very-low-risk and low-risk men compared with the baseline cohort of risk-group-matched men. The rate of RP was 10% lower and the rate of IMRT was 14% lower in the prospective cohort of very-low-risk and low-risk men when GPS was incorporated into treatment decisions compared with the baseline cohort of risk group-matched men  In the 29 GPS untested NCCN intermediate-risk patients, 5 (17.2%) were managed by AS, 12 (41.4%) by RP, 11 (37.9%) by IMRT, and 1 (3.4%) by CyberKnife radiosurgery. In the 29 GPS-tested NCCN favorable intermediate-risk patients, no patient chose AS, 14 (48%) chose RP, 11 (38%) chose RT, 1 (3%) chose brachytherapy, and 3 (10%) chose multimodal treatment. AS utilization decreased and RP slightly increased in NCCN favorable intermediate-risk group patients after using GPS. IMRT usage remained unchanged between the baseline and prospective groups. Comparing payer costs in the first 180 days after diagnosis for the entire NCCN risk population (n = 80), there was an average cost addition of $1023 per patient, including the cost of the GPS at $4520 (total net addition of $81,855 for the entire GPS-tested population). Comparing payer costs in the first 180 days after diagnosis for th baseline and prospective NCCN very-low-risk and low-risk populations, there was an average savings per patient of $2286, including the cost of the GPS.\n",
      "An assessment by the BlueCross BlueShield Technology Evaluation Center (TEC, 2014) concluded that direct evidence is insufficient to establish the analytic validity, clinical validity, or clinical utility of the Oncotype Dx Prostate.\n",
      "The BlueCross BlueShield Technology Evaluation Centerâs assessment on \"Gene Expression Analysis for Prostate Cancer Management\" (BCBSA, 2015) concluded that \"Evidence is insufficient to determine whether . . . Oncotype Dx Prostate testing improves health outcomes in the investigational setting.  Based on the above, neither the Prolaris nor Oncotype Dx Prostate array-based gene expression test meets the TEC criteria\".  The assessment stated: Published evidence is sparse and insufficient to draw conclusions on the . . . clinical validity or utility of Oncotype Dx Prostate in patients under active surveillance program.\"\n",
      "An assessment by Adelaide Health Technology Assessment (Ellery, et al., 2014) concluded that \"there is uncertainty about the clinical utility\" of the Oncotype Dx Prostate and the Prolaris tests, \"even when taking into account the highest level of evidence available\" The assessment stated that it remains to be verified whether genetic expression of the unique gene panels involved are robust to heterogeneous sampling of prostate tissue at the time of biopsy. Also, the need for tissue which has previously been fixed for histological analysis is of some concern. The assessment observed that this is the most obvious reason for the relatively high number of patients for whom a valid test results could not be obtained.\n",
      "European Association of Urology (2015) prostate cancer guidelines state that genomics on the tissue samping appear \"promising,\" but \"further study data will be needed before such markers can be used in standard clinical practice.\"\n",
      "NCCN guidelines on prostate cancer (NCCN, 2016) state: \"These molecular tests listed have been developed with extensive industry support, guidance, and involvement and have been marketed under the less rigorous FDA regulatory pathway for biomarkers. Although full assessment of their clinical utility requires prospective, randomized clinical trials, which are unlikely to be done, the panel believes that men with clinically localized disease may consider the use of tumor-based molecular assays at this time.  Future comparative effectiveness research may allow these tests and others like them to gain additional evidence regarding their utility for better risk stratification of men with prostate cancer.\"\n",
      "More recently, NCCN guidelines on prostate cancer (NCCN, 2019) state: \"These molecular biomarker tests have been developed with extensive industry support, guidance, and involvement, and have been marketed under the less rigorous FDA regulatory pathway for biomarkers. Although full assessment of their clinical utility requires prospective, randomized clinical trials, which are unlikely to be done, the panel believe that men with low or favorable intermediate risk disease may consider the use of Decipher, Oncotype Dx Prostate, Prolaris, or ProMark during initial risk stratification. . . . Future comparative effectiveness research may allow these tests and others like them to gain additional evidence regarding their utility for better risk stratification of men with prostate cancer.\" \n",
      "linical practice guidelines from the American Urological Association (AUA)/American Society for Radiation Oncology (ASTRO)/Society of Urologic Oncology (SUO) and the American Society of Clinical Oncology (ASCO) state that among men with low-risk prostate cancer, tissue-based genomic biomarkers have not shown a clear role in the selection of candidates for active surveillance (Bekelman, et al., 2018; Sanda, et al., 2018a; Sanda, et al., 2018b). \n",
      "Guidelines on prostate cancer from the European Association of Urology (EAU), the European Society for Radiotherapy and Oncology (ESTRO), the European Society of Urogenital Radiology (ESUR), and the International Society of Geriatric Oncology (SIOG)  (Mottet, et al., 2018) state that the Oncotype Dx Prostate has been shown in prospective studies to provide prognostic information in men with clinically localized prostate cancer, additional to conventional clinico-pathological parameters, including pathologic grade and PSA level. \"The results of prospective multicentre studies are awaited before a recommendation can be made regarding their routine application.\"\n",
      "A guideline from the American Society for Clinical Oncology on active surveillance of prostate cancer (Chen, et al., 2016) stated that \"[u]se of ancillary tests beyond DRE, PSA, and biopsy to improve patient selection or as part of monitoring in an AS regimen remains investigational. Although there is a potential for genomic tests that use biopsy tissue to predict patients who are more rather than less likely to have disease progression and cancer-specific mortality and for multiparametric magnetic resonance imaging (mpMRI) to guide biopsies to find more clinically aggressive disease, prospective validation of these tests is needed to assess their impact on patient outcomes such as survival. Selective use of these ancillary tests in patients with discordant clinical and/or pathologic findings may be appropriate.\" \n",
      "Guidelines on localized prostate cancer from the American Urologic Association (Sanda, et al., 2017) state based upon expert opinion that, among most low-risk localized prostate cancer patients, the Oncotype Dx Prostate and other tissue based genomic biomarkers have not shown a clear role in the selection of candidates for active surveillance. The guidelines also indicate that tissue based genomic biomarkers are not necessary for followup. The guidelines state that the Oncotype Dx Prostate test has not been validated as providing substantial benefit in the active surveillance population.\n",
      "In a prospective, multi-institutional study, Eure et al (2017) studied the impact of genomic testing in shared decision-making for men with clinically low-risk PCa.  Patients with clinically low-risk PCa were enrolled in this study of a validated 17-gene tissue-based reverse transcription polymerase chain reaction assay (Genomic Prostate Score [GPS]).  These investigators reported on outcomes in the first 297 patients enrolled in the study with valid 17-gene assay results and decision-change data.  The primary end-points were shared decision on initial management and persistence on active surveillance (AS) at 1 year post-diagnosis; AS utilization and persistence were compared with similar end-points in a group of patients who did not have genomic testing (baseline cohort).  Secondary end-points included perceived utility of the assay and patient decisional conflict before and after testing; 1-year results were available on 258 patients.  Shift between initial recommendation and shared decision occurred in 23 % of patients.  Utilization of AS was higher in the GPS-tested cohort than in the untested baseline cohort (62 % versus 40 %).  The proportion of men who selected and persisted on AS at 1 year was 55 % and 34 % in the GPS and baseline cohorts, respectively.  Physicians reported that GPS was useful in 90 % of cases.  Mean decisional conflict scores declined in patients after GPS testing.  The authors concluded that patients who received GPS testing were more likely to select and persist on AS for initial management compared with a matched baseline group.  They stated that these data indicated that GPS help guide shared decisions in clinically low-risk PCa.\n",
      "The authors stated that this study had several drawbacks.  This study was based on an interim analysis of the first 297 patients enrolled in a large (nâ=â1,200), multi-center prospective trial, and should thus be considered preliminary.  There were marked differences between participating practices in terms of baseline utilization of AS; this may impact change rates in a meaningful fashion.  Definitive surgical pathology was not available in patients who chose AS.  The high rates of initial decision for AS in the GPS-tested cohort could be partly due to a Hawthorne effect; providers may have over-stated their utilization of AS at baseline in this study that was focused on a test for AS decision-making.  The actual rate of AS in routine practice could be lower than what had been collected on the study.  They stated that despite these limitations, this analysis highlighted the potential benefits of incorporating genomic testing into individualized risk estimation and shared decision-making for PCa patients in a real-world setting.\n",
      "Steven and colleagues (2017) stated that many men with low-risk prostate cancer (PCa) receive definitive treatment despite recommendations that have been informed by 2 large, randomized trials encouraging active surveillance (AS).  These researchers conducted a retrospective cohort study using the Optum Research Database (Eden Prairie, MN) of electronic health records (EHR)and administrative claims data to assess AS use for patients tested with a 17-gene Genomic Prostate Score (GPS; Genomic Health, Redwood City, CA) assay and/or prostate magnetic resonance imaging (MRI).  De-identified records were extracted on health plan members enrolled from June 2013 to June 2016 who had greater than or equal to 1 record of PCa (n = 291,876).  Inclusion criteria included age greater than or equal to 18 years, new diagnosis, American Urological Association low-risk PCa (stage T1 to T2a, PSA less than or equal to 10 ng/ml, Gleason score = 6), and clinical activity for at least 12 months before and after diagnosis.  Data included baseline characteristics, use of GPS testing and/or MRI, and definitive procedures.  GPS or MRI testing was performed in 17 % of men (GPS, n = 375, 4 %; MRI, n = 1,174, 13 %).  AS use varied from a low of 43 % for men who only underwent MRI to 89 % for GPS tested men who did not undergo MRI (p < 0.001).  At 6-month follow-up, AS use was 31.0 % higher (95 % CI: 27.6 % to 34.5 %; p < 0.001) for men receiving the GPS test only versus men who did not undergo GPS testing or MRI; the difference was 30.5 % at 12-month follow-up.  The authors concluded that in a large US payer system, the GPS assay was associated with significantly higher AS use at 6 and 12 months compared with men who had MRI only, or no GPS or MRI testing.\n",
      "The authors stated that \"Several limitations were inherent to a study design with an EHR and claims database.  First, some patients followed to 6 months had no follow-up data at 12 months, which may be due to patients seeking care elsewhere, an insurance change, or a small risk of mortality between This definition likely captured the majority of AS patients, but may have permitted some misclassification (some selected patients may not have truly undergone guideline-based AS).  Finally, physicians who ordered the GPS test may have had different, unmeasurable, perspectives toward adoption of emerging medical technologies compared with those who did not order the test and thus may not be representative of the field at large\".\n",
      "Van Den Eeden et al (2018) stated that a 17-gene biopsy-based reverse transcription polymerase chain reaction assay, which provides a Genomic Prostate Score (GPS-scale 0 to 100), has been validated as an independent predictor of adverse pathology and biochemical recurrence after radical prostatectomy (RP) in men with low- and intermediate-risk PCa.  In a retrospective study, these researchers evaluated GPS as a predictor of PCa metastasis and PCa-specific death (PCD) in a large cohort of men with localized PCa and long-term follow-up.  A stratified cohort sampling design was performed in a cohort of men treated with RP within Kaiser Permanente Northern California; RNA from archival diagnostic biopsies was assayed to generate GPS results.  The se researchers assessed the association between GPS and time to metastasis and PCD in pre-specified uni- and multi-variable statistical analyses, based on Cox proportional hazard models accounting for sampling weights.  The final study population consisted of 279 men with low-, intermediate-, and high-risk PCa between 1995 and 2010 (median follow-up of 9.8 years), and included 64 PCD and 79 metastases.  Valid GPS results were obtained for 259 (93 %).  In uni-variable analysis, GPS was strongly associated with time to PCD, hazard ratio (HR)/20 GPS units = 3.23 (95 % CI: 1.84 to 5.65; p < 0.001), and time to metastasis, HR/20 units = 2.75 (95 % CI: 1.63 to 4.63; p < 0.001).  The association between GPS and both end-points remained significant after adjusting for NCCN, AUA, and Cancer of the Prostate Risk Assessment (CAPRA) risks (p < 0.001).  No patient with low- or intermediate-risk disease and a GPS of less than 20 developed metastases or PCD (n = 31).  In receiver operating characteristic analysis of PCD at 10 years, GPS improved the c-statistic from 0.78 (CAPRA alone) to 0.84 (GPS + CAPRA; p < 0.001).  The authors concluded that GPS is a strong independent predictor of long-term outcomes in clinically localized PCa in men treated with RP and may improve risk stratification for men with newly diagnosed disease. \n",
      "The authors stated that it is important to note that all patients in the cohort had undergone definitive therapy with RP, and thus a limitation of the study was that it did not assess outcomes in patients managed with AS or radiation therapy (RT).  Another drawback of the study was that patients were treated during an era when definitive treatment was standard of care with little adoption of active surveillance.\n",
      "Canfield et al (2018) stated that many men with low-risk PCa receive definitive treatment despite recommendations that have been informed by 2 large, randomized trials encouraging AS.  These researchers conducted a retrospective cohort study using the Optum Research Database (Eden Prairie, MN) of electronic health records and administrative claims data to assess AS use for patients tested with a 17-gene GPS (Genomic Health, Redwood City, CA) assay and/or prostate magnetic resonance imaging (MRI).  De-identified records were extracted on health plan members enrolled from June 2013 to June 2016 who had greater than or equal to 1 record of PCa (n = 291,876).  Inclusion criteria included age greater than or equal to 18 years, new diagnosis, AUA low-risk PCa (stage T1 to T2a, prostate-specific antigen less than or equal to 10 ng/ml, Gleason score 5 6), and clinical activity for at least 12 months before and after diagnosis.  Data included baseline characteristics, use of GPS testing and/or MRI, and definitive procedures.  GPS or MRI testing was performed in 17 % of men (GPS, n = 375, 4 %; MRI, n = 1,174, 13 %).  AS use varied from a low of 43 % for men who only underwent MRI to 89 % for GPS tested men who did not undergo MRI (P < 0.001).  At 6-month follow-up, AS use was 31.0 % higher (95 % CI:  27.6 % to 34.5 %; p < 0.001) for men receiving the GPS test only versus men who did not undergo GPS testing or MRI; the difference was 30.5 % at 12-month follow-up.  The authors concluded that in a large US payer system, the GPS assay was associated with significantly higher AS use at 6 and 12 months compared with men who had MRI only, or no GPS or MRI testing.\n",
      "The authors noted that several limitations were inherent to a study design with an EHR and claims database.  First, some patients followed to 6 months had no follow-up data at 12 months, which may be due to patients seeking care elsewhere, an insurance change, or a small risk of mortality between 6 and 12 months.  Second, because no unique CPT code for AS exists, these researchers defined it as no definitive therapy during the observation period.  This definition likely captured the majority of AS patients, but may have allowed some misclassification (some selected patients may not have truly undergone guideline-based AS).  Finally, physicians who ordered the GPS test may have had different, unmeasurable, perspectives toward adoption of emerging medical technologies compared with those who did not order the test, and thus may not be representative of the field at large.\n",
      "Lynch et al (2018) compared management strategies for PCa at 6 Veterans Affairs' medical centers (VAMCs)  before and after introduction of the Oncotype DX Genomic Prostate Score (GPS) assay.  These investigators reviewed records of patients diagnosed with PCa between 2013 and 2014 to identify management patterns in an untested cohort.  From 2015 to 2016, these patients received GPS testing in a prospective study.  Charts from 6 months post-biopsy were reviewed for both cohorts to compare management received in the untested and tested cohorts.  Men who just received their diagnosis and had NCCN very low-, low-, and select cases of intermediate-risk PCa were eligible for this study.  Patient characteristics were generally similar in the untested and tested cohorts.  Active surveillance (AS) utilization was 12 % higher in the tested cohort compared with the untested cohort.  In men younger than 60 years, utilization of AS in tested men was 33 % higher than in untested men; AS in tested men was higher across all NCCN risk groups and races, particular in low-risk men (72 % versus 90 % for untested vs tested, respectively).  Tested veterans exposed to Agent Orange (AO) received less AS than untested veterans.  Tested non-exposed veterans received 19 % more AS than untested veterans.  Median GPS results did not significantly differ as a factor of race or AO exposure.  The authors concluded that men who received GPS testing are more likely to utilize AS within the year post-diagnosis, regardless of age, race, and NCCN risk group.  Median GPS was similar across racial groups and AO exposure groups, suggesting similar biology across these groups.  They stated that  GPS assay may be a useful tool to refine risk assessment of PCa and increase rates of AS among clinically and biologically low-risk patients, which is in line with guideline-based care.  Moreover, they stated that future studies showing the persistence on AS and longer-term outcomes should be considered to further support the utility of the GPS assay.\n",
      "This study has several drawbacks.  The untested cohort included a significantly larger proportion of intermediate-risk patients.  Although this may be construed to imply that the greater utilization of AS in the tested cohort was driven by lower baseline risk, within group changes indicated that there was higher AS utilization in tested patients regardless of baseline risk group; this change was most pronounced for the NCCN low-risk category;  but was present in NCCN very-low and intermediate-risk patients.  The authors considered any patients who did not receive a definitive treatment within 6 months of biopsy as undergoing AS.  Hence, some patients classified as receiving AS may have been simply delaying planned definitive management.  The untested patients were seen in the VA in the 2 years prior to enrollment of the tested patients.  Practice patterns are evolving in the direction of higher AS rates across all practice settings, and some of this shift may be related to temporal tends.  However, these researchers believed it unlikely that the differences between groups in this study would have been achieved in a 1- to 2-year period, particularly since the institutions and providers were the same and the baseline rate of AS was already very high.\n",
      "Hu et al (2018) noted that tissue-based GECs may assist with management decisions in patients with newly diagnosed PCa.  These investigators evaluated the current use of GEC tests and determined how the test results were associated with primary disease management.  In this observational study, patients diagnosed with localized PCa were tracked through the Michigan Urological Surgery Improvement Collaborative registry.  The utilization and results of 3 GECs (Decipher Prostate Biopsy, Oncotype DX Prostate, and Prolaris) were prospectively collected.  Practice patterns, predictors of GEC use, and effect of GEC results on disease management were examined.  Of 3,966 newly diagnosed patients, 747 (18.8 %) underwent GEC testing.  The rate of GEC use in individual practices ranged from 0 % to 93 %, and patients undergoing GEC testing were more likely to have a lower PSA level, lower Gleason score, lower clinical T stage, and fewer positive cores (all p < 0.05).  Among patients with clinical favorable risk of cancer, the rate of AS differed significantly among patients with a GEC result above the threshold (46.2 %), those with a GEC result below the threshold (75.9 %), and those who did not undergo GEC (57.9 %; p < 0.001 for comparison of the 3 groups).  This resulted in an estimate that, for every 9 men with favorable risk of cancer who undergo GEC testing, 1 additional patient may have their disease initially managed with AS.  On multi-variable analysis, patients with favorable-risk PCa who were classified as GEC low risk were more likely to be managed on AS than those without testing (OR, 1.84; p = 0.006).  The authors concluded that there is large variability in practice-level use and GEC tests ordered in patients with newly diagnosed, localized PCa.  In patients with clinical favorable risk of cancer, GEC testing significantly increased the use of AS.  These researchers stated that additional follow-up will help examine if incorporation of GEC testing into initial patient care favorably affects clinical outcomes.\n",
      "Kornberg et al (2019) stated that the OncotypeDx GPS (Genomic Prostate Score) is a 17-gene RNA expression assay intended to help guide treatment decisions in men diagnosed with PCa.  The PI-RADS (Prostate Imaging Reporting and Data System) version 2 was developed to standardize the risk stratification of lesions identified on multi-parametric prostate MRI.  These researchers examined if these tests are associated with an increased risk of biopsy up-grading in men on AS.  They identified all patients on AS at the University of California-San Francisco who had low  /intermediate risk PCa (PSA of 20 ng/ml or less and clinical stage T1/T2) and Gleason score 6 disease who underwent multiple biopsies and had a GPS available and/or had undergone multi-parametric prostate MRI with an available PI-RADS version 2 score.  The primary study outcome was biopsy up-grading, defined as an increase in the Gleason score from 3 + 3 to 3 + 4 or greater, which was analyzed by Cox proportional hazards regression.  Of the men 140 had only GPS test findings, 169 had only a PI-RADS version 2 score and 131 had both data.  Each 5-unit increase in the GPS was associated with an increased risk of biopsy up-grading (HR 1.28, 95 % CI: 1.19 to 1.39, p < 0.01).  PI-RADS scores of 5 versus 1-2 (HR 4.38, 95 % CI: 2.36 to 8.16, p < 0.01) and 4 versus 1-2 (HR 2.62, 95 % CI 1.45 to 4.76, p < 0.01) were also associated with an increased risk of a biopsy up-grade.  On sub-analysis of patients with GPS and PI-RADS version 2 scores the GPS was associated with biopsy up-grading, adding value to the clinical co-variates (partial likelihood ratio p = 0.01).  The authors concluded that a higher GPS or a PI-RADS version 2 score of 4 or 5 was associated with an increased risk of biopsy up-grading. \n",
      "Eggener et al (2019) validated the 17-gene Oncotype DX Genomic Prostate Score (GPS) biopsy-based gene expression assay as a predictor of adverse pathology (AP, Gleason score [pGS] greater than or equal to 4+3 and/or greater than or equal to pT3) in a prospectively enrolled cohort.  Between July 2014 and September 2015, a total of 1,200 men with very low-, low-, and favorable intermediate-risk prostate cancer (PCa) enrolled in a multi-institutional prospective study of the GPS assay.  The subset who proceeded to immediate radical prostatectomy (RP) after GPS testing was included in a pre-specified sub-analysis of GPS on biopsy and its association with surgical AP on RP using logistic regression and receiver operating characteristic curves.  The effect of GPS testing on physicians' and patients' attitudes about decision-making was assessed with the Decisional Conflict Scale.  A total of 114 patients (treated by 59 physicians from 19 sites) elected RP and 40 (35 %) had AP; GPS result was a significant predictor of AP (odds ratio per 20 GPS units [OR/20 units]: 2.2; 95 % CI: 1.2 to 4.1; pâ¯=â¯0.008) in uni-variable analysis and remained significant after adjustment for biopsy Gleason score, clinical T-stage, and logPSA (OR/20 units: 1.9; 95 % CI: 1.0 to 3.8; pâ¯=â¯0.04), or NCCN risk group (OR/20 units: 2.0; 95 % CI: 1.1 to 3.7; pâ¯=â¯0.02).  Mean pre-GPS Decisional Conflict Scale score was 27 (95 % CI: 24 to 31), which improved significantly after GPS testing to 14 (95 % CI: 11 to 17) (p < 0.001).  The authors concluded that in this real-world multi-institutional study, the GPS assay was prospectively confirmed as an independent predictor of AP at surgery; GPS testing was associated with reduced patient decisional conflict.\n",
      "The authors stated that the data presented were not without limitations; RP was performed by 59 surgeons from 19 centers and providers were not coached on how to address PCa decision-making in a uniform fashion.  In addition, biopsy and prostatectomy specimens were evaluated locally without central review.  While the lack of standardization across sites introduced the potential for site-specific variation and inter-observer variability in grading/staging, the fact that these results were derived from diverse practices may increase the external validity and relevance of these findings for routine use of GPS testing in practice.  Although there were only 40 AP events, there was a strong and significant association between the GPS result and AP and it was unlikely additional data points would have substantively modified the result.  This study did not include patients managed with initial AS or radiation therapy; assessments of the GPS assay in men managed with initial AS or radiation therapy are ongoing.  These researchers stated that despite these limitations, this prospective analysis highlighted the added value of GPS testing for prediction of AP in a contemporary, AS-eligible PCa cohort.  The added potential for genomic testing to inform management decisions may help to increase the pool of men who are eligible and appropriate for AS, while identifying men with more aggressive disease who may consider definitive treatment.\n",
      "An accompanying editorial asked \"how do we know that similar improvements in decisional conflict could not have been achieved through the use of free, publically-available decision aids?\" (Singhal, et al., 2019). The editorialists noted that the authors acknowledged that this study did not include men who elected active surveillance. \"While men electing treatment could be expected to experience some level of relief after simply making this decision, the more pertinent question may be how GPS impacts patients pursuing a less definitive, more anxiety-associated approach such as surveillance.\"  The editorialists noted that the study reported on the odds ratio and the overall range of GPS scores, but noted that it is difficult to capture the clinical significance of the odds ratio without a clear illustration of the GPS distribution. Odds ratios considered strong in the research setting are not adequate for discriminating between subjects who do and do not experience the outcome at an individual level. The editorialists also observed that once challenge to tests such as Oncotype Dx Prostate that report on a continuum of risk is the lack of a clear, singular threshold that can rule in or rule out the projected outcome. The editorialist suggested that future studies report threshold values with very high specificity and sensitivity observed in the study population. \"Clinical utility in this setting will continue to be challenged by a need to identify such thresholds for reliable, individual-level decision making.\" The editorialists concluded: \"Additional studies will help to clarify the optimal clinical scenarios for implementing this test and others in this rapidly-evolving arena.\" \n",
      "National Comprehensive Cancer Networkâs clinical practice guideline on \"Prostate cancer\" (Version 4.2019) recommends coverage of  \"Decipher\", \"Oncotype DX Prostate\", \"Prolaris\", and \"ProMark\" (Category 2A).  Oncotype DX Prostate: For post-biopsy based on NCCN very-low-, low-risk, and favorable intermediate-risk patients with greater than 10 year life expectancy who have not received treatment for prostate cancer and are candidates for active surveillance or definitive therapy.\n",
      "Oncotype DX AR-V7 Nucleus Detect Assay for Men with Metastatic Castrate Resistant Prostate Cancer\n",
      "The Oncotype DX nuclear-localized androgen receptor splice variant 7 (AR-V7) Nuclear Detect test is a circulating tumor cell-based, liquid biopsy test, which detects patients with CTCs who have nuclear expression of the AR-V7 truncated protein and is intended to provide information that can help guide treatment selection in patients with metastatic castration resistant prostate cancer (mCRPC). It has been hypothesized that the that detection of androgen-receptor splice variant 7 messenger RNA (AR-V7) in CTCs from men with advanced prostate cancer is associated with resistance to enzalutamide and abiraterone.\n",
      "Antonarakis et al (2014) used a quantitative reverse-transcriptase-polymerase-chain-reaction assay to evaluate AR-V7 in circulating tumor cells from prospectively enrolled patients with metastatic castration-resistant prostate cancer who were initiating treatment with either enzalutamide or abiraterone. The investigators examined associations between AR-V7 status (positive vs. negative) and prostate-specific antigen (PSA) response rates (the primary end point), freedom from PSA progression (PSA progression-free survival), clinical or radiographic progression-free survival, and overall survival. A total of 31 enzalutamide-treated patients and 31 abiraterone-treated patients were enrolled, of whom 39% and 19%, respectively, had detectable AR-V7 in circulating tumor cells. Among men receiving enzalutamide, AR-V7-positive patients had lower PSA response rates than AR-V7-negative patients (0% vs. 53%, P=0.004) and shorter PSA progression-free survival (median, 1.4 months vs. 6.0 months; P<0.001), clinical or radiographic progression-free survival (median, 2.1 months vs. 6.1 months; P<0.001), and overall survival (median, 5.5 months vs. not reached; P=0.002). Similarly, among men receiving abiraterone, AR-V7-positive patients had lower PSA response rates than AR-V7-negative patients (0% vs. 68%, P=0.004) and shorter PSA progression-free survival (median, 1.3 months vs. not reached; P<0.001), clinical or radiographic progression-free survival (median, 2.3 months vs. not reached; P<0.001), and overall survival (median, 10.6 months vs. not reached, P=0.006). The association between AR-V7 detection and therapeutic resistance was maintained after adjustment for expression of full-length androgen receptor messenger RNA. The investigators concluded that detection of AR-V7 in circulating tumor cells from patients with castration-resistant prostate cancer may be associated with resistance to enzalutamide and abiraterone. They stated that these findings require large-scale prospective validation. An accompanying editoral (Nelson, et al., 2014) agreed that \"the small number of patients in the study by Antonarakis and colleagues mandates validation. The proprietary combination of antibodies used to capture circulating tumor cells is a potential limitation. Whether other methods of isolating prostate-cancer cells would yield similar results should be determined.\" \n",
      "Antonarakis et al (2015) investigated whether AR-V7-positive patients would retain sensitivity to taxane chemotherapy and whether AR-V7 status would have a differential impact on taxane-treated men compared with enzalutamide- or abiraterone-treated men. The investigators examined CTCs for AR-V7 mRNA using a reverse-transcription polymerase chain reaction assay. From January 2013 to July 2014, the investigators prospectively enrolled patients with metastatic CRPC initiating taxane chemotherapy (docetaxel or cabazitaxel) at a single academic institution. Their prespecified statistical plan required a sample size of 36 taxane-treated men. The investigators evaluated associations between AR-V7 status and prostate-specific antigen (PSA) response rates, PSA progression-free survival (PSA PFS), and clinical and/or radiographic progression-free survival (PFS). After incorporating updated data from the above study of 62 patients treated with enzalutamide or abiraterone, they also investigated the interaction between AR-V7 status (positive or negative) and treatment type (taxane vs enzalutamide or abiraterone). Of 37 taxane-treated patients enrolled, 17 (46%) had detectable AR-V7 in CTCs. Prostate-specific antigen responses were achieved in both AR-V7-positive and AR-V7-negative men (41% vs 65%; Pâ=â.19). Similarly, PSA PFS (hazard ratio [HR], 1.7, 95% CI, 0.6-5.0; Pâ=â.32) and PFS (HR, 2.7, 95% CI, 0.8-8.8; Pâ=â.11) were comparable in AR-V7-positive and AR-V7-negative patients. A significant interaction was observed between AR-V7 status and treatment type (Pâ<â.001). Clinical outcomes were superior with taxanes compared with enzalutamide or abiraterone therapy in AR-V7-positive men, whereas outcomes did not differ by treatment type in AR-V7-negative men. In AR-V7-positive patients, PSA responses were higher in taxane-treated vs enzalutamide- or abiraterone-treated men (41% vs 0%; Pâ<â.001), and PSA PFS and PFS were significantly longer in taxane-treated men (HR, 0.19 [95% CI, 0.07-0.52] for PSA PFS, Pâ=â.001; HR, 0.21 [95% CI, 0.07-0.59] for PFS, Pâ=â.003). The investigators concluded that detection of AR-V7 in CTCs from men with metastatic CRPC was not associated with primary resistance to taxane chemotherapy. In AR-V7-positive men, taxanes appeared to be more efficacious than enzalutamide or abiraterone therapy, whereas in AR-V7-negative men, taxanes and enzalutamide or abiraterone may have comparable efficacy. The investigators suggested that circulating tumor cell-based AR-V7 detection may serve as a treatment selection biomarker in CRPC. An accompanying editorial (Taplin and Balk, 2015) noted that \"An obvious implication of these AR-V7 studies is that men with AR-V7âpositive CRPC, particularly if they have also been treated previously with abiraterone or enzalutamide, are more likely to respond to taxanes than to an alternative AR-targeted therapy, but this needs to be confirmed in further studies using validated assays.\" They noted that, although the difference in response to taxane chemotherapy was not statistically significant, confidence intervals were wide, and there was a 24% difference (AR-V7âpositive men less likely to respond) and a larger trial would be needed to accurately quantitate any difference. \n",
      "Luo (2016) stated that prostate cancer cells demonstrate a remarkable \"addiction\" to androgen receptor (AR) signaling in all stages of disease progression.  As such, suppression of AR signaling remains the therapeutic goal in systemic treatment of prostate cancer.  A number of molecular alterations arise in patients treated with AR-directed therapies.  These molecular alterations may indicate the emergence of treatment resistance and may be targeted for the development of novel agents for prostate cancer.  The presence of functional androgen receptor splice variants may represent a potential explanation for resistance to abiraterone and enzalutamide, newer AR-directed agents developed to treat mCRPC.  In the last 8 years, many androgen receptor splice variants have been identified and characterized.  Among these, androgen receptor splice variant-7 (AR-V7) has been investigated extensively.  In AR-V7, the entire COOH-terminal ligand-binding domain of the canonical AR is truncated and replaced with a variant-specific peptide of 16 amino acids.  Functionally, AR-V7 is capable of mediating constitutive nuclear localization and androgen receptor signaling in the absence of androgens, or in the presence of enzalutamide.  Methods have been developed to detect AR-V7 in clinical mCRPC specimens.  The author concluded that AR-V7 can be reliably measured in both tissue and CTCs derived from mCRPC patients, making it possible to conduct both cross-sectional and longitudinal clinical correlative studies; current evidence derived from studies focusing on detection of AR-V7 in mCRPC support its potential clinical utility as a treatment selection marker.\n",
      "Ciccarese and co-workers (2016) noted that AR plays a key role in progression to mCRPC.  Despite the recent progress in targeting persistent AR activity with the next-generation hormonal therapies (abiraterone acetate and enzalutamide), resistance to these agents limits therapeutic efficacy for many patients.  Several explanations for response and/or resistance to abiraterone acetate and enzalutamide are emerging, but growing interest is focusing on importance of AR splice variants (AR-Vs) and in particular of AR-V7.  Increasing evidences highlighted the concept that variant expression could be used as a potential predictive biomarker and a therapeutic target in advanced prostate cancer.  The authors concluded that understanding the mechanisms of treatment resistance or sensitivity can help to achieve a more effective management of mCRPC, increasing clinical outcomes and representing a promising and engaging area of prostate cancer research.\n",
      "In a cross-sectional, cohort study, Scher and colleagues (2016) determined if pre-therapy nuclear AR-V7 protein expression and localization on CTCs is a treatment-specific marker for response and outcomes between androgen receptor signaling (ARS) inhibitors and taxanes.  A total of 265 men with progressive mCRPC undergoing a change in treatment were considered; 86 were excluded because they were not initiating ARS or taxane therapy; and 18 were excluded for processing time constraints, leaving 161 patients for analysis.  Between December 2012 and March 2015, blood was collected and processed from patients with progressive mCRPC immediately prior to new line of systemic therapy.  Patients were followed-up to 3 years.  Main outcome measures were (PSA response, time receiving therapy, radiographic PFS (rPFS), and OS.  Overall, of 193 prospectively collected blood samples from 161 men with mCRPC, 191 were evaluable (128 pre-ARS inhibitor and 63 pre-taxane).  AR-V7-positive CTCs were found in 34 samples (18 %), including 3 % of 1st-line, 18 % of 2nd-line, and 31 % of 3rd- or greater line samples.  Patients whose samples had AR-V7-positive CTCs before ARS inhibition had resistant post-therapy PSA changes (PTPC), shorter rPFS, shorter time on therapy, and shorter OS than those without AR-V7-positive CTCs.  Overall, resistant PTPC were seen in 65 of 112 samples (58 %) without detectable AR-V7-positive CTCs prior to ARS inhibition.  There were statistically significant differences in OS but not in PTPC, time on therapy, or rPFS for patients with or without pre-therapy AR-V7-positive CTCs treated with a taxane.  A multi-variable model adjusting for baseline factors associated with survival showed superior OS with taxanes relative to ARS inhibitors when AR-V7-positive CTCs were detected pre-therapy (HR, 0.24; 95 % CI: 0.10 to 0.57; pâ=â0.035).  The authors concluded that these findings validated CTC nuclear expression of AR-V7 protein in men with mCRPC as a treatment-specific biomarker that is associated with superior survival on taxane therapy over ARS-directed therapy in a clinical practice setting.  Moreover, they stated that continued examination of this biomarker in prospective studies will further aid clinical utility.  Moreover, they stated that given the magnitude of sub-stratification and outcome specificity of the nuclear specific AR-V7 protein test in CTCs, a diagnostic-grade test that informs the selection of ARS inhibitors or taxanes has the potential to significantly improve outcomes, by enabling patients to receive treatments to which they are most likely to respond while avoiding the toxic effects and costs associated with an ineffective treatment.  These investigators stated that prospective trials to validate these findings and further elucidate clinical utility are currently in development. An accompanying editorial (Montgomery and Plymate, 2016) noted that the assay used in this study may be less sensitive and specific than the assay used in the previously described studies by Antonorakis, et al. The editorialist observed that there was a lower proportion of patients with AR-V7âpositive CTCs at each line of therapy using the protein-based assay used by Sher, et al. compared with the mRNA-based assay used by Antonorakis, et al. Although the patient groups were not entirely identical, this difference in detection may have reflected a greater sensitivity provided by polymerase chain reaction (PCR) used by Antonorakis versus antibody detection systems used by Sher, et al. The editorialist also noted that issues of specificity of antibodies versus PCR are clearly different. The editorialist cited studies showing that AR-V7 antibodies can react with nonprostate tissue and cell lines in which no AR-V7 can be identified.\n",
      "In a cross-sectional, cohort study, Scher and colleagues (2017) evaluated if expanding the positivity criteria to include both nuclear and cytoplasmic AR-V7 localization (\"nuclear-agnostic\") identifies more patients who would benefit from a taxane over an androgen receptor signaling inhibitor (ARSi) such as abiraterone acetate, apalutamide, and enzalutamide.  Between December 2012 and March 2015,a total of  193 pre-therapy blood samples, 191 of which were evaluable, were collected and processed from 161 unique mCRPC patients before starting a new line of systemic therapy for disease progression at the Memorial Sloan Kettering Cancer Center.  The association between two AR-V7 scoring criteria, post-therapy PTPC and OS following ARSi or taxane treatment, was explored.  One criterion required nuclear-specific AR-V7 localization, and the other required an AR-V7 signal but was agnostic to protein localization in CTCs.  Correlation of AR-V7 status to PTPC and OS was investigated.  Relationships with survival were analyzed using multi-variable Cox regression and log-rank analyses.  A total of 34 (18 %) samples were AR-V7-positive using nuclear-specific criteria, and 56 (29 %) were AR-V7-positive using nuclear-agnostic criteria.  Following ARSi treatment, none of the 16 nuclear-specific AR-V7-positive samples and 6 of the 32 (19 %) nuclear-agnostic AR-V7-positive samples had greater than or equal to 50 % PTPC at 12 weeks.  The strongest baseline factor influencing OS was the interaction between the presence of nuclear-specific AR-V7-positive CTCs and treatment with a taxane (HR 0.24, 95 % CI: 0.078 to 0.79; p = 0.019).  This interaction was not significant when nuclear-agnostic criteria were used.  The authors stated that the results highlighted an important limitation of mRNA-based approaches in CTCs: the inability to determine whether the AR-V7 message has been translated into protein and, if so, whether the protein is present in the nucleus, where it is known to function as an oncogenic driver of tumor growth.  A prospective clinical trial testing the predictive capacity of pooled CTC AR-V7 mRNA and nuclear-specific AR-V7 protein in matched samples is ongoing.  They concluded that to reliably inform treatment selection using an AR-V7 protein biomarker in CTCs, nuclear-specific localization is needed. In an accompanying editorial, Lamb, et al. (2017) noted that a criticism of the study by Sher, et al. is that the âânuclearââ or ââagnosticââ status could be assigned based on a single CTC. \"We await further data to confirm whether a single cell really can reflect the predominant disease burden of an individual. In addition, it will be important to establish the biological explanation for these findings.\"\n",
      "Bernemann et al (2017) found that a subgroup of patients can benefit from abiraterone and/or enzalutamide despite detection of AR-V7 splice variants in their CTCs. The investigators assessed the response in a cohort of 21 AR-V7 PCR positive castration-resistant prostate cancer patients who had received therapy with abiraterone or enzalutamide. The investigators detected a subgroup of six AR-V7 positive patients showing benefit from either abiraterone or enzalutamide. Their progression free survival was 26 days (censored) to 188 days. Four patients displayed a prostate-specific antigen decrease of >50%. When analyzing prior therapies, the investigators noticed that only one of the six patients had received next-generation ADT prior to CTC collection. As a result, the investigators concluded that AR-V7 status in CTC cannot entirely predict nonresponse to next generation ADT, and AR-V7-positive patients should not be systematically denied abiraterone or enzalutamide treatment, especially as effective alternative treatment options are still limited.\n",
      "Scher et al (2018) examined if a validated assay for the AR-V7 protein in circulating tumor cells could determine differential OS among patients with mCRPC treated with taxanes versus ARSi.  This blinded, correlative study conducted from December 31, 2012, to September 1, 2016, included 142 patients with histologically confirmed mCRPC and who were treated at Memorial Sloan Kettering Cancer Center, the Royal Marsden, or the London Health Sciences Centre.  Blood samples were obtained prior to administration of ARS inhibitors or taxanes as a 2nd-line or greater systemic therapy for progressing mCRPC.  Main outcome measures were OS after treatment with an ARSi or taxane in relation to pre-therapy AR-V7 status.  Among the 142 patients in the study (mean [SD] age of 69.5 [9.6] years), 70 were designated as high-risk by conventional prognostic factors.  In this high-risk group, patients positive for AR-V7 who were treated with taxanes had superior OS relative to those treated with ARSi (median OS, 14.3 versus 7.3 months; HR, 0.62; 95 % CI: 0.28 to 1.39; pâ=â0.25).  Patients negative for AR-V7 who were treated with ARSi had superior OS relative to those treated with taxanes (median OS, 19.8 versus 12.8 months; HR, 1.67; 95 % CI: 1.00 to 2.81; pâ=â0.05).  The authors concluded that the findings of this study suggested that nuclear-localized AR-V7 protein in circulating tumor cells could identify patients who may live longer with taxane chemotherapy versus ARSi treatment.  Moreover, these researchers stated that a limitation of this study was that patients were not prospectively randomized to treatment based on the biomarker results, addressed in part through the use of risk scores in the analysis to mitigate confounding between treatment and underlying patient risk for which latent, unknown imbalances might not be captured by the included features.\n",
      "An editorial accompanying the study by Sher et al noted a number of challenges that remain. The stated that intrapatient heterogeneity with contemporaneous AR-V7âpositive and ARV7ânegative CTCs is an issue; follow up tissue studies need to explore intermetastatic and intrametastatic AR-V7 positivity as AR-V7âpositive patients may still have a sensitive cell population subset. The editorialists also stated that confidence in false-negative rates will require concurrent evaluation of CTC AR-V7 positivity in men with AR-V7âpositive tumor biopsies. Such studies must take into account the fact that many patients with CRPC, especially if they are chemotherapy naive, do not have CTCs in their blood, which is a cause of false-negative results. The editorialists stated that, in addition, there is a need to determine whether the AR-V7 assay is prognostic or predictive. Studies must correlate response to treatment with assay positivity, and not just survival data, to ensure that the assay is not simply a prognostic biomarker. The editorialists noted that AR-V7 positivity, in the study by Sher, et al, was associated with higher LDH, alkaline phosphatase, and PSA levels, suggesting a higher disease burden in the taxane arm. This finding indicates that AR-V7 positivity by this assay may be more prognostic, associated with disease burden, than predictive. Finally, this assay used by Sher, etal. is an antibody to AR-V7 cryptic exon 3 reported to be nonspecific, increasing the risk of false-positive results. This concern could be reduced by comparing this CTC AR-V7 protein assay used by Sher et al. with the CTC AR-V7 mRNA assay used in other studies.\n",
      "NCCN's clinical practice guideline on prostate cancer (2019) stated that: \"These clinical experiences suggest that AR-V7 assays are promising predictors of aberaterone and and enzalutamine resistance. Furthermore, the prevalance of AR-V7 positivity is only 3 % in patients prior to treatment with enzalutamide, abiraterone, and taxanes, so the panel believes that AR-V7 detection would not be useful to inform treatment decisions in the naive setting. On the other hand, the prevalence of AR-V7 positivity is higher after progression on abiraterone or enzalutamide (19 % - 39%), but data have already shown that abiraterone / enzalutamide cross-over therapy is rarely effective and taxanes are more effective in this setting. The panel recommends the use of AR-V7 tests can be considered to help guide selection of therapy in the post abiraterone / ensalutamide metastatic CRPC setting.\" \n",
      "Armstrong et al (2019) noted that AR-V7 results in a truncated receptor, which leads to ligand-independent constitutive activation that is not inhibited by anti-androgen therapies, including abiraterone or enzalutamide.  Given that previous reports suggested that circulating tumor cell (CTC) AR-V7 detection is a poor prognostic indicator for the clinical efficacy of secondary hormone therapies, these researchers conducted a prospective multi-center validation study.  PROPHECY is a multi-center, prospective-blinded study of men with high-risk metastatic castration-resistant prostate cancer (mCRPC) starting abiraterone acetate or enzalutamide treatment.  The primary objective was to validate the prognostic significance of baseline CTC AR-V7 on the basis of radiographic or clinical PFS by using the Johns Hopkins University modified-AdnaTest CTC AR-V7 mRNA assay and the Epic Sciences CTC nuclear-specific AR-V7 protein assay; OS and PSA responses were secondary end-points.  These researchers enrolled 118 men with mCRPC who were starting abiraterone or enzalutamide treatment; AR-V7 detection by both the Johns Hopkins and Epic AR-V7 assays was independently associated with shorter PFS (HR, 1.9 [95 % CI: 1.1 to 3.3; p = 0.032] and 2.4 [95 % CI: 1.1 to 5.1; p = 0.020], respectively) and OS (HR, 4.2 [95 % CI: 2.1 to 8.5] and 3.5 [95 % CI: 1.6 to 8.1], respectively) after adjusting for CTC number and clinical prognostic factors.  Men with AR-V7-positive mCRPC had fewer confirmed PSA responses (0 % to 11 %) or soft tissue responses (0 % to 6 %).  The observed percentage agreement between the 2 AR-V7 assays was 82 %.  The authors concluded that detection of AR-V7 in CTCs by 2 blood-based assays was independently associated with shorter PFS and OS with abiraterone or enzalutamide, and such men with mCRPC should be offered alternative treatments.\n",
      "Boeriggter et al (2019) noted that the study by Armstrong, et al. has some important limitations. First, the number of AR-v7 positive patients was relatively small and no alternative treatments were tested. Second, there were discrepancies in AR-v7 positive results between both tests. Eleven patients tested positive for AR-v7 with the Epic Sciences (antibody) test platform, while 28 patients were AR-v7 positive tested using the AdnaTest (PCR test). \"Taken together, AR-v7 detection is a promising predictive biomarker. Detection of AR-v7 in CTCs might be predictive for response to enzalutamide and abiraterone treatment, and it seems that for these patients taxane treatment should be preferred. However, large prospective clinical trials are needed to validate the AdnaTest and Epic Sciences platform to confirm their clinical utility for mCRPC patients.\"\n",
      "Sciarra et al (2019) sought to critically analyze: frequency of the AR-V7 expression in metastatic castration-resistant prostate cancer (mCRPC) cases-impact of AR-V7 expression on abiraterone, enzalutamide, and taxane therapy. The authors searched in the Medline and Cochrane Library database from the literature of the past 10 years. The authors critically evaluated the level of evidence according to the European Association of Urology (EAU) guidelines. Twelve clinical trials were selected. The authors said that the determination of AR-V7 in peripheral blood using circulating tumor cells mRNA seems to be the preferred method. At baseline, the mean percentage of cases with AR-V7 positivity was 18.3% (range 17.8%-28.8%). All data on mCRPC submitted to enzalutamide or abiraterone reported a significantly (Pâ<.05) lower clinical progression-free survival (CPFS) and overall survival (OS) in AR-V7+ than AR-V7- cases (CPFS hazard ratio [HR]: 2.3; 95% CI 1.1-4.9; OS HR: 3.0; 95% CI 1.4-6.3). In mCRPC cases submitted to chemotherapies data are not homogeneous and some studies showed no association between CPFS or OS and AR-V7 status (OS HR 1.6; 95% CI 0.6-4.4; P = .40). The authors concluded that the suggestion is that taxane therapy is more efficacious than abiraterone or enzalutamide for men with AR-V7+ CRPC. The authors found, on the contrary, that clinical outcomes did not seem to differ significantly on the basis of the type of therapy used among AR-V7- cases.\n",
      "Prostavysion\n",
      "ProstaVysion (Bostwick Labs) is a prognostic genetic panel for prostate cancer (Raman, et al., 2013). This test examines two major mechanisms of prostate carcinogenesis: ERG gene fusion/translocation and the loss of the PTEN tumor suppressor gene. This test is a tissue-based panel. By examining these two markers, ProstaVysion is able to provide a molecular analysis of prostate cancer aggressiveness and long-term patient prognosis. ERG gene fusions are found in 40% of primary prostate cancers and are associated with a more aggressive phenotype. Deletion of PTEN occurs in both localized prostate cancers and 60% of metastases.\n",
      "PAM50 and Prosigna\n",
      "PAM50 Breast Cancer Intrinsic Classifier (University of Utah) examines 50 genes and sorts breast cancer into four subtypes (Raman, et al., 2013). Each subtype responds differently to standard therapies, and knowing the subtype allows doctors to tailor treatment for each patient. PAM50 assay can aid profiling for both prognosis and prediction of benefit from adjuvant tamoxifen and has been found superior to immunohistochemistry.\n",
      "A National Institute for Health Research assessment (Ward, et al., 2013) found the evidence for PAM50 to be limited. The report concluded that \"the evidence base for PAM50 is still relatively immature.\"\n",
      "An international working group (Azim, et al, 2013) found insufficient evidence of the analytic and clinical validity of the PAM50. They found insufficient evidence of the clinical utility of the PAM50 or the other breast cancer genomic tests that they assessed.\n",
      "A report by the Belgian Healthcare Knowledge Centre (KCE) (San MIguel, et al., 2015) found that the evidence for PAM50 is limited to studies supporting the prognostic ability (clinical validity) of the test. Most of the evidence is in node-positive patients. The KCE found insufficient evidence on the impact of PAM50 on clinical management (clinical utility).\n",
      "Prosigna is intended for use as a prognostic indicator in conjunction with other clinicopathologic factors for distant recurrenceâfree survival at 10 years in postmenopausal women with hormone receptor (HR)âpositive, lymph nodeânegative/stage I or II, or lymph nodeâpositive (1â3 positive nodes)/stage II breast cancer to be treated with adjuvant endocrine therapy alone. The assay measures the expression profiles of genes included in the PAM50 gene signature, as well as 8 housekeeping genes (for normalization), 6 positive controls and 8 negative controls.\n",
      "The BlueCross BlueShield Association (2015) concluded that the use of Prosigna to determine recurrence risk in women with early-stage breast cancer does not meet the TEC criteria. The evidence is insufficient to permit conclusions regarding health outcomes. Assay performance characteristics of the commercially available version of the test indicate high reproducibility.\n",
      "A medical technology innovation briefing by the National Institute for Health and Clinical Excellence (NICE, 2015) noted that none of the women analyzed in the clinical validation studies (citing Gnant, et al. (2014), Sestak, et al. (2015) and Dowsett et al. (2013)) had chemotherapy as part of their initial treatment. As a result, the prognostic value of the Prosigna ROR score in a chemotherapy-treated population is unknown.  The briefing also noted that the populations included in the patient cohorts included in these clinical validation studies. Sestak, et al. (2015) combined data previously analysed by Dowsett et al. (2013) and Gnant et al. (2014). Dowsett et al. (2013) and Sestak et al. (2015) used the clinical treatment score as a comparator rather than the online tools Adjuvant! Online and PREDICT, or the NPI, which are standard practice in the UK. Similarly, Gnant et al. (2014) used a combination score of clinicopathologic parameters as the comparator for Prosigna. NICE stated that such indices are always incomplete because they may not include all parameters used by clinicians in other health systems to aid clinical decision-making. The NICE briefing also pointed out that all included studies received financial support or disclosed competing interests from the manufacturer, and this introduces the potential for bias in the reporting of outcomes.\n",
      "Guidelines from the American Society for Clinical Oncology (2016) state: \"If a patient has ER/PgR-positive, HER2-negative (node-negative) breast cancer, the clinician may use the PAM50 risk of recurrence (ROR) score (Prosigna Breast Cancer Prognostic Gene Signature Assay; NanoString Technologies, Seattle,WA), in conjunction with other clinicopathologic variables, to guide decisions on adjuvant systemic therapy.\" This is a strong recommendation based upon high quality evidence. The guidelines recommend against the use of PAM50 to guide decisions on adjuvant systemic therapy in patints with ER/PgR-positive, HER2-negative (node-positive) breast cancer. The guidelines also recommend against the use of PAM 50 in HER2-positive breast cancer and TN breast cancer. The guidelines recommended against the use of PAM50 to guide decisions on extended endocrine therapy for patients with ER/PgR-positive, HER-2 negative (node-negative) breast cancer who have had 5 years of endocrine therapy without evidence of recurrence. \n",
      "Cancer Care Ontario Guidelines (Chang, et al., 2016) state: \"Although no assay represents a gold standard, Oncotype DX is supported by the widest range of evidence for prognosis and prediction of chemotherapy benefit, while both Prosigna and EndoPredict have evidence-based validity in providing some of the same or similar clinical information.\"\n",
      "OncotypeDx Colon\n",
      "The Oncotype Dx Colon has been promoted for use in colorectal cancer. However, there is a lack of evidence establishing the clinical utility of this test in colorectal cancer.\n",
      "The results of the Quick and Simple and Reliable Study (QUASAR) were published by Gray et al (2011).  The purpose of the QUASAR study was develop quantitative gene expression assays to assess recurrence risk and benefits from chemotherapy in patients with stage II colon cancer.  Recurrence score (RS) and treatment score (TS) were calculated from gene expression levels of 13 cancer-related genes and from five reference genes.  The results of the study showed risk of recurrence to be significantly associated with RS (95 % confidence interval [CI]: 1.11 to 1.74; p = 0.004).  Recurrence risks at 3 years were 12 %, 18 %, and 22 % for predefined low, intermediate, and high recurrence risk groups, respectively.  Continuous RS was associated with risk of recurrence (p = 0.006), but there was no trend for increased benefit from chemotherapy at higher TS (p = 0.95).  The continuous 12-gene RS has been validated in a prospective study for assessment of recurrence risk in patients with stage II colon cancer after surgery.  RS was also found to provide prognostic value that complements T stage and mismatch repair.\n",
      "Yamanaka et al (2016) evaluated the 12-gene Recurrence Score assay for stage II and III colon cancer without chemotherapy to reveal the natural course of recurrence risk in stage III disease. A cohort-sampling design was used. From 1,487 consecutive patients with stage II to III disease who had surgery alone, 630 patients were sampled for inclusion with a 1:2 ratio of recurrence and nonrecurrence. Sampling was stratified by stage (II v III). The assay was performed on formalin-fixed, paraffin-embedded primary cancer tissue. Association of the Recurrence Score result with recurrence-free interval (RFI) was assessed by using weighted Cox proportional hazards regression. Overall, 597 of 630 patients were analyzable-247 patients had stage II, and 350 had stage III colon cancer. The continuous Recurrence Score was significantly associated with RFI after adjustment for disease stage (hazard ratio for a 25-unit increase in Recurrence Score, 2.05; 95% CI, 1.47 to 2.86; P < .001). With respect to prespecified subgroups, as defined by low (< 30), intermediate (30 to 40), and high (â¥ 41) Recurrence Score risk groups, patients with stage II disease in the high-risk group had a 5-year risk of recurrence similar to patients with stage IIIA to IIIB disease in the low-risk group (19% v 20%), whereas patients with stage IIIA to IIIB disease in the high-risk group had a recurrence risk similar to that of patients with stage IIIC disease in the low-risk group (approximately 38%).\n",
      "The authors concluded that this study provides the first validation of the 12-gene Recurrence Score assay in stage III colon cancer without chemotherapy and showed the heterogeneity of recurrence risks in stage III as well as in stage II colon cancer. \n",
      "The NCCNâs clinical practice guideline on \"Colon cancer\" (Version 2.2015) states that there are insufficient data to recommend the use of multi-gene assays (e.g., the Oncotype DX colon cancer assay) to determine adjuvant therapy.\n",
      "An assessment prepared for the Agency for Healthcare Research and Quality (Meleth,, 2014) stated: \"For CRC, evidence did not adequately support added prognostic value for Oncotype DX Colon. evidence either did not support added prognostic value or we found no studies with sufficiently low RoB to support a conclusion about prognostic value.\"\n",
      "The Institut national dâexcellence en santÃ© et services sociaux (INESSS) (Boily, et al., 2016) reviewed the data on Oncotype Dx Colon, noting that the Oncotype Dx Colon has prognostic value in stage II colon cancer, but is not currently reimbursed in Quebec.\n",
      "Decipher\n",
      "The Decipher test appears to be a RNA biomarkers \"assay\" for prostate cancer.  Decipher does this by measuring the expression levels of 22 RNA biomarkers involved in multiple biological pathways across the genome that are associated with aggressive prostate cancer.\n",
      "Studies of the Decipher genetic test have evaluated its correlation with tumor characteristics (Den, et al., 2016; Klein, et al., 2016) and reported on the use of this gene panel to predict biochemical recurrence, metastatic progression, and disease-specific survival after radical prostatectomy with or without external beam radiotherapy (Ehro, et al., 2013; Den, et al., 2013; Cooperberg, et al., 2015; Ross, et al., 2014; Klein, et al., 2015; Karnes, et al., 2013; Den, et al., 2014; Den, et al., 2015; Lee, et al., 2016; Klein, et al., 2016; Glass, et al., 2016; Freedland, et al., 2016; Ross, et al., 2016). The impact of Decipher was evaluated in a clinical utility study where 21 uro-oncologists were presented 24 patient cases (12 potential candidates for adjuvant and 12 for salvage external beam radiation therapy) and were asked for treatment recommendations with and without information from the genetic test (Badani, et al., 2013). The recommendation changed in 43% of the adjuvant cases and 53% in the salvage setting, suggesting a potentially significant impact on treatment decisions after radical prostatectoy.\n",
      "Michalopoulos et al (2014) reported that the Decipher genomic classifier was useful in the clinic when used as a part of the risk stratification in recommending adjuvant radiation to patients with high-risk pathologic features. In that study, 43% of patients shifted to observation based on information of Decipher genomic classifier after radical prostatectomy. The authors stated that this study had several drawbacks.  First, treatment recommendations were submitted using an electronic DCI, rather than tracking actual treatment administered per review of medical charts.  Thus, treatment recommendations may vary from actual practice patterns.  Second, these researchers were unable to assess the patientâs influence on the treatment decision-making process.  Third, the association between genomic classifier (GC) test results and treatment recommendations was determined using \"early adopters\" of the test.  Therefore, the treatment recommendations may not be indicative of physician decision-making among other physicians in the field; for example, those who were hesitant or reluctant to modify their practice patterns, or those who were not aware of the availability of the GC test.  It should be noted, however, that participants were community-based physicians rather than those with appointments at academic/research centers, and treatment strategies may deviate from standard practice.  Last, the influence of practice management decisions on patient outcomes would involve initially inferences based on \"chain-of-evidence\" and related decision-analytical principles commonly used by the highly regarded USPSTF and recently endorsed in an editorial discussion regarding the research agenda following a large clinical validity trial of a biomarker test for colon cancer screening.  Consensus is emerging that long-term, prospective studies in diverse settings will optimize generalizable knowledge to inform best practices for such technologies.  This approach and other recommendations for evidence development was recently recommended by a diverse and independent group of researchers, insurers, and policy-makers addressing the challenges of translating the promise of tumor biology research into practice.  Studies in other settings, funded by diverse independent sources (e.g., NIH), would be appropriate and warranted.\n",
      "Nguyen et al (2015) examined how the results of the Decipher test altered recommendations of radiation oncologists and urologists for adjuvant treatment of 10 patients status post RP for prostate cancer. Using clinical information alone, observation was recommended in 42% of decisions made by urologists versus 23% by radiation oncologists (P < .0001). The GC test results altered 35% and 45% of treatment recommendations made by radiation oncologists and urologists, respectively. Badani, et al. (2015) reported on a study where 51 urologists provided treatment recommendations for patients with high-risk prostate cancer with and without Decipher GC test results. Each urologist was asked to provide treatment recommendations on 10 cases randomly drawn from a pool of 100 case histories. Without Decipher GC test result knowledge, observation was recommended for 57% (n = 303), adjuvant radiation therapy (ART) for 36% (n = 193) and other treatments for 7% (n = 34) of patients. Overall, 31% (95% CI: 27-35%) of treatment recommendations changed with knowledge of the Decipher GC test results. However, the long-term impact of these changes in management is unknown (Bostrom et al, 2015). \n",
      "Spratt et al (2017) performed an individual patient-level metaanalysis of the performance of the Decipher genomc classifier in high-risk men after prostatectomy to predict the development of metastatic disease. MEDLINE, EMBASE, and the Decipher genomic resource information database were searched for published reports between 2011 and 2016 of men treated by prostatectomy that assessed the benefit of the Decipher test. Multivariable Cox proportional hazards models fit to individual patient data were performed; meta-analyses were conducted by pooling the study-specific hazard ratios (HRs) using random-effects modeling. Extent of heterogeneity between studies was determined with the I(2) test. Five studies (975 total patients, and 855 patients with individual patient-level data) were eligible for analysis, with a median follow-up of 8 years. Of the total cohort, 60.9%, 22.6%, and 16.5% of patients were classified by Decipher as low, intermediate, and high risk, respectively. The 10-year cumulative incidence metastases rates were 5.5%, 15.0%, and 26.7% ( P < .001), respectively, for the three risk classifications. Pooling the study-specific Decipher HRs across the five studies resulted in an HR of 1.52 (95% CI, 1.39 to 1.67; I(2) = 0%) per 0.1 unit. In multivariable analysis of individual patient data, adjusting for clinicopathologic variables, Decipher remained a statistically significant predictor of metastasis (HR, 1.30; 95% CI, 1.14 to 1.47; P < .001) per 0.1 unit. The C-index for 10-year distant metastasis of the clinical model alone was 0.76; this increased to 0.81 with inclusion of Decipher. The authors concluded that the Decipher test can improve prognostication of patients postprostatectomy. The authors stated that future study of how to best incorporate genomic testing in clinical decision-making and subsequent treatment recommendations is warranted.\n",
      "Dalela et al (2017) aimed to develop and internally validate a risk-stratification tool incorporating the Decipher score, along with routinely available clinicopathologic features, to identify patients who would benefit the most from postoperative adjuvant radiation therapy. Patient and Methods Our cohort included 512 patients with prostate cancer treated with radical prostatectomy at one of four US academic centers between 1990 and 2010. All patients had â¥ pT3a disease, positive surgical margins, and/or pathologic lymph node invasion. Multivariable Cox regression analysis tested the relationship between available predictors (including Decipher score) and clinical recurrence (CR), which were then used to develop a novel risk-stratification tool. Overall, 21.9% of patients received adjuvant radiotherapy. Median follow-up in censored patients was 8.3 years. The 10-year CR rate was 4.9% vs. 17.4% in patients treated with adjuvant radiotherapy versus initial observation ( P < .001). Pathologic T3b/T4 stage, Gleason score 8-10, lymph node invasion, and Decipher score > 0.6 were independent predictors of CR (all P < .01). The cumulative number of risk factors was 0, 1, 2, and 3 to 4 in 46.5%, 28.9%, 17.2%, and 7.4% of patients, respectively. Adjuvant radiotherapy was associated with decreased CR rate in patients with two or more risk factors (10-year CR rate 10.1% in aRT v 42.1% in initial observation; P = .012), but not in those with fewer than two risk factors (P = 0.18). The investigators concluded that, using the new model to indicate adjuvant radiotherapy might reduce overtreatment, decrease unnecessary adverse effects, and reduce risk of CR in the subset of patients (approximately 25% of all patients with aggressive pathologic disease in our cohort) who benefit from this therapy. \n",
      "In a review of genomic predictors of outcome in prostate cancer, Bostrom et al (2015) noted that the Decipher test, like other gene panels (Prolaris, Oncotype DX Genomic Prostate Score) have been evaluated in terms of potential prognostic value after RP. The future will tell if this additional information is considered sufficient by the urologic and prostate cancer patients to change practice (Bostrom, et al., 2015; Nguyen, et al., 2015). Bostrom, et al. (2015) commented: \"Although clinical studies have suggested potential benefits with these tests, real clinical use and long-term data are needed to judge the added value.\"\n",
      "Guidelines on localized prostate cancer from the American Urologic Association (Sanda, et al., 2017) state based upon expert opinion that, among most low-risk localized prostate cancer patients, Decipher and other tissue based genomic biomarkers have not shown a clear role in the selection of candidates for active surveillance. The guidelines also indicate that tissue based genomic biomarkers are not necessary for followup. The guidelines state that the Decipher test has not been validated as providing substantial benefit in the active surveillance population.\n",
      "Furthermore, NCCNâs clinical practice guideline on \"Prostate cancer\" (Version 4.2018) states that \"These molecular biomarker tests have been developed with extensive industry support, guidance, and involvement, and have been marketed under the less rigorous FDA regulatory pathway for biomarkers.  Although full assessment of their clinical utility requires prospective randomized clinical trials, which are unlikely to be done, the panel believes that men with low or favorable intermediate disease may consider the use of Decipher, Oncotype DX Prostate, Prolaris, or ProMark during initial risk stratification.  In addition, Decipher may be considered during workup for radical prostatectomy PSA persistence or recurrence (category 2B).  Future comparative effectiveness research may allow these tests and others like them to gain additional evidence regarding their utility for better stratification of men with prostate cancer\".\n",
      "Klein et al (2016) evaluated the ability of the Decipher genomic classifier in predicting metastasis from analysis of prostate needle biopsy diagnostic tumor tissue specimens.  A total of 57 patients with available biopsy specimens were identified from a cohort of 169 men treated with radical prostatectomy (RP) in a previously reported Decipher validation study at Cleveland Clinic.  A Cox multivariable proportional hazards model and survival C-index were used to evaluate the performance of Decipher.  With a median follow-up of 8 years, 8 patients metastasized and 3 died of prostate cancer.  The Decipher plus NCCN model had an improved C-index of 0.88 (95 % CI: 0.77 to 0.96) compared to NCCN alone (C-index 0.75, 95 % CI: 0.64 to 0.87).  On multi-variable analysis (MVA), Decipher was the only significant predictor of metastasis when adjusting for age, pre-operative PSA and biopsy Gleason score (Decipher HR per 10 % increase 1.72, 95 % CI: 1.07 to 2.81, pâ=â0.02).  The authors concluded that biopsy Decipher predicted the risk of metastasis at 10 years post-radical prostatectomy.  These researchers stated that while further validation is needed on larger cohorts, pre-operative knowledge of Decipher risk derived from biopsy could indicate the need for multi-modality therapy and help set patient expectations of therapeutic burden.\n",
      "Nguyen et al (2017a) examined the ability of a biopsy-based 22-marker genomic classifier (GC) to predict for distant metastases after radiation and a median of 6 months of androgen deprivation therapy (ADT).  These researchers studied 100 patients with intermediate-risk (55 %) and high-risk (45 %) prostate cancer who received definitive radiation plus a median of 6 months of ADT (range of 3 to 39 months) from 2001 to 2013 at a single center and had available biopsy tissue.  Six to ten 4-micron sections of the needle biopsy core with the highest Gleason score and percentage of tumor involvement were macro-dissected for RNA extraction; GC scores (range of 0.04 to 0.92) were determined.  The primary end-point of the study was time to distant metastasis.  Median follow-up was 5.1 years.  There were 18 metastases during the study period.  On uni-variable analysis (UVA), each 0.1 unit increase in GC score was significantly associated with time to distant metastasis (HR: 1.40 (1.10 to 1.84), p = 0.006) and remained significant after adjusting for clinical variables on MVA (adjusted HR: 1.36 (1.04 to 1.83), p = 0.024).  The c-index for 5-year distant metastasis was 0.45 (95 % CI: 0.27 to 0.64) for Cancer of the Prostate Risk Assessment score, 0.63 (0.40 to 0.78) for NCCN risk groups, and 0.76 (0.57 to 0.89) for the GC score.  Using pre-specified GC risk categories, the cumulative incidence of metastasis for GC>0.6 reached 20 % at 5 years after radiation (p = 0.02).  The authors believed this was the first demonstration of the ability of the biopsy-based GC score to predict for distant metastases after definitive radiation and ADT for intermediate- and high-risk prostate cancer.  Patients with the highest GC risk (GC greater than 0.6) had high rates of metastasis despite multi-modal therapy suggesting that they could potentially be candidates for treatment intensification and/or enrollment in clinical trials of novel therapy.\n",
      "The authors stated that a limitation of this study was the size of the data-set and length of follow-up, and thus further studies are needed to validate these findings in larger data sets with longer follow-up.  In addition, the hypothesis that patients with very low GC score of less than or equal to 0.2 may be able to omit ADT requires further testing in previously treated cohorts as well as in prospective studies, which are currently being planned.  Finally, while the GC score was prognostic for distant metastasis, it did not have a significant association with biochemical recurrence.  This may reflect the fact that the test was originally developed to specifically predict for distant metastasis and generally only a minority of biochemical recurrences will lead to distant metastasis.  Another consideration was that none of the patients in this study received a multi-parametric MRI, and it has previously been shown that MRI could add some prognostic information to clinical variables through up-staging or through detecting potentially higher grade lesions.  These researchers stated that future studies should evaluate how GC adds to prognostic value when multiparametric MRI has also been performed, although studies with the cell-cycle progression score suggested that the MRI and the genomic information are capturing different types of prognostic information.\n",
      "Nguyen et al (2017b) evaluated the ability of biopsy Decipher to predict metastasis and prostate cancer-specific mortality (PCSM) in primarily intermediate- to high-risk patients treated with RP or radiation therapy (RT).  A total of 235 patients treated with either RP (n = 105) or RT Â± androgen deprivation therapy (n = 130) with available genomic expression profiles generated from diagnostic biopsy specimens from 7 tertiary referral centers were included in this study.  The highest-grade core was sampled and Decipher was calculated based on a locked random forest model.  Metastasis and PCSM were the primary and secondary outcomes of the study, respectively.  Cox analysis and c-index were used to evaluate the performance of Decipher.  With a median follow-up of 6 years among censored patients, 34 patients developed metastases and 11 died of prostate cancer.  On MVA, biopsy Decipher remained a significant predictor of metastasis (HR: 1.37 per 10 % increase in score, 95 % CI]: 1.06 to 1.78, p = 0.018) after adjusting for clinical variables.  For predicting metastasis 5-year post-biopsy, Cancer of the Prostate Risk Assessment score had a c-index of 0.60 (95 % CI: 0.50 to 0.69), while Cancer of the Prostate Risk Assessment plus biopsy Decipher had a c-index of 0.71 (95 % CI: 0.60 to 0.82); NCCN risk group had a c-index of 0.66 (95 % CI: 0.53 to 0.77), while NCCN plus biopsy Decipher had a c-index of 0.74 (95 % CI: 0.66 to 0.82).  Biopsy Decipher was a significant predictor of PCSM (HR: 1.57 per 10 % increase in score, 95 % CI: 1.03 to 2.48, p = 0.037), with a 5-year PCSM rate of 0 %, 0 %, and 9.4 % for Decipher low, intermediate, and high, respectively.  The authors concluded that these findings suggested that patients classified as Decipher high risk have a very high 5-year risk of distant metastasis (21 %) and PCSM (9.4 %), and may therefore be rationally subjected to multi-modal therapy or enrolled into clinical trials targeting men with highest-risk disease.\n",
      "These researchers stated that the cohort size of this study was limited by access to biopsy tissue from community and referral health centers; 93 % of the unavailable cohort were either unavailable or had inadequate tissue and 7.4 % failed RNA extraction.  Of the 909 patients eligible for this study, only 235 had biopsy tissue available from the institution in which the RPs or RTâ",
      "Â±â",
      "ADT were performed.  A larger cohort size with longer follow-up would have strengthened this study and might have given more PCSM events to allow for a MVA of predictors of PCSM rather than only univariable analysis.  Running multi-variable models on a relatively small number of events could also lead to issues with validity, and that was why these investigators fit the Cox models using an adaptation of Firth's penalized approach that was designed to minimize bias in this scenario.  Another limitation of this study was that the majority of patients were NCCN intermediate- or high-risk and the authors were unable to draw conclusions regarding Decipher among low-risk patients, who only represented 10 % of the patients in the study.  Such information would be useful to guide decisions about treatment versus active surveillance.  Finally, research is ongoing to determine the concordance between Decipher scores derived from biopsy versus prostatectomy samples, which has been reported in prior small studies to be 64 %, 75 %, and 86 %.\n",
      "Gore et al (2017) stated that patients with PCa and their providers face uncertainty as they consider ART or salvage radiotherapy (SRT) after undergoing radical prostatectomy.  These researchers prospectively evaluated the impact of the Decipher test, which predicts metastasis risk after radical prostatectomy, on decision-making for ART and SRT.  A total of 150 patients who were considering ART and 115 who were considering SRT were enrolled.  Providers submitted a management recommendation before processing the Decipher test and again at the time of receipt of the test results.  Patients completed validated surveys on PCa-specific decisional effectiveness and PCa-related anxiety.  Before the Decipher test, observation was recommended for 89 % of patients considering ART and 58 % of patients considering SRT.  After Decipher testing, 18 % (95 % CI: 12 % to 25 %) of treatment recommendations changed in the ART arm, including 31 % among high-risk patients; and 32 % (95 % CI: 24 % to 42 %) of management recommendations changed in the salvage arm, including 56 % among high-risk patients.  Decisional Conflict Scale (DCS) scores were better after viewing Decipher test results (ART arm: median DCS before Decipher, 25 and after Decipher, 19 [p < 0.001]; SRT arm: median DCS before Decipher, 27 and after Decipher, 23 [p < 0.001]).  PCa-specific anxiety changed after Decipher testing; fear of PCa disease recurrence in the ART arm (p = 0.02) and PCa-specific anxiety in the SRT arm (p = 0.05) decreased significantly among low-risk patients.  Decipher results reported per 5 % increase in 5-year metastasis probability were associated with the decision to pursue ART (OR, 1.48; 95 % CI: 1.19 to 1.85) and SRT (OR, 1.41; 95 % CI: 1.09 to 1.81) in multi-variable logistic regression analysis.  The authors concluded that knowledge of Decipher test results was associated with treatment decision-making and improved decisional effectiveness among men with PCa who were considering ART and SRT.  These researchers stated that the Decipher test has the potential to be an important adjunct to clinical decision-making in men with adverse pathology or a rising PSA after undergoing radical prostatectomy for PCa.\n",
      "The authors stated that this study had several drawbacks.  First, they were presenting interim data regarding treatment recommendations, which may not correlate with the actual treatment received.  Final analysis of the current study will identify treatments received within 12 months of Decipher testing.  Second, patients were their own controls; these researchers did not include a group unexposed to Decipher testing.  Patients who have additional time to consider their clinical and pathological characteristics may have decisional effectiveness changes parallel with the current study findings.  Third, patients in the SRT arm had heterogeneous time since prostatectomy, which may influence treatment recommendations independent of Decipher testing results.  Last, to the best of their knowledge, no genomic test to-date has been validated against a control to demonstrate that use of the test improved PCa-specific outcomes.\n",
      "Spratt et al (2018) noted that it is clinically challenging to integrate genomic-classifier results that report a numeric risk of recurrence into treatment recommendations for localized prostate cancer, which are founded in the framework of risk groups.  These investigators developed a novel clinical-genomic risk grouping system that can readily be incorporated into treatment guidelines for localized prostate cancer.  Two multi-center cohorts (n = 991) were used for training and validation of the clinical-genomic risk groups, and 2 additional cohorts (n = 5,937) were used for re-classification analyses.  Competing risks analysis was used to estimate the risk of distant metastasis.  Time-dependent c-indices were constructed to compare clinicopathologic risk models with the clinical-genomic risk groups.  With a median follow-up of 8 years for patients in the training cohort, 10-year distant metastasis rates for NCCN low, favorable-intermediate, unfavorable-intermediate, and high-risk were 7.3 %, 9.2 %, 38.0 %, and 39.5 %, respectively.  In contrast, the 3-tier clinical-genomic risk groups had 10-year distant metastasis rates of 3.5 %, 29.4 %, and 54.6 %, for low-, intermediate-, and high-risk, respectively, which were consistent in the validation cohort (0 %, 25.9 %, and 55.2 %, respectively).  C-indices for the clinical-genomic risk grouping system (0.84; 95 % CI: 0.61 to 0.93) were improved over NCCN (0.73; 95 % CI: 0.60 to 0.86) and Cancer of the Prostate Risk Assessment (0.74; 95 % CI: 0.65 to 0.84), and 30 % of patients using NCCN low/intermediate/high would be re-classified by the new 3-tier system and 67 % of patients would be re-classified from NCCN 6-tier (very-low- to very-high-risk) by the new 6-tier system.  The authors concluded that a commercially available genomic classifier in combination with standard clinicopathologic variables could generate a simple-to-use clinical-genomic risk grouping that more accurately identifies patients at low-, intermediate, and high-risk for metastasis and can be easily incorporated into current guidelines to better risk-stratify patients.\n",
      "The authors noted that they did not include a separate NCCN very-high-risk category in their model for several reasons.  First, although these men have poor oncologic outcomes, there is a lack of consensus for the definition of very-high-risk disease and thus, it was not included in American Urological Association/American Society for Radiation Oncology/Society of Urologic Oncology 2017 guidelines.  Second, only 1.5 % of their training cohort was NCCN very-high-risk.  In contrast, 25.7 % of their training cohort was clinical-genomic high-risk, which has significantly worse outcomes than the NCCN high-risk group, and thus, these researchers have identified a much larger group of patients with very poor outcomes.  Lastly, a potential source of bias that was present in this retrospective cohort was that the samples analyzed were typically older than 10 years.  Thus, it was possible that samples with larger tumor burden were more likely to be analyzed successfully.  This may explain why the event rates were generally higher than comparable clinical trial series.  This was in contrast to normal clinical use tissue, which has a high pass rate, even for patients with NCCN at very-low risk.  Given constant stage and grade migration, it was challenging to simultaneously have modern patients who also have long-term outcomes.  For example, 12-year outcomes were recently reported from Radiation Therapy Oncology Group (RTOG) 9601, a trial that started over 20 years ago.  These investigators stated that despite these drawbacks, it will be important for continued validation of their clinical-genomic risk system.\n",
      "Berlin et al (2019) stated that NCCN has recently endorsed the stratification of intermediate-risk prostate cancer (IR-PCa) into favorable and unfavorable subgroups and recommended the addition of ADT to RT for unfavorable IR-PCa.  Recently, more accurate prognostication was demonstrated by integrating a 22-feature GC to the NCCN stratification system.  These researchers tested the utility of the GC to better identify patients with IR-PCa who are sufficiently treated by RT alone.  They identified a novel cohort comprising 121 patients with IR-PCa treated with dose-escalated image guided RT (78 Gy in 39 fractions) without ADT.  GC scores were derived from tumors sampled in diagnostic biopsies; MVAs, including both NCCN sub-classification and GC scores, were performed for biochemical failure (PSA nadir + 2 ng/ml) and metastasis occurrence.  By NCCN sub-classification, 33 (27.3 %) and 87 (71.9 %) of men were classified as having favorable and unfavorable IR-PCa, respectively (1 case unclassifiable).  GC scores were high in 3 favorable IR-PCa and low in 60 unfavorable IR-PCa.  Higher GC scores, but not NCCN risk subgroups, were associated with biochemical relapse (HR, 1.36; 95 % CI: 1.09 to 1.71] per 10 % increase; p = 0.007) and metastasis (HR, 2.05; 95 % CI: 1.24 to 4.24; p = 0.004).  GC predicted biochemical failure at 5 years (area under the curve [AUC], 0.78; 95 % CI: 0.59 to 0.91), and the combinatorial NCCN + GC model significantly out-performed the NCCN alone model for predicting early-onset metastasis (AUC for 5-year metastasis of 0.89 versus 0.86 [GC alone] versus 0.54 [NCCN alone]).  The authors demonstrated the accuracy of the GC for predicting disease recurrence in IR-PCa treated with dose-escalated image guided RT alone.  The authors concluded that these findings high-lighted the need to evaluate this GC in a prospective clinical trial examining the role of ADT-RT in clinicogenomic-defined IR-PCa subgroups.\n",
      "The authors stated that this study had several drawbacks.  First, it was arguable that this study was under-powered given the modest sample size and consequently few metastatic events.  Nonetheless, this patient cohort was identified from a prospective registry with stringent inclusion criteria of adequate diagnostic biopsy tissue for central pathology review and sampling, omission of concurrent ADT with RT, and contemporary RT dose intensity and technique of 78 Gy in 39 fractions delivered using IGRT.  This reflected real life clinical practice, and the fact that the GC was robust for prognosticating these patients was compelling evidence for its routine clinical implementation in men with IR-PCa treated with RT.  Next, although these researchers acknowledged that the addition of ADT to RT for unfavorable IR-PCa disease was considered standard practice by several institutions, it remains debatable whether the reported benefits of the combinatorial approach are maintained in the context of RT dose-escalation.  Presently, the European Organization for Research and Treatment of Cancer 22991 phase-III clinical trial provides the main supportive evidence specific to this clinical conundrum.  However, it must be noted that the trial's cohort also consisted of 25 % NCCN-defined high-risk patients, treatment schedules with minor dose escalation (i.e., 70 Gy or 74 Gy in more than 75 % of cases), and treatment delivered without image guidance; all elements could in part explain the poor outcomes in the EBRT-alone control arm.  Thus, at the time of the present study, the authorsâ practice for clinical management of IR-PCa remained largely unchanged, and the low rates of biochemical relapse and metastatic events observed supported this approach.  For example, the 5-year biochemical relapse-free rates in this series were 94 % and 88 % for the favorable and unfavorable subgroups, respectively, mirroring the 87 % reported in the DE-RT arm from the Radiation Therapy Oncology Group 0126 trial in predominantly favorable IR-PCa.  Nevertheless, the authors could not completely exclude the presence of selection bias within this cohort; in fact, during the last few years their practice has increasingly embraced the combination of DE-IGRT and short-term ADT, especially in IR-PCa harboring unfavorable indices and/or other aggressive features such as intra-ductal and cribiform subpathologies.  Finally, although these researchers have shown the potential utility of the GC test for identifying an unfavorable subgroup of men who likely require treatment intensification beyond DE-IGRT, this study was not positioned to determine the efficacy of combined ADT and DE-IGRT to overcome the adverse prognosis of patients with a GC high risk score.\n",
      "Kim et al (2019) stated that many men diagnosed with prostate cancer are active surveillance (AS) candidates.  However, AS may be associated with increased risk of disease progression and metastasis due to delayed therapy.  Genomic classifiers, e.g., Decipher, may allow better risk-stratify newly diagnosed prostate cancers for AS.  Decipher was initially assessed in a prospective cohort of prostatectomies to explore the correlation with clinically meaningful biologic characteristics and then assessed in diagnostic biopsies from a retrospective multi-center cohort of 266 men with NCCN very low/low and favorable-intermediate risk prostate cancer.  Decipher and Cancer of the Prostate Risk Assessment (CAPRA) were compared as predictors of adverse pathology (AP) for which there is universal agreement that patients with long life-expectancy are not suitable candidates for AS (primary pattern 4 or 5, advanced local stage [pT3b or greater] or lymph node involvement).  Decipher from prostatectomies was significantly associated with adverse pathologic features (pâ<â0.001).  Decipher from the 266 diagnostic biopsies (64.7 % NCCN-very-low/low and 35.3 % favorable-intermediate) was an independent predictor of AP (OR 1.29 per 10 % increase, 95 % CI: 1.03 to 1.61, p-value 0.025) when adjusting for CAPRA. CAPRA AUC was 0.57, (95 % CI: 0.47 to 0.68).  Adding Decipher to CAPRA increased the AUC to 0.65 (95 % CI: 0.58 to 0.70).  Negative predictive value (NPV), which determines the degree of confidence in the absence of AP for patients, was 91 % (95 % CI: 87 to 94 %) and 96 % (95 % CI: 90 to 99 %) for Decipher thresholds of 0.45 and 0.2, respectively.  Using a threshold of 0.2, Decipher was a significant predictor of AP when adjusting for CAPRA (p-value 0.016).  The authors concluded that the Decipher test could be applied to prostate biopsies from NCCN-very-low/low and favorable-intermediate risk patients to predict absence of adverse pathologic features.  These patients are predicted to be good candidates for AS.\n",
      "The authors stated that this study did not have long-term follow-up to consider survival outcomes and the sample size and low number of events did not allow Decipher to be assessed in individual NCCN risk (e.g., favorable intermediate only) risk groups.  These researchers stated that an ongoing multi-institutional study of favorable-intermediate risk patients aims to address this limitation.\n",
      "National Comprehensive Cancer Networkâs clinical practice guideline on \"Prostate cancer\" (Version 4.2019) recommends coverage of  \"Decipher\", \"Oncotype DX Prostate\", \"Prolaris\", and \"ProMark\" (Category 2A).  Decipher : For post-biopsy based on NCCN very-low- and low-risk patients with greater than 10 year life expectancy who have not received treatment for prostate cancer and are candidates for active surveillance or definitive therapy.\n",
      "miRNAs for Prostate Cancer\n",
      "Maugeri-Sacca et al (2013) stated that prostate cancer is one of the most common causes of cancer-related death.  The management of prostate cancer patients has become increasingly complex, consequently calling on the need for identifying and validating prognostic and predictive biomarkers.  Growing evidence indicates that microRNAs play a crucial role in the pathobiology of neoplastic diseases.  The deregulation of the cellular \"miRNome\" in prostate cancer has been connected with multiple tumor-promoting activities such as aberrant activation of growth signals, anti-apoptotic effects, pro-metastatic mechanisms, alteration of the androgen receptor pathway, and regulation of the cancer stem cell phenotype.  With the elucidation of molecular mechanisms controlled by microRNAs, investigations have been conducted in an attempt to exploit these molecules in the clinical setting.  Moreover, the multi-faceted biological activity of microRNAs makes them an attractive candidate as anti-cancer agents.  This review summarized the current knowledge on microRNA deregulation in prostate cancer, and the rationale underlying their exploitation as cancer biomarkers and therapeutics.\n",
      "Yu and Xia (2013) discussed the novel biomarkers of microRNAs in prostate cancer.  The literatures about microRNAs and prostate cancer cited in this review were obtained mainly from PubMed published in English from 2004 to 2012.  Original articles regarding the novel role of microRNAs in prostate cancer were selected.  MicroRNAs play an important role in prostate cancer such as cell differentiation, proliferation, apoptosis, and invasion. Especially microRNAs correlate with prostate cancer cell epithelial-mesenchymal transition (EMT), cancer stem cells (CSCs), drug sensitivity, cancer microenvironment, energy metabolism, androgen independence transformation, and diagnosis prediction.  The authors concluded that microRNAs are involved in various aspects of prostate cancer biology.  Moreover, they state that the role of microRNA in the initiation and development of prostate cancer deserves further study.\n",
      "Chiam et al (2014) noted that epigenome alterations are characteristic of nearly all human malignancies and include changes in DNA methylation, histone modifications and microRNAs (miRNAs).  However, what induces these epigenetic alterations in cancer is largely unknown and their mechanistic role in prostate tumorigenesis is just beginning to be evaluated.  Identification of the epigenetic modifications involved in the development and progression of prostate cancer will not only identify novel therapeutic targets but also prognostic and diagnostic markers.  This review focused on the use of epigenetic modifications as biomarkers for prostate cancer.\n",
      "Furthermore, the National Comprehensive Cancer Networkâs clinical practice guideline on \"Prostate cancer\" (Version 1.2014) does not mention the use of RNA/microRNA biomarker as a management tool.\n",
      "Galectin-3\n",
      "There has been emerging evidence for galectin-3 in the pathogenesis and progression of prostate cancer.  However, there is insufficient evidence for its impact in screening, diagnosis or management.  National Comprehensive Cancer Networkâs clinical practice guideline on \"Prostate cancer\" (Version 1.2015) as well as its Biomarkers Compendium has no recommendation for galectin-3 in prostate cancer. \n",
      "National Comprehensive Cancer Networkâs clinical practice guidelines on \"Bone cancer\" (Version 1.2020) and \"Myelodysplastic syndromes\" (Version 1.2020) do not mention galectin-3 as a management tool.\n",
      "MLH1 Promotor Methylation\n",
      "Metcalf et al (2014) stated that colorectal cancer (CRC) that displays high microsatellite instability (MSI-H) can be caused by either germline mutations in mismatch repair (MMR) genes, or non-inherited transcriptional silencing of the MLH1 promoter.  A correlation between MLH1 promoter methylation, specifically the 'C' region, and BRAF V600E status has been reported in CRC studies.  Germline MMR mutations also greatly increase risk of endometrial cancer (EC), but no systematic review has been undertaken to determine if these tumor markers may be useful predictors of MMR mutation status in EC patients.  Endometrial cancer cohorts meeting review inclusion criteria encompassed 2,675 tumors from 20 studies for BRAF V600E, and 447 tumors from 11 studies for MLH1 methylation testing.  BRAF V600E mutations were reported in 4/2,675 (0.1 %) endometrial tumors of unknown MMR mutation status, and there were 7/823 (0.9 %) total sequence variants in exon 11 and 27/1012 (2.7 %) in exon 15.  Promoter MLH1 methylation was not observed in tumors from 32 MLH1 mutation carriers, or for 13 MSH2 or MSH6 mutation carriers.  MMR mutation-negative individuals with tumor MLH1 and PMS2 IHC loss displayed MLH1 methylation in 48/51 (94 %) of tumors.  These researchers had also detailed specific examples that showed the importance of MLH1 promoter region, assay design, and quantification of methylation.  The authors concluded that this review showed that BRAF mutations occurs so infrequently in endometrial tumors they can be discounted as a useful marker for predicting MMR-negative mutation status, and further studies of endometrial cohorts with known MMR mutation status are needed to quantify the utility of tumor MLH1 promoter methylation as a marker of negative germline MMR mutation status in EC patients.\n",
      "Furthermore, UpToDate reviews on \"Endometrial carcinoma: Clinical features and diagnosis\" (Chen ad Berek, 2015) and \"Overview of endometrial carcinoma\" (Plaxe and Mundt, 2015) as well as NCCNâs clinical practice guideline on \"Uterine neoplasms\" (Version 2.2015) do not mention testing of MLH1 promoter methylation.\n",
      "p16\n",
      "p16 is a tumor suppressor gene that regulates cellular proliferation and growth by acting as a cyclin-dependent kinase 4 (CDK4) inhibitor (Chen, et al. 2006). This test determines if a patient has a p16 gene mutation, indicating a predisposition for melanoma and pancreatic cancer.\n",
      "Chung et al (2014) noted that although p16 protein expression, a surrogate marker of oncogenic human papillomavirus (HPV) infection, is recognized as a prognostic marker in oropharyngeal squamous cell carcinoma (OPSCC), its prevalence and significance have not been well-established in cancer of the oral cavity, hypopharynx, or larynx, collectively referred as non-OPSCC, where HPV infection is less common than in the oropharynx.  p16 expression and high-risk HPV status in non-OPSCCs from RTOG 0129, 0234, and 0522 studies were determined by immunohistochemistry (IHC) and in-situ hybridization (ISH).  Hazard ratios from Cox models were expressed as positive or negative, stratified by trial, and adjusted for clinical characteristics.  p16 expression was positive in 14.1 % (12 of 85), 24.2 % (23 of 95), and 19.0 % (27 of 142) and HPV ISH was positive in 6.5 % (6 of 93), 14.6 % (15 of 103), and 6.9 % (7 of 101) of non-OPSCCs from RTOG 0129, 0234, and 0522 studies, respectively.  Hazard ratios for p16 expression were 0.63 (95 % CI: 0.42 to 0.95; p = 0.03) and 0.56 (95 % CI: 0.35 to 0.89; p = 0.01) for PFS and OS, respectively.  Comparing OPSCC and non-OPSCC, patients with p16-positive OPSCC have better PFS and OS than patients with p16-positive non-OPSCC, but patients with p16-negative OPSCC and non-OPSCC have similar outcomes.  The authors concluded that similar to results in patients with OPSCC, patients with p16-negative non-OPSCC have worse outcomes than patients with p16-positive non-OPSCC, and HPV may also have a role in outcome in a subset of non-OPSCC.  However, these investigators stated that further development of a p16 IHC scoring system in non-OPSCC and improvement of HPV detection methods are needed before broad application in the clinical setting; they noted that additional research using multi-modality testing in non-OPSCC and development of more accurate HPV testing are indicated.\n",
      "MUC5AC\n",
      "Ruzzenente et al (2014) stated that mucin 5AC (MUC5AC) is a glycoprotein found in different epithelial cancers, including biliary tract cancer (BTC).  These researchers examined the role of MUC5AC as serum marker for BTC and its prognostic value after operation with curative intent.  From January 2007 to July 2012, a quantitative assessment of serum MUC5AC was performed with enzyme-linked immunoassay in a total of 88 subjects.  Clinical and biochemical data (including CEA and Ca 19-9) of 49 patients with BTC were compared with a control population that included 23 patients with benign biliary disease (BBD) and 16 healthy control subjects (HCS).  Serum MUC5AC was greater in BTC patients (mean 17.93 Â± 10.39 ng/ml) compared with BBD (mean 5.95 Â± 5.39 ng/ml; p < 0.01) and HCS (mean 2.74 Â± 1.35 ng/ml) (p < 0.01).  Multi-variate analysis showed that MUC5AC was related with the presence of BTC compared with Ca 19-9 and CEA: p < 0.01, p = 0.080, and p = 0.463, respectively.  In the BTC group, serum MUC5AC greater than or equal to 14 ng/ml was associated with lymph-node metastasis (p = 0.050) and American Joint Committee on Cancer and International Union for Cancer Control stage IVb disease (p = 0.047).  Moreover, in patients who underwent operation with curative intent, serum MUC5AC greater than or equal to 14 ng/ml was related to a worse prognosis compared with patients with lesser levels, with 3-year survival rates of 21.5 % and 59.3 %, respectively (p = .039).  The authors concluded that MUC5AC could be proposed as new serum marker for BTC.  Moreover, the quantitative assessment of serum MUC5AC could be related to tumor stage and long-term survival in patients with BTC undergoing operation with curative intent.\n",
      "The authors stated that \"Limitations of this study include the lack of data on serum levels of MUC5AC in patients with obstructive jaundice and with premalignant biliary lesions such as hepatolithiasis, sclerosing cholangitis, and choledochal cysts â¦. Therefore, further studies should be addressed to clarify the diagnostic value of serum MUC5AC also in patients with obstructive jaundice and with premalignant lesions â¦. Our data should be confirmed by well-designed large-scale prospective studies\". \n",
      "Furthermore, NCCNâs clinical practice guideline on \"Hepatobiliary cancers\" (Version 2.2015) does not mention mucin 5AC (MUC5AC) as a management tool.\n",
      "Tp53\n",
      "In a pilot study, Erickson et al (2014) examined if tumor cells could be detected in the vagina of women with serous ovarian cancer through TP53 analysis of DNA samples collected by placement of a vaginal tampon.  Women undergoing surgery for a pelvic mass were identified in the gynecologic oncology clinic.  They placed a vaginal tampon before surgery, which was removed in the operating room.  Cells were isolated and DNA was extracted from both the cells trapped within the tampon and the primary tumor.  In patients with serous carcinoma, the DNA was interrogated for the presence of TP53 mutations using a method capable of detecting rare mutant alleles in a mixture of mutant and wild-type DNA.  A total of 33 patients were enrolled; 8 patients with advanced serous ovarian cancer were included for analysis; and 3 had a prior tubal ligation.  TP53 mutations were identified in all 8 tumor samples.  Analysis of the DNA from the tampons revealed mutations in 3 of the 5 patients with intact tubes (sensitivity 60 %) and in none of the 3 patients with tubal ligation.  In all 3 participants with mutation detected in the tampon specimen, the tumor and the vaginal DNA harbored the exact same TP53 mutation.  The fraction of DNA derived from exfoliated tumor cells ranged from 0.01 % to 0.07 %.  The authors concluded that in this pilot study, DNA derived from tumor was detected in the vaginas of 60 % of patients with ovarian cancer with intact fallopian tubes.  They stated that with further development, this approach may hold promise for the early detection of this deadly disease.  They stated that for this method to ultimately be clinically useful, several factors should be considered -- this approach will have to be shown to be able to adequately detect early states of disease to provide sufficient lead time for an effective intervention.  In this regard, one of the drawbacks of this study was that all samples were obtained from patients with late-stage cancer.  Another limitation was that these researchers did not sequence the DNA from tampons from patients with benign disease.  Thus, specificity could not be calculated.  These investigators stated that larger studies are needed to further validate this method and identify a more precise detection rate.\n",
      "In an editorial that accompanied the afore-mentioned study, Mulch (2014) stated that \"In terms of clinical utility, the sensitivity of this test may be around 60 % in patients with intact tubes and with clinically obvious cancer, but we do not know what it will be in patients with less advanced disease â¦. However, the barrier to ovarian cancer screening is the fact that the prevalence of the disease is so low in the general population that any screening test must have an unrealistic sensitivity and specificity â¦. This technology shows great promise â¦. The technology represented here has the potential to do what other screening tests have not â¦. We must be careful not to endorse it until its usefulness is fully validated\".\n",
      "Furthermore, NCCNâs clinical practice guideline on \"Ovarian cancers\" (Version 3.2014) does not mention TP53 mutation analysis as a management tool.\n",
      "Zhang et al (2015) summarized the potential diagnostic value of 5 serum tumor markers in esophageal cancer.  These researchers systematically searched PubMed, Embase, Chinese National Knowledge Infrastructure (CNKI) and Chinese Biomedical Database (CBM), through February 28, 2013, without language restriction.  Studies were assessed for quality using QUADAS (quality assessment of studies of diagnostic accuracy).  The positive likelihood ratio (PLR) and negative likelihood ratio (NLR) were pooled separately and compared with overall accuracy measures using diagnostic odds ratios (DORs) and symmetric summary receiver operating characteristic (SROC) curves.  Of 4,391 studies initially identified, 44 eligible studies including 5 tumor markers met the inclusion criteria for the meta-analysis, while meta-analysis could not be conducted for 12 other tumor markers.  Approximately 79.55 % (35/44) of the included studies were of relatively high quality (QUADAS score greater than or equal to 7).  The summary estimates of PLR, NLR and DOR for diagnosing EC were as follows: CEA, 5.94/0.76/9.26; Cyfra21-1 (a cytokeratin 19 fragment), 12.110.59/22.27; p53 antibody, 6.71/0.75/9.60; squamous cell carcinoma antigen (SCC-Ag), 7.66/0.68/12.41; and vascular endothelial growth factor C (VEGF-C), 0.74/0.37/8.12.  The estimated SROC curves showed that the performance of all 5 tumor markers was reasonable.  The authors concluded that the current evidence suggested that CEA, Cyfra21-1, p53, SCC-Ag and VEGF-C have a potential diagnostic value for esophageal carcinoma.\n",
      "Ki67\n",
      "There is a strong correlation between proliferation rate and clinical outcome in a variety of tumor types and measurement of cell proliferative activity is an important prognostic marker (Chen, et al., 2006). This marker correlates with flow cytometric S-phase.\n",
      "There is insufficient evidence for Ki67. NCCN guidelines on breast cancer (2015) state: \"The measurement of nuclear antigen, Ki-67 by IHC, gives an estimate of the tumor cells in the proliferative phase (G1, G2 and M phases) of the cell cycle. Studies have demonstrated the prognostic value of Ki-67 as a biomarker and its usefulness in predicting response and clinical outcome. One small study suggests that measurement of Ki-67 after short-term exposure to endocrine treatment may be useful to select patients resistant to endocrine therapy and those who may benefit from additional interventions. However, these data require larger analytic and clinical validation. In addition, standardization of tissue handling and processing is required to improve the reliability and value of Ki-67 testing. At this time, there is no conclusive evidence that Ki-67 alone, especially baseline Ki-67 as an individual biomarker, helps to select the type of endocrine therapy for an individual patient. Therefore, the NCCN Breast Cancer Panel does not currently recommend assessment of Ki-67.\"\n",
      "The p16/KI-67 Dual Stain test (CINtec PLUS) claims to detect virally induced oncogenic molecular changes in the cell through the immune cytochemical double staining of the tumor suppressor gene p16INK4a and the proliferation marker Ki-67 and thereby to improve the triage of women with equivocal cytological results (Kisser, et al., 2014). The Ludwig Boltzmann Institut conducted a systematic review of studies assessing utlity of the p16/Ki-67 Dual Stain test in the triage of equivocal or mild to moderate dysplasia results in cervical cancer screening. The authors of the assessment stated that they could not identify any studies assessing clinical outcomes such as mortality or morbidity and only one high quality study assessing diagnostic accuracy of the test: the evaluation of the clinical utility of the test was therefore not possible (Kisser, et al., 2014). Consequently the test was not recommended for inclusion in the benefits catalogue of public health insurances.\n",
      "Guidelines from the American Society for Clinical Oncology (2016) state: \"Protein encoded by the MKI67 gene labeling index by IHC should not be used to guide choice on adjuvant chemotherapy.\" This is a moderate-strength recommendation based upon intermediate-quality evidence. \n",
      "Veristrat\n",
      "Mass spectrometry based proteomic profiling (such as Veristrat) is a multivariate serum protein test that uses mass spectrometry and proprietary algorithms to analyze proteins in an individualâs serum.\n",
      "NCCN guidelines on non-small cell lung cancer (NCCN, 2015) recommend proteomic testing for patients with NSCLC and wild-type EGFR or with unknown EGFR status. The guidelines state that a patient with a \"poor\" classification should not be offered erlotinib in the second-line setting. For support, NCCN guidelines reference a study by Gregorc, et al. (2014), who reported that serum protein test status (Veristrat) is predictive of differential benefit in overall survival for erlotinib versus chemotherapy in the second-line setting, and that patients classified as likely to have a poor outcome have better outcomes on chemotherapy than on erlotinib. From Feb 26, 2008, to April 11, 2012, patients (aged â¥18 years) with histologically or cytologically confirmed, second-line, stage IIIB or IV non-small-cell lung cancer were enrolled in 14 centrers in Italy. Patients were stratified according to a minimization algorithm by Eastern Cooperative Oncology Group performance status, smoking history, center, and masked pretreatment serum protein test classification, and randomly assigned centrally in a 1:1 ratio to receive erlotinib (150 mg/day, orally) or chemotherapy (pemetrexed 500 mg/m2, intravenously, every 21 days, or docetaxel 75 mg/m2, intravenously, every 21 days). The proteomic test classification was masked for patients and investigators who gave treatments, and treatment allocation was masked for investigators who generated the proteomic classification. The primary endpoint was overall survival and the primary hypothesis was the existence of a significant interaction between the serum protein test classification and treatment. Analyses were done on the per-protocol population.  Investigators randomly assigned 142 patients to chemotherapy and 143 to erlotinib, and 129 (91%) and 134 (94%), respectively, were included in the per-protocol analysis. 88 (68%) patients in the chemotherapy group and 96 (72%) in the erlotinib group had a proteomic test classification of good. Median overall survival was 9Â·0 months (95% CI 6Â·8â10Â·9) in the chemotherapy group and 7Â·7 months (5Â·9â10Â·4) in the erlotinib group. The investigators noted a significant interaction between treatment and proteomic classification (pinteraction = 0Â·017 when adjusted for stratification factors; pinteraction=0Â·031 when unadjusted for stratification factors). The investigators found that patients with a proteomic test classification of poor had worse survival on erlotinib than on chemotherapy (hazard ratio 1Â·72 [95% CI 1Â·08â2Â·74], p=0Â·022). There was no significant difference in overall survival between treatments for patients with a proteomic test classification of good (adjusted HR 1Â·06 [0Â·77â1Â·46], p=0Â·714). In the group of patients who received chemotherapy, the most common grade 3 or 4 toxic effect was neutropenia (19 [15%] vs one [<1%] in the erlotinib group), whereas skin toxicity (one [<1%] vs 22 [16%]) was the most frequent in the erlotinib group.\n",
      "Multiplex Testing for Myeloid Hematopathologic Disorders\n",
      "Multiplex testing/next generation sequencing can assist in the diagnosis of various myeloid hematopathologic disorders, particularly myelodysplastic syndrome (MDS). The International Consensus Working Group (ICWG) (Valent, et al., 2007) recommends that minimal diagnostic criteria for MDS include: A) Prerequisite criteria, including stable cytopenia in one or more cell line, and exclusion of other potential disorders as a primary reason for dysplasia and/or cytopenia; B) MDS-related (decisive) criteria, including significant dysplasia, a blast count of 5-19%, and/or specific MDS cytogenetic abnormalities; and co-criteria for patients fulfilling A) but not B), including clear signs of a monoclonal population utilizing molecular markers (such as DNA mutations) or flow cytometry, or markedly reduced colony formation. In addition, many of the genes have independent prognostic value in various myeloid malignancies including ASXL1, RUNX1, ETV6, EZH2, TP53 in multivariable analysis in MDS. Other critical disease genes such as DNMT3A, CBL, IDH2, IDH1, SRSF2, ZRSR2, NRAS, U2AF1, and SF3B1 have also been shown to be independent predictors of survival in MDS as well as ASXL1, SRSF2, CBL, and IDH2 in chronic myelomonocytic leukemia (CMML), IDH1/2, EZH2, SRSF2, ASXL1 in primary myelofibrosis (PMF), and SETBP1 in atypical chronic myeloid leukemia (aCML).\n",
      "ResponseDx\n",
      "ResponseDX: ColonÂ® (Response Genetics) panel utilizes testing of multiple genes including KRAS mutation, BRAF mutation, ERCC1 expression, MSI, c-Met expression, EGFR expression, VEGFR2 expression, NRAS mutation, PIK3CA mutation, and Thymidylate synthetase (Raman, et al., 2013).  The test predicts disease prognosis and selects patients who might benefit from alternative therapies and aids in selection of metastatic colorectal cancer patients that might benefit from EGFR-targeted monoclonal antibody therapies.\n",
      "ResponseDX:LungÂ® panel (Response Genetics) utilizes testing of multiple genes including ROS1 rearrangements, EGFR mutation, EML4-ALK rearrangement, ALK, ERCC1 expression, RRM1 expression, c-MET expression, TS expression, KRAS mutation, and PIK3CA mutation (Raman, et al., 2013) The test is used in patients with non-small cell lung cancer who are being considered for treatment with the tyrosine kinase inhibitor (TKI) Crizotinib.\n",
      "ResponseDX:MelanomaÂ® panel (Response Genetics) utilizes testing of multiple genes including BRAF mutation, and NRAS mutation (Raman, et al., 2013). The test is performed on formalin-fixed, paraffin embedded (FFPE) biopsy specimen, using fluorescence in situ hybridization (FISH) and polymerase chain reaction (PCR). The test is intended for patients with melanoma who are being considered for treatment with the tyrosine kinase inhibitor (TKI) and EGFR antagonists cetuximab and panitumumab. \n",
      "ResponseDX: GastricÂ® panel (Response Genetics) utilizes testing of multiple genes including HER2 gene amplification, ERCC1 expression, and Thymidilate Synthetase expression (Raman, et al., 2013). This is a PCR-based test performed on formalin-fixed, paraffin-embedded biopsy specimens.   Amplification of the HER2 gene is associated with increased disease recurrence and a worse prognosis. ERCC1 expression predicts the best therapeutic combination of agents including platinum and select patients who might benefit from platinum-based therapies. Thymidylate synthetase (TS) expression predicts the best therapeutic combination of agents including pemetrexed or 5-FU and select patients who might benefit from pemetrexed-based therapies.\n",
      "4K Score\n",
      "4Kscore Test measures the blood plasma levels of four different prostate-derived kallikrein proteins [Total PSA, Free PSA, Intact PSA and Human Kallikrein2 (hK2)] and combines results in an algorithm with age, DRE (nodules, no nodules) and prior biopsy results. The result is purportedly an individualâs specific probability for finding a high-grade, Gleason score 7 or higher prostate cancer upon biopsy.\n",
      "Parekh et al (2015) performed the first prospective evaluation of the 4Kscore in predicting Gleason â¥7 PCa in the USA. The investigators prospectively enrolled 1012 men scheduled for prostate biopsy, regardless of prostate-specific antigen level or clinical findings, from 26 US urology centers between October 2013 and April 2014.  The primary outcome was Gleason â¥7 PCa on prostate biopsy. The area under the receiver operating characteristic curve, risk calibration, and decision curve analysis (DCA) were determined, along with comparisons of probability cutoffs for reducing the number of biopsies and their impact on delaying diagnosis. Gleason â¥7 PCa was found in 231 (23%) of the 1012 patients. The investigators stated that the 4Kscore showed excellent calibration and demonstrated higher discrimination (area under the curve [AUC] 0.82) and net benefit compared to a modified Prostate Cancer  Prevention Trial Risk Calculator 2.0 model and standard of care (biopsy for all men) according to DCA. A possible reduction of 30-58% in the number biopsies was identified with delayed diagnosis in only 1.3-4.7% of Gleason â¥7 PCa cases, depending on the threshold used for biopsy. Pathological assessment was performed according to the standard of care at each site without centralized review. \n",
      "Stattin et al (2015) conducted a case-control study nested within a population-based cohort. PSA and three additional kallikreins (4KScore) were measured in cryopreserved blood from a population-based cohort in VÃ¤sterbotten, Sweden. Of 40,379 men providing blood at ages 40, 50, and 60 years from 1986 to 2009, 12,542 men were followed for >15 yr. From this cohort, the Swedish Cancer Registry identified 1423 incident PCa cases, 235 with distant metastasis. Most metastatic cases occurred in men with PSA in the top quartile at age 50 yr (69%) or 60 yr (74%), whereas 20-yr risk of metastasis  for men with PSA below median was low (â¤0.6%). The investigators reported that, among men with PSA >2 ng/ml, a prespecified model based on four kallikrein markers significantly enhanced the prediction of metastasis compared with PSA alone. About half of all men with PSA >2 ng/ml were defined as low risk by this model and had a â¤1% 15-yr risk of metastasis. The authors concluded that, for men in their fifties, screening should focus on those in the top 10% to 25% of PSA values because the majority of subsequent cases of distant metastasis are found among these men. Testing of four kallikrein markers in men with an elevated PSA could aid biopsy decision making. \n",
      "Voigt et al (2014) conducted a systematic review and meta-analysis to examine the aggregated results from published studies of the Kallikrein Panel. The results of the meta-analysis were used to model the Kallikrein Panel's effect on healthcare costs. The authors reported that meta-analysis demonstrates a statistically significant improvement of 8-10% in predictive accuracy. The authors estimated that 48% to 56% of current prostate biopsies could be avoided and that use of the Kallikrein Panel could result in annual US savings approaching $1 billion. \n",
      "Konety et al (2015) conducted a clinical utility study to assess the influence of the 4Kscore Test on the decision to perform prostate biopsies in men referred to urologists for abnormal PSA and/or DRE results. The study population included 611 patients seen by 35 academic and community urologists in the United States. Urologists ordered the 4Kscore Test as part of their assessment of men referred for abnormal PSA and/or DRE test results. Results for  the patients were stratified into low risk (< 7.5%), intermediate risk (7.5%-19.9%), and high risk (â¥ 20%) for aggressive prostate cancer. The investigators reported that the 4Kscore Test results influenced biopsy decisions in 88.7% of the men. Performing the 4Kscore Test resulted in a 64.6% reduction in prostate biopsies in patients; the  actual percentage of cases not proceeding to biopsy were 94.0%, 52.9%, and 19.0% for men who had low-, intermediate-, and high-risk 4Kscore Test results, respectively. A higher 4Kscore Test was associated with greater likelihood of having a prostate biopsy (P < 0.001). The investigators reported that, among the 171 patients who had a biopsy, the 4Kscore risk category is strongly associated with biopsy pathology.  \n",
      "Lin et al (2016) sought to evaluate the utility of the 4Kscore in predicting the presence of high-grade cancer in men on active surveillance. Plasma collected before the first and subsequent surveillance biopsies was assessed for 718 men prospectively enrolled in the multi-institutional Canary PASS trial. Biopsy data were split 2:1 into training and test sets. The investigators developed statistical models that included clinical information and either the 4Kpanel or serum prostate-specific antigen (PSA). The endpoint was reclassification to Gleason â¥7. The investigators used receiver operating characteristic (ROC) curve analyses and area under the curve (AUC) to assess discriminatory capacity, and decision curve analysis (DCA) to report clinical net benefit. Significant predictors for reclassification were 4Kpanel (odds ratio [OR] 1.54, 95% confidence interval [CI] 1.31-1.81) or PSA (OR 2.11, 95% CI 1.53-2.91), â¥20% cores positive (OR 2.10, 95% CI 1.33-3.32), two or more prior negative biopsies (OR 0.19, 95% CI 0.04-0.85), prostate volume (OR 0.47, 95% CI 0.31-0.70), and body mass index (OR 1.09, 95% CI 1.04-1.14). ROC curve analysis comparing 4K and base models indicated that the 4Kpanel improved accuracy for predicting reclassification (AUC 0.78 vs 0.74) at the first surveillance biopsy. Both models performed comparably for prediction of reclassification at subsequent biopsies (AUC 0.75 vs 0.76). In DCA, both models showed higher net benefit compared to biopsy-all and biopsy-none strategies. Limitations include the single cohort nature of the study and the small numbers; results should be validated in another cohort before clinical use.\n",
      "Guidelines from the National Comprehensive Cancer Network (NCCN, 2016) lists the 4Kscore nonpreferentially among a number of tests (i.e., the percent free PSA and the Prostate Health Index (PHI)) that can be considered for patients prior to biopsy and among several tests (i.e., percent free PSA, PHI, PCA3 and ConfirmMDx) for those with prior negative biopsy for men thought to be at higher risk for clinically significant prostate cancer. The NCCN guidelines state that the 4Kscore cannot be recommended over other tests (i.e., the percent free PSA, the Prostate Health Index (PHI), The NCCN guidelines explain that head-to-head comparisons have been performed in Europe for some of these tests, performed individually or in combinations in the intial or repeat biopsy settings, but sample sizes were small and results varied. The NCCN guidelines stated that the optimal order of biomarker tests and imaging is unknown, and that it remains unclear how to interpret multiple tests in individual patients, especially when results are contradictory. The panel states that it is important for patients and their urologists to understand, however, that no cutoff threshold has been established for the 4KScore.  \n",
      "Recommendations from Memorial Sloan Kettering (Vickers, et al., 2016) state that in biopsy-naive men with PSA â¥3 ng/mL, prostate MRI is the strongest independent predictor of clinically significant prostate cancer, but \"[a]s evidence continues to build, we believe that prostate MRI may emerge as a valuable tool to reduce overdiagnosis of PCa, most likely in concert with newer biomarkers, such as the Prostate Health Index, the 4Kscore, and single nucleotide polymorphism panels.\n",
      "A 2016 MolDx assessment of the 4KScore concluded that \"the intended use population has been inadequately validated; the 4Kscore model has continuously changed; the model has been recurrently tested on potentially inappropriate patients (PSA > 10) and patients with inadequate biopsy sampling; it is unclear how much the hK2 and possibly intact PSA contribute to the model; the value of the 4Kscore model/algorithm is fraught with statistical hypothesis and not prospective outcomes or concordance in a defined patient population likely to be considered for biopsy (eg: PSA 3-10 ng/mL); assumptions are made that no harm will come to following young men with unknown low grade prostate cancer (not on AS); there is significant difficulty equating the model used in the Swedish study to the presently proposed formula; and the incidence of clinically diagnosable prostate cancer in patients with low risk by the model/algorithm at 10 years is very concerning.\" \n",
      "Anceschi et al (2019) stated that in recent years, several biomarkers alternative to standard prostate specific antigen (PSA) for PCa diagnosis have become available.  In a systematic review, these researchers examined current knowledge about alternative serum and urinary biomarkers for the diagnosis of PCa.  A research was conducted in Medline, restricted to English language articles published between December 2014 and June 2018 with the aim to update previously published series on PCa biomarkers.  The preferred reporting items for systematic reviews and meta-analyses (PRISMA) criteria were used for selecting studies with the lowest risk of bias.  Emerging role and actual controversies on serum and urine alternative biomarkers to standard PSA for PCa diagnosis, staging and prognosis assessment, such as prostate health index (PHI), PCA3, ConfirmMDx, Aberrant PSA glycosylation, MiPS, miRNAs were critically presented in the current review.  The authors concluded that although the use of several biomarkers has been recommended or questioned by different international guidelines, larger prospective randomized studies are still necessary to validate their efficacy in PCa detection, discrimination, prognosis and treatment effectiveness.  To-date, only PHI and 4Kscore have shown clinical relevance for discriminating more aggressive PCa.  Furthermore, a new grading classification based on molecular features relevant for PCa risk-stratification and tailoring treatment is still needed.\n",
      "Kim et al (2019) noted that prostate cancer (CaP) is the most common cancer diagnosed among men in the United States and the 5th most common cancer among men in Korea.  Unfortunately, the early stages of CaP may have no symptoms.  Therefore, early detection is very important and physicians managing voiding dysfunction must have awareness regarding CaP.  The traditional tests used for early detection of CaP are the prostate-specific antigen (PSA) blood test and digital rectal examination (DRE).  However, a high PSA level is not specific for CaP.  Benign prostatic hyperplasia (BPH), prostatitis, urinary tract infection (UTI), and urinary retention can all cause a high PSA level.  Thus, no test shows sufficient accuracy to truly be useful for screening men for CaP.  A prostate biopsy is the only method that yields a definitive diagnosis of CaP; however, this test is invasive and uncomfortable.  Recently, new biomarkers for CaP detection have been proposed to improve the accuracy of the PSA test.  These investigators summarized their knowledge of various new biomarkers, including PSA-associated biomarkers (the prostate health index and 4Kscore), molecular biomarkers (PCA3, TMPRSS2: ERG fusion gene, and various miRNAs), and proteomics-associated biomarkers, and the ways in which they may improve the detection rate of CaP.  The authors concluded that until now, there has been many efforts to predict early stage CaP such as PSA associated markers, various molecular markers, miRNA markers, and protein markers.  Unfortunately, the follow-up validation studies are lack due to several reasons.  Thus, future studies of CaP biomarkers need to focus on combinations of molecular biomarkers and clinical variables, rather than on biomarkers alone.\n",
      "Marzouk et al (2019) stated that recent years have seen the development of biomarkers and imaging technologies designed to improve the specificity of PSA.  Widespread implementation of imaging technologies, such as mp-MRI raises considerable logistical challenges.  These researchers evaluated a biopsy strategy that utilizes selective mp-MRI as a follow-up test to biomarkers to improve the detection of significant PCa.  They developed a conceptual approach based on the risk calculated from the 4Kscore using results from the U.S. prospective validation study, multiplied by the likelihood ratio of mp-MRI from the PROMIS trial.  The primary outcome was Gleason grade greater than or equal to 7 (grade group greater than or equal to 2) cancer on biopsy.  Using decision curve analysis, the net benefit was determined for this model and compared with the use of the 4Kscore and mp-MRI independently at various thresholds for biopsy.  For a cut-point of 7.5 % risk of high-grade disease, patients with less than 5 % risk from a blood marker would not have risk of significant PCa sufficiently increased by a positive mp-MRI to warrant biopsy; comparably, patients with a risk of greater than 23 % would not have risk sufficiently reduced by a negative imaging study to forgo biopsy.  From the 4Kscore validation study, 46 % of men considered for biopsy in the U.S. have risks 5 % to 23 %.  Net benefit was highest for the combined strategy, followed by 4Kscore alone.  The authors concluded that selective mp-MRI in men with intermediate scores on a secondary blood test resulted in a biopsy strategy that was more scalable than mp-MRI for all men with elevated PSA.  These researchers stated that prospective validation is needed to examine if the predicted properties of combined blood and imaging testing are empirically confirmed.\n",
      "Falagario et al (2020) stated that the 2019 European Association of Urology guidelines recommended mp-MRI for biopsy-naÃ¯ve patients with clinical suspicion of PCa and avoiding biopsy in patients with negative mp-MRI and low clinical suspicion.  However, consensus on the optimal definition of low clinical suspicion is lacking.  These researchers evaluated 266 biopsy-naÃ¯ve patients who underwent mp-MRI, the 4Kscore test, and prostate biopsy to define the best strategy to avoid unnecessary testing and biopsies.  The European Randomized Study of Screening for Prostate Cancer risk calculator (ERSPC-RC) and PSA density (PSAd) were also considered.  For men with Prostate Imaging-Reporting and Data System v2.0 (PI-RADS) 1â¿¿2 lesions, the highest negative predictive value (NPV) was observed for those with low or intermediate 4Kscore risk (96.9 % and 97.1 %), PSAd < 0.10 ng/ml/cm3 (98.7 %), and ERSPC-RC less than 2 % (98.7 %).  For men with PI-RADS 3â¿¿5 lesions the lowest positive predictive value (PPV) was observed for those with low 4Kscore risk (0 %), PSAd less than 0.10 ng/ml/cm3 (13.2 %), and ERSPC-RC of less than 2 % (12.3 %).  The best biopsy strategy was an initial 4Kscore followed by mp-MRI if the 4Kscore was greater than 7.5 % and a subsequent biopsy if the mp-MRI was positive (PI-RADS 3â¿¿5) or the 4Kscore was â¿¥18 %.  This would result in missing 2.7 % (2/74) of clinically significant PCs (csPCs) and avoiding 34.2 % of biopsies.  Initial mp-MRI followed by biopsy for negative mp-MRI (PI-RADS 1â¿¿2) if the 4Kscore was â¿¥18 % or PSAd was â¿¥0.10 ng/ml/cm3 resulted in a similar percentage of csPC missed (2.7 % [2/74] and 1.3 % [1/74]) but slightly fewer biopsies avoided (25.2 % and 28.1 %).  Physicians should consider clinical risk screening tools when ordering and interpreting mp-MRI results to avoid unnecessary testing and diagnostic errors.  The authors stated that performing the 4Kscore test in conjunction with mp-MRI for men with a clinical suspicion of prostate cancer may help to reduce unnecessary biopsies.  These researchers stated that this study was limited by its small sample size and its retrospective nature; prospective validation of these findings is needed before their implementation in clinical practice.\n",
      "An UpToDate review on âScreening for prostate cancerâ (Hoffman, 2021) states that âReferral for urologic evaluation will not necessarily result in a prostate biopsy.  Other tests (e.g., free to total PSA ratio [f/T PSA], PCA3, 4Kscore test, and/or magnetic resonance imaging [MRI]) may be done by the urologist to help determine the likelihood that the PSA is elevated due to prostate cancer, the PSA may be followed over time, or a biopsy may be performed.  Relevant considerations include the patient's health status, clinical likelihood for harboring significant disease, and personal wishesâ.\n",
      "ConfirmMDx\n",
      "ConfirmMDx (MDxHealth, Irvine, CA) is an epigenetic assay using multiplex polymerase chain reaction (PCR) to measure DNA methylation of gene regions that are associated with cancer.  It is designed to distinguish patients with prostate cancer who have a true-negative biopsy from those who may have occult cancer.  The test supposedly helps urologists rule-out prostate cancer-free men from undergoing unnecessary repeat biopsies and, helps rule-in high-risk patients who may require repeat biopsies and potential treatment.  However, there is inadequate evidence to support the clinical value of ConfirmMDx in patients with prostate cancer.\n",
      "Stewart et al (2013) reported that ConfirmMDx, a multiplex quantitative methylation specific polymerase chain reaction assay determining the methylation status of GSTP1, APC and RASSF1, was strongly associated with repeat biopsy outcome up to 30 months after initial negative biopsy in men with suspicion of prostate cancer.  The investigators blindly tested archived prostate biopsy needle core tissue samples of 498 subjects from the United Kingdom and Belgium with histopathologically negative prostate biopsies, followed by positive (cases) or negative (controls) repeat biopsy within 30 months.  Clinical performance of the epigenetic marker panel, emphasizing negative predictive value, was assessed and cross-validated. Multivariate logistic regression was used to evaluate all risk factors.  The epigenetic assay performed on the first negative biopsies of this retrospective review cohort resulted in a negative predictive value of 90 % (95 % CI: 87 to 93).  In a multivariate model correcting for patient age, prostate specific antigen, digital rectal examination and first biopsy histopathological characteristics the epigenetic assay was a significant independent predictor of patient outcome (OR 3.17, 95 % CI: 1.81 to 5.53).  The investigators stated that adding this epigenetic assay could improve the prostate cancer diagnostic process and decrease unnecessary repeat biopsies.\n",
      "Partin et al (2014) reported that the ConfirmMDx epigenetic assay was a significant, independent predictor of prostate cancer detection in a repeat biopsy collected an average of 13 months after an initial negative result.  The investigators evaluated the archived, cancer negative prostate biopsy core tissue samples of 350 subjects from a total of 5 urological centers in the United States.  All subjects underwent repeat biopsy within 24 months with a negative (controls) or positive (cases) histopathological result.  Centralized blinded pathology evaluation of the 2 biopsy series was performed in all available subjects from each site.  Biopsies were epigenetically profiled for GSTP1, APC and RASSF1 relative to the ACTB reference gene using quantitative methylation specific polymerase chain reaction.  Pre-determined analytical marker cutoffs were used to determine assay performance.  Multivariate logistic regression was used to evaluate all risk factors.  The epigenetic assay resulted in a negative predictive value of 88 % (95 % CI: 85 to 91).  In multivariate models correcting for age, prostate specific antigen, digital rectal examination, first biopsy histopathological characteristics and race the test proved to be the most significant independent predictor of patient outcome (OR 2.69, 95 % CI: 1.60 to 4.51).  The investigators stated that adding this epigenetic assay to other known risk factors may help decrease unnecessary repeat prostate biopsies.\n",
      "Wu and colleagues (2011) noted that PSA screening has low specificity.  Assessment of methylation status in body fluids may complement PSA screening if the test has high specificity.  The purpose of this study was to conduct a meta-analysis of the sensitivity and specificity for prostate cancer detection of glutathione-s-transferase-Ï (GSTP1) methylation in body fluids (plasma, serum, whole blood, urine, ejaculate, and prostatic secretions).  These researchers conducted a comprehensive literature search on Medline (PubMed).  They included studies if they met all 4 of the following criteria:\n",
      "\n",
      "measurement of DNA methylation in body fluids;\n",
      "a case-control or case-only design;\n",
      "publication in an English journal; and\n",
      "adult subjects. \n",
      "\n",
      "Reviewers conducted data extraction independently using a standardized protocol.  A total of 22 studies were finally included in this paper.  Primer sequences and methylation method in each study were summarized and evaluated using meta-analyses.  This paper represented a unique cross-disciplinary approach to molecular epidemiology.  The pooled specificity of GSTP1 promoter methylation measured in plasma, serum, and urine samples from negative-biopsy controls was 0.89 (95 % CI: 0.80 to 0.95).  Stratified analyses consistently showed a high specificity across different sample types and methylation methods (include both primer sequences and location).  The pooled sensitivity was 0.52 (95 % CI: 0.40 to 0.64).  The authors concluded that the pooled specificity of GSTP1 promoter methylation measures in plasma, serum, and urine was excellent and much higher than the specificity of PSA.  The sensitivity of GSTP1 was modest, no higher than that of PSA.  They stated that these findings suggested that measurement of GSTP1 promoter methylation in plasma, serum, or urine samples may complement PSA screening for prostate cancer diagnosis.\n",
      "Van Neste et al (2012a) from the MDxHealth stated that PSA-directed prostate cancer screening leads to a high rate of false-positive identifications and an unnecessary biopsy burden.  Epigenetic biomarkers have proven useful, exhibiting frequent and abundant inactivation of tumor suppressor genes through such mechanisms.  An epigenetic, multiplex PCR test for prostate cancer diagnosis could provide physicians with better tools to help their patients.  Biomarkers like GSTP1, APC and RASSF1 have demonstrated involvement with prostate cancer, with the latter 2 genes playing prominent roles in the field effect.  The epigenetic states of these genes can be used to assess the likelihood of cancer presence or absence.  An initial test cohort of 30 prostate cancer-positive samples and 12 cancer-negative samples was used as basis for the development and optimization of an epigenetic multiplex assay based on the GSTP1, APC and RASSF1 genes, using methylation specific PCR (MSP).  The effect of prostate needle core biopsy sample volume and age of formalin-fixed paraffin-embedded (FFPE) samples was evaluated on an independent follow-up cohort of 51 cancer-positive patients.  Multiplexing affects copy number calculations in a consistent way per assay.  Methylation ratios are therefore altered compared to the respective singleplex assays, but the correlation with patient outcome remains equivalent.  In addition, tissue-biopsy samples as small as 20 Î¼m can be used to detect methylation in a reliable manner.  The age of FFPE-samples does have a negative impact on DNA quality and quantity.  The authors concluded that the developed multiplex assay appears functionally similar to individual singleplex assays, with the benefit of lower tissue requirements, lower cost and decreased signal variation.  This assay can be applied to small biopsy specimens, down to 20 microns, widening clinical applicability.  Increasing the sample volume can compensate the loss of DNA quality and quantity in older samples.\n",
      "Van Neste et al (2012b) noted that prostate cancer is the most common cancer diagnosis in men and a leading cause of death.  Improvements in disease management would have a significant impact and could be facilitated by the development of biomarkers, whether for diagnostic, prognostic, or predictive purposes.  The blood-based prostate biomarker PSA has been part of clinical practice for over 2 decades, although it is surrounded by controversy.  While debates of usefulness are ongoing, alternatives should be explored.  Particularly with recent recommendations against routine PSA-testing, the time is ripe to explore promising biomarkers to yield a more efficient and accurate screening for detection and management of prostate cancer.  Epigenetic changes, more specifically DNA methylation, are among the most common alterations in human cancer.  These changes are associated with transcriptional silencing of genes, leading to an altered cellular biology.  One gene in particular, GSTP1, has been widely studied in prostate cancer.  Thus, a meta-analysis has been conducted to examine the role of this and other genes and the potential contribution to prostate cancer management and screening refinement.  More than 30 independent, peer-reviewed studies have reported a consistently high sensitivity and specificity of GSTP1 hyper-methylation in prostatectomy or biopsy tissue.  The meta-analysis combined and compared these results.  The authors concluded that GSTP1 methylation detection can serve an important role in prostate cancer management.  The meta-analysis clearly confirmed a link between tissue DNA hyper-methylation of this and other genes and prostate cancer.  They stated that detection of DNA methylation in genes, including GSTP1, could serve an important role in clinical practice.\n",
      "Andres et al (2013) synthesized the principal advances in the field of the study of epigenetics and specifically DNA methylation regarding the diagnosis of urological neoplasms.  Review of the literature (PubMed, MEDLINE y COCHRANE) on the study of DNA methylation in urological neoplasms (prostate cancer, bladder cancer, renal cancer and testicular cancer), considering all the studies published up to January 2013 was carried out.  It was possible to determine the state of methylation of many genes in tumor samples.  When these were compared with healthy tissue samples, it was possible to define the specific aberrant methylation patterns for each type of tumor.  The study and definition of specific abnormal methylation patterns of each type of tumor is a tool having potential utility for diagnosis, evaluation, prediction of prognosis and treatment of the different forms of genito-urinary cancer.  The analysis of gene methylation in urine after micturition or post-prostatic massage urine, semen, in the wash plasma or fluid from prostatic biopsies may allow early detection of bladder, prostate, renal and testicular cancer.  In each of the neoplasms, an epigenetic signature that may be detected in the DNA has been identified, obtained from very scarce or not at all invasive specimens, with potential in the diagnosis and evaluation of prognosis.  Validation of these studies will confirm the accuracy, effectiveness, and reproducibility of the results available up to now.  Criteria have still not been developed that determine if a gene panel provides sufficient information in the health care practice to guide an unequivocal diagnosis or therapeutic conduct.  More studies are needed to compare sensitivity, specificity, positive- and negative-predictive values of the test in each case.  Multi-center studies analyzing the real reproducibility of these results in a clinical setting also do not exist.  The authors concluded that the study of aberrant DNA methylation in biological specimens of patients has an enormous potential for the early diagnosis and screening of genitourinary neoplasms.  They stated that more studies are needed to define the series of genes that would mean unequivocal signatures of malignancy.  This methodology also has potential when defining prognostic groups and potential of response to different therapies\n",
      "Lin et al (2013) prostate cancer is the second leading cause of cancer death among men worldwide, and not all men diagnosed with prostate cancer will die from the disease.  A critical challenge, therefore, is to distinguish indolent prostate cancer from more advanced forms to guide appropriate treatment decisions.  These investigators used Enhanced Reduced Representation Bisulfite Sequencing, a genome-wide high-coverage single-base resolution DNA methylation method to profile seven localized prostate cancer samples, 7 matched benign prostate tissues, and 6 aggressive castration-resistant prostate cancer (CRPC) samples.  They integrated these data with RNA-seq and whole-genome DNA-seq data to comprehensively characterize the prostate cancer methylome, detect changes associated with disease progression, and identify novel candidate prognostic biomarkers.  The analyses revealed the correlation of cytosine guanine dinucleotide island (CGI)-specific hyper-methylation with disease severity and association of certain breakpoints (deletion, tandem duplications, and inter-chromosomal translocations) with DNA methylation.  Furthermore, integrative analysis of methylation and single-nucleotide polymorphisms (SNPs) uncovered widespread allele-specific methylation (ASM) for the first time in prostate cancer.  These researchers found that most DNA methylation changes occurred in the context of ASM, suggesting that variations in tumor epigenetic landscape of individuals are partly mediated by genetic differences, which may affect prostate cancer disease progression.  They further selected a panel of 13 CGIs demonstrating increased DNA methylation with disease progression and validated this panel in an independent cohort of 20 benign prostate tissues, 16 prostate cancer a, and 8 aggressive CRPCs.  The authors concluded that these results warrant clinical evaluation in larger cohorts to help distinguish indolent prostate cancer from advanced disease.\n",
      "Wojno and colleagues (2014) reported on an observational study that suggests that the ConfirmMDx may reduce biopsy rates in persons suspected of having prostate cancer.  The investigators noted that the diagnosis of prostate cancer (PCa) is dependent on histologic confirmation in biopsy core tissues.  The biopsy procedure is invasive, puts the patient at risk for complications, and is subject to significant sampling errors.  The authors stated that an epigenetic test that uses methylation-specific PCR to determine the epigenetic status of the PCa-associated genes GSTP1, APC, and RASSF1 has been clinically validated and is used in clinical practice to increase the negative predictive value (NPV) in men with no history of PCa compared with standard histopathology.  The investigators stated that such information can help to avoid unnecessary repeat biopsies.  The investigators posited that the repeat biopsy rate may provide preliminary clinical utility evidence in relation to this assay's potential impact on the number of unnecessary repeat prostate biopsies performed in U.S. urology practices.  The investigators stated that the purpose of this preliminary study was to quantify the number of repeat prostate biopsy procedures to demonstrate a low repeat biopsy rate for men with a history of negative histopathology who received a negative epigenetic assay result on testing of the residual prostate tissue.  In this field observation study, practicing urologists used the ConfirmMDx for PCa to evaluate cancer-negative men considered at risk for PCa.  The authors stated that his test has been previously validated in 2 blinded multi-center studies that showed the superior NPV of the epigenetic test over standard histopathology for cancer detection in prostate biopsies.  A total of 5 clinical urology practices that had ordered a minimum of 40 commercial epigenetic test requisitions for patients with previous, cancer-negative biopsies over the course of the previous 18 months were contacted to assess their interest to participate in the study.  Select demographic and prostate-screening parameter information, as well as the incidence of repeat biopsy, specifically for patients with a negative test result, was collected and merged into 1 collective database.  All men from each of the 5 sites who had negative assay results were included in the analysis.  A total of 138 patients were identified in these urology practices and were included in the analysis.  The median age of the men was 63 years, and the current median serum PSA level was 4.7 ng/ml.  Repeat biopsies had been performed in 6 of the 138 (4.3 %) men with a negative epigenetic assay result, in whom no evidence of cancer was found on histopathology.  The authors concluded that in this study, a low rate of repeat prostatic biopsies was observed in the group of men with previous histopathologically negative biopsies who were considered to be at risk for harboring cancer.  The data suggested that patients managed using the ConfirmMDx for PCa negative results had a low rate of repeat prostate biopsies.  Moreover, they stated that these results warrant a large, controlled, prospective study to further evaluate the clinical utility of the epigenetic test to lower the unnecessary repeat biopsy rate.\n",
      "The PASCUAL Study is a 600 patient randomized, controlled prospective study to track the clinical utility of the ConfirmMDx assay in U.S. urologic practices for the management of patients with a previous histopathologically cancer-negative biopsy, but clinical risk factors suggesting the need for a repeat biopsy (MDxHealth, 2016). The study, initiated in 2014, will compare the rate of repeat biopsies under the standard of care to the rate of repeat biopsies in patients managed with ConfirmMDx test results. The study is expected to be completed in 2017. \n",
      "A review by Bostrom et al (2015) stated that commercially available epigenetic ConfirmMDx may be of value when repeat biopsies are considered after negative initial prostate biopsies.  The review concluded: \"Many new genetic-based tests are newly available or in late stages of clinical development, with potential applications in PCa decisions ranging from the need for repeat biopsy to initial treatment selection, decisions about secondary therapy, and selection of treatment for advanced disease.  Greater understanding of the potential long-term benefits and limitations of these tests is important, and how exactly they should be used in clinical practice to optimize decision making must be the subject of future prospective studies\".\n",
      "A HealthPACT assessment of ConfirmMDX (Foerster et al, 2013) concluded: \"From the evidence base available, prostate cancer detection using DNA methylation assays appears to offer some benefit over existing methods of diagnosis.  In particular, the high NPVs and the low rates of falseânegatives observed indicate that DNA methylation assays may provide a means of reducing the number of healthy men incorrectly diagnosed and subjected to unnecessary biopsy; although, highâquality comparative studies are needed before this can be truly determined.  Further studies investigating the effects of prostate cancer detection using DNA methylation compared with conventional techniques on overall patient survival are also required\".\n",
      "Nguyen and colleagues (2015) stated that over the past several years, multiple biomarkers designed to improve prostate cancer risk stratification have become commercially available, while others are still being developed. In this review, these researchers focused on the evidence supporting recently reported biomarkers, with a focus on gene expression signatures.  Many recently developed biomarkers are able to improve upon traditional risk assessment at nearly all stages of disease.  ConfirmMDx uses gene methylation patterns to improve detection of clinically significant cancer following negative biopsy.  Both the Prolaris and Oncotype DX Genomic Prostate Score tests can improve risk stratification following biopsy, especially among men who are eligible for active surveillance.  Prolaris and the Decipher genomic classifier have been associated with risk of adverse outcome following prostatectomy, while Oncotype DX is being studied in this setting.  Finally, recent reports of the association of androgen receptor-V7 in circulating tumor cells with resistance to enzalutamide and abiraterone raised the possibility of extending the use of genetic biomarkers to advanced disease.  The authors concluded that with the development of multiple genetic expression panels in prostate cancer, careful study and validation of these tests and integration into clinical practice will be critical to realizing the potential of these tools.\n",
      "Guidelines from  2015 from the National Comprehensive Cancer Network on prostate cancer screening (NCCN, 2015) concluded, regarding ConfirmMDx: \"Despite the good NPV of this test, the panel noted that the study population included only patients who were ultimately selected for repeat biopsy.  Furthermore, the panel noted that the NPV of first negative biopsy alone is already in the range of 75% to 80% and questioned the true value added by the test results.  Therefore, until prospective data for this test or data comparing this test to other tests are available, the panel does not recommend its use\".\n",
      "Guidelines from the National Comprehensive Cancer Network (NCCN, 2016) recommend ConfirmMDx nonpreferentially among several markers (percent free PSA, PHI, 4KScore, PCA3 and ConfirmMDx) for men contemplating repeat biopsy because the assay may identify individuals at higher risk of prostate cancer diagnosis on repeat biopsy. The guidelines note that direct comparisons have been performed for some of these tests, used independently or in combinations, in the initial or repat biopsy settings, but sample sizes were small and results varied. Therefore, the NCCN panel concluded that no biomarker test can be recommended over any other at this time. The NCCN panel noted the optimal order of biomarker tests and imaging is unknown; and it remains unclear how to interpret results of multiple tests in individual patients, especially when results are contradictory.\n",
      "A Joint Consensus Statement from the American Urological Association and the Society of Abdominal Radiology (Rosenkrantz, et al., 2016) state: \"Non-imaging markers (i.e., PSA-based measures as well as PCA3) are likely useful in further selecting patients with a negative or low-suspicion MRI (PI-RADS score of 1 or 2, respectively) that may deserve a systematic biopsy despite the MRI results. However, targeted biopsy remains warranted for intermediate or high suspicion MRI lesions despite results from these ancillary markers given the consistently observed strong independent effect of the MRI suspicion score on cancer detection in multivariate models. Further investigation is warranted to identify which of these markers best complements MRI findings in the repeat biopsy setting.\"\n",
      "Zhuang and Johnson (2016) noted that progress has been made in applying genetic information to disease management in the postgenomic era, and precision medicine is emerging in prostate cancer management.  The prostate health index, the 4-kallikrein (4K) score, and the PCA3, TMPRSS2- ERG, and Prostarix tests have potential for refining prostate cancer screening in conjunction with traditional prostate-specific antigen (PSA) testing.  The Confirm MDx and PCA3 tests have shown promise in identifying men who need be re-biopsied after a primary negative biopsy.  Oncotype DX, Prolaris, the biopsy-based Decipher prostate cancer test, and ProMark may improve predictive risk stratification in addition to the traditional Gleason score and tumor stage.  Decipher and Prolaris may predict biochemical recurrence and metastasis after radical prostatectomy and possibly help identify patients who need adjuvant therapy.  Androgen receptor splice variant 7 appears effective in guiding the selection of 2nd hormonal manipulation with abiraterone or enzalutamide versus chemotherapy when treating metastatic castration-resistant prostate cancer.\n",
      "Van Neste et al (2016) noted that prostate cancer (PCa) diagnosis is challenging because efforts for effective, timely treatment of men with significant cancer typically result in over-diagnosis and repeat biopsies.  The presence or absence of epigenetic aberrations, more specifically DNA-methylation of GSTP1, RASSF1, and APC in histopathologically negative prostate core biopsies has resulted in an increased negative predictive value (NPV) of approximately 90 % and thus could lead to a reduction of unnecessary repeat biopsies.  These researchers examined if in methylation-positive men, DNA-methylation intensities could help to identify those men harboring high-grade (Gleason scoreâ[GS] of greater than or equal to 7) PCa, resulting in an improved PPV.  Two cohorts, consisting of men with histopathologically negative index biopsies, followed by a positive or negative repeat biopsy, were combined.  EpiScore, a methylation intensity algorithm was developed in methylation-positive men, using area under the curve (AUC) of the receiver operating characteristic (ROC) as metric for performance.  Next, a risk score was developed combining EpiScore with traditional clinical risk factors to further improve the identification of high-grade (GS of greater than or equal to 7) cancer.  Compared to other risk factors, detection of DNA-methylation in histopathologically negative biopsies was the most significant and important predictor of high-grade cancer, resulting in a NPV of 96 %.  In methylation-positive men, EpiScore was significantly higher for those with high-grade cancer detected upon repeat biopsy, compared to those with either no or low-grade cancer.  The risk score resulted in further improvement of patient risk stratification and was a significantly better predictor compared to currently used metrics as PSA and the prostate cancer prevention trial (PCPT) risk calculator (RC).  A decision curve analysis indicated strong clinical utility for the risk score as decision-making tool for repeat biopsy.  The authors concluded that low DNA-methylation levels in PCa-negative biopsies led to a NPV of 96 % for high-grade cancer.  The risk score, comprising DNA-methylation intensity and traditional clinical risk factors, improved the identification of men with high-grade cancer, with a maximum avoidance of unnecessary repeat biopsies.  This risk score resulted in better patient risk stratification and significantly out-performed current risk prediction models such as PCPTRC and PSA.  The risk score could help to identify patients with histopathologically negative biopsies harboring high-grade PCa. \n",
      "Van Neste et al (2017) noted that early detection of aggressive PCa remains crucial for effective treatment of patients.  However, PCa screening remains controversial due to a high rate of over-diagnosis and over-treatment.  To better reconcile both objectives, more effective methods for assessing disease severity at the time of diagnosis are needed.  The relationship between DNA-methylation and high-grade PCa was examined in a cohort of 102 prospectively enrolled men who received standard 12-core prostate biopsies.  EpiScore, an algorithm that quantifies the relative DNA methylation intensities of GSTP1, RASSF1, and APC in prostate biopsy tissue, was evaluated as a method to compensate for biopsy under-sampling and improve risk stratification at the time of diagnosis.  DNA-methylation intensities of GSTP1, RASSF1, and APC were higher in biopsy cores from men diagnosed with GSâof greater than or equal to â7 cancer compared to men with diagnosed GS of 6 disease.  This was confirmed by EpiScore, which was significantly higher for subjects with high-grade biopsies and higher NCCN risk categories (both pâ<â0.001).  In patients diagnosed with GSâof greater than or equal toâ7, increased levels of DNA-methylation were present, not only in the high-grade biopsy cores, but also in other cores with no or low-grade disease (pâ<â0.001).  By combining EpiScore with traditional clinical risk factors into a logistic regression model, the prediction of high GS reached an AUC of 0.82 (95 % confidence interval [CI]: 0.73 to 0.91) with EpiScore, digital rectal examination (DRE), and atypical histological findings as most important contributors.  The authors concluded that in men diagnosed with PCa, DNA-methylation profiling could detect under-sampled high-risk PCa in prostate biopsy specimens through a field effect.  Predictive accuracy increased when EpiScore was combined with other clinical risk factors.  They stated that these findings  that EpiScore could aid in the detection of occult high-grade disease at the time of diagnosis, thereby improving the selection of candidates for active surveillance.\n",
      "Kretschmer and colleagues (2017) stated that in the era of personalized medicine and precision oncology, innovative genetic biomarkers are of emerging interest to close the diagnostic and prognostic gap that is left by current clinicopathologic risk classifiers.  These investigators summarized evidence regarding prognostic and predictive genetic biomarkers that are currently in widespread clinical use at initial diagnosis as well as following definitive treatment of prostate cancer.  They gave a brief summary about basic principles of biomarker research studies and present current data for the Progensa PC3 test, TMPRSS2:ERG gene fusion, ConfirmMDx, Prolaris gene panel, OncotypeDX Genomic Prostate score, and Decipher classifier.  The authors concluded that evidence regarding those genetic biomarkers has heavily increased recently; however, there is still a lack of large, multi-centric and prospective clinical validation studies.  Furthermore, they stated that comparative studies that investigate the prognostic value of various genetic biomarkers are needed.\n",
      "BioSpeciFx\n",
      "BioSpeciFx is an individualized molecular tumor profiling of a panel of tumor markers to establish a personalized molecular profile of a tumor to recommend treatment options. It is often ordered in combination with an invitro chemosensitivity/chemoresistance assay or ChemoFx. See CPB 0245 - Tumor Chemosensitivity Assays and CPB 0758 - Tumor Chemoresistance Assays. The combination of molecular profiling and invitro drug response marker testing is sometimes referred to as comprehensive tumor profiling.\n",
      "HeproDx-TM\n",
      "mRNA expression testing for hepatocellular carcinoma (HCC) (eg, HeproDx-TM) purportedly incorporates levels of 161 genes, fresh hepatocellular carcinoma tumor tissue, AFP level and an algorithm to report a risk classifier related to HCC recurrence and metastasis.\n",
      "NETest\n",
      "NETest is a multianalyte algorithm PCR-based gene blood test that measures 51 neuroendocrine tumor specific gene transcripts in combination with molecular biomarkers which purportedly allows monitoring of neuroendocrine tumor gene activity levels.\n",
      "PÄczkowska et al (2017) evaluated whether NETEST has clinical utility as a diagnostic and prognostic marker. The investigators conducted a prospective cohort study. Subjects included well-differentiated aragangliomas and pheochromocytomas (PPGLs) (nâ=â32), metastatic (nâ=â4); SDHx mutation (nâ=â25); 12 biochemically active, lanreotide treated (nâ=â4). Age- and gender-matched controls and GEP-NETs were compared. PPGL were NETest positive (100%). All exhibited higher scores than controls (55âÂ±â5% vs 8âÂ±â1%, Pâ=â0.0001), similar to GEP-NETs (47âÂ±â5%). ROC analysis area under curve was 0.98 for differentiating PPGLs/controls (cut-off for normal: 26.7%). Mutation status was not directly linked to NETest. Genetic and molecular clustering was associated (Pâ<â0.04) with NETest scores. Metastatic (80âÂ±â9%) and multicentric (64âÂ±â9%) disease had significantly (Pâ<â0.04) higher scores than localized disease (43âÂ±â7%). Progressive disease (PD) had the highest scores (86âÂ±â2%) vs stable (SD, 41âÂ±â2%) (Pâ<â0.0001). The area under the curve for PD from SD was 0.93 (cut-off for PD: 53%). Proliferation, epigenetic and somatostatin receptor gene expression was elevated (Pâ<â0.03) in PD. Metabolic gene expression was decreased in SDHx mutations. Repeat NETest measurements defined clinical status in the 9 patients (6 SD and 3 PD). Amine measurement was non-informative. Multivariate analysis identified NETest >53% as an independent prognostic factor. \n",
      "Pavel et al (2016) assessed the NETest as a predictive and prognostic marker of progression of gastroenteropancreatic neuroendocrine tumors. GEP-NETs (n = 34) followed for a median 4 years (2.2-5.4) were evaluated. WHO tumor grade/stage grade 1: n = 17, grade 2: n = 14, grade 3: n = 1 (for 2, no grade was available); 31 (91%) were stage IV. Baseline and longitudinal imaging and blood biomarkers were available in all, and progression was defined per standard clinical protocols (RECIST 1.0). The NETest was measured by quantitative PCR of blood and multianalyte algorithmic analysis (disease activity scaled 0-100% with low <40% and high activity risk cutoffs >80%); chromogranin A (CgA) was measured by radioimmunoassay (normal <150 Âµg/l); progression-free survival (PFS) was analyzed by Cox proportional-hazard regression and Kaplan-Meier analysis. At baseline, 100% were NETest positive, and CgA was elevated in 50%. The only baseline variable (Cox modeling) associated with PFS was NETest (hazard ratio = 1.022, 95% confidence interval = 1.005-1.04; p < 0.012). Using Kaplan-Meier analyses, the baseline NETest (>80%) was significantly associated (p = 0.01) with disease progression (median PFS 0.68 vs. 2.78 years with <40% levels). The NETest was more informative (96%) than CgA changes (<under>></under>25%) in consistently predicting disease alterations (40%, p < 2 Ã 10-5, Ï2 = 18). The NETest had an earlier time point change than imaging (1.02 Â± 0.15 years). Baseline NETest levels >40% in stable disease were 100% prognostic of disease progression versus CgA (Ï2 = 5, p < 0.03). Baseline NETest values <40% accurately (100%) predicted stability over 5 years (p = 0.05, Ï2 = 3.8 vs. CgA). \n",
      "Bodei et al (2016)  assessed the accuracy of circulating NET transcripts as a measure of PRRT efficacy, and moreover to identify prognostic gene clusters in pretreatment blood that could be interpolated with relevant clinical features in order to define a biological index for the tumor and a predictive quotient for PRRT efficacy. NET patients (nâ=â54), M: F 37:17, median age 66, bronchial: nâ=â13, GEP-NET: nâ=â35, CUP: nâ=â6 were treated with (177)Lu-based-PRRT (cumulative activity: 6.5-27.8 GBq, median 18.5). At baseline: 47/54 low-grade (G1/G2; bronchial typical/atypical), 31/49 (18)FDG positive and 39/54 progressive. Disease status was assessed by RECIST1.1. Transcripts were measured by real-time quantitative reverse transcription PCR (qRT-PCR) and multianalyte algorithmic analysis (NETest); CgA by enzyme-linked immunosorbent assay (ELISA). Gene cluster (GC) derivations: regulatory network, protein:protein interactome analyses. The disease control rate was 72 %. Median PFS was not achieved (follow-up: 1-33 months, median: 16). Only grading was associated with response (pâ<â0.01). At baseline, 94 % of patients were NETest-positive, while CgA was elevated in 59 %. NETest accurately (89 %, Ï(2)â=â27.4; pâ=â1.2âÃâ10(-7)) correlated with treatment response, while CgA was 24 % accurate. Gene cluster expression (growth-factor signalome and metabolome) had an AUC of 0.74âÂ±â0.08 (z-statisticâ=â2.92, pâ<â0.004) for predicting response (76 % accuracy). Combination with grading reached an AUC: 0.90âÂ±â0.07, irrespective of tumor origin. Circulating transcripts correlated accurately (94 %) with PRRT responders (SD+PR+CR; 97 %) vs. non-responders (91 %). \n",
      "Modlin et al (2016)  examined whether a blood-based multianalyte neuroendocrine gene transcript assay (NETest)  would define tumor cytoreduction and therapeutic efficacy. A total of 35 GEP-NETs in 2 groups were evaluated. I: after surgery (R0, n = 15; residual, n = 12); II: nonsurgery (n = 8: embolization with gel-foam alone [bland: n = 3]), transarterial chemoembolization (n = 2), and radiofrequency embolization (n = 3). Measurement (quantitative real-time-polymerase chain reaction) and chromogranin A (CgA; enzyme-linked immunosorbent assay) were undertaken preoperatively and 1 month after treatment. NETest score was increased in 35 (100%) preoperatively; 14 (40%) had increased CgA (Ï(2) = 30, P < 2 Ã 10(-8)). Resection reduced NETest from 80 Â± 5% to 29% Â± 5, (P < .0001). CgA decrease was insignificant (14.3 Â± 1.6 U/L to 12.2 Â± 1.7 U/L). NETest decreases correlated with diminished tumor volume (R(2) = 0.29, P = .03). Cytoreduction significantly reduced NETest from 82 Â± 3% to 41% Â± 6, P < .0001). CgA was not decreased (21.4 Â± 5.5 U/L to 18.4 Â± 10.1 U/L). Four (36%) of 11 R0s with increased NETest at 1 month developed positive imaging (sensitivity 100%, specificity 20%). One hundred percent (ablated group) were transcript- and image-positive. \n",
      "Modlin et al (2015)  reported the sensitivity and selectivity of the NETest to detect tumors with reference to other benign and malignant gastrointestinal diseases. A total of 179 cases (gastrointestinal tumors: n=81; pancreatic disease: n=98) were prospectively collected and assessed using the NETest or chromogranin A (CgA) to determine metrics for detecting small intestinal and pancreatic NETs. For intestinal carcinoids, the accuracy of the NETest was 93% (all NETs positive and 3 (12%) colorectal tumors were positive). CgA was positive in 80%, but 29% (n=7) of colorectal cancers were CgA positive. For pancreatic disease, the NETest accuracy was 94% (96% NETs positive, 2 (6%) of intraductal papillary  ucinous neoplasms (IPMNs) were positive). The accuracy of CgA was 56% (29% of pancreatic NETs were CgA positive). Overall, the NETest was significantly more sensitive than CgA for the detection of small intestinal (area under the curve 0.98 vs. 0.75 P<0.0001) and pancreatic NETs (0.94 vs. 0.52, P<0.0001). NETest scores were elevated (P<0.05) in extensive disease and were more accurate (76-80%) than CgA levels (20-32%). The metrics of the multianalyte NETest met the performance criteria proposed by the National Institutes of Health for biomarkers, whereas CgA measurement did not. \n",
      "Modlin et al (2014) evaluated a PCR-based 51 transcript signature (NETest) and compared it to chromogranin A (CgA), pancreastatin (PST) and neurokinin A (NKA). The multigene signature was evaluated in two groups: i) a validation set of 40 NETs and controls and ii) a prospectively collected group of NETs (n=41, 61% small intestinal, 50% metastatic, 44% currently treated and 41 age-sex matched controls). Samples were analyzed by a two-step PCR (51 marker genes) protocol and ELISAs for CgA, PST and NKA. Sensitivity comparisons included Ï(2), non-parametric measurements, ROC curves and predictive feature importance (PFAI) analyses. NETest identified 38 of 41 NETs. Performance metrics were: sensitivity 92.8%, specificity 92.8%, positive predictive value 92.8% and negative predictive value 92.8%. Single analyte ELISA metrics were: CgA 76, 59, 65, and 71%; PST 63, 56, 59, and 61% and NKA 39, 93, 84, and 60%. The AUCs (ROC analysis) were: NETest: 0.96Â±0.025, CgA: 0.67Â±0.06, PST 0.56Â±0.06, NKA: 0.66Â±0.06. NETest significantly outperformed single analyte tests (area differences: 0.284-0.403, Z-statistic 4.85-5.9, P<0.0001). PFAI analysis determined NETest had most value (69%) in diagnosis (CgA (13%), PST (9%), and NKA (9%)). Test data were consistent with the validation set (NETest >95% sensitivity and specificity, AUC =0.98 vs single analytes: 59-67% sensitivity, AUCs: 0.58-0.63).\n",
      "Chen et al (2017) commented that NETEST and other novel biomarkers are promising biomarkers in gastroenteropancreatic neuroendocrine tumors with potential clinical benefit, but further research is needed before their clinical application. \n",
      "Pulmotype\n",
      "A multiantibody immunohistochemistry (IHC) assay (eg, Pulmotype) purportedly aids in the differentiation of squamous and adenocarcinoma histology for NSCLC. The assay uses tissue from a lung cancer biopsy to measure five biomarkers: cytokeratin 5/6 (CK5/6), mucin-1 (MUC-1), tripartite motif containing protein 29 (TRIM 29), carcinoembryonic antigen-related cell adhesion molecule (CEACAM) and SLC7A5. The results of these measurements are applied to an algorithm, resulting in a class assignment.\n",
      "NexCourse IHC4\n",
      "NexCourse IHC4 by AQUA Technology is a test proposed to determine the risk of breast cancer recurrence by analyzing protein expression of estrogen receptor (ER)/progesterone receptor (PR), HER2 and Ki67. Determination of ER/PR is performed routinely on all individuals with invasive breast cancer using immunohistochemistry (IHC) to select those individuals who are most likely to respond to hormone therapy.\n",
      "EndoPredict\n",
      "EndoPredict is a multi-gene assay that predicts the likelihood of women with estrogen receptor (ER)-positive and human epidermal growth factor receptor 2 (HER2)-negative breast cancer developing metastases within 10 years of the initial diagnosis (NICE, 2015). The test combines measurements of gene expression (the EndoPredict [EP] score) with nodal status and tumor size to generate a comprehensive risk score (the EPclin score) which is used to identify tumor types that will not benefit from chemotherapy.\n",
      "Guidelines from the American Society for Clinical Oncology state that, \"[i]f a patient has ER/PgR-positive, HER2-negative (node-negative) breast cancer, the clinician may use the 12-gene risk score (EndoPredict; Sividon Diagnostics, KÂ¨oln, Germany) to guide decisions on adjuvant systemic chemotherapy.\" This is amoderate strength recommendation based upon intermediate-quality evidence. The guidelines recommend against the use of the 12-gene scoreto guide decisions onadjuvant systemic chemotherapy in ER/PgR-positive, HER2-negative (node-positive breast cancer. The guidelines also recommend against the use of Endopredict in patients with HER2-positive breast cancer or TN breast cancer. The guidelines recommended against the use of EndoPredict to guide decisions on extended endocrine therapy for patients with ER/PgR-positive, HER-2 negative (node-negative) breast cancer who have had 5 years of endocrine therapy without evidence of recurrence. \n",
      "An assessment by the National Institute for Health and Care Excellence (NICE, 2015) found that the published clinical evidence comes from 3 analytical validation and 5 clinical validation studies in which the test was generally shown to be reproducible and to have prognostic power. In 1 impact evaluation study, EndoPredict results were reported to change treatment decisions. A cost-effectiveness analysis found that using EndoPredict in combination with non-UK clinical guidelines was less costly and more effective than clinical guideline risk stratification alone.\n",
      "Cancer Care Ontario Guidelines (Chang, et al., 2016) state: \"Although no assay represents a gold standard, Oncotype DX is supported by the widest range of evidence for prognosis and prediction of chemotherapy benefit, while both Prosigna and EndoPredict have evidence-based validity in providing some of the same or similar clinical information.\"\n",
      "Immunohistochemistry 4 (IHC4)\n",
      "IHC4 measures the levels of 4 key proteins (ER, PR, HER2 and Ki-67) in addition to classical clinical and pathological variables (for example, age, nodal status, tumour size and grade) and calculates a risk score for distant recurrence using an algorithm (NICE, 2013). Quantitative assessments of ER, PR, and Ki-67 are needed for the IHC4 test. An online calculator for IHC4 is in development. The test uses formalin-fixed paraffin-embedded samples.\n",
      "Guidelines from the American Society for Clinical Oncology (2016) state: \"If a patient has ER/PgR-positive, HER2-negative (node-positive or node-negative) breast cancer, the clinician should not use immunohistochemistry 4 (IHC4) to guide decisions on adjuvant systemic chemotherapy.\" This is a moderate-strength recommendation based upon intermediate-quality evidence. The ASCO guidelines recommend the use of IHC4 to guide decisions on adjuvant systemic therapy for patien with HER2-positive breast cancer or TN breast cancer. The guidelines also recommended against the use of IHC4 to guide decisions on extended endocrine therapy for patients with ER/PgR-positive, HER-2 negative (node-negative) breast cancer who have had 5 years of endocrine therapy without evidence of recurrence.  \n",
      "Tumor Infiltrating Lymphocytes\n",
      "Guidelines from the American Society for Clinical Oncology (2016) state: \"If a patient has ER/PgR-positive, HER2-negative (node-positive or node-negative) breast cancer, the clinician should not use tumor-infiltrating lymphocytes to guide decisions on adjuvant systemic therapy.\" This is a strong recommendation based upon insufficient evidence. The guidelines recommend against the use of tumor-infiltrating lymphocytes to guide decisions on adjuvant systemic therapy in patients with HER2-positive breast cancer or TN breast cancer.  \n",
      "Other Markers to Guide Adjuvant Therapy in Breast Cancer\n",
      "Oncology (2016) recommends against CEP17 duplication, TIMP-1, FOXP3 and microtubule-associated protein Tau mRNA expression or mRNA expression by IHC to guide adjuvant chemotherapy selection. The ASCO guidelines also recommend against CYP2D6 polymorphisms and p27 to guide endocrine therapy selection.  \n",
      "Active Surveillance of Prostate Cancers in Men with \"Favorable\" Intermediate-Risk Disease\n",
      "Mulcahy (2016) stated that the NCCN is the first major organization in the U.S. that has broadened the scope of prostate cancers that qualify for active surveillance to include men with \"favorable\" intermediate-risk disease.  Active surveillance includes ongoing disease monitoring (with PSA testing, biopsies, and imaging), but does not include definitive treatment, such as surgery or radiation, or related harms, such as erectile dysfunction and incontinence.  Previously, the NCCN and other organizations (e.g., the American Urological Association [AUA]) have recommended active surveillance only for very-low-risk and low-risk prostate cancers.  However, the NCCN is now recommending active surveillance for intermediate-risk prostate cancer with a Gleason score of 7 (3+4), which is considered favorable (the grade 3 prostate cancer is predominant accounting for at least 50 % of the biopsied tissue, and the grade 4 is secondary accounting for at least 5 % but less than 50 %).  Favorable intermediate risk also requires that less than 50 % of a patient's biopsy cores are positive, and that he has no more than 1 NCCN intermediate-risk factor.  Those risk factors include a tumor stage of T2b to T2c and a PSA value of 10 to 20 ng/ml.  Moreover, the new NCCN recommendation does not include active surveillance for \"unfavorable\" intermediate-risk cancer, such as that with a Gleason score of 7 (4+3).\n",
      "Dr. James Mohler of the NCCN Prostate Cancer Panel noted that the use of active surveillance in select men with intermediate-risk prostate cancer has been underway for years at major academic centers.  However, the new NCCN recommendation is a \"little more inclusive than what many urologists have used for selecting these [favorable intermediate-risk] patients.  Specifically, it is \"more liberal\" in terms of the percentage of positive biopsies it allows and, thus, will allow an even higher percentage of intermediate-risk patients to be monitored with active surveillance\".  The recommendation is based on a radiation oncology study in which 1,024 patients with intermediate-risk prostate cancer underwent radiotherapy with or without androgen-deprivation therapy (Zumsteg et al, 2013).  In that study, the authors analyzed recurrence-free survival and distant metastases outcomes, and concluded that intermediate-risk disease is \"heterogenous\" with \"favorable and unfavorable subsets\" -- groupings that the NCCN is now using.  The idea that active surveillance should be used in favorable intermediate-risk disease was endorsed by Dr. Ann Raldow and her colleagues of the Harvard Radiation Oncology Program.  In their observational study, Raldow et al (2015) compared 3,972 men with low-risk prostate cancer with 1,608 men with favorable intermediate-risk prostate cancer, all of whom were treated with brachytherapy from 1997 to 2013.  These investigators found that rates of prostate-cancer-specific mortality and all-cause mortality were similar in the 2 groups, and concluded that these findings provided \"evidence to support active surveillance as an initial approach for men with favorable intermediate-risk prostate cancer\".\n",
      "Dr. Mohler stated that the NCCN Prostate Cancer Panel has been advocating active-surveillance for some time.  In 2010, the NCCN was the first organization to recommend active surveillance as the sole initial therapy for many men.  The use of active surveillance has increased in recent years in the U.S.; estimates range from a high of 50 % of men with low-risk prostate cancer in Michigan, where an insurance-industry-funded awareness program is underway, to a low of 8 % nationally.  Medscape Medical News reported that another estimate of the prevalence of the practice came from CaPSURE, a prostate cancer registry that has been collecting data on men managed at 47 clinical, primarily community-based, sites.  An analysis of the data collected from 2008 to 2013 showed that the primary treatment of 38.4 % of men with low-risk tumors was watchful waiting or active surveillance.  Furthermore, Dr. Stacy Loeb of the New York University stated that the U.S. has lagged behind certain European countries in the adoption of active surveillance.  As of 2013, the use of active surveillance in Sweden was 78 % for men with very-low-risk disease and 59 % for men with low-risk disease.\n",
      "CDX2 as a Prognostic Biomarker in Colon Cancer\n",
      "Dalerbra and colleagues (2016) stated that the identification of high-risk stage II colon cancers is key to the selection of patients who require adjuvant treatment after surgery. Microarray-based multigene-expression signatures derived from stem cells and progenitor cells hold promise, but they are difficult to use in clinical practice.  These investigators used a new bioinformatics approach to search for biomarkers of colon epithelial differentiation across gene-expression arrays and then ranked candidate genes according to the availability of clinical-grade diagnostic assays.  With the use of subgroup analysis involving independent and retrospective cohorts of patients with stage II or stage III colon cancer, the top candidate gene was tested for its association with DFS and a benefit from adjuvant chemotherapy.  The transcription factor CDX2 ranked first in the screening test.  A group of 87 of 2,115 tumor samples (4.1 %) lacked CDX2 expression.  In the discovery data set, which included 466 patients, the rate of 5-year DFS was lower among the 32 patients (6.9 %) with CDX2-negative colon cancers than among the 434 (93.1 %) with CDX2-positive colon cancers (HR for disease recurrence, 3.44; 95 % CI: 1.60 to 7.38; p = 0.002).  In the validation data set, which included 314 patients, the rate of 5-year DFS was lower among the 38 patients (12.1 %) with CDX2 protein-negative colon cancers than among the 276 (87.9 %) with CDX2 protein-positive colon cancers (HR, 2.42; 95 % CI: 1.36 to 4.29; p = 0.003).  In both these groups, these findings were independent of the patient's age, sex, and tumor stage and grade.  Among patients with stage II cancer, the difference in 5-year DFS was significant both in the discovery data set (49 % among 15 patients with CDX2-negative tumors versus 87 % among 191 patients with CDX2-positive tumors, p = 0.003) and in the validation data set (51 % among 15 patients with CDX2-negative tumors versus 80 % among 106 patients with CDX2-positive tumors, p = 0.004).  In a pooled database of all patient cohorts, the rate of 5-year DFS was higher among 23 patients with stage II CDX2-negative tumors who were treated with adjuvant chemotherapy than among 25 who were not treated with adjuvant chemotherapy (91 % versus 56 %, p = 0.006).  the authors concluded that lack of CDX2 expression identified a subgroup of patients with high-risk stage II colon cancer who appeared to benefit from adjuvant chemotherapy.  They stated that given the exploratory and retrospective design of the study, these findings need to be validated; they advocated these results to be confirmed within the framework of randomized, clinical trials, in conjunction with genomic DNA sequencing studies.\n",
      "PDGFRB Testing\n",
      "The National Comprehensive Cancer Networkâs Biomarkers Compendium (2016) recommends the following for PDGFRB testing:\n",
      "Myelodysplastic Syndromes (MDS): Helpful in some clinical situations: Evaluate CMML patients for 5q31-33 translocations and/or PDGFR beta gene rearrangements. (Category of Evidence: 2A).\n",
      "Non-Melanoma Skin Cancers - Dermatofibrosarcoma Protuberans (DFSP): Tumors lacking the t(17;22) translocation may not respond to imatinib. Molecular analysis of a tumor using cytogenetics may be useful prior to the institution of imatinib therapy.  (Category of Evidence: 2A).\n",
      "Guardant360\n",
      "The Guardant360 panel analyzes cell-free circulating tumor DNA (liquid biopsy) for 73 genes associated with a wide variety of solid tumors.\n",
      "Noting that data on the influence of hybrid capture (HC)-based NGS on treatment are limited, Rozenblum, et al. (2017) investigated its impact on treatment decisions and clinical outcomes in a series of patients at a cancer center. This retrospective study included patients with advanced lung cancer on whom HC-based NGS with broad gene panels was performed between November 2011 and October 2015. HC-based NGS was performed upon the recommendation of the treating physician, mostly on the basis of young age and smoking history. The results of standard molecular testing for EGFR mutations and ALK rearrangements were negative before HC-based NGS in 80.2% (81 of 101) and 70.3% (71 of 101) of the patients, respectively. Upfront HC-based NGS was performed on 15 patients because of very little biopsy material. HC-based NGS was performed off-site on tumor samples with FoundationOne (Foundation Medicine, Inc., Cambridge, MA) (n = 82) or on blood samples using a liquid biopsy approach with Guardant360 if the tissue sample had been exhausted (n= 18). The study focused on gene analyses (GAs) with potential clinical relevance. Initial analysis (level 1) included GAs associated with U.S. Food and Drug Administrationâapproved anticancer therapies (including off-label drugs) for all cancer types. A subsequent analysis (level 2) included GAs with appropriate evidence-based targeted agents with antidriver activity in lung cancer, as recommended by the National Comprehensive Cancer Network (NCCN) guidelines for NSCLC. GAs associated with investigational treatments were not included in the current analysis. Demographic and clinicopathologic characteristics, treatments, and outcome data were collected. A total of 101 patients were included (median age 63 years [53% females, 45% never-smokers, and 85% with adenocarcinoma]). HC-based NGS was performed upfront and after EGFR/ALK testing yielded negative or inconclusive results in 15% and 85% of patients, respectively. In 51.5% of patients, HC-based NGS was performed before first-line therapy, and in 48.5%, it was performed after treatment failure. HC-based NGS identified clinically actionable genomic alterations in 50% of patients, most frequently in EGFR (18%), Ret proto-oncogene (RET) (9%), ALK (8%), Mesenchymal-epithelial transition factor (MET) receptor tyrosine kinase gene (6%), and erb-b2 receptor tyrosine kinase 2 gene (ERBB2) (5%). In 15 patients, it identified EGFR/ALK aberrations after negative results of prior standard testing. Treatment strategy was changed for 43 patients (42.6%). The overall response rate in these patients was 65% (complete response 14.7%, partial response 50%). Median survival was not reached. Immunotherapy was administered in 33 patients, mostly without an actionable driver, with a presenting disease control rate of 32%, and with an association with tumor mutation burden. The authors noted a number of limitations of this study, including its retrospective nature, its small sample size, and its being a single-center study. In addition, the high percentage of never-smokers, the preponderance of female patients, and the relatively young median age of the patient group represented a selection bias with a high pretest probability for the existence of driver mutation. The authors noted that the results of large prospective trials such as the UK National Lung Matrix Trial and the National Cancer Institute's Molecular Analysis for Therapy Choice Program are thus eagerly anticipated.\n",
      "Kim et al (2017) reported on an interim analysis of an open-label prospective, clinical trial of ctDNA in patients with metastatic NSCLC, gastric cancer (GC), and other cancers. The investigators reported that somatic alterations were detected in 59 patients with GC (78%), and 25 patients (33%) had targetable alterations (ERBB2, n = 11;MET, n = 5; FGFR2, n = 3; PIK3CA, n = 6). In NSCLC, 62 patients (85%) had somatic alterations, and 34 (47%) had targetable alterations (EGFR, n = 29; ALK, n = 2; RET, n = 1; ERBB2, n = 2). A subgroup of subjects (10 with GC and 17 with NSCLC) who had tissue for confirmation of ctDNA findings were treated with targeted therapy. The investigators reported that response rate and disease control rate were 67% and 100%, respectively, in GC and 87% and 100%, respectively, in NSCLC. The authors noted that this is the first prospective study to examine the clinical utility of comprehensive ctDNA genomic testing to guide matched therapy selection. The authors stated that, because this study was not randomized, its primary limitation is the potential for selection bias to enroll patients more likely to benefit. In addition, the cohort is heterogeneous, including patients at varying lines of therapy and with various concomitant treatments, which limits conclusions in this interim analysis. Not all patients with targetable alterations could receive matched therapy because of the various requirements of the multiple parallel matched therapy substudy protocols, performance status, or loss to follow-up. The authors stated that the final analysis will help to address the modest sample size of this interim analysis as well as report on progression-free survival. The authors stated that future studies should examine ctDNAguided matched therapy outcomes in more racially diverse cohorts.\n",
      "Noting that there is a paucity of data on the concordance between plasma cell-free circulating tumor DNA (ctDNA) and tissue-based genomic testing, Villaflor, et al. (2016) reported on a descriptive study of subjects with NSCLC undergoing analysis of ctDNA using Guardant360 next-generation sequencing assay at a single institution. The authors stated that this study is the first clinic-based series of NSCLC patients assessing outcomes of targeted therapies using a commercially available ctDNA assay. Of the 90 patients submitted for ctDNA analysis as part of clinical care, 68 had provided informed consent for inclusion in this study. Thirty-eight samples from the 68 subjects were tested using the 54-gene ctDNA panel while the remaining 31 samples were analyzed on the 68-gene ctDNA panel. Of note, the 54-gene panel did not include ALK, RET or ROS1 fusions. Tissue-based testing was performed on 44 subjects using 9 different testing platforms. Demographic, clinicopathologic information and results from tissue and plasma-based genomic testing were reviewed for each subject. The majority of patients had a diagnosis of lung adenocarcinoma (n = 55, 81%), with the remainder lung squamous cell carcinoma (n = 12, 17.7%) and other lung cancers (n = 1, 1.3%). Over 80% of patients had detectable ctDNA. Thirty-one patients had matched tissue and blood samples; the reason for lack of tissue results for the remaining 37 patients was not routinely documented. In cases with detectable ctDNA and completed tissue analysis, an EGFR activating was found in both tissue and blood in 5 paired samples, and in tissue only in 2 samples (71% concordance). The time between biopsy and blood draw ranged from 0 days to 7 years, with an average of 8.8 months and median of 1.4 years between biopsy and blood draw. The investigators found no correlation between concordance and timing of blood draw versus tissue biopsy. A total of 9 subjects with paired tissue and blood samples had an EGFR driver mutation identified in plasma and tissue (n = 5), plasma only (n = 1) or tissue only (n = 3). Eight of these individuals were treated with erlotinib or afatinib at first or second line. Two patients were still responding to therapy at the time of data analysis. Of the 6 remaining patients, the median progression-free survival was 11.5 months (range 7.5 monthsâ29 months; 95% CIâ5.7â28.7). The investigators stated that these data suggest that biopsy-free ctDNA analysis is a viable first choice when the diagnostic tissue biopsy is insufficient for genotyping or at the time of progression when a repeated invasive tissue biopsy is not possible/preferred. The authors noted however, that the numbers in this series are modest and further research in larger prospective cohorts is needed.\n",
      "Thompson et al (2016) evaluated the feasibility of using cell-free circulating tumor DNA (ctDNA) NGS as a complement or alternative to tissue NGS in a single-center observational study. A total of 112 plasma samples obtained from a consecutive study of 102 prospectively enrolled patients with advanced NSCLC were subjected to ultra-deep sequencing of 68 or 70 genes and matched with tissue samples, when possible. The investigators detected 275 alterations in 45 genes, and at least one alteration in the ctDNA for 86 of 102 patients (84%), with EGFR variants being most common. ctDNA NGS detected 50 driver and 12 resistance mutations, and mutations in 22 additional genes for which experimental therapies, including clinical trials, are available. Although ctDNA NGS was completed for 102 consecutive patients, tissue sequencing was only successful for 50 patients (49%). The overall concordance for all variants covered and detected by both platforms was 60%. When wild-type calls, that is, genes for which no variants were detected, are considered, the overall concordance was 97.5%. Actionable EGFR mutations were detected in 24 tissue and 19 ctDNA samples, yielding concordance of 79%, with a shorter time interval between tissue and blood collection associated with increased concordance (P = 0.038). ctDNA sequencing identified eight patients harboring a resistance mutation who developed progressive disease while on targeted therapy, and for whom tissue sequencing was not possible.\n",
      "Schwaederle et al (2016) extracted plasma from 171 patients with a variety of cancers and analyzed the plasma for ctDNA (54 genes and copy number variants (CNVs) in three genes (EGFR, ERBB2 and MET)). The most represented cancers were lung (23%), breast (23%), and glioblastoma (19%). Ninety-nine patients (58%) had at least one detectable alteration, where actionability was defined as an alteration that was either the direct target or a pathway component that could be targeted by at least one FDA approved or investigational drug in a clinical trial. The most frequent alterations were TP53 (29.8%), followed by EGFR (17.5%), MET (10.5%), PIK3CA (7%), and NOTCH1 (5.8%). In contrast, of 222 healthy volunteers, only one had an aberration (TP53). Ninety patients with non-brain tumors had a discernible aberration (65% of 138 patients; in 70% of non-brain tumor patients with an alteration, the anomaly was potentially actionable). Nine of 33 patients (27%) with glioblastoma had an alteration (6/33 (18%) potentially actionable). Overall, sixty-nine patients had potentially actionable alterations (40% of total; 69.7% of patients (69/99) with alterations); 68 patients (40% of total; 69% of patients with alterations), by an FDA-approved drug. In summary, 65% of diverse cancers (as well as 27% of glioblastomas) had detectable ctDNA aberration(s), with the majority theoretically actionable by an approved agent. The authors noted a number of study limitations. First, this study included a limited number of patients in each histology. Second, clinical annotation was not available since the database was de-identified. Third, the definition of \"actionable\" and the level of evidence needed for such a determination is a matter of debate and in constant evolution. Fourth, the use of tissue-based next generation sequencing as a comparison to establish clinical utility was not accessible for this group of de-identified patients. Finally, whether or not the patients would have responded to these drugs could not be addressed in this study, and will require further investigation.\n",
      "Liang et al (2016) performed a retrospective chart review of 100 patients with stage 4 or high-risk stage 3 breast cancer. Of the 100 patients included in this study, 29 had a tissue analysis done during the course of treatment. Only the specific genomic alterations tested in both the cell-free DNA (cfDNA) and tissue DNA were included in this analysis. Of the 29 patients with tissue analysis, 6 had no evidence of disease at the time of cfDNA analysis and were excluded from the comparative analysis of genomic alterations found between cfDNA and tissue DNA. A total of 55 single nucleotide variants (SNVs) and 4 copy number variants (CNVs) were evaluated for both cfDNA and tissue DNA from the 23 remaining patients. The degree of agreement between genomic alterations found in tumor DNA (tDNA) and cfDNA was determined by Cohen's Kappa. Clinical disease progression was compared to mutant allele frequency using a 2-sided Fisher's exact test. The presence of mutations and mutant allele frequency was correlated with PFS using a Cox proportional hazards model and a log-rank test. The most commonly found genomic alterations were mutations in TP53 and PIK3CA, and amplification of EGFR and ERBB2. PIK3CA mutation and ERBB2 amplification demonstrated robust agreement between tDNA and cfDNA (Cohen's kappa = 0.64 and 0.77, respectively).  TP53 mutation and EGFR amplification demonstrated poor agreement between tDNA and cfDNA (Cohen's kappa = 0.18 and 0.33, respectively).  The directional changes of TP53 and PIK3CA mutant allele frequency were closely associated with response to therapy (p = 0.002).  The investigators stated that the presence of TP53 mutation (p = 0.0004) and PIK3CA mutant allele frequency [p = 0.01, HR 1.074 (95 % CI: 1.018 to 1.134)] was excellent predictors of PFS.  The authors concluded that identification of selected cancer-specific genomic alterations from cfDNA may be a non-invasive way to monitor disease progression, predict PFS, and offer targeted therapy.  They noted that this study was limited by its small sample size and the inherent nature of retrospective data collection of existing genomic information.\n",
      "Aggarwal et al (2019) noted that the clinical implications of adding plasma-based circulating tumor DNA next-generation sequencing (NGS) to tissue NGS for targetable mutation detection in non-small cell lung cancer (NSCLC) have not been formally assessed.  In a prospective, cohort study, these researchers examined if plasma NGS testing was associated with improved mutation detection and enhanced delivery of personalized therapy in a real-world clinical setting.  This trial enrolled 323 patients with metastatic NSCLC who had plasma testing ordered as part of routine clinical management.  Plasma NGS was performed using a 73-gene commercial platform.  Patients were enrolled at the Hospital of the University of Pennsylvania from April 1, 2016, through January 2, 2018.  The database was locked for follow-up and analyses on January 2, 2018, with a median follow-up of 7 months (range of 1 to 21 months).  The number of patients with targetable alterations detected with plasma and tissue NGS; the association between the allele fractions (AFs) of mutations detected in tissue and plasma; and the association of response rate with the plasma AF of the targeted mutations.  Among the 323 patients with NSCLC (60.1 % women; median age of 65 years [range of 33 to 93]), therapeutically targetable mutations were detected in EGFR, ALK, MET, BRCA1, ROS1, RET, ERBB2, or BRAF for 113 (35.0 %) overall; 94 patients (29.1 %) had plasma testing only at the discretion of the treating physician or patient preference.  Among the 94 patients with plasma testing alone, 31 (33.0 %) had a therapeutically targetable mutation detected, thus obviating the need for an invasive biopsy.  Among the remaining 229 patients who had concurrent plasma and tissue NGS or were unable to have tissue NGS, a therapeutically targetable mutation was detected in tissue alone for 47 patients (20.5 %), whereas the addition of plasma testing increased this number to 82 (35.8 %); 36 of 42 patients (85.7 %) who received a targeted therapy based on the plasma result achieved a complete response (CR) or a partial response (PR) or stable disease (SD).  The plasma-based targeted mutation AF had no correlation with depth of Response Evaluation Criteria in Solid Tumors (RECIST) response (râ=â-0.121; pâ=â0.45).  The authors concluded that given the ease of obtaining plasma-based genotyping and the success observed with such a non-invasive approach, these findings argued for incorporation of plasma-based genotyping into routine clinical management of patients with NSCLC.\n",
      "The authors stated that this study had several drawbacks.  This single-center study was conducted among physicians who were comfortable ordering and interpreting plasma NGS tests.  This user bias probably enriched for patients who had plasma NGS only and were likely to have targetable mutations.  A sizeable proportion of patients underwent testing after progression to detect resistance mutations, which likely increased the frequency of patients with EGFR T790M.  Moreover, this study only considered  plasma NGS testing at a single point.  The clinical utility of longitudinal plasma NGS-based monitoring is an area of active study in this group.\n",
      "In an editorial that accompanied the afore-mentioned study by Aggarwal et al (2019), Gyawali and West (2018) noted that \"Putting aside the question of whether and when NGS is appropriate, what does the study by Aggarwal and colleagues demonstrate for the role of plasma vs tissue NGS?  We cannot conclude from this work that plasma testing should obviate the need for tissue NGS in most patients, since 29 % of the patients with a therapeutically targetable mutation and who had undergone NGS testing from both plasma and tissue had the mutation detected in tissue only.  But the study compellingly demonstrates that plasma NGS can obviate the need for tissue NGS in patients in whom plasma testing demonstrates a mutation, given the response and disease control rate among patients who had therapeutically targetable mutations identified from plasma.  The relatively high rate of molecular marker detection from plasma also offers a strong option for patients for whom tissue is not available and challenging to obtain.  These results, combined with the patient satisfaction with the relative ease of providing blood rather than a solid tissue sample, suggest a clinical strategy of pursuing plasma NGS first, then tissue NGS if plasma NGS cannot detect relevant mutations.  Another driver of plasma NGS is the cost-effectiveness of liquid biopsy over tissue biopsy, as suggested in the Statement Paper by the International Association for the Study of Lung Cancer5; however, data to support this claim are still lacking.  In the context of initial workup of advanced non-squamous NSCLC, for which many therapeutically targetable mutations are potentially present, broader clinical use of NGS from one source or another seems reasonable, based on cost and time and tissue efficiency.  However, this may not hold true in other contexts in which the relevant targets are very limited, such asT790Min acquired resistance on an earlier-generation EGFR TKI, or are extremely infrequent and/or are not clinically relevant, as in squamous NSCLC, acquired resistance to various other driver mutations, and many other cancer settings.  In summary, mounting data now support a role for plasma NGS as a helpful tool to supplement or even obviate the need for often scarce and difficult-to-obtain tissue for NGS testing, but this should not circumvent the central question of whether NGS testing will improve clinical outcomes and thus whether it should be performed at all.  Next-generation sequencing should not be presumed to be the right tool for every job â¦ A shotgun approach may be appropriate if there is a sufficient chance of hitting a target suspected to be there, but we do not know exactly where; however, there are more accurate and precise weapons if we have a better idea where the true target is.  If not, and if there is little reason to expect the existence of a real target, merely having a readily available shotgun should not lead us to shoot blindly in the dark without acknowledging that we may do unexpected damage\".\n",
      "Odegaard et al (2018) stated that liquid biopsies are powerful tools that enable non-invasive genotyping of advanced solid tumors; however, comprehensive, structured validation studies employing validated orthogonal comparator methods are lacking.  These researchers analytically and clinically validated a circulating cell-free tumor DNA sequencing test for comprehensive tumor genotyping and demonstrated its clinical feasibility.  Analytical validation was conducted according to established principles and guidelines.  Blood-to-blood clinical validation comprised blinded external comparison to clinical digital droplet PCR across 222 consecutive biomarker-positive clinical samples.  Blood-to-tissue clinical validation comprised comparison of Digital Sequencing calls to those documented in the medical record of 543 consecutive lung cancer patients.  Clinical experience was reported from 10,593 consecutive clinical samples.  Digital sequencing technology enabled variant detection down to 0.02 % to 0.04 % allelic fraction/2.12 copies with less than or equal to 0.3 %/2.24 to 2.76 copies 95 % limits of detection while maintaining high specificity (prevalence-adjusted positive predictive value (PPV) greater than 98 %).  Clinical validation using orthogonal plasma- and tissue-based clinical genotyping across more than 750 patients demonstrated high accuracy and specificity (positive percent agreement (PPAs) and negative percent agreement (NPAs) greater than 99 % and PPVs 92 to 100 %).  Clinical use in 10,593 advanced adult solid tumor patients demonstrated high feasibility (greater than 99.6 % technical success rate) and clinical sensitivity (85.9 %), with high potential actionability (16.7 % with FDA-approved on-label treatment options; 72.0 % with treatment or trial recommendations), particularly in non-small cell lung cancer where 34.5 % of patient samples comprised a directly targetable standard-of-care biomarker.  The authors concluded that high concordance with orthogonal clinical plasma- and tissue-based genotyping methods supported the clinical accuracy of digital sequencing across all 4 types of targetable genomic alterations.  Digital sequencingâs clinical applicability is further supported by high rates of technical success and biomarker target discovery.\n",
      "McCoach et al (2018) stated that patients with advanced NSCLC whose tumors harbor anaplastic lymphoma kinase (ALK) gene fusions benefit from treatment with ALK inhibitors (ALKi).  Analysis of cell-free circulating tumor DNA (cfDNA) may provide a non-invasive way to identify ALK fusions and actionable resistance mechanisms without an invasive biopsy.  The Guardant360 (G360; Guardant Health) de-identified database of NSCLC cases was queried to identify 88 consecutive patients with 96 plasma-detected ALK fusions.  G360 is a clinical cfDNA NGS test that detects point mutations, select copy number gains, fusions, insertions, and deletions in plasma.  Identified fusion partners included EML4 (85.4 %), STRN (6 %), and KCNQ, KLC1, KIF5B, PPM1B, and TGF (totaling 8.3 %); 42 ALK-positive patients had no history of targeted therapy (cohort 1), with tissue ALK molecular testing attempted in 21 (5 negative, 5 positive, and 11 tissue insufficient).  Follow-up of 3 of the 5 tissue-negative patients showed responses to ALKi; 31 patients were tested at known or presumed ALKi progression (cohort 2); 16 samples (53% ) contained 1 to 3 ALK resistance mutations.  In 13 patients, clinical status was unknown (cohort 3), and no resistance mutations or bypass pathways were identified.  In 6 patients with known EGFR-activating mutations, an ALK fusion was identified on progression (cohort 4; 4 STRN, 1 EML4; 1 both STRN and EML4); 5 harbored EGFR T790M. The authors concluded that in this cohort of cfDNA-detected ALK fusions, these researchers demonstrated that comprehensive cfDNA NGS provided a non-invasive means of detecting targetable alterations and characterizing resistance mechanisms on progression.\n",
      "The authors stated that this study had several drawbacks.  First, this was a retrospective analysis reliant on clinical information provided on sample submission.  Thus, complete treatment history and clinical follow-up was not available (and cannot be verified) for all patients.  This included patient demographic information, type and length of prior therapies, local tissue testing modality, and prior molecular testing results both at diagnosis and progression re-biopsy.  Further, there were limitations to the cfDNA platform including the identification of multiple sub-clonal populations, which may not be clinically relevant to resistance.  Additionally, given G360 is a clinical cfDNA assay, only ALK fusion events that occurred with partners with known biologic significance were reported.  Finally, in this study these researchers identified 6 patients in cohort 2 whose ALK fusion were not identified by cfDNA, instead they were identified by the presence of the ALK resistance mutation.  This reflected the complexity of fusion proteins and the fact that ALK has numerous fusion variants that may hinder identification by small fragment cfDNA analyses.  Additionally, these investigators were unable to estimate the true false negative rate of cfDNA in detecting ALK fusions given the database search parameters.\n",
      "Laufer-Geva et al (2018) stated that NGS of cfDNA enables non-invasive genomic analysis of NSCLC patients.  Although plasma-detected genomic alterations (GAs) have been shown to predict targeted therapy response, evidence of durability of response is lacking or limited to small cohorts as is the impact of cfDNA NGS results on clinical decisions.  This retrospective study of stage IIIB/IV NSCLC patients between the years 2014 and 2017 in Israel used cfDNA NGS (Guardant360) to identify targetable GAs.  These researchers consecutively tested 116 NSCLC patients, 41.4 % before 1st-line therapy (group A), 34.5 % upon progression on chemotherapy or immunotherapy (group B1), and 24.1 % upon progression on EGFR tyrosine kinase inhibitors (group B2).  Targetable GAs were found in 31 % of group A (15 of 48 patients), 32.5 % in group B1 (13 of 40 patients) and 71 % in group B2 (20 of 28 patients).  Treatment decision was changed to targeted therapy in 23 % (11 of 48 patients), 25 % (10 of 40 patients) and 32 % (9 of 28 patients), respectively (total cohort 26 %; 30/116).  Objective response rate (Response Evaluation Criteria in Solid Tumors [RECIST]) was 43 % (12 of 28 patients) including 1 CR, PR in 39 % (11 of 28 patients), SD in 32 % (9 of 28 patients), and progressive disease in 25 % (7 of 28 patients).  Disease control rate was 75 % for 5 months median treatment duration.  The authors concluded that comprehensive cfDNA testing impacted clinical decisions in 1/4 to 1/3 of initial and subsequent lines of treatment in advanced NSCLC patients.  This retrospective study extended previous reports by showing that responses based on cfDNA were durable and change treatment decisions at initial presentation and at progression.\n",
      "The authors stated that limitations of this study included its retrospective nature, although the response and disease control rate (DCR) were consistent with 2 previous prospective studies.  Also, more than 50 % of the at-progression patients were in the 3rd-line of treatment or higher, where response rates would be expected to be lower than published studies of 2nd-line targeted therapies.  Response rate, survival, and duration of treatment in this study population, which was enriched for plasma-positive patients with limited or failed tumor tissue genotyping, may introduce theoretical selection bias as cfDNA may be more likely to be positive in patients with aggressively growing metastases whereas single lesion biopsy-based genotyping is indifferent to whether disease is indolent or aggressive.  If true, however, then the plasma-based treatment results become more, rather than less, compelling.  Nonetheless, the clinical outcomes reported in this study reflected the real-life impact.  Because patients were not randomized, and cfDNA testing ordered on those with a higher pre-test probability of mutation (female, non-smoking, etc.) the prevalence of GAs here may be higher than a cohort enrolled by randomization.\n",
      "Lam and associates (2019) stated that major guidelines do not recommend routine molecular profiling of lung squamous-cell carcinoma (LUSC) because the prevalence of actionable alterations is thought to be low.  Increased utilization of next-generation sequencing (NGS), particularly with cfDNA, facilitates re-evaluation of this premise.  These investigators retrospectively evaluated the prevalence of actionable alterations in 2 distinct LUSC cohorts totaling 492 patients.  A total of 410 consecutive patients with stage 3B or 4 LUSC were tested with a targeted cfDNA NGS assay, and 82 patients with LUSC of any stage were tested with a tissue NGS cancer panel.  In the overall cohort, 467 patients (94.9 %) had a diagnosis of LUSC, and 25 patients (5.1 %) had mixed histology with a squamous component.  A total of 10.5 % of the LUSC subgroup had somatic alterations with therapeutic relevance, including in EGFR (2.8 %), ALK/ROS1 (1.3 %), BRAF (1.5 %), and MET amplification or exon 14 skipping (5.1 %); 16 % of patients with mixed histology had an actionable alteration.  In the LUSC subgroup, 3 evaluable patients were treated with targeted therapy for an actionable alteration; all of them experienced partial response.  The authors concluded that in this large, real-world LUSC cohort, they observed a clinically significant prevalence of actionable alterations.  These researchers stated that further evaluation of the genomic landscape in this setting is needed to potentially identify under-appreciated treatment options.\n",
      "Leighl and colleagues (2019) stated that complete and timely tissue genotyping is challenging, leading to significant numbers of patients with newly diagnosed metastatic NSCLC (mNSCLC) being under-genotyped for all 8 genomic biomarkers recommended by professional guidelines.  These researchers attempted to demonstrate non-inferiority of comprehensive cfDNA relative to physician discretion SOC tissue genotyping to identify guideline-recommended biomarkers in patients with mNSCLC.  Prospectively enrolled patients with previously untreated mNSCLC undergoing physician discretion SOC tissue genotyping submitted a pre-treatment blood sample for comprehensive cfDNA analysis (Guardant360).  Among 282 patients, physician discretion SOC tissue genotyping identified a guideline-recommended biomarker in 60 patients versus 77 cfDNA identified patients (21.3 % versus 27.3 %; p < 0.0001 for non-inferiority).  In tissue-positive patients, the biomarker was identified alone (12/60) or concordant with cfDNA (48/60), an 80 % cfDNA clinical sensitivity for any guideline-recommended biomarker.  For FDA-approved targets (EGFR, ALK, ROS1, BRAF) concordance was greater than 98.2 % with 100 % PPV for cfDNA versus tissue (34/34 EGFR, ALK, or BRAF positive patients).  Utilizing cfDNA in addition to tissue increased detection by 48 %, from 60 to 89 patients, including those with negative, not assessed, or insufficient tissue results.  cfDNA median turnaround time was significantly faster than tissue (9 versus 15 days; p < 0.0001).  Guideline-complete genotyping was significantly more likely (268 versus 51; p < 0.0001).  The authors concluded that a comprehensive, sensitive, and specific cfDNA test used in patients with newly diagnosed mNSCLC successfully identified guideline recommended biomarkers at a rate at least as high as SOC tissue testing and returned these results significantly faster and for a significantly higher proportion of the population.  Moreover, cfDNA-detected guideline recommended biomarkers were invariably present in tissue, when tissue was successfully tested, reinforcing that cfDNA genotyping results may be used in clinical management in the same way tissue genotyping results are currently used.  Lastly, when modeled together, these results suggested that initial biomarker assessment using cfDNA rather than tissue (\"blood first\"), reserving tissue for PD-L1 IHC and reflex testing when cfDNA is negative for any known oncogenic driver mutations, improved biomarker discovery rate, turn-around time, and increased the number of patients with newly diagnosed mNSCLC who receive guideline complete biomarker testing.\n",
      "The authors stated that 1 main drawback of this study was that, while cfDNA testing utilized a single platform, tissue genomic assessment was not standardized but was instead left to physicianâs discretion SOC, which included a variety of methodologies, including PCR, FISH, IHC, and/or NGS.  As only 18 % of patients successfully underwent comprehensive tissue genomic profiling, many alterations that were identified in cfDNA alone were in fact a result of incomplete tissue genotyping due to methodology choice and/or tissue testing failure as opposed to analytical discordance between the tests.  As part of the study design, providers were specifically instructed to not make any changes to their SOC tissue genotyping practices, however, these investigators could not rule out the possibility that the receipt of a cfDNA clinical result may have influenced the decision to pursue further tissue genotyping in instances of sequential testing.  Moreover, these findings may not apply to other cfDNA tests that are less sensitive or less comprehensive.  They stated that while this limited certain comparisons, this design was critical to the fundamental question addressed by this study, whether a well-validated cfDNA test can match or even improve upon SOC tissue methods.\n",
      "An accompanying commentary (Meador and Oxnard, 2019) noted that the sensitivity of cfDNA genotyping can be low in patients with lower metastatic burden, likely due to reduced shed of tumor DNA into the plasma. The commentators stated that this insensitivity of cfDNA sequencing must be acknowledged as a significant barrier to its application. \"Ultimately, negative cfDNA sequencing may be better thanno genotyping at all, but it is not sufficient to rule out the presence of targetable driver mutations given the impaired sensitivity of these assays and unknown rate of tumor shed in any given patient.\"\n",
      "Willis et al (2019) sought to analytically validate microsatellite instability (MSI) testing using Guardant360 according to established guidelines and clinically validate it using 1,145 cfDNA samples for which tissue MSI status based on standard-of-care tissue testing was available. The landscape of cfDNA-based MSI across solid tumor types was investigated in a cohort of 28,459 clinical plasma samples. Clinical outcomes for 16 patients with cfDNA MSI-H gastric cancer treated with immunotherapy were evaluated. In evaluable patients, cfDNA testing accurately detected 87% (71/82) of tissue MSI-H and 99.5% of tissue microsatellite stable (863/867) for an overall accuracy of 98.4% (934/949) and a positive predictive value of 95% (71/75). Concordance of cfDNA MSI with tissue PCR and next-generation sequencing was significantly higher than IHC. Prevalence of cfDNA MSI for major cancer types was consistent with those reported for tissue. Finally, robust clinical activity of immunotherapy treatment was seen in patients with advanced gastric cancer positive for MSI by cfDNA, with 63% (10/16) of patients achieving complete or partial remission with sustained clinical benefit. Limitations included the small number of subjects for which clinical outcomes were evaluated.\n",
      "National Comprehensive Cancer Networkâs clinical practice guideline on \"Non-small cell lung cancer\" (Version 3.2019) states that \"The panel feels that cell-free/circulating tumor DNA testing should not be used in lieu of tissue diagnosis.  Standard and guidelines for cell-free DNA (cfDNA)/circulating tumor DNA testing for genetic alterations have not been established, there is up to a 30 % false-negative rate, and alterations can be detected that are not related to the tumor (e.g., clonal hematopoiesis of indeterminate potential [CHIP] â¦ However, cfDNA testing can be used in specific circumstances if the patient is not medically fit for invasive tissue sampling, or there is insufficient tissue for molecular analysis and follow-up tissue-based analysis will be done if an oncogenic driver is not identified.  Given the previous caveats, careful consideration is required to determine whether cfDNA findings reflect a true driver mutation or an unrelated finding\".  Since the Guardant360 includes a panel of 68 genes, and only about 5 of which are actionable.  The clinical value of the of the entire gene panel of Guardant360 has not been established.\n",
      "Turner and colleagues (2020) stated that ctDNA testing might provide a current assessment of the genomic profile of advanced cancer, without the need to repeat tumor biopsy.  In an open-label, multi-center, multi-cohort, phase-IIa platform trial, these researchers examined the accuracy of ctDNA testing in advanced breast cancer and the ability of ctDNA testing to select patients for mutation-directed therapy.  This study was carried out in 18 United Kingdom hospitals.  Participants were women (aged greater than or equal to 18 years) with histologically confirmed advanced breast cancer and an ECOG performance status of 0 to 2.  Patients had completed at least 1 previous line of treatment for advanced breast cancer or relapsed within 12 months of neoadjuvant or adjuvant chemotherapy.  Patients were recruited into 4 parallel treatment cohorts matched to mutations identified in ctDNA: cohort A comprised patients with ESR1 mutations (treated with IM extended-dose fulvestrant 500 mg); cohort B comprised patients with HER2 mutations (treated with oral neratinib 240 mg, and if estrogen receptor-positive with IM standard-dose fulvestrant); cohort C comprised patients with AKT1 mutations and estrogen receptor-positive cancer (treated with oral capivasertib 400 mg plus IM standard-dose fulvestrant); and cohort D comprised patients with AKT1 mutations and estrogen receptor-negative cancer or PTEN mutation (treated with oral capivasertib 480 mg).  Each cohort had a primary endpoint of confirmed ORR.  For cohort A, 13 or more responses among 78 evaluable patients were needed to infer activity and 3 or more among 16 were needed for cohorts B, C, and D.  Recruitment to all cohorts was complete and long-term follow-up is ongoing.  Between December 21, 2016, and April 26, 2019, a total of 1,051 patients registered for the study, with ctDNA results available for 1,034 patients.  Agreement between ctDNA digital PCR and targeted sequencing was 96% to 99 % (n = 800, kappa 0.89 to 0.93).  Sensitivity of digital PCR ctDNA testing for mutations identified in tissue sequencing was 93 % (95 % CI: 83 to 98) overall and 98 % (87 to 100) with contemporaneous biopsies.  In all cohorts, combined median follow-up was 14.4 months (IQR 7.0 to 23.7).  Cohorts B and C met or exceeded the target number of responses, with 5 (25 % [95 % CI: 9 to 49]) of 20 patients in cohort B and 4 (22 % [6 to 48]) of 18 patients in cohort C having a response.  Cohorts A and D did not reach the target number of responses, with 6 (8 % [95 % CI: 3 to 17]) of 74 in cohort A and 2 (11 % [1 to 33]) of 19 patients in cohort D having a response.  The most common grade 3 to 4 AEs were raised gamma-glutamyltransferase (13 [16 %] of 80 patients; cohort A); diarrhea (4 [25 %] of 20; cohort B); fatigue (4 [22 %] of 18; cohort C); and rash (5 [26 %] of 19; cohort D); 17 serious adverse reactions occurred in 11 patients, and there was 1 treatment-related death caused by grade 4 dyspnea (in cohort C).  The authors concluded that ctDNA testing offered accurate, rapid genotyping that enabled the selection of mutation-directed therapies for patients with breast cancer, with sufficient clinical validity for adoption into routine clinical practice.  These researchers stated that these findings demonstrated clinically relevant activity of targeted therapies against rare HER2 and AKT1 mutations, confirming these mutations could be targetable for breast cancer treatment.\n",
      "The authors stated that this study had several drawbacks.  Inclusion of relatively heavily pre-treated patients might reduce activity of the targeted drugs, especially in cohort A, and future ctDNA selection trials might benefit from more restrictive entry criteria.  The study was designed to examine the activity of therapies against specific genomic events; however, it did not target PIK3CA mutations, and as a result relatively few of the patients registered to the trial had a response to therapy (17 [1.6 %] of 1,051 patients).  However, mutation-directed therapy with alpelisib is now approved to target PIK3CA mutations, and this study showed the clinical validity of using ctDNA to direct therapy.  Cohort D was designed as a basket cohort from the outset, to examine the activity of capivasertib against different AKT pathway activating mutations.  Only cohort D allowed entry of patients with previous tissue sequencing results, as it was anticipated that ctDNA testing alone might not recruit sufficient patients.  Although these researchers identified low activity of capivasertib in PTEN-mutant cancers when used as a single agent, AKT inhibition in combination with paclitaxel chemotherapy might be effective in PTEN mutant cancers.  Capivasertib plus fulvestrant might be effective in endocrine-resistant estrogen receptor-positive breast cancer without mutation selection, as shown in the FAKTION trial.  It was not possible to robustly compare plasma MATCH with FAKTION, as patients enrolled in plasma MATCH had more previous lines of treatment, and AKT1 mutations were not assessed and would be few in number in FAKTION.\n",
      "In a retrospective, single-center study, Alvarez et al (2020) reported their experience of cfDNA testing at the time of diagnosis and how this intervention could help avoid further invasive interventions, how it could be used to determine initiation of therapy, and how variation allele frequency of the somatic alteration affects response to subsequent treatment.  This trial included patients with advanced NSCLC who had cfDNA from plasma tested using the Guardant360 panel, which identifies somatic genomic alterations by massive parallel sequencing of target genes.  An institutional Clinical Laboratory Improvement Amendments tissue panel using FISH (for MET, RET, ROS1, and ALK) and NGS for selected genes was employed for tissue analysis.  Actionable mutations are those with FDA-approved targeted therapies (EGFR, ALK, ROS, BRAF, NTRK fusions) or therapies soon to be approved (RET fusions and MET amplifications; or MET exon 14 skipping mutation).  A total of 163 blood samples from 143 patients were evaluated, 82 at diagnosis and 81 at disease progression.  A total of 94 cases had tissue and cfDNA testing performed within 12 weeks of each other; 76 (81 %) of 94 cases were concordant, of which 22 cases were concordantly positive and 54 concordantly negative; 18 (19 %) of 94 cases were discordant, of which 11 had negative blood and positive tissue results, and 7 had positive blood and negative tissue results.  cfDNA testing had a sensitivity of 67 % (95 % CI: 51 % to 83 %), specificity of 89 % (95 % CI: 81 % to 97 %), NPV of 83 % (95 % CI: 74 % to 92 %), and PPV of 76 % (95 % CI: 60 % to 91 %); 19 (21 %) of 82 cfDNA samples analyzed at diagnosis had actionable mutations identified (4 EGFR exon 19 deletion, 2 EGFR exon 21 L858R, 2 EGFR L861Q, 1 L861R, 4 EML4-ALK fusion, 2 CD74-ROS1 fusion, 2 MET exon 14 skipping mutation, 2 KIF5B-RET fusion).  Of the 82 patients with cfDNA testing performed at the time of diagnosis, 8 patients (10 %) initiated targeted therapy on the basis of cfDNA results only, with 6 patients experiencing PR, 1 patient CR, and 1 patient SD.  The response rate for patients who initiated targeted therapies on the basis of cfDNA only at diagnosis was 88 %. Variant allele frequency had no impact on response.  The authors concluded that initiation of targeted therapy for advanced NSCLC was feasible based only on identification of actionable mutations by cfDNA testing in 9 % of the cases for which tissue diagnosis could not be obtained.  Actionable targets were identified by cfDNA in 20 % of the samples sent at diagnosis.  A substantial number of patients benefited from cfDNA testing at initial diagnosis because it identified actionable mutations that led to appropriate targeted treatments.  These researchers stated that cfDNA testing results are being incorporated and accepted in many clinical trials for patient enrollment, which also represents an opportunity to expand patient access to innovative treatments when obtaining further tissue for expanded molecular testing is challenging.  Progressive disease remains a challenge, and cfDNA testing results can provide some insight into the mechanism of resistance to tyrosine kinase inhibitors that in some instances tissue re-biopsy molecular testing might not reflect.  It should also be noted that cfDNA variation allele frequency does not predict depth of response to targeted therapies.\n",
      "In a retrospective data review, Dvir et al (2021) presented their real-world data on the use of liquid biopsies in the routine management of NCSLC patients.  These investigators carried out a review of 279 consecutive patients with NSCLC in the community setting, who had liquid biopsies performed between 2014 and 2019 as part of routine clinical management.  Over a period of 5 years, a total of 337 liquid biopsy samples, taken from 279 patients were sent for plasma NGS testing.  The median age at diagnosis was 73 years (range of 36to 93, SD 10.4), 141 (51 %) were men and 138 (49 %) were women.  The majority were White or Caucasian (80 % versus 8 % Black or African American versus 12 % multi-racial or unknown race) and had a history of smoking (79 %).  Excluding synonymous mutations and variants of unknown significance, 254 AAs were detected in 106 patients.  Commonly detected AAs were EGFR (n = 127, 50%), KRAS (n = 61, 24%), BRAF (n = 24, 9.5%), and MET (n = 23, 9%). Tissue NGS detected actionable aberrations in 45 patients, with EGFR (n = 28, 57 %) and KRAS (n = 10, 20 %) being the most common actionable aberrations.  Concordance agreement between plasma and tissue NGS modalities was detected in 39 of 45 (86.7 %) patients and was demonstrated most commonly in EGFR (n = 25) and KRAS (n = 11).  In 44 of 106 (41.5 %) of patients, for whom tissue NGS was not carried out, additional precision treatment was guided by the actionable aberrations detected through liquid biopsy.  The authors concluded that integration of liquid biopsy into the routine management of patients with NSCLC demonstrated actionable aberrations detection in 44 additional patients, which comprised a 42 % increase in actionable aberrations detection rate, when tissue NGS was not performed.  Moreover, these researchers stated that more powered studies are needed to examine if incremental benefit exists between tissue NGS and liquid biopsy; these findings cautiously showed a role for the use of liquid biopsy as part of routine clinical management.\n",
      "The authors stated that this study was limited by its retrospective nature.  The information regarding tissue NGS and mortality was captured via review of the electronic medical records, which may be incomplete.  The liquid biopsy information was obtained from the Guardant360 database.  Detection bias may be introduced if providers chose to order liquid biopsies in selected cases depending on specific characteristics (e.g., insurance coverage and non-smoking status).  Evaluating whether clinicians had access to the genomic NGS data and offered patients informed therapy was not routinely documented, limiting the understanding of clinician decision-making with broad-based genomic-sequencing results.\n",
      "In a prospective study, Palmero et al (2021) examined comprehensive NGS of cfDNA compared with SOC tissue-based testing to identify guideline-recommended alterations in advanced NSCLC (aNSCLC).  Patients with treatment-naive aNSCLC were tested using a well-validated NGS cfDNA panel, and results were compared with SOC tissue testing.  The primary objective was non-inferiority of cfDNA versus tissue analysis for the detection of 2 guideline-recommended biomarkers (EGFR and ALK) and an additional 6 actionable biomarkers.  Secondary analyses included tissue versus cfDNA biomarker discovery, ORR, PFS to targeted therapy, and PPV of cfDNA.  The primary objective was met with cfDNA identifying actionable mutations in 46 patients versus 48 by tissue (p < 0.05).  In total, 0/186 patients were genotyped for all 8 biomarkers with tissue, compared with 90.8% using cfDNA.  Targetable alterations or KRAS were identified in 80.7 % when cfDNA was used first versus 57.1 % when tissue was used first; PPV for cfDNA-detected EGFR was 100.0 % (25/25); ORR and PFS in patients receiving targeted therapy based on tissue or cfDNA were similar to those previously reported.  The authors concluded that this study confirmed a previous report that comprehensive cfDNA testing was non-inferior to SOC tissue testing in detecting aNSCLC-recommended biomarkers.  Furthermore, cfDNA-based 1st-line therapy produced outcomes similar to tissue-based testing, demonstrating the clinical utility of comprehensive cfDNA genotyping as the initial genotyping modality in patients with treatment-naive aNSCLC when tissue was insufficient or when all actionable biomarkers could not be rapidly assessed.\n",
      "The authors stated that this study had several drawbacks.  First, the lack of a standardized tissue-based testing algorithm precluded direct comparison of comprehensive cfDNA versus tissue testing performance.  Physicians used the tissue assays available to them as per their institutional SOC, as this study was designed to specifically address the critical question of what impact the addition of comprehensive cfDNA-based testing might have on real-world patient care and was not intended to be a head-to-head comparison of tissue and cfDNA NGS testing.  In this study, only 2 patients had comprehensive NGS tissue testing.  Second, although multi-center, this study was limited to Spain and may not reflect results in other patient populations or healthcare systems, although several studies with very similar results have been reported in the U.S. and elsewhere in the world.  Third, as this study was powered for a primary endpoint of non-inferiority, only a small number of patients received targeted therapy for any individual biomarker and were available for central RECIST assessment of response.  Fourth, these researchers did not examine the correlation of clinicopathologic features with cfDNA detection rates and blood-tissue concordance.  This topic has been addressed elsewhere and could be included in future studies.  Finally, the findings of this study were only applicable to liquid biopsies that, like the current assay, perform comprehensive genomic profiling as defined by the MolDx program.\n",
      "Cui et al (2022) noted that genomic sequencing is necessary for 1st-line advanced NSCLC (aNSCLC) treatment decision-making.  While tissue NGS is standard, however, tissue quantity, quality, and time-to-results remains problematic.  These researchers compared upfront cfDNA NGS clinical utility against routine tissue testing in patients with aNSCLC.  cfDNA-NGS was carried out in consecutive, newly identified aNSCLC patients between December 2019 to October 2021 alongside routine tissue genotyping.  Variants were interpreted using AMP/ASCO/CAP guidelines.  The primary endpoint was tier-1 variants detected on cfDNA-NGS.  cfDNA-NGS results were compared to tissue results.  Of 311 patients, 282 (91 %) had an informative cfDNA-NGS test; 118 (38 %) patients had a tier-1 variant identified by cfDNA-NGS.  Of 243 patients with paired tissue-cfDNA tests, 122 (50 %) tissue tests were informative; 85 (35 %) tissue tests identified a tier-1 variant.  cfDNA-NGS detected 39 additional tier-1 variants compared to tissue alone, increasing the tier-1 detection rate by 46 % (from 85 to 124).  The sensitivity of cfDNA-NGS relative to tissue was 75 % (25 % tissue tier-1 variants were not detected on cfDNA-NGS); 33 % of cfDNA tier-1 variants were not identified on tissue tests.  Median time from request-to-report was shorter for cfDNA-NGS versus tissue (8 days versus 22 days; p < 0.0001).  A total of 245 (79 %) patients received 1st-line systemic-therapy: 49 (20 %) with cfDNA-NGS results alone.  Median time from sampling-to-commencement of 1st-line treatment was shorter for cfDNA-NGS blood draw versus 1st tissue biopsy (16 days versus 35 days; p < 0.0001).  The authors concluded that given the ability of cfDNA-NGS to rapidly detect clinically relevant genomic variants, a plasma-first, tissue-next (if no relevant variants are detected on plasma) testing approach could improve the speed and accuracy of therapeutic decision-making and should be considered a key strategy to increase adequacy and timeliness of target identification and treatment for all patients with aNSCLC.\n",
      "The authors stated that this trial was carried out at a single U.K. academic cancer center, a drawback of this study.  Patients were referred from a variety of diagnostic services with different tissue molecular ordering methods and timelines to the authorsâ center.  Thus, to minimize unquantifiable bias, the tissue test turn-around time was calculated from the date of biopsy rather than the date of tissue molecular test request as this could not be accurately ascertained.  In addition, in this real-world study, the tissue testing methods were heterogenous, reflecting their standard referral pathways and use of sequential hierarchical single-gene testing at some sites could have biased the comparisons between cfDNA-NGS and tissue testing times.  Nevertheless, these comparisons did reflect real-world testing scenarios and were, therefore, clinically relevant.  Moreover, these findings were based on a healthcare model of government-funded genotype testing, identifying the potential challenges and benefits of implementing cfDNA-NGS in this setting.\n",
      "Garcia-Pardo et al (2023) stated that clinical management of patients with newly diagnosed advanced NSCLC requires molecular testing of genetic alterations to guide 1st-line treatment.  Next generation sequencing in tumor tissue is recommended by clinical guidelines for diagnosis and molecular profiling for advanced non-squamous NSCLC.  However, despite tissue-NGS is becoming more widely used, access to NGS testing is still limited in many parts of the world due to a lack of resources and infrastructure.  Furthermore, tissue NGS in NSCLC can be challenging due to the need of invasive, limited-tissue content tumor biopsies, which are both time- and resource-consuming.  Inadequate or delayed tumor tissue genotyping is one of the major barriers to access to targeted therapy.  Detection of plasma ctDNA is becoming widely adopted as complementary to tissue tumor genotyping in NSCLC.  Plasma-based molecular profiling is less invasive, faster, requires fewer resources, and it is non-inferior to tumor tissue\n",
      "genotyping in NSCLC.  In a retrospective study, these investigators reported their experience using plasma-based NGS in patients with advanced NSCLC.  They analyzed the frequency of oncogenic drivers detected, the proportion of patients treated with genotype-directed therapy, and survival outcomes.  These researchers analyzed the findings of 109 patients with advanced NSCLC who underwent either Foundation One Liquid (n = 91) or Guardant360 (n = 18) as part of molecular pre-screening for clinical trials.  They noted that plasma-based NGS can potentially identify additional driver alterations in patients with NSCLC compared to single-gene testing for EGFR, ALK and ROS1; this can be relevant when there is limited tissue for molecular profiling or if tissue biopsy is not feasible.  The identification of driver alterations in plasma aids NSCLC patients to be treated with matched targeted therapy, which eventually might lead to better outcomes.  The authors stated that further studies examining the cost-effectiveness of complementary liquid biopsy in addition to tissue genotyping are needed, as well as novel strategies such as a tissue-sparing, plasma-first approach.  Moreover, they stated that timely and equitable access to targeted therapies will also be critical to translate research in precision medicine into improved patient care.\n",
      "The authors stated that this study had several drawbacks.  First, this was a retrospective study, and the sample size was relatively small.  Second, not all patients with advanced NSCLC referred the authorsâ center were systematically enrolled in the molecular pre-screening studies, potentially leading to selection bias; this may account, in part, for the high rate of cases without full-valid tissue genotyping.  Third, although not intended, this trial compared plasma-based NGS to single-gene testing for EGFR, ALK, and ROS1 only, due to access constraints to broad molecular testing in Spain; tissue-based NGS is considered the SOC for molecular profiling in NSCLC, and any kind of NGS (either plasma or tissue-based) is preferred to single-gene testing for EGFR, ALK and ROS1.\n",
      "CancerIntercept\n",
      "CancerIntercept (Pathway Genomics) is a liquid biopsy intended for use as a non-invasive screening test designed for early cancer detection and monitoring. Cell-free DNA (cfDNA) in the blood is tested for the presence of circulating tumor DNA (ctDNA) by screening for specific cancer-associated mutations using polymerase chain reaction (PCR) to amplify both the mutant and wild type DNA, followed by a \"specific enrichment of the mutant and simultaneous removal of the wild type DNA by using a proprietary technology,\" after which the \"mutant DNA is sequenced on Illuminaâs next-generation sequencing platform.\" The tests analyze the presence of 96 frequently occurring DNA mutation hot spots in nine cancer driver genes (BRAF, CTNNB1, EGFR, FOXL2, GNAS, KRAS, NRAS, PIK3CA and TP53) that, when mutated, can cause cancer or contribute to cancer progression. These mutations are commonly associated with lung, breast, ovarian, colorectal cancers and melanoma, and may occur less frequently in other cancer types (such as pancreatic, head and neck, thyroid, gastric and prostate cancers). Clinical trials are ongoing to assess the correlation of liquid biopsy results with the actual presence or absence of these mutations in the tumor itself.\n",
      "The test is offered for two general indications: CancerIntercept Detect is a liqud biopsy designed to detect tumor DNA in high-risk individuals; CancerIntercept Monitor is intended to monitor patients with active or previously diagnosed cancer. \n",
      "There is a lack of adequate clinical validation to justify CancerIntercept Detect's recommended use in screening high-risk patients for cancer. There is a lack of clinical trial evidence showing that CancerIntercept Detect results in earlier diagnosis or decreases mortality from cancer.\n",
      "Clinical trials are also examining the prognostic value of various mutations screened for by the CancerIntercept Monitor test in terms of recurrence, survival, and response to treatment.\n",
      "Circulating Cell-Free Nucleic Acids in Colorectal Cancer\n",
      "Toth and colleagues (2016) stated that screening methods for the most frequent diagnosed malignant tumor, CRC, have limitations.  Total circulating cell-free DNA (cfDNA) analysis came into focus as a potential screening test for CRC.  Detection of epigenetic and genetic alterations of cfDNA as DNA methylation or DNA mutations and related ribonucleic acids may improve cancer detection based on unique, CRC-specific patterns.  These investigators summarized the CRC-specific nucleic acid biomarkers measured in peripheral blood and their potential as screening markers.  Detection of DNA mutation has inadequate sensitivity; however, methylated DNA can be established with higher sensitivity from CRC plasma samples.  The ribonucleic acid based miRNA studies represented higher sensitivity for CRC as compared with mRNA studies.  Recently, isolation of cfDNA has become automated, highly reproducible and a high throughput method.  The authors concluded that with automated possible diagnostic tools, a new approach may be available for CRC screening as liquid biopsy.\n",
      "Spindler (2017) noted that circulating DNA can be used to measure cfDNA and for detection and quantification of tumor-specific genetic alterations in the peripheral blood, and the broad clinical potential of circulating DNA has attracted increasing focus over the past decade.  Concentrations of circulating DNA are high in metastatic CRC, and the total levels of cfDNA have been reported to hold strong prognostic value.  Colorectal tumors are characterized by a high frequency of well known, clinically relevant genetic alteration, which is readily detected in the cfDNA and holds potential for tailoring of palliative therapy and for monitoring during treatment.  These investigators reviewed the current literature that has specifically reported data on the potential utility of cfDNA and on tumor-specific mutations in metastatic CRC (mCRC).  Methodological, biological and clinical aspects were discussed based on the most recent development in this specific setting, and eligible studies were identified by systematic literature searched from PubMed and Embase in addition to conference papers and communications.  The literature regarding cfDNA in CRC is broad and heterogeneous concerning aims, nomenclature, methods, cohorts and clinical end-points and consequently difficult to include in a single systematic search.  However, the available data underline a strong clinical value of measuring both total cfDNA levels and tumor-specific mutations in the plasma of patients with mCRC, pre- and during systemic therapy.  The authors concluded that this paper had gathered the most recent literature on several aspects of cfDNA in mCRC, including methodological, biological and clinical aspects, and discussed the large clinical potential in this specific setting, which needs to be validated in carefully designed prospective studies in statistically relevant cohorts.\n",
      "HMGB1 and RAGE in Cutaneous Malignancy\n",
      "Tesarova and associates (2016) noted that activation of the receptor for advanced glycation end-products (RAGE) due to its increased expression in cancer cells or its stimulation by multiple ligands (AGEs, high-mobility group box-1 [HMGB1], S100 proteins, etc.) may contribute to the proliferation, invasiveness of tumor cells and formation of distant metastases and also to the resistance of cancer to treatment.  RAGE ligands could thus become both useful markers of disease severity and its outcome and, a potential therapeutic target.  The authors concluded that better understanding of the role of RAGE activation in different types of cancer may help to define the role of ligand/RAGE antagonists as promising cancer treatment.\n",
      "Nguyen and colleagues (2017) stated that inflammation and the immune system play a role in the development and progression of melanoma, basal cell carcinoma (BCC), and SCC.  The pro-inflammatory and tumor-promoting effects of HMGB1 protein and RAGE have been investigated in these cutaneous malignancies.  The clinical implication of these molecules is not fully described.  The National Library of Medicine database was searched for articles addressing the clinical relevance of HMGB1 and RAGE in melanoma, BCC, and SCC.  This systematic review included 9 articles, with 6 summarizing RAGE in cutaneous malignancies and 3 involving HMGB1.  RAGE has been found to be up-regulated in SCC lesions, as well as melanoma.  Levels of RAGE were highest in stage IV melanomas.  Lower levels of soluble RAGE have been associated with poor OS in melanoma.  Sporadic extracellular expression of HMGB1 was evident in BCC and SCC lesions, which could be released by necrotic tumor cells.  HMGB1 was found to be a prognostic marker in melanoma, and HMGB1 levels were elevated in patients who were non-responders to ipilimumab treatment.  The authors concluded that HMGB1 and RAGE could serve as potential prognostic markers or therapeutic targets in treating melanoma, BCC, and SCC; however, further research regarding the clinical utility of the HMGB1-RAGE axis in cutaneous malignancies is needed.\n",
      "Ki-67 in Upper Tract Urinary Carcinoma\n",
      "Ley and associates (2015) noted that upper urinary-tract urothelial carcinomas (UTUC) constitute 5 % of urothelial malignancies.  Prognostic biomarkers would allow lower risk surgical approaches for less aggressive UTUCs.  One biomarker, Ki-67/mindbomb E3 ubiquitin protein ligase 1 (Ki-67/MIB-1), showed promise in UTUC, but there have been conflicting findings regarding its prognostic role.  This systematic review and meta-analysis examined the prognostic value of Ki-67/MIB-1 in UTUC in terms of UTUC-specific mortality rate, 5-year DFS, and 5-year OS (including DSS).  A systematic review of the current literature produced 654 records.  A total of 13 studies consisting of 1,030 patients were finally included in the meta-analysis; HRs with 95 % CI were extracted or estimated.  The individual HR estimates were combined into a pooled HR using a fixed-effects model that summed homogeneity of the individual true HRs.  Patients with Ki-67/MIB-1 over-expression displayed significantly higher UTUC-specific mortality rate (pooled HR: 2.14, 95 % CI: 1.73 to 2.64; p < 0.00001), significantly reduced 5-year DFS (pooled HR: 2.27, 95 % CI: 1.79 to 2.92; p < 0.00001), and significantly reduced 5-year OS (pooled HR = 1.77; 95 % CI: 1.39 to 2.23 p < 0.00001).  There was significant heterogeneity detected in the UTUC-specific mortality rate meta-analysis (I(2) = 63 %) and the 5-year DFS meta-analysis (I(2) = 65 %), but there was no significant heterogeneity detected in the 5-year OS meta-analysis (I(2) = 0%).  Egger's testing showed that none of the outcomes was influenced by publication bias (p > 0.05).  The authors concluded that Ki-67/MIB-1 over-expression showed promise as a prognostic biomarker for UTUC patients and required further investigation.\n",
      "Fan and colleagues (2016) stated that UTUC is a relatively uncommon but aggressive disease.  The Ki-67 antigen is a classic marker of cellular proliferation, but there is still controversy regarding the significance and importance of Ki-67 in tumor progression.  In this study, these researchers first detected Ki-67 expression in UTUC patients by immunohistochemistry.  Subsequently, they quantitatively combined the results with those from the published literature in a meta-analysis after searching several databases.  Immunohistochemistry results demonstrated that patients with muscle-invasive tumors (T2-T4) had higher Ki-67 expression than those with non-muscle-invasive tumors (Tis-T1), suggesting that high Ki-67 expression may be associated with the aggressive form of UTUC.  Kaplan-Meier curves showed that patients with high Ki-67 expression had significantly poorer cancer-specific survival (CSS) and DFS.  Furthermore, multi-variate analysis suggested that Ki-67 expression was an independent prognostic factor for CSS (HR = 3.196) and DFS (HR = 3.517) in UTUC patients.  Then, a meta-analysis of the published literature investigating Ki-67 expression and its effects on UTUC prognosis was conducted.  After searching the PubMed, Medline, Embase, Cochrane Library and Scopus databases, a total of 12 articles met the eligibility criteria for this analysis.  The eligible studies included a total of 1,740 patients with a mean number of 82 patients per study (range of 38 to 475).  The combined results showed that increased Ki-67 levels were associated with poor survival and disease progression, with a pooled HR estimate of 2.081 and 2.791, respectively.  In subgroup analysis, the pooled HR was statistically significant for CSS (HR = 2.276), metastasis-free survival (HR = 3.008) and DFS (HR = 6.336).  The authors concluded that high Ki-67 expression was associated with poor survival in patients with UTUC, as well as a high risk of disease progression, although these findings need to be interpreted with caution.  They stated that large-scale, adequately designed, prospective trials are needed to further confirm the value of Ki-67 in prognosis of UTUC patients.\n",
      "Long Non-Coding RNA in Gallbladder Cancer and Non-Small Cell Lung Cancer\n",
      "Ricciuti and associates (2016) stated that recent advances in tiling array and high throughput analyses revealed that at least 87.3 % of the human genome is actively transcribed, though less than 3 % of the human genome encodes proteins.  This unexpected truth suggests that most of the transcriptome is constituted by non-coding RNA.  Among them, high-resolution microarray and massively parallel sequencing analyses identified long non-coding RNAs (lncRNAs) as non-protein-coding transcripts.  lncRNAs are largely poly-adenylated and greater than 200 nucleotides in length transcripts, involved in gene expression through epigenetic and transcriptional regulation, splicing, imprinting and subcellular transport.  Although lncRNAs functions are largely uncharacterized, accumulating data indicate that they are involved in fundamental biological functions.  Conversely, their dysregulation has increasingly been recognized to contribute to the development and progression of several human malignancies, especially lung cancer, which represents the leading cause of cancer-related deaths worldwide.  These researchers conducted a comprehensive review of the published literature focusing on lncRNAs function and disruption in non-small cell lung cancer (NSCLC) biology, also highlighting their value as biomarkers and potential therapeutic targets.  lncRNAs are involved in NSCLC pathogenesis, modulating fundamental cellular processes such as proliferation, cell growth, apoptosis, migration, stem cell maintenance and epithelial to mesenchymal transition, also serving as signaling transducers, molecular decoys and scaffolds.  Furthermore, lncRNAs represent very promising biomarkers in early-stage NSCLC patients and may become particularly useful in non-invasive screening protocols.  lncRNAs may be used as predictive biomarkers for chemotherapy and targeted therapies sensitivity.  In addition, selectively targeting oncogenic lncRNAs could provide a new therapeutic tool in treating NSCLC patients. The authors concluded that lncRNAs disruption plays a pivotal role in NSCLC development and progression.  They stated that these molecules also serve as diagnostic, prognostic and predictive biomarkers; characterization of lncRNA genes and their mechanisms of action will foster development of a more comprehensive clinical approach, with the final goal to benefit patients.\n",
      "Xu and colleagues (2016) noted that lung cancer ranks as the first most common cancer and the first leading cause of cancer-related death in China and worldwide.  Due to the difficulty in early diagnosis and the onset of cancer metastasis, the 5-year survival rate of lung cancer remains extremely low.  Long noncoding RNAs, which lacking protein-coding ability, have recently emerged as pivotal participants in biological processes, often dysregulated in a range of cancers, including lung cancer.  These investigators highlighted the recent findings of lncRNAs in lung cancer pathogenesis.  The authors concluded that while the understanding of lncRNAs in the onset and progression of lung cancer is still in its infancy, there is no doubt that understanding the activities of lncRNAs will certainly secure strong biomarkers and improve treatment options for lung cancer patients.\n",
      "Furthermore, National Comprehensive Cancer Networkâs clinical practice guidelines on \"Non-small cell lung cancer\" (Version 4.2017) does not mention detection of long non-coding RNAs as a management tool.\n",
      "Khandelwal and colleagues (2017) noted that gallbladder cancer (GBC) is the most common and aggressive form of biliary tract carcinoma with an alarmingly low 5-year survival rate.  Despite its high mortality rate, the underlying mechanisms of GBC pathogenesis are not completely understood.  Recently, from a growing volume of literature, long non-coding RNAs (lncRNAs) have emerged as key regulators of gene expression and appear to play vital roles in many human cancers.  To-date, a number of lncRNAs have been implicated in GBC, but their potential roles in GBC have not been systematically examined.  These investigators discussed the emerging roles of lncRNAs in GBC, and the pathways involved.  Specifically, they noted that some lncRNAs show greater expression in T1 and T2 tumor stages compared to T3 and T4 tumor stages and that their dysregulation leads to alterations in cell cycle progression and can cause an increase in GBC cell proliferation or apoptosis.  Furthermore, some lncRNAs control the epithelial-mesenchymal transition process, while others take part in the regulation of ERK/MAPK and Ras cancer-associated signaling pathways.  These researchers also presented their potential utility in diagnosis, prognosis, and/or treatment of GBC.  The authors concluded that the overall goal of this review was to stimulate interest in the role of lncRNAs in GBC, which may open new avenues in the determination of GBC pathogenesis and may lead to the development of new preventive and therapeutic strategies for GBC.\n",
      "MUC1 in Gastric Cancer\n",
      "Wang and colleagues (2016) stated that MUC1, a member of the mucin family, is expressed in tumors of various human organs and may function as an anti-adhesion molecule that inhibits cell-to-cell adhesion, inducing tumor metastasis, and served as a potential biomarker of tumor progression in early gastric cancer. However, its prognostic significance in gastric cancer is still in dispute.  These researchers performed a meta-analysis to evaluate the relationship between MUC1 expression and prognosis of gastric cancer.  A total of 10 eligible studies with 834 cases and 548 controls were included.  MUC1 positive cases were highly positive in intestinal-type carcinomas (OR = 1.76, 95 % CI: 1.27 to 2.44, p = 0.0008 fixed-effect), higher rate of vascular invasion (OR = 1.64, 95 % CI: 1.13 to 2.39, p = 0.009 fixed-effect), and lymph node metastasis (OR = 2.10, 95 % CI: 1.20 to 3.67, p = 0.01 random-effect), as well as lower 5-year survival rate (HR = 0.27, 95 % CI: 0.11 to 0.66, p = 0.004 random-effect).  However, the presence of MUC1 was not associated with gender, tumor size, histologic differentiation, and clinical stage.  The authors concluded that MUC1 is a prognostic factor in gastric cancer, which acts as a marker of poor outcome in patients with gastric cancer; further clinical studies are needed to confirm the role of MUC1 in clinical practice.\n",
      "Furthermore, National Comprehensive Cancer Networkâs clinical practice guidelines on \"Gastric cancer\" (Version 3.2016) did not mention the use of MUC1 as a biomarker.\n",
      "Percepta Bronchial Genomic Classifier (Veracyte)\n",
      "Silvestri et al (2015) stated that bronchoscopy is frequently non-diagnostic in patients with pulmonary lesions suspected to be lung cancer.  This often results in additional invasive testing, although many lesions are benign.  These researchers sought to validate a bronchial-airway gene-expression classifier that could improve the diagnostic performance of bronchoscopy.  Current or former smokers undergoing bronchoscopy for suspected lung cancer were enrolled at 28 centers in 2 multi-center prospective studies (AEGIS-1 and AEGIS-2).  A gene-expression classifier was measured in epithelial cells collected from the normal-appearing main-stem bronchus to assess the probability of lung cancer.  A total of 639 patients in AEGIS-1 (298 patients) and AEGIS-2 (341 patients) met the criteria for inclusion.  A total of 43 % of bronchoscopic examinations were non-diagnostic for lung cancer, and invasive procedures were performed after bronchoscopy in 35 % of patients with benign lesions.  In AEGIS-1, the classifier had an area under the receiver-operating-characteristic curve (AUC) of 0.78 (95 % confidence interval [CI]: 0.73 to 0.83), a sensitivity of 88 % (95 % CI: 83 to 92), and a specificity of 47 % (95 % CI: 37 to 58).  In AEGIS-2, the classifier had an AUC of 0.74 (95 % CI: 0.68 to 0.80), a sensitivity of 89 % (95 % CI: 84 to 92), and a specificity of 47 % (95 % CI: 36 to 59).  The combination of the classifier plus bronchoscopy had a sensitivity of 96 % (95 % CI: 93 to 98) in AEGIS-1 and 98 % (95 % CI: 96 to 99) in AEGIS-2, independent of lesion size and location.  In 101 patients with an intermediate pre-test probability of cancer, the negative predictive value of the classifier was 91 % (95 % CI: 75 to 98) among patients with a non-diagnostic bronchoscopic examination.  The authors concluded that the gene-expression classifier improved the diagnostic performance of bronchoscopy for the detection of lung cancer.  In intermediate-risk patients with a non-diagnostic bronchoscopic examination, a negative classifier score provided support for a more conservative diagnostic approach.\n",
      "The authors noted that there are several important limitations to this study:\n",
      "\n",
      "specimens from 155 patients (11 %) yielded insufficient or poor-quality RNA, precluding measurement of the classifier.  However, similar rates of insufficient RNA quality or quantity have been observed with other gene expression tests that have been integrated into clinical practice, and it may be possible to improve sample quality by decreasing the time between sample collection and RNA isolation.  Patients who were not included in the study for this reason do not appear to differ in terms of cancer prevalence or other clinical features in comparison with the overall study population; however, it cannot be determined whether the classifier has similar performance in this group;\n",
      "9 % of patients were lost to follow-up, and 5 % did not have a definitive diagnosis established at 12 months after bronchoscopic examination.  This rate of loss to follow-up is not unexpected in an observational trial in which the subsequent evaluation after bronchoscopic examination was not mandated to occur at the study center.  Although the follow-up period was limited to 12 months, it is unlikely that they missed a substantial number of cancers that would have been found with an additional year of follow-up.  Although guidelines suggest 24 months of surveillance, these recommendations are based on older studies regarding solitary pulmonary nodules discovered on chest radiography (not computed tomography [CT]).  The high sensitivity of CT makes it unlikely that solid nodules that are stable in the first year will have subsequent growth; this is supported by studies of lung cancer screening in which nodules that were stable for 1 year had a conversion rate to cancer of only 1 per 1,000 during year 2;\n",
      "the exclusion criteria in this study limit the generalizability of these findings among life life-time non-smokers and smokers with a history of lung cancer.  It is unclear whether a similar field of injury exists in people who have never smoked or in very light smokers who have lung cancer and whether the field of injury persists after tumor resection; further studies are needed to evaluate these questions;\n",
      "these investigators considered bronchoscopy to be \"diagnostic\" only when the procedure yielded a lung-cancer diagnosis.  There were 49 bronchoscopic examinations in which a specific benign cause was identified, but 31 of the patients received further invasive testing, including 4 patients who ultimately had lung cancer diagnosed on subsequent lung biopsy; this suggests that the concern for cancer remained elevated despite the initial benign finding on bronchoscopic examination; and\n",
      "these researchers did not assess the accuracy of a model incorporating the classifier in combination with clinical variables.  Although risk-prediction models have been developed for solitary pulmonary nodules, there are no validated models for patients undergoing diagnostic bronchoscopic examination, which includes patients with a broad range of findings, including larger lesions (i.e., greater than 3 cm), infiltrates, or lymphadenopathy.\n",
      "\n",
      "Thus, most patients are selected for bronchoscopy on the basis of the physicianâs qualitative assessment of the probability of lung cancer.  The authors showed that their classifier performed well in patients with an intermediate probability of cancer as assessed by a physician in a process that incorporated the available clinical risk factors.\n",
      "Whitney et al (2015) stated that the gene expression profile of cytologically-normal bronchial airway epithelial cells has previously been shown to be altered in patients with lung cancer.  Although bronchoscopy is often used for the diagnosis of lung cancer, its sensitivity is imperfect, especially for small and peripheral suspicious lesions.  In this study, these researchers derived a gene expression classifier from bronchoscopically-obtained airway epithelial cells that detects the presence of cancer in current and former smokers undergoing bronchoscopy for suspect lung cancer and evaluated its sensitivity to detect lung cancer among patients from an independent cohort.  They collected bronchial epithelial cells (BEC) from the main-stem bronchus of 299 current or former smokers (223 cancer-positive and 76 cancer-free subjects) undergoing bronchoscopy for suspected lung cancer in a prospective, multi-center study.  RNA from these samples was run on gene expression microarrays for training a gene-expression classifier.  A logistic regression model was built to predict cancer status, and the finalized classifier was validated in an independent cohort from a previous study.  These researchers found 232 genes whose expression levels in the bronchial airway were associated with lung cancer.  They then built a classifier based on the combination of 17 cancer genes, gene expression predictors of smoking status, smoking history, and gender, plus patient age.  This classifier had a ROC curve AUC of 0.78 (95 % CI: 0.70 to 0.86) in patients whose bronchoscopy did not lead to a diagnosis of lung cancer (nâ=â134).  In the validation cohort, the classifier had a similar AUC of 0.81 (95 % CI: 0.73 to 0.88) in this same subgroup (nâ=â118).  The classifier performed similarly across a range of mass sizes, cancer histologies and stages.  The negative predictive value was 94 % (95 % CI: 83 to 99 %) in subjects without bronchoscopy-detected lung cancer.  The authors concluded that they developed a gene expression classifier measured in bronchial airway epithelial cells that is able to accurately identify lung cancer in current and former smokers who have undergone bronchoscopy for suspicion of lung cancer.  They stated that due to the high NPV of the classifier, it could potentially inform clinical decisions regarding the need for further invasive testing in patients whose bronchoscopy is non-diagnostic.\n",
      "Ferguson et al (2016) noted that bronchoscopy is frequently used for the evaluation of suspicious pulmonary lesions found on computed tomography, but its sensitivity for detecting lung cancer is limited.  Recently, a bronchial genomic classifier was validated to improve the sensitivity of bronchoscopy for lung cancer detection, demonstrating a high sensitivity and negative predictive value (NPV) among patients at intermediate risk (10 to 60 %) for lung cancer with an inconclusive bronchoscopy.  These researchers examined if a negative genomic classifier result that down-classifies a patient from intermediate risk to low risk (less than 10 %) for lung cancer would reduce the rate that physicians recommend more invasive testing among patients with an inconclusive bronchoscopy.  These researchers conducted a randomized, prospective, decision impact survey study assessing pulmonologist recommendations in patients undergoing work-up for lung cancer who had an inconclusive bronchoscopy.  Cases with an intermediate pretest risk for lung cancer were selected from the AEGIS trials and presented in a randomized fashion to pulmonologists either with or without the patient's bronchial genomic classifier result to determine how the classifier results impacted physician decisions.  A total of 202 physicians provided 1,523 case evaluations on 36 patients.  Invasive procedure recommendations were reduced from 57 % without the classifier result to 18 % with a negative (low risk) classifier result (pâ<â0.001).  Invasive procedure recommendations increased from 50 to 65 % with a positive (intermediate risk) classifier result (pâ<â0.001).  When stratifying by ultimate disease diagnosis, there was an overall reduction in invasive procedure recommendations in patients with benign disease when classifier results were reported (54 to 41 %, pâ<â0.001).  For patients ultimately diagnosed with malignant disease, there was an overall increase in invasive procedure recommendations when the classifier results were reported (50 to 64 %, pâ=â0.003).  The authors concluded that these findings suggested that a negative (low risk) bronchial genomic classifier result reduces invasive procedure recommendations following an inconclusive bronchoscopy and that the classifier overall reduced invasive procedure recommendations among patients ultimately diagnosed with benign disease.  They stated that these results support the potential clinical utility of the classifier to improve management of patients undergoing bronchoscopy for suspect lung cancer by reducing additional invasive procedures in the setting of benign disease.\n",
      "Vachani et al (2016) stated that bronchoscopy is often the initial diagnostic procedure performed in patients with pulmonary lesions suggestive of lung cancer.  A bronchial genomic classifier was previously validated to identify patients at low risk for lung cancer after an inconclusive bronchoscopy.  In this study, these investigators evaluated the potential of the classifier to reduce invasive procedure utilization in patients with suspected lung cancer.  In 2 multi-center trials of patients undergoing bronchoscopy for suspected lung cancer, the classifier was measured in normal-appearing bronchial epithelial cells from a main-stem bronchus.  Among patients with low and intermediate pretest probability of cancer (n = 222), subsequent invasive procedures after an inconclusive bronchoscopy were identified.  Estimates of the ability of the classifier to reduce unnecessary procedures were calculated.  Of the 222 patients, 188 (85 %) had an inconclusive bronchoscopy and follow-up procedure data available for analysis; 77 (41 %) patients underwent an additional 99 invasive procedures, which included surgical lung biopsy in 40 (52 %) patients.  Benign and malignant diseases were ultimately diagnosed in 62 (81 %) and 15 (19 %) patients, respectively.  Among those undergoing surgical biopsy, 20 (50 %) were performed in patients with benign disease.  If the classifier had been used to guide decision making, procedures could have been avoided in 50 % (21 of 42) of patients undergoing further invasive testing.  Furthermore, among 35 patients with an inconclusive index bronchoscopy who were diagnosed with lung cancer, the sensitivity of the classifier was 89 %, with 4 (11 %) patients having a false-negative classifier result.  The authors concluded that invasive procedures after an inconclusive bronchoscopy occur frequently, and most are performed in patients ultimately diagnosed with benign disease.  They stated that using the genomic classifier as an adjunct to bronchoscopy may reduce the frequency and associated morbidity of these invasive procedures.\n",
      "UpToDate reviews on \"Overview of the initial evaluation, diagnosis, and staging of patients with suspected lung cancer\" (Thomas and Gould, 2017a), \"Selection of modality for diagnosis and staging of patients with suspected non-small cell lung cancer\" (Thomas and Gould, 2017b), and \"Overview of the initial evaluation, treatment and prognosis of lung cancer\" (Midthun, 2017) do not mention the use of genomic testing/classifier.\n",
      "Also, an UpToDate review on \"\"Procedures for tissue biopsy in patients with suspected non-small cell lung cancer\" (Thomas and Gould, 2017c) states that \"Although obtaining samples of lavage fluid or tissue for genomic analysis has been studied as a potential diagnostic tool designed to enhance the sensitivity of bronchoscopy for the diagnosis of lung cancer, further study is required before it can be recommended for routine use\".\n",
      "Furthermore, National Comprehensive Cancer Networkâs clinical practice guidelines on \"Non-small cell lung cancer\" (Version 4.2017) and \"Small cell lung cancer\" (Version 2.2017) do not mention the use of genomic testing/classifier.\n",
      "SelectMDx\n",
      "SelectMDx is a reverse transcription PCR (RT-PCR) assay performed on post-DRE, first-void urine specimens from patients with clinical risk factors for PCa, who are being considered for biopsy.  The test measures the mRNA levels of the DLX1 and HOXC6 biomarkers, using KLK3 expression as internal reference, to aid in patient selection for prostate biopsy.  Higher expression levels of DLX1 and HOXC6 mRNA are associated with an increased probability for high-grade (Gleason score (GS) greater than or equal to 7) prostate cancer.  SelectMDx provides the likelihood of detecting PCa upon biopsy, and the probability for high-grade versus low-grade disease, with an AUC of 0.89 (95 % CI: 0.86 to 0.92). \n",
      "Carlsson and Roobol (2017) provided an overview of the current state of the evidence and highlight recent advances in the evaluation and diagnosis of clinically significant PCa, focusing on biomarkers, risk calculators and multi-parametric MRI (mpMRI).  In 2017 there are numerous options to improve early detection as compared to a purely PSA-based approach.  All have strengths and drawbacks.  In addition to repeating the PSA and performing clinical work-up (DRE and estimation of prostate volume), additional tests investigated in the initial biopsy setting are: % free PSA, PHI, 4Kscore, SelectMDx, and Michigan Prostate Score and in the repeat setting: % free PSA, PHI, 4Kscore, Prostate Cancer Antigen 3, and ConfirmMDx.  Risk calculators are available for both biopsy settings and incorporate clinical data with, or without, biomarkers; mpMRI is an important diagnostic adjunct.  The authors concluded that there are numerous tests available that can help increase the specificity of PSA, in the initial and repeat biopsy setting; they all coincide with a small decrease in sensitivity of detecting high-grade cancer.  They noted that cost-effectiveness is crucial; and the way forward is a multi-variable risk assessment on the basis of readily available clinical data, potentially with the addition of PSA sub-forms, preferably at low cost; MRI in the pre-diagnostic setting is promising, but is not ready for \"prime time\".\n",
      "Hendriks and colleagues (2017) noted that the diagnosis of PCa is currently based on serum PSA testing and/or abnormal DRE and histopathologic evaluation of prostate biopsies.  The main drawback of PSA testing is the lack of specificity for PCa.  To improve early detection of PCa more specific biomarkers are needed.  In the past few years, many new promising biomarkers have been identified; however, to-date, only a few have reached clinical practice.  These researchers discussed new blood-based and urinary biomarker models that are promising for usage in PCa detection, follow-up and treatment decision-making.  These include PHI, PCA3, 4-kallikrein panel (4K), transmembrane protease serine 2-ERG (TMPRSS2-ERG), ExoDx Prostate Intelliscore, SelectMDx and the Mi-Prostate score.  Only few head-to-head studies are available for these new fluid-based biomarkers and/or models.  The blood-based PHI and urinary PCA3 are 2 FDA-approved biomarkers for diagnosis of PCa.  These investigators also provided an overview of published studies comparing these 2 available biomarkers for prediction of (i) PCa upon prostate biopsy, (ii) pathological features in radical prostatectomy specimen, and (iii) significant PCa in patients eligible for active surveillance.  Studies showed opposing results in comparison of PHI with PCA3 for prediction of PCa upon initial and repeat prostate biopsy; PHI and PCA3 are able to predict pathological findings on radical prostatectomy specimen, such as tumor volume and Gleason score.  Only PHI could predict seminal vesicle invasion and only PCA3 could predict multi-focality.  There is some evidence that PHI outperformed PCA3 in predicting significant PCa in an active surveillance population.  The authors concluded that future research should focus on independent validation of promising fluid-based biomarkers/models, and prospective comparison of biomarkers with each other.\n",
      "Dijkstra and associates (2017) examined the cost-effectiveness of a new urinary biomarker-based risk score (SelectMDx; MDxHealth, Inc., Irvine, CA) to identify patients for TRUS-guided biopsy and to compare this with the current standard of care (SOC), using only PSA to select for TRUS-guided biopsy.  A decision-tree and Markov model were developed to evaluate the cost-effectiveness of SelectMDx as a reflex test versus SOC in men with a PSA level of greater than 3 ng/ml.  Transition probabilities, utilities and costs were derived from the literature and expert opinion.  Cost-effectiveness was expressed in QALYs and healthcare costs of both diagnostic strategies, simulating the course of patients over a time horizon representing 18 years.  Deterministic sensitivity analyses were performed to address uncertainty in assumptions.  A diagnostic strategy including SelectMDx with a cut-off chosen at a sensitivity of 95.7 % for high-grade PCa resulted in savings of â¬128 and a gain of 0.025 QALY per patient compared to the SOC strategy.  The sensitivity analyses showed that the disutility assigned to active surveillance had a high impact on the QALYs gained and the disutility attributed to TRUS-guided biopsy only slightly influenced the outcome of the model.  The authors concluded that based on the currently available evidence, the reduction of over-diagnosis and over-treatment due to the use of the SelectMDx test in men with PSA levels of greater than 3 ng/ml may lead to a reduction in total costs per patient and a gain in QALYs.\n",
      "Sari Motlagh et al (2022) compared SelectMDx and multiparametric MRI (mpMRI) as a diagnostic test in detecting clinical prostate cancer (PCa) and high grade(HG)-PCa in men suspected to have PCa. The authors conducted a systematic search using major web databases for studies that compared sensitivity, specificity, positive predictive value (PPV), and negative predictive value (NPV) of SelectMDx and/or mpMRI that were published before September 30, 2021. The bivariate random model that plotted sensitivity, specificity, PPV, NPV, and likelihood ratio (LR) for PCa and HG-PCa detection was applied to compare SelectMDx, mpMRI, and combination strategies (both positive and one or both positive). The authors found 7 studies comprising of 1328 patients who had undergone SelectMDx and mpMRI to detect PCa. Regarding PCa detection, SelectMDx had a pooled sensitivity of 81%, specificity of 69.8%, PPV of 64.7%, NPV of 85%, and LRs of +2.68 to -0.27, while mpMRI had a pooled sensitivity of 80.8%, specificity of 73.4%, PPV of 72.4%, NPV of 83.5%, and LRs of +3.03 to -0.26. The one or both positive strategy had the highest sensitivity (96.3%), NPV (95.7%), and the lowest -LR (0.06). While the both positive strategy had the highest specificity (80.9%), the PPV (76.5%) and +LR (3.68). In the scenario of PI-RADS 3 lesions not being biopsied in case of a negative SelectMDx (n = 44), unnecessary biopsies would be reduced by 42% (44/105) while the risk of missing HG-PCa would be 9% (4/44). The authors concluded that the performance of SelectMDx is comparable to that of mpMRI with regards to PCa and HG-PCa detection. In addition, this biomarker could help refine the clinical decision-making regarding the necessity of a biopsy in patients suspected to have PCa.\n",
      "An UpToDate review on \"Prostate biopsy\" (Benway and Andriole, 2021) does not mention SelectMDx.\n",
      "Furthermore, NCCNâs clinical practice guideline on \"Prostate cancer\" (Version 1.2023) does not mention SelectMDx as a management tool.\n",
      "ExoDx Prostate (IntelliScore)\n",
      "ExoDx Prostate (IntelliScore) is a non-invasive urine-based liquid biopsy for PCa.  It is used to identify high-grade prostate cancer (HGPCA) both at the time of biopsy and at surgery.  ExoDx Prostate is an exosomal RNA (exoRNA)-based assay that can be used prior to initial biopsy as well as for sequential monitoring of disease progression in patients enrolled in active surveillance.\n",
      "Di Meo and associates (2017) noted that there is a growing trend towards exploring the use of a minimally invasive \"liquid biopsy\" to identify biomarkers in a number of cancers, including urologic malignancies.  Multiple aspects can be assessed in circulating cell-free DNA, including cell-free DNA levels, integrity, methylation and mutations.  Other prospective liquid biopsy markers include circulating tumor cells, circulating RNAs (microRNA [miRNA], long non-coding RNAs [lncRNAs] and messenger RNA [mRNA]), cell-free proteins, peptides and exosomes have also emerged as non-invasive cancer biomarkers.  These circulating molecules can be detected in various biological fluids, including blood, urine, saliva and seminal plasma.  Liquid biopsies hold great promise for personalized medicine due to their ability to provide multiple non-invasive global snapshots of the primary and metastatic tumors.  The authors noted that although a promising source of cancer biomarkers, few exosomal biomarkers have been implemented into clinical practice.  This is partly due to the lack of accurate isolation and detection methods.  They speculate that the development of sensitive capture platforms is likely to trigger the introduction of novel exosomal biomarkers into the clinic in the near future.\n",
      "Panigrahi and Deep (2017) noted that African American men in the US have higher incidence and mortality rates due to PCa compared to other races.  In 2016 alone, nearly 30,000 cases of PCa in African American men were diagnosed and 4,450 men died from PCa.  The underlying reasons for this health disparity in PCa are complex and include social, economic, and biologic determinants.  To reduce or eliminate this health disparity, one must better understand the biology of the disease in African Americans and then develop novel diagnostic and prognostic biomarkers useful for timely and effective treatment decisions.  Recently, there has been remarkable progress in understanding the role of exosomes (vesicles of 30 to 150 nm diameter) in cancer development and progression.  Exosomes are loaded with unique cargo, including proteins, nucleic acids, lipids, and metabolites, that could predict the cells of their origin.  Thus, circulating exosomes in cancer patients are being used as a type of biopsy to identify novel biomarkers for early diagnosis, prognosis, and therapeutics.  The authors discussed the promising use of exosomes to identify race-related unique biological features of PCa, and discover novel biomarkers for better diagnosis and prognosis of PCa, with the goal of reducing cancer health disparities.\n",
      "Foj and colleagues (2017) stated that miRNAs are non-coding small RNAs, involved in post-transcriptional regulation of many target genes.  In this study, 5 miRNAs that have been consistently found deregulated in PCa (miR-21, miR-141, miR-214, miR-375, and let-7c) were analyzed in urinary pellets from 60 PCa patients and 10 healthy subjects by qRT-PCR.  Besides, urinary exosomes were isolated by differential centrifugation and analyzed for those miRNAs.  Significant up-regulation of miR-21, miR-141, and miR-375 was found comparing PCa patients with healthy subjects in urinary pellets, while miR-214 was found significantly down-regulated.  Regarding urinary exosomes, miR-21 and miR-375 were also significantly up-regulated in PCa but no differences were found for miR-141.  Significant differences were found for let-7c in PCa in urinary exosomes while no differences were observed in urinary pellets.  A panel combining miR-21 and miR-375 is suggested as the best combination to distinguish PCa patients and healthy subjects, with an AUC of 0.872.  Furthermore, the association of miRNAs with clinicopathological characteristics was investigated.  MiR-141 resulted significantly correlated with Gleason score in urinary pellets and let-7c with clinical stage in urinary exosomes.  Additionally, miR-21, miR-141, and miR-214 were found significantly deregulated in intermediate/high-risk PCa versus low-risk/healthy subjects in urinary pellets.  Significant differences between both groups were found in urinary exosomes for miR-21, miR-375, and let-7c.  The authors concluded that these findings suggested that the analysis of miRNAs-especially miRNA-21 and miR-375- in urine could be useful as biomarkers in PCa.\n",
      "Yang and co-workers (2017) stated that exosomes are membrane-bound extracellular vesicles involved in intercellular communication and tumor cell metastasis.  In this study, flow field-flow fractionation (FlFFF) was utilized to separate urinary exosomes by size, demonstrating a significant difference in exosome sizes between healthy controls and patients with PCa.  Exosome fractions of different sizes were collected for microscopic analysis during an FlFFF run and evaluated with exosome marker proteins using Western blot analysis.  The results indicated that exosomes of different sizes originated from different types of cells.  Collected exosome fractions were further examined using nanoflow ultrahigh performance liquid chromatography-electrospray ionization-tandem mass spectrometry (nUPLC-ESI-MS/MS) for lipidomic analysis.  A total of 162 lipids (from 286 identified) were quantified using a selected reaction monitoring (SRM) method.  The overall amount of lipids increased by 1.5- to 2-fold in patients with PCa and degree of increase was more significant in the smaller fractions (diameter less than 150 nm) than in the larger ones (diameter greater than 150 nm) some classes of lipids.  In addition, neutral lipids like diacylglycerol (DAG) and triacylglycerol (TAG) decreased in all exosomes without size dependency.  Moreover, a dramatic increase in 22:6/22:6-phosphatidylglycerol (PG) was observed and significant decrease in (16:0, 16:0)- and (16:1, 18:1)-DAG species (nearly 5-fold) and high abundant TAG species (greater than 2.5-fold) was observed in patients with PCa.  The authors concluded that the findings of this study indicated that FlFFF can be employed for the high-speed screening of urinary exosome sizes in patients with PCa and lipidomic analysis of the fractionated exosomes has potential for developing and distinguishing biomarkers of PCa.\n",
      "Pan and colleagues (2017) stated that exosomes are small vesicular bodies released by a variety of cells.  Exosomes contain miRNAs, mRNAs and proteins with the potential to regulate signaling pathways in recipient cells.  Exosomes deliver nucleic acids and proteins to mediate the communication between cancer cells and stroma cells.  These investigators summarized recent progress in the understanding of the role of exosomes in PCa.  The tumorigenesis, metastasis and drug resistance of PCa are associated with the cargos of exosomes such as miRNA, lncRNAs and proteins.  In addition, PCa cells modulate surrounding stromal cells via the exosomes.  Affected stromal cells employ the exosomes to modulate microenvironment and promote tumor growth and metastasis.  Exosomes derived from PCa cells contribute to cancer chemo-resistance.  The lipid bilayer membrane of the exosomes makes them promising carriers of drugs and other therapeutic molecules targeting PCa.  Furthermore, exosomes can be detected and isolated from various body fluids for the diagnosis of PCa.  The authors concluded that accumulating evidences confirm that exosomes are implicated in the progression and metastasis of PCa.  Many biological molecules are encapsulated in the exosomes from PCa such as miRNAs, lncRNAs and proteins, and their expression levels differ from those of normal prostate cells.  The easy isolation of exosomes from body fluid enables them as potential biomarkers of PCa.  Furthermore, the lipid bilayer membrane of exosomes makes them promising carriers of drugs and other therapeutic molecules to target PCa.  In the near future, it is expected that the power of this nano-sized vesicles would be realized to promote the clinical application of exosomes in PCa diagnosis and therapy.\n",
      "Tutrone et al (2020) noted that the ExoDx Prostate (IntelliScore) (EPI) test is a non-invasive risk assessment tool for detection of high-grade prostate cancer (HGPC) that informs whether to proceed with prostate biopsy.  These researchers examined the impact of EPI on the decision to biopsy in a real-world clinical setting.  They conducted a prospective, randomized, blinded, 2-armed clinical utility study that enrolled 1,094 patients with 72 urologists from 24 urology practices.  Patients were considered for prostate biopsy at enrollment based on standard clinical criteria.  All patients had an EPI test; however, patients were randomized into EPI versus control arms where only the EPI arm received results for their biopsy decision.  In the EPI arm (n = 458), 93 patients received negative EPI scores of which 63 % were recommended to defer biopsy by the urologist and 74 % ultimately deferred.  In contrast, 87 % of patients with positive EPI scores were recommended to undergo biopsy with a 72 % compliance rate to the urologist's recommendation.  This resulted in detection of 30 % more HGPC compared to the control arm, and these investigators estimated that 49 % fewer HGPC were missed due to deferrals compared to standard of care (SOC).  Overall, 68 % of urologists reported that the EPI test influenced their biopsy decision.  The primary reason not to comply with EPI results was rising PSA.  The authors concluded that to their knowledge, this was the 1st report on a PC biomarker utility study with a blinded control arm.  The study demonstrated that the EPI test influenced the overall decision to defer or proceed with a biopsy and improved patient stratification.\n",
      "The authors stated that this study had several drawbacks.  There was a 5.7 % assay failure rate in the EPI arm (30 assay failures of 520 EPI patients).  If these researchers included the patients that were randomized to not receive the EPI test, the failure rate was 7.1 %.  The failed assay controls was representative of the assay quality control procedures and reflected variations in urine exosome concentration.  Although follow-up is ongoing, these researchers currently lack data evaluating long-term outcomes among patients who deferred biopsy after using EPI or any health economics data.  They anticipate both aspects will be addressed in the next year.  Furthermore, despite the innovative study design, the large number of sites and urologists required stream-lined questionnaires, thereby limiting comprehensive feedback assessment.  The authors had a small number of patients (less than 5 %) who underwent pre-biopsy MRI.  A pre-biopsy MRI has the potential to help refine biopsy accuracy and provide additional information regarding EPI test performance.  These researchers also did not use MRI-targeted biopsies in this study as they were not available for them in this real-world clinical setting.  They stated that future studies could include a larger percentage of patients with MRI data available.  (Conflict of interest PT, VT, MN and JS are employees of Bio-Techne.  TM and MJD are consultants for Bio-Techne).\n",
      "National Comprehensive Cancer Networkâs clinical practice guideline on âProstate cancer early detectionâ (Version 1.2023) states that tests that biomarkers that improve the specificity of detection are not, as yet, mandated as 1st-line screening tests in conjunction with serum PSA.  However, there may be some patients who meet PSA standards for consideration of prostate biopsy, but for whom the patient and/or the physician wish to further define risk.  Lower percent-free PSA and/or higher PSA density are associated with a greater risk of high-grade prostate cancer.  The probability of high-grade cancer (Gleason score â¥ 3+4, Grade Group 2 or higher) may be further defined utilizing the Prostate Health Index (PHI), SelectMDx, 4Kscore, ExoDx Prostate Test, MyProstateScore (MPS), and IsoPSA.  Extent of validation of these tests across diverse populations is variable.  It is not yet known how such tests could be applied in optimal combination with MRI.  Results of biomarker assays can be complex and should be interpreted with caution.  Moreover, the NCCN panel believes that ExoDx Prostate (IntelliScore), also called EPI, can be considered as an option for individuals contemplating initial or repeat biopsy.\n",
      "Next-Generation Sequencing and Solid Cancers\n",
      "Forouzanfar and associates (2017) noted that esophageal squamous cell carcinoma is one of the deadliest of all the cancers.  Its metastatic properties portend poor prognosis and high rate of recurrence.  A more advanced method to identify new molecular biomarkers predicting disease prognosis can be whole exome sequencing (WES).  These researchers reported the most effective genetic variants of the Notch signaling pathway in esophageal SCC susceptibility by WES.  These investigators analyzed 9 probands in unrelated familial esophageal SCC pedigrees to identify candidate genes.  Genomic DNA was extracted and WES performed to generate information about genetic variants in the coding regions.  Bioinformatics software applications were utilized to exploit statistical algorithms to demonstrate protein structure and variants conservation.  Polymorphic regions were excluded by false-positive investigations.  Gene-gene interactions were analyzed for Notch signaling pathway candidates.  These researchers identified novel and damaging variants of the Notch signaling pathway through extensive pathway-oriented filtering and functional predictions, which led to the study of 27 candidate novel mutations in all 9 patients.  Detection of the tri-nucleotide repeat containing 6B gene mutation (a slice site alteration) in 5 of the 9 probands, but not in any of the healthy samples, suggested that it may be a susceptibility factor for familial esophageal SCC.  Noticeably, 8 of 27 novel candidate gene mutations (e.g., epidermal growth factor, signal transducer and activator of transcription 3, MET) act in a cascade leading to cell survival and proliferation.  The authors concluded that these findings suggested that the tri-nucleotide repeat containing 6B mutation may be a candidate predisposing gene in esophageal SCC.  In addition, some of the Notch signaling pathway genetic mutations may act as key contributors to esophageal SCC.\n",
      "Kyrochristos and colleagues (2017) stated that hepatobiliary and pancreatic (HBP) cancers are associated with high cancer-related death rates.  Surgery aiming for complete tumor resection (R0) remains the cornerstone of the treatment for HBP cancers.  The current progress in the adjuvant treatment is quite slow, with gemcitabine chemotherapy available only for pancreatic ductal adenocarcinoma (PDA).  In the advanced and metastatic setting, only 2 targeted drugs have been approved by the FDA, which are sorafenib for hepatocellular carcinoma and erlotinib for PDA.  It is a pity that multiple phase III randomized control trials (RCTs) examining the effectiveness of targeted agents have negative results.  Failure in the development of effective drugs probably reflects the poor understanding of genome-wide alterations and molecular mechanisms orchestrating therapeutic resistance and recurrence.  In the post-ENCODE (Encyclopedia of DNA Elements) era, cancer is referred to as a highly heterogeneous and systemic disease of the genome.  The unprecedented potential of NGS technologies to accurately identify genetic and genomic variations has attracted major research and clinical interest.  The applications of NGS include targeted NGS with potential clinical implications, while WES and whole-genome sequencing (WGS) focus on the discovery of both novel cancer driver genes and therapeutic targets.  These advances dictate new designs for clinical trials to validate biomarkers and drugs.  The authors discussed the findings of available NGS studies on HBP cancers and the limitations of genome sequencing analysis to translate genome-based biomarkers and drugs into patient care in the clinic.  They concluded that the validity of NGS technologies to identify tumor heterogeneity-associated therapeutic resistance and relapse gives rise to high expectations for translating these advances into patient-centric trials and clinical benefit.  In the medium-term, targeted NGS enables the conduction of umbrella and basket clinical trials.  The identification of mutated or amplified gene-based patient subgroups and the subsequent tumor-guided treatment with targeted drugs from the list of available FDA-approved agents, matching these specific genetic alterations, could improve personalized patient care.  By contrast, the discovery of novel therapeutic targets by WES and WGS studies raises much higher expectations to substantially broaden the targeted drugs catalogue with a long-term perspective.  However, this concept requires evaluation and confirmation by appropriately designed large-scale clinical trials.  These researchers stated that targeted NGS, WES, and WGS could enable the development of robust biomarkers for tailored treatment; and translational NGS research represents a top prospect for faster progress than any other available technology to achieve precision oncology.\n",
      "Lianos and co-workers (2017) noted that by identifying cancer driver genes involved in tumorigenesis, WES analyses enable the development of robust biomarkers and novel therapeutic targets to reach precision oncology.  In this study, WES analyses were performed in matched gastric cancer-normal gastric tissues from 2 patients.  These researchers compared genes highlighted with those of a database and recent WES/WGS studies.  They identified 32 highlighted gastric cancer genes, 2 of these (DEFB118 and RNF43) may provide future potential clinical implications.  The authors concluded that definitive evidence on extensive genetic heterogeneity suggested the need for large-scale NGS studies to validate gastric cancer driver genes catalog.  This list represents the foundation for developing genome-based biomarkers to guide precision gastric cancer treatment.\n",
      "Oncomine Dx Target Test\n",
      "The Oncomine Dx Target Test (Thermo Fisher Scientific, Inc, Carlsbad, CA) is a qualitative in vitro diagnostic test that uses targeted high-throughput, parallel-sequencing technology to detect sequence variations in 23 genes in DNA and RNA isolated from formalin-fixed, paraffin-embedded tumor (FFPE) tissue samples from patients with non-small cell lung cancer (NSCLC) using the Ion PGM Dx System. In June 2017, the FDA granted premarket approval for Oncomine Dx, the first next-generation sequencing (NGS) âbased test, that simultaneously screens tumor samples for biomarkers within four days, and are associated with three genes that are associated with FDA-approved therapies for non-small cell lung cancer (NSCLC).  Three of the markers provide an aid in selecting patients for approved targeted therapies, while others are currently being investigated in clinical trials and are potentially actionable in the future. Results from analysis of these three genes can be used to identify patients who may be eligible for treatment with one of the following: the combined therapy of TafinlarÂ® and MekinistÂ®, XALKORIÂ®, or IRESSAÂ®. The test became available in the United States in July 2017 (Thermo Fisher, 2017).\n",
      "RAS (KRAS and NRAS) and BRAF Mutations Testing for Colon Cancer\n",
      "NCCNâs clinical practice guidelines on \"Colon cancer\" (Version 1.2018) states that \"All patients with metastatic colorectal cancer should have tumor tissue genotyped for RAS (KRAS and NRAS) and BRAF mutations\".\n",
      "MSK-IMPACT\n",
      "MSK-IMPACT is a 468-gene oncopanel intended to detect gene mutations and other critical genetic aberrations in both rare and common cancers created by the Department of Pathology at Memorial Sloan Kettering Cancer Center (MSK). The MSK-IMPACT assay is a single-site, qualitative in vitro diagnostic test that uses targeted next-generation sequencing of formalin-fixed, paraffin-embedded tumor tissue matched with normal specimens from patients with solid malignant neoplasms to detect tumor gene alterations in a broad multigene panel. The test is intended to provide information on somatic mutations (point mutations and small insertions and deletions) and microsatellite instability (MSI) for use by qualified healthcare professionals in accordance with professional guidelines. It is not conclusive or prescriptive for labeled use of any specific therapeutic product. The FDA reviewed data for MSK-IMPACT through the de novo premarket review pathway, a regulatory pathway for novel, low- to moderate-risk devices that are not substantially equivalent to an already legally marketed device. Following the de novo authorization, the FDA intends to allow future, similar tests to come to the U.S. market as substantial equivalents.\n",
      "FDA authorization was based upon a large-scale, prospective clinical sequencing initiative which evaluated the clinical impact of real-time large-scale tumor sequencing in patients with metastatic cancer. In this trial, Zehir et al (2017) stated tumor molecular profiling is a fundamental component of precision oncology, enabling the identification of genomic alterations in genes and pathways that can be targeted therapeutically. The existence of recurrent targetable alterations across distinct histologically defined tumor types, coupled with an expanding portfolio of molecularly targeted therapies, demands flexible and comprehensive approaches to profile clinically relevant genes across the full spectrum of cancers. The authors established a large-scale, prospective clinical sequencing initiative using a comprehensive assay, MSK-IMPACT, through which we have compiled tumor and matched normal sequence data from a unique cohort of more than 10,000 patients with advanced cancer and available pathological and clinical annotations. Using these data, the authors identified clinically relevant somatic mutations, novel noncoding alterations, and mutational signatures that were shared by common and rare tumor types. The authors reported that nearly 37 percent of patients who had their tumors sequenced through MSK-IMPACT had at least one actionable mutation, meaning that drugs to target these mutations were available either in a clinical trial or as part of the standard of care. In addition, this assay allows for the detection of microsatellite instability (MSI) providing a means to select patients who may benefit from immunotherapy.  Of 5,009 patients tested by MSK-IMPACT prior to July 2015, 1,894 (38%) were enrolled on any clinical trial, 811 (16%) were enrolled on a clinical trial with a targeted agent, and 527 (11%) harbored genomic alterations matching the drug target. Of all matches, 72% occurred after the MSK-IMPACT reports were issued, and the remaining matches were based on the results of prior molecular testing. The authors concluded that while this study represents a first step towards evaluating the clinical impact of large-scale prospective tumor sequencing, more systematic studies are needed to assess the long-term effects of clinical cancer genomics on patient outcomes. These studies will require detailed, longitudinal follow-up. Additionally, data sharing across laboratories and institutions engaged in tumor sequencing is paramount in order to realize the full discovery potential of the resulting datasets. To this end, the authors have deposited their full dataset into the cBioPortal for Cancer Genomics.\n",
      "Merkel SmT Oncoprotein Antibody Titer\n",
      "This is a quantitative test specific to detection of antibodies to the Merkel cell polyoma virus oncoprotein.\n",
      "Paulson et al (2017) stated Merkel cell carcinoma (MCC) is an aggressive skin cancer with a recurrence rate of >40%. Of the 2000 MCC cases per year in the United States, most are caused by the Merkel cell polyomavirus (MCPyV). Antibodies to MCPyV oncoprotein (T-antigens) have been correlated with MCC tumor burden. The present study assesses the clinical utility of MCPyV-oncoprotein antibody titers for MCC prognostication and surveillance. MCPyV-oncoprotein antibody detection was optimized in a clinical laboratory. A cohort of 219 patients with newly diagnosed MCC were followed prospectively (median follow-up, 1.9 years). Among the seropositive patients, antibody titer and disease status were serially tracked. Antibodies to MCPyV oncoproteins were rare among healthy individuals (1%) but were present in most patients with MCC (114 of 219 patients [52%]; Pâ<â.01). Seropositivity at diagnosis independently predicted decreased recurrence risk (hazard ratio, 0.58; Pâ=â.04) in multivariate analyses adjusted for age, sex, stage, and immunosuppression. After initial treatment, seropositive patients whose disease did not recur had rapidly falling titers that became negative by a median of 8.4 months. Among seropositive patients who underwent serial evaluation (71 patients; 282 time points), an increasing oncoprotein titer had a positive predictive value of 66% for clinically evident recurrence, whereas a decreasing titer had a negative predictive value of 97%. The authors concluded determination of oncoprotein antibody titer assists in the clinical management of patients with newly diagnosed MCC by stratifying them into a higher risk seronegative cohort, in which radiologic imaging may play a more prominent role, and into a lower risk seropositive cohort, in which disease status can be tracked in part by oncoprotein antibody titer.\n",
      "Paulson et al (2010) stated Merkel cell polyomavirus (MCPyV) is a common infectious agent that is likely involved in the etiology of most Merkel cell carcinomas (MCC). Serum antibodies recognizing the MCPyV capsid protein VP1 are detectable at high titer in nearly all MCC patients and remain stable over time. Although antibodies to the viral capsid indicate prior MCPyV infection, they provide limited clinical insight into MCC because they are also detected in more than half of the general population. The authors investigated whether antibodies recognizing MCPyV large and small tumor-associated antigens (T-Ag) would be more specifically associated with MCC. Among 530 population control subjects, these antibodies were present in only 0.9% and were of low titer. In contrast, among 205 MCC cases, 40.5% had serum IgG antibodies that recognize a portion of T-Ag shared between small and large T-Ags. Among cases, titers of T-Ag antibodies fell rapidly (â¼8-fold per year) in patients whose cancer did not recur, whereas they rose rapidly in those with progressive disease. Importantly, in several patients who developed metastases, the rise in T-Ag titer preceded clinical detection of disease spread. These results suggest that antibodies recognizing T-Ag are relatively specifically associated with MCC, do not effectively protect against disease progression, and may serve as a clinically useful indicator of disease status.\n",
      "Merkel Virus VP1 Capsid Antibody\n",
      "This test is similar to the Merkel SmT Oncoprotein Antibody Titer test, but this test detects antibodies to the Merkel cell polyoma virus capsid protein (VP1) and the result is reported as positive or negative.\n",
      "Samimi et al (2016) stated Merkel cell polyomavirus (MCPyV) is the main aetiological agent of Merkel cell carcinoma (MCC). Serum antibodies against the major MCPyV capsid protein (VP1) are detected in the general population, whereas antibodies against MCPyV oncoproteins (T antigens) have been reported specifically in patients with MCC. The primary aim was to assess whether detection of serum antibodies against MCPyV proteins at baseline was associated with disease outcome in patients with MCC. The secondary aim was to establish whether evolution of these antibodies during follow-up was associated with the course of the disease. Serum T-antigen and VP1 antibodies were assessed by enzyme-linked immunosorbent assay using recombinant proteins in a cohort of 143 patients with MCC, including 84 patients with serum samples available at baseline. Low titres of VP1 antibodies at baseline (< 10 000) were significantly and independently associated with increased risk of recurrence [hazard ratio (HR) 2Â·71, 95% confidence interval (CI) 1Â·13-6Â·53, P = 0Â·026] and death (HR 3Â·74, 95% CI 1Â·53-9Â·18, P = 0Â·004), whereas T-antigen antibodies were not found to be associated with outcome. VP1 antibodies did not differ between patients in remission and those with recurrence or progression during follow-up. However, T-antigen antibodies were more frequently detected in patients with recurrence or progression at 12 months (P = 0Â·020) and 24 months (P = 0Â·016) after diagnosis. VP1 antibodies constitute a prognostic marker at baseline, whereas T-antigen antibodies constitute a marker of disease recurrence or progression if detected > 12 months after diagnosis.\n",
      "MyAML Next Generation Sequencing Panel\n",
      "MyAML is a 194 targeted next generation sequencing (NGS) gene panel. Coding regions and potential genomic breakpoints within known somatic gene fusions are sequenced with 300bp paired end reads on an Illumina MiSeq instrument to an average depth of coverage >1000x. Using Invivoscribe's proprietary MyInformatics annotation and bioinformatics database, testing idenifies single nucleotide variants (SNVs), indels, inversions and translocations. In addition, allelic frequencies can also be used to investigate potential aneuploidy and clonality.\n",
      "OmniSeq Advance DNA and RNA sequencing (OmniSeq and LabCorp)\n",
      "OmniSeq Advance is an assay that combines targeted therapy and immune-oncology biomarkers into a single clinical test. The OmniSeq Advance Assay uses two different testing methodologies: IHC and DNA-sequencing, RNA-sequencing and MSI performed by NGS. The components of this assay include PD-L1, CD8 by IHC, microsatellite instability (MSI), tumor mutational burden (TMB), RNA-sequencing of over 50 critical immune markers to characterize the tumor micro-environment (TME), and somatic genomic profiling of 144 genes. The assay combines these tests from fixed-formalin paraffin embedded patient samples. The assay is expected to provide benefits for the treatment of patients with solid tumor cancers and the development of new precision therapies. Utilizing data from patients tested with both OmniSeq Comprehensive and OmniSeq Immune Report Card, the company projects that the OmniSeq Advance test will report clinically actionable results for 99% of tested patients. \n",
      "Prostate Cancer Risk Panel (FISH analysis by Mayo Clinic)\n",
      "This laboratory developed fluorescence in situ hybridization test(FISH) by Mayo Clinic is aimed at assisting with determining the probability of a higher prostate tumor grade. The genes analyzed are ASAP1, HDAC9, CHD1, and PTEN.\n",
      "Karnes et al stated in an effort to lessen overtreatment of prostate cancer, further risk stratification on needle biopsy specimens can be critical for patient management. Men with low-risk disease are candidates for active surveillance. Currently, the most important feature for risk stratification is Gleason score; however, sampling error in the needle biopsy procedure results in a significant underestimation of risk in harboring a Gleason grade 4 (Group 2 or 3). The objective of this study was to use genomic features associated with significant prostate cancer previously identified by massively parallel mate-pair next generation sequencing (NGS), and create a model now using fluorescence in situ hybridization (FISH) that could be applied to needle biopsies to improve risk stratification. FISH probes for six genomic alterations associated with significant prostate cancer were applied to 150 contemporary consecutive needle biopsy specimens of men who underwent radical prostatectomy (RP), and a model was constructed that predicted for men with Gleason score (GS) 6 (Group 1) on needle biopsy, the probability of GS 7 and higher (Group >2) in the RP specimen. The final outcome measured was the predicted probability of harboring a significant cancer or GS > 7 (Group >2) in the prostate gland based on a derived formula from FISH analysis of single core needle biopsies. Concordance measures were created using an elastic net model. The application of these probes to needle biopsy specimens confirmed that a model composed of PTEN, CHD1, ASAP1 and HDAC9 was predictive of upgrading (AUC 0.788) from GS 6 on needle biopsy to GS > 7 in RP specimens. The AUC on the biopsies was less than that on earlier discovery and validation sets likely related to inter-tumoral heterogeneity and sampling bias from biopsies. The authors concluded  that use of this model could be clinically useful in risk stratification for patients considering active surveillance for prostate cancer by separating those GS 6 (Group 1) on biopsies into \"lower\" or \"higher\" risk.\n",
      "Tumor Human Papillomavirus (HPV) Testing\n",
      "The National Comprehensive Cancer Network (NCCN) Biomarkers Compendium (2018) include a 2A recommendation for human papillomavirus (HPV) tumor testing in the workup of occult primary cancers and cancer of the oropharynx (e.g., base of tongue, tonsil, posterior pharyngeal wall, soft palate). Tumor HPV association by testing p16 immunohistochemistry (IHC) for the workup of cancer of the oropharynx is preferred. Per NCCN guidelines on \"Cancer of the Oropharynx\" (2018), \"p16 expression is highly correlated with HPV status and prognosis. Expression of p16 as detected by IHC, is a widely available surrogate biomarker that has very good agreement with HPV status as determined by the gold standard of HPV E6/E7 mRNA expression. Other tests include HPV detection through PCR and in situ hybridization (ISH).\n",
      "The NCCN guidelines on \"Head and Neck Cancers\" (version 2.2018) states that there is emerging evidence that shows HPV infection may also be associated with increased risk of SCC of the larynx, and the overall incidence of HPV- positive head and neck cancers is increasing in the U.S. Though there \"is a small proportion of non-oropharyngeal tumor sites that are related to HPV, there is lack of consistent evidence in support of prognostic significance, routine HPV testing or p16 testing of non-oropharyngeal cancers (e.g., paransal sinus, oral cavity, larynx).\n",
      "The NCCN Biomarkers Compendium (2018) for HPV infection include a 2A recommendation for HPV testing in the workup of head and neck cancers that are considered occult primary (e.g., sqaumous cell carcinoma, adenocarcinoma, and anaplastic/undifferentiated epithelial tumors.\n",
      "BreastSentry\n",
      "BreastSentry is a test that measures the serum levels of 2 biomarkers, pro-neurotensin (pro-NT) and pro-enkephalin (pro-ENK), which are supposedly predictive of a womanâs risk for developing breast cancer.  Available evidence to support the use of these markers were observational studies examining the correlation of these markers with incidence of breast cancer.  However, there is no evidence on the use of these markers in a screening program, much less data on what actions would be taken based upon the results of this testing.\n",
      "Melander and colleagues (2014) noted that high fasting plasma pro-NT concentration was associated with the development of breast cancer in the Malmo Diet and Cancer Study (MDCS).  These investigators aimed at replicating the initial finding in an independent second cohort.  The Malmo Preventive Project (MPP) is a population study and comprised 18,240 subjects when examined in 2002 to 2006.  Of women without history of breast cancer at examination, these researchers included all who developed breast cancer during follow-up (n = 130) until December 31, 2010, and a random sample of women without breast cancer until the end of follow-up (n = 1,439) for baseline plasma pro-NT assessment (mean age of 70.0 Â± 4.4 years); pro-NT was measured in fasting plasma samples and was related to the risk of later breast cancer development using multi-variate logistic regression.  Pro-neurotensin [OR per standard deviation (SD) increment of LN-transformed pro-NT] was significantly related to incident breast cancer [OR, 2.09; 95 % CI: 1.79 to 2.44; p < 0.001; adjusted for age, body mass index (BMI), smoking, and hormone replacement therapy].  The effect estimate in the MPP was larger than in the discovery cohort (MDCS), with the main difference between the 2 cohorts being that women of the MPP study were on the average about 10 years older and follow-up time was shorter than that of the MDCS.  The authors concluded that as initially found in the MDCS, fasting plasma pro-NT was significantly associated with the development of breast cancer in the MPP study as well.  They stated that the replication suggested that pro-NT may be of value for the identification of women at high risk of breast cancer development.\n",
      "Melander and associates (2015) stated that in experimental studies, enkephalins (ENKs) and related opioids have been implicated as negative regulators of breast cancer development by enhancing immune-mediated tumoral defense as well as directly inhibiting cancer cells.  These researchers hypothesized that plasma levels of ENKs are predictive of the long-term breast cancer risk. They measured pro-ENK A, a surrogate for mature ENK, and examined its predictive value for the development of breast cancer in a large population of middle-aged women and an independent study population.  These investigators related pro-ENK in fasting plasma samples from 1,929 healthy women (mean age of 57.6 Â± 5.9 years) of the MDCS to breast cancer incidence (n = 123) during a median follow-up of 14.7 years.  For replication, pro-ENK was related to risk of breast cancer (n = 130) in an older independent sample from the MPP consisting of 1,569 women (mean age of 70.0 Â± 4.4 years).  In the MDCS, pro-ENK was inversely related to the risk of incident breast cancer, with a hazard ratio (HR) per each SD increment of logarithm-transformed pro-ENK of 0.72 (95 % CI: 0.62 to 0.85; p < 0.001).  The linear elevation of risk over pro-ENK quartiles 3, 2, and 1, with the 4rth quartile as a reference, was 1.38 (95 % CI: 0.73 to 2.64), 2.29 (95 % CI: 1.26 to 4.15), and 3.16 (95 % CI: 1.78 to 5.60; for the trend, p < 0.001), respectively.  These results were replicated in the MPP, where the continuous OR for incident breast cancer was 0.63 (95 % CI: 0.52 to 0.76; p < 0.001) and the risk over pro-ENK quartiles 3, 2, and 1, where the 4th quartile was the reference, was 2.48 (95 % CI: 1.25 to 4.94), 2.94 (95 % CI: 1.50 to 5.77), and 4.81 (95 % CI: 2.52 to 9.18; for the trend, p < 0.001), respectively.  The authors concluded that low fasting plasma concentration of the opioid precursor peptide pro-ENK was associated with an increased risk of future breast cancer in middle-aged and post-menopausal women.  Moreover, they stated that these findings supported the results of several previous experimental studies and should encourage further research regarding the ENK system as a potential therapeutic target in the prevention and treatment of breast cancer.\n",
      "The authors stated that this study had 2 main drawbacks.  Mammographic screening is performed every 2nd year in Swedish women aged 40 to 74 years to identify individuals with sub-clinical but existing breast cancer, whereas pro-ENK testing may identify higher-risk individuals several years before they develop detectable breast cancer.  Unfortunately, these investigators did not have data regarding the family history of breast cancer and the presence of benign breast disease.  Thus, they could not determine whether the association between low levels of pro-ENK and breast cancer risk may be mediated through, or confounded by, these factors.   These researchers did not have access to individual data from mammographic examinations actually performed, and could not exclude a confounding relationship between pro-ENK and mammographic screening density.  However, the Kaplan-Meier curves of 1st breast cancer events versus pro-ENK data continued to separate throughout follow-up, and most evidently from 10 to 15 years after pro-ENK measurement.  This observation was of interest when the potential biologic explanations of these findings were considered.\n",
      "Furthermore, National Comprehensive Cancer Networkâs clinical practice guideline on \"Breast cancer\" (Version 2.2018) does not mention pro-neurotensin (pro-NT) and pro-enkephalin (pro-ENK).\n",
      "miR-31now\n",
      "miR-31now, a proprietary miRNA-based molecular test by GoPath Laboratories, which predicts the potential clinical benefits associated with anti-EGFR therapy for patients with RAS wild type (WT) metastatic colorectal cancer. The mir-31now test quantifies the expression of the miR-31-3p in formalin-fixed, paraffin-embedded (FFPE) tumor samples using standard quantitative reverse transcription PCR (RT-qPCR) [1]. miR-31now is the first positive theranostic test for patients with metastatic colorectal cancer (mCRC).\n",
      "Laurent-Puig et al (2019) stated MiR-31-3p expression has been shown to be associated with response to anti-EGFR therapy. The authors investigated the predictive role of this biomarker in the FIRE-3 study population, including its ability to differentiate outcomes between patients receiving anti-EGFR and anti-VEGF therapy. MiR-31-3p expression was measured in primary tumors obtained from 340 RAS WT mCRC patients enrolled in the FIRE-3 Trial. This included 164 patients randomized to receive FOLFIRI plus cetuximab (FOLFIRI+Cetux) and 176 to FOLFIRI plus bevacizumab (FOLFIRI+Beva). Patients were divided into subgroups defined by low or high miR-31-3p expression using a pre-specified cut-off and by treatment arm. Analyses were performed to assess treatment efficacy by subgroup. Overall Survival (OS) and Progression Free Survival (PFS) were analyzed using Kaplan-Meier curves and Cox regression models. Investigator-assessed objective response (iOR), early tumor shrinkage at 6 weeks (ETS), and centrally-reviewed objective response (cOR) were analyzed using logistic regression models. The predictive value of miR-31-3p expression level was assessed through a treatment interaction test using multivariate models adjusted for potential confounding factors. Low miR-31-3p expressers benefited from cetuximab compared to bevacizumab for PFS (HR=0.74 [0.55;1.00];P=0.05), OS (HR=0.61 [0.41;0.88];P<0.01), iOR (OR=4.0 [1.9;8.2];P<0.01), ETS (OR=4.0 [2.1;7.7];P<0.01) and cOR (OR=4.9 [2.3;10.5]:P<0.01) in multivariate analyses. There was no difference in outcomes for high expressers between treatment arms. MiR-31-3p expression level was predictive of treatment effect for PFS (P=0.03), OS (P=0.05), iOR (P=0.02), ETS (P=0.04) and cOR (P<0.01). The authors concluded that miR-31-3p expression level was validated as a predictive biomarker of cetuximab therapy efficacy for RAS WT mCRC patients.\n",
      "National Comprehensive Cancer Networkâs clinical practice guideline on \"Colon Cancer\" (Version 3.2018) does not mention miR-31-3p.\n",
      "BBDRisk Dx\n",
      "BBDRisk Dx by Silbiotech, Inc. is a genomic risk test for women diagnosed with precancerous breast tumors that have one or more of the following: Atypical Ductal Hyperplasias (ADH), Atypical Lobular Hyperplasias (ALH), Usual Ductal Hyperplasia (UDH), Papilloma, or Sclerosing Adenosis The test assays a set of cancer biomarker proteins in the tumor tissue and  is intended to predict the likelihood of developing breast cancer in the next five years after diagnosis. Based on the cancer marker levels in the tumor tissue, a âCancer Risk Scoreâ in the range of 0-10 will be provided. The higher the risk score indicates a higher risk of developing breast cancer.\n",
      "Poola et al (2019) discussed the dilemma faced by oncologists in administering preventative measures to \"at risk\" patients diagnosed with atypical and non-atypical hyperplasia due to lack of any molecular means of risk stratification and identifying high-risk subjects.  These investigators examined a 4-marker risk signature, MMP-1, CEACAM6, HYAL1, and HEC1, using 440 hyperplastic tissues for identifying high-risk subjects who will benefit from preventative therapies.  They assayed the markers by IHC and combined their expression levels to obtain a composite value from 0 to 10, which they called a \"Cancer Risk Score\".  These researchers demonstrated that the 4-marker-based risk scores predicted subsequent cancer development with an accuracy of 91 % and 86 % for atypical and non-atypical subjects, respectively.  They have established a correlation between risk scores and cancer rates by stratifying the samples into low-risk (score of less than or equal to 0.5); intermediate-risk (score of less than or equal to 5.4), and high-risk (score of greater than 5.4) groups using Kaplan-Meier survival analysis.  These investigators have evaluated cancer rates at 5, 10, and 15 years.  These findings showed that the average cancer rates in the first 5 years among low- and intermediate-risk groups were 2 % and 15 %, respectively.  Among high-risk group, the average cancer rates at 5 years were 73 % and 34 % for atypical and non-atypical subjects, respectively.  The authors concluded that data were presented in establishing MMP-1-, CEACAM6-, HYAL1-, and HEC1-based risk scoring and risk stratification strategy to classify a hyperplasia subject as low-risk, intermediate-risk, or high-risk for developing future breast cancer.  Furthermore, data on cancer rates for a given risk group at 5 years, 10 years, and beyond were presented for atypical as well as non-atypical patients.  The risk scoring model described here offered for the 1st time a tumor biology-based personalized risk scoring method for making informed treatment decisions.  Prophylactic treatment decisions can now be made based on the cancer risk score and consultation between a patient and her health care professional (e.g., a standard screening for women with low-risk, increased surveillance for subjects of intermediate-risk, and risk-reducing endocrine therapies and/or prophylactic mastectomy for high-risk group similar to BRCA gene mutation carriers).  These investigators stated that the above treatment guidelines were approved for only atypical subjects but not for non-atypical subjects.  These findings presented here on non-atypical samples warrants the need for developing treatment guidelines similar to subjects with atypical growths.  These researchers noted that the findings of this study in conjunction with the available prophylactic measures could prevent approximately 20 % to 25 % of sporadic breast cancers.\n",
      "Furthermore, National Comprehensive Cancer Networkâs clinical practice guideline on âBreast cancerâ (Version 2.2022) does not mention BBDRisk (MMP-1, CEACAM6, HYAL1, or HEC1).\n",
      "DecisionDx-Melanoma\n",
      "Sidiropoulos et al (2014) noted that primary dermal melanoma (PDM) is a subtype of melanoma confined to the dermis that may be morphologically impossible to distinguish from cutaneous metastatic melanoma (CMM). These researchers sought to better characterize PDM by describing the clinical, histologic, and molecular features of 49 cases of PDM and examine if a gene expression-profiling (GEP) test could help distinguish PDM from CMM.  They described 49 cases of PDM and examined if any clinical or histopathologic features had a statistically significant relationship with outcome.  In addition, these investigators performed a melanoma GEP test on a subset of the PDM and CMM cases.  Overall recurrence was infrequent and seen in 9 of 49 cases; 6 patients had loco-regional recurrences and 3 patients had distant metastasis.  None of the clinical or histologic parameters showed a statistically significant relationship with recurrence.  There was a statistically significant association of a class I signature by DecisionDx-Melanoma assay (Castle Biosciences Inc., Friendswood, TX) for PDM whereas CMM were more frequently class II (p = 0.023).  The authors concluded that most conventional staging parameters used for prognosis in cutaneous melanoma have limited applicability to PDM.  They stated that the melanoma prognostic assay may be a useful tool for distinguishing PDM from CMM.\n",
      "Gerami et al (2015) reported on the DecisionDx predictive genetic signature for classifying tumors as class 1 (low risk) or class 2 (high risk) for metastasis. Using earlier studies, they compared differences in the levels of 28 genes, including some control genes, using RT-PCR. The 5-year disease-free survival (DFS) rate for the 164 sample training set was 91% for class 1 and 25% for class 2 (P< 0.0001), while the 5-year DFS rate for the 104 sample validation set (stage IâIV) was 97% for class 1 and 31% for class 2 (P< 0.0001). The signature was used to classify stage I and stage IIA tumors, accurately predicting 120 of 134 tumors without metastases as class 1 (90%) and 24 of 30 tumors with metastases as class 2 (80%).\n",
      "Berger et al (2016) sought to ascertain clinical management changes determined by DecisionDx-Melanoma outcome, which classifies cutaneous melanoma (CM) patients being at low (Class 1) or high (Class 2) risk for recurrence. Medical charts were reviewed from 156âCM patients from six institutions (three dermatology and three surgical oncology practices) who were consecutively tested between May 2013 and December 2015. Clinical management data that were compiled and compared before and after receipt of the 31-gene expression test result included frequency of physical exams, frequency and modality of imaging, and referrals to surgical and medical oncologists. Forty-two percent of patients were Stage I, 47% were Stage II and 8% were Stage III. Overall, 95 patients (61%) were Class 1 and 61 (39%) were Class 2. Documented changes in management were observed in 82 (53%) patients, with the majority of Class 2 patients (77%) undergoing management changes compared to 37% of Class 1 patients (pâ<â0.0001 by Fisher's exact test). The majority (77/82, 94%) of these changes were concordant with the risk indicated by the test result (pâ<â0.0001 by Fisher's exact test), with increased management intensity for Class 2 patients and reduced management intensity for Class 1 patients. The authors concluded that molecular risk classification by gene expression profiling has clinical impact and influences physicians to direct clinical management of CM patients. The vast majority of the changes implemented after the receipt of test results were reflective of the low or high recurrence risk associated with the patient's molecular classification. Because follow-up data was not collected for this patient cohort, the study is limited for the assessment of the impact of gene expression profile based management changes on healthcare resource utilization and patient outcome.\n",
      "Ferris et al (2017) noted that a significant proportion of patients with AJCC-defined early-stage cutaneous melanoma have disease recurrence and die.  A 31-gene expression profile (GEP; DecisionDx-Melanoma) that accurately assesses metastatic risk associated with primary cutaneous melanomas has been described.  These researchers compared accuracy of the GEP in combination with risk determined using the web-based AJCC Individualized Melanoma Patient Outcome Prediction Tool.  GEP results from 205 stage I/II cutaneous melanomas with sufficient clinical data for prognostication using the AJCC tool were classified as low (class 1) or high (class 2) risk.  Two 5-year overall survival (OS) cut-offs (AJCC 79 % and 68 %), reflecting survival for patients with stage IIA or IIB disease, respectively, were assigned for binary AJCC risk.  Cox univariate analysis revealed significant risk classification of distant metastasis-free survival (DMFS) and OS (hazard ratio [HR] range of 3.2 to 9.4, p < 0.001) for both tools.  In all, 43 (21 %) cases had discordant GEP and AJCC classification (using 79 % cut-off); 11 of 13 (85 %) deaths in that group were predicted as high-risk by GEP but low risk by AJCC.  The authors concluded that GEP provided valuable prognostic information and improved identification of high-risk melanomas when used together with the AJCC online prediction tool.   These investigators stated that combining GEP-based classification with AJCC staging may help identify the majority of patients who would benefit from increased clinical and imaging-based surveillance, to identify metastatic disease earlier and ultimately administer the most effective therapies developed to-date to improve patient outcomes. Moreover, the authors stated that the drawbacks of this study included specimens reflected tertiary care center referrals; more effective therapies have been approved for clinical use after accrual.\n",
      "Farberg et al (2017) stated that current guidelines for cutaneous malignant melanoma (CMM) provide general recommendations regarding surveillance while indicating that management should be tailored to patients' individual probability of recurrence.  A 31-gene expression profile (31-GEP) test to predict metastatic risk has been previously validated, and classified patients as either Class 1 (low risk) or Class 2 (high risk).  These researchers determined the impact of the 31-GEP test's result on clinical decision-making.  Dermatology residents who attended a national educational conference were presented with clinical validity evidence for the 31-GEP.  Respondents were given 6 CMM patient vignettes with descriptions of clinical features and answered questions about their willingness to recommend sentinel lymph node biopsy (SLNBx) or imaging based on each scenario.  Additionally, respondents were asked to provide the Breslow thickness (BT), ranging from 0.7 to 1.5mm in 0.1-mm increments, at which they would recommend SLNBx, imaging, or oncology referral.  The number of respondents who would recommend each management modality based upon 3 outcomes (no result, Class 1, or Class 2) was quantified.  Differences between response groups were assessed using Fisher's exact test.  The majority of respondents (62 %, 57 %, and 55 %, respectively) indicated a 1.0-mm BT as the guiding modality, reflecting adherence to current guidelines.  After inclusion of a Class 2 result, the BT used to guide SLNBx, oncology referral, and imaging was changed in 47 %, 50 % and 47 % of the responses, respectively, with 95 %, 84 % and 97 % of the cases, respectively, changed in a risk-appropriate direction (decreased BT).  The authors concluded that based on a 31-GEP Class 1 or Class 2 result, risk appropriate recommendations were more likely to be made for each management modality tested in 5 of the 6 patient vignettes (p < 0.05).  The authors concluded that the 31-GEP test had a significant and appropriate impact on management while remaining within the context of established guidelines.\n",
      "Hsueh et al (2017) carried out a prospective evaluation of the GEP performance in patients enrolled in 2 clinical registries.  A total of 222 cutaneous melanoma (CM) patients enrolled in the EXPAND (NCT02355587) and INTEGRATE (NCT02355574) registries met the criteria of age greater than or equal to 16 years, successful GEP result and greater than or equal to 1 follow-up visit for inclusion in this interim analysis.  Primary end-points were recurrence-free survival (RFS), DMFS, and OS.  Median follow-up was 1.5 years for event-free patients.  Median age for subjects was 58 years (range of 18 to 87) and median Breslow thickness was 1.2 mm (range of 0.2 to 12.0); 88 % (282/322) of cases had stage I/II disease and 74 % (237/322) had a SLN biopsy; 77 % (248/322) had class 1 molecular profiles;  1.5-year RFS, DMFS, and OS rates were 97 versus 77 %, 99 versus 89 %, and 99 versus 92 % for class 1 versus class 2, respectively (p < 0.0001 for each).  Multi-variate Cox regression showed Breslow thickness, mitotic rate, and GEP class to significantly predict recurrence (p < 0.01), while tumor thickness was the only significant predictor of distant metastasis and overall survival in this interim analysis.  The authors concluded that interim analysis of patient outcomes from a combined prospective cohort supports the 31-gene GEP's ability to stratify early-stage CM patients into 2 groups with significantly different metastatic risk; RFS outcomes in this real-world cohort were consistent with previously published analyses with retrospective specimens.  They stated that GEP testing complemented current clinic-pathologic features and increased identification of high-risk patients.  Moreover, these investigators stated that \"While direct evidence of a benefit from surveillance has not been published, considering the rapid time to event observed in this interim analysis, and the accuracy of risk prediction by the GEP test, increased surveillance with imaging for class 2 patients might be useful, especially in those patients who would not be offered surveillance options based on stage\".  It should be noted that this was an interim analysis.\n",
      "Cook et al (2018) noted that the DecisionDx-Melanoma test provides prognostic information for patients with CM.  Using formalin-fixed paraffin-embedded primary tumor tissue, the RT-PCR-based test classifies patients into a low- (Class 1) or high-risk (Class 2) category for recurrence based on expression of 31 genes.  The current study was designed to assess the analytical validity of this test.  Inter-assay, inter-instrument, and inter-operator studies were performed to evaluate reliability of the 31-GEP test results, sample stability and reagent stability.  From March 2013 through June 2016, the GEP test was performed on 8,244 CM tumors.  De-identified data from pathology reports were used to assess technical success.  Robust sample and reagent stability was observed.  Inter-assay concordance on 168 specimens run on 2 consecutive days was 99 % and matched probability scores were significantly correlated (R2â=â0.96).  Inter-instrument concordance was 9 5%, and probability scores had a correlation R2 of 0.99 (pâ<â0.001).  From 8,244 CM specimens submitted since 2013, 85 % (7023) fulfilled pre-specified tumor content parameters.  In these samples with sufficient tumor requirements, the technical success of the test was 98 %.  The authors concluded that DecisionDx-Melanoma is a robust GEP test that demonstrated strong reproducibility between experiments and had high technical reliability on clinical samples.\n",
      "Dillon et al (2018) stated that a 31- GEP test that has been clinically validated identifies melanoma patients with low (Class 1) or high (Class 2) risk of metastasis based on primary tumor biology.  These researchers prospectively evaluated the test impact on clinical management of melanoma patients.  Physicians at 16 dermatology, surgical or medical oncology centers examined patients to assess clinical features of the primary melanoma.  Recommendations for clinical follow-up and surveillance were collected.  Following consent of the patient and performance of the GEP test, recommendations for management were again collected, and pre- and post-test recommendations were assessed to determine changes in management resulting from the addition of GEP testing to traditional clinic-pathologic risk factors.  Post-test management plans changed for 49 % (122 of 247) of cases in the study when compared to pre-test plans; 36 % (66 of 181) of Class 1 cases had a management change, compared to 85 % (56 of 66) of Class 2 cases.  GEP class was a significant factor for change in care during the study (p < 0.001), with Class 1 accounting for 91 % (39 of 43) of cases with decreased management intensity, and Class 2 accounting for 72 % (49 of 68) of cases with increases.  The authors concluded that the reported study showed that the 31-gene GEP test improved net health outcomes in the management of cutaneous melanoma.  Physicians used test results to guide risk-appropriate changes that match the biological risk of the tumor, including directing more frequent and intense surveillance to high-risk, Class 2 patients.\n",
      "Zager et al (2018) stated that the heterogeneous behavior of patients with melanoma makes prognostication challenging.  To address this, a GEP test to predict metastatic risk was previously developed.  This study evaluated the GEP's prognostic accuracy in an independent cohort of cutaneous melanoma patients.  This multi-center study analyzed primary melanoma tumors from 523 patients, using the GEP to classify patients as Class 1 (low risk) and Class 2 (high risk).  Molecular classification was correlated to clinical outcome and assessed along with AJCC v7 staging criteria.  Primary end-points were RFS and DMFS.  The 5-year RFS rates for Class 1 and Class 2 were 88 % and 52 %, respectively, and DMFS rates were 93 % versus 60 %, respectively (P <â0.001).  The GEP was a significant predictor of RFS and DMFS in univariate analysis (HRâ=â5.4 and 6.6, respectively, p <â0.001 for each), along with Breslow thickness, ulceration, mitotic rate, and SLN status (p <â0.001 for each).  GEP, tumor thickness and SLN status were significant predictors of RFS and DMFS in a multi-variate model that also included ulceration and mitotic rate (RFS HRâ=â2.1, 1.2, and 2.5, respectively, p <â0.001 for each; and DMFS HRâ=â2.7, 1.3 and 3.0, respectively, p <â0.01 for each).  The authors concluded that the GEP test is an objective predictor of metastatic risk and provided additional independent prognostic information to traditional staging to help estimate an individual's risk for recurrence.  The assay identified 70 % of stage I and II patients who ultimately developed distant metastasis.  Moreover, they stated that its role in consideration of patients for adjuvant therapy should be examined prospectively.\n",
      "The authors stated that one of the drawbacks of this study was the inclusion of samples in the cohort that were diagnosed prior to widespread standardization of reporting for pathological variables such as Breslow thickness, ulceration and mitosis and therefore some pathology reports did not specify all features.  However, the Cox regression models assessing the association between GEP and those factors accounted for this limitation and only patients with all factors specified were included in this analysis.  Another drawback was the retrospective nature of the study and thus did not take into account recent advances in management of patients with advanced melanoma in the adjuvant and metastatic settings.  However, recently published results of an interim analysis of the GEP test in a prospective cohort showed consistency of results with this another retrospective cohorts.\n",
      "Greenhaw et al (2018) stated that cutaneous melanomas (CMs) with similar clinical and histopathologic features can harbor differing capacities for metastasis.  A validated gene expression profile (GEP) test offers prognostic information by classifying CMs as low-risk (Class 1A/1B) or high-risk (Class 2A/2B) for metastasis.  These researchers performed an independent study of the predictive accuracy of the GEP test, to determine what clinical and histopathologic features predict high-risk classification, and to evaluate how intermediate classes (1B and 2A) performed clinically.  Using their institution's prospectively collected melanoma registry, the authors identified patients who had been treated for CM within the last 5 years and undergone GEP testing.  Clinical, histopathologic, and outcomes data were analyzed.  A sub-cohort of patients with known metastatic disease were identified and tested.  The GEP test accurately identified 77 % of metastatic CMs as high risk (Class 2).  The GEP had a negative predictive value (NPV) of 99 % for Class 1 CMs.  Class 2 CMs were 22 times more likely to metastasize.  The authors concluded that GEP test's performance in their independent cohort corresponded with previous industry-sponsored studies and proved to be a helpful clinical prognostic tool with the potential to direct patient care protocols.\n",
      "Gastman et al (2019) examined risk prediction by the 31-GEP test within 3 low-risk (according to the American Joint Committee on Cancer) populations of patients with CM: those who are SLN-negative, those with stage I to IIA tumors, and those with thin (less than or equal to 1 mm [T1]) tumors.  A total of 3 previous validation studies provided a non-overlapping cohort of 690 patients with 31-GEP results, staging information, and survival outcomes.  Kaplan-Meier and Cox regression analysis were performed.  The results included the identification of 70 % of SLN-negative patients who experienced metastasis as Class 2, the discovery of reduced recurrence-free survival for patients with thin tumors and Class 2B biology compared with that of those with Class 1A biology (p < 0.0001); and determination of the 31-GEP test as an independent predictor of risk compared with traditional staging factors in patients with stage I to IIA tumors.  The authors concluded that the findings of this study confirmed that the 31-GEP test is an independent prognostic factor, and the strongest prognostic indicator, compared to current staging factors.  Managing these patients according to their individual risk (e.g., by more frequent clinical follow-up and increased surveillance/imaging for early identification of metastatic disease) is consistent with current national guidelines.  Furthermore, it is likely that there will be interest in evaluating contemporary adjuvant therapies in stage II patients.  To do this, identifying patient groups with high rates of metastatic events will be necessary for any clinical trial that includes this population.\n",
      "The authors stated that this study was limited by incomplete pathologic staging data owing to variation in contemporaneous reporting standards between 1998 and 2014 and the lack of centralized pathology review.  However, the study cohort reflected the current clinical situation wherein histopathologic assessment of CM may be prone to subjectivity, thus supporting a need for additional methods of risk assessment that are not subject to inter-observer variability.  As Cox regression analyses were performed by using only those cases in which all variables were identified, the number of cases included in each of the analyses, as indicated in each table, was less than the total number of cases.  To address this limitation, Cox regression analysis was also performed including only the co-variates of SLNB and 31-GEP subclass.  Both SLN positivity and 31-GEP Class 2B remained independent predictors of recurrence in patients with T1 tumors (p â¤ 0.005 for both [data not shown]).  A second limitation was the retrospective nature of the sample collection.  Another possible caveat was the proportion of stage III cases within the cohort.  Although the overall cohort exhibited a higher rate of SLN positivity than that typically observed in clinical practice, the MSS outcomes for each stage aligned with the AJCC 8th Edition staging outcomes, which indicated that from the stand-point of stage, the population was representative of contemporary patients with melanoma.\n",
      "Commenting on the study Gastman et al (2019), Marchetti et al (2019) stated that \"Before a prognostic test for cutaneous melanoma is embraced, the test must be demonstrated to be valid and clinically useful in prospective studies with predetermined end points.  Several studies over the past few years have portrayed the 31-GEP test as having analytic and clinical validity, but the clinical utility and cost-effectiveness of the test remain unproved, especially in patients at low risk of metastasis or death.  In our opinion, the data provided by Gastman et al and Greenhaw et al provide compelling evidence against routine clinical use of the 31-GEP test for T1 melanomas.  Its indiscriminate use in the United States has the potential to incorrectly risk-stratify more than 8000 individuals/y with no demonstrable benefit\".\n",
      "Vetto et al (2019) determined if GEP could be used to identify patients with T1-T2 melanoma at low-risk for SLN positivity?  Bioinformatics modeling determined a population in which a 31-GEP test predicted less than 5 % SLN positivity.  Multi-center, prospectively-tested (n = 1,421) and retrospective (n = 690) cohorts were used for validation and outcomes, respectively.  Patients 55 to 64 years and greater than or equal to 65 years with a class 1A (low-risk) profile had SLN positivity rates of 4.9 % and 1.6 %.  Class 2B (high-risk) patients had SLN positivity rates of 30.8 % and 11.9 %.  Melanoma-specific survival was 99.3 % for patients greater than or equal to 55 years with class 1A, T1-T2 tumors and 55.0 % for class 2B, SLN-positive, T1-T2 tumors.  The authors concluded that incorporation of molecular signatures to guide biopsy recommendations is now routine for patients with thyroid, prostate and lung cancers.  These findings showed that a gene expression signature could be applied in melanoma to identify a patient population with less than 5 % predicted probability of a positive SLN with demonstrated high survival rates and therefore has potential utility in guiding SLNB decisions.  Additional multi-center retrospective and prospective studies to confirm and expand these results are ongoing or planned.  If used in this way, the 31-GEP test could potentially reduce a substantial proportion of SLNBs while still maintaining a robust survival rate in those patients with low-risk tumor biology.  These patients could benefit from avoidance of risks associated with surgery and anesthesia; SLN guidance using the GEP test is not meant to deter patients from surgical consultation, as it is always important to discuss all risks/benefits in an individual clinical situation.  Rather, this test may serve as an additional decision-making tool for adhering to national recommendations of personalized care.\n",
      "The authors stated that drawbacks of this study included the fact that long-term follow-up was not available for patients in the prospective cohorts, however outcomes were modeled in the retrospective cohorts which have long-term outcomes.  Another potential drawback was that the study did not include a significant number of T1b-T2 patients who might have been considered for SLNB, but who either decided not to undergo the procedure or the procedure was not performed due to medical contra-indications.  As with all innovative approaches, replication in additional patient cohorts is recommended, to confirm the validity of this approach.  Thus, a 2nd multi-center validation study is ongoing to evaluate patients with T1-T2 melanoma who were clinically tested with the 31-GEP test and their SLNB results.  Additionally, prospective, multi-center studies to track and evaluate clinical outcomes of patients for whom the 31-GEP test is used to guide SLNB decisions under IRB-approved protocols are planned.  Based on results of the MSLT-I trial it is expected that class 1A patients with T1-T2 tumors who forgo the SLNB procedure will not demonstrate a significant difference in melanoma-specific survival rates compared with those who are managed with SLNB.\n",
      "The American Academy of Dermatology (AAD)âs guidelines on \"Care for the management of primary cutaneous melanoma\" (Swetter et al, 2019) state that although ancillary diagnostic molecular techniques (e.g., CGH, FISH, GEP) may be used for equivocal melanocytic neoplasms [Strength of evidence = C; level of evidence = III], routine molecular testing, including GEP, for prognostication is discouraged until better use criteria are defined.  The application of molecular information for clinical management (e.g., sentinel lymph node eligibility, follow-up, and/or therapeutic choice) is not recommended outside of a clinical study or trial\" [Strength of evidence = C, level of evidence = III].  The guidelines explain: \"Diagnostic molecular techniques are still largely investigative and may be appropriate as ancillary tests in equivocal melanocytic neoplasms, but they are not recommended for routine diagnostic use in CM.  These include comparative genomic hybridization, fluorescence in-situ hybridization, gene expression profiling (GEP), and (potentially) next-generation sequencing.  These tests may help to differentiate benign nevi from CM, including atypical Spitz tumors.  In the opinion of the WG, there is also insufficient evidence of benefit to recommend routine use of currently available prognostic molecular tests, including GEP, to provide more accurate prognosis beyond currently known clinicopathologic factors\".\n",
      "Marks and associates (2019) stated that treatment plans for cutaneous melanoma are based upon individual risk of recurrence.  Decisions made post-diagnosis include recommendation for a SLNBx, followed by management decisions such as surveillance, frequency of follow-up, and inter-disciplinary consultations including possible adjuvant therapy use.  These have traditionally been guided by clinicopathologic factors, but discordance exists, as a substantial number of melanoma deaths occur in patients diagnosed with disease considered to be early stage by such factors, including a negative SLNBx.  Molecular testing can be used to apply an objective approach that optimizes individualized patient care.  The 31-gene expression profile (31-GEP) test has been validated in nearly 1,600 patients as an independent predictor of risk of recurrence, distant metastasis and death in Stage I to III melanoma and can guide SLNBx decisions in patient subgroups, as demonstrated in 1,421 patients.  While clinical use of the 31-GEP test has been adopted into routine practice, an evidence-based analysis of a decision point for use in thin, T1 tumors would be clinically useful.  To help define an appropriate population for 31-GEP testing, these investigators evaluated changes in patient management, cumulative differential risk across Breslow thicknesses based on a large data-set, and 31-GEP subclass distribution in a clinically tested cohort.  The authors concluded that these findings suggested a Breslow thickness threshold of 0.3 mm for using the 31-GEP test for guiding management decisions dependent on individual risk of recurrence.\n",
      "Dubin and colleagues (2019) reviewed the current literature and establish the level of evidence for a cutaneous melanoma 31-GEP test.  These researchers carried out a review of 7 development and validation studies for the 31-GEP test.  The respective strengths and weaknesses of each study were applied to the level of evidence criteria from major organizations that publish guidelines for melanoma management: AJCC, NCCN, and AAD.  Evaluating each study led to classifying the 31-GEP test as level I/II, I-IIIB, and IIA according to AJCC, NCCN, and AAD criteria, respectively.  This stands in contrast to the official unrated status conferred by the AJCC and NCCN, and the II/IIIC rating designated by the AAD.  The authors concluded that differences between the authors' findings and official published ratings may be attributed to chronological issues, as many of the studies were not yet published when the afore-mentioned organizations conducted their reviews.  There was also difficulty in applying the NCCN criteria to this prognostic test, as their guidelines were intended for evaluation of predictive markers.  Nevertheless, based upon the most current data available, integration of the 31-GEP test into clinical practice may be warranted in certain clinical situations.  These researchers found the 31-GEP test to be particularly useful for patients with invasive melanoma or older patients with T1/T2 melanomas.  For patients with invasive melanoma, the results of the molecular test may help guide the frequency of skin examinations and utilization of SLNBx or imaging following diagnosis.  Patients aged older than 65 years diagnosed with T1/T2 melanomas may also benefit from molecular testing, especially in the evaluation of the risks and benefits of a SLNBx.\n",
      "Furthermore, NCCNâs clinical practice guideline on \"Cutaneous melanoma\" (Version 2.2019) states that \"Commercially available GEP tests are marketed as being able to classify cutaneous melanoma into separate categories based on risk of metastasis.  However, it remains unclear whether these tests provide clinically actionable prognostic information when used in addition to or in comparison with known clinicopathologic factors or multi-variable nomograms that incorporate patient sex, age, tumor location and thickness, ulceration, mitotic rate, lympho-vascular invasion, microsatellites, and SLNB status.  Furthermore, the impact of these tests on treatment outcomes or follow-up schedules has not been established.\n",
      "Keller et al (2019) stated that gene expression profiling (GEP) has been integrated into cancer treatment decision-making in multiple neoplasms.  These researchers prospectively evaluated the prognostic utility of the 31-GEP test (DecisionDx-Melanoma) in cutaneous melanoma (CM) patients undergoing sentinel node biopsy (SNB).  A total of 159 patients (aged 26 to 88 years) diagnosed with melanoma between 01/2013 and 8/2015 underwent SNB and concurrent GEP testing.  GEP results were reported as low-risk Class 1 (subclasses 1A and 1B) or high-risk Class 2 (subclasses 2A and 2B).  Statistical analyses were performed with Chi-square analysis, t-tests, log-rank tests, and Cox proportional hazard models.  Recurrence-free survival (RFS) and distant metastasis-free survival (DMFS) were estimated using Kaplan-Meier method.  Median follow-up was 44.9 months for event-free cases.  Median Breslow thickness was 1.4 mm (0.2 to 15.0 mm).  There were 117 Class 1 and 42 Class 2 patients.  Gender, age, Breslow thickness, ulceration, SNB positivity, and AJCC stage were significantly associated with GEP classification (p < 0.05 for all).  Recurrence and distant metastasis rates were 5 % and 1 % for Class 1 patients compared with 55 % and 36 % for Class 2 patients.  Sensitivities of Class 2 and SNB for recurrence were 79 % and 34 %, respectively.  Of 10 SNB-positive/Class 2 patients, 9 recurred.  By multi-variate analysis, only SNB result and GEP class were statistically associated with both RFS (p = 0.008 and 0.0001) and DMFS (p = 0.019 and 0.001).  The authors concluded that GEP Class 2 result and SNB positivity were independently associated with recurrence and distant metastasis in primary CM patients.  These researchers stated that GEP testing may have additive prognostic utility in initial staging work-up of these patients.  They stated that GEP testing can identify melanoma patients at high risk for recurrence, however clinical trials are needed for these highârisk patients to determine the optimal treatment strategy, in particular for adjuvant therapy decision-making.\n",
      "Greenhaw and colleagues (2020) stated that multiple studies have reported on the accuracy of the prognostic 31-GEP test for cutaneous melanoma.  Consistency of the test results across studies has not been systematically evaluated.  In a meta-analysis, these researchers examined the robustness of the prognostic value of the 31-GEP test.  Raw data were obtained from studies identified from systematic review.  These investigators determined the overall effect of the 31-GEP test.  Clinical outcome metrics for the 31-GEP test were compared to AJCC staging.  A total of 3 studies met inclusion criteria; data from a novel cohort of 211 patients were included (n = 1,479); 5-year recurrence-free and DMFS rates were 91.4 % and 94.1 % for Class 1A patients and 43.6 % and 55.5 % for Class 2B patients (p < 0.0001).  Meta-analysis results showed that Class 2 was significantly associated with recurrence (HR: 2.90; p < 0.0001) and distant metastasis (HR: 2.75; p < 0.0001).  The 31-GEP test identified AJCC stage I to III patient subsets with high likelihood for recurrence and distant metastasis.  Sensitivity was 76 % [95 % CI: 71 to 80 %] and 76 % [95 % CI: 70 to 82 %] for each end-point, respectively.  When the 31-GEP test and SLNBx results were considered together, sensitivity and NPV for DMFS were both improved.  The authors concluded that the 31-GEP test consistently and accurately identified melanoma patients at increased risk of metastasis, was independent of other clinicopathologic covariates, and augmented current risk stratification by re-classifying patients previously designated as low-risk, for heightened surveillance.  Moreover, these researchers stated that further studies are needed to evaluate appropriate methods and intervals for follow-up of patients identified as high risk by the 31-GEP test, and on therapeutic management, based on risk determined by the 31-GEP test together with other clinicopathologic co-variates.\n",
      "The authors stated that a drawback of this meta-analysis was that studies identified through systematic review were published, so there was the risk that unpublished negative data were not considered.  In addition, the included studies had different study designs, which may have impacted the overall magnitude of the effect of the GEP test because of evolving treatment, management, surveillance, and population differences across the time frame that the samples were collected.  Although 2 cohorts consisted of archived cases and the other 2 consisted of patients tested clinically with the 31-GEP, tests of heterogeneity across studies were not significant.  Furthermore, follow-up time varied among these studies, which should be considered in interpreting the pooled survival estimates.  However, the median follow-up interval for recurrence-free cases was longer than the median time to recurrence.  Finally, while the prospective studies analyzed as part of the meta-analysis had no inclusion/exclusion biases, the study from Greenhaw et al (2018) did not include patients who underwent SLNBx as part of their management protocol.\n",
      "In a commentary on the study by Greenhaw et al (2020), Marchetti et al (2020) stated that these investigators carried out a meta-analysis of the prognostic effect size of the 31-GEP test; and concluded that it had a level 1A evidence ranking under the SORT and Oxford Systems.  However, methodologic shortcomings and the small sample size of studies suggested that the estimated 31-GEP effect size was unlikely to be an accurate measure of the true effect size.  Marchetti et al (2020) stated that the study by Greenhaw et al had several drawbacks including the lack of a pre-specified protocol, the multi-variate analyses did not include important clinicopathologic prognostic factors, such as sex, anatomic site, and mitotic index, incomplete risk of bias assessment, publication bias, and the majority of the study authors reported a financial conflict of interest for this work and were the authors of many (or all) of the included studies.\n",
      "Guidelines from the National Comprehensive Cancer Network (version 3.2020) stated: \"Prognostic gene expression profiling (GEP) to differentiate melanomas at low versus high risk of metastases may provide information on individual risk of recurrence, as an adjunct to standard AJCC Staging. However, the currently available prognostic molecular techniques should not replace pathologic staging procedures, and the use of GEP testing according to specific melanoma stage (before or after SLNB) requires further prospective investigation in large, contemporary data sets of unselected patients\".\n",
      "The Melanoma Prevention Working Group (MPWG) (Grossman et al, 2020) concluded that \"More evidence is needed to support using GEP testing to inform recommendations regarding SLNB, intensity of follow-up or imaging surveillance, and postoperative adjuvant therapy. The MPWG recommends further research to assess the validity and clinical applicability of existing and emerging GEP tests. Decisions on performing GEP testing and patient management based on these results should only be made in the context of discussion of testing limitations with the patient or within a multidisciplinary group\".\n",
      "Hseuh et al (2021) stated that current guidelines for post-operative management of patients with stage I to IIA CM do not recommend routine cross-sectional imaging; however, many of these patients develop metastases.  Methods that complement American Joint Committee on Cancer (AJCC) staging are needed to improve identification and treatment of these patients.  A 31-GEP test predicts metastatic risk as low (class 1) or high (class 2).  Prospective analysis of CM outcomes was carried out to test the hypotheses that the 31-GEP would provide prognostic value for patients with stage I to III CM, and that patients with stage I to IIA melanoma and class 2 31-GEP results have metastatic risk similar to patients for whom surveillance is recommended.  Two multi-center registry studies, INTEGRATE and EXPAND, were initiated under IRB approval, and 323 patients with stage I to III CM and median follow-up time of 3.2 years met inclusion criteria.  Primary endpoints were 3-year RFS, DMFS, and OS.  The 31-GEP was significant for RFS, DMFS, and OS in a univariate analysis and was a significant, independent predictor of RFS, DMFS, and OS in a multi-variable analysis.  GEP class 2 results were significantly associated with lower 3-year RFS, DMFS, and OS in all patients and those with stage I to IIA disease.  Patients with stage I to IIA CM and a class 2 result had recurrence, distant metastasis, and death rates similar to patients with stage IIB to III CM.  Combining 31-GEP results and AJCC staging enhanced sensitivity over each approach alone.  The authors concluded that the findings of this study provided a rationale for using the 31-GEP along with AJCC staging; and suggested that patients with stage I to IIA CM and a class 2 31-GEP signature may be candidates for more intense follow-up.\n",
      "The authors stated that this studyâs outcomes were based on a median follow-up time of 3.2 years for those without an event, which should have captured most events.  However, future additional events may have a small effect on recurrence rates and survival estimates, as previously reported from archival cohorts with longer follow-up.  Furthermore, the number of DMFS and OS events was limited relative to the number of significant variables in the univariate analysis; therefore, Breslow thickness, ulceration, and SLN status were condensed into stage, and inclusion criteria in the multi-variable model were stringent (p < 0.01).  Additionally, OS was evaluated in place of disease-specific survival (DSS) because the cause of death was not documented in some cases.  Lastly, many of the patients in this study underwent SLNBx.  Although it is unlikely SLNBx changed the course of the patientsâ disease, risk stratification by 31-GEP class in this study was consistent with outcomes in a study in which the majority of patients did not undergo SLNBx.\n",
      "Whitman et al (2021) stated that national guidelines recommend SLNB be offered to patients with greater than 10 % likelihood of SLN positivity.  On the other hand, guidelines do not recommend SLNB for patients with T1a tumors without high-risk features who have less than 5 % likelihood of a positive SLN.  However, the decision to perform SLNB is less certain for patients with higher-risk T1 melanomas in which a positive node is expected 5 % to 10 % of the time.  In a retrospective study, these researchers hypothesized that integrating clinicopathologic features with the 31-GEP score using advanced artificial intelligence (AI) techniques would provide more precise SLN risk prediction.  An integrated 31-GEP (i31-GEP) neural network algorithm incorporating clinicopathologic features with the continuous 31-GEP score was developed using a previously reported patient cohort (n = 1,398) and validated using an independent cohort (n = 1,674).  Compared with other co-variates in the i31-GEP, the continuous 31-GEP score had the largest likelihood ratio (G2 = 91.3, p < 0.001) for predicting SLN positivity.  The i31-GEP showed high concordance between predicted and observed SLN positivity rates (linear regression slope = 0.999).  The i31-GEP increased the percentage of patients with T1 to T4 tumors predicted to have less than 5 % SLN-positive likelihood from 8.5 % to 27.7 % with a NPV of 98 %.  More importantly, for patients with T1 tumors originally classified with a likelihood of SLN positivity of 5 % to 10 %, the i31-GEP re-classified 63 % of cases as having less than 5 % or greater than 10 % likelihood of positive SLN, for a more precise, personalized, and clinically actionable SLN-positive likelihood estimate.  The authors concluded that these data suggested the i31-GEP could reduce the number of SLNBs performed by identifying patients with likelihood under the 5 % threshold for performance of SLNB and improve the yield of positive SLNBs by identifying patients more likely to have a positive SLNB.\n",
      "The authors stated that although the i31-GEP developed in this report was independently validated to refine risk assessment within the context of clinical, histologic, and molecular features, there were several drawbacks.  First, the training and validation cohorts were mostly treated at surgical oncology centers, with nearly 80 % undergoing SLNB; thus, patients not referred from a dermatology clinic may not have been included.  Second, some T1a patients were evaluated clinically but did not have SLNB performed; thus, these investigators could not rule out the potential for occult nodal metastases in these patients.  Third, in any retrospective study from various centers, standardization of SLNB, SLN assessment, and event detection bias may influence the results.\n",
      "Brownstone et al (2022) noted that CM is a life-threatening neoplasm where early detection along with appropriately timed intervention could significantly improve outcomes; however, practice gaps still exist for the diagnosis and treatment of CM given the challenges in defining high-risk subsets of lower-risk patients who may die from this cancer.  In addition, decisions to biopsy suspicious lesions are heavily dependent upon subjective visual examinations.  Integrating the non-invasive 31-GEP and 2-GEP tests into clinical practice for evaluating CM diagnosis and prognosis has been shown to enhance accuracy.  For these reasons, GEP technology is becoming an important adjunct in clinical practice to the SOC for the management of patients with CM.\n",
      "Zakria et al (2022) stated that national guidelines for CM suggest avoiding SLNB if the risk of SLN positivity is less than 5 % (T1a with no high-risk features), considering SLNB if the risk is 5 % to 10 % (T1a with additional high-risk features (T1aHR) and T1b), and offering SLNB if the risk is greater than 10 % (T2 to T4).  Because most patients (88 %) who undergo an SLNB have a negative result, novel tools to identify patients who can safely forgo SLNB are important.  The i31-GEP for SLNB test for CM combines tumor molecular biology with clinicopathologic features to provide a precise risk of SLN positivity.  The Melanoma Institute of Australia (MIA) developed a nomogram that uses only clinicopathologic features to predict SLN positivity.  In a retrospective study, these investigators compared the i31-GEP for SLNB to the MIA nomogram in patients with T1 to T2 tumors with complete data (n = 582).  The precision of each tool to identify patients with less than 5 % SLN positivity risk was analyzed using 95 % CIs.  To be considered low-risk, the predicted risk must be less than 5 % and the upper 95 % CI must be 10 % or less, and to be considered high-risk, the predicted risk must be greater than 10 % and the lower 95 % CI of 5% or higher.  The i31-GEP for SLNB identified 28.5 % (166/582) of patients as having a less than 5 % risk of SLN positivity while also having an upper 95 % CI of 10 % or less compared with 0.9 % (5/582, p < 0.001) using the MIA nomogram.  In patients with a pre-test likelihood of SLN positivity of 5 % to 10 % (T1aHR-T1b), the i31-GEP re-classified risk in 60.2 % (171/284) of patients as being less than 5 % or greater than 10% compared to 13.7 % (39/284, p < 0.001) using the MIA nomogram.  In patients with a known SLN status (n = 466), the i31-GEP for SLNB identified 22.1 % (103/466) of patients as having less than 5 % risk, with a 3.9 % (4/103) SLN positivity rate compared to 0.6%  (3/466, p < 0.001) identified by the MIA as having a less than 5 % risk with a 33.3 % (1/3) SLN positivity rate.  The authors concluded that this study showed that the i31-GEP for SLNB outperformed the MIA nomogram for selecting patients for SLNB, given the more precise result provided, and using the i31-GEP for SLNB could lead to a more accurate assignment of the patient into the appropriate NCCN SLN risk grouping.  These researchers stated that the findings of this study suggested that integrating the 31-GEP with clinicopathologic features could improve patient care via better risk-aligned management decisions for SLNBs and reduce the number of unnecessary SLNBs.\n",
      "The authors stated that drawbacks of this study included its retrospective nature and that this selected population from primarily surgical centers may not represent the general population.  Furthermore, comparison to the MIA nomogram limited patient inclusion, primarily due to missing subtypes, in more than 50 % of cases.\n",
      "Jarell et al (2022) noted that many patients with low-stage CM will experience tumor recurrence, metastasis, or death, and many higher staged patients will not.  In a retrospective study, these researchers developed an algorithm by integrating the 31-GEP test with clinicopathologic data for an optimized, personalized risk of recurrence (integrated 31 risk of recurrence [i31-ROR]) or death and use i31-ROR in conjunction with a previously validated algorithm for precise sentinel lymph node positivity risk estimates (i31-SLNB) for optimized treatment plan decisions.  Cox regression models for ROR were developed (n = 1,581) and independently validated (n = 523) on a cohort with stage I to III CM.  Using NCCN cut-points, i31-ROR performance was examined using the mid-point survival rates between patients with stage IIA and stage IIB disease as a risk threshold.  Patients with a low-risk i31-ROR result had significantly higher 5-year RFS (91 % versus 45 %, p < 0.001), DMFS (95 % versus 53 %, p < 0 .001), and melanoma-specific survival (98 % versus 73 %, p < 0.001) than patients with a high-risk i31-ROR result.  A combined i31-SLNB/ROR analysis identified 44 % of patients who could forego SLNBx while maintaining high survival rates (greater than 98 %) or were re-stratified as being at a higher or lower risk of recurrence or death.  The authors concluded that integrating clinicopathologic features with the 31-GEP optimized patient risk stratification compared to clinicopathologic features alone.\n",
      "The authors stated that a drawback of this trial was that due to the lack of reporting of some clinicopathologic variables proposed to impact survival, they were not included in this algorithm.  A 2nd drawback was its retrospective design.  Furthermore, it was unknown which patients received adjuvant therapies, as differences in diagnosis dates may have affected access to these options.  Moreover, these researchers stated that now that they have developed this novel tool, it will be insightful to examine the effect of the i31-ROR on patient management decisions, outcomes, as well as health economics, especially for patients with stage 1 CM, which make up the largest proportion of the newly diagnosed melanomas and a group with a high melanoma-specific survival rate.\n",
      "Williams et al (2022) stated that early detection of CM is instrumental as the 5-year survival decreases from 93.3 % to less than 50 % when metastases are present.  Distinguishing which patients require closer follow-up can be difficult for CM patients.  Developments by Castle Biosciences' DecisionDx-Melanoma (DDx-M) use 31 melanoma associated genes to stratify melanomas into 4 classes with 1A having lowest risk of morbidity and mortality and 2B the highest.  In a retrospective study, these investigators examined the benefit of providing additional 18FDG-PET-CT and brain MRI to genetically high-risk patients who may have otherwise been over-looked.  A total of 297 patients at the authorsâ center had biopsies sent for DDx-M between 2014 and 2021.  Patients found to have Class 2 CM received additional screening with yearly 18FDG-PET-CT scans and brain MRIs.  Patients with Class 2 DDx-M scores and negative SLNB were included in the study; and a total of 66 met inclusion criteria and received imaging.  Within 3 years of follow-up, 8/66 (12.1 %) patients had metastases detected by 18FDG-PET-CT scans.  No patients with stage IA or IB went on to develop metastases.  The authors concluded that 18FDG-PET-CT scans detected metastases in less than 3 % of the time when all stage I and II patients were scanned; however, by using DDx-M in their screening protocols, these investigators achieved a detection rate of 12.1 %.  These patients went on to receive treatment and would have otherwise progressed undetected, leading to higher morbidity and mortality.  The authors suggested all patients with initial stage II or above CM receive a DDx-M score and those with class 2 receive yearly 18FDG-PET-CT/brain MRI imaging.\n",
      "The authors stated that drawbacks of this study included those inherent to retrospective chart review, most importantly, the potential for missing data, especially in a rural area.  These researchers stated that longer follow-up may continue to reveal recurrences past their screening interval with increased imaging.  These investigators do, however, continue to follow these patients with physical examinations and review of systems yearly.  Lastly, this was a descriptive review of the authorsâ institutional experience with increased screening.  They stated that a larger, multi-center study is needed to examine the use of increased imaging.\n",
      "FoundationOne CDx Assa\n",
      "Kato et al (2018) stated that patients with rare tumors may lack approved treatments and clinical trial access.  Although each rare tumor is uncommon, cumulatively they account for approximately 25 % of cancers.  These researchers recently initiated a Rare Tumor Clinic that emphasized a precision medicine strategy.  They examined the first 40 patients presenting at the Rare Tumor Clinic.  Next-generation sequencing (NGS) of tissue and plasma-derived, circulating-tumor DNA (ctDNA), and protein markers were assessed.  Median age was 58 years (range of 31 to 78 years); 70 % (28/40) were women; median number of previous systemic therapies was 2 (range of 0 to 7).  The most common diagnoses were sarcoma (nâ=â7) for solid tumors and Erdheim-Chester disease (nâ=â5) for hematologic malignancies; 20 distinct diagnoses were seen.  Examples of ultra-rare tumors included ameloblastoma, yolk sac liver tumor, ampullary cancer, and Castleman's disease.  Altogether, 32 of 33 patients (97 %) with tissue NGS and 15 of 33 (45 %) with ctDNA sequencing harbored greater than or equal to 1 alteration.  Overall, 92.5 % of patients (37/40) had greater than or equal to 1 actionable target based on either genomic (nâ=â32) or protein (nâ=â27) markers.  In total, 52.5 % (21/40) received matched therapy; 52.4 % (11/21) achieved stable disease (SD) of greater than or equal to 6 months (nâ=â3), partial remission (PR; nâ=â6), or complete remission (CR; nâ=â2).  Matched therapy resulted in significantly longer progression-free survival (PFS) compared with last prior unmatched therapy (hazard ratio [HR] 0.26, 95 % confidence interval [CI]: 0.10 to 0.71, pâ=â0.008).  The authors concluded that identifying genomic and protein markers in patients with rare/ultra-rare tumors was feasible.  When therapies were matched, greater than 50 % of patients attained SD of greater than or equal to 6 months, PR, or CR.  Moreover, they stated that further precision medicine clinical investigations focusing on rare and ultra-rare tumors are urgently needed.  (This study did not mention FoundationOne)\n",
      "Pishvaian et al (2018) noted that to broaden access to and implementation of precision medicine in the care of patients with pancreatic cancer, the Know Your Tumor (KYT) program was initiated using a turn-key precision medicine system.  Patients undergo commercially available multi-omic profiling to determine molecularly rationalized clinical trials and off-label therapies.  Tumor samples were obtained for 640 patients from 287 academic and community practices covering 44 states.  College of American Pathologists/Clinical Laboratory Improvement Amendments-accredited laboratories were used for genomic, proteomic, and phosphoprotein-based molecular profiling.  Tumor samples were adequate for NGS in 96 % and IHC in 91 % of patients.  A tumor board reviewed the results for every patient and found actionable genomic alterations in 50 % of patients (with 27 % highly actionable) and actionable proteomic alterations (excluding chemo-predictive markers) in 5 %.  Actionable alterations commonly found were in DNA repair genes (BRCA1/2 or ATM mutations, 8.4 %) and cell-cycle genes (CCND1/2/3 or CDK4/6 alterations, 8.1 %).  A subset of samples was assessed for actionable phosphoprotein markers.  Among patients with highly actionable biomarkers, those who received matched therapy (n = 17) had a significantly longer median progression-free survival (PFS) than those who received unmatched therapy [n = 18; PFS = 4.1 versus 1.9 months; HR, 0.47; 95 % CI: 0.24 to 0.94; p (adj) = 0.03].  The authors concluded that a comprehensive precision medicine system can be implemented in community and academic settings, with highly actionable findings observed in over 25 % of pancreatic cancers.  Patients whose tumors have highly actionable alterations and receive matched therapy demonstrated significantly increased PFS.  These investigators stated that these findings support further prospective evaluation of precision oncology in pancreatic cancer.  The authors noted that this study was non-randomized with heterogeneously treated patients and clinical history, which may introduce bias, particularly due to selection of patients with an interest in clinical research and the potential for patients with poor performance status being ineligible for trials of targeted therapies.  These investigators stated that as a real-world evidence type study, they believed that programs such as theirs play a critical and important role in precision medicine as a \"signal finding\" effort that could high-light important therapeutic targets-drug combinations that warrant further exploration in more rigorous prospective studies.\n",
      "An assessment of FoundationOne CDx by the Ludwig Boltzmann Institute for Health Technology Assessment (Wild and GrÃ¶ssmann, 2019) concluded: \"Currently, there is no scientific evidence that diagnostics with multi-gene panels for the development of therapy recommendations lead to better clinical outcomes. Few biomarkers are validated and recommended by the EMA, as well as by the FDA. Many more are at a research stage, although many expectations and hopes are being raised for multi-gene panels. It can be predicted that multi-gene panels will have the potential to stimulate a broad off-label use of drugs without having to test them for clinical relevance in clinical trials. These consequences should be given attention, as many approved oncology medications only have a marginal benefit (0-2 according to ESMO Magnitude of Clinical Benefit Scale) and may represent a potential therapeutic option, but have little actual clinical relevance.\"\n",
      "BRAF and EGFR for Esophageal Carcinoma\n",
      "National Comprehensive Cancer network (NCCN)âs Biomarkers Compendium (2019) does not list esophageal and esophagogastric junction cancers to be associated with EGFR or BRAF.\n",
      "CA 19-9 for Ovarian Cyst\n",
      "An UpToDate review on \"Serum biomarkers for evaluation of an adnexal mass for epithelial carcinoma of the ovary, fallopian tube, or peritoneum\" (Ueland and Li, 2019) states that \"Cancer antigen 19-9 -- Cancer antigen 19-9 (CA 19-9) is a mucin protein that may be elevated in ovarian cancer, but is used sparingly in ovarian cancer management.  CA 19-9 is used primarily to monitor disease response to therapy or detect cancer recurrence in patients with a documented gastric cancer, pancreatic cancer, gallbladder cancer, cholangiocarcinoma, or adenocarcinoma of the ampulla of Vater\".  It does not mention the use of CA 19-9 for ovarian cyst.\n",
      "Furthermore, National Comprehensive Cancer Networkâs Biomarkers Compendium (2019) does not list ovarian cyst as a cancer/disease associated with CA19-9 expression.\n",
      "CA 19-9 for Prediction of Prognosis or Treatment Effect in Bladder (Urothelial) Cancer\n",
      "Yaegashi and colleagues (2019) examined if baseline serum CA 19-9 predicts prognosis or treatment effect in patients with advanced urothelial carcinoma (UC).  These researchers retrospectively analyzed data of patients diagnosed with locally advanced or metastatic or recurrent UC between April 2008 and November 2014; CA19-9 was determined using enzyme-linked immunosorbent assay (ELISA) and the relationship between CA19-9 and prognosis were analyzed.  Of 40 patients, 7 with CA19-9 less than or equal to 2 U/ml who were suspected of having Lewis A-negative blood type and 2 patients with advanced metastatic liver disease were excluded.  UC-specific survival in metastatic disease significantly correlated with prognosis (p = 0.018); OS in patients with high serum CA19-9 demonstrated a significantly better prognosis than in those with low concentrations (log-rank test, p = 0.032).  The authors concluded that high baseline serum CA19-9 may predict a good prognosis in patients with advanced UC.\n",
      "Furthermore, National Comprehensive Cancer Networkâs clinical practice guideline on \"Bladder cancer\" (Version 1.2019) does not mention serum CA19-9 as a management tool.\n",
      "Colvera (Circulating Tumor DNA / ctDNA)\n",
      "National Comprehensive Cancer Networkâs clinical practice guidelines on \"Colon cancer\" (Version 4.2018) and \"Rectal cancer\" (Version 3.2018) do not mention circulating tumor DNA (ctDNA) as a management tool.\n",
      "Murray and colleagues (2018) stated that methylation in IKZF1 and BCAT1 are common events in colorectal cancer (CRC).  They are often detected in blood as circulating tumor DNA (ctDNA) at diagnosis and disappear after surgery in most CRC patients.  In a prospective study, these researchers examined the relationship between detection of these markers following surgery and risk for residual disease and for recurrence.  ctDNA status with methylated BCAT1 and IKZF1 was determined within 12 months of surgical resection of CRC; and was related to presence of or risk for residual disease (margins involved, metastases present or nature of node involvement), and to RFS.  Blood was collected from 172 CRC patients after surgery and 28 (16 %) were ctDNA positive.  Recurrence was diagnosed in 23 of the 138 with clinical follow-up after surgery (median follow-up of 23.3 months, inter-quartile range [IQR] 14.3 to 29.5).  Multi-variate modeling indicated that features suggestive of residual disease were an independent predictor of post-surgery ctDNA status: cases with any of 3 features (close resection margins, apical node involved, or distant metastases) were 5.3 times (95 % CI: 1.5 to 18.4, p = 0.008) more likely to be ctDNA positive.  Multi-variate analysis showed that post-surgery ctDNA positivity was independently associated with an increased risk of recurrence (HR 3.8, 1.5 to 9.5, p = 0.004).  The authors concluded that CRC cases positive for methylated ctDNA after surgery are at increased risk of residual disease and subsequently recurrence.  This could have implications for guiding recommendations for adjuvant therapy and surveillance strategies.  Moreover, these researchers stated that randomized studies are now indicated to examine if monitoring cases with these biomarkers would lead to survival benefit.\n",
      "Musher and associates (2020) noted that most recurrences of early-stage CRC detected with current surveillance measures are widespread and incurable; ctDNA may facilitate earlier diagnosis of recurrent CRC and improve cancer-related outcomes.  Plasma from patients undergoing standard surveillance after definitive treatment for stage II/III CRC was assayed with Colvera and CEA at a single time-point.  Results were correlated with radiographic imaging.  Assay performance, including sensitivity and specificity for recurrence, were compared.  Impact of potentially confounding variables was also examined.  A total of 322 patients were included in the final analysis, and 27 recurrences were documented over a median follow-up period of 15 months.  Sensitivity for recurrence was 63 % [CI: 42.4 to 80.6] and 48 % (CI: 28.7 to 68.1) for Colvera and CEA (greater than 5 ng/ml), respectively (p = 0.046), while specificity was 91.5 % (CI: 87.7 to 94.4) and 96.3 % (CI: 93.4 to 98.1), respectively (p = 0.016).  Smoking and age were independent predictors of CEA, but not Colvera positivity.  The authors concluded that Colvera was more sensitive but less specific than CEA in detecting recurrent CRC.  Short median follow-up may have been responsible for apparent false positives in Colvera.  Moreover, these researchers stated that studies with serial sampling and longer follow-up are needed to examine if earlier detection of CRC recurrence would translate into clinical benefit.\n",
      "Symonds and co-workers (2020) stated that the sensitive detection of recurrent CRC by the measurement of ctDNA might improve the chance of a cure.  These researchers compared a quantitative methylated ctDNA test with CEA in the setting of surveillance for recurrence.  Blood samples collected either during surveillance or within 12 months of the confirmation of recurrence were assayed for ctDNA (methylated branched-chain amino acid transaminase 1 [BCAT1]/Ikaros family zinc-finger 1 protein [IKZF1]) and CEA.  The optimal ctDNA threshold was determined by ROC analysis, and the test performance for the detection of recurrence was compared with CEA (5 ng/ml threshold).  The study cohort comprised 144 eligible patients and included 50 recurrence events.  The sensitivity of the methylated ctDNA test for recurrence was 66.0 % (95 % CI: 57.1 % to 69.3 %), which was significantly higher than the sensitivity of CEA (31.9 %; 95 % CI: 22.8 % to 36.6 %; p < 0.001).  The sensitivity for resectable recurrence (n = 20) was also higher (ctDNA, 60.0 %; CEA, 20.0 %; p = 0.01).  The specificity did not differ between the tests (ctDNA, 97.9 %; 95 % CI: 93.2 % to 99.6 %; CEA, 96.4 %; 95 % CI: 91.4 % to 99.0 %).  When adjustments were made for other predictors of the presence of recurrence, a positive ctDNA test was an independent predictor (OR, 155.7; 95 % CI: 17.9 to 1360.6; p < .001), whereas CEA was not (OR, 2.5; 95 % CI: 0.3 to 20.6; p = 0.407).  The authors concluded that the quantitative ctDNA test showed superior sensitivity in comparison with CEA without a difference in the specificity for detecting recurrent CRC.  Moreover, these researchers stated that longitudinal studies are needed to further examine the utility (specifically the survival benefit) of methylated BCAT1/IKZF1 ctDNA in the surveillance of patients with CRC.\n",
      "Galectin-3 for Breast Cancer\n",
      "Zhang et al (2014) stated that galectin-3 has a relatively high level of expression in triple-negative breast cancers and is a potential marker for this disease.  However, the clinical and prognostic implications of galectin-3 expression in breast cancer remain unclear.  These researchers examined mastectomy specimens from 1,086 breast cancer cases and matching, adjacent non-cancerous tissues using immunohistochemistry.  Overall, triple-negative breast cancers expressed galectin-3 more strongly than did other breast cancers types (63.59 % versus 21.36 %, p = 0.001).  Galectin-3 expression was not found to be an independent prognostic factor for breast cancer by Cox regression analysis, but was associated with chemotherapeutic resistance.  Apoptosis was only weakly induced by arsenic trioxide (ATO) treatment in galectin-3-positive breast cancer cells (MDA-MB-231 and MCF-7), although ATO treatment up-regulated galectin-3 expression.  Knock-down of galectin-3 in MDA-MB-231 cells sensitized them to killing by ATO.  The authors concluded that these findings supported a possible role for galectin-3 as a marker for triple-negative breast cancer progression and as a therapeutic target in combination with ATO treatment, although the mechanisms that underlie this synergy require further investigation.\n",
      "White et al (2015) noted that to metastasize, tumor cells often need to migrate through a layer of collagen-containing scar tissue which encapsulates the tumor.  A key component of scar tissue and fibrosing diseases is the monocyte-derived fibrocyte, a collagen-secreting pro-fibrotic cell.  To test the hypothesis that invasive tumor cells may block the formation of the fibrous sheath, these researchers examined if tumor cells secrete factors that inhibit monocyte-derived fibrocyte differentiation.  They found that the human metastatic breast cancer cell line MDA-MB-231 secretes activity that inhibits human monocyte-derived fibrocyte differentiation, whereas less aggressive breast cancer cell lines secrete less of this activity.  Purification indicated that galectin-3 binding protein (LGALS3BP) is the active factor.  Recombinant LGALS3BP inhibits monocyte-derived fibrocyte differentiation, and immuno-depletion of LGALS3BP from MDA-MB 231 conditioned media removes the monocyte-derived fibrocyte differentiation-inhibiting activity.  LGALS3BP inhibits the differentiation of monocyte-derived fibrocytes from wild-type mouse spleen cells, but not from SIGN-R1(-/-) mouse spleen cells, suggesting that CD209/SIGN-R1 is required for the LGALS3BP effect.  These investigators found that galectin-3 and galectin-1, binding partners of LGALS3BP, potentiate monocyte-derived fibrocyte differentiation.  In breast cancer biopsies, increased levels of tumor cell-associated LGALS3BP were observed in regions of the tumor that were invading the surrounding stroma.  The authors concluded that these findings suggested LGALS3BP and galectin-3 as new targets to treat metastatic cancer and fibrosing diseases.\n",
      "Bulten et al (2015) stated that it remains challenging to identify patients at risk of anthracycline-induced cardiotoxicity.  To better understand the different risk-stratifying approaches, these researchers evaluated (123)I-metaiodobenzylguanidine ((123)I-mIBG) scintigraphy and its interrelationship with conventional echocardiography, 2D strain imaging and several biomarkers.  They performed (123)I-mIBG scintigraphy, conventional and strain echocardiography and biomarker (NT-proBNP, TNF-Î±, galectin-3, IL-6, troponin I, ST-2 and sFlt-1) assessment in 59 breast cancer survivors 1 year after anthracycline treatment.  Inter-observer and inter-method variability was calculated on planar and SPECT (123)I-mIBG scintigraphy, using the heart/mediastinum (H/M) ratio and wash-out (WO).  Pearson's r and multi-variate analyses were performed to identify correlations and independent predictors of (123)I-mIBG scintigraphy results.  Delayed planar anterior whole-heart ROI (WH) H/M ratios and WO were the most robust (123)I-mIBG parameters.  Significant correlations were observed between (123)I-mIBG parameters and several conventional echo parameters, global longitudinal and radial strain (GLS and GRS) and galectin-3.  The highest Pearson's r was observed between delayed H/M ratio and GRS (Pearson's r 0.36, p = 0.01).  Multi-variate analysis showed that GRS was the only independent predictor of the delayed WH H/M ratio (p = 0.023).  The authors concluded that the delayed planar H/M ratio was the most robust (123)I-mIBG parameter.  It correlated with several conventional echocardiographic parameters, GLS, GRS and galectin-3.  Of these, only GRS predicts the H/M ratio.\n",
      "Ilmer et al (2016) noted that galectin-3 (Gal3) plays diverse roles in cancer initiation, progression, and drug resistance depending on tumor type characteristics that are also associated with cancer stem cells (CSCs).  Recurrence of breast carcinomas may be attributed to the presence of breast CSCs (BCSCs).  BCSCs exist in mesenchymal-like or epithelial-like states and the transition between these states endows BCSCs with the capacity for tumor progression.  The discovery of a feedback loop with galectins during epithelial-to-mesenchymal transition (EMT) prompted these researchers to examine its role in breast cancer stemness.  To elucidate the role of Gal3 in BCSCs, these investigators performed various in-vitro and in-vivo studies such as sphere-formation assays, Western blotting, flow cytometric apoptosis assays, and limited dilution xenotransplant models.  Histological staining for Gal3 in tissue microarrays of breast cancer patients was performed to analyze the relationship of clinical outcome and Gal3 expression.  These researchers showed in a cohort of 87 node-positive breast cancer patients treated with doxorubicin-based chemotherapy that low Gal3 was associated with increased lympho-vascular invasion and reduced overall survival (OS).  Analysis of in-vitro BCSC models demonstrated that Gal3 knock-down by small hairpin RNA (shRNA) interference in epithelial-like mammary spheres led to EMT, increased sphere-formation ability, drug-resistance, and heightened aldefluor activity.  Furthermore, Gal3-negative BCSCs were associated with enhanced tumorigenicity in orthotopic mouse models.  The authors concluded that in at least some breast cancers, loss of Gal3 might be associated with EMT and cancer stemness-associated traits, predicts poor response to chemotherapy, and poor prognosis.  Moreover, they stated that \"It will be important to evaluate the expression of Gal3 in circulating tumor cells, and its role as a predictive biomarker in determining response to therapy and DSS (disease-specific survival) and OS in prospective clinical trials in a variety of breast cancer patients\"\n",
      "De Iuliis et al (2016) stated that it is important to identify novel plasmatic biomarkers that can contribute to assessing the prognosis and outcome of breast cancer patients.  Neuregulin-1 (NRG1) and Gal3 are proteins that are involved in breast cancer development and patient survival; therefore, these researchers  examined if the serum concentration of these 2 proteins can be correlated to breast cancer progression.  Plasmatic NRG1 and Gal3 were evaluated in 25 healthy controls and 50 breast cancer patients at baseline and at 3 and 6 months after treatment with anthracyclines and taxanes, with or without trastuzumab.  NRG1 and Gal3 were significantly more elevated in cancer patients than in healthy controls; furthermore, NRG1 and Gal3 were significantly increased after chemotherapy and were predictive of mortality at 1 year.  The authors concluded that circulating NRG1 and Gal3 can be additional biomarkers indicative of prognosis and outcomes for breast cancer patients.  This was a relatively small study (n = 500 with only 1-year follow-up.  These findings need to be validated by well-designed studies.\n",
      "Furthermore, National Comprehensive Cancer Networkâs clinical practice guideline on \"Breast cancer\" (Version 4.2018) does not mention galectin-3 as a biomarker.\n",
      "Galectin-3 for Ovarian Cancer\n",
      "National Comprehensive Cancer Networkâs clinical practice guideline on \"Ovarian cancer\" (Version 2.2018) does not mention galectin-3 as a diagnostic tool.\n",
      "Galectin-3 for Pancreatic Cancer\n",
      "Shimamura et al (2002) stated that galectin-3, a member of the beta-galactoside-binding lectin family, has multiple biological functions including cell-cell interactions and cell-extracellular matrix adhesion, cellular proliferation, cellular differentiation, and apoptosis. These researchers determined the relationship of galectin-3 expression to clinicopathological findings and patient prognosis in ductal adenocarcinoma of the pancreas. They examined galectin-3 expression in 104 surgically resected pancreatic ductal adenocarcinoma cases with stages I through IV using immunohistochemistry and investigated the relationship of it to overall survival (OS). Patients were divided into 2 groups: a low expression group, where less than 60 % of tumor cells were positive; and a high expression group, where greater than or equal to 60 % of tumor cells were positive. Cases in the low expression group had a significant tendency to be at later stages, to have distant metastasis, and to have less differentiated tumors, compared with cases in the high expression group (p = 0.001 for stage, p = 0.001 for metastasis, and p = 0.006 for differentiation). Post-operative OS was worse in the low galectin-3 expression group than in the high galectin-3 expression group (p = 0.004). Multi-variate analysis showed that the risk ratio of prognosis was 2.06 among patients in the low galectin-3 expression group compared with the high galectin-3 expression group (p = 0.006). The authors concluded that decreased expression of galectin-3 was associated with advanced stage, tumor de-differentiation, and metastasis in ductal adenocarcinoma of the pancreas; galectin-3 expression might be a useful prognostic marker for survival in ductal adenocarcinoma of the pancreas.\n",
      "Gaida et al (2012) noted that galectin-3 influences neoangiogenesis, tumor cell adhesion, and tumor-immune-escape mechanisms. Hence, the expression of galectin-3 in pancreatic ductal adenocarcinoma (PDAC) was evaluated. Galectin-3 expression in PDAC cell lines was proven by the presence of intracellular protein and by release into the supernatant. Furthermore, galectin-3 was found in the majority of human tissue samples. Serum concentrations of galectin-3 in PDAC patients did not differ significantly from healthy donors and did not correlate with established tumor markers. The authors concluded that galectin-3 was expressed in PDAC tissues suggesting a role in tumor development; however, no relationship between expression and clinical findings could be established.\n",
      "Jiang et al (2014) analyzed galectin-3 immunohistochemical expression in fine-needle aspiration (FNA) cell blocks of PDAC, pancreatic neuroendocrine neoplasms (PNEN), gastro-intestinal stromal tumors (GIST) and non-tumor pancreatic tissue. In parallel, galectin-3 and PTEN levels were evaluated in a tumor tissue microarray (TMA); 44 of 46 PDAC FNA and 32 of 33 PDAC TMA demonstrated tumor-specific galectin-3 positivity. In contrast, galectin-3 was not detected in PNEN and GIST. Total loss of PTEN was displayed by 26 of 33 PDAC, while non-neoplastic tissues all retained PTEN expression. The authors concluded that galectin-3 could be a valuable marker to help diagnose PDAC and rule out PNEN and GIST. In addition, PTEN positivity strongly argued against a diagnosis of PDAC. These data also advocated their potential diagnostic roles in the work up of challenging cytologic cases requiring ancillary test confirmation.\n",
      "Coppin et al (2016) stated that CA 19-9 is the gold standard biomarker of pancreatic adenocarcinoma despite several weaknesses in particular a high rate of false positives or negatives. CA-125 corresponding to MUC16 and galectin-3, a lectin able to interact with mucin-associated carbohydrates, are tumor-associated proteins. These researchers examined if combined measurement of CA 19-9, galectin-3 and CA-125 may help to better discriminate pancreatic adenocarcinoma versus non-malignant pancreatic diseases. They evaluated by immunohistochemistry the expression of MUC4, MUC16 (CA-125) and galectin-3 in 31 pancreatic adenocarcinomas. These investigators measured CA 19-9, CA-125 and Gal-3 in the serum from patients with pancreatic benign diseases (n = 58) or adenocarcinoma (n = 44). Clinical performance of the 3 biomarkers for cancer diagnosis and prognosis was analyzed. By immunohistochemistry, MUC16 and Gal-3 were expressed in 74 % and 84 % of adenocarcinomas versus 0 % and 3.2 % in peri-tumoral regions, respectively. At the serum level, CA 19-9 and CA125 were significantly higher in patients with pancreatic adenocarcinoma whereas Gal-3 levels did not differ. The performance of CA 19-9 for cancer detection was higher than those of CA-125 or Gal-3 by ROC analysis. However, CA-125 offered the highest specificity for malignancy (81 %) because of an absence of false-positives among type 2 diabetic patients. Cancer deaths assessed 6 or 12 months after diagnosis varied according to the initial CA-125 level (p < 0.006). The authors concluded that Gal-3 is not an interesting biomarker for pancreatic adenocarcinoma detection. CA 19-9 alone exhibits the best performance but measuring CA-125 provides complementary information in terms of diagnosis and prognosis.\n",
      "Nigjeh et al (2017) noted that PDAC is a lethal disease characterized by its late diagnosis, poor prognosis and rapid development of drug resistance. Using the data-independent acquisition (DIA) technique, the authors applied a spectral library-based proteomic approach to analyze N-glycosylated peptides in human plasma, in the context of pancreatic cancer study. The authors extended the application of DIA to the quantification of N-glycosylated peptides enriched from plasma specimens from a clinically well-defined cohort that consists of patients with early stage PDAC, chronic pancreatitis and healthy subjects. The analytical platform was evaluated in light of its robustness for quantitative analysis of large-scale clinical specimens. The analysis indicated that the level of N-glycosylated peptides derived from galectin-3 binding proteins (LGALS3BP) were frequently elevated in plasma from PDAC patients, concurrent with the altered N-glycosylation of LGALS3BP observed in the tumor tissue. The authors concluded that the glycosylation form of LGALS3BP influenced its function in the galectin network, which is profoundly involved in cancer progression, immune response and drug resistance. As one of the major binding ligands of galectin network, discovery of site specific N-glycosylation changes of LGALS3BP in association of PDAC may provide useful clues to facilitate cancer detection or phenotype stratification.\n",
      "Furthermore, National Comprehensive Cancer Networkâs clinical practice guideline on \"Pancreatic adenocarcinoma\" (Version 1.2019) does not mention galectin-3.\n",
      "KRAS, NRAS and BRAF for Metastatic Colon Cancer\n",
      "National Comprehensive Cancer Networkâs clinical practice guideline on \"Colon cancer\" (Version 4.2018) recommends RAS and BRAF (individually or part of next-generation sequencing [NGS] panel) for patients with metastatic colon cancer.\n",
      "NF1, RET, and SDHB for Ovarian Cancer\n",
      "Norris and (2018) stated that neurofibromatosis type 1 (NF1) is caused by mutations in the NF1 gene encoding neurofibromin, which negatively regulates Ras signaling.  NF1 patients have an increased risk of developing early onset breast cancer, however, the association between NF1 and high grade serous ovarian cancer (HGSOC) is unclear.  Since most NF1-related tumors exhibit early bi-allelic inactivation of NF1, the authors evaluated the evolution of genetic alterations in HGSOC in an NF1 patient.  Somatic variation analysis of WES of tumor samples from both ovaries and a peritoneal metastasis showed a clonal lineage originating from an ancestral clone within the left adnexa, which exhibited copy number (CN) loss of heterozygosity (LOH) in the region of chromosome 17 containing TP53, NF1, and BRCA1 and mutation of the other TP53 allele.  This event led to bi-allelic inactivation of NF1 and TP53 and LOH for the BRCA1 germline mutation.  Subsequent CN alterations were found in the dominant tumor clone in the left ovary and nearly 100 % of tumor at other sites.  Neurofibromin modeling studies suggested that the germline NF1 mutation could potentially alter protein function.  The authors concluded that these findings demonstrated early, bi-allelic inactivation of neurofibromin in HGSOC and highlighted the potential of targeting RAS signaling in NF1 patient.\n",
      "The NCCN Biomarkers Compendium (2019) does not list ovarian cancer to be associated with RET and SDHB.  While the Compendium lists \"Genetic/Familial High-Risk Assessment: Breast and Ovarian\" as disease associated with NF1 mutation, it only notes \"Increased risk of breast cancer\" under the \"NCCN Recommendation: Clinical Decision\".\n",
      "BTK (Brution's Typrosine Kinase) and PLCG2 (Phospholipase C Gamma 2)\n",
      "The National Comprehensive Cancer Network (NCCN) guidelines for \"Chronic lymphocytic/smalllymphocytic lymphoma\" (v.2.2019) states that testing for BTK and PLCG2 mutations may be useful in patients receiving ibrutinib and suspected of having progression; however, BTK and PLCG2 mutation status alone is not an indication to change treatment. Testing for mutations as screening for resistance is not currently recommended.\n",
      "Lampson and Brown (2018) state that BTK and PLCG2 mutations are found in approximately 80% of CLL patients with acquired resistance to ibrutinib; however, it remains unclear if these mutations are solely associated with disease relapse or is the direct cause. The authors reviewed the properties of both CLL and ibrutinib that complicate attempts to definitively conclude whether BTK/PLCG2 mutations are passengers or drivers of ibrutinib-resistant disease. The authors concluded that while BTK/PLCG2 mutations have characteristics suggesting that these mutations can drive ibrutinib resistance, a definitive answer remains formally unproven until specific inhibition of such mutations is shown to cause regression of ibrutinib-resistant CLL. Furthermore, data suggest that alternative mechanisms of resistance do exist in some patients. The authors further conclude that multiple unanswered questions remain regarding resistance to ibrutinib in CLL, requiring a need for further exploration. Testing the efficacy of drugs that can inhibit the BTK C481S mutation in patients with ibrutinib-resistant disease is warranted.\n",
      "EZH2 (Ehancer of Zeste 2 Polycomb Repressive Complex 2 Subunit)\n",
      "The National Comprehensive Cancer Center (NCCN) Biomarkers Compendium (2019) for \"EZH2\" includes the following category 2A recommendations:\n",
      "\n",
      "Myelodysplastic syndromes (MDS) for somatic mutation of EZH2 for cytopenia(s), suspect myelodysplasia. For initial evaluation, consider genetic testing for somatic mutations (i.e., acquired mutations) in genes associated with MDS.\n",
      "Myeloproliferative neoplasms (MPN) - additional molecular testing using multi-gene NGS panel should be considered to evaluate for higher-risk mutations associated with disease progression in patients with primary myelofibrosis (PMF). Next-generation sequencing (NGS) remains a research tool in many situations. However, it may be useful to establish clonality in selected circumstances (e.g., \"triple negative\" non-mutated JAK2, MPL, and CALR. Identification of \"higher-risk\" mutations may be helpful in the decision-making regarding allogeneic HCT for patients with PMF.\n",
      "\n",
      "The NCCN guidelines on \"B-cell lymphomas\" (v.1.2019) does not provide a recommendation for EZH2 testing. Thus, NCCN does not provide a recommendation for diffuse large B-cell lymphoma (DLBC).\n",
      "Intlekofer et al (2018) state that there is an unmet need to develop genomic biomarker-driven therapeutics to improve outcomes for patients with diffuse large B-cell lymphoma (DLBCL), which currently has a relapse rate of over 30%. The authors sought to define the genomic landscape of DLBCL by using formalin-fixed paraffin-embedded (FFPE) biopsy specimens in order to help underline genomic alterations that characterize DLBCL. Archived FFPE biopsy specimens from 1989 to 2012 were reviewed on 198 patients with DLBCL. Samples were sequenced using the FoundationOne-Heme platform that uses DNA sequencing to interrogate the entire coding sequence of 406 genes, selected introns of 31 genes involved in rearrangements, and utilizes RNA sequencing to interrogate 265 genes known to be somatically altered in human hematologic malignancies. Of 219 FFPE DLBCL samples attempted, 214 were successfully sequenced. The median number of genomic alterations (Gas) per case was 6, with 97% of patients harboring at least one alteration. The most commonly identified single nucleotide variants (SNVs) were in KMT2D (MLL2; 31%, nâ=â62), TP53 (24%, nâ=â48), MYD88 (18%, nâ=â36), CREBBP (18%, nâ=â35), and B2M (Beta-2-microglobulin; 17%; nâ=â33). A cluster of BCL2trans and KMT2Dmut corresponded with a GCB subtype and with high rates of TP53mut, EZH2mut, and TNFRSF14mut (pâ=â0.002). Of note, the largest cluster of 80 patients (40%) did not have a distinct genomic signature. The authors further observed an enrichment in MYD88mut, ETV6mut, and PRDM1mut among non-GCB and EZH2mut among GCB tumors; however, these did not remain significant after correction for FDR. In 41% (nâ=â81) there was a GA targeted by a non-FDA-approved drug with compelling clinical evidence either in DLBCL (level 3A; 33%, nâ=â66; mostly histone deacetylase and EZH2 inhibitors in CREBBPmut, EP300mut, and EZH2mut) or in another indication (level 3B; 8%, nâ=â15). The authors note that prior studies reported EZH2 mutations frequencies as high as 24%, whereas they found EZH2mut in 11% of their cohort, a difference that would have major implications for designing a trial with sequencing-based selection of patients for treatment with EZH2 inhibitors. The authors concluded that despite an accumulating body of research into the genomic landscape of DLBCL, very few GAs have been found to be associated with treatment refractoriness or disease relapse. The authors report that their study confirms prior associations between TP53mut and survival. Though marginally significant, CDKN2A/Bdel and B2Mmut were also found to be associated with shorter OS. As larger sequencing cohorts are assembled, future studies will continue to refine the association between GAs and treatment outcomes.\n",
      "MYD88 (Myeloid Differentiation Primary Response 88)\n",
      "NCCN Biomarkers Compendium (2019) for \"MYD88\" includes the following category 2A recommendations: \n",
      "\n",
      "Gastric MALT lymphoma - Useful under certain circumstances, such as molecular analysis to detect antigen receptor gene rearrangements; MYD88 mutation status to differentiate Waldenstromâs macroglobinemia (WM) versus marginal zone lymphomas (MZL) if plasmacytic differentiation present\n",
      "Nodal marginal zone lymphoma and nongastric MALT lymphoma - Molecular analysis to detect antigen receptor gene rearrangements; MYD88 mutation status to differentiate WM versus MZL if plasmacytic differentiation present; PCR for t(11;18)\n",
      "Splenic marginal zone lymphoma - Useful under certain circumstances, such as molecular analysis to detect antigen receptor gene rearrangements; MYD88 mutation status to differentiate WM versus MZL if plasmacytic differentiation present; BRAF mutation status to differentiate MZL from HCL by IHC or sequencing; PCR for t(11;18).\n",
      "\n",
      "TERT (Telomerase Reverse Transcriptase)\n",
      "The National Comprehensive Cancer Network (NCCN) Biomarkers Compendium (2019) for \"TERT\" includes the following category 2A recommendations:\n",
      "\n",
      "Central nervous system cancers â adult low-grad (WHO Grade II) infiltrative supratentorial, astrocystoma/oligodendroglioma, and anaplastic gliomas/glioblastoma for which TERT mutation testing is recommended but not required for gliomas in the following clinical setting: MRI compatible with infiltrative glioma; pre-adjuvant therapy, surgery or biopsy; or recurrent or progressive low-grade disease, surgery or biopsy.\n",
      "Myelodysplastic syndromes (MDS) â cultured skin fibroblasts specimen type for cytopenia(s) and suspect myelodysplasia. For initial evaluation, consider additional molecular and genetic testing for hereditary hematologic malignancy predisposition in a subset of patients, particularly in younger patients. Shortened telomere length has been associated with diseases of bone marrow failure, including inherited disorders such as DC, particularly in the presence of mutations in the DKC1, TERT, or TERC genes that encode for components of the telomere complex.\n",
      "\n",
      "NCCN guidelines on \"Central nervous system cancers\" (version 2.2018) states that the diagnostic value of TERT mutations are almost invariably present in 1p/19q codeleted oligodendroglioma, and are found in most glioblastomas. TERT mutation, in combination with IDH mutation and 1p/19q codeletion, is characteristic of oligodendroglioma. Absence of TERT mutation, coupled with IDH mutation, designates astrocytoma. For prognostic value, in the absence of an IDH mutation, TERT mutations in diffusely infiltrative gliomas are associated with reduced overall survival compared to gliomas lacking TERT mutations. \n",
      "The NCCN guidelines on \"Thyroid carcinoma\" (v.3.2018) state that some studies have linked the BRAF V600E mutation to poor prognosis, especially when occurring with TERT promoter mutation. The NCCN panel does not make a formal recommendation for TERT mutation testing for thyroid carcinoma. \n",
      "Ibrahimpasis et al. (2019) state that poorly differentiated thyroid cancer (PDTC) is rare and accounts for most fatalities from non-anaplastic follicular cell-derived thyroid cancer. The authors note that there are limited studies on PDTC due to its rarity; however, in light of the evolution of ultra-deep next-generation sequencing (NGS) technologies and through correlation of clinico-pathological and genomic characteristics of PDTC, an improved understanding of the biology of PDTC has been facilitated. Thus, the authors reviewed the diagnostic criteria, clinico-pathological characteristics, management and outcomes in PDTC, as well as genomic drivers in PDTC reported in recent NGS studies, as well as future prospects in improving outcomes in PDTC patients. The authors reviewed various gene mutations drivers in PDTC; however, specifically regarding TERT, the authors state that TERT promoter mutations represent the most common alteration in PDTC (40%). The authors conclude that \"new insights into the clinico-pathological and molecular characteristics of PDTC, together with further advancement in ultra deep sequencing technologies, will be conducive in narrowing the focus in order to develop novel targeted therapies and improve the outcomes in PDTC patients.\"\n",
      "Bournaud et al. (2019) state that TERT promoter mutations are associated with adverse clinico-pathological characteristics in thyroid carcinomas and considered as a major indicator of poor outcomes. Nevertheless, most studies have pooled heterogeneous types of thyroid carcinomas and have been conducted retrospectively. Thus, the authors conducted a prospective, observational study to evaluate the association between TERT promoter mutations and recurrence in a series of 173 intermediate- to-high-risk patients with thyroid carcinoma. Patients were tested for TERT promoter, BRAF, and RAS mutations of their primary tumor. The prevalence of TERT promoter mutations was 20.2% (35/173) in the total population. The authors found that it was significantly higher in tumors harboring aggressive histological features (poorly differentiated carcinoma, tall cell variant of papillary cancer or widely invasive follicular cancer) than in non-aggressive tumors: 32.7% (16/49) versus 15.3% (19/124; p = 0.020). TERT promoter mutations were also strongly associated with age â¥45 years (p = 0.005), pT4 stage (p = 0.015), metastatic disease (p = 0.014), and extra-thyroidal extension (p = 0.002). TERT promoter mutations were associated with poor outcomes in the total population (p < 0.001) but not in the subgroup of non-metastatic patients (p = 0.051). However, they were associated with a worse outcome in patients both free of metastases and devoid of aggressive histological features. Neither BRAF nor RAS mutations were associated with event-free survival in non-metastatic patients. The authors concluded that TERT promoter mutations may help to better define the prognosis of localized thyroid cancer patients without aggressive histology.\n",
      "Liquid Biopsy for Lung Cancer\n",
      "Lindeman and colleagues (2018) noted that in 2013, an evidence-based guideline was published by the College of American Pathologists (CAP), the International Association for the Study of Lung Cancer (IASLC), and the Association for Molecular Pathology (AMP) to set standards for the molecular analysis of lung cancers to guide treatment decisions with targeted inhibitors.  The authors stated that the 2013 guideline was largely re-affirmed with updated recommendations to allow testing of cytology samples, require improved assay sensitivity, and recommend against the use of immunohistochemistry for EGFR testing.  Principal new recommendations included ROS1 testing for all adenocarcinoma patients; the inclusion of additional genes (ERBB2, MET, BRAF, KRAS, and RET) for laboratories that perform NGS panels; immunohistochemistry as an alternative FISH for ALK and/or ROS1 testing; use of 5 % sensitivity assays for EGFR T790M mutations in patients with secondary resistance to EGFR inhibitors; and the use of cell-free DNA to \"rule in\" targetable mutations when tissue is limited or hard to obtain.  In particular, the authors stated that there is currently insufficient evidence to support the use of circulating plasma cfDNA molecular methods for establishing a primary diagnosis of lung adenocarcinoma.  However, in some clinical settings in which tissue is limited and/or insufficient for molecular testing, physicians may use a cfDNA assay to identify EGFR mutations.  Thus, physicians may use plasma cfDNA methods to identify EGFR T790M mutations in lung adenocarcinoma patients with progression or secondary clinical resistance to EGFR-targeted TKIs; testing of the tumor sample is recommended if the plasma result is negative.\n",
      "Kalemkerian and associates (2018) stated that the CAP, the IASLC, and the AMP recently updated their recommendations for molecular testing for the selection of patients with lung cancer for treatment with targeted TKIs.  ASCO has a policy and set of procedures for endorsing clinical practice guidelines that have been developed by other professional organizations.  The molecular testing guideline was reviewed for developmental rigor by methodologists.  Then an ASCO Expert Panel reviewed the content and the recommendations.  The ASCO Expert Panel determined that the recommendations from the CAP/IASLC/AMP molecular testing guideline are clear, thorough, and based upon the most relevant scientific evidence.  ASCO endorsed the guideline with minor modifications.  This update clarified that any sample with adequate cellularity and preservation may be tested and that analytical methods must be able to detect mutation in a sample with as little as 20 % cancer cells.  It strongly recommended against evaluating EGFR expression by immunohistochemistry for selection of patients for EGFR-targeted therapy.  New for 2018 were recommendations for stand-alone ROS1 testing with additional confirmation testing in all patients with advanced lung adenocarcinoma, and RET, ERBB2 (HER2), KRAS, and MET testing as part of larger panels.  ASCO also recommended stand-alone BRAF testing in patients with advanced lung adenocarcinoma.  Recommendations were also provided for testing methods for lung cancers that have a non-adenocarcinoma non-small-cell component, for patients with targetable mutations who have relapsed on targeted therapy, and for testing the presence of circulating cell-free DNA.\n",
      "An ASCO and College of American Pathologists (CAP) joint review of circulating tumor DNA analysis in patients with cancer (Merker, et al., 2018) concluded: \"Some ctDNA assays have demonstrated clinical validity and utility with certain types of advanced cancer; however, there is insufficient evidence of clinical validity and utility for the majority of ctDNA assays in advanced cancer. Evidence shows discordance between the results of ctDNA assays and genotyping tumor specimens and supports tumor tissue genotyping to confirm undetected results from ctDNA tests. There is no evidence of clinical utility and little evidence of clinical validity of ctDNA assays in early-stage cancer, treatment monitoring, or residual disease detection. There is no evidence of clinical validity and clinical utility to suggest that ctDNA assays are useful for cancer screening, outside of a clinical trial.\"\n",
      "BCR/ABL Fluorescent In Situ Hybridization (FISH)\n",
      "The National Comprehensive Cancer network (NCCN) Myeloproliferative Neoplasms guidelines (v.2.2019), state that FISH or a multiplex reverse transcriptase polymerase chain reaction (RT-PCR) on peripheral blood to detect BCR-ABL1 transcripts and exclude the diagnosis of CML is especially recommended for patients with left-shifted leukocytosis and/or thrombocytosis with basophilia.\n",
      "Immunoassay Using Magnetic Nanosensor for Diagnosis of Lung Cancer\n",
      "Gao and colleagues (2017) developed a real-time assay for highly sensitive, label-free, multi-plexed electrical detection of lung cancer biomarkers by using silicon nanowire field-effect (SiNW-FET) devices.  Highly responsive SiNW arrays were fabricated using a CMOS-compatible anisotropic self-stop etching technique with mass reproducibility and low-cost character.  The SiNW nanosensor was integrated with PDMS microfluidic device, which allows rapid analyte delivery, makes the analysis to be conducted using exceedingly small samples and enables potential multi-plexed detection.  The nanowire arrays allowed highly selective and sensitive multi-plexed detection of microRNA (miRNA)-126 and CEA.  Due to high surface-to-volume ratio that the nanowire dimensions confer, the detection floor of single molecule was achieved.  The potential utility in identifying clinical samples for early diagnosis of cancer was demonstrated by analyzing biomarkers in clinical related samples.  The developed nanosensor with capability for multi-plexed real-time monitoring of biomarkers with high sensitivity and selectivity in clinically relevant samples is highly attractive for diagnosis and treatment of cancer and other diseases.\n",
      "Wang and associates (2017) noted that DNA methylation is an important epigenetic modification in human genomes.  These researchers developed a single quantum dot (QD)-based nanosensor for sensitive detection of DNA methylation at both CpG and non-CpG sites using tricyclic ligation chain reaction (LCR)-mediated QD-based fluorescence resonance energy transfer (FRET).  They designed 2 sets of DNA probes (X and Y, X' and Y') for methylated DNA assay.  In the presence of thermostable DNA ligase, probes X and Y may adjacently hybridize with the methylated DNA to obtain the ligated XY products that may function as the templates for probes X' and Y' to generate the X'Y' products.  The resultant X'Y' products may in turn act as the templates to ligate probes X and Y for the generation of XY products, consequently inducing tricyclic LCR amplification under thermal denaturation conditions to generate a large number of XY products.  The subsequent hybridization of XY products with the capture and reporter probes results in the formation of sandwich hybrids that may assemble on the 605QD surface to obtain 605QD-oligonucleotide-Cy5 nanostructures, inducing efficient FRET from the 605QD to Cy5 and the emission of Cy5.  This nanosensor could detect DNA methylation at single 5-methylcytosine (5-mC) resolution with a detection limit of as low as 1.0 aM and a large dynamic range of 7 orders of magnitude.  The authors concluded that this nanosensor could distinguish as low as a 0.01 % methylation level, and it could detect DNA methylation in human lung cancer cells as well, holding great potential for accurate epigenetic evaluation and early cancer diagnosis.\n",
      "Hao and co-workers (2019) presented an electrolyte-gated graphene field effect transistor (GFET) nanosensor using aptamer for rapid, highly sensitive and specific detection of a lung cancer biomarker interleukin-6 (IL-6) with enhanced stability.  The negatively charged aptamer folded into a compact secondary conformation upon binding with IL-6, thus altering the carrier concentration of graphene and yielding a detectable change in the drain-source current Ids.  Aptamer has smaller size than other receptors (e.g., antibodies), making it possible to bring the charged IL-6 more closely to the graphene surface upon affinity binding, thereby enhancing the sensitivity of the detection.  Thanks to the higher stability of aptamer over antibodies, which degraded easily with increasing storage time, consistent sensing performance was obtained by the nanosensor over extended-time (greater than 24 hours) storage at 25 Â°C.  Furthermore, due to the GFET-enabled rapid transduction of the affinity recognition to IL-6, detection of IL-6 could be achieved in several minutes (less than 10 minutes).  The authors concluded that experimental results indicated that this nanosensor could rapidly and specifically respond to the change in IL-6 levels with high consistency after extended-time storage and a detection limit (DL) down to 139 fM; thus, the nanosensor holds great potential for lung cancer diagnosis at its early stage.\n",
      "UpToDate reviews on \"Overview of the initial evaluation, diagnosis, and staging of patients with suspected lung cancer\" (Thomas and Gould, 2019a) and \"Selection of modality for diagnosis and staging of patients with suspected non-small cell lung cancer\" (Thomas and Gould, 2019b) do not mention the use of magnetic nanosensor of as a diagnostic tool.\n",
      "Furthermore, National Comprehensive Cancer Networkâs clinical practice guidelines on \"Non-small cell lung cancer\" (Version 5.2019) and \"Small cell lung cancer\" (Version 1.2019) do not mention the use of magnetic nanosensor of as a diagnostic tool.\n",
      "Measurement of Circulating Tumor Cells (e.g., CellMax Life and FirstSightCRC) for Screening of Colorectal Cancer\n",
      "Yang and colleagues (2017) noted that CTCs have been accepted as a prognostic marker in patients with mCRC (International Union for Cancer Control [UICC] stage IV).  However, the prognostic value of CTCs in patients with non-mCRC (Union for International Cancer Control [UICC] stage I to III) still remains in dispute.  These researchers carried out a meta-analysis to examine the prognostic significance of CTCs detected by the RT-PCR method in patients diagnosed with non-mCRC patients.  A comprehensive literature search for relevant articles was performed in the Embase, PubMed, Ovid, Web of Science, Cochrane library and Google Scholar databases.  The studies were selected according to pre-determined inclusion/exclusion criteria.  Using the random-effects model of Stata software, version12.0 (2011) (Stata Corp, College Station, TX), to conduct the meta-analysis, and the HR, RR and their 95 % CIs were regarded as the effect measures.  Subgroup analyses and meta-regression were also conducted to clarify the heterogeneity.  A total of 12 eligible studies, containing 2,363 patients with non-mCRC, were suitable for final analyses.  The results showed that the OS (HRâ=â3.07, 95 % CI: 2.05 to 4.624, pâ<â0.001; I2â=â55.7 %, pâ=â0.008) and DFS (HRâ=â2.58, 95 % CI: 2.00 to 3.32, pâ<â0.001; I2â=â34.0 %, pâ=â0.085) were poorer in patients with CTC-positive, regardless of the sampling time, adjuvant therapy and TNM stage.  CTC-positive was also significantly associated with regional lymph nodes (RLNs) metastasis (RRâ=â1.62, 95 % CI: 1.17 to 2.23, pâ=â0.003; I2â=â74.6 %, p < 0.001), depth of infiltration (RRâ=â1.41, 95 % CI: 1.03 to 1.92, pâ=â0.03; I2â=â38.3 %, pâ=â0.136), vascular invasion (RRâ=â1.66, 95 % CI: 1.17 to 2.36, pâ=â0.004; I2â=â46.0 %, pâ=â0.135), tumor grade (RRâ=â1.19, 95 % CI: 1.02 to 1.40, pâ=â0.029; I2â=â0 %, pâ=â0.821) and TNM stage (I, II versus III) (RRâ=â0.76, 95 % CI: 0.71 to 0.81, pâ<â0.001; I2â=â0 %, pâ=â0.717).  However, there was no significant relationship between CTC-positive and tumor size (RRâ=â1.08, 95 % CI: 0.94 to 1.24, pâ=â0.30; I2â=â0 %, pâ=â0.528).  The authors concluded that detection of CTCs by RT-PCR method had prognostic value for non-mCRC patients, and CTC-positive was associated with poor prognosis and poor clinicopathological prognostic factors.  However, the prognostic value of CTCs supported the use of CTCs as an indicator of metastatic disease prior to the current classification of mCRC meaning it is detectable by CT/MRI.  This study did not address the use of CTC for screening of CRC.\n",
      "Lopresti and associates (2019) stated that CTCs represent an easy, repeatable and representative access to information regarding solid tumors.  However, their detection remains difficult because of their paucity, their short half-life, and the lack of reliable surface biomarkers.  Flow cytometry (FC) is a fast, sensitive and affordable technique, ideal for rare cells detection.  Adapted to CTCs detection (i.e., extremely rare cells), most FC-based techniques require a time-consuming pre-enrichment step, followed by a 2-hours staining procedure, impeding on the efficiency of CTCs detection.  These researchers overcame these caveats and reduced the procedure to less than 1 hour, with minimal manipulation.  First, cells were simultaneously fixed, permeabilized, then stained.  Second, using low-speed FC acquisition conditions and 2 discriminators (cell size and pan-cytokeratin expression), these investigators suppressed the pre-enrichment step.  Applied to blood from donors with or without known malignant diseases, this protocol ensured a high recovery of the cells of interest independently of their epithelial-mesenchymal plasticity and could predict which samples were derived from cancer donors.  The authors concluded that this proof-of-concept study laid the bases of a sensitive tool to detect CTCs from a small amount of blood upstream of in-depth analyses (colorectal cancer was one of the key words in this study).\n",
      "Baek and co-workers (2019) noted that CTCs in the blood have been used as diagnostic markers in patients with CRC.  In a prospective study, these researchers evaluated a CTC detection system based on cell size to examine CTCs and their potential as early diagnostic and prognostic biomarkers for CRC.  From 2014 to 2015, a total of 88 patients with newly diagnosed CRC, who were scheduled for surgery, and 31 healthy volunteers were enrolled and followed-up in Pusan National University Hospital; CTCs were enriched using a centrifugal microfluidic system with a new fluid-assisted separation technique (FAST) and detected by cytomorphological evaluation using fluorescence microscopy.  Two or more CTCs were detected using FAST in 74 patients and 3 healthy volunteers.  The number of CTCs in the CRC group was significantly higher than that in the healthy volunteers (p < 0.001).  When a ROC curve was created to differentiate patients with CRC from healthy volunteers, the sensitivity and specificity were almost optimized when the critical CTC value was 5/7.5 ml of blood.  When this value was used, the sensitivity and specificity in differentiating patients with CRC from the healthy controls were 75 % and 100 %, respectively.  In patients with CRC with greater than or equal to 5 CTCs, vascular invasion was frequently identified (p = 0.035).  All patients with stage IV were positive for CTCs.  Patients with greater than or equal to 5 CTCs showed a trend toward poor OS and PFS.  The authors concluded that this study demonstrated promising results with the use of FAST-based CTC detection for the early diagnosis and prognosis of CRC.  This study did not address the use of CTC for screening of CRC.\n",
      "UpToDate reviews on \"Screening for colorectal cancer: Strategies in patients at average risk\" (Doubeni, 2019) and \"Screening for colorectal cancer in patients with a family history of colorectal cancer or advanced polyp\" (Ramsey and Grady, 2019) do not mention measurement of circulating tumor cells as a screening tool.\n",
      "Furthermore, National Comprehensive Cancer Networkâs clinical practice guidelines on \"Colon cancer\" (Version 2.2019) and \"Rectal cancer\" (Version 2.2019) do not mention measurement of circulating tumor cells as a screening tool.\n",
      "mRNA Gene Expression Profiling for Cutaneous Melanoma\n",
      "Wang and colleagues (2019) stated that metastatic melanoma of the skin has an aggressive course with high morbidity and mortality.  Thus, an increased understanding of the pathogenesis of metastatic melanoma has gained increasing attention, including the role of epigenetic modification and competing endogenous RNA (ceRNA).  These researchers used bioinformatics data to undertake an integrative analysis of long non-coding RNA (lncRNA), microRNA (miRNA) and messenger RNA (mRNA) expression to construct a ceRNA network in metastatic melanoma.  Data from the Cancer Genome Atlas (TCGA), the Gene Ontology (GO) database, and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway were analyzed.  There were 471 cases that included 103 primary solid tumors and 368 cases of metastatic melanoma that included transcriptome sequencing data (including lncRNA and mRNA); 452 cases had miRNA sequencing data.  Analysis of chip data identified 85 6 mRNAs, 67 miRNAs, and 250 lncRNAs that were differentially expressed in cases of metastatic melanoma, of which 25 miRNAs, 18 lncRNAs, and 18 mRNAs participated in the formation of ceRNAs.  Survival analysis identified 7 differentially expressed mRNAs, 5 differentially expressed miRNAs (miRNA-29c, miRNA-100, miR-142-3p, miR-150, miR-516a-2), and 6 differentially expressed lncRNAs (AC068594.1, C7orf71, FAM41C, GPC5-AS1, MUC19, LINC00402) that were correlated with survival time in patients with metastatic melanoma.  Bioinformatics data and integrative analysis identified lncRNA, miRNA, and mRNA expression to construct a ceRNA and patient survival network in metastatic melanoma.  The authors concluded that these findings supported the need for further studies on the mechanisms involved in the regulation of metastatic melanoma by ceRNAs; and the use of public genomic data might identify potential therapeutic targets and provide new ways to examine the functions and mechanisms of ceRNAs in melanoma and other malignant tumors.\n",
      "Sun and associates (2019) demonstrated altered expression of lncRNA and mRNA in melanoma using data from the TCGA database; GO, KEGG enrichment, and protein-protein interaction (PPI) analyses were conducted.  These investigators also constructed a functional lncRNA-mRNA regulatory network and Kaplan-Meier analysis.  They identified 246 differentially expressed (DE) lncRNAs and 856 DEmRNAs.  A total of 184 DElncRNAs and 428 DEmRNAs were up-regulated in metastatic melanoma, while all others were down-regulated.  Also, these investigators examined the co-expression pattern of 363 genes, among which 26 up-regulated lncRNAs, 9 down- regulated lncRNAs, 49 up-regulated mRNAs and 151 down-regulated mRNAs were identified as being co-expressed with others.  Survival analysis suggested high levels of 14 lncRNAs and 10 mRNAs may significantly increase or decrease OS.  These differentially expressed genes are also potentially prognostic in melanoma.  The authors concluded that these findings suggested potential roles for lncRNAs and mRNAs during melanoma progression and provide candidate biomarkers for further studies.\n",
      "Roca and co-workers (2019) noted that recent studies have demonstrated the existence of multiple copies of E2F1 gene in melanoma specimens which could explain the deregulated E2F1 activity in this type of cancer.  This finding suggested a key role for this transcription factor in the malignant transformation of melanocytes.  Thus, E2F1 has been considered as a potential therapeutic target for this form of skin cancer.  Since germline copy number variations (CNVs) have been associated with increased susceptibility to different types of cancer, these researchers examined germline E2F1 CNV in melanoma patients.  However, CNVs not necessarily lead to gene dosage imbalance, hence, further factors, in association with CNVs, could contribute to clinical manifestations.  Considering that heat stress has been hypothesized as a contributing factor to skin cancer, these investigators also examined the effect of heat stress on E2F1 expression.  E2F1 CNV was measured in genomic DNA isolated from blood of 552 patients diagnosed with melanoma and 520 healthy subjects using TaqMan Copy Number Assays.  E2F1 mRNA expression was also evaluated by RT-qPCR in the melanoma cell line, SK MEL 267, before and after exposure to heat stress.  They found that patients diagnosed with melanoma (1.6 %, 9/552) harbored frequently altered germline E2F1 copies compared to healthy subjects (0 %, 0/520).  Moreover, the difference among the 2 groups was statistically significant (pâ=â0.004).  Furthermore, they found that heat exposure alone could significantly induce E2F1 expression.  The authors concluded that this was the 1st study that showed a relation between germline E2F1 CNV and melanoma, suggesting that altered copies of this gene might be a predisposing factor to skin cancer.  These findings also suggested that environmental insults, such as heat stress, could contribute to an aberrant E2F1 activity by inducing E2F1 mRNA expression.  Therefore, subjects with multiple constitutive copies of E2F1 are at greater risk of developing melanoma when exposed to heat.  Altogether these findings corroborated with the hypothesis that susceptibility to melanoma depends on both the environment and genetic factors.\n",
      "Liu and colleagues (2019) stated that paired box 3 (PAX3) is a transcription factor and critical regulator of pigment cell development during embryonic development.  However, while there have been several studies on PAX3, its expression patterns and precise role remain to be clarified.  These researchers described an in-depth computational study of tumor-associated gene information, with specific emphasis on the expression of PAX3 in melanoma, using Oncomine along with an investigation of corresponding expression profiles in an array of cancer cell lines through Cancer Cell Line Encyclopedia analysis.  Based on Kaplan-Meier analysis, the prognostic value of high PAX3 expression in tissues from patients with melanoma compared with normal tissues was assessed.  PAX3 was more highly expressed in male patients with melanoma compared with female patients with melanoma.  Using Oncomine and Coexpedia analysis, it was demonstrated that PAX3 expression was clearly associated with SRY-box 10 expression.  The survival analysis results revealed that high PAX3 mRNA expression was associated with worse survival rates in patients with melanoma.  The authors concluded that these findings suggested that PAX3 may be a biomarker and essential prognostic factor for melanoma, and provided an important theoretical basis for the development of melanoma treatments.\n",
      "Prabhakar and associates (2019) noted that Wnt/Î²-catenin signaling plays an important role in melanocyte biology, especially in the early stages of melanocyte transformation and melanomagenesis.  Î²-catenin, encoded by the gene CTNNB1, is an intra-cellular signal transducer of Wnt signaling and activates transcription of genes important for cell proliferation and survival.  Wnt/Î²-catenin signaling is frequently activated in melanoma through oncogenic mutations of Î²-catenin and elevated Î²-catenin levels are positively correlated with melanoma aggressiveness.  Molecular mechanisms that regulate Î²-catenin expression in melanoma are not fully understood.  MicroRNA-214 is known to function as a tumor suppressor by targeting Î²-catenin in several types of cancer cells.  These investigators examined the regulation of Î²-catenin by miR-214 and its role in melanoma.  They showed that Î²-catenin mRNA levels were negatively correlated with miR-214 in melanoma.  However, over-expression of miR-214 paradoxically increased Î²-catenin protein levels and promoted malignant properties of melanoma cells including resistance to mitogen-activated protein kinase inhibitors (MAPKi).  RNA-seq analysis revealed that melanoma cells predominantly express a Î²-catenin mRNA isoform lacking miR-214 target site.  Using matched miRNA and mRNA-seq and bioinformatics analysis, these researchers identified novel miR-214 targets, ankyrin repeat domain 6 (ANKRD6) and C-terminal binding protein 1 (CTBP1), that were involved in negative regulation of Wnt signaling.  Over-expression of miR-214 or knockdown of the novel miR-214 targets, ANKRD6 or CTBP1, increased melanoma cell proliferation, migration, and decreased sensitivity to MAPKi.  The authors concluded that these findings suggested that in melanoma cells, Î²-catenin was not regulated by miR-214; and the functions of miR-214 in melanoma were mediated partly by regulating proteins involved in attenuation of Wnt/Î²-catenin signaling.\n",
      "Furthermore, National Comprehensive Cancer Networkâs clinical practice guideline on \"Cutaneous melanoma\" (Version 2.2019) does not mention mRNA gene expression profiling as a management tool.\n",
      "Litchman et al (2020) noted that to decrease morbidity and mortality from melanoma, it is imperative to identify patients who are at high-risk for developing widespread disease.  Gene expression profiling technology may impact melanoma management as physicians are better equipped to measure prognosis.  Many different GEP signatures have been examined.  These investigators searched PubMed, Cochrane CENTRAL, and Embase for studies on GEP in primary melanoma prognosis and examined GEP signatures for prognostic and analytic validity and clinical impact.  The relationship between GEP and survival was measured using HR and OR.  They found 29 articles comprising 9 gene signatures meeting inclusion criteria and carried out a meta-analysis on 6 studies on a 31-gene signature.  High-risk GEP status was associated with poorer RFS (HR = 7.22; 95 % CI: 4.75 to 10.98), DMFS (HR = 6.62; 95 % CI: 4.9 to 8.91), and OS (HR = 7.06; 95 % CI: 4.44 to 11.22); as well as sentinel lymph node biopsy (SNLB) positivity (OR = 2.99; 95 % CI: 2.15 to 4.15).  The authors concluded that the findings of this study had clinical implications for melanoma patients who may benefit from prognostic testing.  These results may be useful to clinicians when ordering GEP testing and help them make better management decisions.\n",
      "The authors stated that this systematic review and meta-analysis had several drawbacks.  Most included studies were conducted retrospectively versus prospectively.  Although a comprehensive search strategy was used, missing relevant studies may be unavoidable, especially those published in non-English language journals.  Some studies did not directly provide HRs with corresponding effect sizes requiring manual derivation from Kaplan-Meier survival curves.  Significant heterogeneity was noted when calculating the pooled HR for RFS and the pooled OR for recurrence.  Furthermore, several permutations of different melanoma gene profiles were being tested and developed; however, these new technologies are still early in their development and additional studies may impact on potential uses and adoption.\n",
      "Hyams et al (2021) defined changes in clinical management resulting from the use of the prognostic 31-gene expression profile (31-GEP) test for cutaneous melanoma in a surgical oncology practice.  Management plans for 112 consecutively tested patients with stage I to III melanoma were examined for duration and number of clinical visits, blood work and imaging.  31-GEP high-risk (class 2; n = 46) patients received increased management compared with low-risk (class 1; n = 66) patients.  Test results were most closely associated with follow-up and imaging.  Of class 1 patients, 65 % received surveillance intensity within guidelines for stage I to IIA patients; 98 % of class 2 patients received surveillance intensity equal to stage IIB to IV patients.  The authors suggested clinical follow-up and metastatic screening be adjusted according to 31-GEP test results. \n",
      "The authors stated that a drawback of this study was the absence of clinical outcomes data associated with the 31-GEP class and an analysis of the completed follow-up and surveillance regimens.  Evaluation of these endpoints was limited by the relatively short follow-up of this cohort; however, the clinical accuracy of the 31-GEP test to predict patient outcomes has been extensively reported, including 2 recent systematic reviews and meta-analyses, including in prospective cohorts.  These researchers stated that longer follow-up of this cohort would allow an evaluation of the combination of the test result and completed management plans to identify recurrence; however, the findings reported in this study were consistent with other investigations in which the 31-GEP test substantially influenced patient management recommendations for class 1 and 2 patients, as a function of their disparate biological risk.  A 2nd drawback was that this cohort included a higher-risk group in which all patients had a SLNB performed, therefore, the majority of management changes reflect up-staging of patients with stage I to IIA tumors, and a class 2 result to more intensive surveillance.  Thus, extrapolation of results to impact on healthcare costs would likely be over-estimated compared with a more diverse melanoma population as was reported by Vetto et al (2019) who showed that class 1A patients over the age of 65 years old had less than 5 % risk of SLN positivity and could therefore forego SLN biopsy and may offset costs incurred by increased surveillance in high-risk patients.\n",
      "Mi-Prostate Score (MiPS)\n",
      "The MiPS assay is a multiplex analysis of TMPRSS2:ERG (T2:ERG) gene fusion, post-DRE urine expression of PCA3, and serum PSA (KLK3). The MiPS assay tests for the presence of two prostate cancer biomarkers: a piece of RNA made from the PCA3 gene, found to be overactive in 95 percent of all prostate cancers, and another RNA marker that is found only when TMPRSS2 and ERG abnormally fuse. TMPRSS2:ERG, or T2-ERG, is a strong indicator of prostate cancer. The MiPS test is not meant to be used alone as a prostate cancer screening tool, nor is it intended to replace PSA. The Mi-Prostate Score test is designed to provide additional information for patients who have undergone PSA testing. The performance in men who have not undergone PSA testing is unknown.\n",
      "The NCCN Prostate Cancer Early Detection Guidelines (v.2.2019) states that rearrangements of the ERG gene are found in approximately half of prostate cancers and early studies suggested that the combination of TMPRSS2:ERG (T2:ERG) gene fusion and PCA3 improved the prediction of prostate cancer on biopsy (Tomlinson 2016; Sanda 2017). Based on these early results, the NCCN panel considers MiPS to be investigational at present time, but will review additional information as it becomes available.\n",
      "Newcomb et al (2019) stated for men on active surveillance for prostate cancer, biomarkers may improve prediction of reclassification to higher grade or volume cancer. This study examined the association of urinary PCA3 and TMPRSS2:ERG (T2:ERG) with biopsy-based reclassification. Urine was collected at baseline, 6, 12, and 24 months in the multi-institutional Canary Prostate Active Surveillance Study (PASS), and PCA3 and T2:ERG levels were quantitated. Reclassification was an increase in Gleason score or ratio of biopsy cores with cancer to â¥34%. The association of biomarker scores, adjusted for common clinical variables, with short- and long-term reclassification was evaluated. Discriminatory capacity of models with clinical variables alone or with biomarkers was assessed using receiver operating characteristic (ROC) curves and decision curve analysis (DCA). Seven hundred and eighty-two men contributed 2069 urine specimens. After adjusting for PSA, prostate size, and ratio of biopsy cores with cancer, PCA3 but not T2:ERG was associated with short-term reclassification at the first surveillance biopsy (ORâ=â1.3; 95% CI 1.0-1.7, pâ=â0.02). The addition of PCA3 to a model with clinical variables improved area under the curve from 0.743 to 0.753 and increased net benefit minimally. After adjusting for clinical variables, neither marker nor marker kinetics was associated with time to reclassification in subsequent biopsies. The authors stated that studies evaluating the use of PCA3 in active surveillance have been limited and sample sizes have been small. In the current study, which is the largest study to date of PCA3 in men using active surveillance, after adjustment for clinical variables available after cancer diagnosis, the authors found a significant association of PCA3 with reclassification at the sBx1 (adjusted OR = 1.3, p = 0.02), but not for subsequent biopsies (adjusted OR = 1.01, p = 0.96). Although they found no association between T2:ERG and biopsy reclassification, some studies have suggested improved performance when PCA3 and T2:ERG are used in combination or combined into a MiPS score for the initial diagnosis of PCa. Thus, the authors combined PCA3 and T2:ERG into a MIPS score, but found little or no improvement over PCA3 alone. The authors concluded that PCA3 but not T2:ERG was associated with cancer reclassification in the first surveillance biopsy but has negligible improvement over clinical variables alone in ROC or DCA analyses. Neither marker was associated with reclassification in subsequent biopsies.\n",
      "Sanda et al (2017) stated potential survival benefits from treating aggressive (Gleason score, â¥7) early-stage prostate cancer are undermined by harms from unnecessary prostate biopsy and overdiagnosis of indolent disease. The objective of their study was to evaluate the a priori primary hypothesis that combined measurement of PCA3 and TMPRSS2:ERG (T2:ERG) RNA in the urine after digital rectal examination would improve specificity over measurement of prostate-specific antigen alone for detecting cancer with Gleason score of 7 or higher. As a secondary objective, to evaluate the potential effect of such urine RNA testing on health care costs. They conducted a prospective, multicenter diagnostic evaluation and validation in academic and community-based ambulatory urology clinics. Participants were a referred sample of men presenting for first-time prostate biopsy without preexisting prostate cancer: 516 eligible participants from among 748 prospective cohort participants in the developmental cohort and 561 eligible participants from 928 in the validation cohort. The interventions included: Urinary PCA3 and T2:ERG RNA measurement before prostate biopsy. The main outcome measure was the presence of prostate cancer having Gleason score of 7 or higher on prostate biopsy. Pathology testing was blinded to urine assay results. In the developmental cohort, a multiplex decision algorithm was constructed using urine RNA assays to optimize specificity while maintaining 95% sensitivity for predicting aggressive prostate cancer at initial biopsy. Findings were validated in a separate multicenter cohort via prespecified analysis, blinded per prospective-specimen-collection, retrospective-blinded-evaluation (PRoBE) criteria. Cost effects of the urinary testing strategy were evaluated by modeling observed biopsy results and previously reported treatment outcomes. Among the 516 men in the developmental cohort (mean age, 62 years; range, 33-85 years) combining testing of urinary T2:ERG and PCA3 at thresholds that preserved 95% sensitivity for detecting aggressive prostate cancer improved specificity from 18% to 39%. Among the 561 men in the validation cohort (mean age, 62 years; range, 27-86 years), analysis confirmed improvement in specificity (from 17% to 33%; lower bound of 1-sided 95% CI, 0.73%; prespecified 1-sided Pâ=â.04), while high sensitivity (93%) was preserved for aggressive prostate cancer detection. Forty-two percent of unnecessary prostate biopsies would have been averted by using the urine assay results to select men for biopsy. Cost analysis suggested that this urinary testing algorithm to restrict prostate biopsy has greater potential cost-benefit in younger men. The authors concluded that combined urinary testing for T2:ERG and PCA3 can avert unnecessary biopsy while retaining robust sensitivity for detecting aggressive prostate cancer with consequent potential health care cost savings.\n",
      "Tomlins et al (2016) state TMPRSS2:ERG (T2:ERG) and prostate cancer antigen 3 (PCA3) are the most advanced urine-based prostate cancer (PCa) early detection biomarkers. The authors aimed to validate logistic regression models, termed Mi-Prostate Score (MiPS), that incorporate serum prostate-specific antigen (PSA; or the multivariate Prostate Cancer Prevention Trial risk calculator version 1.0 [PCPTrc]) and urine T2:ERG and PCA3 scores for predicting PCa and high-grade PCa on biopsy. T2:ERG and PCA3 scores were generated using clinical-grade transcription-mediated amplification assays. Pretrained MiPS models were applied to a validation cohort of whole urine samples prospectively collected after digital rectal examination from 1244 men presenting for biopsy. Area under the curve (AUC) was used to compare the performance of serum PSA (or the PCPTrc) alone and MiPS models. Decision curve analysis (DCA) was used to assess clinical benefit. Among informative validation cohort samples (n=1225 [98%], 80% from patients presenting for initial biopsy), models incorporating T2:ERG had significantly greater AUC than PSA (or PCPTrc) for predicting PCa (PSA: 0.693 vs 0.585; PCPTrc: 0.718 vs 0.639; both p<0.001) or high-grade (Gleason score >6) PCa on biopsy (PSA: 0.729 vs 0.651, p<0.001; PCPTrc: 0.754 vs 0.707, p=0.006). MiPS models incorporating T2:ERG score had significantly greater AUC (all p<0.001) than models incorporating only PCA3 plus PSA (or PCPTrc or high-grade cancer PCPTrc [PCPThg]). DCA demonstrated net benefit of the MiPS_PCPTrc (or MiPS_PCPThg) model compared with the PCPTrc (or PCPThg) across relevant threshold probabilities. The authors concluded that incorporating urine TMPRSS2:ERG (T2:ERG) and PCA3 scores improves the performance of serum PSA (or PCPTrc) for predicting PCa and high-grade PCa on biopsy. The authors noted that limitations of this study, which included included the use of PCPTrc_v1 and PCPThg_v1, rather than updated version 2 (v2) risk calculators, because the MiPS models were locked for subsequent validation studies prior to PCPT_v2 risk calculator development. Of note, PCPTrc_v2 and PCPThg_v2 were poorly calibrated in the validation cohort, with no significant difference in AUCs compared with version 1 (PCPThg_v1 showed significantly increased AUC compared to PCPThg_v2). In addition, the authors observed greater improvement for predicting all cancers, compared with high-grade cancer only, when incorporating T2:ERG plus PCA3 scores. Although overdiagnosis of low-grade cancer drives overtreatment, whether these models show utility in identifying the subset of patients with low-grade cancer who harbor undiagnosed higher grade cancer ( approximately 20â40%) or can be combined with novel imaging or tissue-based prognostic tests should be investigated. Of note, tissue and urine assessment of PCA3 and/or T2:ERG have been variably associated with significant disease and progression, supporting the need for additional investigation in these settings. Last, the validation cohort consisted of men without cancer undergoing biopsy based on current standard of care (i.e., elevated serum PSA), so conclusions regarding performance in men on active surveillance or screening populations is unknown.\n",
      "Salami et al (2013) sought to develop a clinical algorithm combining serum PSA with detection of TMPRSS2:ERG fusion and PCA3 in urine collected after digital rectal exam (post-DRE urine) to predict prostate cancer on subsequent biopsy. Post-DRE urine was collected in 48 consecutive patients before prostate biopsy at 2 centers; quantitative reverse transcription-polymerase chain reaction (qRT-PCR) was used to detect PCA3 and TMPRSS2:ERG fusion transcript expression. Serum PSA was measured by clinical assay. The performance of TMPRSS2:ERG fusion, PCA3, and serum PSA as biomarkers predicting prostate cancer at biopsy was measured; a clinically practical algorithm combining serum PSA with TMPRSS2:ERG and PCA3 in post-DRE urine to predict prostate cancer was developed. Post-DRE urine sediment provided informative RNA in 45 patients; prostate cancer was present on subsequent biopsy in 15. TMPRSS2:ERG in post-DRE urine was associated with prostate cancer (OR = 12.02; P < 0.001). PCA3 had the highest sensitivity in predicting prostate cancer diagnosis (93%), whereas TMPRSS2:ERG had the highest specificity (87%). TMPRSS2:ERG had the greatest discriminatory value in predicting prostate cancer (AUC = 0.77 compared with 0.65 for PCA3 and 0.72 for serum PSA alone). Combining serum PSA, PCA3, and TMPRSS2:ERG in a multivariable algorithm optimized for clinical utility improved cancer prediction (AUC = 0.88; specificity = 90% at 80% sensitivity). The authors concluded that a clinical algorithm specifying biopsy for all patients with PSA â¥ 10 ng/ml, while restricting biopsy among those with PSA <10 ng/ml to only those with detectable PCA3 or TMPRSS2:ERG in post-DRE urine, performed better than the individual biomarkers alone in predicting prostate cancer.\n",
      "Lymph2Cx and Lymph3Cx Lymphoma Molecular Classification Assay \n",
      "The Lymph2Cx assay determines cell-of-origin (COO) of diffuse large B-cell lymphoma (DLBCL) patients by analyzing RNA expression of 20 selected genes. \n",
      "Scott and co-workers (2014) stated that the assignment of DLBCL into COO groups is becoming increasingly important with the emergence of novel therapies that have selective biological activity in germinal center B cell-like (GCB) or activated B cell-like (ABC) groups.  The Lymphoma/Leukemia Molecular Profiling Project's Lymph2Cx assay is a parsimonious digital gene expression (NanoString)-based test for COO assignment in formalin-fixed paraffin-embedded tissue (FFPET).  The 20-gene assay was trained using 51 FFPET biopsies; the locked assay was then validated using an independent cohort of 68 FFPET biopsies.  Comparisons were made with COO assignment using the original COO model on matched frozen tissue.  In the validation cohort, the assay was accurate, with only 1 case with definitive COO being incorrectly assigned, and robust, with greater than 95 % concordance of COO assignment between 2 independent laboratories.  The authors concluded that these qualities, along with the rapid turn-around time, made Lymph2Cx attractive for implementation in clinical trials and, ultimately, patient management.  Moreover, these researchers stated that the predictive and therapeutic values of the COO and risk stratification models developed in this large cohort of DLBCL remain to be determined in future prospective clinical studies.\n",
      "Hwang and colleagues (2018) noted that COO classification of DLBCL is increasingly important due to its prognostic significance and the development of subtype-specific therapeutics.  These researchers compared the clinical utility of the Lymph2Cx assay against 4 widely used IHC algorithms in 150 R-CHOP-treated DLBCL patients using archival tissue.  In contrast to the predominance of GCB subtype in Western populations, Lymph2Cx assay classified more than 50 % of the Korean cases as the ABC subtype (ABC, 83/150 [55.3 %]; GCB, 51/150 [34.0 %]; unclassifiable, 16/150 [10.7 %]).  Predominance of ABC subtype tended to be more pronounced in the nodal lymphomas than in the extra-nodal lymphomas; however, among the primary extra-nodal sites, ABC subgroups predominated in primary testicular, breast, and adrenal gland lymphomas.  The classification of COO by Lymph2Cx assay did not show any significant association with clinical parameters.  The overall concordance rates of the IHC algorithms with the Lymph2Cx ranged from 78.0 % to 84.3 %; however, 47.1 % to 66.7 % of the cases of the Lymph2Cx-defined GCB subgroup were mis-classified as the non-GCB class by the IHC algorithms.  The survival of Lymph2Cx-classified COO subtypes was not significantly different in the present cohort.  Th authors concluded that the ABC subtype predominated over GCB in Korean patients; there were significant discrepancies between the IHC and Lymph2Cx classifications, especially in GCB subtype.\n",
      "Lee and associates (2019) stated that previous studies in Western populations, using IHC methods to subtype DLBCL, suggested that GCBs have improved outcomes; however, data in Asians have been limited and conflicting.  These researchers examined the prognostic impact of COO subtyping by IHC and Lymph2Cx in South-East Asian (SEA) DLBCL patients; and summarized the existing literature.  They carried out a single-center retrospective analysis of 384 DLBCL patients diagnosed between 2013 and 2018 who received rituximab-based chemotherapy.  Hans and Lymph2Cx were used to assign COO and correlated with outcomes.  International Prognostic Index (IPI) score was associated with OS and PFS.  The 5-year OS for non-GCB versus GCB for COO by Hans was 70 % versus 71 %; p = 0.39, while 5-year OS for ABC versus GCB for COO by Lymph2Cx was 74 % versus 92 %; p = 0.19.  The 5-year PFS for non-GCB versus GCB for COO by Hans was 65 % versus 70 %; p=0.26, while 5-year PFS for ABC versus GCB for COO by Lymph2Cx was 64 % versus 86 %; p = 0.07.  The authors concluded that IPI was re-affirmed to be relevant in the rituximab era.  COO by Hans had no prognostic significance, while subtyping by Lymph2Cx trended toward GCBs having better PFS and OS; however, this was not statistically significant in this study cohort.\n",
      "The Lymph3Cx test is considered a modular upgrade to the Lymph2Cx test.  Lymph3Cx is a 58âgene expression assay by fluorescent probe hybridization and applicable to formalinâfixed paraffinâembedded tissue intended to distinguish between primary mediastinal B-cell lymphoma (PMBCL) and diffuse large B-cell lymphoma (DLBCL). Mottok et al (2018) stated primary mediastinal large B-cell lymphoma (PMBCL) is recognized as a distinct entity in the World Health Organization classification. Currently, the diagnosis relies on consensus of histopathology, clinical variables, and presentation, giving rise to diagnostic inaccuracy in routine practice. Previous studies have demonstrated that PMBCL can be distinguished from subtypes of diffuse large B-cell lymphoma (DLBCL) based on gene expression signatures. However, requirement of fresh-frozen biopsy material has precluded the transfer of gene expression-based assays to the clinic. Here, the authors developed a robust and accurate molecular classification assay (Lymph3Cx) for the distinction of PMBCL from DLBCL subtypes based on gene expression measurements in formalin-fixed, paraffin-embedded tissue. A probabilistic model accounting for classification error, comprising 58 gene features, was trained on 68 cases of PMBCL and DLBCL. Performance of the model was subsequently evaluated in an independent validation cohort of 158 cases and showed high agreement of the Lymph3Cx molecular classification with the clinicopathological diagnosis of an expert panel (frank misclassification rate, 3.8%). Furthermore, the authors state they demonstrate reproducibility of the assay with 100% concordance of subtype assignments at 2 independent laboratories. The authors conclude that future studies will determine Lymph3Cx's utility for routine diagnostic purposes and therapeutic decision making.\n",
      "FoundationACT\n",
      "Chung and co-workers (2017) stated that genomic changes that occur in breast cancer during the course of disease have been informed by sequencing of primary and metastatic tumor tissue.  For patients with relapsed and metastatic disease, evolution of the breast cancer genome highlights the importance of using a recent sample for genomic profiling to guide clinical decision-making.  Obtaining a metastatic tissue biopsy can be challenging, and analysis of circulating tumor DNA (ctDNA) from blood may provide a minimally invasive alternative.  In this study, hybrid capture-based genomic profiling was carried out on ctDNA from 254 female patients with ER+ breast cancer.  Peripheral blood samples were submitted by clinicians in the course of routine clinical care between May 2016 and March 2017.  Sequencing of 62 genes was carried out to a median unique coverage depth of 7503Ã.  Genomic alterations (GAs) in ctDNA were evaluated and compared with matched tissue samples and genomic datasets of tissue from breast cancer.  At least 1 GA was reported in 78 % of samples.  Frequently altered genes were TP53 (38 %), ESR1 (31 %) and PIK3CA (31 %).  Temporally matched ctDNA and tissue samples were available for 14 patients; 89 % of mutations detected in tissue were also detected in ctDNA.  Diverse ESR1 GAs including mutation, re-arrangement and amplification, were observed.  Multiple concurrent ESR1 GAs were observed in 40 % of ESR1-altered cases, suggesting polyclonal origin; ESR1 compound mutations were also observed in 2 cases.  ESR1-altered cases harbored co-occurring GAs in PIK3CA (35 %), FGFR1 (16 %), ERBB2 (8 %), BRCA1/2 (5 %), and AKT1 (4 %).  The authors concluded that GAs relevant to relapsed/metastatic breast cancer management were identified, including diverse ESR1 GAs.  Genomic profiling of ctDNA demonstrated sensitive detection of mutations found in tissue.  Detection of amplifications was associated with ctDNA fraction.  These researchers stated that genomic profiling of ctDNA may provide a complementary and possibly alternative approach to tissue-based genomic testing for patients with ER+ metastatic breast cancer.\n",
      "In a retrospective study,  Zhou et al (2018) examined if the plasma ctDNA level and tumor biological features in patients with advanced solid tumors affected the detection of GAs by a plasma ctDNA assay.  Cell-free DNA (cfDNA) extracted from frozen plasma (nâ=â35) or fresh whole blood (nâ=â90) samples were subjected to a 62-gene hybrid capture-based NGS assay FoundationACT.  Concordance was analyzed for 51 matched FoundationACT and FoundationOne (tissue) cases.  The maximum somatic allele frequency (MSAF) was used to estimate the amount of tumor fraction of cfDNA in each sample.  The detection of GAs was correlated with the amount of cfDNA, MSAF, total tumor anatomic burden (dimensional sum), and total tumor metabolic burden (SUVmax sum) of the largest 10 tumor lesions on PET/CT scans.  FoundationACT detected GAs in 69 of 81 (85 %) cases with MSAF greater than 0; 42 of 51 (82 %) cases had greater than or equal toâ1 concordance GAs matched with FoundationOne, and 22 (52 %) matched to the NCCN-recommended molecular targets.  FoundationACT also detected 8 unique molecular targets, which changed the therapy in 7 (88 %) patients who did not have tumor re-biopsy or sufficient tumor DNA for genomic profiling assay.  In all samples (nâ=â81), GAs were detected in plasma cfDNA from cancer patients with high MSAF quantity (pâ=â0.0006) or high tumor metabolic burden (pâ=â0.0006) regardless of cfDNA quantity (pâ=â0.2362).  The authors concluded that the findings of this study supported the utility of using plasma-based genomic assays in cancer patients with high plasma MSAF level or high tumor metabolic burden.  Moreover, these researchers stated that further studies are needed to optimize the clinical application of plasma ctDNA NGS assays in cancer types other than NSCLC.\n",
      "The authors stated that this study had several drawbacks.  First, these investigators included cancer patients with multiple cancer types at initial diagnosis and tumor progression who subsequently underwent different treatment regimens.  The majority (73 % ) of cases in this study was NSCLC.  Further study should consider the impact of different molecular subtypes and cancer types on tumor metabolic activity that might affect the production of plasma ctDNA.  Second, current clinical reports only use SUVmax as the PET biomarker and do not routinely measure SUVmax for all metabolically active tumor lesions, especially in patients with extensive metastatic disease, which can significantly alter the value of the metabolic tumor sum.  Conversely, the quantification of tumor metabolic burden was labor-intensive and subjected to observer variations.  Further investigation should examine automated imaging tools to allow fast and objective quantification of the SUVmax sum and define the detection threshold in each cancer type for each assay for broad clinical application.  Third, FDG PET could not distinguish increased metabolism caused by tumor cells from that caused by infectious or non-infectious inflammation such as pneumonitis.  Development of tumor-specific molecular imaging might improve the distinction of these clinical entities.\n",
      "Clark and colleagues (2018) stated that genomic profiling of circulating tumor DNA derived from cfDNA in blood can provide a non-invasive method for detecting genomic biomarkers to guide clinical decision-making for cancer patients.  These researchers developed a hybrid capture-based NGS assay for genomic profiling of circulating tumor DNA from blood (FoundationACT).  High-sequencing coverage and molecular barcode-based error detection enabled accurate detection of GAs, including short variants (base substitutions, short insertions/deletions) and genomic re-arrangements at low allele frequencies (AFs), and copy number amplifications.  Analytical validation was performed on 2,666 reference alterations.  The assay achieved greater than 99 % overall sensitivity (95 % CI: 99.1 % to 99.4 %) for short variants at AF greater than 0.5 %, greater than 95 % sensitivity (95 % CI: 94.2 % to 95.7 %) for AF 0.25 % to 0.5 %, and 70 % sensitivity (95 % CI: 68.2 % to 71.5 %) for AF 0.125 % to 0.25 %.  No false positives were detected in 62 samples from healthy volunteers.  Genomic alterations detected by FoundationACT demonstrated high concordance with orthogonal assays run on the same clinical cfDNA samples.  In 860 routine clinical FoundationACT cases, genomic alterations were detected in cfDNA at comparable frequencies to tissue; for the subset of cases with temporally matched tissue and blood samples, 75 % of genomic alterations and 83 % of short variant mutations detected in tissue were also detected in cfDNA.  The authors concluded that on the basis of analytical validation results, FoundationACT has been approved for use in their Clinical Laboratory Improvement Amendments-certified/College of American Pathologists-accredited/New York State-approved laboratory.  These researchers stated that the development of this blood-based ctDNA assay may provide an alternative or complementary approach to tissue-based genomic testing for patients with cancer.\n",
      "The authors stated that this clinical series included a broad range of cancer types.  Further tumor type-specific studies are needed to understand the performance of genomic profiling of ctDNA in the context of each cancer type.  For example, a recent study demonstrated robust performance of the FoundationACT assay for estrogen receptor-positive breast cancer (Chung et al, 2017); blood-based genomic profiling may be challenging for disease types, such as glioma, that often do not release sufficient ctDNA into the blood.  Additional studies of clinical validity and clinical utility are needed for ctDNA assays, and the comparison of ctDNA assays to approved tissue-based predictive biomarker tests will inform the relative roles of ctDNA versus tissue-based genomic profiling in the management of patients with cancer.\n",
      "Schrock and associates (2018) noted that genomic profiling of tumor biopsies from advanced GI and anal cancers is increasingly used to inform treatment.  In some cases, tissue biopsy can be prohibitive, and these investigators examined if analysis of blood-derived ctDNA may provide a minimally invasive alternative.  Hybrid capture-based genomic profiling of 62 genes was performed on blood-based ctDNA from 417 patients with GI carcinomas to examine the presence of GA and compare with matched tissue samples.  Evidence of ctDNA was detected in 344 of 417 samples (82 %), and of these, greater than or equal to 1 reportable GA was detected in 89 % (306/344) of samples.  Frequently altered genes were TP53 (72 %), KRAS (35 %), PIK3CA (14 %), BRAF (8 %), and EGFR (7 %).  In temporally matched ctDNA and tissue samples available from 25 patients, 86 % of alterations detected in tissue were also detected in ctDNA, including 95 % of short variants, but only 50 % of amplifications.  Conversely, 63 % of alterations detected in ctDNA were also detected in matched tissue.  The authors noted that the limited accessibility of tumor tissue in advanced cancer patients represents a significant clinical challenge.  Blood-derived ctDNA may provide an alternative approach for genomic profiling in cases where tissue biopsy is prohibitive, and ctDNA testing may have the additional advantage of identifying heterogenous alterations not present in a single tumor site; however, the clinical implications of detection of such alterations requires further investigation.  ctDNA testing is currently limited relative to tissue testing in the detection of gene amplification, and currently available ctDNA assays are typically less comprehensive than available tissue-based assays.  They concluded that genomic profiling of ctDNA detected potentially clinically relevant GAs in a significant subset of patients with GI carcinomas.  In these tumor types, most alterations detected in matched tissue were also detected in ctDNA, and with the exception of amplifications, ctDNA sequencing routinely detected additional alterations not found in matched tissue, consistent with tumor heterogeneity.  These researchers stated that these findings suggested feasibility and utility of ctDNA testing in advanced GI cancers as a complementary approach to tissue testing, and further investigation is needed.\n",
      "Schrock and colleagues (2019) noted that genomic profiling informs selection of matched targeted therapies as part of routine clinical care in NSCLC.  Tissue biopsy is the criterion standard; however, genomic profiling of blood-derived ctDNA has emerged as a minimally invasive alternative.  These researchers evaluated hybrid capture-based genomic profiling of 62 genes on blood-based ctDNA from 1,552 patients with NSCLC.  Evidence of ctDNA was detected in 80 % of samples, and in 86 % of these cases, at least 1 reportable GA was detected.  Frequently altered genes were tumor protein p53 gene (TP53) (59 %), EGFR (25 %), and KRAS (17 %).  Comparative analysis with a tissue genomic database (n = 21,500) showed similar frequencies of GAs per gene, although KRAS mutation and EGFR T790M were more frequent in tissue and ctDNA, respectively (both p < 0.0001), likely reflecting the use of liquid versus tissue biopsy after relapse during targeted therapy.  In temporally matched ctDNA and tissue samples from 33 patients with evidence of ctDNA in their blood, 64 % of GAs detected in tissue were also detected in ctDNA, including 78 % of short variants (58 of 74) and 100 % of re-arrangements (4 of 4), but only 16 % of amplifications (4 of 25).  The authors concluded that genomic profiling of ctDNA detected clinically relevant GAs in a significant subset of NSCLC cases.  Most alterations detected in matched tissue were also detected in ctDNA.  These results suggested the utility of ctDNA testing in advanced NSCLC as a complementary approach to tissue testing.  These researchers stated that blood-based ctDNA testing may be especially useful at the time of progression during targeted therapy.\n",
      "Commenting on the afore-mentioned study by Schrock et al (2019), Lissa and Robles (2019) stated that a growing number of NGS-based liquid biopsy panels indicated to guide personalized therapies are entering the market, and they are rapidly gaining clinical adoption.  Assay sensitivity can be variable in the detection of certain genomic alterations, and the potential for biased assessment and support for different therapy decisions based on the ctDNA platform used is worrisome.  Therefore, robust analytical standards will need to be established and independent comparison of NGS panels will be necessary to clarify the accuracy of this approach to inform patient treatment decisions.  The increased sensitivity of NGS-based liquid biopsy panels also comes with a potential drawback.  Unexpected GAs have been found at low allele frequencies in ctDNA that are not likely derived from the tumor but instead related to clonal hematopoiesis, an aging related gain of somatic mutations in blood cells that can cause false-positive results if not carefully screened.  The authors concluded that comprehensive genomic profiling of ctDNA has the potential to improve the clinical management of patients with advanced NSCLC.  However, detailed and carefully conducted retrospective analyses of tissues for which the genomic profiling has proven clinical utility, or prospective clinical trials, will be needed to evaluate whether a test actually improves patient outcomes.\n",
      "Envisia Genomic Classifier\n",
      "Raghu et al (2019) stated that in the appropriate clinical setting, the diagnosis of idiopathic pulmonary fibrosis (IPF) requires a pattern of usual interstitial pneumonia to be present on high-resolution chest CT (HRCT) or surgical lung biopsy.  A molecular usual interstitial pneumonia signature can be identified by a machine learning algorithm in less-invasive transbronchial lung biopsy samples.  These investigators reported prospective findings for the clinical validity and utility of this molecular test.  They prospectively recruited 237 patients for this study from those enrolled in the Bronchial Sample Collection for a Novel Genomic Test (BRAVE) study in 29 U.S. and European sites.  Patients were undergoing evaluation for interstitial lung disease and had had samples obtained by clinically indicated surgical or transbronchial biopsy or cryobiopsy for pathology.  Histopathological diagnoses were made by experienced pathologists.  Available HRCT scans were reviewed centrally; 3 to 5 transbronchial lung biopsy samples were collected from all patients specifically for this study, pooled by patient, and extracted for transcriptomic sequencing.  After exclusions, diagnostic histopathology and RNA sequence data from 90 patients were used to train a machine learning algorithm (Envisia Genomic Classifier, Veracyte, San Francisco, CA) to identify a usual interstitial pneumonia pattern.  The primary study end-point was validation of the classifier in 49 patients by comparison with diagnostic histopathology.  To assess clinical utility, these researchers compared the agreement and confidence level of diagnosis made by central multi-disciplinary teams based on anonymized clinical information and radiology results plus either molecular classifier or histopathology results.  The classifier identified usual interstitial pneumonia in transbronchial lung biopsy samples from 49 patients with 88 % specificity (95 % confidence interval [CI]: 70 to 98) and 70 % sensitivity (47 to 87).  Among 42 of these patients who had possible or inconsistent usual interstitial pneumonia on HRCT, the classifier showed 81 % positive predictive value (PPV; 95 % CI: 54 to 96) for underlying biopsy-proven usual interstitial pneumonia.  In the clinical utility analysis, these investigators found 86 % agreement (95 % CI: 78 to 92) between clinical diagnoses using classifier results and those using histopathology data.  Diagnostic confidence was improved by the molecular classifier results compared with histopathology results in 18 with IPF diagnoses (proportion of diagnoses that were confident or provisional with high confidence 89 % versus 56 %, p = 0Â·0339) and in all 48 patients with non-diagnostic pathology or non-classifiable fibrosis histopathology (63 % versus 42 %, p = 0Â·0412).  The authors concluded that the molecular test provided an objective method to aid clinicians and multi-disciplinary teams in ascertaining a diagnosis of IPF, particularly for patients without a clear radiological diagnosis, in samples that can be obtained by a less invasive method.  Moreover, these researchers stated that further prospective clinical validation and utility studies are planned.\n",
      "Furthermore, an UpToDate review on \"Clinical manifestations and diagnosis of idiopathic pulmonary fibrosis\" (King, 2019) states that \"Future Directions -- An investigational technology has been developed that analyzes RNA sequence data of 190 genes from transbronchial biopsy samples to classify patients as having a molecular pattern of usual interstitial pneumonia (UIP) or not.  In a validation cohort of 49 patients with new onset interstitial lung disease, 3 to 5 transbronchial biopsies were obtained from each participant for RNA molecular analysis.  The analysis demonstrated 88 % sensitivity (95 % CI 70-98) and 70 % specificity (95 % CI 47-87) for differentiating UIP from non UIP.  Among patients with possible or inconsistent UIP on high resolution computed tomography (HRCT), the RNA molecular test showed a positive predictive value of 81 % (95 % CI 54-96) for biopsy-proven UIP.  Further study is needed to clarify whether the test can reduce the need for surgical biopsies in patients without a pattern of UIP on HRCT or aid in diagnosis when histopathology is not definitive.  It is important to remember that UIP can be found in multiple settings (e.g., hypersensitivity pneumonitis, rheumatoid arthritis, drug-induced lung toxicity), and this test does not distinguish among causes of UIP\".\n",
      "Biodesix BDX-XL2, Nodify Lung, Nodify CDT, or Nodify XL2 Test for Distinguishing Benign from Malignant Lung Nodules\n",
      "BDX-XL2 (Biodesix Inc) is a blood-based classifier designed to identify low-to-moderate risk patients with a likely benign lung nodule.  This classifier supposedly integrates plasma proteins with clinical risk factors associated with lung cancer. \n",
      "Biodesix also offers Nodify Lung testing which consists of two blood-based proteomic tests, the Nodify CDT and Nodify XL2, to help interpret the risk of malignancy of a lung nodule. The tests is obtained by means of a liquid biopsy on a simple blood draw. The Nodify XL2 test is an integrated classifier that measures the ratio of two proteins (LG3BP and C163A) combined with several clinical and radiological factors to help identify those nodules that are likely benign.\n",
      "Silvestri et al (2018) stated that lung nodules are a diagnostic challenge, with an estimated yearly incidence of 1.6 million in the United States.  In a multi-center, observational study, these researchers examined the accuracy of an integrated proteomic classifier in identifying benign nodules in patients with a pre-test probability of cancer (pCA) of less than or equal to 50 %.  This trial included 685 patients with 8- to 30-mm lung nodules.  Multiple reaction monitoring mass spectrometry was used to measure the relative abundance of 2 plasma proteins, LG3BP and C163A.  Results were integrated with a clinical risk prediction model to identify likely benign nodules.  Sensitivity, specificity, and negative predictive value (NPV) were calculated.  Estimates of potential changes in invasive testing had the integrated classifier results been available and acted on were made.  A subgroup of 178 patients with a clinician-assessed pCA of less than or equal to 50 % had a 16 % prevalence of lung cancer.  The integrated classifier demonstrated a sensitivity of 97 % (confidence interval [CI]: 82 to 100), a specificity of 44 % (CI: 36 to 52), and a NPV of 98 % (CI: 92 to 100) in distinguishing benign from malignant nodules.  The classifier performed better than PET, validated lung nodule risk models, and physician cancer probability estimates (p < 0.001).  If the integrated classifier results were used to direct care, 40 % fewer procedures would be performed on benign nodules, and 3 % of malignant nodules would be mis-classified.  The authors concluded that when used in patients with lung nodules with a pCA of less than or equal to 50 %, the integrated classifier accurately identified benign lung nodules with good performance characteristics.  If used in clinical practice, invasive procedures could be reduced by diverting benign nodules to surveillance.  Moreover, these researchers stated that further research is needed to examine the effect of incorporating this test into the diagnostic algorithm for nodule management in the hope of reducing unnecessary procedures in patients without cancer.\n",
      "The authors stated that the present study had several drawbacks.  First, the effect the plasma protein test might have had on test ordering was retrospectively analyzed; thus, a prospective study to assess changes in practice is needed.  Second, community practices were under-represented in this trial; however, based on a previous study, the likely benefit in terms of reduction of invasive testing would be significantly greater than that reported here.  Third, although there was precedent for reporting 1-year outcomes for stable nodules, traditionally, 2 years of nodule stability is what has been required to determine a nodule is benign.  Two-year follow-up data will be reported in the future once finalized.  Fourth, there were 88 patients without follow-up CT scan data at 1 year.  This degree of missing data may not be random, as patients with a low risk of lung cancer may have been less likely to be adherent with follow-up recommendations.\n",
      "An editorial by Al Nasrallah and Sears (2018) comment on the PANOPTIC trial conducted by Silvestri et al (2018). The authors state that the trial represents an important step in development of a molecular profile that will aid in classification of intermediate-risk nodules which will hopefully avoid unnecessary procedures, anxiety, and costs. However, we are not quite ready to put away the biopsy needle just yet. The authors report that âthis well-conducted clinical validation study of two plasma biomarkers [LG3BP and C163A] used in conjunction with well-defined clinical risk factors is a necessary step before determination of clinical utility. There are many lung cancer biomarkers at various stages of development, few of which have progressed to the point of clinical validation. Advantages of this integrated classifier are that the serum biomarkers are easily obtained, the clinical characteristics are readily available, and it could be combined with other risk stratification characteristics available now and in the futureâ.\n",
      "Ostrin et al (2020) noted that lung cancer is the leading worldwide cause of cancer mortality, as it is often detected at an advanced stage.  Since 2011, low-dose CT scan-based screening has promised a 20 % reduction in lung cancer mortality.  However, effectiveness of screening has been limited by eligibility only for a high-risk population of heavy smokers and a large number of false positives generated by CT.  Biomarkers have tremendous potential to improve early detection of lung cancer by refining lung cancer risk, stratifying positive CT scans, and categorizing intermediate-risk pulmonary nodules.  Three biomarker tests (Early CDT-Lung, Nodify XL2, Percepta) have undergone extensive validation and are available to the clinician.  The authors discussed these tests, with their clinical applicability and limitations, current ongoing evaluation, and future directions for biomarkers in lung cancer screening and detection.  Moreover, these researchers stated that biomarkers are playing an emerging role in the early detection of lung cancer.  There are many potential roles for biomarkers, from risk stratification to classification of nodules detected incidentally or through low-dose screening programs.  Several multi-analyte biomarker panels are available and have shown performance in classification of indeterminate pulmonary nodules; however, these panels must be used in an appropriate clinical context.  Furthermore, there is still much work to be done on unfulfilled needs within and outside of CT-based screening.\n",
      "Ost (2021) stated that likelihood ratios (LRs) are a method to evaluate diagnostic test performance and assist in clinical decision-making.  While sensitivity and specificity are useful for binary tests, they cannot be directly applied to tests with greater than 2 possible test results.  LRs can be used for diagnostic tests with 2 or more possible test results and are also suitable for tests with continuous results.  These investigators reviewed the concepts of LRs and how they relate to sensitivity and specificity.  Practical examples from the pulmonary literature of how LRs are used to calculate post-test disease probabilities using Bayes' theorem were provided.  These included examples when there were 3 or more categorical test results that have distinct interpretations (e.g., cytology results from endobronchial ultrasound [US]) as well as continuous test results (e.g., computed tomography [CT] lymph node size and probability of metastasis).  These researchers also highlighted some problems, pitfalls, and misunderstandings regarding LRs in clinical practice.  They employed the example of how the Nodify XL2 test incorrectly calculated and applied LRs, which may lead to falsely low estimates of the probability of cancer in some pulmonary nodules.\n",
      "Tanner et al (2021) noted that the prospective PANOPTIC Trial was carried out at 33 sites to validate the clinical performance of the integrated classifier Nodify XL2 test (Biodesix Inc., Boulder, CO).  Details of trial design and test development have previously been published.  Briefly, patients age 40 years or older with a newly detected nodule 8 to 30 mm in diameter were included.  Cancer diagnosis was by histopathology, and benign diagnosis was made by histopathology, radiographic resolution, or stability.  The 1-year analysis population included 178 patients.  Imaging data at 2 years minimum was available from 161 patients for this report.  The presence and total number of nodules were noted from radiology reports and collected on case report forms.  Of the 392 patients included in the PANOPTIC Trial, 178 had a physician-assessed probability for malignancy (pCA) of at least 50 %.  In patients with a pCA of less than or equal to 50 %, 149 (84 %) were benign at year 1.  At year 2, 10 patients were lost to follow-up, and 7 had final visits that did not extend to 2 years.  These 17 excluded patients had been categorized as benign at the 1-year interval, reducing the number of patients with benign nodules to 132 at the 2-year interval and leaving a total of 161 patients (90 %) with data available for analysis.  All nodules designated as benign at year 1 remained benign by imaging (e.g., stable or resolved) at year 2 with no change in pathologic diagnoses or nodule size by CT.  Patients included in the trial could have more than 1 nodule on imaging studies, with the most suspicious selected as the index lesion.  In the intended use group (n = 178), 101 (57 %) had multiple nodules on CT imaging.  There were on average 3 nodules (range of 1 to 10), and more than 4 indicated a shift to a higher probability of a benign nodule.  Patients with multiple nodules were older than those with a single nodule (p = 0 .002).  There was no significant difference in classifier performance between those with multiple nodules compared with those with 1 nodule (p = 0.164).  These researchers stated that this study, although not powered to be conclusive, suggested that the classifier performed similarly regardless of nodule number.  Furthermore, more than 4 or 5 nodules has been shown to be an indicator of a benign process, and the observational data reported in this trial were supportive.\n",
      "Furthermore, National Comprehensive Cancer Networkâs clinical practice guideline on âNon-small cell lung cancerâ (Version 1.2023) does not mention the use of âlikelihood ratiosâ or \"proteomic\" tests as a management tool.\n",
      "The American Thoracic Society (ATS) Core Curriculum provides a 2021 Pulmonary Core Curriculum update (Garrison et al, 2021) which focuses on lung cancer including risks and prevention, screening, nodules, therapeutics and associated pulmonary toxicities, and malignant pleural effusions. The authors, Ngoc-Tram Ha and Van K. Holden, state that âCurrent risk stratification models categorize nodules into low, intermediate, and high malignancy risk on the basis of the clinician pretest probability of malignancy (pCA). Use of these models can guide follow-up and management, but careful selection and application of available calculators are required. Models may not have been validated outside of the original patient population and may apply only to solid nodulesâ. The authors go on to state that âRecent studies have focused on molecular biomarkers to further risk-stratify nodules, but additional clinical utility studies are neededâ. An example includes integrated plasma proteomic classifier (two plasma proteins and five clinical risk factors) which have been evaluated in patients with undiagnosed 8- to 30-mm lung nodules and a low-to-moderate pCA of less than or equal to 50%, for which a âlikely benignâ result accurately identified patients with benign nodules for up to 2 years of follow-up.\n",
      "Myriad myPath Melanoma\n",
      "Clarke et al (2015) noted that histopathologic examination is sometimes inadequate for accurate and reproducible diagnosis of certain melanocytic neoplasms.  As a result, more sophisticated and objective methods have been sought.  These researchers identified a gene expression signature that reliably differentiated benign and malignant melanocytic lesions and examined its potential clinical applicability.  They described the development of a gene expression signature and its clinical validation using multiple independent cohorts of melanocytic lesions representing a broad spectrum of histopathologic subtypes.  Using quantitative reverse-transcription polymerase chain reaction (RT-PCR) on a selected set of 23 differentially expressed genes, and by applying a threshold value and weighting algorithm, these investigators developed a gene expression signature that produced a score that differentiated benign nevi from malignant melanomas.  The gene expression signature classified melanocytic lesions as benign or malignant with a sensitivity of 89 % and a specificity of 93 % in a training cohort of 464 samples.  The signature was validated in an independent clinical cohort of 437 samples, with a sensitivity of 90 % and specificity of 91 %.  The authors concluded that the performance, objectivity, reliability and minimal tissue requirements of this test suggested that it could have clinical application as an adjunct to histopathology in the diagnosis of melanocytic neoplasms.  Moreover, these researchers stated that additional outcomesâbased and prospective studies are needed to examine the performance of this test.\n",
      "The authors stated that 1 potential drawback of this study was that it was performed with archived formalinâfixed paraffinâembedded (FFPE) tissue.  mRNA extracted from archived FFPE samples is more prone to fragmentation when compared with recently prepared FFPE lesions.  This might explain the relatively high sample failure rate of the signature in older samples . Indeed, these investigators observed a much lower failure rate in contemporary samples.  Samples with low scores had an increased chance of failure if the mRNA was from an older sample and was degraded, which could bias estimates of the sensitivity and specificity.  To mitigate this bias, both the training and validation cohorts consisted of a variety of older archival samples and newer contemporary samples.\n",
      "Clarke et al (2017) stated that recently, a 23-gene signature was developed to produce a melanoma diagnostic score capable of differentiating malignant and benign melanocytic lesions.  These researchers examined the ability of the gene signature to differentiate melanoma from benign nevi in clinically relevant lesions.  A set of 1,400 melanocytic lesions was selected from samples prospectively submitted for gene expression testing at a clinical laboratory.  Each sample was tested and subjected to an independent histopathologic evaluation by 3 experienced dermato-pathologists.  A primary diagnosis (benign or malignant) was assigned to each sample, and diagnostic concordance among the 3 dermato-pathologists was required for inclusion in analyses.  The sensitivity and specificity of the score in differentiating benign and malignant melanocytic lesions were calculated to examine the association between the score and the pathologic diagnosis.  The gene expression signature differentiated benign nevi from malignant melanoma with a sensitivity of 91.5 % and a specificity of 92.5 %.  The authors concluded that these findings reflected the performance of the gene signature in a diverse array of samples encountered in routine clinical practice.  These researchers stated that additional studies with clinical followâup will likely provide additional insight into the performance of this test, as will studies focusing on especially challenging subtypes such as desmoplastic melanoma, nevoid melanoma, and Spitzoid melanoma.\n",
      "The authors stated that the gene signature assessed was intended to provide adjunctive information for the diagnosis of melanoma in ambiguous and difficultâtoâdiagnose lesions.  The prospective cohort used in this study included numerous melanoma and nevus subtypes, including some types known to present significant diagnostic challenges in the clinical setting.  However, the use of a tripleâconcordant diagnostic reference standard did eliminate some of these cases.  For example, desmoplastic and nevoid melanomas were not specifically excluded from the cohort, but none of the cases classified as either of these 2 particular melanoma subtypes received a tripleâconcordant diagnosis by histopathology.  The high frequency of discordance among reviewing dermato-pathologists in this study was similar to that observed in other assessments of clinical cohorts; and highlighted the need for adjunctive diagnostic tools.  An evaluation of the gene signature against clinical outcomes would minimize cohort bias toward straightforward cases, and studies assessing test performance by comparison with clinical outcomes are currently underway.\n",
      "Ko et al (2017) noted that histopathologic examination alone can be inadequate for diagnosis of certain melanocytic neoplasms.  Recently, a 23-gene expression signature was clinically validated as an ancillary diagnostic test to differentiate benign nevi from melanoma.  These researchers examined the performance of this test in an independent cohort of melanocytic lesions against clinically proven outcomes.  Archival tissue from primary cutaneous melanomas and melanocytic nevi was obtained from 4 independent institutions and tested with the gene signature.  Cases were selected according to pre-defined clinical outcome measures.  Malignant lesions were defined as stage I to III primary cutaneous melanomas that produced distant metastases (metastatic to sites other than proximal sentinel lymph node(s)) following diagnosis of the primary lesion.  Melanomas that were metastatic at the time of diagnosis, all re-excisions, and lesions with less than 10 % tumor volume were excluded.  Benign lesions were defined as cutaneous melanocytic lesions with no adverse long-term events reported.  Of 239 submitted samples, 182 met inclusion criteria and produced a valid gene expression result.  This included 99 primary cutaneous melanomas with proven distant metastases and 83 melanocytic nevi.  Median time to melanoma metastasis was 18 months.  Median follow-up time for nevi was 74.9 months.  The gene expression score differentiated melanoma from nevi with a sensitivity of 93.8 % and a specificity of 96.2 %.  The authors concluded that multiple lines of evidence suggested that the gene expression signature differentiated benign and malignant melanocytic lesions with a high degree of accuracy.  The findings of this study showed that the gene signature had high diagnostic accuracy relative to long-term clinical outcomes.  In combination with previous validations that used consensus histopathologic diagnosis as the reference standard, this supports the use of the gene signature as an adjunctive diagnostic test to enable the early and accurate diagnosis of melanoma.  Additional studies on specific melanoma and nevus subtypes are ongoing and will provide additional insight regarding the performance characteristics of the test.\n",
      "Ko et al (2019) stated that a 23-gene expression signature was recently developed as an adjunct to histopathology to differentiate melanocytic nevi from melanoma.  The current study correlated the gene expression signature scores to actual clinical outcomes in cases from the first validation study.  RNA was extracted from 127 archival FFPE tissue sections of melanocytic lesions.  Gene expression was measured using quantitative RT-PCR, and a weighting algorithm was used to generate a numeric score.  Gene expression test results were compared to histopathological diagnoses and development of local recurrence, sentinel lymph node metastases, and distant metastases.  A total of 65 lesions were diagnosed histopathologically as melanoma -- 14 developed metastases.  Gene expression test results were malignant in 61 of 65 (93.8 %) lesions (including all lesions that metastasized), indeterminate in 2 of 65 (3.1 %) lesions, and benign in 2 of 65 (3.1 %) lesions.  The remaining 62 lesions were diagnosed as benign by histopathology.  Gene expression test results were benign in 48 of 62 (77.4 %), indeterminate in 7 of 62 (11.3 %), and malignant in 7 of 62 (11.3 %).  The authors concluded that there was a strong correlation between the gene expression signature test results and clinical outcomes.  All lesions that metastasized were correctly identified by the test as malignant melanoma.\n",
      "The authors stated that this study had several drawbacks.  First, none of the lesions in this cohort was classified as severely dysplastic nevi, nor were any of the 6 Spitz nevi considered âatypicalâ by the participating dermato-pathologists.  Performance of the assay in these and other subtypes that may have âborderlineâ biological potential therefore warrants further study.  Second, although all of the cases that metastasized were apparently correctly diagnosed histopathologically by the submitting dermato-pathologists, a rigorous analysis of their potential for ambiguity (such as a blinded review by 5 or more dermato-pathologists to calculate potential discordance) was not undertaken.  As such, the potential for these lesions to generate diagnostic discordance or be considered ambiguous by histopathologic interpretation is unknown.\n",
      "Castillo et al (2019) stated that melanoma ex blue nevus (MEBN) is a rare, aggressive, and potentially lethal neoplasm.  Distinguishing MEBN from an atypical cellular blue nevus can be very challenging.  These researchers reported a diagnostically difficult case of MEBN with lymph node metastases, in which single nucleotide polymorphism (SNP) array and fluorescence in-situ hybridization (FISH) were used to arrive at the correct diagnosis.  It was also analyzed by the recently-introduced proprietary 23-gene expression signature test (myPath melanoma).  The authors concluded that to the best of their knowledge, this was the 2nd reported case of MEBN analyzed by the 23-gene expression signature, which provided a false-negative result.  These researchers stated that more studies are needed to assess the sensitivity and specificity of this test in various melanocytic proliferations.\n",
      "Fernandez-Flores and Cassarino (2019) noted that chondroid stromal change is very rare in melanocytic nevi.  These investigators presented a severely atypical symmetrical Spitz tumor on the right arm of a 24-years old man.  It showed maturation and occasional mitotic figures.  The melanocytes were intermingled with cartilaginous stroma.  The Ki67 proliferative rate was 5 %.  It was weakly positive for p16 and negative for BRAF/V600E.  BAP1 expression was preserved; and PCR for Myriad myPathÂ® Melanoma test was also negative.\n",
      "Clarke et al (2020) examined the accuracy of a 23-gene expression signature in differentiating benign nevi from melanoma by comparing test results with clinical outcomes.  A total of 7 dermato-pathologists blinded to gene expression test results and clinical outcomes examined 181 lesions to identify diagnostically uncertain cases.  Participants independently recorded diagnoses and responses to questions quantifying diagnostic certainty.  Test accuracy was determined through comparison with clinical outcomes (sensitivity and percent negative agreement).  A total of 125 cases fulfilled criteria for diagnostic uncertainty (69.1 %; 95 % CI: 61.8 % to 75.7 %).  Test sensitivity and percent negative agreement in these cases were 90.4 % (95 % CI: 79.0 % to 96.8 %) and 95.5 % (95 % CI: 87.3 % to 99.1 %), respectively.  The authors concluded that the 23-gene expression signature had high diagnostic accuracy in diagnostically uncertain cases when evaluated against clinical outcomes.  These researchers stated that by aiding in improved diagnostic accuracy for this subset of lesions, the test may allow physicians and patients to more confidently develop appropriate treatment plans.\n",
      "The authors stated that the results of this study should be interpreted within the context of its limitations, most of which were common to investigations of this type. The lack of access to complete clinical and demographic information and reliance on a single diagnostic slide were study parameters that may differ from the typical clinical environment.  However, many melanocytic neoplasms encountered in the clinical setting were represented by a single slide accompanied by only the patient age, gender and anatomic location of the lesion.  The study objective was to identify cases that were uncertain by histopathologic interpretation, and clinical information beyond that provided in the study was rarely necessary when the histopathologic findings were definitive.  Furthermore, since even a diagnosis made via expert consensus may not accurately predict the biological behavior of some histopathologically ambiguous melanocytic neoplasms, documented clinical outcomes were an inclusion criterion for this study.  Distant metastasis was required as evidence of malignancy because it is the most definitive and objective indication that a primary melanocytic neoplasm was indeed malignant melanoma (sentinel lymph node involvement alone is sometimes controversial as an indication of malignancy for atypical Spitz tumors).  Adverse event (AE)-free follow-up was needed as evidence of benignity.  While the absence of metastasis, regardless of follow-up duration, did not conclusively prove a lesion benign, it was nonetheless supportive of a benign diagnosis, particularly for larger lesions.  Thus, while no reference standard is perfect, long-term clinical outcome is the most definitive means of determining the true nature (benign or malignant) of melanocytic neoplasms that are uncertain by histopathologic interpretation.  The 7 participants recruited for this study were experienced dermato-pathologists, all of whom expressed high confidence in their diagnostic ability and 6 of whom serve as expert consultants for the histopathologic interpretation of unusual or difficult melanocytic neoplasms; thus, it was possible that dermato-pathologists with less experience or confidence would consider a greater percentage of the study cases diagnostically uncertain.  As discussed above, clinical outcome was the only independent indication of a melanocytic neoplasmâs true nature (benign or malignant) and the most appropriate reference standard for cases that are uncertain by histopathologic interpretation.  However, while metastasis proves malignancy, the absence of metastases does not prove benignity since not all melanomas metastasize and some may be âcuredâ by removal prior to metastasis.\n",
      "Furthermore, National Comprehensive Cancer Networkâs clinical practice guideline on \"Cutaneous melanoma\" (Version 1.2020) states that \" Prognostic gene expression profiling (GEP) to differentiate melanomas at low versus high risk for metastasis may provide information on individual risk of recurrence, as an adjunct to standard AJCC staging.  However, the currently available prognostic molecular techniques should not replace pathologic staging procedures, and the use of GEP testing according to specific melanoma state (before or after SLNB) requires further prospective investigation in large, contemporary data sets of unselected patients\".\n",
      "Tschen et al (2021) stated that gene expression signature has been validated as an adjunct to traditional methods of differentiating malignant and benign melanocytic neoplasms, and its use in clinical practice warrants further study.  This study followed patients whose melanocytic neoplasms were managed according to a benign result from the gene expression signature (n = 25).  Eligible patients whose tested lesions were classified as benign by the gene expression signature and were subsequently treated as benign by their dermatology providers were observed for a mean follow-up period of 38.5 months.  Results suggested that many patients with melanocytic neoplasms classified as benign by the gene expression signature may safely forego additional surgical excision.  The authors concluded that ancillary methods are emerging as useful tools for the diagnostic evaluation of melanocytic neoplasms that cannot be assigned definitive diagnoses using traditional techniques alone.  The findings of this study suggested that patients with ambiguous melanocytic neoplasms may benefit from diagnoses and treatment decisions aligned with the results of a gene expression test, and that for those with a benign result, simple observation may be a safe alternative to surgical excision.  This expanded upon prior observations of the testâs influence on diagnoses and treatment decisions and supported its role as part of dermatopathologistsâ and dermatologistsâ decision-making process for histopathologically ambiguous melanocytic lesions.\n",
      "The authors stated that this study had several drawbacks.  Obtaining meaningful patient outcome data is a common challenge in healthcare research due to the requisite length of follow-up and sometimes the lack of definitive evidence of AEs.  This is especially difficult for melanocytic neoplasms because of an apparent inclination for patients with benign diagnoses to abandon follow-up and an increasing tendency for even minimal diagnostic uncertainty to prompt complete excision.  Furthermore, the only definitive clinical outcome for melanocytic neoplasms is distant metastasis, which (fortunately for patients) is relatively rare.  Not surprisingly, studies documenting clinical outcomes of patients with ambiguous melanocytic neoplasms tested prospectively with diagnostic adjuncts were scarce, and this studyâs sample size and clinical follow-up compared favorably with the few that exist.  Although most melanomas reveal themselves via recurrence or metastasis within several years of initial biopsy, some are clinically dormant for as long as 10 years following initial detection.  This may be especially true for the small or early-stage lesions that now comprise the majority of biopsied neoplasms, and such events would go undetected by this study and many others.  It also must be recognized that uneventful follow-up, regardless of duration, cannot prove that a biopsied melanocytic neoplasm was benign.  Although only 5 patients had a follow-up time of less than 2 years (the time frame in which most recurrence or metastasis will occur), it cannot be definitively proven that a minimum of 2 years recurrence- or metastasis-free survival indicated a benign lesion.  Many early-stage malignant melanomas are eradicated by complete excision or even by the initial biopsy if margins are uninvolved.  Moreover, these investigators stated that because these limitations are intrinsic to melanocytic neoplasms and current management strategies, they pertain to all investigations seeking insights into biological potential via clinical outcomes.  Similarly, all current diagnostic tools and procedures have the potential for sampling error, including histopathology.  The rarity of adverse outcomes (recurrence and metastasis) in patients with benign test results within this cohort indicated that false-negative results are uncommon, which is further evidenced by a similar rarity of AEs in prior studies of the gene expression signature.\n",
      "These investigators noted that it must be emphasized that the gene expression test, similar to other diagnostic adjuncts, is neither a replacement for histopathologic interpretation nor a substitute for judgment.  As with all tests, it can produce false-positive and false-negative results; thus, it should always be interpreted within the constellation of the many other data points that must be considered when making a distinction between benign nevus and malignant melanoma, including but not limited to patient age, family and personal history of melanoma, anatomic location, clinical features, and histopathologic findings.  As is the case for many diseases, careful consideration of all relevant input is needed to minimize the risk of misdiagnosis that might occur should any single data point prove inaccurate, including the results of adjunctive molecular tests.\n",
      "NantHealth GPS Cancer Panels\n",
      "NantHealth GPS Cancer is a unique molecular test that integrates quantitative targeted proteomics detected by mass spectrometry with whole genome (DNA) and whole transcriptome (RNA) sequencing, of both normal and cancer tissue.  Thus, it not only analyzes the entire DNA sequence, but also the RNA sequence and proteins that are produced.  However, there is a lack of evidence that data generated by this test would alter management of the cancers for which the testing is indicated such that clinical outcomes are improved. \n",
      "ProMark\n",
      "National Comprehensive Cancer Networkâs clinical practice guideline on \"Prostate cancer\" (Version 4.2019) recommends coverage of  \"Decipher\", \"Oncotype DX Prostate\", \"Prolaris\", and \"ProMark\" (Category 2A).  ProMark: For post-biopsy based on NCCN very-low- and low-risk patients with greater than 10 year life expectancy who have not received treatment for prostate cancer and are candidates for active surveillance or definitive therapy.\n",
      "Insight TNBCtype\n",
      "Prat et al (2015) noted that predicting treatment benefit and/or outcome before any therapeutic intervention has taken place would be clinically very useful.  These researchers examined the ability of the intrinsic subtypes and the risk of relapse score at diagnosis to predict survival and response following neoadjuvant chemotherapy.  In addition, they evaluated the ability of the Claudin-low and 7-TNBCtype classifications to predict response within triple-negative breast cancer (TNBC).  Gene expression and clinical-pathological data were evaluated in a combined dataset of 957 breast cancer patients, including 350 with TNBC, treated with sequential anthracycline and anti-microtubule-based neoadjuvant regimens.  Intrinsic subtype, risk of relapse score based on subtype and proliferation (ROR-P), the Claudin-low subtype and the 7-TNBCtype subtype classification were evaluated.  Logistic regression models for pathological complete response (pCR) and Cox models for distant RFS (DRFS) were used.  Basal-like, Luminal A, Luminal B, and HER2-enriched subtypes represented 32.7 %, 30.6 %, 18.2 %, and 10.3 % of cases, respectively.  Intrinsic subtype was independently associated with pCR in all patients, in hormone receptor-positive/HER2-negative disease, in HER2-positive disease, and in TNBC.  The pCR rate of Basal-like disease was greater than 35 % across all clinical cohorts.  Neither the Claudin-low nor the 7-TNBCtype subtype classifications predicted pCR within TNBCs after accounting for intrinsic subtype.  Finally, intrinsic subtype and ROR-P provided independent prognostic information beyond clinicopathological variables and type of pathological response.  A 5-year DRFS of 97.5 % (92.8 to 100.0 %) was observed in these neoadjuvant-treated and clinically node-negative patients predicted to be low risk by ROR-P (i.e., 57.4 % of Luminal A tumors with clinically node-negative disease).  The authors concluded that intrinsic subtyping at diagnosis provided prognostic and predictive information for patients receiving neoadjuvant chemotherapy.  Although these researchers could not exclude a survival benefit of neoadjuvant chemotherapy in patients with early breast cancer with clinically node-negative and ROR-low disease at diagnosis, the absolute benefit of cytotoxic therapy in this group might be rather small (if any).  These researchers stated that further studies are needed to determine the role of intrinsic subtyping in treatment decision-making at diagnosis of breast cancer.\n",
      "The authors stated that this study had several drawbacks.  First, this was a retrospective and exploratory analysis of 4 datasets of patients treated with multi-agent chemotherapy; therefore, these investigators did not test a pre-specified hypothesis.  Second, these researchers used the research-based version of the PAM50 assay and not the standardized version that is currently commercially available.  Third, they could not evaluate the predictive ability of the intrinsic subtypes to specific regimens or schedules.  Fourth, they used the pathological data as provided in each publication and different definitions and cut-offs might have been used to determine the positivity of each biomarker.  Thus, the results might have differed if ER, PR, and HER2 status had been centrally confirmed.  Nonetheless, the authors and others have reported that, even within centrally confirmed TNBC, all the intrinsic molecular subtypes can be identified.  Fifth, Ki-67 by IHC was not available in any of the 4 datasets and thus these investigators could not explore the ability of this biomarker to predict pCR following chemotherapy or survival outcome in the presence of the intrinsic subtypes or histological grade, especially within HR+/HER2â disease.  Sixth, the survival outcomes were only available in one of the data-sets evaluated.  Finally, the cut-offs to define the 3 risk groups of ROR-P were based on a large node-negative cohort that did not receive adjuvant systemic therapy.  These cut-offs might differ from the current standardized PAM50 version that took into account tumor size and that defined the low-risk group as those patients with a risk of distant relapse at 10-years below 3 %.\n",
      "Ring et al (2016) noted that recently, a gene expression algorithm, TNBCtype, was developed that can divide TNBC)into molecularly-defined subtypes.  The algorithm has potential to provide predictive value for TNBC subtype-specific response to various treatments.  TNBCtype used in a retrospective analysis of neoadjuvant clinical trial data of TNBC patients demonstrated that TNBC subtype and pathological complete response to neoadjuvant chemotherapy were significantly associated.  These researchers described an expression algorithm reduced to 101 genes with the power to subtype TNBC tumors similar to the original 2,188-gene expression algorithm and predicted patient outcomes.  The new classification model was built using the same expression data sets used for the original TNBCtype algorithm.  Gene set enrichment followed by shrunken centroid analysis were used for feature reduction, then elastic-net regularized linear modeling was used to identify genes for a centroid model classifying all subtypes, comprised of 101 genes.  The predictive capability of both this new \"lean\" algorithm and the original 2,188-gene model were applied to an independent clinical trial cohort of 139 TNBC patients treated initially with neoadjuvant doxorubicin/cyclophosphamide and then randomized to receive either paclitaxel or ixabepilone to determine association of pathologic complete response within the subtypes.  The new 101-gene expression model reproduced the classification provided by the 2,188-gene algorithm and was highly concordant in the same set of 7 TNBC cohorts used to generate the TNBCtype algorithm (87 %), as well as in the independent clinical trial cohort (88 %), when cases with significant correlations to multiple subtypes were excluded.  Clinical responses to both neoadjuvant treatment arms, found BL2 to be significantly associated with poor response (OR = 0.12, p = 0.03 for the 2,188-gene model; OR = 0.23, p < 0.03 for the 101-gene model).  Additionally, while the BL1 subtype trended towards significance in the 2,188-gene model (OR = 1.91, p = 0.14), the 101-gene model demonstrated significant association with improved response in patients with the BL1 subtype (OR = 3.59, p = 0.02).  The authors concluded that these findings demonstrated that a model using small gene sets could recapitulate the TNBC subtypes identified by the original 2,188-gene model and in the case of standard chemotherapy, the ability to predict therapeutic response.  Moreover, these researchers stated that additional studies are planned comparing both models on randomized clinical trial samples to fully explore the utility of models to identify responsive patient populations.\n",
      "Lehmann et al (2016) stated that TNBC is a heterogeneous disease that can be classified into distinct molecular subtypes by gene expression profiling.  Considered a difficult-to-treat cancer, a fraction of TNBC patients benefit significantly from neoadjuvant chemotherapy and have far better OS.  Outside of BRCA1/2 mutation status, biomarkers do not exist to identify patients most likely to respond to current chemotherapy; and, to-date, no FDA-approved targeted therapies are available for TNBC patients.  Previously, these researchers developed an approach to identify 6molecular subtypes TNBC (TNBCtype), with each subtype displaying unique ontologies and differential response to standard-of-care chemotherapy.  Given the complexity of the varying histological landscape of tumor specimens, these investigators used histopathological quantification and laser-capture microdissection to determine that transcripts in the previously described immunomodulatory (IM) and mesenchymal stem-like (MSL) subtypes were contributed from infiltrating lymphocytes and tumor-associated stromal cells, respectively.  Thus, these researchers refined TNBC molecular subtypes from 6 (TNBCtype) into 4 (TNBCtype-4) tumor-specific subtypes (BL1, BL2, M and LAR) and demonstrated differences in diagnosis age, grade, local and distant disease progression and histopathology.  Using 5 publicly available, neoadjuvant chemotherapy breast cancer gene expression data-sets, these investigators retrospectively evaluated chemotherapy response of over 300 TNBC patients from pre-treatment biopsies subtyped using either the intrinsic (PAM50) or TNBCtype approaches.  Combined analysis of TNBC patients demonstrated that TNBC subtypes significantly differed in response to similar neoadjuvant chemotherapy with 41 % of BL1 patients achieving a pCR compared to 18 % for BL2 and 29 % for LAR with 95 % CIs ([33 to 51], [9 to 28], [17 to 41], respectively).  The authors provided pre-clinical data that could inform clinical trials designed to test the hypothesis that improved outcomes can be achieved for TNBC patients, if selection and combination of existing chemotherapies was directed by knowledge of molecular TNBC subtypes.\n",
      "Guo et al (2018) stated that TNBC is an operational term for breast cancers lacking targetable estrogen receptor expression and HER2 amplifications.  Thus, TNBC is inherently heterogeneous, and is associated with worse prognosis, greater rates of metastasis, and earlier onset.  TNBC displays mutational and transcriptional diversity, and distinct mRNA transcriptional subtypes exhibiting unique biology.  High-throughput sequencing has extended cancer research far beyond protein coding regions that include non-coding small RNAs, such as miRNA, isomiR, tRNA, snoRNAs, snRNA, yRNA, 7SL, and 7SK.  These researchers performed small RNA profiling of 26 TNBC cell lines, and compared the abundance of non-coding RNAs among the transcriptional subtypes of TNBC.  They also examined their co-expression pattern with corresponding mRNAs.  The authors provided a detailed description of small RNA expression in TNBC cell lines that could aid in the development of future biomarker and novel targeted therapies.\n",
      "Funakoshi et al (2019) stated that inflammatory breast cancer (IBC) is an aggressive form of breast cancer.  The triple-negative subtype of IBC (TN-IBC) is particularly aggressive.  Identification of molecular differences between TN-IBC and TN-non-IBC may help clarify the unique clinical behaviors of TN-IBC.  However, the authorsâ previous study comparing gene expression between TN-IBC and TN-non-IBC did not identify any TN-IBC-specific molecular signature.  Lehmann et al (2016) recently reported that the mesenchymal stem-like (MSL) TNBC subtype consisted of infiltrating tumor-associated stromal cells but not cancer cells.  Thus, these investigators compared the gene expression profiles between TN-IBC and TN-non-IBC patient samples not of the MSL subtype.  They classified 88 TNBC samples from the World IBC Consortium into subtypes according to the Vanderbilt classification and Insight TNBCtype, removed samples of MSL and unstable subtype, and compared gene expression profiles between the remaining TN-IBC and TN-non-IBC samples.  In the Vanderbilt analysis, these researchers identified 75 genes significantly differentially expressed between TN-IBC and TN-non-IBC at an FDR of 0.2.  In the Insight TNBCtype analysis, they identified 81 genes significantly differentially expressed between TN-IBC and TN-non-IBC at an FDR of 0.4.  In both analyses, the top canonical pathway was \"Fc Receptor-mediated Phagocytosis in Macrophages and Monocytes\", and the top 10 differentially regulated genes included PADI3 and MCTP1, which were up-regulated, and CDC42EP3, SSR1, RSBN1, and ZC3H13, which were down-regulated.  The authors concluded that these findings suggested that the activity of macrophages might be enhanced in TN-IBC compared with TN-non-IBC.  Moreover, these researchers stated that further pre-clinical and clinical studies are needed to determine the cross-talk between macrophages and IBC cells.\n",
      "MyMRD NGS Panel\n",
      "The MyMRD is a hotspot panel that detects all classes of variants identified in a precisely defined set of targets that commonly drive myeloid malignancies including AML, MPN and MDS.  It can detect single nucleotide variants (SNVs), indels and translocations to the genomic base-pair, giving unparalleled precision and detection of low level mutations in patients.  Testing with MyMRD allows for studying important mutations in known genes implicated in the causation, prognosis, and reoccurrence of myeloid disorders.\n",
      "Carbonell and associates (2019) stated that molecular diagnosis of myeloid neoplasms (MN) is based on the detection of multiple genetic alterations using various techniques.  These researchers analyzed diagnostic samples from 121 patients affected by MN and 10 relapse samples from a subset of AML patients using 2 enrichment-capture NGS gene panels.  Pathogenicity classification of variants was enhanced by the development and application of a custom onco-hematology score.  A total of 278 pathogenic variants were detected in 84 % of patients.  For structural alterations, 82 % of those identified by cytogenetics were detected by NGS, 25 of 31 copy number variants and 3 out of 3 translocations.  The detection of variants using NGS changed the diagnosis of 7 patients and the prognosis of 15 patients and enabled these investigators to identify 44 suitable candidates for clinical trials.  Regarding AML, 6 of the 10 relapsed patients lost or gained variants, comparing with diagnostic samples.  The authors concluded that the use of NGS panels in MN improved genetic characterization of the disease compared with conventional methods, thus demonstrating its potential clinical utility in routine clinical testing.\n",
      "Northrup and colleagues (2020) stated that the implementation of NGS in routine clinical hematology practice remains limited.  These researchers examined the clinical value of NGS in the screening, diagnosis, and follow-up in hematologic neoplasms.  A targeted NGS panel was used to assess a total of 178 patients for questionable or previously diagnosed myeloid neoplasms.  Gene variants were identified in 53 % of patients.  Novel variants were identified in 29 % of patients and variants of unknown significance in 34 %.  Bone marrow samples yielded a higher number of variants than in peripheral blood.  In several cases, NGS played a key role in the screening, diagnostics, prognostic stratification, and the clinical follow-up of a wide variety of myeloid neoplasms.  The authors concluded that NGS is an effective tool in the evaluation of suspected and confirmed hematologic neoplasms and could become part of the routine work-up of patients with hematologic neoplasms.\n",
      "Balagopal and associates (2019) stated that improved systems for detection of measurable residual disease (MRD) in AML are urgently needed, however attempts to utilize broad-scale NGS panels to perform multi-gene surveillance in AML post-induction have been stymied by persistent pre-malignant mutation-bearing clones.  These researchers hypothesized that this technology may be more suitable for evaluation of fully engrafted patients following hematopoietic cell transplantation (HCT).  To address this question, these investigators developed a hybrid-capture NGS panel utilizing unique molecular identifiers (UMIs) to detect variants at 0.1 % variant allelic frequency (VAF) or below across 22 genes frequently mutated in myeloid disorders and applied it to a retrospective sample set of blood and bone marrow DNA samples previously evaluated as negative for disease via standard-of-care short tandem repeat (STR)-based engraftment testing and hematopathology analysis in the authorsâ laboratory.  Of 30 patients who demonstrated trackable mutations in the 22 genes at eventual relapse by standard NGS analysis, they were able to definitively detect relapse-associated mutations in 18/30 (60 %) at previously disease-negative time-points collected 20 to 100 days prior to relapse date.  MRD was detected in both bone marrow (15/28, 53.6 %) and peripheral blood samples (9/18, 50 %), while showing excellent technical specificity in their sample set.  These researchers also confirmed the disappearance of all MRD signal with increasing time prior to relapse (greater than 100 days), indicating true clinical specificity, even using genes commonly associated with clonal hematopoiesis of indeterminate potential (CHIP).  The authors concluded that this study highlighted the efficacy of a highly sensitive, NGS panel-based approach to early detection of relapse in AML and supported the clinical validity of extending MRD analysis across many genes in the post-transplant setting.\n",
      "Aguilera-Diaz and co-workers (2020) stated that the diagnosis of MN has significantly evolved through the last few decades.  These researchers examined the performance of 4 different targeted NGS gene panels based on their technical features and clinical utility.  A total of 32 patient bone marrow samples were accrued and sequenced with 3 commercially available panels and 1 custom panel.  Variants were classified by 2 geneticists based on their clinical relevance in MN.  There was a difference in panel's depth of coverage.  These researchers found 11 discordant clinically relevant variants between panels, with a trend to miss long insertions.  These findings showed that showed that there was a risk of finding different mutations depending on the panel of choice.  This discordance was motivated by panel design and sequencing data analysis.  MN are genetically heterogeneous, therefore choosing a commercial NGS panel needs detailed study of its scope, to be aware of its limitations and to avoid missing the testing of genes relevant to a specific MN subtype.  Based on these findings, the characterization of some genetic regions (CEBPA, CALR, and FLT3) remains a challenge for NGS; this is a major issue, since AML and MPN management strongly depends on their correct detection.  In addition, NGS testing times are hard to harmonize with turnaround time established in current European LeukemiaNet guidelines.  Of note, CEBPA, CALR and FLT3 genes, remains challenging the use of NGS for diagnosis of MN in compliance with current guidelines.  The authors concluded that conventional molecular testing might need to be kept in place for the correct diagnosis of MN for now. \n",
      "Flach and colleagues (2020) stated that reciprocal RUNX1 fusions are traditionally found in up to 10 % of AML patients, usually associated with a translocation (8;21)(q22;q22) corresponding to the RUNX1-RUNX1T1 fusion gene.  So far, alternative RUNX1 re-arrangements have been reported only rarely in AML, and the few reports so far have focused on results based on cytogenetics, FISH, and PCR.  Acknowledging the inherent limitations of these diagnostic techniques, the true incidence of rare RUNX1 re-arrangements may be under-estimated.  These researchers presented 2 cases of adult AML, in which these investigators detected rare RUNX1 re-arrangements not by conventional cytogenetics but rather by NGS panel.  These included t(16;21)(q24;q22)/RUNX1-CBFA2T3 and t(7;21)(p22;q22)/RUNX1-USP42, respectively.  In both patients the AML was therapy-related and associated with additional structural and numerical alterations thereby conferring bad prognosis.  This was in line with previous reports on rare RUNX1 fusions in AML and emphasized the clinical importance of their detection.  The authors concluded that these findings not only confirmed the clinical utility of NGS for diagnostics of rare reciprocal re-arrangements in AML in a real-life scenario but also shed light on the variety and complexity within AML.\n",
      "National Comprehensive Cancer Networkâs clinical practice guideline on \"Acute myeloid leukemia\" (Version 3.2020) states that \"â¦ multiple gene panels and next-generation sequencing analysis are recommended for a comprehensive prognostic assessment\".\n",
      "National Comprehensive Cancer Networkâs clinical practice guideline on \"Myelodysplastic syndromes\" (Version 2.2020) states that \"Next-generation sequencing and chromosome genomic array testing are complementary in detecting both mutations and copy number aberrations and copy neutral loss of heterozygosity in the genes associated with these disorders\".\n",
      "National Comprehensive Cancer Networkâs clinical practice guideline on \"Myeloproliferative neoplasms\" (Version 3.2019) does not mention genetic panel / next-generation sequencing.\n",
      "OncoOmicDx Targeted Proteomic Assay\n",
      "Zhang and colleagues (2017) noted that in breast cancer, p53 could be functionally compromised by interaction with several proteins.  Among those proteins, MDM2 serves as a pivotal negative regulator and counteracts p53 activation.  Therefore, the ability to quantitatively and accurately monitor the changes in level of p53-MDM2 interaction with disease state can enable an improved understanding of this protein-protein interaction (PPI), provide a better insight into cancer development and allow the emergence of advanced treatments.  However, rare studies have evaluated the quantitative extent of PPI including p53-MDM2 interaction so far.  In this study, a LC-MS/MS-based targeted proteomics assay was developed and coupled with co-immunoprecipitation (Co-IP) for the quantification of p53-MDM2 complex.  A p53 antibody with the epitope residing at 156 to 214 residues achieved the greatest IP efficiency.  321KPLDGEYFTLQIR333 (p53) and 327ENWLPEDK334 (MDM2) were selected as surrogate peptides in the targeted analysis.  Stable isotope-labeled synthetic peptides were used as internal standards.  An LOQ (limit of quantification) of 2 ng/ml was obtained.  Then, the assay was applied to quantitatively detect total p53, total MDM2 and p53-MDM2 in breast cells and tissue samples.  Western blotting was performed for a comparison.  Finally, a quantitative time-course analysis in MCF-7 cells with the treatment of nutlin-3 as a PPI inhibitor was also monitored.  These researchers stated that liquid chromatography-tandem mass spectrometry (LC-MS/MS)-based targeted proteomics has shown its potential to study biomolecules because of its high sensitivity, high selectivity and wide dynamic range.  In this study, the author made an effort to develop a LC-MS/MS-based targeted proteomics assay for the quantitative detection of p53-MDM2 interaction in breast cells and tissue samples.\n",
      "Jiang and associates (2018) stated that abnormal expression of C-terminal p53 isoforms Î±, Î², and Î³ can cause the development of cancers including breast cancer.  To-date, much evidence has demonstrated that these isoforms can differentially regulate target genes and modulate their expression.  Therefore, quantification of individual isoforms may help to link clinical outcome to p53 status and to improve cancer patient treatment.  However, there are few studies on accurate determination of p53 isoforms, probably due to sequence homology of these isoforms and also their low abundance.  In this study, a targeted proteomics assay combining molecularly imprinted polymers (MIPs) and LC-MS/MS was developed for simultaneous quantification of C-terminal p53 isoforms.  Isoform-specific surrogate peptides (i.e., KPLDGEYFTLQIR (peptide-Î±) for isoform Î±, KPLDGEYFTLQDQTSFQK (peptide-Î²) for isoform Î², and KPLDGEYFTLQMLLDLR (peptide-Î³) for isoform Î³) were first selected and used in both MIPs enrichment and mass spectrometric detection.  The common sequence KPLDGEYFTLQ of these 3 surrogate peptides was used as single template in MIPs.  In addition to optimization of imprinting conditions and characterization of the prepared MIPs, binding affinity and cross-reactivity of the MIPs for each surrogate peptide were also evaluated.  As a consequence, a LOQ of 5 nM was attained, which was greater than 15-fold more sensitive than that without MIPs.  Finally, the assay was validated and applied to simultaneous quantitative analysis of C-terminal p53 isoforms Î±, Î², and Î³ in several human breast cell lines (i.e., MCF-10A normal cells, MCF-7 and MDA-MB-231 cancer cells, and drug-resistant MCF-7/ADR cancer cells).  The authors concluded that this study was among the 1st to use single template MIPs and cross-reactivity phenomenon to select isoform-specific surrogate peptides and enable simultaneous quantification of protein isoforms in LC-MS/MS-based targeted proteomics.\n",
      "Burat and co-workers (2019) noted that in the field of quantitative proteomics, the Isobaric Tags for Relative and Absolute Quantitation (iTRAQ) technology has demonstrated efficacy for proteome monitoring despite its lack of a consensus for data handling.  In the present study, after peptide and protein identification, these researchers compared the widespread quantitation method based on the calculation of LC-MS/MS reporter ion peaks areas ratios (ProteinPilot) to the alternative method based on the calculation of ratios of the sum of peak intensities (jTRAQx [Quant]) and they processed output data with the in-house Customizable iTRAQ Ratios Calculator (CiR-C) algorithm.  Quantitation based on peak area ratios displayed no significant linear correlation with Western blot quantitation.  In contrast, quantitation based on the sum of peak intensities displayed a significant linear association with Western blot quantitation (non-zero slope; Pearson correlation coefficient test, r = 0.296, p = 0.010) with an average bias of 0.087 Â± 0.500 and 95 % limits of agreement from -0.893 to 1.068.  The authors proposed the Mascot-jTRAQx-CiR-C strategy as a simple yet powerful data processing adjunct to the iTRAQ technology.\n",
      "Resolution ctDx Lung\n",
      "The Resolution ctDx Lung assay was developed to provide clinicians with information that can drive treatment decisions in patients with NSCLC.  The assay includes 23 actionable genes for targeted FDA-approved therapies  or therapies in clinical trials.\n",
      "Paweletz and colleagues (2016) stated that tumor genotyping is a powerful tool for guiding NSCLC care; however, comprehensive tumor genotyping can be logistically cumbersome.  To facilitate genotyping, these researchers developed a NGS assay using a desk-top sequencer to detect actionable mutations and re-arrangements in cfDNA.  An NGS panel was developed targeting 11 driver oncogenes found in NSCLC.  Targeted NGS was performed using a novel methodology that maximizes on-target reads, and minimizes artifact, and was validated on DNA dilutions derived from cell lines.  Plasma NGS was then blindly performed on 48 patients with advanced, progressive NSCLC and a known tumor genotype, and examined in 2 patients with incomplete tumor genotyping.  NGS could identify mutations present in DNA dilutions at greater than or equal to 0.4 % allelic frequency with 100 % sensitivity/specificity.  Plasma NGS detected a broad range of driver and resistance mutations, including ALK, ROS1, and RET re-arrangements, HER2 insertions, and MET amplification, with 100 % specificity.  Sensitivity was 77 % across 62 known driver and resistance mutations from the 48 cases; in 29 cases with common EGFR and KRAS mutations, sensitivity was similar to droplet digital PCR.  In 2 cases with incomplete tumor genotyping, plasma NGS rapidly identified a novel EGFR exon 19 deletion and a missed case of MET amplification.  The authors concluded that blinded to tumor genotype, this plasma NGS approach detected a broad range of targetable genomic alterations in NSCLC with no false positives including complex mutations like re-arrangements and unexpected resistance mutations such as EGFR C797S.  Through use of widely available vacutainers and a desk-top sequencing platform, this assay has the potential to be implemented broadly for patient care and translational research.  These researchers stated that through reducing the barriers between NSCLC patients and genotyping, they hope that plasma NGS will be able to facilitate delivery of targeted therapies and improve outcomes for patients with advanced NSCLC.\n",
      "Li and associates (2019) stated that non-invasive genotyping using plasma cfDNA has the potential to obviate the need for some invasive biopsies in cancer patients while also elucidating disease heterogeneity.  These researchers developed an ultra-deep plasma NGS assay for patients with NSCLC that could detect targetable oncogenic drivers and resistance mutations in patients where tissue biopsy failed to identify an actionable alteration.  Plasma was prospectively collected from patients with advanced, progressive NSCLC.  These investigators performed ultra-deep NGS using cfDNA extracted from plasma and matched white blood cells (WBCs) using a hybrid capture panel covering 37 lung cancer-related genes sequenced to 50,000Ã raw target coverage filtering somatic mutations attributable to clonal hematopoiesis.  Clinical sensitivity and specificity for plasma detection of known oncogenic drivers were calculated and compared with tissue genotyping results.  Orthogonal droplet digital PCR (ddPCR) validation was carried out in a subset of cases.  In 127 assessable patients, plasma NGS detected driver mutations with variant allele fractions ranging from 0.14 % to 52 %.  Plasma ddPCR for EGFR or KRAS mutations revealed findings nearly identical to those of plasma NGS in 21 of 22 patients, with high concordance of variant allele fraction (r = 0.98).  Blinded to tissue genotype, plasma NGS sensitivity for de-novo plasma detection of known oncogenic drivers was 75 % (68/91).  Specificity of plasma NGS in those who were driver-negative by tissue NGS was 100 % (19/19).  In 17 patients with tumor tissue deemed insufficient for genotyping, plasma NGS identified 4 KRAS mutations.  In 23 EGFR mutant cases with acquired resistance to targeted therapy, plasma NGS detected potential resistance mechanisms, including EGFR T790M and C797S mutations and ERBB2 amplification.  The authors concluded that this prospective, multi-center study demonstrated that ultra-deep NGS of plasma cfDNA with clonal hematopoiesis filtering accurately detected a wide variety of oncogenic drivers and resistance mechanisms in patients with advanced lung cancers.  The sensitivity of detection by NGS was comparable to that of established ddPCR methods.  Its high concordance with tissue genotyping and the detection of drivers in settings where tissue biopsy had failed or was not feasible lend credence to the potential clinical use of plasma cfDNA NGS and the development of cfDNA-guided intervention studies.\n",
      "Sabari and co-workers (2019) noted that liquid biopsy for plasma circulating tumor DNA (ctDNA)NGS is commercially available and increasingly adopted in clinical practice despite a paucity of prospective data to support its use.  Patients with advanced lung cancers who had no known oncogenic driver or developed resistance to current targeted therapy (n = 210) underwent plasma NGS, targeting 21 genes.  A subset of patients had concurrent tissue NGS testing using a 468-gene panel (n = 106).  Oncogenic driver detection, test turn-around time (TAT), concordance, and treatment response guided by plasma NGS were measured; all statistical tests were 2-sided.  Somatic mutations were detected in 64.3 % (135/210) of patients.  ctDNA detection was lower in patients who were on systemic therapy at the time of plasma collection compared with those who were not (30/70, 42.9 % versus 105/140, 75.0 %; OR = 0.26, 95 % CI: 0.1 to 0.5, p < 0.001).  The median TAT of plasma NGS was shorter than tissue NGS (9 versus 20 days; p < 0.001).  Overall concordance, defined as the proportion of patients for whom at least 1 identical genomic alteration was identified in both tissue and plasma, was 56.6 % (60/106, 95 % CI: 46.6 % to 66.2 %).  Among patients who tested plasma NGS positive, 89.6 % (60/67; 95 % CI: 79.7 % to 95.7 %) were also concordant on tissue NGS and 60.6 % (60/99; 95 % CI: 50.3 % to 70.3 %) vice-versa.  Patients who tested plasma NGS positive for oncogenic drivers had tissue NGS concordance of 96.1 % (49/51, 95 % CI: 86.5 % to 99.5 %), and directly led to matched targeted therapy in 21.9 % (46/210) with clinical response.  The authors concluded that plasma ctDNA NGS detected a variety of oncogenic drivers with a shorter TAT compared with tissue NGS and matched patients to targeted therapy with clinical response.  Positive findings on plasma NGS were highly concordant with tissue NGS and could guide immediate therapy; however, a negative finding in plasma requires further testing.  These researchers stated that these findings supported the potential incorporation of plasma NGS into practice guidelines.  Moreover, these researchers stated that more research is needed to better understand the kinetics of tumor shedding and the effects of treatment (surgery, radiation, chemotherapy, and immunotherapy) on plasma levels of ctDNA.  Other exciting questions on the future use of plasma-based NGS include detection in earlier stage or minimal residual disease to guide neo-adjuvant or adjuvant therapies and as a screening modality for the early detection of cancers.\n",
      "Furthermore, National Comprehensive Cancer Networkâs clinical practice guideline on \"Non-small cell lung cancer\" (Version 6.2020) states that \"cfDNA can be used in specific circumstances if the patient is not medically fit for invasive sampling, or there is insufficient tissue for molecular analysis and follow-up tissue-based analysis will be done if an oncogenic driver is not identified\".\n",
      "Afirma Xpression Atlas\n",
      "Angell and colleagues (2019) noted that the Afirma Xpression Atlas (XA) detects gene variants and fusions in thyroid nodule FNA samples from a curated panel of 511 genes using whole-transcriptome RNA-sequencing.  Its intended use is among cytologically indeterminate nodules that are Afirma GSC suspicious, Bethesda V/VI nodules, or known thyroid metastases.  These investigators reported its analytical and clinical validation.  DNA and RNA were purified from the same sample across 943 blinded FNAs and compared by multiple methodologies, including whole-transcriptome RNA-seq, targeted RNA-seq, and targeted DNA-seq.  An additional 695 blinded FNAs were used to define performance for fusions between whole-transcriptome RNA-seq and targeted RNA-seq.  These researchers quantified the reproducibility of the whole-transcriptome RNA-seq assay across laboratories and reagent lots.  Finally, variants and fusions were compared to histopathology results.  Of variants detected in DNA at 5 or 20 % variant allele frequency, 74 and 88 % were also detected by XA, respectively.  XA variant detection was 89 % when compared to an alternative RNA-based detection method.  Low levels of expression of the DNA allele carrying the variant, compared with the wild-type allele, was found in some variants not detected by XA; 82% of gene fusions detected in a targeted RNA fusion assay were detected by XA.  Conversely, nearly all variants or fusions detected by XA were confirmed by an alternative method.  Analytical validation studies demonstrated high intra-plate reproducibility (89 % to 94 %), inter-plate reproducibility (86 % to 91 %), and inter-lab accuracy (90 %).  Multiple variants and fusions previously described across the spectrum of thyroid cancers were identified by XA, including some with approved or investigational targeted therapies.  Among 190 Bethesda III/IV nodules, the sensitivity of XA as a stand-alone test was 49 %.  The authors concluded that when the Afirma Genomic Sequencing Classifier (GSC) was first used among Bethesda III/IV nodules as a rule-out test, XA supplemented genomic insight among those that were GSC suspicious.  These researchers stated that the information obtained from variants and fusions assessment may offer new precision medicine insights from diagnostic FNA samples and the opportunity to advance individualized patient care.\n",
      "The authors stated that limitations of measuring variants in expressed RNA included that some variants and fusions identified by an alternative method were not identified by XA.  The reason for these differences was unclear, nor was it known which test should ideally be considered \"correct\".  While imperfect test sensitivity was one possibility, it was also possible that some DNA variants may not be expressed due to gene silencing, or very low expression levels.  Such phenomenon may explain some discrepancies among BRAF V600E variants detected by qPCR that were negative by immunohistochemistry.  By employing a 3rd variant detection methodology that used targeted sequencing of RNA templates, these investigators showed that some samples did have very low expression of the gene variants identified in DNA.  The biological significance of such variants was unknown.  The efficacy of targeted treatment aimed at non-expressed or poorly expressed genomic alterations may be diminished.  Conversely, the vast majority of genomic abnormalities identified by XA were confirmed by the alternative method.  An additional limitation of RNA sequencing was that variants in non-coding regions, such as TERT promotor variants, were not detected by this method.  However, these findings showed that these variants were uncommon among cytologically indeterminate nodules (less than 1 %) and in the vast majority of cases, found in tandem with a RAS variant.  Moreover, these researchers showed that TERT promoter variants in combination with RAS variants could occur in benign lesions in Bethesda IV FNAs.  While current opinion is that nodules with a RAS variant (with or without TERT promoter variant) should be surgically removed given their potential malignant or pre-malignant status, it is unclear if cancers harboring a TERT promoter variant plus a RAS or BRAF variant should be treated differently based on this genomic information independent from traditional prognostic factors for risks of recurrence and death, especially among lower-risk patients.  A DNA based detection method, or development of an RNA expression-based classifier, could be added to XA in the future should reporting of such non-expressed variants be desired.\n",
      "Krane and associates (2020) stated that recent analytical and clinical validation of the Afirma XA demonstrated test reliability and the identification of genomic alterations that may inform patient management.  The updated Afirma Genomic Sequencing Classifier and XA reports aimed to optimize the understanding of these contributions, including decisions regarding observation versus surgery, the need for diseaseâspecific pre-operative testing, associated neoplasm types, prognostics, the identification of molecular targets for systemic therapy, and the recognition of potential hereditary syndromes.  These investigators noted that the increasing ability to leverage knowledge of molecular alterations in thyroid FNA specimens could further encourage the use of FNA, especially for specimen acquisition from patients with locally advanced or metastatic disease.  In such patients, the diagnosis may not be in question, but the opportunity to obtain clinically valuable molecular data in the least invasive manner possible is highly beneficial.  Morphologic analysis in this setting can also serve to document disease progression to more aggressive tumors, such as poorly differentiated or anaplastic carcinomas.  The opportunity to perform cytologic, histologic, and molecular correlation also has the potential to refine and further define cytologic criteria for malignancy on thyroid FNA specimens.  The authors concluded that molecular testing of thyroid FNA specimens is widely practiced and adds considerable insight toward clinical decision-making.  It is exciting to witness genomic insights supplement the traditional factors used in patient management.  Despite this excitement for \"genomics\", clinicians should not forget that the FDA has generally followed a handsâoff policy of enforcement discretion among laboratoryâdeveloped tests.  Therefore, clinicians have had thrust upon them heightened gatekeeper roles of evaluating test quality and safety.  As patient advocates, they must demand validation data for all laboratoryâdeveloped tests and reject those that are unsubstantiated.\n",
      "Furthermore, an UpToDate review on \"Evaluation and management of thyroid nodules with indeterminate cytology\" (Ross, 2020) states that Afirma introduced an Xpression Atlas as an add-on test, available for GSC-suspicious and Bethesda V and VI nodules, which assesses 761 DNA variants and 130 RNA fusions in 500 genes\".  Xpression Atlas is not mentioned in the \"Summary and Recommendations\" section of the review.  In addition, no prospective clinical studies for Afirma Xpression Atlas are available.\n",
      "There is currently insufficient evidence to determine the effects of the Afirma Xpression Atlas on net health outcomes; further investigation is needed to establish its clinical utility. \n",
      "Decipher Bladder\n",
      "Decipher Bladder is a genomic subtyping tool used in the management of patients with locally advanced bladder cancer.  It classifies the molecular subtype of muscle-invasive bladder cancer (MIBC) and informs which patients may benefit from neoadjuvant chemotherapy (NAC) prior to radical cystectomy. \n",
      "da Costa and colleagues (2019) stated that neuroendocrine (NE) bladder carcinoma is a rare and aggressive variant.  Molecular subtyping studies have found that 5 % to 15 % of MIBC have transcriptomic patterns consistent with NE bladder cancer in the absence of NE histology.  The clinical implications of this NE-like subtype have not been explored in depth.  In this study, transcriptome-wide expression profiles were generated for MIBC collected from 7 institutions and clinical-use of Decipher Bladder.  Using unsupervised clustering, these researchers generated a clustering solution on a prospective training cohort (PTC; n = 175), developed single-sample classifiers to predict NE tumors, and evaluated the resultant models on a testing radical cystectomy (RC) cohort (n = 225).  A random forest model was finalized and applied to 5 validation cohorts (n = 1,302).  Uni- and multi-variable survival analyses were used to characterize clinical outcomes.  In the PTC, hierarchical clustering using an 84-gene panel showed a cluster of 8 patients (4.6 %) with highly heterogeneous expression of NE markers in the absence of basal or luminal marker expression.  NE-like tumors were identified in 1 % to 6.6 % of cases in validation cohorts.  Patients with NE-like tumors had significantly worse 1-year PFS (65 % NE-like versus 82 % overall; p = 0.046) and, after adjusting for clinical and pathologic factors, had a 6.4-fold increased risk of all-cause mortality (p = 0.001); IHC confirmed the neuronal character of these tumors.  The authors concluded that a single-patient classifier was developed that identified patients with histologic urothelial cancer harboring a NE transcriptomic profile.  These tumors represent a high-risk subgroup of MIBC, which may require different treatment.  Moreover, these researchers stated that prospective validation is needed before this classification can be used clinically.\n",
      "The authors stated that this study was limited by the lack of central pathology review of all cases, although the majority of cases were assessed by dedicated genitourinary pathologists at the respective tertiary care centers and the NE-like cases were centrally reviewed.  The prospectively collected clinical-use cases underwent stringent sample processing with only the muscle-invasive urothelial component of the tumor being selected and profiled.  Four of these PTC cases had secondary NE variant histology in other regions of the tumor that were not sampled.  An additional limitation was the retrospective nature of the study.\n",
      "Furthermore, National Comprehensive Cancer Networkâs clinical practice guideline on \"Bladder cancer\" (Version 6.2020) does not mention microarray gene expression profiling as a management tool.\n",
      "Mayo Clinic Laboratories Urinary Steroid Profile\n",
      "Mayo Clinicâs Clinical Mass Spectrometry Laboratory (CMSL) added a new, non-invasive test to diagnose malignant adrenal tumors, via urinary steroid profiling.  The test is based on liquid chromatography, high-resolution, and accurate-mass mass spectrometry (HRAM LC-MS) measurement for 25 steroid metabolites in urine. \n",
      "KotÅowska and associates (2009) examined the possible urinary markers of hormonal activity in patients with non-functioning adrenal incidentalomas.  In order to evaluate the endocrine activity of afore-mentioned tumors, urinary steroid metabolite levels were analyzed in samples from patients and controls.  Possible blocks in metabolic pathways of the examined hormones were determined by comparing selected urinary steroid metabolite sums and ratios in both groups of interest.  Urine samples were collected from 20 patients with non-functioning adrenal incidentalomas and from 25 controls matched in terms of age, sex and BMI.  Excretion of 19 major urinary steroid metabolites was analyzed by gas chromatography.  The results were subjected to statistical analysis.  In patients with adrenal incidentalomas, sum of total urinary cortisol metabolites was significantly elevated compared to the control group.  These investigators also observed a shift towards tetrahydro-corticosterone, cortisol and etiocholanolone production in patients.  No significant differences in production of other urinary steroid metabolites were observed in patients with adrenal incidentalomas compared to control group.  The authors concluded that these findings suggested that not only urinary free cortisol but also its metabolite such as tetrahydro-cortisol and other steroids including etiocholanolone and corticosterone tetrahydro-metabolite might be urinary markers for the endocrine activity of adrenal incidentalomas.  Enhanced levels of these urinary steroid metabolites indicated an impairment of 11beta-hydroxysteroid dehydrogenase activity and slightly increased activity of 5beta-reductase in patients with adrenal incidentalomas.  These preliminary findings need to be validated by well-designed studies.\n",
      "Tiu and colleagues (2009) stated that it has been suggested that urinary steroid profiling may be used to provide information aiding the diagnosis and monitoring of adrenocortical carcinoma.  Nonetheless, the abnormal patterns suggestive of adrenal malignancy were not well-defined.  These researchers retrospectively studied the urinary steroid profiles of 5 patients with adrenocortical carcinoma at presentation and at follow-up, and compared these results with those from 76 patients with benign adrenocortical adenoma and 172 healthy controls.  Three abnormal patterns of urinary steroid excretion were identified in patients with adrenocortical carcinoma at presentation and/or follow-up of residual disease: hypersecretion in multiple steroid axes; excretion of unusual metabolites, notably 5-pregnene-3Î±,16Î±,20Î±-triol, 5-pregnene-3Î²,16Î±,20Î±-triol, and neonatal steroid metabolites in the post-neonatal period; increase of tetrahydro-11-deoxycortisol relative to total cortisol metabolites.  The authors concluded that these preliminary findings offered ways in which urinary steroid profiling performed using gas chromatography-mass spectrometry could be helpful in the diagnosis and monitoring of adrenocortical carcinoma.  These preliminary findings need to be validated by well-designed studies.\n",
      "Furthermore, an UpToDate review on \"Evaluation and management of the adrenal incidentaloma\" (Young and Kebebew, 2020) does not mention measurements of urinary steroid metabolites as a management tool. \n",
      "MI Cancer Seek\n",
      "MI Cancer Seek entails whole exome sequencing for DNA mutations, copy number alterations, insertions/deletions, genomic signatures microsatellite instability (MSI) and tumor mutational burden (TMB), and whole transcriptome sequencing for RNA fusions and variant transcripts.  There is currently a lack of evidence that the use of MI Cancer Seek would improve health outcomes; clinical evidence is needed to establish its utility.\n",
      "PreciseDx Breast Cancer Test\n",
      "PreciseDx Breast Cancer Test is designed to provide image analysis with artificial intelligence assessment of 12 histologic and immunohistochemical features, reported as a recurrence score.  There is currently a lack of evidence that the use of PreciseDx Breast Cancer Test would improve health outcomes; clinical evidence is needed to establish its utility.\n",
      "miR Sentinel Prostate Cancer Test\n",
      "The miR Sentinel Prostate Cancer Test (miR Scientific) is a platform of individual assays for both prostate cancer screening and determining the risk level of the disease. The miR Sentinel Prostate Cancer Test applies exosome-based analysis of 442 small noncoding RNAs (sncRNAs) by quantitative reverse transcription polymerase chain reaction (RT-qPCR) from a urine sample. The test is powered by an algorithm-based miR platform with results being reported as molecular evidence of no-, low-, intermediate- or high-risk of prostate cancer.\n",
      "Wu and associates (2020) stated that exosomes are defined as small membranous vesicles.  After RNA content was discovered in exosomes, they emerged as a novel approach for the treatment and diagnosis of cancer.  Long non-coding RNAs (lncRNA), a kind of specific RNA transcript, have been reported to function as tumor growth, metastasis, invasion, and prognosis by regulating the tumor micro-environment in exosomes.  These researchers examined the potential diagnostic of exosomal lncRNA in solid tumors.  They carried out a meta-analysis from January 2000 to October 2019 and identified publications in the English language; all relevant English literature from the Web of Science, Embase, and PubMed data-bases through October 1, 2019 were searched.  The articles were strictly screened by these investigatorsâ criteria and critiqued using the Preferred Reporting Items for Systematic Reviews and Meta-Analyses guidelines.  There were 28 studies with 19 articles (4,017 subjects) identified, including studies on BC, CRC, cholangiocarcinoma, esophageal SCC, gastric cancer, HCC, laryngeal SCC, NSCLC, and PCa.  A meta-analysis showed that the combined value of sensitivity in 29 studies was 0.74 (95 % CI: 0.7 to 0.78), and the combined value of specificity in the studies was 0.81 (95 % CI: 0.78 to 0.83); suggesting the high diagnostic efficacy of liquid exosomes in cancer patients.  It is statistically insignificant in terms of sex, ethnicity, and year.  The diagnostic power of urinary system tumors was found to be higher than that of digestive system tumors by several subgroup analyses.  The authors performed a meta-analysis and literature review of 28 studies that included 4,017 patients with 10 malignant cancer types.  They stated that mechanistically, this study demonstrated that lncRNAs in exosomes could be a promising bio-indicator for the diagnosis and prognosis of solid tumors.  These investigators hoped that their findings would encourage more researchers to examine the prognostic and diagnostic role of lncRNA in exosomes as well as examine the underlying biomechanisms in different cancers.\n",
      "Wang and colleagues (2020) noted that this is the 1st report of the development and performance of a platform that interrogates small non-coding RNAs (sncRNA) isolated from urinary exosomes.  The Sentinel PCa Test classifies patients with PCa from subjects with no evidence of PCa, the miR Sentinel CS Test stratifies patients with PCa between those with low risk PCa (Grade Group 1) from those with intermediate and high risk disease (Grade Group 2-5), and the miR Sentinel HG Test stratifies patients with PCa between those with low and favorable intermediate risk PCa (Grade Group 1 or 2) and those with high risk (Grade Group 3-5) disease.  sncRNAs were extracted from urinary exosomes of 235 subjects and interrogated on miR 4.0 microarrays.  Using proprietary selection and classification algorithms, informative sncRNAs were selected to customize an interrogation OpenArray platform that forms the basis of the tests.  The tests were validated using a case-control sample of 1,436 subjects.  The performance of the miR Sentinel PCa Test demonstrated a sensitivity of 94 % and specificity of 92 %.  The Sentinel CS Test demonstrated a sensitivity of 93 % and specificity of 90 % for prediction of the presence of Grade Group 2 or greater cancer, and the Sentinel HG Test demonstrated a sensitivity of 94 % and specificity of 96 % for the prediction of the presence of Grade Group 3 or greater cancer.  The authors concluded that the Sentinel PCa, CS and HG Tests demonstrated high levels of sensitivity and specificity, highlighting the utility of interrogation of urinary exosomal sncRNAs for non-invasively diagnosing and classifying PCa with high precision.\n",
      "These researchers stated that discordance between the Sentinel test results and the core biopsy outcomes may reflect pathological miss of higher grade cancer or a true test mis-classification.  Given the known false-negative rate of core needle biopsies, these investigators estimated the apparent false-positive rate of the Sentinel PC is 6 % to 12 % based on the 95 % CI.  This compares favorably to the 50 % to 60 % false-positive rate reported for systematic transrectal ultrasound-guided core needle biopsies and 30 % to 40 % false-positive rates reported for various MRI-targeted biopsies.  The combined apparent false-positive and negative rates of the Sentinel HG Test with biopsy outcome is around 10 %.  This performance is within the confidence limits of the well-established rate of mis-attribution of grade resulting from systematic biopsies.  Therefore, it is plausible that in this case the apparent false-positive cases resulting from the Sentinel HG Test may in fact be those who harbor higher grade cancer missed on the systematic biopsy.  An alternative explanation is that some of these represent actual false-positive test results.  To further examine this issue, these researchers are currently conducting a large retrospective study comparing the Sentinel Scores with radical prostatectomy pathology.\n",
      "In an editorial that accompanied the afore-mention study, Helfand (2020) stated that \"Overall, the results appear to be promising new PCa biomarkers for each of these tests.  However, some caution should be made with any novel test, including the requirement to validate the results in other independent cohorts and racially diverse groups.  In addition, because of inherent sampling errors with biopsy results, further comparison to final surgical pathology should be made.  Finally, research should be devoted to further characterize the small noncoding RNAs as they may provide additional insights and/or therapeutic targets into the biology of PCa\".\n",
      "Furthermore, National Comprehensive Cancer Networkâs clinical practice guideline on \"Prostate cancer\" (Version 1.2023) does not mention measurement of non-coding RNAs in urinary exosomes as a management option.\n",
      "Signatera\n",
      "Signatera is a custom-built circulating tumor DNA (ctDNA) test for molecular treatment monitoring and molecular residual disease (MRD) assessment.  It can be used for individuals with stage II/III colorectal cancer who are considering adjuvant chemotherapy (ACT) and/or who are being monitored for relapse post-treatment.  However, there is currently insufficient evidence to support  its clinical effectiveness.\n",
      "Reinert and colleagues (2019) stated that novel sensitive methods for detection and monitoring of residual disease could improve post-operative risk stratification with implications for patient selection for adjuvant chemotherapy (ACT), ACT duration, intensity of radiologic surveillance, and, ultimately, outcome for patients with colorectal cancer (CRC).  In a prospective, multi-center cohort study, these investigators examined the association of circulating tumor DNA (ctDNA) with recurrence using longitudinal data from ultra-deep sequencing of plasma cell-free DNA in patients with CRC before and after surgery, during and after ACT, and during surveillance.  ctDNA was quantified in the pre-operative and post-operative settings of stages I to III CRC by personalized multiplex, PCR-based, next-generation sequencing.  The study enrolled 130 patients at the surgical departments of Aarhus University Hospital, Randers Hospital, and Herning Hospital in Denmark from May 1, 2014, to January 31, 2017.  Plasma samples (n = 829) were collected before surgery, post-operatively at day 30, and every 3rd month for up to 3 years.  Main outcomes were ctDNA measurement, clinical recurrence, and RFS.  A total of 130 patients with stages I to III CRC (mean [SD] age of 67.9 [10.1] years; 74 [56.9 %] men) were enrolled in the study; 5 patients discontinued participation, leaving 125 patients for analysis.  Pre-operatively, ctDNA was detectable in 84 of 94 patients (89.4 %).  After definitive treatment, longitudinal ctDNA analysis identified 14 of 16 relapses (87.5 %).  At post-operative day 30, ctDNA-positive patients were 7 times more likely to relapse than ctDNA-negative patients (HR, 7.2; 95 % CI: 2.7 to 19.0; p < 0.001).  Similarly, shortly after ACT ctDNA-positive patients were 17 times (HR, 17.5; 95 % CI: 5.4 to 56.5; p < 0.001) more likely to relapse.  All 7 patients who were ctDNA positive after ACT experienced relapse.  Monitoring during and after ACT indicated that 3 of the 10 ctDNA-positive patients (30.0 %) were cleared by ACT.  During surveillance after definitive therapy, ctDNA-positive patients were more than 40 times more likely to experience disease recurrence than ctDNA-negative patients (HR, 43.5; 95 % CI: 9.8 to 193.5; p < 0.001).  In all multi-variate analyses, ctDNA status was independently associated with relapse after adjusting for known clinicopathologic risk factors.  Serial ctDNA analyses revealed disease recurrence up to 16.5 months ahead of standard-of-care radiologic imaging (mean of 8.7 months; range of 0.8 to 16.5 months).  Actionable mutations were identified in 81.8 % of the ctDNA-positive relapse samples.  The authors concluded that circulating tumor DNA analysis could potentially change the post-operative management of CRC by enabling risk stratification, ACT monitoring, and early relapse detection.  They stated that there are potential limitations to this study, including the modest sample size of patients with recurrent CRC and the analysis of multiple patient subsets.  Moreover, these researchers stated that these findings suggested many potentially paradigm-changing clinical applications of ctDNA in CRC and provide a framework for future clinical trials to examine the clinical benefits of ctDNA-guided disease management. \n",
      "Tie and associates (2019) noted that ACT in patients with stage III colon cancer prevents recurrence by eradicating MRD.  However, which patients remain at high risk of recurrence after completing standard adjuvant treatment cannot currently be determined.  Post-surgical ctDNA analysis can detect MRD and is associated with recurrence in CRCs.  In a multi-center, population-based, cohort biomarker study, these investigator examined if serial post-surgical and post-chemotherapy ctDNA analysis could provide a real-time indication of adjuvant therapy efficacy in stage III colon cancer.  This trial recruited 100 consecutive patients with newly diagnosed stage III colon cancer planned for 24 weeks of ACT from November 1, 2014, through May 31, 2017.  Patients with another malignant neoplasm diagnosed within the last 3 years were excluded.  Median duration of follow-up was 28.9 months (range of 11.6 to 46.4 months).  Physicians were blinded to ctDNA results.  Data were analyzed from December 10, 2018, through June 23, 2019.  Serial plasma samples were collected after surgery and after chemotherapy.  Somatic mutations in individual patients' tumors were identified via massively parallel sequencing of 15 genes commonly mutated in CRC; personalized assays were designed to quantify ctDNA.  Main outcome measures included detection of ctDNA and recurrence-free interval (RFI).  After 4 exclusions, 96 eligible patients were eligible; median patient age was 64 years (range of 26 to 82); 49 (51 %) were men.  At least 1 somatic mutation was identified in the tumor tissue of all 96 evaluable patients.  Circulating tumor DNA was detectable in 20 of 96 (21 %) post-surgical samples and was associated with inferior RFS (HR, 3.8; 95 % CI: 2.4 to 21.0; p < 0.001); ctDNA was detectable in 15 of 88 (17 %) post-chemotherapy samples.  The estimated 3-year RFI was 30 % when ctDNA was detectable after chemotherapy and 77 % when ctDNA was undetectable (HR, 6.8; 95 % CI: 11.0 to 157.0; p < 0.001).  Post-surgical ctDNA status remained independently associated with RFI after adjusting for known clinicopathologic risk factors (HR, 7.5; 95 % CI: 3.5 to 16.1; p < 0.001).  The authors concluded that the findings of this study suggested that ctDNA analysis after surgery is a promising prognostic marker in stage III colon cancer.  Post-chemotherapy ctDNA analysis may define a patient subset that remains at high risk of recurrence despite completing standard adjuvant treatment.\n",
      "Wang and colleagues (2019) stated that for patients with resected, non-metastatic CRC, the optimal surveillance protocol remains unclear.  These researchers examined if serial ctDNA levels detected disease recurrence earlier, compared with conventional post-operative surveillance, in patients with resected CRC.  This study included patients (n = 58) with stage I, II, or III CRC who underwent radical surgical resection at 4 Swedish hospitals from February 2, 2007, to May 8, 2013; 18 patients received ACT at the discretion of their clinicians, who were blinded to the ctDNA results.  Blood samples were collected at 1 month after the surgical procedure and every 3 to 6 months thereafter for ctDNA analysis.  Patients were followed-up until metachronous metastases were detected, or for a median of 49 months.  Data analysis was performed from March 1, 2009 to June 23, 2018.  Sensitivity and timing of ctDNA positivity were compared with those of conventional surveillance modalities (CT scans and serum CEA tests) for the detection of disease recurrence.  This study included 319 blood samples from 58 patients, with a median (range) age of 69 (47 to 83) years and 34 men (59 %).  The recurrence rate among patients with positive ctDNA levels was 77 % (10 of 13 patients).  Positive ctDNA preceded radiologic and clinical evidence of recurrence by a median of 3 months.  Of the 45 patients with negative ctDNA throughout follow-up, none (0 %; 95 % CI: 0 % to 7.9 %) experienced a relapse, with a median follow-up of 49 months.  However, 3 (6 %; 95 % CI: 1.3 % to 17 %) of the 48 patients without relapse had a positive ctDNA result, which subsequently fell to undetectable levels during follow-up.  The authors concluded that although these findings needed to be validated in a larger, prospective trial, they suggested that ctDNA analysis could complement conventional surveillance strategies as a triage test to stratify patients with resected CRC on the basis of risk of disease recurrence.\n",
      "The authors stated that this study was limited by the sample size, involving only 319 blood samples from 58 patients.  Nevertheless, it was encouraging that the 10 patients who relapsed clinically had positive ctDNA levels that preceded radiographic evidence of recurrence.  Although ctDNA positivity preceded recurrence by a median of 4 months in patients who did not receive adjuvant chemotherapy, the blood samples did not become ctDNA positive until a median of 9 months after surgical resection.  This lead time would not be early enough to affect the decision on ACT; however, it might still be sufficient to allow for earlier implementation of other curative or palliative strategies.  These investigators suspected that the shorter lead time may, in part, be associated with the higher frequency of imaging than recommended by many guidelines for stage II or III disease; 2/3 of the patients in the study underwent CT imaging every 6 months, and the remaining 1/3 was imaged every 12 months.\n",
      "Reece and associates (2019) noted that CRC is one of the most common cancers worldwide and has a high mortality rate following disease recurrence.  Treatment efficacy is maximized by providing tailored cancer treatment, ideally involving surgical resection and personalized neoadjuvant and adjuvant therapies, including chemotherapy, radiotherapy and increasingly, targeted therapy.  Early detection of recurrence or disease progression results in more treatable disease and is essential to improving survival outcomes.  Recent advances in the understanding of tumor genetics have resulted in the discovery of ctDNA.  These investigators carried out a literature search in PubMed to identify all original articles preceding April 2019 that employ ctDNA for the purpose of monitoring response to CRC treatment.  A total of 92 studies were included.  These studies showed that ctDNA is a reliable measure of tumor burden; they demonstrated the utility of ctDNA in assessing the adequacy of surgical tumor clearance and changes in ctDNA levels reflected response to systemic treatments.  ctDNA could be used in the selection of targeted treatments.  The re-appearance or increase in ctDNA, as well as the emergence of new mutations, correlated with disease recurrence, progression, and resistance to therapy, with ctDNA measurement allowing more sensitive monitoring than currently used clinical tools.  The authors concluded that ctDNA showed enormous promise as a sensitive biomarker for monitoring response to many treatment modalities and for targeting therapy; therefore, it is emerging as a new way for guiding treatment decisions-initiating, altering, and ceasing treatments, or prompting investigation into the potential for residual disease.  However, many potentially useful ctDNA markers are available and more work is needed to determine which are best suited for specific purposes and for improving specific outcomes.\n",
      "The authors stated that 1 drawback of this review was the lack of standardization of ctDNA detection methods between studies.  Assay methods include PCR, droplet digital PCR, BEAMing, and NGS.  These methods can vary greatly in cost, and not all technologies are available in standard laboratories.  For standardization it is also important to establish quality controls to account for the inter-laboratory variability.  A recent study showed that through sending ctDNA samples to 32 laboratories across Europe for mutation testing, and using 6 different cell-free DNA extraction methods and 5 different analysis methods, it led to a percentage of errors that could have had implications for clinical decision-making around therapy of 20.1 %.  It has also been shown that artefactual KRAS mutations may occur, depending on the method applied.  Setting a threshold for the mutant allele frequency may reduce the incidence of false results.  Similarly, ultra-sensitive techniques should also be applied, otherwise the absence of detectable ctDNA may indicate a poor quality sample or analysis, rather than the absence of disease.  A further drawback of the studies was that very few established whether ctDNA was an independent predictor for treatment response.  Of 20 studies that calculated a HR of ctDNA for assessing either PFS or OS, only 5 studies performed multi-variate analysis, correcting for other important variables that may affect survival.\n",
      "Wang and co-workers (2020) noted that surgical resection is the primary treatment for patients with non-metastatic CRC.  However, even after undergoing radical resection procedure, 30 to 50 % of patients will still experience relapse.  Circulation tumor DNA, deriving from tumor cells, is shed into the bloodstream and is a potential predictive biomarker of recurrence in CRC.  In a meta-analysis, these researchers examined the clinical value of ctDNA in predicting the recurrence of CRC patients in post-operative.  PubMed, Embase, the Cochrane Library, and Web of Science were searched to identify the studies that reported the function of ctDNA for predicting recurrence in CRC patients.  The eligible studies were pooled to calculate the relative risk (RR) of recurrence in ctDNA positive and negative groups.  The data of ctDNA on RFS were extracted and computed in HR and 95 % CI; subgroup analyses were also performed.  A total of 7 studies including 424 patients were included and analyzed in this meta-analysis.  The results showed that pooled RR was 4.65 (95 % CI: 2.68 to 8.08, p < 0.05), indicating positive ctDNA could predict the recurrence of CRC after curative surgical.  The pooled HR demonstrated strong connection between ctDNA positive and RFS in patients with CRC (HR = 9.14, 95 % CI: 4.02 to 20.75, p < 0.05).  The authors concluded that evidence from the meta-analysis suggested that ctDNA is a promising potential biomarker for predicting post-operative recurrence of CRC.  Moreover, these researchers stated that given the inherent limitations of this study, they look forward to more well-designed clinical studies to validate and update this analysis in the future.\n",
      "Furthermore, National Comprehensive Cancer Networkâs clinical practice guidelines on \"Colon cancer\" (Version 4.2020) and \"Rectal cancer\" (Version 6.2020) do not mention measurement of cell-free DNA as a management tool.\n",
      "Bratman et al (2020) noted that immune checkpoint blockade (ICB) provides clinical benefit to a subset of patients with cancer; however, existing biomarkers do not reliably predict treatment response across diverse cancer types.  Limited data exist to show how serial circulating tumor DNA (ctDNA) testing may perform as a predictive biomarker in patients receiving ICB.  In a prospective, phase-II clinical trial, these researchers examined ctDNA in 5 distinct cohorts of patients with advanced solid tumors treated with pembrolizumab.  They employed bespoke ctDNA assays to 316 serial plasma samples obtained at baseline and every 3 cycles from 94 patients.  Baseline ctDNA concentration correlated with progression-free survival (PFS), overall survival (OS), clinical response and clinical benefit.  This association became stronger when considering ctDNA kinetics during treatment.  All 12 patients with ctDNA clearance during treatment were alive with median 25âmonths follow-up.  The authors concluded that the findings of this phase-II clinical trial suggested broad clinical use for ctDNA-based surveillance in patients treated with ICB.  This is a non-invasive strategy to predict clinical benefit and long-term survival that could be generalizable across cancer types.  They stated that future interventional studies are needed to enable clinical decisions using ctDNA levels to guide ICB treatment as well as surveillance in patients treated with ICB.\n",
      "Loupakis et al (2021) stated that more than 50 % of patients with stage IV colorectal cancer (metastatic colorectal cancer [mCRC]) relapse post-resection.  The effectiveness of post-operative systemic treatment is limited in this setting; thus, these patients would greatly benefit from the use of a reliable prognostic biomarker, such as ctDNA to identify minimal or molecular residual disease (MRD).  In a retrospective study, these researchers analyzed a cohort of 112 patients with mCRC who had undergone metastatic resection with curative intent as part of the PREDATOR clinical trial.  The study examined the prognostic value of ctDNA, correlating MRD status post-surgery with clinical outcomes by using a personalized and tumor-informed ctDNA assay (bespoke multiple PCR, next-generation sequencing [NGS] assay).  Post-resection, systemic therapy was administered to 39.2 % of the patients at the discretion of the treating physician.  Post-surgical, MRD positivity was observed in 54.4 % (61 of 112) of patients, of which 96.7 % (59 of 61) progressed at the time of data cut-off (hazard ratio [HR]: 5.8; 95 % CI: 3.5 to 9.7; p < 0.001).  MRD-positive status was also associated with an inferior OS: HR: 16.0; 95 % CI: 3.9 to 68.0; p < 0.001.  At the time of analyses, 96 % (49 of 51) of patients were alive in the MRD-negative arm compared with 52.4 % (32 of 61) in the MRD-positive arm.  Patients who did not receive systemic therapy and were MRD-negative in the combined ctDNA analysis at 2 time-points had an OS of 100 %.  In the multi-variate analysis, ctDNA-based MRD status was the most significant prognostic factor associated with disease-free survival (DFS; HR: 5.78; 95 % CI: 3.34 to 10.0; p < 0.001).  The authors concluded that the findings of this study suggested that clinical trials centered on patients with mCRC could benefit from the implementation of ctDNA testing in their design.  For example, clinical trials could benefit from patient stratification on the basis of their post-surgical MRD status before randomly assigning patients into a treatment versus placebo arm.  Furthermore, MRD-guided trials could also benefit by enriching patients with high-risk of relapse, leading to significant reductions in trial sample size and unnecessary treatment cost.  In addition, the use of ctDNA as a surrogate endpoint for treatment response monitoring is being actively investigated, wherein an early indication of treatment efficacy (ctDNA clearance) relative to conventional strategies may lead to expedited approval of new therapies.  The results of this trial support the continuous expansion of the number of clinical studies in patients with mCRC using personalized ctDNA-based MRD analysis and provides direct evidence of the predictive and prognostic value of ctDNA, which could help clinicians and researchers with real numbers to design their clinical studies and support therapeutic decisions in the adjuvant setting.  These researchers stated that post-operative MRD analysis could be a strong prognostic biomarker for patients with mCRC undergoing resection of metastases; it holds promises for being implemented in clinical decision-making, informing clinical trial design, and further translational research.\n",
      "The authors stated that drawbacks of this study included a relatively small sample size (n = 112) and the retrospective design with the use of archived samples, because of which these researchers observed a high degree of necrosis and low tumor cellularity in the tissue samples, resulting in a WES QC failure rate of 16.9 %.  This was because some patients received systemic treatment before liver or met resection, resulting in deteriorated sample quality.  However, in real-world, prospective studies with analysis performed on primary or untreated samples, these researchers had observed a WES QC failure rate of less than 3 %, which was compatible with clinical practice.  Another drawback of this study was that these investigators only tested 2 time-points (post-surgical) in this setting.  They stated that their future studies will include monitoring of ctDNA dynamics using serial testing at regular intervals within a subgroup of patients from the present analysis.  Overall, these investigators believed that the use of serial testing could allow for tailoring of treatment regimens, with treatment escalation in patients with progressive disease and the opportunity of early therapeutic interventions with more aggressive follow-up in patients who were ctDNA-positive but have not yet progressed.  Furthermore, previous studies had shown ctDNA clearance to be a proxy of treatment efficacy, especially in the adjuvant setting where post-operative ctDNA negativity showed better outcomes compared with ctDNA-positive patients.  The authors acknowledged the limitation of their small dataset to establish this evidence.  However, they did see a clear pattern, wherein 100 % of the patients who remained positive or turned positive at the 2nd time-point progressed as compared with the ctDNA-negative patients.\n",
      "Powles et al (2021) noted that minimally invasive approaches to detect residual disease following surgery are needed to identify patients with cancer who are at risk for metastatic relapse.  Circulating tumor DNA (ctDNA) holds promise as a biomarker for molecular residual disease and relapse.  These researchers examined outcomes in 581 patients who had undergone surgery and were evaluable for ctDNA from a randomized phase-III clinical trial of adjuvant atezolizumab versus observation in operable urothelial cancer.  This trial did not reach its efficacy endpoint in the intention-to-treat (ITT) population.  These investigators showed that ctDNA testing at the start of therapy (cycle 1 day 1) identified 214 (37 %) patients who were positive for ctDNA and who had poor prognosis (observation arm hazard ratio [HR] = 6.3 (95 % confidence interval [CI]: 4.45 to 8.92); p < 0.0001).  Notably, patients who were positive for ctDNA had improved disease-free survival (DFS) and overall survival (OS) in the atezolizumab arm versus the observation arm (DFS HR = 0.58 (95 % CI: 0.43 to 0.79); p = 0.0024, OS (HR = 0.59 (95 % CI: 0.41 to 0.86)).  No difference in DFS or OS between treatment arms was noted for patients who were negative for ctDNA.  The rate of ctDNA clearance at week 6 was higher in the atezolizumab arm (18 %) than in the observation arm (4 %) (p = 0.0204).  Transcriptomic analysis of tumors from patients who were positive for ctDNA revealed higher expression levels of cell-cycle and keratin genes.  For patients who were positive for ctDNA and who were treated with atezolizumab, non-relapse was associated with immune response signatures and basal-squamous gene features, whereas relapse was associated with angiogenesis and fibroblast TGFÎ² signatures.  The authors concluded that these findings suggested that adjuvant atezolizumab may be associated with improved outcomes compared with observation in patients who were positive for ctDNA and who were at a high risk of relapse.  These findings, if validated in other settings, would shift approaches to post-operative cancer care.\n",
      "The authors stated that this study had 2 main drawbacks.  First, ctDNA status was a prospective but exploratory endpoint; thus, further studies are needed to validate and expand its clinical utility.  Second, this method of ctDNA analysis required whole-exome sequencing (WES) and took approximately 2 to 3 weeks.  This is clinically applicable in the adjuvant setting where patients require a period of time to recover from surgery before starting therapy.\n",
      "In an expert commentary regarding the study by Powles et al (2021), Woldu and Lotan (2021) stated that âWhile this is all quite exciting, the results are far from ideal and further work is required.  As an example, 31 % of patients who were negative for ctDNA in the observation arm in this study experienced relapse.  This lack of sensitivity may be due to technical limitations of the assay, which could reasonably be expected to be mitigated with further advances.  However, there are also likely biologic limitations, such as non-shedding tumors and heterogeneity in tumors that relapse, that will inherently limit the sensitivity of such assays.  In addition, while ctDNA may very well correlate with MRD and have prognostic value, novel biomarkers with predictive capacity that might guide future adjuvant options are neededâ.\n",
      "Kotani et al (2023) stated that despite standard-of-care (SOC) treatment, more than 30 % of patients with resectable colorectal cancer (CRC) relapse; and ctDNA analysis may enable post-surgical risk stratification and adjuvant chemotherapy (ACT) treatment decision-making.  In an observational study, these investigators reported results from GALAXY, which is an observational arm of the ongoing CIRCULATE-Japan study that analyzed pre-surgical and post-surgical ctDNA in patients with stage II-IV resectable CRC (n = 1,039).  In this cohort, with a median follow-up of 16.74 months (range of 0.49 to 24.83 months), post-surgical ctDNA positivity (at 4 weeks after surgery) was associated with higher recurrence risk (hazard ratio (HR) 10.0, p < 0.0001) and was the most significant prognostic factor associated with recurrence risk in patients with stage II or III CRC (HR 10.82, p < 0.001).  Furthermore, post-surgical ctDNA positivity identified patients with stage II or III CRC who derived benefit from ACT (HR 6.59, p < 0.0001).  The authors concluded that in this large, observational study, they showed that post-surgical ctDNA status is a most significant prognostic biomarker than the currently used high-risk clinicopathological features and could potentially be predictive of ACT benefit.  These researchers stated that ongoing prospective, randomized trials will further examine the optimal ctDNA-guided treatment strategy for surgically resectable CRC.\n",
      "The authors stated that potential limitations of these results included the observational nature of the study and the bias in patient characteristics.  To partly mitigate these potential biases, these investigators carried out a multi-variate analysis, demonstrating the clear benefit of ACT for reduction of recurrence risk in ctDNA-positive patients 4âweeks after surgery.  Another limitation of the study was the non-feasibility of performing a randomized trial of ACT versus the observation arm in post-surgical ctDNA-positive patients in Japan.  However, these findings were supportive of the benefit of ACT and may need further investigation in a prospective randomized trial with adequate follow-up to evaluate the non-inferiority of the observation arm versus ACT in ctDNA-negative patients.  These data are supported by the results of the recently published prospective, randomized DYNAMIC trial.  Of 455 patients with stage II colon cancer, 294 underwent ctDNA-guided adjuvant therapy, and 147 underwent standard management (median follow-up of 37 months).  ctDNA-guided management reduced the proportion of patients receiving adjuvant therapy (15 % in the ctDNA-guided arm versus 28 % in the standard management arm), without compromising 2-year recurrence-free survival (RFS; 93.5 % in the ctDNA-guided arm versus 92.4 % in the standard management arm), implying that ctDNA-guided ACT is not inferior to standard management.  Interestingly, patients who were ctDNA-negative were not treated, and the 3-year RFS for ctDNA-negative patients was 92.5 % compared with 86.4 % for ctDNA-positive patients.\n",
      "An assessment of Signatera by the National Institute for Health and Clinical Excellence (2022) stated that \"[k]ey uncertainties around the evidence or technology are that there is no prospective evidence on using Signatera in clinical practice or its effect on treatment decisions or clinical outcomes.. . . Experts advised that there is not enough evidence to support routine use of the technology in the NHS. This is in line with recommendations from the European Society for Medical Oncology on the use of ctDNA. But there are several ongoing trials that may address gaps in the evidence.\"\n",
      "Natera's Signatera Molecular Monitoring (MRD) for Breast Cancer\n",
      "Nateraâs Signatera (RUO) is the first ctDNA assay custom-built for treatment monitoring and molecular residual disease assessment.  The Signatera (RUO) methodology differs from currently available liquid biopsy assays, which test for a panel of genes independent of an individual's tumor.\n",
      "Oellerich et al (2017) stated that high-quality genomic analysis is critical for personalized pharmacotherapy in patients with cancer.  Tumor-specific genomic alterations can be identified in cell-free DNA (cfDNA) from patient blood samples and can complement biopsies for real-time molecular monitoring of treatment, detection of recurrence, and tracking resistance.  cfDNA can be especially useful when tumor tissue is unavailable or insufficient for testing.  For blood-based genomic profiling, next-generation sequencing (NGS) and droplet digital PCR (ddPCR) have been successfully applied.  The Food and Drug Administration (FDA) recently approved the first such \"liquid biopsy\" test for EGFR mutations in patients with non-small cell lung cancer (NSCLC).  Such non-invasive methods allow for the identification of specific resistance mutations selected by treatment, such as EGFR T790M, in patients with NSCLC treated with gefitinib.  Chromosomal aberration pattern analysis by low coverage whole genome sequencing is a more universal approach based on genomic instability.  Gains and losses of chromosomal regions have been detected in plasma tumor-specific cfDNA as copy number aberrations and can be used to compute a genomic copy number instability (CNI) score of cfDNA.  A specific CNI index obtained by massive parallel sequencing discriminated those patients with prostate cancer from both healthy controls and men with benign prostatic disease.  Furthermore, androgen receptor gene aberrations in cfDNA were associated with therapeutic resistance in metastatic castration resistant prostate cancer.  Change in CNI score has been shown to serve as an early predictor of response to standard chemotherapy for various other cancer types (e.g. NSCLC, colorectal cancer, pancreatic ductal adenocarcinomas).  CNI scores have also been shown to predict therapeutic responses to immunotherapy.  Serial genomic profiling can detect resistance mutations up to 16 weeks before radiographic progression.  There is a potential for cost savings when ineffective use of expensive new anticancer drugs is avoided or halted.  Challenges for routine implementation of liquid biopsy tests include the necessity of specialized personnel, instrumentation, and software, as well as further development of quality management (e.g., external quality control).  Validation of blood-based tumor genomic profiling in additional multi-center outcome studies is needed; however, cfDNA monitoring can provide clinically important actionable information for precision oncology approaches.\n",
      "Volckmar et al (2018) noted that recently, many genome-wide profiling studies provided insights into the molecular make-up of major cancer types.  The deeper understanding of these genetic alterations and their functional consequences led to the discovery of novel therapeutic opportunities improving clinical management of cancer patients.  While tissue-based molecular patient stratification is the gold standard for precision medicine, it has certain limitations:  Tissue biopsies are invasive sampling procedures carrying the risk of complications and may not represent the entire tumor due to underlying genetic heterogeneity.  In this context, complementary characterization of genetic information in the blood of cancer patients can serve as minimal-invasive 'liquid biopsy'.  Fragments of cfDNA are released from tissues of healthy individuals as well as cancer patients.  The fraction of cfDNA that is released from primary tumors or metastases (i.e., circulating tumor DNA, ctDNA) represents genetic aberrations in cancer cells, which are a potential source for diagnostic, prognostic, and predictive biomarkers.  Recent studies have demonstrated technical feasibility and clinical applications including detection of drug targets and resistance mutations as well as longitudinal monitoring of tumors under therapy.  To this end, a variety of pre-analytical procedures for blood processing, isolation and quantification of cfDNA are being employed and several analytical methods and technologies ranging from PCR-based single locus assays to genome-wide approaches are available, which considerably differ in sensitivity, specificity, and throughput.  However, broad implementation of ctDNA analysis in daily clinical practice requires a thorough understanding of theoretical, technical, and biological concepts and necessitates standardization and validation of pre-analytical and analytical procedures across different technologies.\n",
      "Oellerich et al (2019) stated that genomic analyses in oncologic care allow for the development of more precise clinical laboratory tests that will be critical for personalized pharmacotherapy.  Traditional biopsy-based approaches are limited by the availability of sequential tissue specimens to detect resistance.  Blood-based genomic profiling (\"liquid biopsy\") is useful for longitudinal monitoring of tumor genomes and can complement biopsies.  Tumor-associated mutations can be identified in ctDNA from patient blood samples and used for monitoring disease activity.  The FDA approved a liquid biopsy test for EGFR-activating mutations in patients with NSCLC as a companion diagnostic for therapy selection.  ctDNA also allows for the identification of mutations selected by treatment such as EGFR T790M in NSCLC.  ctDNA can also detect mutations such as KRAS G12V in colorectal cancer and BRAF V600E/V600K in melanoma.  Chromosomal aberration pattern analysis by low-coverage whole genome sequencing is a new, broader approach.  Genomic imbalances detected in cfDNA can be used to compute a CNI score.  In clinical studies, it was demonstrated that the change in CNI score can serve as an early predictor of therapeutic response to chemotherapy/immunotherapy of many cancer types.  In multi-variable models, it could be shown that the CNI score was superior to clinical parameters for prediction of overall survival in patients with head and neck cancer.  There is emerging evidence for the clinical validity of ctDNA testing regarding identification of candidates for targeted therapies, prediction of therapeutic response, early detection of recurrence, resistance mutation detection, measuring genetic heterogeneity, tumor burden monitoring, and risk stratification.  Improvement of sensitivity to detect tumors at very early stages is difficult due to insufficient mutant DNA fraction of less than or equal to 0.01 %.  The authors stated that further developments will include validation in prospective multi-center interventional outcome studies and the development of digital platforms to integrate diagnostic data.\n",
      "Coombes and colleagues (2019) stated that up to 30 % of patients with breast cancer relapse after primary treatment.  There are no sensitive and reliable tests to monitor these patients and detect distant metastases before overt recurrence.  These researchers reported on the use of personalized circulating tumor DNA (ctDNA) profiling for detection of recurrence in breast cancer.  A total of 49 primary patients with breast cancer were recruited following surgery and adjuvant therapy.  Plasma samples (n = 208) were collected every 6 months for up to 4 years.  Personalized assays targeting 16 variants selected from primary tumor whole-exome data were tested in serial plasma for the presence of ctDNA by ultra-deep sequencing (average greater than 100,000X).  Plasma ctDNA was detected ahead of clinical or radiologic relapse in 16 of the 18 relapsed patients (sensitivity of 89 %); metastatic relapse was predicted with a lead time of up to 2 years (median of 8.9 months; range of 0.5 to 24.0 months).  None of the 31 non-relapsing patients was ctDNA-positive at any time-point across 156 plasma samples (specificity of 100 %).  Of the 2 relapsed patients who were not detected in the study, the 1st had only a local recurrence, whereas the 2nd patient had bone recurrence and had completed chemotherapy just 13 days prior to blood sampling.  The authors concluded that they presented a sensitive and specific clinical test that could be used to identify pre-clinical metastases and follow all patients with breast cancer following therapy irrespective of molecular subtype.  It out-performs conventional means of monitoring and showed promise as a tool for guiding future precision medicine.  Moreover, these researchers stated that future studies will address the issue of the effects of therapy on ctDNA levels in patients with breast cancer.\n",
      "The authors stated that this study had several drawbacks.  The test is not suitable for detecting a 2nd primary breast cancer unless it recurred from the original tumor; this was exemplified by a patient, where a 2nd contralateral primary cancer was detected.  Second, relying on ctDNA required that sufficient molecules were present in the plasma at the time of collection, which might not have been the case in patients with smaller and less aggressive breast cancers.  This was exemplified by a patient who relapsed with local resectable disease but was ctDNA-negative.\n",
      "Hufnagl et al (2020) noted that the clinical decisions made when treating patients with metastatic cancer require knowledge of the current tumor extent and response to therapy.  For the majority of solid tumors, a response assessment, which is based on imaging, is used to guide these decisions.  However, measuring serum protein biomarkers (i.e., tumor markers) may be of additional use.  Furthermore, tumor markers exhibit variable specificity and sensitivity and cannot therefore be solely relied upon when making decisions regarding cancer treatment.  Thus, there is a clinical requirement for the identification of specific, sensitive and quantitative biomarkers.  In recent years, circulating cfDNA and mutation-specific circulating cell-free tumor DNA (cftDNA) have been identified as novel potential biomarkers.  In the current study, cfDNA and cftDNA were compared using imaging-based staging and current tumor markers in 15 patients with metastatic colorectal, pancreatic or breast cancer.  These patients were treated at the Third Medical Department of Paracelsus Medical University Salzburg (Austria).  The results of the current study demonstrated a statistically significant correlation between the concentration changes of cfDNA and cftDNA and response to treatment, which was assessed by imaging.  A correlation was not indicated with current clinically used tumor markers, including carcino-embryonic antigen (CEA), carcinoma antigen (CA) 15-3 and CA 19-9.  The present study also indicated a correlation between cfDNA and cftDNA and the tumor volume of metastatic lesions, which was not observed with the current clinically used tumor markers.  The authors concluded that cfDNA and cftDNA exhibited the potential to become novel biomarkers for the response assessment following cancer treatment, and may serve as a tool for the estimation of tumor volume.  The current study further supports the increasingly important role of cfDNA and cftDNA as new monitoring tools for use during cancer therapy.\n",
      "Moss et al (2020) stated that tumor-derived cfDNA is present in the plasma of individuals with cancer.  Assays aimed at detecting common cancer mutations in cfDNA are being developed for the detection of several cancer types.  In breast cancer, however, such assays have failed to detect the disease at a sensitivity relevant for clinical use, in part due to the absence of multiple common mutations that can be co-detected in plasma.  Unlike individual mutations that exist only in a subset of tumors, unique DNA methylation patterns are universally present in cells of a common type and therefore may be ideal biomarkers.  These researchers described the detection and quantification of breast-derived cfDNA using a breast-specific DNA methylation signature.  They collected plasma from patients with localized breast cancer before and throughout treatment with neoadjuvant chemotherapy and surgery (n = 235 samples).  Pre-treatment breast cfDNA was detected in patients with localized disease with a sensitivity of 80 % at 97 % specificity.  High breast cfDNA levels were associated with aggressive molecular tumor profiles and metabolic activity of the disease.  During neoadjuvant chemotherapy, breast cfDNA levels decreased dramatically.  Importantly, the presence of breast cfDNA towards the end of the chemotherapy regimen reflected the existence of residual disease.  The authors proposed that breast-specific cfDNA is a universal and powerful marker for the detection and monitoring of breast cancer.\n",
      "These researchers stated that in this preliminary study, they provided proof of the concept that breast cfDNA quantification opens a window into the dynamics of breast cancer with a potential for early diagnosis, monitoring of treatment response, and detection of recurrence.  It should be noted that this study was based on data obtained from a small number of individuals who underwent neoadjuvant chemotherapy and further validation in another series of patients receiving neoadjuvant chemotherapy is needed.  Additionally, further expansion of the panel of breast DNA markers used may dramatically increase assay sensitivity and clinical utility.  These investigators also underscored the importance of sufficiently powered studies to explain the variance in breast cfDNA concentrations observed among patients and to test the utility of breast cfDNA in identifying recurrence of the disease.  Finally, while in this study the authors had focused on patients with localized disease at stage II to III, future studies, using a maximally sensitive version of the assay, should examine the relevance of this approach to patients at even earlier stages of the disease.\n",
      "Furthermore, National Comprehensive Cancer Networkâs clinical practice guideline on \"Breast cancer\" (Version 2.2020) does not mention the use of circulating cell-free DNA as a management tool.\n",
      "Magbanua and associates (2021) noted that pathologic complete response (pCR) to neoadjuvant chemotherapy (NAC) is strongly associated with favorable outcome.  These investigators examined the use of serial ctDNA testing for predicting pCR and risk of metastatic recurrence.  Cell-free DNA was isolated from 291 plasma samples of 84 high-risk early breast cancer patients treated in the neoadjuvant I-SPY 2 TRIAL with standard NAC alone or combined with MK-2206 (AKT inhibitor) treatment.  Blood was collected at pre-treatment (T0), 3 weeks after initiation of paclitaxel (T1), between paclitaxel and anthracycline regimens (T2), or prior to surgery (T3).  A personalized ctDNA test was designed to detect up to 16 patient-specific mutations (from whole-exome sequencing of pre-treatment tumor) in cfDNA by ultra-deep sequencing.  The median follow-up time for survival analysis was 4.8 years.  At T0, 61 of 84 (73 %) patients were ctDNA-positive, which decreased over time (T1: 35 %; T2: 14 %; and T3: 9 %).  Patients who remained ctDNA-positive at T1 were significantly more likely to have residual disease after NAC (83 % non-pCR) compared with those who cleared ctDNA (52 % non-pCR; OR 4.33, p = 0.012).  After NAC, all patients who achieved pCR were ctDNA-negative (n = 17, 100 %).  For those who did not achieve pCR (n = 43), ctDNA-positive patients (14 %) had a significantly increased risk of metastatic recurrence [HR 10.4; 95 % CI: 2.3 to 46.6]; interestingly, patients who did not achieve pCR but were ctDNA-negative (86 %) had excellent outcome, similar to those who achieved pCR (HR 1.4; 95 % CI: 0.15 to 13.5).  The authors concluded that the findings of this study showed promise that early response prediction by highly sensitive ctDNA analysis in high-risk early breast cancer patients may facilitate a timely and judicious change in treatment to improve patientsâ chances of achieving favorable long-term outcomes.  The I-SPY 2 TRIAL provided an excellent platform to examine how personalized ctDNA testing could complement imaging and pathologic evaluation of tumor response to fine-tune pCR as a surrogate endpoint for improved survival.  These researchers stated that dynamic monitoring of ctDNA during NAC could facilitate evaluation of new agents by providing an early endpoint of treatment efficacy.  Response over time as measured by imaging and ctDNA in the setting of early (pCR) and late (DRFS) outcomes will provide a robust framework for examining the potential clinical utility of ctDNA in the neoadjuvant setting.\n",
      "FIP1L1-PDGFRA Fusion Oncogene\n",
      "Patnaik et al (2007) noted that systemic mastocytosis is characterized by abnormal growth and accumulation of neoplastic mast cells in various organs.  The clinical presentation is varied and may include skin rash, symptoms related to release of mast cell mediators, and/or organopathy from involvement of bone, liver, spleen, bowel, or bone marrow.  These investigators reviewed pathogenesis, disease classification, clinical features, diagnosis, and treatment of mast cell disorders; they examined pertinent literature emerging during the last 20 years in the field of mast cell disorders.  The authors concluded that the cornerstone of diagnosis is careful bone marrow histologic examination with appropriate immunohistochemical studies.  Ancillary tests such as mast cell immunophenotyping, cytogenetic/molecular studies, and serum tryptase levels assist in confirming the diagnosis.  Patients with cutaneous disease or with low systemic mast cell burden are generally managed symptomatically.  In the patients requiring mast cell cytoreductive therapy, treatment decisions are increasingly being guided by results of molecular studies.  Most patients carry the kit D816V mutation and are predicted to be resistant to imatinib mesylate (Gleevec) therapy.  In contrast, patients carrying the FIP1L1-PDGFRA mutation achieve complete responses with low-dose imatinib therapy.  Other therapeutic options include use of interferon-alpha, chemotherapy (2-chlorodeoxyadenosine), or novel small molecule tyrosine kinase inhibitors currently in clinical trials.\n",
      "Tefferi et al (2008) stated that current classification and diagnosis of systemic mastocytosis, and its distinction from other myeloid malignancies associated with bone marrow mastocytosis, remain challenging for both clinicians and hematopathologists.  In its upcoming revision, due out in 2008, the World Health Organization (WHO) classification system for myeloid malignancies considers mast cell disease as a myeloproliferative neoplasm and systemic mastocytosis as a subcategory of mast cell disease with bone marrow involvement.  At the same time, the WHO document distinguished the usually KIT-mutated systemic mastocytosis from myeloid neoplasms associated with bone marrow mastocytosis and PDGFR mutations (e.g., FIP1L1-PDGFRA, PRKG2-PDGFRB).  The latter are often associated with eosinophilia or basophilia and sensitive to treatment with imatinib.  WHO-defined systemic mastocytosis is sometimes associated with a clonally-related second myeloid neoplasm, which is not surprising considering its origin as a stem cell disease with multi-lineage clonal involvement.  Conversely, an otherwise well-defined myeloid malignancy, such as myelodysplastic syndrome or a non-mast cell disease myeloproliferative neoplasm, might harbor neoplastic mast cells.   The authorsâ approach to diagnosis in systemic mastocytosis starts with bone marrow examination with tryptase staining and mast cell CD25 immunophenotyping.  The former enhances morphologic and the latter immunophenotypic distinction between normal (round and CD25-negative) and abnormal (spindle-shaped and CD25-positive) mast cells.  Bone marrow examination also allows detection of a 2nd hematologic neoplasm, if present.  In addition, in the presence of blood eosinophilia, these investigators screened for FIP1L1-PDGFRA, using either FISH or RT-PCR.  By contrast, they relied on conventional cytogenetics to identify cases of bone marrow mastocytosis associated with a PDGFRB re-arrangement (i.e., chromosomal translocations involving 5q31-32).  In general, the authors considered mutation screening for KITD816V and measurement of serum tryptase or urinary histamine metabolites as being complementary for the diagnosis of mast cell disease.  It is to be noted that the likelihood of detecting a KIT mutation is significantly higher with the use of both highly sensitive PCR-based assay and mast cell-enriched test samples.\n",
      "An UpToDate review on âAdvanced systemic mastocytosis: Management and prognosisâ (Gotlib, 2021) states that âImatinib is generally effective only for unmutated KIT or KIT mutations outside of exon 17.  Case reports have reported sensitivity to imatinib for SM with mutations in exons 8 to 10 of KIT: F522C (transmembrane mutation), germline K509I mutation, deletion of codon 419 in exon 8, and p.A502_Y503dup exon 9 mutation.  It is important to recognize that many previously reported responses to imatinib were likely to be rare KIT mutations that are sensitive to imatinib or misdiagnoses (e.g., FIP1L1-PDGFRA-positive myeloid/lymphoid neoplasms with eosinophilia that can also exhibit an increase in bone marrow MC numbers and elevated serum tryptase levels)â.\n",
      "Furthermore, National Comprehensive Cancer Networkâs clinical practice guideline on âSystemic mastocytosisâ (Version 1.2020) provides the following information:\n",
      "\n",
      "Screen for FIP1L1-PDGFRA if eosinophilia is present\n",
      "Useful in certain circumstances: Imatinib (only if KIT D816V mutation negative or unknown or if eosinophilia is present with FIP1L1-PDGFRA fusion gene.  (In cases with a primarily interstitial pattern of mast cells, peripheral blood eosinophilia, and negativity of KIT D816V mutation, then the FIP1L1-PDGFRA fusion gene should be tested).  The FIP1L1-PDGFRA fusion oncogene should be tested in patients with eosinophilia in peripheral blood who do not have the KIT D816V mutation.\n",
      "\n",
      "Oncotype MAP PanCancer and Oncomap ExTra\n",
      "The Oncotype MAP PanCancer Tissue Test (Paradigm Diagnostics, Inc., Phoenix, AZ) is an NGS-based test that is used to identify genetic alterations among 257 genes to match appropriate targeted therapy for tumor mutation burden of solid tumors.  However, there is currently insufficient evidence to support its clinical value.\n",
      "Kuipers and co-workers (2018) noted that large-scale genomic data highlight the complexity and diversity of the molecular changes that drive cancer progression.  Statistical analysis of cancer data from different tissues could guide drug re-positioning and the design of targeted therapies.  These researchers developed an improved Bayesian network model for tumor mutational profiles and applied it to 8,198 patient samples across 22 cancer types from the Cancer Genome Atlas (TCGA).  For each cancer type, these investigators identified the interactions between mutated genes, capturing signatures beyond mere mutational frequencies.  When comparing mutation networks, they found genes that interact both within and across cancer types.  To detach cancer classification from the tissue type , these researchers carried out de-novo clustering of the PanCancer mutational profiles based on the Bayesian network models.  They found 22 novel clusters that significantly improved survival prediction beyond clinical information.  The authors concluded that the models highlighted key gene interactions for each cluster potentially allowing genomic stratification for clinical trials and identifying drug targets.\n",
      "Li et al (2019) stated that how the immune micro-environment changes during neoadjuvant chemotherapy of primary breast cancer is not well understood.  These researchers analyzed pre- and post-treatment samples from 60 patients using the NanoString PanCancer IO360 assay to measure the expression of 750 immune-related genes corresponding to 14 immune cell types and various immune functions, and assessed tumor infiltrating lymphocyte (TIL) counts and PD-L1 protein expression by immunohistochemistry.  Treatment associated changes in gene expression levels were compared using t-test with Bonferroni correction.  TIL count, PD-L1 protein and immune metagenes were compared using Wilcoxon test.  Baseline immune markers were correlated with pCR using estrogen receptor (ER) and treatment arm adjusted logistic regression.  At baseline, high TIL counts and high expression of chemoattractant cytokines (CCL21, CCL19) and cytotoxic T cell markers were associated with higher pCR rate.  High expression of stromal genes (VEGFB, TGFB3, PDGFB, FGFR1, IGFR1), mast and myeloid inflammatory cell metagenes, stem cell related genes (CD90, WNT11, CTNNB1) and CX3CR1, and IL11RA were associated with residual disease (RD).  After treatment, in cases with pCR, TIL counts and most immune genes decreased significantly.  Among RD cases, TIL counts and PD-L1 expression did not change but cellular stress and hypoxia associated genes (DUSP1, EGR1), and IL6, CD36, CXCL2, CD69 and the IL8/VEGF metagene increased.  The authors concluded that their analysis suggested that CCL19, CCL21 and IL7 signaling play an important role in attracting and activating immune cells in the breast cancer micro-environment and might help convert immune cold cancers to immune hot.  The presence of activated cytotoxic T cells with granzyme and granulysin expression were important for achieving pCR with chemotherapy.  The expression of CXCL1, CXCL3, CXCL2, CCL20, and IL6 were enriched in residual cancer and were associated with lesser response the chemotherapy when highly expressed at baseline.  These molecules represented potential novel targets to increase pCR rates and improve outcome in patients with residual cancer following chemotherapy.\n",
      "The authors stated that this study had several drawbacks.  The small sample size prevented them from adequately powered analysis by ER groups, which possess different immunologic characteristics and chemotherapy sensitivities.  Several other potentially important subset analyses that these investigators performed had limited power.  They observed no significant difference in survival among residual disease cases that showed increase in TIL count from baseline compared to those that had a decline.  Furthermore, they failed to detect significant association between baseline PD-L1 expression and pCR, as they reported earlier in larger cohorts; these findings most likely reflected under-powered analyses.  These investigators also recognized that their pre- and post-treatment sample comparisons may be influenced by unavoidable sampling bias and treatment-related shifts in tumor cellularity.  All baseline tissues were from core needle biopsies and all post-treatment samples were from surgically resected tissues and post-treatment samples generally had lower tumor cellularity even in cases with residual disease.\n",
      "Vishwakarma and colleagues (2020) stated that long non-coding RNAs (lncRNAs) are a group of non-protein-coding RNAs that are longer than 200 nucleotides.  LncRNAs play important roles in epigenetic modification, transcription and post-transcriptional regulation, maintenance of normal tissue development and differentiation.  LncRNA could serve as a biomarker for diagnosis and prognosis as well as a molecular target for therapy in oral squamous cell carcinoma (OSCC).  These researchers examined the expression profile of 5-lncRNAs; namely UCA1, TUG1, HOTAIR, MALAT1, and H19 by quantitative real-time PCR in tumor tissues and adjacent normal tissue of 32 patients with OSCC.  To determine the expression, methylation status and genomic alterations in lncRNAs across pancancer, The Cancer Genome Atlas (TCGA) datasets were analyzed by UALCAN, MEXPRESS and cBioPortal database.  Then, these investigators examined the association between lncRNA expression and clinicopathological attributes of patients by Spearman's rank test.  Expression of UCA1 and TUG1 genes was up-regulated in 54.83 % and 53.12 % OSCC tumors, respectively.  More importantly, expression of MALAT1 and H19 was down-regulated in tumor tissues of 62.5 % and 81.25 %, respectively of patients with OSCC.  Except for MALAT1, these experimental data showed concordance with the TCGA analysis.  Expression of HOTAIR in OSCC tumors was positively correlated with tumor volume, whereas MALAT1 and H19 negatively correlated with the smoking status of patients.\n",
      "The authors state that the Indian subcontinent represents one of the major contributors to oral cancer cases; however, only a few studies have compared the expression of lncRNAs with oral cancer in the Indian subcontinent.  In a study, Linc-RoR has been shown to be over-expressed in undifferentiated oral tumors and showed a strong association with tumor recurrence and poor therapeutic response.  The same group has also shown that OIP5-AS1 lncRNA was over-expressed in oral tumors and in tumors of epithelial origin from the TCGA database.  As only limited data is available in the International Consortium for lncRNA expression in OSCC from India patients, further studies are needed to ascertain their prognostic role in OSCC.\n",
      "Vijayashree Priyadharsini and Paramasivam (2020) noted that abnormalities in the antioxidant pathway are usually associated with inflammatory conditions, followed by tissue damage.  Cancer is one such disease where there is a build-up of reactive oxygen species leading to pathological consequences.  These researchers identified the alteration in genes and proteins associated with the common antioxidant pathways among patients with head and neck SCC (HNSCC).  The study design followed a retrospective approach and employed computational tools to analyze the possible role of genes involved in the anti-oxidation pathways in patients with HNSCC.  The TCGA PanCancer Atlas dataset was employed for the analysis.  The Oncoprint data were analyzed further to obtain information on the type of gene alterations encountered in the HNSCC cases.  Gene amplification and deletions were commonly observed in genes of the thiol reductase pathway, whereas substitutions leading to missense, frameshifts were found in the other pathways assessed.  Gene encoding ceruloplasmin was found to harbor nucleotide variations in approximately 10 % of the patients with OSCC.  The authors concluded that an exhaustive knowledge of the molecular genetic mechanisms underlying the pathways identified could open new avenues in selecting candidate genes that can be used as therapeutic targets against HNSCC.  This study identified and nominated crucial genes from the antioxidant system for further in-vitro experiments.\n",
      "Jordan and colleagues (2020) stated that mutations in the MAP kinase pathway (KRAS, NRAS, BRAF) are common in low-grade serous ovarian carcinoma (LGSOC).  The effect of these and other mutations on RNA transcription in this disease is poorly understood.  These researchers described patterns of somatic mutations and gene transcription in a racially diverse population with LGSOC.  By means of an institutional tumor registry, patients with LGSOC were identified and charts were reviewed.  RNA was extracted from available tumor tissue.  Commercial tumor profiling results were analyzed with PanCancer pathway nanoString mRNA expression data.  Along with nanoString n-Solver software, Chi-squared, Fishers Exact, and Cox proportional hazards models were used for statistical analysis, with significance set at p < 0.05.  A total of 39 patients were identified â 20 % Black, 43 % Hispanic, and 36 % non-Hispanic White; 18 patients had commercial somatic DNA test results, and 23 had available tumor tissue for RNA extraction and nanoString analysis.  The most common somatic alterations identified was KRAS (11 patients, 61 %), followed by ERCC1 and TUBB3 (9 each, 50 %).  KRAS mutations were less common in smokers (14.3 % versus 90.9 %, p = 0.002).  RNA expression analysis demonstrated a greater than 2-fold decrease in expression of HRAS in tumors from older patients (p = 0.04), and a greater than 2-fold decrease in the expression of HRAS in recurrent tumors (p = 0.007).  No significant differences were observed in somatic testing results, RNA expression analysis, or PFS between different racial and ethnic cohorts.  The authors concluded that clinical and molecular understanding of LGSOC continues to rapidly evolve.  Similarities in outcomes and molecular characteristics shared between races and ethnicities in this cohort are encouraging for the applicability of therapeutic strategies across a diverse patient population.  These researchers stated that broader analyses of germline, somatic, and RNA expression data will provide insight into potential therapeutic targets tailored to individual patients and their cancers.\n",
      "The authors stated that the findings of this study are hypothesis-generating, highlighting the need for more research into gene expression alterations in LGSOCs.  Using a commercial NGS database or establishing a large prospective study to analyze a large cohort of known LGSOCs would allow for greater statistical power in comparing somatic NGS data with mRNA expression analysis.  A large tumor registry would facilitate evaluation of primary and recurrent tumors from each patient, allowing direct genetic and mRNA expression analysis between tumors in distinct clinical settings.  Protein expression analysis using tissue microarrays would allow for further characterization of the trends these researchers reported in RNA expression.  Finally, repeating this study using a larger cohort of samples from multiple institutions would provide greater statistical power to detect the clinical implications of differences in mRNA expression across tumors, as well as between racial and ethnic groups.\n",
      "Finn and associates (2021) noted that in the REFLECT study, lenvatinib demonstrated an effect on OS by confirmation of non-inferiority to sorafenib in unresectable HCC.  This analysis examined correlations between serum or tissue biomarkers and efficacy outcomes from the REFLECT study.  Serum biomarkers (VEGF, ANG2, FGF19, FGF21, and FGF23) were measured by ELISA.  Gene expression in tumor tissues was measured by the nCounter PanCancer Pathways Panel.  Pharmacodynamic changes in serum biomarker levels from baseline, and associations of clinical outcomes with baseline biomarker levels were evaluated.  A total of 407 patients were included in the serum analysis set (lenvatinib n = 279, sorafenib n = 128); 58 patients were included in the gene-expression analysis set (lenvatinib n = 34, sorafenib n = 24).  Both treatments were associated with increases in VEGF; only lenvatinib was associated with increases in FGF19 and FGF23 at all timepoints.  Lenvatinib-treated responders had greater increases in FGF19 and FGF23 versus non-responders at C4D1 (FGF19: 55.2 % versus 18.3 %, p = 0.014; FGF23: 48.4 % versus 16.4 %, p = 0.0022, respectively).  Higher baseline VEGF, ANG2, and FGF21 correlated with shorter OS in both treatment groups; OS was longer for lenvatinib than sorafenib (median of 10.9 versus 6.8 months, respectively; HR, 0.53; 95 % CI: 0.33 to 0.85; p = 0.0075; p-interaction = 0.0397) with higher baseline FGF21.  In tumor tissue biomarker analysis, VEGF/FGF enriched groups showed improved OS with lenvatinib versus the intermediate VEGF/FGF group (HR 0.39; 95 % CI: 0.16 to 0.91; p = 0.0253).  The authors concluded that serum biomarker and gene-expression levels appeared to correlate with survival outcomes among patients with evaluable samples from the REFLECT study.  This analysis was limited by the small number of patients with evaluable samples, and the variation in baseline characteristics between the gene-expression analysis set and the ITT population.  However, the differences in baseline characteristics between these groups were understandable and most likely due to the small size of the gene-expression analysis set.  Despite these limitations, multi-variate analysis of important clinical prognostic factors in HCC supports these findings.  Of note, data were analyzed by mRECIST per a blinded independent review.  In patients with unresectable HCC who had not received prior systemic therapy for advanced disease, lenvatinib demonstrated clinical activity based on a mechanism of action that is distinct from sorafenib.  Specifically, lenvatinib demonstrated clinical evidence of FGFR inhibition and stronger inhibition of angiogenesis pathways (VEGFR and TIE-2/ANG2).  ANG2 and TIE-2 are selectively expressed on endothelial cells and are increased with enhanced tumor angiogenesis.  Lenvatinib could lead to decreases in both ANG2 and TIE-2, without direct TIE-2 inhibition, based on its potent angiogenesis inhibition and resultant decrease in endothelial cells.  These researchers stated that these findings suggested that the inhibitory activity of lenvatinib against FGFR may contribute to the increased tumor response, and FGF21 may be a candidate biomarker predictive of longer OS with lenvatinib.  Interestingly, it appeared that Lenvatinib may perform better in the poor prognosis sub-groups independent of the specific pathway, due to its increased activity overall and similarity to the overall study population results.  These results are hypothesis-generating and warrant further study.  The ongoing phase-III LEAP-002 study (NCT03713593), evaluating lenvatinib versus lenvatinib and pembrolizumab in advanced HCC will provide further material for investigation.\n",
      "The Oncomap ExTra (Exact Sciences Inc., Genomic Health Inc.), formerly known as Oncotype Map and GEM ExTra, respectively, is an oncology (neoplasia) test that conducts exome and transcriptome sequence analysis for sequence variants, gene copy number amplifications and deletions, gene rearrangements, microsatellite instability and tumor mutational burden utilizing DNA and RNA from tumor with DNA from normal blood or saliva for subtraction. This test is aimed to report clinically significant mutation(s) with therapy associations.\n",
      "PanGIA Prostate\n",
      "PanGIA Prostate (Genetics Institute of America) is a multi-analyte urine assay with algorithmic analysis that estimates an individualâs risk of having prostate cancer.  The test is marketed as a method to examine if a patient should undergo a prostate biopsy.  Currently, there is a lack of evidence on the clinical value of the PanGIA Prostate.\n",
      "PGDx elio Tissue Complete for Tumor Mutation Profiling\n",
      "The PGDx elio tissue complete assay (Personal Genome Diagnostics, Inc.) is a comprehensive, qualitative in-vitro diagnostic approach that uses targeted NGS of DNA isolated from formalin-fixed, paraffin-embedded tumor tissue from patients with solid malignant neoplasms to detect tumor gene alterations in a broad multi-gene panel.  It is intended to provide tumor mutation profiling information on somatic alterations (SNVs, small insertions and deletions [indels], amplifications, translocations), MSI) as well as TMB for use in oncology for previously diagnosed cancer patients.\n",
      "Labriola and colleagues (2020) noted that immune checkpoint inhibitors (ICIs) have expanded therapeutic options for metastatic RCC (mRCC); however, there are limited predictive biomarkers for response to ICIs in this indication, with programmed death-ligand 1 (PD-L1) status demonstrating little predictive utility in mRCC.  While predictive of ICI response in other tumor types, the use of TMB in mRCC is unclear.  These researchers examined TMB, loss of antigen presentation genes and PD-L1 status correlated with outcomes to ICI treatment in mRCC.  Tumor samples from 34 patients with mRCC treated with ICI therapy at Duke Cancer Institute were retrospectively evaluated using Personal Genome Diagnostics elio tissue complete (RUO version), a tumor genomic profiling assay for somatic variants, TMB, MSI and genomic status of antigen presentation genes.  Tumor samples were also analyzed with the Dako 28-8 PD-L1 immunohistochemistry assay.  De-identified clinical information was extracted from the medical record, and tumor response was examined based on the RECIST V.1.1 criteria.  Patients were stratified by overall response following ICI therapy and designated as progressive disease (PD; n = 18) or disease control groups (DC; n = 16); TMB scores ranged from 0.36 to 12.24 mutations/Mb (mean of 2.83 mutations/Mb) with no significant difference between the PD and DC groups (3.01 versus 2.63 mutations/Mb, respectively; p = 0.7682).  Interestingly, 33 % of PD patients showed loss of heterozygosity of major histocompatibility complex class I genes (LOH-MHC) versus 6 % of DC patients; 9 of 34 samples were PD-L1-positive (4 in the PD group; 5 in the DC group), suggesting no correlation between PD-L1 expression and response to ICI therapy.  Notably, the DC group showed an enrichment of mutations in DNA repair genes (p = 0.04), with 68.8 % exhibiting at least 1 mutated homologous recombination repair (HRR)-related gene compared with only 38.9 % of the PD group (p = 0.03).  The authors concluded that this study associated clinical outcomes of patients with mRCC treated with ICI with tissue analyzed for several biomarkers including genetic correlates, TMB, MHC loss and PD-L1 expression.  TMB alone, PD-L1 status alone, and combined TMB with PD-L1 status did not correlate with ICI outcomes, which is consistent with prior observations.  Interestingly, non-responders showed an increased incidence of LOH-MHC, warranting future investigations to examine if antigen presentation may serve as a predictor of ICI response in patients with mRCC.  Finally, ICI responders had more frequent mutations in DNA damage response genes than non-responders, especially within HRR genes.  Moreover, these researchers stated that further investigation on the tumor micro-environment and engagement of RCC tumors with the immune system is needed to better understand response and predict for ICI treatment outcomes.\n",
      "The authors stated that in this study, single biomarkers such as PD-L1 and TMB status did not correlate with ICI outcomes; however, the correlation of LOH-MHC with ICI resistance and correlation of HRR gene mutations with ICI outcomes in this cohort may be hypothesis-generating for future studies.  While this study did reveal multiple interesting associations, there were several drawbacks and remaining questions that require further inquiry.  First, this analysis was based on archival tissue specimens and carried out retrospectively; thus, these findings should be confirmed in a prospectively collected patient cohort.  A correlation between DNA damage response gene mutations and increased TMB was observed, aligning with previous reports in several indications that defects in HRR/DDR pathways may lead to increased TMB.  It should be noted, however, that in this study, this association may in part be due to the fact that the mutations in these genes contribute to the TMB calculation; thus, it is challenging to specifically attribute increased TMB to HRR/DDR mutations or discern a causal relationship from these data alone.  Given the biological implications of mutations in DNA damage response genes leading to neoantigen generation, patients who were concomitant with LOH-MHC might indicate a subset of non-responders to ICI treatment.  Although this correlation was not observed in this study, only 7 patient samples were identified as LOH-MHC; thus, this study was not sufficiently powered for this analysis, warranting future studies.  Despite TMB scores being relatively low in mRCC, patients benefited from ICI therapy.  Although TMB was merely a surrogate measure for neoantigen burden, the actual estimation remained difficult due to imperfect bioinformatics approaches, highly polymorphic MHC genes and an overall lack of understanding of neoantigen immunogenicity.  Direct assessment of neo-antigenic burden is beyond the scope of this study; however, given the clinical implications, further examination would be beneficial.\n",
      "Immunoscore for Estimating Risk of Recurrence or Determining Adjuvant Therapy in Patients with Colon Cancer\n",
      "Immunoscore is a test that predicts the risk of relapse in patients with localized colon cancer (CC); thus, facilitating the chemotherapy decision-making process.  It is a tissue-based immune assay carried out on formalin-fixed paraffin-embedded (FFPE) tumor tissue samples of primary colon cancer intended to measure the host immune response at the tumor site.  In combination with standard clinicopathologic features, Immunoscore informs adjuvant chemotherapy decision-making for patients with early-stage CC.  Immunoscore values are reported based on pre-defined cut-offs in 5 categorical scores (IS 0 to 4) and in 2 categories of recurrence risk: Immunoscore Low (IS 0 and1) and Immunoscore High (IS 2 to 4), with a higher Immunoscore associated with a lower risk of recurrence\n",
      "Pages and colleagues (2018) noted that the estimation of risk of recurrence for patients with CC must be improved.  A robust immune score quantification is needed to introduce immune parameters into cancer classification.  These researchers examined the prognostic value of total tumor-infiltrating T-cell counts and cytotoxic tumor-infiltrating T-cells counts with the consensus Immunoscore assay in patients with stage I to III CC.  An international consortium of 14 centers in 13 countries, led by the Society for Immunotherapy of Cancer (SIC), evaluated the Immunoscore assay in patients with TNM stage I to III CC.  Patients were randomly assigned to a training set, an internal validation set, or an external validation set.  Paraffin sections of the colon tumor and invasive margin from each patient were processed by immunohistochemistry, and the densities of CD3+ and cytotoxic CD8+ T cells in the tumor and in the invasive margin were quantified by digital pathology.  An Immunoscore for each patient was derived from the mean of 4 density percentiles.  The primary endpoint was to examine the prognostic value of the Immunoscore for time to recurrence, defined as time from surgery to disease recurrence.  Stratified multi-variable Cox models were used to examine the associations between Immunoscore and outcomes, adjusting for potential confounders.  Harrell's C-statistics was used to assess model performance.  Tissue samples from 3,539 patients were processed, and samples from 2,681 patients were included in the analyses after quality controls (700 patients in the training set, 636 patients in the internal validation set, and 1,345 patients in the external validation set).  The Immunoscore assay showed a high level of reproducibility between observers and centers (r = 0.97 for colon tumor; r = 0.97 for invasive margin; p < 0.0001).  In the training set, patients with a high Immunoscore had the lowest risk of recurrence at 5 years (14 [8 %] patients with a high Immunoscore versus 65 (19 %) patients with an intermediate Immunoscore versus 51 (32 %) patients with a low Immunoscore; HR for high versus low Immunoscore 0.20, 95 % CI: 0.10 to 0.38; p < 0.0001).  The findings were confirmed in the 2 validation sets (n = 1,981).  In the stratified Cox multi-variable analysis, the Immunoscore association with time to recurrence was independent of patient age, sex, T stage, N stage, MSI, and existing prognostic factors (p < 0.0001).  Of 1,434 patients with stage II CC, the difference in risk of recurrence at 5 years was significant (HR for high versus low Immunoscore 0.33, 95 % CI: 0.21 to 0.52; p < 0.0001), including in Cox multi-variable analysis (p < 0.0001).  Immunoscore had the highest relative contribution to the risk of all clinical parameters, including the AJCC and UICC TNM classification system.  The authors concluded that the Immunoscore provided a reliable estimate of the risk of recurrence in patients with CC.  These results supported the implementation of the consensus Immunoscore as a new component of a TNM-Immune classification of cancer.\n",
      "Mlecnik and associates (2020) examined the prognostic value of Immunoscore in patients with stage III CC and analyzed its association with the effect of chemotherapy on time to recurrence (TTR).  An international study led by the SIC evaluated the pre-defined consensus Immunoscore in 763 patients with AJCC/UICC TNM stage III CC from cohort 1 (Canada/U.S.) and cohort 2 (Europe/Asia).  CD3+ and cytotoxic CD8+ T lymphocyte densities were quantified in the tumor and invasive margin by digital pathology.  The primary endpoint was TTR.  Secondary endpoints were OS, DFS, prognosis in microsatellite stable (MSS) status, and predictive value of efficacy of chemotherapy.  Patients with a high Immunoscore presented with the lowest risk of recurrence, in both cohorts.  Recurrence-free rates at 3 years were 56.9 % (95 % CI: 50.3 % to 64.4 %), 65.9 % (95 % CI: 60.8 % to 71.4 %), and 76.4 % (95 % CI: 69.3 % to 84.3 %) in patients with low, intermediate, and high Immunoscores, respectively (HR; high versus low, 0.48; 95 % CI: 0.32 to 0.71; p = 0.0003).  Patients with high Immunoscore showed significant association with prolonged TTR, OS, and DFS (all p < 0.001).  In Cox multi-variable analysis stratified by participating center, Immunoscore association with TTR was independent (HR; high versus low, 0.41; 95 % CI: 0.25 to 0.67; p = 0.0003) of patient's sex, T stage, N stage, sidedness, and MSI status.  Significant association of a high Immunoscore with prolonged TTR was also found among MSS patients (HR; high versus low, 0.36; 95 % CI: 0.21 to 0.62; p = 0.0003).  Immunoscore had the strongest contribution Ï2 proportion for influencing survival (TTR and OS).  Chemotherapy was significantly associated with survival in the high-Immunoscore group for both low-risk (HR; chemotherapy versus no chemotherapy, 0.42; 95 % CI: 0.25 to 0.71; p = 0.0011) and high-risk (HR; chemotherapy versus no chemotherapy, 0.5; 95 % CI: 0.33 to 0.77; p = 0.0015) patients, in contrast to the low-Immunoscore group (p > 0.12).  The authors concluded that the findings of this study showed that a high Immunoscore was significantly associated with prolonged survival in stage III CC; these findings suggested that patients with a high Immunoscore would benefit the most from chemotherapy in terms of recurrence risk.\n",
      "Sinicrope et al (2020) noted that the AJCC staging and other prognostic tools fail to account for stage-independent variability in outcome.  These researchers developed a prognostic classifier adding Immunoscore to clinicopathological and molecular features in patients with stage III CC.  Patient (nâ=â559) data from the FOLFOX arm of adjuvant trial NCCTG N0147 were used to construct Cox models for predicting DFS.  Variables included age, sex, T stage, positive lymph nodes (+LNs), N stage, performance status, histologic grade, sidedness, KRAS/BRAF, mismatch repair, and Immunoscore (CD3+, CD8+ T-cell densities).  After determining optimal functional form (continuous or categorical) and within Cox models, backward selection was carried out to analyze all variables as candidate predictors.  All statistical tests were 2-sided.  Poorer DFS was found for tumors that were T4 versus T3 (HR = 1.76, 95 % CI: 1.19 to 2.60; pâ=â0.004), right- versus left-sided (HR = 1.52, 95 % CI: 1.14 to 2.04; pâ=â0.005), BRAF V600E (HR = 1.74, 95 % CI: 1.26 to 2.40; pâ<â0.001), mutant KRAS (HR = 1.66, 95 % CI: 1.08 to 2.55; pâ=â0.02), and low versus high Immunoscore (HR = 1.69, 95 % CI: 1.22 to 2.33; p =â0.001) (all pâ<â0.02).  Increasing numbers of +LNs and lower continuous Immunoscore were associated with poorer DFS that achieved significance (both p <â0.0001).  After number of +LNs, T stage, and BRAF/KRAS, Immunoscore was the most informative predictor of DFS shown multivariately.  Among T1â3 N1 tumors, Immunoscore was the only variable associated with DFS that achieved statistical significance.  A nomogram was generated to determine the likelihood of being recurrence-free at 3âyears.  The authors concluded that the Immunoscore could enhance the accuracy of survival prediction among patients with stage III CC.\n",
      "The authors stated that a potential drawback of this study was the generalizability of results to patients who might not resemble those eligible for enrollment in the clinical trial.  These researchers stated that the final model warrants external validation in an independent cohort of FOLFOX-treated, stage III patients.  Because all patients received adjuvant chemotherapy, these investigators were unable to examine the predictive impact of co-variates for chemotherapy response.  Relevant to this issue are data indicating that oxaliplatin may increase cytotoxic T-cell infiltration and may induce immunogenic cell death.\n",
      "Pages and co-workers (2020) stated that the Immunoscore (IS), which prognostically classifies stage I to III CC patients, was evaluated in the International Duration Evaluation of Adjuvant Therapy (IDEA) France cohort study examining 3 versus 6 months of oxaliplatin-based adjuvant chemotherapy in stage III CC patients.  Densities of CD3+ and CD8+ T cells in the tumor and invasive margin were determined by immunohistochemistry, quantified by digital pathology, and converted to IS.  Mismatch repair status was determined by immunohistochemistry or by pentaplex PCR.  Prediction of DFS by IS was analyzed by a multi-variable Cox regression model in each study arm.  Harrell's C-statistics were used to examine the IS performance.  Samples of 1,322 patients were available.  IS low, intermediate (Int), and high were observed in 43.6 %, 47.0 %, and 9.4 % of patients, respectively.  IS low identified patients at higher risk of relapse or death compared with Int + high [HR = 1.54; 95 % CI: 1.24 to 1.93, p = 0.0001].  The 3-year DFS was 66.80 % (95 % CI: 62.23 to 70.94) for IS low and 77.14 % (95 % CI: 73.50 to 80.35) for IS Int + high.  In multi-variable analysis, IS remained significantly independently associated with DFS (p = 0.003) when adjusted for sex, histological grade, T/N stage, and MSI status.  For mFOLFOX6-treated patients (91.6 % of the cohort), a statistically significant interaction was observed for the predictive value of IS for treatment duration (3 versus 6 months) in terms of DFS (p = 0.057). IS Int + high significantly predicted benefit of 6 months of treatment (HR = 0.53; 95 % CI: 0.37 to 0.75; p = 0.0004), including clinically low- and high-risk stage III CC (all p < 0.001).  Conversely, patients with IS low (46.4 %) did not significantly benefit from the 6-month mFOLFOX6 versus the 3-month mFOLFOX6.  The authors concluded that the prognostic value of IS for DFS was confirmed in patients with stage III CC treated with oxaliplatin-based chemotherapy.  Its predictive value for DFS benefit of longer duration of mFOLFOX6 adjuvant treatment was found in IS Int + high.  Moreover, these researchers stated that these findings will be validated in an external independent cohort.\n",
      "The authors stated that a drawback of this trial was that 90 % of patients in the IDEA France study were treated with the mFOLFOX6 regimen, which precluded any robust conclusion for patients receiving CAPOX.  Furthermore, the median follow-up of the overall mITT population was 4.3 years; thus, it is still impossible to analyze the IS impact on long-term treatment benefit.  They stated that the predictive value of IS needs now to be confirmed in FOLFOX- and/or CAPOX-treated patients in another cohort of the IDEA collaboration to validate the potential use of the IS test in guiding the choice of duration of adjuvant therapy.\n",
      "Zaborowski et al (2021) noted that colorectal cancer (CRC) represents the 2nd leading cause of cancer-related death worldwide.  The therapeutic field of immuno-oncology has rapidly gained momentum, with strikingly promising results observed in clinical practice.  Increasing emphasis has been placed on the role of the immune response in tumorigenesis, therapy and predicting prognosis.  Enhanced understanding of the dynamic and complex tumor-immune microenvironment has enabled the development of molecularly directed, individualized treatment.  Analysis of intra-tumoral lymphocyte infiltration and the dichotomization of CRC into microsatellite stable and unstable disease has important therapeutic and prognostic implications, with potential to capitalize further on these data.  These researchers discussed the latest evidence surrounding the tumor biology and immune landscape of CRC, novel immunotherapies and the interaction of the immune system with each apex of the tripartite of cancer management (oncotherapeutics, radiotherapy and surgery).  By using the synergy of chemotherapeutic agents and immunotherapies, and identifying prognostic and predictive immunological biomarkers, researchers may enter an era of unprecedented disease control, survivorship and cure rates.  These investigators stated that recent data (Pages et al, 2020) suggested intra-tumoral immune response as measured by the Immunoscore may predict the therapeutic benefit of adjuvant oxaliplatin-based chemotherapy in patients with stage III CC.\n",
      "Furthermore, National Comprehensive Cancer Networkâs clinical practice guideline on âColon cancerâ (Version 3.2021) states that âSeveral assays have been developed in hopes of providing prognostic and predictive information to aid in decisions regarding adjuvant therapy in patients with stage II or III colon cancer â¦ the information from these tests can further inform the risk of recurrence over other risk factors, but the panel questions the value added.  Furthermore, evidence of predictive value in terms of the potential benefit of chemotherapy is lacking.  Therefore, the panel believes that there are insufficient data to recommend the use of multigene assays, Immunoscore, or post-surgical ctDNA to estimate risk of recurrence or determine adjuvant therapyâ.\n",
      "OncoSignal Test for Analysis of Solid Tumors\n",
      "According to Protean BioDiagnostics, OncoSignal is a new way to analyze breast cancer and other cancers.  The OncoSignal test uses an advanced molecular and bioinformatics system to measure mRNA expression patterns and calculate the specific activity of 7 key oncogenic driver signal pathways, which include ER (estrogen receptor), AR (androgen receptor), PI3K (Phosphoinositide 3-Kinase), HH (Hedgehog pathway), NOTCH (notch signal pathways), TGFbeta (transforming growth factor receptor beta), and MAPK (mitogen activated protean kinase).  The pathways measure key oncogenic drivers of numerous distinct cancer types including but not limited to breast cancer, prostate cancer, ovarian cancer, colon cancer, lymphoma and more. \n",
      "van de Stolpe et al (2021) stated that precision treatment of cancer requires knowledge on active tumor driving signal transduction pathways (STPs) to select the optimal effective targeted treatment.  Currently, only a subset of patients derives clinical benefit from mutation-based targeted therapy, due to intrinsic and acquired drug resistance mechanisms.  Phenotypic assays to identify the tumor driving pathway based on protein analysis are difficult to multiplex on routine pathology samples.  In contrast, the transcriptome contains information on signaling pathway activity and can complement genomic analyses.  These researchers presented the validation and clinical application of a new knowledge-based mRNA-based diagnostic assay platform (OncoSignal) for measuring activity of relevant signaling pathways simultaneously and quantitatively with high resolution in tissue samples and circulating tumor cells, specifically with very small specimen quantities.  The approach uses mRNA levels of a pathway's direct target genes, selected based on literature for multiple proof points, and used as evidence that a pathway is functionally activated.  Using these validated target genes, a Bayesian network model has been built and calibrated on mRNA measurements of samples with known pathway status, which is used next to calculate a pathway activity score on individual test samples.  Translation to RT-qPCR assays enables broad clinical diagnostic applications, including small analytes.  A large number of cancer samples have been analyzed across a variety of cancer histologies and benchmarked across normal controls.  Assays have been used to characterize cell types in the cancer cell microenvironment, including immune cells in which activated and immunotolerant states can be distinguished.  Results supported the expectation that the assays provide information on cancer driving signaling pathways, which is difficult to derive from next generation DNA sequencing analysis.  The authors concluded that clinical diagnostic applications of STP analysis for prediction of prognosis and response to therapy, including identification of resistance pathways, are currently being further developed in clinical studies.  Prospective, clinical validation studies in various cancer types and basket studies are being initiated with clinical partners, predominantly making use of RT-qPCR-based STP analysis on formalin fixed paraffin embedded (FFPE) tissue samples with considerations toward low quantity specimens including circulating tumor cells.  These researchers stated that in the future, an important focus will lie on measuring the host immune response to cancer, both in blood as well as in cancer tissue samples.  Taken together, measurement of STP activity in cancer, complementary to DNA mutation analysis, is expected to enable development of novel therapies, improve prediction of therapy response and resistance, and improve clinical outcome for a variety of tumor types and treatments, including targeted drugs and immunotherapy.\n",
      "DCISionRT\n",
      "According to the Prelude Corporation, DCISionRT is a risk assessment test for women diagnosed with ductal carcinoma in situ (DCIS). This test is designed to predict the risk of a future DCIS or invasive breast cancer recurrence in the same breast over the next 10 year period. Additionally, a prediction can be made regarding benefit from radiation therapy in reducing the risk of DCIS or invasive disease recurrence. The DCISionRT test can be performed prior to or following breast conserving surgery. The tissue sample, from biopsy or removal during surgery, is assessed for seven different biomarkers associated with key biologic pathways responsible for breast cancer progression as well as four other risk factors. A resultant DCISionRT Score is derived and is a numerical value on a scale of 0.0 to 10.0, and categorized as low or elevated (Prelude, 2021).\n",
      "Bremer and colleagues (2018) investigated the development of the DCISionRT biological signature to assess recurrence and predict radiotherapy (RT) benefit for DCIS patients following breast-conserving surgery (BCS). The calculation of an individualized Decision Score (DS) was developed and cross-validated in 526 DCIS patients treated with BCS Â± RT. A relationship assessment between DS and 10-year risk of invasive breast cancer (IBC) or any ipsilateral breast event (IBE), including IBC or DCIS was performed. Additionally, RT benefit was assessed by risk group and as a function of DS. The results indicated significant association of DS with IBC and IBE risk, HR (per 5 units) of 4.2 and 3.1, respectively. DS identified a low risk group with 10-year IBC risk of 4% (7% IBE) and an elevated risk group with IBC risk of 15% (23% IBE), in patients treated without RT. An examination of DS and RT by group showed the elevated risk group received significant RT benefit, HR of 0.3 for IBC and IBE. DS reclassified 42% of patients into the elevated risk group from a clinicopathologically low-risk subset. Furthermore, patients with an elevated DS exhibited significant RT benefit over baseline. The study concluded that DS was predictive for risk and anticipated RT benefit for DCIS patients. DS classified a clinically meaningful low-risk group and a group with elevated 10-year risks that received considerable RT benefit over baseline.\n",
      "DetermaRx\n",
      "According to Oncocyte Corporation, DetermaRx is a 14-gene molecular stratification test used to identify which patients with stage I and IIA non-squamous non-small cell lung cancer (NSCLC) have a high risk of recurrence and may benefit from adjuvant chemotherapy. Patient eligibility for DetermaRx requires individuals with stage IA, IB, or IIA non-squamous NSCLC with a tumor size less than 5 cm and no lymph node involvement, who underwent surgical resection and are currently under consideration for chemotherapy as a part of their care. Formalin-fixed paraffin-embedded (FFPE) preserved tissue with a tumor area greater than 25% of the blockâs total tissue area from the surgical resection of the tumor undergoes assay testing for 14 genes. The test results are then used to categorize patients into low, intermediate, or high-risk of disease recurrence. Currently, there is an inadequate clinical method for segregating non-squamous NSCLC patients into risk categories for disease recurrence and those who would benefit from adjuvant chemotherapy (Oncocyte, 2021).\n",
      "Kratz and colleagues (2019) evaluated the integration of a clinically validated molecular prognostic classifier into the eight edition of the TNM staging system for patients who underwent resection of non-squamous non-small cell lung cancer (NSCLC). The novel TNMB (with the B denoting biology) staging system, integrated a 14-gene expression assay (i.e., 11 cancer-related genes [BAG1, BRCA1, CDC6, CDK2AP1, ERBB3, FUT3, IL11, LCK, RND3, SH3BGR, WNT3A] and three reference genes [ESD, TBP, YAP1]) utilizing quantitative polymerase chain reaction (PCR) was ran on formalin-fixed paraffin-embedded tissue samples was developed using data from 321 patients with non-squamous NSCLC resected at the University of California, San Francisco. The TNMB staging system then underwent validation in an independent, multicenter cohort of 1373 patients and its implementation was compared with adoption of the seventh and eight edition staging systems utilizing metrics of reclassification. The results were as follows: compared to staging using the eighth edition of the TNM system, the TNMB staging system strengthened the classification of high-risk patients, with a net reclassification improvement of 0.33 (95% confidence interval [CI]; 0.24-0.41). The TNMB better predicted differences in survival, with a relative integrated discrimination improvement of 22.1% (95% CI; 8.8%-35.3%), and it improved agreement between observed and predicted survival, with a decrease in the reclassification calibration statistic from 39 to 21. The study concluded the TNMB staging system significantly improved classification of high-risk patients and survival predictions versus conventional staging which may lead to improved survival of early-stage disease when more effective adjuvant therapy is employed.\n",
      "Woodward and colleagues (2018) evaluated an internationally validated, prognostic, 14-gene quantitative polymerase chain reaction expression assay to classify risk prospectively in 100 consecutive patients with stage IA, IB, and IIA non-squamous non-small cell lung cancer (NSCLC) and application of this molecular-based management strategy in the identification of high-risk patients for adjuvant intervention. Forty-eight patients (48%) were stratified as high-risk according to molecular testing and 36 (36%) met NCCN high-risk criteria. Risk designations diverged in 34 (34%) of all patients. The estimated 5-yar disease-free survival (DFS) was 48.9% for molecular high-risk patients who did not receive adjuvant chemotherapy, 93.8% among untreated molecular low-risk patients, and 91.7% in molecular high-risk patients who did receive chemotherapy (p=0.004).  DFS was 75.2% in untreated NCCN low-risk patients, and 61.9% in untreated NCCN high-risk patients (p=0.183). This prospective, nonrandomized study provided preliminary support that high-risk classification according to the 14-gene prognostic assay predicts benefit from adjuvant chemotherapy for very early stage NSCLC and supported the superiority of molecular stratification compared to NCCN criteria in identification of high-risk patients.\n",
      "mRNA CancerDetect\n",
      "According to Viome Life Sciences, Inc., the mRNA CancerDetect is a metatranscriptomic (RNA sequencing) saliva test utilizes Viomeâs proprietary technology and artificial intelligence (AI) platform to function as a screening tool for oral and throat cancer. This test identifies and quantifies all living microorganisms in the sample down to 1 part per million. After physically removing microbial and human rRNAs from total RNA, all remaining RNAs are sequenced using a full-length transcript. The test produces a quantitative strain-, species-, and genus-level taxonomic classification data and quantitative microbial gene expression data that are grouped into KEGG orhtologies (KOs). The aim of this test is to accurately discover the interaction between microbial activities and human gene expression in the progression of the previously mentioned cancers (Viome Life Sciences, 2021).\n",
      "Banavar and colleagues (2021) developed and evaluated machine-learning classifiers using metatranscriptomic data from saliva samples (n=433) collected from oral premalignant disorders (OPMD), oral cancer (OC) patients (n=71) and normal controls (n=171). The diagnostic classifiers showed a receiver operating characteristics (ROC) area under the curve (AUC) up to 0.9, sensitivity up to 83% (92.3% for stage 1 cancer) and specificity up to 97.9%. The study demonstrated potential clinical utility of an artificial intelligence/machine-learning model in the diagnosis of OC early.\n",
      "Praxis Somatic Whole Genome Sequencing\n",
      "According to Praxis Genomics, LLC, the somatic whole genome sequencing technique compares the genetic content of the patientâs normal and abnormal cells. A patientâs normal sample of blood, buccal swab, saliva, or tissue biopsies from unaffected areas and abnormal cells from the affected organ or region of the body can be cells or tissue samples (e.g., frozen or formalin-fixed paraffin-embedded [FFPE]) and from which the DNA is assessed for diagnostic, therapeutic and predictive purposes. This technique can be utilized to identify the following: 1.) single nucleotide variants (SNVs), 2. small insertions and deletions (InDels), 3. high resolution copy number changes (CNVs) within 100s of base pairs depending on the specific region, 4. changes in mitochondrial genome levels, 5. changes in mitochondrial genome content, and 6 repeat expansions (Praxis Genomics, 2021).\n",
      "Praxis Somatic Optical Genome Mapping\n",
      "According to Praxis Genomics, LLC, the somatic optical genome mapping technology was developed by Bionano Genomics LLC to evaluate large-scale changes in the DNA. Specifically, structural variant detection and analysis is conducted via a genome imaging tool. Samples for evaluation can be obtained from the patientâs blood or tissue biopsies. Specifically, somatic genome optical genome mapping can be used to evaluate the following: 1.) transfer of DNA fragments from one chromosomal position to another, 2. inversion of pieces of a chromosome, 3. complex chromosomal rearrangements, 4.) measuring the size of repetitive regions that control the expression of adjacent areas, and 5.) measuring the size of tandem repeat expansions within the genome (Praxis Genomics, 2021).\n",
      "Praxis Somatic Transcriptome\n",
      "According to Praxis Genomics, LLC, the somatic transcriptome analysis is a technique that enables evaluation of the functional consequences of DNA mutations discovered by Optical Genome Mapping (OGM) or Illumina Short Read (ISR) whole genome sequencing. Samples for evaluation can be obtained from the patientâs blood or tissue biopsies. Specifically, transcriptome analysis can be used to evaluate the following: 1.) deletions, insertions, inversions, translocations affecting regulatory regions, coding regions, imprinted loci, 2.) single nucleotide changes or small insertions deletions affecting splicing, translation initiation and termination, 3.) repeat number changes affecting chromosome structure and adjacent gene expression, and 4.) repeat expansions affecting transcript levels, splicing (Praxis Genomics, 2021).\n",
      "RadTox cfDNA test\n",
      "According to DiaCarta, Inc., the RadTox cfDNA test was developed for the surveillance of radiation therapy toxicity in cancer patients. A liquid biopsy of circulating cell-free DNA (cfDNA) is taken from the cancer patientâs plasma before and after radiation therapy which is then assayed to quantify tissue damage due to radiation therapy. Specifically, the assay analyzes arthrobacter luteus (Alu nucleic acid) repeated DNA sequences within cfDNA which correlates with radiation doses. Additionally, the cfDNA level is proportionally correlated with circulating tumor DNA (ctDNA) shed from tumor cells and therefore is used in the assay. Based on the determination of the cfDNA level in the cancer patientâs blood after radiation, the patientâs sensitivity to radiation and radiation dosage use can be estimated. Currently, the gold standard for clinical evaluation of tissue damage due to radiation therapy relies on the dose volume histogram or maximum tolerated dose radiation and a drug when used in combination (DiaCarta, 2021).\n",
      "Lockney and colleagues (2020) investigated whether cfDNA numbers measured by the RadTox assay (1) correlated with body integral dose, (2) lower with proton radiation therapy (RT) compared with photon RT, and (3) higher with larger prostate cancer RT fields. The study consisted of participants planned to receive proton or photon RT for nonmetastatic prostate cancer who had an intact prostate or after prostatectomy. Plasma collection occurred pre-RT and at 5 additional daily collection points starting 24 hours after initiation of RT. Data analysis was conducted in 54 evaluable participants and showed body integral dose was significantly correlated with the peak post-RT RadTox score (p=0.04). Participants receiving photon RT had a significant elevation in peak post-RT RadTox score (p=0.04), average post-RT RadTox score (p=0.04), and day-2 RadTox score (all minus the pre-RT values for each participant) versus participants receiving proton RT. Field size was not significantly associated with RadTox score. The study concluded that RadTox is correlated with body integral dose and accurately projects which patients receive proton versus photon RT.\n",
      "Lockney and colleagues (2021) conducted a study to evaluate the correlation between cfDNA measured during the first week of radiation therapy (RT)and early and late stage gastrointestinal (GI) and genitourinary (GU) toxicity. The study consisted of participants who received proton or photon RT for nonmetastatic prostate cancer who had an intact prostate or after prostatectomy. Blood sample collection occurred before treatment on sequential treatment days for the first full week of therapy. Toxicity assessments occurred at baseline, weekly during RT, and 6 months and 12 months after RT. The results for the 54 evaluable participants in this study were as follows: four (7%) and 3 (6%) participants displayed acute and late grade 2 GI toxicity, respectively; 22 (41%) and 18 (35%) participants displayed acute and late grade 2 GU toxicity, respectively. The presence of grade 3 or higher toxicity was absent among all participants. Grade 2 acute GI toxicity, but not grade 2 acute GU toxicity, was significantly correlated with pre-RT cfDNA levels and all days 1, 2, 3, 4, and 5 of RT (p < 0.005). Grade 2 late GI toxicity, but not GU toxicity, was significantly correlated with pre-RT cfDNA levels (p=0.021). This preliminary study concluded that cfDNA levels may offer a predictive value for the subset of patients who are likely to develop GI toxicity during prostate cancer treatment.\n",
      "3D Predict Panels\n",
      "The 3D Predict Ovarian Doublet Panel (KIYATEC, Inc.) is an investigational ovarian cancer drug response profile that will be used to help optimize therapeutic decision-making over the course of the disease. This ovarian oncology test uses spheroid cell culture to assess patient-specific response to a panel of 4 oncology drugs (i.e., carboplatin, doxorubicin, gemcitabine, paclitaxel), providing a tumor chemotherapy response prediction for each drug. \n",
      "The 3D Predict Ovarian PARP Panel (KIYATEC, Inc.) is another investigational ovarian cancer drug response profile which uses spheroid cell culture to assess patient-specific response to poly (ADP-ribose) polymerase (PARP) inhibitors (niraparib, olaparib, rucaparib, velparib), providing tumor response prediction for each drug.\n",
      "KIYATEC, Inc. is recruiting patients to participate in a prospective, open-label, non-randomized, clinical trial (3D-PREDICT; NCT03561207) to evaluate a patient-specific ex vivo 3D (EV3D) assay for drug response using a patient's own biopsy or resected tumor tissue for assessing tissue response to therapy in patients with advanced cancers, including ovarian cancer, high-grade gliomas, and high-grade rare tumors (KIYATEC, 2022).\n",
      "Augusta Hematology Optical Genome Mapping\n",
      "The Augusta Hematology Optical Genome Mapping test (Georgia Esoteric and Molecular Labs, Augusta University, Bionano) uses optical genome mapping (via the Saphyr system (Bionano Genomics) for copy number alterations and gene rearrangements utilizing DNA from blood or bone marrow in patients with hematolymphoid neoplasia. The test is aimed to report clinically significant alterations.\n",
      "BostonGene Tumor Portrait Test\n",
      "The BostonGene Tumor Portrait Test (BostonGene, Waltham, MA) utilizes DNA whole exome (WES) and RNA transcriptome (RNAseq) sequencing to identify genomic alterations (e.g., single nucleotide variants (SNV), indels, copy number alterations (CNA), tumor mutational burden (TMB), microsatellite instability (MSI), expressed functions, frameshifts) assess the expression levels of more than 20,000 genes. The test incorporates genomic and transcriptomic analysis coupled with proprietary algorithms to generate an ease of use tumor schematics known as the Molecular Funtional Portrait. This indivualized tumor map serves to faciliate treatment decision-making (e.g., help select the most effective therapy, avoid unnecessary side effects and decrease treatment costs). Furthermore, the test aims to predict individual response to immunotherapy and stratify individuals into responders and non-responders.\n",
      "DecisionDx DiffDx-Melanoma, DecisionDx-Melanoma, and DecisionDx-SCC\n",
      "The DecisionDx DiffDx-Melanoma test (Castle Biosciences, Phoenix, AZ) is based on a neural network, artificial-intelligence configured algorithm providing a low rate of intermediate results. Test validation was performed in cutaneous melanocytic lesions to accurately differentiate between benign and malignant melanocytic lesions of unknown potential established on the expression of 35 genes.\n",
      "The DecisionDx-Melanoma (Castle Biosciences, Phoenix, AZ) is a 31-gene expression profile (GEP) test that was developed to identify the risk of recurrence or metastasis including the likelihood of positive sentinel lymph node in individuals with stage I, II, or III melanoma and inform individual management decisions. Test validation was performed in predicting the risk of recurrence and metastasis and classifying individuals as Class 1A (lowest risk), 1B/2A (indreased risk), or 2B (highest risk).\n",
      "Marchetti and colleagues (2022) conducted a systematic review and meta-analysis assessing the performance of commercially available prognostic gene expression profile (GEP) tests in individuals with American Joint Committee on Cancer (AJCC) stage I or stage II cutaneous melanoma. After applying exclusion criteria, seven studies ( 5 assessing DecisionDx-Melanoma (Castle Biosciences Inc) and 2 assessing MelaGenix (NeraCare GmbH) were included. Both tests attempt to improve on prognostic estimates by classifying individuals as being at high or low risk for recurrence or metastasis. A total of 1450 study participants comprised the 7 included studies. The primary outcomes and measures were the proportion of patients with or without melanoma recurrence correctly classified by the GEP test as being at high or low risk. For both tests, GEP test varied by AJCC stage. For patients tested with DecisionDx-Melanoma, 623 had stage I disease (6 true-positive [TP], 15 false-negative, 61 false-positive, and 541 true-negative [TN] results) and 212 had stage II disease (59 TP, 13 FN, 78 FP, and 62 TN results). For patients with recurrence, DecisionDx-Melanoma correctly classified 29% with stage I disease and 82% with stage II disease. For patients without recurrence, the test correctly classified 90% with stage I disease and 44% with stage II disease. For patients tested with MelaGenix, 88 had stage I disease (7 TP, 15 FN, 15 FP, and 51 TN results) and 245 had stage II disease (59 TP, 19 N, 95 FP, and 72 TN results). For patients with recurrence, MelaGenix correctly classified 32% with stage I disease and 76% with stage II disease. For patients without recurrence, the test correctly classified 77% with stage I disease and 43% with stage II disease. The investigators noted that the reported studies had moderate to high risk of bias due to poor design standards conduct, reporting, and analysis. It was concluded that the prognostic ability of GEP tests for patients with localized melanoma varied by AJCC and seemed to be poor at correctly identifying recurrence in patients with stage I disease, thereby, offering limited potential for clinical utility in these patients.\n",
      "DecisionDx-SCC (Castle Biosciences, Phoenix, AZ) is a 40-gene expression profile (GEP) test that was developed to identify the risk of metastasis in individuals with cutaneous squamous cell carcinoma (SCC) with one or more risk factors based on the biologic makeup of the genes within their tumor tissue.\n",
      "LungLB and LungLife AI\n",
      "The LungLB (LungLife AI, Thousand Oaks, CA) is a 4-color fluorescence in-situ hybridization (FISH) assay employed to detect early circulating tumor cells (CTCs) from peripheral blood draw. This assay was developed as a FISH-based liquid biopsy test utilizing DNA FISH to validate identification of CTC to assist in the clinical assessment of individuals with indeterminate nodules suspicious for lung cancer. \n",
      "LungLife AI also developed the LungAI software, an algorithm to identify FISH probes, utilizing deep learning based on convolutional neural networks (U-Nets) using > 4000 ground-truth probe images manually annotated by domain experts. The aim of the LungAI software is to more accurately identify FISH signals in clinical samples compared to traditional FISH analysis techniques by trained operator, and thereby ehnance the performance of the LungLB test for identifying CTC.\n",
      "PancreaSeq Genomic Classifier (GC)\n",
      "The PancreaSeq Genomic Classifier (GC) test (University of Pittsburgh Medical Center Molecular and Genomic Pathology Laboratory, Pittsburgh, PA) utilizes DNA and mRNA next-generation sequencing analysis of 74 genes and functions as a diagnostic test for pancreatic cysts detected by imaging, including benign cysts, cystic pancreatic neuroendocrine tumors (PNETs), and early detection of pancreatic ductal adenocarcinoma. This test enables the detection of \"single nucleotide variants (SNVs), insertions and deletions (indels) in 20 genes related to pancreatic cysts and pancreatic cancer, including KRAS and GNAS, loss of heterozygosity (LOH) analysis and copy number alterations (CNAs) at 13 chromosomal regions, including the RNF43, SMAD4, TP53, PTEN, VHL, and NF2 tumor suppressor genes. In addition, it detects >150 types of gene fusions involving the ALK, BRAF, FGFR2, NTRK1, NTRK3, RET, ROS1, RAF1, PRKCA, PRKCB genes and expression of specific genetic markers, including KRT7, KRT20, CHRGR, and PGK1.\" Additionally, \"it detects expression of the CEA (CECAM5) gene by qRT-PCR. The test is using a small amount of nucleic acids (DNA and RNA) isolated from pancreatic cyst fluid collected into preservative solution during ultrasound-fine needle aspiration (EUS-FNA) procedure. The findings are subjected to algorithmic analysis to categorize as Negative or Positive for specific alterations associated with different cyst types and risk of progression.\"\n",
      "DAWN IO Melanoma\n",
      "The DAWN IO Melanoma (InterVEnn Biosciences) is a liquid biopsy test that was developed to assess whether a patient is likely to benefit from immune checkpoint inhibitor therapy, specifically nivolumab (Opdivo) in combination with ipilimumab (Yervoy) or pembrolizumab (Keytruda) monotherapy. The test applies artificial intelligence (AI)-enabled quantitative mass spectrometry analysis of 142 unique pairs of glycopeptide and product fragments, obtained from plasma, and produces a predictive algorithm reported as likely, unlikely, or uncertain benefit from immunotherapy agents.\n",
      "EpiSwitch CiRT (Checkpoint-Inhibitor Response Test)\n",
      "The EpiSwitch CiRT (Checkpoint-inhibitor Response Test ) (Oxford BioDynamics, PLC) is a blood test that is used to predict a patient's therapeutic response to checkpoint inhibitor immunotherapy. The test consists of genetic profiling of 8 DNA-regulatory (epigenetic) markers by quantitative polymerase chain reaction (qPCR), from whole blood, and is reported as a high or low probability of responding to immune checkpointâinhibitor therapy.\n",
      "Guardant360 TissueNext\n",
      "The Guardant360 TissueNext (Guardant Health, Inc.) is a next-generation sequencing (NGS)-based pan-cancer tissue test for patients with advanced solid tumor. The test is used to help oncologists identify actionable biomarkers for therapeutic management. The test applies targeted genomic sequence and DNA analysis of 84 or more genes and includes interrogation for sequence variants, gene copy number amplifications, gene rearrangements, microsatellite instability (MSI) and tumor mutational burden (TMB) from formalin-fixed paraffin-embedded (FFPE) tumor tissue.\n",
      "HelioLiver Test\n",
      "The HelioLiver test (Fulgent Genetics, LLC.) is a novel multi-analyte blood test that combines cell-free DNA (cfDNA) methylation patterns and measurement of serum of AFP/AFP-L3 and oncoprotein desgamma-carboxy-prothrombin (DCP) to help detect hepatocellular carcinoma (HCC). The test is intended to be used for surveillance of HCC in adults 21 years and older who are designated to be at high-risk for the disease due to a diagnosis of liver cirrhosis. The test results are based on an algorithm reported as normal or abnormal. The HelioLiver test is not intended as a replacement for diagnostic biopsies or diagnostic imaging by contrast enhanced MRI or CT. \n",
      "IMMray PanCan-d Test\n",
      "The IMMray PanCan-d (Immunovia, Inc.) is a multiplex immunoassay blood test that measures 9 serum biomarkers (C5, C4, cystatin C, factor B, osteoprotegerin (OPG), gelsolin, IGFBP3, CA125, and CA19-9), which is combined in an algorithm and reported qualitatively as positive, negative, or borderline for pancreatic ductal adenocarcinoma. The biomarkers are a combination of immunoregulatory and tumor biomarkers.\n",
      "Brand et al (2022) conducted a prospective cohort study aimed to validate the clinical performance of the IMMray PanCan-d test and to better understand test performance in Lewis-null (le/le) individuals who cannot express CA19-9. Serum samples from 586 individuals were analyzed with the IMMray PanCan-d biomarker signature and CA19-9 assay, including 167 PDAC samples, 203 individuals at high risk of familial/hereditary PDAC, and 216 healthy controls. Samples were collected at 11 sites in the United States and Europe. The study was performed by Immunovia, Inc (Marlborough, MA), and sample identity was blinded throughout the study. Test results were automatically generated using validated custom software with a locked algorithm and predefined decision value cutoffs for sample classification. The authors found that the IMMray PanCan-d test distinguished PDAC stages I and II (n = 56) vs high-risk individuals with 98% specificity and 85% sensitivity and distinguished PDAC stages IâIV vs high-risk individuals with 98% specificity and 87% sensitivity. The authors identified samples with a CA19-9 value of 2.5 U/mL or less as probable Lewis-null (le/le) individuals. Excluding these 55 samples from the analysis increased the IMMray PanCan-d test sensitivity to 92% for PDAC stages IâIV (n = 157) vs controls (n = 379) while maintaining specificity at 99%; test sensitivity for PDAC stages I and II increased from 85% to 89%. The authors concluded that these results demonstrate the IMMray PanCan-d blood test can detect PDAC with high specificity (99%) and sensitivity (92%). The authors noted that their study addresses the diagnostic accuracy of PanCan-d but cannot assess its clinical utility. The authors also state that complete analysis of the prospective PanFAM clinical trial, which has accrued more than 1,000 participants, should provide additional relevant data regarding IMMray PanCan-d test performance and is expected within the next year.\n",
      "IsoPSA\n",
      "The IsoPSA (Cleveland Diagnostics, Inc) is a blood test used to identify prostate cancer risk for men over 50 years of age with elevated prostate-specific antigen (PSA) levels (greater than 4 ng/mL) facing prostate biopsy. The test analyzes all PSA structural isoforms by phase separation and immunoassay, and uses an algorithm to report risk of cancer. IsoPSA is not indicated for men under 50 years of age, men who have had a recent (less than 72 hours) prostate manipulation, including digital rectal exam; recent (less than 2 weeks) UTI and/or prostatitis; recent (less than 30 days) prostate surgery, urinary catheterization, prostate infarction, or endoscopic evaluation; or diagnosis of prostate cancer or other urinary tract malignancy (Cleveland Diagnostics, 2022).\n",
      "Benidir et al (2022) conducted a single center, retrospective review to explore if elevated IsoPSA selects for particular adverse radiographic or histopathologic features among men destined to undergo radical prostatectomy (RP) because of clinically significant prostate cancer identified at biopsy. A consecutive cohort of patients whom had undergone RP within the same period without pre-operative IsoPSA served as controls. Patient had undergone a pre-operative prostate magnetic resonance imaging (MRI). Adverse histopathologic and MRI features were compared between both groups. Concordance, downstaging, and upstaging grade group rates (GG) was evaluated. Pearson Chi-Square test was used to compare categorical variables, Wilcoxon-Rank sum test for quantitative variables, and binary logistic regression to identify predictors of upstaging at RP. Eighty-three patients underwent IsoPSA and RP while 44 patients were controls. The authors found that the IsoPSA group had significantly higher pre-operative PSA (IsoPSA group: 7.8 ng/mL vs Control group: 5.2 ng/mL, p<.001 ). Elevated IsoPSA index (>6.0) did not select for any specific adverse histopathologic features at RP. Excluding PSA density, elevated IsoPSA was not selective for adverse MRI features. There were no differences in concordance, downstaging, and upstaging GG rates from biopsy to RP. IsoPSA testing was not a predictor of GG upstaging (Odds Ratio: 0.63, p .58). The authors concluded that elevated IsoPSA is a diagnostic tool that can detect clinically significant prostate at the time of biopsy. In doing so, it does not select for any particular adverse prostate MRI or pathologic feature at RP. \n",
      "Klein et al (2022) conducted a prospective, multicenter study to validate the diagnostic performance of IsoPSA for High-Grade prostate cancer (CaP) and Any CaP risk on biopsy in men aged 50 years or older with total PSA greater than or equal to 4 ng/ml. The study included 888 men scheduled for prostate biopsy at 8 academic and community sites between August 2015 and August 2020. Participant demographics in the validation study were broadly reflective of contemporary urological practice in the United States and consistent with the recruitment of men at increased risk for CaP representing the targeted intended use population for IsoPSA which included the following demographics for the evaluable cohort of 888 subjects: 103 African-Americans, 712 Caucasians, 7 Hispanics, 29 other, 37 unkown. Receiver operating characteristic and likelihood ratio analysis used to validate diagnostic performance for previously established IsoPSA Index cutoffs for High-Grade CaP (Gleason Score â¥ 7) and Any CaP (Gleason Score â¥ 6), compare IsoPSA to total PSA and % free PSA, and evaluate subgroups (total PSA 4-10 ng/ml, total PSA > 10 ng/ml, biopsy naÃ¯ve, prior negative biopsy). The disease prevalence was 35.6% (High-Grade CaP) and 58.9% (Any CaP). The area under the receiver operating characteristic curve was 0.783 (High-Grade CaP) and 0.770 (Any CaP). IsoPSA outperformed total PSA and % free PSA on area under the receiver operating characteristic curve, specificity, positive and negative predictive value at similar sensitivity. Using selected IsoPSA Index cutoffs, an estimated 46% (High-Grade CaP) and 42% (Any CaP) of biopsies could be avoided in low-risk patients. IsoPSA displayed statistically informative likelihood ratio-based predictive characteristics. IsoPSA maintained accuracy in clinically relevant subgroups. The authors concluded that the IsoPSA diagnostic performance and predictive value is validated for High-Grade CaP and Any CaP in men age 50 years or older with total PSA greater than or equal to 4 ng/ml undergoing diagnostic biopsy. IsoPSA outperforms total and % free PSA in discriminating the risk of prostate cancer on biopsy. IsoPSA has the potential to reduce unnecessary biopsies and improve the risk-benefit ratio for CaP early detection. Potential limitations include decentralized pathology review, lack of standardized biopsy techniques, and the use of specificity to estimate unnecessary biopsies saved which may fail to account for false positive IsoPSA or false negative biopsy (i.e., biopsy template error) results.\n",
      "The National Comprehensive Cancer Network (NCCN) Clinical Practice Guidelines in Oncology on \"Prostate cancer early detection\" Version 1.2022, states that tests like IsoPSA can be considered in patients thought to be higher risk despite a negative prostate biopsy. In addition, NCCN states that \"biomarkers that improve the specificity of detection are not, as yet, mandated as first-line screening tests in conjunction with serum PSA. However, there may be some patients who meet PSA standards for consideration of prostate biopsy, but for whom thdolent cancers. However, the clinical utility of these strategies is uncertain, there is no consensus on using any of these tests, and additional studies for clinical effectiveness are needed. The American Urological Association (AUA) guideline noted the lack of evidence for using any tests other than PSA to determine the need for a referral for biopsy\".\n",
      "An UpToDate review on \"Genetic risk factors for prostate cancer\" (Cheng and Nelson, 2022) states, \"Although an SNP associated with prostate cancer may not reside within a known gene, it can be used to localize candidate genes that require further confirmation. Among the genes that have been suggested in this way are HOXB13, microseminoprotein beta (MSMB), lemur tyrosine kinase 2 (LMTK2), kallikrein-related peptidase 3 (KLK3), copine 3 (CPNE3), interleukin 16 (IL16), cadherin 13 (CDH13), and hepatocyte nuclear factor 1B (HNF1B). \"Additional studies are needed in broader populations to determine whether new models and biomarkers can be combined with PSA levels and other clinical factors (ie, age, race, family history, prior biopsies) to identify males who are at particularly high risk for prostate cancer.\"\n",
      "NavDx for Surveillance of Cancer Recurrence in HPV-Associated Oropharyngeal Cancer\n",
      "In a prospective, multi-center study, Chera et al (2019) identified a profile of circulating tumor human papilloma virus (HPV) DNA (ctHPVDNA) clearance kinetics that is associated with disease control following chemoradiotherapy (CRT) for HPV-associated oropharyngeal squamous cell carcinoma (OPSCC).  This trial was carried out in 103 patients with p16-positive OPSCC; M0 disease; and receipt of definitive CRT.  Blood specimens were collected at baseline, weekly during CRT, and at follow-up visits.  Optimized multi-analyte digital PCR assays were used to quantify ctHPVDNA (types 16/18/31/33/35) in plasma.  A control cohort of 55 healthy volunteers and 60 patients with non-HPV-associated malignancy was also analyzed.  Baseline plasma ctHPVDNA had high specificity (97 %) and high sensitivity (89 %) for detecting newly diagnosed HPV-associated OPSCC.  Pre-treatment ctHPV16DNA copy number correlated with disease burden, tumor HPV copy number, and HPV integration status.  These investigators defined a ctHPV16DNA favorable clearance profile as having high baseline copy number (greater than 200 copies/ml) and greater than 95 % clearance of ctHPV16DNA by day 28 of CRT; 19 of 67 evaluable patients had a ctHPV16DNA favorable clearance profile, and none had persistent or recurrent regional disease after CRT.  In contrast, patients with adverse clinical risk factors (T4 or greater than 10 total pack years [TPY]) and an unfavorable ctHPV16DNA clearance profile had a 35 % actuarial rate of persistent or recurrent regional disease after CRT (p = 0.0049).  The authors concluded that a rapid clearance profile of ctHPVDNA may predict likelihood of disease control in patients with HPV-associated OPSCC patients treated with definitive CRT and may be useful in selecting patients for de-intensified therapy.  These researchers stated that future studies assessing ctHPVDNA as an integral biomarker to guide treatment de-intensification are needed; and may facilitate personalized treatment decisions based on tumor biology in addition to clinical risk factors.  Finally, prospective evaluation of ctHPVDNA as a biomarker in other HPV-associated malignancies (e.g., cervical and anal cancers) should be evaluated.\n",
      "The authors stated that although this study included over 100 patients, it was under-powered due to low occurrence of disease persistence/recurrence and limited follow-up.  Despite these limitations, these investigators observed an early trend in worse regional DFS (RDFS) in patients with greater than 10 TPY and unfavorable ctHPVDNA kinetics.  Validation of these findings using a clinical-grade test in independent and larger patient cohorts is needed to confirm these findings.\n",
      "Chera et al (2020) noted that plasma ctHPVDNA is a sensitive and specific biomarker of HPV-associated OPSCC.  In a prospective, clinical trial, these investigators examined if longitudinal monitoring of ctHPVDNA during post-treatment surveillance could accurately detect clinical disease recurrence.  This study was carried out in patients with non-metastatic HPV-associated (p16-positive) OPSCC.  All patients were treated with curative-intent CRT.  Patients underwent a 3-month post-CRT positron emission tomography/computed tomography (PET/CT) scan and were thereafter clinically evaluated every 2 to 4 months (years 1 to 2), then every 6 months (years 3 to 5).  Chest imaging was carried out every 6 months.  Blood specimens were collected every 6 to 9 months for analysis of plasma ctHPVDNA using a multi-analyte digital PCR assay.  The primary endpoint was to estimate the NPV and PPV of ctHPVDNA surveillance.  A total of 115 patients were enrolled, and 1,006 blood samples were analyzed.  After a median follow-up time of 23 months (range of 6.1 to 54.7 months), 15 patients (13 %) developed disease recurrence; 87 patients had undetectable ctHPVDNA at all post-treatment time-points, and none developed recurrence (NPV, 100 %; 95 % CI: 96 % to 100 %); 28 patients developed a positive ctHPVDNA during post-treatment surveillance, 15 of whom were diagnosed with biopsy-proven recurrence; 16 patients had 2 consecutively positive ctHPVDNA blood tests, 15 of whom developed biopsy-proven recurrence.  Two consecutively positive ctHPVDNA blood tests had a PPV of 94 % (95 % CI: 70 % to 99 %).  Median lead time between ctHPVDNA positivity and biopsy-proven recurrence was 3.9 months (range of 0.37 to 12.9 months).  The authors concluded that post-treatment surveillance of cancer recurrence with plasma ctHPVDNA monitoring has exceptional NPV (100 %) and PPV (94 %) when using 2 consecutively positive ctHPVDNA tests as the criterion for positivity.  Patients who have undetectable ctHPVDNA during clinical follow-up are unlikely to have recurrent disease and may be spared routine radiographic and in-office nasopharyngolaryngoscopic surveillance.  These researchers stated that future studies should examine if ctHPVDNA-based monitoring may help to reduce the financial toxicity of cancer survivorship and whether earlier detection of cancer relapse improves post-recurrence survival outcomes.\n",
      "The authors stated that whether earlier detection of disease recurrence may positively impact survival outcomes in HPV-associated OPSCC remains an open question.  ctHPVDNA-based monitoring may result in a higher rate of identifying oligo-recurrence and potentially a higher rate of salvage with surgery or radiotherapy.  Therefore, clinical trials of early intervention after ctHPVDNA-based detection are needed.  The findings described in this study were specific for the optimized multi-analyte ctHPVDNA assay and may not apply to alternative ctHPVDNA assays.  In addition, detection of 2 consecutively positive ctHPVDNA tests to improve the PPV was not prospectively defined and warrants further validation.  Consistent with NCCN guidelines, there is no âgold standardâ for surveillance imaging.  Therefore, these researchers opted to use a more stringent criterion of biopsy-proven recurrence to define a change in disease status.  A shorter interval between ctHPVDNA testing (e.g., every 3 months) would more precisely define the extent of earlier detection.  Several heterogenous clinical factors (intensity of treatment, tobacco pack-years, use of chemotherapy) may have impacted the pattern and frequency of disease recurrence in this study.  Although this trial showed feasibility of ctHPVDNA-based surveillance for both distant and local-regional recurrences, more research is needed to examine potential differences in detection efficiency.  \n",
      "Misawa et al (2020) noted that HPV-associated oropharyngeal cancer (OPC) is an independent tumor type with regard to cellular, biological, and clinical features.  The use of non-invasive biomarkers such as ctDNA may be relevant in early diagnosis and eventually improve the outcomes of patients with HNSCC.  Genome-wide discovery using RNA sequencing and reduced representation bi-sulfite sequencing yielded 21 candidates for methylation-targeted genes.  A verification study (252 HNSCC patients) using quantitative methylation-specific PCR (Q-MSP) identified 10 genes (ATP2A1, CALML5, DNAJC5G, GNMT, GPT, LY6D, LYNX1, MAL, MGC16275, and MRGPRF) that showed a significant increase recurrence in methylation groups with OPC.  Further investigation on ctDNA using Q-MSP in HPV-associated OPC showed that 3 genes (CALML5, DNAJC5G, and LY6D) had a high predictive ability as emerging biomarkers for a validation set, each capable of discriminating between the plasma of the patients from healthy individuals.  Among the 42 ctDNA samples, methylated CALML5, DNAJC5G, and LY6D were observed in 31 (73.8 %), 19 (45.2 %), and 19 (45.2 %) samples, respectively.  Among pre-treatment ctDNA samples, methylated CALML5, DNAJC5G, and LY6D were observed in 8/8 (100 %), 7/8 (87.5 %), and 7/8 (87.5 %) samples, respectively.  Methylated CALML5, DNAJC5G, and LY6D were found in 2/8 (25.0 %), 0/8 (0 %), and 1/8 (12.5 %) of the final samples in the series, respectively.  The authors presented the relationship between the methylation status of 3 specific genes and cancer recurrence for risk classification of HPV-associated OPC cases.  These researchers concluded that ctDNA analysis has the potential to aid in determining patient prognosis and real-time surveillance for disease recurrences and could serve as an alternative method of screening for HPV-associated OPC.\n",
      "The authors stated that currently the lack of an effective OPC screening program is because there are no identified OPC precursor lesions.  The HPV-associated carcinogenesis initially arises in tonsillar crypts, which may be the most likely reason why the HPV prevalence in tonsillar cells is higher.  In the future, these researchers think that highly sensitive and tissue-specific ctDNA biomarkers of HPV-associated OPC will be discovered, allowing early detection.  This study, entailing human specimens and high-throughput profiling platforms, may be susceptible to measurement bias from various sources; accordingly, the use of methylation markers in clinical practice requires further testing in prospective studies with larger HNSCC cohorts.  Finally, these efforts will lead to the identification of new oncological biomarkers for early detection and outcome prediction, which is a pre-requisite for realizing the advantages of precision medicine.  This trial demonstrated the use of using parallel serial assessment of ctDNA methylation in the treatment evaluation of HPV-associated OPC.\n",
      "Berger et al (2022) stated that despite generally favorable outcomes, 15 % to 25 % of patients with HPV-driven OPSCC will have recurrence.  Current post-treatment surveillance practices rely on physical examinations and imaging and are inconsistently applied.  In a retrospective, clinical case-series study, these researchers examined circulating tumor tissue modified viral (TTMV)-HPV DNA obtained during routine post-treatment surveillance among a large population of real-world patients.  This trial included 1,076 consecutive patients across 108 U.S. sites who were 3 or more months post-treatment for HPV-driven OPSCC and who had 1 or more TTMV-HPV DNA tests (NavDx, Naveris Laboratories) obtained during surveillance between February 6, 2020, and June 29, 2021.  Test results were compared with subsequent clinical evaluations.  Circulating TTMV-HPV DNA was positive in 80 of 1,076 (7.4 %) patients, with follow-up available on all.  At 1st positive surveillance testing, 21 of 80 (26 %) patients had known recurrence while 59 of 80 (74 %) patients were not known to have recurrent disease.  Among these 59 patients, 55 (93 %) subsequently had a confirmed recurrence, 2 patients had clinically suspicious lesions, and 2 had clinically \"no evidence of disease\" (NED) at last follow-up.  To-date, the overall PPV of TTMV-HPV DNA testing for recurrent disease was 95 % (n = 76/80).  Furthermore, the point-in-time NPV was 95 % (n = 1,198/1,256).  The authors concluded that these findings highlighted the clinical potential for circulating TTMV-HPV DNA testing in routine practice.  As a surveillance tool, TTMV-HPV DNA positivity was the 1st indication of recurrence in the majority of cases, pre-dating identification by routine clinical and imaging exams.  These researchers stated that these findings may inform future clinical and guideline-endorsed strategies for HPV-driven malignancy surveillance.\n",
      "The authors stated that this study had several drawbacks.  First, the majority of patients (78 %) had a single surveillance test result obtained during the study period with more than half (55 %) greater than 12 months from completion of therapy.  Sequential values obtained in this cohort over time would be of interest, as would baseline or pre-treatment and on-treatment test results.  The vast majority of positive or detectable cases reflected HPV subtype 16 disease (93 %); therefore, further investigation is needed to validate these findings in non-HPV16 high-risk subtypes, although these investigators expect similar findings.  Second, as this was a cross-sectional analysis of the NPV of the test, whether a negative TTMV-HPV DNA result remains predictive of the absence of recurrence 3, 6, or 12 months after a negative blood test is currently unknown.  Further follow-up is needed to clarify this point and help further refine the role of TTMV-HPV DNA testing in the surveillance setting.  Third, the length of time a negative test remains predictive is the subject of continued study as the cohort ages.\n",
      "An accompanying commentary (Colevas, 2022) stated that \"Lack of prospectively planned follow-up and minimal characterization of the patient population studied complicate interpretation of circulating human papillomavirus (HPV) DNA as a prognostic biomarker for patients with HPV-associated oropharyngeal carcinoma treated with curative intent. Controlled clinical trials with embedded biomarkers will be necessary to optimize the utility of circulating tumor HPV DNA assays\".\n",
      "Rettig et al (2022) stated that circulating tumor tissue-modified viral (TTMV) HPV DNA is a dynamic, clinically relevant biomarker for HPV-positive oropharyngeal squamous cell carcinoma (SCC).  Reasons for its wide pre-treatment inter-patient variability are not well understood.  In a cross-sectional study, these researchers characterized clinicopathologic factors associated with TTMV HPV DNA.  This study included patients evaluated for HPV-positive oropharyngeal SCC at Dana-Farber Cancer Institute (Boston, MA) between December 2019 and January 2022 and who were undergoing curative-intent treatment.  Clinicopathologic characteristics included demographic variables, tumor and nodal staging, HPV genotype, and imaging findings.  Main outcome measures entailed pre-treatment circulating TTMV HPV DNA from 5 genotypes (16, 18, 31, 33, and 35) assessed using a commercially available digital droplet polymerase chain reaction (PCR)-based assay, considered as either detectable/undetectable or a continuous score (fragments/ml).  Among 110 included patients, 96 were men (87 %) and 104 were White (95 %), with a mean (SD) age of 62.2 (9.4) years.  Circulating TTMV HPV DNA was detected in 98 patients (89 %), with a median (inter-quartile range [IQR]) score of 315 (47 to 2,686) fragments/ml (range of 0 to 60,061 fragments/ml).  Most detectable TTMV HPV DNA was genotype 16 (n = 86 [88 %]), while 12 patients (12 %) harbored other genotypes.  Circulating TTMV HPV DNA detection was most strongly associated with clinical N stage.  Although few patients had clinical stage N0 disease, only 4 of these 11 patients (36 %) had detectable DNA compared with 94 of 99 patients (95 %) with clinical stage N1 to N3 disease (proportion difference, 59 %; 95 % confidence interval [CI]: 30 % to 87 %).  Among patients with undetectable TTMV HPV DNA, more than half (7 of 12 [58 %]) had clinical stage N0 disease.  The TTMV HPV DNA prevalence and score increased with progressively higher clinical nodal stage, diameter of largest lymph node, and higher nodal maximum standardized uptake value on positron emission tomography/computed tomography (PET/CT).  In multi-variable analysis, clinical nodal stage and nodal maximum standardized uptake value were each strongly associated with TTMV HPV DNA score.  Among 27 surgically treated patients, more patients with than without lympho-vascular invasion had detectable TTMV HPV DNA (12 of 12 [100 %] versus 9 of 15 [60 %]).  The authors concluded that in this cross-sectional study, circulating TTMV HPV DNA was statistically significantly associated with nodal disease at HPV-positive OPSCC diagnosis.  The few patients with undetectable levels had predominantly clinical stage N0 disease, suggesting assay sensitivity for diagnostic purposes may be lower among patients without cervical lymphadenopathy.  These researchers stated that mechanisms underlying this association, and the use of this biomarker for surveillance of patients with undetectable baseline values, warrant further investigation.\n",
      "Gunning et al (2023) noted that the NavDx blood test analyzes tumor tissue modified viral (TTMV)-HPV DNA to provide a reliable means of detecting and monitoring HPV-driven cancers.  The test has been clinically validated in a large number of independent studies and has been integrated into clinical practice by over 1,000 healthcare providers at over 400 medical sites in the U.S.  This Clinical Laboratory Improvement Amendments (CLIA), high complexity laboratory developed test, has also been accredited by the College of American Pathologists (CAP) and the New York State Department of Health.  These investigators reported a detailed analytical validation of the NavDx assay, including sample stability, specificity as measured by limits of blank (LOBs), and sensitivity illustrated via limits of detection and quantitation (LODs and LOQs).  LOBs were 0 to 0.32 copies/Î¼L, LODs were 0 to 1.10 copies/Î¼L, and LOQs were less than 1.20 to 4.11 copies/Î¼L, demonstrating the high sensitivity and specificity of data provided by NavDx.  In-depth evaluations including accuracy and intra- and inter-assay precision studies were shown to be well within acceptable ranges.  Regression analysis revealed a high degree of correlation between expected and effective concentrations, demonstrating excellent linearity (R2 = 1) across a broad range of analyte concentrations.  The authors concluded that these findings showed that NavDx accurately and reproducibly detects circulating TTMV-HPV DNA, which has been shown to aid in the diagnosis and surveillance of HPV-driven cancers.  It should be noted that all authors are employed by Naveris, Inc. and may hold equity or stock options in Naveris, Inc.  S.K. is listed as an inventor on patents.\n",
      "Furthermore, National Comprehensive Cancer Networkâs clinical practice guideline on âHead and neck cancersâ (Version 2.2022; Version 1.2023) does not mention circulating tumor DNA as a management tool.\n",
      "Thyroid GuidePx\n",
      "The Thyroid GuidePx (Protean BioDiagnostics) is a prognostic test that applies gene-expression profiling via targeted hybrid captureâenrichment RNA sequencing of 82 content genes and 10 housekeeping genes on formalin-fixed paraffin embedded (FFPE) tissue. The test includes an algorithm reported as one of three molecular subtypes. Thyroid GuidePx purports to identify a low risk group with less than 5% recurrence rate.\n",
      "Natera Signatera Minimal Residual Disease Test for Colorectal Cancer\n",
      "In a systematic review and meta-analysis, Gogenur et al (2022) examined the association between circulating tumor DNA (ctDNA) detection at baseline, during and after neoadjuvant treatment, after surgery, and recurrence, in patients with non-metastatic cancer.  These investigators included studies that examined patients undergoing neoadjuvant treatment for non-metastatic cancer and provided recurrence indices stratified for ctDNA status at the following timepoints: baseline, during treatment, post-treatment, and post-surgery.  Study quality was reported with the Newcastle-Ottawa scale, REMARK checklist, and GRADE approach.  PubMed, Embase, Cochrane Library, and Web of Science were the data sources (from inception to June 3, 2021).  The main outcome was risk of recurrence.  They identified 10 studies including 727 patients with rectal, breast, gastric, and bladder cancer.  All studies reported post-treatment ctDNA analysis, while 7, 4, and 6 reported baseline, during treatment, and post-surgery ctDNA analysis, respectively.  ctDNA detection was associated to recurrence across all time-points [baseline: risk ratio (RR) 2.86, 95 % confidence interval (CI): 1.33 to 6.14, during treatment: RR 3.81, 95 % CI: 2.09 to 6.92, post-treatment: RR 4.29, 95 % CI: 2.79 to 6.60, post-surgery: RR 8.03, 95 % CI: 3.16 to 20.43].  Heterogeneity was low-to-moderate.  The authors concluded that this meta-analysis of observational studies found that ctDNA detection in patients undergoing neoadjuvant treatment for non-metastatic cancer was associated with recurrence.  A stronger association was evident in post-treatment and post-surgery time-points; however, some studies reported low negative predictive value (NPV) of pathological complete response, showing that ctDNA-detection-guided escalation and de-escalation studies following neoadjuvant treatment regimens are needed before its role as a treatment guidance can be affirmed.\n",
      "Schraa et al (2022) stated that identification of non-metastatic colorectal cancer (CRC) patients with a high risk of recurrence after tumor resection is important to select patients who might benefit from adjuvant treatment.  Cell-free DNA (cfDNA) and circulating tumor DNA (ctDNA) analyses after surgery are promising biomarkers to predict recurrence in these patients.  However, these analyses face several challenges and do not allow guidance of neoadjuvant treatment, which might become a novel standard option in colon cancer treatment.  The prognostic value of cfDNA/ctDNA before surgery is unclear.  In a systematic review, these researchers examined publications in which the prognostic value of pre-surgery cfDNA/ctDNA in non-metastatic CRC patients was studied and was carried out according to PRISMA guidelines.  A total of 29 out of 1,233 articles were included and categorized into 3 groups that reflected the type of approach: measurement of cfDNA, ctDNA somatic alterations, and ctDNA methylation.  Overall, a clear association between pre-surgery cfDNA/ctDNA and the outcome was not observed, but large studies that primarily focused on the prognostic value of pre-surgery cfDNA/ctDNA are lacking.  The authors concluded that designing and performing studies that focus on the value of pre-surgery cfDNA/ctDNA is needed, in addition to standardization in the reporting of cfDNA/ctDNA results according to existing guidelines to improve comparability and interpretation among studies.\n",
      "Arisi et al (2022) noted that ctDNA is a component of cfDNA that is shed by malignant tumors into the bloodstream and other bodily fluids.  ctDNA can comprise up to 10 % of a patient's cfDNA depending on their tumor type and burden.  The short half-life of ctDNA ensures that its detection captures tumor burden in real-time and offers a non-invasive method of repeatedly evaluating the genomic profile of a patient's tumor.  A challenge in ctDNA detection includes clonal hematopoiesis of indeterminate potential (CHIP), which can be distinguished from tumor variants using a paired whole-blood control.  Most assays for ctDNA quantification rely on measurements of somatic variant allele frequency (VAF), which is a mutation-dependent method.  Patients with certain types of solid tumors, including CRC, can have levels of cfDNA 50 times higher than healthy patients.  ctDNA undergoes a precipitous drop shortly after tumor resection and therapy, and rising levels can fore-shadow radiologic recurrence on the order of months.  The amount of tumor bulk needed for ctDNA detection is lower than that for computed tomography (CT) scan detection, with ctDNA detection preceding radiologic recurrence in many cases.  The authors concluded that cfDNA/ctDNA can be used for tumor molecular profiling to identify resistance mutations when tumor biopsy is not available, to detect minimal residual disease (MRD), to monitor therapy response, and for the detection of tumor relapse.  These researchers stated that although ctDNA is not yet implemented in clinical practice, studies are ongoing to define the appropriate way to use it as a tool in the clinic.\n",
      "Hallermayr et al (2022) stated that analysis of cfDNA is a promising tool for personalized management of CRC patients.  Untargeted cfDNA analysis using whole-genome sequencing (WGS) does not need a priori knowledge of the patientÂ´s mutation profile.  These researchers established Liquid biopsy Fragmentation, Epigenetic signature and Copy Number Alteration analysis (LIFE-CNA) using WGS with approximately 6Ã coverage for detection of ctDNA in CRC patients as a marker for CRC detection and monitoring.  They described the analytical validity and a clinical proof-of-concept of LIFE-CNA using a total of 259 plasma samples collected from 50 patients with stage I to IV CRC and 61 healthy controls.  To reliably distinguish CRC patients from healthy controls, these investigators determined cut-offs for the detection of ctDNA based on global and regional cfDNA fragmentation patterns, transcriptionally active chromatin sites, and somatic copy number alterations.  They further combined global and regional fragmentation pattern into a machine learning (ML) classifier to accurately predict ctDNA for cancer detection.  By following individual patients throughout their course of disease, these researchers showed that LIFE-CNA enabled the reliable prediction of response or resistance to treatment up to 3.5 months before commonly used CEA.  The authors developed and validated a sensitive and cost-effective method for untargeted ctDNA detection at diagnosis as well as for treatment monitoring of all CRC patients based on genetic as well as non-genetic tumor-specific cfDNA features.  These researchers stated that once sensitivity and specificity have been externally validated, LIFE-CNA has the potential to be implemented into clinical practice.  To the best of the authorsâ knowledge, this was the 1st study to consider multiple genetic and non-genetic cfDNA features in combination with ML classifiers and to examine their potential in both cancer detection and treatment monitoring.\n",
      "In a systematic review and meta-analysis, Callesen et al (2022) examined  the current knowledge on ctDNA and its clinical utility in predicting outcomes in patients with metastatic CRC (mCRC).  PubMed, Embase, Cochrane Database of Systematic Reviews and Cochrane Central Register of Controlled Trials were searched.  Last search was performed on December 16, 2020.  These investigators included studies on patients with mCRC reporting the predictive or prognostic value of ctDNA.  They carried out separate random-effects meta-analyses to examine if baseline ctDNA and early changes in ctDNA levels during treatment were associated with survival.  The risk of bias was assessed according to the Quality in Prognosis Studies tool.  A total of 71 studies were included with 6,930 patients; 24 studies were included in meta-analyses.  High baseline ctDNA level was associated with short progression-free survival (PFS) (HR = 2.2; 95 % CI: 1.8 to 2.8; n = 509) and overall survival (OS) (HR = 2.4; 95 % CI: 1.9 to 3.1; n = 1,336).  A small or no early decrease in ctDNA levels during treatment was associated with short PFS (HR = 3.0; 95 % CI: 2.2 to 4.2; n = 479) and OS (HR = 2.8; 95 % CI: 2.1 to 3.9; n = 583).  Results on clonal evolution and lead-time were inconsistent.  A majority of included studies (n = 50/71) had high risk of bias in at least 1 domain.  The authors concluded that plasma ctDNA is a strong prognostic biomarker in mCRC; however, true clinical utility is lacking.\n",
      "Morais et al (2022) noted that the management of locally advanced rectal cancer (LARC) requires a multi-disciplinary approach, with an increasing interest for non-operative strategies.  Liquid biopsy for obtaining ctDNA could provide information on neoadjuvant chemoradiotherapy (nCRT) pathological response and cancer-specific prognosis; thus, might be a promising guide for these treatments.  These investigators carried out a systematic review of the studies available in literature to examine the role of ctDNA as a predictive and prognostic biomarker in LARC patients.  They retrieved 21 publications, of which 17 full-text articles and 4 abstracts.  Results have been categorized into 2 groups: predictive and prognostic.  Data regarding the usefulness of liquid biopsy in this setting is still inconclusive.  However, baseline higher levels of longer fragments of cfDNA and integrity index, tumor-specific mutations and certain methylated genes could predict non-responders.  Furthermore, undetectable baseline ctDNA and decrease of common rectal cancer mutations throughout treatment (dynamic monitoring) were predictive factors of pathological complete response (CR).  The continuous detection of ctDNA in different time-points of treatment (minimal residual disease [MRD]) was consistently associated with worse prognosis.  The authors concluded that ctDNA is a promising biomarker that could aid in predicting treatment response to nCRT and prognosis in patients with LARC.  Moreover, the ideal methods and timings for the liquid biopsy still have to be defined.\n",
      "National Comprehensive Cancer Networkâs clinical practice guideline on âColon Cancerâ (Version 1.2022) stated that ââMultigene Assays, Immunoscore, and Circulating Tumor DNA (ctDNA) -- Several assays have been developed in hopes of providing prognostic and predictive information to aid in decisions regarding adjuvant therapy in patients with stage II or III colon cancer â¦ the information from these tests can further inform the risk of recurrence over other risk factors, but the panel questions the value added.  Furthermore, evidence of predictive value in terms of the potential benefit of chemotherapy is lacking.  Therefore, the panel believes that there are insufficient data to recommend the use of multigene assays, Immunoscore, or post-surgical ctDNA to estimate risk of recurrence or determine adjuvant therapy.  ESMO has released similar recommendations regarding these assays, stating that their role in predicting chemotherapy  benefit is uncertain.  The NCCN Panel encourages enrollment in clinical trials to help with the generation of additional data on these assaysâ.\n",
      "Furthermore, National Comprehensive Cancer Networkâs clinical practice guideline on âRectal Cancerâ (Version 1.2022) does not mention the use of circulating tumor DNA / Signatera molecular monitoring as a management tool.\n",
      "Natera Signatera Minimal Residual Disease Test for Cutaneous Melanoma\n",
      "Rapanotti et al (2020) stated that isolating circulating melanoma cells (CMCs) represents a powerful method to monitor minimal residual disease (MRD).  These researchers documented that MCAM/MUC18/CD146 expression is strongly associated with disease progression.  ABCB5 is melanoma-stem antigen with self-renewal, proliferation, differentiation, tumorigenicity capabilities.  These findings supported these investigators to improve CMC detection, examining MCAM/MUC18/CD146 and ABCB5 as enrichment targets in MM progression.  Moreover, these researchers decided to compare possible molecular diversity of these CMC fractions with metastatic tissue expression, collecting concomitantly cutaneous in transit metastases (CTM).  They enriched CMCs from 8 melanoma patients staged â¥ pT1b AJCC, who developed CTMs at baseline or during follow-up.  These researchers examined a gene expression panel comprising ABCB5, the differentiation markers (tyrosinase, MART1), angiogenic factors (VEGF, bFGF), the cell-cell adhesion molecules (MCAM/MUC18/CD146 5'-portion, long, and short isoforms, E-cadherin, N-cadherin, VE-cadherin) and matrix-metallo-proteinases (MMP2 and MMP9) via high-sensitive RT-PCR.  Preliminary findings defined 3 distinct sub-populations: \"endothelial\" CD45-CD146+CMCs, \"stem\" CD45-ABCB5+CMCs and a \"hybrid stem-endothelial\" - CD45-MCAM+ABCB5+CMCs.  The expression panel documented that -- almost high expression found in CTMs -- like in 73.5 % of CMCs resulted positive for at least 1 transcript at baseline, showing gene-expression variability.  Longitudinal monitoring documented shut-down of all gene-expressions in \"endothelial\"- and \"hybrid stem-endothelial\"-subsets, while persistency or acquisition of MCAM/MUC18/CD146, VE-CADH and MMPs was documented in disease-progression status.  Conversely, a drastic expression shut-down was documented when patients achieved clinical remission.  The \"stem\"-CMCs fraction\" showed quite lower gene expression frequencies.  The authors concluded that MCAM/MUC18/CD146 and ABCB5 as melanoma-specific-targets are effective in the selection of highly primitive CMCs and highlighted those putative genes associated with disease spreading progression.\n",
      "Rapanotti et al (2021) noted that human malignant melanoma shows a high rate of mortality after metastases, and its incidence is continuously rising worldwide.  Several studies have suggested that MCAM/MUC18/CD146 plays an important role in the progression of this malignant disease.  MCAM/MUC18/CD146 is a typical single-spanning trans-membrane glycoprotein, existing as 2 membrane isoforms, long and short, and an additional soluble form, sCD146.  These researchers previously documented that molecular MCAM/MUC18/CD146 expression was strongly associated with disease progression.  Recently, they showed that MCAM/MUC18/CD146 and ABCB5 could serve as melanoma-specific-targets in the selection of highly primitive CMCs, and constitute putative proteins associated with disease spreading progression.  In this study, these investigators analyzed CD146 molecular expression at onset or at disease recurrence in an enlarged melanoma case-series.  For some patients, these researchers also carried out the time courses of molecular monitoring.  Moreover, they examined the role of soluble CD146 in different cohorts of melanoma patients at onset or disease progression, rather than in clinical remission, undergoing immune therapy or free from any clinical treatment.  The authors showed that MCAM/MUC18/CD146 could be considered as: (i) a membrane antigen suitable for identification and enrichment in melanoma liquid biopsy; (ii) a highly effective molecular \"warning\" marker for MRD monitoring; and (iii) a soluble protein index of inflammation and putative response to therapeutic treatments.\n",
      "Gracie et al (2021) stated that ctDNA is an emerging biomarker in melanoma.  In a systematic review and meta-analysis, these investigators examined its clinical use as a prognostic, pharmacodynamic (PD) and predictive biomarker.  They carried out a systematic search from January 2015 to April 2021, of the electronic databases PubMed, Cochrane Library and Ovid Medline to identify studies.  Studies were restricted to those published in English within the past 5 years, examining ctDNA in humans in 10 or more patients.  Survival data were extracted for meta-analysis using pooled treatment effect (TE), i.e., log HRs and corresponding standard error of TE for PFS or OS differences in patients who were ctDNA positive or negative.  PRISMA statement guidelines were followed.  A meta-analysis of 19 studies grouped according to methodology of ctDNA detection, revealed a combined estimate for HR of PFS (13 studies using droplet digital PCR (ddPCR) methodology (n = 1,002) of 2.10 (95 % CI: 1.71 to 2.59) revealing a poorer prognosis when ctDNA was detected.  This result was confirmed in the smaller analysis of (non-ddPCR, n = 347) 5 studies: HR = 2.45 (95 % CI: 1.29 to 4.63).  Similar findings were found in the OS analysis of 9 studies (ddPCR methodology, n = 841) where the combined HR was 2.78 (95 % CI: 2.21 to 3.49) and of the 5 studies (non-ddPCR methodology, n = 326) where the combined HR was 2.58 (95 % CI: 1.74 to 3.84).  Serial ctDNA levels on treatment showed a PD role reflecting response or resistance earlier than radiological assessment.  The authors concluded that ctDNA is a predictive, prognostic and PD biomarker in melanoma. Moreover, these researchers stated that standardization of analysis, data processing and reporting is needed before clinical adoption.Furthermore, National Comprehensive Cancer Networkâs clinical practice guideline on âMelanoma: Cutaneousâ (Version 2.2022) does not mention Signatera / minimal residual disease as a management tool.\n",
      "Natera Signatera Minimal Residual Disease Test for Prostate Cancer\n",
      "Antonarakis et al (2022) stated that liquid biopsy is a powerful tool that could enable treatment decisions for metastatic PCa patients with difficult-to-biopsy tumors.  However, the detection of genomic alterations via liquid biopsy is limited by the fraction (tumor fraction [TF]) of ctDNA within the total cfDNA content.  While previous investigation has preliminarily correlated TF with clinical features of PCa, these researchers sought to validate and provide additional resolution, such that a clinical practitioner might anticipate the probability of successful liquid biopsy profiling leveraging commonly assessed clinical and laboratory features.  A total of 813 liquid biopsy specimens were assessable, with 545 associated with a PSA measurement, collected in SOC settings across approximately 280 U.S. academic- or community-based cancer clinics from September 2018 to July 2021.  Deidentified data were captured into a real-world clinic-genomic database (CGDB).  Comprehensive genomic profiling (CGP) was carried out on extracted cfDNA from liquid biopsy samples.  In multi-variable models, higher PSA level, lower hemoglobin, lower albumin, higher alkaline phosphatase (all p < 0.001), and collection of liquid biopsy blood draw within 60 days of new treatment initiation (p = 0.002) were the most strongly associated features with higher TF.  At PSA levels of less than 5 ng/ml, 43 % of patients had a TF of less than 1 % indicating an increased likelihood of unevaluable results.  Conversely, at PSA levels of greater than 5 ng/ml, 78 % of patients had a TF of at least 1 % and 46 % had a TF of 10 % or greater, suggesting improved sensitivity for detection of targetable alterations.  The authors concluded that universal genomic profiling of PCa will require complementary use of liquid biopsy and tumor tissue profiling for suitable patients.  The likelihood of adequate ctDNA shedding into plasma is one consideration when deciding whether to pursue CGP via liquid biopsy versus tumor profiling.  Their real-world data suggested that PSA of less than 5 ng/ml was associated with lower ctDNA yield on liquid biopsy, potentially increasing the incidence of negative results or a need for confirmation with tissue testing.\n",
      "The authors stated that this analysis had several drawbacks.  First, this was not a prospectively enrolled study.  The patients included in this analysis reflected those for whom liquid biopsies have been ordered in the time period reflected.  It has been previously established that tumor burden and PSA burden correlate with TF.  However, the authorsâ database did not contain quantified measures of metastatic tumor burden, which is often difficult to evaluate given frequent boneâdominant metastases in mCRPC.  Second, PSA levels are influenced by castration status, and this analysis did not interpret PSA levels separately in the hormoneâsensitive and castrationâresistant metastatic settings.  Third, the estimation of TF performed in this analysis relied upon genomic features associated with cancer biology, yet which also could at times be due to other somatic signals such as clonal hematopoiesis.  These researchers hoped that the increasing emergence of tumorâinformed monitoring assays for reliable quantification of tumor content will allow further validation of the clinical associations that they identified in this analysis.\n",
      "ArteraAI Prostate Test for Identification of Patients who May Benefit from Therapy Intensification and Help Guide Treatment Decisions for Men with Localized Prostate Cancer\n",
      "The ArteraAI Prostate Test (Artera Inc.) has not been cleared or approved by the FDA.   The Test employs artificial intelligence (AI) models for identification of patients who may benefit from therapy intensification and help guide treatment decisions for men with localized prostate cancer. \n",
      "Esteva et al (2022) stated that PCa is the most common cancer in men and a leading cause of cancer death.  Determining a patient's optimal therapy is a challenge, where oncologists must select a therapy with the highest likelihood of success and the lowest likelihood of toxicity.  International standards for prognostication rely on non-specific and semi-quantitative tools, commonly leading to over- and under-treatment.  Tissue-based molecular biomarkers have attempted to address this; however, most have limited validation in prospective, randomized trials and expensive processing costs, posing substantial barriers to widespread adoption.  There remains a significant need for accurate and scalable tools to support therapy personalization.  These researchers demonstrated PCa therapy personalization by predicting long-term, clinically relevant outcomes using a multi-modal deep learning architecture and train models using clinical data and digital histopathology from prostate biopsies.  These investigators trained and validated models using 5 randomized, phase-III clinical trials conducted across hundreds of clinical centers.  Histopathological data were available for 5,654 of 7,764 randomized patients (71 %) with a median follow-up of 11.4 years.  Compared to the most common risk -- stratification tool-risk groups developed by the NCCN -- their models have superior discriminatory performance across all endpoints, ranging from 9.2 % to 14.6 % relative improvement in a held-out validation set.  This AI-based tool improves prognostication over standard tools and allows oncologists to computationally predict the likeliest outcomes of specific patients to determine optimal treatment.  Outfitted with digital scanners and internet access, any clinic could offer such capabilities, enabling global access to therapy personalization.\n",
      "Furthermore, NCCNâs clinical practice guideline on âProstate Cancerâ (Version 1.2023) states that âMultivariable models, such as gene expression classifiers or artificial intelligence (AI)-derived digital histopathology biomarkers, can combine clinical, pathologic, and other biomarkers to further improve risk stratificationâ.  However, it does not mention the use of artificial intelligence (AI)-derived digital histopathology biomarkers to guide treatment decisions for men with localized PCa.\n",
      "ColoScape Test (a qPCR-Based Multi-Gene Panel) for Qualitative Detection of Colorectal Cancer-Associated Gene Mutations and Methylation Markers in Liquid Biopsy Samples\n",
      "The ColoScape Test (DiaCarta Clinical Lab, DiaCarta, Inc.) is a novel diagnostic test that uses a multi-plex quantitative polymerase chain reaction (qPCR)-based multi-gene panel for the qualitative detection of CRC-associated gene mutations and methylation markers in liquid biopsy samples.  The test panel targets 61 mutations in 8 genes (APC, BRAF, CTNNB1, KRAS, NRAS, PIK3CA, SMAD4, and TP53) and 7 methylation markers (MYO1G, KCNQ5, C9ORF50, FLI1, CLIP4, ZNF132 and TWIST1) mostly associated with CRC.  The test employs DiaCartaâs proprietary XNA technology that leverages a sequence-specific clamp made by xeno-nucleic acid (XNA) to clamp the wild type sequences.\n",
      "Scimia et al (2018) noted that circulating cfDNA derived from tumors are becoming recognized as clinically significant and extremely useful biomarkers for detection of cancer and for monitoring the progression of targeted drug therapy and immunotherapy.  Screening programs for CRC in Europe employ the Fetal Immunochemical Test (FIT) test as a primary screener.  FIT+ patients are referred to immediate colonoscopy and the PPV is usually 25 %.  In a pilot study, these investigators reported a trial using the ColoScape assay panel to detect mutations in the APC, KRAS, BRAF and CTNNB1 genes, in order to collect preliminary performance indicators and plan a future, larger population study.  The assay was used on 52 prospectively collected whole-blood samples obtained from FIT+ patients enrolled in the CRC screening program of ASL NAPOLI 3 SUD, using colonoscopy as confirmation.  The assay's sensitivity for advanced adenomas was 53.8 % and the specificity was 92.3 %.  The PPV was 70.0 % and the NPV was 85.7 %.  Work-flow optimization was essential to maximize sensitivity.  Of note, 4 of the 6 positive cases missed by ColoScape had a less than sub-optimal DNA input.  Had they been ruled out as inadequate, sensitivity would have increased from 53.8 % to 69 %.  However, as stated previously, this was not a clinical trial, but rather an initial, preliminary technical evaluation.  The authors concluded that the findings of this study showed that ColoScape is a promising tool and further studies are needed to validate its use for the triage of FIT+ patients. \n",
      "Sun et al (2021) stated that CRC is one of the leading causes of cancer-related death.  Early detection is critical to reduce CRC morbidity and mortality.  These researchers developed a molecular clamping assay called the ColoScape assay for early diagnosis of CRC.  A total of 19 mutations in 4 genes (APC, KRAS, BRAF and CTNNB1) associated with early events in CRC pathogenesis were targeted in the ColoScape assay.  Xenonucleic acid (XNA)-mediated qPCR clamping technology was used to minimize the wild-type background amplification in order to improve assay sensitivity of CRC mutation detection.  The assay analytical performance was verified and validated, cfDNA and FFPE CRC patient samples were evaluated, and an ROC curve was employed to evaluate its performance.  The data showed that the assay analytical sensitivity was 0.5 % Variant Allele Frequency, corresponding to approximately 7 to 8 copies of mutant DNA with 5 ng total DNA input per test.  This assay was highly reproducible with intra-assay coefficient of variation (CV) of less than 3 % and inter-assay CV of less than 5 %.  These investigators examined 380 clinical samples including plasma cfDNA and FFPE samples from patients with pre-cancerous and different stages of CRC.  The preliminary assay clinical specificity and sensitivity for CRC cfDNA were: 100 % (95 % CI: 80.3 % to 97.5 %) and 92.2 % (95 % CI: 94.7 % to 100 %), respectively, with AUC of 0.96; 96 % specificity (95 % CI: 77.6 % to 99.7 %) and 92 % sensitivity (95 % CI: 86.1 % to 95.6 %) with AUC of 0.94 for CRC FFPE; 95 % specificity (95 % CI: 82.5 % to 99.1 %) and 62.5 % sensitivity (95 % CI: 35.8 % to 83.7 %) with AUC of 0.79 for pre-cancerous lesions cfDNA.  The authors concluded that they had developed a rapid and sensitive assay to enable the molecular characterization and detection of pre-cancerous CRC and CRC at different stages in a variety of samples.  However, this XNA-based qPCR with only 5 channels available in a qPCR instrument has its limitations.  However, the assay can be improved by targeting more genes and more mutation hotspots.  Recent molecular characterization and NGS analysis of cancer patients has provided unprecedent insight into cancer molecular mechanisms and revealed molecular signatures of different cancer types.  Inclusion of a broader biomarker panel will further improve the assay sensitivity.  This XNA-based technology can also be combined with NGS technology to cover a wide range of variants in many genes, making it applicable to the development of comprehensive DNA-based assays for a wide range of cancer diagnostics.\n",
      "These researchers stated that the preliminary clinical performance of the ColoScape assay showed good sensitivity, including the testing of pre-cancerous screening samples.  Mutations from early cancer patients were detected by the ColoScape assay and were confirmed by Sanger sequencing.  The ColoScape assay and Sanger sequencing results were 98 % concordant.  The ColoScape assay has about 89 % concordance with NGS.  There were 2 samples identified as mutants by the ColoScape assay and confirmed by Sanger sequencing, but not detectable by NGS, possibly due to the higher assay sensitivity (limit of detection [LOD] about 0.1 % VAF) of the ColoScape assay than that of NGS (LOD about 1 % VAF).  The ColoScape assay was used in testing pre-cancerous plasma from FIT-positive patients in a small study and showed 83.3 % PPV and 62.5% assay sensitivity, while FIT testing showed a PPV of 25 %.  Since FIT has such a high false negative rate, the ColoScape assay can potentially be used as a triage test in combination with a FIT test to improve the effectiveness of CRC patient management and treatment.  There are several CRC-related mutation detection and methylation-based screening assays, (e.g., Cologuard (Exact Sciences Corporation) and Epi ProColon (Epigenomics AG)), available in the market.  However, these FDA-approved assays are based only on a single gene methylation detection (Septin 9-Epi ProColon) or 2 methylation biomarkers combined with only 1 target gene mutation assay (Cologuard), which potentially limits the assay sensitivity.  In this preliminary study, these researchers demonstrated that the ColoScape assay could detect 66.6 % of mutations from advanced adenoma (AA) samples (although it was only from 10 samples and needs further confirmation), while the reported Cologuard AA detection rate is 42% to 46 %, suggesting that the ColoScape assay potentially has a comparable sensitivity to that of Cologuard for early colon cancer detection from patient blood samples.  In addition, these investigators showed a sensitivity for cfDNA sample for pre-cancerous lesions of about 62.5 % and specificity of 95.2 %, suggesting that the ColoScape assay has great potential for early pre-cancerous lesions screening.\n",
      "Furthermore, NCCNâs clinical practice guidelines on âColon Cancerâ (Version 3.2022) and âRectal Cancerâ (Version 4.2022) do not mention gene mutations and methylation markers as management tools.\n",
      "Oncuria Detect, Oncuria Monitor, and Oncuria Predict Tests for Bladder Cancer\n",
      "Oncuria is an urine-based molecular diagnostic test that is employed for ruling out the presence of bladder cancer (BCa); thus, it may reduce both unnecessary invasive and non-invasive diagnostics.\n",
      "Grossman et al (2005) noted that a combination of methods is used for diagnosis of bladder cancer (BCa) because no single procedure detects all malignancies.  While urine tests are frequently part of an evaluation, they have either been non-specific for cancer or required specialized analysis at a laboratory.  These researchers examined if a point-of-care (POC) proteomic test that measures the nuclear matrix protein NMP22 in voided urine could enhance detection of malignancy in patients with risk factors or symptoms of BCa.  A total of 23 academic, private practice, and veterans' facilities in 10 states prospectively enrolled consecutive patients from September 2001 to May 2002.  Participants included 1,331 patients at elevated risk for BCa due to factors such as history of smoking or symptoms including hematuria and dysuria.  Patients at risk for malignancy of the urinary tract provided a voided urine sample for analysis of NMP22 protein and cytology before cystoscopy.  The diagnosis of BCa, based on cystoscopy with biopsy, was accepted as the reference standard.  The performance of the NMP22 test was compared with voided urine cytology as an aid to cancer detection.  Testing for the NMP22 tumor marker was conducted in a blinded manner; BCa was diagnosed in 79 patients.  The NMP22 assay was positive in 44 of 79 patients with cancer (sensitivity, 55.7 %; 95 % CI: 44.1 % to 66.7 %), whereas cytology test results were positive in 12 of 76 patients (sensitivity, 15.8 %; 95 % CI: 7.6 % to 24.0 %).  The specificity of the NMP22 assay was 85.7 % (95 % CI: 83.8 % to 87.6 %) compared with 99.2 % (95 % CI: 98.7 % to 99.7 %) for cytology.  The proteomic marker detected 4 cancers that were not visualized during initial endoscopy, including 3 that were muscle invasive and 1 carcinoma in-situ.  The authors concluded that the NMP22 assay may be a useful adjunct to cystoscopy for diagnosing BCa.  Moreover, these researchers stated that studies in different patient populations are needed to further define the role of this assay in patients with risk factors and symptoms suggestive of possible BCa.\n",
      "Shimizu et al (2016) stated that urine-based assays that can non-invasively detect BCa have the potential to reduce unnecessary and invasive procedures.  These researchers developed a multiplex immunoassay that could accurately and simultaneously monitor 10 diagnostic urinary protein biomarkers for use as a non-invasive test for detection of BCa.  A customized electro-chemiluminescent multiplex assay was constructed (Meso Scale Diagnostics, LLC, Rockville, MD) to detect the following urinary proteins; IL8, MMP9, MMP10, ANG, APOE, SDC1, A1AT, PAI1, CA9 and VEGFA.  Voided urine samples from 2 cohorts were collected before cystoscopy and samples were analyzed blinded to the clinical status of the participants.  Means (Â± SD) and ROC curve analysis were used to compare assay performance and to evaluate the diagnostic accuracy of the diagnostic signature.  Comparative diagnostic performance analyses revealed an AUROC value of 0.9258 for the multiplex assay and 0.9467 for the combination of the single-target ELISA assays (p = 0.625); thus, there was no loss of diagnostic utility for the MSD multiplex assay.  Analysis of the independent 200-sample cohort using the multiplex assay achieved an overall diagnostic sensitivity of 0.85, specificity of 0.81, PPV of 0.82 and NPV of 0.84.  The authors concluded that it was technically feasible to simultaneously monitor complex urinary diagnostic signatures in a single assay without loss of performance.  The described protein-based assay has the potential to be developed for the non-invasive detection of BCa.\n",
      "Goodison et al (2016) noted that BCa is among the most commonly diagnosed malignancies globally, and due to the high recurrence of post-operative disease, it is one of the most prevalent in many countries.  The development of non-invasive molecular assays that could accurately detect and monitor BCa would be a major advance, benefiting both patients and healthcare systems.  These researchers have previously identified a urinary protein biomarker panel that is being developed for use in at-risk patient cohorts.  These investigators examined the potential utility of the multiplex assay in a Japanese cohort.  The Japanese study cohort collected from urology clinics at 2 institutions was comprised of a total of 288 subjects.  The protein biomarker panel (IL8, MMP9, MMP10, ANG, APOE, SDC1, A1AT, PAI1, CA9, VEGFA) was monitored in voided urine samples collected before cystoscopy using a custom multiplex ELISA assay.  The diagnostic performance of the biomarker panel was evaluated using ROCs, predictive modeling and descriptive statistics.  Urinary biomarker concentrations were significantly elevated in cases versus controls, and in cases with high-grade and muscle-invasive tumors.  The AUC for the 10-biomarker assay was 0.892 (95 % CI: 0.850 to 0.934), with an overall diagnostic sensitivity and specificity of 0.85 and 0.81, respectively.  A predictive model trained on the larger institutional cohort correctly identified 99 % of the cases from the 2nd institution.  The authors concluded that urinary levels of a 10-biomarker panel enabled discrimination of patients with BCa.  These investigators stated that this multiplex urinary diagnostic assay has the potential to be developed for the non-invasive detection of BCa in at-risk Japanese patients.  They stated that the diagnostic assay achieved encouraging performance values and will be the focus of ongoing studies to examine the potential added value of the multiplex assay if integrated into clinical decision-making.\n",
      "The authors stated that this study had 2 main drawbacks.  First, the cohort was comprised of greater than 50 % cases, but disease prevalence was typically considerably lower in routine urologic practice.  While it was important to initially test enough cases to achieve statistical significance, it will be necessary to perform additional studies that reflect urology clinic presentation to examine predictive value of the assay.  It will also be necessary to include more diverse controls that may be under-represented in this study cohort.  To address these issues, these researchers have launched a prospective, multi-center clinical trial, which will assess the multiplex diagnostic assay in subjects with gross hematuria, microscopic hematuria and history of BCa on tumor surveillance.  Such a study would minimize selection bias, better represent urological disease prevalence, and evaluate potential confounding co-morbidities in the study population.  Second, these investigators focused only on subjects with primary BCa, however, they had previously reported that the multiplex assay was also accurate for the detection of recurrent BCa (sensitivity 79 %, specificity 88 %), outperforming the Urovysion cytogenetic assay and VUC.  The inclusion of Japanese patients on routine surveillance following primary BCa treatment will be an additional objective and will enable evaluation of potential prognostic use of the assay.  These researchers have initiated the development of BCa diagnostic nomograms that incorporate biomarker data with relevant clinical information (e.g., age, sex, race, and tobacco history) in U.S. cohort studies, and this can also be extended to future Japanese cohort studies.\n",
      "Furuya et al (2020) examined the analytical performance of a multiplex assay (Oncuria) to quantify protein biomarkers towards a BCa-associated diagnostic signature in voided urine.  By means of the Luminex xMAP technology, a custom immunoassay was developed to measure the concentrations of 10 urinary analytes (angiogenin, ANG; apolipoprotein E, APOE; alpha-1 antitrypsin, A1AT; carbonic anhydrase 9, CA9; interleukin 8, IL8; matrix metallopeptidase 9, MMP9; matrix metallopeptidase 10, MMP10; plasminogen activator inhibitor 1, PAI1; syndecan 1, SDC1; vascular endothelial growth factor, VEGF).  Selectivity, sensitivity, specificity, precision, linearity, dynamic range, and detection threshold were assessed using recombinant proteins and human urine samples.  Analytical variability with respect to batch size, run, day, operator, and interference were also evaluated.  Analytical evaluation demonstrated that all antigen cross-reactivity was noted to be less than 1 % of the tested concentration; minimal detected dose ranged from 0.295 âpg/ml in IL8 to 31.1 âpg/ml in APOE; highly reproducible and accurate noting coefficient of variation (CV) and relative error (RE) values of below 15 % for all analytes; and minimal interference.  The assay could be completed in less than 5 âhours using as little as 150 âÎ¼l of voided urine.  The authors concluded that to their knowledge, this was the 1st multiplex bead-based immunoassay for the non-invasive detection of BCa that has been analytically validated as a tool with the potential to help clinicians manage patients at risk of harboring BCa.  These investigators stated that implementing the assay clinically is a necessary 1st step to improve on current diagnostic approaches and to demonstrate the clinical utility of advanced molecular diagnostic testing.\n",
      "Hirasawa et al (2021) noted that due to insufficient accuracy, urine-based assays currently have a limited role in the management of patients with BCa.  The identification of multiplex molecular signatures associated with disease has the potential to address this deficiency and to assist with accurate, non-invasive diagnosis and monitoring.  These researchers examined the performance of Oncuria, a multiplex immunoassay for bladder detection in voided urine samples.  The test was examined in a multi-center cohort of 362 prospectively collected subjects presenting for BCa evaluation.  The parallel measurement of 10 biomarkers (A1AT, APOE, ANG, CA9, IL8, MMP9, MMP10, PAI1, SDC1 and VEGFA) was carried out in an independent clinical laboratory.  The ability of the test to identify patients harboring BCa was assessed; BCa status was confirmed by cystoscopy and tissue biopsy.  The association of biomarkers and demographic factors was evaluated using linear discriminant analysis (LDA) and predictive models were derived using supervised learning and cross-validation analyses.  Diagnostic performance was assessed using ROC curves.  The combination of the 10 biomarkers provided an AUROC 0.93 [95 % CI: 0.87 to 0.98], outperforming any single biomarker.  The addition of demographic data (age, sex, and race) into a hybrid signature improved the diagnostic performance AUROC 0.95 [95 % CI: 0.90 to 1.00].  The hybrid signature achieved an overall sensitivity of 0.93, specificity of 0.93, PPV of 0.65 and NPV of 0.99 for BCa classification.  Sensitivity values of the diagnostic panel for high-grade BCa, low-grade BCa, muscle invasive BCa (MIBCa) and non-muscle invasive BCa (NMIBCa) were 0.94, 0.89, 0.97 and 0.93, respectively.  The authors concluded that the development of an accurate and robust urinary test for the detection of BCa would benefit both patients and healthcare systems.  In a multi-center study, the multiplex Oncuria test achieved highly encouraging diagnostic performance.  The test uses established technology enabling rapid uptake in clinical laboratories around the world.  These researchers stated that additional studies are underway to examine the potential added value of the test in clinical decision-making.\n",
      "The authors stated that this study had several drawbacks.  First, as tertiary-care facilities, they tended to see more high-grade, high-stage disease, which was reflected in this study cohort.  To further confirm the robustness of the multiplex assay, subsequent studies must evaluate larger cohorts that include more subjects with low-grade, low-stage disease.  Second, these researchers did not have complete smoking data for all subjects in the cohort; thus, an association with smoking history was not possible.  Third, processed, banked urines were analyzed.  Urines were centrifuged and separated into cellular pellet and supernatant before storage at â 80 Â°C.  It is feasible that freshly voided urine samples may provide different results.  These investigators are currently investigating the performance of the test in urines processed via a number of different protocols, including freshly voided urines.  To address these issues, the Oncuria test is currently being evaluated in 3 large, international, prospective, multi-center clinical trials.  These trials will include 1st-event diagnosis and disease recurrence monitoring.\n",
      "In a systematic review and meta-analysis, Soputro et al (2022) examined the diagnostic performance of FDA-approved urinary biomarkers in the evaluation of primary hematuria for investigation of BCa.  The scientific databases Medline, Embase, PubMed and Web of Science were searched to collect studies.  Studies that examined the diagnostic performance of FDA-approved urinary biomarkers in investigating patients with primary hematuria without a prior history of BCa were included.  Quality of studies was assessed using the JBI Criteria.  Bi-variate mixed-effects regression model was used to calculate pooled sensitivities and specificities for each biomarker.  A total of 18 studies were included in the analysis.  The biomarkers assessed in these studies were CxBladder, AssureMDx, Bladder Tumor Antigen (BTA), NMP22, UroVysion and Immunocyt/uCyt+.  Several biomarkers, such as AssureMDx, CxBladder and Immunocyt, were shown to have better diagnostic performance based on their sensitivity, specificity and diagnostic odds ratio (DOR), as well as PLR and NLR.  Across the 6 biomarkers, sensitivity ranged from 0.659 to 0.973, and the specificity ranged between 0.577 and 0.833.  The authors concluded that despite certain biomarkers demonstrated better performance, current diagnostic abilities of the FDA-approved biomarkers remain insufficient for their general application as a rule out test for BCa diagnosis and as a triage test for cystoscopy in patients with primary hematuria.  These researchers stated that high-quality prospective studies are needed to further analyze this and also analyze the correct scenario in which urinary biomarkers may be best used.\n",
      "The authors stated that this study had several drawbacks.  First, some urinary marker results were based on limited number of studies.  Sample size of most studies were also limited and varying, ranging from only 1 histologically proven BCa to 245.  Therefore, one might question the reproducibility of the results in larger studies with potentially more heterogenous cohort.  Moreover, despite most diagnostic algorithms involved the use of cystoscopy and histopathological confirmation shortly following sample collection and up to 3 months, there may appear to be some heterogeneity in the cystoscopic evaluations that is dependent on technique, imaging modality, and inter-observer variability.  Furthermore, given current evidence that BCa more likely to present with gross hematuria and that microscopic hematuria more likely to be attributed to false positive results, this analysis only included a subâgroup of BCa patients presenting with primary macrohematuria.  Further research is needed to establish the effectiveness of the different biomarkers in setting of microscopic as well as macroscopic hematuria for workâup of BCa.\n",
      "Chen et al (2022) noted that BCa is a biologically heterogeneous disease with variable clinical presentations, outcomes and responses to therapy; therefore, the clinical use of single biomarkers for the detection and prediction of biological behavior of BCa is limited.  These researchers have previously identified and validated a BCa diagnostic signature composed of 10 biomarkers, which has been incorporated into a multiplex immunoassay bladder cancer test, Oncuria.  In a pilot study, these investigators examined if these 10 biomarkers could assist in the prediction of BCa clinical outcomes.  Tumor gene expression and patient survival data from BCa cases from the Cancer Genome Atlas (TCGA) were analyzed.  Alignment between the mRNA expression of 10 biomarkers and the TCGA 2017 subtype classification was assessed.  Kaplan-Meier analysis of multiple gene expression datasets indicated that high expression of the combined 10 biomarkers correlated with a significant reduction in OS.  The analysis of 3 independent, publicly available gene expression datasets confirmed that multiplex prognostic models outperformed single biomarkers.  In total, 8 of the 10 biomarkers from the Oncuria test were significantly associated with either luminal or basal molecular subtypes; therefore, the test has the potential to assist in the prediction of clinical outcome.  These researchers stated that future investigation may enable the construction of algorithms that can define and weigh specific biomarkers for the stratification of patients for clinical management.\n",
      "The authors noted that as a pilot study, a number of drawbacks were evident.  While the use of concise multiplex signatures for prognosis and subtyping is promising, as a retrospective study leveraging publicly available databases, interpretation requires caution.  For example, the treatments that patients have received will influence outcomes and were highly heterogeneous and incomplete across cohorts; therefore, these investigators could not include this information in their analysis.  Improving these aspects for future in-depth studies will enable comprehensive evaluation of the value of multiplex signatures in predicting clinical outcomes.\n",
      "In a pilot study, Murakami et al (2022) examined if Oncuria would influence physicians' use of non-invasive and invasive diagnostic tests for microhematuria, gross hematuria, and BCa surveillance.  These researchers carried out a survey-based study to evaluate physician management of 9 clinical scenarios involving real-world data from patients with gross hematuria, microhematuria, and BCa on surveillance.  They randomly sampled 15 practicing urologists and generated data including 135 patient-by-urologist interactions and 2,160 decision points.  Urologists recommended a selection of diagnostic tests and procedures before and after Oncuria results were provided.  These investigators examined changes in provider use of non-invasive and invasive diagnostic tests after Oncuria results were provided.  Over 90 % of all urologists changed their diagnostic behavior in at least 1 patient case with the addition of Oncuria results.  The total number of diagnostic procedures was reduced by 31 % following the disclosure of a negative Oncuria test and 27 % following the disclosure of a positive Oncuria test.  The authors concluded that the Oncuria urine-based test, a highly promising actionable molecular diagnostic capable of ruling out the presence of bladder cancer, can reduce the number of unnecessary invasive and non-invasive diagnostics recommended by physicians, which has the potential to lower healthcare costs and improve outcomes for patients.\n",
      "The authors stated that this study had several drawbacks.  First, the study sample of 15 randomly selected urologists may not be representative of a national sample of urologists.  Second, the current clinical vignettes presented in this study have not been tested for validity and reliability.  Third, there was a limitation when aggregating data on disparate clinical scenarios.  Despite these drawbacks, the information generated in this study will be helpful to direct future studies.  Such future studies should focus on the longitudinal assessment of changes in the Oncuria risk score, as such changes may give insight into the risk associated with the development of BCa.  Indeed, a movement toward risk-based counseling and decision-making holds the promise of personalizing medicine in a heterogeneous and complex clinical entity such as BCa.\n",
      "Papavasiliou et al (2023) stated that evidence on the use of biomarkers to detect BCa in the general population is scarce.  In a systematic review and meta-analysis, these investigators examined the evidence on the diagnostic performance of biomarkers that might be suitable for use in community and primary care settings.   Database searches on Medline and Embase from January 2000 to May 2022 resulted in 4,914 unique citations, 44 of which met inclusion criteria.  Included studies reported on 112 biomarkers and combinations.  Heterogeneity of designs, populations and outcomes allowed for the meta-analysis of 3 biomarkers identified in at least 5 studies (NMP-22, UroVysion, uCyt+).  These 3 biomarkers showed similar discriminative ability (adjusted AUC estimates ranging from 0.650 to 0.707), although for NMP-22 and UroVysion there was significant unexplained heterogeneity between included studies.  Narrative synthesis revealed the potential of these biomarkers for use in the general population based on their reported clinical utility, including effects on clinicians, patients, and the healthcare system.  These investigators identified some promising novel biomarkers and biomarker combinations (n of less than 3 studies for each biomarker/combination) with NPVs of 90 % or higher.  The authors concluded that these biomarkers have potential for use as a triage tool in community and primary care settings for reducing unnecessary specialist referrals.  Moreover, these researchers stated that despite promising emerging evidence, further validation studies in the general population are needed at different stages within the diagnostic pathway.\n",
      "OvaWatch for Evaluation of the Risk of Ovarian Cancer\n",
      "OvaWatch (Aspira Women's Health, Aspira Labs, Inc.) is used to aid in evaluating the risk of ovarian cancer when the initial clinical assessment of the adnexal mass is either indeterminant or benign.  It entails biochemical assays of 7 proteins (follicle stimulating hormone [FSH], human epididymis protein 4 [HE4], apolipoprotein A-1, transferrin, beta-2 macroglobulin, pre-albumin [i.e., transthyretin], and cancer antigen 125 [Ca 125]).\n",
      "Reilly et al (2022) noted that early detection of ovarian cancer, the deadliest gynecologic cancer, is crucial for reducing mortality.  Current non-invasive risk assessment measures include protein biomarkers in combination with other clinical factors, which vary in their accuracy.  Machine learning (ML) can be employed to optimize the combination of these features, resulting in more accurate assessment of malignancy.  However, the low prevalence of the disease can make rigorous validation of these tests challenging and can result in unbalanced performance.  MIA3G is a deep feed-forward neural network for ovarian cancer risk assessment, using 7 protein biomarkers along with age and menopausal status as input features.  The algorithm was developed on a heterogenous data set of 1,067 serum specimens from women with adnexal masses (prevalence = 31.8 %).  It was subsequently validated on a cohort almost twice that size (n = 2,000).  In the analytical validation data set (prevalence = 4.9 %), MIA3G demonstrated a sensitivity of 89.8 % and a specificity of 84.02 %. The PPV was 22.45 %, and the NPV was 99.38 %.  When stratified by cancer type and stage, MIA3G achieved sensitivities of 94.94 % for epithelial ovarian cancer, 76.92 % for early-stage cancer, and 98.04 % for late-stage cancer.  The authors concluded that the balanced performance of MIA3G resulted in a high sensitivity and high specificity, a combination that may be clinically useful for providers in evaluating the appropriate management strategy for their patients.  These investigators stated that future directions may include the addition of new biomarkers or other modalities to strengthen the algorithms' ability to both detect and rule out malignancy and advance the diagnostic ability of non-invasive testing.\n",
      "The authors stated that the main drawbacks of this study included retrospective nature of the data set and the unequal, albeit random, assignment of histologic subtypes between the training and validation data sets.  Although MIA3G was developed and validated on a highly diverse cohort obtained by merging several studies, most of these studies were retrospective in design with data collected from patients who were confirmed to have an adnexal mass and scheduled for surgery at the time of diagnosis.  To address this, prospective studies are currently underway to validate the algorithm's performance in patients with a variety of clinical presentations.  Furthermore, because of the random assignment of samples to the training and validation data sets, there was no way to match the distribution of cancer types between sets.  For example, by happenstance, 5 of the tumors in the validation set were stromal tumors and 1 was a germ cell tumor, subtypes known to have a different biomarker presentation compared with the more common epithelial types.  In the test set, however, MIA3G demonstrated 100 % sensitivity in non-epithelial malignancies, as sarcomas and carcinosarcomas comprised 4 of 5 non-epithelial malignancies in that set.  These cancer types present more similar to epithelial ovarian cancer in terms of biomarker distribution.  Non-epithelial subtypes are rare presentations of ovarian cancer, comprising approximately 10 % of all ovarian malignancies; thus, their particularly low incidence presents a challenge with regard to generating sufficient data for training and validating ML algorithms.  These researchers stated that future directions should include evaluating how to train an algorithm on multiple subtypes that express different biomarker patterns and achieve consistent test performance across these subtypes.\n",
      "Reilly et al (2023) stated that conservative management of adnexal mass is needed when there is imaging-based and clinical evidence of benign characteristics.  Malignancy risk is, however, a concern due to the mortality rate of ovarian cancer.  Malignancy occurs in 10 % to 15 % of adnexal masses that go to surgery, whereas the rate of malignancy is much lower in masses clinically characterized as benign or indeterminate.  Additional diagnostic tests could assist conservative management of these patients.  These researchers reported the clinical validation of OvaWatch, a multi-variate index assay, with real-world evidence of performance that supports conservative management of adnexal masses.  OvaWatch employs a previously characterized neural network-based algorithm combining serum biomarkers and clinical co-variates and was used to examine malignancy risk in prospective and retrospective samples of patients with an adnexal mass.  Retrospective data sets were assembled from previous studies using patients who had adnexal mass and were scheduled for surgery.  The prospective study was a multi-center trial of women with adnexal mass as identified on clinical examination and indeterminate or asymptomatic by imaging.  The performance to detect ovarian malignancy was evaluated at a previously validated score threshold.  In retrospective, low prevalence (n = 1,453, 1.5 % malignancy rate) data from patients who received an independent physician assessment of benign, OvaWatch has a sensitivity of 81.8 % [95 % CI: 65.1 to 92.7] for identifying a histologically confirmed malignancy, and a NPV of 99.7 %.  OvaWatch identified 18/22 malignancies missed by physician assessment.  A prospective data set had 501 patients where 106 patients with adnexal mass went for surgery.  The prevalence was 2 % (10 malignancies).  The sensitivity of OvaWatch for malignancy was 40 % (95 % CI: 16.8 % to 68.7 %), and the specificity was 87 % (95 % CI: 83.7 % to 89.7 %) when patients were included in the analysis who did not go to surgery and were evaluated as benign.  The NPV remained 98.6 % (95 % CI: 97.0 % to 99.4 %).  An independent analysis set with a high prevalence (45.8 %) the NPV value was 87.8 % (95 % CI: 75.8 % to 94.3 %).  The authors concluded that OvaWatch showed high NPV across diverse data sets and promised utility as an effective diagnostic test supporting management of suspected benign or indeterminate mass to safely decrease or delay unnecessary surgeries.\n",
      "The authors stated that it is important to note that prospective data set was limited in size.  The low likelihood of finding malignancies in patients with incidentally discovered and mostly simple cystic adnexal masses and the rare nature of ovarian cancer impacted the prevalence and likely the accuracy of performance metrics.  Information on imaging and physician impression of the masses was also not recorded frequently enough to allow meaningful stratification on other diagnostic factors and this may hide interesting interactions in the data.  Even though the study did not collect specific information of benign masses, the performance of OvaWatch might support the evaluation of adnexal mass suspicious as endometriosis cysts versus a mass arising from an endometrioid ovarian cancer or metastatic endometrial cancer to the ovary through the testing result of low probability of malignancy versus indeterminate.  Furthermore, these researchers acknowledged that OvaWatch was designed on data that was initially collected to address a higher risk population.  The algorithm was developed and validated on a highly diverse cohort obtained by merging several studies, mostly retrospective, with data collected from patients who were confirmed to have an adnexal mass and scheduled for surgery at the time of diagnosis.  The influence of the higher risk patients on the biomarker values in the train/test sets have led to low sensitivity in the prospective populations, collected from women at lower risk.  However, it is noteworthy how consistent the performance has been over these seemingly disparate data sets.  Finally, not all patients were followed-up with surgery, and so did not receive the âgold standardâ diagnostic outcome.  The assumption of benign mass for many in this study was based on hypotheses regarding the low rate of progression of masses over time and physician assessment.\n",
      "Furthermore, National Comprehensive Cancer Networkâs clinical practice guideline on âOvarian Cancer/Fallopian Tube Cancer/Primary Peritoneal Cancerâ (Version 1.2023) does not mention protein biomarkers as management tools.\n",
      "Solid Tumor Expanded Panel\n",
      "The Solid Tumor Expanded Panel employs NGS to detect mutations present in DNA (523 genes) and RNA (55 genes) extracted from formalin-fixed, paraffin embedded (FFPE) tissue sections of solid tumors.  The test is designed to detect SNVs and small insertions/deletions, as well as whole-gene copy number alterations and translocations in a select group of genes; MSI and/or TMB are/is also evaluated.\n",
      "Vashistha et al (2020) stated that to support the rising need for testing and to standardize tumor DNA sequencing practices within the U.S. Department of Veterans Affairs (VA)'s Veterans Health Administration (VHA), the National Precision Oncology Program (NPOP) was launched in 2016.  These investigators evaluated oncologists' practices, concerns, and perceptions regarding NGS and the NPOP.  By means of a purposive total sampling approach, oncologists who had previously ordered NGS for at least 1 tumor sample via the NPOP were invited to participate in semi-structured interviews.  Questions assessed the following: expectations for the NPOP, procedural requirements, applicability of testing results, and the summative utility of the NPOP.  Interviews were assessed using an open coding approach.  Thematic analysis was carried out to examine the completed codebook.  Themes were defined deductively by reviewing the direct responses to interview questions as well as inductively by identifying emerging patterns of data.  Of the 105 medical oncologists who were invited to participate, 20 (19 %) were interviewed from 19 different VA medical centers in 14 states.  Five recurrent themes were observed.  First: Educational Efforts Regarding Tumor DNA Sequencing Should be Undertaken; Second: Pathology Departments Share a Critical Role in Facilitating Test Completion; Third: Tumor DNA Sequencing via NGS Serves as the Most Comprehensive Testing Modality within Precision Oncologyâ Fourth: The Availability of the NPOP Has Expanded Options for Select Patients; and Fifth: The Completion of Tumor DNA Sequencing through the NPOP Could Help Improve Research Efforts within VHA Oncology Practices.  The authors concluded that medical oncologists believed that the availability of tumor DNA sequencing via the NPOP could potentially result in an improvement in outcomes for veterans with metastatic solid tumors.  Efforts should be directed toward improving oncologists' understanding of sequencing, strengthening collaborative relationships between oncologists and pathologists, and examining the role of comprehensive NGS panels within the battery of precision tests.\n",
      "The authors stated that this study had several drawbacks.  First, the interviewees solely included medical oncologists who served veterans within the VHA.  although many of the perceptions and concerns surrounding tumor DNA sequencing are likely similar between oncologists within the VHA and those employed elsewhere, the familiarity of sequencing may vary across practice settings, and the availability of consultative support similar to that provided by the NPOP is potentially unavailable to oncologists external to the VHA. Thus, the implications of these findings may not be universally applicable.  Second, the purposive sampling approach only sought participants who had requested at least 1 sample for sequencing via the NPOP.  Oncologists who have never requested testing may have a lesser understanding of the indications and benefits of tumor DNA sequencing, and their input could have been valuable to further highlight the need for specific educational initiatives.  Furthermore, oncologists who had not previously ordered testing may have felt that completing multi-gene NGS panels was simply not beneficial, and their perceptions regarding the use of tumor DNA sequencing could have been critically missed.  Third, although the interviewees provided important information regarding the vital role of pathologists towards the completion of molecular sequencing, these researchers excluded providers who were not medical oncologists from this trial, and consequently, the perceptions of pathologists were not included.  These investigators stated that a separate qualitative analysis is needed to examine the opinions and concerns regarding the rising number of requests for multi-gene NGS panels among this critical group of physicians.\n",
      "Sha et al (2020) noted that tumor mutational burden (TMB), defined as the number of somatic mutations per mega-base of interrogated genomic sequence, varies across malignancies.  Panel sequencing-based estimates of TMB have largely replaced WES-derived TMB in the clinic.  Retrospective evidence suggested that TMB could predict the effectiveness of ICIs, and data from KEYNOTE-158 led to the recent FDA approval of pembrolizumab for the TMB-high tumor subgroup.  Unmet needs include prospective validation of TMB cut-offs in relationship to tumor type and patient outcomes.  In addition, standardization and harmonization of TMB measurement across test platforms are important to the successful implementation of TMB in clinical practice.  The authors concluded that evaluation of TMB as a predictive biomarker creates the need to harmonize panel-based TMB estimation and standardize its reporting.  These researchers stated that TMB can improve the predictive accuracy for immunotherapy outcomes; and has the potential to expand the candidate pool of patients for treatment with ICIs. \n",
      "The authors stated that calculation of TMB from panel-based sequencing data has important limitations.  There is a need to harmonize the types of mutations analyzed as missense mutations are included in all panels, but other types can vary.  Another limitation of a pan-cancer TMB cut-off approach is the limited number of patients with high quality outcome data and available TMB data.  These researchers stated that larger datasets for TMB and clinical outcome of ICI-treated patients will facilitate the optimization of TMB cut-offs within specific cancer types and potentially extend the approval of immune therapies to larger patient populations.  In addition, prospective, randomized studies are needed to validate a TMB-high cutoff and to examine optimal TMB cut-offs in specific tumor types.  They stated that TMB combined with other potential biomarkers and computational assistance is paving the way towards a precision immunotherapy approach.\n",
      "Gambardella et al (2021) stated that molecular-matched therapies have revolutionized cancer treatment.  In a retrospective, non-randomized, single-center study, these investigators examined the improvement in clinical outcomes of applying an in-house customized NGS panel in a single center.  Patients with advanced solid tumors were molecularly selected to receive a molecular-matched treatment into early phase clinical trials versus best investigators choice, according to the evaluation of a multi-disciplinary molecular tumor board (MTB).  The primary endpoint was PFS assessed by the ratio of patients presenting 1.3-fold longer PFS on matched therapy (PFS2) than with prior therapy (PFS1).  Of a total of 231 molecularly screened patients, 87 were eligible for analysis.  Patients who received matched therapy had a higher median PFS2 (6.47 months; 95 % CI: 2.24 to 14.43) compared to those who received standard therapy (2.76 months; 95 % CI: 2.14 to 3.91, log-rank p = 0.022).  The proportion of patients with a PFS2/PFS1 ratio over 1.3 was significantly higher in the experimental arm (0.33 versus 0.08; p = 0.008).  The authors demonstrated the pivotal role of the institutional MTB in evaluating the results of a customized NGS panel.  These researchers stated that this process optimized the selection of available therapies, improving disease control.  Moreover, they stated that prospective, randomized trials are needed to confirm this approach.\n",
      "The authors stated that this study had several drawbacks that could have resulted in uncontrolled biases, such as its retrospective and non-randomized design, as well as the single-center setting, which yielded a reduced patient sample.  Another drawback could be the short follow-up, yet this could also had limited the magnitude of benefit in the targeted therapy cohort.  Indeed, at data cut-off (October 15, 2020), treatment is ongoing in 41 % of patients included in the molecular-matched cohort, compared to only 9 % in the standard therapy arm.  In addition, these were early clinical trial results with a median follow-up time of greater than 3 months, classically a key time-point for phase-I development.  Focusing on patients who are still under treatment and had a prior line, however, all those in the standard therapy cohort (6/6) had a PFS2/PFS1 ratio less than 1.3, compared to only 37.5 % (3/8) in the experimental treatment arm.\n",
      "Tarawneh et al (2022) stated that increasing knowledge of cancer biology and an expanding spectrum of molecularly targeted therapies provide the basis for precision oncology.  Despite extensive gene diagnostics, previous reports indicated that less than 10 % of patients benefit from this concept.  In a retrospective study, these researchers analyzed all patients referred to their center's MTB from 2018 to 2021.  Molecular testing by NGS included a 67-gene panel for the detection of short-sequence variants and copy-number alterations, a 53- or 137-gene fusion panel and an ultra-low-coverage whole-genome sequencing for the detection of additional copy-number alterations outside the panel's target regions.  Immunohistochemistry for MSI and PD-L1 expression complemented NGS.  A total of 109 patients were referred to the MTB.  In all, 78 patients received therapeutic proposals (70 based on NGS) and 33 were treated accordingly.  Evaluable patients treated with MTB-recommended therapy (n = 30) had significantly longer PFS than patients treated with other therapies (n = 17) (4.3 versus 1.9 months, p = 0.0094).  A total of 7 patients treated with off-label regimens experienced major clinical benefits.  The authors concluded that the combined focused sequencing assays detected targetable alterations in the majority of patients; and patient benefits appeared to lie in the same range as with large-scale sequencing approaches.  They stated that this approach provides a reasonable 1st step for precision oncology and a good alternative to larger gene panels and WES, especially in centers where the implementation of these technologies is not (yet) feasible in the clinical practice.\n",
      "The authors stated that this study had several drawbacks.  Potential interpretation biases of this study arose from the lack of randomization, its retrospective nature and the small patient population with a large heterogeneity in terms of diagnoses and previous treatments.  Moreover, while these investigators observed a significant prolongation of PFS in patients receiving molecularly matched therapy, 15 % of patients not receiving matched therapy were lost to follow-up, as they continued their treatment at a different oncology clinic or practice; thus, their outcomes were not evaluable.\n",
      "AMBLor Melanoma Prognostic Test\n",
      "Avero Diagnostics (Bellingham, WA and Irving, TX), a laboratory operations and pathology company, developed AMBLor, a laboratory test for the identification of early-stage melanoma at low risk of progression. The AMBLor test's technology can identify the presence of two prognostic biomarker proteins, AMBRA1 and loricrin, which are located on the surface of the skin above the tumor. Absence of both markers in the American Joint Committee on Cancer early-stage (stage I and II) melanoma is associated with tumors at normal or high risk of recurrence, whereas normal expression of one or both markers is associated with a low risk of progression (Avero Diagnostics, 2023).\n",
      "Currently, there is insufficient evidence in the peer-reviewed literature to support the sensitivity or specificity of this test.\n",
      "Grail Galleri Test\n",
      "Grail (Menlo Park, CA), a biotechnology company, developed the Galleri Test which utilizes a single blood test to check for more than 100,000 DNA regions and over a million specific DNA sites to screen for a signal shared by cancers that may otherwise go unnoticed. Specifically, the Galleri test looks for cell-free DNA and identifies whether the source is from healthy or cancer cells (Grail, 2023).\n",
      "Currently, there is insufficient evidence in the peer-reviewed literature to support the sensitivity or specificity of this test.\n",
      "InVisionFirst-Lung\n",
      "NeoGenomics (Fort Myers, FL) developed the InVisionFirst-Lung liquid biopsy assay which is a clinical diagnostic laboratory-developed test validated for the accurate identification of genomic alterations in 37 genes and is intended to aid clinicians in therapeutic decision making for patients diagnosed with advanced non-small cell lung cancer (NSCLC). This test utilizes proprietary enhanced tagged-amplicon next generation sequencing (eTAm-Seq) technology which can detect point mutations, indels, amplifications and gene fusions from plasma cell free DNA (cfDNA) that commonly occur in (NSCLC) (NeoGenomics, 2023).\n",
      "Plagnol, et al. (2018) conducted an analytical validation study for the InVisionFirst assay, a next generation sequencing liquid biopsy assay, for high sensitivity broad molecular profiling. For analytical validation, two 10 mL blood tubes were collected from NSCLC patients and health volunteer donors. In addition, contrived samples were used to represent a wide spectrum of genetic aberrations and variant allele fractions (VAFs). The investigators noted that the InVisionFirst assay demonstrated high sensitivity and specificity for the detection of single nucleotide variants, indels, ALK and ROS1 gene fusions, and DNA amplifications in ERBB2, FGFR1, MET and EGFR. Furthermore, a comparison between InVisionFirst assay and digital PCR in a series cancer patients showed high concordance.\n",
      "OncobiotaLUNG\n",
      "Micronoma (San Diego, CA), a biotech company, developed OncobiotaLUNG assay for early cancer detection with a microbiome-driven liquid biopsy platform. On January 10, the U.S. Food and Drug Administration (FDA) granted a breakthrough device designation for OncobiotaLUNG assay. The device categorizes lung nodule samples into those that are considered high or low-risk for malignancy based off of a blood draw from the patient. An analysis of proteins and circulating tumor microbial DNA (ctmbDNA) is performed and then this data is run through proprietary machine learning algorithms to report the findings to the healthcare provider (Micronoma, 2023).\n",
      "Currently, there is insufficient evidence in the peer-reviewed literature to support the sensitivity or specificity of this test.\n",
      "Pharmaco-oncologic Algorithmic Treatment Ranking Service\n",
      "CureMatch, Inc. (San Diego, CA), a leading company in precision medicine support for oncology, developed a therapy matching and scoring service that is a patient-specific, assistive, rules-based algorithm for ranking pharmaco-oncologic treatment options based on the patient's tumor-specific cancer marker information obtained from prior molecular pathology, immunohistochemical, or other pathology results which have been previously interpreted and reported separately (CureMatch, 2023).\n",
      "Currently, there is insufficient evidence in the peer-reviewed literature to support the sensitivity or specificity of this test.\n",
      "Strata Select\n",
      "Strata Oncology, Inc. (Ann Arbor, MI), a next-generation precision oncology company, developed Strat Select, a molecular profiling test for patients with advanced cancer which features the Immunotherapy Response Score, a novel multivariate predictive biomarker algorithm for PD-1/PD-L1 checkpoint inhibitor immunotherapy benefit. The Immunology Response Score integrates pertinent biological factors (TMB, PD-L1, PD-1, and tumor microenvironment components) into a simple score to predict immunotherapy benefit across all solid tumors. Strata Select analyzes DNA and RNA across 437 genes to provide guidance for genomic-alteration targeted therapies. The targeted RNA sequencing enables analysis of 950 fusion isoforms involving 59 primary driver genes (Strata Oncology, 2023).\n",
      "Currently, there is insufficient evidence in the peer-reviewed literature to support the sensitivity or specificity of this test.\n",
      "\n",
      "Glossary of Terms\n",
      "Table: Glossary of Terms\n",
      "\n",
      "\n",
      "Term\n",
      "Definition\n",
      "\n",
      "\n",
      "Biopsy specimen\n",
      "Tissue removed from the body and examined under a microscope to determine whether disease is present\n",
      "\n",
      "\n",
      "Germline mutation\n",
      "Inherited; can be passed onto offspring\n",
      "\n",
      "\n",
      "Liquid biopsy\n",
      "A noninvasive diagnostic approach involving the isolation of circulating tumor markers such as cell-free nucleic acids and circulating tumor cells from peripheral blood, thus, analyzing tumor DNA in blood called circulating tumor DNA (ctDNA)\n",
      "\n",
      "\n",
      "Mutation\n",
      "Any change in the DNA of a cell\n",
      "\n",
      "\n",
      "Oncogenes\n",
      "A gene which in certain circumstances can transform a cell into a tumor cell\n",
      "\n",
      "\n",
      "Phenotype\n",
      "The observable characteristics in an individual resulting from the expression of genes\n",
      "\n",
      "\n",
      "Serum tumor marker test\n",
      "A blood test that measures the amount of tumor markers (or biomarkers). Tumor markers are released into the blood by tumor cells or by other cells in response to tumor cells. A high level of a tumor marker may be a sign of cancer\n",
      "\n",
      "\n",
      "Somatic mutation\n",
      "Acquired; cannot be inherited by offspring\n",
      "\n",
      "\n",
      "Somatic genomic testing\n",
      "Looks for changes in the genes of cancer cells\n",
      "\n",
      "\n",
      "Tumor\n",
      "A mass formed when normal cells begin to change and grow uncontrollably. A tumor can be benign (noncancerous) or malignant (cancerous, meaning it can spread to other parts of the body). Also called a nodule or mass.\n",
      "\n",
      "\n",
      "Tumor suppressor genes\n",
      "A type of gene that makes a protein called a tumor suppressor protein that helps control cell growth. Mutations in tumor suppressor genes may lead to cancer. Also called antioncogene.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Appendix\n",
      "Table: Acronyms and Abbreviations\n",
      "\n",
      "\n",
      "Abbreviation\n",
      "Description\n",
      "\n",
      "\n",
      "a2-PAG\n",
      "Pregnancy-associated alpha2 glycoprotein\n",
      "\n",
      "\n",
      "BCM\n",
      "Breast cancer mucin\n",
      "\n",
      "\n",
      "BTA\n",
      "Bladder tumor antigen\n",
      "\n",
      "\n",
      "CA19-9\n",
      "Cancer antigen 19-9\n",
      "\n",
      "\n",
      "CA50\n",
      "Cancer antigen 50\n",
      "\n",
      "\n",
      "CA72-4\n",
      "Cancer antigen 72-4\n",
      "\n",
      "\n",
      "CA195\n",
      "Cancer antigen 195\n",
      "\n",
      "\n",
      "CA242\n",
      "Cancer antigen 242\n",
      "\n",
      "\n",
      "CA549\n",
      "Cancer antigen 549\n",
      "\n",
      "\n",
      "CA-SCC\n",
      "Squamous cell carcinoma\n",
      "\n",
      "\n",
      "CAM17-1\n",
      "Monoclonal antimucin antibody 17-1\n",
      "\n",
      "\n",
      "CAM26\n",
      "Monoclonal antimucin antibody 26\n",
      "\n",
      "\n",
      "CAM29\n",
      "Monoclonal antimucin antibody 29\n",
      "\n",
      "\n",
      "CAR3\n",
      "Antigenic determinant recognized by monoclonal antibody AR3\n",
      "\n",
      "\n",
      "DU-PAN-2\n",
      "Sialylated carbohydrate antigen DU-PAN-2\n",
      "\n",
      "\n",
      "FDP\n",
      "Fibrin/fibrinogen degradation products\n",
      "\n",
      "\n",
      "GCC\n",
      "Guanylyl cyclase C\n",
      "\n",
      "\n",
      "MCA\n",
      "Mucin-like carcinoma-associated antigen\n",
      "\n",
      "\n",
      "NMP22\n",
      "Nuclear matrix protein22\n",
      "\n",
      "\n",
      "NSE\n",
      "Neuron-specific enolase\n",
      "\n",
      "\n",
      "PLAP\n",
      "Placental alkaline phosphatase\n",
      "\n",
      "\n",
      "PNA-ELLA\n",
      "Peanut lectin-bonding assay\n",
      "\n",
      "\n",
      "SLEX\n",
      "Sialylated Lewis X-antigen\n",
      "\n",
      "\n",
      "SLX\n",
      "Sialylated SSEA-1 antigen\n",
      "\n",
      "\n",
      "SPAN-1\n",
      "Sialylated carbonated antigen SPAN-1\n",
      "\n",
      "\n",
      "ST-439\n",
      "Sialylated carbonated antigen ST-439\n",
      "\n",
      "\n",
      "TAG12\n",
      "Tumor-associated glycoprotein 12\n",
      "\n",
      "\n",
      "TAG72\n",
      "Tumor-associated glycoprotein 72\n",
      "\n",
      "\n",
      "TAG72.3\n",
      "Tumor-associated glycoprotein 72.3\n",
      "\n",
      "\n",
      "TATI\n",
      "Tumor-associated trypsin inhibitor\n",
      "\n",
      "\n",
      "TNF-a\n",
      "Tumor necrosis factor alpha\n",
      "\n",
      "\n",
      "TPA\n",
      "Tissue polypeptide antigen\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "References\n",
      "The above policy is based on the following references:\n",
      "Aaberg TM Jr, Cook RW, Oelschlager K, et al. Current clinical practice: Differential management of uveal melanoma in the era of molecular tumor analyses. Clin Ophthalmol. 2014;8:2449-2460.\n",
      "Abbosh PH, Plimack ER. Measuring the efficacy and value of urothelial cancer urinary biomarkers. Ann Intern Med. 2015;163(12):954-955.\n",
      "Abeloff MD, Armitage JO, Niederhuber JE, et al. Clinical Oncology. 3rd ed. Philadelphia, PA. Churchill Livingstone; 2004: 2369.\n",
      "Abida W, Patnaik A, Campbell D, et al.; TRITON2 investigators. Rucaparib in men with metastatic castration-resistant prostate cancer harboring a BRCA1 or BRCA2 gene alteration. J Clin Oncol. 2020;38(32):3763-3772.\n",
      "Abrams RA, Grochow LB, Chakravarthy A, et al. Intensified adjuvant therapy for pancreatic and periampullary adenocarcinoma: Survival results and observations regarding patterns of failure, radiotherapy dose and CA19-9 levels. Int J Radiat Oncol Biol Phys. 1999;44(5):1039-1046.\n",
      "Adams JM, Cory S. The Bcl-2 apoptotic switch in cancer development and therapy. Oncogene. 2007;26:1324-1337.\n",
      "Adelstein BA, Dobbins TA, Harris CA, et al. A systematic review and meta-analysis of KRAS status as the determinant of response to anti-EGFR antibodies and the impact of partner chemotherapy in metastatic colorectal cancer. Eur J Cancer. 2011;47(9):1343-1354.\n",
      "Aggarwal C, Thompson JC, Black TA, et al. Clinical implications of plasma-based genotyping with the delivery of personalized therapy in metastatic non-small cell lung cancer. JAMA Oncol. 2019;5(2):173-180.\n",
      "Aguado J. Identification of circulating tumour cells in metastatic breast cancer with the CellSearch (TM) system [summary]. Technological Report. Seville, Spain: Agencia de Evaluacion de Tecnologias Sanitarias de Andalucia (AETSA); 2006.\n",
      "Aguilera-Diaz A, Vazquez I, Ariceta B, et al. Assessment of the clinical utility of four NGS panels in myeloid malignancies. Suggestions for NGS panel choice or design. PLoS One. 2020 ;15(1):e0227986.\n",
      "Al Nasrallah N, Sears CR. Biomarkers in pulmonary nodule diagnosis. Is it time to put away the biopsy needle? Chest. 2018;154(3):467-468.\n",
      "Albain KS, Barlow WE, Shak S, et al; Breast Cancer Intergroup of North America. Prognostic and predictive value of the 21-gene recurrence score assay in postmenopausal women with node-positive, oestrogen-receptor-positive breast cancer on chemotherapy: A retrospective analysis of a randomised trial. Lancet Oncol. 2010;11(1):55-65.\n",
      "Albala D, Kemeter MJ, Febbo PG, et al. Health economic impact and prospective clinical utility of Oncotype DXÂ® Genomic Prostate Score. Rev Urol. 2016;18(3):123-132.\n",
      "Alberta Provincial Thoracic Tumour Team. Non-small cell lung cancer stage IV. Clinical Practice Guideline No. LU-004. Edmonton, AB: Alberta Health Services, Cancer Care; June 2011. \n",
      "Alberts SR, Yu T, Behrens RJ, et al. Real-world comparative economics of a 12-gene assay for prognosis in stage II colon cancer [abstract]. J Clin Oncol. 2012;30(suppl 34):Abstract 391.\n",
      "Al-Haddad M, DeWitt J, Sherman S, et al. Performance characteristics of molecular (DNA) analysis for the diagnosis of mucinous pancreatic cysts. Gastrointest Endosc. 2014;79(1):79-87.\n",
      "Al-Haddad MA, Kowalski T, Siddiqui A, et al. Integrated molecular pathology accurately determines the malignant potential of pancreatic cysts. Endoscopy. 2015;47(2):136-142.\n",
      "Allard WJ, Matera J, Miller MC, et al. Tumor cells circulate in the peripheral blood of all major carcinomas but not in healthy subjects or patients with nonmalignant diseases. Clin Cancer Res. 2004;10(20):6897-6904.\n",
      "Allegra CJ, Jessup JM, Somerfield MR, et al. American Society of Clinical Oncology provisional clinical opinion: Testing for KRAS gene mutations in patients with metastatic colorectal carcinoma to predict response to anti-epidermal growth factor receptor monoclonal antibody therapy. J Clin Oncol. 2009;27(12):2091-2096.\n",
      "Alvarado M, Carter DL, Guenther JM, et al. The impact of genomic testing on the recommendation for radiation therapy in patients with ductal carcinoma in situ: A prospective clinical utility assessment of the 12-gene DCIS scoreâ¢ result. J Surg Oncol. 2015;111(8):935-940.\n",
      "Alvarez JGB, Janse S, Owen DH, et al. Treatment of non-small-cell lung cancer based on circulating cell-free DNA and impact of variation allele frequency. Clin Lung Cancer. 2021;22(4):e519-e527.\n",
      "Amado RG, Wolf M, Peeters M, et al. Wild-type KRAS is required for panitumumab efficacy in patients with metastatic colorectal cancer. J Clin Oncol. 2008;26(10):1626-1634.\n",
      "American Cancer Society (ACS). Ovarian cancer has early symptoms. First national consensus on common warning signs. ACS News Center. Atlanta, GA: ACS; June 14, 2007. \n",
      "American College of Obstetricians and Gynecologists (ACOG). Evaluation and management of adnexal masses. ACOG Practice Bulletin No. 174. Washington, DC: ACOG; November 2016.\n",
      "American College of Obstetricians and Gynecologists (ACOG). Position of the American College of Obstetricians and Gynecologists Committee on Gynecologic Practice Regarding OvaCheck. Washington, DC: ACOG; February 25, 2004.\n",
      "American College of Obstetricians and Gynecologists (ACOG). The role of the generalist obstetrician-gynecologist in the early detection of ovarian cancer. ACOG Committee Opinion No. 280. Washington, DC: ACOG; December 2002.\n",
      "American College of Physicians. Screening for ovarian cancer: Recommendations and rationale. Ann Intern Med. 1994 J;121(2):141-142.\n",
      "American Medical Association (AMA). CPT proprietary laboratory analyses (PLA) codes: long descriptor. April 6, 2022. Available at: https://www.ama-assn.org/system/files/cpt-pla-codes-long.pdf. Accessed April 22, 2022.\n",
      "American Society of Clinical Oncology. 1997 update of recommendations for the use of tumor markers in breast and colorectal cancer. Adopted on November 7, 1997 by the American Society of Clinical Oncology. J Clin Oncol. 1998;16(2):793-795.\n",
      "American Society of Clinical Oncology. Clinical practice guidelines for the use of tumor markers in breast and colorectal cancer. Adopted on May 17, 1996 by the American Society of Clinical Oncology. J Clin Oncol. 1996;14(10):2843-2877.\n",
      "American Thyroid Association (ATA) Guidelines Taskforce on Thyroid Nodules and Differentiated Thyroid Cancer, Cooper DS, Doherty GM, Haugen BR, et al. Revised American Thyroid Association management guidelines for patients with thyroid nodules and differentiated thyroid cancer. Thyroid. 2009;19(11):1167-1214.\n",
      "Anceschi U, Tuderti G, Lugnani F, et al. Novel diagnostic biomarkers of prostate cancer: An update. Curr Med Chem. 2019;26(6):1045-1058.\n",
      "Anderson GG, Weiss LM. Determining tissue of origin for metastatic cancers: Meta-analysis and literature review of immunohistochemistry performance. Appl Immunohistochem Mol Morphol. 2010;18(1):3-8.\n",
      "Andre F, Ismaila N, Allison KH, et al. Biomarkers for adjuvant endocrine and chemotherapy in early-stage breast cancer: ASCO Guideline Update. J Clin Oncol. 2022;40(16):1816-1837.\n",
      "Andreopoulou E, Yee H, Warycha MA, et al. Mucinous cancer of the appendix: challenges in diagnosis and treatment. J Chemother. 2007;19(4):451-454.\n",
      "Andres G, Ashour N, Sanchez-Chapado M, et al. The study of DNA methylation in urological cancer: Present and future. Actas Urol Esp. 2013;37(6):368-375.\n",
      "Andrews E. Ovacheck: Breakthrough or also ran? Nat Rev Med. 2004;1(5).\n",
      "Angell TE, Wirth LJ, Cabanillas ME, et al. Analytical and clinical validation of expressed variants and fusions from the whole transcriptome of thyroid FNA samples. Front Endocrinol (Lausanne). 2019;10:612.\n",
      "Antonarakis ES, Armstrong AJ, Dehm SM, Luo J. Androgen receptor variant-driven prostate cancer: Clinical implications and therapeutic targeting. Prostate Cancer Prostatic Dis. 2016;19(3):231-241.\n",
      "Antonarakis ES, Lu C, Luber B, et al. Androgen receptor splice variant 7 and efficacy of taxane chemotherapy in patients with metastatic castration-resistant prostate cancer. JAMA Oncol. 2015;1(5):582-591.\n",
      "Antonarakis ES, Lu C, Wang H, et al. AR-V7 and resistance to enzalutamide and abiraterone in prostate cancer. N Engl J Med. 2014;371(11):1028-1038. \n",
      "Antonarakis ES, Tierno M, Fisher V, et al. Clinical and pathological features associated with circulating tumor DNA content in real-world patients with metastatic prostate cancer. Prostate. 2022;82(7):867-875.\n",
      "Aoyagi Y, Oguro M, Yanagi M, et al. Clinical significance of simultaneous determinations of alpha-fetoprotein and des-gamma-carboxy prothrombin in monitoring recurrence in patients with hepatocellular carcinoma. Cancer. 1996;77(9):1781-1786.\n",
      "Arisi MF, Dotan E, Fernandez SV. Circulating tumor DNA in precision oncology and its applications in colorectal cancer. Int J Mol Sci. 2022;23(8):4441.\n",
      "Armstrong AJ, Halabi S, Luo J, et al. Prospective multicenter validation of androgen receptor splice variant 7 and hormone Therapy resistance in high-risk castration-resistant prostate cancer: The PROPHECY study. J Clin Oncol. 2019;37(13):1120-1129.\n",
      "Arsenault D, Le Blanc R. Cellules tumorales circulantes dans les cas de cancer du sein: Utilisation clinique du test CellSearch. Note Informative [summary]. Quebec, QC: Institut national dâexcellence en santÃ© et services sociaux (INESSS); Septembre 2016. \n",
      "ASGE Standards of Practice Committee, Muthusamy VR, Chandrasekhara V, Acosta RD, et al. The role of endoscopy in the diagnosis and treatment of cystic pancreatic neoplasms. Gastrointest Endosc. 2016;84(1):1-9.\n",
      "Attard G, Parker C, Eeles RA, et al. Prostate cancer. Lancet. 2016;387(10013):70-82. \n",
      "Aubry W, Lieberthal R, Willis A, et al. Budget impact model: Epigenetic assay can help avoid unnecessary repeated prostate biopsies and reduce healthcare spending. Am Health Drug Benefits. 2013;6(1):15-24.\n",
      "Aureon Laboratories, Inc. Aureon Laboratories announces name change for existing post-surgical prostate cancer recurrence test commercially available since February 2006, Post-Op Px(TM) is the first diagnostic test to predict prostate cancer recurrence. GlobeNewswire. Nasdaz Omx. March 5, 2010, Available at: http://www.globenewswire.com/newsroom/news.html?d=185946. Accessed December 9, 2011.\n",
      "Australia and New Zealand Horizon Scanning Network (ANZHSN). Genesearchâ¢ breast lymph node (BLN) assay. Horizon Scanning Technology Prioritising Summaries. Canberra, ACT: Australian Government, Department of Health and Ageing; May 2008.  \n",
      "Avero Diagnostics. Avero Diagnostics launches melanoma progression test with AMLo. News Release. Avero Diagnostics. Februrary 2, 2023. Available at: https://usa.amlo-biosciences.com/avero-diagnostics-launches-melanoma-progression-test-with-amlo/. Accessed July 31, 2023.\n",
      "Aw A, Kim HT, Fernandes SM, et al. Minimal residual disease detected by immunoglobulin sequencing predicts CLL relapse more effectively than flow cytometry. Leuk Lymphoma. 2018;59(8):1986-1989.\n",
      "Azim HA Jr, Michiels S, Zagouri F, et al. Utility of prognostic genomic tests in breast cancer practice: The IMPAKT 2012 Working Group Consensus Statement. Ann Oncol. 2013;24(3):647-654.\n",
      "Badani K, Thompson DJ, Buerki C, et al. Impact of a genomic classifier of metastatic risk on postoperative treatment recommendations for prostate cancer patients: A report from the DECIDE study group. Oncotarget. 2013;4(4):600-609.\n",
      "Badani KK, Kemeter MJ, Febbo PG, et al. The impact of a biopsy based 17-gene Genomic Prostate Score on treatment recommendations in men with newly diagnosed clinically prostate cancer who are candidates for active surveillance. Urology Practice. 2015;2(4):181-189.\n",
      "Badani KK, Thompson DJ, Brown G, et al. Effect of a genomic classifier test on clinical practice decisions for patients with high-risk prostate cancer after surgery. BJU Int. 2015;115(3):419-429.\n",
      "Baek DH, Kim GH, Song GA, et al. Clinical potential of circulating tumor cells in colorectal cancer: A prospective study. Clin Transl Gastroenterol. 2019;10(7): e00055.\n",
      "Baker M, Gillanders WE, Mikhitarian K, et al. The molecular detection of micrometastatic breast cancer. Am J Surg. 2003;186(4):351-358.\n",
      "Balagopal V, Hantel A, Kadri S, et al. Measurable residual disease monitoring for patients with acute myeloid leukemia following hematopoietic cell transplantation using error corrected hybrid capture next generation sequencing. PLoS One. 2019;14(10):e0224097.\n",
      "Banavar G, Ogundijo O, Toma R, et al. The salivary metatranscriptome as an accurate diagnostic indicator of oral cancer. NPJ Genom Med. 2021;6(1):105. \n",
      "Bareche Y, Venet D, Ignatiadis M, et al. Unravelling triple-negative breast cancer molecular heterogeneity using an integrative multiomic analysis. Ann Oncol. 2018;29(4):895-902.\n",
      "Baron TH, Mallery JS, Hirota WK, et al. The role of endoscopy in the evaluation and treatment of patients with pancreaticobiliary malignancy. Gastrointest Endosc. 2003;58(5):643-649.\n",
      "Barresi L, Tarantino I, Ligresti D, et al. Pancreatic cystic lesions: The challenge of risk stratification for malignancy. Endoscopy. 2015;47(6):562.\n",
      "Bartlett JM, Bayani J, Marshall A, et al; OPTIMA TMG. Comparing breast cancer multiparameter tests in the OPTIMA prelim trial: No test is more equal than the others. J Natl Cancer Inst. 2016;108(9).\n",
      "Bartlett JM, Thomas J, Ross DT, et al. Mammostrat as a tool to stratify breast cancer patients at risk of recurrence during endocrine therapy. Breast Cancer Res. 2010;12(4):R47.\n",
      "Bast RC Jr, Badgwell D, Lu Z, et al. New tumor markers: CA125 and beyond. Int J Gynecol Cancer. 2005;15 Suppl 3:274-281.\n",
      "Bast RC Jr, Brewer M, Zou C, et al. Prevention and early detection of ovarian cancer: Mission impossible? Recent Results Cancer Res. 2007;174:91-100. \n",
      "Bast RC Jr, Hortobagyi GN. Individualized care for patients with cancer - a work in progress. N Engl J Med. 2004;351(27):2865-2867.\n",
      "Bast RC Jr, Ravdin P, Hayes DF, et al. 2000 update of recommendations for the use of tumor markers in breast and colorectal cancer: Clinical practice guidelines of the American Society of Clinical Oncology. J Clin Oncol. 2001;19(6):1865-1878.\n",
      "Bastos DA, Antonarakis ES. AR-V7 and treatment selection in advanced prostate cancer: are we there yet? Precis Cancer Med. 2018 Sep;1. pii: 13. \n",
      "Bates SE. Clinical Applications of serum tumor markers. Ann Intern Med. 1991;115(8):623-638.\n",
      "Bekelman JE, Rumble RB, Chen RC, et al. Clinically localized prostate cancer: ASCO Clinical Practice Guideline endorsement of an American Urological Association/American Society for Radiation Oncology/Society of Urologic Oncology guideline. J Clin Oncol. 2018;36(32):3251-3258.\n",
      "Benidir T, Hofmann M, Lone Z, et al. Elevated IsoPSA selects for clinically significant prostate cancer without a preference for any particular adverse histopathologic or radiographic feature. Urology. 2022;168:150-155.\n",
      "Benjamin H, Schnitzer-Perlman T, Shtabsky A, et al. Analytical validity of a microRNA-based assay for diagnosing indeterminate thyroid FNA smears from routinely prepared cytology slides. Cancer Cytopathol. 2016;124(10):711-721.\n",
      "Benway BM, Andriole GL. Prostate biopsy. UpToDate [online serial]. Waltham, MA: UpToDate; reviewed September 2021.\n",
      "Berchuck A; Society for Gynecologic Oncologists. Ovarian cancer symptoms consensus statement. Chicago, IL: Society for Gynecologic Oncologists; 2007. Available at: http://www.sgo.org/publications/OvarianCancerSymptoms.pdf. Accessed July 6, 2007.\n",
      "Berger AC, Davidson RS, Poitras JK, et al. Clinical impact of a 31-gene expression profile test for cutaneous melanoma in 156 prospectively and consecutively tested patients. Curr Med Res Opin. 2016;32(9):1599-1604.\n",
      "Berger BM, Hanna GJ, Posner MR, et al. Detection of occult recurrence using circulating tumor tissue modified viral HPV DNA among patients treated for HPV-driven oropharyngeal carcinoma. Clin Cancer Res. 2022;28(19):4292-4301.\n",
      "Berlin A, Murgic J, Hosni A, et al. Genomic classifier for guiding treatment of intermediate-risk prostate cancers to dose-escalated image guided radiation therapy without hormone therapy. Int J Radiat Oncol Biol Phys. 2019;103(1):84-91.\n",
      "Bernemann C, Schnoeller TJ, Luedeke M, et al. Expression of AR-V7 in circulating tumour cells does not preclude response to next generation androgen deprivation therapy in patients with castration resistant prostate cancer. Eur Urol. 2017;71(1):1-3. \n",
      "Besse B, Olaussen KA, Soria JC. ERCC1 and RRM1: Ready for prime time? J Clin Oncol. 2013;31(8):1050-1060.\n",
      "Bhatia P, Deniwar A, Friedlander P, et al. Diagnostic potential of ancillary molecular testing in differentiation of benign and malignant thyroid nodules. Anticancer Res. 2015;35(3):1237-1241.\n",
      "Bianchini G, Gianni L. An unmet need: Tailoring extended adjuvant endocrine therapy. Br J Cancer. 2013;109(12):2951-2953.\n",
      "Biol Phys. 2014;89(5):1038-1046.\n",
      "Blue Cross Blue Shield Association (BCBSA), Technology Evaluation Center (TEC). Use of genesearch breast lymph node assay to detect sentinel node metastases in early stage breast cancer. TEC Assessment Program. Chicago, IL:BCBSA; 2007; 22(8). \n",
      "Blue Cross Blue Shield Association (BCBSA), Technology Evaluation Center (TEC). KRAS mutations and epidermal growth factor receptor inhibitor therapy in metastatic colorectal cancer. TEC Assessment Program. Chicago, IL: BCBSA; January 2009;25(6). \n",
      "BlueCross BlueShield Association (BCBSA), Technology Evaluation Center (TEC). Special Report: Pharmacogenomics of cancer-candidate genes.TEC Assessment Program. Chicago, IL: BCBSA; November 2007;22(5). \n",
      "BlueCross BlueShield Association (BCBSA), Technology Evaluation Center (TEC). Special report: Recent developments in prostate cancer genetics and genetic testing. TEC Assessments in Press. Chicago, IL: BCBSA; September 2008.\n",
      "BlueCross BlueShield Association (BCBSA), Technology Evaluation Center (TEC). Gene expression profiling for women with lymph-node-positive breast cancer to select adjuvant chemotherapy treatment. TEC Assessment Program. Chicago, IL: BCBSA; November 2010;25(1).\n",
      "BlueCross BlueShield Association (BCBSA), Technology Evaluation Center (TEC). Special report: Multiple molecular testing of cancers to identify targeted therapies. TEC Assessments in Press. Chicago, IL: BCBSA; February 2013.\n",
      "BlueCross BlueShield Association (BCBSA), Technology Evaluation Center (TEC). Actions Taken by the BlueCross BlueShield Association Medical Advisory Panel (MAP). Chicago, IL: BCBSA; February 19, 2014.\n",
      "BlueCross BlueShield Association (BCBSA), Technology Evaluation Center (TEC). Gene expression analysis for prostate cancer management. TEC Assessment Program. Chicago, IL: BCBSA; April 2014;28(11).\n",
      "BlueCross BlueShield Association (BCBSA), Technology Evaluation Center (TEC). Mutianalyte testing for evaluation of adnexal masses. TEC Assessment Program. Chicago, IL: BCBSA; April 2013;27(8).\n",
      "BlueCross BlueShield Association (BCBSA). Gene expression analysis for prostate cancer management. TEC Assessment. Chicago, IL: BCBSA; January 2015; 29(9).\n",
      "BlueCross BlueShield Association (BCBSA). Gene expression profiling in women with lymph node-negative breast cancer to select adjuvant chemotherapy. TEC Assessment. Chicago, IL: BCBSA; October 2014;29(3).\n",
      "BlueCross BlueShield Association (BCBSA). Technology Evaluation Center (TEC). Multi-analyte testing for the evaluation of adnexal masses. TEC Assessment Program. Chicago, IL: BCBSA; 2013. \n",
      "Blum ES, Yang J, Komatsubara KM, Carvajal RD. Clinical management of uveal and conjunctival melanoma. Oncology (Williston Park). 2016;30(1):29-32, 34-43, 48.\n",
      "Blumencranz P, Whitworth PW, Deck K, et al. Scientific Impact Recognition Award. Sentinel node staging for breast cancer: Intraoperative molecular pathology overcomes conventional histologic sampling errors. Am J Surg. 2007; 194(4):426-432.\n",
      "Bodei L, Kidd M, Modlin IM, et al. Measurement of circulating transcripts and gene cluster analysis predicts and defines therapeutic efficacy of peptide receptor radionuclide therapy (PRRT) in neuroendocrine tumors. Eur J Nucl Med Mol Imaging. 2016;43(5):839-851.\n",
      "Boerrigter E, Groen LN, van Erp NP, et al. Clinical utility of emerging biomarkers in prostate cancer liquid biopsies. Expert Rev Mol Diagn. 2020;20(2):219-230.\n",
      "Boily G, Boulanger J, Goulet S, et al. Algorithmes dâinvestigation, de traitement et de suivi Cancer du cÃ´lon (mise Ã  jour). Quebec, QC: Institut national dâexcellence en santÃ© et en services sociaux (INESSS) et Groupe dâÃ©tude en oncologie du QuÃ©bec (GEOQ); May 2016.\n",
      "Bokemeyer C, Bondarenko I, Hartmann J, et al. KRAS status and efficacy of first-line treatment of patients with metastatic colorectal cancer (mCRC) with FOLFOX with or without cetuximab: The OPUS experience. J Clin Oncol. 2008;26:(May 20 Suppl; Abstr. 4000). \n",
      "Borg A, Bergh J, Kirstoffersson U. Genprofiltest vid brÃ¶stcancer. [Gene expression profiling for breast cancer.] SBU Kommenterar. Publicerad 2010-10-26. Stockholm, Sweden: Swedish Office of Health Technology Assessment (SBU); 2010.\n",
      "Bos JL. Ras oncogenes in human cancer: A review. Cancer Res. 1989;49(17):4682-4689.\n",
      "BostonGene. BostonGene Tumor Portrait Tests. Waltham, MA: BostonGene. Available at: https://bostongene.com/clinical-solutions/bg-portrait-tests/. Accessed July 8, 2022.\n",
      "BostrÃ¶m PJ, Bjartell AS, Catto JW, et al. Genomic predictors of outcome in prostate cancer. Eur Urol. 2015;68(6):1033-1044.\n",
      "Bournaud C, Descotes F, Decaussin-Petrucci M, et al. TERT promoter mutations identify a high-risk group in metastasis-free advanced thyroid carcinoma. Eur J Cancer. 2019;108:41-49.\n",
      "Boyle P, Chapman CJ, Holdenrieder S, et al. Clinical validation of an autoantibody test for lung cancer. Ann Oncol. 2011;22(2):383-389.\n",
      "Bradley LA, Palomaki G, Gutman S, et al. PCA3 testing for the diagnosis and management of prostate cancer. Comparative Effectiveness Review No. 98. Prepared by the Blue Cross and Blue Shield Technology Evaluation Center Evidence-based Practice Center under Contract No. 290-2007-10058-I). AHRQ Publication No. 13-EHC030-EF. Rockville, MD: Agency for Healthcare Research and Quality (AHRQ); April 2013.\n",
      "Brand RE, Persson J, Bratlie SO, et al. Detection of early-stage pancreatic ductal adenocarcinoma from blood samples: Results of a multiplex biomarker signature validation study. Clin Transl Gastroenterol. 2022;13(3):e00468.\n",
      "Brand TC, Zhang N, Crager MR, et al. Patient-specific meta-analysis of 2 clinical validation studies to predict pathologic outcomes in prostate cancer using the 17-gene genomic prostate score. Urology. 2016;89:69-75.\n",
      "Bratman SV, Yang SYC, Iafolla MAJ, et al. Personalized circulating tumor DNA analysis as a predictive biomarker in solid tumor patients treated with pembrolizumab. Nat Cancer. 2020;1:873-881.\n",
      "Bristow RE, Hodeib M, Smith A, et al. Impact of a multivariate index assay on referral patterns for surgical management of an adnexal mass. Am J Obstet Gynecol. 2013;209(6):581.e1-e8.\n",
      "Bristow RE, Smith A, Zhang Z, et al. Ovarian malignancy risk stratification of the adnexal mass using a multivariate index assay. Gynecol Oncol. 2013;128(2):252-259.\n",
      "British Committee for Standards in Haematology, Milligan DW, Grimwade D, Cullis JO, et al. Guidelines on the management of acute myeloid leukaemia in adults. Br J Haematol. 2006;135(4):450-474.\n",
      "Brodsky BS, Owens GM, Scotti DJ, et al. Economic impact of increased utilization of multivariate assay testing to guide the treatment of ovarian cancer: Implications for payers. Am Health Drug Benefits. 2017;10(7):351-359.\n",
      "Brostjan C, Gebhardt K, Gruenberger B, et al. Neoadjuvant treatment of colorectal cancer with bevacizumab: The perioperative angiogenic balance is sensitive to systemic thrombospondin-1 levels. Clin Cancer Res. 2008;14(7):2065-2074.\n",
      "Brown FM. Urine cytology. Is it still the gold standard for screening? Urol Clin North Am. 2000;27(1):25-37.\n",
      "Brownstone N, Zakria D, Dirr MA, et al. Integrating genomic testing for melanoma into your practice. Skin. 2022;6(3):189-194.\n",
      "Bryant RJ, Sjoberg DD, Vickers AJ, et al. Predicting high-grade cancer at ten-core prostate biopsy using four kallikrein markers measured in blood in the ProtecT study. J Natl Cancer Inst. 2015;107(7).\n",
      "Bubendorf L, Grilli B, Sauter G, et al. Multiprobe FISH for enhanced detection of bladder cancer in voided urine specimens and bladder washings. Am J Clin Pathol. 2001;116(1):79-86.\n",
      "Bulten BF, Verberne HJ, Bellersen L, et al. Relationship of promising methods in the detection of anthracycline-induced cardiotoxicity in breast cancer patients. Cancer Chemother Pharmacol. 2015;76(5):957-967.\n",
      "Burat B, Gonzalez J, Sauvage FL, et al. Sum of peak intensities outperforms peak area integration in iTRAQ protein expression measurement by LC-MS/MS using a TripleTOF 5600+ platform. Biosci Rep. 2019;39(6):BSR20190904.\n",
      "Burchardt M, Burchardt T, Shabsigh A, et al. Current concepts in biomarker technology for bladder cancer. Clin Chem. 2000;46(5):595-605.\n",
      "Burstein HJ, Paik S, Ravdin PM, Albain KS. Adjuvant chemotherapy for patients with estrogen receptor-positive breast cancer. 2006 ASCO Educational Sessions. 2006:49-55.\n",
      "Burstein HJ, Temin S, Anderson H, et al. Adjuvant endocrine therapy for women with hormone receptor-positive breast cancer: American Society of Clinical Oncology clinical practice guideline focused update. J Clin Oncol. 2014;32(21):2255-2269\n",
      "Bussemakers MJG, van Bokhoven A, Verheagh GW, et al. DD3: A new prostate-specific gene, highly overexpressed in prostate cancer. Cancer Res. 1999;59:5975-5979.\n",
      "Butts CA. VeriStrat validated in patients with non-small-cell lung cancer. Lancet Oncol. 2014;15(7):671-672.\n",
      "California Technology Assessment Forum (CTAF). Gene expression profiling as a guide for the management of early stage breast cancer. Technology Assessment. San Francisco, CA: CTAF; 2007. \n",
      "Callesen LB, Hamfjord J, Boysen AK, et al. Circulating tumour DNA and its clinical utility in predicting treatment response or survival in patients with metastatic colorectal cancer: A systematic review and meta-analysis. Br J Cancer. 2022;127(3):500-513.\n",
      "Campesato LF, Barroso-Sousa R, Jimenez L, et al. Comprehensive cancer-gene panels can be used to estimate mutational load and predict clinical benefit to PD-1 blockade in clinical practice. Oncotarget. 2015;6(33):34221-34227. \n",
      "Canadian Agency for Drugs and Technologies in Health (CADTH). The âCellSearchâ System for Detecting Circulating Tumour Cells in Advanced Ovarian Cancer: Clinical Benefit and Cost-Effectiveness. Rapid Response Report. Ottawa, ON: CADTH; November 16, 2012.\n",
      "Canfield S, Kemeter MJ, Febbo PG, Hornberger J. Balancing confounding and generalizability using observational, real-world data: 17-gene genomic prostate score assay effect on active surveillance. Rev Urol. 2018;20(2):69-76.\n",
      "Canfield S, Kemeter MJ, Hornberger J, Febbo PG. Active surveillance use among a low-risk prostate cancer population in a large US payer system: 17-gene genomic prostate score versus other risk stratification methods. Rev Urol. 2018;19(4):203-212.\n",
      "Carbonell D, SuÃ¡rez-GonzÃ¡lez J, Chicano M, et al. Next-generation sequencing improves diagnosis, prognosis and clinical management of myeloid neoplasms. Cancers (Basel). 2019;11(9).\n",
      "Cardoso F, vanât Veer LJ, Bogaerts J, et al; MINDACT Investigators. 70-Gene signature as an aid to treatment decisions in early-stage breast cancer. N Engl J Med 2016;375(8):717-729.\n",
      "Carlson KJ. Screening for ovarian cancer. Ann Intern Med. 1994;121(2):124-132\n",
      "Carlson MR. Previstage GCC colorectal cancer staging test: A new molecular test to identify lymph node metastases and provide more accurate information about the stage of patients with colorectal cancer. Mol Diagn Ther. 2009;13(1):11-14.\n",
      "Carlsson SV, Roobol MJ. Improving the evaluation and diagnosis of clinically significant prostate cancer in 2017. Curr Opin Urol. 2017;27(3):198-204.\n",
      "Carmignani CP, Hampton R, Sugarbaker CE, et al. Utility of CEA and CA 19-9 tumor markers in diagnosis and prognostic assessment of mucinous epithelial cancers of the appendix. J Surg Oncol. 2004b;87(4):162-166.\n",
      "Carmignani CP, Sugarbaker PH. Synchronous extraperitoneal and intraperitoneal dissemination of appendix cancer. Eur J Surg Oncol. 2004a;30(8):864-868.\n",
      "Cassarino DS, Lewine N, Cole D, et al. Budget impact analysis of a novel gene expression assay for the diagnosis of malignant melanoma. J Med Econ. 2014;17(11):782-791.\n",
      "Castillo SA, Pham AK, Barton DT, et al. A diagnostically-challenging case of melanoma ex blue nevus with comprehensive molecular analysis, including the 23-gene expression signature (myPath melanoma). J Cutan Pathol. 2019;46(3):226-230.\n",
      "Castle Biosciences. Products. Phoenix, AZ: Castle Biosciences. Available at: https://castlebiosciences.com/. Accessed July 8, 2022.\n",
      "Castro MA, Dal-Pizzol F, Zdanov S, et al. CFL1 expression levels as a prognostic and drug resistance marker in nonsmall cell lung cancer. Cancer. 2010;116(15):3645-3655.\n",
      "Centers for Medicare & Medicare Services (CMS). Local Coverage Determination (LCD): MolDX: Breast Cancer Indexâ  Genetic Assay (L35294). Palmetto GBA â MAC Part B. Medicare Coverage Database. Baltimore, MD: CMS; effective November 3, 2014.\n",
      "Chakravarty D, Johnson A, Sklar J, et al. Somatic Genomic Testing in Patients With Metastatic or Advanced Cancer: ASCO Provisional Clinical Opinion. J Clin Oncol. 2022;40(11):1231-1258.\n",
      "Chakravatri A, Zehr EM, Zietman AL, et al. Thymosin beta-15 predicts for distant failure in patients with clinically localized prostate cancer-results from a pilot study. Urology. 2000;55(5):635-638. \n",
      "Chalmers ZR, Connelly CF, Fabrizio D, et al. Analysis of 100,000 human cancer genomes reveals the landscape of tumor mutational burden. Genome Med. 2017;9(1):34.\n",
      "Chang M, Ismaila N, Kamel-Reid S, et al. Comparison of Oncotype DX with Multi-gene Profiling Assays, (e.g., MammaPrint, PAM50) and Other Tests (e.g., Adjuvant! Online, Ki-67 and IHC4) in Early-stage Breast Cancer. Recommendation Report MOAC-2. Edmonton, ON: Program in Evidence-Based Care, Cancer Care Ontario; November 20, 2013.\n",
      "Chang MC, Souter LH, Kamel-Reid S, et al. Clinical utility of multigene profiling assays in early-stage breast cancer. Program in Evidence-Based Care Recommendation Report No.: MOAC-4. Toronto, ON: Cancer Care Ontario; June 8, 2016.\n",
      "Chappell MC, Char DH, Cole TB, et al. Uveal melanoma: Molecular pattern, clinical features, and radiation response. Am J Ophthalmol. 2012;154(2):227-232.e2.\n",
      "Check E. Proteomics and cancer: Running before we can walk? Nature. 2004;429(6991):496-497.\n",
      "Chen L, Chen M, Chen J. [Advances of circulating biomarkers in gastroenteropancreatic neuroendocrine neoplasms]. Zhonghua Wei Chang Wai Ke Za Zhi. 2017;20(3):357-360.\n",
      "Chen L-M, Berek JS. Endometrial carcinoma: Clinical features and diagnosis. UpToDate [online serial]. Waltham, MA: UpToDate; reviewed January 2015.\n",
      "Chen R, Pagano I, Sun Y, et al. A diagnostic gene expression signature for bladder cancer can stratify cases into prescribed molecular subtypes and predict outcome. Diagnostics (Basel). 2022 5;12(8):1801.\n",
      "Chen RC, Rumble RB, Loblaw DA, et al. Active Surveillance for the Management of Localized Prostate Cancer (Cancer Care Ontario Guideline): American Society of Clinical Oncology Clinical Practice Guideline Endorsement. J Clin Oncol. 2016;34(18):2182-2190.\n",
      "Chen X, Li J, Gray WH, et al. TNBCtype: A subtyping tool for triple-negative breast cancer. Cancer Inform. 2012;11:147-156.\n",
      "Cheng HH, Nelson PS. Genetic risk factors for prostate cancer. UpToDate [online serial]. Waltham, MA: UpToDate; reviewed July 2022. \n",
      "Cheng S, Qu K, Abdool A, et al. A molecular diagnostic panel for thyroid cancer disease management. J Clin Oncol 30, 2012 (suppl; abstr 5510). Available at: http://meetinglibrary.asco.org/content/94089-114. Accessed April 1, 2014.\n",
      "Chera BS, Kumar S, Beaty BT, et al. Rapid clearance profile of plasma circulating tumor HPV type 16 DNA during chemoradiotherapy correlates with disease control in HPV-associated oropharyngeal cancer. Clin Cancer Res. 2019;25(15):4682-4690.\n",
      "Chera BS, Kumar S, Shen C, et al. Plasma circulating tumor HPV DNA for the surveillance of cancer recurrence in HPV-associated oropharyngeal cancer. J Clin Oncol. 2020;38(10):1050-1058.\n",
      "Cherchi PL, Dessole S, Ruiu GA, et al. The value of serum CA 125 and association CA 125/CA 19-9 in endometrial carcinoma. Eur J Gynaecol Oncol. 1999;20(4):315-317.\n",
      "Chiam K, Ricciardelli C, Bianco-Miotto T. Epigenetic biomarkers in prostate cancer: Current and future uses. Cancer Lett. 2014;342(2):248-256.\n",
      "Chou R, Gore JL, Buckley D, et al. Urinary biomarkers for diagnosis of bladder cancer: A systematic review and meta-analysis. Ann Intern Med. 2015;163(12):922-931.\n",
      "Chung CH, Zhang Q, Kong CS, et al. p16 protein expression and human papillomavirus status as prognostic biomarkers of nonoropharyngeal head and neck squamous cell carcinoma. J Clin Oncol. 2014;32(35):3930-3938.\n",
      "Chung JH, Pavlick D, Hartmaier R, et al. Hybrid capture-based genomic profiling of circulating tumor DNA from patients with estrogen receptor-positive metastatic breast cancer. Ann Oncol. 2017;28(11):2866-2873.\n",
      "Cibas EX, Ali SZ. The 2017 Bethesda System for reporting thyroid cytopathology. Thyroid. 2017;27(11):1341-1346.\n",
      "Ciccarese C, Santoni M, Brunelli M, et al. AR-V7 and prostate cancer: The watershed for treatment selection? Cancer Treat Rev. 2016;43:27-35.\n",
      "Clark TA, Chung JH, Kennedy M, et al. Analytical validation of a hybrid capture-based next-generation sequencing clinical assay for genomic profiling of cell-free circulating tumor DNA. J Mol Diagn. 2018;20(5):686-702.\n",
      "Clarke LE, Flake DD, Busam K, et al. An independent validation of a gene expression signature to differentiate malignant melanoma from benign melanocytic nevi. Cancer. 2017;123(4):617-628.\n",
      "Clarke LE, Mabey B, Flake II DD, et al. Clinical validity of a gene expression signature in diagnostically uncertain neoplasms. Pers Med. 2020;17(5):361-371.\n",
      "Clarke LE, Warf MB, Flake DD, et al. Clinical validation of a gene expression signature that differentiates benign nevi from malignant melanoma. J Cutan Pathol. 2015;42(4):244-252.\n",
      "Cleveland Diagnostics, Inc. IsoPSA: Proceed to biopsy only when needed [website]. Cleveland, OH: ClevelandDx; 2022. Available at: https://www.clevelanddx.com/isopsa-testing. Accessed November 2, 2022.\n",
      "Coates AS, Winer EP, Goldhirsch A, et al.; Panel Members. Tailoring therapies-improving the management of early breast cancer: St Gallen International Expert Consensus on the Primary Therapy of Early Breast Cancer 2015. Ann Oncol. 2015;26(8):1533-1546.\n",
      "Cocco E, Bellone S, El-Sahwi K, et al. Serum amyloid A: A novel biomarker for endometrial cancer. Cancer. 2010;116(4):843-851.\n",
      "Cohen SJ, Punt CJ, Iannotti N, et al. Relationship of circulating tumor cells to tumor response, progression-free survival, and overall survival in patients with metastatic colorectal cancer. J Clin Oncol. 2008;26(19):3213-3221.\n",
      "Coleman RL, Herzog TJ, Chan DW, et al. Validation of a second-generation multivariate index assay for malignancy risk of adnexal masses. Am J Obstet Gynecol. 2016;215(1):82.e1-82.e11.\n",
      "Colevas AD. HPV DNA as a biomarker in oropharyngeal cancer: A step in the right direction. Clin Cancer Res. 2022;28(19):4171-4172.\n",
      "Collins LC, Laronga C, Wong JS. Ductal carcinoma in situ: Treatment and prognosis. UpToDate [online serial]. Waltham, MA: UpToDate; reviewed February 2013. \n",
      "Compton CC. Pathology and prognostic determinants of colorectal cancer. UpToDate [online serial]. Waltham, MA: UpToDate; 2008.\n",
      "Cook RW, Middlebrook B, Wilkinson J, et al. Analytic validity of DecisionDx-Melanoma, a gene expression profile test for determining metastatic risk in melanoma patients. Diagn Pathol. 2018;13(1):13.\n",
      "Coombes RC, Page K, Salari R, et al. Personalized detection of circulating tumor DNA antedates breast cancer metastatic recurrence. Clin Cancer Res. 2019;25(14):4255-4263.\n",
      "Cooperberg MR, Davicioni E, Crisan A, et al. Combined value of validated clinical and genomic risk stratification tools for predicting prostate cancer mortality in a high-risk prostatectomy cohort. Eur Urol. 2015;67(2):326-333.\n",
      "Coppin L, Benomar K, Corfiotti F, et al. CA-125, but not galectin-3, complements CA 19-9 for discriminating ductal adenocarcinoma versus non-malignant pancreatic diseases. Pancreatology. 2016;16(1):115-120.\n",
      "Coradini D, Daidone MG. Biomolecular prognostic factors in breast cancer. Curr Opin Obstet Gynecol. 2004;16(1):49-55.\n",
      "CorrÃªa ZM, Augsburger JJ. Independent prognostic significance of gene expression profile class and largest basal diameter of posterior uveal melanomas. Am J Ophthalmol. 2016;162:20-27.\n",
      "Correa ZM, Augsburger JJ. Sufficiency of FNAB aspirates of posterior uveal melanoma for cytologic versus GEP classification in 159 patients, and relative prognostic significance of these classifications. Graefes Arch Clin Exp Ophthalmol. 2014;252(1):131-135. \n",
      "Coveney EC, Geraghty JG, Sherry F, et al. The clinical value of CEA and CA 15-3 in breast cancer management. Int J Biol Markers. 1995;10(1):35-41.\n",
      "Cowling T, Loshak H. An overview of liquid biopsy in screening and early detection of cancer. Issues in Emerging Health Technologies. Issue 179. Ottawa, ON: Canadian Agency for Drugs and Technologies in Health (CADTH); November 2019.\n",
      "Cristofanilli M, Budd GT, Ellis MJ, et al. Circulating tumor cells, disease progression, and survival in metastatic breast cancer. N Engl J Med. 2004;351(8):781-791.\n",
      "Cuadros Celorrio M, Llanos MÃ©ndez A. MicroRNAs como herramienta diagnÃ³stica en el cÃ¡ncer de pulmÃ³n. [MicroRNAs as a diagnostic tool for lung cancer], Summary. AETSA 2010/2-3. Seville, Spain: Andalusian Agency for Health Technology Assessment (AETSA); 2012\n",
      "Cui W, Milner-Watts C, O'Sullivan H, et al. Up-front cell-free DNA next generation sequencing improves target identification in UK first line advanced non-small cell lung cancer (NSCLC) patients. Eur J Cancer. 2022;171:44-54.\n",
      "Cullen J, Rosner IL, Brand TC, et al. A biopsy-based 17-gene Genomic Prostate Score predicts recurrence after radical prostatectomy and adverse surgical pathology in a racially diverse population of men with clinically low- and intermediate-risk prostate cancer.  Eur Urol. 2014;68(1):123-131.\n",
      "CureMatch, Inc. CureMatch granted CPT code by American Medical Association for its Algorithmic Cancer Therapy Ranking Service. Press Release. San Diego, CA: CueMatch; Januarey 4, 2023. Available at: https://www.curematch.com/press/curematch-granted-cpt-code-by-american-medical-association-for-its-algorithmic-cancer-therapy-ranking-service/. Accessed July 31, 2023.\n",
      "ÄwikÅa JB, Bodei L, Kolasinska-ÄwikÅa A, et al. Circulating transcript analysis (NETest) in GEP-NETs treated with somatostatin analogs defines therapy. J Clin Endocrinol Metab. 2015;100(11):E1437-E1445. \n",
      "da Costa JB, Gibb EA, Bivalacqua TJ, et al. Molecular characterization of neuroendocrine-like bladder cancer. Clin Cancer Res. 2019;25(13):3908-3920.\n",
      "Dagan R, Amdur RJ, Dziegielewski PT. Tumors of the nasal cavity. UpToDate [online serial]. Waltham, MA: UpToDate; reviewed February 2021.\n",
      "Dalela D, Santiago-JimÃ©nez M, Yousefi K, et al. Genomic classifier augments the role of pathological fatures in identifying optimal candidates for adjuvant radiation therapy in patients with prostate cancer: Development and internal validation of a multivariable prognostic model. J Clin Oncol. 2017;35(18):1982-1990.\n",
      "Dalerba P, Sahoo D, Paik S, et al. CDX2 as a prognostic biomarker in stage II and stage III colon cancer. N Engl J Med. 2016;374(3):211-222.\n",
      "DallâEra MA, Maddala T, Polychronopoulos L, et al.. Utility of the Oncotype DXÂ® prostate cancer assay in clinical practice for treatment selection in men newly diagnosed with prostate cancer: A retrospective chart review analysis. Urology Practice. 2015;2:343-348.\n",
      "Dall'Era M, Michalopoulos SN, Denes BS, et al. Clinical benefits and costs of a 17-gene assay to assess risk of adverse pathology after positive prostate biopsy. J Urol. 2015;193(4 Suppl):e2-e3.\n",
      "Daly MB, Ozols RF. Symptoms of ovarian cancer--where to set the bar? JAMA. 2004;291(22):2755-2756.\n",
      "Davis R, Jones JS, Barocas DA, et al. Diagnosis, evaluation and follow-up of asymptomatic microhematuria (AMH) in adults: AUA guideline. Linthicum, MD: American Urological Association Education and Research, Inc. (AUA); May 2012.\n",
      "de Boer M, van Deurzen CH, van Dijck JA, et al. Micrometastases or isolated tumor cells and the outcome of breast cancer. N Engl J Med. 2009;361(7):653-663.\n",
      "De Giorgi U, Valero V, Rohren E, et al. Circulating tumor cells and [18F]fluorodeoxyglucose positron emission tomography/computed tomography for outcome prediction in metastatic breast cancer. J Clin Oncol. 2009;27(20):3303-3311.\n",
      "De Iuliis F, Salerno G, Taglieri L, et al. Circulating neuregulin-1 and galectin-3 can be prognostic markers in breast cancer. Int J Biol Markers. 2017;32(3):e333-e336.\n",
      "de Kok JB, Verhaegh GW, Roelofs RW et al. DDS (PCA3), a very sensitive and specific marker to detect prostate tumors. Cancer Res. 2002;62:2695-2698.\n",
      "De Roock W, Piessevaux H, De Schutter J, et al. KRAS wild-type state predicts survival and is associated to early radiological response in metastatic colorectal cancer treated with cetuximab. Ann Oncol. 2008;19(3):508-515.\n",
      "Deeb G, Wang J, Ramnath N, et al. Altered E-cadherin and epidermal growth factor receptor expressions are associated with patient survival in lung cancer: A study utilizing high-density tissue microarray and immunohistochemistry. Mod Pathol. 2004;17(4):430-439.\n",
      "Deftereos G, Finkelstein SD, Jackson SA, et al. The value of mutational profiling of the cytocentrifugation supernatant fluid from fine-needle aspiration of pancreatic solid mass lesions. Mod Pathol. 2014;27(4):594-601.\n",
      "Delahaye LJM, Wehkamp D, Floore AN, et al. Performance characteristics of the MammaPrint breast cancer diagnostic gene signature. Personalized Medicine. 2013;10(8):801.\n",
      "Den RB, Feng FY, Showalter TN, et al. Genomic prostate cancer classifier predicts biochemical failure and metastases in patients after postoperative radiation therapy. Int J Radiat Oncol Biol Phys. 2014;89(5):1038-1046.\n",
      "Den RB, Santiago-Jimenez M, Alter J, et al. Decipher correlation patterns post prostatectomy: Initial experience from 2â342 prospective patients. Prostate Cancer Prostatic Dis. 2016;19(4):374-379.\n",
      "Den RB, Yousefi K, Trabulsi EJ, et al. Genomic classifier identifies men with adverse pathology after radical prostatectomy who benefit from adjuvant radiation therapy. J Clin Oncol. 2015;33(8):944-951.\n",
      "Deverka P, Messner D, Dutta T. Evaluation of clinical validity and clinical utility of actionable molecular diagnostic tests in adult oncology. Effectiveness Guidance Document. Baltimore, MD: Center for Medical Technology Policy (CMTP); 2013.\n",
      "Devlin J, O'Grady J. Indications for referral and assessment in adult liver transplantation: A clinical guideline. British Society of Gastroenterology. Gut. 1999;45 Suppl 6:VI1-VI22.\n",
      "DiaCarta, Inc. RadTox cfDNA test [website]. Pleasanton, CA: DiaCarta; 2018. Available at: https://www.diacarta.com/products/radtox-cfdna-test. Accessed December 29, 2021.\n",
      "Di Fiore F, Blanchard F, Charbonnier F, et al. Clinical relevance of KRAS mutation detection in metastatic colorectal cancer treated by Cetuximab plus chemotherapy. Br J Cancer. 2007;96(8):1166-1169.\n",
      "Di Meo A, Bartlett J, Cheng Y, et al. Liquid biopsy: A step forward towards precision medicine in urologic malignancies. Mol Cancer. 2017;16(1):80.\n",
      "Diamandis EP. OvaCheck: Doubts voiced soon after publication. Nature. 2004;430(7000):611.\n",
      "Dijkstra S, Govers TM, Hendriks RJ, et al. Cost-effectiveness of a new urinary biomarker-based risk score compared to standard of care in prostate cancer diagnostics - a decision analytical model. BJU Int. 2017;120(5):659-665.\n",
      "Dillon LD, Gadzia JE, Davidson RS, et al. Prospective, multicenter clinical impact evaluation of a 31-gene expression profile test for management of melanoma patients. Skin. 2018;2(2):111-121.\n",
      "DiMaio CJ, Kim MK. Adjunctive molecular analysis of pancreatic cyst fluid to determine malignant potential. Gastroenterology. 2015;149(1):249-251.\n",
      "Donovan MJ, Hamann S, Clayton M, et al. Systems pathology approach for the prediction of prostate cancer progression after radical prostatectomy. J Clin Oncol. 2008;26(24):3923-3929.\n",
      "Dorandeu A, Raoul JL, Siriser F, et al. Carcinoma of the ampulla of Vater: prognostic factors after curative surgery: A series of 45 cases. Gut. 1997;40:350-355.\n",
      "Doroshow JH. Selecting systemic cancer therapy one patient at a time: Is there a role for molecular profiling of individual patients with advanced solid tumors? J Clin Oncol. 2010;28(33):4869-4876.\n",
      "Doubeni C. Screening for colorectal cancer: Strategies in patients at average risk. UpToDate [online serial]. Waltham, MA: UpToDate; reviewed June 2019.\n",
      "Dowsett M, Sestak I, et al. Comparison of PAM50 risk of recurrence score with Oncotype DX and IHC4 for predicting risk of distant recurrence after endocrine therapy. J Clin Oncol. 2014;31(22): 2783-2790.\n",
      "Drapkin R, von Horsten HH, Lin Y, et al. Human epididymis protein 4 (HE4) is a secreted glycoprotein that is overexpressed by serous and endometrioid ovarian carcinomas. Cancer Res. 2005;65(6):2162-2169.\n",
      "Drukker CA, Bueno-de-Mesquita JM, RetÃ¨l VP, et al. A prospective evaluation of a breast cancer prognosis signature in the observational RASTER study. Int J Cancer. 2013;133(4):929-936.\n",
      "Dubin DP, Dinehart SM, Farberg AS. Level of evidence review for a gene expression profile test for cutaneous melanoma. Am J Clin Dermatol. 2019;20(6):763-770.\n",
      "Duffy MJ, van Dalen A, Haglund C, et al. Clinical utility of biochemical markers in colorectal cancer: European Group on Tumour Markers (EGTM) guidelines. Eur J Cancer. 2003;39(6):718-727.\n",
      "Duffy MJ, van Dalen A, Haglund C, et al. Tumour markers in colorectal cancer: European Group on Tumour Markers (EGTM) guidelines for clinical use. Eur J Cancer. 2007;43(9):1348-1360.\n",
      "Duffy MJ. PSA as a marker for prostate cancer: A critical review. Ann Clin Biochem. 1996;33(Pt 6):511-519.\n",
      "Dumur CI, Lyons-Weiler M, Sciulli C, et al. Interlaboratory performance of a microarray-based gene expression test to determine tissue of origin in poorly differentiated and undifferentiated cancers. J Mol Diagn. 2008;10(1):67-77.\n",
      "Dunphy CH, Polski JM, Evans HL, Gardner LJ. Evaluation of bone marrow specimens with acute myelogenous leukemia for CD34,CD15, CD117, and myeloperoxidase. Arch Pathol Lab Med. 2001;125(8):1063-1069.\n",
      "Dunton C, Bullock RG, Fritsche H, et al. Ethnic disparity in clinical performance between multivariate index assay and CA125 in detection of ovarian malignancy. Future Oncol. 2019;15(26):3047-3051.\n",
      "Dunton CJ, Eskander RN, Bullock RG, Pappas T. Low-risk multivariate index assay scores, physician referral and surgical choices in women with adnexal masses. Curr Med Res Opin. 2020;36(12):2079-2083.\n",
      "Dunton CJ, Hutchcraft ML, Bullock RG, et al. Salvaging detection of early-stage ovarian malignancies when CA125 is not informative. Diagnostics. 2021;11:1440.\n",
      "Dutch College of Health Insurances (CvZ). Genexpressietest Mammaprint bij de behandeling van het mammacarcinoom. Amsterdam, The Netherlands: CvZ; October 29, 2010.\n",
      "Dvir K, Galarza-Fortuna GM, Haines JM, et al. Real-world data on liquid biopsy use in non-small cell lung cancer in the community setting. J Immunother Precision Oncol. 2021;4 (1):1-5.\n",
      "ECRI Institute. OVA1 and ROMA Multivariate Biomarker Tests for Determining Malignancy Risk of a Pelvic Mass. Technology Assessment. Plymouth Meeting, PA: ECRI; April 3, 2015.      \n",
      "Eggener S, Karsh LI, Richardson T, et al. A 17-gene panel for prediction of adverse prostate cancer pathologic features: Prospective clinical validation and utility. Urology. 2019;126:76-82. \n",
      "Ellery B, Hiller J. CellSearchÂ®: Detection of circulating tumour cells for the prognosis and improved management of cancer patients. Horizon Scanning Technology Prioritising Summary. Canberra, ACT: Department of Health and Ageing; November 2010.\n",
      "Ellery B, Parsons J, Merlin T. Molecular testing for prostate cancer prognosis. Technology Brief. Herston, QLD: Department of Health, Queensland; November 2014.\n",
      "Ellis PM, Blais N, Soulieres D, et al. A systematic review and Canadian consensus recommendations on the use of biomarkers in the treatment of non-small cell lung cancer. J Thorac Oncol. 2011;6(8):1379-1391.\n",
      "Ellsworth EM, Palma JF, Spence WC, et al. Mutational profiling of sporadic versus toxin-associated brain cancer formation: Initial findings using loss of heterozygosity profiling. Int J Hyg Environ. 2012;215(3):427-433.\n",
      "Erho N, Crisan A, Vergara IA, et al. Discovery and validation of a prostate cancer genomic classifier that predicts early metastasis following radical prostatectomy. PLoS One. 2013;8(6):e66855. \n",
      "Erickson BK, Kinde I, Dobbin ZC, et al. Detection of somatic TP53 mutations in tampons of patients with high-grade serous ovarian cancer. Obstet Gynecol. 2014;124(5):881-885.\n",
      "Erovic BM, Pammer J, Hollemann D, et al. Motility-related protein-1/CD9 expression in head and neck squamous cell carcinoma. Head Neck. 2003;25(10):848-857.\n",
      "Eskander RN, Carpenter BA, Wu HG, Wolf JK. The clinical utility of an elevated-risk multivariate index assay score in ovarian cancer patients. Curr Med Res Opin. 2016;32(6):1161-1165.\n",
      "Esserman L, Yau C. Rethinking the standard for ductal carcinoma in situ treatment. JAMA Oncol. 2015;1(7):881-883.\n",
      "Esteva A, Feng J, van der Wal D, et al. Prostate cancer therapy personalization via multi-modal deep learning on randomized phase III clinical trials. NPJ Digit Med. 2022;5(1):71.\n",
      "Esteva FJ, Sahin AA, Coombes K, et al. Multi-gene RT-PCR assay for predicting recurrence in node negative breast cancer patients -- M.D. Anderson Clinical Validation Study [abstract]. Breast Cancer Res Treat. 2003;82 (Suppl 1):S11.\n",
      "Eure G, Germany R, Given R, et al. Use of a 17-gene prognostic assay in contemporary urologic practice: Results of an interim analysis in an observational cohort. Urology. 2017;107:67-75.\n",
      "Evaluation of Genomic Applications in Practice and Prevention (EGAPP) Working Group. Recommendations from the EGAPP Working Group: Can tumor gene expression profiling improve outcomes in patients with breast cancer? Genet Med. 2009;11(1):66-73.\n",
      "Evaluation of Genomic Applications in Practice and Prevention (EGAPP) Working Group. Recommendations from the EGAPP Working Group: Can testing of tumor tissue for mutations in EGFR pathway downstream effector genes in patients with metastatic colorectal cancer improve health outcomes by guiding decisions regarding anti-EGFR therapy? Genet Med. 2013;15(7):517-527.\n",
      "Evaluation of Genomic Applications in Practice and Prevention (EGAPP) Working Group. Recommendations from the EGAPP Working Group: Does PCA3 testing for the diagnosis and management of prostate cancer improve patient health outcomes? Genet Med. 2014;16(4):338-346.\n",
      "Everhard S, Kaloshi G, CriniÃ¨re E, et al. MGMT methylation: A marker of response to temozolomide in low-grade gliomas. Ann Neurol. 2006;60(6):740-743.\n",
      "Falagario UG, Martini A, Wajswol E, et al. Avoiding unnecessary magnetic resonance imaging (MRI) and biopsies: Negative and positive predictive value of MRI according to prostate-specific antigen density, 4Kscore and risk calculators. Eur Urol Oncol. 2020;3(5):700-704.\n",
      "Fan B, Zhang H, Jin H, et al. Is overexpression of Ki-67 a prognostic biomarker of upper tract urinary carcinoma? A retrospective cohort study and meta-analysis. Cell Physiol Biochem. 2016;40(6):1613-1625.\n",
      "Farberg AS, Glazer AM, White R, Rigel DS. Impact of a 31-gene expression profiling test for cutaneous melanoma on dermatologists' clinical management decisions. J Drugs Dermatol. 2017;16(5):428-431.\n",
      "Febbo, PG, Ladanyi, M, Aldape, KD, et al.  NCCN Task Force Report: Evaluating the Clinical Utility of Tumor Markers in Oncology. J Natl Compre Canc Netw. 2011;9:S1-S31.\n",
      "Ferguson JS, Van Wert R, Choi Y, et al. Impact of a bronchial genomic classifier on clinical decision making in patients undergoing diagnostic evaluation for lung cancer. BMC Pulm Med. 2016;16(1):66.\n",
      "Fernandez-Flores A, Cassarino DS. Chondroid atypical spitzoid melanocytic tumor. Rev Esp Patol. 2019;52(3):190-193.\n",
      "Ferraz C, Eszlinger M, Paschke R. Current state and future perspective of molecular diagnosis of fine-needle aspiration biopsy of thyroid nodules. J Clin Endocrinol Metab. 2011;96(7):2016-2026.\n",
      "Ferris LK, Farberg AS, Middlebrook B, et al. Identification of high-risk cutaneous melanoma tumors is improved when combining the online American Joint Committee on Cancer Individualized Melanoma Patient Outcome Prediction Tool with a 31-gene expression profile-based classification. J Am Acad Dermatol. 2017;76(5):818-825.\n",
      "Field MG, Harbour JW. Recent developments in prognostic and predictive testing in uveal melanoma. Curr Opin Ophthalmol. 2014;25(3):234-239.\n",
      "Fields MM, Chevlen E. Ovarian cancer screening: A look at the evidence. Clin J Oncol Nurs. 2006;10(1):77-81.\n",
      "Finkelstein SD, Bibbo M, Kowalski TE, et al. Mutational analysis of cytocentrifugation supernatant fluid from pancreatic solid mass lesions. Diagn Cytopathol. 2014;42(8):719-725.\n",
      "Finkelstein SD, Bibbo M, Loren DE, et al. Molecular analysis of centrifugation supernatant fluid from pancreaticobiliary duct samples can improve cancer detection. Acta Cytol. 2012;56(4):439-447.\n",
      "Finkelstein SD, Marsh W, Demetris AJ, et al. Microdissection-based allelotyping discriminates de novo tumor from intrahepatic spread in hepatocellular carcinoma. Hepatology. 2003;37(4):871-879.\n",
      "Finkelstein SD, Przygodzki R, Pricolo VE, et al. K-ras-2 topographic genotyping of pancreatic adenocarcinoma. Arch Surg. 1994;129(4):367-73.\n",
      "Finkelstein SD, Przygodzki R, Pricolo VE, et al. Prediction of biologic aggressiveness in colorectal cancer by p53/K-ras-2 topographic genotyping. Mol Diagn. 1996;1(1):5-28.\n",
      "Finkelstein SD, Tiffee JC, Bakker A, et al. Malignant transformation in sinonasal papillomas is closely associated with aberrant p53 expression. Mol Diagn. 1998;3(1):37-41.\n",
      "Finn RS, Kudo M, Cheng A-L, et al. Pharmacodynamic biomarkers predictive of survival benefit with lenvatinib in unresectable hepatocellular carcinoma: From the phase 3 REFLECT study. Clin Cancer Res. 2021;27(17):4848-4858. \n",
      "Fischer RN, Wolf J. Tumor mutation burden as emerging biomarker in non-small cell lung cancer and beyond. Translational Cancer Research . 2018;7(Suppl 7):S770-S773.\n",
      "Flach J, Shumilov E, Joncourt R, et al. Detection of rare reciprocal RUNX1 rearrangements by next-generation sequencing in acute myeloid leukemia. Genes Chromosomes Cancer. 2020;59(4):268-274.\n",
      "Fletcher SW. Screening for breast cancer. UpToDate [online serial]. Waltham, MA: UpToDate; reviewed February 2013. \n",
      "Flynn K. Bladder cancer surveillance. Brief Overview. Final Report. Boston, MA: Veterans Health Administration, Office of Patient Care Services, Technology Assessment Program; November 2007.\n",
      "Foerster V, Edgar-Hughes M, Forel D, Hoggan B. An assay to identify biopsy-negative prostate cancer. Technology Brief. Health Policy Advisory Committee on Technology (HealthPACT). Australian Safety and Efficacy Register of New Interventional Procedures - Surgical (ASERPNIP-S). Brisbane, QLD: Queensland Department of Health; February 2013.\n",
      "Foj L, Ferrer F, Serra M, et al. Exosomal and non-exosomal urinary miRNAs in prostate cancer detection and prognosis. Prostate. 2017;77(6):573-583.\n",
      "Forde GK, Hornberger J, Michalopoulos S, Bristow RE. Cost-effectiveness analysis of a multivariate index assay compared to modified American College of Obstetricians and Gynecologists criteria and CA-125 in the triage of women with adnexal masses. Curr Med Res Opin. 2016;32(2):321-329.\n",
      "Forouzanfar N, Baranova A, Milanizadeh S, et al. Novel candidate genes may be possible predisposing factors revealed by whole exome sequencing in familial esophageal squamous cell carcinoma. Tumour Biol. 2017;39(5):1010428317699115.\n",
      "Foukakis T, Bergh J. Prognostic and predictive factors in early, non-metastatic breast cancer. UpToDate [online serial]. Waltham, MA: UpToDate; June 2015.\n",
      "Fradet Y, Saad F, Aprikian A, et al. uPM3, a new molecular urine test for the detection of prostate cancer. Urology. 2004;64(2):311-315; discussion 315-316.\n",
      "Freedland SJ, Choeurng V, Howard L, et al. Utilization of a genomic classifier for prediction of metastasis following salvage radiation therapy after radical prostatectomy. Eur Urol. 2016;70(4):588-596.\n",
      "Friboulet L, Olaussen KA, Pignon JP, et al. ERCC1 isoform expression and DNA repair in non-small-cell lung cancer. N Engl J Med. 2013;368(12):1101-1110.\n",
      "Fujiyama S, Izuno K, Gohshi K, et al. Clinical usefulness of des-gamma-carboxy prothrombin assay in early diagnosis of hepatocellular carcinoma. Dig Dis Sci. 1991;36(12):1787-1792.\n",
      "Funakoshi Y, Wang Y, Semba T, et al. Comparison of molecular profile in triple-negative inflammatory and non-inflammatory breast cancer not of mesenchymal stem-like subtype. PLoS One. 2019;14(9):e0222336.\n",
      "Fung ET. A recipe for proteomics diagnostic test development: the OVA1 test, from biomarker discovery to FDA clearance. Clin Chem. 2010;56(2):327-329.\n",
      "Furuya H, Tabula L, Lee R, et al. Analytical validation of ONCURIAâ¢ a multiplex bead-based immunoassay for the non-invasive bladder cancer detection. Pract Lab Med. 2020;22:e00189.\n",
      "Gaida MM, Bach ST, GÃ¼nther F, et al. Expression of galectin-3 in pancreatic ductal adenocarcinoma. Pathol Oncol Res. 2012;18(2):299-307.\n",
      "Gambardella V, Lombardi P, Carbonell-Asins JA, et al. Molecular profiling of advanced solid tumours. The impact of experimental molecular-matched therapies on cancer patient outcomes in early-phase trials: The MAST study. Br J Cancer. 2021;125(9):1261-1269.\n",
      "Gandini O, Luci L, Stigliano A, et al. Is DD3 a new prostate-specific gene? Anticancer Res. 2003;23:305-308.\n",
      "Gao A, Yang X, Tong J, et al. Multiplexed detection of lung cancer biomarkers in patients serum with CMOS-compatible silicon nanowire arrays. Biosens Bioelectron. 2017;91:482-488.\n",
      "Gao G, Ren S, Li A, et al. Epidermal growth factor receptor-tyrosine kinase inhibitor therapy is effective as first-line treatment of advanced non-small-cell lung cancer with mutated EGFR: A meta-analysis from six phase III randomized controlled trials. Int J Cancer. 2012;131(5):E822-E829.\n",
      "Gao X, Porter AT, Grignon DJ, et al. Diagnostic and prognostic markers for human prostate cancer. Prostate. 1997;31(4):264-281.\n",
      "Garcia AA, Hirte H, Fleming G, et al. Phase II clinical trial of bevacizumab and low-dose metronomic oral cyclophosphamide in recurrent ovarian cancer: A trial of the California, Chicago, and Princess Margaret Hospital phase II consortia. J Clin Oncol. 2008;26(1):76-82.\n",
      "Garcia-Barcina M, Martin Bueno AE, de Bujanda Fernandez Pierola L, et al. Tumour markers in certain cancer locations. D-02-03. Vitoria-Gasteiz, Spain: Basque Office for Health Technology Assessment, Health Department Basque Government (OSTEBA); 2002.\n",
      "Garcia-Pardo M, Aparicio I, Martinez I, et al. Brief report: Clinical outcomes using plasma-based molecular profiling to guide treatment decisions in patients with advanced NSCLC and limited access to broad tissue testing. Clin Lung Cancer. 2023 Feb 9 [Online ahead of print].\n",
      "Garner EI. Advances in the early detection of ovarian carcinoma. J Reprod Med. 2005;50(6):447-453.\n",
      "Garrison GW, Cho JL, Deng JC, et al. ATS Core Curriculum 2021. Adult pulmonary medicine: Thoracic oncology. ATS Sch. 2021;2(3):468-483.\n",
      "Gastman BR, Gerami P, Kurley SJ, et al. Identification of patients at risk of metastasis using a prognostic 31-gene expression profile in subpopulations of melanoma patients with favorable outcomes by standard criteria. J Am Acad Dermatol. 2019;80(1):149-157.\n",
      "Gelmann EP, Henshall SM. Clinically relevant prognostic markers for prostate cancer: The search goes on. Ann Intern Med. 2009;150(9):647-649.\n",
      "Gene-Probe, Inc. Gen-Probe PCA3 assay. Package Insert. San Diego, CA: Gen-Probe; April 2007. \n",
      "George B, Datar RH, Wu L, et al. p53 gene and protein status: The role of p53 alterations in predicting outcome in patients with bladder cancer. J Clin Oncol. 2007;25(34):5352-5358.\n",
      "Gerami P, Cook RW, Russell MC, et al. Gene expression profiling for molecular staging of cutaneous melanoma in patients undergoing sentinel lymph node biopsy. J Am Acad Dermatol. 2015;72(5):780-785.\n",
      "Gerami P, Cook RW, Wilkinson J, et al. Development of a prognostic genetic signature to predict the metastatic risk associated with cutaneous melanoma. Clin Cancer Res. 2015;21(1):175-183.\n",
      "Gevensleben H, Gohring UJ, Buttner R, et al. Comparison of MammaPrint and TargetPrint results with clinical parameters in German patients with early stage breast cancer. Int J Mol Med. 2010;26(6):837-843.\n",
      "Gharib H, Papini E, Garber JR, et al; AACE/ACE/AME Task Force on Thyroid Nodules. American Association of Clinical Endocrinologists, American College of Endocrinology, and Associazione Medici Endocrinologi Medical Guidelines for clinical practice for the diagnosis and management of thyroid nodules - 2016 update. Endocr Pract. 2016;22(5):622-639.\n",
      "Ghoneim C, Soula-Rothhut M, Rothhut B. Thrombospondin-1 in differentiated thyroid cancer: Dr. Jekyll and Mr. Hyde. Connect Tissue Res. 2008;49(3):257-260.\n",
      "Gilbey AM, Burnett D, Coleman RE, Holen I. The detection of circulating breast cancer cells in blood. J Clin Pathol. 2004;57(9):903-911.\n",
      "Glas A S, Roos D, Deutekom M, et al. Tumor markers in the diagnosis of primary bladder cancer: A systematic review. J Urol. 2003;169(6):1975-1982.\n",
      "Glass AG, Leo MC, Haddad Z, et al. Validation of a genomic classifier for predicting post-prostatectomy recurrence in a community based health care setting. J Urol. 2016;195(6):1748-1753.\n",
      "Gnant M, Filipits M, et al. Predicting distant recurrence in receptor-positive breast cancer patients with limited clinicopathological risk: Using the PAM50 Risk of Recurrence score in 1478 postmenopausal patients of the ABCSG-8 trial treated with adjuvant endocrine therapy alone. Ann Oncol. 2014;25(2):339-345.\n",
      "Goff BA, Mandel LS, Melancon CH, Muntz HG. Frequency of symptoms of ovarian cancer in women presenting to primary care clinics. JAMA. 2004;291(22):2705-2712.\n",
      "Gogenur M, Al-Huda Hadi N, Qvortrup C, et al. ctDNA for risk of recurrence assessment in patients treated with neoadjuvant treatment: A systematic review and meta-analysis. Ann Surg Oncol. 2022;29(13):8666-8674.\n",
      "GonÃ§alves A, Esteyries S, Taylor-Smedra B, et al. A polymorphism of EGFR extracellular domain is associated with progression free-survival in metastatic colorectal cancer patients receiving cetuximab-based treatment. BMC Cancer. 2008;8(169). \n",
      "Gonda TA, Viterbo D, Gausman V, et al. Mutation profile and fluorescence in situ hybridization analyses increase detection of malignancies in biliary strictures. Clin Gastroenterol Hepatol. 2017;15(6):913-919.\n",
      "Gong W, Zhang X, Wu J, et al. RRM1 expression and clinical outcome of gemcitabine-containing chemotherapy for advanced non-small-cell lung cancer: A meta-analysis. Lung Cancer. 2012;75(3):374-380.\n",
      "Goodison S, Ogawa O, Matsui Y, et al. A multiplex urinary immunoassay for bladder cancer detection: Analysis of a Japanese cohort. J Transl Med. 2016;14(1):287.\n",
      "Goodman AM, Kato S, Bazhenova L, et al. Tumor mutational burden as an independent predictor of response to immunotherapy in diverse cancers. Mol Cancer Ther. 2017;16(11):2598-2608.\n",
      "Goodman S, Dickerson K, Wilson R. Gene expression profile tests for early-stage breast cancer. Effectiveness Guidance Document: Methodological Guidance for the Design of Comparative Effectiveness Studies. Version 1.0. Baltimore, MD: Center for Medical Technology Policy (CMTP); June 2009.\n",
      "Goodrich ST, Bristow RE, Santoso JT, et al. The effect of ovarian imaging on the clinical interpretation of a multivariate index assay. Am J Obstet Gynecol. 2014;211(1):65.e1-65.e11.\n",
      "Gore JL, du Plessis M, Santiago-JimÃ©nez M, et al. Decipher test impacts decision making among patients considering adjuvant and salvage treatment after radical prostatectomy: Interim results from the multicenter prospective PRO-IMPACT study. Cancer. 2017;123(15):2850-2859.\n",
      "Gotlib J. Advanced systemic mastocytosis: Management and prognosis. UpToDate [online serial]. Waltham, MA: UpToDate; reviewed February 2021.\n",
      "Gracie L, Pan Y, Atenafu EG, et al. Circulating tumour DNA (ctDNA) in metastatic melanoma, a systematic review and meta-analysis. Eur J Cancer. 2021;158:191-207.\n",
      "Gradishar WJ. Male breast cancer. UpToDate [online serial]. Waltham, MA: UpToDate; 2010.\n",
      "Grail. Galleri. Menlo Park, CA: Grail; Available at: https://www.galleri.com/what-is-galleri. Accessed July 31, 2023.\n",
      "Gray RG, Quirke P, Handley K, et al. Validation study of a quantitative multigene reverse transcriptase-polynmerase chain reaction assay for assessment of recurrence risk in patients with stage II colon cancer.  J Clin Oncol. 2011;29:1-13.\n",
      "Green Park Collaborative. Demonstrating the clinical utility of next generation sequencing in clinical oncology. Meeting Summary. Baltimore, MD: Center for Medical Technology Policy (CMTP); July 7, 2014.\n",
      "Greenhaw BN, Covington KR, Kurley SJ, et al. Molecular risk prediction in cutaneous melanoma: A meta-analysis of the 31-gene expression profile prognostic test in 1,479 patients. J Am Acad Dermatol. 2020;83(3):745-753. \n",
      "Greenhaw BN, Zitelli JA, Brodland DG. Estimation of prognosis in invasive cutaneous melanoma: An independent study of the accuracy of a gene expression profile test. Dermatol Surg. 2018;44(12):1494-1500.\n",
      "Gregorc V, Novello S, Lazzari C, et al. Predictive value of a proteomic signature in patients with non-small-cell lung cancer treated with second-line erlotinib or chemotherapy (PROSE): A biomarker-stratified, randomised phase 3 trial. Lancet Oncol. 2014;15(7):713-721.\n",
      "Grocela JA, McDougal WS. Utility of nuclear matrix protein (NMP22) in the detection of recurrent bladder cancer. Urol Clin North Am. 2000;27(1):47-51.\n",
      "Groskopf J, Aubin SM, Deras IL, et al. APTIMA PCA3 molecular urine test: Development of a method to aid in the diagnosis of prostate cancer. Clin Chem. 2006;52(6):1089-1095.\n",
      "Groskopf J, Nakanishi H, Deras IL, et al. The PCA3 score correlates with tumor volume but not prostate size, and can synergize with other patient information for predicting biopsy outcome. Abstrac 1709. Annual Meeting of the American Urological Association. Anaheim, CA; May 19-24, 2007. \n",
      "Grossfeld GD, Litwin MS, Wolf JS Jr, et al. Evaluation of asymptomatic microscopic hematuria in adults: The American Urological Association best practice policy--part II: Patient evaluation, cytology, voided markers, imaging, cystoscopy, nephrology evaluation, and follow-up. Urology. 2001;57(4):604-610.\n",
      "Grossman D, Okwundu N, Bartlett EK, et al. Prognostic gene expression profiling in cutaneous melanoma: Identifying the knowledge gaps and assessing the clinical benefit. JAMA Dermatol. 2020;156(9):1004-1011.\n",
      "Grossman HB, Messing E, Soloway M, et al. Detection of bladder cancer using a point-of-care proteomic assay. JAMA. 2005;293(7):810-816.\n",
      "Gruber A, BjÃ¶rkholm M, Brinch L, et al. A phase I/II study of the MDR modulator Valspodar (PSC 833) combined with daunorubicin and cytarabine in patients with relapsed and primary refractory acute myeloid leukemia. Leuk Res. 2003;27(4):323-328.\n",
      "Gunning A, Kumar S, Williams CK, et al. Analytical validation of NavDx, a cfDNA-Based fragmentomic profiling assay for HPV-driven cancers. Diagnostics (Basel). 2023;13(4):725.\n",
      "Gunven P, RandÃ©n M, Elmberger G, et al. Gene expression profiling guiding diagnosis and therapy of rare mammary-like anogenital gland carcinomas. Med Oncol. 2012;29(1):127-132.\n",
      "Guo Y, Yu H, Wang J, et al. The landscape of small non-coding RNAs in triple-negative breast cancer. Genes (Basel). 2018;9(1).\n",
      "Gupta S, Bent S, Kohlwes J. Test characteristics of alpha-fetoprotein for detecting hepatocellular carcinoma in patients with hepatitis C. Ann Intern Med. 2003;139(1):46-50.\n",
      "Gustavsen G, Schroeder B, Kennedy P, et al. Health economic analysis of Breast Cancer Index in patients with ER+, LN-breast cancer. Am J Manag Care. 2014;20(8):e302-e310.\n",
      "Gyawali B, West H. Plasma vs tissue next-generation sequencing in non-small cell lung cancer -- Either, both, or neither? JAMA Oncol. 2019;5(2):148-149.\n",
      "Habel LA, Sakoda LC, Achacoso N, et al. HOXB13:IL17BR and molecular grade index and risk of breast cancer death among patients with lymph node-negative invasive disease. Breast Cancer Res. 2013;15(2):R24.\n",
      "Habel LA, Shak S, Jacobs MK, et al. A population-based study of tumor gene expression and risk of breast cancer death among lymph node-negative patients. Breast Cancer Res. 2006;8(3):R25.\n",
      "Haese A, Van Poppel H, Marberger M, et al. The value of the PCA3 assay in guiding decision which men with a negative prostate biopsy need immediate repeat biopsy: Preliminary European data. Abstract. 22nd Annual EAU Congress. Berlin, Germany; March 21-24, 2007. \n",
      "Hainsworth JD, Rubin MS, Spigel DR, et al. Molecular gene expression profiling to predict the tissue of origin and direct site-specific therapy in patients With carcinoma of unknown primary site: A prospective trial of the Sarah Cannon Research Institute. J Clin Oncol. 2013;31(2):217-223.\n",
      "Hallermayr A, Wohlfrom T, Steinke-Lange V, et al. Somatic copy number alteration and fragmentation analysis in circulating tumor DNA for cancer screening and treatment monitoring in colorectal cancer patients. J Hematol Oncol. 2022;15(1):125.\n",
      "Hamamura K, Shiratori Y, Shiina S, et al. Unique clinical characteristics of patients with hepatocellular carcinoma who present with high plasma des-gamma-carboxy prothrombin and low serum alpha-fetoprotein. Cancer. 2000;88(7):1557-1564.\n",
      "Hamm CM, Cripps C. Carcinoembryonic antigen in metastatic colorectal cancer. Clin Invest Med. 1998;21(4-5):186-191.\n",
      "Hao Z, Pan Y, Huang C, et al. Sensitive detection of lung cancer biomarkers using an aptameric graphene-based nanosensor with enhanced stability. Biomed Microdevices. 2019;21(3):65.\n",
      "Harris L, Fritsche H, Mennel R, et al; American Society of Clinical Oncology. American Society of Clinical Oncology 2007 update of recommendations for the use of tumor markers in breast cancer. J Clin Oncol. 2007;25(33):5287-5312.\n",
      "Harris LN, Ismaila N, McShane LM, et al. Use of biomarkers to guide decisions on adjuvant systemic therapy for women with early-stage invasive breast cancer: American Society of Clinical Oncology Clinical Practice Guideline. J Clin Oncol. 2016;34(10):1134-1150.\n",
      "Harrison LE, Guillem JG, Paty P, et al. Preoperative carcinoembryonic antigen predicts outcomes in node-negative colon cancer patients: A multivariate analysis of 572 patients. J Am Coll Surg. 1997;185(1):55-59.\n",
      "Haugen BR, Alexander EK, Bible KC, et al. 2015 American Thyroid Association Management Guidelines for Adult Patients with Thyroid Nodules and Differentiated Thyroid Cancer: The American Thyroid Association Guidelines Task Force on Thyroid Nodules and Differentiated Thyroid Cancer. Thyroid. 2016;26(1):1-133.\n",
      "Havrilesky LJ, Whitehead CM, Rubatt JM, et al. Evaluation of biomarker panels for early stage ovarian cancer detection and monitoring for disease recurrence. Gynecol Oncol. 2008;110(3):374-382.\n",
      "Hayes DF. Serum tumor markers for breast cancer. Anticancer Drugs. 1995;6(Suppl 2):26-27.\n",
      "Helfand B. Editorial comment. J Urol. 2020;204(3):474.\n",
      "Hellmann MD, Ciuleanu TE, Pluzanski A, et al. Nivolumab plus ipilimumab in lung cancer with a high tumor mutational burden. N Engl J Med. 2018;378(22):2093-2104.\n",
      "Hellstrom I, Hellstrom KE. SMRP and HE4 as biomarkers for ovarian carcinoma when used alone and in combination with CA125 and/or each other. Adv Exp Med Biol. 2008;622:15-21\n",
      "Hellstrom I, Raycraft J, Hayden-Ledbetter M, et al. The HE4 (WFDC2) protein is a biomarker for ovarian carcinoma. Cancer Res. 2003;63:3695. \n",
      "Helman E, Nguyen M, Karlovich CA, et al. Cell-free DNA NGS prediction of response and resistance to a 3rd-generation EGFR inhibitor. Clin Lung Cancer. 2018;19(6):518-530.\\\n",
      "Hendriks RJ, van Oort IM, Schalken JA. Blood-based and urinary prostate cancer biomarkers: A review and comparison of novel biomarkers for detection and treatment decisions. Prostate Cancer Prostatic Dis. 2017;20(1):12-19.\n",
      "Hessels D, Klein Gunnewiek JM, van Oort I, et al. DD3 (PCA3) based molecular urine analysis for the diagnosis of prostate cancer. Eur Urol. 2003;44:8-16.\n",
      "Hirasawa Y, Pagano I, Chen R, et al. Diagnostic performance of Oncuriaâ¢, a urinalysis test for bladder cancer. J Transl Med. 2021;19(1):141.\n",
      "Hochheiser L, Hornberger J, Turner M, Lyman GH. Multi-gene assays: Effect on chemotherapy use, toxicity and cost in estrogen receptor-positive early stage breast cancer. J Comp Eff Res. 2019;8(5):289-304.\n",
      "Hoffman RM. Screening for prostate cancer. UpToDate [online serial]. Waltham, MA: UpToDate; reviewed May 2022.\n",
      "Hollestelle A, Pelletier C, Hooning M, et al. Prevalence of the variant allele rs61764370 T>G in the 3'UTR of KRAS among Dutch BRCA1, BRCA2 and non-BRCA1/BRCA2 breast cancer families. Breast Cancer Res Treat. 2011;128(1):79-84.\n",
      "Holst VA, Finkelstein S, Colby TV, et al.  p53 and K-ras mutational genotyping in pulmonary carcinosarcoma, spindle cell carcinoma, and pulmonary blastoma: Implications for histogenesis. Am J Surg Pathol. 1997;21(7):801-811.\n",
      "Holst VA, Finkelstein S, Yousem SA. Bronchioloalveolar adenocarcinoma of lung: Monoclonal origin for multifocal disease. Am J Surg Pathol. 1998;22(11):1343-1350.\n",
      "Hsueh EC, DeBloom JR, Lee J, et al. Interim analysis of survival in a prospective, multi-center registry cohort of cutaneous melanoma tested with a prognostic 31-gene expression profile test. J Hematol Oncol. 2017;10(1):152.\n",
      "Hsueh EC, DeBloom JR, Lee JH, et al. Long-term outcomes in a multicenter, prospective cohort evaluating the prognostic 31-gene expression profile for cutaneous melanoma. JCO Precis Oncol. 2021;5:PO.20.00119.\n",
      "Hu JC, Tosoian JT, Qi J, et al. Clinical utility of gene expression classifiers in men with newly diagnosed prostate cancer. JCO Precision Oncology. 2018. \n",
      "Hudis CA, Dickler M. Increasing precision in adjuvant therapy for breast cancer. N Engl J Med. 2016;375(8):790-791.\n",
      "Hufnagl C, Leisch M, Weiss L, et al. Evaluation of circulating cell-free DNA as a molecular monitoring tool in patients with metastatic cancer. Oncol Lett. 2020;19(2):1551-1558.\n",
      "Hultcrantz R, Olsson R, Danielsson A, et al. A 3-year prospective study on serum tumor markers used for detecting cholangiocarcinoma in patients with primary sclerosing cholangitis. J Hepatol. 1999;30(4):669-673.\n",
      "Humphrey LL, Teutsch S, Johnson M; U.S. Preventive Services Task Force. Lung cancer screening with sputum cytologic examination, chest radiography, and computed tomography: An update for the U.S. Preventive Services Task Force. Ann Intern Med. 2004;140(9):740-753.\n",
      "Hutchinson LM, Chang EL, Becker CM, et al. Development of a sensitive and specific enzyme-linked immunosorbent assay for thymosin beta15, a urinary biomarker of human prostate cancer. Clin Biochem. 2005;38(6):558-571.\n",
      "Hwang HS, Yoon DH, Hong JY, et al. The cell-of-origin classification of diffuse large B cell lymphoma in a Korean population by the Lymph2Cx assay and its correlation with immunohistochemical algorithms. Ann Hematol. 2018;97(12):2363-2372.\n",
      "Hyams DM, Covington KR, Johnson CE, et al. Integrating the melanoma 31-gene expression profile test with surgical oncology practice within national guideline and staging recommendations. Future Oncol. 2021;17(5):517-527.\n",
      "Ibrahimpasic T, Ghossein R, Shah JP, et al. Poorly differentiated carcinoma of the thyroid gland: Current status and future prospects. Thyroid. 2019;29(3):311-321.\n",
      "Ignatiadis M, Sotiriou C. Luminal breast cancer: From biology to treatment. Nat Rev Clin Oncol. 2013;10(9):494-506.\n",
      "Ignatiadis M. Multigene assays for late recurrence of breast cancer. Lancet Oncol. 2013;14(11):1029-1030.\n",
      "Ikoma J, Kaito M, Ishihara T, et al. Early diagnosis of hepatocellular carcinoma using a sensitive assay for serum des-gamma-carboxy prothrombin: A prospective study. Hepatogastroenterology. 2002;49(43):235-238.\n",
      "Ilmer M, Mazurek N, Gilcrease MZ, et al. Low expression of galectin-3 is associated with poor survival in node-positive breast cancers and mesenchymal phenotype in breast cancer stem cells. Breast Cancer Res. 2016;18(1):97.\n",
      "Institute for Clinical Systems Improvement (ICSI). Breast cancer treatment. ICSI Healthcare Guidelines. Bloomington, MN: ICSI; September 2004.\n",
      "Intlekofer AM, Joffe E, Batlevi CL, et al. Integrated DNA/RNA targeted genomic profiling of diffuse large B-cell lymphoma using a clinical assay. Blood Cancer J. 2018;8(6):60.\n",
      "Invivoscribe, Inc. MyAMLâ¢ 194 targeted NGS gene panel - CLIA [website]. San Diego, CA: Invivoscribe; 2018. Available at: https://www.invivoscribe.com/clinical-services/myaml-194-targeted-ngs-gene-panel-clia. Accessed July 10, 2018.\n",
      "Jankowitz RC, Cooper K, Erlander MG, et al. Prognostic utility of the breast cancer index and comparison to Adjuvant! Online in a clinical case series of early breast cancer. Breast Cancer Res. 2011;13(5):R98.\n",
      "Janku F, Wheler JJ, Hong DS, Kurzrock R. Bevacizumab-based treatment in colorectal cancer with a NRAS Q61K mutation. Target Oncol. 2013;8(3):183-188.\n",
      "Jarell A, Gastman BR, Dillon LD, et al. Optimizing treatment approaches for patients with cutaneous melanoma by integrating clinical and pathologic features with the 31-gene expression profile test. J Am Acad Dermatol. 2022;87(6):1312-1320.\n",
      "Jarvinen TA, Liu ET. Simultaneous amplification of HER-2 (ERBB2) and topoisomerase IIalpha (TOP2A) genes--molecular basis for combination chemotherapy in cancer. Curr Cancer Drug Targets. 2006;6(7):579-602.\n",
      "Jeffery GM, Hickey BE, Hider P. Follow-up strategies for patients treated for non-metastatic colorectal cancer. Cochrane Database Syst Rev. 2007;(1):CD002200.\n",
      "Jerevall PL, Ma XJ, Li H, Salunga R, et al. Prognostic utility of HOXB13:IL17BR and molecular grade index in early-stage breast cancer patients from the Stockholm trial. Br J Cancer. 2011;104(11):1762-1769.\n",
      "Jessup JM, Loda M. Prognostic markers in rectal carcinoma. Semin Surg Oncol. 1998;15(2):131-140.\n",
      "Jiang K, Lawson D, Cohen C, Siddiqui MT. Galectin-3 and PTEN expression in pancreatic ductal adenocarcinoma, pancreatic neuroendocrine neoplasms and gastrointestinal tumors on fine-needle aspiration cytology. Acta Cytol. 2014;58(3):281-287.\n",
      "Jiang W, Liu L, Chen Y. Simultaneous detection of human C-terminal p53 isoforms by single template molecularly imprinted polymers (MIPs) coupled with liquid chromatography-tandem mass spectrometry (LC-MS/MS)-based targeted proteomics. Anal Chem. 2018;90(5):3058-3066.\n",
      "Johnson PW, Joel SP, Love S, et al. Tumour markers for prediction of survival and monitoring of remission in small cell lung cancer. Br J Cancer. 1993;67(4):760-766.\n",
      "Jones MW, Kounelis S, Hsu C, et al. Prognostic value of p53 and K-ras-2 topographic genotyping in endometrial carcinoma: A clinicopathologic and molecular comparison. Int J Gynecol Pathol. 1997;16(4):354-360.\n",
      "Jones MW, Kounelis S, Papadaki H, et al. The origin and molecular characterization of adenoid basal carcinoma of the uterine cervix. Int J Gynecol Pathol. 1997;16(4):301-306.\n",
      "Jordan SE, Saad H, Covarrubias AS, et al. mRNA expression in low grade serous ovarian cancer: Results of a nanoString assay in a diverse population. Gynecol Oncol. 2020;159(2):554-562.\n",
      "Jorgensen LG,, Osterlind K, Hansen HH, Cooper EH. Serum neuron specific enolase (NSE) is a determinant of response duration in small cell lung cancer. Br J Cancer. 1992;66(3):594-598.\n",
      "Jovanovic B, Sheng Q, Seitz RS, et al. Comparison of triple-negative breast cancer molecular subtyping using RNA from matched fresh-frozen versus formalin-fixed paraffin-embedded tissue. BMC Cancer. 2017;17(1):241.\n",
      "Julian TB, Blumencranz P, Deck K, et al. Novel intraoperative molecular test for sentinel lymph node metastases in patients with early-stage breast cancer. J Clin Oncol. 2008;26(20):3338-3345.\n",
      "Juntermanns B, Kaiser GM, Reis H, et al. Klatskin-mimicking lesions: Still a diagnostical and therapeutical dilemma? Hepatogastroenterology. 2011;58(106):265-269.\n",
      "Kahn HJ, Presta A, Yang LY, et al. Enumeration of circulating tumor cells in the blood of breast cancer patients after filtration enrichment: Correlation with disease stage. Breast Cancer Res Treat. 2004;86(3):237-247.\n",
      "Kalemkerian GP, Narula N, Kennedy EB,et al. Molecular testing guideline for the selection of patients with lung cancer for treatment with targeted tyrosine kinase inhibitors: American Society of Clinical Oncology Endorsement of the College of American Pathologists/International Association for the Study of Lung Cancer/Association for Molecular Pathology Clinical Practice Guideline Update. J Clin Oncol. 2018;36(9):911-919.\n",
      "Kamisawa T, Fukayama M, Koike M, et al. Carcinoma of the ampulla of Vater: Expression of cancer-associated antigens inversely correlated with prognosis. Am J Gastroenterol. 1988;83(10):1118-1123.\n",
      "Kanbour-shakir A, Kounelis S, Papadaki H, et al. Relationship of p53 genotype to second-look recurrence and survival in ovarian epithelial malignancy. Mol Diagn. 1996;1(2):121-129.\n",
      "Karapetis CS, Khambata-Ford S, Jonker DJ, et al. K-ras mutations and benefit from cetuximab in advanced colorectal cancer. N Engl J Med. 2008;359(17):1757-1765.\n",
      "Karnes JR, Wastatzis G, Kosari F, et al. Risk Stratification of Prostate Cancer in Needle Biopsy Specimens: Utilization of a Multi-gene FISH Panel Prostate Cancer Foundation (PCF). Abstract. Prostate Cancer Foundation Annual Scientific Retreat 2017. Washington D.C; October 5-7, 2017. \n",
      "Karnes RJ, Bergstralh EJ, Davicioni E, et al. Validation of a genomic classifier that predicts metastasis following radical prostatectomy in an at risk patient population. J Urol. 2013;190(6):2047-2053.\n",
      "Kato S, Kurasaki K, Ikeda S, Kurzrock R. Rare tumor clinic: The University of California San Diego Moores Cancer Center experience with a precision therapy approach. Oncologist. 2018;23(2):171-178.\n",
      "Kau SY, Shyr YM, Su CH, et al. Diagnostic and prognostic values of CA 19-9 and CEA in periampullary cancers. J Am Coll Surg. 1999;188(4):415-420.\n",
      "Kawai M, Uchiyama K, Tani M, et al. Clinicopathological features of malignant intraductal papillary mucinous tumors of the pancreas: The differential diagnosis from benign entities. Arch Surg. 2004;139(2):188-192.\n",
      "Keller J, Schwartz TL, Lizalek JM, et al. Prospective validation of the prognostic 31-gene expression profiling test in primary cutaneous melanoma. Cancer Med. 2019;8(5):2205-2212.\n",
      "Khalid A, Nodit L, Zahid M, et al. Endoscopic ultrasound fine needle aspirate DNA analysis to differentiate malignant and benign pancreatic masses. Am J Gastroenterol. 2006;101(11):2493-2500.\n",
      "Khalid A, Pal R, Sasatomi E, et al. Use of microsatellite marker loss of heterozygosity in accurate diagnosis of pancreaticobiliary malignancy from brush cytology samples. Gut. 2004;53(12):1860-1865.\n",
      "Khandelwal A, Malhotra A, Jain M, et al. The emerging role of long non-coding RNA in gallbladder cancer pathogenesis. Biochimie. 2017;132:152-160.\n",
      "Khatcheressian JL, Hurley P, Bantug E, et al.; American Society of Clinical Oncology. Breast cancer follow-up and management after primary treatment: American Society of Clinical Oncology clinical practice guideline update. J Clin Oncol. 2013;31(7):961-965.\n",
      "Khatcheressian JL, Wolff AC, Smith TJ, et al; American Society of Clinical Oncology. American Society of Clinical Oncology 2006 update of the breast cancer follow-up and management guidelines in the adjuvant setting. J Clin Oncol. 2006;24(31):5091-5097.\n",
      "Khosravi F, Sachdev M, Alshati A, et al. Mutation profiling impacts clinical decision making and outcomes of patients with solid pancreatic lesions indeterminate by cytology. J Pancreas (Online). 2018;19(1):6-11.\n",
      "Kim HL, Li P, Huang HC, et al. Validation of the Decipher test for predicting adverse pathology in candidates for prostate cancer active surveillance. Prostate Cancer Prostatic Dis. 2019;22(3):399-405\n",
      "Kim KH, Zsebik GN, Straughn JM, Landen CN. Management of complex pelvic masses using a multivariate index assay: A decision analysis. Gynecol Oncol. 2012;126(3):364-368.\n",
      "Kim ST, Banks KC, Lee S-H, et al. Prospective feasibility study for using cell-free circulating tumor DNA-guided therapy in refractory metastatic solid cancers: An interim analysis. JCO Precis Oncol. 2017;1:PO.16.00059.\n",
      "Kim WT, Yun SJ, Kim WJ. For physicians managing voiding dysfunction, improving the detection rate of early prostate cancer and discrimination from benign prostatic hyperplasia, in a molecular biomarker aspects. Int Neurourol J. 2019;23(1):5-12.\n",
      "King TE, Jr. Clinical manifestations and diagnosis of idiopathic pulmonary fibrosis. UpToDate [online serial]. Waltham, MA: UpToDate; reviewed September 2019.\n",
      "King V, Mosbaek C, Carson S, et al. Gene expression profile testing of cancer tissue. Final Evidence Report. Prepared by the Center for Evidence-Based Policy, Oregon Health and Science University, for the Health Technology Assessment Program (HTA). Olympia, WA: Washington State Health Care Authority, Health Technology Assessment Program (HTA); February 16, 2018.\n",
      "Kisser A, Zechmeister-Koss I. A systematic review of p16/Ki-67 immuno-testing for triage of low grade cervical cytology. BJOG. 2015;122(1):64-70.\n",
      "KIYATEC, Inc. 3D prediction of patient-specific response (3D-PREDICT). ClinicalTrials.gov Identifier: NCT03561207. Bethesda, MD: National Library of Medicine; last updated April 8, 2022.\n",
      "Klein EA, Cooperberg MR, Magi-Galluzzi C, et al. A 17-gene assay to predict prostate cancer aggressiveness in the context of Gleason grade heterogeneity, tumor multifocality, and biopsy undersampling. Eur Urol. 2014;66(3):550-560.\n",
      "Klein EA, Haddad Z, Yousefi K, et al. Decipher genomic classifier measured on prostate biopsy predicts metastasis risk. Urology. 2016;90:148-152. (Ref. 599).\n",
      "Klein EA, Partin A, Lotan Y, et al. Clinical validation of IsoPSA, a single parameter, structure-focused assay for improved detection of prostate cancer: A prospective, multicenter study. Urol Oncol. 2022;40(9):408.e9-408.e18.\n",
      "Klein EA, Santiago-JimÃ©nez M, Yousefi K, et al. Molecular analysis of low grade prostate cancer utilizing a genomic classifier of metastatic potential. J Urol. 2016;197(1):122-128.\n",
      "Klein EA, Stephenson AJ, Raghavan D, Dreicer R. Systems pathology and predicting outcome after radical prostatectomy. J Clin Oncol. 2008;26(24):3916-3917.\n",
      "Klein EA, Yousefi K, Haddad Z, . A genomic classifier improves prediction of metastatic disease within 5 years after surgery in node-negative high-risk prostate cancer patients managed by radical prostatectomy without adjuvant therapy. Eur Urol. 2015;67(4):778-786.\n",
      "Klufas MA, Itty S, McCannel CA, et al. Variable results for uveal melanoma-specific gene expression profile prognostic test in choroidal metastasis. JAMA Ophthalmol. 2015;133(9):1073-1076.\n",
      "Knauer M, Mook S, Rutgers EJ, et al. The predictive value of the 70-gene signature for adjuvant chemotherapy in early breast cancer. Breast Cancer Res Treat. 2010;120(3):655-661.\n",
      "Ko JS, Clarke LE, Minca EC, et al. Correlation of melanoma gene expression score with clinical outcomes on a series of melanocytic lesions. Hum Pathol. 2019;86:213-221.\n",
      "Ko JS, Matharoo-Ball B, Billings SD, et al. Diagnostic distinction of malignant melanoma and benign nevi by a gene expression signature and correlation to clinical outcomes. Cancer Epidemiol Biomarkers Prev. 2017;26(7):1107-1113.\n",
      "Konety B, Zappala SM, Parekh DJ, et al. The 4KscoreÂ® Test reduces prostate biopsy rates in community and academic urology practices. Rev Urol. 2015;17(4):231-240.\n",
      "Kornberg Z, Cowan JE, Westphalen AC, et al. Genomic prostate score, PI-RADSâ¢ version 2 and progression in men with prostate cancer on active surveillance. J Urol. 2019;201(2):300-307.\n",
      "Kotani D, Oki E, Nakamura Y, et al. Molecular residual disease and efficacy of adjuvant chemotherapy in patients with colorectal cancer. Nat Med. 2023;29(1):127-134.\n",
      "KotÅowska A, MaliÅski E, Sworczak K, et al. The urinary steroid profile in patients diagnosed with adrenal incidentaloma. Clin Biochem. 2009;42(6):448-454.\n",
      "Kounelis S, Jones MW, Papadaki H, et al. Carcinosarcomas (malignant mixed mullerian tumors) of the female genital tract: Comparative molecular analysis of epithelial and mesenchymal components. Hum Pathol. 1998;29(1):82-87.\n",
      "Kowalski T, Siddiqui A, Loren D, et al. Management of patients with pancreatic cysts: Analysis of possible false-negative cases of malignancy. J Clin Gastroenterol. 2016;50(8):649-657.\n",
      "Krane JF, Cibas ES, Endo M, et al. The Afirma Xpression Atlas for thyroid nodules and thyroid cancer metastases: Insights to inform clinical decision-making from a fine-needle aspiration sample. Cancer Cytopathol. 2020;128(7):452-459.\n",
      "Kratz JR, Haro GJ, Cook NR, et al. Incorporation of a molecular prognostic classifier improves conventional non-small cell lung cancer staging. J Thorac Oncol. 2019;14(7):1223-1232.\n",
      "Kretschmer A, Tolkach Y, Ellinger J, Kristiansen G. Molecular biomarkers and prognostic factors for prostate cancer. Urologe A. 2017;56(7):933-944.\n",
      "Krijgsman O, Roepman P, Zwart W, et al. A diagnostic gene profile for molecular subtyping of breast cancer associated with treatment response. Breast Cancer Res Treat. 2012;133(1):37-47.\n",
      "Krishnamurti U, Sasatomi E, Swalsky PA, et al. Analysis of loss of heterozygosity in atypical and negative bile duct brushing cytology specimens with malignant outcome: Are 'negative' cytologic findings a representation of morphologically subtle molecular alterations? Arch Pathol Lab Med. 2007;131(1):74-80.\n",
      "Krishnamurti U, Sasatomi E, Swalsky PA, et al. Microdissection-based mutational genotyping of serous borderline tumors of the ovary. Int J Gynecol Pathol. 2005;24(1):56-61.\n",
      "Krop I, Ismaila N, Andre F, et al. Use of biomarkers to guide decisions on adjuvant systemic therapy for women with early-stage invasive breast cancer: American Society of Clinical Oncology Clinical Practice Guideline Focused Update. J Clin Oncol. 2017:JCO2017740472.\n",
      "Kruger S, Mess F, Bohle A, Feller AC. Numerical aberrations of chromosome 17 and the 9p21 locus are independent predictors of tumor recurrence in non-invasive transitional cell carcinoma of the urinary bladder. Int J Oncol. 2003;23(1):41-48.\n",
      "Kuipers J, Thurnherr T, Moffa G, et al. Mutational interactions define novel cancer subgroups. Nat Commun. 2018;9(1):4353.\n",
      "Kumaresan K, Kakkar N, Verma A, et al. Diagnostic utility of Î±-methylacyl CoA racemase (P504S) & HMWCK in morphologically difficult prostate cancer. Diagn Pathol. 2010;22(5):83.\n",
      "Kushnir VM, Mullady DK, Das K, et al. The diagnostic yield of malignancy comparing cytology, FISH, and molecular analysis of cell free cytology brush supernatant in patients with biliary strictures undergoing endoscopic retrograde cholangiography (ERC): A prospective study. J Clin Gastroenterol. 2019;53(9):686-692.\n",
      "Kwatra SG, Hines H, Semenov YR, et al. A dermatologist's guide to implementation of gene expression profiling in the management of melanoma. J Clin Aesthet Dermatol. 2020;13(11 Suppl 1):s3-s14.\n",
      "Kyrochristos ID, Glantzounis GK, Ziogas DE, et al. From clinical standards to translating next-generation sequencing research into patient care improvement for hepatobiliary and pancreatic cancers. Int J Mol Sci. 2017;18(1):180.\n",
      "Laboratory Corporation of America Holdings. OmniSeq AdvanceSM Assay. Integrated Oncology [website]. Burlington, NC: Laboratory Corporation of America Holdings; 2018. Available at: https://www.integratedoncology.com/test-menu/omniseq-advance-assay/4e3e0db4-7a52-4c5e-a677-4bc58fc7606f. Accessed July 20, 2018.\n",
      "Labourier E, Shifrin A, Busseniers AE, et al. Molecular testing for miRNA, mRNA, and DNA on fine-needle aspiration improves the preoperative diagnosis of thyroid nodules with indeterminate cytology. J Clin Endocrinol Metab. 2015;100(7):2743-2750.\n",
      "Labourier E. Efficiency versus effectiveness: impact of molecular testing on healthcare costs and clinical outcome of patients with indeterminate thyroid nodules [letter]. Med Oncol. 2016;33(7):79.\n",
      "Labourier E. Utility and cost-effectiveness of molecular testing in thyroid nodules with indeterminate cytology. Clin Endocrinol (Oxf). 2016;85(4):624-631.\n",
      "Labriola MK, Zhu J, Gupta RT, et al. Characterization of tumor mutation burden, PD-L1 and DNA repair genes to assess relationship to immune checkpoint inhibitors response in metastatic renal cell carcinoma. J Immunother Cancer. 2020;8(1):e000319.\n",
      "LaFranchi S. Thyroid nodules and cancer in children. UpToDate [online serial]. Waltham, MA: UpToDate; reviewed January 2016.\n",
      "Lagautriere F, Valvano L, Chaazl M, et al. Prognostic factors in colorectal adenocarcinoma. Ann Ital Chir. 1998;69(4):491-497.\n",
      "Lam VK, Tran HT, Banks KC, et al. Targeted tissue and cell-free tumor DNA sequencing of advanced lung squamous-cell carcinoma reveals clinically significant prevalence of actionable alterations. Clin Lung Cancer. 2019;20(1):30-36.\n",
      "Lamb AD, Lawrence MG, Sandhu S. Practical Polling for Prostate Cancer: AR-V7-based Treatment Selection. Eur Urol. 2017;71(6):883-885. \n",
      "Lamerz R, Stieber P, Fateh-Moghadam A. Serum marker combinations in human breast cancer. In Vivo. 1993;7(6B):607-613.\n",
      "Lampson BL, Brown JR. Are BTK and PLCG2 mutations necessary and sufficient for ibrutinib resistance in chronic lymphocytic leukemia?. Expert Rev Hematol. 2018;11(3):185-194.\n",
      "Lapkus O, Gologan O, Liu Y, et al. Determination of sequential mutation accumulation in pancreas and bile duct brushing cytology. Mod Pathol. 2006;19(7):907-913. \n",
      "Laufer-Geva S, Rozenblum AB, Twito T, et al. The clinical impact of comprehensive genomic testing of circulating cell-free DNA in advanced lung cancer. J Thorac Oncol. 2018;13(11):1705-1716.\n",
      "Laukhtina E, Shim SR, Mori K, et al. Diagnostic accuracy of novel urinary biomarker tests in non-muscle-invasive bladder cancer: A systematic review and network meta-analysis [published correction appears in Eur Urol Oncol. 2022 Jan 19;:]. Eur Urol Oncol. 2021;4(6):927-942.\n",
      "Laurent-Puig P, Grisoni ML, Heinemann V, et al. Validation of miR-31-3p expression to predict cetuximab efficacy when used as first-line treatment in RAS wild-type metastatic colorectal cancer. Clin Cancer Res. 2019;25(1):134-141.\n",
      "Lechner MG, Hershman JM. Thyroid nodules and cancer in the elderly. In: Feingold KR, Anawalt B, Boyce A, et al., eds. Endotext. South Dartmouth, MA: MDText.com, Inc.; January 11, 2022.\n",
      "Lee HJ, Yousefi K, Haddad Z, et al. Evaluation of a genomic classifier in radical prostatectomy patients with lymph node metastasis. Res Rep Urol. 2016;8:77-84.\n",
      "Lee J, Hue SSS, Ko SQ, et al. Clinical impact of the cell-of-origin classification based on immunohistochemistry criteria and Lymph2Cx of diffuse large B-Cell lymphoma patients in a South-east Asian population: A single center experience and review of the literature. Expert Rev Hematol. 2019;12(12):1095-1105.\n",
      "Lehmann BD, Jovanovic B, Chen X, et al. Refinement of triple-negative breast cancer molecular subtypes: Implications for neoadjuvant chemotherapy selection. PLoS One. 2016;11(6):e0157368.\n",
      "Lehmann BD, Pietenpol JA. Clinical implications of molecular heterogeneity in triple negative breast cancer. Breast. 2015;24 Suppl 2:S36-S40.\n",
      "Lehmann BD, Pietenpol JA. Identification and use of biomarkers in treatment strategies for triple-negative breast cancer subtypes. J Pathol. 2014;232(2):142-150.\n",
      "Lei Y, Li Z, Qi L, et al. The prognostic role of Ki-67/MIB-1 in upper urinary-tract urothelial carcinomas: A systematic review and meta-analysis. J Endourol. 2015;29(11):1302-1308.\n",
      "Leibrand CR, Price DK, Figg WD. Androgen receptor splice variant 7 (AR-V7) and drug efficacy in castration-resistant prostate cancer: Biomarker for treatment selection exclusion or inclusion? Cancer Biol Ther. 2016;17(5):467-469. \n",
      "Leighl NB, Page RD, Raymond VM, et al. Clinical utility of comprehensive cell-free DNA analysis to identify genomic biomarkers in patients with newly diagnosed metastatic non-small cell lung cancer. Clin Cancer Res. 2019;25(15):4691-4700.\n",
      "Lein M, Stephan C, Jung K, et al. Relation of free PSA/total PSA in serum for differentiating between patients with prostatic cancer and benign hyperplasia of the prostate: Which cutoff should be used? Cancer Invest. 1998;16(1):45-49.\n",
      "Leung KK, Ross WA, Evans D, et al. Pancreatic cystic neoplasm: The role of cyst morphology, cyst fluid analysis, and expectant management. Ann Surg Oncol. 2009;16(10):2818-2824.\n",
      "Levine M, Moutquin JM, Walton R, Feightner J. Chemoprevention of breast cancer. A joint guideline from the Canadian Task Force on Preventive Health Care and the Canadian Breast Cancer Initiative's Steering Committee on Clinical Practice Guidelines for the Care and Treatment of Breast Cancer. CMAJ. 2001;164(12):1681-1690.\n",
      "Levy C, Lymp J, Angulo P, et al. The value of serum CA 19-9 in predicting cholangiocarcinomas in patients with primary sclerosing cholangitis. Dig Dis Sci. 2005;50(9):1734-1740.\n",
      "Li BT, Janku F, Jung B, et al. Ultra-deep next-generation sequencing of plasma cell-free DNA in patients with advanced lung cancers: Results from the Actionable Genome Consortium. Ann Oncol. 2019;30(4):597-603.\n",
      "Li X, Warren S, Pelekanou V, et al. Immune profiling of pre- and post-treatment breast cancer tissues from the SWOG S0800 neoadjuvant trial. J Immunother Cancer. 2019;7(1):88.\n",
      "Liang DH, Ensor JE, Liu ZB, et al. Cell-free DNA as a molecular tool for monitoring disease progression and response to therapy in breast cancer patients. Breast Cancer Res Treat. 2016;155(1):139-149.\n",
      "Lianos GD, Glantzounis GK, Bali CD, et al. Identification of novel genes by whole-exome sequencing can improve gastric cancer precision oncology. Future Oncol. 2017;13(10):883-892.\n",
      "Litchman GH, Prado G, Teplitz RW, Rigel D. A systematic review and meta-analysis of gene expression profiling for primary cutaneous melanoma prognosis. Skin. 2020;4(3):221-237.\n",
      "Liebman HA, Furie BC, Tong MJ, et al. Des-gamma-carboxy (abnormal) prothrombin as a serum marker of primary hepatocellular carcinoma. NEJM. 1984;310(22):1427-1431.\n",
      "LiÃ¨vre A, Bachet JB, Boige V, et al. KRAS mutations as an independent prognostic factor in patients with advanced colorectal cancer treated with cetuximab. J Clin Oncol. 2008;26(3):374-379.\n",
      "LiÃ¨vre A, Bachet JB, Le Corre D, et al. KRAS mutation status is predictive of response to cetuximab therapy in colorectal cancer. Cancer Res. 2006;66(8):3992-3995.\n",
      "Lin PC, Giannopoulou EG, Park K, et al. Epigenomic alterations in localized and advanced prostate cancer. Neoplasia. 2013;15(4):373-383.\n",
      "Lin X, Finkelstein SD, Zhu B, Silverman JF. Molecular analysis of multifocal papillary thyroid carcinoma. J Mol Endocrinol. 2008;41(4):195-203.\n",
      "Lin X, Zhu B, Finkelstein SD, et al. Significance of loss of heterozygosity in predicting axillary lymph node metastasis of invasive ductal carcinoma of the breast. Appl Immunohistochem Mol Morphol. 2012;20(2):116-123.\n",
      "Linardou H, Dahabreh IJ, Kanaloupiti D, et al. Assessment of somatic k-RAS mutations as a mechanism associated with resistance to EGFR-targeted agents: A systematic review and meta-analysis of studies in advanced non-small-cell lung cancer and metastatic colorectal cancer. Lancet Oncol. 2008;9(10):962-972.\n",
      "Lindeman NI, Cagle PT, Aisner DL, et al. Updated molecular testing guideline for the selection of lung cancer patients for treatment with targeted tyrosine kinase inhibitors: Guideline from the College of American Pathologists, the International Association for the Study of Lung Cancer, and the Association for Molecular Pathology.  Arch Pathol Lab Med. 2018;142(3):321-346.\n",
      "Lissa D, Robles AI. Comprehensive genomic analysis of circulating tumor DNA for patients with advanced non-small cell lung cancer. Ann Transl Med. 2019;7(5):80. \n",
      "List AF, Spier CM. Multidrug resistance in acute leukemia: A conserved physiologic function. Leuk Lymphoma. 1992;8(1-2):9-14.\n",
      "Lithwick-Yanai G, Dromi N, Shtabsky A, et. al. Multicentre validation of a microRNA-based assay for diagnosing indeterminate thyroid nodules utilizing fine needle aspirate smears. J Clin Pathol. 2017;70(6):500-507.\n",
      "Little J, Wilson B, Carter R, et al. Multigene panels in prostate cancer risk assessment. Evidence Report/Technology Assessment No. 209. Prepared by the McMaster University Evidence-based Practice Center (EPC) under contract to the Agency for Healthcare Research and Quality (AHRQ), Contract No. 290-2007-10060-1. AHRQ Publication No. 12-E020-EF. Rockville, MD: Agency for Healthcare Research and Quality (AHRQ); 2012.\n",
      "Littrup PJ. Future benefits and cost-effectiveness of prostate carcinoma screening. American Cancer Society. Cancer. 1997;80(9):1864-1870.\n",
      "Liu Y, Cui S, Li W, et al. PAX3 is a biomarker and prognostic factor in melanoma: Database mining. Oncol Lett. 2019;17(6):4985-4993.\n",
      "Lobo JM, Dicker AP, Buerki C, et al. Evaluating the clinical impact of a genomic classifier in prostate cancer using individualized decision analysis. PLoS One. 2015;10(3):e0116866.\n",
      "Locker GY, Hamilton S, Harris J, et al; ASCO. ASCO 2006 update of recommendations for the use of tumor markers in gastrointestinal cancer. J Clin Oncol. 2006;24(33):5313-5327.\n",
      "Lockney NA, Henderson R, Swarts SG, et al. Circulating cell-free DNA correlates with body integral dose and radiation modality in prostate cancer. Int J Part Ther. 2020;7(2):21-30. \n",
      "Lockney NA, Henderson R, Swarts SG, et al. Measuring radiation toxicity using circulating cell-free DNA in prostate cancer patients. Int J Part Ther. 2021;8(3):28-35. .\n",
      "Longoria TC, Ueland FR, Zhang Z, et al. Clinical performance of a multivariate index assay for detecting early-stage ovarian cancer. Am J Obstet Gynecol. 2014;210(1):78.e1-e9.\n",
      "Lopez SE, Velasco DC, Amaro JAB., A systematic review on efficacy and economic impact of genetic tests in breast cancer and depression treatments. Madrid, Spain: Plan de Calidad para el SNS del MSPSI, Unidad de EvaluaciÃ³n de TecnologÃ­as Sanitarias, Agencia LaÃ­n Entralgo; 2010.\n",
      "Lopresti A, Malergue F, Bertucci F, et al. Sensitive and easy screening for circulating tumor cells by flow cytometry. JCI Insight. 2019;5.\n",
      "Loren D, Kowalski T, Siddiqui A, et al. Influence of integrated molecular pathology test results on real-world management decisions for patients with pancreatic cysts: Analysis of data from a national registry cohort. Diagn Pathol. 2016;11(1):5.\n",
      "Lotan Y, Choueiri TK. Clinical presentation, diagnosis, and staging of bladder cancer. UpToDate [online serial]. Waltham, MA: UpToDate; reviewed December 2013. \n",
      "Loupakis F, Sharma S, Derouazi M, et al. Detection of molecular residual disease using personalized circulating tumor DNA assay in patients with colorectal cancer undergoing resection of metastases. JCO Precis Oncol. 2021;5:PO.21.00101.\n",
      "Lowe KA, Shah C, Wallace E, et al. Effects of personal characteristics on serum CA125, mesothelin, and HE4 levels in healthy postmenopausal women at high-risk for ovarian cancer. Cancer Epidemiol Biomarkers Prev. 2008;17(9):2480-2487.\n",
      "Lu KH, Patterson AP, Wang L, et al. Selection of potential markers for epithelial ovarian cancer with gene expression arrays and recursive descent partition analysis. Clin Cancer Res. 2004;10(10):3291-3300.\n",
      "LÃ¼ftner D, Possinger K. Nuclear matrix proteins as biomarkers for breast cancer. Expert Rev Mol Diagn. 2002;2(1):23-31.\n",
      "LungLifeAI. Lung cancer publications/links. Thousand Oaks, CA: LungLifeAI; 2022. Available at: https://lunglifeai.com/wp-content/uploads/2021/07/IASLC-Liquid-Biopsy-Abstract_Book-FINAL-2020.10.01-GS.pdf#page=52. Accessed July 8, 2022.\n",
      "Luo J. Development of AR-V7 as a putative treatment selection marker for metastatic castration-resistant prostate cancer. Asian J Androl. 2016;18(4):580-585.\n",
      "Lynch JA, Rothney MP, Salup RR, et al. Improving risk stratification among veterans diagnosed with prostate cancer: Impact of the 17-gene prostate score assay. Am J Manag Care. 2018;24(1 Suppl):S4-S10.\n",
      "Ma XJ, Salunga R, Dahiya S, et al. A five-gene molecular grade index and HOXB13:IL17BR are complementary prognostic factors in early stage breast cancer. Clin Cancer Res. 2008;14(9):2601-2608.\n",
      "Ma XJ, Wang Z, Ryan PD, et al. A two-gene expression ratio predicts clinical outcome in breast cancer patients treated with tamoxifen. Cancer Cell. 2004;5(6):607-616.\n",
      "Magbanua MJM, Swigart LB, Wu H-T, et al. Circulating tumor DNA in neoadjuvant-treated breast cancer reflects response and survival. Ann Oncol. 2021;32(2):229-239.\n",
      "Maheshwari V, Tsung A, Lin Y, et al. Analysis of loss of heterozygosity for tumor-suppressor genes can accurately classify and predict the clinical behavior of mucinous tumors arising from the appendix. Ann Surg Oncol. 2006;13(12):1610-1616. \n",
      "Makar AP, Kristensen GB, Bormer OP, Trope CG. CA 125 measured before a second-look laparotomy us an independent prognostic factor for survival in patients with epithelial ovarian cancer. Gynecol Oncol. 1992;45(3):323-328.\n",
      "Malhotra N, Jackson SA, Freed LL, et al. The added value of using mutational profiling in addition to cytology in diagnosing aggressive pancreaticobiliary disease: Review of clinical cases at a single center. BMC Gastroenterol. 2014;14:135. \n",
      "Mamounas EP, Russell CA, Lau A, et al. Clinical relevance of the 21-gene Recurrence ScoreÂ® assay in treatment decisions for patients with node-positive breast cancer in the genomic era. NPJ Breast Cancer. 2018;4:27.\n",
      "Marchetti MA, Bartlett EK, Dusza SW, Bichakjian CK. Use of a prognostic gene expression profile test for T1 cutaneous melanoma: Will it help or harm patients? J Am Acad Dermatol. 2019;80(6):e161-e162.\n",
      "Marchetti MA, Coit DG, Dusza SW, et al. Performance of gene expression profile tests for prognosis in patients with localized cutaneous melanoma: A systematic review and meta-analysis. JAMA Dermatol. 2020;156(9):953-962.\n",
      "Marchetti MA, Dusza SW, Bartlett EK. Problematic methodology in a systematic review and meta-analysis of DecisionDx-Melanoma. J Am Acad Dermatol. 2020;83(5):e357-e358.\n",
      "Marchionni L, Wilson RF, Marinopoulos SS, et al. Impact of gene expression profiling on breast cancer outcomes. Evidence Report/Technology Assessment No. 160. Prepared by the Johns Hopkins University Evidence-based Practice Center for the Agency for Healthcare Research and Quality (AHRQ). AHRQ Publication No. 08-E002. Rockville, MD: AHRQ; January 2008.\n",
      "Marchionni L, Wilson RF, Wolff AC, et al. Systematic review: Gene expression profiling assays in early-stage breast cancer. Ann Intern Med. 2008;148(5):358-369.\n",
      "Marks E, Caruso HG, Kurley SJ, et al. Establishing an evidence-based decision point for clinical use of the 31-gene expression profile test in cutaneous melanoma. Skin. 2019;3(4):239-249.\n",
      "Marks LS, Fradet Y, Deras IL, et al. PCA3 molecular urine assay for prostate cancer in men undergoing repeat biopsy. Urology. 2007;69(3):532-535.\n",
      "MartÃ­nez-FÃ©rez IM, MÃ¡rquez-PelÃ¡ez S, Isabel-GÃ³mez R, BeltrÃ¡n-Calvo C.. Pruebas genÃ³micas para el pronÃ³stico de pacientes con cÃ¡ncer de mama. MammaPrintÂ® y Oncotype DXÂ®. [Prognostic genomic tests in early breast cancer: MammaPrintÂ® and Oncotype DXÂ®]. Summary.  AETSA 2013/3. Seville, Spain: Andalusian Agency for Health Technology Assessment (AETSA). 2014.\n",
      "Martinez-Useros J, Garcia-Foncillas J. The role of BRCA2 mutation status as diagnostic, predictive, and prognosis biomarker for pancreatic cancer. Biomed Res Int. 2016;2016:1869304.\n",
      "Marzouk K, Ehdaie B, Vertosick E, et al. Developing an effective strategy to improve the detection of significant prostate cancer by combining the 4Kscore and multiparametric MRI. Urol Oncol. 2019;37(10):672-677.\n",
      "Mathieu MC, Mazouni C, Kesty NC, et al. Breast Cancer Index predicts pathological complete response and eligibility for breast conserving surgery in breast cancer patients treated with neoadjuvant chemotherapy. Ann Oncol. 2012;23(8):2046-2052.\n",
      "Matorras R, Rodriguez-Escuderoi FJ, Diez J, et al. Monitoring endometrial adenocarcinoma with a four tumor marker combination. CA 125, squamous cell carcinoma antigen, CA 19.9 and CA 15.3. Acta Obstet Gynecol Scand. 1992;71(6):458-464.\n",
      "Matritech, Inc. [website]. Newton, MA; Matritech; 2009. Available at: http://www.matritech.com/. Accessed on January 27, 2009.\n",
      "Matsuo T, Kuriyama K, Miyazaki Y, et al. The percentage of myeloperoxidase-positive blast cells is a strong independent prognostic factor in acute myeloid leukemia, even in the patients with normal karyotype. Leukemia. 2003;17:1538-1543.\n",
      "Maugeri-Sacca M, Coppola V, De Maria R, Bonci D. Functional role of microRNAs in prostate cancer and therapeutic opportunities. Crit Rev Oncog. 2013;18(4):303-315.\n",
      "Mazurek A. Niklinski J. Laudanski T. et al. Clinical tumour markers in ovarian cancer. Eur J Cancer Prev. 1998;7(1):23-35.\n",
      "McCoach CE, Blakely CM, Banks KC, et al. Clinical utility of cell-free DNA for the detection of ALK fusions and genomic mechanisms of ALK inhibitor resistance in non-small cell lung cancer. Clin Cancer Res. 2018;24(12):2758-2770.\n",
      "Meador CB, Oxnard GR. Effective Cancer Genotyping-Many Means to One End. Clin Cancer Res. 2019;25(15):4583-4585.\n",
      "Mehta A, Carpenter JT. How do I recommend extended adjuvant hormonal therapy? Curr Treat Options Oncol. 2014;15(1):55-62.\n",
      "Mejia A, Schulz S, Hyslop T, et al. GUCY2C reverse transcriptase PCR to stage pN0 colorectal cancer patients. Expert Rev Mol Diagn. 2009;9(8):777-785.\n",
      "Mejia A, Waldmana SA. Previstage GCC test for staging patients with colorectal cancer. Expert Rev Mol Diagn. 2008;8(5):571-578.\n",
      "Melander O, Belting M, Manjer J, et al. Validation of plasma proneurotensin as a novel biomarker for the prediction of incident breast cancer. Cancer Epidemiol Biomarkers Prev. 2014;23(8):1672-1676.\n",
      "Melander O, Orho-Melander M, Manjer J, et al. Stable peptide of the endogenous opioid enkephalin precursor and breast cancer risk. J Clin Oncol. 2015;33(24):2632-2638.\n",
      "Meleth S, Reeder-Hayes K, Ashok M, et al. Technology assessment of molecular pathology testing for the estimation of prognosis for common cancers. Technology Assessment Report. Prepared for the RTI International-University of North Carolina at Chapel Hill Evidence Based Practice Center (RTI-UNC EPC) for the Agency for Healthcare Research and Quality (AHRQ), Contract No. HHSA2902007100561.Rockville, MD: Agency for Healthcare Research and Quality (AHRQ); 2014.\n",
      "Meleth S, Whitehead N, Swinson T, Lux L. Technology assessment on genetic testing or molecular pathology testing of cancers with unknown primary site to determine origin. Technology Assessment Report. Final. Prepared by RTI International-University of North Carolina at Chapel Hill Evidence-based Practice Center for the Agency for Healthcare Research and Quality (AHRQ) under contract no. HHSA290200710057I #8. Project ID: CANU5011. Rockville, MD: AHRQ; February 20, 2013. \n",
      "Merker JD, Oxnard GR, Compton C, et al. Circulating Tumor DNA Analysis in Patients With Cancer: American Society of Clinical Oncology and College of American Pathologists Joint Review. J Clin Oncol. 2018;36(16):1631-1641.\n",
      "Messick CA, Sanchez J, Dejulius KL, et al. CEACAM-7: A predictive marker for rectal cancer recurrence. Surgery. 2010;147(5):713-719.\n",
      "Metcalf AM, Spurdle AB. Endometrial tumour BRAF mutations and MLH1 promoter methylation as predictors of germline mismatch repair gene mutation status: A literature review. Fam Cancer. 2014;13(1):1-12.\n",
      "Mhawech P, Dulguerov P, Tschanz E, et al. Motility-related protein-1 (MRP-1/CD9) expression can predict disease-free survival in patients with squamous cell carcinoma of the head and neck. Br J Cancer. 2004;90(2):471-475.\n",
      "Mhawech P, Herrmann F, Coassin M, et al. Motility-related protein 1 (MRP-1/CD9) expression in urothelial bladder carcinoma and its relation to tumor recurrence and progression. Cancer. 2003;98(8):1649-1657.\n",
      "Michalopoulos SN, Kella N, Payne R, et al.; PRO-ACT Study Group. Influence of a genomic classifier on post-operative treatment decisions in high-risk prostate cancer patients: Results from the PRO-ACT study. Curr Med Res Opin. 2014;30(8):1547-1556.\n",
      "Micronoma. Oncobiota Platform. San Diego, California: Micronoma; Available at: https://www.micronoma.com/oncobiota/. Accessed July 31, 2023.\n",
      "Midthun DE. Overview of the initial evaluation, treatment and prognosis of lung cancer. UpToDate [online serial]. Waltham, MA: UpToDate; reviewed January 2017.\n",
      "Mimori K, Kataoka A, Yoshinaga K, et al. Identification of molecular markers for metastasis-related genes in primary breast cancer cells. Clin Exp Metastasis. 2005;22(1):59-67.\n",
      "Ministerio de Salud y ProtecciÃ³n Social, Departamento Administrativo de Ciencia TecnologÃ­a e InnovaciÃ³n en Salud (Colciencias), Instituto Nacional de CancerologÃ­a Empresa Social del Estado-Fedesalud; Instituto de EvaluaciÃ³n TechnolÃ³gica en Salud (IETS), GuÃ­a de PrÃ¡ctica ClÃ­nica para la detecciÃ³n temprana, tratamiento integral, seguimiento y rehabilitaciÃ³n del cÃ¡ncer de mama. GuÃ­a No. GPC-2013-19. BogotÃ¡, Colombia: Instituto Nacional de CancerologÃ­a ESE; 2013.\n",
      "Mirsky R, Prado G, Svoboda R, et al. Management decisions made by physician assistants and nurse practitioners in cutaneous malignant melanoma patients: Impact of a 31-gene expression profile test. J Drugs Dermatol. 2018;17(11):1220-1223.\n",
      "Misawa K, Imai A, Matsui H, et al. Identification of novel methylation markers in HPV-associated oropharyngeal cancer: Genome-wide discovery, tissue verification and validation testing in ctDNA. Oncogene. 2020;39(24):4741-4755.\n",
      "Miyake M, Nakano K, Ieki Y, et al. Motility related protein 1 (MRP-1/CD9) expression: inverse correlation with metastases in breast cancer. Cancer Res. 1995;55(18):4127-4131.\n",
      "Mlecnik B, Bifulco C, Bindea G, et al. Multicenter international society for immunotherapy of cancer study of the consensus Immunoscore for the prediction of survival and response to chemotherapy in stage III colon cancer. J Clin Oncol. 2020;38(31):3638-3651.\n",
      "Modlin IM, Drozdov I, Alaimo D, et al. A multianalyte PCR blood test outperforms single analyte ELISAs (chromogranin A, pancreastatin, neurokinin A) for neuroendocrine tumor detection. Endocr Relat Cancer. 2014;21(4):615-628.\n",
      "Modlin IM, Frilling A, Salem RR, et al. Blood measurement of neuroendocrine gene transcripts defines the effectiveness of operative resection and ablation strategies. Surgery. 2016;159(1):336-347. \n",
      "Modlin IM, Kidd M, Bodei L, et al. The clinical utility of a novel blood-based multi-transcriptome assay for the diagnosis of neuroendocrine tumors of the gastrointestinal tract. Am J Gastroenterol. 2015;110(8):1223-1232.\n",
      "Mogenson O. Prognostic value of CA 125 in advanced ovarian cancer. Gynecol Oncol. 1992;44:207-212.\n",
      "Montgomery RB, Plymate SR. AR-V7 Protein in Circulating Tumor Cells-The Decider for Therapy? JAMA Oncol. 2016;2(11):1450-1451. \n",
      "Monzon FA, Dumur CI, Lyons-Weiler M, et al. Validation of a gene expression-based tissue of origin test applied to poorly differentiated and undifferentiated cancers. Abstract presented at the 13th Annual Meeting of the Association for Molecular Pathology, Los Angeles, CA, November 7-10, 2007.\n",
      "Monzon FA, Koen TJ. Diagnosis of metastatic neoplasms: molecular approaches for identification of tissue of origin. Arch Pathol Lab Med. 2010;134(2):216-224.\n",
      "Monzon FA, Lyons-Weiler M, Buturovic LJ, et al. Multicenter validation of a 1,550-gene expression profile for identification of tumor tissue of origin. J Clin Oncol. 2009;27(15):2503-2508.\n",
      "Moore RG, Brown AK, Miller MC, et al. The use of multiple novel tumor biomarkers for the detection of ovarian carcinoma in patients with a pelvic mass. Gynecol Oncol. 2008;108(2):402-408.\n",
      "Morabito A, Magnani E, Gion M, et al. Prognostic and predictive indicators in operable breast cancer. Clin Breast Cancer. 2003;3(6):381-390.\n",
      "Morais M, Pinto DM, Machado JC, Carneiro S. ctDNA on liquid biopsy for predicting response and prognosis in locally advanced rectal cancer: A systematic review. Eur J Surg Oncol. 2022;48(1):218-227.\n",
      "Morash C, Tey R, Agbassi C, et al. Active surveillance for the management of localized prostate cancer: Guideline recommendations. Can Urol Assoc J. 2015;9(5-6):171-178.\n",
      "Moroni M, Veronese S, Benvenuti S, et al. Gene copy number for epidermal growth factor receptor (EGFR) and clinical response to antiEGFR treatment in colorectal cancer: A cohort study. Lancet Oncol. 2005;6(5):279-286.\n",
      "Moschini M, Spahn M, Mattei A, et al. Incorporation of tissue-based genomic biomarkers into localized prostate cancer clinics. BMC Med. 2016;14:67.\n",
      "Moss J, Zick A, Grinshpun A, et al. Circulating breast-derived DNA allows universal detection and monitoring of localized breast cancer. Ann Oncol. 2020;31(3):395-403.\n",
      "Mottet N, Bellmunt J, Briers E, et al. Guidelines on Prostate Cancer. Arnhem, The Netherlands: European Association of Urology; 2015.\n",
      "Mottok A, Wright G, Rosenwald A, et al. Molecular classification of primary mediastinal large B-cell lymphoma using routinely available tissue specimens. Blood. 2018;132(22):2401-2405.\n",
      "Mulcahy N. Medscape Medical News: Oncology. 'Very big deal': NCCN okays watching more prostate cancer. January 13, 2016. Available at: http://www.medscape.com/viewarticle/857151. Accessed March 1, 2016.\n",
      "Muller CY. Doctor, should I get this new ovarian cancer test-OVA1? Obstet Gynecol. 2010;116(2 Pt 1):246-247.\n",
      "Muller V, Pantel K. Bone marrow micrometastases and circulating tumor cells: Current aspects and future perspectives. Breast Cancer Res. 2004;6(6):258-261.\n",
      "Mundy L, Hiller J. NMP22 BladderChek Diagnostic test for bladder cancer. Horizon Scanning Prioritising Summary Update. Canberra, ACT: Department of Health and Ageing; November 2009.\n",
      "Mundy L, Merlin T, Hiller J. Diagnostic tests for ovarian cancer. Horizon Scanning Technology Prioritising Summary. Canberra, ACT: Department of Health and Ageing; 2010.\n",
      "Mundy L. Gene expression profiling of breast cancer. Technology Brief. Health Policy Advisory Committee on Technology (HealthPACT) Emerging Health Techology. Brisbane, QLD; Queensland Health; May 2012. \n",
      "Muntener M, Epstein JI, Hernandez DJ, et al. Prognostic significance of Gleason score discrepancies between needle biopsy and radical prostatectomy. Eur Urol. 2008;53(4):767-775; discussion 775-776.\n",
      "Murakami K, Pagano I, Furuya H, et al. Clinical utility of Oncuriaâ¢, a multiplexed liquid biopsy for the non-invasive detection of bladder cancer -- A pilot study. Diagnostics (Basel). 2022;12(1):131.\n",
      "Murat A, Migliavacca E, Gorlia T, et al. Stem cell-related \"self-renewal\" signature and high epidermal growth factor receptor expression associated with resistance to concomitant chemoradiotherapy in glioblastoma. J Clin Oncol. 2008;26(18):3015-3024.\n",
      "Murray A, Chapman CJ, Healey G, et al. Technical validation of an autoantibody test for lung cancer. Ann Oncol. 2010;21(8):1687-1693.\n",
      "Murray DH, Symonds EL, Young GP, et al. Relationship between post-surgery detection of methylated circulating tumor DNA with risk of residual disease and recurrence-free survival. J Cancer Res Clin Oncol. 2018;144(9):1741-1750.\n",
      "Murray NP, Calaf GM, Badinez L, et al. P504S expressing circulating prostate cells as a marker for prostate cancer. Oncol Rep. 2010;24(3):687-692.\n",
      "Musher BL, Melson JE, Amato G, et al, Evaluation of circulating tumor DNA for methylated BCAT1 and IKZF1 to detect recurrence of stage II/stage III colorectal cancer (CRC). Cancer Epid Biomarkers Prev. 2020;29(12):2702-2709.\n",
      "Mutch DG. Can molecular diagnostics usher in a new era for screening, diagnosis, and treatment of ovarian cancer? Obstet Gynecol. 2014;124(5):870-872.\n",
      "Myers ER, Havrilesky LJ, Kulasingam SL, et al. Genomic tests for ovarian cancer detection and management. Evidence Report/Technology Assessment 145. Prepared by the Duke Evidence-Based Practice Center for the Agency for Healthcare Research and Quality (AHRQ). AHRQ Publication No. 07-E0001. Rockville, MD: AHRQ; October 2006. \n",
      "Nagaoka S, Yatsuhashi H, Hamada H, et al. The des-gamma-carboxy prothrombin index is a new prognostic indicator for hepatocellular carcinoma. Cancer. 2003;98(12):2671-2677.\n",
      "Nair VS, Maeda LS, Ioannidis JP. Clinical outcome prediction by microRNAs in human cancer: A systematic review. J Natl Cancer Inst. 2012;104(7):528-540.\n",
      "Nakamura S, Nouso K, Sakaguchi K, et al. Sensitivity and specificity of des-gamma-carboxy prothrombin for diagnosis of patients with hepatocellular carcinomas varies according to tumor size. Am J Gastroenterol. 2006;101(9):2038-2043.\n",
      "Narod SA, Iqbal J, Giannakeas V, et al. Breast cancer mortality after a diagnosis of ductal carcinoma in situ. JAMA Oncol. 2015;1(7):888-896.\n",
      "Nathan P, Cohen V, Coupland S, et al.; United Kingdom Uveal Melanoma Guideline Development Working Group. Uveal Melanoma UK National Guidelines. Eur J Cancer. 2015;51(16):2404-2412.\n",
      "National Cancer Institute at the National Institutes of Health (NCI). NCI dictionary of cancer terms. Available at: https://www.cancer.gov/publications/dictionaries/cancer-terms/def/serum-tumor-marker-test. Accessed April 21, 2022.\n",
      "National Cancer Institute (NCI). Ovarian Cancer (PDQ): Screening. Health Professional Version. Bethesda, MD: NCI; updated February 20, 2004. \n",
      "National Cancer Institute (NCI). Questions and answers: OvaCheck â¢ and NCI/FDA Ovarian Cancer Clinical Trials using proteomics technology. NCI News. Bethesda, MD: NCI; March 22, 2004.Available at: http://www.cancer.gov/newscenter/pressreleases/ProteomicsOvarian. Accessed June 4, 2004.\n",
      "National Cancer Institute. Carcinoma of unknown primary treatment (PDQÂ®). Bethesda, MD: NCI; 2008. \n",
      "National Cancer Institute. Hormone therapy with or without combination chemotherapy in treating women who have undergone surgery for node-negative breast cancer (the TAILORx Trial). ClinicalTrials.gov ID. NCT00310180. Bethesda, MD: National Library of Medicine; updated August 11, 2009.\n",
      "National Collaborating Centre for Cancer. Diagnosis and management of metastatic malignant disease of unknown primary origin. Developed for NICE. Cardiff, UK: National Collaborating Centre for Cancer; July 2010. \n",
      "National Comprehensive Cancer Network (NCCN). Acute myeloid leukemia. NCCN Clinical Practice Guidelines in Oncology v.1.2009. Fort Washington, PA: NCCN; September 2008. \n",
      "National Comprehensive Cancer Network (NCCN). Acute myeloid leukemia. NCCN Clinical Practice Guidelines in Oncology v.3.2020. Fort Washington, PA: NCCN 2020\n",
      "National Comprehensive Cancer Network (NCCN). B-cell lymphomas. NCCN Clinical Practice Guidelines in Oncology, version 1.2019. Fort Washington, PA: NCCN; 2018.\n",
      "National Comprehensive Cancer Network (NCCN). Biomarkers Compendium. NCCN: Fort Washington, PA: NCCN; 2019.\n",
      "National Comprehensive Cancer Network. CA 19-9 expression. NCCN Biomarkers Compendium. Plymouth Meeting, PA:NCCN; 2021.\n",
      "National Comprehensive Cancer Network (NCCN). Bladder cancer. NCCN Clinical Practice Guidelines in Oncology, Version 1.2014. Fort Washington, PA: NCCN; 2014.\n",
      "National Comprehensive Cancer Network (NCCN). Bladder cancer. NCCN Clinical Practice Guidelines in Oncology, Version 1.2019. Fort Washington, PA: NCCN; 2019.\n",
      "National Comprehensive Cancer Network (NCCN). Bladder cancer. NCCN Clinical Practice Guidelines in Oncology, Version 6.2020. Fort Washington, PA: NCCN; 2020.\n",
      "National Comprehensive Cancer Network (NCCN). Bone cancer. Clinical Practice Guideline in Oncology, Version 1.2020. Fort Washington, PA: NCCN: 2020.\n",
      "National Comprehensive Cancer Network (NCCN). Breast cancer. NCCN Clinical Practice Guidelines in Oncology, Version 2.2021. Plymouth Meeting, PA: NCCN; 2021.\n",
      "National Comprehensive Cancer Network (NCCN). Breast cancer. NCCN Clinical Practice Guidelines in Oncology. Version 2.2011. Fort Washington, PA: NCCN; 2011.\n",
      "National Comprehensive Cancer Network (NCCN). Breast cancer. NCCN Clinical Practice Guidelines in Oncology, version 2.2013. Fort Washington, PA: NCCN; 2013.\n",
      "National Comprehensive Cancer Network (NCCN). Breast cancer. NCCN Clinical Practice Guidelines in Oncology, version 2.2015. Fort Washington, PA: NCCN; 2015.\n",
      "National Comprehensive Cancer Network (NCCN). Breast cancer. NCCN Clinical Practice Guidelines in Oncology, Version 2.2018. Fort Washington, PA: NCCN; 2018.\n",
      "National Comprehensive Cancer Network (NCCN). Breast cancer. NCCN Clinical Practice Guidelines in Oncology, Version 1.2018. Fort Washington, PA: NCCN; 2018.\n",
      "National Comprehensive Cancer Network (NCCN). Breast cancer. NCCN Clinical Practice Guidelines in Oncology, Version 4.2018. Fort Washington, PA: NCCN; 2018.\n",
      "National Comprehensive Cancer Network (NCCN). Breast cancer. NCCN Clinical Practice Guidelines in Oncology, Version 2.2020. Fort Washington, PA: NCCN; 2020.\n",
      "National Comprehensive Cancer Network (NCCN). Breast cancer. NCCN Clinical Practice Guidelines in Oncology, Version 2.2022. Plymouth Meeting, PA: NCCN; December 20, 2021.\n",
      "National Comprehensive Cancer Network (NCCN). Central nervous system cancers. NCCN Clinical Practice Guidelines in Oncology, version 2.2018. Fort Washington, PA: NCCN;  2018.\n",
      "National Comprehensive Cancer Network (NCCN). Chronic lymphocytic leukemia/small lymphocytic lymphoma. NCCN Clinical Practice Guidelines in Oncology, version 2.2019. Fort Washington, PA: NCCN, 2018.\n",
      "National Comprehensive Cancer Network (NCCN). Colon cancer. NCCN Clinical Practice Guidelines in Oncology, Version 2.2015. Fort Washington, PA: NCCN; 2015.\n",
      "National Comprehensive Cancer Network (NCCN). Colon cancer. NCCN Clinical Practice Guidelines in Oncology, Version 1.2018. Fort Washington, PA: NCCN; 2018.\n",
      "National Comprehensive Cancer Network (NCCN). Colon cancer. NCCN Clinical Practice Guidelines in Oncology, Version 3.2018. Fort Washington, PA: NCCN; 2018.\n",
      "National Comprehensive Cancer Network (NCCN). Colon cancer. NCCN Clinical Practice Guidelines in Oncology, Version 4.2018. Fort Washington, PA: NCCN; 2018.\n",
      "National Comprehensive Cancer Network (NCCN). Colon cancer. NCCN Clinical Practice Guidelines in Oncology, Version 2.2019. Fort Washington, PA: NCCN; 2019.\n",
      "National Comprehensive Cancer Network (NCCN). Colon cancer. NCCN Clinical Practice Guidelines in Oncology, Version 4.2020. Fort Washington, PA: NCCN; 2020.\n",
      "National Comprehensive Cancer Network (NCCN). Colon cancer. NCCN Clinical Practice Guidelines in Oncology, Version 3.2021. Plymouth Meeting, PA: NCCN; 2021.\n",
      "National Comprehensive Cancer Network. Clinical practice guideline: Colon Cancer. Version 1.2022. NCCN: Plymouth Meeting, PA.\n",
      "National Comprehensive Cancer Network. Clinical practice guideline: Colon Cancer. Version 3.2022. NCCN: Plymouth Meeting, PA.\n",
      "National Comprehensive Cancer Network (NCCN). Cutaneous melanoma. NCCN Clinical Practice Guidelines in Oncology, version 1.2019. Fort Washington, PA: NCCN; 2018.\n",
      "National Comprehensive Cancer Network (NCCN). Cutaneous melanoma. NCCN Clinical Practice Guidelines in Oncology, Version 2.2019. Fort Washington, PA: NCCN; 2019.\n",
      "National Comprehensive Cancer Network (NCCN). Cutaneous melanoma. NCCN Clinical Practice Guidelines in Oncology, Version 2.2019. Fort Washington, PA: NCCN; 2019.\n",
      "National Comprehensive Cancer Network (NCCN). Cutaneous melanoma. NCCN Clinical Practice Guidelines in Oncology, Version 1.2020. Fort Washington, PA: NCCN; 2020.\n",
      "National Comprehensive Cancer Network (NCCN). Cutaneous melanoma. NCCN Clinical Practice Guidelines in Oncology, Version 3.2020. Fort Washington, PA: NCCN; 2020.\n",
      "National Comprehensive Cancer Network (NCCN). Esophageal and esophagogastric junction cancers. NCCN Clinical Practice Guidelines in Oncology v.2.2011. Fort Washington, PA: NCCN; May, 2011. \n",
      "National Comprehensive Cancer Network (NCCN). Head and neck cancers. NCCN Clinical Practice Guidelines in Oncology, Version 2.2022. Plymouth Meeting, PA: NCCN, 2022.\n",
      "National Comprehensive Cancer Network (NCCN). Head and neck cancers. NCCN Clinical Practice Guidelines in Oncology, Version 1.2023. Plymouth Meeting, PA: NCCN, 2023.\n",
      "National Comprehensive Cancer Network (NCCN). Hepatobiliary cancer. NCCN Clinical Practice Guidelines in Oncology, Version 1.2021. Plymouth Meeting, PA: NCCN; 2021.\n",
      "National Comprehensive Cancer Network (NCCN). Hepatobiliary cancers. NCCN Clinical Practice Guidelines in Oncology v.1.2010. Fort Washington, PA: NCCN; 2010.\n",
      "National Comprehensive Cancer Network (NCCN). Hepatobiliary cancers. NCCN Clinical Practice Guidelines in Oncology, Version 2.2015. Fort Washington, PA: NCCN; 2015.\n",
      "National Comprehensive Cancer Network. Clinical practice guideline: Kidney cancer. Version 4.2022. NCCN: Plymouth Meeting, PA.\n",
      "National Comprehensive Cancer Network (NCCN). Mekinist. NCCN Drugs & Biologics Compendium. Fort Washington, PA: NCCN; 2018.\n",
      "National Comprehensive Cancer Network (NCCN). Melanoma. NCCN Clinical Practice Guidelines in Oncology, Version 2.2018. Fort Washington, PA: NCCN; 2018.\n",
      "National Comprehensive Cancer Network (NCCN). Melanoma. NCCN Drugs & Biologics Compendium. Fort Washington, PA: NCCN; 2018.\n",
      "National Comprehensive Cancer Network (NCCN). Melanoma: Cutaneous. NCCN Clinical Practice Guidelines in Oncology, Version 2.2022. Plymouth Meeting, PA: NCCN; 2021.\n",
      "National Comprehensive Cancer Network (NCCN). Multiple myeloma. NCCN Clinical Practice Guidelines in Oncology, Version 2.2013. Fort Washington: PA: NCCN; 2013.\n",
      "National Comprehensive Cancer Network (NCCN). Myelodysplastic syndromes. NCCN Clinical Practice Guidelines in Oncology, version 1.2020. Fort Washington, PA: NCCN; 2020.\n",
      "National Comprehensive Cancer Network (NCCN). Myelodysplastic syndromes. NCCN Clinical Practice Guidelines in Oncology, version 2.2020. Fort Washington, PA: NCCN; 2020.\n",
      "National Comprehensive Cancer Network (NCCN). Myeloproliferative neoplasms. NCCN Clinical Practice Guidelines in Oncology, version 2.2019. Fort Washington, PA: NCCN; 2019.\n",
      "National Comprehensive Cancer Network (NCCN). Myeloproliferative neoplasms. NCCN Clinical Practice Guidelines in Oncology, version 3.2019. Fort Washington, PA: NCCN; 2019.\n",
      "National Comprehensive Cancer Network (NCCN). Non-small cell lung cancer. NCCN Clinical Practice Guidelines in Oncology, Version 3.2019; Fort Washington, PA: NCCN; 2019.\n",
      "National Comprehensive Cancer Network (NCCN). Non-small cell lung cancer. NCCN Clinical Practice Guidelines in Oncology, Version 5.2019. Fort Washington, PA: NCCN; 2019.\n",
      "National Comprehensive Cancer Network (NCCN). Non-small cell lung cancer. NCCN Clinical Practice Guidelines in Oncology, Version 6.2020. Fort Washington, PA: NCCN; 2020.\n",
      "National Comprehensive Cancer Network (NCCN). Non-small cell lung cancer. NCCN Clinical Practice Guidelines in Oncology, Version 1.2023. Plymouth Meeting, PA: NCCN; December 22, 2022.\n",
      "National Comprehensive Cancer Network (NCCN). Occult primary (cancer of unknown primary [CUP]). NCCN Clinical Practice Guidelines in Oncology v.1.2010. Fort Washington, PA: NCCN; 2010.\n",
      "National Comprehensive Cancer Network (NCCN). Occult primary. NCCN Clinical Practice Guidelines in Oncology v.1.2009. Fort Washington, PA: NCCN; July 2008.\n",
      "National Comprehensive Cancer Network (NCCN). Ovarian cancer. NCCN Clinical Practice Guidelines in Oncology, version 2.2015. Fort Washington, PA: NCCN; 2015.\n",
      "National Comprehensive Cancer Network (NCCN). Ovarian cancer. NCCN Clinical Practice Guidelines in Oncology, Version 2.2018. Fort Washington, PA: NCCN; 2018.\n",
      "National Comprehensive Cancer Network (NCCN). Ovarian cancer. NCCN Clinical Practice Guidelines in Oncology, Version 3.2014. Fort Washington, PA: NCCN; 2014.\n",
      "National Comprehensive Cancer Network. Clinical practice guideline: Ovarian Cancer/Fallopian Tube Cancer/Primary Peritoneal Cancer. Version 1.2023. NCCN: Plymouth Meeting, PA.\n",
      "National Comprehensive Cancer Network. Clinical practice guideline: Pancreatic adenocarcinoma. Version 1.2022. NCCN: Plymouth Meeting, PA.\n",
      "National Comprehensive Cancer Network (NCCN). Prostate cancer. NCCN Biomarkers Compendium. Fort Washington, PA: NCCN; 2019.\n",
      "National Comprehensive Cancer Network (NCCN). Prostate cancer. NCCN Clinical Practice Guidelines in Oncology, Version 1.2023. Plymouth Meeting, PA: NCCN; September 16, 2022.\n",
      "National Comprehensive Cancer Network (NCCN). Prostate cancer. NCCN Clinical Practice Guidelines in Oncology, Version 1.2015. Fort Washington, PA: NCCN; 2015.\n",
      "National Comprehensive Cancer Network (NCCN). Prostate cancer. NCCN Clinical Practice Guidelines in Oncology, Version 2.2017. Fort Washington, PA: NCCN; 2017.\n",
      "National Comprehensive Cancer Network (NCCN). Prostate cancer. NCCN Clinical Practice Guidelines in Oncology, Version 2.2018. Fort Washington, PA: NCCN; 2018.\n",
      "National Comprehensive Cancer Network (NCCN). Prostate cancer. NCCN Clinical Practice Guidelines in Oncology, Version 4.2018. Fort Washington, PA: NCCN; 2018.\n",
      "National Comprehensive Cancer Network (NCCN). Prostate cancer. NCCN Clinical Practice Guidelines in Oncology, Version 4.2019. Fort Washington, PA: NCCN; 2019.\n",
      "National Comprehensive Cancer Network (NCCN). Prostate cancer. NCCN Clinical Practice Guidelines in Oncology, Version 2.2020. Fort Washington, PA: NCCN; 2020.\n",
      "National Comprehensive Cancer Network. Clinical practice guideline: Prostate Cancer. Version 3.2022. NCCN: Plymouth Meeting, PA.\n",
      "\n",
      "National Comprehensive Cancer Network. Clinical practice guideline: Prostate Cancer. Version 1.2023. NCCN: Plymouth Meeting, PA.\n",
      "National Comprehensive Cancer Network (NCCN). Prostate cancer early detection. NCCN Clinical Practice Guidelines in Oncology, Version 2.2019. Plymouth Meeting, PA: NCCN; 2019.\n",
      "National Comprehensive Cancer Network (NCCN). Prostate cancer early detection. NCCN Clinical Practice Guidelines in Oncology, Version 1.2022. Plymouth Meeting, PA: NCCN; February 16, 2022.\n",
      "National Comprehensive Cancer Network. Clinical practice guideline: Prostate cancer early detection. Version 1.2023. NCCN: Plymouth Meeting, PA.\n",
      "National Comprehensive Cancer Network (NCCN). Rectal cancer. NCCN Clinical Practice Guidelines in Oncology, Version 3.2018. Fort Washington, PA: NCCN; 2018.\n",
      "National Comprehensive Cancer Network (NCCN). Rectal cancer. NCCN Clinical Practice Guidelines in Oncology, Version 2.2019. Fort Washington, PA: NCCN; 2019.\n",
      "National Comprehensive Cancer Network (NCCN). Rectal cancer. NCCN Clinical Practice Guidelines in Oncology, Version 1.2020. Fort Washington, PA: NCCN; 2020.\n",
      "National Comprehensive Cancer Network (NCCN). Rectal cancer. NCCN Clinical Practice Guidelines in Oncology, Version 6.2020. Fort Washington, PA: NCCN; 2020.\n",
      "National Comprehensive Cancer Network. Clinical practice guideline: Rectal cancer. Version 1.2022. NCCN: Plymouth Meeting, PA.\n",
      "National Comprehensive Cancer Network. Clinical practice guideline: Rectal Cancer. Version 4.2022. NCCN: Plymouth Meeting, PA.\n",
      "National Comprehensive Cancer Network (NCCN). Small cell lung cancer. NCCN Clinical Practice Guideline in Oncology, Version 2.2017. Fort Washington, PA: NCCN; 2017.\n",
      "National Comprehensive Cancer Network (NCCN). Small cell lung cancer. NCCN Clinical Practice Guidelines in Oncology, Version 1.2019. Fort Washington, PA: NCCN; 2019.\n",
      "National Comprehensive Cancer Network. Clinical practice guideline: Soft tissue sarcoma. Version 1.2022. NCCN: Plymouth Meeting, PA.\n",
      "National Comprehensive Cancer Network (NCCN). Systemic mastocytosis. NCCN Clinical Practice Guidelines in Oncology, Version 1.2020. Plymouth Meeting, PA: NCCN; 2020.\n",
      "National Comprehensive Cancer Network (NCCN). Thyroid carcinoma. NCCN Clinical Practice Guidelines in Oncology, version 3.2018. Fort Washington, PA: NCCN; 2018.\n",
      "National Comprehensive Cancer Network (NCCN). Uterine neoplasms. NCCN Clinical Practice Guidelines in Oncology, Version 2.2015. Fort Washington, PA: NCCN; 2015.\n",
      "National Comprehensive Cancer Network. Clinical practice guideline: Uterine neoplasms. Version 1.2022. NCCN: Plymouth Meeting, PA.\n",
      "National Comprehensive Cancer Network (NCCN). Gastric cancer. NCCN Clinical Practice Guidelines in Oncology, Version 3.2016. Fort Washington, PA: NCCN; 2016.\n",
      "National Comprehensive Cancer Network. Clinical practice guideline: Gastric cancer. Version 2.2022. NCCN: Plymouth Meeting, PA.\n",
      "National Comprehensive Cancer Network (NCCN). Head and neck cancers. NCCN Clinical Practice Guidelines in Oncology, version 2.2018. Fort Washington, PA:NCCN, 2018.\n",
      "National Comprehensive Cancer Network (NCCN). HPV infection. NCCN Biomarkers Compendium. Fort Washington, PA: NCCN. 2018.\n",
      "National Comprehensive Cancer Network (NCCN). HPV. NCCN Biomarkers Compendium. Fort Washington, PA: NCCN, 2018.\n",
      "National Comprehensive Cancer Network (NCCN). Melanoma. NCCN Clinical Practice Guidelines in Oncology, Version 2.2016. Fort Washington, PA: NCCN; 2016.\n",
      "National Comprehensive Cancer Network (NCCN). Non-small cell lung cancer. NCCN Clinical Practice Guidelines in Oncology, Version 4.2017. Fort Washington, PA: NCCN; 2017.\n",
      "National Comprehensive Cancer Network (NCCN). Pancreatic adenocarcinoma. NCCN Clinical Practice Guidelines in Oncology, Version 2.2016. Fort Washington, PA: NCCN; 2016.\n",
      "National Comprehensive Cancer Network (NCCN). Pancreatic adenocarcinoma. NCCN Clinical Practice Guidelines in Oncology, version 1.2019. Fort Washington, PA: NCCN; 2018.\n",
      "National Comprehensive Cancer Network (NCCN). Prostate cancer. NCCN Clinical Practice Guidelines in Oncology, Version 3.2016. Fort Washington, PA: NCCN; 2016.\n",
      "National Horizon Scanning Center (NHSC). Prostate cancer gene 3 (Progensa PCA3) assay in the diagnosis of prostate cancer. Horizon Scanning Technology Briefing. Birmingham, UK: NHSC; December 2006. \n",
      "National Horizon Scanning Centre (NHSC). NMP22 BladderChek proteomic assay for the detection of bladder cancer - horizon scanning technology note. Birmingham, UK: NHSC; 2006.\n",
      "National Horizon Scanning Centre (NHSC). Update: OVA1â¢ test for the assessment of suspected ovarian cancer. Horizon Scanning Report. Birmingham, UK: National Horizon Scanning Centre, University of Birmingham; 2012.\n",
      "National Horizon Scanning Centre (NHSC). UPDATED: RealTime mS9 Colorectal Cancer (CRC) Assay for the early detection of colorectal cancer. Horizon Scanning Review. Birmingham, UK: National Horizon Scanning Centre (NHSC); 2012.\n",
      "National Institute for Health and Care Excellence (NICE).  Prolaris gene expression assay for assessing long-term risk of prostate cancer progression. MedTech Innovation Briefing [MIB65]. London, UK: NICE; May 2016.\n",
      "National Institute for Health and Care Excellence (NICE). Caris Molecular Intelligence for guiding cancer treatment. Medtech Innovation Briefing [MIB120]. London, UK: NICE; September 2017.\n",
      "National Institute for Health and Care Excellence (NICE). Diagnosing prostate cancer: PROGENSA PCA3 assay and Prostate Health Index. Diagnostics Guidance (DG) No. 17. London, UK: National Institute for Health and Care Excellence (NICE); June 2, 2015.\n",
      "National Institute for Health and Care Excellence (NICE). EndoPredict gene expression profiling assay for assessing risk of breast cancer recurrence. Medtech Innovation Briefing 44. London, UK: NICE; November 2015.\n",
      "National Institute for Health and Care Excellence (NICE). The Prosigna gene expression profiling assay for assessing long-term risk of breast cancer recurrence. Medtech Innovation Briefing 27. London, UK: NICE; March 2015. \n",
      "National Institute for Health and Care Excellence (NICE). Gene expression profiling and expanded immunohistochemistry tests for guiding adjuvant chemotherapy decisions in early breast cancer management: MammaPrint, Oncotype DX, IHC4 and Mammostrat. Diagnostics Guidance (DG) 10. London, UK: NICE; September 2013.\n",
      "National Institute for Health and Care Excellence (NICE). Signatera for detecting molecular residual disease from solid tumour cancers. Medtech Innovation Briefing (MIB) 307. London, UK: NICE: October 4, 2022.\n",
      "National Institute for Health and Clinical Excellence (NICE). Bevacizumab and cetuximab for the treatment of metastatic colorectal cancer. Technology Appraisal Guidance 118. London, UK: NICE; January 2007. \n",
      "National Institute for Health and Clinical Excellence (NICE). Diagnosis and management of metastatic malignant disease of unknown primary origin. Clinical Guideline. Draft for Consultation. London, UK: NICE; December 2009.\n",
      "National Institutes of Health (NIH), Early Detection Research Network (EDRN). MiPS (Mi Prostate Score Urine Test. Biomarkers. Bethesda, MD: NIH; 2019. Available at: https://edrn.nci.nih.gov/biomarkers/mips-mi-prostate-score-urine-test. Accessed October 7, 2019.\n",
      "Nauseef WM, Olsson I, Arnljots K. Biosynthesis and processing of myeloperoxidase--a marker for myeloid cell differentiation. Eur J Haematol. 1988;40(2):97-110.\n",
      "NeoGenomics.InVisionFirst-Lung Technical Sheet. Fort Myers, FL: NeoGenomics; Available at: https://neogenomics.com/invisionfirstr-lung. Accessed July 31, 2023\n",
      "Nehls O, Gregor M, Klump B. Serum and bile markers for cholangiocarcinoma. Semin Liver Dis. 2004;24(2):139-154.\n",
      "Nelson PS. Targeting the androgen receptor in prostate cancer--a resilient foe. N Engl J Med. 2014;371(11):1067-1069. \n",
      "Newcomb LF, Zheng Y, Faino AV, et al. Performance of PCA3 and TMPRSS2:ERG urinary biomarkers in prediction of biopsy outcome in the Canary Prostate Active Surveillance Study (PASS). Prostate Cancer Prostatic Dis. 2019;22(3):438-445.\n",
      "Nguyen AH, Detty SQ, Agrawal DK. Clinical implications of high-mobility group box-1 (HMGB1) and the receptor for advanced glycation end-products (RAGE) in cutaneous malignancy: A systematic review. Anticancer Res. 2017;37(1):1-7.\n",
      "Nguyen HG, Welty CJ, Cooperberg MR. Diagnostic associations of gene expression signatures in prostate cancer tissue. Curr Opin Urol. 2015;25(1):65-70.\n",
      "Nguyen PL, Haddad Z, Ross AE, et al. Ability of a genomic classifier to predict metastasis and prostate cancer-specific mortality after radiation or surgery based on needle biopsy specimens. Eur Urol. 2017b;72(5):845-852.\n",
      "Nguyen PL, Martin NE, Choeurng V, et al. Utilization of biopsy-based genomic classifier to predict distant metastasis after definitive radiation and short-course ADT for intermediate and high-risk prostate cancer. Prostate Cancer Prostatic Dis. 2017a;20(2):186-192.\n",
      "Nguyen PL, Shin H, Yousefi K, et al. Impact of a Genomic classifier of metastatic risk on postprostatectomy treatment recommendations by radiation oncologists and urologists. Urology. 2015;86(1):35-40.\n",
      "Nielsen KV, Ejlertsen B, MÃ¸ller S, et al. The value of TOP2A gene copy number variation as a biomarker in breast cancer: Update of DBCG trial 89D. Acta Oncol. 2008;47(4):725-734.\n",
      "Nigjeh EN, Chen R, Allen-Tamura Y, et al. Spectral library-based glycopeptide analysis-detection of circulating galectin-3 binding protein in pancreatic cancer. Proteomics Clin Appl. 2017;11(9-10):10.1002/prca.201700064.\n",
      "NIHR HSC. Caris Target Nowâ¢ molecular profiling service for solid malignant tumours. Horizon Scanning Review. Birmingham, UK: NIHR Horizon Scanning Centre (NIHR HSC); 2013\n",
      "Nikiforov YE, Ohori NP, Hodak SP, et al. Impact of mutational testing on the diagnosis and management of patients with cytologically indeterminate thyroid nodules: A prospective analysis of 1056 FNA samples. J Clin Endocrinol Metab. 2011;96(11):3390-3397.\n",
      "Nikiforov YE, Steward DL, Robinson-Smith TM, et al. Molecular testing for mutations in improving the fine-needle aspiration diagnosis of thyroid nodules. J Clin Endocrinol Metab. 2009;94(6):2092-2098.\n",
      "Nitz U, Gluz O, Christgen M5, et al. Reducing chemotherapy use in clinically high-risk, genomically low-risk pN0 and pN1 early breast cancer patients: Five-year data from the prospective, randomised phase 3 West German Study Group (WSG) PlanB trial. Breast Cancer Res Treat. 2017;165(3):573-583.\n",
      "No authors listed. Committee opinion no. 477: The role of the obstetrician-gynecologist in the early detection of epithelial ovarian cancer. Obstet Gynecol. 2011;117(3):742-746.\n",
      "No authors listed. Ovarian cancer: Screening, treatment, and followup. NIH Consens Statement. 1994;12(3):1-30.\n",
      "Nodit L, McGrath KM, Zahid M, et al. Endoscopic ultrasound-guided fine needle aspirate microsatellite loss analysis and pancreatic endocrine tumor outcome. Clin Gastroenterol Hepatol. 2006;4(12):1474-14788.\n",
      "Nomura F, Ishijima M, Horikoshi A, et al. Determination of serum des-gamma-carboxy prothrombin levels in patients with small-sized hepatocellular carcinoma: Comparison of the conventional enzyme immunoassay and two modified methods. Am J Gastroenterol. 1996;91(7):1380-1383.\n",
      "Noon AP, VlatkoviÄ N, PolaÅski R, et al. p53 and MDM2 in renal cell carcinoma: Biomarkers for disease progression and future therapeutic targets? Cancer. 2010;116(4):780-790.\n",
      "Norris EJ, Jones WD, Surleac MD, et al. Clonal lineage of high grade serous ovarian cancer in a patient with neurofibromatosis type 1. Gynecol Oncol Rep. 2018;23:41-44.\n",
      "Northrup V, Maybank A, Carson N, Rahmeh T. The value of next-generation sequencing in the screening and evaluation of hematologic neoplasms in clinical practice. Am J Clin Pathol. 2020;153(5):639-645.\n",
      "Odegaard JI, Vincent JJ, Morti SA, et al. Validation of a plasma-based comprehensive cancer genotyping assay utilizing orthogonal tissue- and plasma-based methodologies. Clin Cancer Res. 2018;24(15):3539-3549.\n",
      "Ohori NP, Nikiforova MN, Schoedel KE, et al. Contribution of molecular testing to thyroid fine-needle aspiration cytology of \"follicular lesion of undetermined significance/atypia of undetermined significance\". Cancer Cytopathol. 2010;118(1):17-23.\n",
      "OmniSeq and LabCorp Launch OmniSeq Advanceâ  Assay [website]. Buffalo, NY: OmniSeq; 2018. Available at: https://globenewswire.com/news-release/2018/06/01/1515775/0/en/OmniSeq-and-LabCorp-Launch-OmniSeq-Advance-Assay.html. Accessed July 10, 2018.\n",
      "Oncocyte Corporation. DetermaRx [website]. Irvine, CA: Oncocyte; 2021. Available at: https://oncocyte.com/products/oncologists/. Accessed December 29, 2021.\n",
      "Onken MD, Worley LA, Char DH, et al. Collaborative Ocular Oncology Group report number 1: Prospective validation of a multi-gene prognostic assay in uveal melanoma. Ophthalmology. 2012;119(8):1596-1603. \n",
      "Onken MD, Worley LA, Tuscan MD, Harbour JW. An accurate, clinically feasible multi-gene expression assay for predicting metastasis in uveal melanoma. J Mol Diagn. 2010;12(4):461-468.\n",
      "Oratz R, Kim B, Chao C, et al. Physician survey of the effect of the 21-gene recurrence score assay results on treatment recommendations for patients with lymph node-positive, estrogen receptor-positive breast cancer. J Oncol Practice. 2011;7(2):94-99.\n",
      "O'Sullivan P, Sharples K, Dalphin M, et al. A multigene urine test for the detection and stratification of bladder cancer in patients presenting with hematuria. J Urol. 2012;188(3):741-747.\n",
      "Ost DE. Interpretation and application of the likelihood ratio to clinical practice in thoracic oncology. J Bronchology Interv Pulmonol. 2022;29(1):62-70.\n",
      "Ostrin EJ, Sidransky D, Spira A, Hanash SM. Biomarkers for lung cancer screening and detection. Cancer Epidemiol Biomarkers Prev. 2020;29(12):2411-2415.\n",
      "Otchy D, Hyman NH, Simmang C, et al. Practice parameters for colon cancer. Dis Colon Rectum. 2004;47(8):1269-1284.\n",
      "Oxnard GR, Binder A, JÃ¤nne PA. New targetable oncogenes in non-small-cell lung cancer. J Clin Oncol. 2013;31(8):1097-1104.\n",
      "Pages F, Andre T, Taieb J, et al. Prognostic and predictive value of the Immunoscore in stage III colon cancer patients treated with oxaliplatin in the prospective IDEA France PRODIGE-GERCOR cohort study. Ann Oncol. 2020;31(7):921-929.\n",
      "Pages F, Mlecnik B, Marliot F, et al. International validation of the consensus Immunoscore for the classification of colon cancer: A prognostic and accuracy study. Lancet. 2018;391(10135):2128-2139.\n",
      "Paik S, Tang G, Shak S, et al. Gene expression and benefit of chemotherapy in women with node-negative, estrogen receptor-positive breast cancer J Clin Oncol. 2006;24(23):3726-3734.\n",
      "Paik S, Shak S, Tang G, et al. A multigene assay to predict recurrence of tamoxifen-treated, node-negative breast cancer. N Engl J Med. 2004;351(27):2817-2826.\n",
      "Palanichamy K, Erkkinen M, Chakravarti A. Predictive and prognostic markers in human glioblastomas. Curr Treat Options Oncol. 2006;7(6):490-504.\n",
      "Palmero R, Taus A, Viteri S, et al. Biomarker discovery and outcomes for comprehensive cell-free circulating tumor DNA versus standard-of-care tissue testing in advanced nonâsmall-cell lung cancer.  JCO Precis Oncol. 2021;5:93-102.\n",
      "Pan J, Ding M, Xu K, et al. Exosomes in diagnosis and therapy of prostate cancer. Oncotarget. 2017;8(57):97693-97700.\n",
      "Panigrahi GK, Deep G. Exosomes-based biomarker discovery for diagnosis and prognosis of prostate cancer. Front Biosci (Landmark Ed). 2017;22:1682-1696.\n",
      "Pantel K, Muller V, Auer M, et al. Detection and clinical implications of early systemic tumor cell dissemination in breast cancer. Clin Cancer Res. 2003;9(17):6326-6334.\n",
      "Papadaki H, Kounelis S, Kapadia SB, et al. Relationship of p53 gene alterations with tumor progression and recurrence in olfactory neuroblastoma. Am J Surg Pathol. 1996;20(6):715-721.\n",
      "Papavasiliou E, Sills VA, Calanzani N, et al. Diagnostic performance of biomarkers for bladder cancer detection suitable for community and primary care settings: A systematic review and meta-analysis. Cancers (Basel). 2023;15(3):709.\n",
      "Parekh DJ, Ankerst DP, Troyer D, et al. Biomarkers for prostate cancer detection. J Urol. 2007;178(6):2252-2259.\n",
      "Parekh DJ, Punnen S, Sjoberg DD, et al. A multi-institutional prospective trial in the USA confirms that the 4Kscore accurately identifies men with high-grade prostate cancer. Eur Urol. 2015;68(3):464-470.\n",
      "Park CM, Lee WY, Chun HK, et al. Relationship of polymorphism of the tandem repeat sequence in the thymidylate synthase gene and the survival of stage III colorectal cancer patients receiving adjuvant 5-flurouracil-based chemotherapy. J Surg Oncol. 2010;101(1):22-27.\n",
      "Parrella A, Mundy L, Merlin T, Hiller J. NMP22 BladderChek(TM) point-of-care diagnostic test for bladder cancer. Horizon Scanning Prioritising Summary - Volume 9. Adelaide, SA: Adelaide Health Technology Assessment (AHTA) on behalf of National Horizon Scanning Unit (HealthPACT and MSAC); 2005.\n",
      "Partin AW, Van Neste L, Klein EA, et al. Clinical validation of an epigenetic assay to predict negative histopathological results in repeat prostate biopsies. J Urol. 2014;192(4):1081-1087.\n",
      "Pathwork Diagnostics [website]. Announcing the pathwork tissue of origin test [website]; Sunnyvale, CA. Available at: http://www.pathworkdx.com/index.html.  Accessed October 13, 2008.\n",
      "Patnaik MM, Rindos M, Kouides PA, et al. Systemic mastocytosis: A concise clinical and laboratory review. Arch Pathol Lab Med. 2007;131(5):784-791.\n",
      "Patsner B, Mann WJ, Vissicchio M, Loesch M. Comparison of serum CA-125 and lipid-associated sialic acid (LASA-P) in monitoring patients with invasive ovarian adenocarcinoma. Gynecol Oncol. 1988;30(1):98-103.\n",
      "Paulson KG, Carter JJ, Johnson LG, et al. Antibodies to merkel cell polyomavirus T antigen oncoproteins reflect tumor burden in merkel cell carcinoma patients. Cancer Res. 2010;70(21):8388-97.\n",
      "Paulson KG, Lewis CW, Redman MW, et al. Viral oncoprotein antibodies as a marker for recurrence of Merkel cell carcinoma: A prospective validation study. Cancer. 2017;123(8):1464-1474.\n",
      "Paweletz CP, Sacher AG, Raymond CK, et al. Bias-corrected targeted next-generation sequencing for rapid, multiplexed detection of actionable alterations in cell-free DNA from advanced lung cancer patients. Clin Cancer Res. 2016;22(4):915-922.\n",
      "Peczkowska M, Cwikla J, Kidd M, et al. The clinical utility of circulating neuroendocrine gene transcript analysis in well-differentiated paragangliomas and pheochromocytomas. Eur J Endocrinol. 2017;176(2):143-157. \n",
      "Peled N, Roisman LC, Miron B, et al. Subclonal therapy by two EGFR TKIs guided by sequential plasma cellfree DNA in EGFR-mutated lung cancer. J Thorac Oncol. 2017;12(7):e81-e84.\n",
      "Penzvalto Z, MihÃ¡ly Z, Gyorffy B. Gene expression based multigene prognostic and predictive tests in breast cancer. Magy Onkol. 2009;53(4):351-359.\n",
      "Permanente Medical Group. Ovacheck. Permanente Medical Group Physician Home Pages. San Francisco, CA: Kaiser Permanente; 2004. Available at: http://www.permanente.net/kaiser/pages/f21815.html. Accessed June 4, 2004.\n",
      "Perrot A, Lauwers-Cances V, Corre J, et al. Minimal residual disease negativity using deep sequencing is a major prognostic factor in multiple myeloma. Blood 2018;132(23): 2456-2464.\n",
      "Petricoin EF, Ardekani AM, Hitt BA, et al. Use of proteomic patterns in serum to identify ovarian cancer. Lancet. 2002;359(9306):572-577.\n",
      "Pfister DG, Johnson DH, Azzoli CG, et al. American Society of Clinical Oncology treatment of unresectable non-small-cell lung cancer guideline: Update 2003. J Clin Oncol. 2004;22(2):330-353.\n",
      "Piotrowska Z, Drapkin B, Engelman JA, et al. Plasma T790M result alters treatment options in a previously T790 wild-type EGFR-mutant lung cancer. J Thorac Oncol. 2016;11(8):e95-e97.\n",
      "Pirtskalaishvili G, Konety BR, Getzenberg RH. Update on urine-based markers for bladder cancer. How sensitive and specific are the new noninvasive tests? Postgrad Med. 1999;106(6):85-86, 91-94.\n",
      "Pishvaian MJ, Bender RJ, Halverson D, et al. Molecular profiling of patients with pancreatic cancer: Initial results from the know your tumor initiative. Clin Cancer Res. 2018;24(20):5018-5027.\n",
      "Plagnol V, Woodhouse S, Howarth K, et al. Analytical validation of a next generation sequencing liquid biopsy assay for high sensitivity broad molecular profiling. PLoS One. 2018;13(3):e0193802. Published 2018 Mar 15.\n",
      "Plasseraud KM, Cook RW, Tsai T, et al. Clinical performance and management outcomes with the DecisionDx-UM gene expression profile test in a prospective multicenter study. J Oncol. 2016; 2016: 5325762.\n",
      "Plaxe SC, Mundt AJ. Overview of endometrial carcinoma. UpToDate [online serial]. Waltham, MA: UpToDate; reviewed January 2015.\n",
      "Plymate SR, Sharp A, de Bono JS. Nuclear Circulating Tumor Cell Androgen Receptor Variant 7 in Castration-Resistant Prostate Cancer: The Devil Is in the Detail. JAMA Oncol. 2018;4(9):1187-1188. \n",
      "Podlipnik S, Carrera C, Boada A, et al. Early outcome of a 31-gene expression profile test in 86 ajcc stage ib-ii melanoma patients. A prospective multicentre cohort study. J Eur Acad Dermatol Venereol. 2019;33(5):857-862.\n",
      "Pollack IF, Finkelstein SD, Burnham J, et al.; Children's Cancer Group. Age and TP53 mutation frequency in childhood malignant gliomas: Results in a multi-institutional cohort. Cancer Res. 2001;61(20):7404-7407.\n",
      "Poola I, Yue Q, Gillespie J, et al. Breast hyperplasias, risk signature and breast cancer. Cancer Prev Res (Phila). 2019:12(7):471-480.\n",
      "Powles T, Assaf ZJ, Davarpanah N, et al. ctDNA guiding adjuvant immunotherapy in urothelial carcinoma. Nature. 2021;595(7867):432-437.\n",
      "Prabhakar K, RodrÎ¯guez CI, Jayanthy AS, et al. Role of miR-214 in regulation of Î²-catenin and the malignant phenotype of melanoma. Mol Carcinog. 2019;58(11):1974-1984.\n",
      "Prat A, Fran C, FernÃ¡ndez A, et al. Response and survival of breast cancer intrinsic subtypes following multi-agent neoadjuvant chemotherapy. BMC Med. 2015;13:303.\n",
      "Praxis Genomics, LLC. Genomic testing [website]. Sandy Springs, GA: Praxis Genomics; 2020. Available at: https://www.praxisgenomics.com/test-list. Accessed December 29, 2021.\n",
      "Prelude Corporation. What is your DCISion score? [website]. Laguna Hills, CA: Prelude; 2022. Available at: https://preludedx.com/patients/#dcis-information. Accessed December 29, 2021.\n",
      "Pricolo VE, Finkelstein SD, Bland KI.  Topographic genotyping of colorectal carcinoma: From a molecular carcinogenesis model to clinical relevance. Ann Surg Oncol. 1997;4(3):269-278.\n",
      "Pricolo VE, Finkelstein SD, Wu TT, et al. Prognostic value of TP53 and K-ras-2 mutational analysis in stage III carcinoma of the colon. Am J Surg. 1996;171(1):41-46.\n",
      "Pritchard KI, Messersmith H, Elavathil L, et al. HER-2 and topoisomerase II as predictors of response to chemotherapy. J Clin Oncol. 2008;26(5):736-744.\n",
      "Provista Life Sciences. The BT test and early breast cancer detection [website]. Phoenix, AZ: Provista Life Sciences; 2008. Available at: http://www.provistals.com/BTTestOverview.aspx. Accessed September 8, 2008.\n",
      "Przygodzki RM, Finkelstein SD, Keohavong P, et al. Sporadic and thorotrast-induced angiosarcomas of the liver manifest frequent and multiple point mutations in K-ras-2. Lab Invest. 1997;76(1):153-159.\n",
      "Przygodzki RM, Finkelstein SD, Langer JC, et al. Analysis of p53, K-ras-2, and C-raf-1 in pulmonary neuroendocrine tumors. Correlation with histological subtype and clinical outcome. Am J Pathol. 1996;148(5):1531-1541.\n",
      "Przygodzki RM, Koss MN, Moran CA, et al. Pleomorphic (giant and spindle cell) carcinoma is genetically distinct from adenocarcinoma and squamous cell carcinoma by K-ras-2 and p53 analysis. Am J Clin Pathol. 1996;106(4):487-492.\n",
      "Punnen S, Freedland SJ, Polascik TJ, et al. A multi-institutional prospective trial confirms noninvasive blood test maintains predictive value in African American men. J Urol. 2018;199(6):1459-1463.\n",
      "Oellerich M, Schutz E, Beck J, et al. Using circulating cell-free DNA to monitor personalized cancer therapy. Crit Rev Clin Lab Sci. 2017;54(3):205-218.\n",
      "Oellerich M, Schutz E, Beck J, Walson PD. Circulating cell-free DNA-diagnostic and prognostic applications in personalized cancer therapy. Ther Drug Monit. 2019;41(2):115-120.\n",
      "Raghu G, Flaherty KR, Lederer DJ, et al. Use of a molecular classifier to identify usual interstitial pneumonia in conventional transbronchial lung biopsy samples: A prospective validation study. Lancet Respir Med. 2019;7(6):487-496. \n",
      "Rakovitch E, Nofech-Mozes S, Hanna W, et al. A population-based validation study of the DCIS Score predicting recurrence risk in individuals treated by breast-conserving surgery alone. Breast Cancer Res Treat. 2015;152(2):389-398.\n",
      "Raldow AC, Zhang D, Chen M-H, et al. Risk group and death from prostate cancer. Implications for active surveillance in men with favorable intermediate-risk prostate cancer. JAMA Oncol. 2015;1(3):334-340.\n",
      "Raman G, Wallace B, Patel K, et al. Update on horizon scans of genetic tests currently available for clinical use in cancers. Technology Assessment Report. Final Report. Prepared by the Tufts Evidence-based Practice Center for the Agency for Healthcare Research and Quality (AHRQ) under contract no. HHSA 290-2007-10055-I. Project ID: GEND0508. Rockville, MD: AHRQ; April 15, 2011.\n",
      "Ramsey SD, Grady WM. Screening for colorectal cancer in patients with a family history of colorectal cancer or advanced polyp. UpToDate [online serial]. Waltham, MA: UpToDate; reviewed June 2019.\n",
      "Rapanotti MC, Cugini E, Nuccetelli M, et al. MCAM/MUC18/CD146 as a multifaceted warning marker of melanoma progression in liquid biopsy. Int J Mol Sci. 2021;22(22):12416.\n",
      "Rapanotti MC, Viguria TMS, Spallone G, et al. Minimal residual disease in melanoma: Molecular characterization of in transit cutaneous metastases and circulating melanoma cells recognizes an expression panel potentially related to disease progression. Cancer Treat Res Commun. 2020;25:100262.\n",
      "Rasamny JJ, Allak A, Krook KA, et al. Cyclin D1 and FADD as biomarkers in head and neck squamous cell carcinoma. Otolaryngol Head Neck Surg. 2012;146(6):923-31.\n",
      "Ratner E, Lu L, Boeke M, et al. A KRAS-variant in ovarian cancer acts as a genetic marker of cancer risk. Cancer Res. 2010;70(16):6509-6515.\n",
      "Rawstron AC, Fazi C, Agathangelidis A, et al. A complementary role of multiparameter flow cytometry and high-throughput sequencing for minimal residual disease detection in chronic lymphocytic leukemia: An European Research Initiative on CLL study. Leukemia. 2016;30(4):929-936.\n",
      "Real FX. p53: It has it all, but will it make it to the clinic as a marker in bladder cancer? J Clin Oncol. 2007;25(34):5341-5344.\n",
      "Reece M, Saluja H, Hollington P, et al. The use of circulating tumor DNA to monitor and predict response to treatment in colorectal cancer. Front Genet. 2019;10:1118.\n",
      "Reed JC. Bcl-2-family proteins and hematologic malignancies: history and future prospects. Blood. 2008;111(7):3322-3330.\n",
      "Reilly G, Bullock RG, Greenwood J, et al. Analytical validation of a deep neural network algorithm for the detection of ovarian cancer. JCO Clin Cancer Inform. 2022;6:e2100192.\n",
      "Reilly GP, Dunton CJ, Bullock RG, et al. Validation of a deep neural network-based algorithm supporting clinical management of adnexal mass. Front Med (Lausanne). 2023;10:1102437.\n",
      "Reilly GP, Gregory DA, Scotti DJ, et al. A real-world comparison of the clinical and economic utility of OVA1 and CA125 in assessing ovarian tumor malignancy risk. J Comp Eff Res. 2023;12(6):e230025.\n",
      "Reinert T, Henriksen TV, Christensen E, et al. Analysis of plasma cell-free DNA by ultradeep sequencing in patients with stages I to III colorectal cancer. JAMA Oncol. 2019;5(8):1124-1131.\n",
      "Rettig EM, Wang AA, Tran N-A, et al. Association of pretreatment circulating tumor tissue-modified viral HPV DNA with clinicopathologic factors in HPV-positive oropharyngeal cancer JAMA Otolaryngol Head Neck Surg. 2022;148(12):1120-1130.\n",
      "Ribeiro U Jr, Finkelstein SD, Safatle-Ribeiro AV, et al. p53 sequence analysis predicts treatment response and outcome of patients with esophageal carcinoma. Cancer. 1998;83(1):7-18.\n",
      "Ribeiro U, Safatle-Ribeiro AV, Posner MC, et al. Comparative p53 mutational analysis of multiple primary cancers of the upper aerodigestive tract. Surgery. 1996;120(1):45-53.\n",
      "Ricciuti B, Mencaroni C, Paglialunga L, et al. Long noncoding RNAs: New insights into non-small cell lung cancer biology, diagnosis and therapy. Med Oncol. 2016;33(2):18.\n",
      "Riley RD, Burchill SA, Abrams KR, et al. A systematic review and evaluation of the use of tumour markers in paediatric oncology: Ewing's sarcoma and neuroblastoma. Health Technol Assess. 2003;7(5):1-162\n",
      "Ring A, Smith IE, Dowsett M. Circulating tumour cells in breast cancer. Lancet Oncol. 2004 Feb;5(2):79-88.\n",
      "Ring BZ, Hout DR, Morris SW, et al. Generation of an algorithm based on minimal gene sets to clinically subtype triple negative breast cancer patients. BMC Cancer. 2016;16:143.\n",
      "Ring BZ, Seitz RS, Beck R, et al. Novel prognostic immunohistochemical biomarker panel for estrogen receptor-positive breast cancer. J Clin Oncol. 2006; 24(19):3039-3047.\n",
      "Roberts MC, Miller DP, Shak S, Petkov VI. Breast cancer-specific survival in patients with lymph node-positive hormone receptor-positive invasive breast cancer and Oncotype DX Recurrence Score results in the SEER database. Breast Cancer Res Treat. 2017;163(2):303-310.\n",
      "Rocca MS, Benna C, Mocellin S, et al. E2F1 germline copy number variations and melanoma susceptibility. J Transl Med. 2019;17(1):181.\n",
      "Roepman P, Horlings HM, Krijgsman O, et al. Microarray-based determination of estrogen receptor, progesterone receptor, and HER2 receptor status in breast cancer. Clin Cancer Res. 2009;15(22):7003-7011.\n",
      "Rojas MP, Telaro E, Russo A, et al. Follow-up strategies for women treated for early breast cancer. Cochrane Database Syst Rev. 2000;(4):CD001768.\n",
      "Romeo MJA, MÃ©ndez AL. Gene expression test for breast cancer. OncotypeÂ®. Executive abstract. Informes de EvaluaciÃ³n de TecnologÃ­as Sanitarias AETSA 2007/2-20. Sevilla, Spain: Agencia de EvaluaciÃ³n de TecnologÃ­as Sanitarias de AndalucÃ­a (AETSA); 2010.\n",
      "Rosenkrantz AB, Verma S, Choyke P, et al. Prostate MRI and MRI-targeted biopsy in patients with prior negative biopsy. Collaborative Initiative of the American Urological Association and the Society of Abdominal Radiologyâs Prostate Cancer Disease-Focused Panel. American Urological Association (AUA) and Society of Abdominal Radiology (SAR) Joint Consensus Statement. Approved by the AUA and SAR Board of Directors 2016. Houston, TX: SAR; 2016.\n",
      "Rosenthal AN, Menon U, Jacobs IJ. Screening for ovarian cancer. Clin Obstet Gynecol. 2006;49(3):433-447.\n",
      "Ross AE, Feng FY, Ghadessi M, et al. A genomic classifier predicting metastatic disease progression in men with biochemical recurrence after prostatectomy. Prostate Cancer Prostatic Dis. 2014;17(1):64-69.\n",
      "Ross AE, Johnson MH, Yousefi K, et al. Tissue-based genomics augments post-prostatectomy risk stratification in a natural history cohort of intermediate- and high-risk men. Eur Urol. 2016;69(1):157-165. \n",
      "Ross DS. Diagnostic approach to and treatment of thyroid nodules. UpToDate [online serial]. Waltham, MA: UpToDate; reviewed January 2016a.\n",
      "Ross DS. Evaluation and management of thyroid nodules with indeterminate cytology. UpToDate [online serial]. Waltham, MA: UpToDate; reviewed September 2020.\n",
      "Ross DS. Overview of thyroid nodule formation. UpToDate [online serial]. Waltham, MA: UpToDate; reviewed January 2016b.\n",
      "Ross DT, Kim CY, Tang G, et al. Chemosensitivity and stratification by a five monoclonal antibody immunohistochemistry test in the NSABP B14 and B20 trials. Clin Cancer Res. 2008;14(20):6602-6609.\n",
      "Rozenblum AB, Ilouze M, Dudnik E, et al. Clinical impact of hybrid capture-based next-generation sequencing on changes in treatment decisions in lung cancer. J Thorac Oncol. 2017;12(2):258-268.\n",
      "Ruzzenente A, Iacono C, Conci S, et al. A novel serum marker for biliary tract cancer: Diagnostic and prognostic values of quantitative evaluation of serum mucin 5AC (MUC5AC). Surgery. 2014;155(4):633-639.\n",
      "Saad F. UPM3: Review of a new molecular diagnostic urine test for prostate cancer. Can J Urol. 2005;12 Suppl 1:40-43; discussion 99-100.\n",
      "Saad RS, Denning KL, Finkelstein SD, et al. Diagnostic and prognostic utility of molecular markers in synchronous bilateral breast carcinoma. Mod Pathol. 2008;21(10):1200-1207.\n",
      "Sabari JK, Offin M, Stephens D, et al. A prospective study of circulating tumor DNA to guide matched targeted therapy in lung cancers. J Natl Cancer Inst. 2019;111(6):575-583.\n",
      "Sacher AG, Paweletz C, Dahlberg SE, et al. Prospective validation of rapid plasma genotyping for the detection of EGFR and KRAS mutations in advanced lung cancer. JAMA Oncol. 2016;2(8):1014-1022.\n",
      "Safatle-Ribeiro AV, Ribeiro Junior U, Reynolds JC, et al. Morphologic, histologic, and molecular similarities between adenocarcinomas arising in the gastric stump and the intact stomach. Cancer. 1996;78(11):2288-2299.\n",
      "Salami SS, Schmidt F, Laxman B, et al. Combining urinary detection of TMPRSS2:ERG and PCA3 with serum PSA to predict diagnosis of prostate cancer. Urol Oncol. 2013;31(5):566-71.\n",
      "Salgia R. Molecular markers in non-small cell lung cancer. UpToDate [online serial]. Waltham, MA; UpToDate; 2008.\n",
      "Samimi M, Molet L, Fleury M, et al. Prognostic value of antibodies to Merkel cell polyomavirus T antigens and VP1 protein in patients with Merkel cell carcinoma. Br J Dermatol. 2016;174(4):813-22.\n",
      "San Miguel L, Vlayen J, De Laet C. Gene expression profiling and immunohistochemistry tests for personalized management of adjuvant chemotherapy decisions in early breast cancer â a Rapid Assessment. Health Technology Assessment (HTA). KCE Reports 237.  D/2015/10.273/13. Brussels, Belgium: Belgian Health Care Knowledge Centre (KCE); 2015.\n",
      "Sanda MG, Cadeddu JA, Kirkby E, et al. Clinically Localized Prostate Cancer: AUA/ASTRO/SUO Guideline. Part I: Risk Stratification, Shared Decision Making, and Care Options. J Urol. 2018a;199(3):683-690. \n",
      "Sanda MG, Cadeddu JA, Kirkby E, et al. Clinically Localized Prostate Cancer: AUA/ASTRO/SUO Guideline. Part II: Recommended Approaches and Details of Specific Care Options. J Urol. 2018b;199(4):990-997. \n",
      "Sanda MG, Chen RC, Crispino T, et al. Clinically localized prostate cancer: AUA/ASTRO/SUO guideline. American Urological Association (AUA) / American Society for Radiation Oncology (ASTRO) / Society of Urologic Oncology (SUO). Approved by the AUA Board of Directors April 2017.\n",
      "Sanda MG, Feng Z, Howard DH, et al; and the EDRN-PCA3 Study Group. Association Between Combined TMPRSS2:ERG and PCA3 RNA Urinary Testing and Detection of Aggressive Prostate Cancer. JAMA Oncol. 2017;3(8):1085-1093.\n",
      "Santos ES, Raez LE, Castillero LDC, et al.. Genomic tissue analysis and liquid biopsy profiles from patients diagnosed with advanced adenocarcinoma of the lung. Clin Oncol. 2016;1:1-4.\n",
      "Sanz-Pamplona R, Berenguer A, Cordero D, et al. Clinical value of prognosis gene expression signatures in colorectal cancer: A systematic review. PLoS One. 2012;7(11):e48877.\n",
      "Sapino A, Roepman P, Linn SC, et al.  MammaPrint molecular diagnostics on formalin-fixed, paraffin-embedded tissue. J Mol Diagn. 2014;16(2):190-197.\n",
      "Sarbia M, Borchard F, Hengels KJ. Histogenetical investigations on adenocarcinomas of the esophagogastric junction. An immunohistochemical study. Pathol Res Pract. 1993;189(5):530-535.\n",
      "Sari Motlagh R, Yanagisawa T, Kawada T, et al. Accuracy of SelectMDx compared to mpMRI in the diagnosis of prostate cancer: A systematic review and diagnostic meta-analysis. Prostate Cancer Prostatic Dis. 2022;25(2):187-198.\n",
      "Sarosdy MF, Schellhammer P, Bokinsky G, et al. Clinical evaluation of a multi-target fluorescent in situ hybridization assay for detection of bladder cancer. J Urol. 2002;168(5):1950-1954.\n",
      "Sawczuk IS, Lee B. The mechanism and clinical applications of the NMP22 tumor marker immunoassay: A review. Am Clin Lab. 1999;18(3):24-26.\n",
      "Schalken JA, Hessels D, Verhaegh G. New targetes for therapy in prostate cancer: Differential display code 3 (DD3/PCA3), a highly prostate cancer specific gene. Urology. 2003;62(suppl 5A):34-43.\n",
      "Scheiman JM, Hwang JH, Moayyedi P. American Gastroenterological Association technical review on the diagnosis and management of asymptomatic neoplastic pancreatic cysts. Gastroenterology. 2015;148(4):824-848.\n",
      "Scher HI, Graf RP, Schreiber NA, et al. Assessment of the validity of nuclear-localized androgen receptor splice variant 7 in circulating tumor cells as a predictive biomarker for castration-resistant prostate cancer. JAMA Oncol. 2018;4(9):1179-1186.\n",
      "Scher HI, Graf RP, Schreiber NA, et al. Nuclear-specific AR-V7 protein localization is necessary to guide treatment selection in metastatic castration-resistant prostate cancer. Eur Urol. 2017;71(6):874-882.\n",
      "Scher HI, Heller G, Molina A, et al. Circulating tumor cell biomarker panel as an individual-level surrogate for survival in metastatic castration-resistant prostate cancer. J Clin Oncol. 2015;33(12):1348-1355.\n",
      "Scher HI, Lu D, Schreiber NA, et al. Association of AR-V7 on circulating tumor cells as a treatment-specific biomarker with outcomes and survival in castration-resistant prostate cancer. JAMA Oncol. 2016;2(11):1441-1449.\n",
      "Scholler N, Crawford M, Sato A, et al. Bead-based ELISA for validation of ovarian cancer early detection markers. Clin Cancer Res. 2006;12(7 Pt 1):2117-2124.\n",
      "Schraa SJ, van Rooijen KL, Koopman M, et al. Cell-free circulating (tumor) DNA before surgery as a prognostic factor in non-metastatic colorectal cancer: A systematic review. Cancers (Basel). 2022;14(9):2218.\n",
      "Schrock AB, Pavlick D, Klempner SJ, et al. Hybrid capture-based genomic profiling of circulating tumor DNA from patients with advanced cancers of the gastrointestinal tract or anus. Clin Cancer Res. 2018;24(8):1881-1890.\n",
      "Schrock AB, Welsh A, Chung JH, et al. Hybrid capture-based genomic profiling of circulating tumor DNA from patients with advanced non-small cell lung cancer. J Thorac Oncol. 2019;14(2):255-264.\n",
      "Schuitevoerder D, Heath M, Cook RW, et al. Impact of gene expression profiling on decision-making in clinically node negative melanoma patients after surgical staging. J Drugs Dermatol. 2018;17(2):196-199.\n",
      "Schwaederle M, Husain H, Fanta PT, et al. Detection rate of actionable mutations in diverse cancers using a biopsy-free (blood) circulating tumor cell DNA assay. Oncotarget. 2016;7(9):9707-9717.\n",
      "Schwaederle MC, Patel SP, Husain H, et al. Utility of genomic assessment of blood-derived circulating tumor DNA (ctDNA) in patients with advanced lung adenocarcinoma. Clin Cancer Res. 2017;23(17):5101-5111.\n",
      "Schwartz JM, Carithers RL. Clinical features and diagnosis of primary hepatocellular carcinoma. UpToDate [online serial]. Waltham, MA: UpToDate; 2008.\n",
      "Sciarra A, Gentilucci A, Silvestri I, et al. Androgen receptor variant 7 (AR-V7) in sequencing therapeutic agents for castratrion resistant prostate cancer: A critical review. Medicine (Baltimore). 2019;98(19):e15608.\n",
      "Scimia M, Du J, Pepe F, et al. Evaluation of a novel liquid biopsy-based ColoScape assay for mutational analysis of colorectal neoplasia and triage of FIT+ patients: A pilot study. J Clin Pathol. 2018;71(12):1123-1126.\n",
      "Scott DW, Wright GW, Williams PM, et al. Determining cell-of-origin subtypes of diffuse large B-cell lymphoma using gene expression in formalin-fixed paraffin-embedded tissue. Blood. 2014;123(8):1214-1217.\n",
      "Scottish Intercollegiate Guidelines Network (SIGN). Epithelial ovarian cancer. A national clinical guideline. SIGN Pub. No. 75. Edinburgh, Scotland: Scottish Intercollegiate Guidelines Network (SIGN); October 2003.\n",
      "Secord AA, Darcy KM, Hutson A, et al; Gynecologic Oncology Group study. Co-expression of angiogenic markers and associations with prognosis in advanced epithelial ovarian cancer: a Gynecologic Oncology Group study. Gynecol Oncol. 2007;106(1):221-232.\n",
      "Segal R, Lukka H, Klotz LH, et al.; Cancer Care Ontario Practice Guidelines Initiative Genitourinary Cancer Disease Site Group. Surveillance programs for early stage non-seminomatous testicular cancer: A practice guideline. Can J Urol. 2001;8(1):1184-1192.\n",
      "Seifert JK Morris DI. Indicators of recurrence following cryotherapy for hepatic metastases from colorectal cancer. Br J Surg. 1999;86(2):234-240.\n",
      "Senkus E, Kyriakides S, Penault-Llorca F, et al.; ESMO Guidelines Working Group. Primary breast cancer: ESMO Clinical Practice Guidelines for diagnosis, treatment and follow-up. Ann Oncol. 2013;24 Suppl 6:vi7-23.\n",
      "Sepulveda AR, Hamilton SR, Allegra CJ, et al, Molecular biomarkers for the evaluation of colorectal cancer: Guideline from the American Society for Clinical Pathology, College of American Pathologists, Association for Molecular Pathology, and American Society of Clinical Oncology. J Mol Diagn. 2017;19(2):187-225.\n",
      "Sestak I, Cuzick J, et al. Prediction of late distant recurrence after 5 years of endocrine treatment: A combined analysis of patients from the Austrian Breast and Colorectal Cancer Study Group 8 and Arimidex, Tamoxifen Alone or in Combination Randomized Trials using the PAM50 Risk of Recurrence Score. J Clin Oncol. 2015;33(8):916-922.\n",
      "Sestak I, Cuzick J. Markers for the identification of late breast cancer recurrence. Breast Cancer Res. 2015;17(1):10.\n",
      "Sestak I, Dowsett M, Sgroi D. Comparison of fibe different scores for the prediction of late recurrence for oestrogen receptor-positive breast cancer. Ann Oncol. 2013; 24 (suppl 3): iii29.\n",
      "Sgroi DC, Carney E, Zarrella E, et al. Prediction of late disease recurrence and extended adjuvant letrozole benefit by the HOXB13/IL17BR biomarker. J Natl Cancer Inst. 2013;105(14):1036-1042.\n",
      "Sgroi DC, Sestak I, Cuzick J, et al. Prediction of late distant recurrence in patients with oestrogen-receptor-positive breast cancer: A prospective comparison of the breast-cancer index (BCI) assay, 21-gene recurrence score, and IHC4 in the TransATAC study population. Lancet Oncol. 2013;14(11):1067-1076.\n",
      "Sha D, Jin Z, Budczies J, et al. Tumor mutational burden as a predictive biomarker in solid tumors. Cancer Discov. 2020;10(12):1808-1825.\n",
      "Shak S. Palmer G, Baehner FL, et al; Genomic Health, Redwood City, CA; Indiana University School of Medicine, Indianapolis, IN. Molecular characterization of male breast cancer by standardized quantitative RT-PCR analysis: First large genomic study of 347 male breast cancers compared to 82,434 female breast cancers. J Clin Oncol. 2009;27:15s (suppl; abstr 549).\n",
      "Shankaran V, Wisinski KB, Mulcahy MF, et al. The role of molecular markers in predicting response to therapy in patients with colorectal cancer. Mol Diagn Ther. 2008;12(2):87-98.\n",
      "Shanmugam C, Jhala NC, Katkoori VR, et al. Prognostic value of mucin 4 expression in colorectal adenocarcinomas. Cancer. 2010;116(15):3577-3586.\n",
      "Sharma S, Zippe CD, Pandrangi L, et al. Exclusion criteria enhance the specificity and positive predictive value of NMP22 and BTA Stat. J Urol. 1999;162(1):53-57.\n",
      "Shen J, Brugge WR, Dimaio CJ, Pitman MB. Molecular analysis of pancreatic cyst fluid: A comparative analysis with current practice of diagnosis. Cancer Cytopathol. 2009;117(3):217-227.\n",
      "Sherman M. Surveillance for hepatocellular carcinoma in adults with chronic liver disease. UpToDate [online serial]. Waltham, MA: UpToDate; 2008.\n",
      "Shimamura T, Sakamoto M, Ino Y, et al. Clinicopathological significance of galectin-3 expression in ductal adenocarcinoma of the pancreas. Clin Cancer Res. 2002;8(8):2570-2575.\n",
      "Shimizu Y, Furuya H, Greenwood PB, et al. A multiplex immunoassay for the non-invasive detection of bladder cancer. J Transl Med. 2016;14:31.\n",
      "Shin SJ, Hwang JW, Ahn JB, et al. Circulating vascular endothelial growth factor receptor 2/pAkt-positive cells as a functional pharmacodynamic marker in metastatic colorectal cancers treated with antiangiogenic agent. Invest New Drugs. 2013;31(1):1-13.\n",
      "Shirabe K, Itoh S, Yoshizumi T, et al. The predictors of microvascular invasion in candidates for liver transplantation with hepatocellular carcinoma-with special reference to the serum levels of des-gamma-carboxy prothrombin. J Surg Oncol. 2007;95(3):235-340.\n",
      "Shore N, Concepcion R, Saltzstein D, et al. Clinical utility of a biopsy-based cell cycle gene expression assay in localized prostate cancer. Curr Med Res Opin. 2014;30(4):547-553.\n",
      "Siddiqui AA, Kowalski TE, Kedika R, et al. EUS-guided pancreatic fluid aspiration for DNA analysis of KRAS and GNAS mutations for the evaluation of pancreatic cystic neoplasia: A pilot study. Gastrointest Endosc. 2013;77(4):669-670.\n",
      "Sidiropoulos M, Obregon R, Cooper C, et al. Primary dermal melanoma: A unique subtype of melanoma to be distinguished from cutaneous metastatic melanoma: A clinical, histologic, and gene expression-profiling study. J Am Acad Dermatol. 2014;71(6):1083-1092.\n",
      "Silvestri GA, Tanner NT, Kearney P, et al; PANOPTIC Trial Team. Assessment of plasma proteomics biomarkerâs ability to distinguish benign from malignant lung nodules: : Results of the PANOPTIC (Pulmonary Nodule Plasma Proteomic Classifier) Trial. Chest. 2018;154(3):491-500.\n",
      "Silvestri GA, Vachani A, Whitney D, et al; for the AEGIS Study Team. A bronchial genomic classifier for the diagnostic evaluation of lung cancer.  N Engl J Med. 2015;373(3):243-251.\n",
      "Sinakos E, Saenger AK, Keach J, et al. Many patients with primary sclerosing cholangitis and increased serum levels of carbohydrate antigen 19-9 do not have cholangiocarcinoma. Clin Gastroenterol Hepatol. 2011;9(5):434-439.\n",
      "Singhal U, Morgan TM, Tosoian JJ. Editorial comment. Urology. 2019;126:82.\n",
      "Singletary SE, Allred C, Ashley P, et al. Revision of the American Joint Committee on Cancer staging system for breast cancer. J Clin Oncol. 2002;20(17):3628-3636.\n",
      "Sinicrope FA, Shi Q, Hermitte F, et al. Contribution of Immunoscore and molecular features to survival prediction in stage III colon cancer. JNCI Cancer Spectrum. 2020;4(3).\n",
      "Smartt P. A comparison of gene expression profiling tests for breast cancer. A systematic review update. HSAC Report. Christchurch, New Zealand; Health Services Assessment Collaboration (HSAC); November 2009;3 (16).\n",
      "Smeets A, Christiaens M. Implications of the sentinel lymph node procedure for local and systemic adjuvant treatment. Curr Opin Oncol. 2005; 17:539-544.\n",
      "Smith IE, Yeo B, Schiavon, G. The optimal duration and selection of adjuvant endocrine therapy for breast cancer: How long is enough? ASCO Educational Book 2014. ASCO University. Alexandria, VA: American Society for Clinical Oncology (ASCO); 2014.\n",
      "Smith RA, Cokkinides V, von Eschenbach AC, et al. American Cancer Society guidelines for the early detection of cancer. CA Cancer J Clin. 2002;52(1):8-22.\n",
      "Society for Surgery of the Alimentary Tract (SSAT). Operative treatment for chronic pancreatitis. Manchester, MA: SSAT; 2004.\n",
      "Society of Gynecologic Oncologists (SGO). Society of Gynecologic Oncologists statement regarding OvaCheckâ¢. Position Statements. Chicago, IL: SGO; February 7, 2004. Available at: http://www.sgo.org/policy/position_statement.cfm. Accessed June 4, 2004.\n",
      "Society of Gynecologic Oncologists (SGO).. Statement regarding OvaSure. Chicago, IL: SGO; July 2, 2008. Available at: http://blog.targethealth.com/?p=1492. Accessed September 2, 2008.\n",
      "Solary E, Drenou B, Campos L, et al; Groupe Ouest Est LeucÃ©mies AiguÃ«s MyÃ©loblastiques. Quinine as a multidrug resistance inhibitor: A phase 3 multicentric randomized study in adult de novo acute myelogenous leukemia. Blood. 2003;15;102(4):1202-1210.\n",
      "Soputro NA, Gracias DN, Dias BH, et al. Utility of urinary biomarkers in primary haematuria: Systematic review and meta-analysis. BJUI Compass. 2022;3(5):334-343.\n",
      "Sparano JA, Gray RJ, Makower DF, et al. Adjuvant chemotherapy guided by a 21-gene expression assay in breast cancer. N Engl J Med. 2018;379(2):111-121.\n",
      "Spindler KG. Methodological, biological and clinical aspects of circulating free DNA in metastatic colorectal cancer. Acta Oncol. 2017;56(1):7-16.\n",
      "Spratt DE, Yousefi K, Deheshi S, et al. Individual patient-level meta-analysis of the performance of the Decipher genomic classifier in high-risk men after prostatectomy to predict development of metastatic disease. J Clin Oncol. 2017;35(18):1991-1998.\n",
      "Spratt DE, Zhang J, Santiago-Jimenez M, et al. Development and validation of a novel integrated clinical-genomic risk group classification for localized prostate cancer. J Clin Oncol. 2018;36(6):581-590.\n",
      "Srivastava G, Renfro LA, Behrens RJ, et al. Prospective evaluation of a 12-gene assay on treatment recommendations in patients with stage II colon cancer [abstract]. J Clin Oncol. 2012;30(suppl 34):Abstract 453.\n",
      "Stark A, Donahue TR, Reber HA, Hines O. Pancreatic cyst disease: A review. JAMA. 2016;315(17):1882-1893.\n",
      "Stattin P, Vickers AJ, Sjoberg DD, et al. Improving the specificity of screening for lethal prostate cancer using prostate-specific antigen and a panel of kallikrein markers: A nested case-control study. Eur Urol. 2015;68(2):207-213.\n",
      "Steinberg W. The clinical utility of the CA19-9 tumor associated antigen. Am J Gastroenterol. 1990;85:350-355.\n",
      "Stemmer SM, Steiner M, Rizel S, et al. Clinical outcomes in patients with node-negative breast cancer treated based on the recurrence score results: Evidence from a large prospectively designed registry. NPJ Breast Cancer. 2017;3:33.\n",
      "Steven C, Michael JK, John H, Phillip GF. Active surveillance use among a low-risk prostate cancer population in a large US payer system: 17-gene genomic prostate score versus other risk stratification methods. Rev Urol. 2017;19(4):203-212.\n",
      "Stewart GD, Van Neste L, Delvenne P, et al. Clinical utility of an epigenetic assay to detect occult prostate cancer in histopathologically negative biopsies: Results of the MATLOC study. J Urol. 2013;189(3):1110-1116.\n",
      "Stewart SL, Townsend JS, Puckett MC, Rim SH. Adherence of primary care physicians to evidence-based recommendations to reduce ovarian cancer mortality. J Womens Health (Larchmt). 2016;25(3):235-241.\n",
      "Storr J, Dolan G, Coustan-Smith E, et al. Value of monoclonal anti-myeloperoxidase (MPO7) for diagnosing acute leukaemia. J Clin Pathol. 1990;43(10):847-849.\n",
      "Stoyanova R, Pollack A, Takhar M, et al. Association of multiparametric MRI quantitative imaging features with prostate cancer gene expression in MRI-targeted prostate biopsies. Oncotarget. 2016;7(33):53362-53376.\n",
      "Strata Oncology, Inc. Strata Oncology announces launch and Medicare coverage for Strata Select, a first-of-its-kind immunotherapy test. News and Publications. Ann Arbor, MI: Strata Oncology; April 25, 2023. Available at: https://strataoncology.com/publications/042523/. Accessed July 31, 2023.\n",
      "Stupp R, Hegi ME, Gilbert MR, et al. Chemoradiotherapy in malignant glioma: standard of care and future directions. J Clin Oncol. 2007;25(26):4127-4136.\n",
      "Sullivan MG. Validity testing indefinitely delays Ovacheck release. FDA raises regulatory issues. Ob.Gyn.News Online. 2004;39(7):5. \n",
      "Sumi T, Katsumata K, Tsuchida A, et al. Correlations of clinicopathological factors with protein expression levels of thymidylate synthase, dihydropyrimidine dehydrogenase and orotate phosphoribosyltransferase in colorectal cancer. Chemotherapy. 2010;56(2):120-126.\n",
      "Sun L, Guan Z, Wei S, et al. Identification of long non-coding and messenger RNAs differentially expressed between primary and metastatic melanoma. Front Genet. 2019;10:292.\n",
      "Sun Q, Pastor L, Du J, et al. A novel xenonucleic acid-mediated molecular clamping technology for early colorectal cancer screening. PLoS One. 2021;16(10):e0244332.\n",
      "Sutcliffe P, Hummel S, Simpson E, et al. Use of classical and novel biomarkers as prognostic risk factors for localised prostate cancer: A systematic review. Health Technol Assess. 2009;13(5):iii, xi-xiii 1-219.\n",
      "Svatek RS, Karam J, Karakiewicz PI, et al. Role of urinary cathepsin B and L in the detection of bladder urothelial cell carcinoma. J Urol. 2008;179(2):478-484; discussion 484.\n",
      "Svoboda RM, Glazer AM, Farberg AS, Rigel DS. Factors affecting dermatologists' use of a 31-gene expression profiling test as an adjunct for predicting metastatic risk in cutaneous melanoma. J Drugs Dermatol. 2018;17(5):544-547.\n",
      "Swetter SM, Tsao H, Bichakjian CK, et al. Guidelines of care for the management of primary cutaneous melanoma. J Am Acad Dermatol. 2019;80(1):208-250.\n",
      "Symonds EL, Pedersen SK, Murray D, et al. Circulating epigenetic biomarkers for detection of recurrent colorectal cancer. Cancer. 2020;126(7):1460-1469.\n",
      "Takara K, Sakaeda T, Okumura K. An update on overcoming MDR1-mediated multidrug resistance in cancer chemotherapy. Curr Pharm Des. 2006;12(3):273-286.\n",
      "Tanaka  M, Fernandez-del Castillo C, Adsay V, et al; International Association of Pancreatology. International consensus guidelines 2012 for the management of IPMN and MCN of the pancreas. Pancreatology. 2012;12(3):183-197.\n",
      "Tanner NT, Springmeyer SC, Porter A, et al. Assessment of integrated classifier's ability to distinguish benign from malignant lung nodules: Extended analyses and 2-year follow-up results of the PANOPTIC (Pulmonary Nodule Plasma Proteomic Classifier) Trial. Chest. 2021;159(3):1283-1287.\n",
      "Taplin ME, Balk SP. Has the Time Arrived for Biomarker-Directed Therapy in Castration-Resistant Prostate Cancer? JAMA Oncol. 2015;1(5):577-579.\n",
      "Tarawneh TS, Rodepeter FR, Teply-Szymanski J, et al. Combined focused next-generation sequencing assays to guide precision oncology in solid tumors: A retrospective analysis from an institutional molecular tumor board. Cancers (Basel). 2022;14(18):4430.\n",
      "Taylor CR. Cirrhosis. eMedicine Medicine Topic 175. Omaha, NE: eMedicine.com. January 7, 2009. Available at: http://emedicine.medscape.com/article/366426-overview. Accessed January 13, 2009.\n",
      "Tefferi A. Overview of the myeloproliferative neoplasms. UpToDate [online serial]. Waltham, MA: UpToDate; reviewed December 2019.\n",
      "Tefferi A, Verstovsek S, Pardanani A. How we diagnose and treat WHO-defined systemic mastocytosis in adults. Haematologica. 2008;93(1):6-9.\n",
      "Tesarova P, Kalousova M, Zima T, Tesar V. HMGB1, S100 proteins and other RAGE ligands in cancer - markers, mediators and putative therapeutic targets. Biomed Pap Med Fac Univ Palacky Olomouc Czech Repub. 2016;160(1):1-10.\n",
      "Thermo Fisher Scientific, Inc. FDA approves first companion diagnostic test to simultaneously screen for multiple non-small cell lung cancer therapies. Press Release. Carlsbad, CA: Thermo Fisher Scientific; June 22, 2017.\n",
      "Thermo Fisher Scientific, Inc. Oncomine Dx target test technical and validation information. Oncology Diagnostics [website]. Carlsbad, CA: Thermo Fisher Scientific; 2017. Available at: https://www.thermofisher.com/us/en/home/clinical/diagnostic-testing/condition-disease-diagnostics/oncology-diagnostics/oncomine-dx-target-test/oncomine-dx-target-test-us-only/oncomine-dx-target-test-technical-validation-information.html. Accessed October 2, 2017. \n",
      "Thomas KW, Gould MK. Overview of the initial evaluation, diagnosis, and staging of patients with suspected lung cancer. UpToDate [online serial]. Waltham, MA: UpToDate; reviewed January 2017a.\n",
      "Thomas KW, Gould MK. Overview of the initial evaluation, diagnosis, and staging of patients with suspected lung cancer. UpToDate [online serial]. Waltham, MA: NCCN; reviewed June 2019a.\n",
      "Thomas KW, Gould MK. Procedures for tissue biopsy in patients with suspected non-small cell lung cancer. UpToDate [online serial]. Waltham, MA: UpToDate; reviewed January 2017c.\n",
      "Thomas KW, Gould MK. Selection of modality for diagnosis and staging of patients with suspected non-small cell lung cancer. UpToDate [online serial]. Waltham, MA: UpToDate; reviewed January 2017b.\n",
      "Thomas KW, Gould MK. Selection of modality for diagnosis and staging of patients with suspected non-small cell lung cancer. UpToDate [online serial]. Waltham, MA. Last reviewed June 2019b.\n",
      "Thompson JC, Yee SS, Troxel AB, et al. Detection of therapeutically targetable driver and resistance mutations in lung cancer patients by next-generation sequencing of cell-free circulating tumor DNA. Clin Cancer Res. 2016;22(23):5772-5782.\n",
      "Tice JA. The 70-gene signature (MammaPrint) as a guide for the management of early stage breast cancer. Technology Assessment. San Francisco, CA: California Technology Assessment Forum (CTAF); June 2, 2010.\n",
      "Tie J, Cohen JD, Wang Y, et al. Circulating tumor DNA analyses as markers of recurrence risk and benefit of adjuvant therapy for stage III colon cancer. JAMA Oncol. 2019;5(12):1710-1717.\n",
      "Tinzl M, Marberger M, Horvath S, Chypre C. DD3PCA3 RNA analysis in urine--a new perspective for detecting prostate cancer. Eur Urol. 2004;46(2):182-186; discussion 187.\n",
      "Tiu SC, Chan AOK, Taylor NF, et al. Use of urinary steroid profiling for diagnosing and monitoring adrenocortical tumours. Hong Kong Med J. 2009;15(6):463-470.\n",
      "Tiwana SK, Smith A, Leggett L, et al. Use of Oncotype DX for guiding adjuvant chemotherapy decisions in early stage invasive breast cancer patients in Alberta. A Health Technology Assessment. Calgary, AB: Health Technology Assessment Unit, Institute for Public Health, University of Calgary; August 7, 2013.\n",
      "Tomlins SA, Day JR, Lonigro RJ, et al. Urine TMPRSS2:ERG Plus PCA3 for individualized prostate cancer risk assessment. Eur Urol. 2016;70(1):45-53.\n",
      "Tosoian JJ, Loeb S, Kettermann A, et al. Accuracy of PCA3 measurement in predicting short-term biopsy progression in an active surveillance program. J Urol. 2010;183(2):534-538.\n",
      "Toth K, Bartak BK, Tulassay Z, Molnar B. Circulating cell-free nucleic acids as biomarkers in colorectal cancer screening and diagnosis. Expert Rev Mol Diagn. 2016;16(2):239-252.\n",
      "Toyoda H, Kumada T, Kiriyama S, et al. Prognostic significance of simultaneous measurement of three tumor markers in patients with hepatocellular carcinoma. Clin Gastroenterol Hepatol. 2006;4(1):111-117.\n",
      "Trikalinos TA, Terasawa T, Raman G, et al. A systematic review of loss-of-heterozygosity based topographic genotyping with PathfinderTG. Technology Assessment Report. Project ID: GEND0308. Prepared by the Tufts Evidence-based Practice Center for the Agency for Healthcare Research and Quality (AHRQ) under Contract No. HHSA 290 2007 10055 I. Rockville, MD: AHRQ; March 1, 2010.\n",
      "Trock BJ, Brotzman MJ, Mangold LA, et al. Evaluation of GSTP1 and APC methylation as indicators for repeat biopsy in a high-risk cohort of men with negative initial prostate biopsies. BJU Int. 2012;110(1):56-62.\n",
      "Troyer DA, Lucia MS, de BruÃ¯ne AP, et al. Prostate cancer detected by methylated gene markers in histopathologically cancer-negative tissues from men with subsequent positive biopsies. Cancer Epidemiol Biomarkers Prev. 2009;18(10):OF1-OF6.\n",
      "Tschen J, Davies P, Meek SM, Clarke LE. Clinical use of a diagnostic gene expression signature for melanocytic neoplasms. Cutis. 2021;107(5):264-269.\n",
      "Tucker ME. New test distinguishes benign from malignant thyroid nodules. Medscape Medical News. New York, NY: Medscape; March 17, 2015. Available at: http://www.medscape.com/viewarticle/844818.\n",
      "Turner NC, Kingston B, Kilburn LS, et al. Circulating tumour DNA analysis to direct therapy in advanced breast cancer (plasmaMATCH): A multicentre, multicohort, phase 2a, platform trial. Lancet Oncol. 2020;21(10):1296-1308.\n",
      "Tutrone R, Donovan MJ, Torkler P, et al. Clinical utility of the exosome based ExoDx Prostate (IntelliScore) EPI test in men presenting for initial biopsy with a PSA 2-10 ng/ml. Prostate Cancer Prostatic Dis. 2020;23(4):607-614.\n",
      "Underwood JJ, Quadri RS, Kalva SP et al. Liquid biopsy for cancer: Review and implications for the radiologist. RSNA Radiology. 2019;294(1). \n",
      "UPMC Molecular and Genomic Pathology Laboratory. Pancrea genomic classifier - pancreatic cyst fluid NGS analysis. Pittsburgh, PA: UPMC; 2022. Available at: https://mgp.upmc.com/Home/Test/PanSeq_details. Accessed July 8, 2022.\n",
      "U.S. Food and Drug Administration. GeneSearch BLN Testkit. P060017. Rockville, MD: FDA. July 16, 2007. \n",
      "U.S. Food and Drug Administration (FDA), Center for Devices and Radiologic Health (CDRH), Office of In Vitro Diagnostic Device Evaluation and Safety (OVID). Letter to Laboratory Corporation of America. Re: Ovacheck - Ovarian Cancer Screen. Rockville, MD: FDA; March 2, 2004. \n",
      "U.S. Food and Drug Administration (FDA), Center for Devices and Radiologic Health (CDRH). CellSearchâ¢ Epithelial Cell Kit / CellSpotterâ¢ Analyzer. 510(k) no. K031588. Rockville, MD: FDA; January 21, 2004.\n",
      "U.S. Food and Drug Administration (FDA), Center for Devices and Radiologic Health (CDRH), Office of In Vitro Diagnostic Device Evaluation and Safety (OVID). Warning Letter to Laboratory Corporation of America. Rockville, MD: FDA; September 29, 2008. \n",
      "U.S. Food and Drug Administration (FDA). FDA approves encorafenib and binimetinib in combination for unresectable or metastatic melanoma with BRAF mutations. Press Release. Silver Spring, MD: FDA; June 27, 2018. \n",
      "U.S. Food and Drug Administration (FDA). FDA approves new uses for two drugs administered together for the treatment of BRAF-positive anaplastic thyroid cancer. Press Release. Silver Spring, MD: FDA; May 4, 2018. \n",
      "U.S. Food and Drug Administration (FDA). FDA clears test that helps identify type of cancer in tumor sample. FDA News. Rockville, MD: FDA; July 31, 2008. \n",
      "U.S. Food and Drug Administration (FDA). FDA grants regular approval to dabrafenib and trametinib combination for metastatic NSCLC with BRAF V600E mutation. Approved Drugs. Silver Spring, MD: FDA; June 22, 2017. \n",
      "U.S. Food and Drug Administration (FDA). FDA unveils a streamlined path for the authorization of tumor profiling tests alongside its latest product action. Press Release. Silver Spring, MD: FDA; November 15, 2017. \n",
      "U.S. Food and Drug Administration (FDA). Ovarian cancer screening tests: Safety communication - FDA recommends against use. Safety Alerts for Human Medical Products. Silver Spring, MD: FDA; September 7, 2016.\n",
      "U.S. Food and Drug Administration CDRH Consumer Information. Genesearch BLN Test Kit - P060017. Rockville, MD: FDA. August 3, 2007. \n",
      "U.S. Preventive Services Task Force (USPSTF). Screening for bladder cancer in adults. In: Guide to Clinical Preventive Services. Report of the U.S. Preventive Services Task Force. 3rd ed. Rockville, MD: Agency for Healthcare Research and Quality (AHRQ); June 2004.\n",
      "U.S. Preventive Services Task Force (USPSTF). Screening for ovarian cancer. In: Guide to Clinical Preventive Services. Report of the U.S. Preventive Services Task Force. 3rd ed. Rockville, MD: Agency for Healthcare Research and Quality (AHRQ); May 2004.\n",
      "U.S. Preventive Services Task Force. Chemoprevention of breast cancer: Recommendations and rationale. Ann Intern Med. 2002;137(1):56-58.\n",
      "U.S. Preventive Services Task Force. Lung cancer screening: Recommendation statement. Ann Intern Med. 2004;140(9):738-739.\n",
      "Ueland FR, Li AJ. Serum biomarkers for evaluation of an adnexal mass for epithelial carcinoma of the ovary, fallopian tube, or peritoneum. UpToDate [online serial]. Waltham, MA: NCCN; reviewed January 2019.\n",
      "Ueland, FR, Desimone, CP, Seamon LG, et al. Effectiveness of a multivariate index assay in the preoperative assessment of ovarian tumors. Obstet Gynecol. 2011;117(6):1289-1297.\n",
      "Urban N, McIntosh MW, Andersen M, Karlan BY. Ovarian cancer screening. Hematol Oncol Clin North Am. 2003;17(4):989-1005.\n",
      "Urban RR, Smith A, Agnew K, et al.  Evaluation of a validated biomarker test in combination with a symptom index to predict ovarian malignancy. Int J Gynecol Cancer. 2017;27(2):233-238.\n",
      "Vachani A, Whitney DH, Parsons EC, et al. Clinical utility of a bronchial genomic classifier in patients with suspected lung cancer. Chest. 2016;150(1):210-218.\n",
      "Valent P, Horny HP, Bennett JM, et al. Definitions and standards in the diagnosis and treatment of the myelodysplastic syndromes: Consensus statements and report from a working conference. Leuk Res. 2007 Jun;31(6):727-736.\n",
      "Van Cutsem E, Lang I, D'haens G, et al. KRAS status and efficacy in the first-line treatment of patients with metastatic colorectal cancer (mCRC) treated with FOLFIRI with or without cetuximab: The CRYSTAL experience. J Clin Oncol. 2008;26: Abstr. 2. \n",
      "van de Stolpe A, Verhaegh W, Blay J-Y, et al. RNA based approaches to profile oncogenic pathways from low quantity samples to drive precision oncology strategies. Front Genet. 2021;11:598118.\n",
      "van de Vijver, He YD, vanât Veer LJ, et al. A gene-expression signature as a predictor of survival in breast cancer. N Engl J Med. 2002;347(25):1999-2009.\n",
      "van den Bent MJ, Kros JM. Predictive and prognostic markers in neuro-oncology. J Neuropathol Exp Neurol. 2007;66(12):1074-1081.\n",
      "Van den Bulcke, M, San Miguel L, Salgado R,, et al. Next generation sequencing panels for target therapy in oncology and haematology-oncology. KCE Report 240. Brussels, Belgium: Belgian Healthcare Knowledge Centre (KCE); March 19, 2015.\n",
      "Van Den Eeden SK, Lu R, Zhang N,. et al., A biopsy-based 17-gene genomic prostate score as a predictor of metastases and prostate cancer death in surgically treated men with clinically localized disease. Eur Urol. 2018;73(1):129-138.\n",
      "van Gils MP, Hessels D, van Hooij O, et al The time-resolved fluorescence-based PCA3 test on urinary sediments after digital rectal examination; A Dutch multicenter validation of the diagnostic performance. Clin Cancer Res. 2007;13(3):939-943.\n",
      "Van Neste L, Bigley J, Toll A, et al. A tissue biopsy-based epigenetic multiplex PCR assay for prostate cancer detection. BMC Urol. 2012a; 12:16.\n",
      "Van Neste L, Groskopf J, Grizzle WE, et al. Epigenetic risk score improves prostate cancer risk assessment. Prostate. 2017;77(12):1259-1264.\n",
      "Van Neste L, Herman JG, Otto G, et al. The epigenetic promise for prostate cancer diagnosis. Prostate. 2012b;72(11):1248-1261.\n",
      "Van Neste L, Partin AW, Stewart GD, et al. Risk score predicts high-grade prostate cancer in DNA-methylation positive, histopathologically negative biopsies. Prostate. 2016;76(12):1078-1087.\n",
      "Vardiman JW, Harris NL, Brunning RD. The World Health Organization (WHO) classification of the myeloid neoplasms. Blood. 2002;100(7):2292-2302.\n",
      "Vashistha V, Poonnen PJ, Snowdon JL, et al. Medical oncologists' perspectives of the Veterans Affairs National Precision Oncology Program. PLoS One. 2020;15(7):e0235861.\n",
      "Vege SS, Ziring B, Jain R, Moayyedi P; Clinical Guidelines Committee. American Gastroenterological Association Institute guideline on the diagnosis and management of asymptomatic neoplastic pancreatic cysts. Gastroenterology. 2015;148(4):819-822.\n",
      "Venook AP, Niedzwiecki D, Lopatin M, et al. Biologic determinants of tumor recurrence in stage II colon cancer: Validation study of the 12-gene recurrence score in cancer and leukemia group B (CALGB) 9581. J Clin Oncol. 2013;31(14):1775-1781.\n",
      "Veridex, LLC. CellSearchâ¢ [website]. Warren, NJ: Veridex; 2005. Available at: http://www.veridex.com. Accessed December 17, 2004.\n",
      "Vetto JT, Hsueh EC, Gastman BR, et al. Guidance of sentinel lymph node biopsy decisions in patients with t1-t2 melanoma using gene expression profiling. Future Oncol. 2019;15(11):1207-1217.\n",
      "Viale G, Bosari S, Maxxarol G, et al. Intra-operative examination of axillary sentinel lymph nodes in breast carcinoma patients. Cancer 1999; 85(11):2433-2438.\n",
      "Viale G, Dell'Orto P, Biasi MO, et al. Comparative evaluation of an extensive histopathologic examination and a real-time reverse-transcription-polymerase chain reaction assay for mammaglobin and cytokeratin 19 on axillary sentinel lymph nodes of breast carcinoma patients. Ann Surg. 2008;247(1):136-142.\n",
      "Vickers AJ, Eastham JA, Scardino PT, Lilja H. The Memorial Sloan Kettering Cancer Center Recommendations for Prostate Cancer Screening. Urology. 2016;91:12-18.\n",
      "Vijayashree Priyadharsini J, Paramasivam A. Virtual screening of mutations in antioxidant genes and its putative association with HNSCC: An in silico approach. Mutat Res. 2020;821:111710.\n",
      "Villaflor V, Won B, Nagy R, et al. Biopsy-free circulating tumor DNA assay identifies actionable mutations in lung cancer. Oncotarget. 2016;7(41):66880-66891.\n",
      "Viome Life Sciences. Our tests [website]. Bothell, WA: Viome Life Sciences; 2022. Available at: https://www.viomelifesciences.com/tests. Accessed December 29, 2021.\n",
      "Vishwakarma S, Pandey R, Singh R, et al. Expression of H19 long non-coding RNA is down-regulated in oral squamous cell carcinoma. J Biosci. 2020;45:145.\n",
      "Visintin I, Feng Z, Longton G, et al. Diagnostic markers for early detection of ovarian cancer. Clin Cancer Res. 2008;14(4):1065-1072.\n",
      "Voigt JD, Zappala SM, Vaughan ED, Wein AJ. The Kallikrein Panel for prostate cancer screening: Its economic impact. Prostate. 2014;74(3):250-259.\n",
      "Volckmar AL, Sultmann H, Riediger A, et al. A field guide for cancer diagnostics using cell-free DNA: From principles to practice and clinical applications. Genes Chromosomes Cancer. 2018;57(3):123-139.\n",
      "Von Hoff DD, Stephenson JJ Jr, Rosen P, et al. Pilot study using molecular profiling of patients' tumors to find potential targets and select treatments for their refractory cancers. J Clin Oncol. 2010;28(33):4877-4883.\n",
      "Wang LX, Wan C, Dong ZB, et al. Integrative analysis of long noncoding RNA (lncRNA), microRNA (miRNA) and mRNA expression and construction of a competing endogenous RNA (ceRNA) network in metastatic melanoma. Med Sci Monit. 2019;25:2896-2907.\n",
      "Wang R, Zhao A, Cao N, et al. The value of circulation tumor DNA in predicting postoperative recurrence of colorectal cancer: A meta-analysis. Int J Colorectal Dis. 2020;35(8):1463-1475.\n",
      "Wang WLW, Sorokin I, Aleksic I, et al. Expression of small noncoding RNAs in urinary exosomes classifies prostate cancer into indolent and aggressive disease. J Urol. 2020;204(3):466-475.\n",
      "Wang XT, Kong FB, Mai W, et al. MUC1 immunohistochemical expression as a prognostic factor in gastric cancer: Meta-analysis. Dis Markers. 2016;2016:9421571.\n",
      "Wang Y, Li L, Cohen JD, et al. Prognostic potential of circulating tumor DNA measurement in postoperative surveillance of nonmetastatic colorectal cancer. JAMA Oncol. 2019 5(8):1118-1123.\n",
      "Wang Z, Chen JQ, Liu JL, et al. Polymorphisms in ERCC1, GSTs, TS and MTHFR predict clinical outcomes of gastric cancer patients treated with platinum/5-Fu-based chemotherapy: A systematic review. BMC Gastroenterol. 2012;12:137.\n",
      "Wang ZY, Wang LJ, Zhang Q, et al. Single quantum dot-based nanosensor for sensitive detection of 5-methylcytosine at both CpG and non-CpG sites. Chem Sci. 2017;9(5):1330-1338.\n",
      "Ward S, Scope A, Rafia R, et al. Gene expression profiling and expanded immunohistochemistry tests to guide the use of adjuvant chemotherapy in breast cancer management: A systematic review and cost-effectiveness analysis. Health Technol Assess. 2013;17(44):1-302.\n",
      "Ware, MR, Smith, A, DeSimone CP, et al. Performance of the American College of Obstetricians and Gynecologists' ovarian tumor referral guidelines with a multivariate index assay. Obstet Gynecol. 2011;117(6):1298-1306.\n",
      "Watine J, Charet JC. Are the ATS (American Thoracic Society) and the ERS (European Respiratory Society) correct in not recommending routine tumor marker assays for screening, staging, or evaluation of non-small cell lung cancer? Rev Mal Respir. 1999;16(2):139-149.\n",
      "Weigelt B, Bosma AJ, Hart AA, et al. Marker genes for circulating tumour cells predict survival in metastasized breast cancer patients. Br J Cancer. 2003;88(7):1091-1094.\n",
      "Weitz IC; Liebman HA. Des-gamma-carboxy (abnormal) prothrombin and hepatocellular carcinoma: A critical review. Hepatology. 1993;18(4):990-997.\n",
      "Welch S, Kocha RB, Rumble K, et al., Gastrointestinal Cancer Disease Site Group. Cancer Care Ontario. The role of bevacizumab (avastin) combined with chemotherapy in the treatment of patients with advanced colorectal cancer: Guideline recommendations.  Evidence-based Series #2-25: Section 1. Toronto, ON: Cancer Care Ontario; May 2008.\n",
      "Werner M, Faser C, Silverberg M. Clinical utility and validation of emerging biochemical markers for mammary adenocarcinoma. Clin Chem. 1993;39(11 Pt 2):2386-2396.\n",
      "Westwood M, Ramaekers B, Armstrong A, et al.. Tests in secondary care to identify people at high risk of ovarian cancer. PROSPERO. 2016:CRD42016053326.\n",
      "Williams A, Hamilton O, Likar C, et al. \"The benefit of positron emission tomography/computed tomography in stage I and stage II melanomas with high-risk Decisiondx-Melanoma scores\". Am Surg. 2022;88(7):1446-1451.\n",
      "White MJ, Roife D, Gomer RH. Galectin-3 binding protein secreted by breast cancer cells inhibits monocyte-derived fibrocyte differentiation. J Immunol. 2015;195(4):1858-1867.\n",
      "White NM, Zhao SG, Zhang J, et al. Multi-institutional analysis shows that low PCAT-14 expression associates with poor outcomes in prostate cancer. Eur Urol. 2016 Jul 22.\n",
      "Whitman ED, Koshenkov VP, Gastman BR, et al. Integrating 31-gene expression profiling with clinicopathologic features to optimize cutaneous melanoma sentinel lymph node metastasis prediction. JCO Precis Oncol. 2021;5:PO.21.00162.\n",
      "Whitney DH, Elashoff MR, Porta-Smith K, et al. Derivation of a bronchial genomic classifier for lung cancer in a prospective study of patients undergoing diagnostic bronchoscopy. BMC Med Genomics. 2015;8(1):18.\n",
      "Wild C, GrÃ¶ssmann N. FoundationOne CDx: Genetic profiling of solid tumours. Rapid Assessment 014. Vienna, Austria: Ludwig Boltzmann Institute for Health Technology Assessment; 2019.\n",
      "Willis J, Lefterova MI, Artyomenko A, et al. Validation of microsatellite instability detection using a comprehensive plasma-based genotyping panel. Clin Cancer Res. 2019;25(23):7035-7045.\n",
      "Wobbes T, Thomas CM, Segers MF, Nagengast FM. Evaluation of seven tumor markers in the treatment sera of patients with gastric carcinoma. Cancer. 1992;69(8):2036-2041.\n",
      "Wojno KJ, Costa FJ, Cornell RJ, et al. Reduced rate of repeated prostate biopsies observed in ConfirmMDx clinical utility field study. Am Health Drug Benefits. 2014;7(3):129-134.\n",
      "Woldu SL, Lotan Y.  Re: ctDNA guiding adjuvant immunotherapy in urothelial carcinoma. Euro Urol. 2021;80(4):517-518.\n",
      "Wolf I, Ben-Baruch N, Shapira-Frommer R, et al. Association between standard clinical and pathologic characteristics and the 21-gene recurrence score in breast cancer patients: A population-based study. Cancer. 2008;112(4):731-736. \n",
      "Woodard GA, Wang SX, Kratz JR, et al. Adjuvant chemotherapy guided by molecular profiling and improved outcomes in early stage, non-small-cell lung cancer. Clin Lung Cancer. 2018;19(1):58-64.\n",
      "Worley LA, Onken MD, Person E, et al. Transcriptomic versus chromosomal prognostic markers and clinical outcome in uveal melanoma. Clin Cancer Res. 2007;13(5):1466-1471.\n",
      "Woyach JA, Ruppert AS, Guinn D, et al. BTKC481S-mediated resistance to ibrutinib in chronic lymphocytic leukemia. J Clin Oncol. 2017;35(13):1437-1443.\n",
      "Wright T, McGechan A. Breast cancer: New technologies for risk assessment and diagnosis. Mol Diagn. 2003;7(1):49-55.\n",
      "Wu T, Giovannucci E, Welge J, et al. Measurement of GSTP1 promoter methylation in body fluids may complement PSA screening: A meta-analysis. Br J Cancer. 2011;105(1):65-73.\n",
      "Wu Z, Xu Z, Yu B, et al. The potential diagnostic value of exosomal long noncoding RNAs in solid tumors: A meta-analysis and systematic review. Biomed Res Int. 2020;2020:6786875.\n",
      "Xu YJ, Du Y, Fan Y. Long noncoding RNAs in lung cancer: What we know in 2015. Clin Transl Oncol. 2016;18(7):660-665.\n",
      "Yaegashi H, Izumi K, Kadomoto S, et al. High serum CA19-9 concentration indicates high chemosensitivity and better survival in advanced urothelial carcinoma. Anticancer Res. 2019;39(1):375-380.\n",
      "Yamanaka T, Oki E, Yamazaki K, et al. 12-gene recurrence score assay stratifies the recurrence risk in stage II/III colon cancer with surgery alone: The SUNRISE Study. J Clin Oncol. 2016;34(24):2906-2913.\n",
      "Yang C, Zou K, Zheng L, Xiong B. Prognostic and clinicopathological significance of circulating tumor cells detected by RT-PCR in non-metastatic colorectal cancer: A meta-analysis and systematic review. BMC Cancer. 2017;17(1):725. \n",
      "Yang JS, Lee JC, Byeon SK, et al. Size dependent lipidomic analysis of urinary exosomes from patients with prostate cancer by flow field-flow fractionation and nanoflow liquid chromatography-tandem mass spectrometry. Anal Chem. 2017;89(4):2488-2496.\n",
      "Yardley DA, Kaufman PA, Huang W, et al. Quantitative measurement of HER2 expression in breast cancers: Comparison with 'real-world' routine HER2 testing in a multicenter Collaborative Biomarker Study and correlation with overall survival. Breast Cancer Res. 2015;17:41.  \n",
      "Yoshida A, Tsuta K, Nitta H, et al. Bright-field dual-color chromogenic in situ hybridization for diagnosing echinoderm microtubule-associated protein-like 4-anaplastic lymphoma kinase-positive lung adenocarcinomas. Am J Surg Pathol. 2011;35(8):1226-1234.\n",
      "Young WF, Jr, Kebebew E. Evaluation and management of the adrenal incidentaloma. UpToDate [online serial]. Waltham, MA: UpToDate; reviewed September 2020.\n",
      "Yu D, Shi J, Sun T, et al. Pharmacogenetic role of ERCC1 genetic variants in treatment response of platinum-based chemotherapy among advanced non-small cell lung cancer patients. Tumour Biol. 2012;33(3):877-884.\n",
      "Yu JJ, Xia SJ. Novel role of microRNAs in prostate cancer. Chin Med J (Engl). 2013;126(15):2960-2964.\n",
      "Yu Z, Li Z, Cai B, et al. Association between the GSTP1 Ile105Val polymorphism and prostate cancer risk: A systematic review and meta-analysis. Tumour Biol. 2013 34(3):1855-1863.\n",
      "Yuan LW, Tang W, Kokudo N, et al. Measurement of des-gamma-carboxy prothrombin levels in cancer and non-cancer tissue in patients with hepatocellular carcinoma. Oncol Rep. 2004;12(2):269-273.\n",
      "Yuen AR, Sikic BI. Multidrug resistance in lymphomas. J Clin Oncol. 1994;12(11):2453-2459.\n",
      "Yuen MF, Lai CL. Serological markers of liver cancer. Best Pract Res Clin Gastroenterol. 2005;19(1):91-99.\n",
      "Yurkovetsky Z, Skates S, Lomakin A, et al. Development of a multimarker assay for early detection of ovarian cancer. J Clin Oncol. 2010;28(13):2159-2166.\n",
      "Zaborowski AM, Winter DC , Lynch L, et al. The therapeutic and prognostic implications of immunobiology in colorectal cancer: A review. Br J Cancer. 2021;125:1341-1349.\n",
      "Zager JS, Gastman BR, Leachman S, et al. Performance of a prognostic 31-gene expression profile in an independent cohort of 523 cutaneous melanoma patients. BMC Cancer. 2018;18(1):130.\n",
      "Zakria D, Brownstone N, Rigel DS. The integrated 31-gene expression profile (i31-GEP) test for cutaneous melanoma outperforms a clinicopathologic-only nomogram at identifying patients who can forego sentinel lymph node biopsy. Skin. 2022;6(6):453-463.\n",
      "Zehir A, Benayed R, Shah RH, et al. Mutational landscape of metastatic cancer revealed from prospective clinical sequencing of 10,000 patients. Nat Med. 2017;23(6):703-713.\n",
      "Zhang H, Luo M, Liang X, et al. Galectin-3 as a marker and potential therapeutic target in breast cancer. PLoS One. 2014;9(9):e103482.\n",
      "Zhang J, Zhu Z, Liu Y, et al. Diagnostic value of multiple tumor markers for patients with esophageal carcinoma. PLoS One. 2015;10(2):e0116951.\n",
      "Zhang W, Zhong T, Chen Y, et al. LC-MS/MS-based targeted proteomics quantitatively detects the interaction between p53 and MDM2 in breast cancer. J Proteomics. 2017;152:172-180.\n",
      "Zhang Y, Schnabel CA, Schroeder BE, et al. Breast cancer index identifies early-stage estrogen receptor-positive breast cancer patients at risk for early- and late-distant recurrence. Clin Cancer Res. 2013;19(15):4196-4205.\n",
      "Zhang Z. An in vitro diagnostic multivariate index assay (IVDMIA) for ovarian cancer: Harvesting the power of multiple biomarkers. Rev Obstet Gynecol. 2012;5(1):35-41.\n",
      "Zhou C, Yuan Z, Ma W, et al. Clinical utility of tumor genomic profiling in patients with high plasma circulating tumor DNA burden or metabolically active tumors. J Hematol Oncol. 2018;11(1):129.\n",
      "Zhuang L, Johnson MT. How precisely can prostate cancer be managed? Int Neurourol J. 2016;20(Suppl 2):S120-S130.\n",
      "Zumsteg ZS, Spratt DE, Pei I, et al. A new risk classification system for therapeutic decision making with intermediate-risk prostate cancer patients undergoing dose-escalated external-beam radiation therapy. Eur Urol. 2013;64(6):895-902.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      " \n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Policy History\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Last Review\n",
      "               10/03/2023\n",
      "              Last Review 10/03/2023\n",
      "\n",
      "                Effective: 09/13/1999       \n",
      "                \n",
      "\n",
      "                Next Review: 03/21/2024      \n",
      "              \n",
      "\n",
      "\n",
      "\n",
      " \n",
      "\n",
      "\n",
      "Review History\n",
      "\n",
      "\n",
      "Review History\n",
      "\n",
      "\n",
      "\n",
      " \n",
      "\n",
      "Definitions\n",
      "\n",
      "Definitions\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Additional Information\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Clinical Policy Bulletin Notes\n",
      "\n",
      "Clinical Policy Bulletin Notes\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      " \n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      " \n",
      "\n",
      "\n",
      "\n",
      "\n",
      "  \n",
      "\n",
      "             \n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\t\t\t\n",
      "\t\t\t\tCopyright Aetna Inc. All rights reserved. Clinical Policy Bulletins are developed by Aetna to assist in administering plan benefits and constitute neither offers of coverage nor medical advice. This Clinical Policy Bulletin contains only a partial, general description of plan or program benefits and does not constitute a contract. Aetna does not provide health care services and, therefore, cannot guarantee any results or outcomes. Participating providers are independent contractors in private practice and are neither employees nor agents of Aetna or its affiliates. Treating providers are solely responsible for medical advice and treatment of members. This Clinical Policy Bulletin may be updated and therefore is subject to change. \n",
      "\t\t\t\n",
      "\n",
      "\n",
      "\n",
      "\t\t\t\t \n",
      "\t\t\t\n",
      "\n",
      "\n",
      "\n",
      "\n",
      " Glossary \n",
      " Aetna Mobile App \n",
      " Careers \n",
      "\n",
      "\n",
      "\n",
      "\n",
      " Accessibility Services \n",
      " Terms of Use \n",
      " Investor Info \n",
      "\n",
      "\n",
      "\n",
      "\n",
      " FAQs \n",
      " Program Provisions \n",
      " Interest-Based Ads Policy \n",
      "\n",
      "\n",
      "\n",
      "\n",
      " Legal Notices \n",
      " Plan Disclosures \n",
      " Nondiscrimination Notice \n",
      "\n",
      "\n",
      "\n",
      "\n",
      " Site Map \n",
      " Privacy Center \n",
      " State Directory \n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Copyright © 2001-[current-year] Aetna Inc.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "                  Language services can be provided by calling the number on your member ID card. For additional language assistance:  \n",
      "                  \n",
      "EspaÃ±ol\n",
      "ä¸­æ\n",
      "Tiáº¿ng Viá»t\n",
      "íêµ­ì´\n",
      "Tagalog\n",
      "PÑÑÑÐºÐ¸Ð¹\n",
      "Ø§ÙØ¹Ø±Ø¨ÙØ©\n",
      "KreyÃ²l\n",
      "FranÃ§ais\n",
      "Polski\n",
      "PortuguÃªs\n",
      "Italiano\n",
      "Deutsch\n",
      "æ¥æ¬èª\n",
      "ÙØ§Ø±Ø³Û\n",
      "Other Languagesâ¦\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "close\n",
      "You are now leaving the Aetna website.\n",
      "\n",
      "\n",
      "\t\t\t\n",
      "\t\t\tLinks to various non-Aetna sites are provided for your convenience only. Aetna Inc. and its subsidiary companies are not responsible or liable for the content, accuracy, or privacy practices of linked sites, or for products or services described on these sites. \n",
      "\t\t\t\n",
      "\t\t\n",
      "\n",
      "Continue \n",
      "\n",
      "\n",
      " \n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "import requests\n",
    "from bs4 import BeautifulSoup\n",
    "import re\n",
    "\n",
    "headers = {'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/118.0.0.0 Safari/537.36 Edg/118.0.2088.46'}\n",
    "    \n",
    "response = requests.get(url, headers=headers)    \n",
    "if response.status_code == 200:\n",
    "    html_content = response.text\n",
    "    soup = BeautifulSoup(html_content, 'html.parser')\n",
    "    soup.prettify()\n",
    "    text = soup.get_text()\n",
    "    \n",
    "        \n",
    "print(text) \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "id": "a18b4b4e",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'Tumor Markers - Medical Clinical Policy Bulletins | Aetna'"
      ]
     },
     "execution_count": 45,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "soup.find(\"head\").title.text"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "64ce35b5",
   "metadata": {},
   "source": [
    "### Anthem text"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "id": "553895f4",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'https://www.anthem.com/dam/medpolicies/abc/active/policies/mp_pw_a049877.html'"
      ]
     },
     "execution_count": 33,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "url=url[148]\n",
    "url"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 70,
   "id": "ef27e492",
   "metadata": {},
   "outputs": [],
   "source": [
    "url='https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 71,
   "id": "d43909b6",
   "metadata": {},
   "outputs": [
    {
     "ename": "ConnectionError",
     "evalue": "HTTPSConnectionPool(host='mcgs.bcbsfl.com', port=443): Max retries exceeded with url: /MCG?mcgId=05-82000-28&pv=false (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002574FCCDE20>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[1;31mTimeoutError\u001b[0m                              Traceback (most recent call last)",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\urllib3\\connection.py:174\u001b[0m, in \u001b[0;36mHTTPConnection._new_conn\u001b[1;34m(self)\u001b[0m\n\u001b[0;32m    173\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[1;32m--> 174\u001b[0m     conn \u001b[38;5;241m=\u001b[39m connection\u001b[38;5;241m.\u001b[39mcreate_connection(\n\u001b[0;32m    175\u001b[0m         (\u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_dns_host, \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mport), \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mtimeout, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39mextra_kw\n\u001b[0;32m    176\u001b[0m     )\n\u001b[0;32m    178\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m SocketTimeout:\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\urllib3\\util\\connection.py:95\u001b[0m, in \u001b[0;36mcreate_connection\u001b[1;34m(address, timeout, source_address, socket_options)\u001b[0m\n\u001b[0;32m     94\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m err \u001b[38;5;129;01mis\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m \u001b[38;5;28;01mNone\u001b[39;00m:\n\u001b[1;32m---> 95\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m err\n\u001b[0;32m     97\u001b[0m \u001b[38;5;28;01mraise\u001b[39;00m socket\u001b[38;5;241m.\u001b[39merror(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mgetaddrinfo returns an empty list\u001b[39m\u001b[38;5;124m\"\u001b[39m)\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\urllib3\\util\\connection.py:85\u001b[0m, in \u001b[0;36mcreate_connection\u001b[1;34m(address, timeout, source_address, socket_options)\u001b[0m\n\u001b[0;32m     84\u001b[0m     sock\u001b[38;5;241m.\u001b[39mbind(source_address)\n\u001b[1;32m---> 85\u001b[0m \u001b[43msock\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mconnect\u001b[49m\u001b[43m(\u001b[49m\u001b[43msa\u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m     86\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m sock\n",
      "\u001b[1;31mTimeoutError\u001b[0m: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond",
      "\nDuring handling of the above exception, another exception occurred:\n",
      "\u001b[1;31mNewConnectionError\u001b[0m                        Traceback (most recent call last)",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\urllib3\\connectionpool.py:703\u001b[0m, in \u001b[0;36mHTTPConnectionPool.urlopen\u001b[1;34m(self, method, url, body, headers, retries, redirect, assert_same_host, timeout, pool_timeout, release_conn, chunked, body_pos, **response_kw)\u001b[0m\n\u001b[0;32m    702\u001b[0m \u001b[38;5;66;03m# Make the request on the httplib connection object.\u001b[39;00m\n\u001b[1;32m--> 703\u001b[0m httplib_response \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_make_request\u001b[49m\u001b[43m(\u001b[49m\n\u001b[0;32m    704\u001b[0m \u001b[43m    \u001b[49m\u001b[43mconn\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    705\u001b[0m \u001b[43m    \u001b[49m\u001b[43mmethod\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    706\u001b[0m \u001b[43m    \u001b[49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    707\u001b[0m \u001b[43m    \u001b[49m\u001b[43mtimeout\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mtimeout_obj\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    708\u001b[0m \u001b[43m    \u001b[49m\u001b[43mbody\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mbody\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    709\u001b[0m \u001b[43m    \u001b[49m\u001b[43mheaders\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mheaders\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    710\u001b[0m \u001b[43m    \u001b[49m\u001b[43mchunked\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mchunked\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    711\u001b[0m \u001b[43m\u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m    713\u001b[0m \u001b[38;5;66;03m# If we're going to release the connection in ``finally:``, then\u001b[39;00m\n\u001b[0;32m    714\u001b[0m \u001b[38;5;66;03m# the response doesn't need to know about the connection. Otherwise\u001b[39;00m\n\u001b[0;32m    715\u001b[0m \u001b[38;5;66;03m# it will also try to release it and we'll have a double-release\u001b[39;00m\n\u001b[0;32m    716\u001b[0m \u001b[38;5;66;03m# mess.\u001b[39;00m\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\urllib3\\connectionpool.py:386\u001b[0m, in \u001b[0;36mHTTPConnectionPool._make_request\u001b[1;34m(self, conn, method, url, timeout, chunked, **httplib_request_kw)\u001b[0m\n\u001b[0;32m    385\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[1;32m--> 386\u001b[0m     \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_validate_conn\u001b[49m\u001b[43m(\u001b[49m\u001b[43mconn\u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m    387\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m (SocketTimeout, BaseSSLError) \u001b[38;5;28;01mas\u001b[39;00m e:\n\u001b[0;32m    388\u001b[0m     \u001b[38;5;66;03m# Py2 raises this as a BaseSSLError, Py3 raises it as socket timeout.\u001b[39;00m\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\urllib3\\connectionpool.py:1042\u001b[0m, in \u001b[0;36mHTTPSConnectionPool._validate_conn\u001b[1;34m(self, conn)\u001b[0m\n\u001b[0;32m   1041\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m \u001b[38;5;28mgetattr\u001b[39m(conn, \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124msock\u001b[39m\u001b[38;5;124m\"\u001b[39m, \u001b[38;5;28;01mNone\u001b[39;00m):  \u001b[38;5;66;03m# AppEngine might not have  `.sock`\u001b[39;00m\n\u001b[1;32m-> 1042\u001b[0m     \u001b[43mconn\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mconnect\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m   1044\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m conn\u001b[38;5;241m.\u001b[39mis_verified:\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\urllib3\\connection.py:363\u001b[0m, in \u001b[0;36mHTTPSConnection.connect\u001b[1;34m(self)\u001b[0m\n\u001b[0;32m    361\u001b[0m \u001b[38;5;28;01mdef\u001b[39;00m \u001b[38;5;21mconnect\u001b[39m(\u001b[38;5;28mself\u001b[39m):\n\u001b[0;32m    362\u001b[0m     \u001b[38;5;66;03m# Add certificate verification\u001b[39;00m\n\u001b[1;32m--> 363\u001b[0m     \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39msock \u001b[38;5;241m=\u001b[39m conn \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_new_conn\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m    364\u001b[0m     hostname \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mhost\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\urllib3\\connection.py:186\u001b[0m, in \u001b[0;36mHTTPConnection._new_conn\u001b[1;34m(self)\u001b[0m\n\u001b[0;32m    185\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m SocketError \u001b[38;5;28;01mas\u001b[39;00m e:\n\u001b[1;32m--> 186\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m NewConnectionError(\n\u001b[0;32m    187\u001b[0m         \u001b[38;5;28mself\u001b[39m, \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mFailed to establish a new connection: \u001b[39m\u001b[38;5;132;01m%s\u001b[39;00m\u001b[38;5;124m\"\u001b[39m \u001b[38;5;241m%\u001b[39m e\n\u001b[0;32m    188\u001b[0m     )\n\u001b[0;32m    190\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m conn\n",
      "\u001b[1;31mNewConnectionError\u001b[0m: <urllib3.connection.HTTPSConnection object at 0x000002574FCCDE20>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond",
      "\nDuring handling of the above exception, another exception occurred:\n",
      "\u001b[1;31mMaxRetryError\u001b[0m                             Traceback (most recent call last)",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\requests\\adapters.py:489\u001b[0m, in \u001b[0;36mHTTPAdapter.send\u001b[1;34m(self, request, stream, timeout, verify, cert, proxies)\u001b[0m\n\u001b[0;32m    488\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m chunked:\n\u001b[1;32m--> 489\u001b[0m     resp \u001b[38;5;241m=\u001b[39m \u001b[43mconn\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43murlopen\u001b[49m\u001b[43m(\u001b[49m\n\u001b[0;32m    490\u001b[0m \u001b[43m        \u001b[49m\u001b[43mmethod\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mrequest\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mmethod\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    491\u001b[0m \u001b[43m        \u001b[49m\u001b[43murl\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    492\u001b[0m \u001b[43m        \u001b[49m\u001b[43mbody\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mrequest\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mbody\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    493\u001b[0m \u001b[43m        \u001b[49m\u001b[43mheaders\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mrequest\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mheaders\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    494\u001b[0m \u001b[43m        \u001b[49m\u001b[43mredirect\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43;01mFalse\u001b[39;49;00m\u001b[43m,\u001b[49m\n\u001b[0;32m    495\u001b[0m \u001b[43m        \u001b[49m\u001b[43massert_same_host\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43;01mFalse\u001b[39;49;00m\u001b[43m,\u001b[49m\n\u001b[0;32m    496\u001b[0m \u001b[43m        \u001b[49m\u001b[43mpreload_content\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43;01mFalse\u001b[39;49;00m\u001b[43m,\u001b[49m\n\u001b[0;32m    497\u001b[0m \u001b[43m        \u001b[49m\u001b[43mdecode_content\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43;01mFalse\u001b[39;49;00m\u001b[43m,\u001b[49m\n\u001b[0;32m    498\u001b[0m \u001b[43m        \u001b[49m\u001b[43mretries\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mmax_retries\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    499\u001b[0m \u001b[43m        \u001b[49m\u001b[43mtimeout\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mtimeout\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    500\u001b[0m \u001b[43m    \u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m    502\u001b[0m \u001b[38;5;66;03m# Send the request.\u001b[39;00m\n\u001b[0;32m    503\u001b[0m \u001b[38;5;28;01melse\u001b[39;00m:\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\urllib3\\connectionpool.py:787\u001b[0m, in \u001b[0;36mHTTPConnectionPool.urlopen\u001b[1;34m(self, method, url, body, headers, retries, redirect, assert_same_host, timeout, pool_timeout, release_conn, chunked, body_pos, **response_kw)\u001b[0m\n\u001b[0;32m    785\u001b[0m     e \u001b[38;5;241m=\u001b[39m ProtocolError(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mConnection aborted.\u001b[39m\u001b[38;5;124m\"\u001b[39m, e)\n\u001b[1;32m--> 787\u001b[0m retries \u001b[38;5;241m=\u001b[39m \u001b[43mretries\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mincrement\u001b[49m\u001b[43m(\u001b[49m\n\u001b[0;32m    788\u001b[0m \u001b[43m    \u001b[49m\u001b[43mmethod\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43merror\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43me\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43m_pool\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43mself\u001b[39;49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43m_stacktrace\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43msys\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mexc_info\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\u001b[43m[\u001b[49m\u001b[38;5;241;43m2\u001b[39;49m\u001b[43m]\u001b[49m\n\u001b[0;32m    789\u001b[0m \u001b[43m\u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m    790\u001b[0m retries\u001b[38;5;241m.\u001b[39msleep()\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\urllib3\\util\\retry.py:592\u001b[0m, in \u001b[0;36mRetry.increment\u001b[1;34m(self, method, url, response, error, _pool, _stacktrace)\u001b[0m\n\u001b[0;32m    591\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m new_retry\u001b[38;5;241m.\u001b[39mis_exhausted():\n\u001b[1;32m--> 592\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m MaxRetryError(_pool, url, error \u001b[38;5;129;01mor\u001b[39;00m ResponseError(cause))\n\u001b[0;32m    594\u001b[0m log\u001b[38;5;241m.\u001b[39mdebug(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mIncremented Retry for (url=\u001b[39m\u001b[38;5;124m'\u001b[39m\u001b[38;5;132;01m%s\u001b[39;00m\u001b[38;5;124m'\u001b[39m\u001b[38;5;124m): \u001b[39m\u001b[38;5;132;01m%r\u001b[39;00m\u001b[38;5;124m\"\u001b[39m, url, new_retry)\n",
      "\u001b[1;31mMaxRetryError\u001b[0m: HTTPSConnectionPool(host='mcgs.bcbsfl.com', port=443): Max retries exceeded with url: /MCG?mcgId=05-82000-28&pv=false (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002574FCCDE20>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))",
      "\nDuring handling of the above exception, another exception occurred:\n",
      "\u001b[1;31mConnectionError\u001b[0m                           Traceback (most recent call last)",
      "Cell \u001b[1;32mIn[71], line 7\u001b[0m\n\u001b[0;32m      3\u001b[0m \u001b[38;5;28;01mimport\u001b[39;00m \u001b[38;5;21;01mre\u001b[39;00m\n\u001b[0;32m      5\u001b[0m headers \u001b[38;5;241m=\u001b[39m {\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mUser-Agent\u001b[39m\u001b[38;5;124m'\u001b[39m: \u001b[38;5;124m'\u001b[39m\u001b[38;5;124mMozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/118.0.0.0 Safari/537.36 Edg/118.0.2088.46\u001b[39m\u001b[38;5;124m'\u001b[39m}\n\u001b[1;32m----> 7\u001b[0m response \u001b[38;5;241m=\u001b[39m \u001b[43mrequests\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mget\u001b[49m\u001b[43m(\u001b[49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mheaders\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mheaders\u001b[49m\u001b[43m)\u001b[49m    \n\u001b[0;32m      8\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m response\u001b[38;5;241m.\u001b[39mstatus_code \u001b[38;5;241m==\u001b[39m \u001b[38;5;241m200\u001b[39m:\n\u001b[0;32m      9\u001b[0m     html_content \u001b[38;5;241m=\u001b[39m response\u001b[38;5;241m.\u001b[39mtext\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\requests\\api.py:73\u001b[0m, in \u001b[0;36mget\u001b[1;34m(url, params, **kwargs)\u001b[0m\n\u001b[0;32m     62\u001b[0m \u001b[38;5;28;01mdef\u001b[39;00m \u001b[38;5;21mget\u001b[39m(url, params\u001b[38;5;241m=\u001b[39m\u001b[38;5;28;01mNone\u001b[39;00m, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39mkwargs):\n\u001b[0;32m     63\u001b[0m     \u001b[38;5;124mr\u001b[39m\u001b[38;5;124;03m\"\"\"Sends a GET request.\u001b[39;00m\n\u001b[0;32m     64\u001b[0m \n\u001b[0;32m     65\u001b[0m \u001b[38;5;124;03m    :param url: URL for the new :class:`Request` object.\u001b[39;00m\n\u001b[1;32m   (...)\u001b[0m\n\u001b[0;32m     70\u001b[0m \u001b[38;5;124;03m    :rtype: requests.Response\u001b[39;00m\n\u001b[0;32m     71\u001b[0m \u001b[38;5;124;03m    \"\"\"\u001b[39;00m\n\u001b[1;32m---> 73\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m request(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mget\u001b[39m\u001b[38;5;124m\"\u001b[39m, url, params\u001b[38;5;241m=\u001b[39mparams, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39mkwargs)\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\requests\\api.py:59\u001b[0m, in \u001b[0;36mrequest\u001b[1;34m(method, url, **kwargs)\u001b[0m\n\u001b[0;32m     55\u001b[0m \u001b[38;5;66;03m# By using the 'with' statement we are sure the session is closed, thus we\u001b[39;00m\n\u001b[0;32m     56\u001b[0m \u001b[38;5;66;03m# avoid leaving sockets open which can trigger a ResourceWarning in some\u001b[39;00m\n\u001b[0;32m     57\u001b[0m \u001b[38;5;66;03m# cases, and look like a memory leak in others.\u001b[39;00m\n\u001b[0;32m     58\u001b[0m \u001b[38;5;28;01mwith\u001b[39;00m sessions\u001b[38;5;241m.\u001b[39mSession() \u001b[38;5;28;01mas\u001b[39;00m session:\n\u001b[1;32m---> 59\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m session\u001b[38;5;241m.\u001b[39mrequest(method\u001b[38;5;241m=\u001b[39mmethod, url\u001b[38;5;241m=\u001b[39murl, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39mkwargs)\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\requests\\sessions.py:587\u001b[0m, in \u001b[0;36mSession.request\u001b[1;34m(self, method, url, params, data, headers, cookies, files, auth, timeout, allow_redirects, proxies, hooks, stream, verify, cert, json)\u001b[0m\n\u001b[0;32m    582\u001b[0m send_kwargs \u001b[38;5;241m=\u001b[39m {\n\u001b[0;32m    583\u001b[0m     \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mtimeout\u001b[39m\u001b[38;5;124m\"\u001b[39m: timeout,\n\u001b[0;32m    584\u001b[0m     \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mallow_redirects\u001b[39m\u001b[38;5;124m\"\u001b[39m: allow_redirects,\n\u001b[0;32m    585\u001b[0m }\n\u001b[0;32m    586\u001b[0m send_kwargs\u001b[38;5;241m.\u001b[39mupdate(settings)\n\u001b[1;32m--> 587\u001b[0m resp \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39msend(prep, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39msend_kwargs)\n\u001b[0;32m    589\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m resp\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\requests\\sessions.py:701\u001b[0m, in \u001b[0;36mSession.send\u001b[1;34m(self, request, **kwargs)\u001b[0m\n\u001b[0;32m    698\u001b[0m start \u001b[38;5;241m=\u001b[39m preferred_clock()\n\u001b[0;32m    700\u001b[0m \u001b[38;5;66;03m# Send the request\u001b[39;00m\n\u001b[1;32m--> 701\u001b[0m r \u001b[38;5;241m=\u001b[39m adapter\u001b[38;5;241m.\u001b[39msend(request, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39mkwargs)\n\u001b[0;32m    703\u001b[0m \u001b[38;5;66;03m# Total elapsed time of the request (approximately)\u001b[39;00m\n\u001b[0;32m    704\u001b[0m elapsed \u001b[38;5;241m=\u001b[39m preferred_clock() \u001b[38;5;241m-\u001b[39m start\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\requests\\adapters.py:565\u001b[0m, in \u001b[0;36mHTTPAdapter.send\u001b[1;34m(self, request, stream, timeout, verify, cert, proxies)\u001b[0m\n\u001b[0;32m    561\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28misinstance\u001b[39m(e\u001b[38;5;241m.\u001b[39mreason, _SSLError):\n\u001b[0;32m    562\u001b[0m         \u001b[38;5;66;03m# This branch is for urllib3 v1.22 and later.\u001b[39;00m\n\u001b[0;32m    563\u001b[0m         \u001b[38;5;28;01mraise\u001b[39;00m SSLError(e, request\u001b[38;5;241m=\u001b[39mrequest)\n\u001b[1;32m--> 565\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m \u001b[38;5;167;01mConnectionError\u001b[39;00m(e, request\u001b[38;5;241m=\u001b[39mrequest)\n\u001b[0;32m    567\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m ClosedPoolError \u001b[38;5;28;01mas\u001b[39;00m e:\n\u001b[0;32m    568\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m \u001b[38;5;167;01mConnectionError\u001b[39;00m(e, request\u001b[38;5;241m=\u001b[39mrequest)\n",
      "\u001b[1;31mConnectionError\u001b[0m: HTTPSConnectionPool(host='mcgs.bcbsfl.com', port=443): Max retries exceeded with url: /MCG?mcgId=05-82000-28&pv=false (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002574FCCDE20>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))"
     ]
    }
   ],
   "source": [
    "import requests\n",
    "from bs4 import BeautifulSoup\n",
    "import re\n",
    "\n",
    "headers = {'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/118.0.0.0 Safari/537.36 Edg/118.0.2088.46'}\n",
    "    \n",
    "response = requests.get(url, headers=headers)    \n",
    "if response.status_code == 200:\n",
    "    html_content = response.text\n",
    "    soup = BeautifulSoup(html_content, 'html.parser')\n",
    "    soup.prettify()\n",
    "    text = soup.get_text()\n",
    "    \n",
    "        \n",
    "print(text)   "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "id": "9d07d447",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'GENE.00009 Gene Expression Profiling and Genomic Biomarker Tests for Prostate Cancer'"
      ]
     },
     "execution_count": 39,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "soup.find(\"head\").title.text.strip()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "id": "b353bab8",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "array(['Aetna', 'Anthem', 'BCBSFL', 'BCBSNC', 'Geisinger', 'Wellmark',\n",
       "       '(arkansasblue OR BCBS Arkansas)',\n",
       "       '(Asuris  OR Asuris Northwest Health)',\n",
       "       '(BCBSLA OR BCBS Louisiana)', '(bcbsri OR BCBS Rhode Island)',\n",
       "       '(bcidaho OR Blue Cross Idaho)',\n",
       "       '(bridgespanhealth OR Bridgespan Health)',\n",
       "       '(empireblue OR Empire BCBS)',\n",
       "       '(keystonefirstpa OR Keystone First)',\n",
       "       '(Providence Health Plan OR providencehealthplan)',\n",
       "       '(southcarolinablues OR BCBS South Carolina)',\n",
       "       '(ssmhealth OR SSM Health)', '(wellsense OR BMC Healthnet)',\n",
       "       '(FEPBlue  OR BCBS Federal Employee Plan)',\n",
       "       '(Healthnet OR Centene)', '(UHC OR UnitedHealthcare)',\n",
       "       'BCBS Michigan', 'AIM Specialty Health/Carelon Medical Benefits',\n",
       "       '(\"bcbsms\" OR BCBS Mississippi)', '\"HealthPartners\"',\n",
       "       '(\"upmchealthplan\" OR UPMC Health Plan)',\n",
       "       '(\"Triple-S Salud\" OR Triple-S Health Plan)', '\"qualchoice\"',\n",
       "       'Horizon BCBS NJ', 'Independence Blue Cross'], dtype=object)"
      ]
     },
     "execution_count": 50,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data_html[\"Payors/LBM\"].unique()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "84fa1d19",
   "metadata": {},
   "source": [
    "### Independence Blue Cross"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 90,
   "id": "c618c60f",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_231364'"
      ]
     },
     "execution_count": 90,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "ind_url=[]\n",
    "for i in data_html[data_html[\"Payors/LBM\"]=='\"HealthPartners\"']['Policy url']:\n",
    "    ind_url.append(i)\n",
    "    \n",
    "url=ind_url[0] \n",
    "url"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "id": "e7d40d34",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[<h2 style=\"background-color: #BED6DB!important; text-transform: uppercase;\"><a id=\"Summary\"><b style=\"color: #003359;\">Summary</b></a></h2>, <h2 style=\"background-color: #BED6DB!important; text-transform: uppercase;\"><b style=\"color: #003359;\"><a id=\"Objective\"></a></b></h2>, <h2 style=\"background-color: #BED6DB!important; text-transform: uppercase;\"><b style=\"color: #003359;\"><a id=\"Objective\"><b style=\"color: #003359;\">Objective</b></a></b></h2>, <h2 style=\"background-color: #BED6DB!important; text-transform: uppercase;\"><a id=\"Policy\"></a><b style=\"color: #003359;\"><a id=\"Policy Statements\"></a></b></h2>, <h2 style=\"background-color: #BED6DB!important; text-transform: uppercase;\"><b style=\"color: #003359;\"><a id=\"Policy Statements\"><b style=\"color: #003359;\">Policy Statements </b></a></b></h2>, <h2 style=\"background-color: #BED6DB!important; text-transform: uppercase;\"><a id=\"Policy Guidelines\"></a><b style=\"color: #003359;\"><a id=\"Policy Guidelines\"></a></b></h2>, <h2 style=\"background-color: #BED6DB!important; text-transform: uppercase;\"><b style=\"color: #003359;\"><a id=\"Policy Guidelines\"><b style=\"color: #003359;\">Policy Guidelines </b></a></b></h2>, <h2 style=\"background-color: #BED6DB!important; text-transform: uppercase;\"><a id=\"Benefit Application\"></a><b style=\"color: #003359;\"><a id=\"Benefit Application\"></a></b></h2>, <h2 style=\"background-color: #BED6DB!important; text-transform: uppercase;\"><b style=\"color: #003359;\"><a id=\"Benefit Application\"><b style=\"color: #003359;\">Benefit Application </b></a></b></h2>, <h2 style=\"background-color: #BED6DB!important; text-transform: uppercase;\"><a id=\"Background\"></a><b style=\"color: #003359;\"><a id=\"Background\"></a></b></h2>, <h2 style=\"background-color: #BED6DB!important; text-transform: uppercase;\"><b style=\"color: #003359;\"><a id=\"Background\"><b style=\"color: #003359;\">Background</b></a></b></h2>, <h2 style=\"background-color: #BED6DB!important; text-transform: uppercase;\"><a id=\"Rationale\"></a><b style=\"color: #003359;\"><a id=\"Regulatory Status\"></a></b><b style=\"color: #003359;\"> <a id=\"Rationale\"></a></b></h2>, <h2 style=\"background-color: #BED6DB!important; text-transform: uppercase;\"><b style=\"color: #003359;\"><a id=\"Rationale\"><b style=\"color: #003359;\">Rationale</b></a></b></h2>, <h2 style=\"background-color: #BED6DB!important; text-transform: uppercase;\"><a id=\"Supplemental Information\"></a></h2>, <h2 style=\"background-color: #BED6DB!important; text-transform: uppercase;\"><b style=\"color: #003359;\"><a id=\"Supplemental Information\"><b style=\"color: #003359;\">Supplemental Information</b></a></b></h2>, <h2 style=\"break-inside: avoid; break-after: avoid; font-family: Arial !important; font-size: 20px !important; color: rgb(0, 51, 89) !important; font-weight: bold !important; text-transform: uppercase; background-color: rgb(190, 214, 219) !important;\">Medicare National Coverage</h2>, <h2 style=\"background-color: #BED6DB!important; text-transform: uppercase;\"><a id=\"References\"></a><b style=\"color: #003359;\"><a id=\"References\"></a></b></h2>, <h2 style=\"background-color: #BED6DB!important; text-transform: uppercase;\"><b style=\"color: #003359;\"><a id=\"References\"><b style=\"color: #003359;\">References</b></a></b></h2>, <h2 style=\"background-color: #BED6DB!important; text-transform: uppercase;\"><a id=\"Codes\"></a><b style=\"color: #003359;\"><a id=\"Codes\"></a></b></h2>, <h2 style=\"background-color: #BED6DB!important; text-transform: uppercase;\"><b style=\"color: #003359;\"><a id=\"Codes\"><b style=\"color: #003359;\">Codes</b></a></b></h2>, <h2 style=\"background-color: #BED6DB!important; text-transform: uppercase;\"><b style=\"color: #003359;\"><a id=\"Applicable Modifiers\"><b style=\"color: #003359;\">Applicable Modifiers </b></a></b></h2>, <h2 style=\"background-color: #BED6DB!important; text-transform: uppercase;\"><b style=\"color: #003359;\"><a id=\"Policy History\"><b style=\"color: #003359;\"> Policy History </b></a></b></h2>]\n"
     ]
    }
   ],
   "source": [
    "import requests\n",
    "from bs4 import BeautifulSoup\n",
    "import re\n",
    "headers = {'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/118.0.0.0 Safari/537.36 Edg/118.0.2088.46'}\n",
    "\n",
    "response = requests.get(url, headers=headers)    \n",
    "if response.status_code == 200:\n",
    "    html_content = response.text\n",
    "    soup = BeautifulSoup(html_content, 'html.parser')\n",
    "    soup.prettify()\n",
    "    text = soup.get_text()\n",
    "    print(soup.find_all('h2'))        "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 80,
   "id": "5401ae8d",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "common method of title extraction :\n",
      " IBC Medical Policies\n",
      "Overall_title :\n",
      " IBC Medical Policies\n",
      "Required title of doc: \n",
      " Rapid Whole Exome Sequencing (rWES) and Rapid Whole Genome Sequencing (rWGS) for Diagnosis of Genetic Disorders\n"
     ]
    }
   ],
   "source": [
    "print(\"common method of title extraction :\\n\",soup.find(\"head\").title.text.strip())\n",
    "print('Overall_title :\\n',soup.find(\"head\").title.text.strip())\n",
    "print('Required title of doc: \\n',soup.find(class_='title-content').text.strip())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "2e1e37c1",
   "metadata": {},
   "outputs": [],
   "source": [
    "soup.find_all(class_='')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 65,
   "id": "e52da34c",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Class Names: None\n"
     ]
    }
   ],
   "source": [
    "# Find the div element\n",
    "div_element = soup.find('div')\n",
    "\n",
    "# Get the class names as a list\n",
    "class_names = div_element.get('class')\n",
    "\n",
    "# Print the class names\n",
    "print(\"Class Names:\", class_names)\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 70,
   "id": "c87ea4bc",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'Rapid Whole Exome Sequencing (rWES) and Rapid Whole Genome Sequencing (rWGS) for Diagnosis of Genetic Disorders'"
      ]
     },
     "execution_count": 70,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "soup.find(class_='title-content').text.strip()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "dfd1f23e",
   "metadata": {},
   "source": [
    "### Next policy : (arkansasblue OR BCBS Arkansas)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 87,
   "id": "6677f0f8",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "array([146, 144, 143, 142, 140, 138, 137], dtype=int64)"
      ]
     },
     "execution_count": 87,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data_html[\"Payors/LBM\"].value_counts().values[0:7]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 104,
   "id": "229a49aa",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2010015&viewIntro=yes\n"
     ]
    }
   ],
   "source": [
    "ind_url=[]\n",
    "for i in data_html[data_html[\"Payors/LBM\"]=='(arkansasblue OR BCBS Arkansas)']['Policy url']:\n",
    "    ind_url.append(i)\n",
    "    \n",
    "url=ind_url[0] \n",
    "print(url)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 103,
   "id": "b11f7528",
   "metadata": {},
   "outputs": [
    {
     "ename": "IndexError",
     "evalue": "list index out of range",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[1;31mIndexError\u001b[0m                                Traceback (most recent call last)",
      "Cell \u001b[1;32mIn[103], line 5\u001b[0m\n\u001b[0;32m      2\u001b[0m \u001b[38;5;28;01mfor\u001b[39;00m i \u001b[38;5;129;01min\u001b[39;00m data_pdf[data_pdf[\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mPayors/LBM\u001b[39m\u001b[38;5;124m\"\u001b[39m]\u001b[38;5;241m==\u001b[39m\u001b[38;5;124m'\u001b[39m\u001b[38;5;124m(arkansasblue OR BCBS Arkansas)\u001b[39m\u001b[38;5;124m'\u001b[39m][\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mPolicy url\u001b[39m\u001b[38;5;124m'\u001b[39m]:\n\u001b[0;32m      3\u001b[0m     pdf_url\u001b[38;5;241m.\u001b[39mappend(i)\n\u001b[1;32m----> 5\u001b[0m url\u001b[38;5;241m=\u001b[39m\u001b[43mpdf_url\u001b[49m\u001b[43m[\u001b[49m\u001b[38;5;241;43m4\u001b[39;49m\u001b[43m]\u001b[49m \n\u001b[0;32m      6\u001b[0m \u001b[38;5;28mprint\u001b[39m(url)\n",
      "\u001b[1;31mIndexError\u001b[0m: list index out of range"
     ]
    }
   ],
   "source": [
    "pdf_url=[]\n",
    "for i in data_pdf[data_pdf[\"Payors/LBM\"]=='(bluecrossma OR BCBS Massachusetts)']['Policy url']:\n",
    "    pdf_url.append(i)\n",
    "    \n",
    "url=pdf_url[4] \n",
    "print(url)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "3bae3693",
   "metadata": {},
   "source": [
    "### NOTE  PAYOR:(empireblue OR Empire BCBS) 'https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2015009&viewIntro=yes'\n",
    "this kind of Link not working"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 93,
   "id": "221bb20e",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "HTML - Working    143\n",
       "Name: link_status, dtype: int64"
      ]
     },
     "execution_count": 93,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data_html[data_html[\"Payors/LBM\"]=='(arkansasblue OR BCBS Arkansas)']['link_status'].value_counts()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "1da3e757",
   "metadata": {},
   "source": []
  },
  {
   "cell_type": "markdown",
   "id": "47715cc0",
   "metadata": {},
   "source": [
    "### Notes:HTML\n",
    "* 4.Payor: (southcarolinablues OR BCBS South Carolina) : title working with common method\n",
    "* 5.Payor:(empireblue OR Empire BCBS): ANTHEM payor are same \n",
    "* 6.Payor: (\"Triple-S Salud\" OR Triple-S Health Plan)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "7f3eb288",
   "metadata": {},
   "outputs": [],
   "source": [
    "import requests\n",
    "from bs4 import BeautifulSoup\n",
    "import re\n",
    "\n",
    "\n",
    "def extract_text(url):\n",
    "    headers = {'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/118.0.0.0 Safari/537.36 Edg/118.0.2088.46'}\n",
    "    \n",
    "    response = requests.get(url, headers=headers)    \n",
    "    if response.status_code == 200:\n",
    "        html_content = response.text\n",
    "        soup = BeautifulSoup(html_content, 'html.parser')\n",
    "        soup.prettify()\n",
    "        text = soup.get_text()\n",
    "\n",
    "        print(text)\n",
    "        print('****************************')\n",
    "        #print(text)\n",
    "    else:\n",
    "        print(response.status_code())\n",
    "        \n",
    "        \n",
    "def extract_html_file_url(main_html_url):\n",
    "    response = requests.get(main_html_url)\n",
    "    if response.status_code == 200:\n",
    "        soup = BeautifulSoup(response.content, 'html.parser')\n",
    "        # Assuming the linked HTML file is specified using an anchor tag with attribute 'href'\n",
    "        link_tag = soup.find('a', href=True)\n",
    "        if link_tag:\n",
    "            linked_html_url = link_tag['href']\n",
    "            return linked_html_url\n",
    "    return None       \n",
    "                            \n",
    "        \n",
    "def extract_title_dates(url):\n",
    "    headers = {'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/118.0.0.0 Safari/537.36 Edg/118.0.2088.46'}\n",
    "    \n",
    "    response = requests.get(url, headers=headers)    \n",
    "    if response.status_code == 200:\n",
    "        content_type=response.headers.get('Content-Type','')\n",
    "        \n",
    "        # Ignore character encoding information in Content-Type header\n",
    "        #content_type = content_type.split(';')[0]\n",
    "        \n",
    "        if 'text/html' in content_type:\n",
    "            html_content = response.text\n",
    "            soup = BeautifulSoup(html_content, 'html.parser')\n",
    "            soup.prettify()\n",
    "            text = soup.get_text()\n",
    "            title_pattern = r'Title:\\s*(.*?)\\n'\n",
    "            policy_name_pattern = r'Policy Name:\\s*(.*?)\\n'\n",
    "            \n",
    "            # Use the regular expression to extract the \"Subject\"\n",
    "            match_title = re.search(title_pattern, text)\n",
    "            match_policy_name=re.search(policy_name_pattern,text)\n",
    "            \n",
    "            if match_title:\n",
    "                Title = match_title.group(1).strip()\n",
    "                return \"title_tag_html: \",Title\n",
    "            elif match_policy_name:\n",
    "                Title_policy = match_policy_name.group(1).strip()\n",
    "                return \"title_tag_html: \",Title_policy\n",
    "        \n",
    "                \n",
    "            else:\n",
    "                title_tag=soup.title.text.strip()\n",
    "                if title_tag:\n",
    "                    print(\"title_tag: \", title_tag,\"\\nPolicy URL:\",url)\n",
    "                else:\n",
    "                    print(\"No Title_tag by this soup.title Method \\n Try other method\")\n",
    "                    #print('Overall_title :\\n',soup.find(\"head\").title.text.strip())\n",
    "                    #print('Required title of doc: \\n',soup.find(class_='title-content').text.strip())\n",
    "                    \n",
    "                    \n",
    "             \n",
    "                    \n",
    "            print('****************************')\n",
    "            #print(text)\n",
    "        \n",
    "        elif 'application/pdf' in content_type:\n",
    "            import fitz\n",
    "            pdf_bytes = response.content\n",
    "            pdf_document = fitz.open(\"pdf\", pdf_bytes)\n",
    "\n",
    "            text = \"\"\n",
    "            for page_num in range(pdf_document.page_count):\n",
    "                page = pdf_document.load_page(page_num)\n",
    "                text += page.get_text()\n",
    "            subject_pattern = r'Subject:\\s*(.*?)\\n'\n",
    "            \n",
    "\n",
    "            # Use the regular expression to extract the \"Subject\"\n",
    "            match = re.search(subject_pattern, text)\n",
    "            if match:\n",
    "                subject = match.group(1)\n",
    "                return \"title_tag_pdf_subject: \",subject,\"\\nPolicy URL:\",url\n",
    "            else:\n",
    "                metadata = pdf_document.metadata\n",
    "                title = metadata.get(\"title\", \"Title not found in metadata\")\n",
    "                return \"title_tag_pdf_met :\",title #,\"\\nPolicy URL:\",url\n",
    "                print('****************************')\n",
    "                \n",
    "            \n",
    "\n",
    "\n",
    "        else:\n",
    "            # Handle other content types if needed\n",
    "            print(\"Unsupported content type:\", content_type)   \n",
    "    else:\n",
    "        print(response.status_code())        \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 100,
   "id": "1b45cd8c",
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Index(['Aetna', 'Independence Blue Cross', '(arkansasblue OR BCBS Arkansas)',\n",
       "       '(\"bcbsms\" OR BCBS Mississippi)',\n",
       "       '(southcarolinablues OR BCBS South Carolina)',\n",
       "       '(empireblue OR Empire BCBS)',\n",
       "       '(\"Triple-S Salud\" OR Triple-S Health Plan)', 'Anthem',\n",
       "       'Horizon BCBS NJ', '\"HealthPartners\"',\n",
       "       '(\"upmchealthplan\" OR UPMC Health Plan)', '\"qualchoice\"',\n",
       "       '(bcidaho OR Blue Cross Idaho)', '(keystonefirstpa OR Keystone First)',\n",
       "       'AIM Specialty Health/Carelon Medical Benefits',\n",
       "       '(BCBSLA OR BCBS Louisiana)', '(bcbsri OR BCBS Rhode Island)',\n",
       "       '(Healthnet OR Centene)', 'BCBSFL', 'Wellmark', 'BCBSNC',\n",
       "       '(bridgespanhealth OR Bridgespan Health)', 'Geisinger',\n",
       "       '(wellsense OR BMC Healthnet)',\n",
       "       '(FEPBlue  OR BCBS Federal Employee Plan)', '(UHC OR UnitedHealthcare)',\n",
       "       'BCBS Michigan', '(Providence Health Plan OR providencehealthplan)',\n",
       "       '(Asuris  OR Asuris Northwest Health)', '(ssmhealth OR SSM Health)'],\n",
       "      dtype='object')"
      ]
     },
     "execution_count": 100,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data_html[\"Payors/LBM\"].value_counts().index"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 146,
   "id": "c0ea556e",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://www.aetna.com/cpb/medical/data/300_399/0352.html\n"
     ]
    }
   ],
   "source": [
    "ind_url=[]\n",
    "for i in data_html[data_html[\"Payors/LBM\"]=='Aetna' ]['Policy url']:\n",
    "    ind_url.append(i)\n",
    "    \n",
    "url=ind_url[1] \n",
    "print(url)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 149,
   "id": "2d0766b7",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n"
     ]
    }
   ],
   "source": [
    "pdf_url=[]\n",
    "for i in data_pdf[data_pdf[\"Payors/LBM\"]=='(bluecrossma OR BCBS Massachusetts)']['Policy url']:\n",
    "    pdf_url.append(i)\n",
    "    \n",
    "url=pdf_url[4] \n",
    "print(url)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "6cba7c0b",
   "metadata": {},
   "source": [
    "pdf_url=[]\n",
    "for i in data_pdf[data_pdf[\"Payors/LBM\"]=='(bluecrossma OR BCBS Massachusetts)']['Policy url']:\n",
    "    pdf_url.append(i)\n",
    "    \n",
    "url=pdf_url[77] \n",
    "print(url)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 134,
   "id": "d324a7cf",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Medical Policies  -  Horizon Blue Cross Blue Shield of New Jersey\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Skip to main content\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "NASCO Data Security IncidentOn May 30, 2023, NASCO experienced a data security incident in which a threat actor acquired data from NASCO’s MOVEit platform.\r\n",
      "Notification_close\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Search For‌ALL\n",
      "Select an audience to restrict the search\n",
      "ALLMEMBERSPROVIDERSBROKERSEMPLOYERS\n",
      "\n",
      "\n",
      "Enter the terms you wish to search for\n",
      "\n",
      "\n",
      "\n",
      "search button\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "HomeMembersProvidersBrokersEmployers\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "alert\n",
      "\n",
      "\n",
      "Contact Us\n",
      "About Us\n",
      "CAREERS\n",
      "\n",
      "search button\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "NAVINET SIGN IN\n",
      "DENTAL SIGN IN\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Alert\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Toggle menu\n",
      "\n",
      "\n",
      "BACKback to www.horizonblue.com\n",
      "PROVIDERS \n",
      "\n",
      "COVID-19 InformationCOVID-19 Information\n",
      "\n",
      " COVID-19 Information \n",
      "\n",
      "Important Information for New COVID-19 Vaccine ClaimsImportant Information for New COVID-19 Vaccine Claims\n",
      "\n",
      "\n",
      "Code Terminations as the PHE EndsCode Terminations as the PHE Ends\n",
      "\n",
      "\n",
      "PHE Update: Prescription Limitation Change for Braven Health, HMO D-SNP and Medicare Part D membersPHE Update: Prescription Limitation Change for Braven Health, HMO D-SNP and Medicare Part D members\n",
      "\n",
      "\n",
      "Important Update: Additional Policy Changes as PHE EndsImportant Update: Additional Policy Changes as PHE Ends\n",
      "\n",
      "\n",
      "Some Benefits Changes as the PHE EndsSome Benefits Changes as the PHE Ends\n",
      "\n",
      "\n",
      "Referral Requirements for All ServicesReferral Requirements for All Services\n",
      "\n",
      "\n",
      "Referral Requirements for Services Not Related to COVID-19Referral Requirements for Services Not Related to COVID-19\n",
      "\n",
      "\n",
      "Reminder: Select one method for COVID-19 and Influenza TestingReminder: Select one method for COVID-19 and Influenza Testing\n",
      "\n",
      "\n",
      "Antibody testing: FDA and CDC do not recommend use to determine immunityAntibody testing: FDA and CDC do not recommend use to determine immunity\n",
      "\n",
      "\n",
      "COVID-19 vaccine administration reimbursement at UCCsCOVID-19 vaccine administration reimbursement at UCCs\n",
      "\n",
      "\n",
      "Coverage for antibody infusion therapyCoverage for antibody infusion therapy\n",
      "\n",
      "\n",
      "Reminder to use specific codes when evaluating for COVID-19Reminder to use specific codes when evaluating for COVID-19\n",
      "\n",
      "\n",
      "COVID-19 Update: Telemedicine Reimbursement Policy AddendaCOVID-19 Update: Telemedicine Reimbursement Policy Addenda\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Why JoinWhy Join\n",
      "\n",
      " Why Join \n",
      "\n",
      "Our CommitmentOur Commitment\n",
      "\n",
      "\n",
      "Join Our NetworksJoin Our Networks\n",
      "\n",
      " Join Our Networks\n",
      "\n",
      "Doula Services PractitionersDoula Services Practitioners\n",
      "\n",
      "\n",
      "Join a Horizon BCBSNJ or Horizon NJ Health Network (Physicians and Other Healthcare Professionals )Join a Horizon BCBSNJ or Horizon NJ Health Network (Physicians and Other Healthcare Professionals )\n",
      "\n",
      " Join a Horizon BCBSNJ or Horizon NJ Health Network (Physicians and Other Healthcare Professionals )\n",
      "\n",
      "Credentialing Frequently Asked QuestionsCredentialing Frequently Asked Questions\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Join the Horizon BCBSNJ Network (Ancillary Providers)Join the Horizon BCBSNJ Network (Ancillary Providers)\n",
      "\n",
      "\n",
      "Join the Horizon NJ Health Network (Ancillary Providers)Join the Horizon NJ Health Network (Ancillary Providers)\n",
      "\n",
      "\n",
      "Join the Horizon Dental NetworkJoin the Horizon Dental Network\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Our Healthcare FacilitiesOur Healthcare Facilities\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Products & ProgramsProducts & Programs\n",
      "\n",
      " Products & Programs \n",
      "\n",
      "Braven Health&#8480;Braven Health&#8480;\n",
      "\n",
      " Braven Health&#8480;\n",
      "\n",
      "Braven Health Supportive ProgramBraven Health Supportive Program\n",
      "\n",
      "\n",
      "Braven Health&#8480; Drug FormularyBraven Health&#8480; Drug Formulary\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Horizon Behavioral Health&#8480;Horizon Behavioral Health&#8480;\n",
      "\n",
      " Horizon Behavioral Health&#8480;\n",
      "\n",
      "Integrated System of Care (ISC) ProgramIntegrated System of Care (ISC) Program\n",
      "\n",
      "\n",
      "Peer Support ProgramPeer Support Program\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Horizon Network InformationHorizon Network Information\n",
      "\n",
      " Horizon Network Information\n",
      "\n",
      "BlueCard ProgramBlueCard Program\n",
      "\n",
      " BlueCard Program\n",
      "\n",
      "BlueCard ClaimsBlueCard Claims\n",
      "\n",
      "\n",
      "BlueCard Medical Policy/Pre-Certification InfoBlueCard Medical Policy/Pre-Certification Info\n",
      "\n",
      "\n",
      "BlueCard Program FAQsBlueCard Program FAQs\n",
      "\n",
      "\n",
      "BlueCard Program OverviewBlueCard Program Overview\n",
      "\n",
      "\n",
      "BlueCard Program TutorialBlueCard Program Tutorial\n",
      "\n",
      "\n",
      "GeoBlue: International CoverageGeoBlue: International Coverage\n",
      "\n",
      "\n",
      "How to Identify BlueCard MembersHow to Identify BlueCard Members\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Dental ServicesDental Services\n",
      "\n",
      "\n",
      "ProductsProducts\n",
      "\n",
      " Products\n",
      "\n",
      "Managed Care PlansManaged Care Plans\n",
      "\n",
      "\n",
      "PPO and Indemnity ProductsPPO and Indemnity Products\n",
      "\n",
      "\n",
      "OMNIA Health PlansOMNIA Health Plans\n",
      "\n",
      "\n",
      "NJ SHBP/SEHBP Benefit PlansNJ SHBP/SEHBP Benefit Plans\n",
      "\n",
      "\n",
      "FEPFEP\n",
      "\n",
      "\n",
      "Braven Health&#8480; Medicare Advantage PlansBraven Health&#8480; Medicare Advantage Plans\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Our PledgeOur Pledge\n",
      "\n",
      "\n",
      "Patient Care SupportPatient Care Support\n",
      "\n",
      "\n",
      "PharmacyPharmacy\n",
      "\n",
      " Pharmacy\n",
      "\n",
      "Medical Injectables ProgramMedical Injectables Program\n",
      "\n",
      " Medical Injectables Program\n",
      "\n",
      "Medical Injectables Program Code ListsMedical Injectables Program Code Lists\n",
      "\n",
      " Medical Injectables Program Code Lists\n",
      "\n",
      "Injectable Medication Codes Subject to MRxM Claim Review/EditingInjectable Medication Codes Subject to MRxM Claim Review/Editing\n",
      "\n",
      "\n",
      "Injectable Medications subject to Medical Necessity and Appropriateness ReviewInjectable Medications subject to Medical Necessity and Appropriateness Review\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Medical Injectables Program Provider Questions and AnswersMedical Injectables Program Provider Questions and Answers\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Pharmacy ProgramsPharmacy Programs\n",
      "\n",
      " Pharmacy Programs\n",
      "\n",
      "E-Prescribing & IncentivesE-Prescribing & Incentives\n",
      "\n",
      "\n",
      "Medicare Part DMedicare Part D\n",
      "\n",
      "\n",
      "Pharmacy GuidelinesPharmacy Guidelines\n",
      "\n",
      " Pharmacy Guidelines\n",
      "\n",
      "Oral Anticancer Medication Coverage LawOral Anticancer Medication Coverage Law\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Preferred Medical DrugsPreferred Medical Drugs\n",
      "\n",
      "\n",
      "Site of Administration Program for Infusion/Injectable MedicationsSite of Administration Program for Infusion/Injectable Medications\n",
      "\n",
      " Site of Administration Program for Infusion/Injectable Medications\n",
      "\n",
      "Site of Administration Program Code ListSite of Administration Program Code List\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Specialty Pharmaceuticals for Office AdministrationSpecialty Pharmaceuticals for Office Administration\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Risk Adjustment OverviewRisk Adjustment Overview\n",
      "\n",
      " Risk Adjustment Overview\n",
      "\n",
      "Commercial Risk Adjustment (CRA) ICD-10-CM Clinical Documentation Improvement for Chronic ConditionsCommercial Risk Adjustment (CRA) ICD-10-CM Clinical Documentation Improvement for Chronic Conditions\n",
      "\n",
      "\n",
      "How Risk Adjustment Benefits You and Your PatientsHow Risk Adjustment Benefits You and Your Patients\n",
      "\n",
      "\n",
      "Risk Adjustment Documentation & Coding Improvement Reference InformationRisk Adjustment Documentation & Coding Improvement Reference Information\n",
      "\n",
      "\n",
      "Risk Adjustment In DepthRisk Adjustment In Depth\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Utilization Management ProgramsUtilization Management Programs\n",
      "\n",
      " Utilization Management Programs\n",
      "\n",
      "Carelon Medical Benefits ManagementCarelon Medical Benefits Management\n",
      "\n",
      " Carelon Medical Benefits Management\n",
      "\n",
      "Diagnostic Imaging ProgramDiagnostic Imaging Program\n",
      "\n",
      "\n",
      "Musculoskeletal ProgramMusculoskeletal Program\n",
      "\n",
      "\n",
      "Oncology ProgramOncology Program\n",
      "\n",
      " Oncology Program\n",
      "\n",
      "Oncology Program Code ListOncology Program Code List\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Sleep Disorder Management ProgramSleep Disorder Management Program\n",
      "\n",
      "\n",
      "Groups Enrolled With CarelonGroups Enrolled With Carelon\n",
      "\n",
      "\n",
      "Procedure Codes Reviewed by CarelonProcedure Codes Reviewed by Carelon\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Horizon Care@HomeHorizon Care@Home\n",
      "\n",
      " Horizon Care@Home\n",
      "\n",
      "About Horizon Care@HomeAbout Horizon Care@Home\n",
      "\n",
      "\n",
      "Find A ProviderFind A Provider\n",
      "\n",
      "\n",
      "Home Infusion TherapyHome Infusion Therapy\n",
      "\n",
      "\n",
      "Prior Authorization/Pre-Service RegistrationPrior Authorization/Pre-Service Registration\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Horizon Supportive Care and Readmission ProgramHorizon Supportive Care and Readmission Program\n",
      "\n",
      "\n",
      "Surgical and Implantable Device Management ProgramSurgical and Implantable Device Management Program\n",
      "\n",
      " Surgical and Implantable Device Management Program\n",
      "\n",
      "About the Surgical and Implantable Device Management ProgramAbout the Surgical and Implantable Device Management Program\n",
      "\n",
      "\n",
      "Cardiac ServicesCardiac Services\n",
      "\n",
      " Cardiac Services\n",
      "\n",
      "Cardiac Services Procedure CodesCardiac Services Procedure Codes\n",
      "\n",
      "\n",
      "Surgical and Implantable Device Management Program for Cardiac Surgeries  Frequently Asked Questions Updated: March 25, 2021Surgical and Implantable Device Management Program for Cardiac Surgeries  Frequently Asked Questions Updated: March 25, 2021\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Orthopedic ServicesOrthopedic Services\n",
      "\n",
      " Orthopedic Services\n",
      "\n",
      "Orthopedic Services Procedure CodesOrthopedic Services Procedure Codes\n",
      "\n",
      "\n",
      "Safety and Quality Award Program Q&ASafety and Quality Award Program Q&A\n",
      "\n",
      "\n",
      "Surgical and Implantable Device Management Program Orthopedic Services Frequently Asked QuestionsSurgical and Implantable Device Management Program Orthopedic Services Frequently Asked Questions\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Spine ServicesSpine Services\n",
      "\n",
      " Spine Services\n",
      "\n",
      "Spine Procedure CodesSpine Procedure Codes\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "TurningPoint Safety and Quality Award ProgramTurningPoint Safety and Quality Award Program\n",
      "\n",
      "\n",
      "Medical Policy Criteria and GuidelinesMedical Policy Criteria and Guidelines\n",
      "\n",
      "\n",
      "PA/MND ProcessPA/MND Process\n",
      "\n",
      "\n",
      "Using Out-of-Network Providers in Surgical ServicesUsing Out-of-Network Providers in Surgical Services\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "eviCore HealthCareeviCore HealthCare\n",
      "\n",
      " eviCore HealthCare\n",
      "\n",
      "Cardiology Imaging ProgramCardiology Imaging Program\n",
      "\n",
      " Cardiology Imaging Program\n",
      "\n",
      "Cardiology Imaging Program Provider Questions and AnswersCardiology Imaging Program Provider Questions and Answers\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Molecular and Genomic Testing ProgramMolecular and Genomic Testing Program\n",
      "\n",
      " Molecular and Genomic Testing Program\n",
      "\n",
      "Medical Information Requirements for Programs Administered by eviCoreMedical Information Requirements for Programs Administered by eviCore\n",
      "\n",
      "\n",
      "Medical Necessity Determination (MND) Review of Molecular and Genomic Diagnostic Testing Services Frequently Asked QuestionsMedical Necessity Determination (MND) Review of Molecular and Genomic Diagnostic Testing Services Frequently Asked Questions\n",
      "\n",
      "\n",
      "Molecular and Genomic Testing Procedure CodesMolecular and Genomic Testing Procedure Codes\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Musculoskeletal Program for Pain Management ServicesMusculoskeletal Program for Pain Management Services\n",
      "\n",
      "\n",
      "Radiation Therapy ProgramRadiation Therapy Program\n",
      "\n",
      " Radiation Therapy Program\n",
      "\n",
      "Medical Information Requirements for Programs Administered by eviCoreMedical Information Requirements for Programs Administered by eviCore\n",
      "\n",
      "\n",
      "Radiation Therapy Program Questions and AnswersRadiation Therapy Program Questions and Answers\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Radiology/Imaging ServicesRadiology/Imaging Services\n",
      "\n",
      " Radiology/Imaging Services\n",
      "\n",
      "72 Hour Rule72 Hour Rule\n",
      "\n",
      "\n",
      "Appropriate Use of Modifier 26Appropriate Use of Modifier 26\n",
      "\n",
      "\n",
      "Cardiology & Radiology Imaging Procedure CodesCardiology & Radiology Imaging Procedure Codes\n",
      "\n",
      "\n",
      "Contrast Agents and RadiopharmaceuticalsContrast Agents and Radiopharmaceuticals\n",
      "\n",
      " Contrast Agents and Radiopharmaceuticals\n",
      "\n",
      "Contrast Agents and RadiopharmaceuticalsContrast Agents and Radiopharmaceuticals\n",
      "\n",
      "\n",
      "Codes Considered Inclusive to an Imaging ServiceCodes Considered Inclusive to an Imaging Service\n",
      "\n",
      "\n",
      "Codes are Active/Invoice is RequiredCodes are Active/Invoice is Required\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Maternal Fetal Medicine Evaluation CodingMaternal Fetal Medicine Evaluation Coding\n",
      "\n",
      "\n",
      "Medical Information Requirements for Programs Administered by eviCoreMedical Information Requirements for Programs Administered by eviCore\n",
      "\n",
      "\n",
      "Radiology/Imaging Guidelines for Emergency Room Preliminary Reads (Wet Reads)Radiology/Imaging Guidelines for Emergency Room Preliminary Reads (Wet Reads)\n",
      "\n",
      "\n",
      "Radiology/Imaging Program Guidelines for Use of Modifier 59Radiology/Imaging Program Guidelines for Use of Modifier 59\n",
      "\n",
      "\n",
      "Radiology Imaging Program Q &amp; ARadiology Imaging Program Q &amp; A\n",
      "\n",
      "\n",
      "Correct Coding Rules Bank for Radiology, Cardiology and Ultrasound ServicesCorrect Coding Rules Bank for Radiology, Cardiology and Ultrasound Services\n",
      "\n",
      " Correct Coding Rules Bank for Radiology, Cardiology and Ultrasound Services\n",
      "\n",
      "Correct Coding Rules BankCorrect Coding Rules Bank\n",
      "\n",
      "\n",
      "Code Pairs Added to this List Effective January 1, 2023Code Pairs Added to this List Effective January 1, 2023\n",
      "\n",
      "\n",
      "Code Pairs Removed from this List Effective December 31, 2022Code Pairs Removed from this List Effective December 31, 2022\n",
      "\n",
      "\n",
      "Code Description ChangesCode Description Changes\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Code Bundling RulesCode Bundling Rules\n",
      "\n",
      " Code Bundling Rules\n",
      "\n",
      "Code Bundling Rules for Radiology, Cardiology and Ultrasound ServicesCode Bundling Rules for Radiology, Cardiology and Ultrasound Services\n",
      "\n",
      "\n",
      "Code Bundling Rules for PET/MRI ServicesCode Bundling Rules for PET/MRI Services\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Value-Based ProgramsValue-Based Programs\n",
      "\n",
      " Value-Based Programs\n",
      "\n",
      "Quality vs. Fee for ServiceQuality vs. Fee for Service\n",
      "\n",
      "\n",
      "Tools and ResourcesTools and Resources\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Policies & ProceduresPolicies & Procedures\n",
      "\n",
      " Policies & Procedures \n",
      "\n",
      "HEDIS ResourcesHEDIS Resources\n",
      "\n",
      " HEDIS Resources\n",
      "\n",
      "Behavioral Health HEDIS Webinar SeriesBehavioral Health HEDIS Webinar Series\n",
      "\n",
      " Behavioral Health HEDIS Webinar Series\n",
      "\n",
      "Introduction to Behavioral Health HEDISIntroduction to Behavioral Health HEDIS\n",
      "\n",
      "\n",
      "Child Behavioral Health HEDIS MeasuresChild Behavioral Health HEDIS Measures\n",
      "\n",
      "\n",
      "Adult Behavioral Health HEDIS MeasuresAdult Behavioral Health HEDIS Measures\n",
      "\n",
      "\n",
      "Substance Use Disorder HEDIS MeasuresSubstance Use Disorder HEDIS Measures\n",
      "\n",
      "\n",
      "Follow-Up Care for Children Prescribed ADHD Medication & Metabolic Monitoring for Children and Adolescents on Antipsychotics HEDIS MeasuresFollow-Up Care for Children Prescribed ADHD Medication & Metabolic Monitoring for Children and Adolescents on Antipsychotics HEDIS Measures\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Claim Submission & BillingClaim Submission & Billing\n",
      "\n",
      " Claim Submission & Billing\n",
      "\n",
      "Billable Service ExceptionsBillable Service Exceptions\n",
      "\n",
      " Billable Service Exceptions\n",
      "\n",
      "Eligible Laboratory Procedures Rendered by a PracticeEligible Laboratory Procedures Rendered by a Practice\n",
      "\n",
      "\n",
      "PCP Billable ListsPCP Billable Lists\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Claim Editing PoliciesClaim Editing Policies\n",
      "\n",
      "\n",
      "Claim OverpaymentsClaim Overpayments\n",
      "\n",
      "\n",
      "Claim ReimbursementClaim Reimbursement\n",
      "\n",
      "\n",
      "Claim SubmissionClaim Submission\n",
      "\n",
      " Claim Submission\n",
      "\n",
      "Claim Submission InstructionsClaim Submission Instructions\n",
      "\n",
      " Claim Submission Instructions\n",
      "\n",
      "Dental ServicesDental Services\n",
      "\n",
      "\n",
      "InstitutionalInstitutional\n",
      "\n",
      "\n",
      "ProfessionalProfessional\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Tips for Electronic Claim SubmissionTips for Electronic Claim Submission\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Electronic Claim AdjustmentsElectronic Claim Adjustments\n",
      "\n",
      "\n",
      "Explanation of PaymentExplanation of Payment\n",
      "\n",
      "\n",
      "PCP Billable ListsPCP Billable Lists\n",
      "\n",
      "\n",
      "Pre-payment Correct Coding ReviewsPre-payment Correct Coding Reviews\n",
      "\n",
      "\n",
      "Prompt Pay GuidelinesPrompt Pay Guidelines\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Demographic UpdatesDemographic Updates\n",
      "\n",
      " Demographic Updates\n",
      "\n",
      "The Importance of Demographic UpdatesThe Importance of Demographic Updates\n",
      "\n",
      "\n",
      "Provider Directory Management PolicyProvider Directory Management Policy\n",
      "\n",
      "\n",
      "CMS Audits to Validate Directory InformationCMS Audits to Validate Directory Information\n",
      "\n",
      "\n",
      "How to Make Demographic Updates: Participating Ancillary ProvidersHow to Make Demographic Updates: Participating Ancillary Providers\n",
      "\n",
      " How to Make Demographic Updates: Participating Ancillary Providers\n",
      "\n",
      "Horizon Data Submission Template for Ancillary ProvidersHorizon Data Submission Template for Ancillary Providers\n",
      "\n",
      "\n",
      "Exceptions to Using Horizon Data Submission TemplateExceptions to Using Horizon Data Submission Template\n",
      "\n",
      "\n",
      "Supporting Documentation for Ancillary Provider Demographic UpdatesSupporting Documentation for Ancillary Provider Demographic Updates\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "How to Make Demographic Updates: Participating PracticesHow to Make Demographic Updates: Participating Practices\n",
      "\n",
      " How to Make Demographic Updates: Participating Practices\n",
      "\n",
      "Provider Data Maintenance ToolProvider Data Maintenance Tool\n",
      "\n",
      "\n",
      "Horizon Data Submission TemplateHorizon Data Submission Template\n",
      "\n",
      "\n",
      "Specific Criteria You Should Confirm is Accurate and Up to DateSpecific Criteria You Should Confirm is Accurate and Up to Date\n",
      "\n",
      "\n",
      "Supporting Documentation Requirements for Practice-level Demographic UpdatesSupporting Documentation Requirements for Practice-level Demographic Updates\n",
      "\n",
      "\n",
      "Supporting Documentation Requirements for Practitioner Demographic UpdatesSupporting Documentation Requirements for Practitioner Demographic Updates\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Initiating Demographic Updates: Nonparticipating ProvidersInitiating Demographic Updates: Nonparticipating Providers\n",
      "\n",
      "\n",
      "Using CAQH ProView&trade;Using CAQH ProView&trade;\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Inquiries, Complaints & AppealsInquiries, Complaints & Appeals\n",
      "\n",
      " Inquiries, Complaints & Appeals\n",
      "\n",
      "InquiriesInquiries\n",
      "\n",
      "\n",
      "ComplaintsComplaints\n",
      "\n",
      "\n",
      "Time Limits for Filing Inquiries/ComplaintsTime Limits for Filing Inquiries/Complaints\n",
      "\n",
      "\n",
      "Resolving Inquiries/ComplaintsResolving Inquiries/Complaints\n",
      "\n",
      "\n",
      "Appeals of Non-Utilization Management DeterminationsAppeals of Non-Utilization Management Determinations\n",
      "\n",
      "\n",
      "Appeals of Utilization Management/Medical Management DeterminationsAppeals of Utilization Management/Medical Management Determinations\n",
      "\n",
      "\n",
      "Appeals of Post Service Medical Necessity DeterminationsAppeals of Post Service Medical Necessity Determinations\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "PoliciesPolicies\n",
      "\n",
      " Policies\n",
      "\n",
      "Medical PoliciesMedical Policies\n",
      "\n",
      " Medical Policies\n",
      "\n",
      "Allied HealthAllied Health\n",
      "\n",
      "\n",
      "DMEDME\n",
      "\n",
      "\n",
      "DrugsDrugs\n",
      "\n",
      "\n",
      "IntroductionIntroduction\n",
      "\n",
      "\n",
      "MedicineMedicine\n",
      "\n",
      "\n",
      "ObstetricsObstetrics\n",
      "\n",
      "\n",
      "PathologyPathology\n",
      "\n",
      "\n",
      "RadiologyRadiology\n",
      "\n",
      "\n",
      "SurgerySurgery\n",
      "\n",
      "\n",
      "TreatmentTreatment\n",
      "\n",
      "\n",
      "Hosted Medical Policy ContentHosted Medical Policy Content\n",
      "\n",
      "\n",
      "MCG Care GuidelinesMCG Care Guidelines\n",
      "\n",
      "\n",
      "Documentation Submission GuidelinesDocumentation Submission Guidelines\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Administrative PoliciesAdministrative Policies\n",
      "\n",
      " Administrative Policies\n",
      "\n",
      "Allowable Practice Locations for PathologistsAllowable Practice Locations for Pathologists\n",
      "\n",
      "\n",
      "Appointment Availability Access Standards for Primary Care-Type Providers, ObGyns, Specialists and Behavioral Health ProvidersAppointment Availability Access Standards for Primary Care-Type Providers, ObGyns, Specialists and Behavioral Health Providers\n",
      "\n",
      "\n",
      "Credentialing and Recredentialing Policy for Ancillary and Managed Long Term Support Service (MLTSS) ProvidersCredentialing and Recredentialing Policy for Ancillary and Managed Long Term Support Service (MLTSS) Providers\n",
      "\n",
      " Credentialing and Recredentialing Policy for Ancillary and Managed Long Term Support Service (MLTSS) Providers\n",
      "\n",
      "Initial CredentialingInitial Credentialing\n",
      "\n",
      "\n",
      "RecredentialingRecredentialing\n",
      "\n",
      "\n",
      "Standards for ParticipationStandards for Participation\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Credentialing and Recredentialing Policy for Participating Physicians and Healthcare ProfessionalsCredentialing and Recredentialing Policy for Participating Physicians and Healthcare Professionals\n",
      "\n",
      "\n",
      "Diagnostic Imaging Privileging by Participating Provider Practice SpecialtyDiagnostic Imaging Privileging by Participating Provider Practice Specialty\n",
      "\n",
      "\n",
      "Digital Member ID Card PolicyDigital Member ID Card Policy\n",
      "\n",
      "\n",
      "EDI and NaviNet Claims Submission RequirementEDI and NaviNet Claims Submission Requirement\n",
      "\n",
      "\n",
      "Electronic Funds Transfer (EFT)Electronic Funds Transfer (EFT)\n",
      "\n",
      "\n",
      "Material Adverse Change (MAC) Notification PolicyMaterial Adverse Change (MAC) Notification Policy\n",
      "\n",
      "\n",
      "Medical Records Documentation StandardsMedical Records Documentation Standards\n",
      "\n",
      "\n",
      "Medicare Advantage ReadmissionMedicare Advantage Readmission\n",
      "\n",
      "\n",
      "Never EventsNever Events\n",
      "\n",
      "\n",
      "Out-of-Network Referral PolicyOut-of-Network Referral Policy\n",
      "\n",
      "\n",
      "Outlier Audit Programs: Post Payment and Pre-PaymentOutlier Audit Programs: Post Payment and Pre-Payment\n",
      "\n",
      "\n",
      "Physician and Healthcare Professional Counseling and Termination Policy - Professional CompetencyPhysician and Healthcare Professional Counseling and Termination Policy - Professional Competency\n",
      "\n",
      "\n",
      "Participation Status in Products that Utilize Tiering and/or Subset of an Existing Horizon NetworkParticipation Status in Products that Utilize Tiering and/or Subset of an Existing Horizon Network\n",
      "\n",
      "\n",
      "Pass Through Billing (Modifier 90)Pass Through Billing (Modifier 90)\n",
      "\n",
      "\n",
      "Practice Location ReinstatementsPractice Location Reinstatements\n",
      "\n",
      "\n",
      "Practice Locations LimitPractice Locations Limit\n",
      "\n",
      "\n",
      "Practitioner Office Site Quality and Medical/Behavioral Health Record Keeping StandardsPractitioner Office Site Quality and Medical/Behavioral Health Record Keeping Standards\n",
      "\n",
      "\n",
      "Provider Directory ManagementProvider Directory Management\n",
      "\n",
      "\n",
      "Provider Outlier Program Frequently Asked QuestionsProvider Outlier Program Frequently Asked Questions\n",
      "\n",
      "\n",
      "Retainer Based MedicineRetainer Based Medicine\n",
      "\n",
      "\n",
      "SHBP/SEHBP Inpatient Readmission ReimbursementSHBP/SEHBP Inpatient Readmission Reimbursement\n",
      "\n",
      "\n",
      "Standards for All Types of Medical and Dental Diagnostic Radiology and Imaging FacilitiesStandards for All Types of Medical and Dental Diagnostic Radiology and Imaging Facilities\n",
      "\n",
      "\n",
      "Tier Awareness PolicyTier Awareness Policy\n",
      "\n",
      "\n",
      "Use of Horizon Hospital Network Performance DataUse of Horizon Hospital Network Performance Data\n",
      "\n",
      "\n",
      "Use of Practitioner Performance DataUse of Practitioner Performance Data\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "BlueCard Medical PoliciesBlueCard Medical Policies\n",
      "\n",
      "\n",
      "Reimbursement Policies & GuidelinesReimbursement Policies & Guidelines\n",
      "\n",
      " Reimbursement Policies & Guidelines\n",
      "\n",
      "Add-on Payment for COVID-19 Diagnostic Testing Run on High Throughput Technology (U0005)Add-on Payment for COVID-19 Diagnostic Testing Run on High Throughput Technology (U0005)\n",
      "\n",
      "\n",
      "After-hours and Weekend CareAfter-hours and Weekend Care\n",
      "\n",
      "\n",
      "Allergy ServicesAllergy Services\n",
      "\n",
      "\n",
      "Ambulance ServicesAmbulance Services\n",
      "\n",
      "\n",
      "Ambulatory Electrocardiographic MonitoringAmbulatory Electrocardiographic Monitoring\n",
      "\n",
      "\n",
      "Anesthesia Reimbursement GuidelinesAnesthesia Reimbursement Guidelines\n",
      "\n",
      "\n",
      "ASC Multiple Procedure MethodologyASC Multiple Procedure Methodology\n",
      "\n",
      "\n",
      "Assistant at SurgeryAssistant at Surgery\n",
      "\n",
      "\n",
      "Balloon SinuplastyBalloon Sinuplasty\n",
      "\n",
      "\n",
      "Bariatric Surgery Billed With Hiatal Hernia RepairBariatric Surgery Billed With Hiatal Hernia Repair\n",
      "\n",
      "\n",
      "Behavioral Health Services Rendered by Supervised PractitionersBehavioral Health Services Rendered by Supervised Practitioners\n",
      "\n",
      "\n",
      "Bilateral ProceduresBilateral Procedures\n",
      "\n",
      "\n",
      "Billing Guidelines for Maternity ServicesBilling Guidelines for Maternity Services\n",
      "\n",
      "\n",
      "Biologics CodingBiologics Coding\n",
      "\n",
      "\n",
      "Cardiac Event DetectionCardiac Event Detection\n",
      "\n",
      "\n",
      "Cardiovascular Implant Device Monitoring ServicesCardiovascular Implant Device Monitoring Services\n",
      "\n",
      "\n",
      "Casting, Strapping and SplintsCasting, Strapping and Splints\n",
      "\n",
      "\n",
      "Chemotherapy AdministrationChemotherapy Administration\n",
      "\n",
      "\n",
      "Chronic Care Management ServicesChronic Care Management Services\n",
      "\n",
      "\n",
      "Claim Editing PoliciesClaim Editing Policies\n",
      "\n",
      "\n",
      "Claims Requiring Additional DocumentationClaims Requiring Additional Documentation\n",
      "\n",
      "\n",
      "ClaimsXten Editing RulesClaimsXten Editing Rules\n",
      "\n",
      "\n",
      "Clinical Trials Support ProgramClinical Trials Support Program\n",
      "\n",
      "\n",
      "Co-Surgeon ReimbursementCo-Surgeon Reimbursement\n",
      "\n",
      "\n",
      "Collaborative Care Management ServicesCollaborative Care Management Services\n",
      "\n",
      "\n",
      "Colonoscopy with Modifier 59Colonoscopy with Modifier 59\n",
      "\n",
      "\n",
      "Conscious SedationConscious Sedation\n",
      "\n",
      "\n",
      "Consultation Services PaymentConsultation Services Payment\n",
      "\n",
      "\n",
      "Consumable Medical SuppliesConsumable Medical Supplies\n",
      "\n",
      "\n",
      "Continuous Positive Airway Pressure or Bi-level Positive airway Pressure (CPAP/BiPAP) SuppliesContinuous Positive Airway Pressure or Bi-level Positive airway Pressure (CPAP/BiPAP) Supplies\n",
      "\n",
      "\n",
      "COVID 19 Antibody TestingCOVID 19 Antibody Testing\n",
      "\n",
      "\n",
      "COVID-19 Testing and Testing Related ServicesCOVID-19 Testing and Testing Related Services\n",
      "\n",
      "\n",
      "Daily Management of Epidural or Subarachnoid Continuous Drug AdministrationDaily Management of Epidural or Subarachnoid Continuous Drug Administration\n",
      "\n",
      "\n",
      "Daily Maximum Units for Surgical Pathology and Microscopic ExaminationDaily Maximum Units for Surgical Pathology and Microscopic Examination\n",
      "\n",
      "\n",
      "Determination of Refractive StateDetermination of Refractive State\n",
      "\n",
      "\n",
      "Diabetic Screening ServicesDiabetic Screening Services\n",
      "\n",
      "\n",
      "Diabetic SuppliesDiabetic Supplies\n",
      "\n",
      "\n",
      "Distinct Procedural Service Modifiers (59, XE, XP, XS, XU)Distinct Procedural Service Modifiers (59, XE, XP, XS, XU)\n",
      "\n",
      "\n",
      "DME Rent to PurchaseDME Rent to Purchase\n",
      "\n",
      "\n",
      "Doula ServicesDoula Services\n",
      "\n",
      "\n",
      "Drug and Immunization PricingDrug and Immunization Pricing\n",
      "\n",
      "\n",
      "Duplex ScanningDuplex Scanning\n",
      "\n",
      "\n",
      "Drug Wastage – Modifier JWDrug Wastage – Modifier JW\n",
      "\n",
      "\n",
      "Duplicate Claim Logic for Independent Laboratory ServicesDuplicate Claim Logic for Independent Laboratory Services\n",
      "\n",
      "\n",
      "Evaluation and Management Services with Chiropractic Manipulative TreatmentEvaluation and Management Services with Chiropractic Manipulative Treatment\n",
      "\n",
      "\n",
      "Evaluation and Management Services with Osteopathic Manipulative TreatmentEvaluation and Management Services with Osteopathic Manipulative Treatment\n",
      "\n",
      "\n",
      "Evaluation and Management Services billed with Global Radiology, Stress Test, Stress Echo, Myocardial Profusion ImagingEvaluation and Management Services billed with Global Radiology, Stress Test, Stress Echo, Myocardial Profusion Imaging\n",
      "\n",
      "\n",
      "False ClaimsFalse Claims\n",
      "\n",
      "\n",
      "Free Flap Breast ReconstructionFree Flap Breast Reconstruction\n",
      "\n",
      "\n",
      "Frequency of Care Coordination Services and ESRD ProceduresFrequency of Care Coordination Services and ESRD Procedures\n",
      "\n",
      "\n",
      "Frequency of G0179Frequency of G0179\n",
      "\n",
      "\n",
      "Hip ArthroscopyHip Arthroscopy\n",
      "\n",
      "\n",
      "Horizon Fee Schedule Updates based on Third Party SourcesHorizon Fee Schedule Updates based on Third Party Sources\n",
      "\n",
      "\n",
      "Hospital Non-Patient Laboratory ServicesHospital Non-Patient Laboratory Services\n",
      "\n",
      " Hospital Non-Patient Laboratory Services\n",
      "\n",
      "Hospital Non-Patient Laboratory Services Sample FeesHospital Non-Patient Laboratory Services Sample Fees\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Hot or Cold Pack TherapyHot or Cold Pack Therapy\n",
      "\n",
      "\n",
      "Inpatient ConsultationsInpatient Consultations\n",
      "\n",
      "\n",
      "Knee ArthroscopyKnee Arthroscopy\n",
      "\n",
      "\n",
      "Lab Panel RebundlingLab Panel Rebundling\n",
      "\n",
      "\n",
      "Laboratory Services Billed by PhysiciansLaboratory Services Billed by Physicians\n",
      "\n",
      "\n",
      "Laser Treatment of Psoriasis or ParapsoriasisLaser Treatment of Psoriasis or Parapsoriasis\n",
      "\n",
      "\n",
      "Maternity ReimbursementMaternity Reimbursement\n",
      "\n",
      "\n",
      "Maximum Units for AnesthesiaMaximum Units for Anesthesia\n",
      "\n",
      "\n",
      "Medical Nutrition Therapy (MNT)Medical Nutrition Therapy (MNT)\n",
      "\n",
      "\n",
      "Medicare Advantage Hospital Sequestration ReimbursementMedicare Advantage Hospital Sequestration Reimbursement\n",
      "\n",
      "\n",
      "Microsurgery and Robotic SurgeryMicrosurgery and Robotic Surgery\n",
      "\n",
      "\n",
      "Modifier 25Modifier 25\n",
      "\n",
      "\n",
      "Modifier 50 Bilateral GuidelinesModifier 50 Bilateral Guidelines\n",
      "\n",
      "\n",
      "Modifier 52Modifier 52\n",
      "\n",
      "\n",
      "Modifier 53Modifier 53\n",
      "\n",
      "\n",
      "Modifier 54Modifier 54\n",
      "\n",
      "\n",
      "Modifier 55Modifier 55\n",
      "\n",
      "\n",
      "Modifier 56Modifier 56\n",
      "\n",
      "\n",
      "Modifier 57Modifier 57\n",
      "\n",
      "\n",
      "Modifier 76Modifier 76\n",
      "\n",
      "\n",
      "Modifier 77Modifier 77\n",
      "\n",
      "\n",
      "Modifier 78Modifier 78\n",
      "\n",
      "\n",
      "Modifier SUModifier SU\n",
      "\n",
      "\n",
      "Modifiers CQ/COModifiers CQ/CO\n",
      "\n",
      "\n",
      "Multiple Procedure ReductionsMultiple Procedure Reductions\n",
      "\n",
      "\n",
      "Mutually and Non-Mutually Exclusive NCCI Supplemental EditsMutually and Non-Mutually Exclusive NCCI Supplemental Edits\n",
      "\n",
      "\n",
      "Non-ESRD ESA Level ReportingNon-ESRD ESA Level Reporting\n",
      "\n",
      "\n",
      "Noncovered Related ServicesNoncovered Related Services\n",
      "\n",
      "\n",
      "Outpatient ConsultationsOutpatient Consultations\n",
      "\n",
      "\n",
      "Outpatient Facility Claim Coding RequirementsOutpatient Facility Claim Coding Requirements\n",
      "\n",
      "\n",
      "Outpatient Facility Code Edits: Bundling and Revenue CodesOutpatient Facility Code Edits: Bundling and Revenue Codes\n",
      "\n",
      " Outpatient Facility Code Edits: Bundling and Revenue Codes\n",
      "\n",
      "Bundled ServicesBundled Services\n",
      "\n",
      "\n",
      "Lab codes when billed with other servicesLab codes when billed with other services\n",
      "\n",
      "\n",
      "Revenue Codes Requiring HCPCS CodesRevenue Codes Requiring HCPCS Codes\n",
      "\n",
      "\n",
      "Typically packaged codesTypically packaged codes\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Outpatient Laboratory Claims: Referring Practitioner RequiredOutpatient Laboratory Claims: Referring Practitioner Required\n",
      "\n",
      "\n",
      "Outpatient Services Prior to Admission or Same-Day SurgeryOutpatient Services Prior to Admission or Same-Day Surgery\n",
      "\n",
      "\n",
      "Outpatient Therapy Daily MaximumOutpatient Therapy Daily Maximum\n",
      "\n",
      "\n",
      "Physician Extenders Non-Surgical ServicesPhysician Extenders Non-Surgical Services\n",
      "\n",
      "\n",
      "Pre-Payment Coding Reviews Documentation RequestsPre-Payment Coding Reviews Documentation Requests\n",
      "\n",
      "\n",
      "Pre-Payment Documentation Requests: Facility ClaimsPre-Payment Documentation Requests: Facility Claims\n",
      "\n",
      "\n",
      "Post Payment Documentation Requests: Facility ClaimsPost Payment Documentation Requests: Facility Claims\n",
      "\n",
      "\n",
      "Pulmonary Diagnostic Procedures when billed with E&M CodesPulmonary Diagnostic Procedures when billed with E&M Codes\n",
      "\n",
      "\n",
      "Radiology, Preliminary and Double ReadsRadiology, Preliminary and Double Reads\n",
      "\n",
      "\n",
      "RadiopharmaceuticalsRadiopharmaceuticals\n",
      "\n",
      "\n",
      "Reimbursement and Billing Guidelines for Anesthesia ClaimsReimbursement and Billing Guidelines for Anesthesia Claims\n",
      "\n",
      "\n",
      "Removal of Impacted Cerumen Requiring InstrumentationRemoval of Impacted Cerumen Requiring Instrumentation\n",
      "\n",
      "\n",
      "Screening and Diagnostic Mammography & 3D TomosynthesisScreening and Diagnostic Mammography & 3D Tomosynthesis\n",
      "\n",
      "\n",
      "Site of Service DifferentialSite of Service Differential\n",
      "\n",
      "\n",
      "Smoking CessationSmoking Cessation\n",
      "\n",
      "\n",
      "Status N CodesStatus N Codes\n",
      "\n",
      "\n",
      "Telemedicine ServicesTelemedicine Services\n",
      "\n",
      " Telemedicine Services\n",
      "\n",
      "Telemedicine Services Reimbursement Policy: Temporary Addendum for Horizon BCBSNJ Commercial/ASO plans and productsTelemedicine Services Reimbursement Policy: Temporary Addendum for Horizon BCBSNJ Commercial/ASO plans and products\n",
      "\n",
      "\n",
      "Telemedicine Services Reimbursement Policy: Temporary Addendum for Horizon Medicare AdvantageTelemedicine Services Reimbursement Policy: Temporary Addendum for Horizon Medicare Advantage\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Ulcer Debridement and Ulcer StagesUlcer Debridement and Ulcer Stages\n",
      "\n",
      "\n",
      "Urgent Care Center Billing RequirementsUrgent Care Center Billing Requirements\n",
      "\n",
      "\n",
      "Urinalysis with Evaluation and Management (E&M) ServicesUrinalysis with Evaluation and Management (E&M) Services\n",
      "\n",
      "\n",
      "Urine Drug Screening/TestingUrine Drug Screening/Testing\n",
      "\n",
      "\n",
      "Vascular CodingVascular Coding\n",
      "\n",
      "\n",
      "Vitamin D TestingVitamin D Testing\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Utilization ManagementUtilization Management\n",
      "\n",
      " Utilization Management\n",
      "\n",
      "Utilization Management Request ToolUtilization Management Request Tool\n",
      "\n",
      "\n",
      "Prior Authorization Procedure Search ToolPrior Authorization Procedure Search Tool\n",
      "\n",
      "\n",
      "BlueCard MembersBlueCard Members\n",
      "\n",
      "\n",
      "FEP MembersFEP Members\n",
      "\n",
      "\n",
      "UNITE HERE HEALTHUNITE HERE HEALTH\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "ResourcesResources\n",
      "\n",
      " Resources \n",
      "\n",
      "Patient Quality & Outcome ResourcesPatient Quality & Outcome Resources\n",
      "\n",
      " Patient Quality & Outcome Resources\n",
      "\n",
      "Clinical Practice GuidelinesClinical Practice Guidelines\n",
      "\n",
      "\n",
      "Consumer Assessment of Healthcare Providers and Systems (CAHPS)Consumer Assessment of Healthcare Providers and Systems (CAHPS)\n",
      "\n",
      " Consumer Assessment of Healthcare Providers and Systems (CAHPS)\n",
      "\n",
      "Focusing on Your Horizon and Braven Health Patients’ Experience: Tools to Help YouFocusing on Your Horizon and Braven Health Patients’ Experience: Tools to Help You\n",
      "\n",
      "\n",
      "Discussion Checklist for CAHPS and HOS SurveysDiscussion Checklist for CAHPS and HOS Surveys\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Cultural CompetencyCultural Competency\n",
      "\n",
      "\n",
      "HEDIS ResourcesHEDIS Resources\n",
      "\n",
      " HEDIS Resources\n",
      "\n",
      "HEDIS Measure Guidelines for Behavioral Health ProvidersHEDIS Measure Guidelines for Behavioral Health Providers\n",
      "\n",
      " HEDIS Measure Guidelines for Behavioral Health Providers\n",
      "\n",
      "Follow-Up Care for Children Prescribed ADHD Medication (ADD)Follow-Up Care for Children Prescribed ADHD Medication (ADD)\n",
      "\n",
      "\n",
      "Antidepressant Medication Management (AMM)Antidepressant Medication Management (AMM)\n",
      "\n",
      "\n",
      "Metabolic Monitoring for Children and Adolescents on Antipsychotics (APM)Metabolic Monitoring for Children and Adolescents on Antipsychotics (APM)\n",
      "\n",
      "\n",
      "Use of First-Line Psychosocial Care for Children and Adolescents on Antipsychotics (APP)Use of First-Line Psychosocial Care for Children and Adolescents on Antipsychotics (APP)\n",
      "\n",
      "\n",
      "Follow-Up After Emergency Department Visit for Substance Use (FUA)Follow-Up After Emergency Department Visit for Substance Use (FUA)\n",
      "\n",
      "\n",
      "Follow-Up After Hospitalization for Mental Illness (FUH)Follow-Up After Hospitalization for Mental Illness (FUH)\n",
      "\n",
      "\n",
      "Follow-Up After High-Intensity Care for Substance Use Disorder (FUI)Follow-Up After High-Intensity Care for Substance Use Disorder (FUI)\n",
      "\n",
      "\n",
      "Follow-Up After Emergency Department Visit for Mental Illness (FUM)Follow-Up After Emergency Department Visit for Mental Illness (FUM)\n",
      "\n",
      "\n",
      "Initiation and Engagement of Substance Use Disorder Treatment (IET)Initiation and Engagement of Substance Use Disorder Treatment (IET)\n",
      "\n",
      "\n",
      "Adherence to Antipsychotic Medications for Individuals with Schizophrenia (SAA)Adherence to Antipsychotic Medications for Individuals with Schizophrenia (SAA)\n",
      "\n",
      "\n",
      "Diabetes Screening for People with Schizophrenia or Bipolar Disorder Who Are Using Antipsychotic Medications (SSD)Diabetes Screening for People with Schizophrenia or Bipolar Disorder Who Are Using Antipsychotic Medications (SSD)\n",
      "\n",
      "\n",
      "Cardiovascular Monitoring for People with Cardiovascular Disease and Schizophrenia (SMC)Cardiovascular Monitoring for People with Cardiovascular Disease and Schizophrenia (SMC)\n",
      "\n",
      "\n",
      "Diabetes Monitoring for People with Diabetes and Schizophrenia (SMD)Diabetes Monitoring for People with Diabetes and Schizophrenia (SMD)\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "HEDIS Measurement Year (MY) 2023 Provider Tips for Optimizing HEDIS ResultsHEDIS Measurement Year (MY) 2023 Provider Tips for Optimizing HEDIS Results\n",
      "\n",
      " HEDIS Measurement Year (MY) 2023 Provider Tips for Optimizing HEDIS Results\n",
      "\n",
      "Best Practices Overall for CodingBest Practices Overall for Coding\n",
      "\n",
      "\n",
      "Adherence to Antipsychotic Medications for Individuals With Schizophrenia (SAA)Adherence to Antipsychotic Medications for Individuals With Schizophrenia (SAA)\n",
      "\n",
      "\n",
      "Adult Immunization Status (AIS-E)Adult Immunization Status (AIS-E)\n",
      "\n",
      "\n",
      "Adults’ Access to Preventive/Ambulatory Health Services (AAP)Adults’ Access to Preventive/Ambulatory Health Services (AAP)\n",
      "\n",
      "\n",
      "Advanced Care Planning (ACP)Advanced Care Planning (ACP)\n",
      "\n",
      "\n",
      "Antibiotic Utilization for Respiratory Conditions (AXR)Antibiotic Utilization for Respiratory Conditions (AXR)\n",
      "\n",
      "\n",
      "Antidepressant Medication Management (AMM)Antidepressant Medication Management (AMM)\n",
      "\n",
      "\n",
      "Appropriate Testing for Pharyngitis (CWP)Appropriate Testing for Pharyngitis (CWP)\n",
      "\n",
      "\n",
      "Appropriate Treatment for Upper Respiratory Infection (URI)Appropriate Treatment for Upper Respiratory Infection (URI)\n",
      "\n",
      "\n",
      "Asthma Medication Ratio (AMR)Asthma Medication Ratio (AMR)\n",
      "\n",
      "\n",
      "Avoidance of Antibiotic Treatment for Acute Bronchitis/ Bronchiolitis (AAB)Avoidance of Antibiotic Treatment for Acute Bronchitis/ Bronchiolitis (AAB)\n",
      "\n",
      "\n",
      "Blood Pressure Control for Patients With Diabetes (BPD)Blood Pressure Control for Patients With Diabetes (BPD)\n",
      "\n",
      "\n",
      "Breast Cancer Screening (BCS-E)Breast Cancer Screening (BCS-E)\n",
      "\n",
      "\n",
      "Cardiovascular Monitoring for People With Cardiovascular Disease and Schizophrenia (SMC)Cardiovascular Monitoring for People With Cardiovascular Disease and Schizophrenia (SMC)\n",
      "\n",
      "\n",
      "Care for Older Adults (COA)Care for Older Adults (COA)\n",
      "\n",
      "\n",
      "Cervical Cancer Screening (CCS) (CCS-E)Cervical Cancer Screening (CCS) (CCS-E)\n",
      "\n",
      "\n",
      "Child and Adolescent Well-Care Visits (WCV)Child and Adolescent Well-Care Visits (WCV)\n",
      "\n",
      "\n",
      "Childhood Immunization Status (CIS) (CIS-E)Childhood Immunization Status (CIS) (CIS-E)\n",
      "\n",
      "\n",
      "Chlamydia Screening in Women (CHL)Chlamydia Screening in Women (CHL)\n",
      "\n",
      "\n",
      "Colorectal Cancer Screening (COL) (COL-E)Colorectal Cancer Screening (COL) (COL-E)\n",
      "\n",
      "\n",
      "Controlling High Blood Pressure (CBP)Controlling High Blood Pressure (CBP)\n",
      "\n",
      "\n",
      "Deprescribing of Benzodiazepines in Older Adults (DBO)Deprescribing of Benzodiazepines in Older Adults (DBO)\n",
      "\n",
      "\n",
      "Depression Remission or Response for Adolescents and Adults (DRR-E)Depression Remission or Response for Adolescents and Adults (DRR-E)\n",
      "\n",
      "\n",
      "Depression Screening and Follow-Up for Adolescents and Adults (DSF-E)Depression Screening and Follow-Up for Adolescents and Adults (DSF-E)\n",
      "\n",
      "\n",
      "Diabetes Monitoring for People With Diabetes and Schizophrenia (SMD)Diabetes Monitoring for People With Diabetes and Schizophrenia (SMD)\n",
      "\n",
      "\n",
      "Diabetes Screening for People With Schizophrenia or Bipolar Disorder Who Are Using Antipsychotic Medications (SSD)Diabetes Screening for People With Schizophrenia or Bipolar Disorder Who Are Using Antipsychotic Medications (SSD)\n",
      "\n",
      "\n",
      "Eye Exam for Patients With Diabetes (EED)Eye Exam for Patients With Diabetes (EED)\n",
      "\n",
      "\n",
      "Follow-Up After Emergency Department Visit for Alcohol and Other Drug Abuse or Dependence (FUA)Follow-Up After Emergency Department Visit for Alcohol and Other Drug Abuse or Dependence (FUA)\n",
      "\n",
      "\n",
      "Follow-Up After Emergency Department Visit for Mental Illness (FUM)Follow-Up After Emergency Department Visit for Mental Illness (FUM)\n",
      "\n",
      "\n",
      "Follow-Up After Emergency Department Visit for People With Multiple High-Risk Chronic Conditions (FMC)Follow-Up After Emergency Department Visit for People With Multiple High-Risk Chronic Conditions (FMC)\n",
      "\n",
      "\n",
      "Follow-Up After High- Intensity Care for Substance Use Disorder (FUI)Follow-Up After High- Intensity Care for Substance Use Disorder (FUI)\n",
      "\n",
      "\n",
      "Follow-Up After Hospitalization for Mental Illness (FUH)Follow-Up After Hospitalization for Mental Illness (FUH)\n",
      "\n",
      "\n",
      "Follow-Up Care for Children Prescribed ADHD Medication (ADD-E)Follow-Up Care for Children Prescribed ADHD Medication (ADD-E)\n",
      "\n",
      "\n",
      "Hemoglobin A1c Control for Patients With Diabetes (HBD)Hemoglobin A1c Control for Patients With Diabetes (HBD)\n",
      "\n",
      "\n",
      "Immunizations for Adolescents  (IMA) (IMA-E)Immunizations for Adolescents  (IMA) (IMA-E)\n",
      "\n",
      "\n",
      "Initiation and Engagement of Substance Use Disorder Treatment (IET)Initiation and Engagement of Substance Use Disorder Treatment (IET)\n",
      "\n",
      "\n",
      "Kidney Health Evaluation for Patients with Diabetes (KED)Kidney Health Evaluation for Patients with Diabetes (KED)\n",
      "\n",
      "\n",
      "Lead Screening in Children (LSC) - EPSDTLead Screening in Children (LSC) - EPSDT\n",
      "\n",
      "\n",
      "Lead Screening in Children (LSC) - NCQALead Screening in Children (LSC) - NCQA\n",
      "\n",
      "\n",
      "Metabolic Monitoring for Children and Adolescents on Antipsychotics (APM-E)Metabolic Monitoring for Children and Adolescents on Antipsychotics (APM-E)\n",
      "\n",
      "\n",
      "Osteoporosis Management in Women Who Had a Fracture (OMW)Osteoporosis Management in Women Who Had a Fracture (OMW)\n",
      "\n",
      "\n",
      "Osteoporosis Screening in Older Women (OSW)Osteoporosis Screening in Older Women (OSW)\n",
      "\n",
      "\n",
      "Persistence of Beta- Blocker Treatment After a Heart Attack (PBH)Persistence of Beta- Blocker Treatment After a Heart Attack (PBH)\n",
      "\n",
      "\n",
      "Pharmacotherapy Management of COPD Exacerbation (PCE)Pharmacotherapy Management of COPD Exacerbation (PCE)\n",
      "\n",
      "\n",
      "Postpartum Depression Screening and Follow-Up (PDS-E)Postpartum Depression Screening and Follow-Up (PDS-E)\n",
      "\n",
      "\n",
      "Prenatal Depression Screening and Follow-Up (PND-E)Prenatal Depression Screening and Follow-Up (PND-E)\n",
      "\n",
      "\n",
      "Prenatal Immunization Status (PRS-E)Prenatal Immunization Status (PRS-E)\n",
      "\n",
      "\n",
      "Prenatal and Postpartum Care (PPC)Prenatal and Postpartum Care (PPC)\n",
      "\n",
      "\n",
      "Risk of Continued Opioid Use (COU)Risk of Continued Opioid Use (COU)\n",
      "\n",
      "\n",
      "Statin Therapy for Patients with Cardiovascular Disease (SPC)Statin Therapy for Patients with Cardiovascular Disease (SPC)\n",
      "\n",
      "\n",
      "Statin Therapy for Patients with Diabetes (SPD)Statin Therapy for Patients with Diabetes (SPD)\n",
      "\n",
      "\n",
      "Topical Fluoride for Children (TFC)Topical Fluoride for Children (TFC)\n",
      "\n",
      "\n",
      "Transitions of Care (TRC)Transitions of Care (TRC)\n",
      "\n",
      "\n",
      "Unhealthy Alcohol Use Screening and Follow-Up (ASF-E)Unhealthy Alcohol Use Screening and Follow-Up (ASF-E)\n",
      "\n",
      "\n",
      "Use of First-Line Psychosocial Care for Children and Adolescents on Antipsychotics (APP)Use of First-Line Psychosocial Care for Children and Adolescents on Antipsychotics (APP)\n",
      "\n",
      "\n",
      "Use of Imaging Studies for Low Back Pain (LBP)Use of Imaging Studies for Low Back Pain (LBP)\n",
      "\n",
      "\n",
      "Use of Opioids at High Dosage (HDO)Use of Opioids at High Dosage (HDO)\n",
      "\n",
      "\n",
      "Use of Opioids from Multiple Providers (UOP)Use of Opioids from Multiple Providers (UOP)\n",
      "\n",
      "\n",
      "Use of Spirometry Testing in the Assessment and Diagnosis of COPD (SPR)Use of Spirometry Testing in the Assessment and Diagnosis of COPD (SPR)\n",
      "\n",
      "\n",
      "Utilization of the PHQ-9 to Monitor Depression Symptoms for Adolescents and Adults (DMS-E)Utilization of the PHQ-9 to Monitor Depression Symptoms for Adolescents and Adults (DMS-E)\n",
      "\n",
      "\n",
      "Weight Assessment and Counseling for Nutrition and Physical  Activity for Children/Adolescents (WCC)Weight Assessment and Counseling for Nutrition and Physical  Activity for Children/Adolescents (WCC)\n",
      "\n",
      "\n",
      "Well-Child Visits in the First 30 Months of Life (W30)Well-Child Visits in the First 30 Months of Life (W30)\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "HealthSphereHealthSphere\n",
      "\n",
      " HealthSphere\n",
      "\n",
      "HealthSphere TrainingHealthSphere Training\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "HorizonDocsHorizonDocs\n",
      "\n",
      " HorizonDocs\n",
      "\n",
      "HorizonDocs 2023 TrainingHorizonDocs 2023 Training\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Quality Resource CenterQuality Resource Center\n",
      "\n",
      " Quality Resource Center\n",
      "\n",
      "Educational MaterialsEducational Materials\n",
      "\n",
      " Educational Materials\n",
      "\n",
      "Achieving Success in Quality ImprovementAchieving Success in Quality Improvement\n",
      "\n",
      "\n",
      "Quality Management 2023 WebinarsQuality Management 2023 Webinars\n",
      "\n",
      " Quality Management 2023 Webinars\n",
      "\n",
      "R&R OverviewR&R Overview\n",
      "\n",
      "\n",
      "Adult MeasuresAdult Measures\n",
      "\n",
      "\n",
      "Pediatric MeasuresPediatric Measures\n",
      "\n",
      "\n",
      "Optimizing Monthly R&R ReportsOptimizing Monthly R&R Reports\n",
      "\n",
      "\n",
      "Overview of Horizon DocsOverview of Horizon Docs\n",
      "\n",
      "\n",
      "Quality Care Gap ClosureQuality Care Gap Closure\n",
      "\n",
      "\n",
      "Lead OverviewLead Overview\n",
      "\n",
      "\n",
      "Asthma Medication RatioAsthma Medication Ratio\n",
      "\n",
      "\n",
      "CWP OverviewCWP Overview\n",
      "\n",
      "\n",
      "Risk Adjustment OverviewRisk Adjustment Overview\n",
      "\n",
      "\n",
      "CAHPS OverviewCAHPS Overview\n",
      "\n",
      "\n",
      "TelehealthTelehealth\n",
      "\n",
      "\n",
      "Women’s HealthWomen’s Health\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "What’s the Code?What’s the Code?\n",
      "\n",
      "\n",
      "Colorectal Cancer ScreeningColorectal Cancer Screening\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Electronic Claim Data System (ECDS) Supplemental Data File FeedElectronic Claim Data System (ECDS) Supplemental Data File Feed\n",
      "\n",
      "\n",
      "Results & Recognition ProgramResults & Recognition Program\n",
      "\n",
      " Results & Recognition Program\n",
      "\n",
      "Women’s Health Results and Recognition ProgramWomen’s Health Results and Recognition Program\n",
      "\n",
      "\n",
      "Provider Guidelines: Non-Standard (Medical Record) Supplemental Data for HEDIS Gap ClosureProvider Guidelines: Non-Standard (Medical Record) Supplemental Data for HEDIS Gap Closure\n",
      "\n",
      " Provider Guidelines: Non-Standard (Medical Record) Supplemental Data for HEDIS Gap Closure\n",
      "\n",
      "Adult HEDIS MeasuresAdult HEDIS Measures\n",
      "\n",
      "\n",
      "Pediatric HEDIS MeasuresPediatric HEDIS Measures\n",
      "\n",
      "\n",
      "How to Submit Supplemental Data to HorizonHow to Submit Supplemental Data to Horizon\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Health Outcomes Survey: How You Can Drive ResultsHealth Outcomes Survey: How You Can Drive Results\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Provider Self-Service ToolsProvider Self-Service Tools\n",
      "\n",
      " Provider Self-Service Tools\n",
      "\n",
      "Drug Authorizations ToolDrug Authorizations Tool\n",
      "\n",
      "\n",
      "Electronic Data Interchange (EDI)Electronic Data Interchange (EDI)\n",
      "\n",
      " Electronic Data Interchange (EDI)\n",
      "\n",
      "Braven Health&#8480; Electronic Data Interchange (EDI)Braven Health&#8480; Electronic Data Interchange (EDI)\n",
      "\n",
      "\n",
      "Horizon BCBSNJ Electronic Data Interchange (EDI)Horizon BCBSNJ Electronic Data Interchange (EDI)\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Electronic Funds Transfer (EFT)Electronic Funds Transfer (EFT)\n",
      "\n",
      "\n",
      "Eligibility and Benefits Cost Share EstimatorEligibility and Benefits Cost Share Estimator\n",
      "\n",
      "\n",
      "Interactive Voice Response System (IVR)Interactive Voice Response System (IVR)\n",
      "\n",
      " Interactive Voice Response System (IVR)\n",
      "\n",
      "Referral Process Using the Interactive Voice Response SystemReferral Process Using the Interactive Voice Response System\n",
      "\n",
      "\n",
      "Effective use of the Interactive Voice Response SystemEffective use of the Interactive Voice Response System\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "NaviNetNaviNet\n",
      "\n",
      "\n",
      " Prior Authorization Procedure Search Tool Prior Authorization Procedure Search Tool\n",
      "\n",
      "\n",
      "Provider Data Maintenance ToolProvider Data Maintenance Tool\n",
      "\n",
      "\n",
      "Utilization Management Request ToolUtilization Management Request Tool\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Working with Us: Information & Education ResourcesWorking with Us: Information & Education Resources\n",
      "\n",
      " Working with Us: Information & Education Resources\n",
      "\n",
      "Educational WebinarsEducational Webinars\n",
      "\n",
      " Educational Webinars\n",
      "\n",
      "Behavioral Health Training WebinarsBehavioral Health Training Webinars\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Manuals & User GuidesManuals & User Guides\n",
      "\n",
      " Manuals & User Guides\n",
      "\n",
      "AgreementsAgreements\n",
      "\n",
      "\n",
      "Ancillary ManualAncillary Manual\n",
      "\n",
      " Ancillary Manual\n",
      "\n",
      "The BlueCard ProgramThe BlueCard Program\n",
      "\n",
      "\n",
      "Claims Submissions and ReimbursementClaims Submissions and Reimbursement\n",
      "\n",
      "\n",
      "Coordination of BenefitsCoordination of Benefits\n",
      "\n",
      "\n",
      "HIPAAHIPAA\n",
      "\n",
      "\n",
      "Horizon Behavioral HealthHorizon Behavioral Health\n",
      "\n",
      "\n",
      "Horizon Hospital NetworkHorizon Hospital Network\n",
      "\n",
      "\n",
      "IdentificationIdentification\n",
      "\n",
      "\n",
      "Inquiries, Complaints and AppealsInquiries, Complaints and Appeals\n",
      "\n",
      "\n",
      "Member Rights and ResponsibilitiesMember Rights and Responsibilities\n",
      "\n",
      "\n",
      "Patient-Centered ProgramsPatient-Centered Programs\n",
      "\n",
      "\n",
      "Patient Health SupportPatient Health Support\n",
      "\n",
      "\n",
      "Policies, Procedures and General GuidelinesPolicies, Procedures and General Guidelines\n",
      "\n",
      "\n",
      "ProductsProducts\n",
      "\n",
      "\n",
      "Provider ResponsibilitiesProvider Responsibilities\n",
      "\n",
      "\n",
      "Quality ManagementQuality Management\n",
      "\n",
      "\n",
      "Quality Recognition ProgramsQuality Recognition Programs\n",
      "\n",
      "\n",
      "ReferralsReferrals\n",
      "\n",
      "\n",
      "ServiceService\n",
      "\n",
      "\n",
      "Utilization ManagementUtilization Management\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Hospital ManualHospital Manual\n",
      "\n",
      " Hospital Manual\n",
      "\n",
      "Claims Submission and ReimbursementClaims Submission and Reimbursement\n",
      "\n",
      "\n",
      "Coordination of BenefitsCoordination of Benefits\n",
      "\n",
      "\n",
      "HIPAAHIPAA\n",
      "\n",
      "\n",
      "Horizon Behavioral HealthHorizon Behavioral Health\n",
      "\n",
      "\n",
      "IdentificationIdentification\n",
      "\n",
      "\n",
      "Inquiries, Complaints and AppealsInquiries, Complaints and Appeals\n",
      "\n",
      "\n",
      "Member Rights and ResponsibilitiesMember Rights and Responsibilities\n",
      "\n",
      "\n",
      "Patient Health SupportPatient Health Support\n",
      "\n",
      "\n",
      "Patient-Centered ProgramsPatient-Centered Programs\n",
      "\n",
      "\n",
      "Policies, Procedures and General GuidelinesPolicies, Procedures and General Guidelines\n",
      "\n",
      "\n",
      "ProductsProducts\n",
      "\n",
      "\n",
      "Programs Administered by eviCore healthcarePrograms Administered by eviCore healthcare\n",
      "\n",
      "\n",
      "Provider ResponsibilitiesProvider Responsibilities\n",
      "\n",
      "\n",
      "Quality ManagementQuality Management\n",
      "\n",
      "\n",
      "Quality Recognition ProgramsQuality Recognition Programs\n",
      "\n",
      "\n",
      "ServiceService\n",
      "\n",
      "\n",
      "The BlueCard ProgramThe BlueCard Program\n",
      "\n",
      "\n",
      "Utilization ManagementUtilization Management\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Important Resources for ProvidersImportant Resources for Providers\n",
      "\n",
      "\n",
      "Participating Physician and Other Health Care Professional Office ManualParticipating Physician and Other Health Care Professional Office Manual\n",
      "\n",
      " Participating Physician and Other Health Care Professional Office Manual\n",
      "\n",
      "At Your ServiceAt Your Service\n",
      "\n",
      "\n",
      "The BlueCard ProgramThe BlueCard Program\n",
      "\n",
      "\n",
      "Claims Submissions and ReimbursementClaims Submissions and Reimbursement\n",
      "\n",
      "\n",
      "Coordination of BenefitsCoordination of Benefits\n",
      "\n",
      "\n",
      "HIPAAHIPAA\n",
      "\n",
      "\n",
      "Horizon Behavioral HealthHorizon Behavioral Health\n",
      "\n",
      "\n",
      "Horizon Hospital NetworkHorizon Hospital Network\n",
      "\n",
      "\n",
      "IdentificationIdentification\n",
      "\n",
      "\n",
      "Inquiries, Complaints and AppealsInquiries, Complaints and Appeals\n",
      "\n",
      "\n",
      "Member Rights and ResponsibilitiesMember Rights and Responsibilities\n",
      "\n",
      "\n",
      "Patient-Centered ProgramsPatient-Centered Programs\n",
      "\n",
      "\n",
      "Patient Health SupportPatient Health Support\n",
      "\n",
      "\n",
      "Payment Summaries and VouchersPayment Summaries and Vouchers\n",
      "\n",
      "\n",
      "Policies, Procedures and General GuidelinesPolicies, Procedures and General Guidelines\n",
      "\n",
      "\n",
      "ProductsProducts\n",
      "\n",
      "\n",
      "Provider ResponsibilitiesProvider Responsibilities\n",
      "\n",
      "\n",
      "Quality ManagementQuality Management\n",
      "\n",
      "\n",
      "Quality Recognition ProgramsQuality Recognition Programs\n",
      "\n",
      "\n",
      "ReferralsReferrals\n",
      "\n",
      "\n",
      "ServiceService\n",
      "\n",
      "\n",
      "Special ProgramsSpecial Programs\n",
      "\n",
      "\n",
      "Utilization ManagementUtilization Management\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Welcome KitWelcome Kit\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Network SpecialistsNetwork Specialists\n",
      "\n",
      " Network Specialists\n",
      "\n",
      "Medical Network Specialist AssignmentsMedical Network Specialist Assignments\n",
      "\n",
      "\n",
      "Behavioral Health Network Specialist AssignmentsBehavioral Health Network Specialist Assignments\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Recognition Programs and PartnershipsRecognition Programs and Partnerships\n",
      "\n",
      " Recognition Programs and Partnerships\n",
      "\n",
      "Blue Distinction CentersBlue Distinction Centers\n",
      "\n",
      "\n",
      "New Jersey Infection Prevention PartnershipNew Jersey Infection Prevention Partnership\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "TrainingTraining\n",
      "\n",
      " Training\n",
      "\n",
      "CMS Training RequirementsCMS Training Requirements\n",
      "\n",
      " CMS Training Requirements\n",
      "\n",
      "Compliance Training for ProvidersCompliance Training for Providers\n",
      "\n",
      "\n",
      "Health Care Fraud, Waste and AbuseHealth Care Fraud, Waste and Abuse\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "EducationEducation\n",
      "\n",
      " Education\n",
      "\n",
      "Prepayment Coding Audit ReviewPrepayment Coding Audit Review\n",
      "\n",
      "\n",
      "ProductsProducts\n",
      "\n",
      " Products\n",
      "\n",
      "Horizon BlueCardHorizon BlueCard\n",
      "\n",
      "\n",
      "Horizon EPOHorizon EPO\n",
      "\n",
      "\n",
      "Horizon Indemnity/PPOHorizon Indemnity/PPO\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "NavinetNavinet\n",
      "\n",
      "\n",
      "Online ToolsOnline Tools\n",
      "\n",
      " Online Tools\n",
      "\n",
      "Electronic Funds TransferElectronic Funds Transfer\n",
      "\n",
      "\n",
      "Health Risk AssessmentHealth Risk Assessment\n",
      "\n",
      "\n",
      "Radiation Therapy Medical Necessity DeterminationRadiation Therapy Medical Necessity Determination\n",
      "\n",
      "\n",
      "Utilization Management Request ToolUtilization Management Request Tool\n",
      "\n",
      "\n",
      "Virtual ID CardsVirtual ID Cards\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "FormsForms\n",
      "\n",
      " Forms \n",
      "\n",
      "Forms by Plan TypeForms by Plan Type\n",
      "\n",
      " Forms by Plan Type\n",
      "\n",
      "DentalDental\n",
      "\n",
      "\n",
      "MedicalMedical\n",
      "\n",
      "\n",
      "PharmacyPharmacy\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Forms by Specialty TypeForms by Specialty Type\n",
      "\n",
      " Forms by Specialty Type\n",
      "\n",
      "Horizon Behavioral HealthHorizon Behavioral Health\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Forms by TypeForms by Type\n",
      "\n",
      " Forms by Type\n",
      "\n",
      "Appeal / DisputeAppeal / Dispute\n",
      "\n",
      "\n",
      "Assess / Evaluate / ExamineAssess / Evaluate / Examine\n",
      "\n",
      "\n",
      "AuthorizationsAuthorizations\n",
      "\n",
      "\n",
      "Braven Health FormsBraven Health Forms\n",
      "\n",
      "\n",
      "ClaimClaim\n",
      "\n",
      "\n",
      "ConsentConsent\n",
      "\n",
      "\n",
      "CredentialingCredentialing\n",
      "\n",
      "\n",
      "Enroll / Elect / ApplyEnroll / Elect / Apply\n",
      "\n",
      "\n",
      "Horizon NJ TotalCare (HMO D-SNP) FormsHorizon NJ TotalCare (HMO D-SNP) Forms\n",
      "\n",
      "\n",
      "Inquiry / RequestInquiry / Request\n",
      "\n",
      "\n",
      "Prescription Drug Mail OrderPrescription Drug Mail Order\n",
      "\n",
      "\n",
      "Reimbursement / PaymentReimbursement / Payment\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Frequently Used FormsFrequently Used Forms\n",
      "\n",
      "\n",
      "MiscellaneousMiscellaneous\n",
      "\n",
      "\n",
      "W9 Form-DentalW9 Form-Dental\n",
      "\n",
      "\n",
      "W9 Form-MedicalW9 Form-Medical\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "NewsNews\n",
      "\n",
      " News \n",
      "\n",
      "Blue ReviewBlue Review\n",
      "\n",
      " Blue Review\n",
      "\n",
      "Blue Review September 2023Blue Review September 2023\n",
      "\n",
      "\n",
      "ArchivesArchives\n",
      "\n",
      " Archives\n",
      "\n",
      "Blue Review March 2022Blue Review March 2022\n",
      "\n",
      "\n",
      "Blue Review December 2021Blue Review December 2021\n",
      "\n",
      "\n",
      "Blue Review September 2022Blue Review September 2022\n",
      "\n",
      "\n",
      "Blue Review September 2021Blue Review September 2021\n",
      "\n",
      "\n",
      "Blue Review June 2021Blue Review June 2021\n",
      "\n",
      "\n",
      "Blue Review March 2021Blue Review March 2021\n",
      "\n",
      "\n",
      "Blue Review June 2023Blue Review June 2023\n",
      "\n",
      "\n",
      "Blue Review December 2022Blue Review December 2022\n",
      "\n",
      "\n",
      "Blue Review June 2022Blue Review June 2022\n",
      "\n",
      "\n",
      "Blue Review March 2023Blue Review March 2023\n",
      "\n",
      "\n",
      "Blue Review 2020Blue Review 2020\n",
      "\n",
      "\n",
      "Blue Review 2019Blue Review 2019\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "Feature StoriesFeature Stories\n",
      "\n",
      " Feature Stories\n",
      "\n",
      "Pregnancy-Newborn Episodes of Care Program: Helping to Improve Outcomes for Moms and NewbornsPregnancy-Newborn Episodes of Care Program: Helping to Improve Outcomes for Moms and Newborns\n",
      "\n",
      "\n",
      "As Mental Health Needs Continue To Rise, So Do Innovative Virtual ServicesAs Mental Health Needs Continue To Rise, So Do Innovative Virtual Services\n",
      "\n",
      "\n",
      "Treat Knee, Back, and Hip Pain with Orthotic Device that Helps Avoid Invasive ProceduresTreat Knee, Back, and Hip Pain with Orthotic Device that Helps Avoid Invasive Procedures\n",
      "\n",
      "\n",
      "Horizon Neighbors in Health Program Supports Struggling FamiliesHorizon Neighbors in Health Program Supports Struggling Families\n",
      "\n",
      "\n",
      "Bariatric Surgery Value-Based Program Helps Members with Weight LossBariatric Surgery Value-Based Program Helps Members with Weight Loss\n",
      "\n",
      "\n",
      "Dental Providers Benefit from Dedicated Horizon LiaisonsDental Providers Benefit from Dedicated Horizon Liaisons\n",
      "\n",
      "\n",
      "Connecting with parents on the importance of early childhood health screenings and vaccinationsConnecting with parents on the importance of early childhood health screenings and vaccinations\n",
      "\n",
      "\n",
      "Episodes of Care Program Gives Cancer Patients the Care They NeedEpisodes of Care Program Gives Cancer Patients the Care They Need\n",
      "\n",
      "\n",
      "HealthSphere gives a behavioral health provider the full patient viewHealthSphere gives a behavioral health provider the full patient view\n",
      "\n",
      "\n",
      "Home-Delivered Meals Help Braven Health&#8480; PatientsHome-Delivered Meals Help Braven Health&#8480; Patients\n",
      "\n",
      "\n",
      "Horizon Neighbors in Health Program Helps At-Risk Members in CamdenHorizon Neighbors in Health Program Helps At-Risk Members in Camden\n",
      "\n",
      "\n",
      "How a value-based primary care provider helps the New Jersey Vaccination ProgramHow a value-based primary care provider helps the New Jersey Vaccination Program\n",
      "\n",
      "\n",
      "Improving Health Equity through Increased Access to Prenatal Care Across New JerseyImproving Health Equity through Increased Access to Prenatal Care Across New Jersey\n",
      "\n",
      "\n",
      "Making Pathways in Innovating and Advancing Maternal HealthMaking Pathways in Innovating and Advancing Maternal Health\n",
      "\n",
      "\n",
      "Pharmacy Collaboration leads to better patient outcomes and cost savingsPharmacy Collaboration leads to better patient outcomes and cost savings\n",
      "\n",
      "\n",
      "Providing Innovative Cancer Care - Expanding Episodes of CareProviding Innovative Cancer Care - Expanding Episodes of Care\n",
      "\n",
      "\n",
      "Telehealth after COVID-19 – Many doctors agree it&#39;s here to stayTelehealth after COVID-19 – Many doctors agree it&#39;s here to stay\n",
      "\n",
      "\n",
      "Value-based care -- transforming health care with better collaboration and improved health outcomesValue-based care -- transforming health care with better collaboration and improved health outcomes\n",
      "\n",
      "\n",
      "When planning, collaboration and crisis merge - a medical practice's successful response to COVID-19When planning, collaboration and crisis merge - a medical practice's successful response to COVID-19\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "News & Legal NoticesNews & Legal Notices\n",
      "\n",
      "\n",
      "Provider AlertsProvider Alerts\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "FIND A DOCTOR\n",
      "MEMBER SIGN IN\n",
      "SHOP FOR A PLAN\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "COVID-19 Information \n",
      "\n",
      "\n",
      "\n",
      "Important Information for New COVID-19 Vaccine Claims\n",
      "\n",
      "\n",
      "Code Terminations as the PHE Ends\n",
      "\n",
      "\n",
      "PHE Update: Prescription Limitation Change for Braven Health, HMO D-SNP and Medicare Part D members\n",
      "\n",
      "\n",
      "Important Update: Additional Policy Changes as PHE Ends\n",
      "\n",
      "\n",
      "Some Benefits Changes as the PHE Ends\n",
      "\n",
      "\n",
      "Referral Requirements for All Services\n",
      "\n",
      "\n",
      "Referral Requirements for Services Not Related to COVID-19\n",
      "\n",
      "\n",
      "Reminder: Select one method for COVID-19 and Influenza Testing\n",
      "\n",
      "\n",
      "Antibody testing: FDA and CDC do not recommend use to determine immunity\n",
      "\n",
      "\n",
      "COVID-19 vaccine administration reimbursement at UCCs\n",
      "\n",
      "\n",
      "Coverage for antibody infusion therapy\n",
      "\n",
      "\n",
      "Reminder to use specific codes when evaluating for COVID-19\n",
      "\n",
      "\n",
      "COVID-19 Update: Telemedicine Reimbursement Policy Addenda\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Why Join \n",
      "\n",
      "\n",
      "\n",
      "Our Commitment\n",
      "\n",
      "\n",
      "Join Our Networks\n",
      "\n",
      "\n",
      "Our Healthcare Facilities\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Products & Programs \n",
      "\n",
      "\n",
      "\n",
      "Braven Health℠\n",
      "\n",
      "\n",
      "Horizon Behavioral Health℠\n",
      "\n",
      "\n",
      "Horizon Network Information\n",
      "\n",
      "\n",
      "Our Pledge\n",
      "\n",
      "\n",
      "Patient Care Support\n",
      "\n",
      "\n",
      "Pharmacy\n",
      "\n",
      "\n",
      "Risk Adjustment Overview\n",
      "\n",
      "\n",
      "Utilization Management Programs\n",
      "\n",
      "\n",
      "Value-Based Programs\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Policies & Procedures \n",
      "\n",
      "\n",
      "\n",
      "HEDIS Resources\n",
      "\n",
      "\n",
      "Claim Submission & Billing\n",
      "\n",
      "\n",
      "Demographic Updates\n",
      "\n",
      "\n",
      "Inquiries, Complaints & Appeals\n",
      "\n",
      "\n",
      "Policies\n",
      "\n",
      "\n",
      "Utilization Management\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Resources \n",
      "\n",
      "\n",
      "\n",
      "Patient Quality & Outcome Resources\n",
      "\n",
      "\n",
      "Provider Self-Service Tools\n",
      "\n",
      "\n",
      "Working with Us: Information & Education Resources\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Forms \n",
      "\n",
      "\n",
      "\n",
      "Forms by Plan Type\n",
      "\n",
      "\n",
      "Forms by Specialty Type\n",
      "\n",
      "\n",
      "Forms by Type\n",
      "\n",
      "\n",
      "Frequently Used Forms\n",
      "\n",
      "\n",
      "Miscellaneous\n",
      "\n",
      "\n",
      "W9 Form-Dental\n",
      "\n",
      "\n",
      "W9 Form-Medical\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "News \n",
      "\n",
      "\n",
      "\n",
      "Blue Review\n",
      "\n",
      "\n",
      "Feature Stories\n",
      "\n",
      "\n",
      "News & Legal Notices\n",
      "\n",
      "\n",
      "Provider Alerts\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Breadcrumb\n",
      "\n",
      "\n",
      "Home\n",
      "\n",
      "\n",
      "Providers\n",
      "\n",
      "\n",
      "Policies & Procedures\n",
      "\n",
      "\n",
      "Policies\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "COVID-19\n",
      "Stay informed. Get the latest information on COVID-19.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Important - Please Read \n",
      "\n",
      "By accessing this Medical Policy Manual, you acknowledge receipt and agreement with the information below.\n",
      "The purpose of the Horizon Medical Policy Manual is to provide clinical policies applicable to the administration of health benefits insured or administered by Horizon Blue Cross Blue Shield of New Jersey, Horizon Healthcare of New Jersey, Inc., Horizon Insurance Company, and Healthier New Jersey Insurance Company (collectively “Horizon BCBSNJ”), either directly or through one of their delegated vendors.\n",
      "If the benefits available to a member pursuant to the member’s benefit plan differ from what is stated in a medical policy, the benefit plan prevails.\n",
      "Although a service, supply, or procedure may be medically necessary, it nevertheless may be subject to limitations and/or exclusions under a member’s benefit plan.\n",
      "If a service, treatment, procedure, equipment, device, supply, or drug is not covered, and if the member nevertheless proceeds to obtain any of them, the member may be responsible for the cost to the extent permitted under the member’s benefit plan and applicable law.  For information about their benefits and payment of those benefits, the member should consult their plan document.\n",
      "The policies in this database are not intended to direct the course of clinical care, and they do not replace a physician’s or other health care professional’s independent clinical judgment or duty to exercise special knowledge and skill in the treatment of Horizon BCBSNJ members.  Decisions regarding treatment and treatment plans are the responsibility of physicians and other health care professionals.  Horizon BCBSNJ is not responsible for, does not provide, and does not hold itself out as a provider of medical care.  Physicians and other health care professionals remain responsible for the quality and type of health care services provided to a Horizon BCBSNJ member.\n",
      "Horizon BCBSNJ medical policies do not constitute medical advice, authorization, certification, approval, explanation of benefits, offer of coverage, contract, or a guarantee of payment.  Payment is determined based on all terms, conditions, limitations, and exclusions of the member’s benefit plan, including the medical necessity of the services provided; the member’s eligibility at the time the services are provided; the agreement between the physician or other health care professional and Horizon BCBSNJ, if applicable; and the out-of-network fee schedule for the member’s benefit plan, if applicable.\n",
      "The medical policies are written taking into consideration information from such sources as applicable Federal and State regulations, evidence-based scientific literature, standards of practice, and opinions of community physicians and professional societies.  Although the medical policies are specifically written to address the clinical circumstances of the majority of people, Horizon BCBSNJ will consider an individual’s unique clinical circumstances, as well as the sources noted in the respective medical policies, when making these decisions on a case-by-case basis.\n",
      "Medical policies are highly technical and are designed for use by Horizon BCBSNJ professional staff in making coverage determinations and by physicians and other health care professionals in understanding those decisions.  Members who are provided with a copy of a medical policy should discuss the medical policy with their treating provider and should refer to their specific benefit plan for the terms, conditions, limitations, and exclusions of their coverage.\n",
      "The Horizon BCBSNJ Medical Policy Manual is proprietary.  Horizon BCBSNJ reserves the right to update or change the contents of this Medical Policy Manual without notice, as the law allows.  Horizon BCBSNJ makes benefit determinations based on the medical policies in existence at the time Horizon BCBSNJ receives  a request (e.g., prior authorization or prior determination) or based on the actual date of service on a claim for the service, treatment, procedure, equipment, device, supply, or drug.  Horizon BCBSNJ will not later revise its benefit determinations as the result of any subsequent updates or changes in medical policy.\n",
      "Horizon BCBSNJ has various categories of medical policies.  Policy criteria may also vary by line of business, and notes in the respective policies will indicate the specific guidance.   In addition, it should be noted that, in addition to this Medical Policy Manual, Horizon BCBSNJ’s delegated vendors also maintain libraries of medical policies that have been reviewed and adopted by Horizon BCBSNJ.\n",
      "The Horizon Blue Cross Blue Shield pharmacy policy guidelines are available at\n",
      " https://www.myprime.com/v/HBCBSNJ/COMMERCIAL/NJCLASSIC/en/forms/coverage-determination/prior-authorization.html for Classic formulary;\n",
      " https://www.myprime.com/v/HBCBSNJ/COMMERCIAL/NJIADVANT/en/forms/coverage-determination/prior-authorization.html for Classic with Health Insurance Marketplace formulary;\n",
      "https://www.myprime.com/content/dam/prime/memberportal/forms/2019/FullyQualified/Other/ALL/HBCBSNJ/MEDICARE_D/NJMBNGCPPO/Prior_Authorization_Criteria.pdf for Medicare Part D formulary\n",
      "https://www.myprime.com/content/dam/prime/memberportal/forms/2019/FullyQualified/Other/ALL/HBCBSNJ/MEDICARE_D/NJSNPHMO/Prior_Authorization_Criteria.pdf for Medicare DSNP formulary\n",
      "Horizon NJ Health policies (Medicaid) are available at https://services3.horizon-bcbsnj.com/ddn/NJhealthWeb.nsf\n",
      "Horizon Blue Cross Blue Shield of New Jersey is an independent licensee of the Blue Cross Blue Shield Association.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Are you sure you want to leave this website?\n",
      "\n",
      "\n",
      "You are leaving the Horizon Blue Cross Blue Shield of New Jersey website. Please click Continue to leave this website. Or, if you would like to remain in the current site, click Cancel.\n",
      "\n",
      "CONTINUE  linkCANCEL\n",
      "\n",
      "‌close\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "‌\n",
      "‌\n",
      "‌\n",
      "‌ \n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "This website is operated by Horizon Blue Cross Blue Shield of New Jersey and is not New Jersey’s Health Insurance Marketplace. This website does not display all Qualified Health Plans available through Get Covered NJ. To see all available Qualified Health Plan options, go to the New Jersey Health Insurance Marketplace at Get Covered NJ.\n",
      "Products and services are provided by Horizon Blue Cross Blue Shield of New Jersey, Horizon Insurance Company, Horizon Healthcare of New Jersey, Braven Health, and/or Horizon Healthcare Dental, Inc., each an independent licensee of the Blue Cross Blue Shield Association. Communications may be issued by Horizon Blue Cross Blue Shield of New Jersey in its capacity as administrator of programs and provider relations for all its companies. The Blue Cross® and Blue Shield® name and symbols are registered marks of the Blue Cross Blue Shield Association. The Horizon® name and symbols are registered marks of Horizon Blue Cross Blue Shield of New Jersey. The Braven Health℠ name and symbols are service marks of Braven Health.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Information in Other Languages\n",
      "\n",
      "Español\n",
      "Polski\n",
      "عربي\n",
      "中文 \n",
      "Italiano\n",
      "اردو\n",
      "한국어\n",
      "Русский язык\n",
      "Português\n",
      "Tagalog\n",
      "हिंदी\n",
      "Kreyòl ayisyen\n",
      "ગુજરાતી\n",
      "Français\n",
      "Tiếng Việt\n",
      "\n",
      "\n",
      "\n",
      "Family of Companies\n",
      "\n",
      "Horizon BCBSNJ Dental\n",
      "Horizon Casualty Services\n",
      "Horizon NJ Health\n",
      "\n",
      "\n",
      "\n",
      "Additional Links\n",
      "\n",
      "About Us\n",
      "Privacy Center\n",
      "Careers\n",
      "Find a Doctor\n",
      "Contact Us\n",
      "Claims Payment Policies and Other Information\n",
      "Notice of Non-Discrimination \n",
      "\n",
      "FAQs\n",
      "Horizon BCBSNJ Retirees\n",
      "Interoperability Developer Portal\n",
      "Transparency in Coverage\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "® 2023 Horizon Blue Cross Blue Shield of New Jersey, Three Penn Plaza East, Newark, New Jersey 07105.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      " Applicable Products:Commercial PPO/EPO &Exchange POS/EPO ‌    Applicable Products: Commercial HMO & POS‌    \n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "Please note that these forms are to be used by Federal Employee Program Members only\n",
      "\n",
      "CONTINUE  linkCANCEL\n",
      "\n",
      "‌close\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "****************************\n"
     ]
    }
   ],
   "source": [
    "extract_t('https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/74f5260b-b1d4-7d46-f716-d824691117a6')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 140,
   "id": "e51fe11b",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "title_tag:  Tumor Markers - Medical Clinical Policy Bulletins | Aetna \n",
      "Policy URL: https://www.aetna.com/cpb/medical/data/300_399/0352.html\n",
      "****************************\n"
     ]
    }
   ],
   "source": [
    "extract_title_dates(url)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 151,
   "id": "0ccb2806",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n"
     ]
    }
   ],
   "source": [
    "print(url)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 150,
   "id": "7e40acdc",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "title_tag_pdf_met : 957 AIM Genetic Testing Management Program CPT and HCPCS Codes\n",
      "****************************\n",
      " \n",
      "1 \n",
      " \n",
      " \n",
      "Medical Policy \n",
      "AIM Genetic Testing Management Program CPT and HCPCS Codes \n",
      "Policy Number: 957 \n",
      "BCBSA Reference Number: N/A \n",
      "NCD/LCD:  N/A \n",
      "Effective Date:  January 1, 2019 \n",
      "Related Policies   \n",
      "• \n",
      "AIM Genetic Testing Management Program, #954   \n",
      "• \n",
      "Biomarkers for the Diagnosis and Cancer Risk Assessment of Prostate Cancer, #336 \n",
      "• \n",
      "Cardiovascular Risk Panels, #664 \n",
      "• \n",
      "Preimplantation Genetic Testing, #088 \n",
      " \n",
      "Commercial Products \n",
      "The following CPT and HCPCS codes are in-scope under the AIM Genetic Testing Management Program for \n",
      "Commercial products only.  For medical necessity criteria, see AIM Clinical Appropriateness Guidelines. \n",
      " \n",
      "Medicare Advantage Products \n",
      "For specific genetic testing guidelines for the following CPT and HCPCS codes, please see the appropriate \n",
      "National Coverage Determination (NCD) or Local Coverage Determination (LCD) through the CMS website.  \n",
      "Prior authorization through AIM Specialty Health is not required for Medicare Advantage products.   \n",
      " \n",
      "CPT Codes: \n",
      "CPT codes Code Description  \n",
      "81120 \n",
      "IDH1 (isocitrate dehydrogenase 1 [NADP+], soluble) (eg, glioma), common variants (eg, \n",
      "R132H, R132C) \n",
      "81121 \n",
      "IDH2 (isocitrate dehydrogenase 2 [NADP+], mitochondrial) (eg, glioma), common variants \n",
      "(eg, R140W, R172M) \n",
      "81161 \n",
      "DMD (dystrophin) (eg, Duchenne/Becker muscular dystrophy) deletion analysis, and \n",
      "duplication analysis, if performed \n",
      "81162 \n",
      "BRCA1 (BRCA1, DNA repair associated), BRCA2 (BRCA2, DNA repair associated) (eg, \n",
      "hereditary breast and ovarian cancer) gene analysis; full sequence analysis and full \n",
      "duplication/deletion analysis (ie, detection of large gene rearrangements) \n",
      "81163  \n",
      "BRCA1 (BRCA1, DNA repair associated), BRCA2 (BRCA2, DNA repair associated) (eg, \n",
      "hereditary breast and ovarian cancer) gene analysis; full sequence analysis \n",
      "81164 \n",
      "BRCA1 (BRCA1, DNA repair associated), BRCA2 (BRCA2, DNA repair associated) (eg, \n",
      "hereditary breast and ovarian cancer) gene analysis; full duplication/deletion analysis (ie, \n",
      "detection of large gene rearrangements) \n",
      "81165 \n",
      "BRCA1 (BRCA1, DNA repair associated) (eg, hereditary breast and ovarian cancer) gene \n",
      "analysis; full sequence analysis \n",
      "81166 \n",
      "BRCA1 (BRCA1, DNA repair associated) (eg, hereditary breast and ovarian cancer) gene \n",
      "analysis; full duplication/deletion analysis (ie, detection of large gene rearrangements) \n",
      "81167 \n",
      "BRCA2 (BRCA2, DNA repair associated) (eg, hereditary breast and ovarian cancer) gene \n",
      "analysis; full duplication/deletion analysis (ie, detection of large gene rearrangements) \n",
      " \n",
      "2 \n",
      " \n",
      "81168 \n",
      "CCND1/IGH (t(11;14)) (eg, mantle cell lymphoma) translocation analysis, major breakpoint, \n",
      "qualitative and quantitative, if performed \n",
      "81170 \n",
      "ABL1 (ABL proto-oncogene 1, non-receptor tyrosine kinase) (eg, acquired imatinib tyrosine \n",
      "kinase inhibitor resistance), gene analysis, variants in the kinase domain \n",
      "81171 \n",
      "AFF2 (AF4/FMR2 family, member 2 [FMR2]) (eg, fragile X mental retardation 2 [FRAXE]) \n",
      "gene analysis; evaluation to detect abnormal (eg, expanded) alleles \n",
      "81172 \n",
      "AFF2 (AF4/FMR2 family, member 2 [FMR2]) (eg, fragile X mental retardation 2 [FRAXE]) \n",
      "gene analysis; characterization of alleles (eg, expanded size and methylation status) \n",
      "81173 \n",
      "AR (androgen receptor) (eg, spinal and bulbar muscular atrophy, Kennedy disease, X \n",
      "chromosome inactivation) gene analysis; full gene sequence \n",
      "81174 \n",
      "AR (androgen receptor) (eg, spinal and bulbar muscular atrophy, Kennedy disease, X \n",
      "chromosome inactivation) gene analysis; known familial variant \n",
      "81175 \n",
      "ASXL1 (additional sex combs like 1, transcriptional regulator) (eg, myelodysplastic syndrome, \n",
      "myeloproliferative neoplasms, chronic myelomonocytic leukemia), gene analysis; full gene \n",
      "sequence \n",
      "81176 \n",
      "ASXL1 (additional sex combs like 1, transcriptional regulator) (eg, myelodysplastic syndrome, \n",
      "myeloproliferative neoplasms, chronic myelomonocytic leukemia), gene analysis; targeted \n",
      "sequence analysis (eg, exon 12) \n",
      "81177 \n",
      "ATN1 (atrophin 1) (eg, dentatorubral-pallidoluysian atrophy) gene analysis, evaluation to \n",
      "detect abnormal (eg, expanded) alleles \n",
      "81178 \n",
      "ATXN1 (ataxin 1) (eg, spinocerebellar ataxia) gene analysis, evaluation to detect abnormal \n",
      "(eg, expanded) alleles \n",
      "81179 \n",
      "ATXN2 (ataxin 2) (eg, spinocerebellar ataxia) gene analysis, evaluation to detect abnormal \n",
      "(eg, expanded) alleles \n",
      "81180 \n",
      "ATXN3 (ataxin 3) (eg, spinocerebellar ataxia, Machado-Joseph disease) gene analysis, \n",
      "evaluation to detect abnormal (eg, expanded) alleles \n",
      "81181 \n",
      "ATXN7 (ataxin 7) (eg, spinocerebellar ataxia) gene analysis, evaluation to detect abnormal \n",
      "(eg, expanded) alleles \n",
      "81182 \n",
      "ATXN8OS (ATXN8 opposite strand [non-protein coding]) (eg, spinocerebellar ataxia) gene \n",
      "analysis, evaluation to detect abnormal (eg, expanded) alleles \n",
      "81183 \n",
      "ATXN10 (ataxin 10) (eg, spinocerebellar ataxia) gene analysis, evaluation to detect abnormal \n",
      "(eg, expanded) alleles \n",
      "81184 \n",
      "CACNA1A (calcium voltage-gated channel subunit alpha1 A) (eg, spinocerebellar ataxia) \n",
      "gene analysis; evaluation to detect abnormal (eg, expanded) alleles \n",
      "81185 \n",
      "CACNA1A (calcium voltage-gated channel subunit alpha1 A) (eg, spinocerebellar ataxia) \n",
      "gene analysis; full gene sequence \n",
      "81186 \n",
      "CACNA1A (calcium voltage-gated channel subunit alpha1 A) (eg, spinocerebellar ataxia) \n",
      "gene analysis; known familial variant \n",
      "81187 \n",
      "CNBP (CCHC-type zinc finger nucleic acid binding protein) (eg, myotonic dystrophy type 2) \n",
      "gene analysis, evaluation to detect abnormal (eg, expanded) alleles \n",
      "81188 \n",
      "CSTB (cystatin B) (eg, Unverricht-Lundborg disease) gene analysis; evaluation to detect \n",
      "abnormal (eg, expanded) alleles \n",
      "81189 \n",
      "CSTB (cystatin B) (eg, Unverricht-Lundborg disease) gene analysis; full gene sequence \n",
      "81190 \n",
      "CSTB (cystatin B) (eg, Unverricht-Lundborg disease) gene analysis; known familial variant(s) \n",
      "81191 \n",
      "NTRK1 (neurotrophic receptor tyrosine kinase 1) (eg, solid tumors) translocation analysis \n",
      "81192 \n",
      "NTRK2 (neurotrophic receptor tyrosine kinase 2) (eg, solid tumors) translocation analysis \n",
      "81193 \n",
      "NTRK3 (neurotrophic receptor tyrosine kinase 3) (eg, solid tumors) translocation analysis \n",
      "81194 \n",
      "NTRK (neurotrophic-tropomyosin receptor tyrosine kinase 1, 2, and 3) (eg, solid tumors) \n",
      "translocation analysis \n",
      "81200 \n",
      "ASPA (aspartoacylase) (eg, Canavan disease) gene analysis, common variants (eg, E285A, \n",
      "Y231X) \n",
      "81201 \n",
      "APC (adenomatous polyposis coli) (eg, familial adenomatosis polyposis [FAP], attenuated \n",
      "FAP) gene analysis; full gene sequence \n",
      "81202 \n",
      "APC (adenomatous polyposis coli) (eg, familial adenomatosis polyposis [FAP], attenuated \n",
      "FAP) gene analysis; known familial variants \n",
      "81203 \n",
      "APC (adenomatous polyposis coli) (eg, familial adenomatosis polyposis [FAP], attenuated \n",
      "FAP) gene analysis; duplication/deletion variants \n",
      " \n",
      "3 \n",
      " \n",
      "81204 \n",
      "AR (androgen receptor) (eg, spinal and bulbar muscular atrophy, Kennedy disease, X \n",
      "chromosome inactivation) gene analysis; characterization of alleles (eg, expanded size or \n",
      "methylation status) \n",
      "81205 \n",
      "BCKDHB (branched-chain keto acid dehydrogenase E1, beta polypeptide) (eg, maple syrup \n",
      "urine disease) gene analysis, common variants (eg, R183P, G278S, E422X) \n",
      "81206 \n",
      "BCR/ABL1 (t(9;22)) (eg, chronic myelogenous leukemia) translocation analysis; major \n",
      "breakpoint, qualitative or quantitative \n",
      "81207 \n",
      "BCR/ABL1 (t(9;22)) (eg, chronic myelogenous leukemia) translocation analysis; minor \n",
      "breakpoint, qualitative or quantitative \n",
      "81208 \n",
      "BCR/ABL1 (t(9;22)) (eg, chronic myelogenous leukemia) translocation analysis; other \n",
      "breakpoint, qualitative or quantitative \n",
      "81209 \n",
      "BLM (Bloom syndrome, RecQ helicase-like) (eg, Bloom syndrome) gene analysis, \n",
      "2281del6ins7 variant \n",
      "81210 \n",
      "BRAF (B-Raf proto-oncogene, serine/threonine kinase) (eg, colon cancer, melanoma), gene \n",
      "analysis, V600 variant(s) \n",
      "81212 \n",
      "BRCA1 (BRCA1, DNA repair associated), BRCA2 (BRCA2, DNA repair associated) (eg, \n",
      "hereditary breast and ovarian cancer) gene analysis; 185delAG, 5385insC, 6174delT variants \n",
      "81215 \n",
      "BRCA1 (BRCA1, DNA repair associated) (eg, hereditary breast and ovarian cancer) gene \n",
      "analysis; known familial variant \n",
      "81216 \n",
      "BRCA2 (BRCA2, DNA repair associated) (eg, hereditary breast and ovarian cancer) gene \n",
      "analysis; full sequence analysis \n",
      "81217 \n",
      "BRCA2 (BRCA2, DNA repair associated) (eg, hereditary breast and ovarian cancer) gene \n",
      "analysis; known familial variant \n",
      "81218 \n",
      "CEBPA (CCAAT/enhancer binding protein [C/EBP], alpha) (eg, acute myeloid leukemia), \n",
      "gene analysis, full gene sequence \n",
      "81219 \n",
      "CALR (calreticulin) (eg, myeloproliferative disorders), gene analysis, common variants in \n",
      "exon 9 \n",
      "81220 \n",
      "CFTR (cystic fibrosis transmembrane conductance regulator) (eg, cystic fibrosis) gene \n",
      "analysis; common variants (eg, ACMG/ACOG guidelines) \n",
      "81221 \n",
      "CFTR (cystic fibrosis transmembrane conductance regulator) (eg, cystic fibrosis) gene \n",
      "analysis; known familial variants \n",
      "81222 \n",
      "CFTR (cystic fibrosis transmembrane conductance regulator) (eg, cystic fibrosis) gene \n",
      "analysis; duplication/deletion variants \n",
      "81223 \n",
      "CFTR (cystic fibrosis transmembrane conductance regulator) (eg, cystic fibrosis) gene \n",
      "analysis; full gene sequence \n",
      "81224 \n",
      "CFTR (cystic fibrosis transmembrane conductance regulator) (eg, cystic fibrosis) gene \n",
      "analysis; intron 8 poly-T analysis (eg, male infertility) \n",
      "81225 \n",
      "CYP2C19 (cytochrome P450, family 2, subfamily C, polypeptide 19) (eg, drug metabolism), \n",
      "gene analysis, common variants (eg, *2, *3, *4, *8, *17) \n",
      "81226 \n",
      "CYP2D6 (cytochrome P450, family 2, subfamily D, polypeptide 6) (eg, drug metabolism), \n",
      "gene analysis, common variants (eg, *2, *3, *4, *5, *6, *9, *10, *17, *19, *29, *35, *41, *1XN, \n",
      "*2XN, *4XN) \n",
      "81227 \n",
      "CYP2C9 (cytochrome P450, family 2, subfamily C, polypeptide 9) (eg, drug metabolism), \n",
      "gene analysis, common variants (eg, *2, *3, *5, *6) \n",
      "81228 \n",
      "Cytogenomic constitutional (genome-wide) microarray analysis; interrogation of genomic \n",
      "regions for copy number variants (eg, bacterial artificial chromosome [BAC] or oligo-based \n",
      "comparative genomic hybridization [CGH] microarray analysis) \n",
      "81229 \n",
      "Cytogenomic constitutional (genome-wide) microarray analysis; interrogation of genomic \n",
      "regions for copy number and single nucleotide polymorphism (SNP) variants for \n",
      "chromosomal abnormalities \n",
      "81230 \n",
      "CYP3A4 (cytochrome P450 family 3 subfamily A member 4) (eg, drug metabolism), gene \n",
      "analysis, common variant(s) (eg, *2, *22) \n",
      "81231 \n",
      "CYP3A5 (cytochrome P450 family 3 subfamily A member 5) (eg, drug metabolism), gene \n",
      "analysis, common variants (eg, *2, *3, *4, *5, *6, *7) \n",
      "81232 \n",
      "DPYD (dihydropyrimidine dehydrogenase) (eg, 5-fluorouracil/5-FU and capecitabine drug \n",
      "metabolism), gene analysis, common variant(s) (eg, *2A, *4, *5, *6) \n",
      " \n",
      "4 \n",
      " \n",
      "81233 \n",
      "BTK (Bruton's tyrosine kinase) (eg, chronic lymphocytic leukemia) gene analysis, common \n",
      "variants (eg, C481S, C481R, C481F) \n",
      "81234 \n",
      "DMPK (DM1 protein kinase) (eg, myotonic dystrophy type 1) gene analysis; evaluation to \n",
      "detect abnormal (expanded) alleles \n",
      "81235 \n",
      "EGFR (epidermal growth factor receptor) (eg, non-small cell lung cancer) gene analysis, \n",
      "common variants (eg, exon 19 LREA deletion, L858R, T790M, G719A, G719S, L861Q) \n",
      "81236 \n",
      "EZH2 (enhancer of zeste 2 polycomb repressive complex 2 subunit) (eg, myelodysplastic \n",
      "syndrome, myeloproliferative neoplasms) gene analysis, full gene sequence \n",
      "81237 \n",
      "EZH2 (enhancer of zeste 2 polycomb repressive complex 2 subunit) (eg, diffuse large B-cell \n",
      "lymphoma) gene analysis, common variant(s) (eg, codon 646) \n",
      "81238 \n",
      "F9 (coagulation factor IX) (eg, hemophilia B), full gene sequence \n",
      "81239 \n",
      "DMPK (DM1 protein kinase) (eg, myotonic dystrophy type 1) gene analysis; characterization \n",
      "of alleles (eg, expanded size) \n",
      "81240 \n",
      "F2 (prothrombin, coagulation factor II) (eg, hereditary hypercoagulability) gene analysis, \n",
      "20210G>A variant \n",
      "81241 \n",
      "F5 (coagulation factor V) (eg, hereditary hypercoagulability) gene analysis, Leiden variant \n",
      "81242 \n",
      "FANCC (Fanconi anemia, complementation group C) (eg, Fanconi anemia, type C) gene \n",
      "analysis, common variant (eg, IVS4+4A>T) \n",
      "81243 \n",
      "FMR1 (fragile X mental retardation 1) (eg, fragile X mental retardation) gene analysis; \n",
      "evaluation to detect abnormal (eg, expanded) alleles \n",
      "81244 \n",
      "FMR1 (fragile X mental retardation 1) (eg, fragile X mental retardation) gene analysis; \n",
      "characterization of alleles (eg, expanded size and promoter methylation status) \n",
      "81245 \n",
      "FLT3 (fms-related tyrosine kinase 3) (eg, acute myeloid leukemia), gene analysis; internal \n",
      "tandem duplication (ITD) variants (ie, exons 14, 15) \n",
      "81246 \n",
      "FLT3 (fms-related tyrosine kinase 3) (eg, acute myeloid leukemia), gene analysis; tyrosine \n",
      "kinase domain (TKD) variants (eg, D835, I836) \n",
      "81247 \n",
      "G6PD (glucose-6-phosphate dehydrogenase) (eg, hemolytic anemia, jaundice), gene \n",
      "analysis; common variant(s) (eg, A, A-) \n",
      "81248 \n",
      "G6PD (glucose-6-phosphate dehydrogenase) (eg, hemolytic anemia, jaundice), gene \n",
      "analysis; known familial variant(s) \n",
      "81249 \n",
      "G6PD (glucose-6-phosphate dehydrogenase) (eg, hemolytic anemia, jaundice), gene \n",
      "analysis; full gene sequence \n",
      "81250 \n",
      "G6PC (glucose-6-phosphatase, catalytic subunit) (eg, Glycogen storage disease, type 1a, \n",
      "von Gierke disease) gene analysis, common variants (eg, R83C, Q347X) \n",
      "81251 \n",
      "GBA (glucosidase, beta, acid) (eg, Gaucher disease) gene analysis, common variants (eg, \n",
      "N370S, 84GG, L444P, IVS2+1G>A) \n",
      "81252 \n",
      "GJB2 (gap junction protein, beta 2, 26kDa, connexin 26) (eg, nonsyndromic hearing loss) \n",
      "gene analysis; full gene sequence \n",
      "81253 \n",
      "GJB2 (gap junction protein, beta 2, 26kDa, connexin 26) (eg, nonsyndromic hearing loss) \n",
      "gene analysis; known familial variants \n",
      "81254 \n",
      "GJB6 (gap junction protein, beta 6, 30kDa, connexin 30) (eg, nonsyndromic hearing loss) \n",
      "gene analysis, common variants (eg, 309kb [del(GJB6-D13S1830)] and 232kb [del(GJB6-\n",
      "D13S1854)]) \n",
      "81255 \n",
      "HEXA (hexosaminidase A [alpha polypeptide]) (eg, Tay-Sachs disease) gene analysis, \n",
      "common variants (eg, 1278insTATC, 1421+1G>C, G269S) \n",
      "81256 \n",
      "HFE (hemochromatosis) (eg, hereditary hemochromatosis) gene analysis, common variants \n",
      "(eg, C282Y, H63D) \n",
      "81257 \n",
      "HBA1/HBA2 (alpha globin 1 and alpha globin 2) (eg, alpha thalassemia, Hb Bart hydrops \n",
      "fetalis syndrome, HbH disease), gene analysis; common deletions or variant (eg, Southeast \n",
      "Asian, Thai, Filipino, Mediterranean, alpha3.7, alpha4.2, alpha20.5, Constant Spring) \n",
      "81258 \n",
      "HBA1/HBA2 (alpha globin 1 and alpha globin 2) (eg, alpha thalassemia, Hb Bart hydrops \n",
      "fetalis syndrome, HbH disease), gene analysis; known familial variant \n",
      "81259 \n",
      "HBA1/HBA2 (alpha globin 1 and alpha globin 2) (eg, alpha thalassemia, Hb Bart hydrops \n",
      "fetalis syndrome, HbH disease), gene analysis; full gene sequence \n",
      "81260 \n",
      "IKBKAP (inhibitor of kappa light polypeptide gene enhancer in B-cells, kinase complex-\n",
      "associated protein) (eg, familial dysautonomia) gene analysis, common variants (eg, \n",
      "2507+6T>C, R696P) \n",
      " \n",
      "5 \n",
      " \n",
      "81261 \n",
      "IGH@ (Immunoglobulin heavy chain locus) (eg, leukemias and lymphomas, B-cell), gene \n",
      "rearrangement analysis to detect abnormal clonal population(s); amplified methodology (eg, \n",
      "polymerase chain reaction) \n",
      "81262 \n",
      "IGH@ (Immunoglobulin heavy chain locus) (eg, leukemias and lymphomas, B-cell), gene \n",
      "rearrangement analysis to detect abnormal clonal population(s); direct probe methodology \n",
      "(eg, Southern blot) \n",
      "81263 \n",
      "IGH@ (Immunoglobulin heavy chain locus) (eg, leukemia and lymphoma, B-cell), variable \n",
      "region somatic mutation analysis \n",
      "81264 \n",
      "IGK@ (Immunoglobulin kappa light chain locus) (eg, leukemia and lymphoma, B-cell), gene \n",
      "rearrangement analysis, evaluation to detect abnormal clonal population(s) \n",
      "81265 \n",
      "Comparative analysis using Short Tandem Repeat (STR) markers; patient and comparative \n",
      "specimen (eg, pre-transplant recipient and donor germline testing, post-transplant non-\n",
      "hematopoietic recipient germline [eg, buccal swab or other germline tissue sample] \n",
      "81266 \n",
      "Comparative analysis using Short Tandem Repeat (STR) markers; each additional specimen \n",
      "(eg, additional cord blood donor, additional fetal samples from different cultures, or additional \n",
      "zygosity in multiple birth pregnancies) (List separately in addition to code for primary \n",
      "procedure) \n",
      "81269 \n",
      "HBA1/HBA2 (alpha globin 1 and alpha globin 2) (eg, alpha thalassemia, Hb Bart hydrops \n",
      "fetalis syndrome, HbH disease), gene analysis; duplication/deletion variants \n",
      "81270 \n",
      "JAK2 (Janus kinase 2) (eg, myeloproliferative disorder) gene analysis, p.Val617Phe (V617F) \n",
      "variant \n",
      "81271 \n",
      "HTT (huntingtin) (eg, Huntington disease) gene analysis; evaluation to detect abnormal (eg, \n",
      "expanded) alleles \n",
      "81272 \n",
      "KIT (v-kit Hardy-Zuckerman 4 feline sarcoma viral oncogene homolog) (eg, gastrointestinal \n",
      "stromal tumor [GIST], acute myeloid leukemia, melanoma), gene analysis, targeted sequence \n",
      "analysis (eg, exons 8, 11, 13, 17, 18) \n",
      "81273 \n",
      "KIT (v-kit Hardy-Zuckerman 4 feline sarcoma viral oncogene homolog) (eg, mastocytosis), \n",
      "gene analysis, D816 variant(s) \n",
      "81274 \n",
      "HTT (huntingtin) (eg, Huntington disease) gene analysis; characterization of alleles (eg, \n",
      "expanded size) \n",
      "81275 \n",
      "KRAS (Kirsten rat sarcoma viral oncogene homolog) (eg, carcinoma) gene analysis; variants \n",
      "in exon 2 (eg, codons 12 and 13) \n",
      "81276 \n",
      "KRAS (Kirsten rat sarcoma viral oncogene homolog) (eg, carcinoma) gene analysis; \n",
      "additional variant(s) (eg, codon 61, codon 146) \n",
      "81277 \n",
      "Cytogenomic neoplasia (genome-wide) microarray analysis, interrogation of genomic regions \n",
      "for copy number and loss-of-heterozygosity variants for chromosomal abnormalities \n",
      "81278 \n",
      "IGH@/BCL2 (t(14;18)) (eg, follicular lymphoma) translocation analysis, major breakpoint \n",
      "region (MBR) and minor cluster region (mcr) breakpoints, qualitative or quantitative \n",
      "81279 \n",
      "JAK2 (Janus kinase 2) (eg, myeloproliferative disorder) targeted sequence analysis (eg, \n",
      "exons 12 and 13) \n",
      "81283 \n",
      "IFNL3 (interferon, lambda 3) (eg, drug response), gene analysis, rs12979860 variant \n",
      "81284 \n",
      "FXN (frataxin) (eg, Friedreich ataxia) gene analysis; evaluation to detect abnormal \n",
      "(expanded) alleles \n",
      "81285 \n",
      "FXN (frataxin) (eg, Friedreich ataxia) gene analysis; characterization of alleles (eg, expanded \n",
      "size) \n",
      "81286 \n",
      "FXN (frataxin) (eg, Friedreich ataxia) gene analysis; full gene sequence \n",
      "81287 \n",
      "MGMT (O-6-methylguanine-DNA methyltransferase) (eg, glioblastoma multiforme) promoter \n",
      "methylation analysis \n",
      "81288 \n",
      "MLH1 (mutL homolog 1, colon cancer, nonpolyposis type 2) (eg, hereditary non-polyposis \n",
      "colorectal cancer, Lynch syndrome) gene analysis; promoter methylation analysis \n",
      "81289 \n",
      "FXN (frataxin) (eg, Friedreich ataxia) gene analysis; known familial variant(s) \n",
      "81290 \n",
      "MCOLN1 (mucolipin 1) (eg, Mucolipidosis, type IV) gene analysis, common variants (eg, \n",
      "IVS3-2A>G, del6.4kb) \n",
      "81291 \n",
      "MTHFR (5,10-methylenetetrahydrofolate reductase) (eg, hereditary hypercoagulability) gene \n",
      "analysis, common variants (eg, 677T, 1298C) \n",
      "81292 \n",
      "MLH1 (mutL homolog 1, colon cancer, nonpolyposis type 2) (eg, hereditary non-polyposis \n",
      "colorectal cancer, Lynch syndrome) gene analysis; full sequence analysis \n",
      " \n",
      "6 \n",
      " \n",
      "81293 \n",
      "MLH1 (mutL homolog 1, colon cancer, nonpolyposis type 2) (eg, hereditary non-polyposis \n",
      "colorectal cancer, Lynch syndrome) gene analysis; known familial variants \n",
      "81294 \n",
      "MLH1 (mutL homolog 1, colon cancer, nonpolyposis type 2) (eg, hereditary non-polyposis \n",
      "colorectal cancer, Lynch syndrome) gene analysis; duplication/deletion variants \n",
      "81295 \n",
      "MSH2 (mutS homolog 2, colon cancer, nonpolyposis type 1) (eg, hereditary non-polyposis \n",
      "colorectal cancer, Lynch syndrome) gene analysis; full sequence analysis \n",
      "81296 \n",
      "MSH2 (mutS homolog 2, colon cancer, nonpolyposis type 1) (eg, hereditary non-polyposis \n",
      "colorectal cancer, Lynch syndrome) gene analysis; known familial variants \n",
      "81297 \n",
      "MSH2 (mutS homolog 2, colon cancer, nonpolyposis type 1) (eg, hereditary non-polyposis \n",
      "colorectal cancer, Lynch syndrome) gene analysis; duplication/deletion variants \n",
      "81298 \n",
      "MSH6 (mutS homolog 6 [E. coli]) (eg, hereditary non-polyposis colorectal cancer, Lynch \n",
      "syndrome) gene analysis; full sequence analysis \n",
      "81299 \n",
      "MSH6 (mutS homolog 6 [E. coli]) (eg, hereditary non-polyposis colorectal cancer, Lynch \n",
      "syndrome) gene analysis; known familial variants \n",
      "81300 \n",
      "MSH6 (mutS homolog 6 [E. coli]) (eg, hereditary non-polyposis colorectal cancer, Lynch \n",
      "syndrome) gene analysis; duplication/deletion variants \n",
      "81301 \n",
      "Microsatellite instability analysis (eg, hereditary non-polyposis colorectal cancer, Lynch \n",
      "syndrome) of markers for mismatch repair deficiency (eg, BAT25, BAT26), includes \n",
      "comparison of neoplastic and normal tissue, if performed \n",
      "81302 \n",
      "MECP2 (methyl CpG binding protein 2) (eg, Rett syndrome) gene analysis; full sequence \n",
      "analysis \n",
      "81303 \n",
      "MECP2 (methyl CpG binding protein 2) (eg, Rett syndrome) gene analysis; known familial \n",
      "variant \n",
      "81304 \n",
      "MECP2 (methyl CpG binding protein 2) (eg, Rett syndrome) gene analysis; \n",
      "duplication/deletion variants \n",
      "81305 \n",
      "MYD88 (myeloid differentiation primary response 88) (eg, Waldenstrom's macroglobulinemia, \n",
      "lymphoplasmacytic leukemia) gene analysis, p.Leu265Pro (L265P) variant \n",
      "81306 \n",
      "NUDT15 (nudix hydrolase 15) (eg, drug metabolism) gene analysis, common variant(s) (eg, \n",
      "*2, *3, *4, *5, *6) \n",
      "81307 \n",
      "PALB2 (partner and localizer of BRCA2) (eg, breast and pancreatic cancer) gene analysis; \n",
      "full gene sequence \n",
      "81308 \n",
      "PALB2 (partner and localizer of BRCA2) (eg, breast and pancreatic cancer) gene analysis; \n",
      "known familial variant \n",
      "81309 \n",
      "PIK3CA (phosphatidylinositol-4, 5-biphosphate 3-kinase, catalytic subunit alpha) (eg, \n",
      "colorectal and breast cancer) gene analysis, targeted sequence analysis (eg, exons 7, 9, 20) \n",
      "81310 \n",
      "NPM1 (nucleophosmin) (eg, acute myeloid leukemia) gene analysis, exon 12 variants \n",
      "81311 \n",
      "NRAS (neuroblastoma RAS viral [v-ras] oncogene homolog) (eg, colorectal carcinoma), gene \n",
      "analysis, variants in exon 2 (eg, codons 12 and 13) and exon 3 (eg, codon 61) \n",
      "81312 \n",
      "PABPN1 (poly[A] binding protein nuclear 1) (eg, oculopharyngeal muscular dystrophy) gene \n",
      "analysis, evaluation to detect abnormal (eg, expanded) alleles \n",
      "81313 \n",
      "PCA3/KLK3 (prostate cancer antigen 3 [non-protein coding]/kallikrein-related peptidase 3 \n",
      "[prostate specific antigen]) ratio (eg, prostate cancer) \n",
      "81314 \n",
      "PDGFRA (platelet-derived growth factor receptor, alpha polypeptide) (eg, gastrointestinal \n",
      "stromal tumor [GIST]), gene analysis, targeted sequence analysis (eg, exons 12, 18) \n",
      "81315 \n",
      "PML/RARalpha, (t(15;17)), (promyelocytic leukemia/retinoic acid receptor alpha) (eg, \n",
      "promyelocytic leukemia) translocation analysis; common breakpoints (eg, intron 3 and intron \n",
      "6), qualitative or quantitative \n",
      "81316 \n",
      "PML/RARalpha, (t(15;17)), (promyelocytic leukemia/retinoic acid receptor alpha) (eg, \n",
      "promyelocytic leukemia) translocation analysis; single breakpoint (eg, intron 3, intron 6 or \n",
      "exon 6), qualitative or quantitative \n",
      "81317 \n",
      "PMS2 (postmeiotic segregation increased 2 [S. cerevisiae]) (eg, hereditary non-polyposis \n",
      "colorectal cancer, Lynch syndrome) gene analysis; full sequence analysis \n",
      "81318 \n",
      "PMS2 (postmeiotic segregation increased 2 [S. cerevisiae]) (eg, hereditary non-polyposis \n",
      "colorectal cancer, Lynch syndrome) gene analysis; known familial variants \n",
      "81319 \n",
      "PMS2 (postmeiotic segregation increased 2 [S. cerevisiae]) (eg, hereditary non-polyposis \n",
      "colorectal cancer, Lynch syndrome) gene analysis; duplication/deletion variants \n",
      " \n",
      "7 \n",
      " \n",
      "81320 \n",
      "PLCG2 (phospholipase C gamma 2) (eg, chronic lymphocytic leukemia) gene analysis, \n",
      "common variants (eg, R665W, S707F, L845F \n",
      "81321 \n",
      "PTEN (phosphatase and tensin homolog) (eg, Cowden syndrome, PTEN hamartoma tumor \n",
      "syndrome) gene analysis; full sequence analysis \n",
      "81322 \n",
      "PTEN (phosphatase and tensin homolog) (eg, Cowden syndrome, PTEN hamartoma tumor \n",
      "syndrome) gene analysis; known familial variant \n",
      "81323 \n",
      "PTEN (phosphatase and tensin homolog) (eg, Cowden syndrome, PTEN hamartoma tumor \n",
      "syndrome) gene analysis; duplication/deletion variant \n",
      "81324 \n",
      "PMP22 (peripheral myelin protein 22) (eg, Charcot-Marie-Tooth, hereditary neuropathy with \n",
      "liability to pressure palsies) gene analysis; duplication/deletion analysis \n",
      "81325 \n",
      "PMP22 (peripheral myelin protein 22) (eg, Charcot-Marie-Tooth, hereditary neuropathy with \n",
      "liability to pressure palsies) gene analysis; full sequence analysis \n",
      "81326 \n",
      "PMP22 (peripheral myelin protein 22) (eg, Charcot-Marie-Tooth, hereditary neuropathy with \n",
      "liability to pressure palsies) gene analysis; known familial variant \n",
      "81327 \n",
      "SEPT9 (Septin9) (eg, colorectal cancer) promoter methylation analysis \n",
      "81328 \n",
      "SLCO1B1 (solute carrier organic anion transporter family, member 1B1) (eg, adverse drug \n",
      "reaction), gene analysis, common variant(s) (eg, *5) \n",
      "81329 \n",
      "SMN1 (survival of motor neuron 1, telomeric) (eg, spinal muscular atrophy) gene analysis; \n",
      "dosage/deletion analysis (eg, carrier testing), includes SMN2 (survival of motor neuron 2, \n",
      "centromeric) analysis, if performed \n",
      "81330 \n",
      "SMPD1(sphingomyelin phosphodiesterase 1, acid lysosomal) (eg, Niemann-Pick disease, \n",
      "Type A) gene analysis, common variants (eg, R496L, L302P, fsP330) \n",
      "81331 \n",
      "SNRPN/UBE3A (small nuclear ribonucleoprotein polypeptide N and ubiquitin protein ligase \n",
      "E3A) (eg, Prader-Willi syndrome and/or Angelman syndrome), methylation analysis \n",
      "81332 \n",
      "SERPINA1 (serpin peptidase inhibitor, clade A, alpha-1 antiproteinase, antitrypsin, member \n",
      "1) (eg, alpha-1-antitrypsin deficiency), gene analysis, common variants (eg, *S and *Z) \n",
      "81333 \n",
      "TGFBI (transforming growth factor beta-induced) (eg, corneal dystrophy) gene analysis, \n",
      "common variants (eg, R124H, R124C, R124L, R555W, R555Q) \n",
      "81334 \n",
      "RUNX1 (runt related transcription factor 1) (eg, acute myeloid leukemia, familial platelet \n",
      "disorder with associated myeloid malignancy), gene analysis, targeted sequence analysis \n",
      "(eg, exons 3-8) \n",
      "81335 \n",
      "TPMT (thiopurine S-methyltransferase) (eg, drug metabolism), gene analysis, common \n",
      "variants (eg, *2, *3) \n",
      "81336 \n",
      "SMN1 (survival of motor neuron 1, telomeric) (eg, spinal muscular atrophy) gene analysis; full \n",
      "gene sequence \n",
      "81337 \n",
      "SMN1 (survival of motor neuron 1, telomeric) (eg, spinal muscular atrophy) gene analysis; \n",
      "known familial sequence variant(s) \n",
      "81338 \n",
      "MPL (MPL proto-oncogene, thrombopoietin receptor) (eg, myeloproliferative disorder) gene \n",
      "analysis; common variants (eg, W515A, W515K, W515L, W515R) \n",
      "81339 \n",
      "MPL (MPL proto-oncogene, thrombopoietin receptor) (eg, myeloproliferative disorder) gene \n",
      "analysis; sequence analysis, exon 10 \n",
      "81340 \n",
      "TRB@ (T cell antigen receptor, beta) (eg, leukemia and lymphoma), gene rearrangement \n",
      "analysis to detect abnormal clonal population(s); using amplification methodology (eg, \n",
      "polymerase chain reaction) \n",
      "81341 \n",
      "TRB@ (T cell antigen receptor, beta) (eg, leukemia and lymphoma), gene rearrangement \n",
      "analysis to detect abnormal clonal population(s); using direct probe methodology (eg, \n",
      "Southern blot) \n",
      "81342 \n",
      "TRG@ (T cell antigen receptor, gamma) (eg, leukemia and lymphoma), gene rearrangement \n",
      "analysis, evaluation to detect abnormal clonal population(s) \n",
      "81343 \n",
      "PPP2R2B (protein phosphatase 2 regulatory subunit Bbeta) (eg, spinocerebellar ataxia) gene \n",
      "analysis, evaluation to detect abnormal (eg, expanded) alleles \n",
      "81344 \n",
      "TBP (TATA box binding protein) (eg, spinocerebellar ataxia) gene analysis, evaluation to \n",
      "detect abnormal (eg, expanded) alleles \n",
      "81345 \n",
      "TERT (telomerase reverse transcriptase) (eg, thyroid carcinoma, glioblastoma multiforme) \n",
      "gene analysis, targeted sequence analysis (eg, promoter region) \n",
      "81346 \n",
      "TYMS (thymidylate synthetase) (eg, 5-fluorouracil/5-FU drug metabolism), gene analysis, \n",
      "common variant(s) (eg, tandem repeat variant) \n",
      " \n",
      "8 \n",
      " \n",
      "81347 \n",
      "SF3B1 (splicing factor [3b] subunit B1) (eg, myelodysplastic syndrome/acute myeloid \n",
      "leukemia) gene analysis, common variants (eg, A672T, E622D, L833F, R625C, R625L) \n",
      "81348 \n",
      "SRSF2 (serine and arginine-rich splicing factor 2) (eg, myelodysplastic syndrome, acute \n",
      "myeloid leukemia) gene analysis, common variants (eg, P95H, P95L) \n",
      "81349 \n",
      "Cytogenomic (genome-wide) analysis for constitutional chromosomal abnormalities; \n",
      "interrogation of genomic regions for copy number and loss-of-heterozygosity variants, low-\n",
      "pass sequencing analysis - Cytogenomic Constitutional Analysis Low Pass Sequencing \n",
      "81350 \n",
      "UGT1A1 (UDP glucuronosyltransferase 1 family, polypeptide A1) (eg, irinotecan metabolism), \n",
      "gene analysis, common variants (eg, *28, *36, *37) \n",
      "81351 \n",
      "TP53 (tumor protein 53) (eg, Li-Fraumeni syndrome) gene analysis; full gene sequence \n",
      "81352 \n",
      "TP53 (tumor protein 53) (eg, Li-Fraumeni syndrome) gene analysis; targeted sequence \n",
      "analysis (eg, 4 oncology) \n",
      "81353 \n",
      "TP53 (tumor protein 53) (eg, Li-Fraumeni syndrome) gene analysis; known familial variant \n",
      "81355 \n",
      "VKORC1 (vitamin K epoxide reductase complex, subunit 1) (eg, warfarin metabolism), gene \n",
      "analysis, common variant(s) (eg, -1639G>A, c.173+1000C>T) \n",
      "81357 \n",
      "U2AF1 (U2 small nuclear RNA auxiliary factor 1) (eg, myelodysplastic syndrome, acute \n",
      "myeloid leukemia) gene analysis, common variants (eg, S34F, S34Y, Q157R, Q157P) \n",
      "81360 \n",
      "ZRSR2 (zinc finger CCCH-type, RNA binding motif and serine/arginine-rich 2) (eg, \n",
      "myelodysplastic syndrome, acute myeloid leukemia) gene analysis, common variant(s) (eg, \n",
      "E65fs, E122fs, R448fs) \n",
      "81361 \n",
      "HBB (hemoglobin, subunit beta) (eg, sickle cell anemia, beta thalassemia, \n",
      "hemoglobinopathy); common variant(s) (eg, HbS, HbC, HbE) \n",
      "81362 \n",
      "HBB (hemoglobin, subunit beta) (eg, sickle cell anemia, beta thalassemia, \n",
      "hemoglobinopathy); known familial variant(s) \n",
      "81363 \n",
      "HBB (hemoglobin, subunit beta) (eg, sickle cell anemia, beta thalassemia, \n",
      "hemoglobinopathy); duplication/deletion variant(s) \n",
      "81364 \n",
      "HBB (hemoglobin, subunit beta) (eg, sickle cell anemia, beta thalassemia, \n",
      "hemoglobinopathy); full gene sequence \n",
      "81400 \n",
      "Molecular pathology procedure, Level 1(eg, identification of single germline variant [eg, SNP] \n",
      "by techniques such as restriction enzyme digestion or melt curve analysis) \n",
      "81401 \n",
      "Molecular pathology procedure, Level 2 (eg, 2-10 SNPs, 1 methylated variant, or 1 somatic \n",
      "variant [typically using nonsequencing target variant analysis], or detection of a dynamic \n",
      "mutation disorder/triplet repeat) \n",
      "81402 \n",
      "Molecular pathology procedure, Level 3 (eg, >10 SNPs, 2-10 methylated variants, or 2-10 \n",
      "somatic variants [typically using non-sequencing target variant analysis], immunoglobulin and \n",
      "T-cell receptor gene rearrangements, duplication/deletion variants of 1 exon, loss of \n",
      "heterozygosity [LOH], uniparental disomy [UPD]) \n",
      "81403 \n",
      "Molecular pathology procedure, Level 4 (eg, analysis of single exon by DNA sequence \n",
      "analysis, analysis of >10 amplicons using multiplex PCR in 2 or more independent reactions, \n",
      "mutation scanning or duplication/deletion variants of 2-5 exons) \n",
      "81404 \n",
      "Molecular pathology procedure, Level 5 (eg, analysis of 2-5 exons by DNA sequence \n",
      "analysis, mutation scanning or duplication/deletion variants of 6-10 exons, or characterization \n",
      "of a dynamic mutation disorder/triplet repeat by Southern blot analysis) \n",
      "81405 \n",
      "Molecular pathology procedure, Level 6 (eg, analysis of 6-10 exons by DNA sequence \n",
      "analysis, mutation scanning or duplication/deletion variants of 11-25 exons, regionally \n",
      "targeted cytogenomic array analysis) \n",
      "81406 \n",
      "Molecular pathology procedure, Level 7 (eg, analysis of 11-25 exons by DNA sequence \n",
      "analysis, mutation scanning or duplication/deletion variants of 26-50 exons, cytogenomic \n",
      "array analysis for neoplasia) \n",
      "81407 \n",
      "Molecular pathology procedure, Level 8 (eg, analysis of 26-50 exons by DNA sequence \n",
      "analysis, mutation scanning or duplication/deletion variants of >50 exons, sequence analysis \n",
      "of multiple genes on one platform) \n",
      "81408 \n",
      "Molecular pathology procedure, Level 9 (eg, analysis of >50 exons in a single gene by DNA \n",
      "sequence analysis) \n",
      "81410 \n",
      "Aortic dysfunction or dilation (eg, Marfan syndrome, Loeys Dietz syndrome, Ehler Danlos \n",
      "syndrome type IV, arterial tortuosity syndrome); genomic sequence analysis panel, must \n",
      " \n",
      "9 \n",
      " \n",
      "include sequencing of at least 9 genes, including FBN1, TGFBR1, TGFBR2, COL3A1, \n",
      "MYH11, ACTA2, SLC2A10, SMAD3, and MYLK \n",
      "81411 \n",
      "Aortic dysfunction or dilation (eg, Marfan syndrome, Loeys Dietz syndrome, Ehler Danlos \n",
      "syndrome type IV, arterial tortuosity syndrome); duplication/deletion analysis panel, must \n",
      "include analyses for TGFBR1, TGFBR2, MYH11, and COL3A1 \n",
      "81412 \n",
      "Ashkenazi Jewish associated disorders (eg, Bloom syndrome, Canavan disease, cystic \n",
      "fibrosis, familial dysautonomia, Fanconi anemia group C, Gaucher disease, Tay-Sachs \n",
      "disease), genomic sequence analysis panel, must include sequencing of at least 9 genes, \n",
      "including ASPA, BLM, CFTR, FANCC, GBA, HEXA, IKBKAP, MCOLN1, and SMPD1 \n",
      "81413 \n",
      "Cardiac ion channelopathies (eg, Brugada syndrome, long QT syndrome, catecholaminergic \n",
      "polymorphic ventricular tachycardia); genomic sequence analysis panel, must include \n",
      "sequencing of at least 10 genes, including ANK2, CASQ2, CAV3, KCNE1, KCNE2, KCNH2, \n",
      "KCNJ2, KCNQ1, RYR2, and SCN5A \n",
      "81414 \n",
      "Cardiac ion channelopathies (eg, Brugada syndrome, long QT syndrome, catecholaminergic \n",
      "polymorphic ventricular tachycardia); duplication/deletion gene analysis panel, must include \n",
      "analysis of at least 2 genes, including KCNH2 and KCNQ1 \n",
      "81415 \n",
      "Exome (eg, unexplained constitutional or heritable disorder or syndrome); sequence analysis \n",
      "81416 \n",
      "Exome (eg, unexplained constitutional or heritable disorder or syndrome); sequence analysis, \n",
      "each comparator exome (eg, parents, siblings) (List separately in addition to code for primary \n",
      "procedure) \n",
      "81417 \n",
      "Exome (eg, unexplained constitutional or heritable disorder or syndrome); re-evaluation of \n",
      "previously obtained exome sequence (eg, updated knowledge or unrelated \n",
      "condition/syndrome) \n",
      "81418 \n",
      "Drug metabolism (eg, pharmacogenomics) genomic sequence analysis panel, must include \n",
      "testing of at least 6 genes, including CYP2C19, CYP2D6, and CYP2D6 duplication/deletion \n",
      "analysis \n",
      "81419 \n",
      "Epilepsy genomic sequence analysis panel, must include analyses for ALDH7A1, CACNA1A, \n",
      "CDKL5, CHD2, GABRG2, GRIN2A, KCNQ2, MECP2, PCDH19, POLG, PRRT2, SCN1A, \n",
      "SCN1B, SCN2A, SCN8A, SLC2A1, SLC9A6, STXBP1, SYNGAP1, TCF4, TPP1, TSC1, \n",
      "TSC2, and ZEB2 \n",
      "81420 \n",
      "Fetal chromosomal aneuploidy (eg, trisomy 21, monosomy X) genomic sequence analysis \n",
      "panel, circulating cell-free fetal DNA in maternal blood, must include analysis of \n",
      "chromosomes 13, 18, and 21 \n",
      "81422 \n",
      "Fetal chromosomal microdeletion(s) genomic sequence analysis (eg, DiGeorge syndrome, \n",
      "Cri-du-chat syndrome), circulating cell-free fetal DNA in maternal blood \n",
      "81425 \n",
      "Genome (eg, unexplained constitutional or heritable disorder or syndrome); sequence \n",
      "analysis \n",
      "81426 \n",
      "Genome (eg, unexplained constitutional or heritable disorder or syndrome); sequence \n",
      "analysis, each comparator genome (eg, parents, siblings) (List separately in addition to code \n",
      "for primary procedure) \n",
      "81427 \n",
      "Genome (eg, unexplained constitutional or heritable disorder or syndrome); re-evaluation of \n",
      "previously obtained genome sequence (eg, updated knowledge or unrelated \n",
      "condition/syndrome) \n",
      "81430 \n",
      "Hearing loss (eg, nonsyndromic hearing loss, Usher syndrome, Pendred syndrome); genomic \n",
      "sequence analysis panel, must include sequencing of at least 60 genes, including CDH23, \n",
      "CLRN1, GJB2, GPR98, MTRNR1, MYO7A, MYO15A, PCDH15, OTOF, SLC26A4, TMC1, \n",
      "TMPRSS3, USH1C, USH1G, USH2A, and WFS1 \n",
      "81431 \n",
      "Hearing loss (eg, nonsyndromic hearing loss, Usher syndrome, Pendred syndrome); \n",
      "duplication/deletion analysis panel, must include copy number analyses for STRC and \n",
      "DFNB1 deletions in GJB2 and GJB6 genes \n",
      "81432 \n",
      "Hereditary breast cancer-related disorders (eg, hereditary breast cancer, hereditary ovarian \n",
      "cancer, hereditary endometrial cancer); genomic sequence analysis panel, must include \n",
      "sequencing of at least 10 genes, always including BRCA1, BRCA2, CDH1, MLH1, MSH2, \n",
      "MSH6, PALB2, PTEN, STK11, and TP53 \n",
      "81433 \n",
      "Hereditary breast cancer-related disorders (eg, hereditary breast cancer, hereditary ovarian \n",
      "cancer, hereditary endometrial cancer); duplication/deletion analysis panel, must include \n",
      "analyses for BRCA1, BRCA2, MLH1, MSH2, and STK11 \n",
      " \n",
      "10 \n",
      " \n",
      "81434 \n",
      "Hereditary retinal disorders (eg, retinitis pigmentosa, Leber congenital amaurosis, cone-rod \n",
      "dystrophy), genomic sequence analysis panel, must include sequencing of at least 15 genes, \n",
      "including ABCA4, CNGA1, CRB1, EYS, PDE6A, PDE6B, PRPF31, PRPH2, RDH12, RHO, \n",
      "RP1, RP2, RPE65, RPGR, and USH2A \n",
      "81435 \n",
      "Hereditary colon cancer disorders (eg, Lynch syndrome, PTEN hamartoma syndrome, \n",
      "Cowden syndrome, familial adenomatosis polyposis); genomic sequence analysis panel, \n",
      "must include sequencing of at least 10 genes, including APC, BMPR1A, CDH1, MLH1, \n",
      "MSH2, MSH6, MUTYH, PTEN, SMAD4, and STK11 \n",
      "81436 \n",
      "Hereditary colon cancer disorders (eg, Lynch syndrome, PTEN hamartoma syndrome, \n",
      "Cowden syndrome, familial adenomatosis polyposis); duplication/deletion analysis panel, \n",
      "must include analysis of at least 5 genes, including MLH1, MSH2, EPCAM, SMAD4, and \n",
      "STK11 \n",
      "81437 \n",
      "Hereditary neuroendocrine tumor disorders (eg, medullary thyroid carcinoma, parathyroid \n",
      "carcinoma, malignant pheochromocytoma or paraganglioma); genomic sequence analysis \n",
      "panel, must include sequencing of at least 6 genes, including MAX, SDHB, SDHC, SDHD, \n",
      "TMEM127, and VHL \n",
      "81438 \n",
      "Hereditary neuroendocrine tumor disorders (eg, medullary thyroid carcinoma, parathyroid \n",
      "carcinoma, malignant pheochromocytoma or paraganglioma); duplication/deletion analysis \n",
      "panel, must include analyses for SDHB, SDHC, SDHD, and VHL \n",
      "81439 \n",
      "Hereditary cardiomyopathy (eg, hypertrophic cardiomyopathy, dilated cardiomyopathy, \n",
      "arrhythmogenic right ventricular cardiomyopathy), genomic sequence analysis panel, must \n",
      "include sequencing of at least 5 cardiomyopathy-related genes (eg, DSG2, MYBPC3, MYH7, \n",
      "PKP2, TTN) \n",
      "81440 \n",
      "Nuclear encoded mitochondrial genes (eg, neurologic or myopathic phenotypes), genomic \n",
      "sequence panel, must include analysis of at least 100 genes, including BCS1L, C10orf2, \n",
      "COQ2, COX10, DGUOK, MPV17, OPA1, PDSS2, POLG, POLG2, RRM2B, SCO1, SCO2, \n",
      "SLC25A4, SUCLA2, SUCLG1, TAZ, TK2, and TYMP \n",
      "81441 \n",
      "Inherited bone marrow failure syndromes (IBMFS) (eg, Fanconi anemia, dyskeratosis \n",
      "congenita, Diamond-Blackfan anemia, Shwachman-Diamond syndrome, GATA2 deficiency \n",
      "syndrome, congenital amegakaryocytic thrombocytopenia) sequence analysis panel, must \n",
      "include sequencing of at least 30 genes, including BRCA2, BRIP1, DKC1, FANCA, FANCB, \n",
      "FANCC, FANCD2, FANCE, FANCF, FANCG, FANCI, FANCL, GATA1, GATA2, MPL, NHP2, \n",
      "NOP10, PALB2, RAD51C, RPL11, RPL35A, RPL5, RPS10, RPS19, RPS24, RPS26, RPS7, \n",
      "SBDS, TERT, and TINF2 \n",
      "81442 \n",
      "Noonan spectrum disorders (eg, Noonan syndrome, cardio-facio-cutaneous syndrome, \n",
      "Costello syndrome, LEOPARD syndrome, Noonan-like syndrome), genomic sequence \n",
      "analysis panel, must include sequencing of at least 12 genes, including BRAF, CBL, HRAS, \n",
      "KRAS, MAP2K1, MAP2K2, NRAS, PTPN11, RAF1, RIT1, SHOC2, and SOS1 \n",
      "81443 \n",
      "Genetic testing for severe inherited conditions (eg, cystic fibrosis, Ashkenazi Jewish-\n",
      "associated disorders [eg, Bloom syndrome, Canavan disease, Fanconi anemia type C, \n",
      "mucolipidosis type VI, Gaucher disease, Tay-Sachs disease], beta hemoglobinopathies, \n",
      "phenylketonuria, galactosemia), genomic sequence analysis panel, must include sequencing \n",
      "of at least 15 genes (eg, ACADM, ARSA, ASPA, ATP7B, BCKDHA, BCKDHB, BLM, CFTR, \n",
      "DHCR7, FANCC, G6PC, GAA, GALT, GBA, GBE1, HBB, HEXA, IKBKAP, MCOLN1, PAH) \n",
      "81445 \n",
      "Targeted genomic sequence analysis panel, solid organ neoplasm, 5-50 genes (eg, ALK, \n",
      "BRAF, CDKN2A, EGFR, ERBB2, KIT, KRAS, MET, NRAS, PDGFRA, PDGFRB, PGR, \n",
      "PIK3CA, PTEN, RET), interrogation for sequence variants and copy number variants or \n",
      "rearrangements, if performed; DNA analysis or combined DNA and RNA analysis \n",
      "81448 \n",
      "Hereditary peripheral neuropathies (eg, Charcot-Marie-Tooth, spastic paraplegia), genomic \n",
      "sequence analysis panel, must include sequencing of at least 5 peripheral neuropathy-\n",
      "related genes (eg, BSCL2, GJB1, MFN2, MPZ, REEP1, SPAST, SPG11, SPTLC1) \n",
      "81449 \n",
      "Targeted genomic sequence analysis panel, solid organ neoplasm, 5-50 genes (eg, ALK, \n",
      "BRAF, CDKN2A, EGFR, ERBB2, KIT, KRAS, MET, NRAS, PDGFRA, PDGFRB, PGR, \n",
      "PIK3CA, PTEN, RET), interrogation for sequence variants and copy number variants or \n",
      "rearrangements, if performed; RNA analysis \n",
      "81450 \n",
      "Targeted genomic sequence analysis panel, hematolymphoid neoplasm or disorder, 5-50 \n",
      "genes (eg, BRAF, CEBPA, DNMT3A, EZH2, FLT3, IDH1, IDH2, JAK2, KIT, KRAS, MLL, \n",
      " \n",
      "11 \n",
      " \n",
      "NOTCH1, NPM1, NRAS), interrogation for sequence variants, and copy number variants or \n",
      "rearrangements, or isoform expression or mRNA expression levels, if performed; DNA \n",
      "analysis or combined DNA and RNA analysis \n",
      "81451 \n",
      "Targeted genomic sequence analysis panel, hematolymphoid neoplasm or disorder, 5-50 \n",
      "genes (eg, BRAF, CEBPA, DNMT3A, EZH2, FLT3, IDH1, IDH2, JAK2, KIT, KRAS, MLL, \n",
      "NOTCH1, NPM1, NRAS), interrogation for sequence variants, and copy number variants or \n",
      "rearrangements, or isoform expression or mRNA expression levels, if performed; RNA \n",
      "analysis \n",
      "81455 \n",
      "Targeted genomic sequence analysis panel, solid organ or hematolymphoid neoplasm or \n",
      "disorder, 51 or greater genes (eg, ALK, BRAF, CDKN2A, CEBPA, DNMT3A, EGFR, ERBB2, \n",
      "EZH2, FLT3, IDH1, IDH2, JAK2, KIT, KRAS, MET, MLL, NOTCH1, NPM1, NRAS, PDGFRA, \n",
      "PDGFRB, PGR, PIK3CA, PTEN, RET), interrogation for sequence variants and copy number \n",
      "variants or rearrangements, or isoform expression or mRNA expression levels, if performed; \n",
      "DNA analysis or combined DNA and RNA analysis \n",
      "81456 \n",
      "Targeted genomic sequence analysis panel, solid organ or hematolymphoid neoplasm or \n",
      "disorder, 51 or greater genes (eg, ALK, BRAF, CDKN2A, CEBPA, DNMT3A, EGFR, ERBB2, \n",
      "EZH2, FLT3, IDH1, IDH2, JAK2, KIT, KRAS, MET, MLL, NOTCH1, NPM1, NRAS, PDGFRA, \n",
      "PDGFRB, PGR, PIK3CA, PTEN, RET), interrogation for sequence variants and copy number \n",
      "variants or rearrangements, or isoform expression or mRNA expression levels, if performed; \n",
      "RNA analysis \n",
      "81460 \n",
      "Whole mitochondrial genome (eg, Leigh syndrome, mitochondrial encephalomyopathy, lactic \n",
      "acidosis, and stroke-like episodes [MELAS], myoclonic epilepsy with ragged-red fibers \n",
      "[MERFF], neuropathy, ataxia, and retinitis pigmentosa [NARP], Leber hereditary optic \n",
      "neuropathy [LHON]), genomic sequence, must include sequence analysis of entire \n",
      "mitochondrial genome with heteroplasmy detection \n",
      "81465 \n",
      "Whole mitochondrial genome large deletion analysis panel (eg, Kearns-Sayre syndrome, \n",
      "chronic progressive external ophthalmoplegia), including heteroplasmy detection, if \n",
      "performed \n",
      "81470 \n",
      "X-linked intellectual disability (XLID) (eg, syndromic and non-syndromic XLID); genomic \n",
      "sequence analysis panel, must include sequencing of at least 60 genes, including ARX, \n",
      "ATRX, CDKL5, FGD1, FMR1, HUWE1, IL1RAPL, KDM5C, L1CAM, MECP2, MED12, MID1, \n",
      "OCRL, RPS6KA3, and SLC16A2 \n",
      "81471 \n",
      "X-linked intellectual disability (XLID) (eg, syndromic and non-syndromic XLID); \n",
      "duplication/deletion gene analysis, must include analysis of at least 60 genes, including ARX, \n",
      "ATRX, CDKL5, FGD1, FMR1, HUWE1, IL1RAPL, KDM5C, L1CAM, MECP2, MED12, MID1, \n",
      "OCRL, RPS6KA3, and SLC16A2 \n",
      "81479 \n",
      "Unlisted molecular pathology procedure \n",
      "81493 \n",
      "Coronary artery disease, mRNA, gene expression profiling by real-time RT-PCR of 23 genes, \n",
      "utilizing whole peripheral blood, algorithm reported as a risk score \n",
      "81504 \n",
      "Oncology (tissue of origin), microarray gene expression profiling of > 2000 genes, utilizing \n",
      "formalin-fixed paraffin-embedded tissue, algorithm reported as tissue similarity scores \n",
      "81507 \n",
      "Fetal aneuploidy (trisomy 21, 18, and 13) DNA sequence analysis of selected regions using \n",
      "maternal plasma, algorithm reported as a risk score for each trisomy \n",
      "81518 \n",
      "Oncology (breast), mRNA, gene expression profiling by real-time RT-PCR of 11 genes (7 \n",
      "content and 4 housekeeping), utilizing formalin-fixed paraffin-embedded tissue, algorithms \n",
      "reported as percentage risk for metastatic recurrence and likelihood of benefit from extended \n",
      "endocrine therapy \n",
      "81519 \n",
      "Oncology (breast), mRNA, gene expression profiling by real-time RT-PCR of 21 genes, \n",
      "utilizing formalin-fixed paraffin embedded tissue, algorithm reported as recurrence score \n",
      "81520 \n",
      "Oncology (breast), mRNA gene expression profiling by hybrid capture of 58 genes (50 \n",
      "content and 8 housekeeping), utilizing formalin-fixed paraffin-embedded tissue, algorithm \n",
      "reported as a recurrence risk score \n",
      "81521 \n",
      "Oncology (breast), mRNA, microarray gene expression profiling of 70 content genes and 465 \n",
      "housekeeping genes, utilizing fresh frozen or formalin-fixed paraffin-embedded tissue, \n",
      "algorithm reported as index related to risk of distant metastasis \n",
      " \n",
      "12 \n",
      " \n",
      "81522 \n",
      "Oncology (breast), mRNA, gene expression profiling by RT-PCR of 12 genes (8 content and \n",
      "4 housekeeping), utilizing formalin-fixed paraffin-embedded tissue, algorithm reported as \n",
      "recurrence risk score (Endopredict) \n",
      "81523 \n",
      "Oncology (breast), mRNA, next-generation sequencing gene expression profiling of 70 \n",
      "content genes and 31 housekeeping genes, utilizing formalin-fixed paraffin-embedded tissue, \n",
      "algorithm reported as index related to risk to distant metastasis - MAAA Breast Cancer \n",
      "Metastasis RNA Sequencing \n",
      "81525 \n",
      "Oncology (colon), mRNA, gene expression profiling by real-time RT-PCR of 12 genes (7 \n",
      "content and 5 housekeeping), utilizing formalin-fixed paraffin-embedded tissue, algorithm \n",
      "reported as a recurrence score \n",
      "81529 \n",
      "Oncology (cutaneous melanoma), mRNA, gene expression profiling by real-time RT-PCR of \n",
      "31 genes (28 content and 3 housekeeping), utilizing formalin-fixed paraffin-embedded tissue, \n",
      "algorithm reported as recurrence risk, including likelihood of sentinel lymph node metastasis \n",
      "81540 \n",
      "Oncology (tumor of unknown origin), mRNA, gene expression profiling by real-time RT-PCR \n",
      "of 92 genes (87 content and 5 housekeeping) to classify tumor into main cancer type and \n",
      "subtype, utilizing formalin-fixed paraffin-embedded tissue, algorithm reported \n",
      "81541 \n",
      "Oncology (prostate), mRNA gene expression profiling by real-time RT-PCR of 46 genes (31 \n",
      "content and 15 housekeeping), utilizing formalin-fixed paraffin-embedded tissue, algorithm \n",
      "reported as a disease-specific mortality risk score \n",
      "81542 \n",
      "Oncology (prostate), mRNA, microarray gene expression profiling of 22 content genes, \n",
      "utilizing formalin-fixed paraffin-embedded tissue, algorithm reported as metastasis risk score \n",
      "(Decipher) \n",
      "81546 \n",
      "Oncology (thyroid), mRNA, gene expression analysis of 10,196 genes, utilizing fine needle \n",
      "aspirate, algorithm reported as a categorical result (eg, benign or suspicious) \n",
      "81551 \n",
      "Oncology (prostate), promoter methylation profiling by real-time PCR of 3 genes (GSTP1, \n",
      "APC, RASSF1), utilizing formalin-fixed paraffin-embedded tissue, algorithm reported as a \n",
      "likelihood of prostate cancer detection on repeat biopsy \n",
      "81552 \n",
      "Oncology (uveal melanoma), mRNA, gene expression profiling by real-time RT-PCR of 15 \n",
      "genes (12 content and 3 housekeeping), utilizing fine needle aspirate or formalin-fixed \n",
      "paraffin-embedded tissue, algorithm reported as risk of metastasis \n",
      "81554 \n",
      "Pulmonary disease (idiopathic pulmonary fibrosis [IPF]), mRNA, gene expression analysis of \n",
      "190 genes, utilizing transbronchial biopsies, diagnostic algorithm reported as categorical \n",
      "result (eg, positive or negative for high probability of usual interstitial pneumonia [UIP]) \n",
      "81595 \n",
      "Cardiology (heart transplant), mRNA, gene expression profiling by real-time quantitative PCR \n",
      "of 20 genes (11 content and 9 housekeeping), utilizing subfraction of peripheral blood, \n",
      "algorithm reported as a rejection risk score \n",
      " \n",
      " \n",
      "0001U \n",
      "Red blood cell antigen typing, DNA, human erythrocyte antigen gene analysis of 35 antigens \n",
      "from 11 blood groups, utilizing whole blood, common RBC alleles reported \n",
      "0004M \n",
      "Scoliosis, DNA analysis of 53 single nucleotide polymorphisms (SNPs), using saliva, \n",
      "prognostic algorithm reported as a risk score \n",
      "0005U \n",
      "Oncology (prostate) gene expression profile by real-time RT-PCR of 3 genes (ERG, PCA3, \n",
      "and SPDEF), urine, algorithm reported as risk score \n",
      "0006M \n",
      "Oncology (hepatic), mRNA expression levels of 161 genes, utilizing fresh hepatocellular \n",
      "carcinoma tumor tissue, with alpha-fetoprotein level, algorithm reported as a risk classifier \n",
      "0007M \n",
      "Oncology (gastrointestinal neuroendocrine tumors), real-time PCR expression analysis of 51 \n",
      "genes, utilizing whole peripheral blood, algorithm reported as a nomogram of tumor disease \n",
      "index \n",
      "0011M \n",
      "Oncology, prostate cancer, mRNA expression assay of 12 genes (10 content and 2 \n",
      "housekeeping), RT-PCR test utilizing blood plasma and/or urine, algorithms to predict high-\n",
      "grade prostate cancer risk \n",
      "0012M \n",
      "Oncology (urothelial), mRNA, gene expression profiling by real-time quantitative PCR of five \n",
      "genes (MDK, HOXA13, CDC2 [CDK1], IGFBP5, and XCR2), utilizing urine, algorithm \n",
      "reported as a risk score for having urothelial carcinoma \n",
      "0016M \n",
      "Oncology (bladder), mRNA, microarray gene expression profiling of 209 genes, utilizing \n",
      "formalinfixed paraffin-embedded tissue, algorithm reported as molecular subtype (luminal, \n",
      "luminal infiltrated, basal, basal claudin-low, neuroendocrine-like) \n",
      " \n",
      "13 \n",
      " \n",
      "0017M \n",
      "Oncology (diffuse large B-cell lymphoma [DLBCL]), mRNA, gene expression profiling by \n",
      "fluorescent probe hybridization of 20 genes, formalin-fixed paraffin-embedded tissue, \n",
      "algorithm reported as cell of origin \n",
      "0013U \n",
      "Oncology (solid organ neoplasia), gene rearrangement detection by whole genome next-\n",
      "generation sequencing, DNA, fresh or frozen tissue or cells, report of specific gene \n",
      "rearrangement(s) \n",
      "0016U \n",
      "Oncology (hematolymphoid neoplasia), RNA, BCR/ABL1 major and minor breakpoint fusion \n",
      "transcripts, quantitative PCR amplification, blood or bone marrow, report of fusion not \n",
      "detected or detected with quantitation \n",
      "0017U \n",
      "Oncology (hematolymphoid neoplasia), JAK2 mutation, DNA, PCR amplification of exons 12-\n",
      "14 and sequence analysis, blood or bone marrow, report of JAK2 mutation not detected or \n",
      "detected \n",
      "0018U \n",
      "Oncology (thyroid), microRNA profiling by RT-PCR of 10 microRNA sequences, utilizing fine \n",
      "needle aspirate, algorithm reported as a positive or negative result for moderate to high risk \n",
      "of malignancy \n",
      "0019U \n",
      "Oncology, RNA, gene expression by whole transcriptome sequencing, formalin-fixed paraffin \n",
      "embedded tissue or fresh frozen tissue, predictive algorithm reported as potential targets for \n",
      "therapeutic agents \n",
      "0022U \n",
      "Targeted genomic sequence analysis panel, non-small cell lung neoplasia, DNA and RNA \n",
      "analysis, 23 genes, interrogation for sequence variants and rearrangements, reported as \n",
      "presence/absence of variants and associated therapy(ies) to consider \n",
      "0023U \n",
      "Oncology (acute myelogenous leukemia), DNA, genotyping of internal tandem duplication, \n",
      "p.D835, p.I836, using mononuclear cells, reported as detection or nondetection of FLT3 \n",
      "mutation and indication for or against the use of midostaurin \n",
      "0026U \n",
      "Oncology (thyroid), DNA and mRNA of 112 genes, next-generation sequencing, fine needle \n",
      "aspirate of thyroid nodule, algorithmic analysis reported as a categorical result (\"Positive, \n",
      "high probability of malignancy\" or \"Negative, low probability of malignancy\") \n",
      "0027U \n",
      "JAK2 (Janus kinase 2) (eg, myeloproliferative disorder) gene analysis, targeted sequence \n",
      "analysis exons 12-15 \n",
      "0029U \n",
      "Drug metabolism (adverse drug reactions and drug response), targeted sequence analysis \n",
      "(ie, CYP1A2, CYP2C19, CYP2C9, CYP2D6, CYP3A4, CYP3A5, CYP4F2, SLCO1B1, \n",
      "VKORC1 and rs12777823) \n",
      "0030U \n",
      "Drug metabolism (warfarin drug response), targeted sequence analysis (ie, CYP2C9, \n",
      "CYP4F2, VKORC1, rs12777823) \n",
      "0031U \n",
      "CYP1A2 (cytochrome P450 family 1, subfamily A, member 2)(eg, drug metabolism) gene \n",
      "analysis, common variants (ie, *1F, *1K, *6, *7) \n",
      "0032U \n",
      "COMT (catechol-O-methyltransferase)(drug metabolism) gene analysis, c.472G>A (rs4680) \n",
      "variant \n",
      "0033U \n",
      "HTR2A (5-hydroxytryptamine receptor 2A), HTR2C (5-hydroxytryptamine receptor 2C) (eg, \n",
      "citalopram metabolism) gene analysis, common variants (ie, HTR2A rs7997012 [c.614-\n",
      "2211T>C], HTR2C rs3813929 [c.- 759C>T] and rs1414334 [c.551-3008C>G]) \n",
      "0034U \n",
      "TPMT (thiopurine S-methyltransferase), NUDT15 (nudix hydroxylase 15)(eg, thiopurine \n",
      "metabolism), gene analysis, common variants (ie, TPMT *2, *3A, *3B, *3C, *4, *5, *6, *8, *12; \n",
      "NUDT15 *3, *4, *5) \n",
      "0036U \n",
      "Exome (ie, somatic mutations), paired formalin-fixed paraffin-embedded tumor tissue and \n",
      "normal specimen, sequence analyses \n",
      "0037U \n",
      "Targeted genomic sequence analysis, solid organ neoplasm, DNA analysis of 324 genes, \n",
      "interrogation for sequence variants, gene copy number amplifications, gene rearrangements, \n",
      "microsatellite instability and tumor mutational burden \n",
      "0040U \n",
      "BCR/ABL1 (t(9;22)) (eg, chronic myelogenous leukemia) translocation analysis, major \n",
      "breakpoint, quantitative \n",
      "0045U \n",
      "Oncology (breast ductal carcinoma in situ), mRNA, gene expression profiling by realtime RT-\n",
      "PCR of 12 genes (7 content and 5 housekeeping), utilizing formalin-fixed paraffin-embedded \n",
      "tissue, algorithm reported as recurrence score \n",
      "0046U \n",
      "FLT3 (fms-related tyrosine kinase 3) (eg, acute myeloid leukemia) internal tandem duplication \n",
      "(ITD) variants, quantitative \n",
      " \n",
      "14 \n",
      " \n",
      "0047U \n",
      "Oncology (prostate), mRNA, gene expression profiling by real-time RT-PCR of 17 genes (12 \n",
      "content and 5 housekeeping), utilizing formalin-fixed paraffin-embedded tissue, algorithm \n",
      "reported as a risk score \n",
      "0048U \n",
      "Oncology (solid organ neoplasia), DNA, targeted sequencing of protein-coding exons of 468 \n",
      "cancer-associated genes, including interrogation for somatic mutations and microsatellite \n",
      "instability, matched with normal specimens, utilizing formalin-fixed paraffin-embedded tumor \n",
      "tissue, report of clinically significant mutation(s) \n",
      "0049U \n",
      "NPM1 (nucleophosmin) (eg, acute myeloid leukemia) gene analysis, quantitative \n",
      "0050U \n",
      "Targeted genomic sequence analysis panel, acute myelogenous leukemia, DNA analysis, \n",
      "194 genes, interrogation for sequence variants, copy number variants or rearrangements \n",
      "0055U \n",
      "Cardiology (heart transplant), cell-free DNA, PCR assay of 96 DNA target sequences (94 \n",
      "single nucleotide polymorphism targets and two control targets), plasma \n",
      "0060U \n",
      "Twin zygosity, genomic targeted sequence analysis of chromosome 2, using circulating cell-\n",
      "free fetal DNA in maternal blood \n",
      "0069U \n",
      "Oncology (colorectal), microRNA, RT-PCR expression profiling of miR-31-3p, formalin-fixed \n",
      "paraffin-embedded tissue, algorithm reported as an expression score \n",
      "0070U \n",
      "CYP2D6 (cytochrome P450, family 2, subfamily D, polypeptide 6) (eg, drug metabolism) \n",
      "gene analysis, common and select rare variants (ie, *2, *3, *4, *4N, *5, *6, *7, *8, *9, *10, *11, \n",
      "*12, *13, *14A, *14B, *15, *17, *29, *35, *36, *41, *57, *61, *63, *68, *83, *xN) \n",
      "0071U \n",
      "CYP2D6 (cytochrome P450, family 2, subfamily D, polypeptide 6) (eg, drug metabolism) \n",
      "gene analysis, full gene sequence (List separately in addition to code for primary procedure) \n",
      "0072U \n",
      "CYP2D6 (cytochrome P450, family 2, subfamily D, polypeptide 6) (eg, drug metabolism) \n",
      "gene analysis, targeted sequence analysis (ie, CYP2D6-2D7 hybrid gene) (List separately in \n",
      "addition to code for primary procedure) \n",
      "0073U \n",
      "CYP2D6 (cytochrome P450, family 2, subfamily D, polypeptide 6) (eg, drug metabolism) \n",
      "gene analysis, targeted sequence analysis (ie, CYP2D7-2D6 hybrid gene) (List separately in \n",
      "addition to code for primary procedure)  \n",
      "0074U \n",
      "CYP2D6 (cytochrome P450, family 2, subfamily D, polypeptide 6) (eg, drug metabolism) \n",
      "gene analysis, targeted sequence analysis (ie, non-duplicated gene when \n",
      "duplication/multiplication is trans) (List separately in addition to code for primary procedure)  \n",
      "0075U \n",
      "CYP2D6 (cytochrome P450, family 2, subfamily D, polypeptide 6) (eg, drug metabolism) \n",
      "gene analysis, targeted sequence analysis (ie, 5’ gene duplication/multiplication) (List \n",
      "separately in addition to code for primary procedure)  \n",
      "0076U \n",
      "CYP2D6 (cytochrome P450, family 2, subfamily D, polypeptide 6) (eg, drug metabolism) \n",
      "gene analysis, targeted sequence analysis (ie, 3’ gene duplication/ multiplication) (List \n",
      "separately in addition to code for primary procedure) \n",
      "0078U \n",
      "Pain management (opioid-use disorder) genotyping panel, 16 common variants (ie, ABCB1, \n",
      "COMT, DAT1, DBH, DOR, DRD1, DRD2, DRD4, GABA, GAL, HTR2A, HTTLPR, MTHFR, \n",
      "MUOR, OPRK1, OPRM1), buccal swab or other germline tissue sample, algorithm reported \n",
      "as positive or negative risk of opioid-use disorder \n",
      "0079U \n",
      "Comparative DNA analysis using multiple selected single-nucleotide polymorphisms (SNPs), \n",
      "urine and buccal DNA, for specimen identity verification \n",
      "0081U \n",
      "Oncology (uveal melanoma), mRNA, gene-expression profiling by real-time RT-PCR of 15 \n",
      "genes (12 content and 3 housekeeping genes), utilizing fine needle aspirate or formalin-fixed \n",
      "paraffin-embedded tissue, algorithm reported as risk of metastasis \n",
      "0087U \n",
      "Cardiology (heart transplant), mRNA gene expression profiling by microarray of 1283 genes, \n",
      "transplant biopsy tissue, allograft rejection and injury algorithm reported as a probability score \n",
      "0088U \n",
      "Transplantation medicine (kidney allograft rejection), microarray gene expression profiling of \n",
      "1494 genes, utilizing transplant biopsy tissue, algorithm reported as a probability score for \n",
      "rejection \n",
      "0089U \n",
      "Oncology (melanoma), gene expression profiling by RTqPCR, PRAME and LINC00518, \n",
      "superficial collection using adhesive patch(es) \n",
      "0090U \n",
      "Oncology (cutaneous melanoma), mRNA gene expression profiling by RT-PCR of 23 genes \n",
      "(14 content and 9 housekeeping), utilizing formalin-fixed paraffin-embedded tissue, algorithm \n",
      "reported as a categorical result (ie, benign, indeterminate, malignant) \n",
      "0094U \n",
      "Genome (eg, unexplained constitutional or heritable disorder or syndrome), rapid sequence \n",
      "analysis \n",
      " \n",
      "15 \n",
      " \n",
      "0101U \n",
      "Hereditary colon cancer disorders (eg, Lynch syndrome, PTEN hamartoma syndrome, \n",
      "Cowden syndrome, familial adenomatosis polyposis), genomic sequence analysis panel \n",
      "utilizing a combination of NGS, Sanger, MLPA, and array CGH, with MRNA analytics to \n",
      "resolve variants of unknown significance when indicated (15 genes [sequencing and \n",
      "deletion/duplication], EPCAM and GREM1 [deletion/duplication only]) \n",
      "0102U \n",
      "Hereditary breast cancer-related disorders (eg, hereditary breast cancer, hereditary ovarian \n",
      "cancer, hereditary endometrial cancer), genomic sequence analysis panel utilizing a \n",
      "combination of NGS, Sanger, MLPA, and array CGH, with MRNA analytics to resolve \n",
      "variants of unknown significance when indicated (17 genes [sequencing and \n",
      "deletion/duplication]) \n",
      "0103U \n",
      "Hereditary ovarian cancer (eg, hereditary ovarian cancer, hereditary endometrial cancer), \n",
      "genomic sequence analysis panel utilizing a combination of NGS, Sanger, MLPA, and array \n",
      "CGH, with MRNA analytics to resolve variants of unknown significance when indicated (24 \n",
      "genes [sequencing and deletion/duplication], EPCAM [deletion/duplication only]) \n",
      "0111U \n",
      "Oncology (colon cancer), targeted KRAS (codons 12, 13, and 61) and NRAS (codons 12, 13, \n",
      "and 61) gene analysis utilizing formalin-fixed paraffin-embedded tissue \n",
      "0113U \n",
      " \n",
      " \n",
      "0113U Oncology (prostate), measurement of PCA3 and TMPRSS2-ERG in urine and PSA in \n",
      "serum following prostatic massage, by RNA amplification and fluorescence-based detection, \n",
      "algorithm reported as risk score \n",
      "0114U \n",
      "Gastroenterology (Barrett’s esophagus), VIM and CCNA1 methylation analysis, esophageal \n",
      "cells, algorithm reported as likelihood for Barrett’s esophagus \n",
      "0118U \n",
      "Transplantation medicine, quantification of donor-derived cell-free DNA using whole genome \n",
      "next-generation sequencing, plasma, reported as percentage of donor derived cell-free DNA \n",
      "in the total cell-free DNA \n",
      "0120U \n",
      "Oncology (B-cell lymphoma classification), mRNA, gene expression profiling by fluorescent \n",
      "probe hybridization of 58 genes (45 content and 13 housekeeping genes), formalin-fixed \n",
      "paraffin-embedded tissue, algorithm reported as likelihood for primary mediastinal B-cell \n",
      "lymphoma (PMBCL) and diffuse large B-cell lymphoma (DLBCL) with cell of origin subtyping \n",
      "in the latter \n",
      "0129U \n",
      "Hereditary breast cancer–related disorders (eg, hereditary breast cancer, hereditary ovarian \n",
      "cancer, hereditary endometrial cancer), genomic sequence analysis and deletion/duplication \n",
      "analysis panel (ATM, BRCA1, BRCA2, CDH1, CHEK2, PALB2, PTEN, and TP53) \n",
      "0130U \n",
      "Hereditary colon cancer disorders (eg, Lynch syndrome, PTEN hamartoma syndrome, \n",
      "Cowden syndrome, familial adenomatosis polyposis), targeted mRNA sequence analysis \n",
      "panel (APC, CDH1, CHEK2, MLH1, MSH2, MSH6, MUTYH, PMS2, PTEN, and TP53) (List \n",
      "separately in addition to code for primary procedure) (Use 0130U in conjunction with 81435, \n",
      "0101U) \n",
      "0131U \n",
      "Hereditary breast cancer–related disorders (eg, hereditary breast cancer, hereditary ovarian \n",
      "cancer, hereditary endometrial cancer), targeted mRNA sequence analysis panel (13 genes) \n",
      "(List separately in addition to code for primary procedure) (Use 0131U in conjunction with \n",
      "81162, 81432, 0102U) \n",
      "0132U \n",
      "Hereditary ovarian cancer–related disorders (eg, hereditary breast cancer, hereditary ovarian \n",
      "cancer, hereditary endometrial cancer), targeted mRNA sequence analysis panel (17 genes) \n",
      "(List separately in addition to code for primary procedure) (Use 0132U in conjunction with \n",
      "81162, 81432, 0103U) \n",
      "0133U \n",
      "Hereditary prostate cancer–related disorders, targeted mRNA sequence analysis panel (11 \n",
      "genes) (List separately in addition to code for primary procedure) (Use 0133U in conjunction \n",
      "with 81162) \n",
      "0134U \n",
      "Hereditary pan cancer (eg, hereditary breast and ovarian cancer, hereditary endometrial \n",
      "cancer, hereditary colorectal cancer), targeted mRNA sequence analysis panel (18 genes) \n",
      "(List separately in addition to code for primary procedure) (Use 0134U in conjunction with \n",
      "81162, 81432, 81435) \n",
      "0135U \n",
      "Hereditary gynecological cancer (eg, hereditary breast and ovarian cancer, hereditary \n",
      "endometrial cancer, hereditary colorectal cancer), targeted mRNA sequence analysis panel \n",
      "(12 genes) (List separately in addition to code for primary procedure) (Use 0135U in \n",
      "conjunction with 81162 \n",
      " \n",
      "16 \n",
      " \n",
      "0136U \n",
      "ATM (ataxia telangiectasia mutated) (eg, ataxia telangiectasia) mRNA sequence analysis \n",
      "(List separately in addition to code for primary procedure) (Use 0136U in conjunction with \n",
      "81408) \n",
      "0137U \n",
      "PALB2 (partner and localizer of BRCA2) (eg, breast and pancreatic cancer) mRNA sequence \n",
      "analysis (List separately in addition to code for primary procedure) (Use 0137U in conjunction \n",
      "with 81406) \n",
      "0138U \n",
      "BRCA1 (BRCA1, DNA repair associated), BRCA2 (BRCA2, DNA repair associated) (eg, \n",
      "hereditary breast and ovarian cancer) mRNA sequence analysis (List separately in addition to \n",
      "code for primary procedure) (Use 0138U in conjunction with 81162) \n",
      "0153U \n",
      "Oncology (breast), mRNA, gene expression profiling by next-generation sequencing of 101  \n",
      "genes, utilizing formalin-fixed paraffin-embedded tissue, algorithm reported as a triple negative \n",
      "breast cancer clinical subtype(s) with information on immune cell involvement \n",
      "0154U \n",
      "FGFR3 (fibroblast growth factor receptor 3) gene analysis (ie, p.R248C [c.742C>T], p.S249C \n",
      "[c.746C>G], p.G370C [c.1108G>T], p.Y373C [c.1118A>G], FGFR3-TACC3v1, and FGFR3-\n",
      "TACC3v3) \n",
      "0155U \n",
      "PIK3CA (phosphatidylinositol-4,5- bisphosphate 3-kinase, catalytic subunit alpha) (eg, breast \n",
      "cancer) gene analysis (ie, p.C420R, p.E542K, p.E545A, p.E545D [g.1635G>T only], p.E545G, \n",
      "p.E545K, p.Q546E, p.Q546R, p.H1047L, p.H1047R, p.H1047Y) \n",
      "0156U \n",
      "Copy number (eg, intellectual disability, dysmorphology), sequence analysis \n",
      "0157U \n",
      "APC (APC regulator of WNT signaling pathway) (eg, familial adenomatosis polyposis [FAP]) \n",
      "mRNA sequence analysis (List separately in addition to code for primary procedure) (Use  \n",
      "0157U in conjunction with 81201) \n",
      "0158U \n",
      "MLH1 (mutL homolog 1) (eg, hereditary non-polyposis colorectal cancer, Lynch syndrome) \n",
      "mRNA sequence analysis (List separately in addition to code for primary procedure) (Use 0158U \n",
      "in conjunction with 81292) \n",
      "0159U \n",
      "MSH2 (mutS homolog 2) (eg, hereditary colon cancer, Lynch syndrome) mRNA sequence  \n",
      "analysis (List separately in addition to code for primary procedure) (Use 0159U in conjunction \n",
      "with 81295) \n",
      "0160U \n",
      "MSH6 (mutS homolog 6) (eg, hereditary colon cancer, Lynch syndrome) mRNA sequence  \n",
      "analysis (List separately in addition to code for primary procedure) (Use 0160U in conjunction \n",
      "with 81298) \n",
      "0161U \n",
      "PMS2 (PMS1 homolog 2, mismatch repair system component) (eg, hereditary nonpolyposis \n",
      "colorectal cancer, Lynch syndrome) mRNA sequence analysis (List separately in addition to  \n",
      "code for primary procedure) (Use 0161U in conjunction with 81317) \n",
      "0162U \n",
      "Hereditary colon cancer (Lynch syndrome), targeted mRNA sequence analysis panel (MLH1, \n",
      "MSH2, MSH6, PMS2) (List separately in addition to code for primary procedure) \n",
      "0169U \n",
      "NUDT15 (nudix hydrolase 15) and TPMT (thiopurine S-methyltransferase) (eg, drug metabolism) \n",
      "gene analysis, common variants \n",
      "0170U \n",
      "Neurology (autism spectrum disorder [ASD]), RNA, next-generation sequencing, saliva, \n",
      "algorithmic analysis, and results reported as predictive probability of ASD diagnosis \n",
      "0171U \n",
      "Targeted genomic sequence analysis panel, acute myeloid leukemia, myelodysplastic syndrome, \n",
      "and myeloproliferative neoplasms, DNA analysis, 23 genes, interrogation for sequence variants, \n",
      "rearrangements and minimal residual disease, reported as presence/absence \n",
      "0172U \n",
      "Oncology (solid tumor as indicated by the label), somatic mutation analysis of BRCA1 (BRCA1, \n",
      "DNA repair associated), BRCA2 (BRCA2, DNA repair associated) and analysis of homologous \n",
      "recombination deficiency pathways, DNA, formalin-fixed paraffin-embedded tissue, algorithm \n",
      "quantifying tumor genomic instability score \n",
      "0173U \n",
      "Psychiatry (ie, depression, anxiety), genomic analysis panel, includes variant analysis of 14 \n",
      "genes \n",
      "0175U \n",
      "Psychiatry (eg, depression, anxiety), genomic analysis panel, variant analysis of 15 genes \n",
      "0177U \n",
      "Oncology (breast cancer), DNA, PIK3CA (phosphatidylinositol-4,5-bisphosphate 3- kinase \n",
      "catalytic subunit alpha) gene analysis of 11 gene variants utilizing plasma, reported as PIK3CA \n",
      "gene mutation status \n",
      "0179U \n",
      "Oncology (non-small cell lung cancer), cell-free DNA, targeted sequence analysis of 23 genes \n",
      "(single nucleotide variations, insertions and deletions, fusions without prior knowledge of \n",
      "partner/breakpoint, copy number variations), with report of significant mutation(s) \n",
      " \n",
      "17 \n",
      " \n",
      "0203U \n",
      "Autoimmune (inflammatory bowel disease), mRNA, gene expression profiling by quantitative RT-\n",
      "PCR, 17 genes (15 target and 2 reference genes), whole blood, reported as a continuous risk \n",
      "score and classification of inflammatory \n",
      "0204U \n",
      "Oncology (thyroid), mRNA, gene expression analysis of 593 genes (including BRAF, RAS, RET, \n",
      "PAX8, and NTRK) for sequence variants and rearrangements, utilizing fine needle aspirate, \n",
      "reported as detected or not detected \n",
      "0205U \n",
      "Ophthalmology (age-related macular degeneration), analysis of 3 gene variants (2 CFH gene, 1 \n",
      "ARMS2 gene), using PCR and MALDI-TOF, buccal swab, reported as positive or negative for \n",
      "neovascular agerelated macular-degeneration risk associated with zinc supplements \n",
      "0208U \n",
      "Oncology (medullary thyroid carcinoma), mRNA, gene expression analysis of 108 genes, utilizing \n",
      "fine needle aspirate, algorithm reported as positive or negative for medullary thyroid carcinoma \n",
      "0209U \n",
      "Cytogenomic constitutional (genome-wide) analysis, interrogation of genomic regions for copy \n",
      "number, structural changes and areas of homozygosity for chromosomal abnormalities \n",
      "0211U \n",
      "Oncology (pan-tumor), DNA and RNA by next-generation sequencing, utilizing formalin-fixed \n",
      "paraffin-embedded tissue, interpretative report for single nucleotide variants, copy number \n",
      "alterations, tumor mutational burden, and microsatellite instability, with therapy association \n",
      "0212U \n",
      "Rare diseases (constitutional/heritable disorders), whole genome and mitochondrial DNA \n",
      "sequence analysis, including small sequence changes, deletions, duplications, short tandem \n",
      "repeat gene expansions, and variants in non-uniquely mappable regions, blood or saliva, \n",
      "identification and categorization of genetic variants, proband (Do not report 0212U in conjunction \n",
      "with 81425) \n",
      "0213U \n",
      "Rare diseases (constitutional/heritable disorders), whole genome and mitochondrial DNA \n",
      "sequence analysis, including small sequence changes, deletions, duplications, short tandem \n",
      "repeat gene expansions, and variants in non-uniquely mappable regions, blood or saliva, \n",
      "identification and categorization of genetic variants, each comparator genome (eg, parent, \n",
      "sibling) (Do not report 0213U in conjunction with 81426) \n",
      "0214U \n",
      "Rare diseases (constitutional/heritable disorders), whole exome and mitochondrial DNA \n",
      "sequence analysis, including small sequence changes, deletions, duplications, short tandem \n",
      "repeat gene expansions, and variants in non-uniquely mappable regions, blood or saliva, \n",
      "identification and categorization of genetic variants, proband (Do not report 0214U in conjunction \n",
      "with 81415) \n",
      "0215U \n",
      "Rare diseases (constitutional/heritable disorders), whole exome and mitochondrial DNA \n",
      "sequence analysis, including small sequence changes, deletions, duplications, short tandem \n",
      "repeat gene expansions, and variants in non-uniquely mappable regions, blood or saliva, \n",
      "identification and categorization of genetic variants, each comparator exome (eg, parent, sibling) \n",
      "(Do not report 0215U in conjunction with 81416) \n",
      "0216U \n",
      "Neurology (inherited ataxias), genomic DNA sequence analysis of 12 common genes including \n",
      "small sequence changes, deletions, duplications, short tandem repeat gene expansions, and \n",
      "variants in non-uniquely mappable regions, blood or saliva, identification and categorization of \n",
      "genetic variants \n",
      "0217U \n",
      "Neurology (inherited ataxias), genomic DNA sequence analysis of 51 genes including small \n",
      "sequence changes, deletions, duplications, short tandem repeat gene expansions, and variants \n",
      "in non-uniquely mappable regions, blood or saliva, identification and categorization of genetic \n",
      "variants \n",
      "0218U \n",
      "Neurology (muscular dystrophy), DMD gene sequence analysis, including small sequence \n",
      "changes, deletions, duplications, and variants in non-uniquely mappable regions, blood or saliva, \n",
      "identification and characterization of genetic variants \n",
      "0228U \n",
      "Oncology (prostate), multianalyte molecular profile by photometric detection of  \n",
      "macromolecules adsorbed on nanosponge array slides with machine learning, utilizing first \n",
      "morning voided urine, algorithm reported as likelihood of prostate cancer \n",
      "0229U \n",
      "BCAT1 (Branched chain amino acid transaminase 1) or IKZF1 (IKAROS family zinc finger 1) \n",
      " (eg, colorectal cancer) promoter methylation analysis \n",
      "0230U \n",
      "AR (androgen receptor) (eg, spinal and bulbar muscular atrophy, Kennedy disease, X \n",
      "chromosome inactivation), full sequence analysis, including small sequence changes in  \n",
      "exonic and intronic regions, deletions, duplications, short tandem repeat (STR) expansions, \n",
      "mobile element insertions, and variants in non-uniquely mappable regions \n",
      " \n",
      "18 \n",
      " \n",
      "0231U \n",
      "CACNA1A (calcium voltage-gated channel subunit alpha 1A) (eg, spinocerebellar ataxia), full \n",
      "gene analysis, including small sequence changes in exonic and intronic regions, deletions, \n",
      "duplications, short tandem repeat (STR) gene expansions, mobile element insertions, and \n",
      "variants in non-uniquely mappable regions \n",
      "0232U \n",
      "CSTB (cystatin B) (eg, progressive myoclonic epilepsy type 1A, Unverricht-Lundborg disease), \n",
      "full gene analysis, including small sequence changes in exonic and intronic regions, deletions, \n",
      "duplications, short tandem repeat (STR) expansions, mobile element insertions, and variants  \n",
      "in non-uniquely mappable regions \n",
      "0233U \n",
      "FXN (frataxin) (eg, Friedreich ataxia), gene analysis, including small sequence changes in  \n",
      "exonic and intronic regions, deletions, duplications, short tandem repeat (STR) expansions, \n",
      "mobile element insertions, and variants in non-uniquely mappable regions \n",
      "0234U \n",
      "MECP2 (methyl CpG binding protein 2) (eg, Rett syndrome), full gene analysis, including  \n",
      "small sequence changes in exonic and intronic regions, deletions, duplications, mobile  \n",
      "element insertions, and variants in non-uniquely mappable regions \n",
      "0235U \n",
      "PTEN (phosphatase and tensin homolog) (eg, Cowden syndrome, PTEN hamartoma tumor \n",
      "syndrome), full gene analysis, including small sequence changes in exonic and intronic  \n",
      "regions, deletions, duplications, mobile element insertions, and variants in non-uniquely \n",
      "mappable regions \n",
      "0236U \n",
      "SMN1 (survival of motor neuron 1, telomeric) and SMN2 (survival of motor neuron 2, \n",
      "centromeric) (eg, spinal muscular atrophy) full gene analysis, including small sequence  \n",
      "changes in exonic and intronic regions, duplications and deletions, and mobile element \n",
      " insertions \n",
      "0237U \n",
      "Cardiac ion channelopathies (eg, Brugada syndrome, long QT syndrome, short QT syndrome, \n",
      "catecholaminergic polymorphic ventricular tachycardia), genomic sequence analysis panel \n",
      "including ANK2, CASQ2, CAV3, KCNE1, KCNE2, KCNH2, KCNJ2, KCNQ1, RYR2, and  \n",
      "SCN5A, including small sequence changes in exonic and intronic regions, deletions,  \n",
      "duplications, mobile element insertions, and variants in non-uniquely mappable regions \n",
      "0238U \n",
      "Oncology (Lynch syndrome), genomic DNA sequence analysis of MLH1, MSH2, MSH6,  \n",
      "PMS2, and EPCAM, including small sequence changes in exonic and intronic regions,  \n",
      "deletions, duplications, mobile element insertions, and variants in non-uniquely mappable  \n",
      "regions \n",
      "0239U \n",
      "Targeted genomic sequence analysis panel, solid organ neoplasm, cell-free DNA, analysis of  \n",
      "311 or more genes, interrogation for sequence variants, including substitutions, insertions, \n",
      "deletions, select rearrangements, and copy number variations \n",
      "0242U \n",
      "Targeted genomic sequence analysis panel, solid organ neoplasm, cell-free circulating DNA \n",
      "analysis of 55-74 genes, interrogation for sequence variants, gene copy number amplifications, \n",
      "and gene rearrangements \n",
      "0244U \n",
      "Oncotype MAP PanCancer Tissue Test (Oncology (solid organ), DNA, comprehensive genomic \n",
      "profiling, 257 genes, interrogation for single-nucleotide variants, insertions/deletions, copy \n",
      "number alterations, gene rearrangements, tumor-mutational burden and microsatellite instability, \n",
      "utilizing formalin-fixed paraffin embedded tumor tissue) \n",
      "0245U \n",
      "Oncology (thyroid), mutation analysis of 10 genes and 37 RNA fusions and expression of 4 \n",
      "mRNA markers using next-generation sequencing, fine needle aspirate, report includes \n",
      "associated risk of malignancy expressed as a percentage \n",
      "0250U \n",
      "Oncology (solid organ neoplasm), targeted genomic sequence DNA analysis of 505 genes, \n",
      "interrogation for somatic alterations (SNVs [single nucleotide variant], small insertions and \n",
      "deletions, one amplification, and four translocations), microsatellite instability and tumor-mutation \n",
      "burden \n",
      "0252U \n",
      "Fetal aneuploidy short tandem–repeat comparative analysis, fetal DNA from products of \n",
      "conception, reported as normal (euploidy), monosomy, trisomy, or partial deletion/duplications, \n",
      "mosaicism, and segmental aneuploidy \n",
      "0253U \n",
      "Reproductive medicine (endometrial receptivity analysis), RNA gene expression profile, 238 \n",
      "genes by next-generation sequencing, endometrial tissue, predictive algorithm reported as \n",
      "endometrial window of implantation (eg, pre-receptive, receptive, post-receptive) \n",
      "0254U \n",
      "Reproductive medicine (preimplantation genetic assessment), analysis of 24 chromosomes using \n",
      "embryonic DNA genomic sequence analysis for aneuploidy, and a mitochondrial DNA score in \n",
      " \n",
      "19 \n",
      " \n",
      "euploid embryos, results reported as normal (euploidy), monosomy, trisomy, or partial \n",
      "deletion/duplications, mosaicism, and segmental aneuploidy, per embryo tested \n",
      "0258U \n",
      "Autoimmune (psoriasis), mRNA, next generation sequencing, gene expression profiling of  \n",
      "50-100 genes, skin-surface collection using adhesive patch, algorithm reported as likelihood  \n",
      "of response to psoriasis biologics \n",
      "0260U \n",
      "Rare diseases (constitutional/heritable disorders), identification of copy number variations, \n",
      "inversions, insertions, translocations, and other structural variants by optical genome mapping \n",
      "0262U \n",
      "Oncology (solid tumor), gene expression profiling by real-time RT-PCR of 7 gene pathways  \n",
      "(ER, AR, PI3K, MAPK, HH, TGFB, Notch), formalin-fixed paraffin embedded (FFPE),  \n",
      "algorithm reported as gene pathway activity score \n",
      " \n",
      "0264U \n",
      "Rare diseases (constitutional/heritable disorders), identification of copy number variations, \n",
      "inversions, insertions, translocations, and other structural variants by optical genome mapping \n",
      "0265U \n",
      "Rare constitutional and other heritable disorders, whole genome and mitochondrial DNA \n",
      "sequence analysis, blood, frozen and formalin-fixed paraffin embedded (FFPE) tissue, saliva, \n",
      "buccal swabs or cell lines, identification of single nucleotide and copy number variants \n",
      "0266U \n",
      "Unexplained constitutional or other heritable disorders or syndromes, tissue specific gene \n",
      "expression by whole transcriptome and next-generation sequencing, blood, formalin-fixed  \n",
      "paraffin embedded (FFPE) tissue or fresh frozen tissue, reported as presence or absence of \n",
      "splicing or expression changes \n",
      "0267U \n",
      "Rare constitutional and other heritable disorders, identification of copy number variations, \n",
      "inversions, insertions, translocations, and other structural variants by optical genome mapping \n",
      "and whole genome sequencing \n",
      "0268U \n",
      "Hematology (atypical hemolytic uremic syndrome [aHUS]), genomic sequence analysis of 15 \n",
      "genes, blood, buccal swab, or amniotic fluid \n",
      "0269U \n",
      "Hematology (autosomal dominant congenital thrombocytopenia), genomic sequence analysis  \n",
      "of 14 genes, blood, buccal swab, or amniotic fluid \n",
      "0270U \n",
      "Hematology (congenital coagulation disorders), genomic sequence analysis of 20 genes,  \n",
      "blood, buccal swab, or amniotic fluid \n",
      "0271U \n",
      "Hematology (congenital neutropenia), genomic sequence analysis of 23 genes, blood, buccal \n",
      "swab, or amniotic fluid \n",
      "0272U \n",
      "Hematology (genetic bleeding disorders), genomic sequence analysis of 51 genes, blood,  \n",
      "buccal swab, or amniotic fluid, comprehensive \n",
      "0273U \n",
      "Hematology (genetic hyperfibrinolysis, delayed bleeding), genomic sequence analysis of 8  \n",
      "genes (F13A1, F13B, FGA, FGB, FGG, SERPINA1, SERPINE1, SERPINF2, PLAU), blood, \n",
      "buccal swab, or amniotic fluid \n",
      "0274U \n",
      "Hematology (genetic platelet disorders), genomic sequence analysis of 43 genes, blood,  \n",
      "buccal swab, or amniotic fluid \n",
      "0276U \n",
      "Hematology (inherited thrombocytopenia), genomic sequence analysis of 23 genes, blood,  \n",
      "buccal swab, or amniotic fluid \n",
      "0277U \n",
      "Hematology (genetic platelet function disorder), genomic sequence analysis of 31 genes,  \n",
      "blood, buccal swab, or amniotic fluid \n",
      "0278U \n",
      "Hematology (genetic thrombosis), genomic sequence analysis of 12 genes, blood, buccal  \n",
      "swab, or amniotic fluid \n",
      "0285U \n",
      "Oncology, response to radiation, cell-free DNA, quantitative branched chain DNA amplification, \n",
      "plasma, reported as a radiation toxicity score - RadTox™ cfDNA test \n",
      "0286U \n",
      "CEP72 (centrosomal protein, 72-KDa), NUDT15 (nudix hydrolase 15) and TPMT (thiopurine S-\n",
      "methyltransferase) (eg, drug metabolism) gene analysis, common variants - CNT (CEP72, \n",
      "NUDT15 and TPMT) Genotyping Panel \n",
      "0287U \n",
      "Oncology (thyroid), DNA and mRNA, nextgeneration sequencing analysis of 112 genes, fine \n",
      "needle aspirate or formalinfixed paraffin-embedded (FFPE) tissue, algorithmic prediction of \n",
      "cancer recurrence, reported as a categorical risk result (low, intermediate, high) - ThyroSeq® \n",
      "CRC \n",
      "0288U \n",
      "Oncology (lung), mRNA, quantitative PCR analysis of 11 genes (BAG1, BRCA1, CDC6, \n",
      "CDK2AP1, ERBB3, FUT3, IL11, LCK, RND3, SH3BGR, WNT3A) and 3 reference genes (ESD, \n",
      "TBP, YAP1), formalin-fixed paraffin-embedded (FFPE) tumor tissue, algorithmic interpretation \n",
      "reported as a recurrence risk score - DetermaRx™ \n",
      " \n",
      "20 \n",
      " \n",
      "0289U \n",
      "Neurology (Alzheimer disease), mRNA, gene expression profiling by RNA sequencing of 24 \n",
      "genes, whole blood, algorithm reported as predictive risk score - MindX Blood Test™ - \n",
      "Memory/Alzheimer's \n",
      " \n",
      "0290U \n",
      "Pain management, mRNA, gene expression profiling by RNA sequencing of 36 genes, whole \n",
      "blood, algorithm reported as predictive risk score - MindX Blood Test™ - Pain \n",
      "0291U \n",
      "Psychiatry (mood disorders), mRNA, gene expression profiling by RNA sequencing of 144 genes, \n",
      "whole blood, algorithm reported as predictive risk score - MindX Blood Test™ - Mood \n",
      "0292U \n",
      "Psychiatry (stress disorders), mRNA, gene expression profiling by RNA sequencing of 72 genes, \n",
      "whole blood, algorithm reported as predictive risk score - MindX Blood Test™ - Stress \n",
      "0293U \n",
      "Psychiatry (suicidal ideation), mRNA, gene expression profiling by RNA sequencing of 54 genes, \n",
      "whole blood, algorithm reported as predictive risk score - MindX Blood Test™ - Suicidality \n",
      "0294U \n",
      "Longevity and mortality risk, mRNA, gene expression profiling by RNA sequencing of 18 genes, \n",
      "whole blood, algorithm reported as predictive risk score - MindX Blood Test™ - Longevity \n",
      "0296U \n",
      "Oncology (oral and/or oropharyngeal cancer), gene expression profiling by RNA sequencing at \n",
      "least 20 molecular features (eg, human and/or microbial mRNA), saliva, algorithm reported as \n",
      "positive or negative for signature associated with malignancy - mRNA CancerDetect™ \n",
      "0297U \n",
      "Oncology (pan tumor), whole genome sequencing of paired malignant and normal DNA \n",
      "specimens, fresh or formalinfixed paraffin-embedded (FFPE) tissue, blood or bone marrow, \n",
      "comparative sequence analyses and variant identification - Praxis Somatic Whole Genome \n",
      "Sequencing \n",
      "0298U \n",
      "Oncology (pan tumor), whole transcriptome sequencing of paired malignant and normal RNA \n",
      "specimens, fresh or formalin-fixed paraffin-embedded (FFPE) tissue, blood or bone marrow, \n",
      "comparative sequence analyses and expression level and chimeric transcript identification - \n",
      "Praxis Somatic Transcriptome \n",
      "0299U \n",
      "Oncology (pan tumor), whole genome optical genome mapping of paired malignant and normal \n",
      "DNA specimens, fresh frozen tissue, blood, or bone marrow, comparative structural variant \n",
      "identification - Praxis Somatic Optical Genome Mapping \n",
      "0300U \n",
      "Oncology (pan tumor), whole genome sequencing and optical genome mapping of paired \n",
      "malignant and normal DNA specimens, fresh tissue, blood, or bone marrow, comparative \n",
      "sequence analyses and variant identification - Praxis Somatic Combined Whole Genome \n",
      "Sequencing and Optical Genome Mapping \n",
      "0306U \n",
      "Oncology (minimal residual disease [MRD]), next-generation targeted sequencing analysis,  \n",
      "cell-free DNA, initial (baseline) assessment to determine a patient specific panel for future \n",
      "comparisons to evaluate for MRD (Do not report 0306U in conjunction with 0307U) \n",
      "0307U \n",
      "Oncology (minimal residual disease [MRD]), next-generation targeted sequencing analysis of  \n",
      "a patient-specific panel, cell-free DNA, subsequent assessment with comparison to previously \n",
      "analyzed patient specimens to evaluate for MRD (Do not report 0307U in conjunction with  \n",
      "0306U \n",
      "0313U \n",
      "Oncology (pancreas), DNA and mRNA next-generation sequencing analysis of 74 genes and \n",
      "analysis of CEA (CEACAM5) gene expression, pancreatic cyst fluid, algorithm reported as a \n",
      "categorical result (ie, negative, low probability of neoplasia or positive, high probability of \n",
      "neoplasia) \n",
      "0314U \n",
      "Oncology (cutaneous melanoma), mRNA gene expression profiling by RT-PCR of 35 genes (32 \n",
      "content and 3 housekeeping), utilizing formalin-fixed paraffin-embedded (FFPE) tissue, algorithm \n",
      "reported as a categorical result (ie, benign, intermediate, malignant) \n",
      "0315U \n",
      "Oncology (cutaneous squamous cell carcinoma), mRNA gene expression profiling by RT-PCR of \n",
      "40 genes (34 content and 6 housekeeping), utilizing formalin-fixed paraffin embedded tissue, \n",
      "algorithm reported as a categorical risk result (ie, Class 1, Class 2A, Class 2B) \n",
      "0317U \n",
      "Oncology (lung cancer), four-probe FISH (3q29, 3p22.1, 10q22.3, 10cen) assay, whole blood, \n",
      "predictive algorithm generated evaluation reported as decreased or increased risk for lung  \n",
      "cancer \n",
      "0318U \n",
      "Pediatrics (congenital epigenetic disorders), whole genome methylation analysis by microarray \n",
      "for 50 or more genes, blood \n",
      "0319U \n",
      "Nephrology (renal transplant), RNA expression by select transcriptome sequencing, using \n",
      "pretransplant peripheral blood, algorithm reported as a risk score for early acute rejection \n",
      " \n",
      "21 \n",
      " \n",
      "0320U \n",
      "Nephrology (renal transplant), RNA expression by select transcriptome sequencing, using \n",
      "posttransplant peripheral blood, algorithm reported as a risk score for acute cellular rejection \n",
      "0326U \n",
      "Targeted genomic sequence analysis panel, solid organ neoplasm, cell-free circulating DNA \n",
      "analysis of 83 or more genes, interrogation for sequence variants, gene copy number \n",
      "amplifications, gene rearrangements, microsatellite instability and tumor mutational burden \n",
      "0327U  \n",
      "Fetal aneuploidy (trisomy 13, 18, and 21), DNA sequence analysis of selected regions using \n",
      "maternal plasma, algorithm reported as a risk score for each trisomy, includes sex reporting, if \n",
      "performed \n",
      "0329U \n",
      "Oncology (neoplasia), exome and transcriptome sequence analysis for sequence variants,  \n",
      "gene copy number amplifications and deletions, gene rearrangements, microsatellite instability \n",
      "and tumor mutational burden utilizing DNA and RNA from tumor with DNA from normal blood  \n",
      "or saliva for subtraction, report of clinically significant mutation(s) with therapy associations \n",
      "0331U  \n",
      "Oncology (hematolymphoid neoplasia), optical genome mapping for copy number alterations  \n",
      "and gene rearrangements utilizing DNA from blood or bone marrow, report of clinically  \n",
      "significant alternations \n",
      "0332U \n",
      "Oncology (pan-tumor), genetic profiling of 8 DNA-regulatory (epigenetic) markers by quantitative \n",
      "polymerase chain reaction (qPCR), whole blood, reported as a high or low probability of \n",
      "responding to immune checkpoint–inhibitor therapy \n",
      "0333U \n",
      "Oncology (liver), surveillance for hepatocellular carcinoma (HCC) in highrisk patients, analysis of \n",
      "methylation patterns on circulating cell-free DNA (cfDNA) plus measurement of serum of \n",
      "AFP/AFP-L3 and oncoprotein des-gammacarboxy-prothrombin (DCP), algorithm reported as \n",
      "normal or abnormal result \n",
      "0334U \n",
      "Oncology (solid organ), targeted genomic sequence analysis, formalin-fixed paraffinembedded \n",
      "(FFPE) tumor tissue, DNA analysis, 84 or more genes, interrogation for sequence variants, gene \n",
      "copy number amplifications, gene rearrangements, microsatellite instability and tumor mutational \n",
      "burden \n",
      "0335U \n",
      "Rare diseases (constitutional/heritable disorders), whole genome sequence analysis, including \n",
      "small sequence changes, copy number variants, deletions, duplications, mobile element \n",
      "insertions, uniparental disomy (UPD), inversions, aneuploidy, mitochondrial genome sequence \n",
      "analysis with heteroplasmy and large deletions, short tandem repeat (STR) gene expansions, \n",
      "fetal sample, identification and categorization of genetic variants (Do not report 0335U in \n",
      "conjunction with 81425, 0212U) \n",
      "0336U \n",
      "Rare diseases (constitutional/heritable disorders), whole genome sequence analysis, including \n",
      "small sequence changes, copy number variants, deletions, duplications, mobile element \n",
      "insertions, uniparental disomy (UPD), inversions, aneuploidy, mitochondrial genome sequence \n",
      "analysis with heteroplasmy and large deletions, short tandem repeat (STR) gene expansions, \n",
      "blood or saliva, identification and categorization of genetic variants, each comparator genome \n",
      "(eg, parent) (Do not report 0336U in conjunction with 81426, 0213U) \n",
      "0339U \n",
      "Oncology (prostate), mRNA expression profiling of HOXC6 and DLX1, reverse transcription \n",
      "polymerase chain reaction (RT-PCR), first-void urine following digital rectal examination, \n",
      "algorithm reported as probability of high-grade cancer \n",
      "0340U \n",
      "Oncology (pan-cancer), analysis of minimal residual disease (MRD) from plasma, with assays \n",
      "personalized to each patient based on prior next-generation sequencing of the patient’s tumor \n",
      "and germline DNA, reported as absence or presence of MRD, with disease-burden correlation, if \n",
      "appropriate \n",
      "0341U \n",
      "Fetal aneuploidy DNA sequencing comparative analysis, fetal DNA from products of conception, \n",
      "reported as normal (euploidy), monosomy, trisomy, or partial deletion/duplication, mosaicism, and \n",
      "segmental aneuploidy \n",
      "0343U \n",
      "Oncology (prostate), exosome-based analysis of 442 small noncoding RNAs (sncRNAs) by \n",
      "quantitative reverse transcription polymerase chain reaction (RT-qPCR), urine, reported as \n",
      "molecular evidence of no-, low-, intermediate- or high-risk of prostate cancer \n",
      "0345U \n",
      "Psychiatry (eg, depression, anxiety, attention deficit hyperactivity disorder [ADHD]), genomic \n",
      "analysis panel, variant analysis of 15 genes, including deletion/duplication analysis of CYP2D6 \n",
      "0347U \n",
      "Drug metabolism or processing (multiple conditions), whole blood or buccal specimen, DNA \n",
      "analysis, 16 gene report, with variant analysis and reported phenotypes \n",
      "0348U \n",
      "Drug metabolism or processing (multiple conditions), whole blood or buccal specimen, DNA \n",
      "analysis, 25 gene report, with variant analysis and reported phenotypes \n",
      " \n",
      "22 \n",
      " \n",
      "0349U \n",
      "Drug metabolism or processing (multiple conditions), whole blood or buccal specimen, DNA \n",
      "analysis, 27 gene report, with variant analysis, including reported phenotypes and impacted \n",
      "gene-drug interactions \n",
      "0350U \n",
      "Drug metabolism or processing (multiple conditions), whole blood or buccal specimen, DNA \n",
      "analysis, 27 gene report, with variant analysis and reported phenotypes \n",
      "0355U \n",
      "APOL1 (apolipoprotein L1) (eg, chronic kidney disease), risk variants (G1, G2) \n",
      "0356U \n",
      "Oncology (oropharyngeal), evaluation of 17 DNA biomarkers using droplet digital PCR  \n",
      "(ddPCR), cell-free DNA, algorithm reported as a prognostic risk score for cancer recurrence \n",
      "0362U \n",
      "Oncology (papillary thyroid cancer), gene-expression profiling via targeted hybrid capture–\n",
      "enrichment RNA sequencing of 82 content genes and 10 housekeeping genes, formalin-fixed \n",
      "paraffin embedded (FFPE) tissue, algorithm reported as one of three molecular subtypes \n",
      " \n",
      "0363U \n",
      "Oncology (urothelial), mRNA, geneexpression profiling by real-time quantitative PCR of 5  \n",
      "genes (MDK, HOXA13, CDC2 [CDK1], IGFBP5, and CXCR2), utilizing urine, algorithm \n",
      "incorporates age, sex, smoking history, and macrohematuria frequency, reported as a risk  \n",
      "score for having urothelial carcinoma \n",
      " \n",
      "HCPCS Codes \n",
      "HCPCS  \n",
      "codes \n",
      "Code Description \n",
      "G9143 \n",
      "Warfarin responsiveness testing by genetic technique using any method, any number of \n",
      "specimen(s) \n",
      "S3800 \n",
      "Genetic testing for amyotrophic lateral sclerosis (ALS) \n",
      "S3840 \n",
      "DNA analysis for germline mutations of the RET proto-oncogene for susceptibility to multiple \n",
      "endocrine neoplasia type 2 \n",
      "S3841 \n",
      "Genetic testing for retinoblastoma \n",
      "S3842 \n",
      "Genetic testing for Von Hippel-Lindau disease \n",
      "S3844 \n",
      "DNA analysis of the connexin 26 gene (GJB2) for susceptibility to congenital, profound \n",
      "deafness \n",
      "S3845 \n",
      "Genetic testing for alpha-thalassemia \n",
      "S3846 \n",
      "Genetic testing for hemoglobin E beta-thalassemia \n",
      "S3849 \n",
      "Genetic testing for Niemann-Pick disease \n",
      "S3850 \n",
      "Genetic testing for sickle cell anemia \n",
      "S3852 \n",
      "DNA analysis for APOE epsilon 4 allele for susceptibility to Alzheimer's disease \n",
      "S3853 \n",
      "Genetic testing for myotonic muscular dystrophy \n",
      "S3854 \n",
      "Gene expression profiling panel for use in the management of breast cancer treatment \n",
      "S3861 \n",
      "Genetic testing, sodium channel, voltage-gated, type V, alpha subunit (SCN5A) and variants \n",
      "for suspected Brugada Syndrome \n",
      "S3865 \n",
      "Comprehensive gene sequence analysis for hypertrophic cardiomyopathy \n",
      "S3866 \n",
      "Genetic analysis for a specific gene mutation for hypertrophic cardiomyopathy (HCM) in an \n",
      "individual with a known HCM mutation in the family \n",
      "S3870 \n",
      "Comparative genomic hybridization (CGH) microarray testing for developmental delay, \n",
      "autism spectrum disorder and/or intellectual disability \n",
      " \n",
      "Policy History \n",
      "1/2023 \n",
      "CPT codes were added: 0355U; 0356U; 0362U; 0363U; 81418; 81441; 81449; 81451; \n",
      "81456.  Effective 1/1/2023. \n",
      "10/2022 \n",
      "CPT codes were deleted (retired codes).  0012U; 0013U; 0014U; 0056U.  Effective \n",
      "9/30/2022. CPT codes were added: 0332U;0333U;0334U;0335U;0336U;0339U;0340U \n",
      "0341U;0343U;0345U;0347U;0348U;0349U; 0350U. Effective 10/1/2022. \n",
      "7/2022 \n",
      "CPT codes were added: 0326U; 0327U; 0329U; 0331U.  Effective 7/1/2022. \n",
      " \n",
      "4/2022 \n",
      "CPT codes were added: 0306U; 0307U; 0313U; 0314U; 0315U; 0317U; 0318U; 0319U; \n",
      "0320U.  Effective 4/1/2022. \n",
      "1/2022 \n",
      "CPT codes were added: 81349; 81523; 0285U;0286U;0287U;0288U;0289U;0290U;0291U \n",
      "0292U;0293U;0294U;0296U;0297U; 0298U;0299U;0300U.  Effective 1/1/2022. \n",
      " \n",
      "23 \n",
      " \n",
      "10/2021 \n",
      "CPT codes were added: 0258U; 0260U; 0262U; 0264U; 0265U; 0266U; 0267U; 0268U; \n",
      "0269U; 0270U; 0271U; 0272U; 0273U; 0274U; 0276U; 0277U; 0278U.  Deleted code 0168U \n",
      "Effective 10/1/2021. \n",
      "7/2021 \n",
      "CPT codes were added: 0250U; 0252U; 0253U; 0254U.  Effective 7/1/2021. \n",
      "4/2021 \n",
      "CPT codes were added: 0242U;0245U. \n",
      "3/2021 \n",
      "CPT code was added: 0017M \n",
      "1/2021 \n",
      "CPT codes were added:  81168; 81191; 81192; 81193; 81194; 81278; 81279; 81338; 81339; \n",
      "81347; 81348; 81351; 81352; 81353; 81357; 81360; 81419; 81529; 81546; 81554; 0228U; \n",
      "0229U; 0230U; 0231U; 0232U; 0233U; 0234U; 0235U; 0236U; 0237U; 0238U; 0239U.  \n",
      "Deleted codes 81545 and 0009M. Effective 1/1/2021. \n",
      "10/2020 \n",
      "CPT codes were added:  0016M; 0203U; 0204U; 0205U; 0208U; 0209U; 0211U; 0212U; \n",
      "0213U; 0214U; 0215U; 0216U; 0217U; 0218U.  Effective 10/1/2020. \n",
      "7/2020 \n",
      "CPT codes were added:  0172U, 0173U, 0175U, 0177U, 0179U.  Effective 7/1/2020. \n",
      "4/2020 \n",
      "CPT codes were added: 0168U;0169U; 0170U;0171U.  Effective 4/1/2020. \n",
      "1/2020 \n",
      "CPT codes were added: 0153U; 0154U; 0155U; 0156U; 0157U; 0158U; 0159U; 0160U; \n",
      "0161U; 0162U; 81277; 81307; 81308; 81309; 81522; 81542; 81552.  Effective 1/1/2020. \n",
      "10/2019 \n",
      "CPT codes were added: 0111U; 0113U; 0114U; 0118U; 0120U; 0129U; 0130U; 0131U; \n",
      "0132U; 0133U; 0134U; 0135U; 0136U; 0137U; 0138U.  Deleted code: 0104U. Effective \n",
      "10/1/2019. \n",
      "7/2019 \n",
      "CPT codes were added: 0087U; 0088U; 0089U; 0090U; 0094U; 0101U; 0102U; 0103U; \n",
      "0104U. Deleted code 0057U. Effective 7/1/2019. \n",
      "3/2019 \n",
      "HCPCS code S3620 removed. 3/22/2019 \n",
      "3/2019 \n",
      "HCPCS codes G0452; G9840 & G9841 removed. \n",
      "1/2019 \n",
      "New 1/1/2019 CPT codes added. Code 0081U added. Deleted codes:  81211, 81213, 81214.  \n",
      "Effective 1/1/2019. \n",
      "11/2018 \n",
      "New document #957 issued.  Effective 1/1/2019 \n",
      " \n",
      "Disclaimer:  \n",
      "Coverage is subject to applicable benefit contract. Specific benefits may vary by product and/or employer \n",
      "group. Please reference appropriate member materials (e.g., Benefit Handbook, Certificate of Coverage) for \n",
      "member-specific benefit information.  \n",
      " \n",
      "Member’s medical records must document that services are medically necessary for the care provided. BCBS \n",
      "MA maintains the right to audit the services provided to our members, regardless of the participation status of \n",
      "the provider. All documentation must be available upon request. Failure to produce the requested information \n",
      "may result in denial or retraction of payment. \n",
      " \n",
      "\n"
     ]
    }
   ],
   "source": [
    "extract_title_text(url)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 112,
   "id": "3cdea132",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "ename": "ConnectionError",
     "evalue": "HTTPSConnectionPool(host='www.bcbsms.com', port=443): Max retries exceeded with url: /medical-policy-search (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001B786B09580>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[1;31mTimeoutError\u001b[0m                              Traceback (most recent call last)",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\urllib3\\connection.py:174\u001b[0m, in \u001b[0;36mHTTPConnection._new_conn\u001b[1;34m(self)\u001b[0m\n\u001b[0;32m    173\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[1;32m--> 174\u001b[0m     conn \u001b[38;5;241m=\u001b[39m connection\u001b[38;5;241m.\u001b[39mcreate_connection(\n\u001b[0;32m    175\u001b[0m         (\u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_dns_host, \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mport), \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mtimeout, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39mextra_kw\n\u001b[0;32m    176\u001b[0m     )\n\u001b[0;32m    178\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m SocketTimeout:\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\urllib3\\util\\connection.py:95\u001b[0m, in \u001b[0;36mcreate_connection\u001b[1;34m(address, timeout, source_address, socket_options)\u001b[0m\n\u001b[0;32m     94\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m err \u001b[38;5;129;01mis\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m \u001b[38;5;28;01mNone\u001b[39;00m:\n\u001b[1;32m---> 95\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m err\n\u001b[0;32m     97\u001b[0m \u001b[38;5;28;01mraise\u001b[39;00m socket\u001b[38;5;241m.\u001b[39merror(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mgetaddrinfo returns an empty list\u001b[39m\u001b[38;5;124m\"\u001b[39m)\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\urllib3\\util\\connection.py:85\u001b[0m, in \u001b[0;36mcreate_connection\u001b[1;34m(address, timeout, source_address, socket_options)\u001b[0m\n\u001b[0;32m     84\u001b[0m     sock\u001b[38;5;241m.\u001b[39mbind(source_address)\n\u001b[1;32m---> 85\u001b[0m \u001b[43msock\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mconnect\u001b[49m\u001b[43m(\u001b[49m\u001b[43msa\u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m     86\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m sock\n",
      "\u001b[1;31mTimeoutError\u001b[0m: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond",
      "\nDuring handling of the above exception, another exception occurred:\n",
      "\u001b[1;31mNewConnectionError\u001b[0m                        Traceback (most recent call last)",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\urllib3\\connectionpool.py:703\u001b[0m, in \u001b[0;36mHTTPConnectionPool.urlopen\u001b[1;34m(self, method, url, body, headers, retries, redirect, assert_same_host, timeout, pool_timeout, release_conn, chunked, body_pos, **response_kw)\u001b[0m\n\u001b[0;32m    702\u001b[0m \u001b[38;5;66;03m# Make the request on the httplib connection object.\u001b[39;00m\n\u001b[1;32m--> 703\u001b[0m httplib_response \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_make_request\u001b[49m\u001b[43m(\u001b[49m\n\u001b[0;32m    704\u001b[0m \u001b[43m    \u001b[49m\u001b[43mconn\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    705\u001b[0m \u001b[43m    \u001b[49m\u001b[43mmethod\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    706\u001b[0m \u001b[43m    \u001b[49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    707\u001b[0m \u001b[43m    \u001b[49m\u001b[43mtimeout\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mtimeout_obj\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    708\u001b[0m \u001b[43m    \u001b[49m\u001b[43mbody\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mbody\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    709\u001b[0m \u001b[43m    \u001b[49m\u001b[43mheaders\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mheaders\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    710\u001b[0m \u001b[43m    \u001b[49m\u001b[43mchunked\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mchunked\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    711\u001b[0m \u001b[43m\u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m    713\u001b[0m \u001b[38;5;66;03m# If we're going to release the connection in ``finally:``, then\u001b[39;00m\n\u001b[0;32m    714\u001b[0m \u001b[38;5;66;03m# the response doesn't need to know about the connection. Otherwise\u001b[39;00m\n\u001b[0;32m    715\u001b[0m \u001b[38;5;66;03m# it will also try to release it and we'll have a double-release\u001b[39;00m\n\u001b[0;32m    716\u001b[0m \u001b[38;5;66;03m# mess.\u001b[39;00m\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\urllib3\\connectionpool.py:386\u001b[0m, in \u001b[0;36mHTTPConnectionPool._make_request\u001b[1;34m(self, conn, method, url, timeout, chunked, **httplib_request_kw)\u001b[0m\n\u001b[0;32m    385\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[1;32m--> 386\u001b[0m     \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_validate_conn\u001b[49m\u001b[43m(\u001b[49m\u001b[43mconn\u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m    387\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m (SocketTimeout, BaseSSLError) \u001b[38;5;28;01mas\u001b[39;00m e:\n\u001b[0;32m    388\u001b[0m     \u001b[38;5;66;03m# Py2 raises this as a BaseSSLError, Py3 raises it as socket timeout.\u001b[39;00m\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\urllib3\\connectionpool.py:1042\u001b[0m, in \u001b[0;36mHTTPSConnectionPool._validate_conn\u001b[1;34m(self, conn)\u001b[0m\n\u001b[0;32m   1041\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m \u001b[38;5;28mgetattr\u001b[39m(conn, \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124msock\u001b[39m\u001b[38;5;124m\"\u001b[39m, \u001b[38;5;28;01mNone\u001b[39;00m):  \u001b[38;5;66;03m# AppEngine might not have  `.sock`\u001b[39;00m\n\u001b[1;32m-> 1042\u001b[0m     \u001b[43mconn\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mconnect\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m   1044\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m conn\u001b[38;5;241m.\u001b[39mis_verified:\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\urllib3\\connection.py:363\u001b[0m, in \u001b[0;36mHTTPSConnection.connect\u001b[1;34m(self)\u001b[0m\n\u001b[0;32m    361\u001b[0m \u001b[38;5;28;01mdef\u001b[39;00m \u001b[38;5;21mconnect\u001b[39m(\u001b[38;5;28mself\u001b[39m):\n\u001b[0;32m    362\u001b[0m     \u001b[38;5;66;03m# Add certificate verification\u001b[39;00m\n\u001b[1;32m--> 363\u001b[0m     \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39msock \u001b[38;5;241m=\u001b[39m conn \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_new_conn\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m    364\u001b[0m     hostname \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mhost\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\urllib3\\connection.py:186\u001b[0m, in \u001b[0;36mHTTPConnection._new_conn\u001b[1;34m(self)\u001b[0m\n\u001b[0;32m    185\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m SocketError \u001b[38;5;28;01mas\u001b[39;00m e:\n\u001b[1;32m--> 186\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m NewConnectionError(\n\u001b[0;32m    187\u001b[0m         \u001b[38;5;28mself\u001b[39m, \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mFailed to establish a new connection: \u001b[39m\u001b[38;5;132;01m%s\u001b[39;00m\u001b[38;5;124m\"\u001b[39m \u001b[38;5;241m%\u001b[39m e\n\u001b[0;32m    188\u001b[0m     )\n\u001b[0;32m    190\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m conn\n",
      "\u001b[1;31mNewConnectionError\u001b[0m: <urllib3.connection.HTTPSConnection object at 0x000001B786B09580>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond",
      "\nDuring handling of the above exception, another exception occurred:\n",
      "\u001b[1;31mMaxRetryError\u001b[0m                             Traceback (most recent call last)",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\requests\\adapters.py:489\u001b[0m, in \u001b[0;36mHTTPAdapter.send\u001b[1;34m(self, request, stream, timeout, verify, cert, proxies)\u001b[0m\n\u001b[0;32m    488\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m chunked:\n\u001b[1;32m--> 489\u001b[0m     resp \u001b[38;5;241m=\u001b[39m \u001b[43mconn\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43murlopen\u001b[49m\u001b[43m(\u001b[49m\n\u001b[0;32m    490\u001b[0m \u001b[43m        \u001b[49m\u001b[43mmethod\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mrequest\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mmethod\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    491\u001b[0m \u001b[43m        \u001b[49m\u001b[43murl\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    492\u001b[0m \u001b[43m        \u001b[49m\u001b[43mbody\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mrequest\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mbody\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    493\u001b[0m \u001b[43m        \u001b[49m\u001b[43mheaders\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mrequest\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mheaders\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    494\u001b[0m \u001b[43m        \u001b[49m\u001b[43mredirect\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43;01mFalse\u001b[39;49;00m\u001b[43m,\u001b[49m\n\u001b[0;32m    495\u001b[0m \u001b[43m        \u001b[49m\u001b[43massert_same_host\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43;01mFalse\u001b[39;49;00m\u001b[43m,\u001b[49m\n\u001b[0;32m    496\u001b[0m \u001b[43m        \u001b[49m\u001b[43mpreload_content\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43;01mFalse\u001b[39;49;00m\u001b[43m,\u001b[49m\n\u001b[0;32m    497\u001b[0m \u001b[43m        \u001b[49m\u001b[43mdecode_content\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43;01mFalse\u001b[39;49;00m\u001b[43m,\u001b[49m\n\u001b[0;32m    498\u001b[0m \u001b[43m        \u001b[49m\u001b[43mretries\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mmax_retries\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    499\u001b[0m \u001b[43m        \u001b[49m\u001b[43mtimeout\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mtimeout\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    500\u001b[0m \u001b[43m    \u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m    502\u001b[0m \u001b[38;5;66;03m# Send the request.\u001b[39;00m\n\u001b[0;32m    503\u001b[0m \u001b[38;5;28;01melse\u001b[39;00m:\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\urllib3\\connectionpool.py:787\u001b[0m, in \u001b[0;36mHTTPConnectionPool.urlopen\u001b[1;34m(self, method, url, body, headers, retries, redirect, assert_same_host, timeout, pool_timeout, release_conn, chunked, body_pos, **response_kw)\u001b[0m\n\u001b[0;32m    785\u001b[0m     e \u001b[38;5;241m=\u001b[39m ProtocolError(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mConnection aborted.\u001b[39m\u001b[38;5;124m\"\u001b[39m, e)\n\u001b[1;32m--> 787\u001b[0m retries \u001b[38;5;241m=\u001b[39m \u001b[43mretries\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mincrement\u001b[49m\u001b[43m(\u001b[49m\n\u001b[0;32m    788\u001b[0m \u001b[43m    \u001b[49m\u001b[43mmethod\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43merror\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43me\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43m_pool\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43mself\u001b[39;49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43m_stacktrace\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43msys\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mexc_info\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\u001b[43m[\u001b[49m\u001b[38;5;241;43m2\u001b[39;49m\u001b[43m]\u001b[49m\n\u001b[0;32m    789\u001b[0m \u001b[43m\u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m    790\u001b[0m retries\u001b[38;5;241m.\u001b[39msleep()\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\urllib3\\util\\retry.py:592\u001b[0m, in \u001b[0;36mRetry.increment\u001b[1;34m(self, method, url, response, error, _pool, _stacktrace)\u001b[0m\n\u001b[0;32m    591\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m new_retry\u001b[38;5;241m.\u001b[39mis_exhausted():\n\u001b[1;32m--> 592\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m MaxRetryError(_pool, url, error \u001b[38;5;129;01mor\u001b[39;00m ResponseError(cause))\n\u001b[0;32m    594\u001b[0m log\u001b[38;5;241m.\u001b[39mdebug(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mIncremented Retry for (url=\u001b[39m\u001b[38;5;124m'\u001b[39m\u001b[38;5;132;01m%s\u001b[39;00m\u001b[38;5;124m'\u001b[39m\u001b[38;5;124m): \u001b[39m\u001b[38;5;132;01m%r\u001b[39;00m\u001b[38;5;124m\"\u001b[39m, url, new_retry)\n",
      "\u001b[1;31mMaxRetryError\u001b[0m: HTTPSConnectionPool(host='www.bcbsms.com', port=443): Max retries exceeded with url: /medical-policy-search (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001B786B09580>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))",
      "\nDuring handling of the above exception, another exception occurred:\n",
      "\u001b[1;31mConnectionError\u001b[0m                           Traceback (most recent call last)",
      "Cell \u001b[1;32mIn[112], line 1\u001b[0m\n\u001b[1;32m----> 1\u001b[0m response \u001b[38;5;241m=\u001b[39m \u001b[43mrequests\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mget\u001b[49m\u001b[43m(\u001b[49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mheaders\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mheaders\u001b[49m\u001b[43m)\u001b[49m  \n\u001b[0;32m      2\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m response\u001b[38;5;241m.\u001b[39mstatus_code\u001b[38;5;241m!=\u001b[39m\u001b[38;5;241m200\u001b[39m:\n\u001b[0;32m      3\u001b[0m          \u001b[38;5;28mprint\u001b[39m(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mLink not working\u001b[39m\u001b[38;5;124m\"\u001b[39m)\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\requests\\api.py:73\u001b[0m, in \u001b[0;36mget\u001b[1;34m(url, params, **kwargs)\u001b[0m\n\u001b[0;32m     62\u001b[0m \u001b[38;5;28;01mdef\u001b[39;00m \u001b[38;5;21mget\u001b[39m(url, params\u001b[38;5;241m=\u001b[39m\u001b[38;5;28;01mNone\u001b[39;00m, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39mkwargs):\n\u001b[0;32m     63\u001b[0m     \u001b[38;5;124mr\u001b[39m\u001b[38;5;124;03m\"\"\"Sends a GET request.\u001b[39;00m\n\u001b[0;32m     64\u001b[0m \n\u001b[0;32m     65\u001b[0m \u001b[38;5;124;03m    :param url: URL for the new :class:`Request` object.\u001b[39;00m\n\u001b[1;32m   (...)\u001b[0m\n\u001b[0;32m     70\u001b[0m \u001b[38;5;124;03m    :rtype: requests.Response\u001b[39;00m\n\u001b[0;32m     71\u001b[0m \u001b[38;5;124;03m    \"\"\"\u001b[39;00m\n\u001b[1;32m---> 73\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m request(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mget\u001b[39m\u001b[38;5;124m\"\u001b[39m, url, params\u001b[38;5;241m=\u001b[39mparams, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39mkwargs)\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\requests\\api.py:59\u001b[0m, in \u001b[0;36mrequest\u001b[1;34m(method, url, **kwargs)\u001b[0m\n\u001b[0;32m     55\u001b[0m \u001b[38;5;66;03m# By using the 'with' statement we are sure the session is closed, thus we\u001b[39;00m\n\u001b[0;32m     56\u001b[0m \u001b[38;5;66;03m# avoid leaving sockets open which can trigger a ResourceWarning in some\u001b[39;00m\n\u001b[0;32m     57\u001b[0m \u001b[38;5;66;03m# cases, and look like a memory leak in others.\u001b[39;00m\n\u001b[0;32m     58\u001b[0m \u001b[38;5;28;01mwith\u001b[39;00m sessions\u001b[38;5;241m.\u001b[39mSession() \u001b[38;5;28;01mas\u001b[39;00m session:\n\u001b[1;32m---> 59\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m session\u001b[38;5;241m.\u001b[39mrequest(method\u001b[38;5;241m=\u001b[39mmethod, url\u001b[38;5;241m=\u001b[39murl, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39mkwargs)\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\requests\\sessions.py:587\u001b[0m, in \u001b[0;36mSession.request\u001b[1;34m(self, method, url, params, data, headers, cookies, files, auth, timeout, allow_redirects, proxies, hooks, stream, verify, cert, json)\u001b[0m\n\u001b[0;32m    582\u001b[0m send_kwargs \u001b[38;5;241m=\u001b[39m {\n\u001b[0;32m    583\u001b[0m     \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mtimeout\u001b[39m\u001b[38;5;124m\"\u001b[39m: timeout,\n\u001b[0;32m    584\u001b[0m     \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mallow_redirects\u001b[39m\u001b[38;5;124m\"\u001b[39m: allow_redirects,\n\u001b[0;32m    585\u001b[0m }\n\u001b[0;32m    586\u001b[0m send_kwargs\u001b[38;5;241m.\u001b[39mupdate(settings)\n\u001b[1;32m--> 587\u001b[0m resp \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39msend(prep, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39msend_kwargs)\n\u001b[0;32m    589\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m resp\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\requests\\sessions.py:701\u001b[0m, in \u001b[0;36mSession.send\u001b[1;34m(self, request, **kwargs)\u001b[0m\n\u001b[0;32m    698\u001b[0m start \u001b[38;5;241m=\u001b[39m preferred_clock()\n\u001b[0;32m    700\u001b[0m \u001b[38;5;66;03m# Send the request\u001b[39;00m\n\u001b[1;32m--> 701\u001b[0m r \u001b[38;5;241m=\u001b[39m adapter\u001b[38;5;241m.\u001b[39msend(request, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39mkwargs)\n\u001b[0;32m    703\u001b[0m \u001b[38;5;66;03m# Total elapsed time of the request (approximately)\u001b[39;00m\n\u001b[0;32m    704\u001b[0m elapsed \u001b[38;5;241m=\u001b[39m preferred_clock() \u001b[38;5;241m-\u001b[39m start\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\requests\\adapters.py:565\u001b[0m, in \u001b[0;36mHTTPAdapter.send\u001b[1;34m(self, request, stream, timeout, verify, cert, proxies)\u001b[0m\n\u001b[0;32m    561\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28misinstance\u001b[39m(e\u001b[38;5;241m.\u001b[39mreason, _SSLError):\n\u001b[0;32m    562\u001b[0m         \u001b[38;5;66;03m# This branch is for urllib3 v1.22 and later.\u001b[39;00m\n\u001b[0;32m    563\u001b[0m         \u001b[38;5;28;01mraise\u001b[39;00m SSLError(e, request\u001b[38;5;241m=\u001b[39mrequest)\n\u001b[1;32m--> 565\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m \u001b[38;5;167;01mConnectionError\u001b[39;00m(e, request\u001b[38;5;241m=\u001b[39mrequest)\n\u001b[0;32m    567\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m ClosedPoolError \u001b[38;5;28;01mas\u001b[39;00m e:\n\u001b[0;32m    568\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m \u001b[38;5;167;01mConnectionError\u001b[39;00m(e, request\u001b[38;5;241m=\u001b[39mrequest)\n",
      "\u001b[1;31mConnectionError\u001b[0m: HTTPSConnectionPool(host='www.bcbsms.com', port=443): Max retries exceeded with url: /medical-policy-search (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000001B786B09580>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))"
     ]
    }
   ],
   "source": [
    "response = requests.get(url, headers=headers)  \n",
    "if response.status_code!=200:\n",
    "         print(\"Link not working\")\n",
    "else:\n",
    "    print('Connection setup problem')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "59064f61",
   "metadata": {},
   "outputs": [],
   "source": [
    "extract_html_file_url()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 69,
   "id": "54a52c77",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://www.aetna.com/cpb/medical/data/300_399/0352.html\n",
      "https://www.aetna.com/cpb/medical/data/300_399/0352.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/300_399/0352.html\n",
      "https://www.aetna.com/cpb/medical/data/300_399/0352.html\n",
      "https://www.aetna.com/cpb/medical/data/300_399/0352.html\n",
      "https://www.aetna.com/cpb/medical/data/300_399/0352.html\n",
      "https://www.aetna.com/cpb/medical/data/300_399/0352.html\n",
      "https://www.aetna.com/cpb/medical/data/300_399/0352.html\n",
      "https://www.aetna.com/cpb/medical/data/300_399/0352.html\n",
      "https://www.aetna.com/cpb/medical/data/300_399/0352.html\n",
      "https://www.aetna.com/cpb/medical/data/300_399/0352.html\n",
      "https://www.aetna.com/cpb/medical/data/500_599/0586.html\n",
      "https://www.aetna.com/cpb/medical/data/200_299/0227.html\n",
      "https://www.aetna.com/cpb/medical/data/300_399/0352.html\n",
      "https://www.aetna.com/cpb/medical/data/300_399/0352.html\n",
      "https://www.aetna.com/cpb/medical/data/300_399/0352.html\n",
      "https://www.aetna.com/cpb/medical/data/600_699/0698.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/300_399/0381.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/300_399/0352.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/900_999/0911.html\n",
      "https://www.aetna.com/cpb/medical/data/300_399/0352.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/800_899/0866.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/800_899/0835.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/600_699/0650.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/300_399/0352.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/300_399/0352.html\n",
      "https://www.aetna.com/cpb/medical/data/500_599/0516.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0188.html\n",
      "https://www.aetna.com/cpb/medical/data/300_399/0352.html\n",
      "https://www.aetna.com/cpb/medical/data/300_399/0352.html\n",
      "https://www.aetna.com/cpb/medical/data/300_399/0352.html\n",
      "https://www.aetna.com/cpb/medical/data/300_399/0352.html\n",
      "https://www.aetna.com/cpb/medical/data/300_399/0352.html\n",
      "https://www.aetna.com/cpb/medical/data/200_299/0227.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/300_399/0352.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/300_399/0352.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/200_299/0227.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/300_399/0352.html\n",
      "https://www.aetna.com/cpb/medical/data/200_299/0227.html\n",
      "https://www.aetna.com/cpb/medical/data/600_699/0650.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/600_699/0690.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/1_99/0038.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/300_399/0352.html\n",
      "https://www.aetna.com/cpb/medical/data/300_399/0358.html\n",
      "https://www.aetna.com/cpb/medical/data/300_399/0352.html\n",
      "https://www.aetna.com/cpb/medical/data/300_399/0352.html\n",
      "https://www.aetna.com/cpb/medical/data/700_799/0715.html\n",
      "https://www.aetna.com/cpb/medical/data/500_599/0521.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/700_799/0715.html\n",
      "https://www.aetna.com/cpb/medical/data/700_799/0715.html\n",
      "https://www.aetna.com/cpb/medical/data/300_399/0352.html\n",
      "https://www.aetna.com/cpb/medical/data/400_499/0464.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/700_799/0715.html\n",
      "https://www.aetna.com/cpb/medical/data/200_299/0227.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/200_299/0227.html\n",
      "https://www.aetna.com/cpb/medical/data/300_399/0352.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/700_799/0715.html\n",
      "https://www.aetna.com/cpb/medical/data/400_499/0464.html\n",
      "https://www.aetna.com/cpb/medical/data/400_499/0464.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/400_499/0464.html\n",
      "https://www.aetna.com/cpb/medical/data/600_699/0650.html\n",
      "https://www.aetna.com/cpb/medical/data/600_699/0650.html\n",
      "https://www.aetna.com/cpb/medical/data/600_699/0650.html\n",
      "https://www.aetna.com/cpb/medical/data/300_399/0352.html\n",
      "https://www.aetna.com/cpb/medical/data/600_699/0650.html\n",
      "https://www.aetna.com/cpb/medical/data/700_799/0787.html\n",
      "https://www.aetna.com/cpb/medical/data/300_399/0352.html\n",
      "https://www.aetna.com/cpb/medical/data/600_699/0650.html\n",
      "https://www.aetna.com/cpb/medical/data/500_599/0586.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/200_299/0227.html\n",
      "https://www.aetna.com/cpb/medical/data/200_299/0227.html\n",
      "https://www.aetna.com/cpb/medical/data/600_699/0650.html\n",
      "https://www.aetna.com/cpb/medical/data/300_399/0352.html\n",
      "https://www.aetna.com/cpb/medical/data/600_699/0650.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/300_399/0352.html\n",
      "https://www.aetna.com/cpb/medical/data/800_899/0830.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/300_399/0352.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://qawww.aetna.com/cpb/medical/data/400_499/0493.html\n",
      "https://www.aetna.com/cpb/medical/data/500_599/0521.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/700_799/0715.html\n",
      "https://www.aetna.com/cpb/medical/data/600_699/0643.html\n",
      "https://www.aetna.com/cpb/medical/data/300_399/0352.html\n",
      "https://www.aetna.com/cpb/medical/data/500_599/0516.html\n",
      "https://www.aetna.com/cpb/medical/data/200_299/0227.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/200_299/0227.html\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.aetna.com/cpb/medical/data/300_399/0352.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/policies/mp_pw_e000224.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000972.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/policies/mp_pw_a049877.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000972.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/policies/mp_pw_a049877.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/policies/mp_pw_a049877.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/policies/mp_pw_a049877.html\n",
      "https://www.anthem.com/dam/medpolicies/abcbs/active/policies/mp_pw_c135466.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/policies/mp_pw_e000224.html\n",
      "https://www.anthem.com/dam/medpolicies/abcbs/active/policies/mp_pw_a049883.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/policies/mp_pw_e000224.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000235.html\n",
      "https://www.anthem.com/dam/medpolicies/abcbs/active/guidelines/gl_pw_d056660.html\n",
      "https://www.anthem.com/dam/medpolicies/abcbs/active/policies/mp_pw_a046259.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/policies/mp_pw_a049877.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/policies/mp_pw_c148391.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000232.html\n",
      "https://www.anthem.com/dam/medpolicies/abcbs/active/policies/mp_pw_a049883.html\n",
      "https://www.anthem.com/dam/medpolicies/abcbs/active/policies/mp_pw_c197570.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/policies/mp_pw_e000971.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/policies/mp_pw_e000971.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000232.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/policies/mp_pw_e000224.html\n",
      "1\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000235.html\n",
      "https://www.anthem.com/dam/medpolicies/abcbs/active/policies/mp_pw_d090898.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/policies/mp_pw_e000224.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000232.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/policies/mp_pw_e000224.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/policies/mp_pw_c193684.html\n",
      "https://www.anthem.com/dam/medpolicies/abcbs/active/policies/mp_pw_e000224.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/policies/mp_pw_e000224.html\n",
      "1\n",
      "https://www.anthem.com/dam/medpolicies/abcbs/active/policies/mp_pw_e000224.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000232.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/policies/mp_pw_e000224.html\n",
      "https://www.anthem.com/individual-and-family/insurance-basics/health-insurance/costs\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000235.html\n",
      "https://www.anthem.com/dam/medpolicies/abcbs/active/guidelines/gl_pw_e000974.html\n",
      "https://www.anthem.com/dam/medpolicies/abcbs/active/guidelines/gl_pw_e000233.html\n",
      "https://www.anthem.com/dam/medpolicies/abcbs/active/policies/mp_pw_e000224.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://providers.anthem.com/docs/gpp/KY_CAID_ColorectalCancerScreening.pdf?v=202301031849\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/policies/mp_pw_a050311.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000232.html\n",
      "https://www.anthem.com/dam/medpolicies/abcbs_va/active/guidelines/gl_pw_e000232.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000232.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000235.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/policies/mp_pw_a050309.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000232.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000232.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/policies/mp_pw_a045973.html\n",
      "https://www.anthem.com/dam/medpolicies/abcbs/active/guidelines/gl_pw_e000974.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000232.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000234.html\n",
      "https://www.anthem.com/dam/medpolicies/abcbs_va/active/guidelines/gl_pw_e000233.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000232.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/policies/mp_pw_a045973.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000234.html\n",
      "1\n",
      "1\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/policies/mp_pw_a049877.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_d094156.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000235.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000232.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000234.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000813.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000232.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000234.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000813.html\n",
      "https://www.anthem.com/dam/medpolicies/abcbs_va/active/guidelines/gl_pw_e000233.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/policies/mp_pw_e000224.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000813.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_d087066.html\n",
      "https://www.anthem.com/dam/medpolicies/abcbs/active/guidelines/gl_pw_e001870.html\n",
      "https://www.anthem.com/dam/medpolicies/abcbs/active/guidelines/gl_pw_e000974.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/policies/mp_pw_d082650.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/policies/mp_pw_d082650.html\n",
      "https://www.anthem.com/dam/medpolicies/abcbs/active/guidelines/gl_pw_e000974.html\n",
      "https://www.anthem.com/dam/medpolicies/abcbs/active/policies/mp_pw_e000975.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "1\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000232.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/policies/mp_pw_e000224.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/policies/mp_pw_e000224.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000232.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000232.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000235.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000232.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000235.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000235.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000234.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000235.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/policies/mp_pw_e000224.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_d094156.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/policies/mp_pw_c148391.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000972.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/policies/mp_pw_a049877.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/policies/mp_pw_e000224.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000813.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/policies/mp_pw_a049877.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_d087066.html\n",
      "https://www.anthem.com/dam/medpolicies/abcbs/active/guidelines/gl_pw_d091788.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/policies/mp_pw_a049877.html\n",
      "https://www.anthem.com/dam/medpolicies/abcbs/active/guidelines/gl_pw_d056660.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/policies/mp_pw_e000224.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/policies/mp_pw_c148391.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/policies/mp_pw_c148391.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000232.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.anthem.com/dam/medpolicies/abcbs/active/policies/mp_pw_e001735.html\n",
      "https://www.anthem.com/dam/medpolicies/abcbs/active/guidelines/gl_pw_e000974.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/policies/mp_pw_c128862.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "1\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000972.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/policies/mp_pw_a050309.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/policies/mp_pw_e000224.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000813.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_d091788.html\n",
      "1\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/policies/mp_pw_a050309.html\n",
      "https://www.anthem.com/dam/medpolicies/abcbs/active/policies/mp_pw_e000975.html\n",
      "https://www.anthem.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-86000-22&pv=false\n",
      "http://mcgs.bcbsfl.com/MCG?mcgId=05-86000-26&pv=false\n",
      "http://mcgs.bcbsfl.com/MCG?mcgId=05-86000-26&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "1\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-86000-22&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-86000-22&pv=false\n",
      "http://mcgs.bcbsfl.com/MCG?mcgId=05-86000-26&pv=false\n",
      "http://mcgs.bcbsfl.com/MCG?mcgId=05-86000-29&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-86000-22&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-86000-22&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "http://mcgs.bcbsfl.com/MCG?mcgId=05-82000-30&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-86000-22&pv=false\n",
      "http://mcgs.bcbsfl.com/MCG?activity=openSearchedDocMcg&imgId=75S1HNGWR0AZTVW6MK1\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-86000-22&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-86000-22&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?activity=openSearchedDocMcg&imgId=765OOVDMO2ZZVDGKXB5\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-86000-22&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-86000-22&pv=false\n",
      "1\n",
      "http://mcgs.bcbsfl.com/MCG?mcgId=05-82000-31&pv=false\n",
      "1\n",
      "https://mcgs.bcbsfl.com/MCG?activity=openSearchedDocMcg&imgId=765KYA5LE7QLR5DLFSX\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "1\n",
      "1\n",
      "http://mcgs.bcbsfl.com/MCG?mcgId=05-86000-28&pv=false\n",
      "1\n",
      "https://mcgs.bcbsfl.com/MCG?activity=openSearchedDocMcg&imgId=765OOVDMO2ZZVDGKXB5\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-86000-22&pv=false\n",
      "http://mcgs.bcbsfl.com/MCG?mcgId=05-82000-27&pv=false\n",
      "1\n",
      "http://mcgs.bcbsfl.com/MCG?mcgId=05-82000-31&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "http://mcgs.bcbsfl.com/MCG?mcgId=05-82000-30&pv=false\n",
      "http://mcgs.bcbsfl.com/MCG?mcgId=05-82000-31&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-86000-22&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "http://mcgs.bcbsfl.com/MCG?mcgId=05-82000-27&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "https://fl-policies.exploremyplan.com/portal/web/medical-policies/-/mp-548\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "http://mcgs.bcbsfl.com/MCG?mcgId=09-J0000-39&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?activity=openSearchedDocMcg&imgId=765OOVDMO2ZZVDGKXB5\n",
      "1\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=09-J3000-93&pv=false\n",
      "http://mcgs.bcbsfl.com/MCG?activity=openSearchedDocMcg&imgId=75S1HNGWR0AZTVW6MK1\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=09-J4000-35&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "1\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=03-59000-18&pv=false\n",
      "http://mcgs.bcbsfl.com/MCG?mcgId=05-82000-31&pv=false\n",
      "http://mcgs.bcbsfl.com/MCG?mcgId=05-82000-30&pv=false\n",
      "1\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-86000-22&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "http://mcgs.bcbsfl.com/MCG?mcgId=05-82000-30&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?activity=openSearchedDocMcg&imgId=75WUVKCSS8C3BONIG7T\n",
      "https://mcgs.bcbsfl.com/MCG?activity=openSearchedDocMcg&imgId=765OOVDMO2ZZVDGKXB5\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "http://mcgs.bcbsfl.com/MCG?mcgId=05-82000-31&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?activity=openSearchedDocMcg&imgId=765OOVDMO2ZZVDGKXB5\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "http://mcgs.bcbsfl.com/MCG?mcgId=05-82000-31&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?activity=openSearchedDocMcg&imgId=765OOVDMO2ZZVDGKXB5\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=03-59000-18&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "https://fl-policies.exploremyplan.com/portal/web/medical-policies/-/mp-548\n",
      "https://fl-policies.exploremyplan.com/portal/web/medical-policies/-/mp-548\n",
      "https://fl-policies.exploremyplan.com/portal/web/medical-policies/-/mp-548\n",
      "https://fl-policies.exploremyplan.com/portal/web/medical-policies/-/mp-548\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=03-59000-18&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-86000-22&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "http://mcgs.bcbsfl.com/MCG?mcgId=05-82000-31&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-86000-22&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-86000-22&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-86000-22&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-86000-22&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?activity=openSearchedDocMcg&imgId=765KYA5LE7QLR5DLFSX\n",
      "http://mcgs.bcbsfl.com/MCG?mcgId=05-86000-28&pv=false\n",
      "http://mcgs.bcbsfl.com/MCG?mcgId=05-86000-26&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "1\n",
      "http://mcgs.bcbsfl.com/MCG?activity=openSearchedDocMcg&imgId=75S1HNGWR0AZTVW6MK1\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-86000-22&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=09-A0000-03&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "http://mcgs.bcbsfl.com/MCG?mcgId=05-86000-26&pv=false\n",
      "1\n",
      "https://fl-policies.exploremyplan.com/portal/web/medical-policies/-/mp-548\n",
      "http://mcgs.bcbsfl.com/MCG?mcgId=05-82000-31&pv=false\n",
      "http://mcgs.bcbsfl.com/MCG?mcgId=05-86000-24&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-86000-22&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-86000-22&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "1\n",
      "https://fl-policies.exploremyplan.com/portal/web/medical-policies/-/mp-548\n",
      "http://mcgs.bcbsfl.com/MCG?activity=openSearchedDocMcg&imgId=75S1HNGWR0AZTVW6MK1\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "http://mcgs.bcbsfl.com/MCG?mcgId=05-86000-24&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-86000-22&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "http://mcgs.bcbsfl.com/MCG?mcgId=09-J1000-87&pv=false\n",
      "http://mcgs.bcbsfl.com/MCG?activity=openSearchedDocMcg&imgId=75S1HNGWR0AZTVW6MK1\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-86000-22&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=03-59000-18&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false\n",
      "https://www.bluecrossnc.com/sites/default/files/document/attachment/services/public/pdfs/medicalpolicy/colorectal_cancer_screening.pdf\n",
      "https://www.bluecrossnc.com/document/genetic-cancer-susceptibility-panels-using-next-generation-sequencing-ahs-m2066\n",
      "https://www.bluecrossnc.com/document/genetic-cancer-susceptibility-panels-using-next-generation-sequencing-ahs-m2066\n",
      "https://www.bluecrossnc.com/document/serum-testing-hepatic-fibrosis-evaluation-and-monitoring-chronic-liver-disease\n",
      "https://www.bluecrossnc.com/document/brca-ahs-m2003\n",
      "https://www.bluecrossnc.com/providers/medical-policies-and-coverage/search-medical-policy/medical-guidelines-categorical-index\n",
      "https://www.bluecrossnc.com/sites/default/files/document/attachment/services/public/pdfs/medicalpolicy/pathogen_panel_testing.pdf\n",
      "https://www.bluecrossnc.com/providers/medical-policies-and-coverage/search-medical-policy/medical-guidelines-alphabetical-index\n",
      "https://www.bluecrossnc.com/sites/default/files/document/attachment/services/public/pdfs/medicalpolicy/carrier_screening_for_genetic_diseases_0.pdf\n",
      "https://www.bluecrossnc.com/sites/default/files/document/attachment/services/public/pdfs/medicalpolicy/carrier_screening_for_genetic_diseases_0.pdf\n",
      "https://www.bluecrossnc.com/document/genetic-testing-neurodegenerative-disorders-ahs-%E2%80%93-m2167\n",
      "https://www.bluecrossnc.com/document/whole-genome-and-whole-exome-sequencing-ahs-%E2%80%93-m2032-0\n",
      "https://www.bluecrossnc.com/sites/default/files/document/attachment/services/public/pdfs/medicalpolicy/general_genetic_testing_germline_disorders_notification_2.pdf\n",
      "https://www.bluecrossnc.com/document/genetic-testing-connective-tissue-disorders-ahs-%E2%80%93-m2144\n",
      "https://www.bluecrossnc.com/document/blue-link-2003-pdfs-1\n",
      "https://www.bluecrossnc.com/document/general-genetic-testing-somatic-disorders-ahs-m2146\n",
      "https://www.bluecrossnc.com/sites/default/files/document/attachment/services/public/pdfs/medicalpolicy/genetic_testing_for_ducenne_becker_facioscapulohumeral_and_limb_girdle_muscular_dystrophies.pdf\n",
      "https://www.bluecrossnc.com/document/lynch-syndrome-ahs-m2004\n",
      "https://www.bluecrossnc.com/content/dam/bcbsnc/pdf/providers/policies-guidelines-codes/policies/commercial/laboratory/genetic_testing_for_inherited_cardiomyopathies_and_channelopathies.pdf\n",
      "https://www.bluecrossnc.com/document/genetic-testing-polyposis-syndrome-ahs-%E2%80%93-m2024\n",
      "https://www.bluecrossnc.com/providers/medical-policies-and-coverage/search-medical-policy/medical-guidelines-alphabetical-index\n",
      "https://www.bluecrossnc.com/document/brca-ahs-m2003\n",
      "https://www.bluecrossnc.com/document/general-genetic-testing-somatic-disorders-ahs-m2146\n",
      "1\n",
      "https://www.bluecrossnc.com/document/liquid-biopsy-ahs-g2054\n",
      "https://www.bluecrossnc.com/document/kras-nras-and-braf-mutation-analysis-colorectal-cancer\n",
      "https://www.bluecrossnc.com/document/services-public-pdfs-medicalpolicy-genetictestingforacutemyeloidleukemianotificationpdf\n",
      "https://www.bluecrossnc.com/document/celiac-disease-testing-ahs-g2043\n",
      "https://www.bluecrossnc.com/document/brca-ahs-m2003\n",
      "https://www.bluecrossnc.com/document/prenatal-screening-genetic-ahs-m2179\n",
      "https://www.bluecrossnc.com/document/services-public-pdfs-medicalpolicy-genetictestingforacutemyeloidleukemianotificationpdf\n",
      "https://www.bluecrossnc.com/sites/default/files/document/attachment/services/public/pdfs/medicalpolicy/pharmacogenetics_testing_notification.pdf\n",
      "https://www.bluecrossnc.com/document/bcr-abl-1-testing-ahs-%E2%80%93-m2027\n",
      "https://www.bluecrossnc.com/document/gene-expression-profiling-and-protein-biomarkers-prostate-cancer-ahs-m2166\n",
      "https://www.bluecrossnc.com/document/genetic-testing-familial-cutaneous-malignant-melanoma-ahs-%E2%80%93-m2037\n",
      "https://www.bluecrossnc.com/document/bcr-abl-1-testing-ahs-%E2%80%93-m2027\n",
      "https://www.bluecrossnc.com/document/services-public-pdfs-medicalpolicy-genetictestingforacutemyeloidleukemianotificationpdf\n",
      "https://www.bluecrossnc.com/sites/default/files/document/attachment/services/public/pdfs/medicalpolicy/gene_expression_testing_for_breast_cancer_prognosis.pdf\n",
      "https://www.bluecrossnc.com/document/genetic-testing-alpha-and-beta-thalassemia-ahs-%E2%80%93-m2131\n",
      "https://www.bluecrossnc.com/sites/default/files/document/attachment/services/public/pdfs/medicalpolicy/prostate_cancer_screening.pdf\n",
      "1\n",
      "1\n",
      "https://www.bluecrossnc.com/sites/default/files/document/attachment/services/public/pdfs/medicalpolicy/gene_expression_testing_for_breast_cancer_prognosis.pdf\n",
      "https://www.bluecrossnc.com/document/gene-expression-profiling-and-protein-biomarkers-prostate-cancer-ahs-m2166\n",
      "https://www.bluecrossnc.com/sites/default/files/document/attachment/services/public/pdfs/medicalpolicy/gene_expression_testing_for_breast_cancer_prognosis.pdf\n",
      "https://www.bluecrossnc.com/document/molecular-diagnosis-breast-cancer-prognosis-ahs-m2020\n",
      "https://www.bluecrossnc.com/sites/default/files/document/attachment/services/public/pdfs/medicalpolicy/general_genetic_testing_somatic_disorders.pdf\n",
      "https://www.bluecrossnc.com/document/molecular-markers-fine-needle-aspirates-thyroid\n",
      "https://www.bluecrossnc.com/sites/default/files/document/attachment/services/public/pdfs/medicalpolicy/gene_expression_profiling_and_protein_biomarkers_for_prostate_cancer_management.pdf\n",
      "https://www.bluecrossnc.com/providers/medical-policies-and-coverage/medical-policy-updates/medical-policy-update-may-17-2022\n",
      "https://www.bluecrossnc.com/document/services-public-pdfs-medicalpolicy-35\n",
      "https://www.bluecrossnc.com/document/transplant-rejection-testing-ahs-m2091\n",
      "https://www.bluecrossnc.com/document/services-public-pdfs-medicalpolicy-36\n",
      "https://www.bluecrossnc.com/sites/default/files/document/attachment/services/public/pdfs/medicalpolicy/gene_expression_testing_for_breast_cancer_prognosis.pdf\n",
      "https://www.bluecrossnc.com/document/services-public-pdfs-medicalpolicy-35\n",
      "https://www.bluecrossnc.com/document/genetic-testing-and-genetic-expression-profiling-patients-cutaneous-melanoma-ahs-m2029\n",
      "https://www.bluecrossnc.com/document/general-genetic-testing-germline-disorders-ahs-m2145\n",
      "https://www.bluecrossnc.com/sites/default/files/document/attachment/services/public/pdfs/medicalpolicy/vectra_da_blood_test_for_rheumatoid_arthritis_notification.pdf\n",
      "https://www.bluecrossnc.com/document/services-public-pdfs-medicalpolicy-35\n",
      "https://www.bluecrossnc.com/document/cardiovascular-disease-risk-assessment-ahs-g2050\n",
      "https://www.bluecrossnc.com/document/services-public-pdfs-medicalpolicy-35\n",
      "https://www.bluecrossnc.com/document/brca-ahs-m2003\n",
      "https://www.bluecrossnc.com/document/services-public-pdfs-medicalpolicy-lynchsyndromenotificationpdf\n",
      "https://www.bluecrossnc.com/document/genetic-testing-neurodegenerative-disorders-ahs-%E2%80%93-m2167\n",
      "https://www.bluecrossnc.com/document/services-public-pdfs-medicalpolicy-35\n",
      "https://www.bluecrossnc.com/sites/default/files/document/attachment/services/public/pdfs/medicalpolicy/prostate_cancer_screening.pdf\n",
      "https://www.bluecrossnc.com/document/prostate-biopsies-ahs-g2007\n",
      "https://www.bluecrossnc.com/document/genetic-testing-and-genetic-expression-profiling-patients-cutaneous-melanoma-ahs-m2029\n",
      "https://www.bluecrossnc.com/document/serum-tumor-markers-malignancies-ahs-g2124\n",
      "https://www.bluecrossnc.com/document/genetic-testing-inherited-cardiomyopathies-and-channelopathies-ahs-m2025\n",
      "https://www.bluecrossnc.com/document/pharmacogenetics-testing-ahs-m2021\n",
      "https://www.bluecrossnc.com/document/genetic-testing-inherited-cardiomyopathies-and-channelopathies-ahs-m2025\n",
      "https://www.bluecrossnc.com/providers/medical-policies-and-coverage/search-medical-policy/medical-guidelines-categorical-index\n",
      "https://www.bluecrossnc.com/content/dam/bcbsnc/pdf/providers/policies-guidelines-codes/policies/commercial/laboratory/genetic_testing_for_PTEN_hamartoma_tumor_syndrome.pdf\n",
      "https://www.bluecrossnc.com/document/preimplantation-genetic-testing-ahs-m2039\n",
      "https://www.bluecrossnc.com/document/services-public-pdfs-medicalpolicy-genetictestingforacutemyeloidleukemianotificationpdf\n",
      "https://www.bluecrossnc.com/document/pharmacogenetics-testing-ahs-m2021\n",
      "https://www.bluecrossnc.com/document/genetic-testing-inherited-cardiomyopathies-and-channelopathies-ahs-m2025\n",
      "https://www.bluecrossnc.com/document/services-public-pdfs-medicalpolicy-21\n",
      "https://www.bluecrossnc.com/document/genetic-cancer-susceptibility-panels-using-next-generation-sequencing-ahs-m2066\n",
      "https://www.bluecrossnc.com/document/genetic-testing-connective-tissue-disorders-ahs-%E2%80%93-m2144\n",
      "https://www.bluecrossnc.com/document/genetic-testing-neurodegenerative-disorders-ahs-%E2%80%93-m2167\n",
      "https://www.bluecrossnc.com/document/genetic-testing-ophthalmologic-conditions-ahs-m2083\n",
      "https://www.bluecrossnc.com/document/services-public-pdfs-medicalpolicy-lynchsyndromenotificationpdf\n",
      "https://www.bluecrossnc.com/document/celiac-disease-testing-ahs-g2043\n",
      "https://www.bluecrossnc.com/document/brca-ahs-m2003\n",
      "https://www.bluecrossnc.com/document/services-public-pdfs-medicalpolicy-genetictestingforacutemyeloidleukemianotificationpdf\n",
      "https://www.bluecrossnc.com/document/services-public-pdfs-medicalpolicy-35\n",
      "https://www.bluecrossnc.com/sites/default/files/document/attachment/services/public/pdfs/medicalpolicy/prenatal_screening_for_fetal_aneuploidy_notification.pdf\n",
      "https://www.bluecrossnc.com/document/genetic-testing-mitochondrial-disorders-ahs-m2085\n",
      "https://www.bluecrossnc.com/document/prenatal-testing-fetal-aneuploidy-ahs-g2055\n",
      "https://www.bluecrossnc.com/document/services-public-pdfs-medicalpolicy-lynchsyndromenotificationpdf\n",
      "https://www.bluecrossnc.com/document/chromosomal-microarray-ahs-m2033\n",
      "https://www.bluecrossnc.com/document/genetic-cancer-susceptibility-panels-using-next-generation-sequencing-ahs-m2066\n",
      "https://www.bluecrossnc.com/sites/default/files/document/attachment/services/public/pdfs/medicalpolicy/whole_exome_and_whole_genome_sequencing_for_diagnosis_of_genetic_disorders_1.pdf\n",
      "https://www.bluecrossnc.com/document/brca-ahs-m2003\n",
      "https://www.bluecrossnc.com/document/celiac-disease-testing-ahs-g2043\n",
      "https://www.bluecrossnc.com/document/genetic-testing-mitochondrial-disorders-ahs-m2085\n",
      "https://www.bluecrossnc.com/document/services-public-pdfs-medicalpolicy-lynchsyndromenotificationpdf\n",
      "https://www.bluecrossnc.com/document/testing-targeted-therapy-non-small-cell-lung-cancer-ahs-m2030\n",
      "https://www.bluecrossnc.com/document/genetic-testing-rett-syndrome\n",
      "https://www.bluecrossnc.com/document/prenatal-screening-genetic-ahs-m2179\n",
      "https://www.bluecrossnc.com/document/genetic-cancer-susceptibility-panels-using-next-generation-sequencing-ahs-m2066\n",
      "https://www.bluecrossnc.com/sites/default/files/document/attachment/services/public/pdfs/medicalpolicy/testing_for_targeted_therapy_of_non_small_cell_lung_cancer.pdf\n",
      "https://www.bluecrossnc.com/document/diagnostic-testing-sexually-transmitted-infections-ahs-g2157\n",
      "https://www.bluecrossnc.com/document/genetic-testing-cystic-fibrosis-ahs-m2017\n",
      "https://www.bluecrossnc.com/document/microsatellite-instability-and-tumor-mutational-burden-testing-ahs-m2178\n",
      "https://www.bluecrossnc.com/document/general-genetic-testing-somatic-disorders-ahs-m2146\n",
      "https://www.bluecrossnc.com/document/identification-microorganisms-using-nucleic-acid-probes\n",
      "https://www.bluecrossnc.com/sites/default/files/document/attachment/services/public/pdfs/medicalpolicy/transplant_rejection_testing.pdf\n",
      "https://www.bluecrossnc.com/sites/default/files/document/attachment/services/public/pdfs/medicalpolicy/whole_exome_and_whole_genome_sequencing_for_diagnosis_of_genetic_disorders_1.pdf\n",
      "https://www.bluecrossnc.com/document/liquid-biopsy-ahs-g2054\n",
      "https://www.bluecrossnc.com/document/gene-expression-profiling-and-protein-biomarkers-prostate-cancer-ahs-m2166\n",
      "https://www.bluecrossnc.com/document/pharmacogenetics-testing-ahs-m2021\n",
      "https://www.bluecrossnc.com/document/liquid-biopsy-ahs-g2054\n",
      "https://www.bluecrossnc.com/document/molecular-testing-pulmonary-disease-ahs-m2160\n",
      "https://www.bluecrossnc.com/document/pathogen-panel-testing-ahs-g2149\n",
      "https://www.bluecrossnc.com/document/genetic-testing-mitochondrial-disorders-ahs-m2085\n",
      "https://www.bluecrossnc.com/document/liquid-biopsy-ahs-g2054\n",
      "https://www.bluecrossnc.com/document/brca-ahs-m2003\n",
      "https://www.bluecrossnc.com/document/transplant-rejection-testing-ahs-m2091\n",
      "https://www.bluecrossnc.com/document/coronavirus-testing-outpatient-setting\n",
      "https://www.bluecrossnc.com/document/blue-book-provider-manual\n",
      "https://www.bluecrossnc.com/sites/default/files/document/attachment/services/public/pdfs/medicalpolicy/prenatal_screening_for_fetal_aneuploidy_notification.pdf\n",
      "https://www.bluecrossnc.com/document/genetic-testing-cystic-fibrosis-ahs-m2017\n",
      "https://www.bluecrossnc.com/document/diagnosis-vaginitis-including-multi-target-pcr-testing-ahs-m2057\n",
      "https://www.bluecrossnc.com/document/gene-expression-profiling-and-protein-biomarkers-prostate-cancer-ahs-m2166\n",
      "https://www.bluecrossnc.com/document/cost-quality-designation-methodology\n",
      "https://www.bluecrossnc.com/document/cost-quality-designation-methodology\n",
      "https://www.bluecrossnc.com/sites/default/files/document/attachment/services/public/pdfs/medicalpolicy/urinary_tumor_markers_for_bladder_cancer.pdf\n",
      "https://www.bluecrossnc.com/document/molecular-profiling-cancers-unknown-primary-origin-ahs-m2065\n",
      "https://www.bluecrossnc.com/document/pharmacogenetics-testing-ahs-m2021\n",
      "https://www.bluecrossnc.com/document/kras-nras-and-braf-mutation-analysis-colorectal-cancer\n",
      "https://www.bluecrossnc.com/document/genetic-testing-pten-hamartoma-tumor-syndrome\n",
      "https://www.bluecrossnc.com/document/services-public-pdfs-medicalpolicy-35\n",
      "https://www.bluecrossnc.com/document/serum-tumor-markers-malignancies-ahs-g2124\n",
      "https://www.bluecrossnc.com/document/whole-genome-and-whole-exome-sequencing-ahs-%E2%80%93-m2032-0\n",
      "https://www.bluecrossnc.com/document/molecular-markers-fine-needle-aspirates-thyroid\n",
      "https://www.bluecrossnc.com/document/brca-ahs-m2003\n",
      "https://www.bluecrossnc.com/document/genetic-testing-and-genetic-expression-profiling-patients-cutaneous-melanoma-ahs-m2029\n",
      "https://www.bluecrossnc.com/document/pharmacogenetics-testing-ahs-m2021\n",
      "https://www.bluecrossnc.com/document/services-public-pdfs-medicalpolicy-genetictestingforacutemyeloidleukemianotificationpdf\n",
      "https://www.bluecrossnc.com/providers/medical-policies-and-coverage/search-medical-policy/medical-guidelines-alphabetical-index\n",
      "https://www.bluecrossnc.com/document/kras-nras-and-braf-mutation-analysis-colorectal-cancer\n",
      "https://www.bluecrossnc.com/document/services-public-pdfs-medicalpolicy-35\n",
      "https://www.bluecrossnc.com/document/services-public-pdfs-medicalpolicy-genetictestingforcysticfibrosisnotificationpdf\n",
      "https://www.bluecrossnc.com/document/general-genetic-testing-somatic-disorders-ahs-m2146\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/Genetic_Testing_Collateral_Document.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0520_coveragepositioncriteria_tumor_profiling.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0520_coveragepositioncriteria_tumor_profiling.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0520_coveragepositioncriteria_tumor_profiling.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0520_coveragepositioncriteria_tumor_profiling.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0520_coveragepositioncriteria_tumor_profiling.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0520_coveragepositioncriteria_tumor_profiling.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0519_coveragepositioncriteria_exome_genome_sequence.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0520_coveragepositioncriteria_tumor_profiling.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0520_coveragepositioncriteria_tumor_profiling.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/Genetic_Testing_Collateral_Document.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0520_coveragepositioncriteria_tumor_profiling.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0520_coveragepositioncriteria_tumor_profiling.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0520_coveragepositioncriteria_tumor_profiling.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0520_coveragepositioncriteria_tumor_profiling.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0520_coveragepositioncriteria_tumor_profiling.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0520_coveragepositioncriteria_tumor_profiling.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/Genetic_Testing_Collateral_Document.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0052_coveragepositioncriteria_genetic_testing.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/Genetic_Testing_Collateral_Document.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0465_coveragepositioncriteria_blood_tests_heart_transplantation_rejection.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0518_coveragepositioncriteria_genetic_cancer_syndromes.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0520_coveragepositioncriteria_tumor_profiling.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0052_coveragepositioncriteria_genetic_testing.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0504_coveragepositioncriteria_omnibus_codes.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/Genetic_Testing_Collateral_Document.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/Collateral_Genetic_Molecular_Tests_NMN_EIU.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0518_coveragepositioncriteria_genetic_cancer_syndromes.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/Genetic_Testing_Collateral_Document.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0530_coveragepositioncriteria_microbe_test_STD.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0518_coveragepositioncriteria_genetic_cancer_syndromes.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0520_coveragepositioncriteria_tumor_profiling.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0518_coveragepositioncriteria_genetic_cancer_syndromes.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0518_coveragepositioncriteria_genetic_cancer_syndromes.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0052_coveragepositioncriteria_genetic_testing.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0518_coveragepositioncriteria_genetic_cancer_syndromes.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0520_coveragepositioncriteria_tumor_profiling.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_Future/mm_0557_coveragepositioncriteria_covid-19_in_vitro_diagnostic_testing.pdf\n",
      "1\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0520_coveragepositioncriteria_tumor_profiling.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0520_coveragepositioncriteria_tumor_profiling.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0514_coveragepositioncriteria_genetic_testing_repro_carrier_prenatal.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/Genetic_Testing_Collateral_Document.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0518_coveragepositioncriteria_genetic_cancer_syndromes.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0520_coveragepositioncriteria_tumor_profiling.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/Collateral_Genetic_Molecular_Tests_NMN_EIU.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0520_coveragepositioncriteria_tumor_profiling.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0148_coveragepositioncriteria_colorectal_cancer_screening.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0514_coveragepositioncriteria_genetic_testing_repro_carrier_prenatal.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0533_coveragepositioncriteria_stem_cell_transplant_blood_cancers.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0518_coveragepositioncriteria_genetic_cancer_syndromes.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/Genetic_Testing_Collateral_Document.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0518_coveragepositioncriteria_genetic_cancer_syndromes.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0520_coveragepositioncriteria_tumor_profiling.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0518_coveragepositioncriteria_genetic_cancer_syndromes.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0520_coveragepositioncriteria_tumor_profiling.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0520_coveragepositioncriteria_tumor_profiling.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0520_coveragepositioncriteria_tumor_profiling.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0148_coveragepositioncriteria_colorectal_cancer_screening.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0518_coveragepositioncriteria_genetic_cancer_syndromes.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0520_coveragepositioncriteria_tumor_profiling.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0520_coveragepositioncriteria_tumor_profiling.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0500_coveragepositioncriteria_pharmacogenetic_testing.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0520_coveragepositioncriteria_tumor_profiling.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0518_coveragepositioncriteria_genetic_cancer_syndromes.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0052_coveragepositioncriteria_genetic_testing.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0500_coveragepositioncriteria_pharmacogenetic_testing.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0520_coveragepositioncriteria_tumor_profiling.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0514_coveragepositioncriteria_genetic_testing_repro_carrier_prenatal.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0518_coveragepositioncriteria_genetic_cancer_syndromes.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0518_coveragepositioncriteria_genetic_cancer_syndromes.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0500_coveragepositioncriteria_pharmacogenetic_testing.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0518_coveragepositioncriteria_genetic_cancer_syndromes.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0514_coveragepositioncriteria_genetic_testing_repro_carrier_prenatal.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0148_coveragepositioncriteria_colorectal_cancer_screening.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0519_coveragepositioncriteria_exome_genome_sequence.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0517_coveragepositioncriteria_genetic_cardiac_syndromes.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0557_coveragepositioncriteria_covid-19_in_vitro_diagnostic_testing.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0530_coveragepositioncriteria_microbe_test_STD.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0514_coveragepositioncriteria_genetic_testing_repro_carrier_prenatal.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0520_coveragepositioncriteria_tumor_profiling.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0514_coveragepositioncriteria_genetic_testing_repro_carrier_prenatal.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0530_coveragepositioncriteria_microbe_test_STD.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0514_coveragepositioncriteria_genetic_testing_repro_carrier_prenatal.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0519_coveragepositioncriteria_exome_genome_sequence.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0520_coveragepositioncriteria_tumor_profiling.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0465_coveragepositioncriteria_blood_tests_heart_transplantation_rejection.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/Genetic_Testing_Collateral_Document.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/Genetic_Testing_Collateral_Document.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/Genetic_Testing_Collateral_Document.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0520_coveragepositioncriteria_tumor_profiling.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0465_coveragepositioncriteria_blood_tests_heart_transplantation_rejection.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0520_coveragepositioncriteria_tumor_profiling.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0518_coveragepositioncriteria_genetic_cancer_syndromes.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0530_coveragepositioncriteria_microbe_test_STD.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0514_coveragepositioncriteria_genetic_testing_repro_carrier_prenatal.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/Genetic_Testing_Collateral_Document.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/Genetic_Testing_Collateral_Document.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0520_coveragepositioncriteria_tumor_profiling.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0514_coveragepositioncriteria_genetic_testing_repro_carrier_prenatal.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/Genetic_Testing_Collateral_Document.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0520_coveragepositioncriteria_tumor_profiling.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0518_coveragepositioncriteria_genetic_cancer_syndromes.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0520_coveragepositioncriteria_tumor_profiling.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0052_coveragepositioncriteria_genetic_testing.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0518_coveragepositioncriteria_genetic_cancer_syndromes.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0518_coveragepositioncriteria_genetic_cancer_syndromes.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0052_coveragepositioncriteria_genetic_testing.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0518_coveragepositioncriteria_genetic_cancer_syndromes.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0520_coveragepositioncriteria_tumor_profiling.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0517_coveragepositioncriteria_genetic_cardiac_syndromes.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/Genetic_Testing_Collateral_Document.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0148_coveragepositioncriteria_colorectal_cancer_screening.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0518_coveragepositioncriteria_genetic_cancer_syndromes.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0518_coveragepositioncriteria_genetic_cancer_syndromes.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0052_coveragepositioncriteria_genetic_testing.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_Future/mm_0557_coveragepositioncriteria_covid-19_in_vitro_diagnostic_testing.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0518_coveragepositioncriteria_genetic_cancer_syndromes.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0520_coveragepositioncriteria_tumor_profiling.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/Genetic_Testing_Collateral_Document.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0530_coveragepositioncriteria_microbe_test_STD.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0520_coveragepositioncriteria_tumor_profiling.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0520_coveragepositioncriteria_tumor_profiling.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0514_coveragepositioncriteria_genetic_testing_repro_carrier_prenatal.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0514_coveragepositioncriteria_genetic_testing_repro_carrier_prenatal.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0520_coveragepositioncriteria_tumor_profiling.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0514_coveragepositioncriteria_genetic_testing_repro_carrier_prenatal.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0518_coveragepositioncriteria_genetic_cancer_syndromes.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0530_coveragepositioncriteria_microbe_test_STD.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0518_coveragepositioncriteria_genetic_cancer_syndromes.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0530_coveragepositioncriteria_microbe_test_STD.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0148_coveragepositioncriteria_colorectal_cancer_screening.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0520_coveragepositioncriteria_tumor_profiling.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/cnf/cnf_268_coveragepositioncriteria_hepatitis_c_virus_direct-acting_antivirals_psm.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0518_coveragepositioncriteria_genetic_cancer_syndromes.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0520_coveragepositioncriteria_tumor_profiling.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/Genetic_Testing_Collateral_Document.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0052_coveragepositioncriteria_genetic_testing.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/Genetic_Testing_Collateral_Document.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0514_coveragepositioncriteria_genetic_testing_repro_carrier_prenatal.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0518_coveragepositioncriteria_genetic_cancer_syndromes.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/Collateral_Genetic_Molecular_Tests_NMN_EIU.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0519_coveragepositioncriteria_exome_genome_sequence.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0517_coveragepositioncriteria_genetic_cardiac_syndromes.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0052_coveragepositioncriteria_genetic_testing.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0052_coveragepositioncriteria_genetic_testing.pdf\n",
      "https://static.cigna.com/assets/chcp/pdf/coveragePolicies/medical/mm_0514_coveragepositioncriteria_genetic_testing_repro_carrier_prenatal.pdf\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/251-300/MP280-Whole-Exome-Sequencing.pdf?la=en\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/301-350/MP321-Gene-Expression-Profiling-for-Cutaneous-Melanoma.pdf?la=en\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/151-200/MP170-Gene-Expression-Profiling-for-Breast-Cancer-Treatment.pdf?sc_lang=en&hash=5A55B2E142D76AF0B1D80E42110EC596\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/151-200/MP170-Gene-Expression-Profiling-for-Breast-Cancer-Treatment.pdf?sc_lang=en&hash=5A55B2E142D76AF0B1D80E42110EC596\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/251-300/MP273-Gene-based-Testing-and-or-Protein-Biomarkers-for-Diagnosis-and-Management-of-Prostate-Cancer.pdf?sc_lang=en&hash=E0BE31E9831C08ED92AF2199F09416CA\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/301-350/MP323-Molecular-Profiling-of-Malignant-Tumors-to-Identify-Targeted-Therapies.pdf?sc_lang=en&hash=2412EEC6A272A60A8AA3620AB305535D\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/151-200/MP170-Gene-Expression-Profiling-for-Breast-Cancer-Treatment.pdf?sc_lang=en&hash=5A55B2E142D76AF0B1D80E42110EC596\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/301-350/MP324-Genetic-Testing-for-Non-cancer-Heritable-Disease-Carrier-Status.pdf?sc_lang=en&hash=C98C951FDCF6D9425DF5ACE8BDD23801\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/301-350/MP321-Gene-Expression-Profiling-for-Cutaneous-Melanoma.pdf?la=en\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/251-300/MP262-Microarray-based-Gene-Expression-testing-for-Cancer-of-Unknown-Origin.pdf?sc_lang=en&hash=EF44835E22AAF901BC02FDFAC8998110\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/351-400/MP363-Urinary-Tumor-Markers-for-Bladder-Cancer.pdf?sc_lang=en&hash=E3AA4C003D8AE047B9B6768BA21E9BCD\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/151-200/MP168-Non-invasive-Testing-for-Organ-Transplant-Rejection.pdf?sc_lang=en&hash=1BCBC019A87149D8A3C19E58D83B1ECB\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/301-350/MP334-Genetic-Testing-for-Macular-Degeneration.pdf?sc_lang=en&hash=3A65DD1BE3ED8D0F87D7F02EAE450BC6\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/301-350/MP303-Genomic-Analysis-to-Predict-Thyroid-Malignancy-in-FNA.pdf?sc_lang=en&hash=8663394755286FC1344B6890FE8F3D07\n",
      "https://www.geisinger.org/-/media/onegeisinger/pdfs/ghs/patient-care/patients-and-visitors/billing-and-insurance/hosp-charges-pdf/pdf-cdm-geisinger-lewistown-hospital-03-25-19\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/201-250/MP246-Oncotype-Dx---Multi-gene-Expression-Assay-for-Predicting-Recurrence-in-Colon-Cancer.pdf?la=en\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/51-100/MP097-Genetic-Testing-for-BRCA1-or-BRCA2-for-Breast-or-Ovarian-Cancer.pdf?sc_lang=en&hash=B344B03B7AED26104C45DB501786A08E\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/251-300/MP265-Proteomic-Serum-Analysis.pdf?sc_lang=en&hash=8EBD37834935913D6501CCAC446387AC\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/301-350/MP324-Genetic-Testing-for-Non-cancer-Heritable-Disease-Carrier-Status.pdf?sc_lang=en&hash=C98C951FDCF6D9425DF5ACE8BDD23801\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/301-350/MP328-Genetic-Susceptibility-Cancer-Panels.pdf?sc_lang=en&hash=D553029B015A60AE4184DE16EA370A7E\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/301-350/MP328-Genetic-Susceptibility-Cancer-Panels.pdf?sc_lang=en&hash=D553029B015A60AE4184DE16EA370A7E\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/301-350/MP304-Genetic-Testing-for-Inherited-Cardiomyopathies-and-Channelopathies.pdf?sc_lang=en&hash=8562D9547B5F629A2044201D571A82EB\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/301-350/MP304-Genetic-Testing-for-Inherited-Cardiomyopathies-and-Channelopathies.pdf?sc_lang=en&hash=8562D9547B5F629A2044201D571A82EB\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/301-350/MP328-Genetic-Susceptibility-Cancer-Panels.pdf?sc_lang=en&hash=D553029B015A60AE4184DE16EA370A7E\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/Whats-New-Updates/Whats-New-April-2023-MP.pdf?sc_lang=en&hash=187D07148268F1319F20A926C2BFFA85\n",
      "https://www.geisinger.org/-/media/OneGeisinger/pdfs/ghs/patient-care/patients-and-visitors/billing-and-insurance/hosp-charges-pdf/PDF-CDM-Geisinger-Wyoming-Valley-Medical-Ctr-07-01-19.pdf\n",
      "https://www.geisinger.org/-/media/OneGeisinger/pdfs/ghs/research/mycode/Latest-MyCode-Results-Reported.pdf?la=en\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/251-300/MP255-Comparative-Genomic-Hybridization-or-Chromosomal-Microarray-Analysis-CMA-for-Evaluation-of-Developmental-Delay.pdf?sc_lang=en&hash=E3D273DD2B314F8B1C1D9FE3EEF8AB23\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/301-350/MP304-Genetic-Testing-for-Inherited-Cardiomyopathies-and-Channelopathies.pdf?sc_lang=en&hash=8562D9547B5F629A2044201D571A82EB\n",
      "1\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/Whats-New-Updates/Whats-New-February-2022-MP.pdf?la=en\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MBP/151-200/MBP-172-Trisenox-arsenic-trioxide.pdf\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/301-350/MP323-Molecular-Profiling-of-Malignant-Tumors-to-Identify-Targeted-Therapies.pdf?sc_lang=en&hash=2412EEC6A272A60A8AA3620AB305535D\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/251-300/MP265-Proteomic-Serum-Analysis.pdf?sc_lang=en&hash=8EBD37834935913D6501CCAC446387AC\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/Whats-New-Updates/Whats-New-April-2023-MP.pdf?sc_lang=en&hash=187D07148268F1319F20A926C2BFFA85\n",
      "https://www.geisinger.org/about-geisinger/news-and-media/news-releases/2020/10/16/16/20/geisinger-opens-dedicated-cold-and-flu-walk-in-care-clinics\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/Whats-New-Updates/Whats-New-April-2023-MP.pdf?sc_lang=en&hash=187D07148268F1319F20A926C2BFFA85\n",
      "https://www.geisinger.org/-/media/OneGeisinger/pdfs/ghs/patient-care/patients-and-visitors/billing-and-insurance/hosp-charges-pdf/PDF-CDM-Geisinge-Comm-Medical-Ctr-07-01-19.pdf?la=en\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/PDFs/Shared-PDFs/Formulary-Updates/June-2020-Commercial-Marketplace-GHP-Kids-Minutes.pdf?la=en\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MBP/101-150/MBP-148-Exondys-51-eteplirsen.pdf\n",
      "https://www.geisinger.org/-/media/onegeisinger/pdfs/ghs/patient-care/patients-and-visitors/billing-and-insurance/hosp-charges-pdf/pdf-cdm-geisinger-lewistown-hospital-03-25-19\n",
      "1\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/301-350/MP323-Molecular-Profiling-of-Malignant-Tumors-to-Identify-Targeted-Therapies.pdf?sc_lang=en&hash=2412EEC6A272A60A8AA3620AB305535D\n",
      "https://www.geisinger.org/-/media/onegeisinger/pdfs/ghs/patient-care/patients-and-visitors/billing-and-insurance/hosp-charges-pdf/pdf-cdm-geisinger-wyoming-valley-med-ctr-01-01-19\n",
      "1\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/351-400/MP355-Plasma-based-Proteomic-Testing-in-the-Management-of-Pulmonary-Nodules.pdf?sc_lang=en&hash=03384CCB312DD466626C98C785A25E12\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/Whats-New-Updates/Whats-New-February-2023-MP.pdf?sc_lang=en&hash=A270C120FF81DC7D3DA0EE40A35142EB\n",
      "https://www.geisinger.org/patient-care/conditions-treatments-specialty/colon-cancer\n",
      "https://www.geisinger.org/-/media/onegeisinger/pdfs/ghs/research/find-an-investigator/pdfs/find-an-investigator/boscarino/boscarino-cv-012414-1\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/301-350/MP334-Genetic-Testing-for-Macular-Degeneration.pdf?sc_lang=en&hash=3A65DD1BE3ED8D0F87D7F02EAE450BC6\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/301-350/MP328-Genetic-Susceptibility-Cancer-Panels.pdf?sc_lang=en&hash=D553029B015A60AE4184DE16EA370A7E\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/Whats-New-Updates/Whats-New-February-2023-MP.pdf?sc_lang=en&hash=A270C120FF81DC7D3DA0EE40A35142EB\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/Whats-New-Updates/Whats-New-February-2022-MP.pdf?la=en\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/201-250/MP239-Pharmacogenetic-Testing.pdf?la=en\n",
      "https://www.geisinger.org/-/media/onegeisinger/pdfs/ghs/patient-care/patients-and-visitors/billing-and-insurance/hosp-charges-pdf/pdf-cdm-geisinger-lewistown-hospital-03-25-19\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/301-350/MP324-Genetic-Testing-for-Non-cancer-Heritable-Disease-Carrier-Status.pdf?sc_lang=en&hash=C98C951FDCF6D9425DF5ACE8BDD23801\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/Whats-New-Updates/Whats-New-February-2022-MP.pdf?la=en\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/301-350/MP323-Molecular-Profiling-of-Malignant-Tumors-to-Identify-Targeted-Therapies.pdf?la=en\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/PDFs/GHP-Family/Pharmaceutical-and-Therapy-Committee-Notes/05-17-2022-Medicaid-Minutes.pdf?sc_lang=en&hash=6ED1172F9A6A5FD057D9B0CD81ECC510\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/301-350/MP323-Molecular-Profiling-of-Malignant-Tumors-to-Identify-Targeted-Therapies.pdf?sc_lang=en&hash=2412EEC6A272A60A8AA3620AB305535D\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/251-300/MP271-Non-Invasive-Testing-for-Fetal-Aneuploidy.pdf?sc_lang=en&hash=6FFB2C9EE46D7801D1BA5BA95879FE67\n",
      "https://www.geisinger.org/precision-health/mycode\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/51-100/MP097-Genetic-Testing-for-BRCA1-or-BRCA2-for-Breast-or-Ovarian-Cancer.pdf?sc_lang=en&hash=B344B03B7AED26104C45DB501786A08E\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/Whats-New-Updates/Whats-New-December-2020-MP.pdf?la=en\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/301-350/MP324-Genetic-Testing-for-Non-cancer-Heritable-Disease-Carrier-Status.pdf?sc_lang=en&hash=C98C951FDCF6D9425DF5ACE8BDD23801\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/201-250/MP239-Pharmacogenetic-Testing.pdf?la=en\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/Whats-New-Updates/Whats-New-February-2023-MP.pdf?sc_lang=en&hash=A270C120FF81DC7D3DA0EE40A35142EB\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/51-100/MP097-Genetic-Testing-for-BRCA1-or-BRCA2-for-Breast-or-Ovarian-Cancer.pdf?sc_lang=en&hash=B344B03B7AED26104C45DB501786A08E\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/351-400/MP356-Genetic-Testing-for-Mitochondrial-Disorders.pdf?sc_lang=en&hash=DBE389D080162B71F5496819A3FB8015\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/301-350/MP328-Genetic-Susceptibility-Cancer-Panels.pdf?sc_lang=en&hash=D553029B015A60AE4184DE16EA370A7E\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/301-350/MP324-Genetic-Testing-for-Non-cancer-Heritable-Disease-Carrier-Status.pdf?sc_lang=en&hash=C98C951FDCF6D9425DF5ACE8BDD23801\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/301-350/MP323-Molecular-Profiling-of-Malignant-Tumors-to-Identify-Targeted-Therapies.pdf?sc_lang=en&hash=2412EEC6A272A60A8AA3620AB305535D\n",
      "https://www.geisinger.org/-/media/OneGeisinger/pdfs/ghs/patient-care/patients-and-visitors/billing-and-insurance/hosp-charges-pdf/PDF-CDM-Geisinge-Comm-Medical-Ctr-07-01-19.pdf?la=en\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/301-350/MP324-Genetic-Testing-for-Non-cancer-Heritable-Disease-Carrier-Status.pdf?sc_lang=en&hash=C98C951FDCF6D9425DF5ACE8BDD23801\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/301-350/MP324-Genetic-Testing-for-Non-cancer-Heritable-Disease-Carrier-Status.pdf?sc_lang=en&hash=C98C951FDCF6D9425DF5ACE8BDD23801\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/151-200/MP170-Gene-Expression-Profiling-for-Breast-Cancer-Treatment.pdf?sc_lang=en&hash=5A55B2E142D76AF0B1D80E42110EC596\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/51-100/MP097-Genetic-Testing-for-BRCA1-or-BRCA2-for-Breast-or-Ovarian-Cancer.pdf?sc_lang=en&hash=B344B03B7AED26104C45DB501786A08E\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/301-350/MP324-Genetic-Testing-for-Non-cancer-Heritable-Disease-Carrier-Status.pdf?sc_lang=en&hash=C98C951FDCF6D9425DF5ACE8BDD23801\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/251-300/MP273-Gene-based-Testing-and-or-Protein-Biomarkers-for-Diagnosis-and-Management-of-Prostate-Cancer.pdf?sc_lang=en&hash=E0BE31E9831C08ED92AF2199F09416CA\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/51-100/MP097-Genetic-Testing-for-BRCA1-or-BRCA2-for-Breast-or-Ovarian-Cancer.pdf?sc_lang=en&hash=B344B03B7AED26104C45DB501786A08E\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/251-300/MP273-Gene-based-Testing-and-or-Protein-Biomarkers-for-Diagnosis-and-Management-of-Prostate-Cancer.pdf?sc_lang=en&hash=E0BE31E9831C08ED92AF2199F09416CA\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/201-250/MP239-Pharmacogenetic-Testing.pdf?la=en\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/251-300/MP271-Non-Invasive-Testing-for-Fetal-Aneuploidy.pdf?sc_lang=en&hash=6FFB2C9EE46D7801D1BA5BA95879FE67\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/251-300/MP280-Whole-Exome-Sequencing.pdf?la=en\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/301-350/MP323-Molecular-Profiling-of-Malignant-Tumors-to-Identify-Targeted-Therapies.pdf?sc_lang=en&hash=2412EEC6A272A60A8AA3620AB305535D\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/201-250/MP239-Pharmacogenetic-Testing.pdf?la=en\n",
      "https://www.geisinger.org/-/media/onegeisinger/pdfs/ghs/patient-care/patients-and-visitors/billing-and-insurance/hosp-charges-pdf/pdf-cdm-geisinger-medical-center-hospital-01-01-19\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/301-350/MP356-Genetic-Testing-for-Mitochondrial-Disorders.pdf?sc_lang=en&hash=DBEAC7DC1A1C271D60FE3A9F7FDA58CA\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/201-250/MP239-Pharmacogenetic-Testing.pdf?la=en\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/Whats-New-Updates/Whats-New-February-2022-MP.pdf?la=en\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/301-350/MP324-Genetic-Testing-for-Non-cancer-Heritable-Disease-Carrier-Status.pdf?sc_lang=en&hash=C98C951FDCF6D9425DF5ACE8BDD23801\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/201-250/MP239-Pharmacogenetic-Testing.pdf?la=en\n",
      "1\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/351-400/MP360-Minimal-Residual-Disease-NGS-Testing.pdf?sc_lang=en&hash=4E4B3E90CA1AFFCABDADDFA52C5F9EDF\n",
      "1\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/301-350/MP323-Molecular-Profiling-of-Malignant-Tumors-to-Identify-Targeted-Therapies.pdf?sc_lang=en&hash=2412EEC6A272A60A8AA3620AB305535D\n",
      "1\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/151-200/MP168-Non-invasive-Testing-for-Organ-Transplant-Rejection.pdf?sc_lang=en&hash=1BCBC019A87149D8A3C19E58D83B1ECB\n",
      "https://www.geisinger.org/-/media/OneGeisinger/pdfs/ghs/patient-care/patients-and-visitors/billing-and-insurance/hosp-charges-pdf/PDF-CDM-Geisinger-Holy-Spirit-Hospital-07-01-19.pdf?la=en\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/201-250/MP239-Pharmacogenetic-Testing.pdf?la=en\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/51-100/MP098-Genetic-Testing-Related-to-Colorectal-Cancer.pdf?sc_lang=en&hash=CC524BF79A7B0B6F0AB0D5822C399B73\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/51-100/MP097-Genetic-Testing-for-BRCA1-or-BRCA2-for-Breast-or-Ovarian-Cancer.pdf?sc_lang=en&hash=B344B03B7AED26104C45DB501786A08E\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/Whats-New-Updates/Whats-New-February-2023-MBP.pdf?sc_lang=en&hash=079099C8947CAB9C197B53D3E13E3DEA\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/301-350/MP323-Molecular-Profiling-of-Malignant-Tumors-to-Identify-Targeted-Therapies.pdf?sc_lang=en&hash=2412EEC6A272A60A8AA3620AB305535D\n",
      "https://www.geisinger.org/about-geisinger/news-and-media/news-releases/2018/05/07/12/18/dna-sequencing-to-become-part-of-geisingers-routine-clinical-care\n",
      "https://www.geisinger.org/-/media/OneGeisinger/pdfs/ghs/patient-care/patients-and-visitors/billing-and-insurance/hosp-charges-pdf/PDF-CDM-Geisinge-Comm-Medical-Ctr-07-01-19.pdf?la=en\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/Whats-New-Updates/Whats-New-July-2022-MP.pdf?sc_lang=en&hash=065C6196C83304404EBF0967A4DE825E\n",
      "https://www.geisinger.org/-/media/OneGeisinger/pdfs/ghs/research/mycode/Latest-MyCode-Results-Reported.pdf?la=en\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/301-350/MP324-Genetic-Testing-for-Non-cancer-Heritable-Disease-Carrier-Status.pdf?sc_lang=en&hash=C98C951FDCF6D9425DF5ACE8BDD23801\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/51-100/MP098-Genetic-Testing-Related-to-Colorectal-Cancer.pdf?sc_lang=en&hash=CC524BF79A7B0B6F0AB0D5822C399B73\n",
      "https://www.geisinger.org/-/media/OneGeisinger/pdfs/ghs/patient-care/patients-and-visitors/billing-and-insurance/hosp-charges-pdf/PDF-CDM-Geisinge-Comm-Medical-Ctr-07-01-19.pdf?la=en\n",
      "https://www.geisinger.org/-/media/OneGeisinger/pdfs/ghs/patient-care/patients-and-visitors/billing-and-insurance/hosp-charges-pdf/PDF-CDM-Geisinger-Wyoming-Valley-Medical-Ctr-07-01-19.pdf\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/301-350/MP323-Molecular-Profiling-of-Malignant-Tumors-to-Identify-Targeted-Therapies.pdf?sc_lang=en&hash=2412EEC6A272A60A8AA3620AB305535D\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/301-350/MP323-Molecular-Profiling-of-Malignant-Tumors-to-Identify-Targeted-Therapies.pdf?sc_lang=en&hash=2412EEC6A272A60A8AA3620AB305535D\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/301-350/MP323-Molecular-Profiling-of-Malignant-Tumors-to-Identify-Targeted-Therapies.pdf?sc_lang=en&hash=2412EEC6A272A60A8AA3620AB305535D\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/Whats-New-Updates/Whats-New-July-2022-MP.pdf?sc_lang=en&hash=065C6196C83304404EBF0967A4DE825E\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/PDFs/Geisinger-Gold/Monthly-Pharmacy-Updates/GHP-803-Medicare-Prior-Authorization-Criteria-Marketing.pdf?sc_lang=en&hash=2A877519B62CD5CD4AC591894E0E68E5\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/Whats-New-Updates/Whats-New-February-2022-MP.pdf?la=en\n",
      "https://www.geisinger.org/-/media/OneGeisinger/pdfs/ghs/patient-care/patients-and-visitors/billing-and-insurance/hosp-charges-pdf/PDF-CDM-Geisinger-Holy-Spirit-Hospital-07-01-19.pdf?la=en\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/301-350/MP326-Biomarker-Testing-for-Rheumatoid-Arthritis.pdf?sc_lang=en&hash=1F7F4FB89EFC8434ECA5CC4ADDECB5EE\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/151-200/MP170-Gene-Expression-Profiling-for-Breast-Cancer-Treatment.pdf?sc_lang=en&hash=5A55B2E142D76AF0B1D80E42110EC596\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/251-300/MP273-Gene-based-Testing-and-or-Protein-Biomarkers-for-Diagnosis-and-Management-of-Prostate-Cancer.pdf?sc_lang=en&hash=E0BE31E9831C08ED92AF2199F09416CA\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/Whats-New-Updates/Whats-New-July-2022-MP.pdf?sc_lang=en&hash=065C6196C83304404EBF0967A4DE825E\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/151-200/MP168-Non-invasive-Testing-for-Organ-Transplant-Rejection.pdf?sc_lang=en&hash=1BCBC019A87149D8A3C19E58D83B1ECB\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/Whats-New-Updates/Whats-New-February-2022-MP.pdf?la=en\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/351-400/MP363-Urinary-Tumor-Markers-for-Bladder-Cancer.pdf?sc_lang=en&hash=E3AA4C003D8AE047B9B6768BA21E9BCD\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/Whats-New-Updates/Whats-New-March-2022-MP.pdf?la=en\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/51-100/MP097-Genetic-Testing-for-BRCA1-or-BRCA2-for-Breast-or-Ovarian-Cancer.pdf?sc_lang=en&hash=B344B03B7AED26104C45DB501786A08E\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/Whats-New-Updates/Whats-New-February-2023-MP.pdf?sc_lang=en&hash=A270C120FF81DC7D3DA0EE40A35142EB\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/301-350/MP328-Genetic-Susceptibility-Cancer-Panels.pdf?sc_lang=en&hash=D553029B015A60AE4184DE16EA370A7E\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/51-100/MP097-Genetic-Testing-for-BRCA1-or-BRCA2-for-Breast-or-Ovarian-Cancer.pdf?sc_lang=en&hash=B344B03B7AED26104C45DB501786A08E\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/251-300/MP279-Gene-Expression-testing-to-Predict-Coronary-Artery-Disease.pdf?sc_lang=en&hash=1B024DF78D561D5736AE3ECD7368E1DA\n",
      "https://www.geisinger.org/about-geisinger/news-and-media/news-releases/2020/10/16/16/20/geisinger-opens-dedicated-cold-and-flu-walk-in-care-clinics\n",
      "https://www.geisinger.org/-/media/OneGeisinger/pdfs/ghs/patient-care/patients-and-visitors/billing-and-insurance/hosp-charges-pdf/PDF-CDM-Geisinger-Holy-Spirit-Hospital-07-01-19.pdf?la=en\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/Whats-New-Updates/Whats-New-May-2022-MP.pdf?sc_lang=en&hash=F7150C3F16FECA7F7534F8DAE61A54E5\n",
      "1\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/251-300/MP273-Gene-based-Testing-and-or-Protein-Biomarkers-for-Diagnosis-and-Management-of-Prostate-Cancer.pdf?sc_lang=en&hash=E0BE31E9831C08ED92AF2199F09416CA\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/251-300/MP265-Proteomic-Serum-Analysis.pdf?sc_lang=en&hash=8EBD37834935913D6501CCAC446387AC\n",
      "1\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/201-250/MP239-Pharmacogenetic-Testing.pdf?la=en\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/301-350/MP303-Genomic-Analysis-to-Predict-Thyroid-Malignancy-in-FNA.pdf?sc_lang=en&hash=8663394755286FC1344B6890FE8F3D07\n",
      "https://www.geisinger.org/-/media/onegeisinger/pdfs/ghs/research/find-an-investigator/pdfs/find-an-investigator/boscarino/boscarino-cv-012414-1\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/251-300/MP255-Comparative-Genomic-Hybridization-or-Chromosomal-Microarray-Analysis-CMA-for-Evaluation-of-Developmental-Delay.pdf?sc_lang=en&hash=E3D273DD2B314F8B1C1D9FE3EEF8AB23\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/301-350/MP304-Genetic-Testing-for-Inherited-Cardiomyopathies-and-Channelopathies.pdf?sc_lang=en&hash=8562D9547B5F629A2044201D571A82EB\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/Whats-New-Updates/Whats-New-February-2022-MP.pdf?la=en\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/301-350/MP323-Molecular-Profiling-of-Malignant-Tumors-to-Identify-Targeted-Therapies.pdf?sc_lang=en&hash=2412EEC6A272A60A8AA3620AB305535D\n",
      "https://www.geisinger.org/-/media/OneGeisinger/Files/Policy-PDFs/MP/301-350/MP321-Gene-Expression-Profiling-for-Cutaneous-Melanoma.pdf?la=en\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Algorithmic-Testing-MP9605\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-intell-develop-MP9587\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-Immune-MP9597\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-Cancer-MP9596\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Algorithmic-Testing-MP9605\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Algorithmic-Testing-MP9605\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Algorithmic-Testing-MP9605\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Molecular-Analysis-9608\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-Neuro-Epil-MP9591\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Algorithmic-Testing-MP9605\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-9610\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-intell-develop-MP9587\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-9610\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-Hemat-MP9595\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Algorithmic-Testing-MP9605\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-Cancer-9596\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-Cardiac-9589\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Algorithmic-Testing-MP9605\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Lab-Testing-9539\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Algorithmic-Testing-MP9605\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Algorithmic-Testing-MP9605\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Molecular-Analysis-9608\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Algorithmic-Testing-MP9605\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Cancer-Screening-MP9606\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-intell-develop-MP9587\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Algorithmic-Testing-MP9605\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Molecular-Analysis-9608\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Algorithmic-Testing-MP9605\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Molecular-Analysis-9608\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Molecular-Analysis-9608\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Algorithmic-Testing-MP9605\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Algorithmic-Testing-MP9605\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Algorithmic-Testing-MP9605\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-Aortopathies-Connect-Tissue-9588\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Algorithmic-Testing-MP9605\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Algorithmic-Testing-MP9605\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Algorithmic-Testing-MP9605\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Tumor-DNA-9609\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-Kidney-MP9598\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Algorithmic-Testing-MP9605\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Algorithmic-Testing-MP9605\n",
      "1\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Algorithmic-Testing-MP9605\n",
      "1\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Molecular-Analysis-9608\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-Cancer-MP9596\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-Cardiac-9589\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-intell-develop-MP9587\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Algorithmic-Testing-MP9605\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-Neuro-Epil-MP9591\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Algorithmic-Testing-MP9605\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-Prenatal-9575\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Molecular-Analysis-9608\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Molecular-Analysis-9608\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-intell-develop-MP9587\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Algorithmic-Testing-MP9605\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Molecular-Analysis-9608\n",
      "1\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Molecular-Analysis-9608\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-Neuro-Epil-MP9591\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Molecular-Analysis-9608\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-intell-develop-MP9587\n",
      "1\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Molecular-Analysis-9608\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Molecular-Analysis-9608\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Algorithmic-Testing-MP9605\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-Prenatal-Diagnosis-9576\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-9610\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-Neuro-Epil-MP9591\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-Cancer-9596\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-9610\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-Cardiac-MP9589\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-Cancer-9596\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-9610\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-Cancer-MP9596\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Molecular-Analysis-9608\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Molecular-Analysis-9608\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Molecular-Analysis-9608\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Molecular-Analysis-9608\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-Pharmaco-MP9602\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-Pharmaco-MP9602\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-Gastro-MP9593\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Molecular-Analysis-9608\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Molecular-Analysis-9608\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Algorithmic-Testing-MP9605\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Molecular-Analysis-9608\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-intell-develop-MP9587\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Molecular-Analysis-9608\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Molecular-Analysis-9608\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Molecular-Analysis-9608\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-9610\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Molecular-Analysis-9608\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Algorithmic-Testing-MP9605\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-9610\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-intell-develop-MP9587\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medicare/Medicare-2023/MAPD-PA-criteria-2023\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-Metabolic-MP9600\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-NIPS-9573\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-NIPS-9573\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Molecular-Analysis-9608\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-9610\n",
      "https://www.prevea360.com/Document-Library/PDF/covered-genetic-testing-that-does-not-require-a-pa.aspx\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Molecular-Analysis-9608\n",
      "1\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-intell-develop-MP9587\n",
      "https://www.prevea360.com/Document-Library/PDF/covered-genetic-testing-that-does-not-require-a-pa.aspx\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-NIPS-9573\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-intell-develop-MP9587\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-Metabolic-MP9600\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Molecular-Analysis-9608\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-9610\n",
      "1\n",
      "1\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-Cancer-MP9596\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Lab-Testing-9539\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-9610\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Molecular-Analysis-9608\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-Cancer-MP9596\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-NIPS-9573\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-Neuro-Epil-MP9591\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-Prenatal-9575\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Molecular-Analysis-9608\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-9610\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-Cancer-MP9596\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-Cancer-9596\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-Prenatal-9575\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-Hematologic-9595\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-Cancer-MP9596\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-Pharmaco-MP9602\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-Prenatal-9575\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-Exome-9586\n",
      "1\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-9610\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-Pharmacogen-9602\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-Kidney-MP9598\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-NIPS-9573\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-Eye-MP9592\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-Cancer-MP9596\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-NIPS-9573\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-Kidney-MP9598\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Molecular-Analysis-9608\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Algorithmic-Testing-MP9605\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-Pharmaco-MP9602\n",
      "1\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Genetic-Testing-Cancer-MP9596\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Molecular-Analysis-9608\n",
      "https://www.prevea360.com/DocumentLibrary/PDF/Medical-Policies/Oncology-Cancer-Screening-MP9606\n",
      "https://blue.regence.com/trgmedpol/lab/lab73.pdf\n",
      "https://blue.regence.com/medicare/gt/m-gt20.pdf\n",
      "https://blue.regence.com/medicare/gt/m-gt64.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt17.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt42.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt11.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt42.pdf\n",
      "https://blue.regence.com/trgmedpol/lab/lab60.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt86.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt06.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt42.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt42.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt17.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt59.pdf\n",
      "https://blue.regence.com/medicare/gt/m-gt20.pdf\n",
      "https://blue.regence.com/medicare/gt/m-gt64.pdf\n",
      "https://blue.regence.com/trgmedpol/transplant/tra45.31.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt59.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt17.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt59.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt20.pdf\n",
      "https://blue.regence.com/medicare/lab/m-lab74.pdf\n",
      "https://blue.regence.com/trgmedpol/lab/lab51.pdf\n",
      "https://blue.regence.com/medicare/gt/m-gt64.pdf\n",
      "https://blue.regence.com/trgmedpol/transplant/tra45.24.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt69.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt77.pdf\n",
      "https://blue.regence.com/medicare/gt/m-gt20.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt52.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt06.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt06.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt06.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt29.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt15.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt81.pdf\n",
      "https://blue.regence.com/medicare/gt/m-gt20.pdf\n",
      "https://blue.regence.com/trgmedpol/lab/lab47.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt76.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt64.pdf\n",
      "https://blue.regence.com/medicare/gt/m-gt64.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt63.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt13.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt02.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt78.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt42.pdf\n",
      "https://blue.regence.com/medicare/gt/m-gt20.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt64.pdf\n",
      "https://blue.regence.com/medicare/gt/m-gt64.pdf\n",
      "https://blue.regence.com/medicare/gt/m-gt20.pdf\n",
      "https://blue.regence.com/medicare/gt/m-gt20.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt20.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt06.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt83.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt59.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt02.pdf\n",
      "https://blue.regence.com/medicare/gt/m-gt64.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt17.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt13.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt19.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt22.pdf\n",
      "https://blue.regence.com/trgmedpol/lab/lab47.pdf\n",
      "https://blue.regence.com/medicare/gt/m-gt64.pdf\n",
      "https://blue.regence.com/medicare/gt/m-gt64.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt11.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt17.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt02.pdf\n",
      "https://blue.regence.com/medicare/gt/m-gt20.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt29.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt06.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt02.pdf\n",
      "https://blue.regence.com/trgmedpol/lab/lab67.pdf\n",
      "https://blue.regence.com/medicare/gt/m-gt64.pdf\n",
      "https://blue.regence.com/trgmedpol/lab/lab72.pdf\n",
      "https://blue.regence.com/medicare/gt/m-gt20.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt29.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt81.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt53.pdf\n",
      "https://blue.regence.com/medicare/gt/m-gt64.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt83.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt17.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt44.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt17.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt77.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt64.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt06.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt06.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt44.pdf\n",
      "https://blue.regence.com/medicare/gt/m-gt64.pdf\n",
      "https://blue.regence.com/medicare/gt/m-gt20.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt78.pdf\n",
      "https://blue.regence.com/medicare/gt/m-gt64.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt78.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt85.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt10.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt02.pdf\n",
      "https://blue.regence.com/medicare/gt/m-gt20.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt56.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt02.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt59.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt54.pdf\n",
      "https://blue.regence.com/trgmedpol/lab/lab60.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt83.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt06.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt63.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt13.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt85.pdf\n",
      "https://blue.regence.com/trgmedpol/lab/lab72.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt12.pdf\n",
      "https://blue.regence.com/medicare/gt/m-gt20.pdf\n",
      "https://blue.regence.com/medicare/gt/m-gt64.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt64.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt54.pdf\n",
      "https://blue.regence.com/medicare/lab/m-lab74.pdf\n",
      "https://blue.regence.com/medicare/lab/m-lab74.pdf\n",
      "https://blue.regence.com/medicare/gt/m-gt20.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt81.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt59.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt59.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt64.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt44.pdf\n",
      "https://blue.regence.com/trgmedpol/lab/lab51.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt85.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt06.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt83.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt10.pdf\n",
      "https://blue.regence.com/medicare/gt/m-gt64.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt29.pdf\n",
      "https://blue.regence.com/medicare/gt/m-gt20.pdf\n",
      "https://blue.regence.com/trgmedpol/lab/lab51.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt02.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt21.pdf\n",
      "https://blue.regence.com/medicare/gt/m-gt20.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt02.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt83.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt76.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt66.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt68.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt19.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt83.pdf\n",
      "https://blue.regence.com/medicare/gt/m-gt64.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt06.pdf\n",
      "https://blue.regence.com/medicare/lab/m-lab74.pdf\n",
      "https://blue.regence.com/trgmedpol/lab/lab46.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt76.pdf\n",
      "https://blue.regence.com/medicare/gt/m-gt20.pdf\n",
      "https://blue.regence.com/trgmedpol/lab/lab41.pdf\n",
      "https://blue.regence.com/trgmedpol/geneticTesting/gt02.pdf\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/gene-express-profiling-prostate-cancer-management\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/gene-expression-profiling-breast-cancer\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/gene-expression-profiling-breast-cancer\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/gene-expression-profiling-breast-cancer\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/genetic-testing-brca-outside-breast-ovarian-cancer\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/gene-express-profiling-prostate-cancer-management\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/gene-expression-profiling-breast-cancer\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/genetic-assays-colon-cancer\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/Molecular-markers-fine-needle-aspirate\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/laboratory-testing-transplant-rejection\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/gene-expression-uveal-melanoma\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/Molecular-markers-fine-needle-aspirate\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/expanded-genetic-panels-cancer-therapy\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/next-generation-sequencing-assessment-mrd\n",
      "1\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/gene-expression-assays-cancer-unknown-primary\n",
      "1\n",
      "1\n",
      "1\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/genetic-testing-myeloproliferative-disease\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/molecular-analysis-targeted-therapy\n",
      "1\n",
      "1\n",
      "1\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/kras-nras-braf-mutation-analysis\n",
      "1\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/circulating-tumor-dna-non-small-cell-lung-cancer\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/genetic-testing-variants-leukemia\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/genetic-testing-BCR-ABL-fusion-gene\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/multimarker-serum-testing-ovarian-cancer\n",
      "1\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/noninvasive-prenatal-screening-fetal-aneuploidies\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/expanded-genetic-panels-cancer-risk\n",
      "1\n",
      "https://www.wellmark.com/provider/medical-policies-authorizations/medical-policies-a-z\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/testing-common-genetic-variants-non-familial-breast\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/genetic-testing-for-marfan-syndrome-thoracic-aortic-aneurysms-and-dissections-and-related-disorders\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/Pharmacogenomic-test-drug-metabolism\n",
      "1\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/expanded-genetic-panels-cancer-risk\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/genetic-testing-brca-outside-breast-ovarian-cancer\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/Pharmacogenomic-test-drug-metabolism\n",
      "1\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/genetic-testing-determine-cardiac-disease\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/molecular-analysis-targeted-therapy\n",
      "https://www.wellmark.com/-/media/sites/public/files/provider/blueink/blueink-0822.pdf?sc_lang=en&hash=2F5A630CAB93429CCA9D7FD238268D8A\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/genetic-testing-variants-leukemia\n",
      "1\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/genetic-testing-brca-outside-breast-ovarian-cancer\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/genetic-testing-determine-cardiac-disease\n",
      "https://www.wellmark.com/provider/medical-policies-authorizations/medical-policies-a-z\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/expanded-genetic-panels-cancer-therapy\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/whole-exome-whole-genome-sequencing\n",
      "1\n",
      "https://www.wellmark.com/provider/medical-policies-authorizations/medical-policies-a-z\n",
      "1\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/genetic-testing-myeloproliferative-disease\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/genetic-protein-biomarker\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/genetic-testing-determine-cardiac-disease\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/expanded-genetic-panels-cancer-risk\n",
      "https://www.wellmark.com/provider/medical-policies-authorizations/medical-policies-a-z\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/genetic-testing-for-marfan-syndrome-thoracic-aortic-aneurysms-and-dissections-and-related-disorders\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/expanded-genetic-panels-cancer-risk\n",
      "1\n",
      "1\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/genetic-testing-determine-cardiac-disease\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/proteomic-testing-systemic-therapy-non-small-cell-lung-cancer\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/expanded-genetic-panels-cancer-risk\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/gene-expression-cutaneous-melanoma\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/genetic-protein-biomarker\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/molecular-analysis-targeted-therapy\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/expanded-genetic-panels-cancer-risk\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/cardiovascular-disease-risk-tests\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/multimarker-serum-testing-ovarian-cancer\n",
      "1\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/molecular-analysis-targeted-therapy\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/biomarker-testing-autoimmune-disease\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/expanded-genetic-panels-cancer-therapy\n",
      "1\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/genetic-testing-variants-leukemia\n",
      "1\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/Pharmacogenomic-test-drug-metabolism\n",
      "https://www.wellmark.com/provider/medical-policies-authorizations/medical-policies-a-z\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/genetic-testing-myeloproliferative-disease\n",
      "1\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/genetic-protein-biomarker\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/genetic-testing-brca-outside-breast-ovarian-cancer\n",
      "1\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/Pharmacogenomic-test-drug-metabolism\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/next-generation-sequencing-assessment-mrd\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/genetic-testing-variants-leukemia\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/kras-nras-braf-mutation-analysis\n",
      "https://www.wellmark.com/finder\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/whole-exome-whole-genome-sequencing\n",
      "https://www.wellmark.com/-/media/sites/public/files/provider/medicare-advantage/medical-prior-authorization-list.pdf?sc_lang=en&hash=2D4BBFF4D0CEA0F7EC9F9BBAC0FFA71E\n",
      "https://www.wellmark.com/-/media/sites/public/files/provider/pharmacy/luxturna.pdf?sc_lang=en&hash=A238E571481FF31CC306DE07DA354DE9\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/expanded-genetic-panels-cancer-risk\n",
      "1\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/noninvasive-prenatal-screening-fetal-aneuploidies\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/molecular-analysis-targeted-therapy\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/egfr-epidermal-growth-factor-receptor-testing\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/genetic-testing-brca-outside-breast-ovarian-cancer\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/whole-exome-whole-genome-sequencing\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/whole-exome-whole-genome-sequencing\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/molecular-analysis-targeted-therapy\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/expanded-genetic-panels-cancer-therapy\n",
      "1\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/noninvasive-prenatal-screening-fetal-aneuploidies\n",
      "1\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/whole-exome-whole-genome-sequencing\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/genetic-protein-biomarker\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/noninvasive-prenatal-screening-fetal-aneuploidies\n",
      "1\n",
      "1\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/gene-expression-cutaneous-melanoma\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/circulating-tumor-dna-cancer-management\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/circulating-tumor-dna-cancer-management\n",
      "1\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/laboratory-testing-transplant-rejection\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/laboratory-testing-transplant-rejection\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/circulating-tumor-dna-cancer-management\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/genetic-testing-brca-outside-breast-ovarian-cancer\n",
      "1\n",
      "1\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/expanded-genetic-panels-cancer-risk\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/molecular-analysis-targeted-therapy\n",
      "1\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/expanded-genetic-panels-cancer-therapy\n",
      "1\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/genetic-testing-brca-outside-breast-ovarian-cancer\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/genetic-testing-brca-outside-breast-ovarian-cancer\n",
      "1\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/gene-expression-profiling-breast-cancer\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/whole-exome-whole-genome-sequencing\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/noninvasive-prenatal-screening-fetal-aneuploidies\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/whole-exome-whole-genome-sequencing\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/expanded-genetic-panels-cancer-risk\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/Pharmacogenomic-test-drug-metabolism\n",
      "1\n",
      "1\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/expanded-genetic-panels-cancer-therapy\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/gene-expression-cutaneous-melanoma\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/exondys-51\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/gene-express-profiling-prostate-cancer-management\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/molecular-testing-pulmonary-nodules\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/noninvasive-prenatal-screening-fetal-aneuploidies\n",
      "https://www.wellmark.com/-/media/sites/public/files/medical-policies/genetic-testing-BCR-ABL-fusion-gene\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2010015&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2015009&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/providers/coverage_policy.aspx\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2012011&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2012005\n",
      "https://secure.arkansasbluecross.com/providers/coverage_policy.aspx\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2018015&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2013044&viewIntro=yes\n",
      "https://www.arkansasbluecross.com/members/coverage_policy.aspx\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2004017\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=1997185&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2012003&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/providers/coverage_policy.aspx\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2015009&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2012005\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2013020&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=1998051&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=1997185&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2011060&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=1998051&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2004029&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2013035\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2010039&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2008022&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2004029&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2004029&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2017019&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2004017\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2004029&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2012003&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2012005\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2012005\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2015009&viewIntro=yes\n",
      "1\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policynumber=2008010\n",
      "https://secure.arkansasbluecross.com/providers/coverage_policy.aspx\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2004029&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2012025&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=1998051&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2015009&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2016023&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/providers/coverage_policy.aspx\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2007016&viewIntro=yes\n",
      "https://www.arkansasbluecross.com/members/coverage_policy.aspx\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2013022&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2015008\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2013045&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2016004&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2013023&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2011045&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2015009&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/providers/coverage_policy.aspx\n",
      "https://secure.arkansasbluecross.com/providers/coverage_policy.aspx\n",
      "https://www.arkansasbluecross.com/members/coverage_policy.aspx\n",
      "https://secure.arkansasbluecross.com/providers/coverage_policy.aspx\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2015008\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2015009&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2002002&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2008014&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2005003&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2012068&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2015004&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/providers/coverage_policy.aspx\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=1998051&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/providers/coverage_policy.aspx\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2015009&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=1998051&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2013022&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2005003&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2015009&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/providers/coverage_policy.aspx\n",
      "https://secure.arkansasbluecross.com/providers/coverage_policy.aspx\n",
      "https://secure.arkansasbluecross.com/providers/coverage_policy.aspx\n",
      "https://secure.arkansasbluecross.com/providers/coverage_policy.aspx\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2012043&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2005021&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/providers/coverage_policy.aspx\n",
      "https://secure.arkansasbluecross.com/providers/coverage_policy.aspx\n",
      "https://secure.arkansasbluecross.com/providers/coverage_policy.aspx\n",
      "https://secure.arkansasbluecross.com/providers/coverage_policy.aspx\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2015008&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/providers/coverage_policy.aspx\n",
      "https://secure.arkansasbluecross.com/providers/coverage_policy.aspx\n",
      "https://www.arkansasbluecross.com/members/coverage_policy.aspx\n",
      "https://secure.arkansasbluecross.com/providers/coverage_policy.aspx\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2007016&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/providers/coverage_policy.aspx\n",
      "https://secure.arkansasbluecross.com/providers/coverage_policy.aspx\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2012019&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2015009&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/providers/coverage_policy.aspx\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=1998051&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/providers/coverage_policy.aspx\n",
      "https://secure.arkansasbluecross.com/providers/coverage_policy.aspx\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2018015&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2015007&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2015007&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2016004&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2022040&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2004017\n",
      "https://secure.arkansasbluecross.com/providers/coverage_policy.aspx\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2015008\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2022040&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2022044&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2013046&viewIntro=yes\n",
      "https://www.arkansasbluecross.com/members/coverage_policy.aspx\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2012005\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2014012&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2013007&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2012014&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2015008\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2014012&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2005021&viewIntro=yes\n",
      " https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2007011\n",
      "https://secure.arkansasbluecross.com/providers/coverage_policy.aspx\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2005021&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2010014&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2012049\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2013035\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2016004&viewIntro=yes\n",
      "1\n",
      "https://secure.arkansasbluecross.com/providers/coverage_policy.aspx\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2014012\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=1998051&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2016004&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2016004&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/providers/coverage_policy.aspx\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=1998051&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2018015&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2010030&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/providers/coverage_policy.aspx\n",
      "https://secure.arkansasbluecross.com/providers/coverage_policy.aspx\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2022017&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/providers/coverage_policy.aspx\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2022040&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2015009&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2010014&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/providers/coverage_policy.aspx\n",
      "https://secure.arkansasbluecross.com/providers/coverage_policy.aspx\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2015009&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2016004&viewIntro=yes\n",
      "https://www.arkansasbluecross.com/docs/librariesprovider9/providers-news/january-2023.pdf?sfvrsn=b8724afd_3\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2012068&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2015009&viewIntro=yes\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2013045\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2004017\n",
      "https://secure.arkansasbluecross.com/members/report.aspx?policyNumber=2018015&viewIntro=yes\n",
      "https://www.policy.asuris.com/medicare/gt/m-gt64.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt17.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt42.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt29.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt29.pdf\n",
      "https://www.policy.asuris.com/medicare/gt/m-gt64.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt42.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt22.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt77.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt17.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt42.pdf\n",
      "https://www.policy.asuris.com/medicare/gt/m-gt20.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt06.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt17.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt29.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt42.pdf\n",
      "https://www.policy.asuris.com/medicare/gt/m-gt64.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt17.pdf\n",
      "https://www.policy.asuris.com/medicare/gt/m-gt64.pdf\n",
      "https://www.policy.asuris.com/lab/lab72.pdf\n",
      "https://www.policy.asuris.com/lab/lab67.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt64.pdf\n",
      "https://www.policy.asuris.com/lab/lab51.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt06.pdf\n",
      "https://www.policy.asuris.com/medicare/gt/m-gt64.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt02.pdf\n",
      "https://www.policy.asuris.com/lab/lab72.pdf\n",
      "https://www.policy.asuris.com/medicare/gt/m-gt64.pdf\n",
      "https://www.policy.asuris.com/medicare/gt/m-gt64.pdf\n",
      "https://www.policy.asuris.com/lab/lab73.pdf\n",
      "https://www.policy.asuris.com/lab/lab60.pdf\n",
      "https://www.policy.asuris.com/lab/lab73.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt17.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt17.pdf\n",
      "https://www.policy.asuris.com/medicare/gt/m-gt20.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt83.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt83.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt11.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt66.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt63.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt02.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt20.pdf\n",
      "https://www.policy.asuris.com/medicare/gt/m-gt20.pdf\n",
      "https://www.policy.asuris.com/medicare/lab/m-lab74.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt76.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt12.pdf\n",
      "https://www.policy.asuris.com/medicare/gt/m-gt64.pdf\n",
      "https://www.policy.asuris.com/medicare/gt/m-gt20.pdf\n",
      "https://www.policy.asuris.com/lab/lab46.pdf\n",
      "https://www.policy.asuris.com/medicare/lab/m-lab74.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt10.pdf\n",
      "https://www.policy.asuris.com/medicare/lab/m-lab74.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt06.pdf\n",
      "https://www.policy.asuris.com/medicare/gt/m-gt20.pdf\n",
      "https://www.policy.asuris.com/medicare/gt/m-gt64.pdf\n",
      "https://www.policy.asuris.com/medicare/gt/m-gt20.pdf\n",
      "https://www.policy.asuris.com/lab/lab47.pdf\n",
      "https://www.policy.asuris.com/medicare/gt/m-gt64.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt06.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt85.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt81.pdf\n",
      "https://www.policy.asuris.com/lab/lab60.pdf\n",
      "https://www.policy.asuris.com/medicare/gt/m-gt20.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt83.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt06.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt68.pdf\n",
      "https://www.policy.asuris.com/medicare/gt/m-gt64.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt06.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt17.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt64.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt77.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt06.pdf\n",
      "https://www.policy.asuris.com/lab/lab51.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt06.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt85a.pdf\n",
      "https://www.policy.asuris.com/transplant/tra45.28.pdf\n",
      "https://www.policy.asuris.com/medicare/gt/m-gt20.pdf\n",
      "http://www.policy.asuris.com/transplant/tra45.28.pdf\n",
      "https://www.policy.asuris.com/medicare/contents/g.html\n",
      "https://www.policy.asuris.com/geneticTesting/gt13.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt02.pdf\n",
      "https://www.policy.asuris.com/medicare/gt/m-gt20.pdf\n",
      "https://www.policy.asuris.com/lab/lab47.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt64.pdf\n",
      "https://www.policy.asuris.com/medicare/gt/m-gt64.pdf\n",
      "https://www.policy.asuris.com/medicare/gt/m-gt20.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt19.pdf\n",
      "http://www.policy.asuris.com/transplant/tra45.28.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt54.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt64.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt02.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt69.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt81.pdf\n",
      "https://www.policy.asuris.com/medicare/gt/m-gt20.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt64.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt83.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt76.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt85.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt02.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt13.pdf\n",
      "https://www.policy.asuris.com/transplant/tra45.31.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt02.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt13.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt10.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt64.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt52.pdf\n",
      "https://www.policy.asuris.com/lab/lab51.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt44.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt02.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt76.pdf\n",
      "https://www.policy.asuris.com/medicare/lab/m-lab74.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt53.pdf\n",
      "https://www.policy.asuris.com/medicare/gt/m-gt20.pdf\n",
      "https://www.policy.asuris.com/medicare/gt/m-gt20.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt79.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt06.pdf\n",
      "https://www.policy.asuris.com/medicare/gt/m-gt20.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt56.pdf\n",
      "https://www.policy.asuris.com/lab/lab46.pdf\n",
      "https://www.policy.asuris.com/medicare/gt/m-gt64.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt83.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt63.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt29.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt19.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt02.pdf\n",
      "https://www.policy.asuris.com/medicare/gt/m-gt20.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt86.pdf\n",
      "https://www.policy.asuris.com/medicare/gt/m-gt20.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt06.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt54.pdf\n",
      "https://www.policy.asuris.com/medicare/gt/m-gt20.pdf\n",
      "https://www.policy.asuris.com/medicare/gt/m-gt20.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt44.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt44.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt58.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt06.pdf\n",
      "https://www.policy.asuris.com/medicare/gt/m-gt64.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt81.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt42.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt20.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt81.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt76.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt11.pdf\n",
      "https://www.policy.asuris.com/geneticTesting/gt21.pdf\n",
      "https://www.policy.asuris.com/medicare/gt/m-gt64.pdf\n",
      "https://www.policy.asuris.com/medicare/gt/m-gt64.pdf\n",
      "https://www.policy.asuris.com/medicare/gt/m-gt20.pdf\n",
      "https://www.bcbsks.com/medical-policies/assays-genetic-expression-tumor-tissue-technique-determine-prognosis-patients\n",
      "https://www.bcbsks.com/medical-policies/molecular-markers-fine-needle-aspiration-thyroid-2023-01-03\n",
      "https://www.bcbsks.com/medical-policies/assays-genetic-expression-tumor-tissue-technique-determine-prognosis-patients\n",
      "https://www.bcbsks.com/medical-policies/gene-expression-based-assays-cancers-unknown-primary-2023-04-25\n",
      "1\n",
      "https://www.bcbsks.com/medical-policies/assays-genetic-expression-tumor-tissue-technique-determine-prognosis-patients\n",
      "https://www.bcbsks.com/medical-policies/assays-genetic-expression-tumor-tissue-technique-determine-prognosis-patients\n",
      "https://www.bcbsks.com/medical-policies/laboratory-tests-post-transplant-and-heart-failure-2022-12-13\n",
      "https://www.bcbsks.com/medical-policies/somatic-biomarker-testing-including-liquid-biopsy-targeted-treatment-and-2023-04-03\n",
      "https://www.bcbsks.com/medical-policies/germline-genetic-testing-hereditary-breastovarian-cancer-syndrome-and-other-high-2023-01-24\n",
      "https://www.bcbsks.com/medical-policies/gene-expression-profiling-uveal-melanoma-0\n",
      "https://www.bcbsks.com/medical-policies/somatic-biomarker-testing-including-liquid-biopsy-targeted-treatment-and-2023-04-03\n",
      "https://www.bcbsks.com/medical-policies/genetic-testing-lynch-syndrome-and-other-inherited-colon-cancer-syndromes\n",
      "https://www.bcbsks.com/medical-policies/molecular-markers-fine-needle-aspiration-thyroid-2023-01-03\n",
      "https://www.bcbsks.com/medical-policies/somatic-biomarker-testing-including-liquid-biopsy-targeted-treatment-and-2023-04-03\n",
      "https://www.bcbsks.com/medical-policies/general-approach-evaluating-utility-genetic-panels-2023-01-03\n",
      "https://www.bcbsks.com/medical-policies/genetic-and-protein-biomarkers-diagnosis-and-cancer-risk-assessment-prostate\n",
      "https://www.bcbsks.com/medical-policies/genetic-and-protein-biomarkers-diagnosis-and-cancer-risk-assessment-prostate\n",
      "https://www.bcbsks.com/medical-policies/general-approach-evaluating-utility-genetic-panels-2023-01-03\n",
      "https://www.bcbsks.com/medical-policies/gene-expression-profiling-cutaneous-melanoma-2022-07-12\n",
      "https://www.bcbsks.com/medical-policies/gene-expression-profiling-cutaneous-melanoma-2022-07-12\n",
      "1\n",
      "https://www.bcbsks.com/medical-policies/genetic-and-protein-biomarkers-diagnosis-and-cancer-risk-assessment-prostate\n",
      "1\n",
      "https://www.bcbsks.com/medical-policies/genetic-cancer-susceptibility-panels-using-next-generation-sequencing-2022-11-22\n",
      "https://www.bcbsks.com/medical-policies/general-approach-evaluating-utility-genetic-panels-2023-01-03\n",
      "https://www.bcbsks.com/medical-policies/genetic-testing-lynch-syndrome-and-other-inherited-colon-cancer-syndromes\n",
      "https://www.bcbsks.com/medical-policies/genetic-testing-cardiac-ion-channelopathies-2023-02-28\n",
      "https://www.bcbsks.com/medical-policies/somatic-biomarker-testing-including-liquid-biopsy-targeted-treatment-and-2023-04-03\n",
      "https://www.bcbsks.com/medical-policies/genetic-testing-marfan-syndrome-thoracic-aortic-aneurysms-and-dissections-and-2023-03-28\n",
      "https://www.bcbsks.com/medical-policies/genetic-testing-marfan-syndrome-thoracic-aortic-aneurysms-and-dissections-and-2023-03-28\n",
      "https://www.bcbsks.com/medical-policies/genetic-cancer-susceptibility-panels-using-next-generation-sequencing-2022-11-22\n",
      "https://www.bcbsks.com/medical-policies/identification-microorganisms-using-nucleic-acid-testing\n",
      "https://www.bcbsks.com/medical-policies/genetic-and-protein-biomarkers-diagnosis-and-cancer-risk-assessment-prostate\n",
      "https://www.bcbsks.com/medical-policies/genetic-testing-cardiac-ion-channelopathies-2023-02-28\n",
      "https://www.bcbsks.com/medical-policies/analysis-human-dna-stool-samples-technique-colorectal-cancer-screening-2022-12-29\n",
      "https://www.bcbsks.com/medical-policies/germline-genetic-testing-hereditary-breastovarian-cancer-syndrome-and-other-high-2023-01-24\n",
      "https://www.bcbsks.com/medical-policies/assays-genetic-expression-tumor-tissue-technique-determine-prognosis-patients\n",
      "https://www.bcbsks.com/medical-policies/genetic-cancer-susceptibility-panels-using-next-generation-sequencing-2022-11-22\n",
      "https://www.bcbsks.com/medical-policies/identification-microorganisms-using-nucleic-acid-testing-2023-04-03\n",
      "https://www.bcbsks.com/medical-policies/noninvasive-techniques-evaluation-and-monitoring-patients-chronic-liver-disease-2022-12-29\n",
      "https://www.bcbsks.com/medical-policies/germline-genetic-testing-hereditary-breastovarian-cancer-syndrome-and-other-high-2023-01-24\n",
      "1\n",
      "https://www.bcbsks.com/medical-policies/noninvasive-techniques-evaluation-and-monitoring-patients-chronic-liver-disease-2022-12-29\n",
      "https://www.bcbsks.com/medical-policies/identification-microorganisms-using-nucleic-acid-testing-2023-04-03\n",
      "https://www.bcbsks.com/medical-policies/general-approach-genetic-testing\n",
      "https://www.bcbsks.com/medical-policies/general-approach-evaluating-utility-genetic-panels-2023-01-03\n",
      "https://www.bcbsks.com/medical-policies/general-approach-evaluating-utility-genetic-panels-2023-01-03\n",
      "https://www.bcbsks.com/medical-policies/genetic-testing-predisposition-inherited-hypertrophic-cardiomyopathy-2023-04-25\n",
      "1\n",
      "https://www.bcbsks.com/medical-policies/general-approach-genetic-testing\n",
      "https://www.bcbsks.com/medical-policies/genetic-testing-lynch-syndrome-and-other-inherited-colon-cancer-syndromes\n",
      "https://www.bcbsks.com/medical-policies/general-approach-evaluating-utility-genetic-panels-2023-01-03\n",
      "https://www.bcbsks.com/medical-policies/general-approach-genetic-testing\n",
      "https://www.bcbsks.com/medical-policies/general-approach-evaluating-utility-genetic-panels-2023-01-03\n",
      "https://www.bcbsks.com/medical-policies/genetic-cancer-susceptibility-panels-using-next-generation-sequencing-2022-11-22\n",
      "https://www.bcbsks.com/medical-policies/genetic-testing-diagnosis-inherited-peripheral-neuropathies-2023-02-28\n",
      "https://www.bcbsks.com/medical-policies/genetic-testing-lynch-syndrome-and-other-inherited-colon-cancer-syndromes\n",
      "https://www.bcbsks.com/medical-policies/general-approach-evaluating-utility-genetic-panels-2023-01-03\n",
      "https://www.bcbsks.com/medical-policies/general-approach-evaluating-utility-genetic-panels-2023-01-03\n",
      "https://www.bcbsks.com/medical-policies/general-approach-evaluating-utility-genetic-panels-2023-01-03\n",
      "https://www.bcbsks.com/medical-policies/somatic-biomarker-testing-including-liquid-biopsy-targeted-treatment-and-2022-10-28\n",
      "https://www.bcbsks.com/medical-policies/genetic-cancer-susceptibility-panels-using-next-generation-sequencing-2022-11-22\n",
      "https://www.bcbsks.com/medical-policies/somatic-biomarker-testing-including-liquid-biopsy-targeted-treatment-and-2023-04-03\n",
      "https://www.bcbsks.com/medical-policies/genetic-cancer-susceptibility-panels-using-next-generation-sequencing-2022-11-22\n",
      "https://www.bcbsks.com/medical-policies/somatic-biomarker-testing-including-liquid-biopsy-targeted-treatment-and-2022-10-28\n",
      "https://www.bcbsks.com/medical-policies/general-approach-evaluating-utility-genetic-panels-2023-01-03\n",
      "https://www.bcbsks.com/medical-policies/general-approach-evaluating-utility-genetic-panels-2023-01-03\n",
      "https://www.bcbsks.com/medical-policies/genetic-testing-lynch-syndrome-and-other-inherited-colon-cancer-syndromes\n",
      "https://www.bcbsks.com/medical-policies/somatic-biomarker-testing-including-liquid-biopsy-targeted-treatment-and-2022-10-28\n",
      "https://www.bcbsks.com/medical-policies/germline-genetic-testing-hereditary-breastovarian-cancer-syndrome-and-other-high-2023-01-24\n",
      "https://www.bcbsks.com/medical-policies/general-approach-evaluating-utility-genetic-panels-2023-01-03\n",
      "https://www.bcbsks.com/medical-policies/general-approach-evaluating-utility-genetic-panels-2023-01-03\n",
      "https://www.bcbsks.com/medical-policies/somatic-biomarker-testing-including-liquid-biopsy-targeted-treatment-and-2023-04-03\n",
      "https://www.bcbsks.com/medical-policies/miscellaneous-genetic-and-molecular-diagnostic-tests-2022-08-23\n",
      "1\n",
      "https://www.bcbsks.com/medical-policies/noninvasive-prenatal-screening-fetal-aneuploidies-microdeletions-single-gene\n",
      "https://www.bcbsks.com/medical-policies/cytochrome-p450-genotype-guided-treatment-strategy-2023-01-03\n",
      "https://www.bcbsks.com/medical-policies/cytochrome-p450-genotype-guided-treatment-strategy-2023-01-03\n",
      "https://www.bcbsks.com/medical-policies/general-approach-evaluating-utility-genetic-panels-2023-01-03\n",
      "https://www.bcbsks.com/medical-policies/noninvasive-prenatal-screening-fetal-aneuploidies-microdeletions-single-gene\n",
      "https://www.bcbsks.com/medical-policies/general-approach-genetic-testing\n",
      "https://www.bcbsks.com/medical-policies/germline-genetic-testing-hereditary-breastovarian-cancer-syndrome-and-other-high-2023-01-24\n",
      "https://www.bcbsks.com/medical-policies/genetic-testing-lynch-syndrome-and-other-inherited-colon-cancer-syndromes\n",
      "https://www.bcbsks.com/medical-policies/general-approach-evaluating-utility-genetic-panels-2023-01-03\n",
      "1\n",
      "https://www.bcbsks.com/medical-policies/general-approach-genetic-testing\n",
      "1\n",
      "https://www.bcbsks.com/medical-policies/germline-genetic-testing-hereditary-breastovarian-cancer-syndrome-and-other-high-2023-01-24\n",
      "https://www.bcbsks.com/medical-policies/general-approach-evaluating-utility-genetic-panels-2023-01-03\n",
      "https://www.bcbsks.com/medical-policies/germline-genetic-testing-hereditary-breastovarian-cancer-syndrome-and-other-high-2023-01-24\n",
      "https://www.bcbsks.com/medical-policies/genetic-testing-diagnosis-inherited-peripheral-neuropathies-2023-02-28\n",
      "https://www.bcbsks.com/medical-policies/general-approach-evaluating-utility-genetic-panels-2023-01-03\n",
      "https://www.bcbsks.com/medical-policies/genetic-cancer-susceptibility-panels-using-next-generation-sequencing-2022-11-22\n",
      "https://www.bcbsks.com/medical-policies/genetic-cancer-susceptibility-panels-using-next-generation-sequencing-2022-11-22\n",
      "https://www.bcbsks.com/medical-policies/general-approach-evaluating-utility-genetic-panels-2023-01-03\n",
      "https://www.bcbsks.com/medical-policies/general-approach-evaluating-utility-genetic-panels-2023-01-03\n",
      "https://www.bcbsks.com/medical-policies/noninvasive-prenatal-screening-fetal-aneuploidies-microdeletions-single-gene\n",
      "https://www.bcbsks.com/medical-policies/cytochrome-p450-genotype-guided-treatment-strategy-2023-01-03\n",
      "https://www.bcbsks.com/medical-policies/germline-genetic-testing-hereditary-breastovarian-cancer-syndrome-and-other-high-2023-01-24\n",
      "https://www.bcbsks.com/medical-policies/genetic-cancer-susceptibility-panels-using-next-generation-sequencing-2022-11-22\n",
      "https://www.bcbsks.com/medical-policies/general-approach-evaluating-utility-genetic-panels-2023-01-03\n",
      "https://www.bcbsks.com/medical-policies/miscellaneous-genetic-and-molecular-diagnostic-tests-2022-08-23\n",
      "1\n",
      "https://www.bcbsks.com/medical-policies/somatic-biomarker-testing-including-liquid-biopsy-targeted-treatment-and-2023-04-03\n",
      "https://www.bcbsks.com/medical-policies/noninvasive-prenatal-screening-fetal-aneuploidies-microdeletions-single-gene\n",
      "https://www.bcbsks.com/medical-policies/somatic-biomarker-testing-including-liquid-biopsy-targeted-treatment-and-2023-04-03\n",
      "https://www.bcbsks.com/medical-policies/miscellaneous-genetic-and-molecular-diagnostic-tests-2022-08-23\n",
      "https://www.bcbsks.com/medical-policies/comprehensive-genomic-profiling-selecting-targeted-cancer-therapies\n",
      "https://www.bcbsks.com/medical-policies/genetic-and-protein-biomarkers-diagnosis-and-cancer-risk-assessment-prostate\n",
      "https://www.bcbsks.com/medical-policies/identification-microorganisms-using-nucleic-acid-testing-2023-04-03\n",
      "https://www.bcbsks.com/medical-policies/general-approach-evaluating-utility-genetic-panels-2023-01-03\n",
      "https://www.bcbsks.com/medical-policies/identification-microorganisms-using-nucleic-acid-testing-2023-04-03\n",
      "https://www.bcbsks.com/medical-policies/identification-microorganisms-using-nucleic-acid-testing-2023-04-03\n",
      "1\n",
      "https://www.bcbsks.com/medical-policies/identification-microorganisms-using-nucleic-acid-testing-2023-04-03\n",
      "https://www.bcbsks.com/medical-policies/germline-and-somatic-biomarker-testing-including-liquid-biopsy-targeted-treatment-2022-10-28\n",
      "https://www.bcbsks.com/medical-policies/gene-expression-profiling-cutaneous-melanoma-2022-07-12\n",
      "https://www.bcbsks.com/medical-policies/somatic-biomarker-testing-including-liquid-biopsy-targeted-treatment-and-2023-04-03\n",
      "https://www.bcbsks.com/medical-policies/noninvasive-prenatal-screening-fetal-aneuploidies-microdeletions-single-gene\n",
      "https://www.bcbsks.com/medical-policies/identification-microorganisms-using-nucleic-acid-testing-2023-04-03\n",
      "https://www.bcbsks.com/medical-policies/laboratory-tests-post-transplant-and-heart-failure-2022-12-13\n",
      "https://www.bcbsks.com/medical-policies/general-approach-genetic-testing\n",
      "https://www.bcbsks.com/medical-policies/somatic-biomarker-testing-including-liquid-biopsy-targeted-treatment-and-2022-10-28\n",
      "https://www.bcbsks.com/medical-policies/miscellaneous-genetic-and-molecular-diagnostic-tests-2022-08-23\n",
      "https://www.bcbsks.com/medical-policies/general-approach-evaluating-utility-genetic-panels-2023-01-03\n",
      "https://www.bcbsks.com/medical-policies/identification-microorganisms-using-nucleic-acid-testing-2023-04-03\n",
      "https://www.bcbsks.com/medical-policies/noninvasive-prenatal-screening-fetal-aneuploidies-microdeletions-single-gene\n",
      "https://www.bcbsks.com/medical-policies/genetic-testing-lynch-syndrome-and-other-inherited-colon-cancer-syndromes\n",
      "https://www.bcbsks.com/medical-policies/general-approach-evaluating-utility-genetic-panels-2023-01-03\n",
      "1\n",
      "https://www.bcbsks.com/medical-policies/genetic-and-protein-biomarkers-diagnosis-and-cancer-risk-assessment-prostate\n",
      "https://www.bcbsks.com/medical-policies/general-approach-evaluating-utility-genetic-panels-2023-01-03\n",
      "https://www.bcbsks.com/medical-policies/gene-expression-profile-testing-and-circulating-tumor-dna-testing-predicting-2022-09-27\n",
      "https://www.bcbsks.com/medical-policies/general-approach-genetic-testing\n",
      "https://www.bcbsks.com/medical-policies/general-approach-evaluating-utility-genetic-panels-2023-01-03\n",
      "https://www.bcbsks.com/medical-policies/genetic-and-protein-biomarkers-diagnosis-and-cancer-risk-assessment-prostate\n",
      "https://www.bcbsks.com/medical-policies/general-approach-evaluating-utility-genetic-panels-2023-01-03\n",
      "https://www.bcbsks.com/medical-policies/general-approach-genetic-testing\n",
      "1\n",
      "https://www.bcbsks.com/medical-policies/genetic-testing-lynch-syndrome-and-other-inherited-colon-cancer-syndromes\n",
      "https://www.bcbsks.com/medical-policies/genetic-testing-lynch-syndrome-and-other-inherited-colon-cancer-syndromes\n",
      "https://www.bcbsks.com/medical-policies/general-approach-genetic-testing\n",
      "https://www.bcbsks.com/medical-policies/genetic-cancer-susceptibility-panels-using-next-generation-sequencing-2022-11-22\n",
      "https://www.bcbsks.com/medical-policies/laboratory-tests-post-transplant-and-heart-failure-2022-12-13\n",
      "https://www.bcbsks.com/medical-policies/genetic-testing-lynch-syndrome-and-other-inherited-colon-cancer-syndromes\n",
      "https://www.bcbsks.com/medical-policies/circulating-tumor-dna-management-non-small-cell-lung-cancer-liquid-biopsy-2023-04-11u\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/36/WholeExomeandWholeGenomeSequencingforDiagnosisofGeneticDisorders.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/GeneExpressionBasedAssaysforCancersofUnknownPrimary.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2022/12/30/17/38/Germline%20and%20Somatic%20Biomarker%20Testing%20Including%20Liquid%20Biopsy%20for%20Targeted00810%2020230101accessible.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/33/AssaysofGeneticExpressioninTumorTissueasaTechniquetoDeterminePrognosisinPatientswithBreastCancer.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/33/AssaysofGeneticExpressioninTumorTissueasaTechniquetoDeterminePrognosisinPatientswithBreastCancer.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/35/Miscellaneous%20Genetic%20and%20Molecular%20Diagnostic%20Tests%20CC%2000577%2020181017%20pdf_accessible%20pdf.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/GeneExpressionProfilingandProteinBiomarkersforProstateCancerManagement.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/33/AssaysofGeneticExpressioninTumorTissueasaTechniquetoDeterminePrognosisinPatientswithBreastCancer.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/33/AssaysofGeneticExpressioninTumorTissueasaTechniquetoDeterminePrognosisinPatientswithBreastCancer.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/06/15/58/Noninvasive%20Prenatal%20Screening%20for%20Fetal%20Aneuploidies%20and%20Coding%2000345%2020200413_accessible%20pdf.pdf\n",
      "https://providers.bcbsla.com/-/media/Files/Providers/M2091%20Transplant%20Rejection%20Testing%20pdf.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/Gene%20Expression%20Profiling%20for%20Cutaneous%20Melanoma%20CC%2000622%2020180815_accessible%20pdf.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/35/Molecular%20Testing%20in%20the%20Management%20of%20Pulmonary%20CC%2000562%2020170823%20pdf.pdf\n",
      "https://www.bcbsla.com/medicalpolicies\n",
      "https://www.bcbsla.com/medicalpolicies\n",
      "https://www.bcbsla.com/medicalpolicies\n",
      "https://www.bcbsla.com/medicalpolicies\n",
      "1\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/06/15/56/Comprehensive%20Genomic%20Profiling%20for%20Selecting%20Targeted%20Cancer%20Therapies%2000423%2020200713accessible%20pdf.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/GeneExpressionProfilingandProteinBiomarkersforProstateCancerManagement.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/06/15/56/Comprehensive%20Genomic%20Profiling%20for%20Selecting%20Targeted%20Cancer%20Therapies%2000423%2020200713accessible%20pdf.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2021/05/11/17/48/00332%20Molecular%20Markers%20in%20Fine%20Needle%20Aspirates%20of%20the%20Thyroid.pdf\n",
      "1\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/35/Multigene%20Expression%20Assays%20for%20Predicting%20Recurrence%20in%20Colon%20CC%2000257%2020190320_accessible%20pdf.pdf\n",
      "1\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/06/15/56/Comprehensive%20Genomic%20Profiling%20for%20Selecting%20Targeted%20Cancer%20Therapies%2000423%2020200713accessible%20pdf.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/GeneExpressionProfilingandProteinBiomarkersforProstateCancerManagement.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/Genetic%20and%20Protein%20Biomarkers%20for%20the%20Diagnosis%20and%20Cancer%20CC%2000272%2020190401_accessible%20pdf.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/06/15/57/Genetic%20Testing%20for%20BRCA1%20or%20BRCA2%20for%20Hereditary%20Breast%20Ovarian%20Coding%2000047%2020200511accessible%20pdf.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/GeneticCancerSusceptibilityPanelsUsingNextGenerationSequencing.pdf\n",
      "1\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/Genetic%20Testing%20for%20Predisposition%20to%20Inherited%20Hypertrophic%20CC%2000270%2020190320_accessible%20pdf.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/Genetic%20and%20Protein%20Biomarkers%20for%20the%20Diagnosis%20and%20Cancer%20CC%2000272%2020190401_accessible%20pdf.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/06/15/56/Comprehensive%20Genomic%20Profiling%20for%20Selecting%20Targeted%20Cancer%20Therapies%2000423%2020200713accessible%20pdf.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/06/15/57/Genetic%20Testing%20for%20BRCA1%20or%20BRCA2%20for%20Hereditary%20Breast%20Ovarian%20Coding%2000047%2020200511accessible%20pdf.pdf\n",
      "https://www.bcbsla.com/medicalpolicies\n",
      "https://www.bcbsla.com/medicalpolicies\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/06/15/57/Genetic%20Testing%20for%20BRCA1%20or%20BRCA2%20for%20Hereditary%20Breast%20Ovarian%20Coding%2000047%2020200511accessible%20pdf.pdf\n",
      "1\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/33/AnalysisofHumanDNAinStoolSamplesasaTechniqueforColorectalCancerScreening.pdf\n",
      "1\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/GeneticTestingforPTENHamartomaTumorSyndrome.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/36/WholeExomeandWholeGenomeSequencingforDiagnosisofGeneticDisorders.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2021/05/11/17/48/00332%20Molecular%20Markers%20in%20Fine%20Needle%20Aspirates%20of%20the%20Thyroid.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/06/15/57/Genetic%20Testing%20for%20BRCA1%20or%20BRCA2%20for%20Hereditary%20Breast%20Ovarian%20Coding%2000047%2020200511accessible%20pdf.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/GeneticTestingforCardiacIonChannelopathies.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/GeneticTestingforMitochondrialDisorders.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/GeneticCancerSusceptibilityPanelsUsingNextGenerationSequencing.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/GeneticTestingforCardiacIonChannelopathies.pdf\n",
      "https://providers.bcbsla.com/-/media/Files/Providers/New%20Revised%20Lab%20Reimbursement%20Policies/G2036%20Hepatitis%20C%20pdf.pdf\n",
      "1\n",
      "https://www.bcbsla.com/medicalpolicies\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/35/Miscellaneous%20Genetic%20and%20Molecular%20Diagnostic%20Tests%20CC%2000577%2020181017%20pdf_accessible%20pdf.pdf\n",
      "1\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/GeneticTestingforEpilepsy.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/GeneticTestingforHeterozygousFamilialHypercholesterolemia.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/GeneticTestingforLynchSyndromeandOtherInheritedColonCancerSyndromes.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/GeneticTestingforLynchSyndromeandOtherInheritedColonCancerSyndromes.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/33/Chimeric%20Antigen%20Receptor%20T%20cell%20Therapy%20CART%20CC%2000605%2020180221_accessible%20pdf.pdf\n",
      "1\n",
      "https://www.bcbsla.com/medicalpolicies\n",
      "1\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/GeneticTestingforDuchenneandBeckerMuscularDystrophy.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/35/Neurodiagnostics.pdf\n",
      "https://providers.bcbsla.com/-/media/Files/Providers/2020-08%20RPP%20Covid%20Testing%20Letter%20pdf.pdf\n",
      "1\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/GeneticTestingforFLT3NPM1andCEBPAMutationsinAcuteMyeloidLeukemia.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/36/WholeExomeandWholeGenomeSequencingforDiagnosisofGeneticDisorders.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/33/Cytochromep450Genotyping.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/06/15/57/Genetic%20Testing%20for%20BRCA1%20or%20BRCA2%20for%20Hereditary%20Breast%20Ovarian%20Coding%2000047%2020200511accessible%20pdf.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/GeneticTestingforRettSyndrome.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/GeneticCancerSusceptibilityPanelsUsingNextGenerationSequencing.pdf\n",
      "https://www.bcbsla.com/medicalpolicies\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/06/15/57/Genetic%20Testing%20for%20BRCA1%20or%20BRCA2%20for%20Hereditary%20Breast%20Ovarian%20Coding%2000047%2020200511accessible%20pdf.pdf\n",
      "https://www.bcbsla.com/medicalpolicies\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/ivacaftorKalydeco.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/GeneticCancerSusceptibilityPanelsUsingNextGenerationSequencing.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/Hematopoietic%20Cell%20Transplantation%20for%20Plasma%20Cell%20CC%2000060%2020190320_accessible%20pdf.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/GeneticTestingforFLT3NPM1andCEBPAMutationsinAcuteMyeloidLeukemia.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/06/15/57/Genetic%20Testing%20for%20BRCA1%20or%20BRCA2%20for%20Hereditary%20Breast%20Ovarian%20Coding%2000047%2020200511accessible%20pdf.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/GeneticTestingforHereditaryPancreatitis%2000394%2020181121_accessible%20pdf.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/06/15/58/Noninvasive%20Prenatal%20Screening%20for%20Fetal%20Aneuploidies%20and%20Coding%2000345%2020200413_accessible%20pdf.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/GeneticTestingforMitochondrialDisorders.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/06/15/56/Comprehensive%20Genomic%20Profiling%20for%20Selecting%20Targeted%20Cancer%20Therapies%2000423%2020200713accessible%20pdf.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2021/03/09/19/37/Biomarker%20Testing%20Including%20Liquid%20Biopsy%20for%20Targeted%2000731%2020210308_accessible%20pdf.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/06/15/58/Immune%20Globulin%20Therapy%20Coding%2000170%2020200511_accessible%20pdf.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/GeneticTestingforMitochondrialDisorders.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/GeneticTestingforLynchSyndromeandOtherInheritedColonCancerSyndromes.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/GeneticTestingforMitochondrialDisorders.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/EpidermalGrowthFactorReceptorEGFRMutationAnalysisforPatientswithNonSmallCellLungCancerNSCLCReplacedPolicy.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2022/07/11/17/15/Tumor%20Informed%20Circulating%20Tumor%20DNA%20Testing%20for%20Cancer%20Management%2000792%2020220711_accessibile%20pdf.pdf\n",
      "https://provider.healthybluela.com/docs/gpp/LA_CAID_RP_STITesting.pdf?v=202110261731\n",
      "1\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/06/15/58/Noninvasive%20Prenatal%20Screening%20for%20Fetal%20Aneuploidies%20and%20Coding%2000345%2020200413_accessible%20pdf.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/Gene%20Expression%20Profiling%20for%20Cutaneous%20Melanoma%20CC%2000622%2020180815_accessible%20pdf.pdf\n",
      "1\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/GeneticTestingforLynchSyndromeandOtherInheritedColonCancerSyndromes.pdf\n",
      "https://www.bcbsla.com/-/media/Files/Find%20a%20Doctor/2023_Closed_3_Tier_HCR_508c%20pdf.pdf\n",
      "1\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/06/15/56/Comprehensive%20Genomic%20Profiling%20for%20Selecting%20Targeted%20Cancer%20Therapies%2000423%2020200713accessible%20pdf.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/06/15/57/Genetic%20Testing%20for%20Hereditary%20Pancreatic%20Cancer%2000706%2020200511_accessible%20pdf.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/35/KRASNRASandBRAFMutationAnalysisinMetastaticColorectalCancer.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/33/ChromosomalMicroarrayAnalysisfortheEvaluationofPregnancyLoss.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/06/15/56/Comprehensive%20Genomic%20Profiling%20for%20Selecting%20Targeted%20Cancer%20Therapies%2000423%2020200713accessible%20pdf.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/36/WholeExomeandWholeGenomeSequencingforDiagnosisofGeneticDisorders.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/GeneticTestingforFLT3NPM1andCEBPAMutationsinAcuteMyeloidLeukemia.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/Genetic%20and%20Protein%20Biomarkers%20for%20the%20Diagnosis%20and%20Cancer%20CC%2000272%2020190401_accessible%20pdf.pdf\n",
      "https://providers.bcbsla.com/-/media/Files/Providers/M2097%20Identification%20Of%20Microorganisms%20Using%20Nucleic%20Acid%20Probes%20pdf.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/33/Chimeric%20Antigen%20Receptor%20T%20cell%20Therapy%20CART%20CC%2000605%2020180221_accessible%20pdf.pdf\n",
      "https://providers.bcbsla.com/-/media/Files/Providers/2020-08%20RPP%20Covid%20Testing%20Letter%20pdf.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/GeneticTestingforFLT3NPM1andCEBPAMutationsinAcuteMyeloidLeukemia.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/GeneticTestingforFLT3NPM1andCEBPAMutationsinAcuteMyeloidLeukemia.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/35/KRASNRASandBRAFMutationAnalysisinMetastaticColorectalCancer.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/33/Cytochromep450Genotyping.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/GeneticTestingforFLT3NPM1andCEBPAMutationsinAcuteMyeloidLeukemia.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/JAK2andMPLMutationAnalysisinMyeloproliferativeNeoplasms.pdf\n",
      "https://www.bcbsla.com/medicalpolicies\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/GeneticTestingforFLT3NPM1andCEBPAMutationsinAcuteMyeloidLeukemia.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/35/PharmacogenomicandMetaboliteMarkersforPatientsTreatedwithThiopurines.pdf\n",
      "1\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/GeneticTestingforLynchSyndromeandOtherInheritedColonCancerSyndromes.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/06/15/58/Noninvasive%20Prenatal%20Screening%20for%20Fetal%20Aneuploidies%20and%20Coding%2000345%2020200413_accessible%20pdf.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/33/AssaysofGeneticExpressioninTumorTissueasaTechniquetoDeterminePrognosisinPatientswithBreastCancer.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/33/AllogeneicHematopoieticCellTransplantationforMyelodysplasticSyndromesandMyeloproliferativeNeoplasms.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/GeneticTestingforMarfanSyndromeThoracicAorticAneurysmsandDissectionsandRelatedDisorders.pdf\n",
      "https://providers.bcbsla.com/-/media/Files/Providers/2020-08%20RPP%20Covid%20Testing%20Letter%20pdf.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/Genetic%20Testing%20for%20Limb%20Girdle%20Muscular%20Dystrophies%2000489%2020170719%20pdf.pdf\n",
      "1\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/06/15/57/Genetic%20Testing%20for%20Diagnosis%20and%20Management%20of%20Mental%20Health%20Coding%2000402%2020190814_accessible%20pdf.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/35/LaboratoryTestsforHeartTransplantRejection.pdf\n",
      "1\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/35/Miscellaneous%20Genetic%20and%20Molecular%20Diagnostic%20Tests%20CC%2000577%2020181017%20pdf_accessible%20pdf.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/JAK2andMPLMutationAnalysisinMyeloproliferativeNeoplasms.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/06/15/57/Genetic%20Testing%20for%20BRCA1%20or%20BRCA2%20for%20Hereditary%20Breast%20Ovarian%20Coding%2000047%2020200511accessible%20pdf.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/06/15/58/Noninvasive%20Prenatal%20Screening%20for%20Fetal%20Aneuploidies%20and%20Coding%2000345%2020200413_accessible%20pdf.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/06/15/58/Noninvasive%20Prenatal%20Screening%20for%20Fetal%20Aneuploidies%20and%20Coding%2000345%2020200413_accessible%20pdf.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/JAK2andMPLMutationAnalysisinMyeloproliferativeNeoplasms.pdf\n",
      "https://www.bcbsla.com/medicalpolicies\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/GeneticTestingforMarfanSyndromeThoracicAorticAneurysmsandDissectionsandRelatedDisorders.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/35/LaboratoryTestsforHeartTransplantRejection.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/35/PharmacogeneticTestingforPainManagement.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/06/15/56/Comprehensive%20Genomic%20Profiling%20for%20Selecting%20Targeted%20Cancer%20Therapies%2000423%2020200713accessible%20pdf.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/GeneticTestingforLynchSyndromeandOtherInheritedColonCancerSyndromes.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/06/15/57/Genetic%20Testing%20for%20BRCA1%20or%20BRCA2%20for%20Hereditary%20Breast%20Ovarian%20Coding%2000047%2020200511accessible%20pdf.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/03/17/34/Gene%20Expression%20Profiling%20for%20Cutaneous%20Melanoma%20CC%2000622%2020180815_accessible%20pdf.pdf\n",
      "https://www.bcbsla.com/-/media/Medical%20Policies/2020/08/06/15/57/Genetic%20Testing%20for%20BRCA1%20or%20BRCA2%20for%20Hereditary%20Breast%20Ovarian%20Coding%2000047%2020200511accessible%20pdf.pdf\n",
      "1\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2022/12/2022%20DRAFT%20Assays%20of%20Genetic%20Expression%20in%20Tumor%20Tissue%20as%20a%20Technique%20to%20Determine%20Prognosis%20in%20Patients%20with%20Breast%20Cancer_0.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2022/12/2022%20DRAFT%20Assays%20of%20Genetic%20Expression%20in%20Tumor%20Tissue%20as%20a%20Technique%20to%20Determine%20Prognosis%20in%20Patients%20with%20Breast%20Cancer_0.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2022/12/2022%20DRAFT%20Assays%20of%20Genetic%20Expression%20in%20Tumor%20Tissue%20as%20a%20Technique%20to%20Determine%20Prognosis%20in%20Patients%20with%20Breast%20Cancer_0.pdf\n",
      "https://www.bcbsri.com/sites/default/files/polices/Genetic%20Testing%20Services.pdf\n",
      "https://www.bcbsri.com/sites/default/files/polices/Genetic%20Testing%20Services.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/support/2023/08/2023%20Genetic%20Testing%20Services%20Codes%20and%20Coverage%20-%20Effective%2009.01.23.xlsx\n",
      "https://www.bcbsri.com/providers/sites/providers/files/support/2023/08/2023%20Genetic%20Testing%20Services%20Codes%20and%20Coverage%20-%20Effective%2009.01.23.xlsx\n",
      "https://www.bcbsri.com/providers/sites/providers/files/support/2023/08/2023%20Genetic%20Testing%20Services%20Codes%20and%20Coverage%20-%20Effective%2009.01.23.xlsx\n",
      "https://www.bcbsri.com/sites/default/files/polices/PLA-Codes-and-Coverage-July-2018.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2022/01/2021%20Serologic%20Genetic%20and%20Molecular%20Screening%20for%20Colorectal%20Cancer.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2022/12/2022%20DRAFT%20Comprehensive%20Genomic%20Profiling%20for%20Selecting%20Targeted%20Cancer%20Therapies%20Eff%204.1.23.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/support/2023/08/2023%20Genetic%20Testing%20Services%20Codes%20and%20Coverage%20-%20Effective%2009.01.23.xlsx\n",
      "https://www.bcbsri.com/providers/sites/providers/files/support/2020/09/2020%20Genetic%20Testing%20Services%20Codes%20and%20Coverage%209.20.xlsx\n",
      "https://www.bcbsri.com/sites/default/files/polices/Molecular%20Testing%20in%20the%20Management%20of%20Pulmonary%20Nodules_0.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/support/2023/08/2023%20Genetic%20Testing%20Services%20Codes%20and%20Coverage%20-%20Effective%2009.01.23.xlsx\n",
      "https://www.bcbsri.com/sites/default/files/polices/GeneticTestingLynchSyndrome.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2022/10/2022%20DRAFT%20Molecular%20Markers%20in%20Fine%20Needle%20Aspiration%20of%20the%20Thyroid.pdf\n",
      "https://www.bcbsri.com/sites/default/files/polices/GeneticTestingLynchSyndrome.pdf\n",
      "https://www.bcbsri.com/sites/default/files/polices/GeneticTestingLynchSyndrome.pdf\n",
      "https://www.bcbsri.com/sites/default/files/polices/Genetic_Testing_Hereditary_Breast_and_Ovarian_Cancer_0.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2020/10/2020%20Exondys%20%28eteplersin%29%20Duchenne%20Muscular%20Dystrophy.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2023/03/2022%20Gene%20Expression%20Profiling%20for%20Cutaneous%20Melanoma.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2022/07/2022%20Gene%20Expression%20Profiling%20and%20Protein%20Biomarkers%20for%20Prostate%20Cancer%20Management.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2021/01/2020%20TEMPORARY%20COVID-19%20Diagnostic%20Testing_1.12.21.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2022/10/2022%20DRAFT%20Molecular%20Markers%20in%20Fine%20Needle%20Aspiration%20of%20the%20Thyroid.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2019/06/UPDATE%202019%20DRAFT%20Laboratory%20Tests%20for%20Heart%20and%20Kidney%20Transplant%20Rejection.pdf\n",
      "https://www.bcbsri.com/sites/default/files/polices/Genetic_Testing_for_PTEN_PREAUTH_DRAFT.pdf\n",
      "https://www.bcbsri.com/sites/default/files/polices/Genetic%20Testing%20Services.pdf\n",
      "https://www.bcbsri.com/providers/medicalpolicies\n",
      "https://www.bcbsri.com/sites/default/files/polices/bluechip_for_medicare_laboratory_network_exemption_list_2.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2023/02/2023%20DRAFT%20Genetic%20Testing%20Services%20-%20Effective%2006.1.2023.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2022/12/2022%20DRAFT%20Assays%20of%20Genetic%20Expression%20in%20Tumor%20Tissue%20as%20a%20Technique%20to%20Determine%20Prognosis%20in%20Patients%20with%20Breast%20Cancer_0.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/support/2022/07/2022%20PLA%20Codes%20and%20Coverage%20July%202022%20New%20Codes.xlsx\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2021/10/2021%20PLA%20Codes%20and%20Coverage.xlsx\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2022/12/2022%20DRAFT%20Comprehensive%20Genomic%20Profiling%20for%20Selecting%20Targeted%20Cancer%20Therapies%20Eff%204.1.23.pdf\n",
      "https://www.bcbsri.com/providers/medicalpolicies\n",
      "https://www.bcbsri.com/sites/default/files/polices/GeneticTestingLynchSyndrome.pdf\n",
      "https://www.bcbsri.com/providers/medicalpolicies/revision/1081\n",
      "https://www.bcbsri.com/sites/default/files/polices/Vectra-DA-Blood-Test-for-Rheumatoid-Arthritis.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2022/12/2022%20DRAFT%20Comprehensive%20Genomic%20Profiling%20for%20Selecting%20Targeted%20Cancer%20Therapies%20Eff%204.1.23.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/support/2023/08/2023%20Genetic%20Testing%20Services%20Codes%20and%20Coverage%20-%20Effective%2009.01.23.xlsx\n",
      "https://www.bcbsri.com/providers/sites/providers/files/support/2020/09/2020%20PLA%20Codes%20and%20Coverage%209.9.20.xlsx\n",
      "https://www.bcbsri.com/sites/default/files/polices/GeneticTestingLynchSyndrome.pdf\n",
      "https://www.bcbsri.com/sitefiles/system/files/2022-02/2020-Quality-Management-Program-Evaluation.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/support/2023/08/2023%20Genetic%20Testing%20Services%20Codes%20and%20Coverage%20-%20Effective%2009.01.23.xlsx\n",
      "https://www.bcbsri.com/providers/sites/providers/files/support/2023/08/2023%20Genetic%20Testing%20Services%20Codes%20and%20Coverage%20-%20Effective%2009.01.23.xlsx\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2020/09/2020%20Genetic%20Testing%20Services%20.pdf\n",
      "https://www.bcbsri.com/providers/medicalpolicies/files/urinary-biomarkers-markers-cancer-screening-diagnosis-and-surveillance\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2022/07/2022%20Gene%20Expression%20Profiling%20and%20Protein%20Biomarkers%20for%20Prostate%20Cancer%20Management.pdf\n",
      "https://www.bcbsri.com/sites/default/files/polices/Genetic_Testing_for_PTEN_PREAUTH_DRAFT.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/support/2023/08/2023%20Genetic%20Testing%20Services%20Codes%20and%20Coverage%20-%20Effective%2009.01.23.xlsx\n",
      "https://www.bcbsri.com/sites/default/files/polices/PLA-Codes-and-Coverage-July-2018.pdf\n",
      "https://www.bcbsri.com/sites/default/files/polices/bluechip_for_medicare_laboratory_network_exemption_list_2.pdf\n",
      "https://www.bcbsri.com/sites/default/files/polices/GeneticTestingLynchSyndrome.pdf\n",
      "https://www.bcbsri.com/sites/default/files/polices/bluechip_for_medicare_laboratory_network_exemption_list_2.pdf\n",
      "https://www.bcbsri.com/providers/medicalpolicies/files/urinary-biomarkers-markers-cancer-screening-diagnosis-and-surveillance\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2022/12/2022%20DRAFT%20Genetic%20and%20Protein%20Biomarkers%20for%20the%20Diagnosis%20and%20Cancer%20Risk%20Assessment%20of%20Prostate%20Cancer%20Eff%204.1.23.pdf\n",
      "https://www.bcbsri.com/providers/medicalpolicies\n",
      "https://www.bcbsri.com/providers/sites/providers/files/support/2023/08/2023%20Genetic%20Testing%20Services%20Codes%20and%20Coverage%20-%20Effective%2009.01.23.xlsx\n",
      "https://www.bcbsri.com/providers/sites/providers/files/support/2023/08/2023%20Genetic%20Testing%20Services%20Codes%20and%20Coverage%20-%20Effective%2009.01.23.xlsx\n",
      "https://www.bcbsri.com/sites/default/files/polices/GeneticTestingLynchSyndrome.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/support/2023/08/2023%20Genetic%20Testing%20Services%20Codes%20and%20Coverage%20-%20Effective%2009.01.23.xlsx\n",
      "https://www.bcbsri.com/providers/sites/providers/files/support/2023/08/2023%20Genetic%20Testing%20Services%20Codes%20and%20Coverage%20-%20Effective%2009.01.23.xlsx\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2018/12/2018%20Expanded%20Molecular%20Panel%20Testing%20of%20Cancers_1.pdf\n",
      "https://www.bcbsri.com/sites/default/files/polices/Blue%20CHiP%20for%20Medicare%20Laboratory%20Network%20Allowable%20Test%20Listing.pdf\n",
      "https://www.bcbsri.com/providers/medicalpolicies/revision/4461\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2021/01/2021%20Multimarker%20Serum%20Testing%20Related%20to%20Ovarian%20Cancer.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2023/07/2023%20Comprehensive%20Genomic%20Profiling%20for%20Selecting%20Targeted%20Cancer%20Therapies%20-%20Effective%208.1.2023.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2022/10/2021%20Genetic%20and%20Protein%20Biomarkers%20for%20the%20Diagnosis%20and%20Cancer%20Risk%20Assessment%20of%20Prostate%20Cancer.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2022/07/2022%20Gene%20Expression%20Profiling%20and%20Protein%20Biomarkers%20for%20Prostate%20Cancer%20Management.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2022/11/2021%20Gene%20Expression%20Profiling%20for%20Cutaneous%20Melanoma.pdf\n",
      "https://www.bcbsri.com/providers/medicalpolicies/files/novel-biomarkers-risk-assessment-and-management-cardiovascular-disease\n",
      "https://www.bcbsri.com/sites/default/files/polices/Genetic-Test-H-Pyloril-Coverage-Policy.pdf\n",
      "https://www.bcbsri.com/sites/default/files/polices/GeneticTestingInheritedBRCA1orBRCA2Mutations_0.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2022/12/2022%20DRAFT%20Genetic%20and%20Protein%20Biomarkers%20for%20the%20Diagnosis%20and%20Cancer%20Risk%20Assessment%20of%20Prostate%20Cancer%20Eff%204.1.23.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/support/2023/08/2023%20Genetic%20Testing%20Services%20Codes%20and%20Coverage%20-%20Effective%2009.01.23.xlsx\n",
      "https://www.bcbsri.com/providers/sites/providers/files/support/2023/08/2023%20Genetic%20Testing%20Services%20Codes%20and%20Coverage%20-%20Effective%2009.01.23.xlsx\n",
      "https://www.bcbsri.com/providers/update/hints-hedisr-and-more-13\n",
      "https://www.bcbsri.com/providers/medicalpolicies\n",
      "https://www.bcbsri.com/sites/default/files/2013BMEDGroupPACriteria10-1.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2022/12/2022%20Noninvasive%20Techniques%20for%20the%20Evaluation%20and%20Monitoring%20of%20Patients%20With%20Chronic%20Liver%20Disease.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/support/2023/08/2023%20Genetic%20Testing%20Services%20Codes%20and%20Coverage%20-%20Effective%2009.01.23.xlsx\n",
      "https://www.bcbsri.com/sites/default/files/polices/GeneticTestingLynchSyndrome.pdf\n",
      "https://www.bcbsri.com/sites/default/files/polices/Blue%20CHiP%20for%20Medicare%20Laboratory%20Network%20Allowable%20Test%20Listing.pdf\n",
      "https://www.bcbsri.com/sites/default/files/polices/Blue%20CHiP%20for%20Medicare%20Laboratory%20Network%20Allowable%20Test%20Listing.pdf\n",
      "https://www.bcbsri.com/sites/default/files/polices/GeneticTestingLynchSyndrome.pdf\n",
      "https://www.bcbsri.com/sites/default/files/polices/Genetic_Testing_for_Cytochrome_p450_Genotyping.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/support/2023/08/2023%20Genetic%20Testing%20Services%20Codes%20and%20Coverage%20-%20Effective%2009.01.23.xlsx\n",
      "https://www.bcbsri.com/sites/default/files/polices/GeneticTestingA_0.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2020/09/2020%20Genetic%20Testing%20Services%20.pdf\n",
      "https://www.bcbsri.com/sites/default/files/polices/GeneticTestingA_0.pdf\n",
      "https://www.bcbsri.com/sites/default/files/polices/GeneticTestingLynchSyndrome.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/support/2023/08/2023%20Genetic%20Testing%20Services%20Codes%20and%20Coverage%20-%20Effective%2009.01.23.xlsx\n",
      "https://www.bcbsri.com/providers/sites/providers/files/support/2020/09/2020%20PLA%20Codes%20and%20Coverage%209.9.20.xlsx\n",
      "https://www.bcbsri.com/sites/default/files/polices/GeneticTestingInheritedBRCA1orBRCA2Mutations_0.pdf\n",
      "https://www.bcbsri.com/sites/default/files/polices/Genetic_Testing_Hereditary_Breast_and_Ovarian_Cancer_0.pdf\n",
      "https://www.bcbsri.com/sites/default/files/polices/GeneticTestingLynchSyndrome.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2020/09/2020%20Genetic%20Testing%20Services%20.pdf\n",
      "https://www.bcbsri.com/sites/default/files/polices/GeneticTestingInheritedBRCA1orBRCA2Mutations_0.pdf\n",
      "https://www.bcbsri.com/sites/default/files/polices/Grid-of-PLA-codes-Coverage.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2021/01/2020%20TEMPORARY%20COVID-19%20Diagnostic%20Testing_1.12.21.pdf\n",
      "https://www.bcbsri.com/sites/default/files/polices/GeneticTestingInheritedBRCA1orBRCA2Mutations_0.pdf\n",
      "https://www.bcbsri.com/sites/default/files/providers/resources/providers-update/2016/8/Provider-Update-2014-December.pdf\n",
      "https://www.bcbsri.com/sites/default/files/polices/Grid-of-GT-codes-Coverage-for-Policy.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2022/12/2022%20DRAFT%20Comprehensive%20Genomic%20Profiling%20for%20Selecting%20Targeted%20Cancer%20Therapies%20Eff%204.1.23.pdf\n",
      "https://www.bcbsri.com/providers/update/genetic-testing-services-1\n",
      "https://www.bcbsri.com/sites/default/files/polices/BlueCHiPforMedicareLaboratoryTestsAllowedOutsideof.pdf\n",
      "https://www.bcbsri.com/sites/default/files/polices/Genetic_Testing_to_Determine_Trisomy_21_from_Maternal_Plasma_DNA.pdf\n",
      "1\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2022/01/2021%20Medicare%20Advantage%20Plans%20Laboratory%20Network%20Hospital%20Outpatient%20Allowable%20List%20Effec%206.1.2021docx.pdf\n",
      "https://www.bcbsri.com/sites/default/files/polices/PLA_fianl_092017_0.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/support/2023/08/2023%20Genetic%20Testing%20Services%20Codes%20and%20Coverage%20-%20Effective%2009.01.23.xlsx\n",
      "https://www.bcbsri.com/providers/sites/providers/files/support/2020/09/2020%20PLA%20Codes%20and%20Coverage%209.9.20.xlsx\n",
      "https://www.bcbsri.com/sites/default/files/forms/BCRI%205-Tier%202016_v22_PACriteriaMemberFile_20160819%20Final.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2022/07/2022%20Laboratory%20Tests%20Post%20Transplant%20and%20for%20Heart%20Failure.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2021/01/2020%20TEMPORARY%20COVID-19%20Diagnostic%20Testing_1.12.21.pdf\n",
      "https://www.bcbsri.com/sites/default/files/polices/Genetic_Testing_to_Determine_Trisomy_21_from_Maternal_Plasma_DNA.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2018/12/2018%20Expanded%20Molecular%20Panel%20Testing%20of%20Cancers_1.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2022/12/2022%20DRAFT%20Comprehensive%20Genomic%20Profiling%20for%20Selecting%20Targeted%20Cancer%20Therapies%20Eff%204.1.23.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2022/12/2022%20DRAFT%20Comprehensive%20Genomic%20Profiling%20for%20Selecting%20Targeted%20Cancer%20Therapies%20Eff%204.1.23.pdf\n",
      "https://www.bcbsri.com/providers/medicalpolicies/revision/1081\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2023/03/2022%20Gene%20Expression%20Profiling%20for%20Cutaneous%20Melanoma.pdf\n",
      "https://www.bcbsri.com/sites/default/files/polices/Grid-of-GT-codes-Coverage-for-Policy.pdf\n",
      "https://www.bcbsri.com/sites/default/files/polices/Genetic_Testing_Hereditary_Breast_and_Ovarian_Cancer_0.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/support/2023/08/2023%20Genetic%20Testing%20Services%20Codes%20and%20Coverage%20-%20Effective%2009.01.23.xlsx\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2023/02/2023%20DRAFT%20Minimal%20Residual%20Disease%20Testing%20for%20Cancer.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/support/2023/08/2023%20Genetic%20Testing%20Services%20Codes%20and%20Coverage%20-%20Effective%2009.01.23.xlsx\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2021/01/2020%20TEMPORARY%20COVID-19%20Diagnostic%20Testing_1.12.21.pdf\n",
      "1\n",
      "https://www.bcbsri.com/sites/default/files/polices/Genetic_Testing_to_Determine_Trisomy_21_from_Maternal_Plasma_DNA.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2018/12/2018%20Expanded%20Molecular%20Panel%20Testing%20of%20Cancers_1.pdf\n",
      "https://www.bcbsri.com/sites/default/files/polices/Genetic_Testing_Multi_Gene_Expression_Assay_for_Predicting_Recurrence_in_Colon_Cancer_DRAFT.pdf\n",
      "https://www.bcbsri.com/sites/default/files/polices/GeneticTestingInheritedSusceptibilitytoColonCancerIncludingMicrosatelliteInstability.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2023/02/2023%20DRAFT%20Minimal%20Residual%20Disease%20Testing%20for%20Cancer.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2019/01/2019%20DRAFT%20Laboratory%20Tests%20for%20Heart%20and%20Kidney%20Transplant%20Rejection.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2022/12/2022%20DRAFT%20Assays%20of%20Genetic%20Expression%20in%20Tumor%20Tissue%20as%20a%20Technique%20to%20Determine%20Prognosis%20in%20Patients%20with%20Breast%20Cancer_0.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2022/12/2022%20DRAFT%20Comprehensive%20Genomic%20Profiling%20for%20Selecting%20Targeted%20Cancer%20Therapies%20Eff%204.1.23.pdf\n",
      "https://www.bcbsri.com/sites/default/files/polices/Genetic%20Testing%20Services.pdf\n",
      "https://www.bcbsri.com/sites/default/files/polices/GeneticTestingInheritedBRCA1orBRCA2Mutations_0.pdf\n",
      "https://www.bcbsri.com/sites/default/files/polices/bluechip_for_medicare_laboratory_network_exemption_list_2.pdf\n",
      "https://www.bcbsri.com/providers/medicalpolicies/files/molecular-testing-management-pulmonary-nodules\n",
      "https://www.bcbsri.com/providers/medicalpolicies/revision/4461\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2020/09/2020%20Genetic%20Testing%20Services%20.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2022/10/2021%20Genetic%20and%20Protein%20Biomarkers%20for%20the%20Diagnosis%20and%20Cancer%20Risk%20Assessment%20of%20Prostate%20Cancer.pdf\n",
      "https://www.bcbsri.com/sites/default/files/polices/GeneticTestingInheritedBRCA1orBRCA2Mutations_0.pdf\n",
      "https://www.bcbsri.com/sites/default/files/polices/bluechip_for_medicare_laboratory_network_exemption_list_2.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2021/01/2021%20Multimarker%20Serum%20Testing%20Related%20to%20Ovarian%20Cancer.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.078_08-25-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/7%20-%20Surg/07.01.121_07-28-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.129_02-23-23.pdf\n",
      "1\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.061_08-25-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.139_10-27-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.151_03-30-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.093_10-27-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/9%20-%20Other/09.01.502_05-15-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.119_06-30-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.091_05-25-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.033_11-17-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.115_10-27-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/9%20-%20Other/09.01.502_05-15-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/9%20-%20Other/09.01.502_03-31-23_Updates.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/9%20-%20Other/09.01.502_05-15-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/9%20-%20Other/09.01.502_03-31-23_Updates.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.008_09-29-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.120_02-23-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.093_10-27-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.060_08-25-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.536_03-30-23.pdf\n",
      "1\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.115_10-27-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.148_02-23-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.147_03-30-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.139_10-27-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/9%20-%20Other/09.01.502_05-15-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/9%20-%20Other/09.01.503_06-15-23_Updates.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.124_01-26-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.053_10-25-22.pdf\n",
      "1\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.124_01-26-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/9%20-%20Other/09.01.502_03-31-23_Updates.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.045_02-20-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/7%20-%20Surg/07.01.122_06-30-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.01.068_01-30-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.536_03-30-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/8%20-%20Therapy/08.01.024_01-27-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.107_09-29-22.pdf\n",
      "1\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.093_10-27-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.008_09-29-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.124_01-26-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/9%20-%20Other/09.01.502_03-31-23_Updates.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/8%20-%20Therapy/08.01.017_01-26-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.147_03-30-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.008_09-29-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.089_01-26-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/4%20-%20OBGYN/04.01.021_11-30-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.053_10-25-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/4%20-%20OBGYN/04.01.021_11-30-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.068_03-30-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/9%20-%20Other/09.01.502_05-15-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.124_01-26-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.008_09-29-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.085_10-27-22.pdf\n",
      "1\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.093_10-27-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.131_11-17-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.078_08-25-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.091_05-26-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.102_03-24-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.088_02-23-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.107_09-29-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.150_07-28-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.144_01-26-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.147_03-30-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.093_10-27-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.126_11-30-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.043_01-26-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.093_10-27-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/9%20-%20Other/09.01.502_05-15-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.01.068_01-30-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/7%20-%20Surg/07.01.044_05-26-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.093_10-27-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.147_03-30-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.019_11-17-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.573_01-15-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.141_08-25-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.088_02-23-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.081_05-26-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.095_11-17-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/9%20-%20Other/09.01.502_03-31-23_Updates.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.148_02-23-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.114_02-23-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/4%20-%20OBGYN/04.02.005_08-25-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.093_10-27-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.139_10-27-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.141_08-25-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.573_01-15-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.093_10-27-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.151_03-30-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.131_11-17-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.008_09-29-22.pdf\n",
      "1\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.093_10-27-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.147_03-30-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.110_07-28-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/4%20-%20OBGYN/04.02.005_08-25-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/4%20-%20OBGYN/04.02.510_12-22-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.129_02-24-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.137_01-26-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/7%20-%20Surg/07.03.014_07-28-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.155_01-30-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.045_02-20-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.122_09-29-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/7%20-%20Surg/07.03.006_08-25-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.150_07-28-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.148_02-23-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.093_10-27-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/4%20-%20OBGYN/04.01.021_11-30-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/4%20-%20OBGYN/04.01.021_11-30-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.127_12-22-22.pdf\n",
      "1\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/4%20-%20OBGYN/04.02.005_08-25-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.010_06-30-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.148_02-23-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.127_12-22-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/9%20-%20Other/09.01.502_03-31-23_Updates.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.045_02-20-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/9%20-%20Other/09.01.502_03-31-23_Updates.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/4%20-%20OBGYN/04.01.021_11-30-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.010_06-30-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.086_03-30-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/9%20-%20Other/09.01.502_05-15-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.102_03-24-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/9%20-%20Other/09.01.502_03-31-23_Updates.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.115_10-27-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.115_10-27-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.125_11-17-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.536_03-30-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.095_11-17-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.137_01-26-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.061_08-25-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.117_09-29-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/9%20-%20Other/09.01.502_06-15-23_Updates.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.054_03-30-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.156_01-30-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.573_01-15-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.008_09-29-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.536_03-30-23.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/9%20-%20Other/09.01.502_03-31-23_Updates.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.095_11-17-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/2%20-%20Med/02.04.093_10-27-22.pdf\n",
      "https://providers.bcidaho.com/resources/pdfs/medical-management/Medical%20Policy%20PDF/9%20-%20Other/09.01.502_06-15-23_Updates.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/029%20Molecular%20Testing%20in%20the%20Management%20of%20Pulmonary%20Nodules%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/709%20Proteomic%20Testing%20for%20Systemic%20Therapy%20in%20Non-Small-Cell%20Lung%20Cancer%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/212%20Hematopoietic%20Cell%20Transplantation%20for%20Chronic%20Myeloid%20Leukemia%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/555%20Identification%20of%20Microorganisms%20Using%20Nucleic%20Acid%20Probe%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/138%20Serological%20Diagnosis%20of%20Celiac%20Disease%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/075%20Hematopoietic%20Cell%20Transplantation%20for%20Plasma%20Cell%20Dyscracias%2C%20Including%20Multiple%20Myeloma%20and%20POEMS%20Syndrome%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/249%20Multimarker%20Serum%20Testing%20Related%20to%20Ovarian%20Cancer%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/249%20Multimarker%20Serum%20Testing%20Related%20to%20Ovarian%20Cancer%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/677%20Multibiomarker%20Disease%20Activity%20Blood%20Test%20for%20Rheumatoid%20Arthritis%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.com/common/en_US/medical_policies/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/336%20Biomarkers%20for%20Diagnosis%20%26%20Cancer%20Risk%20Assessment%20of%20Prostate%20Cancer%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20Carelon%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20Carelon%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/921%20Noninvasive%20Techniques%20for%20the%20Evaluation%20and%20Monitoring%20of%20Patients%20With%20Chronic%20Liver%20Disease%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/557%20Analysis%20of%20Human%20DNA%20in%20Stool%20Samples%20as%20a%20Technique%20for%20Colorectal%20Cancer%20Screening%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20Carelon%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/790Molecular_Panel_Testing_Cancers.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/555%20Identification%20of%20Microorganisms%20Using%20Nucleic%20Acid%20Probe%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/400%20Medical%20Technology%20Assessment%20NonCovered%20Services%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/555%20Identification%20of%20Microorganisms%20Using%20Nucleic%20Acid%20Probe%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/797Circulating_Tumor_DNA_For_Cancer.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/555%20Identification%20of%20Microorganisms%20Using%20Nucleic%20Acid%20Probe%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/555%20Identification%20of%20Microorganisms%20Using%20Nucleic%20Acid%20Probe%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/074%20Hematopoietic%20Cell%20Transplantation%20for%20Chronic%20Lymphocytic%20Leukemia%20and%20Small%20Lymphocytic%20Lymphoma%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/954%20Genetic%20Testing%20for%20Commercial%20and%20Medicare%20Advantage%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20Carelon%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/530%20Laboratory%20Tests%20for%20Heart%20and%20Kidney%20Transplant%20Rejection%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20Carelon%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/555%20Identification%20of%20Microorganisms%20Using%20Nucleic%20Acid%20Probe%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/555%20Identification%20of%20Microorganisms%20Using%20Nucleic%20Acid%20Probe%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/344%20Pharmacy%20Hepatitis%20C%20Medication%20Management%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/555%20Identification%20of%20Microorganisms%20Using%20Nucleic%20Acid%20Probe%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt42.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt42.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt42.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt76.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt29.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt17.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt42.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt42.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt17.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt15.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt17.pdf\n",
      "1\n",
      "1\n",
      "1\n",
      "https://www.policy.bridgespanhealth.com/lab/lab41.pdf\n",
      "1\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt83.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt22.pdf\n",
      "https://www.policy.bridgespanhealth.com/lab/lab73.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt77.pdf\n",
      "https://www.policy.bridgespanhealth.com/lab/lab60.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/index.html\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt44.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt83.pdf\n",
      "https://www.bridgespanhealth.com/provider/library/policies-guidelines/reimbursement-policy/covid19-testing\n",
      "https://www.policy.bridgespanhealth.com/transplant/tra45.23.pdf\n",
      "1\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt64.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt64.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt20.pdf\n",
      "https://www.policy.bridgespanhealth.com/lab/lab67.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt59.pdf\n",
      "https://www.policy.bridgespanhealth.com/lab/lab72.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt06.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt64.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt02.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt06.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt78.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/index.html\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt09.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt81.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt11.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt06.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt54.pdf\n",
      "https://www.policy.bridgespanhealth.com/lab/lab69.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt06.pdf\n",
      "https://www.policy.bridgespanhealth.com/lab/lab47.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt29.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt56.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt53.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt64.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt83.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt44.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt44.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt54.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt02.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt29.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt02.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt64.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt83.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt59.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt17.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt69.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt19.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt76.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt64.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt06.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt02.pdf\n",
      "https://www.policy.bridgespanhealth.com/lab/lab51.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt10.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt02.pdf\n",
      "https://www.policy.bridgespanhealth.com/lab/lab74.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt06.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt20.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/index.html\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt59.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt10.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt78.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt06.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt13.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt10.pdf\n",
      "https://www.policy.bridgespanhealth.com/lab/lab69.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt64.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt02.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt12.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt59.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt63.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt59.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt64.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt81.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt59.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt66.pdf\n",
      "https://www.policy.bridgespanhealth.com/lab/lab72.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt85.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt21.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt59.pdf\n",
      "1\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt06.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt81.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt77.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt63.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt76.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt83.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt20.pdf\n",
      "1\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt85.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt19.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt59.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt68.pdf\n",
      "https://www.policy.bridgespanhealth.com/lab/lab74.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt06.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt64.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt29.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt64.pdf\n",
      "https://www.policy.bridgespanhealth.com/lab/lab51.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt13.pdf\n",
      "https://www.policy.bridgespanhealth.com/transplant/tra45.16.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt20.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt76.pdf\n",
      "https://www.policy.bridgespanhealth.com/lab/lab46.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt64.pdf\n",
      "https://www.policy.bridgespanhealth.com/lab/lab60.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt64.pdf\n",
      "https://www.policy.bridgespanhealth.com/lab/lab46.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt02.pdf\n",
      "https://www.policy.bridgespanhealth.com/lab/lab74.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt85.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt64.pdf\n",
      "https://www.policy.bridgespanhealth.com/lab/lab51.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt06.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt11.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt17.pdf\n",
      "https://www.policy.bridgespanhealth.com/lab/lab47.pdf\n",
      "https://www.policy.bridgespanhealth.com/transplant/tra45.25.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt52.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt81.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt83.pdf\n",
      "https://www.bridgespanhealth.com/provider/library/policies-guidelines/reimbursement-policy/covid19-testing\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt02.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt06.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt86.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt64.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt81.pdf\n",
      "https://www.policy.bridgespanhealth.com/lab/lab46.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt44.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt85.pdf\n",
      "https://www.policy.bridgespanhealth.com/geneticTesting/gt13.pdf\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/b0aa3bd0-9221-492c-8121-17898c73ae4d/medical-policy-2-324.pdf?MOD=AJPERES\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/4-22-2021/MED%20Remove%20and%20Replace/20436%20Assays%20of%20Genetic%20Expression%20in%20Tumor%20Tissue%20as%20a%20Technique%20to.pdf\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/74e54657-525e-4c7a-9d3f-8c5db5a6dd5b/medical-policy-1-036.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/18786a15-58ae-4c7a-bd1b-7f726b0e7e8b/medical-policy-2-326.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/18786a15-58ae-4c7a-bd1b-7f726b0e7e8b/medical-policy-2-326.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/18786a15-58ae-4c7a-bd1b-7f726b0e7e8b/medical-policy-2-326.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/d67f98d9-4a4b-42af-b54a-8f7cefe54830/medical-policy-2-337.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/564112d2-feb2-44a7-afcd-a48676059b9f/luxturna.pdf?MOD=AJPERES&CVID=ovnwlWz\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/58cdd01f-01a5-430d-a676-5a61b2c5bf17/medical-policy-2-328.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/18786a15-58ae-4c7a-bd1b-7f726b0e7e8b/medical-policy-2-326.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/1e6bfff2-4d3c-4767-9335-34d84849db50/medical-policy-2-323.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/184b8f33-4219-4528-a2a6-2d72cc5a29a7/medical-policy-2-325.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/18786a15-58ae-4c7a-bd1b-7f726b0e7e8b/medical-policy-2-326.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/184b8f33-4219-4528-a2a6-2d72cc5a29a7/medical-policy-2-325.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/df5bc913-6ef7-4e52-9aa1-9ad00c0a5c81/medical-policy-2-253.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/4f6a5278-8506-4cea-adc4-a70dc8fcd84f/medical-policy-2-259.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/9bea6211-f951-4b8b-9a2f-d625fc299b22/medical-policy-2-241.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/64e48204-59b9-49df-99af-353ea59d2434/medical-policy-2-211.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/21f72fee-131c-4d5f-85bb-6f08588e7b74/medical-policy-2-234.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/64e48204-59b9-49df-99af-353ea59d2434/medical-policy-2-211.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/a32510ba-3f56-4e9d-9609-7f7545fc4470/medical-policy-2-345.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/e79b6984-a21c-449c-b99d-17a75ab4616d/medical-policy-2-256.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/184b8f33-4219-4528-a2a6-2d72cc5a29a7/medical-policy-2-325.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/18786a15-58ae-4c7a-bd1b-7f726b0e7e8b/medical-policy-2-326.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/18786a15-58ae-4c7a-bd1b-7f726b0e7e8b/medical-policy-2-326.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/64e48204-59b9-49df-99af-353ea59d2434/medical-policy-2-211.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/184b8f33-4219-4528-a2a6-2d72cc5a29a7/medical-policy-2-325.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/18786a15-58ae-4c7a-bd1b-7f726b0e7e8b/medical-policy-2-326.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/184b8f33-4219-4528-a2a6-2d72cc5a29a7/medical-policy-2-325.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/4f6a5278-8506-4cea-adc4-a70dc8fcd84f/medical-policy-2-259.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/735e66ad-16f0-4d49-9401-75e281d1f566/medical-policy-2-270.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/9ae0109b-7d8e-4df2-842f-b8a5599cfae4/medical-policy-2-233.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/4f6a5278-8506-4cea-adc4-a70dc8fcd84f/medical-policy-2-259.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/21f72fee-131c-4d5f-85bb-6f08588e7b74/medical-policy-2-234.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/54f71e2d-5876-435b-9b8d-2f2d84dd734b/medical-policy-2-331.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/18786a15-58ae-4c7a-bd1b-7f726b0e7e8b/medical-policy-2-326.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/7940937a-b1ed-4d6e-926f-0980148b0d33/laboratory-service-prostate-biopsies.pdf?MOD=AJPERES&CACHEID=ROOTWORKSPACE.Z18_4G00HA41L8PI50ALUD09N53000-7940937a-b1ed-4d6e-926f-0980148b0d33-nUgERwK\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/3f9ed8ea-b049-4255-8222-d23e2dc2eefe/actemra-sq.pdf?MOD=AJPERES&CVID=ovnwgdn\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/d67f98d9-4a4b-42af-b54a-8f7cefe54830/medical-policy-2-337.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/18786a15-58ae-4c7a-bd1b-7f726b0e7e8b/medical-policy-2-326.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/b0aa3bd0-9221-492c-8121-17898c73ae4d/medical-policy-2-324.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/1e6bfff2-4d3c-4767-9335-34d84849db50/medical-policy-2-323.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/54f71e2d-5876-435b-9b8d-2f2d84dd734b/medical-policy-2-331.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/9bea6211-f951-4b8b-9a2f-d625fc299b22/medical-policy-2-241.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/9da2a5c0-ab75-4540-8b9e-7884478a19f2/medical-policy-2-304.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/3e154640-41c5-414b-945a-017546fc6ec0/medical-policy-4-024.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/1cc8381e-1183-44e6-ae8e-c6a30fc64f03/reimbursement-policy-failed-colonoscopy.pdf?MOD=AJPERES&CVID=nAuRrtK\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/bd8e6228-93e7-4fe6-a2cb-a89aca0f17f5/medical-policy-2-252.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/184b8f33-4219-4528-a2a6-2d72cc5a29a7/medical-policy-2-325.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/2be5a6ec-f10b-4032-8750-e141fdf502b7/medical-policy-2-267.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/1cec044b-110d-4629-9e5a-c48dce09356c/medical-policy-2-278.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/4e987155-a64f-47c0-ba26-d78a43509b5a/medical-policy-9-001.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/7c4ecb90-de64-4ca4-a4a5-00c25d4e40ed/medical-policy-2-276.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/64e48204-59b9-49df-99af-353ea59d2434/medical-policy-2-211.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/9b8b9984-c9e9-49c1-b588-43a3650f23d8/medical-policy-2-360.pdf?MOD=AJPERES&CVID=mwlzy5a\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/a933b911-65af-414b-a603-2330125028bb/medical-policy-2-318.pdf?MOD=AJPERES&CACHEID=ROOTWORKSPACE.Z18_4G00HA41L8PI50ALUD09N53000-a933b911-65af-414b-a603-2330125028bb-nKxnXKZ\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/64e48204-59b9-49df-99af-353ea59d2434/medical-policy-2-211.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/9b8b9984-c9e9-49c1-b588-43a3650f23d8/medical-policy-2-360.pdf?MOD=AJPERES&CVID=mwlzy5a\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/18786a15-58ae-4c7a-bd1b-7f726b0e7e8b/medical-policy-2-326.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/8db40575-fc0c-4acd-8442-55deaac65cfb/laboratory-service-celiac-disease-testing.pdf?MOD=AJPERES&CVID=nqVWLzo\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/739a3c7d-fbd5-4efb-bc91-ac26afd47fe7/laboratory-service-identification-of-microorganisms-using-nucleic-acid-probes.pdf?MOD=AJPERES&CACHEID=ROOTWORKSPACE.Z18_4G00HA41L8PI50ALUD09N53000-739a3c7d-fbd5-4efb-bc91-ac26afd47fe7-ovnArYv\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/fe028994-2d11-4713-a43c-0b421c452e59/laboratory-service-pathogen-panel-testing.pdf?MOD=AJPERES&CVID=myDrJd-\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/18786a15-58ae-4c7a-bd1b-7f726b0e7e8b/medical-policy-2-326.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/18786a15-58ae-4c7a-bd1b-7f726b0e7e8b/medical-policy-2-326.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/289c722c-60da-42c1-9e7e-f6cee7868c15/medical-policy-2-273.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/033936f4-cf58-490d-a8ca-368e5142bc1c/medical-policy-9-007.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/bd8e6228-93e7-4fe6-a2cb-a89aca0f17f5/medical-policy-2-252.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/2be5a6ec-f10b-4032-8750-e141fdf502b7/medical-policy-2-267.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/64e48204-59b9-49df-99af-353ea59d2434/medical-policy-2-211.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/b0aa3bd0-9221-492c-8121-17898c73ae4d/medical-policy-2-324.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/9a824fb1-69fc-48ba-938b-02fc01c0a0c2/laboratory-service-transplant-rejection-testing.pdf?MOD=AJPERES&CVID=novws8b\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/18786a15-58ae-4c7a-bd1b-7f726b0e7e8b/medical-policy-2-326.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/4072c0d8-e53d-4e8f-8697-5a33a926b7e3/medical-policy-6-054.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/18786a15-58ae-4c7a-bd1b-7f726b0e7e8b/medical-policy-2-326.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/18786a15-58ae-4c7a-bd1b-7f726b0e7e8b/medical-policy-2-326.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/632568d0-a38e-4a60-b97d-447e7de57311/Molecular_Markers_in_Fine-Needle-Aspirates_of_the_Thyroid_6-1-17.pdf?MOD=AJPERES&CVID=lTxlEo5\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/18786a15-58ae-4c7a-bd1b-7f726b0e7e8b/medical-policy-2-326.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/64e48204-59b9-49df-99af-353ea59d2434/medical-policy-2-211.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/184b8f33-4219-4528-a2a6-2d72cc5a29a7/medical-policy-2-325.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/184b8f33-4219-4528-a2a6-2d72cc5a29a7/medical-policy-2-325.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/35d388b5-918f-4f3e-8b3e-74166eb925d5/medical-policy-2-355.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/18786a15-58ae-4c7a-bd1b-7f726b0e7e8b/medical-policy-2-326.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/289c722c-60da-42c1-9e7e-f6cee7868c15/medical-policy-2-273.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/18786a15-58ae-4c7a-bd1b-7f726b0e7e8b/medical-policy-2-326.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/e79b6984-a21c-449c-b99d-17a75ab4616d/medical-policy-2-256.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/18786a15-58ae-4c7a-bd1b-7f726b0e7e8b/medical-policy-2-326.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/184b8f33-4219-4528-a2a6-2d72cc5a29a7/medical-policy-2-325.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/289c722c-60da-42c1-9e7e-f6cee7868c15/medical-policy-2-273.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/c8b5e93d-0df0-4124-a067-982f35912095/service-policy-jemperli.pdf?MOD=AJPERES&CVID=ovYKXGM\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/0f6367cf-f1c2-4ca1-a32b-3b530e923d86/comprehensive-performance-measures-guide-5-0.pdf?MOD=AJPERES&CVID=o3O42vT\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/18786a15-58ae-4c7a-bd1b-7f726b0e7e8b/medical-policy-2-326.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/9b8b9984-c9e9-49c1-b588-43a3650f23d8/medical-policy-2-360.pdf?MOD=AJPERES&CVID=mwlzy5a\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/184b8f33-4219-4528-a2a6-2d72cc5a29a7/medical-policy-2-325.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/18786a15-58ae-4c7a-bd1b-7f726b0e7e8b/medical-policy-2-326.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/184b8f33-4219-4528-a2a6-2d72cc5a29a7/medical-policy-2-325.pdf?MOD=AJPERES\n",
      "1\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/18786a15-58ae-4c7a-bd1b-7f726b0e7e8b/medical-policy-2-326.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/029529d1-4fe4-448b-a32d-ed0cbdcc14be/medical-policy-2-218.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/1cec044b-110d-4629-9e5a-c48dce09356c/medical-policy-2-278.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/735e66ad-16f0-4d49-9401-75e281d1f566/medical-policy-2-270.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/18786a15-58ae-4c7a-bd1b-7f726b0e7e8b/medical-policy-2-326.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/e79b6984-a21c-449c-b99d-17a75ab4616d/medical-policy-2-256.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/e79b6984-a21c-449c-b99d-17a75ab4616d/medical-policy-2-256.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/21f72fee-131c-4d5f-85bb-6f08588e7b74/medical-policy-2-234.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/0b58974d-5db5-48e0-a8c4-a240ca2fa587/laboratory-service-urinary-tumor-markers.pdf?MOD=AJPERES&CVID=my-lvQL\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/18786a15-58ae-4c7a-bd1b-7f726b0e7e8b/medical-policy-2-326.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/1cec044b-110d-4629-9e5a-c48dce09356c/medical-policy-2-278.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/184b8f33-4219-4528-a2a6-2d72cc5a29a7/medical-policy-2-325.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/184b8f33-4219-4528-a2a6-2d72cc5a29a7/medical-policy-2-325.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/184b8f33-4219-4528-a2a6-2d72cc5a29a7/medical-policy-2-325.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/f19f8c27-9163-4b4c-9f41-32b87e3e7aaa/medical-policy-2-264.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/35d388b5-918f-4f3e-8b3e-74166eb925d5/medical-policy-2-355.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/9bea6211-f951-4b8b-9a2f-d625fc299b22/medical-policy-2-241.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/18786a15-58ae-4c7a-bd1b-7f726b0e7e8b/medical-policy-2-326.pdf?MOD=AJPERES\n",
      "1\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/4b2e5540-9491-4249-9f8e-e1baae342813/medical-policy-2-257.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/18786a15-58ae-4c7a-bd1b-7f726b0e7e8b/medical-policy-2-326.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/1e6bfff2-4d3c-4767-9335-34d84849db50/medical-policy-2-323.pdf?MOD=AJPERES\n",
      "https://www.capbluecross.com/wps/wcm/connect/prod_nws.capblue.com29556/2be5a6ec-f10b-4032-8750-e141fdf502b7/medical-policy-2-267.pdf?MOD=AJPERES\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000972.html\n",
      "https://www.empireblue.com/dam/medpolicies/ebcbs/active/policies/mp_pw_a049883.html\n",
      "https://www.empireblue.com/dam/medpolicies/ebcbs/active/guidelines/gl_pw_d056660.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000232.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000972.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/policies/pw_e000224.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/policies/mp_pw_c148391.html\n",
      "https://www.empireblue.com/dam/medpolicies/ebcbs/active/policies/mp_pw_d090898.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/policies/mp_pw_e000971.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/policies/mp_pw_a049885.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000972.html\n",
      "https://www.empireblue.com/dam/medpolicies/ebcbs/active/guidelines/gl_pw_d056660.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/policies/mp_pw_c148391.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/policies/mp_pw_a049877.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000232.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000235.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/policies/pw_e000224.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/policies/pw_e000224.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/policies/mp_pw_a049877.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/policies/mp_pw_a049877.html\n",
      "https://www.empireblue.com/dam/medpolicies/ebcbs/active/guidelines/gl_pw_e000235.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/policies/mp_pw_e000971.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/policies/mp_pw_c128862.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/policies/pw_e000224.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000235.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000234.html\n",
      "https://www.empireblue.com/dam/medpolicies/ebcbs/active/policies/mp_pw_e002062.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000232.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000232.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000234.html\n",
      "https://www.empireblue.com/dam/medpolicies/ebcbs/active/policies/mp_pw_c197570.html\n",
      "https://www.empireblue.com/dam/medpolicies/ebcbs/active/policies/mp_pw_c135466.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000972.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000232.html\n",
      "https://www.empireblue.com/dam/medpolicies/ebcbs/active/guidelines/gl_pw_e000235.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/policies/mp_pw_a049877.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/policies/pw_e000224.html\n",
      "https://www.empireblue.com/dam/medpolicies/ebcbs/active/policies/mp_pw_c193684.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000234.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000232.html\n",
      "https://www.empireblue.com/dam/medpolicies/ebcbs/active/guidelines/gl_pw_c119565.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000232.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/policies/pw_e000224.html\n",
      "https://www.empireblue.com/dam/medpolicies/ebcbs/active/guidelines/gl_pw_e000235.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000235.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000972.html\n",
      "https://www.empireblue.com/dam/medpolicies/ebcbs/active/guidelines/gl_pw_e000974.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000234.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000232.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000235.html\n",
      "https://www.empireblue.com/dam/medpolicies/ebcbs/active/guidelines/gl_pw_d094156.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000232.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000232.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000234.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/policies/mp_pw_c148391.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000234.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/policies/pw_e000224.html\n",
      "https://www.empireblue.com/dam/medpolicies/ebcbs/active/guidelines/gl_pw_d094156.html\n",
      "https://www.empireblue.com/ms/pharmacyinformation/drugsalerts.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000232.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000232.html\n",
      "https://www.empireblue.com/dam/medpolicies/ebcbs/active/policies/mp_pw_a045963.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "1\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/policies/pw_e000224.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/policies/pw_e000224.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000232.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "1\n",
      "https://www.empireblue.com/dam/medpolicies/ebcbs/active/guidelines/gl_pw_d094156.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/policies/pw_e000224.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000234.html\n",
      "https://www.empireblue.com/dam/medpolicies/ebcbs/active/guidelines/gl_pw_e000813.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.empireblue.com/dam/medpolicies/ebcbs/active/guidelines/gl_pw_d094176.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "1\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/policies/mp_pw_a049877.html\n",
      "https://www.empireblue.com/dam/medpolicies/ebcbs/active/policies/mp_pw_a045960.html\n",
      "https://www.empireblue.com/dam/medpolicies/ebcbs/active/guidelines/gl_pw_e000813.html\n",
      "https://www.empireblue.com/dam/medpolicies/ebcbs/active/guidelines/gl_pw_e000974.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.empireblue.com/dam/medpolicies/ebcbs/active/guidelines/gl_pw_e000974.html\n",
      "https://www.empireblue.com/dam/medpolicies/ebcbs/active/guidelines/gl_pw_d087066.html\n",
      "1\n",
      "https://www.empireblue.com/dam/medpolicies/ebcbs/active/guidelines/gl_pw_e000813.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/policies/pw_e000224.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/policies/mp_pw_d082650.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000234.html\n",
      "https://www.empireblue.com/dam/medpolicies/ebcbs/active/guidelines/gl_pw_d094176.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/policies/mp_pw_c148391.html\n",
      "https://www.empireblue.com/dam/medpolicies/ebcbs/active/guidelines/gl_pw_d087066.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/policies/mp_pw_d082650.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.empireblue.com/dam/medpolicies/ebcbs/active/guidelines/gl_pw_d094156.html\n",
      "https://www.empireblue.com/dam/medpolicies/ebcbs/active/guidelines/gl_pw_e000233.html\n",
      "1\n",
      "https://www.empireblue.com/dam/medpolicies/ebcbs/active/guidelines/gl_pw_e000974.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000232.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000232.html\n",
      "https://www.empireblue.com/dam/medpolicies/ebcbs/active/guidelines/gl_pw_e001870.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/policies/pw_e000224.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/policies/pw_e000224.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/policies/pw_e000224.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000234.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000232.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.empireblue.com/dam/medpolicies/ebcbs/active/guidelines/gl_pw_d094156.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/policies/pw_e000224.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000234.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000235.html\n",
      "https://www.empireblue.com/dam/medpolicies/ebcbs/active/guidelines/gl_pw_d094156.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000235.html\n",
      "https://www.empireblue.com/dam/medpolicies/ebcbs/active/guidelines/gl_pw_d087066.html\n",
      "https://www.empireblue.com/dam/medpolicies/ebcbs/active/guidelines/gl_pw_e000813.html\n",
      "https://www.empireblue.com/dam/medpolicies/ebcbs/active/policies/mp_pw_e000975.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000235.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000234.html\n",
      "1\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.empireblue.com/dam/medpolicies/ebcbs/active/policies/mp_pw_e000975.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000232.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000235.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000232.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/policies/mp_pw_d082650.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/policies/pw_e000224.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/policies/pw_e000224.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/policies/mp_pw_a049877.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000233.html\n",
      "https://www.empireblue.com/dam/medpolicies/abc/active/policies/pw_e000224.html\n",
      "https://www.keystonefirstpa.com/pdf/provider/resources/clinical/policies-20230217/ccp1121-genetic-testing-prostate-cancer-prognosis.pdf\n",
      "https://www.keystonefirstpa.com/pdf/provider/resources/clinical/policies-20200423/ccp1037-genetic-testing-long-qt-syndrome.pdf\n",
      "https://www.keystonefirstpa.com/pdf/provider/resources/clinical/policies-20230217/ccp_1456_Molecular-analysis-tx-primary-brain-tumors.pdf\n",
      "https://www.keystonefirstpa.com/pdf/provider/resources/clinical/policies-20230217/ccp_1235_Genetic-testing-hereditary-cancer-susceptibility.pdf\n",
      "https://www.keystonefirstpa.com/pdf/provider/resources/clinical/policies-20230217/ccp_1235_Genetic-testing-hereditary-cancer-susceptibility.pdf\n",
      "https://www.keystonefirstpa.com/pdf/provider/resources/clinical/policies-20230217/ccp1503-serum-proteomic-cancer-risk-classification-indeterminate-pulmonary-nodules.pdf\n",
      "https://www.keystonefirstpa.com/pdf/provider/resources/clinical/policies-20230217/ccp1277-fluorescence-spectroscopy-prostate-cancer-dx.pdf\n",
      "https://www.keystonefirstpa.com/pdf/provider/resources/clinical/policies-20230217/ccp1099-afirma-thyroid-fna-analysis-indeterminate-thyroid-nodules.pdf\n",
      "https://www.keystonefirstpa.com/pdf/provider/resources/clinical/policies-20230217/ccp1050-familial-polyposis-gene-testing.pdf\n",
      "https://www.keystonefirstpa.com/pdf/provider/resources/clinical/policies-20230217/ccp1045-Gene-expression-profile-testing-breast-cancer.pdf\n",
      "https://www.keystonefirstpa.com/pdf/provider/resources/clinical/policies-20201215/ccp1124-genetic-testing-autism-spectrum-disorders.pdf\n",
      "https://www.keystonefirstpa.com/pdf/provider/resources/clinical/policies-20201215/ccp1124-genetic-testing-autism-spectrum-disorders.pdf\n",
      "https://www.keystonefirstpa.com/pdf/provider/resources/clinical/policies-20230217/ccp_1235_Genetic-testing-hereditary-cancer-susceptibility.pdf\n",
      "https://www.keystonefirstpa.com/pdf/provider/resources/clinical/policies-20201215/ccp1218-molecular-analysis-targeted-therapy-lung-cancer.pdf\n",
      "https://www.keystonefirstpa.com/pdf/provider/resources/clinical/policies-20230217/ccp1121-genetic-testing-prostate-cancer-prognosis.pdf\n",
      "https://www.keystonefirstpa.com/pdf/provider/resources/clinical/policies-20201215/ccp1002-maternal-genetic-testing.pdf\n",
      "https://www.keystonefirstpa.com/pdf/provider/resources/clinical/policies-20230217/ccp1277-fluorescence-spectroscopy-prostate-cancer-dx.pdf\n",
      "https://www.keystonefirstpa.com/pdf/provider/resources/clinical/policies-20200423/ccp1384-pharmacogenomic-testing-psychoses.pdf\n",
      "https://www.keystonefirstpa.com/pdf/provider/resources/clinical/policies-20200423/ccp1384-pharmacogenomic-testing-psychoses.pdf\n",
      "https://www.keystonefirstpa.com/pdf/provider/communications/fastfacts/2021/reimbursement-guidelines-sti-tests.pdf\n",
      "https://www.keystonefirstpa.com/pdf/provider/resources/clinical/policies-20230217/ccp1181-pharmacogenetic-testing-cardiac-meds.pdf\n",
      "https://www.keystonefirstpa.com/pdf/provider/resources/clinical/policies-20201215/ccp1124-genetic-testing-autism-spectrum-disorders.pdf\n",
      "https://www.keystonefirstpa.com/pdf/provider/resources/clinical/policies-20230217/ccp_1235_Genetic-testing-hereditary-cancer-susceptibility.pdf\n",
      "https://www.keystonefirstpa.com/pdf/provider/resources/clinical/policies-20230217/ccp1516-circulating_-tumor-dna-circulating-tumor-cells-cancer-management-liquid-biopsy.pdf\n",
      "https://www.keystonefirstpa.com/pdf/provider/resources/clinical/policies-20230217/ccp1516-circulating_-tumor-dna-circulating-tumor-cells-cancer-management-liquid-biopsy.pdf\n",
      "https://www.keystonefirstvipchoice.com/assets/pdf/provider/resources/clinical/policies-20230217/ccp1153-genetic-testing-cystic-fibrosis.pdf\n",
      "https://www.keystonefirstpa.com/pdf/provider/resources/clinical/policies-20201215/ccp1235-genetic-testing-hereditary-cancer-susceptibility.pdf\n",
      "https://www.keystonefirstpa.com/pdf/provider/resources/clinical/policies-20201215/ccp1002-maternal-genetic-testing.pdf\n",
      "https://www.keystonefirstpa.com/pdf/provider/resources/clinical/policies-20201215/ccp1235-genetic-testing-hereditary-cancer-susceptibility.pdf\n",
      "https://www.keystonefirstpa.com/pdf/provider/resources/clinical/policies-20201215/ccp1235-genetic-testing-hereditary-cancer-susceptibility.pdf\n",
      "https://www.keystonefirstpa.com/pdf/provider/resources/clinical/policies-20201215/ccp1235-genetic-testing-hereditary-cancer-susceptibility.pdf\n",
      "https://www.keystonefirstpa.com/pdf/provider/resources/clinical/policies-20200423/ccp1037-genetic-testing-long-qt-syndrome.pdf\n",
      "https://www.keystonefirstpa.com/pdf/provider/resources/clinical/policies-20230217/ccp_1235_Genetic-testing-hereditary-cancer-susceptibility.pdf\n",
      "https://www.keystonefirstpa.com/pdf/provider/resources/clinical/policies-20230217/ccp_1235_Genetic-testing-hereditary-cancer-susceptibility.pdf\n",
      "https://www.keystonefirstpa.com/pdf/provider/resources/clinical/policies-20230217/ccp_1235_Genetic-testing-hereditary-cancer-susceptibility.pdf\n",
      "https://www.keystonefirstpa.com/pdf/provider/resources/clinical/policies-20201215/ccp1002-maternal-genetic-testing.pdf\n",
      "https://www.keystonefirstpa.com/covid-19/index.aspx\n",
      "https://www.keystonefirstpa.com/pdf/provider/resources/clinical/policies-20230217/ccp_1235_Genetic-testing-hereditary-cancer-susceptibility.pdf\n",
      "https://www.keystonefirstpa.com/pdf/pharmacy/forms/injectable/hepatitis-c-agents.pdf\n",
      "https://www.keystonefirstvipchoice.com/assets/pdf/provider/resources/clinical/policies-20230217/ccp1282-genetic-testing-duchenne-muscular-dystrophy.pdf\n",
      "https://www.keystonefirstpa.com/pdf/provider/resources/clinical/policies-20230217/ccp_1235_Genetic-testing-hereditary-cancer-susceptibility.pdf\n",
      "https://www.keystonefirstpa.com/pdf/provider/resources/clinical/policies-20200423/ccp1233-genetic-testing-alzheimer's-disease.pdf\n",
      "https://www.keystonefirstpa.com/pdf/provider/resources/clinical/policies-20201215/ccp1079-serum-biomarkers-liver-fibrosis.pdf\n",
      "https://www.keystonefirstpa.com/pdf/provider/resources/clinical/policies-20230217/ccp_1235_Genetic-testing-hereditary-cancer-susceptibility.pdf\n",
      "https://www.keystonefirstpa.com/pdf/provider/resources/clinical/policies-20201215/ccp1252-genetic-testing-hereditary-cardiomyopathy.pdf\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "https://resources.massgeneralbrighamhealthplan.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "https://resources.massgeneralbrighamhealthplan.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "https://resources.massgeneralbrighamhealthplan.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "1\n",
      "1\n",
      "1\n",
      "https://resources.allwayshealthpartners.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "https://resources.massgeneralbrighamhealthplan.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "1\n",
      "1\n",
      "https://resources.allwayshealthpartners.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "https://resources.allwayshealthpartners.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "https://resources.massgeneralbrighamhealthplan.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "1\n",
      "1\n",
      "1\n",
      "https://resources.massgeneralbrighamhealthplan.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "https://resources.massgeneralbrighamhealthplan.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "https://resources.massgeneralbrighamhealthplan.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "1\n",
      "1\n",
      "https://resources.allwayshealthpartners.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "1\n",
      "https://resources.massgeneralbrighamhealthplan.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "https://resources.allwayshealthpartners.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "https://resources.massgeneralbrighamhealthplan.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "1\n",
      "1\n",
      "1\n",
      "https://resources.allwayshealthpartners.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "https://resources.massgeneralbrighamhealthplan.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "1\n",
      "https://resources.massgeneralbrighamhealthplan.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "https://resources.massgeneralbrighamhealthplan.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "https://resources.allwayshealthpartners.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "1\n",
      "1\n",
      "https://resources.massgeneralbrighamhealthplan.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "1\n",
      "https://resources.massgeneralbrighamhealthplan.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "https://resources.massgeneralbrighamhealthplan.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "https://resources.allwayshealthpartners.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "https://resources.allwayshealthpartners.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "https://resources.allwayshealthpartners.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "https://resources.massgeneralbrighamhealthplan.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "1\n",
      "1\n",
      "https://resources.massgeneralbrighamhealthplan.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "https://resources.massgeneralbrighamhealthplan.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "https://resources.allwayshealthpartners.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "1\n",
      "https://resources.massgeneralbrighamhealthplan.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "1\n",
      "1\n",
      "1\n",
      "https://resources.massgeneralbrighamhealthplan.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "https://resources.allwayshealthpartners.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "1\n",
      "1\n",
      "1\n",
      "https://resources.massgeneralbrighamhealthplan.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "https://resources.massgeneralbrighamhealthplan.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "1\n",
      "https://resources.massgeneralbrighamhealthplan.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "1\n",
      "1\n",
      "1\n",
      "https://resources.massgeneralbrighamhealthplan.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "https://resources.massgeneralbrighamhealthplan.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "https://resources.massgeneralbrighamhealthplan.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "1\n",
      "https://resources.massgeneralbrighamhealthplan.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "1\n",
      "https://resources.massgeneralbrighamhealthplan.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "https://resources.massgeneralbrighamhealthplan.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "https://resources.allwayshealthpartners.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "https://resources.allwayshealthpartners.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "https://resources.massgeneralbrighamhealthplan.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "https://resources.massgeneralbrighamhealthplan.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "https://resources.massgeneralbrighamhealthplan.org/medicalpolicy/ExperimentalandInvestigational.pdf\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-algorithmic-testing.pdf?la=en&hash=4139EE47E5A0305B84C85D8AD8774B44\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-algorithmic-testing.pdf?la=en&hash=4139EE47E5A0305B84C85D8AD8774B44\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-algorithmic-testing.pdf?la=en&hash=4139EE47E5A0305B84C85D8AD8774B44\n",
      "https://partner.medica.com/-/media/documents/provider/utilization-management-policies/iii-dia-13-um-policy.pdf?la=en&hash=BCAD857162D2772C6D648753468B479F\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-algorithmic-testing.pdf?la=en&hash=4139EE47E5A0305B84C85D8AD8774B44\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-algorithmic-testing.pdf?la=en&hash=4139EE47E5A0305B84C85D8AD8774B44\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-algorithmic-testing.pdf?la=en&hash=4139EE47E5A0305B84C85D8AD8774B44\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-algorithmic-testing.pdf?la=en&hash=4139EE47E5A0305B84C85D8AD8774B44\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-for-cardiac-disorders.pdf?la=en&hash=4BB5D4F795DFF68BFEC2DC7F80968055\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-algorithmic-testing.pdf?la=en&hash=4139EE47E5A0305B84C85D8AD8774B44\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-algorithmic-testing.pdf?la=en&hash=4139EE47E5A0305B84C85D8AD8774B44\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-algorithmic-testing.pdf?la=en&hash=4139EE47E5A0305B84C85D8AD8774B44\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-algorithmic-testing.pdf?la=en&hash=4139EE47E5A0305B84C85D8AD8774B44\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-hereditary-cancer-susceptibility.pdf?la=en&hash=089EC258E606591DEA2EF0290D5CD5A2\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-algorithmic-testing.pdf?la=en&hash=4139EE47E5A0305B84C85D8AD8774B44\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-algorithmic-testing.pdf?la=en&hash=4139EE47E5A0305B84C85D8AD8774B44\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-for-aortopathies-connective-tissue.pdf?la=en&hash=74FF6DA77E89A11203E9E149E8B10E4E\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-general-approach-to-genetic-testing.pdf?la=en&hash=B71D2199B66A946C0C3D8899DAE3CD4F\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-algorithmic-testing.pdf?la=en&hash=4139EE47E5A0305B84C85D8AD8774B44\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-algorithmic-testing.pdf?la=en&hash=4139EE47E5A0305B84C85D8AD8774B44\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-skeletal-dysplasia-and-rare-bone-disorders.pdf?la=en\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-algorithmic-testing.pdf?la=en&hash=4139EE47E5A0305B84C85D8AD8774B44\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-molecular-analysis-of-solid-tumors-and-hematologic-malignancies.pdf?la=en&hash=6E509F72DAE1A3D9AFAC1F33872D6C23\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-for-epilepsy-neurodegenerative-and-neuromuscular-conditions.pdf?la=en&hash=D29F67B52F94935A831EF5AA6D6D216A\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-hereditary-cancer-susceptibility.pdf?la=en&hash=089EC258E606591DEA2EF0290D5CD5A2\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-for-epilepsy-neurodegenerative-and-neuromuscular-conditions.pdf?la=en&hash=D29F67B52F94935A831EF5AA6D6D216A\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-algorithmic-testing.pdf?la=en&hash=4139EE47E5A0305B84C85D8AD8774B44\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-hereditary-cancer-susceptibility.pdf?la=en&hash=089EC258E606591DEA2EF0290D5CD5A2\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-algorithmic-testing.pdf?la=en&hash=4139EE47E5A0305B84C85D8AD8774B44\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-molecular-analysis-of-solid-tumors-and-hematologic-malignancies.pdf?la=en&hash=6E509F72DAE1A3D9AFAC1F33872D6C23\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-general-approach-to-genetic-testing.pdf?la=en&hash=B71D2199B66A946C0C3D8899DAE3CD4F\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-hereditary-cancer-susceptibility.pdf?la=en&hash=089EC258E606591DEA2EF0290D5CD5A2\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-immune-autoimmune-and-rheumatoid-disorders.pdf?la=en&hash=3435761134D9C6F0AAFED5F7591C43AF\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-algorithmic-testing.pdf?la=en&hash=4139EE47E5A0305B84C85D8AD8774B44\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-hereditary-cancer-susceptibility.pdf?la=en&hash=089EC258E606591DEA2EF0290D5CD5A2\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-for-cardiac-disorders.pdf?la=en&hash=4BB5D4F795DFF68BFEC2DC7F80968055\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-for-cardiac-disorders.pdf?la=en&hash=4BB5D4F795DFF68BFEC2DC7F80968055\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-algorithmic-testing.pdf?la=en&hash=4139EE47E5A0305B84C85D8AD8774B44\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-hereditary-cancer-susceptibility.pdf?la=en&hash=089EC258E606591DEA2EF0290D5CD5A2\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/biochem-markers-liver-cp.pdf?la=en&hash=8D0B9CE58F91752A855AEE25B5B165AB\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-hereditary-cancer-susceptibility.pdf?la=en&hash=089EC258E606591DEA2EF0290D5CD5A2\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-hereditary-cancer-susceptibility.pdf?la=en&hash=089EC258E606591DEA2EF0290D5CD5A2\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-molecular-analysis-of-solid-tumors-and-hematologic-malignancies.pdf\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-for-epilepsy-neurodegenerative-and-neuromuscular-conditions.pdf?la=en&hash=D29F67B52F94935A831EF5AA6D6D216A\n",
      "1\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-cancer-screening.pdf?la=en&hash=E21E9F376AFFF0E457908CE464158B9E\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-general-approach-to-genetic-testing.pdf?la=en&hash=B71D2199B66A946C0C3D8899DAE3CD4F\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-for-epilepsy-neurodegenerative-and-neuromuscular-conditions.pdf?la=en&hash=D29F67B52F94935A831EF5AA6D6D216A\n",
      "1\n",
      "https://partner.medica.com/-/media/documents/provider/geneticandmolecularlaboratorytesting.pdf?la=en&hash=9156802186E01F1A953BC972C019ABE0\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-hereditary-cancer-susceptibility.pdf?la=en&hash=089EC258E606591DEA2EF0290D5CD5A2\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-hereditary-cancer-susceptibility.pdf?la=en&hash=089EC258E606591DEA2EF0290D5CD5A2\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-hereditary-cancer-susceptibility.pdf?la=en&hash=089EC258E606591DEA2EF0290D5CD5A2\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-hereditary-cancer-susceptibility.pdf?la=en&hash=089EC258E606591DEA2EF0290D5CD5A2\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-for-cardiac-disorders.pdf?la=en&hash=4BB5D4F795DFF68BFEC2DC7F80968055\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-for-epilepsy-neurodegenerative-and-neuromuscular-conditions.pdf?la=en&hash=D29F67B52F94935A831EF5AA6D6D216A\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-for-epilepsy-neurodegenerative-and-neuromuscular-conditions.pdf?la=en&hash=D29F67B52F94935A831EF5AA6D6D216A\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-general-approach-to-genetic-testing.pdf?la=en&hash=B71D2199B66A946C0C3D8899DAE3CD4F\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-cancer-screening.pdf?la=en&hash=E21E9F376AFFF0E457908CE464158B9E\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-molecular-analysis-of-solid-tumors-and-hematologic-malignancies.pdf\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-hereditary-cancer-susceptibility.pdf?la=en&hash=089EC258E606591DEA2EF0290D5CD5A2\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-molecular-analysis-of-solid-tumors-and-hematologic-malignancies.pdf?la=en&hash=6E509F72DAE1A3D9AFAC1F33872D6C23\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-molecular-analysis-of-solid-tumors-and-hematologic-malignancies.pdf?la=en&hash=6E509F72DAE1A3D9AFAC1F33872D6C23\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-general-approach-to-genetic-testing.pdf?la=en&hash=B71D2199B66A946C0C3D8899DAE3CD4F\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-general-approach-to-genetic-testing.pdf?la=en&hash=B71D2199B66A946C0C3D8899DAE3CD4F\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-for-hematologic-conditions-non-cancerous.pdf?la=en&hash=1CD491C45090F6F1DA3DEDA54412B767\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-pharmacogenetics.pdf?la=en&hash=DDF01783618647B8CE2135536820529D\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-hereditary-cancer-susceptibility.pdf?la=en&hash=089EC258E606591DEA2EF0290D5CD5A2\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-for-epilepsy-neurodegenerative-and-neuromuscular-conditions.pdf?la=en&hash=D29F67B52F94935A831EF5AA6D6D216A\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-for-gastroenterologic-disorders-non-cancerous.pdf?la=en&hash=8DA5E8ABA42690C03DCBDAF88B33EF46\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-general-approach-to-genetic-testing.pdf?la=en&hash=B71D2199B66A946C0C3D8899DAE3CD4F\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-general-approach-to-genetic-testing.pdf?la=en&hash=B71D2199B66A946C0C3D8899DAE3CD4F\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-hereditary-cancer-susceptibility.pdf?la=en&hash=089EC258E606591DEA2EF0290D5CD5A2\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-hereditary-cancer-susceptibility.pdf?la=en&hash=089EC258E606591DEA2EF0290D5CD5A2\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-molecular-analysis-of-solid-tumors-and-hematologic-malignancies.pdf?la=en&hash=6E509F72DAE1A3D9AFAC1F33872D6C23\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-general-approach-to-genetic-testing.pdf?la=en&hash=B71D2199B66A946C0C3D8899DAE3CD4F\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-general-approach-to-genetic-testing.pdf?la=en&hash=B71D2199B66A946C0C3D8899DAE3CD4F\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-pharmacogenetics.pdf?la=en&hash=DDF01783618647B8CE2135536820529D\n",
      "1\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-algorithmic-testing.pdf?la=en&hash=4139EE47E5A0305B84C85D8AD8774B44\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-prenatal-diagnosis-via-amniocentesis-cvs-or-pubs-and-pregnancy-loss.pdf?la=en&hash=F708B1733A39119E710FF523F1C1EC39\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-molecular-analysis-of-solid-tumors-and-hematologic-malignancies.pdf?la=en&hash=6E509F72DAE1A3D9AFAC1F33872D6C23\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-general-approach-to-genetic-testing.pdf?la=en&hash=B71D2199B66A946C0C3D8899DAE3CD4F\n",
      "1\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-metabolic-endocrine-and-mitochondrial-disorders.pdf?la=en&hash=DBC984C777D5760C5E329486D6841ABD\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-general-approach-to-genetic-testing.pdf?la=en&hash=B71D2199B66A946C0C3D8899DAE3CD4F\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-for-gastroenterologic-disorders-non-cancerous.pdf?la=en&hash=8DA5E8ABA42690C03DCBDAF88B33EF46\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-prenatal-diagnosis-via-amniocentesis-cvs-or-pubs-and-pregnancy-loss.pdf?la=en&hash=F708B1733A39119E710FF523F1C1EC39\n",
      "1\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-general-approach-to-genetic-testing.pdf?la=en&hash=B71D2199B66A946C0C3D8899DAE3CD4F\n",
      "https://partner.medica.com/-/media/documents/provider/clinical-policies/routine-prenatal-care-vigyn02.pdf?la=en&hash=D5AB2264E41D76254B233992D529E8DE\n",
      "https://partner.medica.com/-/media/documents/provider/covid-19-testing.pdf?la=en&hash=ADA9836AB9582B58D07613220B641038\n",
      "1\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-kidney-disorders.pdf?la=en&hash=792E452AE103442218E4D21ABCB5D385\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-circulating-tumor-dna-and-circulating-tumor-cells-liquid-biopsy.pdf\n",
      "1\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-kidney-disorders.pdf?la=en&hash=792E452AE103442218E4D21ABCB5D385\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-circulating-tumor-dna-and-circulating-tumor-cells-liquid-biopsy.pdf\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-algorithmic-testing.pdf?la=en&hash=4139EE47E5A0305B84C85D8AD8774B44\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-algorithmic-testing.pdf?la=en&hash=4139EE47E5A0305B84C85D8AD8774B44\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-general-approach-to-genetic-testing.pdf?la=en&hash=B71D2199B66A946C0C3D8899DAE3CD4F\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-molecular-analysis-of-solid-tumors-and-hematologic-malignancies.pdf\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-molecular-analysis-of-solid-tumors-and-hematologic-malignancies.pdf\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-molecular-analysis-of-solid-tumors-and-hematologic-malignancies.pdf\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-molecular-analysis-of-solid-tumors-and-hematologic-malignancies.pdf\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-prenatal-diagnosis-via-amniocentesis-cvs-or-pubs-and-pregnancy-loss.pdf?la=en&hash=F708B1733A39119E710FF523F1C1EC39\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-general-approach-to-genetic-testing.pdf?la=en&hash=B71D2199B66A946C0C3D8899DAE3CD4F\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-molecular-analysis-of-solid-tumors-and-hematologic-malignancies.pdf\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-hereditary-cancer-susceptibility.pdf?la=en&hash=089EC258E606591DEA2EF0290D5CD5A2\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-molecular-analysis-of-solid-tumors-and-hematologic-malignancies.pdf\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-hereditary-cancer-susceptibility.pdf?la=en&hash=089EC258E606591DEA2EF0290D5CD5A2\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/non-invasive-prenatal-testing.pdf?la=en&hash=EAC82FFAE10AE8E7DD5E5D65C0F37BC1\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-pharmacogenetics.pdf?la=en&hash=DDF01783618647B8CE2135536820529D\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-eye-disorders.pdf?la=en&hash=A83F6BE7EA6897D1DF0F94D961E76015\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/prenatal-and-preconception-carrier-screening.pdf?la=en&hash=D6ACA6304BC2FD45309E7EB42E1D5C73\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-hereditary-cancer-susceptibility.pdf?la=en&hash=089EC258E606591DEA2EF0290D5CD5A2\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-for-exome-and-genome-sequencing-for-the-diagnosis-of-genetic-disorders.pdf?la=en&hash=8B95C37C7CF5A10C9117AEEB18A46058\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-pharmacogenetics.pdf?la=en&hash=DDF01783618647B8CE2135536820529D\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-for-hematologic-conditions-non-cancerous.pdf?la=en&hash=1CD491C45090F6F1DA3DEDA54412B767\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-general-approach-to-genetic-testing.pdf?la=en&hash=B71D2199B66A946C0C3D8899DAE3CD4F\n",
      "1\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-hereditary-cancer-susceptibility.pdf?la=en&hash=089EC258E606591DEA2EF0290D5CD5A2\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-molecular-analysis-of-solid-tumors-and-hematologic-malignancies.pdf?la=en&hash=6E509F72DAE1A3D9AFAC1F33872D6C23\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/prenatal-and-preconception-carrier-screening.pdf?la=en&hash=D6ACA6304BC2FD45309E7EB42E1D5C73\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-algorithmic-testing.pdf?la=en&hash=4139EE47E5A0305B84C85D8AD8774B44\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-hereditary-cancer-susceptibility.pdf?la=en&hash=089EC258E606591DEA2EF0290D5CD5A2\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-hereditary-cancer-susceptibility.pdf?la=en&hash=089EC258E606591DEA2EF0290D5CD5A2\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-for-aortopathies-connective-tissue.pdf?la=en&hash=74FF6DA77E89A11203E9E149E8B10E4E\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-general-approach-to-genetic-testing.pdf?la=en&hash=B71D2199B66A946C0C3D8899DAE3CD4F\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-molecular-analysis-of-solid-tumors-and-hematologic-malignancies.pdf?la=en\n",
      "1\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-prenatal-diagnosis-via-amniocentesis-cvs-or-pubs-and-pregnancy-loss.pdf?la=en&hash=F708B1733A39119E710FF523F1C1EC39\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-prenatal-diagnosis-via-amniocentesis-cvs-or-pubs-and-pregnancy-loss.pdf?la=en&hash=F708B1733A39119E710FF523F1C1EC39\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-circulating-tumor-dna-and-circulating-tumor-cells-liquid-biopsy.pdf\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/whole-genome-sequencing-cp.pdf?la=en&hash=59A2B0315FE3B0AAC5E48EA4A1A92CB2\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-algorithmic-testing.pdf?la=en&hash=4139EE47E5A0305B84C85D8AD8774B44\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/genetic-testing-for-exome-and-genome-sequencing-for-the-diagnosis-of-genetic-disorders.pdf?la=en&hash=8B95C37C7CF5A10C9117AEEB18A46058\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/whole-genome-sequencing-cp.pdf?la=en&hash=59A2B0315FE3B0AAC5E48EA4A1A92CB2\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-algorithmic-testing.pdf?la=en&hash=4139EE47E5A0305B84C85D8AD8774B44\n",
      "https://partner.medica.com/-/media/documents/provider/coverage-policies/oncology-molecular-analysis-of-solid-tumors-and-hematologic-malignancies.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://www.modahealth.com/pdfs/Commercial_PA_not_covered_list.pdf\n",
      "https://www.modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://www.modahealth.com/pdfs/medical/Moda_medicare_external_PA_list.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://www.modahealth.com/pdfs/Commercial_PA_list.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://www.modahealth.com/pdfs/Commercial_PA_list.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://www.modahealth.com/pdfs/Commercial_PA_list.pdf\n",
      "https://www.modahealth.com/pdfs/Commercial_PA_not_covered_list.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://www.modahealth.com/pdfs/Commercial_PA_not_covered_list.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://www.modahealth.com/pdfs/Commercial_PA_list.pdf\n",
      "1\n",
      "https://www.modahealth.com/pdfs/med_criteria/Minimal%20Residue%20Disease%20Testing.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://www.modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://www.modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://www.modahealth.com/pdfs/medical/Moda_medicare_external_PA_list.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://www.modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://www.modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://www.modahealth.com/pdfs/med_criteria/Non-invasiveTestingLiverFibrosis.pdf\n",
      "1\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://www.modahealth.com/pdfs/Commercial_PA_not_covered_list.pdf\n",
      "https://www.modahealth.com/pdfs/medical/Moda_medicare_external_PA_list.pdf\n",
      "https://www.modahealth.com/pdfs/medical/Moda_medicare_external_PA_list.pdf\n",
      "https://www.modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://www.modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://www.modahealth.com/pdfs/Commercial_PA_list.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://www.modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://www.modahealth.com/pdfs/Commercial_PA_not_covered_list.pdf\n",
      "https://www.modahealth.com/pdfs/Commercial_PA_not_covered_list.pdf\n",
      "https://www.modahealth.com/pdfs/medical/Moda_medicare_external_PA_list.pdf\n",
      "https://www.modahealth.com/pdfs/Commercial_PA_not_covered_list.pdf\n",
      "https://www.modahealth.com/pdfs/medicare/assets/Moda-Health-MA-prior-authorization-guidelines.pdf\n",
      "https://www.modahealth.com/pdfs/Commercial_PA_list.pdf\n",
      "1\n",
      "https://www.modahealth.com/pdfs/medical/Moda_medicare_external_PA_list.pdf\n",
      "https://www.modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "1\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://www.modahealth.com/pdfs/Commercial_PA_not_covered_list.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://www.modahealth.com/pdfs/Commercial_PA_not_covered_list.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "nan\n",
      "https://www.modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "1\n",
      "https://www.modahealth.com/pdfs/Commercial_PA_list.pdf\n",
      "https://www.modahealth.com/pdfs/med_criteria/Tecentriq.pdf\n",
      "https://www.modahealth.com/pdfs/Commercial_PA_list.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "1\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://www.modahealth.com/pdfs/Commercial_PA_list.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "1\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://www.modahealth.com/pdfs/Commercial_PA_not_covered_list.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://www.modahealth.com/pdfs/medical/Moda_medicare_external_PA_list.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://www.modahealth.com/pdfs/med_criteria/Minimal%20Residue%20Disease%20Testing.pdf\n",
      "1\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://www.modahealth.com/pdfs/Commercial_PA_not_covered_list.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "1\n",
      "https://www.modahealth.com/pdfs/Commercial_PA_not_covered_list.pdf\n",
      "1\n",
      "1\n",
      "https://www.modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://www.modahealth.com/pdfs/Commercial_PA_not_covered_list.pdf\n",
      "https://www.modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "1\n",
      "https://www.modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://www.modahealth.com/pdfs/Commercial_PA_not_covered_list.pdf\n",
      "https://www.modahealth.com/pdfs/medical/Moda_medicare_external_PA_list.pdf\n",
      "https://www.modahealth.com/pdfs/medical/Moda_medicare_external_PA_list.pdf\n",
      "https://www.modahealth.com/pdfs/medicare/assets/Moda-Health-MA-prior-authorization-guidelines.pdf\n",
      "https://www.modahealth.com/pdfs/Commercial_PA_list.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://www.modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://www.modahealth.com/pdfs/medical/Moda_medicare_external_PA_list.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://www.modahealth.com/pdfs/prvdr_man_med.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://www.modahealth.com/pdfs/Commercial_PA_list.pdf\n",
      "https://modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://www.modahealth.com/pdfs/prvdr_man_med.pdf\n",
      "https://www.modahealth.com/pdfs/medical/Moda_medicare_external_PA_list.pdf\n",
      "https://www.modahealth.com/pdfs/med_criteria/GeneticTesting.pdf\n",
      "https://www.phs.org/providers/Documents/Next-Generation-Sequencing-MPM-29-0.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Breast-Cancer-Recurrence-Predictive-Genetic-Testing-MPM-33-0.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Testing-Prostate-Cancer-MPM-7-8.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Testing-Prostate-Cancer-MPM-7-8.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Breast-Cancer-Recurrence-Predictive-Genetic-Testing-MPM-33-0.pdf\n",
      "https://www.phs.org/providers/Documents/Breast-Cancer-Recurrence-Predictive-Genetic-Testing-MPM-33-0.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Testing-Uveal-Melanoma-MPM-7-9.pdf\n",
      "https://www.phs.org/providers/Documents/Breast-Cancer-Recurrence-Predictive-Genetic-Testing-MPM-33-0.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Testing-Prostate-Cancer-MPM-7-8.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Next-Generation-Sequencing-MPM-29-0.pdf\n",
      "https://www.phs.org/providers/Documents/Breast-Cancer-Recurrence-Predictive-Genetic-Testing-MPM-33-0.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Testing-Prostate-Cancer-MPM-7-8.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/GENESIGHT-Assay-Refractory-Depression-Medicare-MPM-30-0.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Testing-Lynch-Syndrome-MPM-7-5.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Multi-biomarker-Vectra-DA-Test-Rheumatoid-Arthritis-MPM-42-0.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Next-Generation-Sequencing-MPM-29-0.pdf\n",
      "https://www.phs.org/providers/Documents/Investigative-List-Non-Covered-Services-MPM-36-0.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Testing-Prostate-Cancer-MPM-7-8.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Testing-Cologuard-MPM-7-4.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Testing-Prostate-Cancer-MPM-7-8.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/COVID-19-Testing-MPM-43-0.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Testing-Prostate-Cancer-MPM-7-8.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Next-Generation-Sequencing-MPM-29-0.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Testing-Lynch-Syndrome-MPM-7-5.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "1\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/GENESIGHT-Assay-Refractory-Depression-Medicare-MPM-30-0.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Testing-Lynch-Syndrome-MPM-7-5.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Testing-Plasma-Based-Genomic-Profiling-Solid-Tumor-MPM-39-0.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Testing-Lynch-Syndrome-MPM-7-5.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Testing-Lynch-Syndrome-MPM-7-5.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Testing-Prostate-Cancer-MPM-7-8.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Testing-Lynch-Syndrome-MPM-7-5.pdf\n",
      "https://www.phs.org/providers/Documents/GENESIGHT-Assay-Refractory-Depression-Medicare-MPM-30-0.pdf\n",
      "https://www.phs.org/providers/Documents/Non-Invasive-Prenatal-Testing-MPM-20-15.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Testing-Lynch-Syndrome-MPM-7-5.pdf\n",
      "https://www.phs.org/providers/Documents/PHP-Submitting-COVID-Claims.pdf\n",
      "https://www.phs.org/providers/Documents/PHP-Submitting-COVID-Claims.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Testing-Lynch-Syndrome-MPM-7-5.pdf\n",
      "https://www.phs.org/providers/Documents/Non-Invasive-Prenatal-Testing-MPM-20-15.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "1\n",
      "1\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Testing-Prostate-Cancer-MPM-7-8.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Testing-Whole-Exome-Sequencing-MPM-7-12.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Non-Invasive-Prenatal-Testing-MPM-20-15.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "1\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Testing-Lynch-Syndrome-MPM-7-5.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Testing-Prostate-Cancer-MPM-7-8.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Testing-Lynch-Syndrome-MPM-7-5.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Next-Generation-Sequencing-MPM-29-0.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Testing-Cologuard-MPM-7-4.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Testing-Prostate-Cancer-MPM-7-8.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/COVID-19-Testing-MPM-43-0.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Testing-Lynch-Syndrome-MPM-7-5.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Testing-Prostate-Cancer-MPM-7-8.pdf\n",
      "https://www.phs.org/providers/Documents/Next-Generation-Sequencing-MPM-29-0.pdf\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.phs.org/providers/Documents/Non-Invasive-Prenatal-Testing-MPM-20-15.pdf\n",
      "https://www.phs.org/providers/Documents/Non-Invasive-Prenatal-Testing-MPM-20-15.pdf\n",
      "1\n",
      "https://www.phs.org/providers/Documents/Genetic-Genomic-Testing-MPM-7-1.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp47.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp317.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp213.pdf?sc_lang=en&rev=778486e80fe64e7cb3dd3d241077b57a&hash=625BC2F16D7BCAE08B13102B0E4141DF\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp47.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp213.pdf?sc_lang=en&rev=778486e80fe64e7cb3dd3d241077b57a&hash=625BC2F16D7BCAE08B13102B0E4141DF\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp352.pdf?rev=cbdde670b9cb472b801d39f3c0ddc3e5&sc_lang=en&hash=71A9CDF4619FCEA7BCB1053EE7E5A974\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp252.pdf?sc_lang=en&rev=ddcc04b2d63243f58c2e9bbb0d894a49&hash=6FAF3352DAA6511E64004BB54A35F471\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp96.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp47.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp96.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp39.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp213.pdf?sc_lang=en&rev=778486e80fe64e7cb3dd3d241077b57a&hash=625BC2F16D7BCAE08B13102B0E4141DF\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp194.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp148.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp213.pdf?sc_lang=en&rev=778486e80fe64e7cb3dd3d241077b57a&hash=625BC2F16D7BCAE08B13102B0E4141DF\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp47.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp89.pdf?sc_lang=en&rev=d460a38da01145488c409f03fecf45bb&hash=122174079114624A36F4C2A95A05E268\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp252.pdf?sc_lang=en&rev=ddcc04b2d63243f58c2e9bbb0d894a49&hash=6FAF3352DAA6511E64004BB54A35F471\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp143.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp194.pdf?sc_lang=en&rev=587ae6140e134bdba23b512a6db50a1e&hash=15AB445401C5372692E25F19A4D857B7\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp213.pdf?sc_lang=en&rev=778486e80fe64e7cb3dd3d241077b57a&hash=625BC2F16D7BCAE08B13102B0E4141DF\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp213.pdf?sc_lang=en&rev=778486e80fe64e7cb3dd3d241077b57a&hash=625BC2F16D7BCAE08B13102B0E4141DF\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp120.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp96.pdf?sc_lang=en&rev=8a67f290529a429896bbb4db41c86ac9&hash=CD33094A5DD4230D7A63D3DDDE70EFD1\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp39.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp215.pdf?sc_lang=en&rev=de2629b119e9404aa567c613878ac4d4&hash=8D1E339FA34E88F92D21FF62CDD33B0E\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp213.pdf?sc_lang=en&rev=778486e80fe64e7cb3dd3d241077b57a&hash=625BC2F16D7BCAE08B13102B0E4141DF\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp252.pdf?sc_lang=en&rev=ddcc04b2d63243f58c2e9bbb0d894a49&hash=6FAF3352DAA6511E64004BB54A35F471\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp213.pdf?sc_lang=en&rev=778486e80fe64e7cb3dd3d241077b57a&hash=625BC2F16D7BCAE08B13102B0E4141DF\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp115.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp213.pdf?sc_lang=en&rev=778486e80fe64e7cb3dd3d241077b57a&hash=625BC2F16D7BCAE08B13102B0E4141DF\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp115.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp215.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp115.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp215.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp213.pdf?sc_lang=en&rev=778486e80fe64e7cb3dd3d241077b57a&hash=625BC2F16D7BCAE08B13102B0E4141DF\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp143.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp106.pdf?sc_lang=en&rev=5c0b70561d5f49a38e3798329e472779&hash=02D558FB9AB20882E20A8DADB0F2D466\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp78.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp143.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp213.pdf?sc_lang=en&rev=778486e80fe64e7cb3dd3d241077b57a&hash=625BC2F16D7BCAE08B13102B0E4141DF\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp215.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp213.pdf?sc_lang=en&rev=778486e80fe64e7cb3dd3d241077b57a&hash=625BC2F16D7BCAE08B13102B0E4141DF\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/prior-authorization/php_prior_authorization_code_list.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp43.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/prior-authorization/php_prior_authorization_code_list.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp143.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp220.pdf?sc_lang=en&rev=2e93104396b44579b7145a0e53d68fa7&hash=65B8145C4B2A122B6E9C87BB4295287C\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp345.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp352.pdf?rev=cbdde670b9cb472b801d39f3c0ddc3e5&sc_lang=en&hash=71A9CDF4619FCEA7BCB1053EE7E5A974\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp256.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp143.pdf\n",
      "1\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp115.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/prior-authorization/php_prior_authorization_code_list.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp256.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp143.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/prior-authorization/php_prior_authorization_code_list.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp345.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp143.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp106.pdf?sc_lang=en&rev=5c0b70561d5f49a38e3798329e472779&hash=02D558FB9AB20882E20A8DADB0F2D466\n",
      "1\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/prior-authorization/php_prior_authorization_code_list.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp194.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp215.pdf?sc_lang=en&rev=de2629b119e9404aa567c613878ac4d4&hash=8D1E339FA34E88F92D21FF62CDD33B0E\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp213.pdf?sc_lang=en&rev=778486e80fe64e7cb3dd3d241077b57a&hash=625BC2F16D7BCAE08B13102B0E4141DF\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp215.pdf?rev=dc776bf730064b4ba3e94d43317a510e&sc_lang=en\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp143.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp78.pdf?rev=69b361ab0a11417bb5fc74b0fe62c8cd&sc_lang=en\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp115.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp345.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp215.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp215.pdf?sc_lang=en&rev=de2629b119e9404aa567c613878ac4d4&hash=8D1E339FA34E88F92D21FF62CDD33B0E\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp345.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp345.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp95.pdf?sc_lang=en&rev=8e20b1d9b9af4812a011fdeed3547d56&hash=0E53695F627B5DEF5551C3B1803988FA\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp213.pdf?sc_lang=en&rev=778486e80fe64e7cb3dd3d241077b57a&hash=625BC2F16D7BCAE08B13102B0E4141DF\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp213.pdf?sc_lang=en&rev=778486e80fe64e7cb3dd3d241077b57a&hash=625BC2F16D7BCAE08B13102B0E4141DF\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp115.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp115.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp194.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp96.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/prior-authorization/php_prior_authorization_code_list.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp213.pdf?sc_lang=en&rev=778486e80fe64e7cb3dd3d241077b57a&hash=625BC2F16D7BCAE08B13102B0E4141DF\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/prior-authorization/php_prior_authorization_code_list.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/prior-authorization/php_prior_authorization_code_list.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp256.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp78.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp345.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/prior-authorization/php_non_covered.pdf?sc_lang=en&rev=790b8060dea042ff9ae2e862b2587eda&hash=DEE76D6F6C7067C3D53B29E4A3B0ED13\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp143.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp78.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp213.pdf?sc_lang=en&rev=778486e80fe64e7cb3dd3d241077b57a&hash=625BC2F16D7BCAE08B13102B0E4141DF\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp78.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp78.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp78.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp143.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp143.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp219.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/prior-authorization/php_prior_authorization_code_list.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp313.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp313.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp72.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp213.pdf?sc_lang=en&rev=778486e80fe64e7cb3dd3d241077b57a&hash=625BC2F16D7BCAE08B13102B0E4141DF\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp352.pdf?rev=cbdde670b9cb472b801d39f3c0ddc3e5&sc_lang=en&hash=71A9CDF4619FCEA7BCB1053EE7E5A974\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp213.pdf?sc_lang=en&rev=778486e80fe64e7cb3dd3d241077b57a&hash=625BC2F16D7BCAE08B13102B0E4141DF\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp213.pdf?sc_lang=en&rev=778486e80fe64e7cb3dd3d241077b57a&hash=625BC2F16D7BCAE08B13102B0E4141DF\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp213.pdf?sc_lang=en&rev=778486e80fe64e7cb3dd3d241077b57a&hash=625BC2F16D7BCAE08B13102B0E4141DF\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp213.pdf?sc_lang=en&rev=778486e80fe64e7cb3dd3d241077b57a&hash=625BC2F16D7BCAE08B13102B0E4141DF\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp344.pdf?rev=571811fecdc54b859a4e1b5eef2982e2&sc_lang=en&hash=46C2C19657EA9466B2CB56C9B1F317BD\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp23.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp194.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp256.pdf\n",
      "https://www.providencehealthplan.com/providers/medical-policy-rx-pharmacy-and-provider-information\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp143.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp72.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp306.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp115.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp72.pdf\n",
      "https://www.bcbsri.com/providers/sites/providers/files/policies/2019/01/2019%20DRAFT%20Laboratory%20Tests%20for%20Heart%20and%20Kidney%20Transplant%20Rejection.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp213.pdf?sc_lang=en&rev=778486e80fe64e7cb3dd3d241077b57a&hash=625BC2F16D7BCAE08B13102B0E4141DF\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp78.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp23.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp143.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/prior-authorization/php_prior_authorization_code_list.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp256.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp213.pdf?sc_lang=en&rev=778486e80fe64e7cb3dd3d241077b57a&hash=625BC2F16D7BCAE08B13102B0E4141DF\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp213.pdf?sc_lang=en&rev=778486e80fe64e7cb3dd3d241077b57a&hash=625BC2F16D7BCAE08B13102B0E4141DF\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp115.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp110.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp252.pdf?rev=ebb114cb3ac446388961fd9f22826dfb&sc_lang=en\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp78.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp43.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp122.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp213.pdf?sc_lang=en&rev=778486e80fe64e7cb3dd3d241077b57a&hash=625BC2F16D7BCAE08B13102B0E4141DF\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp213.pdf?sc_lang=en&rev=778486e80fe64e7cb3dd3d241077b57a&hash=625BC2F16D7BCAE08B13102B0E4141DF\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp47.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp213.pdf?rev=947c1373fa27485e8d3395bb6f986830&sc_lang=en\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp23.pdf?rev=4c24dedac7fd4f09bd2d09964245316a&sc_lang=en\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp345.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp96.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp215.pdf\n",
      "1\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp345.pdf\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp213.pdf?sc_lang=en&rev=778486e80fe64e7cb3dd3d241077b57a&hash=625BC2F16D7BCAE08B13102B0E4141DF\n",
      "https://www.providencehealthplan.com/-/media/providence/website/pdfs/providers/medical-policy-and-provider-information/medical-policies/mp213.pdf?sc_lang=en&rev=778486e80fe64e7cb3dd3d241077b57a&hash=625BC2F16D7BCAE08B13102B0E4141DF\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/molecular-diagnostics-for-breast-cancer-prognosis/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/molecular-diagnostics-for-breast-cancer-prognosis/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/genetic-testing-and-genetic-expression-profiling-in-patients-with-uveal-melanoma/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/molecular-testing-of-pulmonary-specimens/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/gene-expression-profiling-and-protein-biomarkers-for-prostate-cancer/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/brca/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/molecular-diagnostics-for-breast-cancer-prognosis/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external/external-policies/molecular-profiling-for-cancers-of-unknown-primary-origin/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/testing-for-targeted-therapy-of-non-small-cell-lung-cancer/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/molecular-markers-in-fine-needle-aspirates-of-the-thyroid\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/brca/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/microsatellite-instability-and-tumor-mutational-burden-testing/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/testing-for-targeted-therapy-of-non-small-cell-lung-cancer/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/general-genetic-testing-germline-disorders/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/molecular-diagnostics-for-breast-cancer-prognosis/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/genetic-testing-and-genetic-expression-profiling-in-patients-with-cutaneous-melanoma/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/genetic-testing-for-neurodegenerative-disorders/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/molecular-diagnostics-for-breast-cancer-prognosis/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/gene-expression-profiling-and-protein-biomarkers-for-prostate-cancer/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/gene-expression-profiling-and-protein-biomarkers-for-prostate-cancer/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/molecular-testing-of-pulmonary-specimens/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/serum-tumor-markers-for-malignancies/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/lynch-syndrome-testing/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/gene-expression-profiling-and-protein-biomarkers-for-prostate-cancer/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/genetic-testing-for-fanconi-anemia/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/gene-expression-profiling-and-protein-biomarkers-for-prostate-cancer/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/testing-for-targeted-therapy-of-non-small-cell-lung-cancer/\n",
      "http://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/genetic-testing-for-inherited-cardiomyopathies-and-channelopathies/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external/external-policies/urinary-tumor-markers-for-bladder-cancer/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/genetic-testing-for-connective-tissue-disorders/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external/external-policies/urinary-tumor-markers-for-bladder-cancer/\n",
      "http://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/genetic-testing-for-inherited-cardiomyopathies-and-channelopathies/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/transplant-rejection-testing/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/genetic-testing-for-fanconi-anemia/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/noninvasive-techniques-for-the-evaluation-and-monitoring-of-patients-with-chronic-liver-disease/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/genetic-testing-for-pten-hamartoma-tumor-syndrome/\n",
      "http://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/genetic-testing-for-inherited-cardiomyopathies-and-channelopathies/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/gene-expression-profiling-and-protein-biomarkers-for-prostate-cancer/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/brca/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/genetic-testing-for-connective-tissue-disorders/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/lynch-syndrome-testing/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/general-genetic-testing-germline-disorders/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/general-genetic-testing-germline-disorders/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/genetic-testing-for-polyposis-syndromes/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/brca/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/genetic-testing-for-duchenne-becker-facioscapulohumeral-and-limb-girdle-muscular-dystrophies/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external/external-policies/genetic-testing-for-hereditary-pancreatitis/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/colorectal-cancer-screening/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/kras-nras-and-braf-mutation-analysis-in-metastatic-colorectal-cancer/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/colorectal-cancer-screening/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/kras-nras-and-braf-mutation-analysis-in-metastatic-colorectal-cancer/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/allogeneic-hematopoietic-stem-cell-transplantation-for-genetic-diseases-and-acquired-anemias/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/lynch-syndrome-testing/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/hematopoietic-cell-transplantation-for-chronic-lymphocytic-leukemiasmall-lymphocytic-lymphoma/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/genetic-testing-for-polyposis-syndromes/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/fecal-analysis-in-the-diagnosis-of-intestinal-dysbiosis-and-fecal-microbiota-transplant-testing/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/molecular-analysis-for-gliomas/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/celiac-disease-testing/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/genetic-testing-for-hereditary-hemochromatosis/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/lynch-syndrome-testing/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/redblood-cell-molecular-testing/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/bcr-abl1-testing/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/genetic-testing-for-familial-hypercholesterolemia/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/hepatitis-testing/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/genetic-testing-for-acute-myeloid-leukemia/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/molecular-analysis-for-gliomas/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/genetic-testing-for-acute-myeloid-leukemia/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/lynch-syndrome-testing/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/whole-genome-and-whole-exome-sequencing/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/lynch-syndrome-testing/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/genetic-testing-for-acute-myeloid-leukemia/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/brca/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/genetic-testing-for-germline-mutations-of-the-ret-proto-oncogene/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/lynch-syndrome-testing/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/genetic-testing-for-germline-mutations-of-the-ret-proto-oncogene/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/genetic-testing-for-familial-hypercholesterolemia/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/gene-expression-profiling-and-protein-biomarkers-for-prostate-cancer/\n",
      "1\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/microsatellite-instability-and-tumor-mutational-burden-testing/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/testing-for-targeted-therapy-of-non-small-cell-lung-cancer/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/diagnostic-testing-of-influenza/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/identification-of-microorganisms-using-nucleic-acid-probes/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/pharmacogenetic-testing/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/allogeneic-hematopoietic-stem-cell-transplantation-for-genetic-diseases-and-acquired-anemias/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/genetic-testing-for-cystic-fibrosis/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/transplant-rejection-testing/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/genetic-testing-of-mitochondrial-disorders/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/genetic-testing-and-genetic-expression-profiling-in-patients-with-cutaneous-melanoma/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/genetic-testing-for-acute-myeloid-leukemia/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/genetic-testing-for-neurodegenerative-disorders/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/prenatal-testing-for-fetal-aneuploidy/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/transplant-rejection-testing/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/redblood-cell-molecular-testing/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/testing-for-targeted-therapy-of-non-small-cell-lung-cancer/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/brca/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/mutation-analysis-in-myeloproliferative-neoplasms/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/genetic-testing-and-genetic-expression-profiling-in-patients-with-cutaneous-melanoma/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external/external-policies/general-genetic-testing-somatic-disorders/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/prenatal-testing-for-fetal-aneuploidy/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/prenatal-testing-for-fetal-aneuploidy/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/genetic-testing-of-mitochondrial-disorders/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/lyme-disease-testing/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/chromosomal-microarray/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/genetic-testing-of-mitochondrial-disorders/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/brca/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/testing-for-targeted-therapy-of-non-small-cell-lung-cancer\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/pharmacogenetic-testing/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external/external-policies/coronavirus-testing-in-the-outpatient-setting/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/pharmacogenetic-testing/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/diagnosis-of-vaginitis-including-multi-target-pcr-testing/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/lynch-syndrome-testing/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/brca/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/general-genetic-testing-germline-disorders/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/genetic-testing-for-pten-hamartoma-tumor-syndrome/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/lynch-syndrome-testing/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/genetic-testing-for-ophthalmologic-conditions/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/pharmacogenetic-testing/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/genetic-testing-for-cystic-fibrosis/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/pharmacogenetic-testing/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/whole-genome-and-whole-exome-sequencing/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/genetic-testing-for-rett-syndrome/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/genetic-testing-for-acute-myeloid-leukemia/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/prenatal-testing-for-fetal-aneuploidy/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/genetic-testing-for-cystic-fibrosis/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/mutation-analysis-in-myeloproliferative-neoplasms/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/testing-for-targeted-therapy-of-non-small-cell-lung-cancer/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/genetic-testing-for-the-diagnosis-of-inherited-peripheral-neuropathies/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/brca/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/genetic-testing-for-germline-mutations-of-the-ret-proto-oncogene/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/hematopoietic-stem-cell-transplantation-for-non-hodgkin-lymphomas/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/molecular-markers-in-fine-needle-aspirates-of-the-thyroid\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/identification-of-microorganisms-using-nucleic-acid-probes/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/genetic-testing-and-genetic-expression-profiling-in-patients-with-cutaneous-melanoma/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/bcr-abl1-testing/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/whole-genome-and-whole-exome-sequencing/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/genetic-markers-for-assessing-risk-of-cardiovascular-disease/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/diagnostic-testing-of-influenza/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/molecular-markers-in-fine-needle-aspirates-of-the-thyroid\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/general-genetic-testing-germline-disorders/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/hematopoietic-cell-transplantation-for-chronic-lymphocytic-leukemiasmall-lymphocytic-lymphoma/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/vectra-da-blood-test-for-rheumatoid-arthritis/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external/external-policies/pathogen-panel-testing/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/lynch-syndrome-testing/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/prenatal-testing-for-fetal-aneuploidy/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/general-genetic-testing-somatic-disorders/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/genetic-testing-for-alpha--and-beta-thalassemia/\n",
      "https://www.southcarolinablues.com/web/public/brands/medicalpolicy/external-policies/whole-genome-and-whole-exome-sequencing/\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Algorithmic-Testing-9605\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Algorithmic-Testing-9605\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Algorithmic-Testing-9605\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Algorithmic-Testing-9605\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Algorithmic-Testing-9605\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Algorithmic-Testing-9605\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Algorithmic-Testing-9605\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Algorithmic-Testing-9605\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Algorithmic-Testing-9605\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Algorithmic-Testing-9605\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Algorithmic-Testing-9605\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Algorithmic-Testing-9605\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Algorithmic-Testing-9605\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Algorithmic-Testing-9605\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-Pharmaco-9602\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-Cancer-9596\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Test-Cancer-9596\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-9610\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Algorithmic-Testing-9605\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Test-Cancer-9596\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Algorithmic-Testing-9605\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-Epilepsy-9591\n",
      "1\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-Intell-9587\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Algorithmic-Testing-9605\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-General-9610\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Algorithmic-Testing-9605\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Algorithmic-Testing-9605\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-9610\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-9610\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Test-Cancer-9596\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-Cancer-9596\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-9610\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Cancer-Screening-9606\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-Hematologic-9595\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-Cancer-9596\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Cytogenetic-9607\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-Cancer-9596\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-Cancer-9596\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Test-Pharmacogenetic-9602\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Prenatal-9575\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-9610\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Algorithmic-Testing-9605\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Algorithmic-Testing-9605\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-Eye-9592\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Prenatal-Amnio-9576\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-Pharmaco-9602\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-Pharmaco-9602\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-9610\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Molecular-9608\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-Metabolic-9600\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-Hematologic-9595\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Cancer-Screening-9606\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Algorithmic-Testing-9605\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-Pharmaco-9602\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-test-preimplantation-9574\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-Epilepsy-9591\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-Metabolic-9600\n",
      "1\n",
      "1\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Algorithmic-Testing-9605\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-Cancer-9596\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-9610\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-Cancer-9596\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-Kidney-9598\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-Kidney-9598\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-9610\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-Cancer-9596\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-9610\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Molecular-9608\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-9610\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Prenatal-9575\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-Gastro-9593\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-Gastro-9593\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Test-Kidney-9598\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-Cardiac-9589\n",
      "https://www.wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Prenatal-Amnio-9576\n",
      "https://www.ssmhealth.com/SSMHealth/media/Documents/Locations/Non-Hospital/consultants-laboratory-wisconsin-directory-of-services.pdf\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Algorithmic-Testing-9605\n",
      "https://www.ssmhealth.com/cardinal-glennon/services/pediatric-pulmonology/cystic-fibrosis/diagnosis-treatment\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Algorithmic-Testing-9605\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-9610\n",
      "https://www.wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Prenatal-Amnio-9576\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-Cardiac-9589\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-Eye-9592\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-Hematologic-9595\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-9610\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Test-Pharmacogenetic-9602\n",
      "https://www.wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-Epilepsy-9591\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Test-Pharmacogenetic-9602\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-General-9610\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Molecular-Anal-9608\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Molecular-9608\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Test-Metabolic-Endo-9600\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-Hematologic-9595\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Molecular-9608\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Cancer-Screen-9606\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Molecular-9608\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Molecular-9608\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Algor-Test-9605\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Molecular-9608\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Test-Pharmacogenetic-9602\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Molecular-9608\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Prenatal-Amnio-9576\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Molecular-9608\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-Epilepsy-9591\n",
      "https://www.wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-Epilepsy-9591\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-NIPS-9573\n",
      "https://www.wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-Exome-9586\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Test-Metabolic-Endo-9600\n",
      "https://staging.wellfirstbenefits.com/Document-Library/PDF/Medical-Policies/Genetic-Testing-for-Cowden-Syndrome-9488\n",
      "1\n",
      "https://www.ssmhealth.com/getmedia/3401c151-1eba-4b30-b1bf-1d7600d3e845/clinical-practice-guideline-respiratory-pathogen-panel.pdf\n",
      "1\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Algor-Test-9605\n",
      "https://www.ssmhealth.com/SSMHealth/media/Documents/Locations/Non-Hospital/consultants-laboratory-wisconsin-directory-of-services.pdf\n",
      "https://www.ssmhealth.com/SSMHealth/media/Documents/Locations/Non-Hospital/consultants-laboratory-wisconsin-directory-of-services.pdf\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Molecular-9608\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Test-Cancer-9596\n",
      "https://www.ssmhealth.com/getmedia/3401c151-1eba-4b30-b1bf-1d7600d3e845/clinical-practice-guideline-respiratory-pathogen-panel.pdf\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-NIPS-9573\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-General-9610\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Test-Cancer-9596\n",
      "https://www.wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-Aortopathies-9588\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Molecular-9608\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Test-Kidney-9598\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-NIPS-9573\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Molecular-9608\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Test-Kidney-9598\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-General-9610\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Test-Cancer-9596\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-General-9610\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Molecular-Anal-9608\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-General-9610\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Molecular-9608\n",
      "https://staging.wellfirstbenefits.com/Document-Library/PDF/Medical-Policies/Genetic-Testing-for-Cowden-Syndrome-9488\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Molecular-9608\n",
      "https://wellfirstbenefits.com/Document-Library/PDF/Providers/Provider-newsletter-winter-2021\n",
      "https://www.ssmhealth.com/SSMHealth/media/Documents/Locations/Non-Hospital/consultants-laboratory-wisconsin-directory-of-services.pdf\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Prenatal-Preconception-9575\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Test-GI-DX-9593\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-General-9610\n",
      "https://www.wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-Epilepsy-9591\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Oncology-Molecular-9608\n",
      "https://wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Test-Kidney-9598\n",
      "https://www.wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-Aortopathies-9588\n",
      "https://www.wellfirstbenefits.com/Document-Library/pdf/Medical-Policies/Genetic-Testing-Epilepsy-9591\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Documents%20and%20Forms/Provider%20Notices/WellSense%20MA%20Network%20Notification%20M227.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/Network%20Notifications/WellSense%20Mass%20NN%20231%20-%20January%20Medical%20Policy%20Network%20Notifications.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Documents%20and%20Forms/Prior%20Authorization%20Resources/CPT%20Code%20Lookup%20Tool_Updated%2006072022.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Policies/Medical%20Policies/August%202022/Genetic%20Genomic%20Pharmacogenetics%20Effective%2008_01_22.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Documents%20and%20Forms/Provider%20Notices/WellSense%20MA%20Network%20Notification%20M227.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/NH/Provider%20Notifications/WellSense%20NH%20Network%20Notification%2074.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Documents%20and%20Forms/Prior%20Authorization%20Resources/CPT%20Code%20Lookup%20Tool_Updated%2006072022.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/NH/Provider%20Notifications/WellSense%20NH%20Network%20Notification%2074.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Documents%20and%20Forms/Provider%20Notices/WellSense%20MA%20Network%20Notification%20M227.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Documents%20and%20Forms/Provider%20Notices/WellSense%20MA%20Network%20Notification%20M227.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/NH/Provider%20Notifications/WellSense%20NH%20Network%20Notification%2074.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Documents%20and%20Forms/Provider%20Notices/WellSense%20MA%20Network%20Notification%20M227.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/Network%20Notifications/WellSense%20Mass%20NN%20231%20-%20January%20Medical%20Policy%20Network%20Notifications.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Documents%20and%20Forms/Prior%20Authorization%20Resources/CPT%20Code%20Lookup%20Tool_Updated%2006072022.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Documents%20and%20Forms/Prior%20Authorization%20Resources/CPT%20Code%20Lookup%20Tool_Updated%2006072022.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/Network%20Notifications/WellSense%20Mass%20NN%20231%20-%20January%20Medical%20Policy%20Network%20Notifications.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Documents%20and%20Forms/Provider%20Notices/WellSense%20MA%20Network%20Notification%20M227.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/NH/Provider%20Notifications/WellSense%20NH%20Network%20Notification%2074.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Documents%20and%20Forms/Provider%20Notices/WellSense%20MA%20Network%20Notification%20M227.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Documents%20and%20Forms/Prior%20Authorization%20Resources/CPT%20Code%20Lookup%20Tool_Updated%2006072022.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Documents%20and%20Forms/Prior%20Authorization%20Resources/CPT%20Code%20Lookup%20Tool_Updated%2006072022.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/NH/Provider%20Notifications/WellSense%20NH%20Network%20Notification%2074.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Documents%20and%20Forms/Provider%20Notices/WellSense%20MA%20Network%20Notification%20M227.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Documents%20and%20Forms/Provider%20Notices/WellSense%20MA%20Network%20Notification%20M227.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/NH/Provider%20Notifications/WellSense%20NH%20Network%20Notification%2074.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Documents%20and%20Forms/Prior%20Authorization%20Resources/CPT%20Code%20Lookup%20Tool_Updated%2006072022.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/NH/Provider%20Notifications/WellSense%20NH%20Network%20Notification%2074.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Documents%20and%20Forms/Provider%20Notices/WellSense%20MA%20Network%20Notification%20M227.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Documents%20and%20Forms/Provider%20Notices/WellSense%20MA%20Network%20Notification%20M227.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/NH/Provider%20Notifications/WellSense%20NH%20Network%20Notification%2074.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Documents%20and%20Forms/Provider%20Notices/WellSense%20MA%20Network%20Notification%20M227.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Policies/Medical%20Policies/Oct%202022/Chromosomal%20Microarray%20Analysis%20Effective%2002_01_22%20and%20Retired%2011_30_22.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/NH/Provider%20Notifications/WellSense%20NH%20Network%20Notification%2074.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Policies/Medical%20Policies/August%202022/Genetic%20Genomic%20Pharmacogenetics%20Effective%2008_01_22.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/NH/Provider%20Notifications/WellSense%20NH%20Network%20Notification%2074.pdf\n",
      "https://bostonmedicalcenter.policytech.com/dotNet/documents/?docid=5796&public=true\n",
      "https://www.wellsense.org/hubfs/Provider/NH/Provider%20Notifications/WellSense%20NH%20Network%20Notification%2074.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Documents%20and%20Forms/Prior%20Authorization%20Resources/CPT%20Code%20Lookup%20Tool_Updated%2006072022.pdf\n",
      "1\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Documents%20and%20Forms/Provider%20Notices/WellSense%20MA%20Network%20Notification%20M227.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Documents%20and%20Forms/Prior%20Authorization%20Resources/WellSense%20MA%20CPT%2001_03_23.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Documents%20and%20Forms/Prior%20Authorization%20Resources/CPT%20Code%20Lookup%20Tool_Updated%2006072022.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Documents%20and%20Forms/Prior%20Authorization%20Resources/CPT%20Code%20Lookup%20Tool_Updated%2006072022.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Documents%20and%20Forms/Prior%20Authorization%20Resources/CPT%20Code%20Lookup%20Tool_Updated%2006072022.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Policies/Medical%20Policies/Oct%202022/Genetic%20Testing%20Fragile%20X-Associated%20Disorders%20Effective%2002_01_22%20and%20Retired%2011_30_22.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Documents%20and%20Forms/Prior%20Authorization%20Resources/CPT%20Code%20Lookup%20Tool_Updated%2006072022.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/NH/Provider%20Notifications/WellSense%20NH%20Network%20Notification%2074.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/NH/Provider%20Notifications/WellSense%20NH%20Network%20Notification%2074.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Documents%20and%20Forms/Prior%20Authorization%20Resources/CPT%20Code%20Lookup%20Tool_Updated%2006072022.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Documents%20and%20Forms/Prior%20Authorization%20Resources/WellSense%20MA%20CPT%2001_03_23.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/NH/Provider%20Notifications/WellSense%20NH%20Network%20Notification%2074.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/NH/Provider%20Notifications/WellSense%20NH%20Network%20Notification%2074.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Policies/Medical%20Policies/August%202022/Genetic%20Genomic%20Pharmacogenetics%20Effective%2008_01_22.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/NH/Provider%20Notifications/WellSense%20NH%20Network%20Notification%2074.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/NH/Provider%20Notifications/WellSense%20NH%20Network%20Notification%2074.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/NH/Provider%20Notifications/WellSense%20NH%20Network%20Notification%2074.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Documents%20and%20Forms/Provider%20Notices/WellSense%20MA%20Network%20Notification%20M227.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Documents%20and%20Forms/Provider%20Notices/WellSense%20MA%20Network%20Notification%20M227.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Documents%20and%20Forms/Provider%20Notices/WellSense%20MA%20Network%20Notification%20M227.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Documents%20and%20Forms/Prior%20Authorization%20Resources/CPT%20Code%20Lookup%20Tool_Updated%2006072022.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Documents%20and%20Forms/Provider%20Notices/WellSense%20MA%20Network%20Notification%20M227.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/NH/Provider%20Notifications/WellSense%20NH%20Network%20Notification%2074.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/Network%20Notifications/WellSense%20Mass%20NN%20231%20-%20January%20Medical%20Policy%20Network%20Notifications.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/Network%20Notifications/WellSense%20Mass%20NN%20231%20-%20January%20Medical%20Policy%20Network%20Notifications.pdf\n",
      "1\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Documents%20and%20Forms/Prior%20Authorization%20Resources/WellSense%20MA%20CPT%2001_03_23.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Policies/Medical%20Policies/Oct%202022/Chromosomal%20Microarray%20Analysis%20Effective%2002_01_22%20and%20Retired%2011_30_22.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/NH/Provider%20Notifications/WellSense%20NH%20Network%20Notification%2074.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/NH/Provider%20Notifications/WellSense%20NH%20Network%20Notification%2074.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Documents%20and%20Forms/Provider%20Notices/WellSense%20MA%20Network%20Notification%20M227.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/NH/Provider%20Notifications/WellSense%20NH%20Network%20Notification%2074.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/NH/Provider%20Notifications/WellSense%20NH%20Network%20Notification%2074.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/NH/Provider%20Notifications/WellSense%20NH%20Network%20Notification%2074.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/NH/Provider%20Notifications/WellSense%20NH%20Network%20Notification%2074.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Policies/Medical%20Policies/Oct%202022/Chromosomal%20Microarray%20Analysis%20Effective%2002_01_22%20and%20Retired%2011_30_22.pdf\n",
      "https://www.wellsense.org/covid-19-resources-providers-massachusetts\n",
      "https://www.wellsense.org/hubfs/Provider/NH/Provider%20Notifications/WellSense%20NH%20Network%20Notification%2074.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Documents%20and%20Forms/Provider%20Notices/WellSense%20MA%20Network%20Notification%20M227.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/Network%20Notifications/WellSense%20Mass%20NN%20231%20-%20January%20Medical%20Policy%20Network%20Notifications.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Documents%20and%20Forms/Prior%20Authorization%20Resources/CPT%20Code%20Lookup%20Tool_Updated%2006072022.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Policies/Medical%20Policies/August%202022/Genetic%20Genomic%20Pharmacogenetics%20Effective%2008_01_22.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Documents%20and%20Forms/Provider%20Notices/WellSense%20MA%20Network%20Notification%20M227.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Policies/Medical%20Policies/August%202022/Genetic%20Genomic%20Pharmacogenetics%20Effective%2008_01_22.pdf\n",
      "1\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Documents%20and%20Forms/Provider%20Notices/WellSense%20MA%20Network%20Notification%20M227.pdf\n",
      "1\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Documents%20and%20Forms/Provider%20Notices/WellSense%20MA%20Network%20Notification%20M227.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Documents%20and%20Forms/Prior%20Authorization%20Resources/CPT%20Code%20Lookup%20Tool_Updated%2006072022.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Policies/Medical%20Policies/August%202022/Genetic%20Genomic%20Pharmacogenetics%20Effective%2008_01_22.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Documents%20and%20Forms/Prior%20Authorization%20Resources/CPT%20Code%20Lookup%20Tool_Updated%2006072022.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Documents%20and%20Forms/Prior%20Authorization%20Resources/CPT%20Code%20Lookup%20Tool_Updated%2006072022.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Policies/Medical%20Policies/August%202022/Genetic%20Genomic%20Pharmacogenetics%20Effective%2008_01_22.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Documents%20and%20Forms/Provider%20Notices/WellSense%20MA%20Network%20Notification%20M227.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/Network%20Notifications/WellSense%20Mass%20NN%20231%20-%20January%20Medical%20Policy%20Network%20Notifications.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Policies/Medical%20Policies/August%202022/Genetic%20Genomic%20Pharmacogenetics%20Effective%2008_01_22.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Documents%20and%20Forms/Provider%20Notices/WellSense%20MA%20Network%20Notification%20M227.pdf\n",
      "https://www.wellsense.org/hubfs/Provider/Prior%20Authorization/WellSense%20NH%20CPT.pdf?hsLang=en\n",
      "https://www.wellsense.org/hubfs/Provider/MA/Policies/Medical%20Policies/Oct%202022/Chromosomal%20Microarray%20Analysis%20Effective%2002_01_22%20and%20Retired%2011_30_22.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/07-06-2022/July-Medical-Remove-and-Replace/204102%20Whole%20Exome%20and%20Whole%20Genome.pdf\n",
      "https://www.fepblue.org/-/media/FEPBlue-Sitecore-10-Media/PDFs/04-01-2023/Med%20Policy%20Remove%20and%20Replace/204137%20Genetic%20Testing%20for%20Neurofibromatosis.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/Jan-2023-Med-Replace/20402%20Germline%20Genetic%20Testing%20Hereditary%20BRCA.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/7-13-2021/204120%20Gene%20Expression%20Profiling%20for%20Uveal%20Melanoma.pdf\n",
      "https://www.fepblue.org/-/media/FEPBlue-Sitecore-10-Media/PDFs/04-01-2023/Med%20Policy%20Remove%20and%20Replace/20436%20Assays%20of%20Genetic%20Expression.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/1-11-2021/20408%20Genetic%20Testing%20for%20Lynch%20Syndrome%20and%20Other%20Inherited%20Colon%20Cancer%20Syndromes.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/Jan-2023-Med-Replace/20478%20Molecular%20Markers%20in%20Fine.pdf\n",
      "1\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/Jan-2023-Med-Replace/20485%20BCR%20ABL1%20Testing%20in%20CML.pdf\n",
      "https://www.fepblue.org/-/media/FEPBlue-Sitecore-10-Media/PDFs/04-01-2023/Med%20Policy%20Remove%20and%20Replace/204111%20Gene%20Expression%20Profiliing.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/Jan-2023-Med-New/204156%20Germline%20and%20Somatic%20Biomarker%20Ovarian.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/3-18-2022/Medical_Policies/20489%20Genetic%20Testing%20for%20the%20Diagnosis.pdf\n",
      "1\n",
      "https://www.fepblue.org/-/media/FEPBlue-Sitecore-10-Media/PDFs/04-01-2023/Med%20Policy%20Remove%20and%20Replace/204111%20Gene%20Expression%20Profiliing.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/4-22-2021/MED%20Remove%20and%20Replace/20436%20Assays%20of%20Genetic%20Expression%20in%20Tumor%20Tissue%20as%20a%20Technique%20to.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/4-22-2021/MED%20Remove%20and%20Replace/204125%20Proteomic%20Testing%20for%20Targeted%20Therapy%20in%20NonSmallCell%20Lung%20Cancer.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/Jan-2023-Med-Replace/20478%20Molecular%20Markers%20in%20Fine.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/10-20-2021/20230%20Transcatheter%20Mitral%20Valve%20Repair.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/10-12-2022/Med%20Remove%20and%20Replace/204101%20Genetic%20Testing%20for%20Li.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/10-22-2020/20453%20KRAS%20NRAS%20BRAF%20Variant%20Analysis%20in%20Metastatic%20Colorectal%20Cancer.pdf\n",
      "https://www.fepblue.org/-/media/FEPBlue-Sitecore-10-Media/PDFs/04-01-2023/Phar%20Policy%20Remove%20and%20Replace/521144%20Pemazyre%20pemigatinib.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/1-6-2022/204115%20Comprehensive%20Genomic%20Profiling%20for%20Selecting.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/1-6-2022/204115%20Comprehensive%20Genomic%20Profiling%20for%20Selecting.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/4-22-2021/MED%20Remove%20and%20Replace/20436%20Assays%20of%20Genetic%20Expression%20in%20Tumor%20Tissue%20as%20a%20Technique%20to.pdf\n",
      "1\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/1-6-2022/204115%20Comprehensive%20Genomic%20Profiling%20for%20Selecting.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/Jan-2023-Med-Replace/20461%20Gene%20Expression%20Profile%20Testing.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/4-22-2021/MED%20Remove%20and%20Replace/20433%20Genetic%20and%20Protein%20Biomarkers%20for%20the%20Diagnosis%20and.pdf\n",
      "https://www.fepblue.org/-/media/FEPBlue-Sitecore-10-Media/PDFs/04-01-2023/Med%20Policy%20Remove%20and%20Replace/204124%20Genetic%20Testing%20for%20FLT3.pdf\n",
      "https://www.fepblue.org/benefit-plans/medical-policies-and-utilization-management-guidelines/-/media/pdfs/medical%20policies/04-04-2019/2_04_33%20genetic%20and%20protein%20biomarkers%20for%20the%20diagnosis%20and%20cancer%20risk%20assessment%20of%20prostate%20cancer.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/3-18-2022/Medical_Policies/20462%20Multimarker%20Serum%20Testing.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/4-22-2021/MED%20Remove%20and%20Replace/20407%20Urinary%20Biomarkers.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/3-18-2022/Medical_Policies/204124%20Genetic%20Testing%20for%20FLT3.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/4-22-2021/MED%20Remove%20and%20Replace/20433%20Genetic%20and%20Protein%20Biomarkers%20for%20the%20Diagnosis%20and.pdf\n",
      "https://www.fepblue.org/-/media/FEPBlue-Sitecore-10-Media/PDFs/04-01-2023/Med%20Policy%20Remove%20and%20Replace/204111%20Gene%20Expression%20Profiliing.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/10-22-2020/20453%20KRAS%20NRAS%20BRAF%20Variant%20Analysis%20in%20Metastatic%20Colorectal%20Cancer.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/1-11-2021/20408%20Genetic%20Testing%20for%20Lynch%20Syndrome%20and%20Other%20Inherited%20Colon%20Cancer%20Syndromes.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/4-22-2021/MED%20Remove%20and%20Replace/20407%20Urinary%20Biomarkers.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/June-2023/Med%20Remove-Replace/204148%20Germline%20Genetic%20Testing%20Pancreatic.pdf\n",
      "https://www.fepblue.org/-/media/FEPBlue-Sitecore-10-Media/PDFs/04-01-2023/Med%20Policy%20Remove%20and%20Replace/20436%20Assays%20of%20Genetic%20Expression.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/10-20-2021/20438%20Cytochrome%20P450%20GenotypeGuided%20Treatment%20Strategy.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/4-22-2021/MED%20Remove%20and%20Replace/204124%20Genetic%20Testing%20for%20FLT3%20NPM1%20and%20CEBPA%20Variants.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/3-18-2022/Medical_Policies/20462%20Multimarker%20Serum%20Testing.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/07-06-2022/July-Medical-Remove-and-Replace/204102%20Whole%20Exome%20and%20Whole%20Genome.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/Jan-2023-Med-Replace/20402%20Germline%20Genetic%20Testing%20Hereditary%20BRCA.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/4-22-2021/MED%20Remove%20and%20Replace/204100%20Cardiovascular%20Risk%20Panels.pdf\n",
      "https://www.fepblue.org/-/media/FEPBlue-Sitecore-10-Media/PDFs/04-01-2023/Med%20Policy%20Remove%20and%20Replace/20436%20Assays%20of%20Genetic%20Expression.pdf\n",
      "1\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/4-22-2021/MED%20Remove%20and%20Replace/204124%20Genetic%20Testing%20for%20FLT3%20NPM1%20and%20CEBPA%20Variants.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/1-11-2021/20168%20Laboratory%20Tests%20Post%20Transplant%20and%20for%20Heart%20Failure.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/10-22-2020/20481%20Genetic%20Testing%20for%20Rett%20Syndrome.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/1-11-2021/20408%20Genetic%20Testing%20for%20Lynch%20Syndrome%20and%20Other%20Inherited%20Colon%20Cancer%20Syndromes.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/3-18-2022/Medical_Policies/204124%20Genetic%20Testing%20for%20FLT3.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/3-18-2022/Medical_Policies/204124%20Genetic%20Testing%20for%20FLT3.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/7-13-2021/20488%20Genetic%20Testing%20for%20PTEN%20Hamartoma%20Tumor%20Syndrome.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/Jan-2023-Med-New/204156%20Germline%20and%20Somatic%20Biomarker%20Ovarian.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/Jan-2023-Med-Replace/40121%20%20Noninvasive%20Prenatal%20Screening%20for.pdf\n",
      "1\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/04-10-2020/204100%20Cardiovascular%20Risk%20Panels.pdf\n",
      "1\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/07-10-2020/20454%20Gene%20ExpressionBased%20Assays%20for%20Cancers%20of%20Unknown.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/7-13-2021/204102%20Whole%20Exome%20and%20Whole%20Genome%20Sequencing%20for%20Diagnosis%20of%20Genetic%20Disorders.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/10-22-2020/204119%20Multibiomarker%20Disease%20Activity%20Blood%20Test%20for%20Rheumatoid%20Arthritis.pdf\n",
      "1\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/4-22-2021/MED%20Remove%20and%20Replace/20433%20Genetic%20and%20Protein%20Biomarkers%20for%20the%20Diagnosis%20and.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/3-18-2022/Medical_Policies/20419%20Pharmacogenomic%20and%20Metabolite%20Markers.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/1-11-2021/20408%20Genetic%20Testing%20for%20Lynch%20Syndrome%20and%20Other%20Inherited%20Colon%20Cancer%20Syndromes.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/10-20-2021/204150%20Serologic%20Genetic%20and%20Molecular%20Screening%20for%20CRC.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/Jan-2023-Med-Replace/20402%20Germline%20Genetic%20Testing%20Hereditary%20BRCA.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/10-12-2022/Med%20Remove%20and%20Replace/20438%20Cytochrome%20P450%20Genotype.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/3-18-2022/Medical_Policies/204124%20Genetic%20Testing%20for%20FLT3.pdf\n",
      "1\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/3-18-2022/Medical_Policies/204131%20Pharmacogenetic%20Testing%20for%20Pain.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/10-12-2022/Med%20Remove%20and%20Replace/20482%20Genetic%20Testing%20for%20Inherited%20Thrombophilia.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/1-11-2021/20485%20BCRABL1%20Testing%20in%20Chronic%20Myelogenous%20Leukemia%20and%20Acute%20Lymphoblastic%20Leukemia.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/07-10-2020/20499%20Genetic%20Testing%20for%20Hereditary%20Pancreatitis.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/10-20-2021/204142%20Molecular%20Testing%20in%20the%20Management%20of%20Pulmonary%20Nodules.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/Jan-2023-Med-Replace/20459%20Genetic%20Testing%20for%20Development.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/07-10-2020/20488%20Genetic%20Testing%20for%20PTEN%20Hamartoma%20Tumor%20Syndrome.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/10-20-2021/204146%20Gene%20Expression%20Profiling%20for%20Cutaneous%20Melanoma.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/Jan-2023-Med-Replace/20408%20Genetic%20Testing%20for%20Lynch%20Syndrome.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/4-22-2021/MED%20Remove%20and%20Replace/20443%20Genetic%20Testing%20for%20Cardiac%20Ion%20Channelopathies.pdf\n",
      "1\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/1-6-2022/20408%20Genetic%20Testing%20for%20Lynch%20Syndrome.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/1-11-2021/20402%20Genetic%20Testing%20for%20BRCA1%20and%20BRCA2%20for%20Hereditary%20Breast%20or%20Ovarian%20Cancer%20Syndrome%20and%20Other.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/1-11-2021/20408%20Genetic%20Testing%20for%20Lynch%20Syndrome%20and%20Other%20Inherited%20Colon%20Cancer%20Syndromes.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/07-10-2020/20486%20Genetic%20Testing%20for%20Duchenne%20and%20Becker%20Muscular%20Dystrophy.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/Jan-2023-Med-Replace/20402%20Germline%20Genetic%20Testing%20Hereditary%20BRCA.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/1-11-2021/20408%20Genetic%20Testing%20for%20Lynch%20Syndrome%20and%20Other%20Inherited%20Colon%20Cancer%20Syndromes.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/3-18-2022/Medical_Policies/20489%20Genetic%20Testing%20for%20the%20Diagnosis.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/07-10-2020/20499%20Genetic%20Testing%20for%20Hereditary%20Pancreatitis.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/10-20-2021/20410%20Identification%20of%20Microorganisms%20Using%20Nucleic%20Acid%20Probes.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/7-13-2021/204148%20Germline%20Genetic%20Testing%20for%20Pancreatic%20Cancer%20Susceptibility%20Genes.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/1-11-2021/5_01_01%20Intron%20A%20Hepatitis%20B.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/4-22-2021/MED%20Remove%20and%20Replace/20443%20Genetic%20Testing%20for%20Cardiac%20Ion%20Channelopathies.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/1-11-2021/5_01_47%20Pegasys%20Pegintron%20Ribavirin.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/7-13-2021/204129%20Genetic%20Testing%20for%20Marfan%20Syndrome%20Thoracic%20Aortic%20Aneurysms%20and%20Dissections%20and%20Related%20Dis.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/7-13-2021/204109%20Genetic%20Testing%20for%20Epilepsy.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/Jan-2023-Med-Replace/20402%20Germline%20Genetic%20Testing%20Hereditary%20BRCA.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/Jan-2023-Med-Replace/20168%20Laboratory%20Tests%20Post%20Transplant.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/07-10-2020/204148%20Genetic%20Testing%20for%20Hereditary%20Pancreatic%20Cancer.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/1-6-2022/20408%20Genetic%20Testing%20for%20Lynch%20Syndrome.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/10-20-2021/204146%20Gene%20Expression%20Profiling%20for%20Cutaneous%20Melanoma.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/07-06-2022/July-Medical-Remove-and-Replace/204109%20Genetic%20Testing%20for%20Epilepsy.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/10-20-2021/20410%20Identification%20of%20Microorganisms%20Using%20Nucleic%20Acid%20Probes.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/07-10-2020/204114%20Genetic%20Testing%20for%20Dilated%20Cardiomyopathy.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/Jan-2023-Med-New/204155%20Germline%20Genetic%20Testing%20Prostate.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/3-18-2022/Medical_Policies/204124%20Genetic%20Testing%20for%20FLT3.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/12_19_19/40121%20Noninvasive%20Prenatal%20Screening%20for%20Fetal%20Aneuploidies%20and%20Microdeletions%20Using%20CellFree%20Fetal%20DNA.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/12_19_19/20168%20Laboratory%20Tests%20for%20Heart%20and%20Kidney%20Transplant%20Rejection.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/Jan-2023-Med-Replace/204141%20Circulating%20Tumor%20DNA%20and.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/07-10-2020/204148%20Genetic%20Testing%20for%20Hereditary%20Pancreatic%20Cancer.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/3-18-2022/Medical_Policies/20402%20Genetic%20Testing%20for%20BRCA.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/4-22-2021/MED%20Remove%20and%20Replace/20429%20Analysis%20of%20Human%20DNA%20in%20Stool%20Samples%20as%20a%20Technique%20for%20CRC.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/1-6-2022/20408%20Genetic%20Testing%20for%20Lynch%20Syndrome.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/1-11-2021/20408%20Genetic%20Testing%20for%20Lynch%20Syndrome%20and%20Other%20Inherited%20Colon%20Cancer%20Syndromes.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/10-20-2021/20410%20Identification%20of%20Microorganisms%20Using%20Nucleic%20Acid%20Probes.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/07-10-2020/20499%20Genetic%20Testing%20for%20Hereditary%20Pancreatitis.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/Jan-2023-Med-Replace/20402%20Germline%20Genetic%20Testing%20Hereditary%20BRCA.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/04-10-2020/521139%20Ayvakit%20avapritinib.pdf\n",
      "https://www.fepblue.org/-/media/FEPBlue-Sitecore-10-Media/PDFs/04-01-2023/Med%20Policy%20Remove%20and%20Replace/204124%20Genetic%20Testing%20for%20FLT3.pdf\n",
      "http://www.fepblue.org/en/coronavirus-updates/coronavirus-updates\n",
      "https://www.fepblue.org/-/media/PDFs/Brochures/2023/Standard-and-Basic-Option-brochure_2023.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/10-20-2021/20410%20Identification%20of%20Microorganisms%20Using%20Nucleic%20Acid%20Probes.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/1-11-2021/204117%20Genetic%20Testing%20for%20Mitochondrial%20Disorders.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/1-11-2021/40121%20Noninvasive%20Prenatal%20Screening%20for.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/Jan-2023-Med-Replace/20168%20Laboratory%20Tests%20Post%20Transplant.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/4-22-2021/MED%20Remove%20and%20Replace/20445%20Molecular%20Analysis%20for%20Targeted%20Therapy%20or%20Immunotherapy%20of%20NSCLC.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/1-6-2022/20408%20Genetic%20Testing%20for%20Lynch%20Syndrome.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/4-22-2021/MED%20Remove%20and%20Replace/204143%20Circulating%20Tumor%20DNA%20Management%20of.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/07-10-2020/204102%20Whole%20Exome%20and%20Whole%20Genome%20Sequencing.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/3-18-2022/Medical_Policies/204124%20Genetic%20Testing%20for%20FLT3.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/10-20-2021/204110%20Genetic%20Testing%20for%20Diagnosis%20and%20Management%20of%20MH%20Conditions.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/10-12-2022/Med%20Remove%20and%20Replace/20410%20Identification%20of%20Microorganisms%20Using.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/10-20-2021/20410%20Identification%20of%20Microorganisms%20Using%20Nucleic%20Acid%20Probes.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/1-6-2022/204122%20Chromosomal%20Microarray%20Testing%20for%20Evaluation.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/1-11-2021/40121%20Noninvasive%20Prenatal%20Screening%20for.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/Jan-2023-Med-Replace/204117%20Genetic%20Testing%20for%20Mitochondrial.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/4-22-2021/MED%20Remove%20and%20Replace/20456%20Immune%20Cell%20Function%20Assay.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/1-11-2021/40121%20Noninvasive%20Prenatal%20Screening%20for.pdf\n",
      "https://www.fepblue.org/benefit-plans/medical-policies-and-utilization-management-guidelines/-/media/pdfs/medical%20policies/09-24-19/204146%20gene%20expression%20profiling%20for%20cutaneous%20melanoma.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/12_19_19/204115%20Comprehensive%20Genomic%20Profiling%20for%20Selecting%20Targeted%20Cancer%20Therapies.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/07-06-2022/July-Medical-Remove-and-Replace/20499%20Genetic%20Testing%20for%20Hereditary.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/Jan-2023-Med-Replace/204117%20Genetic%20Testing%20for%20Mitochondrial.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/1-6-2022/20408%20Genetic%20Testing%20for%20Lynch%20Syndrome.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/1-6-2022/20408%20Genetic%20Testing%20for%20Lynch%20Syndrome.pdf\n",
      "https://www.fepblue.org/-/media/PDFs/Medical%20Policies/10-12-2022/Med%20Remove%20and%20Replace/20410%20Identification%20of%20Microorganisms%20Using.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.219.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.237.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.237.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.237.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.237.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.237.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.237.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.237.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CG%20Oncology%20Algorithmic%20Testing.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.237.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.237.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.241.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.237.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.216.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.237.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.235.pdf\n",
      "https://www.healthnet.com/content/healthnet/en_us/providers/working-with-hn/medical_policies.html\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.241.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.218.pdf\n",
      "https://www.healthnet.com/provcom/pdf/40925.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.225.pdf\n",
      "https://www.healthnet.com/static/general/unprotected/pdfs/national/policies/CardiacRiskAssessmentLaboratoryTests.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.237.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.237.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.181.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.218.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.222.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.225.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.237.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.225.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.241.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.225.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.226.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.222.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.222.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/pharmacy-policies/CP.PHAR.547.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.241.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.225.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.238.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.237.pdf\n",
      "https://www.healthnet.com/static/general/unprotected/pdfs/national/policies/NonmyeloablativeallogeneicSCT.pdf\n",
      "https://www.healthnet.com/content/dam/centene/healthnet/pdfs/medicare/groups/egwp-pa-core-2022.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.225.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.225.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.237.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.225.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.237.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.237.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.241.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.216.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.221.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.225.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.237.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.225.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.225.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.216.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.215.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.181.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.125_06.21.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.225.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.225.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.227.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.218.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.225.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.235.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.235.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.241.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.237.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.222.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.225.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.241.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.237.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.232.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.241.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.232.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.225.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.84.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.218.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.224.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.225.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.220.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.234.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.232.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.241.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.232.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.230.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.225.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.241.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.239.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/pharmacy-policies/CP.PHAR.65.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.225.pdf\n",
      "https://www.healthnet.com/static/employer/unprotected/pdfs/ca/sbg/welcome_guide.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.222.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.232.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.225.pdf\n",
      "https://www.healthnet.com/content/healthnet/en_us/providers/working-with-hn/medical_policies.html\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.225.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.239.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.229.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.234.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CG_Non-Invasive_Prenatal_Screening.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.219.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.181.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.222.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.181.pdf\n",
      "https://www.healthnet.com/content/healthnet/en_us/covid-19-updates/providers.html\n",
      "https://www.healthnet.com/content/healthnet/en_us/providers/working-with-hn/medical_policies.html\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.241.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.237.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.241.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.235.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.227.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.225.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.216.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.241.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.222.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.237.pdf\n",
      "https://www.healthnet.com/content/healthnet/en_us/providers/working-with-hn/medical_policies.html\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.215.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.224.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.237.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.241.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.234.pdf\n",
      "https://www.healthnet.com/static/general/unprotected/html/national/pa_guidelines/sovaldi_natl.html\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.237.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.241.pdf\n",
      "https://www.healthnet.com/static/general/unprotected/html/national/pa_guidelines/2353.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.84.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.225.pdf\n",
      "https://www.healthnet.com/provcom/pdf/40925.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.234.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.227.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.225.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.239.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.237.pdf\n",
      "https://www.healthnet.com/content/healthnet/en_us/providers/working-with-hn/medical_policies.html\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.225.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.84.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.222.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.241.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.97.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.225.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.241.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.241.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.209.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.219.pdf\n",
      "https://www.healthnet.com/content/dam/centene/policies/clinical-policies/CP.MP.225.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/medicaid-comm-plan/genetic-testing-neuromuscular-disorders-cs.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/genetic-testing-hereditary-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/cell-free-fetal-dna-testing.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/genetic-testing-hereditary-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/genetic-testing-hereditary-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/genetic-testing-hereditary-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/medadv-guidelines/m/molecular-pathology-diagnostics-genetic-testing.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/carrier-testing-for-genetic-diseases.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/pharmacogenetic-testing.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/genetic-testing-hereditary-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/genetic-testing-hereditary-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/carrier-testing-for-genetic-diseases.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/genetic-testing-hereditary-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/attachments/medadv/molecular-pathology-diagnostics-genetic-testing-cpt-hcpcs-codes.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/genetic-testing-hereditary-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/genetic-testing-hereditary-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/molecular-oncology-testing-for-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/medadv-guidelines/p/pharmacogenomics-testing.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/genetic-testing-hereditary-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/medadv-guidelines/m/molecular-pathology-procedures-hla-typing.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/attachments/medadv/molecular-pathology-diagnostics-genetic-testing-cpt-hcpcs-codes.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/molecular-oncology-testing-for-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/genetic-testing-hereditary-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/medadv-guidelines/m/molecular-pathology-procedures-hla-typing.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/medadv-guidelines/m/molecular-diagnostic-infectious-disease-testing.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/medadv-guidelines/m/molecular-path-genetic-test-unlisted-codes.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/carrier-testing-for-genetic-diseases.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/prior-auth/radiology/MEDADV-Radiology-Cardiology-Guidelines-06-22-2022.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/medicaid-comm-plan-reimbursement/UHCCP-Respiratory-Viral-Panel-Testing-Policy-Professional-and-Facility.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/whole-exome-and-whole-genome-sequencing.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/genetic-testing-hereditary-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/genetic-testing-neuromuscular-disorders.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/genetic-testing-neuromuscular-disorders.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-reimbursement/COMM-Laboratory-Services-Respiratory-Viral-Panel-Testing-Policy-Facility.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/genetic-testing-hereditary-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/medadv-guidelines/m/molecular-path-genetic-test-unlisted-codes.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/genetic-testing-hereditary-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/genetic-testing-hereditary-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/clinical-guidelines/transplant-review-guidelines-hematopoietic-stem-cell-transplantation.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/genetic-testing-hereditary-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/attachments/medadv/molecular-pathology-diagnostics-genetic-testing-cpt-hcpcs-codes.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/whole-exome-and-whole-genome-sequencing.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/molecular-oncology-testing-for-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/medadv-coverage-sum/genetic-testing.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/attachments/medadv/molecular-pathology-diagnostics-genetic-testing-cpt-hcpcs-codes.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/molecular-oncology-testing-for-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/molecular-oncology-testing-for-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/molecular-oncology-testing-for-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/medadv-guidelines/m/molecular-pathology-diagnostics-genetic-testing.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/medadv-coverage-sum/genetic-testing.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/molecular-oncology-testing-for-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/molecular-oncology-testing-for-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/genetic-testing-hereditary-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/medadv-guidelines/t/tier-2-molecular-pathology-procedures.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/molecular-oncology-testing-for-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/molecular-oncology-testing-for-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/molecular-oncology-testing-for-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/molecular-oncology-testing-for-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/medadv-guidelines/m/molecular-pathology-procedures-hla-typing.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/molecular-oncology-testing-for-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/medadv-guidelines/m/molecular-path-genetic-test-unlisted-codes.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/molecular-oncology-testing-for-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/medadv-guidelines/g/genetic-testing-cardiovascular-disease.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/molecular-oncology-testing-for-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/clinical-guidelines/transplant-review-guidelines-hematopoietic-stem-cell-transplantation.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/index/mapg/molecular-pathology-diagnostics-genetic-testing-01122022.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/prior-auth/pa-requirements/commercial/UHC-Commercial-Advance-Notification-Prior-Authorization-Requirements-11-1-2021.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/medadv-guidelines/p/pharmacogenomics-testing.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/cell-free-fetal-dna-testing.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/attachments/medadv/molecular-pathology-diagnostics-genetic-testing-cpt-hcpcs-codes.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/molecular-oncology-testing-for-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/molecular-oncology-testing-for-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/medadv-coverage-sum/genetic-testing.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/pharmacogenetic-testing.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/signaturevalue-mmg/genetic-testing-cardiac-disease-sv.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/orencia-abatacept-injection-intravenous-infusion.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/genetic-testing-cardiac-disease.pdf\n",
      "https://www.uhcprovider.com/en/resource-library/news/2022/tx-medicaid-genetic-testing-codes.html\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/medadv-guidelines/b/biomarkers-cardiovascular-risk-assessment.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/medadv-guidelines/m/molecular-pathology-diagnostics-genetic-testing.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/molecular-oncology-testing-for-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/molecular-oncology-testing-for-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/molecular-oncology-testing-for-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/medadv-guidelines/m/molecular-pathology-diagnostics-genetic-testing.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/molecular-oncology-testing-for-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/health-plans/uhcw-surgical-codes.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/genetic-testing-hereditary-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/attachments/medadv/molecular-pathology-diagnostics-genetic-testing-cpt-hcpcs-codes.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/genetic-testing-hereditary-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/genetic-testing-cardiac-disease.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/genetic-testing-cardiac-disease.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/molecular-oncology-testing-for-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/genetic-testing-cardiac-disease.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/genetic-testing-hereditary-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/index/commercial/preventive-care-services-10012019.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/genetic-testing-hereditary-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/chromosome-microarray-testing.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/molecular-oncology-testing-for-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/genetic-testing-neuromuscular-disorders.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/cell-free-fetal-dna-testing.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/cell-free-fetal-dna-testing.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/genetic-testing-hereditary-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/cell-free-fetal-dna-testing.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/molecular-oncology-testing-for-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/medadv-guidelines/m/molecular-diagnostic-infectious-disease-testing.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/molecular-oncology-testing-for-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-reimbursement/COMM-Laboratory-Services-Respiratory-Viral-Panel-Testing-Policy-Facility.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/gastrointestinal-pathogen-testing-infectious-diarrhea.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/molecular-oncology-testing-for-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/molecular-oncology-testing-for-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/medadv-guidelines/m/molecular-diagnostic-infectious-disease-testing.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/carrier-testing-for-genetic-diseases.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/hepatitis-screening.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/molecular-oncology-testing-for-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/genetic-testing-hereditary-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/genetic-testing-hereditary-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/molecular-oncology-testing-for-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/molecular-oncology-testing-for-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/molecular-oncology-testing-for-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/molecular-oncology-testing-for-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/molecular-oncology-testing-for-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/genetic-testing-cardiac-disease.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/medadv-guidelines/m/molecular-diagnostic-infectious-disease-testing.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/molecular-oncology-testing-for-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/medadv-guidelines/t/tier-2-molecular-pathology-procedures.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/medadv-guidelines/m/molecular-diagnostic-infectious-disease-testing.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/medicaid-comm-plan/genetic-testing-neuromuscular-disorders-cs.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/molecular-oncology-testing-for-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/genetic-testing-hereditary-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/molecular-oncology-testing-for-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/medadv-coverage-sum/genetic-testing.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/genetic-testing-hereditary-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/molecular-oncology-testing-for-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/molecular-oncology-testing-for-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/molecular-oncology-testing-for-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/medadv-guidelines/m/molecular-path-genetic-test-unlisted-codes.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/genetic-testing-hereditary-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/genetic-testing-hereditary-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/molecular-oncology-testing-for-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/genetic-testing-hereditary-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/molecular-oncology-testing-for-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/genetic-testing-hereditary-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/molecular-oncology-testing-for-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/molecular-oncology-testing-for-cancer.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/medadv-coverage-sum/genetic-testing.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/medadv-coverage-sum/genetic-testing.pdf\n",
      "https://www.uhcprovider.com/content/dam/provider/docs/public/policies/comm-medical-drug/molecular-oncology-testing-for-cancer.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2093421.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2102816.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/86868.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2041154.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2065123.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/86868.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2065123.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2047879.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/86868.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2065123.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2007398.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2017374.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/104426.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2062778.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2047879.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2017522.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2037144.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2093421.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2093421.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2001657.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2047879.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2005167.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2005615.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2041154.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2095722.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2093421.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/76833.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/76833.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2009436.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/76833.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2047879.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/77281.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/77281.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2056100.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2001657.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2028455.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2005615.pdf\n",
      "1\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2065129.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2158372.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2102816.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2158372.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2093421.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/76833.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2065129.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2017522.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2017522.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/providers/documents/duchenne-muscular-dystrophy-medical-policy.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2003575.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2158372.pdf\n",
      "1\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/76833.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2028455.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2017522.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2093421.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2027405.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2093421.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2093421.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2129915.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2093421.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2047879.pdf\n",
      "https://www.aetna.com/cpb/medical/data/100_199/0140.html\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2001657.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/76833.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2027400.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/76833.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2093421.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2009436.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2117516.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2129915.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2129915.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2027400.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2027400.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/76833.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2093421.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2093421.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/97530.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2001657.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2003175.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2117516.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2026685.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2056100.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/76833.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2003173.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2158372.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2001657.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2158372.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2037144.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2001657.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2034822.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2177085.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2058206.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2158372.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2102924.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2002479.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2093421.pdf\n",
      "1\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2027400.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2017374.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/104426.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2056247.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2158372.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2001657.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2001657.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2027400.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2001657.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2117516.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2003175.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2003175.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2017522.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2032143.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/76833.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/76833.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2036291.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2093421.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2093421.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2017522.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2001657.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2031570.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/76822.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2093421.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/76833.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2047882.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2001657.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2097726.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2001657.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/86868.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2158372.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2158372.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2093421.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2017522.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2047879.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2093421.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2017522.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2001657.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2158372.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2093421.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2093421.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2093421.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/104426.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2093421.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2026685.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2047879.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2065123.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2093421.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2002479.pdf\n",
      "https://www.bcbsm.com/amslibs/content/dam/public/mpr/mprsearch/pdf/2102816.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/gene-expression-profiling-medical-policy-emblemhealth.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/MG_Gene_Expression_Profiling_for_Breast_Cancer_Consolidated.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/MG_Gene_Expression_Profiling_for_Breast_Cancer_Consolidated.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/gene-expression-profiling-medical-policy-emblemhealth.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/gene-expression-profiling-medical-policy-emblemhealth.pdf\n",
      "https://www.emblemhealth.com/content/dam/global/pdfs/provider/reimbursement-policies/serum-tumor.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/gene-expression-profiling-medical-policy-emblemhealth.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/EH-Preventive-Services-List_Commercial-%20Effective10-01-2021.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/gene-expression-profiling-medical-policy-emblemhealth.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/gene-expression-profiling-medical-policy-emblemhealth.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/gene-expression-profiling-medical-policy-emblemhealth.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/m/MG_Medical_Necessity_Guideines_NIS.pdf\n",
      "https://www.emblemhealth.com/content/dam/global/pdfs/provider/reimbursement-policies/urinary-tumor.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/c/MG_Carrier_Screening_for_Parents.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/m/MG_Medical_Necessity_Guideines_NIS.pdf\n",
      "https://stage-en.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/MG_Genetic_Testing_Colorectal_Cancer.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/gene-expression-profiling-medical-policy-emblemhealth.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/MedTech_Database1.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/MG_Gene_Expression_Profiling_for_Breast_Cancer_Consolidated.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/m/MG_Medical_Necessity_Guideines_NIS.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/m/MG_Medical_Necessity_Guideines_NIS.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/gene-expression-profiling-medical-policy-emblemhealth.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/m/MG_Medical_Necessity_Guideines_NIS.pdf\n",
      "https://stage-en.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/MG_Genetic_Testing_Colorectal_Cancer.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/gene-expression-profiling-medical-policy-emblemhealth.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/MG_Gene_Expression_Profiling_for_Breast_Cancer_Consolidated.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/MG_Genetic_Testing_for_Hereditary_Cardiomyopathy.pdf\n",
      "1\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/gene-expression-profiling-medical-policy-emblemhealth.pdf\n",
      "https://www.emblemhealth.com/content/dam/global/pdfs/provider/reimbursement-policies/prostate-testing.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/m/MG_Medical_Necessity_Guideines_NIS.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/provider-manual/Appendix%20A-%20Pre-authorization%20List.pdf\n",
      "https://www.emblemhealth.com/content/dam/global/pdfs/provider/reimbursement-policies/urinary-tumor.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/c/MG_Clotting_Disorder_Therapy.pdf\n",
      "1\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/gene-expression-profiling-medical-policy-emblemhealth.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/MedTech_Database1.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/EH-Preventive-Services-List_Commercial-%20Effective10-01-2021.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/gene-expression-profiling-medical-policy-emblemhealth.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/gene-expression-profiling-medical-policy-emblemhealth.pdf\n",
      "https://www.emblemhealth.com/content/dam/global/pdfs/provider/reimbursement-policies/Coronavirus-Testing-Outpatient-Setting.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/MG_Gene_Expression_Profiling_for_Breast_Cancer_Consolidated.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/gene-expression-profiling-medical-policy-emblemhealth.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/gene-expression-profiling-medical-policy-emblemhealth.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/EH-Preventive-Services-List_Commercial-%20Effective10-01-2021.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/MG_Genetic_Testing_for_Hereditary_Cardiomyopathy.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/MedTech_Database1.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/MedTech_Database1.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/MG_Genetic_Testing_for_Hereditary_Cardiomyopathy.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/gene-expression-profiling-medical-policy-emblemhealth.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/m/MG_MYvantage_Hereditary_Comprehensive_Cancer_Panel.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/MG_Genetic_Testing_for_Hereditary_Pancreatitis.pdf\n",
      "1\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/provider-manual/Appendix%20A-%20Pre-authorization%20List.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/EH-Preventive-Services-List_Commercial-%20Effective10-01-2021.pdf\n",
      "https://www.connecticare.com/content/dam/global/pdfs/provider/reimbursement-policies/serum-testing.pdf\n",
      "https://www.emblemhealth.com/content/dam/global/pdfs/provider/reimbursement-policies/pathogen-panel.pdf\n",
      "https://www.emblemhealth.com/content/dam/global/pdfs/provider/reimbursement-policies/vectra-da.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/MedTech_Database1.pdf\n",
      "1\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/gene-expression-profiling-medical-policy-emblemhealth.pdf\n",
      "1\n",
      "https://stage-en.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/MG_Genetic_Testing_Colorectal_Cancer.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/EmblemHealth_Preauthorization_List.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/EmblemHealth_Preauthorization_List.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/MedTech_Database1.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/m/MG_Medical_Necessity_Guideines_NIS.pdf\n",
      "https://stage-en.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/MG_Genetic_Testing_Colorectal_Cancer.pdf\n",
      "https://www.emblemhealth.com/content/dam/global/pdfs/provider/reimbursement-policies/Coronavirus-Testing-Outpatient-Setting.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/c/MG_Carrier_Screening_for_Parents.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/MedTech_Database1.pdf\n",
      "1\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/gene-expression-profiling-medical-policy-emblemhealth.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/MG_Genetic_Testing_Colorectal_Cancer.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/MG_Genetic_Testing_Colorectal_Cancer.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/c/MG_Carrier_Screening_for_Parents.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/provider-manual/Appendix%20A-%20Pre-authorization%20List.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/EH-Preventive-Services-List_Commercial-%20Effective10-01-2021.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/b/MG_BRAF_Mutation_Analysis.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/EmblemHealth_Preauthorization_List.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/MedTech_Database1.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/MedTech_Database1.pdf\n",
      "https://stage-en.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/MG_Genetic_Testing_Colorectal_Cancer.pdf\n",
      "1\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/MedTech_Database1.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/EmblemHealth_Preauthorization_List.pdf\n",
      "1\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/c/MG_Carrier_Screening_for_Parents.pdf\n",
      "https://stage-en.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/MG_Genetic_Testing_Colorectal_Cancer.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/EmblemHealth_Preauthorization_List.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/a/analysis-of-KRAS-status-medical-policy-emblemhealth.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/EmblemHealth_Preauthorization_List.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/gene-expression-profiling-medical-policy-emblemhealth.pdf\n",
      "1\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/EmblemHealth_Preauthorization_List.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/c/MG_Chromosomal_Microarray_Analysis.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/MedTech_Database1.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/EmblemHealth_Preauthorization_List.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/gene-expression-profiling-medical-policy-emblemhealth.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/MedTech_Database1.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/EmblemHealth_Preauthorization_List.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/gene-expression-profiling-medical-policy-emblemhealth.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/EmblemHealth_Preauthorization_List.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/gene-expression-profiling-medical-policy-emblemhealth.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/EmblemHealth_Preauthorization_List.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/MedTech_Database1.pdf\n",
      "https://stage-en.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/MG_Genetic_Testing_Colorectal_Cancer.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/MedTech_Database1.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/c/MG_Chromosomal_Microarray_Analysis.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/n/MG_Noninvasive_Prenatal_Testing_NIPT.pdf\n",
      "https://stage-en.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/MG_Genetic_Testing_Colorectal_Cancer.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/c/MG_Carrier_Screening_for_Parents.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/EmblemHealth_Preauthorization_List.pdf\n",
      "https://www.emblemhealth.com/content/dam/global/pdfs/provider/reimbursement-policies/identification-nucleic-acid.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/n/MG_Noninvasive_Prenatal_Testing_NIPT.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/gene-expression-profiling-medical-policy-emblemhealth.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/n/MG_Noninvasive_Prenatal_Testing_NIPT.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/MedTech_Database1.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/gene-expression-profiling-medical-policy-emblemhealth.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/provider-manual/Appendix%20A-%20Pre-authorization%20List.pdf\n",
      "https://www.emblemhealth.com/content/dam/global/pdfs/provider/reimbursement-policies/Coronavirus-Testing-Outpatient-Setting.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/gene-expression-profiling-medical-policy-emblemhealth.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/m/MG_Medical_Necessity_Guideines_NIS.pdf\n",
      "https://www.emblemhealth.com/content/dam/global/pdfs/provider/reimbursement-policies/vaginitis.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/gene-expression-profiling-medical-policy-emblemhealth.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/m/MG_Medical_Necessity_Guideines_NIS.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/EH-Preventive-Services-List_Commercial-%20Effective10-01-2021.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/EmblemHealth_Preauthorization_List.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/EmblemHealth_Preauthorization_List.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/EmblemHealth_Preauthorization_List.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/gene-expression-profiling-medical-policy-emblemhealth.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/MedTech_Database1.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/c/MG_Carrier_Screening_for_Parents.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/EH-Preventive-Services-List_Commercial-%20Effective10-01-2021.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/m/MG_Medical_Necessity_Guideines_NIS.pdf\n",
      "https://www.emblemhealth.com/content/dam/global/pdfs/provider/reimbursement-policies/pathogen-panel.pdf\n",
      "https://stage-en.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/MG_Genetic_Testing_Colorectal_Cancer.pdf\n",
      "https://www.emblemhealth.com/content/dam/global/pdfs/provider/reimbursement-policies/serum-tumor.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/EmblemHealth_Preauthorization_List.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/gene-expression-profiling-medical-policy-emblemhealth.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/a/analysis-of-KRAS-status-medical-policy-emblemhealth.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/gene-expression-profiling-medical-policy-emblemhealth.pdf\n",
      "https://stage-en.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/MG_Genetic_Testing_Colorectal_Cancer.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/gene-expression-profiling-medical-policy-emblemhealth.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/medical_policies/g/gene-expression-profiling-medical-policy-emblemhealth.pdf\n",
      "https://www.emblemhealth.com/content/dam/emblemhealth/pdfs/provider/MedTech_Database1.pdf\n",
      "https://www.emblemhealth.com/content/dam/global/pdfs/provider/reimbursement-policies/serum-tumor.pdf\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/wgs-thpp\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/genetic-mol-testing\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/genetic-mol-testing\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/genetic-mol-testing\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/breast-cancer-index\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/genetic-mol-testing\n",
      "https://tuftshealthplan.com/documents/providers/general/carelon-thpp-cpt-code-list\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/genetic-mol-testing\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/genetic-mol-testing\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/noncovered-investigational-services-me\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/general/carelon-thpp-cpt-code-list\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/noncovered-investigational-services-me\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/noncovered-investigational-services-me\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/general/carelon-thpp-cpt-code-list\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/noncovered-investigational-services-me\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/noncovered-investigational-services-me\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/noncovered-investigational-services-me\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/genetic-mol-testing\n",
      "https://tuftshealthplan.com/documents/providers/lists/covid19-lab-testing-codes\n",
      "https://tuftshealthplan.com/documents/providers/general/carelon-thpp-cpt-code-list\n",
      "https://tuftshealthplan.com/documents/providers/general/carelon-thpp-cpt-code-list\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/noncovered-investigational-services-me\n",
      "https://tuftshealthplan.com/documents/providers/payment-policies/preventive-services\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/noncovered-investigational-services-me\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/general/carelon-thpp-cpt-code-list\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/human-leukocyte-antigen-genotyping\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/genetic-mol-testing\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/payment-policies/preventive-services\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/noncovered-investigational-services-me\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/general/carelon-thpp-cpt-code-list\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-test-trisomy\n",
      "https://tuftshealthplan.com/documents/providers/general/carelon-thpp-cpt-code-list\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/genetic-mol-testing\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/general/carelon-thpp-cpt-code-list\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "1\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/genetic-mol-testing\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/general/carelon-thpp-cpt-code-list\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-test-trisomy\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/genetic-mol-testing\n",
      "https://tuftshealthplan.com/documents/providers/general/carelon-thpp-cpt-code-list\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/guardant-360\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-testing-brca1-and-brca-2?lang=en-US\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/genetic-mol-testing\n",
      "https://tuftshealthplan.com/documents/providers/general/carelon-thpp-cpt-code-list\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/noncovered-investigational-services-me\n",
      "https://tuftshealthplan.com/documents/providers/general/carelon-thpp-cpt-code-list\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/genetic-mol-testing\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/noncovered-investigational-services-me\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-test-prenatal-diag-carrier-screening\n",
      "https://tuftshealthplan.com/documents/providers/lists/covid19-lab-testing-codes\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/noncovered-investigational-services-me\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/wgs-thpp\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-testing-brca1-and-brca-2?lang=en-US\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-testing-brca1-and-brca-2?lang=en-US\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/genetic-mol-testing\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/genetic-mol-testing\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-test-prenatal-diag-carrier-screening\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/hla-genotyping\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-testing-brca1-and-brca-2?lang=en-US\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/lists/covid19-lab-testing-codes\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-test-prenatal-diag-carrier-screening\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/genetic-mol-testing\n",
      "https://tuftshealthplan.com/documents/providers/general/carelon-thpp-cpt-code-list\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/hla-genotyping\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/genetic-mol-testing\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/genetic-mol-testing\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/genetic-mol-testing\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/noncovered-investigational-services-me\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/genetic-mol-testing\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/genetic-mol-testing\n",
      "https://tuftshealthplan.com/documents/providers/lists/covid19-lab-testing-codes\n",
      "https://tuftshealthplan.com/documents/providers/general/carelon-thpp-cpt-code-list\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/noncovered-investigational-services-me\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-testing-brca1-and-brca-2?lang=en-US\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/genetic-mol-testing\n",
      "https://tuftshealthplan.com/documents/providers/general/carelon-thpp-cpt-code-list\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/noncovered-investigational-services-me\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-test-prenatal-diag-carrier-screening\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/general/carelon-thpp-cpt-code-list\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/general/carelon-thpp-cpt-code-list\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/genetic-mol-testing\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/noncovered-investigational-services-me\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/noncovered-investigational-services-me\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/genetic-mol-testing\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/genetic-mol-testing\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/noncovered-investigational-services-me\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/general/carelon-thpp-cpt-code-list\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-molec-diag-test\n",
      "https://tuftshealthplan.com/documents/providers/guidelines/medical-necessity-guidelines/gen-testing-brca1-and-brca-2?lang=en-US\n",
      "https://www.avalonhcs.com/wp-content/uploads/optimahealth/G2124%20Serum%20Tumor%20Markers%20for%20Malignancies.pdf\n",
      "https://cdn1-originals.webdamdb.com/14017_139613294?cache=1683669088&response-content-disposition=inline;filename=AHS%25E2%2580%2593M2071%2520Genetic%2520Testing%2520and%2520Genetic%2520Expression%2520Profiling%2520in%2520Patients%2520with%2520Uveal%2520Melanoma_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjEzMjk0P2NhY2hlPTE2ODM2NjkwODgmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTJTI1RTIlMjU4MCUyNTkzTTIwNzElMjUyMEdlbmV0aWMlMjUyMFRlc3RpbmclMjUyMGFuZCUyNTIwR2VuZXRpYyUyNTIwRXhwcmVzc2lvbiUyNTIwUHJvZmlsaW5nJTI1MjBpbiUyNTIwUGF0aWVudHMlMjUyMHdpdGglMjUyMFV2ZWFsJTI1MjBNZWxhbm9tYV8yMDIzLTA2MDEucGRmJnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1hcHBsaWNhdGlvbi9wZGYiLCJDb25kaXRpb24iOnsiRGF0ZUxlc3NUaGFuIjp7IkFXUzpFcG9jaFRpbWUiOjIxNDc0MTQ0MDB9fX1dfQ__&Signature=pCtUJoin3Jf7ySIf4TmBmeE8O-NFDd~EGAhmR3hHoStkoAUbJd14wd2L6zWUMY2xACsv~Unzs05R7orvv3WrqRA6rTLjaousVHM5Z4y2ELk8osvEJ6Oa3WsJ~zotr5qp1A9TAOR5FX2TWKO2HTF7VaBREkhgsnRDXNxDo8qD6DlFXWFNUsA0Puc2S0qhQyOSWftPyVKol~yqEDQltRNJuBiYp9fPfdzvJe7znP3kUjmdKt01D0nuTxFv80MbWZ-gt8GjJjjSXgAZiLupmHp0~p3ZYebLO~Mftl-n-JH-G-WT1-r0-ztbtJjlVRvDhxqro24i0UYIVNUZYWRajyOcwA__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://www.avalonhcs.com/wp-content/uploads/bcbsla/M2168%20Proteogenomic%20Testing%20of%20Individuals%20with%20Cancer.pdf\n",
      "https://www.avalonhcs.com/wp-content/uploads/optimahealth/G2124%20Serum%20Tumor%20Markers%20for%20Malignancies.pdf\n",
      "https://www.avalonhcs.com/wp-content/uploads/optimahealth/G2124%20Serum%20Tumor%20Markers%20for%20Malignancies.pdf\n",
      "https://cdn1-originals.webdamdb.com/14017_139612978?cache=1683670541&response-content-disposition=inline;filename=AHS%25E2%2580%2593M2166%2520Gene%2520Expression%2520Profiling%2520and%2520Protein%2520Biomarkers%2520for%2520Prostate%2520Cancer_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjEyOTc4P2NhY2hlPTE2ODM2NzA1NDEmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTJTI1RTIlMjU4MCUyNTkzTTIxNjYlMjUyMEdlbmUlMjUyMEV4cHJlc3Npb24lMjUyMFByb2ZpbGluZyUyNTIwYW5kJTI1MjBQcm90ZWluJTI1MjBCaW9tYXJrZXJzJTI1MjBmb3IlMjUyMFByb3N0YXRlJTI1MjBDYW5jZXJfMjAyMy0wNjAxLnBkZiZyZXNwb25zZS1jb250ZW50LXR5cGU9YXBwbGljYXRpb24vcGRmIiwiQ29uZGl0aW9uIjp7IkRhdGVMZXNzVGhhbiI6eyJBV1M6RXBvY2hUaW1lIjoyMTQ3NDE0NDAwfX19XX0_&Signature=XzgveijaVpJg~UP0ji6dYUfgczjFGkgbMZWYRdRJkrrX-zO-8dbw0CADVz4G8r-uLtxmEflt5xqyQrEJ~JBrAuGI5rRNaBTe-b4iiwCbBzhA68flfVpLwCTNcnusyQcem-IomQkTtZyUVQxuv3sT2xxSL0T4rccnPUAqhWTeLYeMmuXPuHcLftkWIfPMSQ19NPup-wMgZZag4ggNrf4Wxta2cvytVs6dLX1wUj4kPb~6WsYmvE0dZ4CWhKVLE6plYXchtyDs-Jqumr6k1CE4gk2I9ZC~NGTAzG5BUxRlYbCmS8cCfCtV2PLQgTVTZa2Egiepl6bFKlkGzVRtWQBstg__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://www.avalonhcs.com/wp-content/uploads/bcbsla/M2168%20Proteogenomic%20Testing%20of%20Individuals%20with%20Cancer.pdf\n",
      "https://cdn1-originals.webdamdb.com/14017_139612978?cache=1683670541&response-content-disposition=inline;filename=AHS%25E2%2580%2593M2166%2520Gene%2520Expression%2520Profiling%2520and%2520Protein%2520Biomarkers%2520for%2520Prostate%2520Cancer_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjEyOTc4P2NhY2hlPTE2ODM2NzA1NDEmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTJTI1RTIlMjU4MCUyNTkzTTIxNjYlMjUyMEdlbmUlMjUyMEV4cHJlc3Npb24lMjUyMFByb2ZpbGluZyUyNTIwYW5kJTI1MjBQcm90ZWluJTI1MjBCaW9tYXJrZXJzJTI1MjBmb3IlMjUyMFByb3N0YXRlJTI1MjBDYW5jZXJfMjAyMy0wNjAxLnBkZiZyZXNwb25zZS1jb250ZW50LXR5cGU9YXBwbGljYXRpb24vcGRmIiwiQ29uZGl0aW9uIjp7IkRhdGVMZXNzVGhhbiI6eyJBV1M6RXBvY2hUaW1lIjoyMTQ3NDE0NDAwfX19XX0_&Signature=XzgveijaVpJg~UP0ji6dYUfgczjFGkgbMZWYRdRJkrrX-zO-8dbw0CADVz4G8r-uLtxmEflt5xqyQrEJ~JBrAuGI5rRNaBTe-b4iiwCbBzhA68flfVpLwCTNcnusyQcem-IomQkTtZyUVQxuv3sT2xxSL0T4rccnPUAqhWTeLYeMmuXPuHcLftkWIfPMSQ19NPup-wMgZZag4ggNrf4Wxta2cvytVs6dLX1wUj4kPb~6WsYmvE0dZ4CWhKVLE6plYXchtyDs-Jqumr6k1CE4gk2I9ZC~NGTAzG5BUxRlYbCmS8cCfCtV2PLQgTVTZa2Egiepl6bFKlkGzVRtWQBstg__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://cdn1-originals.webdamdb.com/14017_139612978?cache=1683670541&response-content-disposition=inline;filename=AHS%25E2%2580%2593M2166%2520Gene%2520Expression%2520Profiling%2520and%2520Protein%2520Biomarkers%2520for%2520Prostate%2520Cancer_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjEyOTc4P2NhY2hlPTE2ODM2NzA1NDEmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTJTI1RTIlMjU4MCUyNTkzTTIxNjYlMjUyMEdlbmUlMjUyMEV4cHJlc3Npb24lMjUyMFByb2ZpbGluZyUyNTIwYW5kJTI1MjBQcm90ZWluJTI1MjBCaW9tYXJrZXJzJTI1MjBmb3IlMjUyMFByb3N0YXRlJTI1MjBDYW5jZXJfMjAyMy0wNjAxLnBkZiZyZXNwb25zZS1jb250ZW50LXR5cGU9YXBwbGljYXRpb24vcGRmIiwiQ29uZGl0aW9uIjp7IkRhdGVMZXNzVGhhbiI6eyJBV1M6RXBvY2hUaW1lIjoyMTQ3NDE0NDAwfX19XX0_&Signature=XzgveijaVpJg~UP0ji6dYUfgczjFGkgbMZWYRdRJkrrX-zO-8dbw0CADVz4G8r-uLtxmEflt5xqyQrEJ~JBrAuGI5rRNaBTe-b4iiwCbBzhA68flfVpLwCTNcnusyQcem-IomQkTtZyUVQxuv3sT2xxSL0T4rccnPUAqhWTeLYeMmuXPuHcLftkWIfPMSQ19NPup-wMgZZag4ggNrf4Wxta2cvytVs6dLX1wUj4kPb~6WsYmvE0dZ4CWhKVLE6plYXchtyDs-Jqumr6k1CE4gk2I9ZC~NGTAzG5BUxRlYbCmS8cCfCtV2PLQgTVTZa2Egiepl6bFKlkGzVRtWQBstg__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://cdn1-originals.webdamdb.com/14017_139613324?cache=1683670084&response-content-disposition=inline;filename=AHS-M2108%2520Molecular%2520Markers%2520in%2520Fine%2520Needle%2520Aspirates%2520of%2520the%2520Thyroid_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjEzMzI0P2NhY2hlPTE2ODM2NzAwODQmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTLU0yMTA4JTI1MjBNb2xlY3VsYXIlMjUyME1hcmtlcnMlMjUyMGluJTI1MjBGaW5lJTI1MjBOZWVkbGUlMjUyMEFzcGlyYXRlcyUyNTIwb2YlMjUyMHRoZSUyNTIwVGh5cm9pZF8yMDIzLTA2MDEucGRmJnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1hcHBsaWNhdGlvbi9wZGYiLCJDb25kaXRpb24iOnsiRGF0ZUxlc3NUaGFuIjp7IkFXUzpFcG9jaFRpbWUiOjIxNDc0MTQ0MDB9fX1dfQ__&Signature=Vy2HjxZcEerCB3OzvdT61SRigvtIItO9PdHnfqqwtN1KnmYXvLnXc5YfdRYK80CuXGN4873HnrhIl5O2eEFaahggFRu6peFWbjZandJ0OUFPDfl5mMqHNiAeAlOoo3BAClWwKRwdspkCYDOrpTJnEJqI7-2teRgjcAnLVYgUp17KE3m7bDIQ0yj2IlbQclKiYqNPFsca8BHnl63g-Cn3zqWJVo028coTjv-czgYo8IzBQdsgHlZLcssmEtzl9XSrhizlIVYRVgze72mLdlHXyn5Sn8J8Rj9Bh0RiBgCTaIX8bl4YHwalayGYIeU1RxKRbcn2~lzfuM8amYJ9Cb6Rhw__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://cdn1-originals.webdamdb.com/14017_140263898?cache=1683670517&response-content-disposition=inline;filename=AHS%25E2%2580%2593M2160%2520Molecular%2520Testing%2520for%2520Pulmonary%2520Disease_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTQwMjYzODk4P2NhY2hlPTE2ODM2NzA1MTcmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTJTI1RTIlMjU4MCUyNTkzTTIxNjAlMjUyME1vbGVjdWxhciUyNTIwVGVzdGluZyUyNTIwZm9yJTI1MjBQdWxtb25hcnklMjUyMERpc2Vhc2VfMjAyMy0wNjAxLnBkZiZyZXNwb25zZS1jb250ZW50LXR5cGU9YXBwbGljYXRpb24vcGRmIiwiQ29uZGl0aW9uIjp7IkRhdGVMZXNzVGhhbiI6eyJBV1M6RXBvY2hUaW1lIjoyMTQ3NDE0NDAwfX19XX0_&Signature=Tq4I9k31x5jfJMNCihaomN11ccLenYpyV75-RRaxWNPArxxYyqYZcDkvXzL5dcBn9n-t30UW3T38vVp7WbyC4EcdmQaTzrnP5GvIp~Rl-2KnGm-atYwtwKbf5xsfJTS2hJaZo76rYBnjaPMtUazlb~wSbuSp4VXiYlwGvJAOOV7rRjyXSmbCInJjukr8ujWipTVxzARFlW~kTAKzP02~fCEtMR30kkg51En~7Hj-BjWtU6gk1GGi-byNG5Gatx5jRaZBt6doQlu8T5osyg5eUsbiSnQp4MvmNO122sqLLZDi-hmUuRMSY034jq5xOD3l6WpbPteyO~rT4e2jWBl6VA__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://cdn1-originals.webdamdb.com/14017_139613293?cache=1683669489&response-content-disposition=inline;filename=AHS-M2065%2520Molecular%2520Profiling%2520for%2520Cancers%2520of%2520Unknown%2520Primary%2520Origin_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjEzMjkzP2NhY2hlPTE2ODM2Njk0ODkmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTLU0yMDY1JTI1MjBNb2xlY3VsYXIlMjUyMFByb2ZpbGluZyUyNTIwZm9yJTI1MjBDYW5jZXJzJTI1MjBvZiUyNTIwVW5rbm93biUyNTIwUHJpbWFyeSUyNTIwT3JpZ2luXzIwMjMtMDYwMS5wZGYmcmVzcG9uc2UtY29udGVudC10eXBlPWFwcGxpY2F0aW9uL3BkZiIsIkNvbmRpdGlvbiI6eyJEYXRlTGVzc1RoYW4iOnsiQVdTOkVwb2NoVGltZSI6MjE0NzQxNDQwMH19fV19&Signature=dgFZ8MQdyngm1JdzNljeUxU5zG4~~D825zMXhlkeC8~dcmJlPHnkee7BlG5V8Xhm6mFM4ScO1TFFyvRg2qIKAyDQyMOO1keOWO3reWzvOcFCezVCOPd71r-1U2DNGiel5CL3wFNUrw~DiBwOCYl8iaBUxwxAu50RccrueQnuzEhvXwRqTDoxt9t7X4EvTNNmjgrqbBbuTtia9LDWg5oLaAz3hAbomXVlOtA8HHEgLWQkcjEDa~EynnWWofollchga-lbhP9iiOIrOiTrNhGk7a4TsZzThdgjB4SVzX5lZCfNs11Ha7AMDToXT67fsdYRcdDtfvpHyy-8Zhtv7-dl2Q__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://www.avalonhcs.com/wp-content/uploads/bcbsla/M2091%20Transplant%20Rejection%20Testing.pdf\n",
      "https://cdn1-originals.webdamdb.com/14017_139602789?cache=1683670497&response-content-disposition=inline;filename=AHS%25E2%2580%2593M2146%2520General%2520Genetic%2520Testing%252C%2520Somatic%2520Disorders_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjAyNzg5P2NhY2hlPTE2ODM2NzA0OTcmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTJTI1RTIlMjU4MCUyNTkzTTIxNDYlMjUyMEdlbmVyYWwlMjUyMEdlbmV0aWMlMjUyMFRlc3RpbmclMjUyQyUyNTIwU29tYXRpYyUyNTIwRGlzb3JkZXJzXzIwMjMtMDYwMS5wZGYmcmVzcG9uc2UtY29udGVudC10eXBlPWFwcGxpY2F0aW9uL3BkZiIsIkNvbmRpdGlvbiI6eyJEYXRlTGVzc1RoYW4iOnsiQVdTOkVwb2NoVGltZSI6MjE0NzQxNDQwMH19fV19&Signature=EfftKO~wr1cTDHl5ylJ7TVxOIXtsQCj6Vkapb~23NE0DgB7~PO1zzkUQDwRZ8G~G90Awiz7kd7i8BFJyCAPpo6HWE1NPwOO3HgU2qJ6-hZOmmkUNF9KDqRif0P11l0EjMnMXsOZwBdKXlozCjqzUcLptZUWzgoXLPrTwaoGjZDIHN~Goa8vMHm3FgND2k~Gzq2GJGnClXIFz0-xLQBgBFV5YBaVDsCUHY7w4k6PxdGvHHKYZ0nxDJG3~2bRc6NATCiCSfhwtapUKp3SQ53NcknrX4a65YdL2HSNWO8Niq6YRR2YRAxtbdlRKLyquhLt624WOCjdVEeVIec-Q2gEm7w__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://www.avalonhcs.com/wp-content/uploads/optimahealth/G2124%20Serum%20Tumor%20Markers%20for%20Malignancies.pdf\n",
      "https://cdn1-originals.webdamdb.com/14017_139613322?cache=1683670446&response-content-disposition=inline;filename=AHS-M2178%2520Microsatellite%2520Instability%2520and%2520Tumor%2520Mutational%2520Burden%2520Testing_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjEzMzIyP2NhY2hlPTE2ODM2NzA0NDYmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTLU0yMTc4JTI1MjBNaWNyb3NhdGVsbGl0ZSUyNTIwSW5zdGFiaWxpdHklMjUyMGFuZCUyNTIwVHVtb3IlMjUyME11dGF0aW9uYWwlMjUyMEJ1cmRlbiUyNTIwVGVzdGluZ18yMDIzLTA2MDEucGRmJnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1hcHBsaWNhdGlvbi9wZGYiLCJDb25kaXRpb24iOnsiRGF0ZUxlc3NUaGFuIjp7IkFXUzpFcG9jaFRpbWUiOjIxNDc0MTQ0MDB9fX1dfQ__&Signature=jVXe9gY04jCoM56dnykw8~ry-VtcraelaClP~MIroRHA-uHhHfbZPrfzL1U6kPI6rYxG6TwQLPUhgZKtWmiHiCH6QfKuC~vGBt7D68GxndXnp3X6UJtNpDGPi5Ny8RYC24FVhgrZFOwCexlmOh44Yp1twfVWamYaULabB-RUzule59TTmqixjRtmEZK-OWBd9NNcF-I7Prz6IfdQnKaYm4MAiWzPl0n7aNIfF4oxWyl0kUvP-0bR~0h23PJYzlQsrUoJPkG~ndMq0~u5ZykxYwJZJO-FVyD~UUomfqftFLou5zny-3VRnkZ4eHxxR2Z7WM7gXrJmkIDGnV6Ck5Kchg__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://cdn1-originals.webdamdb.com/14017_139613324?cache=1683670084&response-content-disposition=inline;filename=AHS-M2108%2520Molecular%2520Markers%2520in%2520Fine%2520Needle%2520Aspirates%2520of%2520the%2520Thyroid_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjEzMzI0P2NhY2hlPTE2ODM2NzAwODQmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTLU0yMTA4JTI1MjBNb2xlY3VsYXIlMjUyME1hcmtlcnMlMjUyMGluJTI1MjBGaW5lJTI1MjBOZWVkbGUlMjUyMEFzcGlyYXRlcyUyNTIwb2YlMjUyMHRoZSUyNTIwVGh5cm9pZF8yMDIzLTA2MDEucGRmJnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1hcHBsaWNhdGlvbi9wZGYiLCJDb25kaXRpb24iOnsiRGF0ZUxlc3NUaGFuIjp7IkFXUzpFcG9jaFRpbWUiOjIxNDc0MTQ0MDB9fX1dfQ__&Signature=Vy2HjxZcEerCB3OzvdT61SRigvtIItO9PdHnfqqwtN1KnmYXvLnXc5YfdRYK80CuXGN4873HnrhIl5O2eEFaahggFRu6peFWbjZandJ0OUFPDfl5mMqHNiAeAlOoo3BAClWwKRwdspkCYDOrpTJnEJqI7-2teRgjcAnLVYgUp17KE3m7bDIQ0yj2IlbQclKiYqNPFsca8BHnl63g-Cn3zqWJVo028coTjv-czgYo8IzBQdsgHlZLcssmEtzl9XSrhizlIVYRVgze72mLdlHXyn5Sn8J8Rj9Bh0RiBgCTaIX8bl4YHwalayGYIeU1RxKRbcn2~lzfuM8amYJ9Cb6Rhw__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://cdn1-originals.webdamdb.com/14017_139612978?cache=1683670541&response-content-disposition=inline;filename=AHS%25E2%2580%2593M2166%2520Gene%2520Expression%2520Profiling%2520and%2520Protein%2520Biomarkers%2520for%2520Prostate%2520Cancer_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjEyOTc4P2NhY2hlPTE2ODM2NzA1NDEmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTJTI1RTIlMjU4MCUyNTkzTTIxNjYlMjUyMEdlbmUlMjUyMEV4cHJlc3Npb24lMjUyMFByb2ZpbGluZyUyNTIwYW5kJTI1MjBQcm90ZWluJTI1MjBCaW9tYXJrZXJzJTI1MjBmb3IlMjUyMFByb3N0YXRlJTI1MjBDYW5jZXJfMjAyMy0wNjAxLnBkZiZyZXNwb25zZS1jb250ZW50LXR5cGU9YXBwbGljYXRpb24vcGRmIiwiQ29uZGl0aW9uIjp7IkRhdGVMZXNzVGhhbiI6eyJBV1M6RXBvY2hUaW1lIjoyMTQ3NDE0NDAwfX19XX0_&Signature=XzgveijaVpJg~UP0ji6dYUfgczjFGkgbMZWYRdRJkrrX-zO-8dbw0CADVz4G8r-uLtxmEflt5xqyQrEJ~JBrAuGI5rRNaBTe-b4iiwCbBzhA68flfVpLwCTNcnusyQcem-IomQkTtZyUVQxuv3sT2xxSL0T4rccnPUAqhWTeLYeMmuXPuHcLftkWIfPMSQ19NPup-wMgZZag4ggNrf4Wxta2cvytVs6dLX1wUj4kPb~6WsYmvE0dZ4CWhKVLE6plYXchtyDs-Jqumr6k1CE4gk2I9ZC~NGTAzG5BUxRlYbCmS8cCfCtV2PLQgTVTZa2Egiepl6bFKlkGzVRtWQBstg__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://www.avalonhcs.com/wp-content/uploads/bcbsla/M2168%20Proteogenomic%20Testing%20of%20Individuals%20with%20Cancer.pdf\n",
      "https://cdn1-originals.webdamdb.com/14017_139612976?cache=1683670271&response-content-disposition=inline;filename=AHS-M2145%2520General%2520Genetic%2520Testing%252C%2520Germline%2520Disorders_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjEyOTc2P2NhY2hlPTE2ODM2NzAyNzEmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTLU0yMTQ1JTI1MjBHZW5lcmFsJTI1MjBHZW5ldGljJTI1MjBUZXN0aW5nJTI1MkMlMjUyMEdlcm1saW5lJTI1MjBEaXNvcmRlcnNfMjAyMy0wNjAxLnBkZiZyZXNwb25zZS1jb250ZW50LXR5cGU9YXBwbGljYXRpb24vcGRmIiwiQ29uZGl0aW9uIjp7IkRhdGVMZXNzVGhhbiI6eyJBV1M6RXBvY2hUaW1lIjoyMTQ3NDE0NDAwfX19XX0_&Signature=IV06fFxT2Ug5T0yhRFUzyPvrwU1Ur4md7KpjkEKwlHh6dvxMXL7cgQlk8ya5~1cipLCfB1oIzifugQjRUPF4MWh3058eBuIWRj48NS0TT3jNkqL8bLjyZV4iobELNQatMv1U~eTS0dzVW8AH8ctznIy1IBDLgK3TJk9o0zPZjKQuQR5acSaXL9cNWP8~KitClE2dL0X6v83UV0gS3pvJ0FAvL1ROVPpH-NEsONu67nBLhFMAIneOLxsms1iKhDbSV4gcsacw5nfDrPlHiQk0xbC5noF5Vul1uuEXhQ9kGaeiL7cEP5Igfi98EwHAgbIT83jokFh5CHdJAE0nbu1FmA__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://www.avalonhcs.com/wp-content/uploads/bcbsvt/G2050%20Cardiovascular%20Disease%20Risk%20Assessment.pdf\n",
      "https://cdn1-originals.webdamdb.com/14017_140263870?cache=1683670555&response-content-disposition=inline;filename=AHS%25E2%2580%2593M2167%2520Genetic%2520Testing%2520for%2520Neurodegenerative%2520Disorders_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTQwMjYzODcwP2NhY2hlPTE2ODM2NzA1NTUmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTJTI1RTIlMjU4MCUyNTkzTTIxNjclMjUyMEdlbmV0aWMlMjUyMFRlc3RpbmclMjUyMGZvciUyNTIwTmV1cm9kZWdlbmVyYXRpdmUlMjUyMERpc29yZGVyc18yMDIzLTA2MDEucGRmJnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1hcHBsaWNhdGlvbi9wZGYiLCJDb25kaXRpb24iOnsiRGF0ZUxlc3NUaGFuIjp7IkFXUzpFcG9jaFRpbWUiOjIxNDc0MTQ0MDB9fX1dfQ__&Signature=BJBRMxlI~QwSXel1Us78ZqGSojxLYXuxSqY1GAzSmUsq6l0sw6Cq~2PxO8srjnJYcB00RqlsgB7nNK7xsr0XLL8PVKf~ImuB-1XlaqXBQfb-mtB8PkJHxY0VkUVSCqPqJiqGoMfJFhDHzm-mGsh8~flA6D0U3X0P8CneG2mGV3hC8q9j49l4TAbIwr4mbMRrJhNdL2evj8JNtiNAYL2s3DTZqp6btnxvH7-d37Y~GpIGe~WGoEIUU3RATt20SJG3h8HPFLsQ8LzTOehtyHnj~bdVw8KqZbbmxnGIec510wp9gs1b~KM-ktdo5vti1G8hUaF2fi9rOuppm3Md2IxZmQ__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://cdn1-originals.webdamdb.com/14017_139613303?cache=1683669815&response-content-disposition=inline;filename=AHS-M2084%2520Genetic%2520Testing%2520for%2520Mental%2520Health%2520Disorders_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjEzMzAzP2NhY2hlPTE2ODM2Njk4MTUmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTLU0yMDg0JTI1MjBHZW5ldGljJTI1MjBUZXN0aW5nJTI1MjBmb3IlMjUyME1lbnRhbCUyNTIwSGVhbHRoJTI1MjBEaXNvcmRlcnNfMjAyMy0wNjAxLnBkZiZyZXNwb25zZS1jb250ZW50LXR5cGU9YXBwbGljYXRpb24vcGRmIiwiQ29uZGl0aW9uIjp7IkRhdGVMZXNzVGhhbiI6eyJBV1M6RXBvY2hUaW1lIjoyMTQ3NDE0NDAwfX19XX0_&Signature=o2rvl07wx4pG8yZRzFxxg6qJdPFGHo1icuUZQXSyH1DUSwQkcMaUuWnj0FCi37RgzvDDddU0WFOvzcX-gYxGFYVk-Hsm7Jt4iZJctAS-l06Fk6Su04EFW6D~0luv-NpEfZx8pjtQtONATRcjY86ji0gAUQQzS~O0QzMNb9OAGzw-lf9QBzRlhIsqyIV1faHe2zDn648xl6yTrfi6NmcPovJgoVAkkL2Mx2QVoB99l1OQ~S5al7LY6jDa-WWJf193zznX7Ly84oPLasi9Jjyt2shVFkKrRup13BTT0r8kyvo8u4apOPopTvKA5FqTO8YSlloTLI51Uvh4G61tN4Y8tg__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://www.avalonhcs.com/wp-content/uploads/CareSource/KYDSNP/G2050%20Cardiovascular%20Disease%20Risk%20Assessment%20efd;%2009-01-2022.pdf\n",
      "https://www.avalonhcs.com/wp-content/uploads/CareSource/KYDSNP/G2023%20Pre%20Operative%20Testing%20efd;%2001-01-2022%20to%2004-30-2022.pdf\n",
      "https://www.avalonhcs.com/wp-content/uploads/optimahealth/G2124%20Serum%20Tumor%20Markers%20for%20Malignancies.pdf\n",
      "https://cdn1-originals.webdamdb.com/14017_139612976?cache=1683670271&response-content-disposition=inline;filename=AHS-M2145%2520General%2520Genetic%2520Testing%252C%2520Germline%2520Disorders_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjEyOTc2P2NhY2hlPTE2ODM2NzAyNzEmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTLU0yMTQ1JTI1MjBHZW5lcmFsJTI1MjBHZW5ldGljJTI1MjBUZXN0aW5nJTI1MkMlMjUyMEdlcm1saW5lJTI1MjBEaXNvcmRlcnNfMjAyMy0wNjAxLnBkZiZyZXNwb25zZS1jb250ZW50LXR5cGU9YXBwbGljYXRpb24vcGRmIiwiQ29uZGl0aW9uIjp7IkRhdGVMZXNzVGhhbiI6eyJBV1M6RXBvY2hUaW1lIjoyMTQ3NDE0NDAwfX19XX0_&Signature=IV06fFxT2Ug5T0yhRFUzyPvrwU1Ur4md7KpjkEKwlHh6dvxMXL7cgQlk8ya5~1cipLCfB1oIzifugQjRUPF4MWh3058eBuIWRj48NS0TT3jNkqL8bLjyZV4iobELNQatMv1U~eTS0dzVW8AH8ctznIy1IBDLgK3TJk9o0zPZjKQuQR5acSaXL9cNWP8~KitClE2dL0X6v83UV0gS3pvJ0FAvL1ROVPpH-NEsONu67nBLhFMAIneOLxsms1iKhDbSV4gcsacw5nfDrPlHiQk0xbC5noF5Vul1uuEXhQ9kGaeiL7cEP5Igfi98EwHAgbIT83jokFh5CHdJAE0nbu1FmA__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://cdn1-originals.webdamdb.com/14017_139605663?cache=1683669624&response-content-disposition=inline;filename=AHS-M2074%2520Genetic%2520Testing%2520for%2520Duchenne%252C%2520Becker%252C%2520Facioscapulohumeral%252C%2520and%2520Limb-Girdle%2520Muscular%2520Dystrophies_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjA1NjYzP2NhY2hlPTE2ODM2Njk2MjQmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTLU0yMDc0JTI1MjBHZW5ldGljJTI1MjBUZXN0aW5nJTI1MjBmb3IlMjUyMER1Y2hlbm5lJTI1MkMlMjUyMEJlY2tlciUyNTJDJTI1MjBGYWNpb3NjYXB1bG9odW1lcmFsJTI1MkMlMjUyMGFuZCUyNTIwTGltYi1HaXJkbGUlMjUyME11c2N1bGFyJTI1MjBEeXN0cm9waGllc18yMDIzLTA2MDEucGRmJnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1hcHBsaWNhdGlvbi9wZGYiLCJDb25kaXRpb24iOnsiRGF0ZUxlc3NUaGFuIjp7IkFXUzpFcG9jaFRpbWUiOjIxNDc0MTQ0MDB9fX1dfQ__&Signature=bTrnKi8QLtVd4X3kz~f6e3aw9CPdLAKRYDdZWPG3rbmnBdhtckyVHzxiAH6qFSE2imLQ70UjW69syN6G-coZT6Ux8R1zmpRe5J--~gYR~AY0JqCLxGa1rRlGXEMtR6OjYHrWnEVg39emJHuYs1WZ3O7UJkYWKqZySaenegMbcPLQWzIYueu18qFnXlX9QPnQeoAVvnbJ3V34YWXcvmCpe5CXfRH65gaWrwDvX1fqtAxWuWQvjg6QfZYz-F7eagSjsONzpubU9fJT3~UAfjKaIssWvyWhnVNOxQRK9X0jP8Qf19-db47FEFg7DYbwj6YWD50Tus44usPpW9Nv760JzQ__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://cdn1-originals.webdamdb.com/14017_140263855?cache=1683668593&response-content-disposition=inline;filename=AHS%25E2%2580%2593M2029%2520Genetic%2520Testing%2520and%2520Genetic%2520Expression%2520Profiling%2520in%2520Patients%2520with%2520Cutaneous%2520Melanoma_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTQwMjYzODU1P2NhY2hlPTE2ODM2Njg1OTMmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTJTI1RTIlMjU4MCUyNTkzTTIwMjklMjUyMEdlbmV0aWMlMjUyMFRlc3RpbmclMjUyMGFuZCUyNTIwR2VuZXRpYyUyNTIwRXhwcmVzc2lvbiUyNTIwUHJvZmlsaW5nJTI1MjBpbiUyNTIwUGF0aWVudHMlMjUyMHdpdGglMjUyMEN1dGFuZW91cyUyNTIwTWVsYW5vbWFfMjAyMy0wNjAxLnBkZiZyZXNwb25zZS1jb250ZW50LXR5cGU9YXBwbGljYXRpb24vcGRmIiwiQ29uZGl0aW9uIjp7IkRhdGVMZXNzVGhhbiI6eyJBV1M6RXBvY2hUaW1lIjoyMTQ3NDE0NDAwfX19XX0_&Signature=l3BspsvwqNee~wXIcBnV9EYBEFT2mP4~aD890l3xKvHechjaocITHdhYZGUljKyXghmoOAcsjGvOsJ1lkSCkwVh8GiMmPr8Gc9nDftVJNwDW~X0VljHkYK8lMWvfvksnKaybZqG~wSYxS1O4N9SLWvJjPtd9fFV6U5gvwNobUh0PGSP5bBCFRKNfETVs9NkevRNIPAEn4AXA8FSqFMwPeoOtFttPkfhM7s7diBWIRA56S0pqiE1rDlTS4NTe2kr4mE2NJkm3KX~Iv8LERBnQ0ypNWu~ZvYr63Dtlx10KU09oy7WqxlL1w4Kbp8wptOIR-vatyGSFKQUknE0d8Gvzug__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://www.avalonhcs.com/wp-content/uploads/CareSource/ARPASSE/G2055%20Prenatal%20Screening%20for%20Fetal%20Aneuploidy%20efd;%2001-01-2022.pdf\n",
      "https://cdn1-originals.webdamdb.com/14017_139612976?cache=1683670271&response-content-disposition=inline;filename=AHS-M2145%2520General%2520Genetic%2520Testing%252C%2520Germline%2520Disorders_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjEyOTc2P2NhY2hlPTE2ODM2NzAyNzEmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTLU0yMTQ1JTI1MjBHZW5lcmFsJTI1MjBHZW5ldGljJTI1MjBUZXN0aW5nJTI1MkMlMjUyMEdlcm1saW5lJTI1MjBEaXNvcmRlcnNfMjAyMy0wNjAxLnBkZiZyZXNwb25zZS1jb250ZW50LXR5cGU9YXBwbGljYXRpb24vcGRmIiwiQ29uZGl0aW9uIjp7IkRhdGVMZXNzVGhhbiI6eyJBV1M6RXBvY2hUaW1lIjoyMTQ3NDE0NDAwfX19XX0_&Signature=IV06fFxT2Ug5T0yhRFUzyPvrwU1Ur4md7KpjkEKwlHh6dvxMXL7cgQlk8ya5~1cipLCfB1oIzifugQjRUPF4MWh3058eBuIWRj48NS0TT3jNkqL8bLjyZV4iobELNQatMv1U~eTS0dzVW8AH8ctznIy1IBDLgK3TJk9o0zPZjKQuQR5acSaXL9cNWP8~KitClE2dL0X6v83UV0gS3pvJ0FAvL1ROVPpH-NEsONu67nBLhFMAIneOLxsms1iKhDbSV4gcsacw5nfDrPlHiQk0xbC5noF5Vul1uuEXhQ9kGaeiL7cEP5Igfi98EwHAgbIT83jokFh5CHdJAE0nbu1FmA__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://www.avalonhcs.com/wp-content/uploads/bcbsla/G2127%20Vectra%20DA%20Blood%20Test%20for%20Rheumatoid%20Arthritis.pdf\n",
      "https://www.avalonhcs.com/wp-content/uploads/CareSource/ARPASSE/M2112%20Nerve%20Fiber%20Density%20Testing%20efd;%2001-01-2022.pdf\n",
      "https://www.avalonhcs.com/wp-content/uploads/CareSource/OHMarketplace/G2036%20Hepatitis%20C%20efd;%2008-01-2022.pdf\n",
      "https://www.avalonhcs.com/wp-content/uploads/bcbsla/M2168%20Proteogenomic%20Testing%20of%20Individuals%20with%20Cancer.pdf\n",
      "https://cdn1-originals.webdamdb.com/14017_140273002?cache=1683668206&response-content-disposition=inline;filename=AHS%25E2%2580%2593M2004%2520Lynch%2520Syndrome_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTQwMjczMDAyP2NhY2hlPTE2ODM2NjgyMDYmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTJTI1RTIlMjU4MCUyNTkzTTIwMDQlMjUyMEx5bmNoJTI1MjBTeW5kcm9tZV8yMDIzLTA2MDEucGRmJnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1hcHBsaWNhdGlvbi9wZGYiLCJDb25kaXRpb24iOnsiRGF0ZUxlc3NUaGFuIjp7IkFXUzpFcG9jaFRpbWUiOjIxNDc0MTQ0MDB9fX1dfQ__&Signature=amCzCM0wkM9vnkbMjflAAhMLnFE6rJ3E4bUq2aaHRKYnOMZFv~8WpAx~YJh3tAJxmXHvMEq2-Y16KT~GOPuLjFwswGYBcDQ9OfiJf7Nq0I0M0m6MEwX8gactYavZGo52XY0V2qW9gvnH2eG1V5ukRl5fQ8jCRArupWDAi0VoKR3UOSLHKVaUYqWp6KCJrUCJnUL01pmtAdNkLMm-WDWNPFlZpJhd7QRS~i4tskcuPU0ksuaosDB5ACeMMRL1qJM~CGWIbe5GkmDrjlJWNKglHlK~NIpgEnbE52MCAKa1PiE30uUMft6c7XJzWTh~wUbbYzj2~VMRb-svVukFbwlFqA__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://cdn1-originals.webdamdb.com/14017_140273002?cache=1683668206&response-content-disposition=inline;filename=AHS%25E2%2580%2593M2004%2520Lynch%2520Syndrome_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTQwMjczMDAyP2NhY2hlPTE2ODM2NjgyMDYmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTJTI1RTIlMjU4MCUyNTkzTTIwMDQlMjUyMEx5bmNoJTI1MjBTeW5kcm9tZV8yMDIzLTA2MDEucGRmJnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1hcHBsaWNhdGlvbi9wZGYiLCJDb25kaXRpb24iOnsiRGF0ZUxlc3NUaGFuIjp7IkFXUzpFcG9jaFRpbWUiOjIxNDc0MTQ0MDB9fX1dfQ__&Signature=amCzCM0wkM9vnkbMjflAAhMLnFE6rJ3E4bUq2aaHRKYnOMZFv~8WpAx~YJh3tAJxmXHvMEq2-Y16KT~GOPuLjFwswGYBcDQ9OfiJf7Nq0I0M0m6MEwX8gactYavZGo52XY0V2qW9gvnH2eG1V5ukRl5fQ8jCRArupWDAi0VoKR3UOSLHKVaUYqWp6KCJrUCJnUL01pmtAdNkLMm-WDWNPFlZpJhd7QRS~i4tskcuPU0ksuaosDB5ACeMMRL1qJM~CGWIbe5GkmDrjlJWNKglHlK~NIpgEnbE52MCAKa1PiE30uUMft6c7XJzWTh~wUbbYzj2~VMRb-svVukFbwlFqA__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://www.avalonhcs.com/wp-content/uploads/optimahealth/G2149%20Pathogen%20Panel%20Testing.pdf\n",
      "https://www.avalonhcs.com/wp-content/uploads/ohio/G2035_Prenatal_Screening.pdf\n",
      "1\n",
      "https://cdn1-originals.webdamdb.com/14017_139613287?cache=1683668528&response-content-disposition=inline;filename=AHS-M2026%2520KRAS%2520NRAS%2520and%2520BRAF%2520Mutation%2520Analysis%2520In%2520Colorectal%2520Cancer_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjEzMjg3P2NhY2hlPTE2ODM2Njg1MjgmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTLU0yMDI2JTI1MjBLUkFTJTI1MjBOUkFTJTI1MjBhbmQlMjUyMEJSQUYlMjUyME11dGF0aW9uJTI1MjBBbmFseXNpcyUyNTIwSW4lMjUyMENvbG9yZWN0YWwlMjUyMENhbmNlcl8yMDIzLTA2MDEucGRmJnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1hcHBsaWNhdGlvbi9wZGYiLCJDb25kaXRpb24iOnsiRGF0ZUxlc3NUaGFuIjp7IkFXUzpFcG9jaFRpbWUiOjIxNDc0MTQ0MDB9fX1dfQ__&Signature=GfbHWRSsqAk73u--Pz3lQVmGSvHIBIycb79Z1wKeeXl6klR9Ch6Cdtjiol~oszxD6wGG0gg3DyScmcfP4x519hr0IzDatsJYnyAazCuJKBAtwo3ivup0mCf6YmHWV0LNU3SXnxEDIh1Ycor3H7nddI6Y3B8IRFhAzOvdkohvjp2mUImTux4t3SvWhSEbJHdtk2KAOjIHKtFgIzCs05F53BsiMPVdq5Pky7aVpndEUQFuCMv4VgMZQWoYY67OdbmrRU3s3~N80zTypPAwgVpDud128PLIEJcfu9OVvQtsVDjDpXV8cJcVjz9roKrKWE6kQ2blOYU7MpAziWiXHZOxdg__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://cdn1-originals.webdamdb.com/14017_139605667?cache=1683669471&response-content-disposition=inline;filename=AHS-M2062%2520Genetic%2520Testing%2520for%2520Acute%2520Myeloid%2520Leukemia_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjA1NjY3P2NhY2hlPTE2ODM2Njk0NzEmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTLU0yMDYyJTI1MjBHZW5ldGljJTI1MjBUZXN0aW5nJTI1MjBmb3IlMjUyMEFjdXRlJTI1MjBNeWVsb2lkJTI1MjBMZXVrZW1pYV8yMDIzLTA2MDEucGRmJnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1hcHBsaWNhdGlvbi9wZGYiLCJDb25kaXRpb24iOnsiRGF0ZUxlc3NUaGFuIjp7IkFXUzpFcG9jaFRpbWUiOjIxNDc0MTQ0MDB9fX1dfQ__&Signature=Nmb3T8vTlVLonA1RQhIOSheD666p9jPfrAYTAx4p9d~HA6Bbd38Tecean1k2oNBlDwtFBeWA5HUuxO8kfsGSGKh5D~iUyPtf~s-1qkxdXjf4bMZm-RQfMSYHOPNw~yeTANVI7DZjx9QPSi9Sk0Msr0zhsUPQ0hqX5bovD9lEndPUit6uwDhY8lPDeoBhLVHyVEFjTICesh4jLykWhM75z5lAFY6rypu-e-w0spISK~DkTpYmspewQ9TPMS1D-N1gTfmy6V9G7YYtKHOKNA0NIgmelTn7~7egoUE~B5NLLDWul65~NRAZggPKgOO5lpfcOZCxX-3By9GtWpUmZa02Rw__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://www.avalonhcs.com/wp-content/uploads/bcbsla/G2009%20Preventive%20Screening%20in%20Adults.pdf\n",
      "https://www.avalonhcs.com/wp-content/uploads/bcbsla/G2009%20Preventive%20Screening%20in%20Adults.pdf\n",
      "1\n",
      "https://cdn1-originals.webdamdb.com/14017_140273002?cache=1683668206&response-content-disposition=inline;filename=AHS%25E2%2580%2593M2004%2520Lynch%2520Syndrome_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTQwMjczMDAyP2NhY2hlPTE2ODM2NjgyMDYmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTJTI1RTIlMjU4MCUyNTkzTTIwMDQlMjUyMEx5bmNoJTI1MjBTeW5kcm9tZV8yMDIzLTA2MDEucGRmJnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1hcHBsaWNhdGlvbi9wZGYiLCJDb25kaXRpb24iOnsiRGF0ZUxlc3NUaGFuIjp7IkFXUzpFcG9jaFRpbWUiOjIxNDc0MTQ0MDB9fX1dfQ__&Signature=amCzCM0wkM9vnkbMjflAAhMLnFE6rJ3E4bUq2aaHRKYnOMZFv~8WpAx~YJh3tAJxmXHvMEq2-Y16KT~GOPuLjFwswGYBcDQ9OfiJf7Nq0I0M0m6MEwX8gactYavZGo52XY0V2qW9gvnH2eG1V5ukRl5fQ8jCRArupWDAi0VoKR3UOSLHKVaUYqWp6KCJrUCJnUL01pmtAdNkLMm-WDWNPFlZpJhd7QRS~i4tskcuPU0ksuaosDB5ACeMMRL1qJM~CGWIbe5GkmDrjlJWNKglHlK~NIpgEnbE52MCAKa1PiE30uUMft6c7XJzWTh~wUbbYzj2~VMRb-svVukFbwlFqA__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://cdn1-originals.webdamdb.com/14017_140263855?cache=1683668593&response-content-disposition=inline;filename=AHS%25E2%2580%2593M2029%2520Genetic%2520Testing%2520and%2520Genetic%2520Expression%2520Profiling%2520in%2520Patients%2520with%2520Cutaneous%2520Melanoma_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTQwMjYzODU1P2NhY2hlPTE2ODM2Njg1OTMmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTJTI1RTIlMjU4MCUyNTkzTTIwMjklMjUyMEdlbmV0aWMlMjUyMFRlc3RpbmclMjUyMGFuZCUyNTIwR2VuZXRpYyUyNTIwRXhwcmVzc2lvbiUyNTIwUHJvZmlsaW5nJTI1MjBpbiUyNTIwUGF0aWVudHMlMjUyMHdpdGglMjUyMEN1dGFuZW91cyUyNTIwTWVsYW5vbWFfMjAyMy0wNjAxLnBkZiZyZXNwb25zZS1jb250ZW50LXR5cGU9YXBwbGljYXRpb24vcGRmIiwiQ29uZGl0aW9uIjp7IkRhdGVMZXNzVGhhbiI6eyJBV1M6RXBvY2hUaW1lIjoyMTQ3NDE0NDAwfX19XX0_&Signature=l3BspsvwqNee~wXIcBnV9EYBEFT2mP4~aD890l3xKvHechjaocITHdhYZGUljKyXghmoOAcsjGvOsJ1lkSCkwVh8GiMmPr8Gc9nDftVJNwDW~X0VljHkYK8lMWvfvksnKaybZqG~wSYxS1O4N9SLWvJjPtd9fFV6U5gvwNobUh0PGSP5bBCFRKNfETVs9NkevRNIPAEn4AXA8FSqFMwPeoOtFttPkfhM7s7diBWIRA56S0pqiE1rDlTS4NTe2kr4mE2NJkm3KX~Iv8LERBnQ0ypNWu~ZvYr63Dtlx10KU09oy7WqxlL1w4Kbp8wptOIR-vatyGSFKQUknE0d8Gvzug__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://www.avalonhcs.com/wp-content/uploads/CareSource/GAMedicaid/F2019%20Flow%20Cytometry%20v3%20efd;%2002-01-2023.pdf\n",
      "https://www.avalonhcs.com/wp-content/uploads/CareSource/OHMA/G2125%20Urinary%20Tumor%20Markers%20For%20Bladder%20Cancer%20efd;%2011-01-2022.pdf\n",
      "https://cdn1-originals.webdamdb.com/14017_139612978?cache=1683670541&response-content-disposition=inline;filename=AHS%25E2%2580%2593M2166%2520Gene%2520Expression%2520Profiling%2520and%2520Protein%2520Biomarkers%2520for%2520Prostate%2520Cancer_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjEyOTc4P2NhY2hlPTE2ODM2NzA1NDEmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTJTI1RTIlMjU4MCUyNTkzTTIxNjYlMjUyMEdlbmUlMjUyMEV4cHJlc3Npb24lMjUyMFByb2ZpbGluZyUyNTIwYW5kJTI1MjBQcm90ZWluJTI1MjBCaW9tYXJrZXJzJTI1MjBmb3IlMjUyMFByb3N0YXRlJTI1MjBDYW5jZXJfMjAyMy0wNjAxLnBkZiZyZXNwb25zZS1jb250ZW50LXR5cGU9YXBwbGljYXRpb24vcGRmIiwiQ29uZGl0aW9uIjp7IkRhdGVMZXNzVGhhbiI6eyJBV1M6RXBvY2hUaW1lIjoyMTQ3NDE0NDAwfX19XX0_&Signature=XzgveijaVpJg~UP0ji6dYUfgczjFGkgbMZWYRdRJkrrX-zO-8dbw0CADVz4G8r-uLtxmEflt5xqyQrEJ~JBrAuGI5rRNaBTe-b4iiwCbBzhA68flfVpLwCTNcnusyQcem-IomQkTtZyUVQxuv3sT2xxSL0T4rccnPUAqhWTeLYeMmuXPuHcLftkWIfPMSQ19NPup-wMgZZag4ggNrf4Wxta2cvytVs6dLX1wUj4kPb~6WsYmvE0dZ4CWhKVLE6plYXchtyDs-Jqumr6k1CE4gk2I9ZC~NGTAzG5BUxRlYbCmS8cCfCtV2PLQgTVTZa2Egiepl6bFKlkGzVRtWQBstg__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://www.avalonhcs.com/wp-content/uploads/CareSource/OHMA/G2125%20Urinary%20Tumor%20Markers%20For%20Bladder%20Cancer%20efd;%2011-01-2022.pdf\n",
      "https://cdn1-originals.webdamdb.com/14017_139612977?cache=1683668559&response-content-disposition=inline;filename=AHS%25E2%2580%2593M2027%2520BCR-ABL%25201%2520Testing_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjEyOTc3P2NhY2hlPTE2ODM2Njg1NTkmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTJTI1RTIlMjU4MCUyNTkzTTIwMjclMjUyMEJDUi1BQkwlMjUyMDElMjUyMFRlc3RpbmdfMjAyMy0wNjAxLnBkZiZyZXNwb25zZS1jb250ZW50LXR5cGU9YXBwbGljYXRpb24vcGRmIiwiQ29uZGl0aW9uIjp7IkRhdGVMZXNzVGhhbiI6eyJBV1M6RXBvY2hUaW1lIjoyMTQ3NDE0NDAwfX19XX0_&Signature=diE4LHzrN-fgqgfIL1VcRiaed6UCOBHQL1saST0sesZXgo88Pi0pyNYt9syJ8MBxKUi-O5bUZECDIZJH~h4wNkQpz-vRMGlwxfSeXM7L0LITYtDJV-lyLUuGbEpnbzsaxsuU5tNkDWhn463kZ4dEdW~s1dpyZLB~E6rqEUITMI3KeXwybpE2gFbbqmWUNSOt317zUZajTHy5OHBxVh2McJ5k28xjj8a1K6plw3ptzIUFAgnofMO7swPlPPFgMIymk6YOY6vPCdtDLVLtsU7CkGX-55O5hP88a24rqfyM99iWoUdTs5ORl1hg423Z2GV19-KoLKmODhV1Yant2TfMIg__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://cdn1-originals.webdamdb.com/14017_139602776?cache=1683669513&response-content-disposition=inline;filename=AHS-M2066%2520Genetic%2520Cancer%2520Susceptibility%2520Using%2520Next%2520Generation%2520Sequencing_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjAyNzc2P2NhY2hlPTE2ODM2Njk1MTMmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTLU0yMDY2JTI1MjBHZW5ldGljJTI1MjBDYW5jZXIlMjUyMFN1c2NlcHRpYmlsaXR5JTI1MjBVc2luZyUyNTIwTmV4dCUyNTIwR2VuZXJhdGlvbiUyNTIwU2VxdWVuY2luZ18yMDIzLTA2MDEucGRmJnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1hcHBsaWNhdGlvbi9wZGYiLCJDb25kaXRpb24iOnsiRGF0ZUxlc3NUaGFuIjp7IkFXUzpFcG9jaFRpbWUiOjIxNDc0MTQ0MDB9fX1dfQ__&Signature=lhqgHWKtY3699inz4taRBf3DLGFzdVNZkDFRE9qbK6UADLgqYE~dL9eacDYgah10oImDO2MiX8e4MXbphiAd3bwe9fY5dTA50pudr3GmMnB2VbLXHDW6kZ3MM4SI6QBk0B-Lly41CDOrU~~ywnd~MxMJuPLVRXzy951gMDXJTx5ar-z-ILwW2eQobT3JRqP4LwIWABp9zEPKC1LRulSonAXZIJmXwXb4I1~GIpE~rIy818I1WmC5p0g5ki0BKh4MTw30skRQs07TZidaZnhS8Mxi3hsWvCbUhBriMmKdDBTJTJh8WN9iLCzrQ2M0BDchrEYNN7xvVshIpoBzDxgRuA__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://www.avalonhcs.com/wp-content/uploads/bcbsvt/M2176%20Testing%20for%20Autism%20Spectrum%20Disorder%20and%20Developmental%20Delay.pdf\n",
      "https://www.avalonhcs.com/wp-content/uploads/CareSource/GAMedicaid/P2018%20Immunohistochemistry%20v2%20efd;%2010-01-2022.pdf\n",
      "https://cdn1-originals.webdamdb.com/14017_140273002?cache=1683668206&response-content-disposition=inline;filename=AHS%25E2%2580%2593M2004%2520Lynch%2520Syndrome_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTQwMjczMDAyP2NhY2hlPTE2ODM2NjgyMDYmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTJTI1RTIlMjU4MCUyNTkzTTIwMDQlMjUyMEx5bmNoJTI1MjBTeW5kcm9tZV8yMDIzLTA2MDEucGRmJnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1hcHBsaWNhdGlvbi9wZGYiLCJDb25kaXRpb24iOnsiRGF0ZUxlc3NUaGFuIjp7IkFXUzpFcG9jaFRpbWUiOjIxNDc0MTQ0MDB9fX1dfQ__&Signature=amCzCM0wkM9vnkbMjflAAhMLnFE6rJ3E4bUq2aaHRKYnOMZFv~8WpAx~YJh3tAJxmXHvMEq2-Y16KT~GOPuLjFwswGYBcDQ9OfiJf7Nq0I0M0m6MEwX8gactYavZGo52XY0V2qW9gvnH2eG1V5ukRl5fQ8jCRArupWDAi0VoKR3UOSLHKVaUYqWp6KCJrUCJnUL01pmtAdNkLMm-WDWNPFlZpJhd7QRS~i4tskcuPU0ksuaosDB5ACeMMRL1qJM~CGWIbe5GkmDrjlJWNKglHlK~NIpgEnbE52MCAKa1PiE30uUMft6c7XJzWTh~wUbbYzj2~VMRb-svVukFbwlFqA__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://cdn1-originals.webdamdb.com/14017_140273002?cache=1683668206&response-content-disposition=inline;filename=AHS%25E2%2580%2593M2004%2520Lynch%2520Syndrome_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTQwMjczMDAyP2NhY2hlPTE2ODM2NjgyMDYmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTJTI1RTIlMjU4MCUyNTkzTTIwMDQlMjUyMEx5bmNoJTI1MjBTeW5kcm9tZV8yMDIzLTA2MDEucGRmJnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1hcHBsaWNhdGlvbi9wZGYiLCJDb25kaXRpb24iOnsiRGF0ZUxlc3NUaGFuIjp7IkFXUzpFcG9jaFRpbWUiOjIxNDc0MTQ0MDB9fX1dfQ__&Signature=amCzCM0wkM9vnkbMjflAAhMLnFE6rJ3E4bUq2aaHRKYnOMZFv~8WpAx~YJh3tAJxmXHvMEq2-Y16KT~GOPuLjFwswGYBcDQ9OfiJf7Nq0I0M0m6MEwX8gactYavZGo52XY0V2qW9gvnH2eG1V5ukRl5fQ8jCRArupWDAi0VoKR3UOSLHKVaUYqWp6KCJrUCJnUL01pmtAdNkLMm-WDWNPFlZpJhd7QRS~i4tskcuPU0ksuaosDB5ACeMMRL1qJM~CGWIbe5GkmDrjlJWNKglHlK~NIpgEnbE52MCAKa1PiE30uUMft6c7XJzWTh~wUbbYzj2~VMRb-svVukFbwlFqA__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://cdn1-originals.webdamdb.com/14017_139612976?cache=1683670271&response-content-disposition=inline;filename=AHS-M2145%2520General%2520Genetic%2520Testing%252C%2520Germline%2520Disorders_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjEyOTc2P2NhY2hlPTE2ODM2NzAyNzEmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTLU0yMTQ1JTI1MjBHZW5lcmFsJTI1MjBHZW5ldGljJTI1MjBUZXN0aW5nJTI1MkMlMjUyMEdlcm1saW5lJTI1MjBEaXNvcmRlcnNfMjAyMy0wNjAxLnBkZiZyZXNwb25zZS1jb250ZW50LXR5cGU9YXBwbGljYXRpb24vcGRmIiwiQ29uZGl0aW9uIjp7IkRhdGVMZXNzVGhhbiI6eyJBV1M6RXBvY2hUaW1lIjoyMTQ3NDE0NDAwfX19XX0_&Signature=IV06fFxT2Ug5T0yhRFUzyPvrwU1Ur4md7KpjkEKwlHh6dvxMXL7cgQlk8ya5~1cipLCfB1oIzifugQjRUPF4MWh3058eBuIWRj48NS0TT3jNkqL8bLjyZV4iobELNQatMv1U~eTS0dzVW8AH8ctznIy1IBDLgK3TJk9o0zPZjKQuQR5acSaXL9cNWP8~KitClE2dL0X6v83UV0gS3pvJ0FAvL1ROVPpH-NEsONu67nBLhFMAIneOLxsms1iKhDbSV4gcsacw5nfDrPlHiQk0xbC5noF5Vul1uuEXhQ9kGaeiL7cEP5Igfi98EwHAgbIT83jokFh5CHdJAE0nbu1FmA__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://www.avalonhcs.com/wp-content/uploads/bcbsla/G2124%20Serum%20Tumor%20Markers%20for%20Malignancies.pdf\n",
      "https://cdn1-originals.webdamdb.com/14017_139613289?cache=1683668042&response-content-disposition=inline;filename=AHS-M2025%2520Genetic%2520Testing%2520for%2520Inherited%2520Cardiomyopathies%2520and%2520Channelopathies_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjEzMjg5P2NhY2hlPTE2ODM2NjgwNDImcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTLU0yMDI1JTI1MjBHZW5ldGljJTI1MjBUZXN0aW5nJTI1MjBmb3IlMjUyMEluaGVyaXRlZCUyNTIwQ2FyZGlvbXlvcGF0aGllcyUyNTIwYW5kJTI1MjBDaGFubmVsb3BhdGhpZXNfMjAyMy0wNjAxLnBkZiZyZXNwb25zZS1jb250ZW50LXR5cGU9YXBwbGljYXRpb24vcGRmIiwiQ29uZGl0aW9uIjp7IkRhdGVMZXNzVGhhbiI6eyJBV1M6RXBvY2hUaW1lIjoyMTQ3NDE0NDAwfX19XX0_&Signature=k9cIhxfVDa1lOqQ1iK77k8Gi5NNPHXPvd~mw9FHBlC39VOC91pEKA0dIQ4PaO1~2OHPkZv5tRBpFW2nFQB8kO6nZsGMzQ8wKhw9sF1ynsr1cwGNZdNYhlQrNpyfcXXREN-RPHqGShYrlYKmI2YTUfEYc~ZLqP-IgfoVMy3ge542JlUW-vEPGstUOwPgOCnE~sGlDgeqQ2YA8Ak1V-06FKWOoWsdv8AOZen78vwPrutZoBXIkPt9QbWMj2SZ39Y3VZYO6WuBAvgewd7XOz2rOZqDp97hkKBU17yNFeznsitROiIOLxXlP-5GFlRBhdQ5x3BKObxwOxn-4yZUsoNQMqg__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://cdn1-originals.webdamdb.com/14017_139613289?cache=1683668042&response-content-disposition=inline;filename=AHS-M2025%2520Genetic%2520Testing%2520for%2520Inherited%2520Cardiomyopathies%2520and%2520Channelopathies_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjEzMjg5P2NhY2hlPTE2ODM2NjgwNDImcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTLU0yMDI1JTI1MjBHZW5ldGljJTI1MjBUZXN0aW5nJTI1MjBmb3IlMjUyMEluaGVyaXRlZCUyNTIwQ2FyZGlvbXlvcGF0aGllcyUyNTIwYW5kJTI1MjBDaGFubmVsb3BhdGhpZXNfMjAyMy0wNjAxLnBkZiZyZXNwb25zZS1jb250ZW50LXR5cGU9YXBwbGljYXRpb24vcGRmIiwiQ29uZGl0aW9uIjp7IkRhdGVMZXNzVGhhbiI6eyJBV1M6RXBvY2hUaW1lIjoyMTQ3NDE0NDAwfX19XX0_&Signature=k9cIhxfVDa1lOqQ1iK77k8Gi5NNPHXPvd~mw9FHBlC39VOC91pEKA0dIQ4PaO1~2OHPkZv5tRBpFW2nFQB8kO6nZsGMzQ8wKhw9sF1ynsr1cwGNZdNYhlQrNpyfcXXREN-RPHqGShYrlYKmI2YTUfEYc~ZLqP-IgfoVMy3ge542JlUW-vEPGstUOwPgOCnE~sGlDgeqQ2YA8Ak1V-06FKWOoWsdv8AOZen78vwPrutZoBXIkPt9QbWMj2SZ39Y3VZYO6WuBAvgewd7XOz2rOZqDp97hkKBU17yNFeznsitROiIOLxXlP-5GFlRBhdQ5x3BKObxwOxn-4yZUsoNQMqg__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://cdn1-originals.webdamdb.com/14017_140263828?cache=1683668191&response-content-disposition=inline;filename=AHS%25E2%2580%2593M2003%2520BRCA_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTQwMjYzODI4P2NhY2hlPTE2ODM2NjgxOTEmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTJTI1RTIlMjU4MCUyNTkzTTIwMDMlMjUyMEJSQ0FfMjAyMy0wNjAxLnBkZiZyZXNwb25zZS1jb250ZW50LXR5cGU9YXBwbGljYXRpb24vcGRmIiwiQ29uZGl0aW9uIjp7IkRhdGVMZXNzVGhhbiI6eyJBV1M6RXBvY2hUaW1lIjoyMTQ3NDE0NDAwfX19XX0_&Signature=n3PMMNTenneHMc-uX8iQt5CxY0IrvS8WW3YUkNm3T9xkSuQy~xKyzrcNc5m-zvl~9MhR7bT0~AZd3xXCIfKR-DHgxVUvPang1rs8aQOCi8pmDFXGf-4ewUtksy5i8aNrhbYW2BC-1UcH380bNmKJPSZoxfwndyDvX090dhP3SprmpfC~5-PfQ5GfLn19KalH7ioZhJuBqJkzrYHTHXKNfGvF1KMYSzJWU-x3UVm-68SNfo~k64l2fxtWTSZ4p31FI-Xi1B4cZsLgMJAY52OHcIhUWeXVmbiGjtdh6XiL~jyW1zxBITTCVWLj~1t5BMkf5xnjoiq4yDdqMjn2czYhRQ__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://www.avalonhcs.com/wp-content/uploads/bcbsla/G2050%20Cardiovascular%20Disease%20Risk%20Assessment.pdf\n",
      "https://cdn1-originals.webdamdb.com/14017_140273002?cache=1683668206&response-content-disposition=inline;filename=AHS%25E2%2580%2593M2004%2520Lynch%2520Syndrome_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTQwMjczMDAyP2NhY2hlPTE2ODM2NjgyMDYmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTJTI1RTIlMjU4MCUyNTkzTTIwMDQlMjUyMEx5bmNoJTI1MjBTeW5kcm9tZV8yMDIzLTA2MDEucGRmJnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1hcHBsaWNhdGlvbi9wZGYiLCJDb25kaXRpb24iOnsiRGF0ZUxlc3NUaGFuIjp7IkFXUzpFcG9jaFRpbWUiOjIxNDc0MTQ0MDB9fX1dfQ__&Signature=amCzCM0wkM9vnkbMjflAAhMLnFE6rJ3E4bUq2aaHRKYnOMZFv~8WpAx~YJh3tAJxmXHvMEq2-Y16KT~GOPuLjFwswGYBcDQ9OfiJf7Nq0I0M0m6MEwX8gactYavZGo52XY0V2qW9gvnH2eG1V5ukRl5fQ8jCRArupWDAi0VoKR3UOSLHKVaUYqWp6KCJrUCJnUL01pmtAdNkLMm-WDWNPFlZpJhd7QRS~i4tskcuPU0ksuaosDB5ACeMMRL1qJM~CGWIbe5GkmDrjlJWNKglHlK~NIpgEnbE52MCAKa1PiE30uUMft6c7XJzWTh~wUbbYzj2~VMRb-svVukFbwlFqA__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://www.avalonhcs.com/wp-content/uploads/CareSource/GAMarketplace/G2130%20ST2%20Assay%20for%20Chronic%20Heart%20Failure%20efd;%2008-01-2022.pdf\n",
      "1\n",
      "https://cdn1-originals.webdamdb.com/14017_139602776?cache=1683669513&response-content-disposition=inline;filename=AHS-M2066%2520Genetic%2520Cancer%2520Susceptibility%2520Using%2520Next%2520Generation%2520Sequencing_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjAyNzc2P2NhY2hlPTE2ODM2Njk1MTMmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTLU0yMDY2JTI1MjBHZW5ldGljJTI1MjBDYW5jZXIlMjUyMFN1c2NlcHRpYmlsaXR5JTI1MjBVc2luZyUyNTIwTmV4dCUyNTIwR2VuZXJhdGlvbiUyNTIwU2VxdWVuY2luZ18yMDIzLTA2MDEucGRmJnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1hcHBsaWNhdGlvbi9wZGYiLCJDb25kaXRpb24iOnsiRGF0ZUxlc3NUaGFuIjp7IkFXUzpFcG9jaFRpbWUiOjIxNDc0MTQ0MDB9fX1dfQ__&Signature=lhqgHWKtY3699inz4taRBf3DLGFzdVNZkDFRE9qbK6UADLgqYE~dL9eacDYgah10oImDO2MiX8e4MXbphiAd3bwe9fY5dTA50pudr3GmMnB2VbLXHDW6kZ3MM4SI6QBk0B-Lly41CDOrU~~ywnd~MxMJuPLVRXzy951gMDXJTx5ar-z-ILwW2eQobT3JRqP4LwIWABp9zEPKC1LRulSonAXZIJmXwXb4I1~GIpE~rIy818I1WmC5p0g5ki0BKh4MTw30skRQs07TZidaZnhS8Mxi3hsWvCbUhBriMmKdDBTJTJh8WN9iLCzrQ2M0BDchrEYNN7xvVshIpoBzDxgRuA__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://www.avalonhcs.com/wp-content/uploads/bcbsla/G2174%20Coronavirus%20Testing%20in%20the%20Outpatient%20Setting.pdf\n",
      "https://cdn1-originals.webdamdb.com/14017_139612976?cache=1683670271&response-content-disposition=inline;filename=AHS-M2145%2520General%2520Genetic%2520Testing%252C%2520Germline%2520Disorders_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjEyOTc2P2NhY2hlPTE2ODM2NzAyNzEmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTLU0yMTQ1JTI1MjBHZW5lcmFsJTI1MjBHZW5ldGljJTI1MjBUZXN0aW5nJTI1MkMlMjUyMEdlcm1saW5lJTI1MjBEaXNvcmRlcnNfMjAyMy0wNjAxLnBkZiZyZXNwb25zZS1jb250ZW50LXR5cGU9YXBwbGljYXRpb24vcGRmIiwiQ29uZGl0aW9uIjp7IkRhdGVMZXNzVGhhbiI6eyJBV1M6RXBvY2hUaW1lIjoyMTQ3NDE0NDAwfX19XX0_&Signature=IV06fFxT2Ug5T0yhRFUzyPvrwU1Ur4md7KpjkEKwlHh6dvxMXL7cgQlk8ya5~1cipLCfB1oIzifugQjRUPF4MWh3058eBuIWRj48NS0TT3jNkqL8bLjyZV4iobELNQatMv1U~eTS0dzVW8AH8ctznIy1IBDLgK3TJk9o0zPZjKQuQR5acSaXL9cNWP8~KitClE2dL0X6v83UV0gS3pvJ0FAvL1ROVPpH-NEsONu67nBLhFMAIneOLxsms1iKhDbSV4gcsacw5nfDrPlHiQk0xbC5noF5Vul1uuEXhQ9kGaeiL7cEP5Igfi98EwHAgbIT83jokFh5CHdJAE0nbu1FmA__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://www.avalonhcs.com/wp-content/uploads/CareSource/KYDSNP/G2110%20Serum%20Testing%20for%20Hepatic%20Fibrosis%20in%20the%20Evaluation%20and%20Monitoring%20of%20Chronic%20Liver%20Disease%20efd;%2008-01-2022.pdf\n",
      "1\n",
      "https://www.avalonhcs.com/wp-content/uploads/optimahealth/G2043%20Celiac%20Disease%20Testing.pdf\n",
      "https://www.avalonhcs.com/wp-content/uploads/bcbsla/M2168%20Proteogenomic%20Testing%20of%20Individuals%20with%20Cancer.pdf\n",
      "https://www.avalonhcs.com/wp-content/uploads/CareSource/ARPASSE/M2176%20Testing%20for%20Autism%20Spectrum%20Disorder%20and%20Developmental%20Delay%20efd;%2001-01-2022.pdf\n",
      "https://www.avalonhcs.com/wp-content/uploads/bcbsla/G2124%20Serum%20Tumor%20Markers%20for%20Malignancies.pdf\n",
      "https://www.avalonhcs.com/wp-content/uploads/CareSource/ARPASSE/G2055%20Prenatal%20Screening%20for%20Fetal%20Aneuploidy%20efd;%2001-01-2022.pdf\n",
      "https://www.avalonhcs.com/wp-content/uploads/bcbsla/M2168%20Proteogenomic%20Testing%20of%20Individuals%20with%20Cancer.pdf\n",
      "https://www.avalonhcs.com/wp-content/uploads/bcbsla/M2168%20Proteogenomic%20Testing%20of%20Individuals%20with%20Cancer.pdf\n",
      "https://cdn1-originals.webdamdb.com/14017_139613288?cache=1683668492&response-content-disposition=inline;filename=AHS-M2021%2520Pharmacogenetic%2520Testing_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjEzMjg4P2NhY2hlPTE2ODM2Njg0OTImcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTLU0yMDIxJTI1MjBQaGFybWFjb2dlbmV0aWMlMjUyMFRlc3RpbmdfMjAyMy0wNjAxLnBkZiZyZXNwb25zZS1jb250ZW50LXR5cGU9YXBwbGljYXRpb24vcGRmIiwiQ29uZGl0aW9uIjp7IkRhdGVMZXNzVGhhbiI6eyJBV1M6RXBvY2hUaW1lIjoyMTQ3NDE0NDAwfX19XX0_&Signature=iGP67Nqte82fhDLy80P3M8P5DzSheJKT4DXzB~ds7hamJyFv6Fb96D-PfSQUGPThqKw1S0Mr70y~IASbVbOE0wmUYWwH8sVd94IKlee2WT7dsxPwR1LQXVYmURnLwX01OWPJ5aXGJcwu2wkz~ZBXGPnf1bFWVVzLywxlMXLChWQEVvHhLYQ2F2iWPqiFE~3uH9E9FLKFfhIr-~uyhbqcUG3F7GKmS1ij4plLrPrTkc8tdegj2fLRE0dQWGkWGHloPzsTn8VUsAesTzjrlyN5dIQ1823SrbZcQhvxDz0n62x74N3a5ej25PGKC5K6NTr3olsH-U3Ng1V2SFA~OgAWHQ__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://cdn1-originals.webdamdb.com/14017_139612978?cache=1683670541&response-content-disposition=inline;filename=AHS%25E2%2580%2593M2166%2520Gene%2520Expression%2520Profiling%2520and%2520Protein%2520Biomarkers%2520for%2520Prostate%2520Cancer_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjEyOTc4P2NhY2hlPTE2ODM2NzA1NDEmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTJTI1RTIlMjU4MCUyNTkzTTIxNjYlMjUyMEdlbmUlMjUyMEV4cHJlc3Npb24lMjUyMFByb2ZpbGluZyUyNTIwYW5kJTI1MjBQcm90ZWluJTI1MjBCaW9tYXJrZXJzJTI1MjBmb3IlMjUyMFByb3N0YXRlJTI1MjBDYW5jZXJfMjAyMy0wNjAxLnBkZiZyZXNwb25zZS1jb250ZW50LXR5cGU9YXBwbGljYXRpb24vcGRmIiwiQ29uZGl0aW9uIjp7IkRhdGVMZXNzVGhhbiI6eyJBV1M6RXBvY2hUaW1lIjoyMTQ3NDE0NDAwfX19XX0_&Signature=XzgveijaVpJg~UP0ji6dYUfgczjFGkgbMZWYRdRJkrrX-zO-8dbw0CADVz4G8r-uLtxmEflt5xqyQrEJ~JBrAuGI5rRNaBTe-b4iiwCbBzhA68flfVpLwCTNcnusyQcem-IomQkTtZyUVQxuv3sT2xxSL0T4rccnPUAqhWTeLYeMmuXPuHcLftkWIfPMSQ19NPup-wMgZZag4ggNrf4Wxta2cvytVs6dLX1wUj4kPb~6WsYmvE0dZ4CWhKVLE6plYXchtyDs-Jqumr6k1CE4gk2I9ZC~NGTAzG5BUxRlYbCmS8cCfCtV2PLQgTVTZa2Egiepl6bFKlkGzVRtWQBstg__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "1\n",
      "https://cdn1-originals.webdamdb.com/14017_139612977?cache=1683668559&response-content-disposition=inline;filename=AHS%25E2%2580%2593M2027%2520BCR-ABL%25201%2520Testing_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjEyOTc3P2NhY2hlPTE2ODM2Njg1NTkmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTJTI1RTIlMjU4MCUyNTkzTTIwMjclMjUyMEJDUi1BQkwlMjUyMDElMjUyMFRlc3RpbmdfMjAyMy0wNjAxLnBkZiZyZXNwb25zZS1jb250ZW50LXR5cGU9YXBwbGljYXRpb24vcGRmIiwiQ29uZGl0aW9uIjp7IkRhdGVMZXNzVGhhbiI6eyJBV1M6RXBvY2hUaW1lIjoyMTQ3NDE0NDAwfX19XX0_&Signature=diE4LHzrN-fgqgfIL1VcRiaed6UCOBHQL1saST0sesZXgo88Pi0pyNYt9syJ8MBxKUi-O5bUZECDIZJH~h4wNkQpz-vRMGlwxfSeXM7L0LITYtDJV-lyLUuGbEpnbzsaxsuU5tNkDWhn463kZ4dEdW~s1dpyZLB~E6rqEUITMI3KeXwybpE2gFbbqmWUNSOt317zUZajTHy5OHBxVh2McJ5k28xjj8a1K6plw3ptzIUFAgnofMO7swPlPPFgMIymk6YOY6vPCdtDLVLtsU7CkGX-55O5hP88a24rqfyM99iWoUdTs5ORl1hg423Z2GV19-KoLKmODhV1Yant2TfMIg__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://cdn1-originals.webdamdb.com/14017_139613288?cache=1683668492&response-content-disposition=inline;filename=AHS-M2021%2520Pharmacogenetic%2520Testing_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjEzMjg4P2NhY2hlPTE2ODM2Njg0OTImcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTLU0yMDIxJTI1MjBQaGFybWFjb2dlbmV0aWMlMjUyMFRlc3RpbmdfMjAyMy0wNjAxLnBkZiZyZXNwb25zZS1jb250ZW50LXR5cGU9YXBwbGljYXRpb24vcGRmIiwiQ29uZGl0aW9uIjp7IkRhdGVMZXNzVGhhbiI6eyJBV1M6RXBvY2hUaW1lIjoyMTQ3NDE0NDAwfX19XX0_&Signature=iGP67Nqte82fhDLy80P3M8P5DzSheJKT4DXzB~ds7hamJyFv6Fb96D-PfSQUGPThqKw1S0Mr70y~IASbVbOE0wmUYWwH8sVd94IKlee2WT7dsxPwR1LQXVYmURnLwX01OWPJ5aXGJcwu2wkz~ZBXGPnf1bFWVVzLywxlMXLChWQEVvHhLYQ2F2iWPqiFE~3uH9E9FLKFfhIr-~uyhbqcUG3F7GKmS1ij4plLrPrTkc8tdegj2fLRE0dQWGkWGHloPzsTn8VUsAesTzjrlyN5dIQ1823SrbZcQhvxDz0n62x74N3a5ej25PGKC5K6NTr3olsH-U3Ng1V2SFA~OgAWHQ__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://cdn1-originals.webdamdb.com/14017_139613288?cache=1683668492&response-content-disposition=inline;filename=AHS-M2021%2520Pharmacogenetic%2520Testing_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjEzMjg4P2NhY2hlPTE2ODM2Njg0OTImcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTLU0yMDIxJTI1MjBQaGFybWFjb2dlbmV0aWMlMjUyMFRlc3RpbmdfMjAyMy0wNjAxLnBkZiZyZXNwb25zZS1jb250ZW50LXR5cGU9YXBwbGljYXRpb24vcGRmIiwiQ29uZGl0aW9uIjp7IkRhdGVMZXNzVGhhbiI6eyJBV1M6RXBvY2hUaW1lIjoyMTQ3NDE0NDAwfX19XX0_&Signature=iGP67Nqte82fhDLy80P3M8P5DzSheJKT4DXzB~ds7hamJyFv6Fb96D-PfSQUGPThqKw1S0Mr70y~IASbVbOE0wmUYWwH8sVd94IKlee2WT7dsxPwR1LQXVYmURnLwX01OWPJ5aXGJcwu2wkz~ZBXGPnf1bFWVVzLywxlMXLChWQEVvHhLYQ2F2iWPqiFE~3uH9E9FLKFfhIr-~uyhbqcUG3F7GKmS1ij4plLrPrTkc8tdegj2fLRE0dQWGkWGHloPzsTn8VUsAesTzjrlyN5dIQ1823SrbZcQhvxDz0n62x74N3a5ej25PGKC5K6NTr3olsH-U3Ng1V2SFA~OgAWHQ__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://cdn1-originals.webdamdb.com/14017_139605667?cache=1683669471&response-content-disposition=inline;filename=AHS-M2062%2520Genetic%2520Testing%2520for%2520Acute%2520Myeloid%2520Leukemia_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjA1NjY3P2NhY2hlPTE2ODM2Njk0NzEmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTLU0yMDYyJTI1MjBHZW5ldGljJTI1MjBUZXN0aW5nJTI1MjBmb3IlMjUyMEFjdXRlJTI1MjBNeWVsb2lkJTI1MjBMZXVrZW1pYV8yMDIzLTA2MDEucGRmJnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1hcHBsaWNhdGlvbi9wZGYiLCJDb25kaXRpb24iOnsiRGF0ZUxlc3NUaGFuIjp7IkFXUzpFcG9jaFRpbWUiOjIxNDc0MTQ0MDB9fX1dfQ__&Signature=Nmb3T8vTlVLonA1RQhIOSheD666p9jPfrAYTAx4p9d~HA6Bbd38Tecean1k2oNBlDwtFBeWA5HUuxO8kfsGSGKh5D~iUyPtf~s-1qkxdXjf4bMZm-RQfMSYHOPNw~yeTANVI7DZjx9QPSi9Sk0Msr0zhsUPQ0hqX5bovD9lEndPUit6uwDhY8lPDeoBhLVHyVEFjTICesh4jLykWhM75z5lAFY6rypu-e-w0spISK~DkTpYmspewQ9TPMS1D-N1gTfmy6V9G7YYtKHOKNA0NIgmelTn7~7egoUE~B5NLLDWul65~NRAZggPKgOO5lpfcOZCxX-3By9GtWpUmZa02Rw__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://www.avalonhcs.com/wp-content/uploads/CareSource/ARPASSE/G2055%20Prenatal%20Screening%20for%20Fetal%20Aneuploidy%20efd;%2001-01-2022.pdf\n",
      "https://cdn1-originals.webdamdb.com/14017_140273002?cache=1683668206&response-content-disposition=inline;filename=AHS%25E2%2580%2593M2004%2520Lynch%2520Syndrome_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTQwMjczMDAyP2NhY2hlPTE2ODM2NjgyMDYmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTJTI1RTIlMjU4MCUyNTkzTTIwMDQlMjUyMEx5bmNoJTI1MjBTeW5kcm9tZV8yMDIzLTA2MDEucGRmJnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1hcHBsaWNhdGlvbi9wZGYiLCJDb25kaXRpb24iOnsiRGF0ZUxlc3NUaGFuIjp7IkFXUzpFcG9jaFRpbWUiOjIxNDc0MTQ0MDB9fX1dfQ__&Signature=amCzCM0wkM9vnkbMjflAAhMLnFE6rJ3E4bUq2aaHRKYnOMZFv~8WpAx~YJh3tAJxmXHvMEq2-Y16KT~GOPuLjFwswGYBcDQ9OfiJf7Nq0I0M0m6MEwX8gactYavZGo52XY0V2qW9gvnH2eG1V5ukRl5fQ8jCRArupWDAi0VoKR3UOSLHKVaUYqWp6KCJrUCJnUL01pmtAdNkLMm-WDWNPFlZpJhd7QRS~i4tskcuPU0ksuaosDB5ACeMMRL1qJM~CGWIbe5GkmDrjlJWNKglHlK~NIpgEnbE52MCAKa1PiE30uUMft6c7XJzWTh~wUbbYzj2~VMRb-svVukFbwlFqA__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://cdn1-originals.webdamdb.com/14017_140273002?cache=1683668206&response-content-disposition=inline;filename=AHS%25E2%2580%2593M2004%2520Lynch%2520Syndrome_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTQwMjczMDAyP2NhY2hlPTE2ODM2NjgyMDYmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTJTI1RTIlMjU4MCUyNTkzTTIwMDQlMjUyMEx5bmNoJTI1MjBTeW5kcm9tZV8yMDIzLTA2MDEucGRmJnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1hcHBsaWNhdGlvbi9wZGYiLCJDb25kaXRpb24iOnsiRGF0ZUxlc3NUaGFuIjp7IkFXUzpFcG9jaFRpbWUiOjIxNDc0MTQ0MDB9fX1dfQ__&Signature=amCzCM0wkM9vnkbMjflAAhMLnFE6rJ3E4bUq2aaHRKYnOMZFv~8WpAx~YJh3tAJxmXHvMEq2-Y16KT~GOPuLjFwswGYBcDQ9OfiJf7Nq0I0M0m6MEwX8gactYavZGo52XY0V2qW9gvnH2eG1V5ukRl5fQ8jCRArupWDAi0VoKR3UOSLHKVaUYqWp6KCJrUCJnUL01pmtAdNkLMm-WDWNPFlZpJhd7QRS~i4tskcuPU0ksuaosDB5ACeMMRL1qJM~CGWIbe5GkmDrjlJWNKglHlK~NIpgEnbE52MCAKa1PiE30uUMft6c7XJzWTh~wUbbYzj2~VMRb-svVukFbwlFqA__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://www.avalonhcs.com/wp-content/uploads/ohio/M2112_Nerve_Fiber_Density_Testing.pdf\n",
      "https://cdn1-originals.webdamdb.com/14017_139613311?cache=1683669780&response-content-disposition=inline;filename=AHS-M2083%2520Genetic%2520Testing%2520for%2520Ophthalmologic%2520Conditions_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjEzMzExP2NhY2hlPTE2ODM2Njk3ODAmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTLU0yMDgzJTI1MjBHZW5ldGljJTI1MjBUZXN0aW5nJTI1MjBmb3IlMjUyME9waHRoYWxtb2xvZ2ljJTI1MjBDb25kaXRpb25zXzIwMjMtMDYwMS5wZGYmcmVzcG9uc2UtY29udGVudC10eXBlPWFwcGxpY2F0aW9uL3BkZiIsIkNvbmRpdGlvbiI6eyJEYXRlTGVzc1RoYW4iOnsiQVdTOkVwb2NoVGltZSI6MjE0NzQxNDQwMH19fV19&Signature=d-zGqX70GlZi8LQZfwRiQzzb1y6-ATgJvXOWUXb0fvPwFUhc1RC2kT~O~0eHkvSsZa8snGqMxi2eaLBadQYj20uM18MDhAGdR8ie0ZbRWYxYgqV5ew0Q1fMDjD-WaDn0fwQj-fRgXT5FElA4Gm2YdWiij4AYQM3d0yqUfe5-wcImThsI-356Yp-BzJ~Z7IxQMe8MuSO9I5~judJhobLv87zyIRZ6UAoN4RzKkf34pK05MUcab8h0Ntpli1W4VGq9kSXkz6R54b3R5MXvA-yhTB9~MNB-34-KNrB0ZDzQbzFqjmAz4xLB29Gkhvq63wKPhlHpOsRdR5i7gU-Ur1F~IQ__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://www.avalonhcs.com/wp-content/uploads/ohio/G2035_Prenatal_Screening.pdf\n",
      "https://cdn1-originals.webdamdb.com/14017_139613302?cache=1683669854&response-content-disposition=inline;filename=AHS-M2088%2520Genetic%2520Testing%2520for%2520Rett%2520Syndrome_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjEzMzAyP2NhY2hlPTE2ODM2Njk4NTQmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTLU0yMDg4JTI1MjBHZW5ldGljJTI1MjBUZXN0aW5nJTI1MjBmb3IlMjUyMFJldHQlMjUyMFN5bmRyb21lXzIwMjMtMDYwMS5wZGYmcmVzcG9uc2UtY29udGVudC10eXBlPWFwcGxpY2F0aW9uL3BkZiIsIkNvbmRpdGlvbiI6eyJEYXRlTGVzc1RoYW4iOnsiQVdTOkVwb2NoVGltZSI6MjE0NzQxNDQwMH19fV19&Signature=ElwjVUkY8xgBAWcvKaM7smCvboMo0li4IsOdy4RIxPmGs1SFaGeMh5zz4jtGo7zGTZHR831gZIDnLGVKxBDBLCovW~VVDGYaYVHh~Ehr9bNsykwL6MI2EeySm8R-hMAeGGXwyD-4kd~oBZOF3UQJDlU9jRVgg58wPomH5oqcgyg0FBztVRrYPBnK-MHtBOKUs41PwfTf1B1IDSMexAtZzJFfTI1k5MQ3gyKrJWQOYuDnVJb7~henHWRBkL6XYLL3id0Posb5sU-d7wk4MVgRwaaSsT08L~ZHljTkZyfanhH0vzt5erUF5DE-mltftMoKxbI7IlAbOWe4JByIzrEDBg__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://cdn1-originals.webdamdb.com/14017_139613288?cache=1683668492&response-content-disposition=inline;filename=AHS-M2021%2520Pharmacogenetic%2520Testing_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjEzMjg4P2NhY2hlPTE2ODM2Njg0OTImcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTLU0yMDIxJTI1MjBQaGFybWFjb2dlbmV0aWMlMjUyMFRlc3RpbmdfMjAyMy0wNjAxLnBkZiZyZXNwb25zZS1jb250ZW50LXR5cGU9YXBwbGljYXRpb24vcGRmIiwiQ29uZGl0aW9uIjp7IkRhdGVMZXNzVGhhbiI6eyJBV1M6RXBvY2hUaW1lIjoyMTQ3NDE0NDAwfX19XX0_&Signature=iGP67Nqte82fhDLy80P3M8P5DzSheJKT4DXzB~ds7hamJyFv6Fb96D-PfSQUGPThqKw1S0Mr70y~IASbVbOE0wmUYWwH8sVd94IKlee2WT7dsxPwR1LQXVYmURnLwX01OWPJ5aXGJcwu2wkz~ZBXGPnf1bFWVVzLywxlMXLChWQEVvHhLYQ2F2iWPqiFE~3uH9E9FLKFfhIr-~uyhbqcUG3F7GKmS1ij4plLrPrTkc8tdegj2fLRE0dQWGkWGHloPzsTn8VUsAesTzjrlyN5dIQ1823SrbZcQhvxDz0n62x74N3a5ej25PGKC5K6NTr3olsH-U3Ng1V2SFA~OgAWHQ__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://cdn1-originals.webdamdb.com/14017_140263828?cache=1683668191&response-content-disposition=inline;filename=AHS%25E2%2580%2593M2003%2520BRCA_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTQwMjYzODI4P2NhY2hlPTE2ODM2NjgxOTEmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTJTI1RTIlMjU4MCUyNTkzTTIwMDMlMjUyMEJSQ0FfMjAyMy0wNjAxLnBkZiZyZXNwb25zZS1jb250ZW50LXR5cGU9YXBwbGljYXRpb24vcGRmIiwiQ29uZGl0aW9uIjp7IkRhdGVMZXNzVGhhbiI6eyJBV1M6RXBvY2hUaW1lIjoyMTQ3NDE0NDAwfX19XX0_&Signature=n3PMMNTenneHMc-uX8iQt5CxY0IrvS8WW3YUkNm3T9xkSuQy~xKyzrcNc5m-zvl~9MhR7bT0~AZd3xXCIfKR-DHgxVUvPang1rs8aQOCi8pmDFXGf-4ewUtksy5i8aNrhbYW2BC-1UcH380bNmKJPSZoxfwndyDvX090dhP3SprmpfC~5-PfQ5GfLn19KalH7ioZhJuBqJkzrYHTHXKNfGvF1KMYSzJWU-x3UVm-68SNfo~k64l2fxtWTSZ4p31FI-Xi1B4cZsLgMJAY52OHcIhUWeXVmbiGjtdh6XiL~jyW1zxBITTCVWLj~1t5BMkf5xnjoiq4yDdqMjn2czYhRQ__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://cdn1-originals.webdamdb.com/14017_139613298?cache=1683668640&response-content-disposition=inline;filename=AHS-M2032%2520Whole%2520Genome%2520and%2520Whole%2520Exome%2520Sequencing_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjEzMjk4P2NhY2hlPTE2ODM2Njg2NDAmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTLU0yMDMyJTI1MjBXaG9sZSUyNTIwR2Vub21lJTI1MjBhbmQlMjUyMFdob2xlJTI1MjBFeG9tZSUyNTIwU2VxdWVuY2luZ18yMDIzLTA2MDEucGRmJnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1hcHBsaWNhdGlvbi9wZGYiLCJDb25kaXRpb24iOnsiRGF0ZUxlc3NUaGFuIjp7IkFXUzpFcG9jaFRpbWUiOjIxNDc0MTQ0MDB9fX1dfQ__&Signature=oKjta5SIVP~321lx10LTZ9j-XgWqggeAxBhvXTNm62Sj5Fm6FWXJ26iaIFCW5tsBgJ664nPHuOci5BOhxBFiXTIkhwz3VYSLII1Q3VgJUfGx3kqs0Lr0Lx1vwgiCYHxtWXzrCmJ8LJ38wb60OIDQIT~YlbWcOqRDlfVQ9BxcfpoIZIozw7x15LLhdV7Rm~UPZSFz9odIfg2QWyMcgjr4RYyr38zraR6l8KyS-N~FjCtuP47wYWbszHgY~b8oNv-ky0eQ~HmqTGr4kNkKWjPyyCJDc6i-8Se8KPA-b3ZBIxNEOBjidc6CiyOcFAwTNEuIc-bqF~iUTUn5aROUaZ~qlA__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://www.avalonhcs.com/wp-content/uploads/CareSource/GAMedicaid/P2018%20Immunohistochemistry%20v2%20efd;%2010-01-2022.pdf\n",
      "https://cdn1-originals.webdamdb.com/14017_140273002?cache=1683668206&response-content-disposition=inline;filename=AHS%25E2%2580%2593M2004%2520Lynch%2520Syndrome_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTQwMjczMDAyP2NhY2hlPTE2ODM2NjgyMDYmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTJTI1RTIlMjU4MCUyNTkzTTIwMDQlMjUyMEx5bmNoJTI1MjBTeW5kcm9tZV8yMDIzLTA2MDEucGRmJnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1hcHBsaWNhdGlvbi9wZGYiLCJDb25kaXRpb24iOnsiRGF0ZUxlc3NUaGFuIjp7IkFXUzpFcG9jaFRpbWUiOjIxNDc0MTQ0MDB9fX1dfQ__&Signature=amCzCM0wkM9vnkbMjflAAhMLnFE6rJ3E4bUq2aaHRKYnOMZFv~8WpAx~YJh3tAJxmXHvMEq2-Y16KT~GOPuLjFwswGYBcDQ9OfiJf7Nq0I0M0m6MEwX8gactYavZGo52XY0V2qW9gvnH2eG1V5ukRl5fQ8jCRArupWDAi0VoKR3UOSLHKVaUYqWp6KCJrUCJnUL01pmtAdNkLMm-WDWNPFlZpJhd7QRS~i4tskcuPU0ksuaosDB5ACeMMRL1qJM~CGWIbe5GkmDrjlJWNKglHlK~NIpgEnbE52MCAKa1PiE30uUMft6c7XJzWTh~wUbbYzj2~VMRb-svVukFbwlFqA__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://www.avalonhcs.com/wp-content/uploads/ohio/G2035_Prenatal_Screening.pdf\n",
      "https://www.avalonhcs.com/wp-content/uploads/ohio/G2035_Prenatal_Screening.pdf\n",
      "https://cdn1-originals.webdamdb.com/14017_139605667?cache=1683669471&response-content-disposition=inline;filename=AHS-M2062%2520Genetic%2520Testing%2520for%2520Acute%2520Myeloid%2520Leukemia_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjA1NjY3P2NhY2hlPTE2ODM2Njk0NzEmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTLU0yMDYyJTI1MjBHZW5ldGljJTI1MjBUZXN0aW5nJTI1MjBmb3IlMjUyMEFjdXRlJTI1MjBNeWVsb2lkJTI1MjBMZXVrZW1pYV8yMDIzLTA2MDEucGRmJnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1hcHBsaWNhdGlvbi9wZGYiLCJDb25kaXRpb24iOnsiRGF0ZUxlc3NUaGFuIjp7IkFXUzpFcG9jaFRpbWUiOjIxNDc0MTQ0MDB9fX1dfQ__&Signature=Nmb3T8vTlVLonA1RQhIOSheD666p9jPfrAYTAx4p9d~HA6Bbd38Tecean1k2oNBlDwtFBeWA5HUuxO8kfsGSGKh5D~iUyPtf~s-1qkxdXjf4bMZm-RQfMSYHOPNw~yeTANVI7DZjx9QPSi9Sk0Msr0zhsUPQ0hqX5bovD9lEndPUit6uwDhY8lPDeoBhLVHyVEFjTICesh4jLykWhM75z5lAFY6rypu-e-w0spISK~DkTpYmspewQ9TPMS1D-N1gTfmy6V9G7YYtKHOKNA0NIgmelTn7~7egoUE~B5NLLDWul65~NRAZggPKgOO5lpfcOZCxX-3By9GtWpUmZa02Rw__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://www.avalonhcs.com/wp-content/uploads/CareSource/GAMedicaid/P2018%20Immunohistochemistry%20v2%20efd;%2010-01-2022.pdf\n",
      "https://cdn1-originals.webdamdb.com/14017_140263828?cache=1683668191&response-content-disposition=inline;filename=AHS%25E2%2580%2593M2003%2520BRCA_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTQwMjYzODI4P2NhY2hlPTE2ODM2NjgxOTEmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTJTI1RTIlMjU4MCUyNTkzTTIwMDMlMjUyMEJSQ0FfMjAyMy0wNjAxLnBkZiZyZXNwb25zZS1jb250ZW50LXR5cGU9YXBwbGljYXRpb24vcGRmIiwiQ29uZGl0aW9uIjp7IkRhdGVMZXNzVGhhbiI6eyJBV1M6RXBvY2hUaW1lIjoyMTQ3NDE0NDAwfX19XX0_&Signature=n3PMMNTenneHMc-uX8iQt5CxY0IrvS8WW3YUkNm3T9xkSuQy~xKyzrcNc5m-zvl~9MhR7bT0~AZd3xXCIfKR-DHgxVUvPang1rs8aQOCi8pmDFXGf-4ewUtksy5i8aNrhbYW2BC-1UcH380bNmKJPSZoxfwndyDvX090dhP3SprmpfC~5-PfQ5GfLn19KalH7ioZhJuBqJkzrYHTHXKNfGvF1KMYSzJWU-x3UVm-68SNfo~k64l2fxtWTSZ4p31FI-Xi1B4cZsLgMJAY52OHcIhUWeXVmbiGjtdh6XiL~jyW1zxBITTCVWLj~1t5BMkf5xnjoiq4yDdqMjn2czYhRQ__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://cdn1-originals.webdamdb.com/14017_139602789?cache=1683670497&response-content-disposition=inline;filename=AHS%25E2%2580%2593M2146%2520General%2520Genetic%2520Testing%252C%2520Somatic%2520Disorders_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjAyNzg5P2NhY2hlPTE2ODM2NzA0OTcmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTJTI1RTIlMjU4MCUyNTkzTTIxNDYlMjUyMEdlbmVyYWwlMjUyMEdlbmV0aWMlMjUyMFRlc3RpbmclMjUyQyUyNTIwU29tYXRpYyUyNTIwRGlzb3JkZXJzXzIwMjMtMDYwMS5wZGYmcmVzcG9uc2UtY29udGVudC10eXBlPWFwcGxpY2F0aW9uL3BkZiIsIkNvbmRpdGlvbiI6eyJEYXRlTGVzc1RoYW4iOnsiQVdTOkVwb2NoVGltZSI6MjE0NzQxNDQwMH19fV19&Signature=EfftKO~wr1cTDHl5ylJ7TVxOIXtsQCj6Vkapb~23NE0DgB7~PO1zzkUQDwRZ8G~G90Awiz7kd7i8BFJyCAPpo6HWE1NPwOO3HgU2qJ6-hZOmmkUNF9KDqRif0P11l0EjMnMXsOZwBdKXlozCjqzUcLptZUWzgoXLPrTwaoGjZDIHN~Goa8vMHm3FgND2k~Gzq2GJGnClXIFz0-xLQBgBFV5YBaVDsCUHY7w4k6PxdGvHHKYZ0nxDJG3~2bRc6NATCiCSfhwtapUKp3SQ53NcknrX4a65YdL2HSNWO8Niq6YRR2YRAxtbdlRKLyquhLt624WOCjdVEeVIec-Q2gEm7w__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://www.avalonhcs.com/wp-content/uploads/bcbsla/M2168%20Proteogenomic%20Testing%20of%20Individuals%20with%20Cancer.pdf\n",
      "https://www.avalonhcs.com/wp-content/uploads/bcbsla/M2168%20Proteogenomic%20Testing%20of%20Individuals%20with%20Cancer.pdf\n",
      "1\n",
      "https://www.avalonhcs.com/wp-content/uploads/CareSource/OHMedicaid/G2055%20Prenatal%20Screening%20for%20Fetal%20Aneuploidy%20efd;%2011-01-2022%20to%2002-28-2023.pdf\n",
      "https://www.avalonhcs.com/wp-content/uploads/CareSource/ARPASSE/G2055%20Prenatal%20Screening%20for%20Fetal%20Aneuploidy%20efd;%2001-01-2022.pdf\n",
      "https://www.avalonhcs.com/wp-content/uploads/bcbsla/M2168%20Proteogenomic%20Testing%20of%20Individuals%20with%20Cancer.pdf\n",
      "https://cdn1-originals.webdamdb.com/14017_139613309?cache=1683669816&response-content-disposition=inline;filename=AHS-M2085%2520Genetic%2520Testing%2520of%2520Mitochondrial%2520Disorders_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjEzMzA5P2NhY2hlPTE2ODM2Njk4MTYmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTLU0yMDg1JTI1MjBHZW5ldGljJTI1MjBUZXN0aW5nJTI1MjBvZiUyNTIwTWl0b2Nob25kcmlhbCUyNTIwRGlzb3JkZXJzXzIwMjMtMDYwMS5wZGYmcmVzcG9uc2UtY29udGVudC10eXBlPWFwcGxpY2F0aW9uL3BkZiIsIkNvbmRpdGlvbiI6eyJEYXRlTGVzc1RoYW4iOnsiQVdTOkVwb2NoVGltZSI6MjE0NzQxNDQwMH19fV19&Signature=bn8GCRVWT1YAeonu5mPxVKNeIXxzLs1WyK2d4hZGUMMAVl7enFw~7pHWKoeoPgAHBXq9yMifQI~lq1wlsjXaxYQqKCeShSPZ-FLqPZeTdtYGzUhkuCNtyAs9Jxi1KIle7MpDOzKgSKhQgsA9GrDFUCAO-TzU8sck45lQo0DQv9~AMCUPzLkIRxVyT3Tz33brdZYkzwWtfYrynJyAbehq-4Bgvf7o5k~LxG-boQkf51Xzbxjav63jzlPNHTSqlp~2L8mfpjmarZJxdM57dYvEeS~VficrtecCQ29WPpMh9XAqC2F0J1zZAjLCEfQoB4eneCKr-pq3IxYDW5VBpIxv5g__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://www.avalonhcs.com/wp-content/uploads/CareSource/OHMA/G2055%20Prenatal%20Screening%20for%20Fetal%20Aneuploidy%20efd;%2011-01-2022.pdf\n",
      "https://www.avalonhcs.com/wp-content/uploads/CareSource/KYDSNP/G2149%20Pathogen%20Panel%20Testing%20efd;%2009-01-2022.pdf\n",
      "https://www.avalonhcs.com/wp-content/uploads/CareSource/KYDSNP/G2149%20Pathogen%20Panel%20Testing%20efd;%2009-01-2022.pdf\n",
      "https://www.avalonhcs.com/wp-content/uploads/CareSource/KYDSNP/G2149%20Pathogen%20Panel%20Testing%20efd;%2009-01-2022.pdf\n",
      "1\n",
      "https://www.avalonhcs.com/wp-content/uploads/optimahealth/G2149%20Pathogen%20Panel%20Testing.pdf\n",
      "https://www.avalonhcs.com/wp-content/uploads/CareSource/ARPASSE/G2035%20Prenatal%20Screening%20efd;%2001-01-2022.pdf\n",
      "https://cdn1-originals.webdamdb.com/14017_139613283?cache=1683668076&response-content-disposition=inline;filename=AHS-G2054%2520Liquid%2520Biopsy_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjEzMjgzP2NhY2hlPTE2ODM2NjgwNzYmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTLUcyMDU0JTI1MjBMaXF1aWQlMjUyMEJpb3BzeV8yMDIzLTA2MDEucGRmJnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1hcHBsaWNhdGlvbi9wZGYiLCJDb25kaXRpb24iOnsiRGF0ZUxlc3NUaGFuIjp7IkFXUzpFcG9jaFRpbWUiOjIxNDc0MTQ0MDB9fX1dfQ__&Signature=bH0R3bcP~KZa4BvJJ282CfhWpE5A8lhkzVKRdC7KLiChnJ~VxlvZLg03wJaVm7Hn65uR63rpQqmuq8WXR~dPlswmGNIGKH9mH7QMVLKptNh1yWD-Zs8iTfhoysqdT0UeJVCOA-DpZYk0U1zSWsedksU~KUyHcMvUSyLlceBXtbPGNmnVjSBbx37nCDVSS78F9OjZLHwyFe3anZDosZ-s5OFRLi15RT45lRCRqki5MsQpIPd8tdjWYqMNZ~~UZUfzqaAwrOgE4G8A1h2Q1PSSPBw5QB5lchBlIJ~f981K687PivdXJL-xBAoRwInxq1qr7dIdCv1yny4pYZ0RwwOf6w__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://www.avalonhcs.com/wp-content/uploads/optimahealth/G2149%20Pathogen%20Panel%20Testing.pdf\n",
      "https://www.avalonhcs.com/wp-content/uploads/bcbsla/M2091%20Transplant%20Rejection%20Testing.pdf\n",
      "https://cdn1-originals.webdamdb.com/14017_139613309?cache=1683669816&response-content-disposition=inline;filename=AHS-M2085%2520Genetic%2520Testing%2520of%2520Mitochondrial%2520Disorders_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjEzMzA5P2NhY2hlPTE2ODM2Njk4MTYmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTLU0yMDg1JTI1MjBHZW5ldGljJTI1MjBUZXN0aW5nJTI1MjBvZiUyNTIwTWl0b2Nob25kcmlhbCUyNTIwRGlzb3JkZXJzXzIwMjMtMDYwMS5wZGYmcmVzcG9uc2UtY29udGVudC10eXBlPWFwcGxpY2F0aW9uL3BkZiIsIkNvbmRpdGlvbiI6eyJEYXRlTGVzc1RoYW4iOnsiQVdTOkVwb2NoVGltZSI6MjE0NzQxNDQwMH19fV19&Signature=bn8GCRVWT1YAeonu5mPxVKNeIXxzLs1WyK2d4hZGUMMAVl7enFw~7pHWKoeoPgAHBXq9yMifQI~lq1wlsjXaxYQqKCeShSPZ-FLqPZeTdtYGzUhkuCNtyAs9Jxi1KIle7MpDOzKgSKhQgsA9GrDFUCAO-TzU8sck45lQo0DQv9~AMCUPzLkIRxVyT3Tz33brdZYkzwWtfYrynJyAbehq-4Bgvf7o5k~LxG-boQkf51Xzbxjav63jzlPNHTSqlp~2L8mfpjmarZJxdM57dYvEeS~VficrtecCQ29WPpMh9XAqC2F0J1zZAjLCEfQoB4eneCKr-pq3IxYDW5VBpIxv5g__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://cdn1-originals.webdamdb.com/14017_140263828?cache=1683668191&response-content-disposition=inline;filename=AHS%25E2%2580%2593M2003%2520BRCA_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTQwMjYzODI4P2NhY2hlPTE2ODM2NjgxOTEmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTJTI1RTIlMjU4MCUyNTkzTTIwMDMlMjUyMEJSQ0FfMjAyMy0wNjAxLnBkZiZyZXNwb25zZS1jb250ZW50LXR5cGU9YXBwbGljYXRpb24vcGRmIiwiQ29uZGl0aW9uIjp7IkRhdGVMZXNzVGhhbiI6eyJBV1M6RXBvY2hUaW1lIjoyMTQ3NDE0NDAwfX19XX0_&Signature=n3PMMNTenneHMc-uX8iQt5CxY0IrvS8WW3YUkNm3T9xkSuQy~xKyzrcNc5m-zvl~9MhR7bT0~AZd3xXCIfKR-DHgxVUvPang1rs8aQOCi8pmDFXGf-4ewUtksy5i8aNrhbYW2BC-1UcH380bNmKJPSZoxfwndyDvX090dhP3SprmpfC~5-PfQ5GfLn19KalH7ioZhJuBqJkzrYHTHXKNfGvF1KMYSzJWU-x3UVm-68SNfo~k64l2fxtWTSZ4p31FI-Xi1B4cZsLgMJAY52OHcIhUWeXVmbiGjtdh6XiL~jyW1zxBITTCVWLj~1t5BMkf5xnjoiq4yDdqMjn2czYhRQ__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://cdn1-originals.webdamdb.com/14017_140263828?cache=1683668191&response-content-disposition=inline;filename=AHS%25E2%2580%2593M2003%2520BRCA_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTQwMjYzODI4P2NhY2hlPTE2ODM2NjgxOTEmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTJTI1RTIlMjU4MCUyNTkzTTIwMDMlMjUyMEJSQ0FfMjAyMy0wNjAxLnBkZiZyZXNwb25zZS1jb250ZW50LXR5cGU9YXBwbGljYXRpb24vcGRmIiwiQ29uZGl0aW9uIjp7IkRhdGVMZXNzVGhhbiI6eyJBV1M6RXBvY2hUaW1lIjoyMTQ3NDE0NDAwfX19XX0_&Signature=n3PMMNTenneHMc-uX8iQt5CxY0IrvS8WW3YUkNm3T9xkSuQy~xKyzrcNc5m-zvl~9MhR7bT0~AZd3xXCIfKR-DHgxVUvPang1rs8aQOCi8pmDFXGf-4ewUtksy5i8aNrhbYW2BC-1UcH380bNmKJPSZoxfwndyDvX090dhP3SprmpfC~5-PfQ5GfLn19KalH7ioZhJuBqJkzrYHTHXKNfGvF1KMYSzJWU-x3UVm-68SNfo~k64l2fxtWTSZ4p31FI-Xi1B4cZsLgMJAY52OHcIhUWeXVmbiGjtdh6XiL~jyW1zxBITTCVWLj~1t5BMkf5xnjoiq4yDdqMjn2czYhRQ__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://www.avalonhcs.com/wp-content/uploads/optimahealth/G2149%20Pathogen%20Panel%20Testing.pdf\n",
      "https://cdn1-originals.webdamdb.com/14017_139613320?cache=1683670067&response-content-disposition=inline;filename=AHS-M2101%2520Mutation%2520Analysis%2520in%2520Myeloproliferative%2520Neoplasms_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjEzMzIwP2NhY2hlPTE2ODM2NzAwNjcmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTLU0yMTAxJTI1MjBNdXRhdGlvbiUyNTIwQW5hbHlzaXMlMjUyMGluJTI1MjBNeWVsb3Byb2xpZmVyYXRpdmUlMjUyME5lb3BsYXNtc18yMDIzLTA2MDEucGRmJnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1hcHBsaWNhdGlvbi9wZGYiLCJDb25kaXRpb24iOnsiRGF0ZUxlc3NUaGFuIjp7IkFXUzpFcG9jaFRpbWUiOjIxNDc0MTQ0MDB9fX1dfQ__&Signature=Djv6OmQGsbwjyCrelr6EyWEyzq3s7Is8OcarbOiXfkeZvGYh-8lHNdxC9kX5n10X4aXr9RvlNguLnmTUg3I1JPODE1ofSNm~sUpYy3o4BgJd1yADDvM75RNDiKp911xqRDUpK-BTW3ICvq9rrlLA-imxhRwpYAAoX1FYWyoMCqrG-NZAbAzlfh5Ip2TJcf0w7K7~i63PhuREj1CDw3M2fWmjg9KppI0wvE~lacT9ihduV64-ouusaGs6YDfLmYCssAnSDfGJH3u7m0PwZu5gf93txpl43P3VH35nnFxAhnCmLXCzo0oRXdf1kx4dH80AG6GC5qFIP9SIOloTpTwazw__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://cdn1-originals.webdamdb.com/14017_140263887?cache=1683670018&response-content-disposition=inline;filename=AHS%25E2%2580%2593M2175%2520Minimal%2520Residual%2520Disease_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTQwMjYzODg3P2NhY2hlPTE2ODM2NzAwMTgmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTJTI1RTIlMjU4MCUyNTkzTTIxNzUlMjUyME1pbmltYWwlMjUyMFJlc2lkdWFsJTI1MjBEaXNlYXNlXzIwMjMtMDYwMS5wZGYmcmVzcG9uc2UtY29udGVudC10eXBlPWFwcGxpY2F0aW9uL3BkZiIsIkNvbmRpdGlvbiI6eyJEYXRlTGVzc1RoYW4iOnsiQVdTOkVwb2NoVGltZSI6MjE0NzQxNDQwMH19fV19&Signature=UWyFxJBKLEVlM0cNlOnvkRJ6l3e5TtPyW4kxcTGZoc9r~agVhlp~MDHuPYLS9hBKyK09B1WbCndR89-TJYIkL6z1ERMM-OPcrMOt0uHULdxR~cmHBkPpOxV37LycMO3ckbdn5dSbrFoUlN1hSRqJ5x6Hv9sTF89V-1LfWRk1pIW2BGSjjq9zbUwKhtdHV5iFhdV2Uj6ECkysRnsingEb7kur9YezNCjLlDLQQsKDnfRDrYNmtYvNRw8F9x2I-CKSSIaI3SyjjOJiXr~24WZO1wKoLPzkygNxUFpOjwRNO8V9oH9282cYhK0yhcM~vJn1S1ZRirstsaD1sHM5ASLRLg__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://cdn1-originals.webdamdb.com/14017_139613322?cache=1683670446&response-content-disposition=inline;filename=AHS-M2178%2520Microsatellite%2520Instability%2520and%2520Tumor%2520Mutational%2520Burden%2520Testing_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjEzMzIyP2NhY2hlPTE2ODM2NzA0NDYmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTLU0yMTc4JTI1MjBNaWNyb3NhdGVsbGl0ZSUyNTIwSW5zdGFiaWxpdHklMjUyMGFuZCUyNTIwVHVtb3IlMjUyME11dGF0aW9uYWwlMjUyMEJ1cmRlbiUyNTIwVGVzdGluZ18yMDIzLTA2MDEucGRmJnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1hcHBsaWNhdGlvbi9wZGYiLCJDb25kaXRpb24iOnsiRGF0ZUxlc3NUaGFuIjp7IkFXUzpFcG9jaFRpbWUiOjIxNDc0MTQ0MDB9fX1dfQ__&Signature=jVXe9gY04jCoM56dnykw8~ry-VtcraelaClP~MIroRHA-uHhHfbZPrfzL1U6kPI6rYxG6TwQLPUhgZKtWmiHiCH6QfKuC~vGBt7D68GxndXnp3X6UJtNpDGPi5Ny8RYC24FVhgrZFOwCexlmOh44Yp1twfVWamYaULabB-RUzule59TTmqixjRtmEZK-OWBd9NNcF-I7Prz6IfdQnKaYm4MAiWzPl0n7aNIfF4oxWyl0kUvP-0bR~0h23PJYzlQsrUoJPkG~ndMq0~u5ZykxYwJZJO-FVyD~UUomfqftFLou5zny-3VRnkZ4eHxxR2Z7WM7gXrJmkIDGnV6Ck5Kchg__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://cdn1-originals.webdamdb.com/14017_140263855?cache=1683668593&response-content-disposition=inline;filename=AHS%25E2%2580%2593M2029%2520Genetic%2520Testing%2520and%2520Genetic%2520Expression%2520Profiling%2520in%2520Patients%2520with%2520Cutaneous%2520Melanoma_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTQwMjYzODU1P2NhY2hlPTE2ODM2Njg1OTMmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTJTI1RTIlMjU4MCUyNTkzTTIwMjklMjUyMEdlbmV0aWMlMjUyMFRlc3RpbmclMjUyMGFuZCUyNTIwR2VuZXRpYyUyNTIwRXhwcmVzc2lvbiUyNTIwUHJvZmlsaW5nJTI1MjBpbiUyNTIwUGF0aWVudHMlMjUyMHdpdGglMjUyMEN1dGFuZW91cyUyNTIwTWVsYW5vbWFfMjAyMy0wNjAxLnBkZiZyZXNwb25zZS1jb250ZW50LXR5cGU9YXBwbGljYXRpb24vcGRmIiwiQ29uZGl0aW9uIjp7IkRhdGVMZXNzVGhhbiI6eyJBV1M6RXBvY2hUaW1lIjoyMTQ3NDE0NDAwfX19XX0_&Signature=l3BspsvwqNee~wXIcBnV9EYBEFT2mP4~aD890l3xKvHechjaocITHdhYZGUljKyXghmoOAcsjGvOsJ1lkSCkwVh8GiMmPr8Gc9nDftVJNwDW~X0VljHkYK8lMWvfvksnKaybZqG~wSYxS1O4N9SLWvJjPtd9fFV6U5gvwNobUh0PGSP5bBCFRKNfETVs9NkevRNIPAEn4AXA8FSqFMwPeoOtFttPkfhM7s7diBWIRA56S0pqiE1rDlTS4NTe2kr4mE2NJkm3KX~Iv8LERBnQ0ypNWu~ZvYr63Dtlx10KU09oy7WqxlL1w4Kbp8wptOIR-vatyGSFKQUknE0d8Gvzug__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "1\n",
      "https://cdn1-originals.webdamdb.com/14017_139612976?cache=1683670271&response-content-disposition=inline;filename=AHS-M2145%2520General%2520Genetic%2520Testing%252C%2520Germline%2520Disorders_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjEyOTc2P2NhY2hlPTE2ODM2NzAyNzEmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTLU0yMTQ1JTI1MjBHZW5lcmFsJTI1MjBHZW5ldGljJTI1MjBUZXN0aW5nJTI1MkMlMjUyMEdlcm1saW5lJTI1MjBEaXNvcmRlcnNfMjAyMy0wNjAxLnBkZiZyZXNwb25zZS1jb250ZW50LXR5cGU9YXBwbGljYXRpb24vcGRmIiwiQ29uZGl0aW9uIjp7IkRhdGVMZXNzVGhhbiI6eyJBV1M6RXBvY2hUaW1lIjoyMTQ3NDE0NDAwfX19XX0_&Signature=IV06fFxT2Ug5T0yhRFUzyPvrwU1Ur4md7KpjkEKwlHh6dvxMXL7cgQlk8ya5~1cipLCfB1oIzifugQjRUPF4MWh3058eBuIWRj48NS0TT3jNkqL8bLjyZV4iobELNQatMv1U~eTS0dzVW8AH8ctznIy1IBDLgK3TJk9o0zPZjKQuQR5acSaXL9cNWP8~KitClE2dL0X6v83UV0gS3pvJ0FAvL1ROVPpH-NEsONu67nBLhFMAIneOLxsms1iKhDbSV4gcsacw5nfDrPlHiQk0xbC5noF5Vul1uuEXhQ9kGaeiL7cEP5Igfi98EwHAgbIT83jokFh5CHdJAE0nbu1FmA__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://www.avalonhcs.com/wp-content/uploads/CareSource/GAMedicaid/G2124%20Serum%20Tumor%20Markers%20for%20Malignancies%20v2%20efd;%2002-01-2023.pdf\n",
      "https://www.avalonhcs.com/wp-content/uploads/optimahealth/G2043%20Celiac%20Disease%20Testing.pdf\n",
      "https://www.avalonhcs.com/wp-content/uploads/bcbsla/M2091%20Transplant%20Rejection%20Testing.pdf\n",
      "https://cdn1-originals.webdamdb.com/14017_139612978?cache=1683670541&response-content-disposition=inline;filename=AHS%25E2%2580%2593M2166%2520Gene%2520Expression%2520Profiling%2520and%2520Protein%2520Biomarkers%2520for%2520Prostate%2520Cancer_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjEyOTc4P2NhY2hlPTE2ODM2NzA1NDEmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTJTI1RTIlMjU4MCUyNTkzTTIxNjYlMjUyMEdlbmUlMjUyMEV4cHJlc3Npb24lMjUyMFByb2ZpbGluZyUyNTIwYW5kJTI1MjBQcm90ZWluJTI1MjBCaW9tYXJrZXJzJTI1MjBmb3IlMjUyMFByb3N0YXRlJTI1MjBDYW5jZXJfMjAyMy0wNjAxLnBkZiZyZXNwb25zZS1jb250ZW50LXR5cGU9YXBwbGljYXRpb24vcGRmIiwiQ29uZGl0aW9uIjp7IkRhdGVMZXNzVGhhbiI6eyJBV1M6RXBvY2hUaW1lIjoyMTQ3NDE0NDAwfX19XX0_&Signature=XzgveijaVpJg~UP0ji6dYUfgczjFGkgbMZWYRdRJkrrX-zO-8dbw0CADVz4G8r-uLtxmEflt5xqyQrEJ~JBrAuGI5rRNaBTe-b4iiwCbBzhA68flfVpLwCTNcnusyQcem-IomQkTtZyUVQxuv3sT2xxSL0T4rccnPUAqhWTeLYeMmuXPuHcLftkWIfPMSQ19NPup-wMgZZag4ggNrf4Wxta2cvytVs6dLX1wUj4kPb~6WsYmvE0dZ4CWhKVLE6plYXchtyDs-Jqumr6k1CE4gk2I9ZC~NGTAzG5BUxRlYbCmS8cCfCtV2PLQgTVTZa2Egiepl6bFKlkGzVRtWQBstg__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://cdn1-originals.webdamdb.com/14017_140273002?cache=1683668206&response-content-disposition=inline;filename=AHS%25E2%2580%2593M2004%2520Lynch%2520Syndrome_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTQwMjczMDAyP2NhY2hlPTE2ODM2NjgyMDYmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTJTI1RTIlMjU4MCUyNTkzTTIwMDQlMjUyMEx5bmNoJTI1MjBTeW5kcm9tZV8yMDIzLTA2MDEucGRmJnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1hcHBsaWNhdGlvbi9wZGYiLCJDb25kaXRpb24iOnsiRGF0ZUxlc3NUaGFuIjp7IkFXUzpFcG9jaFRpbWUiOjIxNDc0MTQ0MDB9fX1dfQ__&Signature=amCzCM0wkM9vnkbMjflAAhMLnFE6rJ3E4bUq2aaHRKYnOMZFv~8WpAx~YJh3tAJxmXHvMEq2-Y16KT~GOPuLjFwswGYBcDQ9OfiJf7Nq0I0M0m6MEwX8gactYavZGo52XY0V2qW9gvnH2eG1V5ukRl5fQ8jCRArupWDAi0VoKR3UOSLHKVaUYqWp6KCJrUCJnUL01pmtAdNkLMm-WDWNPFlZpJhd7QRS~i4tskcuPU0ksuaosDB5ACeMMRL1qJM~CGWIbe5GkmDrjlJWNKglHlK~NIpgEnbE52MCAKa1PiE30uUMft6c7XJzWTh~wUbbYzj2~VMRb-svVukFbwlFqA__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://cdn1-originals.webdamdb.com/14017_139605667?cache=1683669471&response-content-disposition=inline;filename=AHS-M2062%2520Genetic%2520Testing%2520for%2520Acute%2520Myeloid%2520Leukemia_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjA1NjY3P2NhY2hlPTE2ODM2Njk0NzEmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTLU0yMDYyJTI1MjBHZW5ldGljJTI1MjBUZXN0aW5nJTI1MjBmb3IlMjUyMEFjdXRlJTI1MjBNeWVsb2lkJTI1MjBMZXVrZW1pYV8yMDIzLTA2MDEucGRmJnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1hcHBsaWNhdGlvbi9wZGYiLCJDb25kaXRpb24iOnsiRGF0ZUxlc3NUaGFuIjp7IkFXUzpFcG9jaFRpbWUiOjIxNDc0MTQ0MDB9fX1dfQ__&Signature=Nmb3T8vTlVLonA1RQhIOSheD666p9jPfrAYTAx4p9d~HA6Bbd38Tecean1k2oNBlDwtFBeWA5HUuxO8kfsGSGKh5D~iUyPtf~s-1qkxdXjf4bMZm-RQfMSYHOPNw~yeTANVI7DZjx9QPSi9Sk0Msr0zhsUPQ0hqX5bovD9lEndPUit6uwDhY8lPDeoBhLVHyVEFjTICesh4jLykWhM75z5lAFY6rypu-e-w0spISK~DkTpYmspewQ9TPMS1D-N1gTfmy6V9G7YYtKHOKNA0NIgmelTn7~7egoUE~B5NLLDWul65~NRAZggPKgOO5lpfcOZCxX-3By9GtWpUmZa02Rw__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://www.avalonhcs.com/wp-content/uploads/CareSource/KYMarketplace/M2097%20v2%20Identification%20of%20Microorganisms%20Using%20Nucleic%20Acid%20Probes%20efd;%2006-01-2023.pdf\n",
      "https://cdn1-originals.webdamdb.com/14017_139613283?cache=1683668076&response-content-disposition=inline;filename=AHS-G2054%2520Liquid%2520Biopsy_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjEzMjgzP2NhY2hlPTE2ODM2NjgwNzYmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTLUcyMDU0JTI1MjBMaXF1aWQlMjUyMEJpb3BzeV8yMDIzLTA2MDEucGRmJnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1hcHBsaWNhdGlvbi9wZGYiLCJDb25kaXRpb24iOnsiRGF0ZUxlc3NUaGFuIjp7IkFXUzpFcG9jaFRpbWUiOjIxNDc0MTQ0MDB9fX1dfQ__&Signature=bH0R3bcP~KZa4BvJJ282CfhWpE5A8lhkzVKRdC7KLiChnJ~VxlvZLg03wJaVm7Hn65uR63rpQqmuq8WXR~dPlswmGNIGKH9mH7QMVLKptNh1yWD-Zs8iTfhoysqdT0UeJVCOA-DpZYk0U1zSWsedksU~KUyHcMvUSyLlceBXtbPGNmnVjSBbx37nCDVSS78F9OjZLHwyFe3anZDosZ-s5OFRLi15RT45lRCRqki5MsQpIPd8tdjWYqMNZ~~UZUfzqaAwrOgE4G8A1h2Q1PSSPBw5QB5lchBlIJ~f981K687PivdXJL-xBAoRwInxq1qr7dIdCv1yny4pYZ0RwwOf6w__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://www.avalonhcs.com/wp-content/uploads/bcbsla/G2009%20Preventive%20Screening%20in%20Adults.pdf\n",
      "https://cdn1-originals.webdamdb.com/14017_140263828?cache=1683668191&response-content-disposition=inline;filename=AHS%25E2%2580%2593M2003%2520BRCA_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTQwMjYzODI4P2NhY2hlPTE2ODM2NjgxOTEmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTJTI1RTIlMjU4MCUyNTkzTTIwMDMlMjUyMEJSQ0FfMjAyMy0wNjAxLnBkZiZyZXNwb25zZS1jb250ZW50LXR5cGU9YXBwbGljYXRpb24vcGRmIiwiQ29uZGl0aW9uIjp7IkRhdGVMZXNzVGhhbiI6eyJBV1M6RXBvY2hUaW1lIjoyMTQ3NDE0NDAwfX19XX0_&Signature=n3PMMNTenneHMc-uX8iQt5CxY0IrvS8WW3YUkNm3T9xkSuQy~xKyzrcNc5m-zvl~9MhR7bT0~AZd3xXCIfKR-DHgxVUvPang1rs8aQOCi8pmDFXGf-4ewUtksy5i8aNrhbYW2BC-1UcH380bNmKJPSZoxfwndyDvX090dhP3SprmpfC~5-PfQ5GfLn19KalH7ioZhJuBqJkzrYHTHXKNfGvF1KMYSzJWU-x3UVm-68SNfo~k64l2fxtWTSZ4p31FI-Xi1B4cZsLgMJAY52OHcIhUWeXVmbiGjtdh6XiL~jyW1zxBITTCVWLj~1t5BMkf5xnjoiq4yDdqMjn2czYhRQ__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://www.avalonhcs.com/wp-content/uploads/bcbsvt/G2050%20Cardiovascular%20Disease%20Risk%20Assessment.pdf\n",
      "https://cdn1-originals.webdamdb.com/14017_139613286?cache=1683668518&response-content-disposition=inline;filename=AHS-M2024%2520Genetic%2520Testing%2520for%2520Polyposis%2520Syndrome_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjEzMjg2P2NhY2hlPTE2ODM2Njg1MTgmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTLU0yMDI0JTI1MjBHZW5ldGljJTI1MjBUZXN0aW5nJTI1MjBmb3IlMjUyMFBvbHlwb3NpcyUyNTIwU3luZHJvbWVfMjAyMy0wNjAxLnBkZiZyZXNwb25zZS1jb250ZW50LXR5cGU9YXBwbGljYXRpb24vcGRmIiwiQ29uZGl0aW9uIjp7IkRhdGVMZXNzVGhhbiI6eyJBV1M6RXBvY2hUaW1lIjoyMTQ3NDE0NDAwfX19XX0_&Signature=D07CD9jCA~Fwa2zFXluaWGYwC~ug~F~CpTp3bPCl20eCyjW9U~68a9X0Zfhl632-pZBIZU55DNQr9DW3sJVDU-Gr6IO44NdqGp3n-TY8cfCuklacYuiejVdNMthVYvRedTssAeYOZnJ1vYY0yEudZVvzYK~NCfGpRMyliPAdxOTjHF~IHnAb5wtNgxBPjyMz6DitMrnGAcIMwN8gVvZ2jdw4kCWvB109CBhKGgRE~3zqCFqvCe8HDzHoVQzzi595h3G8tb5x34xBKbNv0Yga7eInvUO0uKGMvjdir7KD0uaRxHNOOlXz0SxFISBziayW7eL6Xepi3ygpboDMHt6Hpw__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://cdn1-originals.webdamdb.com/14017_139612976?cache=1683670271&response-content-disposition=inline;filename=AHS-M2145%2520General%2520Genetic%2520Testing%252C%2520Germline%2520Disorders_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjEyOTc2P2NhY2hlPTE2ODM2NzAyNzEmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTLU0yMTQ1JTI1MjBHZW5lcmFsJTI1MjBHZW5ldGljJTI1MjBUZXN0aW5nJTI1MkMlMjUyMEdlcm1saW5lJTI1MjBEaXNvcmRlcnNfMjAyMy0wNjAxLnBkZiZyZXNwb25zZS1jb250ZW50LXR5cGU9YXBwbGljYXRpb24vcGRmIiwiQ29uZGl0aW9uIjp7IkRhdGVMZXNzVGhhbiI6eyJBV1M6RXBvY2hUaW1lIjoyMTQ3NDE0NDAwfX19XX0_&Signature=IV06fFxT2Ug5T0yhRFUzyPvrwU1Ur4md7KpjkEKwlHh6dvxMXL7cgQlk8ya5~1cipLCfB1oIzifugQjRUPF4MWh3058eBuIWRj48NS0TT3jNkqL8bLjyZV4iobELNQatMv1U~eTS0dzVW8AH8ctznIy1IBDLgK3TJk9o0zPZjKQuQR5acSaXL9cNWP8~KitClE2dL0X6v83UV0gS3pvJ0FAvL1ROVPpH-NEsONu67nBLhFMAIneOLxsms1iKhDbSV4gcsacw5nfDrPlHiQk0xbC5noF5Vul1uuEXhQ9kGaeiL7cEP5Igfi98EwHAgbIT83jokFh5CHdJAE0nbu1FmA__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://www.avalonhcs.com/wp-content/uploads/optimahealth/G2043%20Celiac%20Disease%20Testing.pdf\n",
      "https://cdn1-originals.webdamdb.com/14017_139612976?cache=1683670271&response-content-disposition=inline;filename=AHS-M2145%2520General%2520Genetic%2520Testing%252C%2520Germline%2520Disorders_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjEyOTc2P2NhY2hlPTE2ODM2NzAyNzEmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTLU0yMTQ1JTI1MjBHZW5lcmFsJTI1MjBHZW5ldGljJTI1MjBUZXN0aW5nJTI1MkMlMjUyMEdlcm1saW5lJTI1MjBEaXNvcmRlcnNfMjAyMy0wNjAxLnBkZiZyZXNwb25zZS1jb250ZW50LXR5cGU9YXBwbGljYXRpb24vcGRmIiwiQ29uZGl0aW9uIjp7IkRhdGVMZXNzVGhhbiI6eyJBV1M6RXBvY2hUaW1lIjoyMTQ3NDE0NDAwfX19XX0_&Signature=IV06fFxT2Ug5T0yhRFUzyPvrwU1Ur4md7KpjkEKwlHh6dvxMXL7cgQlk8ya5~1cipLCfB1oIzifugQjRUPF4MWh3058eBuIWRj48NS0TT3jNkqL8bLjyZV4iobELNQatMv1U~eTS0dzVW8AH8ctznIy1IBDLgK3TJk9o0zPZjKQuQR5acSaXL9cNWP8~KitClE2dL0X6v83UV0gS3pvJ0FAvL1ROVPpH-NEsONu67nBLhFMAIneOLxsms1iKhDbSV4gcsacw5nfDrPlHiQk0xbC5noF5Vul1uuEXhQ9kGaeiL7cEP5Igfi98EwHAgbIT83jokFh5CHdJAE0nbu1FmA__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "https://cdn1-originals.webdamdb.com/14017_139613309?cache=1683669816&response-content-disposition=inline;filename=AHS-M2085%2520Genetic%2520Testing%2520of%2520Mitochondrial%2520Disorders_2023-0601.pdf&response-content-type=application/pdf&Policy=eyJTdGF0ZW1lbnQiOlt7IlJlc291cmNlIjoiaHR0cCo6Ly9jZG4xLW9yaWdpbmFscy53ZWJkYW1kYi5jb20vMTQwMTdfMTM5NjEzMzA5P2NhY2hlPTE2ODM2Njk4MTYmcmVzcG9uc2UtY29udGVudC1kaXNwb3NpdGlvbj1pbmxpbmU7ZmlsZW5hbWU9QUhTLU0yMDg1JTI1MjBHZW5ldGljJTI1MjBUZXN0aW5nJTI1MjBvZiUyNTIwTWl0b2Nob25kcmlhbCUyNTIwRGlzb3JkZXJzXzIwMjMtMDYwMS5wZGYmcmVzcG9uc2UtY29udGVudC10eXBlPWFwcGxpY2F0aW9uL3BkZiIsIkNvbmRpdGlvbiI6eyJEYXRlTGVzc1RoYW4iOnsiQVdTOkVwb2NoVGltZSI6MjE0NzQxNDQwMH19fV19&Signature=bn8GCRVWT1YAeonu5mPxVKNeIXxzLs1WyK2d4hZGUMMAVl7enFw~7pHWKoeoPgAHBXq9yMifQI~lq1wlsjXaxYQqKCeShSPZ-FLqPZeTdtYGzUhkuCNtyAs9Jxi1KIle7MpDOzKgSKhQgsA9GrDFUCAO-TzU8sck45lQo0DQv9~AMCUPzLkIRxVyT3Tz33brdZYkzwWtfYrynJyAbehq-4Bgvf7o5k~LxG-boQkf51Xzbxjav63jzlPNHTSqlp~2L8mfpjmarZJxdM57dYvEeS~VficrtecCQ29WPpMh9XAqC2F0J1zZAjLCEfQoB4eneCKr-pq3IxYDW5VBpIxv5g__&Key-Pair-Id=APKAI2ASI2IOLRFF2RHA\n",
      "1\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Whole-Exome-Sequencing-and-Whole-Genome-Sequencing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Genetic-Testing-for-Inherited-Conditions-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Genetic-Testing-for-Inherited-Conditions-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Carrier-Screening-in-the-Prenatal-Setting-and-Preimplantation-Genetic-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Genetic-Testing-for-Inherited-Conditions-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Genetic-Testing-for-Inherited-Conditions-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Polygenic-Risk-Scores-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Genetic-Testing-for-Inherited-Conditions-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Genetic-Testing-for-Inherited-Conditions-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Genetic-Testing-for-Inherited-Conditions-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2021/11/ARCHIVED-Reproductive-Carrier-Screening-and-Prenatal-Diagnosis-03-31-19-to-09-08-19.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Genetic-Testing-for-Inherited-Conditions-2023-02-12.pdf\n",
      "1\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Genetic-Testing-for-Inherited-Conditions-2023-02-12.pdf\n",
      "1\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Hereditary-Cancer-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Hereditary-Cancer-Testing-2023-02-12.pdf\n",
      "1\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/carrier-screening-in-the-prenatal-setting-and-preimplantation-genetic-testing-2023-02-12/\n",
      "1\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2021/11/ARCHIVED-Molecular-Testing-Solid-and-Hematologic-Tumors-01-04-21-to-02-07-21.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Hereditary-Cancer-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Hereditary-Cancer-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2021/11/ARCHIVED-Molecular-Testing-Solid-and-Hematologic-Tumors-01-04-21-to-02-07-21.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Hereditary-Cancer-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Hereditary-Cancer-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Hereditary-Cancer-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Hereditary-Cancer-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Genetic-Testing-for-Inherited-Conditions-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2021/01/ARCHIVED_AIM_Laboratory_medicine_eff-01-01-21-to-3-31-21.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2021/11/ARCHIVED-Hereditary-Cardiac-Disease-03-08-21-to-09-05-21.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2021/11/ARCHIVED-Hereditary-Cardiac-Disease-03-08-21-to-09-05-21.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Hereditary-Cancer-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Genetic-Testing-for-Inherited-Conditions-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Hereditary-Cancer-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2021/11/ARCHIVED-Hereditary-Cardiac-Disease-03-08-21-to-09-05-21.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Hereditary-Cancer-Testing-2023-02-12.pdf\n",
      "1\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Hereditary-Cancer-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2021/01/ARCHIVED_AIM_Laboratory_medicine_eff-01-01-21-to-3-31-21.pdf\n",
      "1\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Pharmacogenomic-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Hereditary-Cancer-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2021/11/ARCHIVED-Reproductive-Carrier-Screening-and-Prenatal-Diagnosis-09-01-20-to-01-03-21.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/pharmacogenomic-testing-2023-02-12/\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "1\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/pharmacogenomic-testing-2023-02-12/\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2021/11/ARCHIVED-Pharmacogenomic-and-Thrombophilia-Testing-09-09-19-to-03-02-20.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/prenatal-testing-using-cell-free-dna-2023-02-12/\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Hereditary-Cancer-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Hereditary-Cancer-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Genetic-Testing-for-Inherited-Conditions-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Genetic-Testing-for-Inherited-Conditions-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/carrier-screening-in-the-prenatal-setting-and-preimplantation-genetic-testing-2023-02-12/\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Genetic-Testing-for-Inherited-Conditions-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/pharmacogenomic-testing-2023-02-12/\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Hereditary-Cancer-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/whole-exome-sequencing-and-whole-genome-sequencing-2023-02-12/\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2021/11/ARCHIVED-Hereditary-Cancer-Susceptibility-03-31-19-to-09-08-19.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Hereditary-Cancer-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/carrier-screening-in-the-prenatal-setting-and-preimplantation-genetic-testing-2023-02-12/\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/carrier-screening-in-the-prenatal-setting-and-preimplantation-genetic-testing-2023-02-12/\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2021/11/ARCHIVED-Hereditary-Cancer-Susceptibility-03-31-19-to-09-08-19.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Hereditary-Cancer-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/whole-exome-sequencing-and-whole-genome-sequencing-2023-02-12/\n",
      "1\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/prenatal-testing-using-cell-free-dna-2023-02-12/\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/prenatal-testing-using-cell-free-dna-2023-02-12/\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/whole-exome-sequencing-and-whole-genome-sequencing-2023-02-12/\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Prenatal-Testing-using-cell-free-DNA-2023-02-12.pdf\n",
      "1\n",
      "1\n",
      "1\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/cell-free-dna-testing-for-the-management-of-cancer-2023-02-12/\n",
      "1\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/chromosomal-microarray-analysis-2023-02-12/\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/cell-free-dna-testing-for-the-management-of-cancer-2023-02-12/\n",
      "1\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Genetic-Testing-for-Inherited-Conditions-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Genetic-Testing-for-Inherited-Conditions-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Hereditary-Cancer-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Hereditary-Cancer-Testing-2023-02-12.pdf\n",
      "1\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2021/11/ARCHIVED-Molecular-Testing-Solid-and-Hematologic-Tumors-09-09-19-to-12-08-19.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "1\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Genetic-Testing-for-Inherited-Conditions-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "1\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Genetic-Testing-for-Inherited-Conditions-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Hereditary-Cancer-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "1\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/cell-free-dna-testing-for-the-management-of-cancer-2023-02-12/\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2021/11/ARCHIVED-Molecular-Testing-Solid-and-Hematologic-Tumors-01-04-21-to-02-07-21.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Hereditary-Cancer-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Hereditary-Cancer-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Genetic-Testing-for-Inherited-Conditions-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2021/11/ARCHIVED-Hereditary-Cancer-Susceptibility-03-31-19-to-09-08-19.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Pharmacogenomic-Testing-2023-02-12.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2021/11/ARCHIVED-Hereditary-Cancer-Susceptibility-03-31-19-to-09-08-19.pdf\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/whole-exome-sequencing-and-whole-genome-sequencing-2023-02-12/\n",
      "https://guidelines.carelonmedicalbenefitsmanagement.com/wp-content/uploads/2023/03/PDF-Somatic-Tumor-Testing-2023-02-12.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://d23l36htrrhty7.cloudfront.net/s3fs-public/clinical-guidelines/2023-06/Highmark_Lab-Management-Guidelines_V202023_eff07012023_pub03012023.pdf\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=6a9931dd-4769-44e9-86e3-690fe0dfd2f7\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=54249238-02a3-48ee-9c5b-ecb806f973a8\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=fb7da3a5-c85d-41d5-bc5c-7b7592e7472f\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=fb7da3a5-c85d-41d5-bc5c-7b7592e7472f\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=fb7da3a5-c85d-41d5-bc5c-7b7592e7472f\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=6ebceae4-5dc8-4b30-8c14-8d1f267aa616\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=fb7da3a5-c85d-41d5-bc5c-7b7592e7472f\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=6ebceae4-5dc8-4b30-8c14-8d1f267aa616\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=fb7da3a5-c85d-41d5-bc5c-7b7592e7472f\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=6ebceae4-5dc8-4b30-8c14-8d1f267aa616\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=ad8461b0-dd52-4f30-a660-79d8fca5a17c\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=14cbcdbe-fb34-4db4-a173-47bf52cdb310\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=001857ca-7484-41e9-89b1-3aa1396ed898\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=5326bd53-6b6a-402d-8375-bb2aaa560e93\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=14cbcdbe-fb34-4db4-a173-47bf52cdb310\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=85f5fb11-3dcf-4d6c-9605-1ac3dac4c423\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=919bfda8-3bc5-4b9f-a3bd-4a12cf7116bc\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=001857ca-7484-41e9-89b1-3aa1396ed898\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=6623fdea-3430-4289-9eb8-5cc2437a7276\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=105114c5-2e9e-4386-a4df-3c7678aa2b47\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=75348eb0-c9cc-4b60-b479-23bc3d90470d\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=6ebceae4-5dc8-4b30-8c14-8d1f267aa616\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=794287a4-1e7c-4542-a9b9-46c226e8063b\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=b406ba34-a005-41d7-b00a-d308b528c0c4\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=d743794d-2f7a-4bca-8a59-a2b1eac8e7e3\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=794287a4-1e7c-4542-a9b9-46c226e8063b\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=6e5b693c-3600-4eb4-b91e-9e54605208e0\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=083e0cf4-0fb5-42bf-990a-f4f17dbecf16\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=794287a4-1e7c-4542-a9b9-46c226e8063b\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=79ce77af-986e-4af6-beac-dbb54a1fc3b1\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=d743794d-2f7a-4bca-8a59-a2b1eac8e7e3\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=bcdd89b3-5a3f-47ed-9160-415e4330cc2e\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=75348eb0-c9cc-4b60-b479-23bc3d90470d\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=bcdd89b3-5a3f-47ed-9160-415e4330cc2e\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=b406ba34-a005-41d7-b00a-d308b528c0c4\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=2b5ad913-804a-47e2-8113-fbb024e012c1\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=6ebceae4-5dc8-4b30-8c14-8d1f267aa616\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=7469f818-17af-4cce-ac1b-6ab01e4dc470\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=2c1f1752-f857-408e-a26a-5dffec11c030\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=d743794d-2f7a-4bca-8a59-a2b1eac8e7e3\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=d743794d-2f7a-4bca-8a59-a2b1eac8e7e3\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=794287a4-1e7c-4542-a9b9-46c226e8063b\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=a2df8279-e40b-4e6b-ba7e-e7274e2f02b7\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=2cf1159e-a7ac-478c-98d5-20649b7ccaa8\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=4c7c4c0a-5b1e-406c-805d-0d45218e95c9\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=4c7c4c0a-5b1e-406c-805d-0d45218e95c9\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=2c1f1752-f857-408e-a26a-5dffec11c030\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=2c1f1752-f857-408e-a26a-5dffec11c030\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=8d526ca6-1472-4b98-8ea0-31bad69cf3c6\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=794287a4-1e7c-4542-a9b9-46c226e8063b\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=a43a6c74-c058-430c-9b66-bc3228399346\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=d743794d-2f7a-4bca-8a59-a2b1eac8e7e3\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=d743794d-2f7a-4bca-8a59-a2b1eac8e7e3\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=9d1f0915-9009-46bd-90ff-60a01213a392\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=9d1f0915-9009-46bd-90ff-60a01213a392\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=d743794d-2f7a-4bca-8a59-a2b1eac8e7e3\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=794287a4-1e7c-4542-a9b9-46c226e8063b\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=2c1f1752-f857-408e-a26a-5dffec11c030\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=d743794d-2f7a-4bca-8a59-a2b1eac8e7e3\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=794287a4-1e7c-4542-a9b9-46c226e8063b\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=d743794d-2f7a-4bca-8a59-a2b1eac8e7e3\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=794287a4-1e7c-4542-a9b9-46c226e8063b\n",
      "1\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=d743794d-2f7a-4bca-8a59-a2b1eac8e7e3\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=9d1f0915-9009-46bd-90ff-60a01213a392\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=d743794d-2f7a-4bca-8a59-a2b1eac8e7e3\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=39a8f104-b097-4825-a08d-4e38c2cd04a4\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=d743794d-2f7a-4bca-8a59-a2b1eac8e7e3\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=794287a4-1e7c-4542-a9b9-46c226e8063b\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=794287a4-1e7c-4542-a9b9-46c226e8063b\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=d743794d-2f7a-4bca-8a59-a2b1eac8e7e3\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=794287a4-1e7c-4542-a9b9-46c226e8063b\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=794287a4-1e7c-4542-a9b9-46c226e8063b\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=794287a4-1e7c-4542-a9b9-46c226e8063b\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=decfecb3-fa3e-4733-955b-6ce8807d5e3e\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=decfecb3-fa3e-4733-955b-6ce8807d5e3e\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=223c3f16-84a0-43e1-9582-c2014ff73fa4\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=980a501d-128c-417f-b3f6-f73f9fab87c3\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=794287a4-1e7c-4542-a9b9-46c226e8063b\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=19e655d2-f782-447e-8f52-207ed4da5e87\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=39a8f104-b097-4825-a08d-4e38c2cd04a4\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=794287a4-1e7c-4542-a9b9-46c226e8063b\n",
      "1\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=794287a4-1e7c-4542-a9b9-46c226e8063b\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=7469f818-17af-4cce-ac1b-6ab01e4dc470\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=2b5ad913-804a-47e2-8113-fbb024e012c1\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=223c3f16-84a0-43e1-9582-c2014ff73fa4\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=223c3f16-84a0-43e1-9582-c2014ff73fa4\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=223c3f16-84a0-43e1-9582-c2014ff73fa4\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=794287a4-1e7c-4542-a9b9-46c226e8063b\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=794287a4-1e7c-4542-a9b9-46c226e8063b\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=d743794d-2f7a-4bca-8a59-a2b1eac8e7e3\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=a2df8279-e40b-4e6b-ba7e-e7274e2f02b7\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=19e655d2-f782-447e-8f52-207ed4da5e87\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=794287a4-1e7c-4542-a9b9-46c226e8063b\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=f9408263-7cf9-4638-a4ee-4fc88fcdd07e\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=223c3f16-84a0-43e1-9582-c2014ff73fa4\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=794287a4-1e7c-4542-a9b9-46c226e8063b\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=0efdf548-0da9-4777-8b96-47930a3fa1cd\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=d743794d-2f7a-4bca-8a59-a2b1eac8e7e3\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=9d3d6a38-011f-415c-b267-e875a498011e\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=d743794d-2f7a-4bca-8a59-a2b1eac8e7e3\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=b13aab9e-54d4-4e86-8d75-fdac95bba63a\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=794287a4-1e7c-4542-a9b9-46c226e8063b\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=794287a4-1e7c-4542-a9b9-46c226e8063b\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=794287a4-1e7c-4542-a9b9-46c226e8063b\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=d743794d-2f7a-4bca-8a59-a2b1eac8e7e3\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=6a9931dd-4769-44e9-86e3-690fe0dfd2f7\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=794287a4-1e7c-4542-a9b9-46c226e8063b\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=6aebb87b-54cb-4919-a359-72de8de210a6\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=794287a4-1e7c-4542-a9b9-46c226e8063b\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=d743794d-2f7a-4bca-8a59-a2b1eac8e7e3\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=0efdf548-0da9-4777-8b96-47930a3fa1cd\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=75348eb0-c9cc-4b60-b479-23bc3d90470d\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=75348eb0-c9cc-4b60-b479-23bc3d90470d\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=794287a4-1e7c-4542-a9b9-46c226e8063b\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=d743794d-2f7a-4bca-8a59-a2b1eac8e7e3\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=d743794d-2f7a-4bca-8a59-a2b1eac8e7e3\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=d743794d-2f7a-4bca-8a59-a2b1eac8e7e3\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=0efdf548-0da9-4777-8b96-47930a3fa1cd\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=10caec14-a29c-44d8-88af-ae111919bf4a\n",
      "1\n",
      "1\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=9d3d6a38-011f-415c-b267-e875a498011e\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=9d3d6a38-011f-415c-b267-e875a498011e\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=794287a4-1e7c-4542-a9b9-46c226e8063b\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=decfecb3-fa3e-4733-955b-6ce8807d5e3e\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=794287a4-1e7c-4542-a9b9-46c226e8063b\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=f01d67b3-7ada-4db2-b803-b3a83def7e9e\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=d591922d-f445-45a1-946b-e98c30a5d085\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=b13aab9e-54d4-4e86-8d75-fdac95bba63a\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=b13aab9e-54d4-4e86-8d75-fdac95bba63a\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=b13aab9e-54d4-4e86-8d75-fdac95bba63a\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=6a9931dd-4769-44e9-86e3-690fe0dfd2f7\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=794287a4-1e7c-4542-a9b9-46c226e8063b\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=9d1f0915-9009-46bd-90ff-60a01213a392\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=9d1f0915-9009-46bd-90ff-60a01213a392\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=9d1f0915-9009-46bd-90ff-60a01213a392\n",
      "1\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=9d1f0915-9009-46bd-90ff-60a01213a392\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=9d1f0915-9009-46bd-90ff-60a01213a392\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=f084d0b4-ecd2-4490-82c0-b3c2b0a6cdd6\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=14cbcdbe-fb34-4db4-a173-47bf52cdb310\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=ee32284e-abcd-4044-a61d-fea3688d793f\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=b406ba34-a005-41d7-b00a-d308b528c0c4\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=85f5fb11-3dcf-4d6c-9605-1ac3dac4c423\n",
      "https://www.bcbsms.com/medical-policy-search#/policy-detail?id=85f5fb11-3dcf-4d6c-9605-1ac3dac4c423\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Gen_Testing_Exome_Genome_Sequencing.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Oncology_Algorithmic_Expression_Testing.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Oncology_Algorithmic_Expression_Testing.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Oncology_Algorithmic_Expression_Testing.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Oncology_Algorithmic_Expression_Testing.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Oncology_Algorithmic_Expression_Testing.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Oncology_Algorithmic_Expression_Testing.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Oncology_Algorithmic_Expression_Testing.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Oncology_Algorithmic_Expression_Testing.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Oncology_Algorithmic_Expression_Testing.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Oncology_Algorithmic_Expression_Testing.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Oncology_Algorithmic_Expression_Testing.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Oncology_Algorithmic_Expression_Testing.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Oncology_Algorithmic_Expression_Testing.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Oncology_Molecular_Analysis_Solid_Hema_Malignancies.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Lab_Test_HTR.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Oncology_Algorithmic_Expression_Testing.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Oncology_Algorithmic_Expression_Testing.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Oncology_Molecular_Analysis_Solid_Hema_Malignancies.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Oncology_Algorithmic_Expression_Testing.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Genappro_GenPnls.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Oncology_Algorithmic_Expression_Testing.pdf\n",
      "nan\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Oncology_Algorithmic_Expression_Testing.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Gen_BRCA1_BRCA2.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Genappro_GenPnls.pdf\n",
      "1\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Genappro_GenPnls.pdf\n",
      "1\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_VectraBldTst_RA.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_GenAppr_GeneticTst.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Oncology_Algorithmic_Expression_Testing.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Oncology_Molecular_Analysis_Solid_Hema_Malignancies.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Oncology_Algorithmic_Expression_Testing.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Oncology_Algorithmic_Expression_Testing.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Oncology_Algorithmic_Expression_Testing.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Oncology_Algorithmic_Expression_Testing.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Oncology_Algorithmic_Expression_Testing.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Gen_BRCA1_BRCA2.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_GenTst_LynchColon.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Oncology_Molecular_Analysis_Solid_Hema_Malignancies.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_GenAppr_GeneticTst.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Genappro_GenPnls.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Genetic_Tst_HyptropCard.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Genetic_Tst_Cardiac_Ion.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Genetic_Tst_Cardiac_Ion.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_GenTst_LynchColon.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_GenTst_LynchColon.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Oncology_Cancer_Screening.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Genappro_GenPnls.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Noninvasive_Tech_EvalMonit_Pts_With%20CLD.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Gen_BRCA1_BRCA2.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Gen_BRCA1_BRCA2.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Ident_Microorg.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Ident_Microorg.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Genappro_GenPnls.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Genappro_GenPnls.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Genappro_GenPnls.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Gen_BRCA1_BRCA2.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Gen_Test_Duchenne_Becker_Mus_Dystrophy.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Genappro_GenPnls.pdf\n",
      "1\n",
      "1\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_GenTst_LynchColon.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Ident_Microorg.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_GenTst_LynchColon.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_GenTst_LynchColon.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Genappro_GenPnls.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Genappro_GenPnls.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Genappro_GenPnls.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_GenTst_LynchColon.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Genappro_GenPnls.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Genappro_GenPnls.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Genappro_GenPnls.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Genappro_GenPnls.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Genappro_GenPnls.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Oncology_Molecular_Analysis_Solid_Hema_Malignancies.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Serologic_Gen_Molecular_Screen_Colorectal_Cancer.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Oncology_Molecular_Analysis_Solid_Hema_Malignancies.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Oncology_Molecular_Analysis_Solid_Hema_Malignancies.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_GenTst_LynchColon.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_GenAppr_GeneticTst.pdf\n",
      "1\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Genappro_GenPnls.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Oncology_Algorithmic_Expression_Testing.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_GenAppr_GeneticTst.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_GenAppr_GeneticTst.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_GenAppr_GeneticTst.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Genappro_GenPnls.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Genappro_GenPnls.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Genappro_GenPnls.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Gen_BRCA1_BRCA2.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Genappro_GenPnls.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Oncology_Molecular_Analysis_Solid_Hema_Malignancies.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Genappro_GenPnls.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Oncology_Molecular_Analysis_Solid_Hema_Malignancies.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Oncology_Molecular_Analysis_Solid_Hema_Malignancies.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Pharmacogen_Metabolite_Markers_IBD.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Cytochrome_p450.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_GenTst_LynchColon.pdf\n",
      "1\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_GenTst_LynchColon.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Gen_Test_Prenatal_Screen_NIPT.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_GenAppr_GeneticTst.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Genappro_GenPnls.pdf\n",
      "1\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Gen_BRCA1_BRCA2.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Gen_Testing_Exome_Genome_Sequencing.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_GenTest_Prenatal_Preconception_Screening.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Genetic_Testing_Rett_Syndrome.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_GenTest_Prenatal_Preconception_Screening.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Gen_BRCA1_BRCA2.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Cytochrome_p450.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_GeneticCncrPnls.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_GeneticCncrPnls.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_GenTest_Prenatal_Preconception_Screening.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_GenTst_LynchColon.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Gen_BRCA1_BRCA2.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Gen_BRCA1_BRCA2.pdf\n",
      "1\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Oncology_Molecular_Analysis_Solid_Hema_Malignancies.pdf\n",
      "1\n",
      "1\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Oncology_Molecular_Analysis_Solid_Hema_Malignancies.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Genappro_GenPnls.pdf\n",
      "1\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Oncology_Molecular_Analysis_Solid_Hema_Malignancies.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_GeneticTst_MitochondrialDisorder.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_GenTest_PrenatalDX_Preg_Loss.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_GeneticTst_MitochondrialDisorder.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Gen_Test_Prenatal_Screen_NIPT.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Gen_Test_Prenatal_Screen_NIPT.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Gen_Test_Prenatal_Screen_NIPT.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Gen_Testing_Exome_Genome_Sequencing.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_GeneticTst_MitochondrialDisorder.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Ident_Microorg.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Ident_Microorg.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Ident_Microorg.pdf\n",
      "1\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Ident_Microorg.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Ident_Microorg.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Oncology_Circulating_Tumor_DNA_Cells_Liquid_Biopsy.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Oncology_Circulating_Tumor_DNA_Cells_Liquid_Biopsy.pdf\n",
      "1\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Gene_Expression_Cutaneous_Melanoma.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Lab_Test_HTR.pdf\n",
      "https://www.blueshieldca.com/bsca/bsc/public/common/PortalComponents/provider/StreamDocumentServlet?fileName=PRV_Lab_Test_HTR.pdf\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Whole+Exome+and+Whole+Genome+Sequencing+for+the+Diagnosis+of+Genetic+Disorders.pdf/187140bc-9ba3-7a80-5e66-85e18760976a?t=1660851213225\n",
      "1\n",
      "https://provider.univerahealthcare.com/documents/d/global/uni-prv-genetic-assay-of-tumor-tissue-to-determine-prognosis-of-breast-cancer-oncotype-dx-breast-recurrence-score-mammaprint-\n",
      "https://provider.univerahealthcare.com/documents/d/global/uni-prv-genetic-assay-of-tumor-tissue-to-determine-prognosis-of-breast-cancer-oncotype-dx-breast-recurrence-score-mammaprint-\n",
      "https://provider.univerahealthcare.com/documents/d/global/uni-prv-genetic-assay-of-tumor-tissue-to-determine-prognosis-of-breast-cancer-oncotype-dx-breast-recurrence-score-mammaprint-\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/gene_express_prost_ca%2Btac%2B18%2B%233.pdf/feb1ef7c-1ad3-7df7-3b67-843d27f7e351?t=1676917919954\n",
      "https://provider.univerahealthcare.com/documents/d/global/uni-prv-genetic-assay-of-tumor-tissue-to-determine-prognosis-of-breast-cancer-oncotype-dx-breast-recurrence-score-mammaprint-\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/gene_express_prost_ca%2Btac%2B18%2B%233.pdf/feb1ef7c-1ad3-7df7-3b67-843d27f7e351?t=1676917919954\n",
      "https://provider.univerahealthcare.com/documents/d/global/uni-prv-genetic-assay-of-tumor-tissue-to-determine-prognosis-of-breast-cancer-oncotype-dx-breast-recurrence-score-mammaprint-\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/gene_express_prost_ca%2Btac%2B18%2B%233.pdf/feb1ef7c-1ad3-7df7-3b67-843d27f7e351?t=1676917919954\n",
      "1\n",
      "https://provider.univerahealthcare.com/documents/d/global/uni-prv-molecular-testing-of-tumor-tissue-to-identify-targeted-therapies-for-cancers\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Molecular+Markers+in+Fine+Needle+Aspirates+of+the+Thyroid.pdf/4bc4174a-ed4a-59b5-4dd8-d6982ecb7063?t=1667317129962\n",
      "1\n",
      "https://provider.univerahealthcare.com/documents/d/global/uni-prv-molecular-testing-of-tumor-tissue-to-identify-targeted-therapies-for-cancers\n",
      "https://provider.univerahealthcare.com/documents/d/global/uni-prv-02-02-55-lab_transplt_reject-tac-23\n",
      "1\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Molecular+Markers+in+Fine+Needle+Aspirates+of+the+Thyroid.pdf/4bc4174a-ed4a-59b5-4dd8-d6982ecb7063?t=1667317129962\n",
      "https://provider.univerahealthcare.com/documents/d/global/uni-prv-molecular-testing-of-tumor-tissue-to-identify-targeted-therapies-for-cancers\n",
      "1\n",
      "1\n",
      "https://provider.univerahealthcare.com/documents/d/global/uni-prv-10-01-05-pros_ca_screen-tac-23\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Genetic+Testing+for+Inherited+Disorders.pdf/8280aec0-7ba6-2cd7-c418-edf7ea7f3353?t=1679926070232\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Gene+Expression+Profiling+for+Cutaneous+Melanoma.pdf/68cc9c89-ff32-761d-f79f-cfe3ea6253e9?t=1668782151186\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Genetic+Testing+for+Hereditary+BRCA+Mutations.pdf/f5183915-4f28-491b-622c-66c973943e2c?t=1678895163082\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Genetic+Testing+for+Inherited+Disorders.pdf/8280aec0-7ba6-2cd7-c418-edf7ea7f3353?t=1679926070232\n",
      "1\n",
      "https://provider.univerahealthcare.com/documents/d/global/uni-prv-genetic-testing-for-cardiac-ion-channelopathies\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Genetic+Testing+for+Inherited+Disorders.pdf/8280aec0-7ba6-2cd7-c418-edf7ea7f3353?t=1679926070232\n",
      "1\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Genetic+Testing+for+Inherited+Susceptibility+to+Colorectal+Cancer.pdf/4fd0d36e-9214-8bc1-1c04-36f4f0421d6f?t=1667316960975\n",
      "https://provider.univerahealthcare.com/documents/d/global/uni-prv-02-02-12-urin_tum_mrkr-archive-tac-23\n",
      "https://provider.univerahealthcare.com/documents/d/global/jak2-mpl-calr-tac-23-1\n",
      "https://provider.univerahealthcare.com/documents/d/global/uni-prv-02-02-12-urin_tum_mrkr-archive-tac-23\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Gene+Expression+Profiling+for+Cutaneous+Melanoma.pdf/68cc9c89-ff32-761d-f79f-cfe3ea6253e9?t=1668782151186\n",
      "https://provider.univerahealthcare.com/documents/d/global/uni-prv-10-01-05-pros_ca_screen-tac-23\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/gene_express_prost_ca%2Btac%2B18%2B%233.pdf/feb1ef7c-1ad3-7df7-3b67-843d27f7e351?t=1676917919954\n",
      "https://provider.univerahealthcare.com/documents/d/global/uni-prv-proteomics-based-testing-for-the-evaluation-of-ovarian-adnexal-masses\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Genetic+Testing+for+Hereditary+BRCA+Mutations.pdf/f5183915-4f28-491b-622c-66c973943e2c?t=1678895163082\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Genetic+Testing+for+Inherited+Susceptibility+to+Colorectal+Cancer.pdf/4fd0d36e-9214-8bc1-1c04-36f4f0421d6f?t=1667316960975\n",
      "https://provider.univerahealthcare.com/documents/d/global/uni-prv-molecular-testing-of-tumor-tissue-to-identify-targeted-therapies-for-cancers\n",
      "1\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Genetic+Testing+for+Susceptibility+to+Hereditary+Cancers.pdf/2bdca948-5961-4ecd-7f1f-a0c5bd5b02a2?t=1676639394409\n",
      "1\n",
      "https://provider.univerahealthcare.com/documents/d/global/uni-prv-genetic-testing-for-cardiac-ion-channelopathies\n",
      "https://provider.univerahealthcare.com/documents/d/global/uni-prv-genetic-testing-for-cardiac-ion-channelopathies\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Genetic+Testing+for+Inherited+Susceptibility+to+Colorectal+Cancer.pdf/4fd0d36e-9214-8bc1-1c04-36f4f0421d6f?t=1667316960975\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Genetic+Testing+for+Inherited+Susceptibility+to+Colorectal+Cancer.pdf/4fd0d36e-9214-8bc1-1c04-36f4f0421d6f?t=1667316960975\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Colorectal+Cancer+Screening.pdf/e8f52bc8-245c-e763-423d-9a92bc2d2a62?t=1663090853820\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Genetic+Testing+for+Inherited+Disorders.pdf/8280aec0-7ba6-2cd7-c418-edf7ea7f3353?t=1679926070232\n",
      "1\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Genetic+Testing+for+Hereditary+BRCA+Mutations.pdf/f5183915-4f28-491b-622c-66c973943e2c?t=1678895163082\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Genetic+Testing+for+Susceptibility+to+Hereditary+Cancers.pdf/2bdca948-5961-4ecd-7f1f-a0c5bd5b02a2?t=1676639394409\n",
      "1\n",
      "1\n",
      "https://provider.univerahealthcare.com/documents/d/global/uni-prv-genetic-testing-for-cardiac-ion-channelopathies\n",
      "https://provider.univerahealthcare.com/documents/d/global/uni-prv-prenatal-genetic-testing\n",
      "https://provider.univerahealthcare.com/documents/d/global/uni-prv-genetic-testing-for-familial-alzheimer-s-disease\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Genetic+Testing+for+Susceptibility+to+Hereditary+Cancers.pdf/2bdca948-5961-4ecd-7f1f-a0c5bd5b02a2?t=1676639394409\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Genetic+Testing+for+Inherited+Disorders.pdf/8280aec0-7ba6-2cd7-c418-edf7ea7f3353?t=1679926070232\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Genetic+Testing+for+Inherited+Disorders.pdf/8280aec0-7ba6-2cd7-c418-edf7ea7f3353?t=1679926070232\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Genetic+Testing+for+Inherited+Disorders.pdf/8280aec0-7ba6-2cd7-c418-edf7ea7f3353?t=1679926070232\n",
      "1\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Genetic+Testing+for+Inherited+Susceptibility+to+Colorectal+Cancer.pdf/4fd0d36e-9214-8bc1-1c04-36f4f0421d6f?t=1667316960975\n",
      "1\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Genetic+Testing+for+Inherited+Susceptibility+to+Colorectal+Cancer.pdf/4fd0d36e-9214-8bc1-1c04-36f4f0421d6f?t=1667316960975\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Genetic+Testing+for+Inherited+Susceptibility+to+Colorectal+Cancer.pdf/4fd0d36e-9214-8bc1-1c04-36f4f0421d6f?t=1667316960975\n",
      "https://provider.univerahealthcare.com/documents/d/global/uni-prv-genetic-testing-for-cardiac-ion-channelopathies\n",
      "1\n",
      "1\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Genetic+Testing+for+Inherited+Susceptibility+to+Colorectal+Cancer.pdf/4fd0d36e-9214-8bc1-1c04-36f4f0421d6f?t=1667316960975\n",
      "1\n",
      "1\n",
      "1\n",
      "https://provider.univerahealthcare.com/documents/d/global/uni-prv-molecular-testing-of-tumor-tissue-to-identify-targeted-therapies-for-cancers\n",
      "https://provider.univerahealthcare.com/documents/d/global/uni-prv-molecular-testing-of-tumor-tissue-to-identify-targeted-therapies-for-cancers\n",
      "https://provider.univerahealthcare.com/documents/d/global/uni-prv-molecular-testing-of-tumor-tissue-to-identify-targeted-therapies-for-cancers\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Colorectal+Cancer+Screening.pdf/e8f52bc8-245c-e763-423d-9a92bc2d2a62?t=1663090853820\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Colorectal+Cancer+Screening.pdf/e8f52bc8-245c-e763-423d-9a92bc2d2a62?t=1663090853820\n",
      "1\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Genetic+Testing+for+Inherited+Susceptibility+to+Colorectal+Cancer.pdf/4fd0d36e-9214-8bc1-1c04-36f4f0421d6f?t=1667316960975\n",
      "1\n",
      "1\n",
      "1\n",
      "https://provider.univerahealthcare.com/documents/d/global/uni-prv-proteomics-based-testing-for-the-evaluation-of-ovarian-adnexal-masses\n",
      "https://provider.univerahealthcare.com/documents/d/global/uni-prv-10-01-05-pros_ca_screen-tac-23\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "https://provider.univerahealthcare.com/documents/d/global/uni-prv-prenatal-genetic-testing\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Genetic+Testing+for+Susceptibility+to+Hereditary+Cancers.pdf/2bdca948-5961-4ecd-7f1f-a0c5bd5b02a2?t=1676639394409\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Genetic+Testing+for+Susceptibility+to+Hereditary+Cancers.pdf/2bdca948-5961-4ecd-7f1f-a0c5bd5b02a2?t=1676639394409\n",
      "1\n",
      "1\n",
      "https://provider.univerahealthcare.com/documents/d/global/uni-prv-molecular-testing-of-tumor-tissue-to-identify-targeted-therapies-for-cancers\n",
      "https://provider.univerahealthcare.com/documents/d/global/uni-prv-molecular-testing-of-tumor-tissue-to-identify-targeted-therapies-for-cancers\n",
      "1\n",
      "https://provider.univerahealthcare.com/documents/d/global/uni-prv-pharmacogenetics\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Genetic+Testing+for+Inherited+Susceptibility+to+Colorectal+Cancer.pdf/4fd0d36e-9214-8bc1-1c04-36f4f0421d6f?t=1667316960975\n",
      "1\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Genetic+Testing+for+Inherited+Susceptibility+to+Colorectal+Cancer.pdf/4fd0d36e-9214-8bc1-1c04-36f4f0421d6f?t=1667316960975\n",
      "https://provider.univerahealthcare.com/documents/d/global/uni-non-invasive-prenatal-testing-terminates-on-2023-09-15-1\n",
      "1\n",
      "1\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Genetic+Testing+for+Inherited+Disorders.pdf/8280aec0-7ba6-2cd7-c418-edf7ea7f3353?t=1679926070232\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Genetic+Testing+for+Hereditary+BRCA+Mutations.pdf/f5183915-4f28-491b-622c-66c973943e2c?t=1678895163082\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Whole+Exome+and+Whole+Genome+Sequencing+for+the+Diagnosis+of+Genetic+Disorders.pdf/187140bc-9ba3-7a80-5e66-85e18760976a?t=1660851213225\n",
      "https://provider.univerahealthcare.com/documents/d/global/uni-prv-02-02-17-gen_test_cf-tac-23\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Genetic+Testing+for+Inherited+Disorders.pdf/8280aec0-7ba6-2cd7-c418-edf7ea7f3353?t=1679926070232\n",
      "https://provider.univerahealthcare.com/documents/d/global/uni-prv-02-02-17-gen_test_cf-tac-23\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Genetic+Testing+for+Hereditary+BRCA+Mutations.pdf/f5183915-4f28-491b-622c-66c973943e2c?t=1678895163082\n",
      "https://provider.univerahealthcare.com/documents/d/global/uni-prv-pharmacogenetics\n",
      "1\n",
      "1\n",
      "https://provider.univerahealthcare.com/documents/d/global/uni-prv-02-02-17-gen_test_cf-tac-23\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Genetic+Testing+for+Inherited+Susceptibility+to+Colorectal+Cancer.pdf/4fd0d36e-9214-8bc1-1c04-36f4f0421d6f?t=1667316960975\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Genetic+Testing+for+Hereditary+BRCA+Mutations.pdf/f5183915-4f28-491b-622c-66c973943e2c?t=1678895163082\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Genetic+Testing+for+Hereditary+BRCA+Mutations.pdf/f5183915-4f28-491b-622c-66c973943e2c?t=1678895163082\n",
      "1\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Genetic+Testing+for+Inherited+Susceptibility+to+Colorectal+Cancer.pdf/4fd0d36e-9214-8bc1-1c04-36f4f0421d6f?t=1667316960975\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "https://provider.univerahealthcare.com/documents/d/global/uni-prv-molecular-testing-of-tumor-tissue-to-identify-targeted-therapies-for-cancers\n",
      "1\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/CMA%2Btac%2B18.pdf/2d85289e-7e20-55c4-8741-f45923d73df4?t=1673644339620\n",
      "1\n",
      "https://provider.univerahealthcare.com/documents/d/global/uni-non-invasive-prenatal-testing-terminates-on-2023-09-15-1\n",
      "https://provider.univerahealthcare.com/documents/d/global/uni-non-invasive-prenatal-testing-terminates-on-2023-09-15-1\n",
      "https://provider.univerahealthcare.com/documents/d/global/uni-non-invasive-prenatal-testing-terminates-on-2023-09-15-1\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Whole+Exome+and+Whole+Genome+Sequencing+for+the+Diagnosis+of+Genetic+Disorders.pdf/187140bc-9ba3-7a80-5e66-85e18760976a?t=1660851213225\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "https://provider.univerahealthcare.com/documents/d/global/uni-prv-circulating-tumor-dna-for-management-of-cancer-liquid-biopsy-\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Gene+Expression+Profiling+for+Cutaneous+Melanoma.pdf/68cc9c89-ff32-761d-f79f-cfe3ea6253e9?t=1668782151186\n",
      "https://provider.univerahealthcare.com/documents/d/global/uni-prv-measurable-residual-disease-assessment-testing\n",
      "https://provider.univerahealthcare.com/documents/20152/127124/Gene+Expression+Profiling+for+Cutaneous+Melanoma.pdf/68cc9c89-ff32-761d-f79f-cfe3ea6253e9?t=1668782151186\n",
      "https://provider.univerahealthcare.com/documents/d/global/uni-prv-02-02-55-lab_transplt_reject-tac-23\n",
      "https://provider.univerahealthcare.com/documents/d/global/uni-prv-02-02-55-lab_transplt_reject-tac-23\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_231364\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232428\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232428\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232428\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232428\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232428\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232428\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232428\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232428\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232428\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232428\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232428\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232428\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232428\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232482\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232485\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232428\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232428\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232482\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232428\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232711\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232428\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232484\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232428\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_234351\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232484\n",
      "1\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232484\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_234412\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232491\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232707\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232428\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232482\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232428\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232428\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232428\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232428\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232428\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232707\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232707\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232482\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232489\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232707\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232485\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232485\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232485\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232707\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232707\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=AENTRY_046106\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232484\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232488\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_234351\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232707\n",
      "1\n",
      "1\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232484\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232482\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232484\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232707\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232485\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232484\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_234412\n",
      "1\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232707\n",
      "1\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232707\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232707\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232484\n",
      "1\n",
      "1\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232707\n",
      "1\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232489\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232482\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232482\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232482\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232482\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232429\n",
      "1\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232482\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232707\n",
      "1\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232488\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232483\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232428\n",
      "1\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232482\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232482\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232482\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232712\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232483\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232707\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232707\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232431\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232482\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232482\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232482\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232712\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232712\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232707\n",
      "1\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232707\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232426\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232489\n",
      "1\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_234412\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_234351\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_231364\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232417\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_234412\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232488\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_234351\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232712\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232707\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232707\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232488\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232707\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_234351\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_234351\n",
      "1\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232482\n",
      "1\n",
      "1\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232482\n",
      "1\n",
      "1\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232482\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232710\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232711\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232710\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232426\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232426\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232426\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_231364\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232710\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232430\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232430\n",
      "1\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232428\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232708\n",
      "https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232708\n",
      "https://embed.widencdn.net/pdf/plus/upmc/pipcd9vggo/MP.PA.099.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/kmjvgfuvbo/MP.117.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/0pxm3nshak/MP.PA.018.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/0pxm3nshak/MP.PA.018.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/0pxm3nshak/MP.PA.018.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/bijymqsoml/MP.PA.130.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/0pxm3nshak/MP.PA.018.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/bijymqsoml/MP.PA.130.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/0pxm3nshak/MP.PA.018.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/bijymqsoml/MP.PA.130.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/bijymqsoml/MP.PA.130.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/y9ctklvo9h/MP.PA.055.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/5iuxtdui8o/MP.128.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/jvp6ubxqcz/MP.115.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/bijymqsoml/MP.PA.130.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/hsns0dlcyr/MP.A063.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/shonttrvg6/MP.088.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/5iuxtdui8o/MP.128.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/bijymqsoml/MP.PA.130.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/jvp6ubxqcz/MP.115.pdf\n",
      "1\n",
      "https://embed.widencdn.net/pdf/plus/upmc/tfvfd05fhb/MP.155.pdf\n",
      "1\n",
      "https://embed.widencdn.net/pdf/plus/upmc/kmjvgfuvbo/MP.117.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/y9ctklvo9h/MP.PA.055.pdf\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "https://embed.widencdn.net/pdf/plus/upmc/jltfwatxnk/MP.A040.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/qjjc5fmjcz/MP.042.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/wflbcswstw/MP.A039.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/bijymqsoml/MP.PA.130.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/wflbcswstw/MP.A039.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/kmjvgfuvbo/MP.117.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/tfvfd05fhb/MP.155.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/bijymqsoml/MP.PA.130.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/bijymqsoml/MP.PA.130.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/y9ctklvo9h/MP.PA.055.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/qjjc5fmjcz/MP.042.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/bijymqsoml/MP.PA.130.pdf\n",
      "1\n",
      "https://embed.widencdn.net/pdf/plus/upmc/h8rhzntidh/MP.PA.110.pdf\n",
      "1\n",
      "1\n",
      "1\n",
      "https://embed.widencdn.net/pdf/plus/upmc/qjjc5fmjcz/MP.042.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/qjjc5fmjcz/MP.042.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/qsclgja98j/MP.A075.pdf\n",
      "1\n",
      "1\n",
      "https://embed.widencdn.net/pdf/plus/upmc/y9ctklvo9h/MP.PA.055.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/y9ctklvo9h/MP.PA.055.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/flwajpwyvw/MP.A058.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/mdefbarizr/MP.A046.pdf\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "https://embed.widencdn.net/pdf/plus/upmc/8x1fc0cmkj/MP.A015.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/qjjc5fmjcz/MP.042.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/mdefbarizr/MP.A046.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/qjjc5fmjcz/MP.042.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/qjjc5fmjcz/MP.042.pdf\n",
      "1\n",
      "1\n",
      "1\n",
      "https://embed.widencdn.net/pdf/plus/upmc/qjjc5fmjcz/MP.042.pdf\n",
      "1\n",
      "1\n",
      "1\n",
      "https://embed.widencdn.net/pdf/plus/upmc/shonttrvg6/MP.088.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/shonttrvg6/MP.088.pdf\n",
      "1\n",
      "https://embed.widencdn.net/pdf/plus/upmc/qsclgja98j/MP.A075.pdf\n",
      "1\n",
      "1\n",
      "https://embed.widencdn.net/pdf/plus/upmc/qjjc5fmjcz/MP.042.pdf\n",
      "1\n",
      "1\n",
      "1\n",
      "nan\n",
      "https://embed.widencdn.net/pdf/plus/upmc/tfvfd05fhb/MP.155.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/1vd0ceswee/MP.PA.097.pdf\n",
      "1\n",
      "1\n",
      "https://embed.widencdn.net/pdf/plus/upmc/o3402gqrpo/MP.143.pdf\n",
      "1\n",
      "1\n",
      "https://embed.widencdn.net/pdf/plus/upmc/h8rhzntidh/MP.PA.110.pdf\n",
      "1\n",
      "1\n",
      "https://embed.widencdn.net/pdf/plus/upmc/shonttrvg6/MP.088.pdf\n",
      "1\n",
      "https://embed.widencdn.net/pdf/plus/upmc/o3402gqrpo/MP.143.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/o3402gqrpo/MP.143.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/qjjc5fmjcz/MP.042.pdf\n",
      "1\n",
      "https://embed.widencdn.net/pdf/plus/upmc/qjjc5fmjcz/MP.042.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/c8z6opk9xe/MP.A023.pdf\n",
      "1\n",
      "1\n",
      "1\n",
      "https://embed.widencdn.net/pdf/plus/upmc/y9ctklvo9h/MP.PA.055.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/pipcd9vggo/MP.PA.099.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/zvj9in3sno/MP.116.pdf\n",
      "1\n",
      "https://embed.widencdn.net/pdf/plus/upmc/zvj9in3sno/MP.116.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/y9ctklvo9h/MP.PA.055.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/o3402gqrpo/MP.143.pdf\n",
      "1\n",
      "1\n",
      "https://embed.widencdn.net/pdf/plus/upmc/zvj9in3sno/MP.116.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/o3402gqrpo/MP.143.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/y9ctklvo9h/MP.PA.055.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/y9ctklvo9h/MP.PA.055.pdf\n",
      "1\n",
      "https://embed.widencdn.net/pdf/plus/upmc/shonttrvg6/MP.088.pdf\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "https://embed.widencdn.net/pdf/plus/upmc/jvp6ubxqcz/MP.115.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/tfvfd05fhb/MP.155.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/cfb3ennouv/MP.PA.098.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/tfvfd05fhb/MP.155.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/c8z6opk9xe/MP.A023.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/c8z6opk9xe/MP.A023.pdf\n",
      "1\n",
      "https://embed.widencdn.net/pdf/plus/upmc/pipcd9vggo/MP.PA.099.pdf\n",
      "1\n",
      "https://embed.widencdn.net/pdf/plus/upmc/mdefbarizr/MP.A046.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/mdefbarizr/MP.A046.pdf\n",
      "1\n",
      "1\n",
      "https://embed.widencdn.net/pdf/plus/upmc/mdefbarizr/MP.A046.pdf\n",
      "1\n",
      "https://embed.widencdn.net/pdf/plus/upmc/bijymqsoml/MP.PA.130.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/bijymqsoml/MP.PA.130.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/bijymqsoml/MP.PA.130.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/kmjvgfuvbo/MP.117.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/hsns0dlcyr/MP.A063.pdf\n",
      "https://embed.widencdn.net/pdf/plus/upmc/hsns0dlcyr/MP.A063.pdf\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.053.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.090.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.035.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.035.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.035.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.051.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.035.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.051.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.035.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.051.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.047.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.030.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.052.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.087.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.026.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.001.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.015.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.052.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.009.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.075.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.023.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.034.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.023.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.097.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.030.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.072.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.021.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.002.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.050.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.076.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.028.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.001.013.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.064.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.001.013.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.097.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.034.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.034.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.003.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.030.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.028.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.026.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.023.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.016.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.084.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.042.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.042.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.028.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.028.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.032.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.072.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.001.009.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.030.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.030.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.001.007.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.001.007.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.059.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.023.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.050.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.027.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.066.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.050.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.070.html\n",
      "1\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.028.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.001.007.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.028.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.028.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.107.html\n",
      "1\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.060.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.028.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.023.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.056.html\n",
      "1\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.004.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.004.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.060.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.133.html\n",
      "1\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.063.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.028.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.023.html\n",
      "1\n",
      "1\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.003.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.034.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.060.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.060.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.060.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.073.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.023.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.027.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.016.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.063.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.023.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.004.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.060.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.039.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.008.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.028.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.060.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.028.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.078.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.070.html\n",
      "1\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.068.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.030.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.053.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.070.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.013.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.070.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.030.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.008.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.064.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.064.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.070.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.028.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.030.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.030.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.049.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.004.html\n",
      "1\n",
      "1\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.060.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.009.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.060.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.009.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.093.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.018.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.093.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.078.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.078.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.078.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.053.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.093.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.001.007.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.001.007.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.001.007.html\n",
      "1\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.001.007.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.001.007.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.089.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.089.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.136.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.097.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.001.html\n",
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.001.html\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI698:00\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI038:00\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI129:00\n",
      "1\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI129:00\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI038:00\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI129:00\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI038:00\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI129:00\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI038:00\n",
      "1\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI508:00\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI038:00\n",
      "1\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI698:00\n",
      "1\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI129:00\n",
      "1\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI698:00\n",
      "1\n",
      "1\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI129:00\n",
      "1\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI038:00\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI508:00\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI698:00\n",
      "1\n",
      "1\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI129:00\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI129:00\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI355:00\n",
      "1\n",
      "1\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI698:00\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI062:00\n",
      "1\n",
      "1\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI508:00\n",
      "1\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI685:00\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI222:00\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI698:00\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI698:00\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI698:00\n",
      "1\n",
      "1\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI698:00\n",
      "1\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI222:00\n",
      "1\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI685:00\n",
      "1\n",
      "1\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI698:00\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI698:00\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI698:00\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI698:00\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI698:00\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI062:00\n",
      "1\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI698:00\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI129:00\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI698:00\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI698:00\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI698:00\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI581:00\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI698:00\n",
      "1\n",
      "1\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI698:00\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI698:00\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI698:00\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI698:00\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI581:00\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI581:00\n",
      "1\n",
      "1\n",
      "1\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI205:00\n",
      "1\n",
      "1\n",
      "1\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI508:00\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI698:00\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI617:00\n",
      "1\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI617:00\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI508:00\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI581:00\n",
      "1\n",
      "1\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI617:00\n",
      "1\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI508:00\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI508:00\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI581:00\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI698:00\n",
      "1\n",
      "1\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI698:00\n",
      "1\n",
      "1\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI698:00\n",
      "1\n",
      "1\n",
      "1\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI205:00\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI205:00\n",
      "1\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI698:00\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI222:00\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI685:00\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI222:00\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI697:00\n",
      "https://documents.qualchoice.com/area/shared/MedicalPolicies.aspx?sop=BI697:00\n",
      "1\n",
      "1\n",
      "1\n",
      "1\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/dac88083-74b7-7488-37d6-2209a0452261\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/a52fe0cd-cab7-9c16-dfc5-73d27ce091d1\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/8e02d63e-ffae-dab6-87c9-400027432008\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/8e02d63e-ffae-dab6-87c9-400027432008\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/8e02d63e-ffae-dab6-87c9-400027432008\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/b3760f5f-28c1-a4d7-b307-f9d78f780b9b\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/8e02d63e-ffae-dab6-87c9-400027432008\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/b3760f5f-28c1-a4d7-b307-f9d78f780b9b\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/8e02d63e-ffae-dab6-87c9-400027432008\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/b3760f5f-28c1-a4d7-b307-f9d78f780b9b\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/f11c75be-64e6-f3db-0c47-1f350014dd09\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/803b4942-79bd-521d-0eb6-b686e68f9c5b\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/71cd6c08-ebbf-9898-a894-25a9eae64419\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/bc68d9cc-6a71-e827-df0c-c6e1a4629677\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/e66d2639-94fc-fd85-b385-278ea7cdd546\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/64db30f1-4fe6-7757-1697-7d4c9e16a386\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/2b1f8c00-fcbd-021f-674f-492999193c2b\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/71cd6c08-ebbf-9898-a894-25a9eae64419\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/e66d2639-94fc-fd85-b385-278ea7cdd546\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/bd89c577-1ccb-a8c7-d9d5-bdee366177f1\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/44594848-7508-767b-4104-fa6a86dd52aa\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/c4603529-e17a-ceda-b8e6-7136d2e0f075\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/6047081a-2660-2bd2-de9c-4e8a9b563569\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/5ead5531-e3bf-811e-1604-1a9ae3b61a05\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/803b4942-79bd-521d-0eb6-b686e68f9c5b\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/1d8ae3cb-f65e-c96e-5e28-a9fa691525c5\n",
      "1\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/6047081a-2660-2bd2-de9c-4e8a9b563569\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/7f312b62-8c3d-da41-bde0-68ba7dfc9a5b\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/c753941c-5f62-f603-2792-b71836bdaed5\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/00e2bf4a-0305-2c9e-caf9-779e090d2ad0\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/b2194861-398e-baf2-1292-a7e93f23acba\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/e66d2639-94fc-fd85-b385-278ea7cdd546\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/b2194861-398e-baf2-1292-a7e93f23acba\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/5ead5531-e3bf-811e-1604-1a9ae3b61a05\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/c4603529-e17a-ceda-b8e6-7136d2e0f075\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/c4603529-e17a-ceda-b8e6-7136d2e0f075\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/bca23301-ce79-d3c7-9b35-623d0dc4bad4\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/803b4942-79bd-521d-0eb6-b686e68f9c5b\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/00e2bf4a-0305-2c9e-caf9-779e090d2ad0\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/e66d2639-94fc-fd85-b385-278ea7cdd546\n",
      "1\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/ee07721f-23b9-8209-2c67-7f3272f07c9c\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/213c295b-4e2f-a5a3-625b-76b16b1e3ef3\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/6047081a-2660-2bd2-de9c-4e8a9b563569\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/6047081a-2660-2bd2-de9c-4e8a9b563569\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/00e2bf4a-0305-2c9e-caf9-779e090d2ad0\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/00e2bf4a-0305-2c9e-caf9-779e090d2ad0\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/518cd66a-56e5-21df-7dd6-50bd457315b8\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/1d8ae3cb-f65e-c96e-5e28-a9fa691525c5\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/0f487b45-c387-ed3a-2bbc-d773c9e693f5\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/803b4942-79bd-521d-0eb6-b686e68f9c5b\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/803b4942-79bd-521d-0eb6-b686e68f9c5b\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/4a2e5d32-8700-4d2d-be24-ef36c6678511\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/4a2e5d32-8700-4d2d-be24-ef36c6678511\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/2b1e130f-972f-f6e5-8c62-2fe16984018d\n",
      "1\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/2b1e130f-972f-f6e5-8c62-2fe16984018d\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/803b4942-79bd-521d-0eb6-b686e68f9c5b\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/e5963175-6879-b530-30c5-da7b301392f0\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/7f312b62-8c3d-da41-bde0-68ba7dfc9a5b\n",
      "1\n",
      "1\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/00e2bf4a-0305-2c9e-caf9-779e090d2ad0\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/4a2e5d32-8700-4d2d-be24-ef36c6678511\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/00e2bf4a-0305-2c9e-caf9-779e090d2ad0\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/00e2bf4a-0305-2c9e-caf9-779e090d2ad0\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/6047081a-2660-2bd2-de9c-4e8a9b563569\n",
      "1\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/74f5260b-b1d4-7d46-f716-d824691117a6\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/00e2bf4a-0305-2c9e-caf9-779e090d2ad0\n",
      "1\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/44e8e81b-8828-8f3b-3a55-0c70edcdd9ec\n",
      "1\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/b9d8beca-4d68-f601-394e-6b00bea56880\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/b9d8beca-4d68-f601-394e-6b00bea56880\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/74f5260b-b1d4-7d46-f716-d824691117a6\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/de982e1c-2ca2-0b30-3f93-b4208b6e0e31\n",
      "1\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/d309fdc5-56f5-6004-8add-25842c857e61\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/00e2bf4a-0305-2c9e-caf9-779e090d2ad0\n",
      "1\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/e14e4cfd-5033-dd20-111f-4063490ece1e\n",
      "1\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/bca23301-ce79-d3c7-9b35-623d0dc4bad4\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/c4603529-e17a-ceda-b8e6-7136d2e0f075\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/74f5260b-b1d4-7d46-f716-d824691117a6\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/74f5260b-b1d4-7d46-f716-d824691117a6\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/74f5260b-b1d4-7d46-f716-d824691117a6\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/b670c16d-72d6-8a9d-67bb-3b69611fafc5\n",
      "1\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/803b4942-79bd-521d-0eb6-b686e68f9c5b\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/ee07721f-23b9-8209-2c67-7f3272f07c9c\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/d309fdc5-56f5-6004-8add-25842c857e61\n",
      "1\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/b9d8beca-4d68-f601-394e-6b00bea56880\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/74f5260b-b1d4-7d46-f716-d824691117a6\n",
      "1\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/fe71f357-3242-0ce4-d356-3df00cc07c7f\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/00e2bf4a-0305-2c9e-caf9-779e090d2ad0\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/61af99d8-290b-52ba-ae85-091bec60f081\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/00e2bf4a-0305-2c9e-caf9-779e090d2ad0\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/76cdf453-f0ff-f66a-6dba-a604f0446e43\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/e5b59643-a571-30ba-1f71-7a8beb9dada3\n",
      "1\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/d3cef78b-0ff1-8df9-3f6a-138a610303e8\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/803b4942-79bd-521d-0eb6-b686e68f9c5b\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/dac88083-74b7-7488-37d6-2209a0452261\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/86c068d3-c6ba-5839-8ee0-eb41512f6859\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/5e47b118-c178-8400-95c3-27b4fc305dfd\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/86c068d3-c6ba-5839-8ee0-eb41512f6859\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/803b4942-79bd-521d-0eb6-b686e68f9c5b\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/fe71f357-3242-0ce4-d356-3df00cc07c7f\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/d9429b59-4259-c078-0e74-ece0207ebc87\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/d9429b59-4259-c078-0e74-ece0207ebc87\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/86c068d3-c6ba-5839-8ee0-eb41512f6859\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/00e2bf4a-0305-2c9e-caf9-779e090d2ad0\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/803b4942-79bd-521d-0eb6-b686e68f9c5b\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/803b4942-79bd-521d-0eb6-b686e68f9c5b\n",
      "1\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/dcfed128-eea5-8941-0d0b-2fc817cf820e\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/e14e4cfd-5033-dd20-111f-4063490ece1e\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/e14e4cfd-5033-dd20-111f-4063490ece1e\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/e66d2639-94fc-fd85-b385-278ea7cdd546\n",
      "1\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/74f5260b-b1d4-7d46-f716-d824691117a6\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/51eb8d19-8261-e441-1ba4-a69ba964e708\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/d358870c-8917-dd73-90f3-ea3372bab751\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/9fb1ca85-fb92-83fa-7a66-88444705d10d\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/d358870c-8917-dd73-90f3-ea3372bab751\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/76cdf453-f0ff-f66a-6dba-a604f0446e43\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/76cdf453-f0ff-f66a-6dba-a604f0446e43\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/76cdf453-f0ff-f66a-6dba-a604f0446e43\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/dac88083-74b7-7488-37d6-2209a0452261\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/d358870c-8917-dd73-90f3-ea3372bab751\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/4a2e5d32-8700-4d2d-be24-ef36c6678511\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/4a2e5d32-8700-4d2d-be24-ef36c6678511\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/4a2e5d32-8700-4d2d-be24-ef36c6678511\n",
      "1\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/4a2e5d32-8700-4d2d-be24-ef36c6678511\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/4a2e5d32-8700-4d2d-be24-ef36c6678511\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/51eb8d19-8261-e441-1ba4-a69ba964e708\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/51eb8d19-8261-e441-1ba4-a69ba964e708\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/0347e12a-aacc-f780-4de6-808f6ceaa6f8\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/5ead5531-e3bf-811e-1604-1a9ae3b61a05\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/64db30f1-4fe6-7757-1697-7d4c9e16a386\n",
      "https://www.horizonblue.com/providers/policies-procedures/policies/medical-policies/detail/64db30f1-4fe6-7757-1697-7d4c9e16a386\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/de0bc2c8-a1f7-4a51-98a3-015a70fc4d8d.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/93d8a8f8-ea45-4d58-8963-0a37b95b1ec7.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/07b6094a-2341-4bed-a1b8-841e3fe33dcc.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/07b6094a-2341-4bed-a1b8-841e3fe33dcc.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/07b6094a-2341-4bed-a1b8-841e3fe33dcc.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/ffe67992-36e2-4b42-869e-6819c4b8b7da.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/07b6094a-2341-4bed-a1b8-841e3fe33dcc.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/ffe67992-36e2-4b42-869e-6819c4b8b7da.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/07b6094a-2341-4bed-a1b8-841e3fe33dcc.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/ffe67992-36e2-4b42-869e-6819c4b8b7da.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/3f9b0311-f500-4001-a312-ab00093be372.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/890ec48f-0c2a-4d5d-9ae9-0a5d7d109085.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/93d8a8f8-ea45-4d58-8963-0a37b95b1ec7.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/fec06f5d-e205-4439-ae58-f7a7d1444ffa.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/3f9b0311-f500-4001-a312-ab00093be372.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/418d9a77-331d-4b0e-81fd-b74f8fdcec75.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/56cef71d-72dd-45ed-ab0c-e989f8be35a9.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/3f9b0311-f500-4001-a312-ab00093be372.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/3f9b0311-f500-4001-a312-ab00093be372.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/d2fef196-2c5b-426c-97f6-7ca9da8f5ded.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/3f9b0311-f500-4001-a312-ab00093be372.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/93d8a8f8-ea45-4d58-8963-0a37b95b1ec7.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/d926697a-f48a-43ac-8d16-0854c42eb35a.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/3f9b0311-f500-4001-a312-ab00093be372.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/890ec48f-0c2a-4d5d-9ae9-0a5d7d109085.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/3f9b0311-f500-4001-a312-ab00093be372.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/3f9b0311-f500-4001-a312-ab00093be372.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/d926697a-f48a-43ac-8d16-0854c42eb35a.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/93d8a8f8-ea45-4d58-8963-0a37b95b1ec7.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/0d3808ad-e83f-456d-bef7-28411893c198.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/051c4dde-6bb6-481c-9df8-85ac1605e940.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/ffe67992-36e2-4b42-869e-6819c4b8b7da.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/3f9b0311-f500-4001-a312-ab00093be372.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/ffe67992-36e2-4b42-869e-6819c4b8b7da.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/3f9b0311-f500-4001-a312-ab00093be372.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/d1cb83f2-a22c-48f7-94e4-1031607efea8.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/3f9b0311-f500-4001-a312-ab00093be372.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/a3d1db2d-92eb-4b1e-b074-b095bb9d2eb3.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/3f9b0311-f500-4001-a312-ab00093be372.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/051c4dde-6bb6-481c-9df8-85ac1605e940.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/3f9b0311-f500-4001-a312-ab00093be372.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/93d8a8f8-ea45-4d58-8963-0a37b95b1ec7.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/93d8a8f8-ea45-4d58-8963-0a37b95b1ec7.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/93d8a8f8-ea45-4d58-8963-0a37b95b1ec7.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/354c8f95-2e3a-438f-8983-cbfc8cd3b309.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/354c8f95-2e3a-438f-8983-cbfc8cd3b309.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/051c4dde-6bb6-481c-9df8-85ac1605e940.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/051c4dde-6bb6-481c-9df8-85ac1605e940.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/5679ec89-5fbf-40f0-923c-6684f1f0cde1.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/3f9b0311-f500-4001-a312-ab00093be372.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/fd101826-54fe-400c-890f-b730a793bf36.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/890ec48f-0c2a-4d5d-9ae9-0a5d7d109085.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/3f9b0311-f500-4001-a312-ab00093be372.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/NewsArticles/89689bbb-5b37-4282-b90a-69b3d315792d.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/93d8a8f8-ea45-4d58-8963-0a37b95b1ec7.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/d926697a-f48a-43ac-8d16-0854c42eb35a.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/d926697a-f48a-43ac-8d16-0854c42eb35a.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/d926697a-f48a-43ac-8d16-0854c42eb35a.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/93d8a8f8-ea45-4d58-8963-0a37b95b1ec7.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/93d8a8f8-ea45-4d58-8963-0a37b95b1ec7.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/d926697a-f48a-43ac-8d16-0854c42eb35a.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/93d8a8f8-ea45-4d58-8963-0a37b95b1ec7.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/d926697a-f48a-43ac-8d16-0854c42eb35a.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/93d8a8f8-ea45-4d58-8963-0a37b95b1ec7.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/d926697a-f48a-43ac-8d16-0854c42eb35a.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/051c4dde-6bb6-481c-9df8-85ac1605e940.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/051c4dde-6bb6-481c-9df8-85ac1605e940.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/d926697a-f48a-43ac-8d16-0854c42eb35a.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/d926697a-f48a-43ac-8d16-0854c42eb35a.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/d926697a-f48a-43ac-8d16-0854c42eb35a.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/051c4dde-6bb6-481c-9df8-85ac1605e940.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/93d8a8f8-ea45-4d58-8963-0a37b95b1ec7.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/93d8a8f8-ea45-4d58-8963-0a37b95b1ec7.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/d926697a-f48a-43ac-8d16-0854c42eb35a.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/93d8a8f8-ea45-4d58-8963-0a37b95b1ec7.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/93d8a8f8-ea45-4d58-8963-0a37b95b1ec7.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/93d8a8f8-ea45-4d58-8963-0a37b95b1ec7.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/5679ec89-5fbf-40f0-923c-6684f1f0cde1.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/93d8a8f8-ea45-4d58-8963-0a37b95b1ec7.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/d926697a-f48a-43ac-8d16-0854c42eb35a.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/051c4dde-6bb6-481c-9df8-85ac1605e940.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/93d8a8f8-ea45-4d58-8963-0a37b95b1ec7.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/d926697a-f48a-43ac-8d16-0854c42eb35a.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/d926697a-f48a-43ac-8d16-0854c42eb35a.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/a3d1db2d-92eb-4b1e-b074-b095bb9d2eb3.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/93d8a8f8-ea45-4d58-8963-0a37b95b1ec7.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/d926697a-f48a-43ac-8d16-0854c42eb35a.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/93d8a8f8-ea45-4d58-8963-0a37b95b1ec7.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/93d8a8f8-ea45-4d58-8963-0a37b95b1ec7.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/93d8a8f8-ea45-4d58-8963-0a37b95b1ec7.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/d926697a-f48a-43ac-8d16-0854c42eb35a.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/93d8a8f8-ea45-4d58-8963-0a37b95b1ec7.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/93d8a8f8-ea45-4d58-8963-0a37b95b1ec7.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/93d8a8f8-ea45-4d58-8963-0a37b95b1ec7.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/d926697a-f48a-43ac-8d16-0854c42eb35a.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/93d8a8f8-ea45-4d58-8963-0a37b95b1ec7.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/93d8a8f8-ea45-4d58-8963-0a37b95b1ec7.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/93d8a8f8-ea45-4d58-8963-0a37b95b1ec7.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/5d0b97aa-679f-499f-8b73-6e16b6e5ba94.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/051c4dde-6bb6-481c-9df8-85ac1605e940.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/93d8a8f8-ea45-4d58-8963-0a37b95b1ec7.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/051c4dde-6bb6-481c-9df8-85ac1605e940.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/3f9b0311-f500-4001-a312-ab00093be372.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/93d8a8f8-ea45-4d58-8963-0a37b95b1ec7.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/3f9b0311-f500-4001-a312-ab00093be372.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/3f9b0311-f500-4001-a312-ab00093be372.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/890ec48f-0c2a-4d5d-9ae9-0a5d7d109085.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/de0bc2c8-a1f7-4a51-98a3-015a70fc4d8d.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/93d8a8f8-ea45-4d58-8963-0a37b95b1ec7.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/93d8a8f8-ea45-4d58-8963-0a37b95b1ec7.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/93d8a8f8-ea45-4d58-8963-0a37b95b1ec7.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/890ec48f-0c2a-4d5d-9ae9-0a5d7d109085.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/5d0b97aa-679f-499f-8b73-6e16b6e5ba94.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/3f9b0311-f500-4001-a312-ab00093be372.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/3f9b0311-f500-4001-a312-ab00093be372.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/93d8a8f8-ea45-4d58-8963-0a37b95b1ec7.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/051c4dde-6bb6-481c-9df8-85ac1605e940.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/890ec48f-0c2a-4d5d-9ae9-0a5d7d109085.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/890ec48f-0c2a-4d5d-9ae9-0a5d7d109085.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/3f9b0311-f500-4001-a312-ab00093be372.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/051c4dde-6bb6-481c-9df8-85ac1605e940.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/d926697a-f48a-43ac-8d16-0854c42eb35a.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/d926697a-f48a-43ac-8d16-0854c42eb35a.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/93d8a8f8-ea45-4d58-8963-0a37b95b1ec7.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/93d8a8f8-ea45-4d58-8963-0a37b95b1ec7.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/93d8a8f8-ea45-4d58-8963-0a37b95b1ec7.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/93d8a8f8-ea45-4d58-8963-0a37b95b1ec7.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/3f9b0311-f500-4001-a312-ab00093be372.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/93d8a8f8-ea45-4d58-8963-0a37b95b1ec7.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/3f9b0311-f500-4001-a312-ab00093be372.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/3d222c19-6bda-4fa0-af3c-0a8e4340e2d9.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/3d222c19-6bda-4fa0-af3c-0a8e4340e2d9.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/93d8a8f8-ea45-4d58-8963-0a37b95b1ec7.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/d926697a-f48a-43ac-8d16-0854c42eb35a.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/3f9b0311-f500-4001-a312-ab00093be372.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/3f9b0311-f500-4001-a312-ab00093be372.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/93d8a8f8-ea45-4d58-8963-0a37b95b1ec7.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/NewsArticles/89689bbb-5b37-4282-b90a-69b3d315792d.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/77d84dc3-aba8-46c8-b12c-fe6fc6698b0e.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/d926697a-f48a-43ac-8d16-0854c42eb35a.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/77d84dc3-aba8-46c8-b12c-fe6fc6698b0e.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/93d8a8f8-ea45-4d58-8963-0a37b95b1ec7.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/93d8a8f8-ea45-4d58-8963-0a37b95b1ec7.aspx\n",
      "1\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/3f9b0311-f500-4001-a312-ab00093be372.aspx\n",
      "https://medpolicy.ibx.com/ibc/Commercial/Pages/Policy/3f9b0311-f500-4001-a312-ab00093be372.aspx\n",
      "1\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n",
      "nan\n"
     ]
    }
   ],
   "source": [
    "payors_info={}\n",
    "payors=[]\n",
    "URL=[]\n",
    "for i in data['Policy url']:\n",
    "    print(i)\n",
    "    '''\n",
    "    if i[-4:]==\"aspx\":\n",
    "        payors.append(data[\"Payors/LBM\"])\n",
    "        URL.append(data['Policy url'])\n",
    "       \n",
    "        #print(\"Payor: \",data[\"Payors/LBM\"],\"\\nURL: \",data['Policy url'])'''"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "6b1d1b30",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "9dc0c801",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "ae471b1b",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "e417919a",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "126089e0",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "c66d3ea2",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "4f1fbfa4",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "082c711f",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "37aeddb1",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "raw",
   "id": "f3567f7f",
   "metadata": {},
   "source": []
  },
  {
   "cell_type": "markdown",
   "id": "21893a65",
   "metadata": {},
   "source": [
    "## Keyword Matching and extraction"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "09ea440d",
   "metadata": {},
   "source": [
    "#### Working on the new flagged working link data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "id": "8bac9c60",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Unnamed: 0</th>\n",
       "      <th>Keywords</th>\n",
       "      <th>Payors/LBM</th>\n",
       "      <th>Policy url</th>\n",
       "      <th>Policy URL Correct/Incorrect Flag</th>\n",
       "      <th>\"Covered/Yes\" Language</th>\n",
       "      <th>\"Not Covered/No\" Language</th>\n",
       "      <th>Vague / non-No</th>\n",
       "      <th>\"Conditional/Need PA\" language</th>\n",
       "      <th>Notable patterns within document</th>\n",
       "      <th>Unnamed: 9</th>\n",
       "      <th>extracted text</th>\n",
       "      <th>Validity Status</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0</td>\n",
       "      <td>81521 OR MammaPrint Breast Cancer Recurrence S...</td>\n",
       "      <td>Aetna</td>\n",
       "      <td>https://www.aetna.com/cpb/medical/data/300_399...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>BrCA testing  is covered,</td>\n",
       "      <td>NaN</td>\n",
       "      <td>not listed in investigational and experimental</td>\n",
       "      <td>NaN</td>\n",
       "      <td>81521 and MammaPrint do not appear in policy, ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GENE.00052 Whole Genome Sequencing, Whole Exom...</td>\n",
       "      <td>HTML - Working</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1</td>\n",
       "      <td>81552 OR DecisionDx-UM OR Gene expression prof...</td>\n",
       "      <td>Aetna</td>\n",
       "      <td>https://www.aetna.com/cpb/medical/data/300_399...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>?</td>\n",
       "      <td></td>\n",
       "      <td></td>\n",
       "      <td>NaN</td>\n",
       "      <td>81552  and test name do not appear in list of ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GENE.00052 Whole Genome Sequencing, Whole Exom...</td>\n",
       "      <td>HTML - Working</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   Unnamed: 0                                           Keywords Payors/LBM  \\\n",
       "0           0  81521 OR MammaPrint Breast Cancer Recurrence S...      Aetna   \n",
       "1           1  81552 OR DecisionDx-UM OR Gene expression prof...      Aetna   \n",
       "\n",
       "                                          Policy url  \\\n",
       "0  https://www.aetna.com/cpb/medical/data/300_399...   \n",
       "1  https://www.aetna.com/cpb/medical/data/300_399...   \n",
       "\n",
       "   Policy URL Correct/Incorrect Flag      \"Covered/Yes\" Language  \\\n",
       "0                                NaN  BrCA testing  is covered,    \n",
       "1                                NaN                           ?   \n",
       "\n",
       "  \"Not Covered/No\" Language                                  Vague / non-No  \\\n",
       "0                       NaN  not listed in investigational and experimental   \n",
       "1                                                                             \n",
       "\n",
       "  \"Conditional/Need PA\" language  \\\n",
       "0                            NaN   \n",
       "1                            NaN   \n",
       "\n",
       "                    Notable patterns within document Unnamed: 9  \\\n",
       "0  81521 and MammaPrint do not appear in policy, ...        NaN   \n",
       "1  81552  and test name do not appear in list of ...        NaN   \n",
       "\n",
       "                                      extracted text Validity Status  \n",
       "0  GENE.00052 Whole Genome Sequencing, Whole Exom...  HTML - Working  \n",
       "1  GENE.00052 Whole Genome Sequencing, Whole Exom...  HTML - Working  "
      ]
     },
     "execution_count": 18,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data_filtered=pd.read_excel('Filtered_data.xlsx')\n",
    "data_filtered.head(2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "e581bd82",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(7588, 13)"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data_filtered.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "598ddbaf",
   "metadata": {},
   "outputs": [
    {
     "ename": "NameError",
     "evalue": "name 'data' is not defined",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[1;31mNameError\u001b[0m                                 Traceback (most recent call last)",
      "Cell \u001b[1;32mIn[5], line 1\u001b[0m\n\u001b[1;32m----> 1\u001b[0m \u001b[43mdata\u001b[49m\u001b[38;5;241m.\u001b[39mcolumns\n",
      "\u001b[1;31mNameError\u001b[0m: name 'data' is not defined"
     ]
    }
   ],
   "source": [
    "data.columns"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "71430f43",
   "metadata": {},
   "outputs": [
    {
     "ename": "NameError",
     "evalue": "name 'data' is not defined",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[1;31mNameError\u001b[0m                                 Traceback (most recent call last)",
      "Cell \u001b[1;32mIn[6], line 1\u001b[0m\n\u001b[1;32m----> 1\u001b[0m \u001b[43mdata\u001b[49m[\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mlink_status\u001b[39m\u001b[38;5;124m'\u001b[39m]\u001b[38;5;241m.\u001b[39mvalue_counts()\n",
      "\u001b[1;31mNameError\u001b[0m: name 'data' is not defined"
     ]
    }
   ],
   "source": [
    "data['link_status'].value_counts()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "0e473190",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "PDF - Working                     3937\n",
       "HTML - Working                    1703\n",
       "Invalid Link                      1592\n",
       "Not Working (Status Code: 404)     187\n",
       "Not Working                        136\n",
       "Other - Working                     32\n",
       "Not Working (Status Code: 403)       1\n",
       "Name: Validity Status, dtype: int64"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data_filtered['Validity Status'].value_counts()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "id": "926c3ef6",
   "metadata": {},
   "outputs": [],
   "source": [
    "data_pdf=data_filtered[data_filtered['Validity Status']=='PDF - Working'].copy(deep=True)\n",
    "data_html=data_filtered[data_filtered['Validity Status']==\"HTML - Working\"].copy(deep=True)\n",
    "data_valid=data_filtered[(data_filtered['Validity Status']!=\"HTML - Working\")|(data_filtered['Validity Status']==\"PDF - Working\")|(data_filtered['Validity Status']==\"Other - Working\")].copy(deep=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 119,
   "id": "266be0e2",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Index(['Aetna', '(arkansasblue OR BCBS Arkansas)',\n",
       "       '(\"bcbsms\" OR BCBS Mississippi)', 'Independence Blue Cross',\n",
       "       '(southcarolinablues OR BCBS South Carolina)',\n",
       "       '(empireblue OR Empire BCBS)', 'Anthem',\n",
       "       '(\"Triple-S Salud\" OR Triple-S Health Plan)', 'Horizon BCBS NJ',\n",
       "       '\"HealthPartners\"', '(\"upmchealthplan\" OR UPMC Health Plan)',\n",
       "       '\"qualchoice\"', '(bcidaho OR Blue Cross Idaho)',\n",
       "       '(keystonefirstpa OR Keystone First)',\n",
       "       'AIM Specialty Health/Carelon Medical Benefits',\n",
       "       '(BCBSLA OR BCBS Louisiana)', '(bcbsri OR BCBS Rhode Island)', 'BCBSFL',\n",
       "       '(Healthnet OR Centene)', 'Wellmark', 'BCBSNC', 'Geisinger',\n",
       "       '(bridgespanhealth OR Bridgespan Health)',\n",
       "       '(wellsense OR BMC Healthnet)', '(ssmhealth OR SSM Health)',\n",
       "       'BCBS Michigan', '(UHC OR UnitedHealthcare)',\n",
       "       '(FEPBlue  OR BCBS Federal Employee Plan)',\n",
       "       '(Presbyterian Healthcare Services)',\n",
       "       '(Asuris  OR Asuris Northwest Health)',\n",
       "       '(Providence Health Plan OR providencehealthplan)'],\n",
       "      dtype='object')"
      ]
     },
     "execution_count": 119,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data_html[\"Payors/LBM\"].value_counts().index"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 126,
   "id": "5239fa6f",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000234.html\n"
     ]
    }
   ],
   "source": [
    "ind_url=[]\n",
    "for i in data_html[data_html[\"Payors/LBM\"]=='(empireblue OR Empire BCBS)' ]['Policy url']:\n",
    "    ind_url.append(i)\n",
    "    \n",
    "url=ind_url[77] \n",
    "print(url)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 157,
   "id": "8bd8d568",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['https://www.healthpartners.com/public/coverage-criteria/policy.html?contentid=ENTRY_232428'] \n",
      " 81552 OR DecisionDx-UM OR Gene expression profile testing for uveal melanoma OR uveal melanoma prognostic algorithmic test\n"
     ]
    }
   ],
   "source": [
    "keywords=[]\n",
    "ind_url=[]\n",
    "for i,j in zip(data_html['Keywords'],data_html[data_html[\"Payors/LBM\"]=='\"HealthPartners\"' ]['Policy url']):\n",
    "    ind_url.append(i)\n",
    "    keywords.append(j)\n",
    "    \n",
    "url=ind_url[1] \n",
    "keyword=keywords[1].split('OR')\n",
    "print(keyword,\"\\n\",url,)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 127,
   "id": "a657407a",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "title_tag:  CG-GENE-15 Genetic Testing for Lynch Syndrome, Familial Adenomatous Polyposis (FAP), Attenuated FAP and MYH-associated Polyposis \n",
      "Policy URL: https://www.empireblue.com/dam/medpolicies/abc/active/guidelines/gl_pw_e000234.html\n",
      "****************************\n"
     ]
    }
   ],
   "source": [
    "extract_title_dates(url)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "80f4d188",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "2724f95f",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "id": "3a1e035f",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 72,
   "id": "52c697ed",
   "metadata": {},
   "outputs": [],
   "source": [
    "import re\n",
    "import requests\n",
    "from bs4 import BeautifulSoup\n",
    "\n",
    "def single_Match_description_from_html(html_content, key):\n",
    "    headers = {'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/118.0.0.0 Safari/537.36 Edg/118.0.2088.46'}\n",
    "    response = requests.get(url, headers=headers)\n",
    " \n",
    "    html_content = response.text\n",
    "    soup = BeautifulSoup(html_content, 'html.parser')\n",
    "    text = soup.get_text()\n",
    "\n",
    "    pattern = fr\"({re.escape(key)})\\s+(.+)\"\n",
    "    # Use re.search to find the first match\n",
    "    match = re.search(pattern, text)\n",
    "\n",
    "    if match:\n",
    "        code = match.group(1)  # Extract the code\n",
    "        description = match.group(2)  # Extract the description\n",
    "        return description\n",
    "    else:\n",
    "        return \"NOT AVAILABLE IN HTML\"\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 73,
   "id": "8961a201",
   "metadata": {},
   "outputs": [],
   "source": [
    "import re\n",
    "import requests\n",
    "from bs4 import BeautifulSoup\n",
    "\n",
    "def Two_Match_descriptions_from_html(html_content, key):\n",
    "    headers = {\n",
    "        'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/118.0.0.0 Safari/537.36 Edg/118.0.2088.46'\n",
    "    }\n",
    "    \n",
    "    response = requests.get(url, headers=headers)\n",
    " \n",
    "    html_content = response.text\n",
    "    soup = BeautifulSoup(html_content, 'html.parser')\n",
    "    text = soup.get_text()\n",
    "\n",
    "    pattern = fr\"({re.escape(key)})\\s+(.+)\"\n",
    "    matches = re.findall(pattern, text)\n",
    "\n",
    "    descriptions_2 = []\n",
    "    for match in matches:\n",
    "        description = match[1]  # Extract the description\n",
    "        descriptions_2.append(description)\n",
    "\n",
    "    return descriptions_2\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 74,
   "id": "53160909",
   "metadata": {},
   "outputs": [],
   "source": [
    "import re\n",
    "import requests\n",
    "from bs4 import BeautifulSoup\n",
    "\n",
    "def all_Match_descriptions_from_html(url, key):\n",
    "    headers = {\n",
    "        'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/118.0.0.0 Safari/537.36 Edg/118.0.2088.46'\n",
    "    }\n",
    "    \n",
    "    response = requests.get(url, headers=headers)\n",
    " \n",
    "    html_content = response.text\n",
    "    soup = BeautifulSoup(html_content, 'html.parser')\n",
    "    text = soup.get_text()\n",
    "\n",
    "    pattern = fr\"({re.escape(key)})\\s+(.+)\"\n",
    "    matches = re.findall(pattern, text)\n",
    "\n",
    "    descriptions_all = []\n",
    "    for match in matches:\n",
    "        code = match[0]  # Extract the code\n",
    "        description = match[1]  # Extract the description\n",
    "        descriptions_all.append(description)\n",
    "\n",
    "    return descriptions_all\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 79,
   "id": "967d75bf",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[]"
      ]
     },
     "execution_count": 79,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "all_Match_descriptions_from_html(\"https://www.bluecrossnc.com/content/dam/bcbsnc/pdf/providers/policies-guidelines-codes/policies/commercial/laboratory/brca.pdf\",\"81212\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 76,
   "id": "a58d7103",
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "def all_match_descriptions_from_html(url, key):\n",
    "    headers = {\n",
    "        'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.110 Safari/537.36'\n",
    "    }\n",
    "\n",
    "    try:\n",
    "        response = requests.get(url, headers=headers)\n",
    "        response.raise_for_status()  # Raise an HTTPError for bad requests\n",
    "\n",
    "        soup = BeautifulSoup(response.text, 'html.parser')\n",
    "        text = soup.get_text()\n",
    "\n",
    "        pattern = fr\"{re.escape(key)}\\s+(.+)\"\n",
    "        match = re.search(pattern, text)\n",
    "        \n",
    "\n",
    "\n",
    "        if match:\n",
    "            code = match.group(1)  # Extract the code\n",
    "            description = match.group(1)  # Extract the description\n",
    "        print(\" First_match: \",description)\n",
    "        print(\"\\n********************************************\") \n",
    "    \n",
    "    except requests.exceptions.HTTPError as errh:\n",
    "        print(\"HTTP Error:\", errh)\n",
    "    except requests.exceptions.ConnectionError as errc:\n",
    "        print(\"Error Connecting:\", errc)\n",
    "    except requests.exceptions.Timeout as errt:\n",
    "        print(\"Timeout Error:\", errt)\n",
    "    except requests.exceptions.RequestException as err:\n",
    "        print(\"Something went wrong:\", err)\n",
    "\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 78,
   "id": "5a48c338",
   "metadata": {},
   "outputs": [],
   "source": [
    "import requests\n",
    "\n",
    "def all_Match_descriptions_from_html_refined(html_content, key):\n",
    "    headers = {\n",
    "        'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/118.0.0.0 Safari/537.36 Edg/118.0.2088.46'\n",
    "    }\n",
    "    \n",
    "    response = requests.get(url, headers=headers)\n",
    " \n",
    "    html_content = response.text\n",
    "    soup = BeautifulSoup(html_content, 'html.parser')\n",
    "    text = soup.get_text()\n",
    "\n",
    "    pattern = fr\"({re.escape(key)})\\s+(.+)\"\n",
    "    matches = re.findall(pattern, text)\n",
    "\n",
    "    descriptions = []\n",
    "    key_match = []\n",
    "    for match in matches:\n",
    "        code = match[0]  # Extract the code\n",
    "        description = match[1]  # Extract the description\n",
    "        descriptions.append(description)\n",
    "\n",
    "    if match:\n",
    "        key_match.append(\"Yes\")\n",
    "    else:\n",
    "        key_match.append(\"No\")\n",
    "\n",
    "    return f\"Keyword_found: {key_match} , 'Matched_sentences'{descriptions}\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "8e6e2676",
   "metadata": {},
   "outputs": [],
   "source": [
    "df=pd.DataFrame(columns=[\"keywords Found\",\"Text found\"],values)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "73cb010e",
   "metadata": {},
   "outputs": [],
   "source": [
    "data_filtered"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 94,
   "id": "354ad6d4",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Keyword_found: ['Yes'] , 'Matched_sentences'['(Castle Biosciences, Phoenix, AZ)\\xa0for risk stratification of persons with localized uveal melanoma;', \"The DecisionDx test is a gene expression profile that determines the molecular signature of a patient's melanoma. The results of the test provide knowledge regarding the risk of near term metastasis (5 years). Tumors with a Class 1 signature are associated with a good prognosis and a low potential to spread (or metastasize), while tumors with a Class 2 signature have a high potential to spread.\\xa0\", 'GEP reports between January 2012 to December 2014 from intra-operative FNA biopsy of choroidal tumors undergoing brachytherapy.\\xa0 A total of 4 patients were identified to have cytopathology consistent with a non-melanoma primary.\\xa0 All 4 patients presented with a unilateral, single choroidal tumor, which was treated with iodine-125 brachytherapy and underwent intra-operative FNA biopsy for cytopathology and UM-specific GEP testing for molecular prognostication.\\xa0 Gene expression profile testing of the choroidal tumor in each patient revealed class 1A in 3 patients and class 2 in 1 patient.\\xa0 The authors concluded that DecisionDx-UM GEP may be a helpful test for molecular prognostication in patients with UM; however, class 1 and class 2 test results are indeed possible in the setting of a non-melanoma malignancy.\\xa0 They recommended that cytopathology and/or other melanoma-specific testing be performed in all cases of suspected choroidal melanoma because GEP with this assay is unable to rule out the diagnosis of a choroidal melanoma.', 'testing. Thirty-seven patients in the prospective study were Class 1 and 33 were Class 2. Class 1 patients had 100% 3-year metastasis-free survival compared to 63% for Class 2 (log rank test ð\\x9d\\x91\\x9d = 0.003) with 27.3 median follow-up months in this interim analysis. Class 2 patients received significantly higher-intensity monitoring and more oncology/clinical trial referrals compared to Class 1 patients (Fisherâ\\x80\\x99s exact test ð\\x9d\\x91\\x9d = 2.1 Ã\\x97 10 â\\x88\\x9213 and ð\\x9d\\x91\\x9d = 0.04, respectively.). The investigators concluded that the results of this study provide additional, prospective evidence in an independent cohort of patients that Class 1 and Class 2 patients are managed according to the differential metastatic risk indicated by DecisionDx-UM.', 'gene expression profile test in a prospective multicenter study. J Oncol. 2016; 2016: 5325762.']\n"
     ]
    }
   ],
   "source": [
    "key_search='DecisionDx-UM'\n",
    "descriptions = all_Match_descriptions_from_html_refined(url,key_search)\n",
    "print(descriptions)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 175,
   "id": "e62b812a",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Index(['Unnamed: 0', 'Keywords', 'Payors/LBM', 'Policy url',\n",
       "       'Policy URL Correct/Incorrect Flag', '\"Covered/Yes\" Language',\n",
       "       '\"Not Covered/No\" Language', 'Vague / non-No',\n",
       "       '\"Conditional/Need PA\" language', 'Notable patterns within document',\n",
       "       'Unnamed: 9', 'extracted text', 'Validity Status'],\n",
       "      dtype='object')"
      ]
     },
     "execution_count": 175,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data_html.columns"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 131,
   "id": "ffe6a982",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Index(['Aetna', '(arkansasblue OR BCBS Arkansas)',\n",
       "       '(\"bcbsms\" OR BCBS Mississippi)', 'Independence Blue Cross',\n",
       "       '(southcarolinablues OR BCBS South Carolina)',\n",
       "       '(empireblue OR Empire BCBS)', 'Anthem',\n",
       "       '(\"Triple-S Salud\" OR Triple-S Health Plan)', 'Horizon BCBS NJ',\n",
       "       '\"HealthPartners\"', '(\"upmchealthplan\" OR UPMC Health Plan)',\n",
       "       '\"qualchoice\"', '(bcidaho OR Blue Cross Idaho)',\n",
       "       '(keystonefirstpa OR Keystone First)',\n",
       "       'AIM Specialty Health/Carelon Medical Benefits',\n",
       "       '(BCBSLA OR BCBS Louisiana)', '(bcbsri OR BCBS Rhode Island)', 'BCBSFL',\n",
       "       '(Healthnet OR Centene)', 'Wellmark', 'BCBSNC', 'Geisinger',\n",
       "       '(bridgespanhealth OR Bridgespan Health)',\n",
       "       '(wellsense OR BMC Healthnet)', '(ssmhealth OR SSM Health)',\n",
       "       'BCBS Michigan', '(UHC OR UnitedHealthcare)',\n",
       "       '(FEPBlue  OR BCBS Federal Employee Plan)',\n",
       "       '(Presbyterian Healthcare Services)',\n",
       "       '(Asuris  OR Asuris Northwest Health)',\n",
       "       '(Providence Health Plan OR providencehealthplan)'],\n",
       "      dtype='object')"
      ]
     },
     "execution_count": 131,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data_html[\"Payors/LBM\"].value_counts().index"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "id": "da361d5d",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['81552 ', ' DecisionDx-UM ', ' Gene expression profile testing for uveal melanoma ', ' uveal melanoma prognostic algorithmic test'] https://www.aetna.com/cpb/medical/data/300_399/0352.html\n"
     ]
    }
   ],
   "source": [
    "keywords=[]\n",
    "ind_url=[]\n",
    "for i,j in zip(data_html['Keywords'],data_html[data_html[\"Payors/LBM\"]=='Aetna' ]['Policy url']):\n",
    "    keywords.append(i)\n",
    "    ind_url.append(j)\n",
    "    \n",
    "url=ind_url[1] \n",
    "keword=keywords[1].split('OR')\n",
    "print(keword,url)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "8a9a76a0",
   "metadata": {},
   "source": [
    "#### Two Match keyword extraction"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "146fd12e",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "id": "dfd1654f",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(Castle Biosciences, Phoenix, AZ) for risk stratification of persons with localized uveal melanoma;\n"
     ]
    }
   ],
   "source": [
    "key_to_search = 'DecisionDx-UM'\n",
    "headers = {\n",
    "        'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/118.0.0.0 Safari/537.36 Edg/118.0.2088.46'\n",
    "    }\n",
    "    \n",
    "response = requests.get(url, headers=headers)\n",
    "html_content = response.text\n",
    "First_match=single_Match_description_from_html(html_content,key_to_search)\n",
    "\n",
    "print(First_match)\n",
    "\n",
    "single_Match_description_from_html\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "2d74b59d",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "id": "c605b090",
   "metadata": {},
   "source": [
    "#### All match Keyword extraction"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "db58d9aa",
   "metadata": {},
   "outputs": [
    {
     "ename": "NameError",
     "evalue": "name 'html_content' is not defined",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[1;31mNameError\u001b[0m                                 Traceback (most recent call last)",
      "Cell \u001b[1;32mIn[14], line 2\u001b[0m\n\u001b[0;32m      1\u001b[0m key_to_search \u001b[38;5;241m=\u001b[39m \u001b[38;5;124m'\u001b[39m\u001b[38;5;124mDecisionDx-UM\u001b[39m\u001b[38;5;124m'\u001b[39m\n\u001b[1;32m----> 2\u001b[0m descriptions \u001b[38;5;241m=\u001b[39m all_Match_descriptions_from_html(\u001b[43mhtml_content\u001b[49m, key_to_search)\n\u001b[0;32m      3\u001b[0m \u001b[38;5;28mprint\u001b[39m(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mAll Match Descriptions:\u001b[39m\u001b[38;5;124m\"\u001b[39m)\n\u001b[0;32m      4\u001b[0m \u001b[38;5;28;01mfor\u001b[39;00m description \u001b[38;5;129;01min\u001b[39;00m descriptions:\n",
      "\u001b[1;31mNameError\u001b[0m: name 'html_content' is not defined"
     ]
    }
   ],
   "source": [
    "key_to_search = 'DecisionDx-UM'\n",
    "descriptions = all_Match_descriptions_from_html(html_content, key_to_search)\n",
    "print(\"All Match Descriptions:\")\n",
    "for description in descriptions:\n",
    "    print(\"- \" + description)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "id": "522741a3",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "200\n"
     ]
    }
   ],
   "source": [
    "headers = {'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/118.0.0.0 Safari/537.36 Edg/118.0.2088.46'}\n",
    "\n",
    "response = requests.get(url, headers=headers)    \n",
    "print(response.status_code)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "2380e7c9",
   "metadata": {},
   "source": [
    "# Final Draft"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 121,
   "id": "7155b39e",
   "metadata": {},
   "outputs": [],
   "source": [
    "import requests\n",
    "\n",
    "def all_Match_descriptions_from_html_refined(html_content, key):\n",
    "    headers = {\n",
    "        'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/118.0.0.0 Safari/537.36 Edg/118.0.2088.46'\n",
    "    }\n",
    "    \n",
    "    response = requests.get(url, headers=headers)\n",
    "    content_type=response.headers.get('Content-Type','')\n",
    "        \n",
    "        # Ignore character encoding information in Content-Type header\n",
    "        #content_type = content_type.split(';')[0]\n",
    "        \n",
    "    if 'text/html' in content_type:\n",
    "            html_content = response.text\n",
    "            soup = BeautifulSoup(html_content, 'html.parser')\n",
    "            text = soup.get_text()\n",
    "\n",
    "            pattern = fr\"({re.escape(key)})\\s+(.+)\"\n",
    "            matches = re.findall(pattern, text)\n",
    "\n",
    "            descriptions = []\n",
    "            key_match = []\n",
    "            for match in matches:\n",
    "                code = match[0]  # Extract the code\n",
    "                description = match[1]  # Extract the description\n",
    "                descriptions.append(description)\n",
    "\n",
    "            if match:\n",
    "                key_match.append(\"Yes\")\n",
    "            else:\n",
    "                key_match.append(\"No\")\n",
    "\n",
    "            return f\"Keyword_found: {key_match} , 'Matched_sentences'{descriptions}\"\n",
    "        \n",
    "    elif 'application/pdf' in content_type:\n",
    "            import fitz\n",
    "            pdf_bytes = response.content\n",
    "            pdf_document = fitz.open(\"pdf\", pdf_bytes)\n",
    "\n",
    "            text = \"\"\n",
    "            for page_num in range(pdf_document.page_count):\n",
    "                page = pdf_document.load_page(page_num)\n",
    "                text += page.get_text()\n",
    "                \n",
    "                pattern = fr\"({re.escape(key)})\\s+(.+)\"\n",
    "                matches = re.findall(pattern, text)\n",
    "                \n",
    "                descriptions = []\n",
    "                key_match = []\n",
    "                for match2 in matches:\n",
    "                    code = match2[0]  # Extract the code\n",
    "                    description = match2[1]  # Extract the description\n",
    "                    descriptions.append(description)\n",
    "\n",
    "                if match2:\n",
    "                    key_match.append(\"Yes\")\n",
    "                else:\n",
    "                    key_match.append(\"No\")\n",
    "\n",
    "                return f\"Keyword_found: {key_match} , 'Matched_sentences'{descriptions}\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 97,
   "id": "c5436a1a",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['81552 ', ' DecisionDx-UM ', ' Gene expression profile testing for uveal melanoma ', ' uveal melanoma prognostic algorithmic test'] https://www.aetna.com/cpb/medical/data/300_399/0352.html\n"
     ]
    }
   ],
   "source": [
    "keywords=[]\n",
    "ind_url=[]\n",
    "for i,j in zip(data_html['Keywords'],data_html[data_html[\"Payors/LBM\"]=='Aetna' ]['Policy url']):\n",
    "    keywords.append(i)\n",
    "    ind_url.append(j)\n",
    "    \n",
    "url=ind_url[1] \n",
    "keword=keywords[1].split('OR')\n",
    "print(keword,url)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 105,
   "id": "da928431",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Index(['regence', 'eviCore', '(bluecrossma OR BCBS Massachusetts)',\n",
       "       '(Asuris  OR Asuris Northwest Health)', 'Tufts Health Plan',\n",
       "       'BCBS Michigan', '(Providence Health Plan OR providencehealthplan)',\n",
       "       '(UHC OR UnitedHealthcare)', '(FEPBlue  OR BCBS Federal Employee Plan)',\n",
       "       '(bridgespanhealth OR Bridgespan Health)',\n",
       "       '(modahealth OR Moda Health)', 'emblemhealth', '(Healthnet OR Centene)',\n",
       "       '(\"BSCA\" OR \"blueshieldca\" OR Blue Shield of California)', 'Geisinger',\n",
       "       '(medica OR Medica Health Plans)', 'Avalon hcs', 'BCBSFL', 'prevea360',\n",
       "       'Cigna', '(Presbyterian Healthcare Services)',\n",
       "       '(bcbsks OR BCBS Kansas)', '(BCBSLA OR BCBS Louisiana)',\n",
       "       'AIM Specialty Health/Carelon Medical Benefits',\n",
       "       '(ssmhealth OR SSM Health)', '(bcbsri OR BCBS Rhode Island)',\n",
       "       'Wellmark', 'Univera Healthcare', '(wellsense OR BMC Healthnet)',\n",
       "       'BCBSNC', '(bcidaho OR Blue Cross Idaho)',\n",
       "       '(massgeneralbrighamhealthplan OR  Mass General Brigham Health OR AllWays Health Partners)',\n",
       "       '(keystonefirstpa OR Keystone First)',\n",
       "       '(capbluecross OR Capital Blue Cross)',\n",
       "       '(arkansasblue OR BCBS Arkansas)', 'Anthem'],\n",
       "      dtype='object')"
      ]
     },
     "execution_count": 105,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data_pdf[\"Payors/LBM\"].value_counts().index"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 122,
   "id": "f36bd08c",
   "metadata": {},
   "outputs": [
    {
     "ename": "UnboundLocalError",
     "evalue": "local variable 'match2' referenced before assignment",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[1;31mUnboundLocalError\u001b[0m                         Traceback (most recent call last)",
      "Cell \u001b[1;32mIn[122], line 2\u001b[0m\n\u001b[0;32m      1\u001b[0m key_search\u001b[38;5;241m=\u001b[39m\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mDecisionDx-UM\u001b[39m\u001b[38;5;124m'\u001b[39m\n\u001b[1;32m----> 2\u001b[0m descriptions \u001b[38;5;241m=\u001b[39m \u001b[43mall_Match_descriptions_from_html_refined\u001b[49m\u001b[43m(\u001b[49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\u001b[43mkey_search\u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m      3\u001b[0m \u001b[38;5;28mprint\u001b[39m(descriptions)\n",
      "Cell \u001b[1;32mIn[121], line 56\u001b[0m, in \u001b[0;36mall_Match_descriptions_from_html_refined\u001b[1;34m(html_content, key)\u001b[0m\n\u001b[0;32m     53\u001b[0m     description \u001b[38;5;241m=\u001b[39m match2[\u001b[38;5;241m1\u001b[39m]  \u001b[38;5;66;03m# Extract the description\u001b[39;00m\n\u001b[0;32m     54\u001b[0m     descriptions\u001b[38;5;241m.\u001b[39mappend(description)\n\u001b[1;32m---> 56\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m \u001b[43mmatch2\u001b[49m:\n\u001b[0;32m     57\u001b[0m     key_match\u001b[38;5;241m.\u001b[39mappend(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mYes\u001b[39m\u001b[38;5;124m\"\u001b[39m)\n\u001b[0;32m     58\u001b[0m \u001b[38;5;28;01melse\u001b[39;00m:\n",
      "\u001b[1;31mUnboundLocalError\u001b[0m: local variable 'match2' referenced before assignment"
     ]
    }
   ],
   "source": [
    "key_search='DecisionDx-UM'\n",
    "descriptions = all_Match_descriptions_from_html_refined(url,key_search)\n",
    "print(descriptions)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 125,
   "id": "c64635dc",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['81546 ', ' Afirma Assay and Afirma Genomic Sequenc Classifer ', ' Gene expression classifier GEC thyroid cancer ', ' Thyroid FNA analysis to classify indeterminate nodules'] https://blue.regence.com/medicare/gt/m-gt64.pdf\n"
     ]
    }
   ],
   "source": [
    "keywords=[]\n",
    "pdf_url=[]\n",
    "for i,j in zip(data_pdf['Keywords'],data_pdf[data_pdf[\"Payors/LBM\"]=='regence' ]['Policy url']):\n",
    "    keywords.append(i)\n",
    "    pdf_url.append(j)\n",
    "    \n",
    "url=pdf_url[2] \n",
    "keword=keywords[2].split('OR')\n",
    "print(keword,url)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "35891c8e",
   "metadata": {},
   "outputs": [],
   "source": [
    "pdf_url=[]\n",
    "for i in data_pdf[data_pdf[\"Payors/LBM\"]=='(bluecrossma OR BCBS Massachusetts)']['Policy url']:\n",
    "    pdf_url.append(i)\n",
    "    \n",
    "url=pdf_url[4] \n",
    "print(url)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 126,
   "id": "8d44f709",
   "metadata": {},
   "outputs": [],
   "source": [
    "import re\n",
    "import requests\n",
    "import fitz  # PyMuPDF\n",
    "from bs4 import BeautifulSoup\n",
    "\n",
    "def all_match_descriptions_from_html_or_pdf(url, key):\n",
    "    headers = {\n",
    "        'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.110 Safari/537.36'\n",
    "    }\n",
    "    \n",
    "    try:\n",
    "        response = requests.get(url, headers=headers)\n",
    "        response.raise_for_status()  # Raise an HTTPError for bad requests\n",
    "\n",
    "        content_type = response.headers.get('Content-Type', '')\n",
    "\n",
    "        if 'text/html' in content_type:\n",
    "            html_content = response.text\n",
    "            soup = BeautifulSoup(html_content, 'html.parser')\n",
    "            text = soup.get_text()\n",
    "        elif 'application/pdf' in content_type:\n",
    "            pdf_bytes = response.content\n",
    "            pdf_document = fitz.open(\"pdf\", pdf_bytes)\n",
    "            text = \"\"\n",
    "            for page_num in range(pdf_document.page_count):\n",
    "                page = pdf_document.load_page(page_num)\n",
    "                text += page.get_text()\n",
    "\n",
    "        else:\n",
    "            return \"Unsupported Content Type\"\n",
    "\n",
    "        pattern = fr\"({re.escape(key)})\\s+(.+)\"\n",
    "        matches = re.findall(pattern, text)\n",
    "\n",
    "        descriptions = []\n",
    "        key_match = []\n",
    "        key_match\n",
    "        for match in matches:\n",
    "            code = match[0]  # Extract the code\n",
    "            description = match[1]  # Extract the description\n",
    "            descriptions.append(description)\n",
    "\n",
    "            key_match.append(\"Yes\")\n",
    "\n",
    "        if not key_match:\n",
    "            key_match.append(\"No\")\n",
    "\n",
    "        return f\"Keyword_found: {key_match}, 'Matched_sentences': {descriptions}\"\n",
    "\n",
    "    except requests.exceptions.HTTPError as errh:\n",
    "        return f\"HTTP Error: {errh}\"\n",
    "    except requests.exceptions.ConnectionError as errc:\n",
    "        return f\"Error Connecting: {errc}\"\n",
    "    except requests.exceptions.Timeout as errt:\n",
    "        return f\"Timeout Error: {errt}\"\n",
    "    except requests.exceptions.RequestException as err:\n",
    "        return f\"Something went wrong: {err}\"\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "7018da71",
   "metadata": {},
   "source": [
    "## With Matched Data Type"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 128,
   "id": "5c7e839e",
   "metadata": {},
   "outputs": [],
   "source": [
    "import re\n",
    "import requests\n",
    "import fitz  # PyMuPDF\n",
    "from bs4 import BeautifulSoup\n",
    "\n",
    "def all_match_descriptions_from_html_or_pdf(url, key):\n",
    "    headers = {\n",
    "        'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.110 Safari/537.36'\n",
    "    }\n",
    "    \n",
    "    try:\n",
    "        response = requests.get(url, headers=headers)\n",
    "        response.raise_for_status()  # Raise an HTTPError for bad requests\n",
    "\n",
    "        content_type = response.headers.get('Content-Type', '')\n",
    "\n",
    "        if 'text/html' in content_type:\n",
    "            html_content = response.text\n",
    "            soup = BeautifulSoup(html_content, 'html.parser')\n",
    "            text = soup.get_text()\n",
    "            content_type = 'HTML'\n",
    "        elif 'application/pdf' in content_type:\n",
    "            pdf_bytes = response.content\n",
    "            pdf_document = fitz.open(\"pdf\", pdf_bytes)\n",
    "            text = \"\"\n",
    "            for page_num in range(pdf_document.page_count):\n",
    "                page = pdf_document.load_page(page_num)\n",
    "                text += page.get_text()\n",
    "            content_type = 'PDF'\n",
    "        else:\n",
    "            return \"Unsupported Content Type\"\n",
    "\n",
    "        pattern = fr\"({re.escape(key)})\\s+(.+)\"\n",
    "        matches = re.findall(pattern, text)\n",
    "\n",
    "        descriptions = []\n",
    "        key_match = []\n",
    "        for match in matches:\n",
    "            code = match[0]  # Extract the code\n",
    "            description = match[1]  # Extract the description\n",
    "            descriptions.append(description)\n",
    "            key_data_type = type(match[0]).__name__  # Get data type of the matched keyword\n",
    "            key_match.append({\"Keyword\": match[0], \"DataType\": key_data_type})\n",
    "\n",
    "        if not key_match:\n",
    "            key_match.append(\"No Match Found\")\n",
    "\n",
    "        return {\"Content Type\": content_type, \"Keyword Matches\": key_match, \"Matched Sentences\": descriptions}\n",
    "\n",
    "    except requests.exceptions.HTTPError as errh:\n",
    "        return f\"HTTP Error: {errh}\"\n",
    "    except requests.exceptions.ConnectionError as errc:\n",
    "        return f\"Error Connecting: {errc}\"\n",
    "    except requests.exceptions.Timeout as errt:\n",
    "        return f\"Timeout Error: {errt}\"\n",
    "    except requests.exceptions.RequestException as err:\n",
    "        return f\"Something went wrong: {err}\"\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 140,
   "id": "8657212a",
   "metadata": {},
   "outputs": [],
   "source": [
    "import re\n",
    "import requests\n",
    "import fitz  # PyMuPDF\n",
    "from bs4 import BeautifulSoup\n",
    "\n",
    "def is_numeric(value):\n",
    "    try:\n",
    "        float(value)\n",
    "        return True\n",
    "    except ValueError:\n",
    "        return False\n",
    "\n",
    "def all_match_descriptions_from_html_or_pdf(url, key):\n",
    "    headers = {\n",
    "        'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.110 Safari/537.36'\n",
    "    }\n",
    "    \n",
    "    try:\n",
    "        response = requests.get(url, headers=headers)\n",
    "        response.raise_for_status()  # Raise an HTTPError for bad requests\n",
    "\n",
    "        content_type = response.headers.get('Content-Type', '')\n",
    "\n",
    "        if 'text/html' in content_type:\n",
    "            html_content = response.text\n",
    "            soup = BeautifulSoup(html_content, 'html.parser')\n",
    "            text = soup.get_text()\n",
    "            content_type = 'HTML'\n",
    "        elif 'application/pdf' in content_type:\n",
    "            pdf_bytes = response.content\n",
    "            pdf_document = fitz.open(\"pdf\", pdf_bytes)\n",
    "            text = \"\"\n",
    "            for page_num in range(pdf_document.page_count):\n",
    "                page = pdf_document.load_page(page_num)\n",
    "                text += page.get_text()\n",
    "            content_type = 'PDF'\n",
    "        else:\n",
    "            return \"Unsupported Content Type\"\n",
    "\n",
    "        pattern = fr\"({re.escape(key)})\\s+(.+)\"\n",
    "        matches = re.findall(pattern, text)\n",
    "\n",
    "        descriptions = []\n",
    "        key_match = []\n",
    "        for match in matches:\n",
    "            code = match[0]  # Extract the code\n",
    "            description = match[1]  # Extract the description\n",
    "            key_value = match[0]\n",
    "            key_data_type = \"Numeric\" if is_numeric(key_value) else \"Text\"  # Determine real datatype of the matched keyword\n",
    "            descriptions.append(description)\n",
    "            key_match.append({\"Keyword\": key_value, \"DataType\": key_data_type})\n",
    "\n",
    "        if not key_match:\n",
    "            key_match.append(\"No Match Found\")\n",
    "\n",
    "        return {\"Content Type\": content_type, \"Keyword Matches\": key_match, \"Matched Sentences\": descriptions}\n",
    "\n",
    "    except requests.exceptions.HTTPError as errh:\n",
    "        return f\"HTTP Error: {errh}\"\n",
    "    except requests.exceptions.ConnectionError as errc:\n",
    "        return f\"Error Connecting: {errc}\"\n",
    "    except requests.exceptions.Timeout as errt:\n",
    "        return f\"Timeout Error: {errt}\"\n",
    "    except requests.exceptions.RequestException as err:\n",
    "        return f\"Something went wrong: {err}\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 138,
   "id": "28e2a13f",
   "metadata": {},
   "outputs": [],
   "source": [
    "import re\n",
    "import requests\n",
    "import fitz  # PyMuPDF\n",
    "from bs4 import BeautifulSoup\n",
    "\n",
    "def is_numeric(value):\n",
    "    try:\n",
    "        float(value)\n",
    "        return True\n",
    "    except ValueError:\n",
    "        return False\n",
    "\n",
    "def all_match_descriptions_from_html_or_pdf(url, key):\n",
    "    headers = {\n",
    "        'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.110 Safari/537.36'\n",
    "    }\n",
    "    \n",
    "    keywords = key.split(\" OR \")  # Split keywords based on \"OR\" separator\n",
    "\n",
    "    try:\n",
    "        response = requests.get(url, headers=headers)\n",
    "        response.raise_for_status()  # Raise an HTTPError for bad requests\n",
    "\n",
    "        content_type = response.headers.get('Content-Type', '')\n",
    "\n",
    "        if 'text/html' in content_type:\n",
    "            html_content = response.text\n",
    "            soup = BeautifulSoup(html_content, 'html.parser')\n",
    "            text = soup.get_text()\n",
    "            content_type = 'HTML'\n",
    "        elif 'application/pdf' in content_type:\n",
    "            pdf_bytes = response.content\n",
    "            pdf_document = fitz.open(\"pdf\", pdf_bytes)\n",
    "            text = \"\"\n",
    "            for page_num in range(pdf_document.page_count):\n",
    "                page = pdf_document.load_page(page_num)\n",
    "                text += page.get_text()\n",
    "            content_type = 'PDF'\n",
    "        else:\n",
    "            return \"Unsupported Content Type\"\n",
    "\n",
    "        descriptions = []\n",
    "        key_matches = []\n",
    "        for keyword in keywords:\n",
    "            pattern = fr\"({re.escape(keyword)})\\s+(.+)\"\n",
    "            matches = re.findall(pattern, text)\n",
    "            for match in matches:\n",
    "                key_value = match[0]\n",
    "                key_data_type = \"Numeric\" if is_numeric(key_value) else \"Text\"\n",
    "                description = match[1]\n",
    "                key_matches.append({\"Keyword\": key_value, \"DataType\": key_data_type, \"Description\": description})\n",
    "                descriptions.append(description)\n",
    "\n",
    "        if not key_matches:\n",
    "            key_matches.append(\"No Match Found\")\n",
    "\n",
    "        return {\"Content Type\": content_type, \"Keyword Matches\": key_matches, \"Matched Sentences\": descriptions}\n",
    "\n",
    "    except requests.exceptions.HTTPError as errh:\n",
    "        return f\"HTTP Error: {errh}\"\n",
    "    except requests.exceptions.ConnectionError as errc:\n",
    "        return f\"Error Connecting: {errc}\"\n",
    "    except requests.exceptions.Timeout as errt:\n",
    "        return f\"Timeout Error: {errt}\"\n",
    "    except requests.exceptions.RequestException as err:\n",
    "        return f\"Something went wrong: {err}\"\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "51b05815",
   "metadata": {},
   "source": [
    "# final anD very final coDE"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "f1e9fc9a",
   "metadata": {},
   "source": [
    "### So we will create 2 columns...\n",
    "1. Keyword Present - Yes or No\n",
    "2. Keyword Type - Numeric or Text"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "id": "29fa57e7",
   "metadata": {},
   "outputs": [],
   "source": [
    "import re\n",
    "import requests\n",
    "import fitz  # PyMuPDF\n",
    "from bs4 import BeautifulSoup\n",
    "\n",
    "def is_numeric(value):\n",
    "    try:\n",
    "        float(value)\n",
    "        return True\n",
    "    except ValueError:\n",
    "        return False\n",
    "\n",
    "def all_match_descriptions_from_html_or_pdf(url, key):\n",
    "    headers = {\n",
    "        'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.110 Safari/537.36'\n",
    "    }\n",
    "    \n",
    "    keywords = key.split(\" OR \")  # Split keywords based on \"OR\" separator\n",
    "    found_keywords = []  # List to store found keywords\n",
    "    keyword_not_found = True  # Flag to indicate if any keyword is found\n",
    "\n",
    "    try:\n",
    "        response = requests.get(url, headers=headers)\n",
    "        response.raise_for_status()  # Raise an HTTPError for bad requests\n",
    "\n",
    "        content_type = response.headers.get('Content-Type', '')\n",
    "\n",
    "        if 'text/html' in content_type:\n",
    "            html_content = response.text\n",
    "            soup = BeautifulSoup(html_content, 'html.parser')\n",
    "            text = soup.get_text()\n",
    "            content_type = 'HTML'\n",
    "        elif 'application/pdf' in content_type:\n",
    "            pdf_bytes = response.content\n",
    "            pdf_document = fitz.open(\"pdf\", pdf_bytes)\n",
    "            text = \"\"\n",
    "            for page_num in range(pdf_document.page_count):\n",
    "                page = pdf_document.load_page(page_num)\n",
    "                text += page.get_text()\n",
    "            content_type = 'PDF'\n",
    "        else:\n",
    "            return \"Unsupported Content Type\"\n",
    "\n",
    "        descriptions = []\n",
    "        key_matches = []\n",
    "        for keyword in keywords:\n",
    "            pattern = fr\"({re.escape(keyword)})\\s+(.+)\"\n",
    "            matches = re.findall(pattern, text)\n",
    "            for match in matches:\n",
    "                key_value = match[0]\n",
    "                key_data_type = \"Numeric\" if is_numeric(key_value) else \"Text\"\n",
    "                description = match[1]\n",
    "                key_matches.append({\"Keyword\": key_value, \"DataType\": key_data_type, \"Description\": description})\n",
    "                descriptions.append(description)\n",
    "                found_keywords.append(\"Yes\")\n",
    "                keyword_not_found = False\n",
    "\n",
    "        if keyword_not_found:\n",
    "            found_keywords.append(\"No\")\n",
    "\n",
    "        return {\"Content Type\": content_type, \"Keyword Matches\": key_matches, \"Matched Sentences\": descriptions, \"Found Keywords\": found_keywords}\n",
    "\n",
    "    except requests.exceptions.HTTPError as errh:\n",
    "        return f\"HTTP Error: {errh}\"\n",
    "    except requests.exceptions.ConnectionError as errc:\n",
    "        return f\"Error Connecting: {errc}\"\n",
    "    except requests.exceptions.Timeout as errt:\n",
    "        return f\"Timeout Error: {errt}\"\n",
    "    except requests.exceptions.RequestException as err:\n",
    "        return f\"Something went wrong: {err}\"\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "42a9054a",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 61,
   "id": "06623464",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "81528 OR Cologuard OR DNA analysis of stool to screen for colorectal cancer OR Fecal DNA screening test for colon cancer https://blue.regence.com/trgmedpol/lab/lab73.pdf\n"
     ]
    }
   ],
   "source": [
    "keywords=[]\n",
    "pdf_url=[]\n",
    "for i,j in zip(data_pdf['Keywords'],data_pdf[data_pdf[\"Payors/LBM\"]=='regence' ]['Policy url']):\n",
    "    keywords.append(i)\n",
    "    pdf_url.append(j)\n",
    "    \n",
    "url=pdf_url[0] \n",
    "key_search=keywords[0]\n",
    "print(key_search,url)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 57,
   "id": "e369a868",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "81493 OR Corus CAD OR Gene expression profiling for cardiac risk assesment https://www.aetna.com/cpb/medical/data/300_399/0381.html\n"
     ]
    }
   ],
   "source": [
    "keywords=[]\n",
    "ind_url=[]\n",
    "for i,j in zip(data_html['Keywords'],data_html[data_html[\"Payors/LBM\"]=='Aetna' ]['Policy url']):\n",
    "    keywords.append(i)\n",
    "    ind_url.append(j)\n",
    "    \n",
    "url=ind_url[23] \n",
    "key_search=keywords[23]\n",
    "print(key_search,url)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 62,
   "id": "2393701d",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'Content Type': 'PDF', 'Keyword Matches': [], 'Matched Sentences': [], 'Found Keywords': ['No']}\n",
      "<class 'dict'>\n"
     ]
    }
   ],
   "source": [
    "descriptions = all_match_descriptions_from_html_or_pdf(url,key_search)\n",
    "print(descriptions)\n",
    "print(type(descriptions))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "c5d2d588",
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "\n",
    "\n",
    "\n",
    "def check_keywords(row):\n",
    "    keyword_list = row['Keyword'].split(',')  # Assuming keywords are separated by comma in the 'Keyword' column\n",
    "    url = row['URL']\n",
    "    keyword_found_flag = False\n",
    "    for key_search in keyword_list:\n",
    "        descriptions = all_match_descriptions_from_html_or_pdf(url, key_search.strip())  # Remove leading/trailing spaces from keywords\n",
    "        if descriptions['Keyword Matches']:\n",
    "            keyword_found_flag = True\n",
    "            break  # Exit loop if any keyword is found for the current URL\n",
    "    return 'Yes' if keyword_found_flag else 'No'\n",
    "\n",
    "# Apply the function to each row and create a new column 'Keyword Found'\n",
    "data_html['Keyword Found'] = df.apply(check_keywords, axis=1)\n",
    "\n",
    "print(data_html)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "eaf9a9c9",
   "metadata": {},
   "outputs": [],
   "source": [
    "Keyword_found = []\n",
    "\n",
    "# Assuming data_html is a dictionary with keys as URLs and values as keyword lists\n",
    "data_html = {\n",
    "    'data_html['Policy url']:data_html,\n",
    "    'https://example2.com': ['Keyword3', 'Keyword4'],\n",
    "    # Add more URLs and their corresponding keyword lists as needed\n",
    "}\n",
    "\n",
    "for url, keyword_list in data_html.items():\n",
    "    keyword_found_flag = False\n",
    "    for key_search in keyword_list:\n",
    "        descriptions = all_match_descriptions_from_html_or_pdf(url, key_search)\n",
    "        if descriptions['Keyword Matches']:\n",
    "            keyword_found_flag = True\n",
    "            break  # Exit loop if any keyword is found for the current URL\n",
    "    if keyword_found_flag:\n",
    "        Keyword_found.append(\"Yes\")\n",
    "    else:\n",
    "        Keyword_found.append(\"No\")\n",
    "\n",
    "print(Keyword_found)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 68,
   "id": "041126f8",
   "metadata": {},
   "outputs": [
    {
     "ename": "KeyboardInterrupt",
     "evalue": "",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[1;31mKeyboardInterrupt\u001b[0m                         Traceback (most recent call last)",
      "Cell \u001b[1;32mIn[68], line 3\u001b[0m\n\u001b[0;32m      1\u001b[0m Keyword_found\u001b[38;5;241m=\u001b[39m[]\n\u001b[0;32m      2\u001b[0m \u001b[38;5;28;01mfor\u001b[39;00m i \u001b[38;5;129;01min\u001b[39;00m data_html[\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mPolicy url\u001b[39m\u001b[38;5;124m'\u001b[39m]:\n\u001b[1;32m----> 3\u001b[0m     descriptions \u001b[38;5;241m=\u001b[39m \u001b[43mall_match_descriptions_from_html_or_pdf\u001b[49m\u001b[43m(\u001b[49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\u001b[43mkey_search\u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m      4\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m descriptions[\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mKeyword Matches\u001b[39m\u001b[38;5;124m'\u001b[39m]:\n\u001b[0;32m      5\u001b[0m         Keyword_found\u001b[38;5;241m.\u001b[39mappend(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mYes\u001b[39m\u001b[38;5;124m\"\u001b[39m)\n",
      "Cell \u001b[1;32mIn[30], line 23\u001b[0m, in \u001b[0;36mall_match_descriptions_from_html_or_pdf\u001b[1;34m(url, key)\u001b[0m\n\u001b[0;32m     20\u001b[0m keyword_not_found \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;01mTrue\u001b[39;00m  \u001b[38;5;66;03m# Flag to indicate if any keyword is found\u001b[39;00m\n\u001b[0;32m     22\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[1;32m---> 23\u001b[0m     response \u001b[38;5;241m=\u001b[39m \u001b[43mrequests\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mget\u001b[49m\u001b[43m(\u001b[49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mheaders\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mheaders\u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m     24\u001b[0m     response\u001b[38;5;241m.\u001b[39mraise_for_status()  \u001b[38;5;66;03m# Raise an HTTPError for bad requests\u001b[39;00m\n\u001b[0;32m     26\u001b[0m     content_type \u001b[38;5;241m=\u001b[39m response\u001b[38;5;241m.\u001b[39mheaders\u001b[38;5;241m.\u001b[39mget(\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mContent-Type\u001b[39m\u001b[38;5;124m'\u001b[39m, \u001b[38;5;124m'\u001b[39m\u001b[38;5;124m'\u001b[39m)\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\requests\\api.py:73\u001b[0m, in \u001b[0;36mget\u001b[1;34m(url, params, **kwargs)\u001b[0m\n\u001b[0;32m     62\u001b[0m \u001b[38;5;28;01mdef\u001b[39;00m \u001b[38;5;21mget\u001b[39m(url, params\u001b[38;5;241m=\u001b[39m\u001b[38;5;28;01mNone\u001b[39;00m, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39mkwargs):\n\u001b[0;32m     63\u001b[0m     \u001b[38;5;124mr\u001b[39m\u001b[38;5;124;03m\"\"\"Sends a GET request.\u001b[39;00m\n\u001b[0;32m     64\u001b[0m \n\u001b[0;32m     65\u001b[0m \u001b[38;5;124;03m    :param url: URL for the new :class:`Request` object.\u001b[39;00m\n\u001b[1;32m   (...)\u001b[0m\n\u001b[0;32m     70\u001b[0m \u001b[38;5;124;03m    :rtype: requests.Response\u001b[39;00m\n\u001b[0;32m     71\u001b[0m \u001b[38;5;124;03m    \"\"\"\u001b[39;00m\n\u001b[1;32m---> 73\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m request(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mget\u001b[39m\u001b[38;5;124m\"\u001b[39m, url, params\u001b[38;5;241m=\u001b[39mparams, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39mkwargs)\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\requests\\api.py:59\u001b[0m, in \u001b[0;36mrequest\u001b[1;34m(method, url, **kwargs)\u001b[0m\n\u001b[0;32m     55\u001b[0m \u001b[38;5;66;03m# By using the 'with' statement we are sure the session is closed, thus we\u001b[39;00m\n\u001b[0;32m     56\u001b[0m \u001b[38;5;66;03m# avoid leaving sockets open which can trigger a ResourceWarning in some\u001b[39;00m\n\u001b[0;32m     57\u001b[0m \u001b[38;5;66;03m# cases, and look like a memory leak in others.\u001b[39;00m\n\u001b[0;32m     58\u001b[0m \u001b[38;5;28;01mwith\u001b[39;00m sessions\u001b[38;5;241m.\u001b[39mSession() \u001b[38;5;28;01mas\u001b[39;00m session:\n\u001b[1;32m---> 59\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m session\u001b[38;5;241m.\u001b[39mrequest(method\u001b[38;5;241m=\u001b[39mmethod, url\u001b[38;5;241m=\u001b[39murl, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39mkwargs)\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\requests\\sessions.py:587\u001b[0m, in \u001b[0;36mSession.request\u001b[1;34m(self, method, url, params, data, headers, cookies, files, auth, timeout, allow_redirects, proxies, hooks, stream, verify, cert, json)\u001b[0m\n\u001b[0;32m    582\u001b[0m send_kwargs \u001b[38;5;241m=\u001b[39m {\n\u001b[0;32m    583\u001b[0m     \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mtimeout\u001b[39m\u001b[38;5;124m\"\u001b[39m: timeout,\n\u001b[0;32m    584\u001b[0m     \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mallow_redirects\u001b[39m\u001b[38;5;124m\"\u001b[39m: allow_redirects,\n\u001b[0;32m    585\u001b[0m }\n\u001b[0;32m    586\u001b[0m send_kwargs\u001b[38;5;241m.\u001b[39mupdate(settings)\n\u001b[1;32m--> 587\u001b[0m resp \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39msend(prep, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39msend_kwargs)\n\u001b[0;32m    589\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m resp\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\requests\\sessions.py:701\u001b[0m, in \u001b[0;36mSession.send\u001b[1;34m(self, request, **kwargs)\u001b[0m\n\u001b[0;32m    698\u001b[0m start \u001b[38;5;241m=\u001b[39m preferred_clock()\n\u001b[0;32m    700\u001b[0m \u001b[38;5;66;03m# Send the request\u001b[39;00m\n\u001b[1;32m--> 701\u001b[0m r \u001b[38;5;241m=\u001b[39m adapter\u001b[38;5;241m.\u001b[39msend(request, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39mkwargs)\n\u001b[0;32m    703\u001b[0m \u001b[38;5;66;03m# Total elapsed time of the request (approximately)\u001b[39;00m\n\u001b[0;32m    704\u001b[0m elapsed \u001b[38;5;241m=\u001b[39m preferred_clock() \u001b[38;5;241m-\u001b[39m start\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\requests\\adapters.py:489\u001b[0m, in \u001b[0;36mHTTPAdapter.send\u001b[1;34m(self, request, stream, timeout, verify, cert, proxies)\u001b[0m\n\u001b[0;32m    487\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[0;32m    488\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m chunked:\n\u001b[1;32m--> 489\u001b[0m         resp \u001b[38;5;241m=\u001b[39m \u001b[43mconn\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43murlopen\u001b[49m\u001b[43m(\u001b[49m\n\u001b[0;32m    490\u001b[0m \u001b[43m            \u001b[49m\u001b[43mmethod\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mrequest\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mmethod\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    491\u001b[0m \u001b[43m            \u001b[49m\u001b[43murl\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    492\u001b[0m \u001b[43m            \u001b[49m\u001b[43mbody\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mrequest\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mbody\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    493\u001b[0m \u001b[43m            \u001b[49m\u001b[43mheaders\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mrequest\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mheaders\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    494\u001b[0m \u001b[43m            \u001b[49m\u001b[43mredirect\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43;01mFalse\u001b[39;49;00m\u001b[43m,\u001b[49m\n\u001b[0;32m    495\u001b[0m \u001b[43m            \u001b[49m\u001b[43massert_same_host\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43;01mFalse\u001b[39;49;00m\u001b[43m,\u001b[49m\n\u001b[0;32m    496\u001b[0m \u001b[43m            \u001b[49m\u001b[43mpreload_content\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43;01mFalse\u001b[39;49;00m\u001b[43m,\u001b[49m\n\u001b[0;32m    497\u001b[0m \u001b[43m            \u001b[49m\u001b[43mdecode_content\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43;01mFalse\u001b[39;49;00m\u001b[43m,\u001b[49m\n\u001b[0;32m    498\u001b[0m \u001b[43m            \u001b[49m\u001b[43mretries\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mmax_retries\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    499\u001b[0m \u001b[43m            \u001b[49m\u001b[43mtimeout\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mtimeout\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    500\u001b[0m \u001b[43m        \u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m    502\u001b[0m     \u001b[38;5;66;03m# Send the request.\u001b[39;00m\n\u001b[0;32m    503\u001b[0m     \u001b[38;5;28;01melse\u001b[39;00m:\n\u001b[0;32m    504\u001b[0m         \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28mhasattr\u001b[39m(conn, \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mproxy_pool\u001b[39m\u001b[38;5;124m\"\u001b[39m):\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\urllib3\\connectionpool.py:703\u001b[0m, in \u001b[0;36mHTTPConnectionPool.urlopen\u001b[1;34m(self, method, url, body, headers, retries, redirect, assert_same_host, timeout, pool_timeout, release_conn, chunked, body_pos, **response_kw)\u001b[0m\n\u001b[0;32m    700\u001b[0m     \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_prepare_proxy(conn)\n\u001b[0;32m    702\u001b[0m \u001b[38;5;66;03m# Make the request on the httplib connection object.\u001b[39;00m\n\u001b[1;32m--> 703\u001b[0m httplib_response \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_make_request\u001b[49m\u001b[43m(\u001b[49m\n\u001b[0;32m    704\u001b[0m \u001b[43m    \u001b[49m\u001b[43mconn\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    705\u001b[0m \u001b[43m    \u001b[49m\u001b[43mmethod\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    706\u001b[0m \u001b[43m    \u001b[49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    707\u001b[0m \u001b[43m    \u001b[49m\u001b[43mtimeout\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mtimeout_obj\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    708\u001b[0m \u001b[43m    \u001b[49m\u001b[43mbody\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mbody\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    709\u001b[0m \u001b[43m    \u001b[49m\u001b[43mheaders\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mheaders\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    710\u001b[0m \u001b[43m    \u001b[49m\u001b[43mchunked\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mchunked\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    711\u001b[0m \u001b[43m\u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m    713\u001b[0m \u001b[38;5;66;03m# If we're going to release the connection in ``finally:``, then\u001b[39;00m\n\u001b[0;32m    714\u001b[0m \u001b[38;5;66;03m# the response doesn't need to know about the connection. Otherwise\u001b[39;00m\n\u001b[0;32m    715\u001b[0m \u001b[38;5;66;03m# it will also try to release it and we'll have a double-release\u001b[39;00m\n\u001b[0;32m    716\u001b[0m \u001b[38;5;66;03m# mess.\u001b[39;00m\n\u001b[0;32m    717\u001b[0m response_conn \u001b[38;5;241m=\u001b[39m conn \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m release_conn \u001b[38;5;28;01melse\u001b[39;00m \u001b[38;5;28;01mNone\u001b[39;00m\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\urllib3\\connectionpool.py:386\u001b[0m, in \u001b[0;36mHTTPConnectionPool._make_request\u001b[1;34m(self, conn, method, url, timeout, chunked, **httplib_request_kw)\u001b[0m\n\u001b[0;32m    384\u001b[0m \u001b[38;5;66;03m# Trigger any extra validation we need to do.\u001b[39;00m\n\u001b[0;32m    385\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[1;32m--> 386\u001b[0m     \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_validate_conn\u001b[49m\u001b[43m(\u001b[49m\u001b[43mconn\u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m    387\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m (SocketTimeout, BaseSSLError) \u001b[38;5;28;01mas\u001b[39;00m e:\n\u001b[0;32m    388\u001b[0m     \u001b[38;5;66;03m# Py2 raises this as a BaseSSLError, Py3 raises it as socket timeout.\u001b[39;00m\n\u001b[0;32m    389\u001b[0m     \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_raise_timeout(err\u001b[38;5;241m=\u001b[39me, url\u001b[38;5;241m=\u001b[39murl, timeout_value\u001b[38;5;241m=\u001b[39mconn\u001b[38;5;241m.\u001b[39mtimeout)\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\urllib3\\connectionpool.py:1042\u001b[0m, in \u001b[0;36mHTTPSConnectionPool._validate_conn\u001b[1;34m(self, conn)\u001b[0m\n\u001b[0;32m   1040\u001b[0m \u001b[38;5;66;03m# Force connect early to allow us to validate the connection.\u001b[39;00m\n\u001b[0;32m   1041\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m \u001b[38;5;28mgetattr\u001b[39m(conn, \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124msock\u001b[39m\u001b[38;5;124m\"\u001b[39m, \u001b[38;5;28;01mNone\u001b[39;00m):  \u001b[38;5;66;03m# AppEngine might not have  `.sock`\u001b[39;00m\n\u001b[1;32m-> 1042\u001b[0m     \u001b[43mconn\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mconnect\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m   1044\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m conn\u001b[38;5;241m.\u001b[39mis_verified:\n\u001b[0;32m   1045\u001b[0m     warnings\u001b[38;5;241m.\u001b[39mwarn(\n\u001b[0;32m   1046\u001b[0m         (\n\u001b[0;32m   1047\u001b[0m             \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mUnverified HTTPS request is being made to host \u001b[39m\u001b[38;5;124m'\u001b[39m\u001b[38;5;132;01m%s\u001b[39;00m\u001b[38;5;124m'\u001b[39m\u001b[38;5;124m. \u001b[39m\u001b[38;5;124m\"\u001b[39m\n\u001b[1;32m   (...)\u001b[0m\n\u001b[0;32m   1052\u001b[0m         InsecureRequestWarning,\n\u001b[0;32m   1053\u001b[0m     )\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\urllib3\\connection.py:363\u001b[0m, in \u001b[0;36mHTTPSConnection.connect\u001b[1;34m(self)\u001b[0m\n\u001b[0;32m    361\u001b[0m \u001b[38;5;28;01mdef\u001b[39;00m \u001b[38;5;21mconnect\u001b[39m(\u001b[38;5;28mself\u001b[39m):\n\u001b[0;32m    362\u001b[0m     \u001b[38;5;66;03m# Add certificate verification\u001b[39;00m\n\u001b[1;32m--> 363\u001b[0m     \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39msock \u001b[38;5;241m=\u001b[39m conn \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_new_conn\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m    364\u001b[0m     hostname \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mhost\n\u001b[0;32m    365\u001b[0m     tls_in_tls \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;01mFalse\u001b[39;00m\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\urllib3\\connection.py:174\u001b[0m, in \u001b[0;36mHTTPConnection._new_conn\u001b[1;34m(self)\u001b[0m\n\u001b[0;32m    171\u001b[0m     extra_kw[\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124msocket_options\u001b[39m\u001b[38;5;124m\"\u001b[39m] \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39msocket_options\n\u001b[0;32m    173\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[1;32m--> 174\u001b[0m     conn \u001b[38;5;241m=\u001b[39m connection\u001b[38;5;241m.\u001b[39mcreate_connection(\n\u001b[0;32m    175\u001b[0m         (\u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_dns_host, \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mport), \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mtimeout, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39mextra_kw\n\u001b[0;32m    176\u001b[0m     )\n\u001b[0;32m    178\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m SocketTimeout:\n\u001b[0;32m    179\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m ConnectTimeoutError(\n\u001b[0;32m    180\u001b[0m         \u001b[38;5;28mself\u001b[39m,\n\u001b[0;32m    181\u001b[0m         \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mConnection to \u001b[39m\u001b[38;5;132;01m%s\u001b[39;00m\u001b[38;5;124m timed out. (connect timeout=\u001b[39m\u001b[38;5;132;01m%s\u001b[39;00m\u001b[38;5;124m)\u001b[39m\u001b[38;5;124m\"\u001b[39m\n\u001b[0;32m    182\u001b[0m         \u001b[38;5;241m%\u001b[39m (\u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mhost, \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mtimeout),\n\u001b[0;32m    183\u001b[0m     )\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\urllib3\\util\\connection.py:85\u001b[0m, in \u001b[0;36mcreate_connection\u001b[1;34m(address, timeout, source_address, socket_options)\u001b[0m\n\u001b[0;32m     83\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m source_address:\n\u001b[0;32m     84\u001b[0m         sock\u001b[38;5;241m.\u001b[39mbind(source_address)\n\u001b[1;32m---> 85\u001b[0m     \u001b[43msock\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mconnect\u001b[49m\u001b[43m(\u001b[49m\u001b[43msa\u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m     86\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m sock\n\u001b[0;32m     88\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m socket\u001b[38;5;241m.\u001b[39merror \u001b[38;5;28;01mas\u001b[39;00m e:\n",
      "\u001b[1;31mKeyboardInterrupt\u001b[0m: "
     ]
    }
   ],
   "source": [
    "\n",
    "Keyword_found=[]\n",
    "for i in data_html['Policy url']:\n",
    "    descriptions = all_match_descriptions_from_html_or_pdf(url,key_search)\n",
    "    if descriptions['Keyword Matches']:\n",
    "        Keyword_found.append(\"Yes\")\n",
    "    else :\n",
    "        Keyword_found.append(\"No\")\n",
    "print(Keyword_found) \n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 63,
   "id": "2c857f4e",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Keyword Found: No\n"
     ]
    },
    {
     "ename": "IndexError",
     "evalue": "list index out of range",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[1;31mIndexError\u001b[0m                                Traceback (most recent call last)",
      "Cell \u001b[1;32mIn[63], line 7\u001b[0m\n\u001b[0;32m      4\u001b[0m     keyword_found \u001b[38;5;241m=\u001b[39m \u001b[38;5;124m'\u001b[39m\u001b[38;5;124mYes\u001b[39m\u001b[38;5;124m'\u001b[39m\n\u001b[0;32m      5\u001b[0m \u001b[38;5;28mprint\u001b[39m(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mKeyword Found:\u001b[39m\u001b[38;5;124m\"\u001b[39m, keyword_found)\n\u001b[1;32m----> 7\u001b[0m Data_Type\u001b[38;5;241m=\u001b[39m\u001b[43mdescriptions\u001b[49m\u001b[43m[\u001b[49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[38;5;124;43mKeyword Matches\u001b[39;49m\u001b[38;5;124;43m'\u001b[39;49m\u001b[43m]\u001b[49m\u001b[43m[\u001b[49m\u001b[38;5;241;43m0\u001b[39;49m\u001b[43m]\u001b[49m[\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mDataType\u001b[39m\u001b[38;5;124m'\u001b[39m]\n\u001b[0;32m      8\u001b[0m \u001b[38;5;28mprint\u001b[39m(keyword_found)\n\u001b[0;32m      9\u001b[0m required_info\n",
      "\u001b[1;31mIndexError\u001b[0m: list index out of range"
     ]
    }
   ],
   "source": [
    "\n",
    "Found_keywords=descriptions['Found Keywords']\n",
    "keyword_found = 'No'\n",
    "if 'Yes' in Found_keywords:\n",
    "    keyword_found = 'Yes'\n",
    "print(\"Keyword Found:\", keyword_found)\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "e8eb33fb",
   "metadata": {},
   "outputs": [],
   "source": [
    "if\n",
    "Data_Type=descriptions['Keyword Matches'][0]['DataType']\n",
    "print(keyword_found)\n",
    "required_info"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "934d0a14",
   "metadata": {},
   "outputs": [],
   "source": [
    "for i in data_html[data_html[\"Payors/LBM\"]=='Aetna' ]['Policy url']:\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "id": "5e6a7fc4",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Content Type</th>\n",
       "      <th>Keyword Matches</th>\n",
       "      <th>Matched Sentences</th>\n",
       "      <th>Found Keywords</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>HTML</td>\n",
       "      <td>{'Keyword': '81448', 'DataType': 'Numeric', 'D...</td>\n",
       "      <td>Hereditary peripheral neuropathies (eg, Charco...</td>\n",
       "      <td>Yes</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  Content Type                                    Keyword Matches  \\\n",
       "0         HTML  {'Keyword': '81448', 'DataType': 'Numeric', 'D...   \n",
       "\n",
       "                                   Matched Sentences Found Keywords  \n",
       "0  Hereditary peripheral neuropathies (eg, Charco...            Yes  "
      ]
     },
     "execution_count": 32,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df=pd.DataFrame(descriptions)\n",
    "df.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 157,
   "id": "9b458d23",
   "metadata": {},
   "outputs": [
    {
     "ename": "NameError",
     "evalue": "name 'priont' is not defined",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[1;31mNameError\u001b[0m                                 Traceback (most recent call last)",
      "Cell \u001b[1;32mIn[157], line 1\u001b[0m\n\u001b[1;32m----> 1\u001b[0m \u001b[43mpriont\u001b[49m\n",
      "\u001b[1;31mNameError\u001b[0m: name 'priont' is not defined"
     ]
    }
   ],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "5ae17963",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "a3ff3edd",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "4424cf0d",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "b5b06145",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "e78041fd",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "647fe7fe",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "9f32a2aa",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "a46bdf1d",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 162,
   "id": "663200ef",
   "metadata": {},
   "outputs": [],
   "source": [
    "def extract_title_text(url):\n",
    "    headers = {'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/118.0.0.0 Safari/537.36 Edg/118.0.2088.46'}\n",
    "    \n",
    "    response = requests.get(url, headers=headers)    \n",
    "    if response.status_code == 200:\n",
    "        content_type=response.headers.get('Content-Type','')\n",
    "        \n",
    "        # Ignore character encoding information in Content-Type header\n",
    "        #content_type = content_type.split(';')[0]\n",
    "        \n",
    "        if 'text/html' in content_type:\n",
    "            html_content = response.text\n",
    "            soup = BeautifulSoup(html_content, 'html.parser')\n",
    "            soup.prettify()\n",
    "            text = soup.get_text()\n",
    "            title_pattern = r'Title:\\s*(.*?)\\n'\n",
    "            policy_name_pattern = r'Policy Name:\\s*(.*?)\\n'\n",
    "            \n",
    "            # Use the regular expression to extract the \"Subject\"\n",
    "            match_title = re.search(title_pattern, text)\n",
    "            match_policy_name=re.search(policy_name_pattern,text)\n",
    "            \n",
    "            if match_title:\n",
    "                Title = match_title.group(1).strip()\n",
    "                print(\"title_tag_html: \",Title)\n",
    "            elif match_policy_name:\n",
    "                Title_policy = match_policy_name.group(1).strip()\n",
    "                print( \"title_tag_html: \",Title_policy)\n",
    "        \n",
    "                \n",
    "            else:\n",
    "                title_tag=soup.title.text.strip()\n",
    "                if title_tag:\n",
    "                    print(\"title_tag: \", title_tag,\"\\nPolicy URL:\",url)\n",
    "                else:\n",
    "                    print(\"No Title_tag by this soup.title Method \\n Try other method\")\n",
    "                    print('Overall_title :\\n',soup.find(\"head\").title.text.strip())\n",
    "                    print('Required title of doc: \\n',soup.find(class_='title-content').text.strip())\n",
    "            print('****************************')\n",
    "            print(text)\n",
    "        \n",
    "        elif 'application/pdf' in content_type:\n",
    "            import fitz\n",
    "            pdf_bytes = response.content\n",
    "            pdf_document = fitz.open(\"pdf\", pdf_bytes)\n",
    "\n",
    "            text = \"\"\n",
    "            for page_num in range(pdf_document.page_count):\n",
    "                page = pdf_document.load_page(page_num)\n",
    "                text += page.get_text()\n",
    "            subject_pattern = r'Subject:\\s*(.*?)\\n'\n",
    "            \n",
    "            \n",
    "\n",
    "            # Use the regular expression to extract the \"Subject\"\n",
    "            match = re.search(subject_pattern, text)\n",
    "            if match:\n",
    "                subject = match.group(1)\n",
    "                print(\"title_tag_pdf_subject: \",subject,\"\\nPolicy URL:\",url)\n",
    "            else:\n",
    "                metadata = pdf_document.metadata\n",
    "                title = metadata.get(\"title\", \"Title not found in metadata\")\n",
    "                print(\"title_tag_pdf_met :\",title) #,\"\\nPolicy URL:\",url)\n",
    "            print('****************************')\n",
    "            print(text) \n",
    "\n",
    "\n",
    "        else:\n",
    "            # Handle other content types if needed\n",
    "            print(\"Unsupported content type:\", content_type)   \n",
    "    else:\n",
    "        print(response.status_code())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "e0308158",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "39a43806",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "fe34e8ef",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "e7102f97",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "264a700c",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "cc254e5c",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "21350719",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "2c6b4107",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "7717e64e",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "8dcc719f",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "e032adac",
   "metadata": {},
   "outputs": [],
   "source": [
    "import requests\n",
    "from bs4 import BeautifulSoup\n",
    "\n",
    "# Function to extract title from HTML content\n",
    "def extract_title_from_html(html_content):\n",
    "    soup = BeautifulSoup(html_content, 'html.parser')\n",
    "    title_tag = soup.title\n",
    "    if title_tag:\n",
    "        return title_tag.text.strip()\n",
    "    else:\n",
    "        return None\n",
    "\n",
    "# Function to download PDF content\n",
    "def download_pdf(url, filename):\n",
    "    with open(filename, 'wb') as pdf_file:\n",
    "        response = requests.get(url)\n",
    "        pdf_file.write(response.content)\n",
    "\n",
    "# Function to handle URLs\n",
    "def handle_url(url):\n",
    "    response = requests.head(url)\n",
    "    content_type = response.headers.get('Content-Type', '')\n",
    "\n",
    "    # Ignore character encoding information in Content-Type header\n",
    "    content_type = content_type.split(';')[0]\n",
    "\n",
    "    if content_type == 'text/html':\n",
    "        # If the content type is HTML\n",
    "        html_content = requests.get(url).content\n",
    "        title = extract_title_from_html(html_content)\n",
    "        if title:\n",
    "            print(\"Title:\", title)\n",
    "        else:\n",
    "            print(\"No title found in the HTML.\")\n",
    "    \n",
    "    elif content_type == 'application/pdf':\n",
    "        # If the content type is PDF\n",
    "        pdf_filename = 'downloaded_pdf.pdf'  # You can customize the filename here\n",
    "        download_pdf(url, pdf_filename)\n",
    "        print(\"PDF downloaded as\", pdf_filename)\n",
    "    \n",
    "    else:\n",
    "        # Handle other content types if needed\n",
    "        print(\"Unsupported content type:\", content_type)\n",
    "\n",
    "# Example usage\n",
    "if __name__ == \"__main__\":\n",
    "    url = \"https://example.com\"  # Replace with the URL of the website you want to handle\n",
    "    handle_url(url)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "27981fe0",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "4ffc08e1",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "ef18e383",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "3071efd5",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "ea18a9f3",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "2fb22580",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "e4edc4a7",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "a78cfa53",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "fe34a08f",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "eb7e63e9",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "8c5544da",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "8eeaf534",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 155,
   "id": "4da4f9c6",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://salud.grupotriples.com/medicalpolicy/docs/11.003.035.html\n"
     ]
    }
   ],
   "source": [
    "print(url)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 159,
   "id": "166d8358",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "No title found in the HTML.\n"
     ]
    }
   ],
   "source": [
    "import requests\n",
    "from bs4 import BeautifulSoup\n",
    "\n",
    "# Function to extract title from HTML content\n",
    "def extract_title_from_html(url):\n",
    "    headers = {'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/118.0.0.0 Safari/537.36 Edg/118.0.2088.46'}\n",
    "    \n",
    "    response = requests.get(url, headers=headers)    \n",
    "    if response.status_code == 200:\n",
    "        html_content = response.text\n",
    "        soup = BeautifulSoup(html_content, 'html.parser')\n",
    "        title_tag = soup.title\n",
    "        if title_tag:\n",
    "            return title_tag.text.strip()\n",
    "        else:\n",
    "            return None\n",
    "\n",
    "# Function to fetch HTML content from a URL\n",
    "def fetch_html_content(url):\n",
    "    try:\n",
    "        headers = {'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/118.0.0.0 Safari/537.36 Edg/118.0.2088.46'}\n",
    "    \n",
    "        response = requests.get(url, headers=headers)    \n",
    "        response.raise_for_status()  # Raise HTTPError for bad requests\n",
    "        return response.content\n",
    "    except requests.exceptions.HTTPError as errh:\n",
    "        print(\"HTTP Error:\", errh)\n",
    "    except requests.exceptions.ConnectionError as errc:\n",
    "        print(\"Error Connecting:\", errc)\n",
    "    except requests.exceptions.Timeout as errt:\n",
    "        print(\"Timeout Error:\", errt)\n",
    "    except requests.exceptions.RequestException as err:\n",
    "        print(\"Oops! Something went wrong:\", err)\n",
    "    return None\n",
    "\n",
    "# Example usage\n",
    "if __name__ == \"__main__\":\n",
    "    url = url # Replace with the URL of the website you want to scrape\n",
    "    html_content = fetch_html_content(url)\n",
    "\n",
    "    if html_content:\n",
    "        title = extract_title_from_html(html_content)\n",
    "        if title:\n",
    "            print(\"Title:\", title)\n",
    "        else:\n",
    "            print(\"No title found in the HTML.\")\n",
    "    else:\n",
    "        print(\"Failed to fetch HTML content.\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 153,
   "id": "4cee11b2",
   "metadata": {},
   "outputs": [],
   "source": [
    "extract_title_from_html(url)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 199,
   "id": "3b54a5c3",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "89\n"
     ]
    }
   ],
   "source": [
    "print(len(soup.find_all('table')))\n",
    "table=soup.find_all('table')[0]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 201,
   "id": "74e814b1",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['Policy Num: \\xa0\\xa0\\xa0\\xa011.003.035\\nPolicy Name:\\xa0 \\xa0 Assays of Genetic Expression in Tumor Tissue as a Technique to Determine Prognosis in Patients with Breast Cancer\\nPolicy ID:\\xa0\\xa0\\xa0\\xa0\\xa0\\xa0\\xa0\\xa0\\xa0\\xa0[11.003.035]\\xa0 [Ac / B /\\xa0M+ / P+]\\xa0 [2.04.36]']"
      ]
     },
     "execution_count": 201,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "info = []\n",
    "a = soup.find_all('p')\n",
    "for i in a:\n",
    "    info.append([i.text.strip()])\n",
    "    \n",
    "info[1]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 200,
   "id": "69b7b1c0",
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[<p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Individuals:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Interventions of interest are:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Comparators of interest are:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Relevant outcomes include:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Individuals:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Interventions of interest are:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Comparators of interest are:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Relevant outcomes include:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Individuals:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Interventions of interest are:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Comparators of interest are:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Relevant outcomes include:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Individuals:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Interventions of interest are:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Comparators of interest are:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Relevant outcomes include:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Individuals:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Interventions of interest are:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Comparators of interest are:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Relevant outcomes include:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Individuals:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Interventions of interest are:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Comparators of interest are:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Relevant outcomes include:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Individuals:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Interventions of interest are:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Comparators of interest are:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Relevant outcomes include:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Individuals</strong>:</p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Interventions of interest are:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Comparators of interest are:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Relevant outcomes include:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Individuals:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Interventions of interest are:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Comparators of interest are:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Relevant outcomes include:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Individuals:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Interventions of interest are:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Comparators of interest are:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Relevant outcomes include:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Individuals:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Interventions of interest are:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Comparators of interest are:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Relevant outcomes include:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Individuals:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Interventions of interest are:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Comparators of interest are:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Relevant outcomes include:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Individuals:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Interventions of interest are:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Comparators of interest are:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Relevant outcomes include:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Individuals:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Interventions of interest are:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Comparators of interest are:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Relevant outcomes include:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Individuals:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Interventions of interest are:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Comparators of interest are:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Relevant outcomes include:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Individuals:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Interventions of interest are:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Comparators of interest are:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Relevant outcomes include:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Individuals:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Interventions of interest are:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Comparators of interest are:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Relevant outcomes include:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Individuals:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Interventions of interest are:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Comparators of interest are:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Relevant outcomes include:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Individuals:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Interventions of interest are:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Comparators of interest are:</strong></p>,\n",
       " <p style=\"line-height: 1.3; font-family: Arial !important; font-size: 10pt !important; color: rgb(0, 0, 0) !important; font-weight: normal !important;\"><strong>Relevant outcomes include:</strong></p>]"
      ]
     },
     "execution_count": 200,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "table.find_all('p')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 166,
   "id": "d1b0ff28",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<p><img src=\"https://services.ssspr.com/sssportal/images/logo-triple-s-salud-en.png\" style=\"width: 300px; float: right;\"/><strong style=\"color: rgb(52, 152, 219); font-size: 36px;\">Medical Policy</strong></p>"
      ]
     },
     "execution_count": 166,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "soup.find('p')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 165,
   "id": "85d5a0db",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "common method of title extraction :\n",
      " Codes\n"
     ]
    }
   ],
   "source": [
    "print(\"common method of title extraction :\\n\",soup.find(\"th\").text.strip())\n",
    "#print('Required title of doc: \\n',soup.find(class_='title-content').text.strip())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 221,
   "id": "cda28f3d",
   "metadata": {},
   "outputs": [
    {
     "ename": "ConnectionError",
     "evalue": "HTTPSConnectionPool(host='www.bcbsms.com', port=443): Max retries exceeded with url: /medical-policy-search (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000025538DAD250>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[1;31mTimeoutError\u001b[0m                              Traceback (most recent call last)",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\urllib3\\connection.py:174\u001b[0m, in \u001b[0;36mHTTPConnection._new_conn\u001b[1;34m(self)\u001b[0m\n\u001b[0;32m    173\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[1;32m--> 174\u001b[0m     conn \u001b[38;5;241m=\u001b[39m connection\u001b[38;5;241m.\u001b[39mcreate_connection(\n\u001b[0;32m    175\u001b[0m         (\u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_dns_host, \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mport), \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mtimeout, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39mextra_kw\n\u001b[0;32m    176\u001b[0m     )\n\u001b[0;32m    178\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m SocketTimeout:\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\urllib3\\util\\connection.py:95\u001b[0m, in \u001b[0;36mcreate_connection\u001b[1;34m(address, timeout, source_address, socket_options)\u001b[0m\n\u001b[0;32m     94\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m err \u001b[38;5;129;01mis\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m \u001b[38;5;28;01mNone\u001b[39;00m:\n\u001b[1;32m---> 95\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m err\n\u001b[0;32m     97\u001b[0m \u001b[38;5;28;01mraise\u001b[39;00m socket\u001b[38;5;241m.\u001b[39merror(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mgetaddrinfo returns an empty list\u001b[39m\u001b[38;5;124m\"\u001b[39m)\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\urllib3\\util\\connection.py:85\u001b[0m, in \u001b[0;36mcreate_connection\u001b[1;34m(address, timeout, source_address, socket_options)\u001b[0m\n\u001b[0;32m     84\u001b[0m     sock\u001b[38;5;241m.\u001b[39mbind(source_address)\n\u001b[1;32m---> 85\u001b[0m \u001b[43msock\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mconnect\u001b[49m\u001b[43m(\u001b[49m\u001b[43msa\u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m     86\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m sock\n",
      "\u001b[1;31mTimeoutError\u001b[0m: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond",
      "\nDuring handling of the above exception, another exception occurred:\n",
      "\u001b[1;31mNewConnectionError\u001b[0m                        Traceback (most recent call last)",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\urllib3\\connectionpool.py:703\u001b[0m, in \u001b[0;36mHTTPConnectionPool.urlopen\u001b[1;34m(self, method, url, body, headers, retries, redirect, assert_same_host, timeout, pool_timeout, release_conn, chunked, body_pos, **response_kw)\u001b[0m\n\u001b[0;32m    702\u001b[0m \u001b[38;5;66;03m# Make the request on the httplib connection object.\u001b[39;00m\n\u001b[1;32m--> 703\u001b[0m httplib_response \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_make_request\u001b[49m\u001b[43m(\u001b[49m\n\u001b[0;32m    704\u001b[0m \u001b[43m    \u001b[49m\u001b[43mconn\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    705\u001b[0m \u001b[43m    \u001b[49m\u001b[43mmethod\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    706\u001b[0m \u001b[43m    \u001b[49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    707\u001b[0m \u001b[43m    \u001b[49m\u001b[43mtimeout\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mtimeout_obj\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    708\u001b[0m \u001b[43m    \u001b[49m\u001b[43mbody\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mbody\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    709\u001b[0m \u001b[43m    \u001b[49m\u001b[43mheaders\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mheaders\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    710\u001b[0m \u001b[43m    \u001b[49m\u001b[43mchunked\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mchunked\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    711\u001b[0m \u001b[43m\u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m    713\u001b[0m \u001b[38;5;66;03m# If we're going to release the connection in ``finally:``, then\u001b[39;00m\n\u001b[0;32m    714\u001b[0m \u001b[38;5;66;03m# the response doesn't need to know about the connection. Otherwise\u001b[39;00m\n\u001b[0;32m    715\u001b[0m \u001b[38;5;66;03m# it will also try to release it and we'll have a double-release\u001b[39;00m\n\u001b[0;32m    716\u001b[0m \u001b[38;5;66;03m# mess.\u001b[39;00m\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\urllib3\\connectionpool.py:386\u001b[0m, in \u001b[0;36mHTTPConnectionPool._make_request\u001b[1;34m(self, conn, method, url, timeout, chunked, **httplib_request_kw)\u001b[0m\n\u001b[0;32m    385\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[1;32m--> 386\u001b[0m     \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_validate_conn\u001b[49m\u001b[43m(\u001b[49m\u001b[43mconn\u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m    387\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m (SocketTimeout, BaseSSLError) \u001b[38;5;28;01mas\u001b[39;00m e:\n\u001b[0;32m    388\u001b[0m     \u001b[38;5;66;03m# Py2 raises this as a BaseSSLError, Py3 raises it as socket timeout.\u001b[39;00m\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\urllib3\\connectionpool.py:1042\u001b[0m, in \u001b[0;36mHTTPSConnectionPool._validate_conn\u001b[1;34m(self, conn)\u001b[0m\n\u001b[0;32m   1041\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m \u001b[38;5;28mgetattr\u001b[39m(conn, \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124msock\u001b[39m\u001b[38;5;124m\"\u001b[39m, \u001b[38;5;28;01mNone\u001b[39;00m):  \u001b[38;5;66;03m# AppEngine might not have  `.sock`\u001b[39;00m\n\u001b[1;32m-> 1042\u001b[0m     \u001b[43mconn\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mconnect\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m   1044\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m conn\u001b[38;5;241m.\u001b[39mis_verified:\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\urllib3\\connection.py:363\u001b[0m, in \u001b[0;36mHTTPSConnection.connect\u001b[1;34m(self)\u001b[0m\n\u001b[0;32m    361\u001b[0m \u001b[38;5;28;01mdef\u001b[39;00m \u001b[38;5;21mconnect\u001b[39m(\u001b[38;5;28mself\u001b[39m):\n\u001b[0;32m    362\u001b[0m     \u001b[38;5;66;03m# Add certificate verification\u001b[39;00m\n\u001b[1;32m--> 363\u001b[0m     \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39msock \u001b[38;5;241m=\u001b[39m conn \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_new_conn\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m    364\u001b[0m     hostname \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mhost\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\urllib3\\connection.py:186\u001b[0m, in \u001b[0;36mHTTPConnection._new_conn\u001b[1;34m(self)\u001b[0m\n\u001b[0;32m    185\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m SocketError \u001b[38;5;28;01mas\u001b[39;00m e:\n\u001b[1;32m--> 186\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m NewConnectionError(\n\u001b[0;32m    187\u001b[0m         \u001b[38;5;28mself\u001b[39m, \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mFailed to establish a new connection: \u001b[39m\u001b[38;5;132;01m%s\u001b[39;00m\u001b[38;5;124m\"\u001b[39m \u001b[38;5;241m%\u001b[39m e\n\u001b[0;32m    188\u001b[0m     )\n\u001b[0;32m    190\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m conn\n",
      "\u001b[1;31mNewConnectionError\u001b[0m: <urllib3.connection.HTTPSConnection object at 0x0000025538DAD250>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond",
      "\nDuring handling of the above exception, another exception occurred:\n",
      "\u001b[1;31mMaxRetryError\u001b[0m                             Traceback (most recent call last)",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\requests\\adapters.py:489\u001b[0m, in \u001b[0;36mHTTPAdapter.send\u001b[1;34m(self, request, stream, timeout, verify, cert, proxies)\u001b[0m\n\u001b[0;32m    488\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m chunked:\n\u001b[1;32m--> 489\u001b[0m     resp \u001b[38;5;241m=\u001b[39m \u001b[43mconn\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43murlopen\u001b[49m\u001b[43m(\u001b[49m\n\u001b[0;32m    490\u001b[0m \u001b[43m        \u001b[49m\u001b[43mmethod\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mrequest\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mmethod\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    491\u001b[0m \u001b[43m        \u001b[49m\u001b[43murl\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    492\u001b[0m \u001b[43m        \u001b[49m\u001b[43mbody\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mrequest\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mbody\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    493\u001b[0m \u001b[43m        \u001b[49m\u001b[43mheaders\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mrequest\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mheaders\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    494\u001b[0m \u001b[43m        \u001b[49m\u001b[43mredirect\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43;01mFalse\u001b[39;49;00m\u001b[43m,\u001b[49m\n\u001b[0;32m    495\u001b[0m \u001b[43m        \u001b[49m\u001b[43massert_same_host\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43;01mFalse\u001b[39;49;00m\u001b[43m,\u001b[49m\n\u001b[0;32m    496\u001b[0m \u001b[43m        \u001b[49m\u001b[43mpreload_content\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43;01mFalse\u001b[39;49;00m\u001b[43m,\u001b[49m\n\u001b[0;32m    497\u001b[0m \u001b[43m        \u001b[49m\u001b[43mdecode_content\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43;01mFalse\u001b[39;49;00m\u001b[43m,\u001b[49m\n\u001b[0;32m    498\u001b[0m \u001b[43m        \u001b[49m\u001b[43mretries\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mmax_retries\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    499\u001b[0m \u001b[43m        \u001b[49m\u001b[43mtimeout\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mtimeout\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    500\u001b[0m \u001b[43m    \u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m    502\u001b[0m \u001b[38;5;66;03m# Send the request.\u001b[39;00m\n\u001b[0;32m    503\u001b[0m \u001b[38;5;28;01melse\u001b[39;00m:\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\urllib3\\connectionpool.py:787\u001b[0m, in \u001b[0;36mHTTPConnectionPool.urlopen\u001b[1;34m(self, method, url, body, headers, retries, redirect, assert_same_host, timeout, pool_timeout, release_conn, chunked, body_pos, **response_kw)\u001b[0m\n\u001b[0;32m    785\u001b[0m     e \u001b[38;5;241m=\u001b[39m ProtocolError(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mConnection aborted.\u001b[39m\u001b[38;5;124m\"\u001b[39m, e)\n\u001b[1;32m--> 787\u001b[0m retries \u001b[38;5;241m=\u001b[39m \u001b[43mretries\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mincrement\u001b[49m\u001b[43m(\u001b[49m\n\u001b[0;32m    788\u001b[0m \u001b[43m    \u001b[49m\u001b[43mmethod\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43merror\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43me\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43m_pool\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43mself\u001b[39;49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43m_stacktrace\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43msys\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mexc_info\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\u001b[43m[\u001b[49m\u001b[38;5;241;43m2\u001b[39;49m\u001b[43m]\u001b[49m\n\u001b[0;32m    789\u001b[0m \u001b[43m\u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m    790\u001b[0m retries\u001b[38;5;241m.\u001b[39msleep()\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\urllib3\\util\\retry.py:592\u001b[0m, in \u001b[0;36mRetry.increment\u001b[1;34m(self, method, url, response, error, _pool, _stacktrace)\u001b[0m\n\u001b[0;32m    591\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m new_retry\u001b[38;5;241m.\u001b[39mis_exhausted():\n\u001b[1;32m--> 592\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m MaxRetryError(_pool, url, error \u001b[38;5;129;01mor\u001b[39;00m ResponseError(cause))\n\u001b[0;32m    594\u001b[0m log\u001b[38;5;241m.\u001b[39mdebug(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mIncremented Retry for (url=\u001b[39m\u001b[38;5;124m'\u001b[39m\u001b[38;5;132;01m%s\u001b[39;00m\u001b[38;5;124m'\u001b[39m\u001b[38;5;124m): \u001b[39m\u001b[38;5;132;01m%r\u001b[39;00m\u001b[38;5;124m\"\u001b[39m, url, new_retry)\n",
      "\u001b[1;31mMaxRetryError\u001b[0m: HTTPSConnectionPool(host='www.bcbsms.com', port=443): Max retries exceeded with url: /medical-policy-search (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000025538DAD250>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))",
      "\nDuring handling of the above exception, another exception occurred:\n",
      "\u001b[1;31mConnectionError\u001b[0m                           Traceback (most recent call last)",
      "Cell \u001b[1;32mIn[221], line 3\u001b[0m\n\u001b[0;32m      1\u001b[0m headers \u001b[38;5;241m=\u001b[39m {\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mUser-Agent\u001b[39m\u001b[38;5;124m'\u001b[39m: \u001b[38;5;124m'\u001b[39m\u001b[38;5;124mMozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/118.0.0.0 Safari/537.36 Edg/118.0.2088.46\u001b[39m\u001b[38;5;124m'\u001b[39m}\n\u001b[0;32m      2\u001b[0m url\u001b[38;5;241m=\u001b[39m\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mhttps://www.bcbsms.com/medical-policy-search#/policy-detail?id=6a9931dd-4769-44e9-86e3-690fe0dfd2f7\u001b[39m\u001b[38;5;124m'\u001b[39m\n\u001b[1;32m----> 3\u001b[0m response \u001b[38;5;241m=\u001b[39m \u001b[43mrequests\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mget\u001b[49m\u001b[43m(\u001b[49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mheaders\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mheaders\u001b[49m\u001b[43m)\u001b[49m    \n\u001b[0;32m      4\u001b[0m \u001b[38;5;28mprint\u001b[39m(response\u001b[38;5;241m.\u001b[39mstatus_code)\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\requests\\api.py:73\u001b[0m, in \u001b[0;36mget\u001b[1;34m(url, params, **kwargs)\u001b[0m\n\u001b[0;32m     62\u001b[0m \u001b[38;5;28;01mdef\u001b[39;00m \u001b[38;5;21mget\u001b[39m(url, params\u001b[38;5;241m=\u001b[39m\u001b[38;5;28;01mNone\u001b[39;00m, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39mkwargs):\n\u001b[0;32m     63\u001b[0m     \u001b[38;5;124mr\u001b[39m\u001b[38;5;124;03m\"\"\"Sends a GET request.\u001b[39;00m\n\u001b[0;32m     64\u001b[0m \n\u001b[0;32m     65\u001b[0m \u001b[38;5;124;03m    :param url: URL for the new :class:`Request` object.\u001b[39;00m\n\u001b[1;32m   (...)\u001b[0m\n\u001b[0;32m     70\u001b[0m \u001b[38;5;124;03m    :rtype: requests.Response\u001b[39;00m\n\u001b[0;32m     71\u001b[0m \u001b[38;5;124;03m    \"\"\"\u001b[39;00m\n\u001b[1;32m---> 73\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m request(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mget\u001b[39m\u001b[38;5;124m\"\u001b[39m, url, params\u001b[38;5;241m=\u001b[39mparams, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39mkwargs)\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\requests\\api.py:59\u001b[0m, in \u001b[0;36mrequest\u001b[1;34m(method, url, **kwargs)\u001b[0m\n\u001b[0;32m     55\u001b[0m \u001b[38;5;66;03m# By using the 'with' statement we are sure the session is closed, thus we\u001b[39;00m\n\u001b[0;32m     56\u001b[0m \u001b[38;5;66;03m# avoid leaving sockets open which can trigger a ResourceWarning in some\u001b[39;00m\n\u001b[0;32m     57\u001b[0m \u001b[38;5;66;03m# cases, and look like a memory leak in others.\u001b[39;00m\n\u001b[0;32m     58\u001b[0m \u001b[38;5;28;01mwith\u001b[39;00m sessions\u001b[38;5;241m.\u001b[39mSession() \u001b[38;5;28;01mas\u001b[39;00m session:\n\u001b[1;32m---> 59\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m session\u001b[38;5;241m.\u001b[39mrequest(method\u001b[38;5;241m=\u001b[39mmethod, url\u001b[38;5;241m=\u001b[39murl, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39mkwargs)\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\requests\\sessions.py:587\u001b[0m, in \u001b[0;36mSession.request\u001b[1;34m(self, method, url, params, data, headers, cookies, files, auth, timeout, allow_redirects, proxies, hooks, stream, verify, cert, json)\u001b[0m\n\u001b[0;32m    582\u001b[0m send_kwargs \u001b[38;5;241m=\u001b[39m {\n\u001b[0;32m    583\u001b[0m     \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mtimeout\u001b[39m\u001b[38;5;124m\"\u001b[39m: timeout,\n\u001b[0;32m    584\u001b[0m     \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mallow_redirects\u001b[39m\u001b[38;5;124m\"\u001b[39m: allow_redirects,\n\u001b[0;32m    585\u001b[0m }\n\u001b[0;32m    586\u001b[0m send_kwargs\u001b[38;5;241m.\u001b[39mupdate(settings)\n\u001b[1;32m--> 587\u001b[0m resp \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39msend(prep, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39msend_kwargs)\n\u001b[0;32m    589\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m resp\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\requests\\sessions.py:701\u001b[0m, in \u001b[0;36mSession.send\u001b[1;34m(self, request, **kwargs)\u001b[0m\n\u001b[0;32m    698\u001b[0m start \u001b[38;5;241m=\u001b[39m preferred_clock()\n\u001b[0;32m    700\u001b[0m \u001b[38;5;66;03m# Send the request\u001b[39;00m\n\u001b[1;32m--> 701\u001b[0m r \u001b[38;5;241m=\u001b[39m adapter\u001b[38;5;241m.\u001b[39msend(request, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39mkwargs)\n\u001b[0;32m    703\u001b[0m \u001b[38;5;66;03m# Total elapsed time of the request (approximately)\u001b[39;00m\n\u001b[0;32m    704\u001b[0m elapsed \u001b[38;5;241m=\u001b[39m preferred_clock() \u001b[38;5;241m-\u001b[39m start\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\requests\\adapters.py:565\u001b[0m, in \u001b[0;36mHTTPAdapter.send\u001b[1;34m(self, request, stream, timeout, verify, cert, proxies)\u001b[0m\n\u001b[0;32m    561\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28misinstance\u001b[39m(e\u001b[38;5;241m.\u001b[39mreason, _SSLError):\n\u001b[0;32m    562\u001b[0m         \u001b[38;5;66;03m# This branch is for urllib3 v1.22 and later.\u001b[39;00m\n\u001b[0;32m    563\u001b[0m         \u001b[38;5;28;01mraise\u001b[39;00m SSLError(e, request\u001b[38;5;241m=\u001b[39mrequest)\n\u001b[1;32m--> 565\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m \u001b[38;5;167;01mConnectionError\u001b[39;00m(e, request\u001b[38;5;241m=\u001b[39mrequest)\n\u001b[0;32m    567\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m ClosedPoolError \u001b[38;5;28;01mas\u001b[39;00m e:\n\u001b[0;32m    568\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m \u001b[38;5;167;01mConnectionError\u001b[39;00m(e, request\u001b[38;5;241m=\u001b[39mrequest)\n",
      "\u001b[1;31mConnectionError\u001b[0m: HTTPSConnectionPool(host='www.bcbsms.com', port=443): Max retries exceeded with url: /medical-policy-search (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000025538DAD250>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))"
     ]
    }
   ],
   "source": [
    "headers = {'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/118.0.0.0 Safari/537.36 Edg/118.0.2088.46'}\n",
    "url='https://www.bcbsms.com/medical-policy-search#/policy-detail?id=6a9931dd-4769-44e9-86e3-690fe0dfd2f7'\n",
    "response = requests.get(url, headers=headers)    \n",
    "print(response.status_code)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 100,
   "id": "e306bfcf",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Index(['Unnamed: 0', 'Keywords', 'Payors/LBM', 'Policy url',\n",
       "       'Policy URL Correct/Incorrect Flag', '\"Covered/Yes\" Language',\n",
       "       '\"Not Covered/No\" Language', 'Vague / non-No',\n",
       "       '\"Conditional/Need PA\" language', 'Notable patterns within document',\n",
       "       'Unnamed: 9', 'extracted text', 'Validity Status'],\n",
       "      dtype='object')"
      ]
     },
     "execution_count": 100,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data_filtered.columns"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "36edbc17",
   "metadata": {},
   "source": [
    "# PDF files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "77fb1214",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Unnamed: 0</th>\n",
       "      <th>Keywords</th>\n",
       "      <th>Payors/LBM</th>\n",
       "      <th>Policy url</th>\n",
       "      <th>Policy URL Correct/Incorrect Flag</th>\n",
       "      <th>\"Covered/Yes\" Language</th>\n",
       "      <th>\"Not Covered/No\" Language</th>\n",
       "      <th>Vague / non-No</th>\n",
       "      <th>\"Conditional/Need PA\" language</th>\n",
       "      <th>Notable patterns within document</th>\n",
       "      <th>Unnamed: 9</th>\n",
       "      <th>extracted text</th>\n",
       "      <th>Validity Status</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>190</th>\n",
       "      <td>190</td>\n",
       "      <td>81528 OR Cologuard OR DNA analysis of stool to...</td>\n",
       "      <td>Anthem</td>\n",
       "      <td>https://providers.anthem.com/docs/gpp/KY_CAID_...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>Coverage can be inferred in this policy as Col...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GENE.00052 Whole Genome Sequencing, Whole Exom...</td>\n",
       "      <td>PDF - Working</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>294</th>\n",
       "      <td>294</td>\n",
       "      <td>81416 OR Exome sequencing OR WES OR Exome sequ...</td>\n",
       "      <td>BCBSFL</td>\n",
       "      <td>https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&amp;...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>GENE.00052 Whole Genome Sequencing, Whole Exom...</td>\n",
       "      <td>PDF - Working</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "     Unnamed: 0                                           Keywords Payors/LBM  \\\n",
       "190         190  81528 OR Cologuard OR DNA analysis of stool to...     Anthem   \n",
       "294         294  81416 OR Exome sequencing OR WES OR Exome sequ...     BCBSFL   \n",
       "\n",
       "                                            Policy url  \\\n",
       "190  https://providers.anthem.com/docs/gpp/KY_CAID_...   \n",
       "294  https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&...   \n",
       "\n",
       "     Policy URL Correct/Incorrect Flag  \\\n",
       "190                                NaN   \n",
       "294                                NaN   \n",
       "\n",
       "                                \"Covered/Yes\" Language  \\\n",
       "190  Coverage can be inferred in this policy as Col...   \n",
       "294                                                NaN   \n",
       "\n",
       "    \"Not Covered/No\" Language Vague / non-No \"Conditional/Need PA\" language  \\\n",
       "190                       NaN            NaN                            NaN   \n",
       "294                       NaN            NaN                            NaN   \n",
       "\n",
       "    Notable patterns within document Unnamed: 9  \\\n",
       "190                              NaN        NaN   \n",
       "294                              NaN        NaN   \n",
       "\n",
       "                                        extracted text Validity Status  \n",
       "190  GENE.00052 Whole Genome Sequencing, Whole Exom...   PDF - Working  \n",
       "294  GENE.00052 Whole Genome Sequencing, Whole Exom...   PDF - Working  "
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data_pdf=data_filtered[data_filtered['Validity Status']=='PDF - Working'].copy()\n",
    "data_pdf.head(2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "id": "6619a9b7",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(bluecrossma OR BCBS Massachusetts)                                                          147\n",
       "regence                                                                                      147\n",
       "eviCore                                                                                      147\n",
       "(Asuris  OR Asuris Northwest Health)                                                         146\n",
       "Tufts Health Plan                                                                            142\n",
       "BCBS Michigan                                                                                142\n",
       "(Providence Health Plan OR providencehealthplan)                                             142\n",
       "(UHC OR UnitedHealthcare)                                                                    141\n",
       "(FEPBlue  OR BCBS Federal Employee Plan)                                                     137\n",
       "(Presbyterian Healthcare Services)                                                           137\n",
       "emblemhealth                                                                                 135\n",
       "(bridgespanhealth OR Bridgespan Health)                                                      135\n",
       "(modahealth OR Moda Health)                                                                  134\n",
       "(Healthnet OR Centene)                                                                       134\n",
       "(\"BSCA\" OR \"blueshieldca\" OR Blue Shield of California)                                      133\n",
       "Geisinger                                                                                    132\n",
       "(medica OR Medica Health Plans)                                                              131\n",
       "Avalon hcs                                                                                   127\n",
       "BCBSFL                                                                                       127\n",
       "prevea360                                                                                    126\n",
       "Cigna                                                                                        125\n",
       "(bcbsks OR BCBS Kansas)                                                                      121\n",
       "(BCBSLA OR BCBS Louisiana)                                                                   117\n",
       "AIM Specialty Health/Carelon Medical Benefits                                                112\n",
       "(bcbsri OR BCBS Rhode Island)                                                                108\n",
       "(ssmhealth OR SSM Health)                                                                    108\n",
       "Wellmark                                                                                      98\n",
       "Univera Healthcare                                                                            96\n",
       "(wellsense OR BMC Healthnet)                                                                  92\n",
       "BCBSNC                                                                                        90\n",
       "(bcidaho OR Blue Cross Idaho)                                                                 75\n",
       "(massgeneralbrighamhealthplan OR  Mass General Brigham Health OR AllWays Health Partners)     47\n",
       "(keystonefirstpa OR Keystone First)                                                           20\n",
       "(capbluecross OR Capital Blue Cross)                                                           1\n",
       "(arkansasblue OR BCBS Arkansas)                                                                1\n",
       "Anthem                                                                                         1\n",
       "Name: Payors/LBM, dtype: int64"
      ]
     },
     "execution_count": 32,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data_pdf[\"Payors/LBM\"].value_counts()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 209,
   "id": "b36a2ebd",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "https://www.bluecrossma.org/medical-policies/sites/g/files/csphws2091/files/acquiadam-assets/957%20AIM%20Genetic%20Testing%20Management%20Program%20CPT%20and%20HCPCS%20Codes%20prn.pdf\n"
     ]
    }
   ],
   "source": [
    "pdf_url=[]\n",
    "for i in data_pdf[data_pdf[\"Payors/LBM\"]=='(bluecrossma OR BCBS Massachusetts)']['Policy url']:\n",
    "    pdf_url.append(i)\n",
    "    \n",
    "url=pdf_url[4] \n",
    "print(url)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 211,
   "id": "1a15427a",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Text from https://providers.anthem.com/docs/gpp/KY_CAID_ColorectalCancerScreening.pdf?v=202301031849:\n",
      "Anthem Blue Cross and Blue Shield Medicaid \n",
      " \n",
      "Provider Bulletin \n",
      "December 2022 \n",
      "1 Centers for Disease Control and Prevention. An Update on Cancer Deaths in the United States. Atlanta, GA: US Department of Health and Human \n",
      "Services, Centers for Disease Control and Prevention, Division of Cancer Prevention and Control; 2022. \n",
      "2 American Cancer Society. Colorectal Cancer Facts & Figures 2020-2022. Atlanta, American Cancer Society; 2020. \n",
      " \n",
      " \n",
      "HEDIS® is a registered trademark of the National Committee for Quality Assurance (NCQA). \n",
      "https://providers.anthem.com/ky  \n",
      "Anthem Blue Cross and Blue Shield Medicaid is the trade name of Anthem Kentucky Managed Care Plan, Inc., independent licensee of the Blue Cross \n",
      "and Blue Shield Association. Anthem is a registered trademark of Anthem Insurance Companies, Inc.  \n",
      "KYBCBS-CD-008128-22 December 2022 \n",
      "Colorectal cancer screening and the benefits of early detection \n",
      " \n",
      "Colorectal cancer is the second leading cause of cancer deaths in the United States. Colon and rectal cancers \n",
      "accounted for 9% of total cancer death in 2020.¹ The occurrences of colorectal cancer have decreased over time \n",
      "due to early detection. Colorectal cancer begins with a polyp or growth on the inner lining of the colon or \n",
      "rectum. Polyps can take as long as 10 to 15 years to develop into cancer. Through visual screenings, polyps can \n",
      "be removed before they have a chance to develop into cancer. \n",
      " \n",
      "Unfortunately, only 4 out of 10 colorectal cancers are found at an early stage. Once the cancer has spread \n",
      "beyond the colon or rectum, the survival rate decreases.² \n",
      "  \n",
      "Begin screening at 45 \n",
      "While most colorectal cancer diagnoses are among people aged 50 and older, incidence is rising in younger \n",
      "adults. To address these gaps and support more equitable care, NCQA recently announced several important \n",
      "updates to the widely used Colorectal Cancer Screening HEDIS® measure. It is now recommended to start \n",
      "colorectal cancer screening at 45 instead of 50. To include broader populations, NCQA has updated the \n",
      "specifications of the measure to include Medicaid recipients. \n",
      " \n",
      "Educate patients about the importance of early detection  \n",
      "Discuss the importance of colorectal cancer screenings and ensure your patients are up to date with their \n",
      "colorectal cancer screening. Ensure medical record documentation includes past medical, surgical history, and \n",
      "procedures including dates and results when possible.  \n",
      "Types of screenings \n",
      "Stool-based tests – home test kits: \n",
      "• Fecal immunochemical test (FIT) every year. \n",
      "• Guaiac-based fecal occult blood test (gFOBT) every year. \n",
      "• Multi-targeted stool DNA test (Cologuard) every three years. \n",
      "Visual exams: \n",
      "• Colonoscopy every 10 years. \n",
      "• CT colonography (visual colonoscopy) every five years. \n",
      "• Flexible sigmoidoscopy every five years. \n",
      "If a person chooses to be screened with a test other than a colonoscopy, an abnormal test result should be \n",
      "followed up with a timely colonoscopy.  \n",
      " \n",
      "Anthem Blue Cross and Blue Shield Medicaid  \n",
      "Colorectal cancer screening and the benefits of early detection \n",
      "Page 2 of 2 \n",
      " \n",
      " \n",
      "Screening codes \n",
      "Screening \n",
      "CPT® codes \n",
      "HCPCS \n",
      "Flexible sigmoidoscopy \n",
      "45330—45335, 45337-45342, 45346, 45347, \n",
      "45349, 45350 \n",
      "G0104 \n",
      "Occult blood test \n",
      "(FOBT, FIT) \n",
      "82270, 82274 \n",
      "G0328 \n",
      "Colonoscopy \n",
      "44388-44394, 44401-44408, 45378-45386, \n",
      "45386, 45398, 45388-45639 \n",
      "G0105, G0121 \n",
      "CT Colonography \n",
      "74261, 74262 \n",
      " \n",
      "FIT-DNA \n",
      "81528 \n",
      " \n",
      " \n",
      " \n",
      " \n",
      " \n",
      " \n",
      "Email is the quickest and most direct way to receive important  \n",
      "information from Anthem Blue Cross and Blue Shield Medicaid. \n",
      "To start receiving email from us (including some sent in lieu of fax or mail), submit your  \n",
      "information using the QR code to the right or via our online form (https://bit.ly/3lZ9oEj). \n",
      "\n",
      "Failed to extract text from https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false. Error: HTTPSConnectionPool(host='mcgs.bcbsfl.com', port=443): Max retries exceeded with url: /MCG?mcgId=05-82000-28&pv=false (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000025538E49550>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))\n",
      "Failed to extract text from https://mcgs.bcbsfl.com/MCG?mcgId=05-86000-22&pv=false. Error: HTTPSConnectionPool(host='mcgs.bcbsfl.com', port=443): Max retries exceeded with url: /MCG?mcgId=05-86000-22&pv=false (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000025538E40EE0>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))\n",
      "Failed to extract text from http://mcgs.bcbsfl.com/MCG?mcgId=05-86000-26&pv=false. Error: HTTPConnectionPool(host='mcgs.bcbsfl.com', port=80): Max retries exceeded with url: /MCG?mcgId=05-86000-26&pv=false (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000025538E4CF40>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))\n",
      "Failed to extract text from http://mcgs.bcbsfl.com/MCG?mcgId=05-86000-26&pv=false. Error: HTTPConnectionPool(host='mcgs.bcbsfl.com', port=80): Max retries exceeded with url: /MCG?mcgId=05-86000-26&pv=false (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000025538E4C490>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))\n",
      "Failed to extract text from https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false. Error: HTTPSConnectionPool(host='mcgs.bcbsfl.com', port=443): Max retries exceeded with url: /MCG?mcgId=05-82000-28&pv=false (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000025538E40CA0>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))\n",
      "Failed to extract text from https://mcgs.bcbsfl.com/MCG?mcgId=05-86000-22&pv=false. Error: HTTPSConnectionPool(host='mcgs.bcbsfl.com', port=443): Max retries exceeded with url: /MCG?mcgId=05-86000-22&pv=false (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000025538E49E20>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))\n",
      "Failed to extract text from https://mcgs.bcbsfl.com/MCG?mcgId=05-86000-22&pv=false. Error: HTTPSConnectionPool(host='mcgs.bcbsfl.com', port=443): Max retries exceeded with url: /MCG?mcgId=05-86000-22&pv=false (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000025536BD4130>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))\n",
      "Failed to extract text from http://mcgs.bcbsfl.com/MCG?mcgId=05-86000-26&pv=false. Error: HTTPConnectionPool(host='mcgs.bcbsfl.com', port=80): Max retries exceeded with url: /MCG?mcgId=05-86000-26&pv=false (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000025538EA9760>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Failed to extract text from http://mcgs.bcbsfl.com/MCG?mcgId=05-86000-29&pv=false. Error: HTTPConnectionPool(host='mcgs.bcbsfl.com', port=80): Max retries exceeded with url: /MCG?mcgId=05-86000-29&pv=false (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000025538EA9F70>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))\n",
      "Failed to extract text from https://mcgs.bcbsfl.com/MCG?mcgId=05-86000-22&pv=false. Error: HTTPSConnectionPool(host='mcgs.bcbsfl.com', port=443): Max retries exceeded with url: /MCG?mcgId=05-86000-22&pv=false (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000025538EB17C0>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))\n",
      "Failed to extract text from https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false. Error: HTTPSConnectionPool(host='mcgs.bcbsfl.com', port=443): Max retries exceeded with url: /MCG?mcgId=05-82000-28&pv=false (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000025536BD4190>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))\n",
      "Failed to extract text from https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false. Error: HTTPSConnectionPool(host='mcgs.bcbsfl.com', port=443): Max retries exceeded with url: /MCG?mcgId=05-82000-28&pv=false (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000025538E49FA0>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))\n",
      "Failed to extract text from https://mcgs.bcbsfl.com/MCG?mcgId=05-86000-22&pv=false. Error: HTTPSConnectionPool(host='mcgs.bcbsfl.com', port=443): Max retries exceeded with url: /MCG?mcgId=05-86000-22&pv=false (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000025538EA9850>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))\n",
      "Failed to extract text from https://mcgs.bcbsfl.com/MCG?mcgId=05-82000-28&pv=false. Error: HTTPSConnectionPool(host='mcgs.bcbsfl.com', port=443): Max retries exceeded with url: /MCG?mcgId=05-82000-28&pv=false (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000025538EA9250>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))\n",
      "Failed to extract text from http://mcgs.bcbsfl.com/MCG?mcgId=05-82000-30&pv=false. Error: HTTPConnectionPool(host='mcgs.bcbsfl.com', port=80): Max retries exceeded with url: /MCG?mcgId=05-82000-30&pv=false (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000025538E4C430>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))\n"
     ]
    },
    {
     "ename": "KeyboardInterrupt",
     "evalue": "",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[1;31mTimeoutError\u001b[0m                              Traceback (most recent call last)",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\urllib3\\util\\connection.py:85\u001b[0m, in \u001b[0;36mcreate_connection\u001b[1;34m(address, timeout, source_address, socket_options)\u001b[0m\n\u001b[0;32m     84\u001b[0m     sock\u001b[38;5;241m.\u001b[39mbind(source_address)\n\u001b[1;32m---> 85\u001b[0m \u001b[43msock\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mconnect\u001b[49m\u001b[43m(\u001b[49m\u001b[43msa\u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m     86\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m sock\n",
      "\u001b[1;31mTimeoutError\u001b[0m: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond",
      "\nDuring handling of the above exception, another exception occurred:\n",
      "\u001b[1;31mKeyboardInterrupt\u001b[0m                         Traceback (most recent call last)",
      "Cell \u001b[1;32mIn[211], line 8\u001b[0m\n\u001b[0;32m      5\u001b[0m url \u001b[38;5;241m=\u001b[39m row[\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mPolicy url\u001b[39m\u001b[38;5;124m'\u001b[39m]  \u001b[38;5;66;03m# Get the URL\u001b[39;00m\n\u001b[0;32m      7\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[1;32m----> 8\u001b[0m     \u001b[38;5;28;01mwith\u001b[39;00m \u001b[43mrequests\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mget\u001b[49m\u001b[43m(\u001b[49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mstream\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43;01mTrue\u001b[39;49;00m\u001b[43m)\u001b[49m \u001b[38;5;28;01mas\u001b[39;00m response:\n\u001b[0;32m      9\u001b[0m         response\u001b[38;5;241m.\u001b[39mraise_for_status()\n\u001b[0;32m     10\u001b[0m         pdf_bytes \u001b[38;5;241m=\u001b[39m \u001b[38;5;124mb\u001b[39m\u001b[38;5;124m'\u001b[39m\u001b[38;5;124m'\u001b[39m\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\requests\\api.py:73\u001b[0m, in \u001b[0;36mget\u001b[1;34m(url, params, **kwargs)\u001b[0m\n\u001b[0;32m     62\u001b[0m \u001b[38;5;28;01mdef\u001b[39;00m \u001b[38;5;21mget\u001b[39m(url, params\u001b[38;5;241m=\u001b[39m\u001b[38;5;28;01mNone\u001b[39;00m, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39mkwargs):\n\u001b[0;32m     63\u001b[0m     \u001b[38;5;124mr\u001b[39m\u001b[38;5;124;03m\"\"\"Sends a GET request.\u001b[39;00m\n\u001b[0;32m     64\u001b[0m \n\u001b[0;32m     65\u001b[0m \u001b[38;5;124;03m    :param url: URL for the new :class:`Request` object.\u001b[39;00m\n\u001b[1;32m   (...)\u001b[0m\n\u001b[0;32m     70\u001b[0m \u001b[38;5;124;03m    :rtype: requests.Response\u001b[39;00m\n\u001b[0;32m     71\u001b[0m \u001b[38;5;124;03m    \"\"\"\u001b[39;00m\n\u001b[1;32m---> 73\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m request(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mget\u001b[39m\u001b[38;5;124m\"\u001b[39m, url, params\u001b[38;5;241m=\u001b[39mparams, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39mkwargs)\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\requests\\api.py:59\u001b[0m, in \u001b[0;36mrequest\u001b[1;34m(method, url, **kwargs)\u001b[0m\n\u001b[0;32m     55\u001b[0m \u001b[38;5;66;03m# By using the 'with' statement we are sure the session is closed, thus we\u001b[39;00m\n\u001b[0;32m     56\u001b[0m \u001b[38;5;66;03m# avoid leaving sockets open which can trigger a ResourceWarning in some\u001b[39;00m\n\u001b[0;32m     57\u001b[0m \u001b[38;5;66;03m# cases, and look like a memory leak in others.\u001b[39;00m\n\u001b[0;32m     58\u001b[0m \u001b[38;5;28;01mwith\u001b[39;00m sessions\u001b[38;5;241m.\u001b[39mSession() \u001b[38;5;28;01mas\u001b[39;00m session:\n\u001b[1;32m---> 59\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m session\u001b[38;5;241m.\u001b[39mrequest(method\u001b[38;5;241m=\u001b[39mmethod, url\u001b[38;5;241m=\u001b[39murl, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39mkwargs)\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\requests\\sessions.py:587\u001b[0m, in \u001b[0;36mSession.request\u001b[1;34m(self, method, url, params, data, headers, cookies, files, auth, timeout, allow_redirects, proxies, hooks, stream, verify, cert, json)\u001b[0m\n\u001b[0;32m    582\u001b[0m send_kwargs \u001b[38;5;241m=\u001b[39m {\n\u001b[0;32m    583\u001b[0m     \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mtimeout\u001b[39m\u001b[38;5;124m\"\u001b[39m: timeout,\n\u001b[0;32m    584\u001b[0m     \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mallow_redirects\u001b[39m\u001b[38;5;124m\"\u001b[39m: allow_redirects,\n\u001b[0;32m    585\u001b[0m }\n\u001b[0;32m    586\u001b[0m send_kwargs\u001b[38;5;241m.\u001b[39mupdate(settings)\n\u001b[1;32m--> 587\u001b[0m resp \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39msend(prep, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39msend_kwargs)\n\u001b[0;32m    589\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m resp\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\requests\\sessions.py:701\u001b[0m, in \u001b[0;36mSession.send\u001b[1;34m(self, request, **kwargs)\u001b[0m\n\u001b[0;32m    698\u001b[0m start \u001b[38;5;241m=\u001b[39m preferred_clock()\n\u001b[0;32m    700\u001b[0m \u001b[38;5;66;03m# Send the request\u001b[39;00m\n\u001b[1;32m--> 701\u001b[0m r \u001b[38;5;241m=\u001b[39m adapter\u001b[38;5;241m.\u001b[39msend(request, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39mkwargs)\n\u001b[0;32m    703\u001b[0m \u001b[38;5;66;03m# Total elapsed time of the request (approximately)\u001b[39;00m\n\u001b[0;32m    704\u001b[0m elapsed \u001b[38;5;241m=\u001b[39m preferred_clock() \u001b[38;5;241m-\u001b[39m start\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\requests\\adapters.py:489\u001b[0m, in \u001b[0;36mHTTPAdapter.send\u001b[1;34m(self, request, stream, timeout, verify, cert, proxies)\u001b[0m\n\u001b[0;32m    487\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[0;32m    488\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m chunked:\n\u001b[1;32m--> 489\u001b[0m         resp \u001b[38;5;241m=\u001b[39m \u001b[43mconn\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43murlopen\u001b[49m\u001b[43m(\u001b[49m\n\u001b[0;32m    490\u001b[0m \u001b[43m            \u001b[49m\u001b[43mmethod\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mrequest\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mmethod\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    491\u001b[0m \u001b[43m            \u001b[49m\u001b[43murl\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    492\u001b[0m \u001b[43m            \u001b[49m\u001b[43mbody\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mrequest\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mbody\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    493\u001b[0m \u001b[43m            \u001b[49m\u001b[43mheaders\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mrequest\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mheaders\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    494\u001b[0m \u001b[43m            \u001b[49m\u001b[43mredirect\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43;01mFalse\u001b[39;49;00m\u001b[43m,\u001b[49m\n\u001b[0;32m    495\u001b[0m \u001b[43m            \u001b[49m\u001b[43massert_same_host\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43;01mFalse\u001b[39;49;00m\u001b[43m,\u001b[49m\n\u001b[0;32m    496\u001b[0m \u001b[43m            \u001b[49m\u001b[43mpreload_content\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43;01mFalse\u001b[39;49;00m\u001b[43m,\u001b[49m\n\u001b[0;32m    497\u001b[0m \u001b[43m            \u001b[49m\u001b[43mdecode_content\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43;01mFalse\u001b[39;49;00m\u001b[43m,\u001b[49m\n\u001b[0;32m    498\u001b[0m \u001b[43m            \u001b[49m\u001b[43mretries\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mmax_retries\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    499\u001b[0m \u001b[43m            \u001b[49m\u001b[43mtimeout\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mtimeout\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    500\u001b[0m \u001b[43m        \u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m    502\u001b[0m     \u001b[38;5;66;03m# Send the request.\u001b[39;00m\n\u001b[0;32m    503\u001b[0m     \u001b[38;5;28;01melse\u001b[39;00m:\n\u001b[0;32m    504\u001b[0m         \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;28mhasattr\u001b[39m(conn, \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mproxy_pool\u001b[39m\u001b[38;5;124m\"\u001b[39m):\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\urllib3\\connectionpool.py:703\u001b[0m, in \u001b[0;36mHTTPConnectionPool.urlopen\u001b[1;34m(self, method, url, body, headers, retries, redirect, assert_same_host, timeout, pool_timeout, release_conn, chunked, body_pos, **response_kw)\u001b[0m\n\u001b[0;32m    700\u001b[0m     \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_prepare_proxy(conn)\n\u001b[0;32m    702\u001b[0m \u001b[38;5;66;03m# Make the request on the httplib connection object.\u001b[39;00m\n\u001b[1;32m--> 703\u001b[0m httplib_response \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_make_request\u001b[49m\u001b[43m(\u001b[49m\n\u001b[0;32m    704\u001b[0m \u001b[43m    \u001b[49m\u001b[43mconn\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    705\u001b[0m \u001b[43m    \u001b[49m\u001b[43mmethod\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    706\u001b[0m \u001b[43m    \u001b[49m\u001b[43murl\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    707\u001b[0m \u001b[43m    \u001b[49m\u001b[43mtimeout\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mtimeout_obj\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    708\u001b[0m \u001b[43m    \u001b[49m\u001b[43mbody\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mbody\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    709\u001b[0m \u001b[43m    \u001b[49m\u001b[43mheaders\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mheaders\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    710\u001b[0m \u001b[43m    \u001b[49m\u001b[43mchunked\u001b[49m\u001b[38;5;241;43m=\u001b[39;49m\u001b[43mchunked\u001b[49m\u001b[43m,\u001b[49m\n\u001b[0;32m    711\u001b[0m \u001b[43m\u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m    713\u001b[0m \u001b[38;5;66;03m# If we're going to release the connection in ``finally:``, then\u001b[39;00m\n\u001b[0;32m    714\u001b[0m \u001b[38;5;66;03m# the response doesn't need to know about the connection. Otherwise\u001b[39;00m\n\u001b[0;32m    715\u001b[0m \u001b[38;5;66;03m# it will also try to release it and we'll have a double-release\u001b[39;00m\n\u001b[0;32m    716\u001b[0m \u001b[38;5;66;03m# mess.\u001b[39;00m\n\u001b[0;32m    717\u001b[0m response_conn \u001b[38;5;241m=\u001b[39m conn \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m release_conn \u001b[38;5;28;01melse\u001b[39;00m \u001b[38;5;28;01mNone\u001b[39;00m\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\urllib3\\connectionpool.py:386\u001b[0m, in \u001b[0;36mHTTPConnectionPool._make_request\u001b[1;34m(self, conn, method, url, timeout, chunked, **httplib_request_kw)\u001b[0m\n\u001b[0;32m    384\u001b[0m \u001b[38;5;66;03m# Trigger any extra validation we need to do.\u001b[39;00m\n\u001b[0;32m    385\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[1;32m--> 386\u001b[0m     \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_validate_conn\u001b[49m\u001b[43m(\u001b[49m\u001b[43mconn\u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m    387\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m (SocketTimeout, BaseSSLError) \u001b[38;5;28;01mas\u001b[39;00m e:\n\u001b[0;32m    388\u001b[0m     \u001b[38;5;66;03m# Py2 raises this as a BaseSSLError, Py3 raises it as socket timeout.\u001b[39;00m\n\u001b[0;32m    389\u001b[0m     \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_raise_timeout(err\u001b[38;5;241m=\u001b[39me, url\u001b[38;5;241m=\u001b[39murl, timeout_value\u001b[38;5;241m=\u001b[39mconn\u001b[38;5;241m.\u001b[39mtimeout)\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\urllib3\\connectionpool.py:1042\u001b[0m, in \u001b[0;36mHTTPSConnectionPool._validate_conn\u001b[1;34m(self, conn)\u001b[0m\n\u001b[0;32m   1040\u001b[0m \u001b[38;5;66;03m# Force connect early to allow us to validate the connection.\u001b[39;00m\n\u001b[0;32m   1041\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m \u001b[38;5;28mgetattr\u001b[39m(conn, \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124msock\u001b[39m\u001b[38;5;124m\"\u001b[39m, \u001b[38;5;28;01mNone\u001b[39;00m):  \u001b[38;5;66;03m# AppEngine might not have  `.sock`\u001b[39;00m\n\u001b[1;32m-> 1042\u001b[0m     \u001b[43mconn\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mconnect\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m   1044\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m conn\u001b[38;5;241m.\u001b[39mis_verified:\n\u001b[0;32m   1045\u001b[0m     warnings\u001b[38;5;241m.\u001b[39mwarn(\n\u001b[0;32m   1046\u001b[0m         (\n\u001b[0;32m   1047\u001b[0m             \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mUnverified HTTPS request is being made to host \u001b[39m\u001b[38;5;124m'\u001b[39m\u001b[38;5;132;01m%s\u001b[39;00m\u001b[38;5;124m'\u001b[39m\u001b[38;5;124m. \u001b[39m\u001b[38;5;124m\"\u001b[39m\n\u001b[1;32m   (...)\u001b[0m\n\u001b[0;32m   1052\u001b[0m         InsecureRequestWarning,\n\u001b[0;32m   1053\u001b[0m     )\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\urllib3\\connection.py:363\u001b[0m, in \u001b[0;36mHTTPSConnection.connect\u001b[1;34m(self)\u001b[0m\n\u001b[0;32m    361\u001b[0m \u001b[38;5;28;01mdef\u001b[39;00m \u001b[38;5;21mconnect\u001b[39m(\u001b[38;5;28mself\u001b[39m):\n\u001b[0;32m    362\u001b[0m     \u001b[38;5;66;03m# Add certificate verification\u001b[39;00m\n\u001b[1;32m--> 363\u001b[0m     \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39msock \u001b[38;5;241m=\u001b[39m conn \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;43mself\u001b[39;49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43m_new_conn\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m    364\u001b[0m     hostname \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mhost\n\u001b[0;32m    365\u001b[0m     tls_in_tls \u001b[38;5;241m=\u001b[39m \u001b[38;5;28;01mFalse\u001b[39;00m\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\urllib3\\connection.py:174\u001b[0m, in \u001b[0;36mHTTPConnection._new_conn\u001b[1;34m(self)\u001b[0m\n\u001b[0;32m    171\u001b[0m     extra_kw[\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124msocket_options\u001b[39m\u001b[38;5;124m\"\u001b[39m] \u001b[38;5;241m=\u001b[39m \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39msocket_options\n\u001b[0;32m    173\u001b[0m \u001b[38;5;28;01mtry\u001b[39;00m:\n\u001b[1;32m--> 174\u001b[0m     conn \u001b[38;5;241m=\u001b[39m connection\u001b[38;5;241m.\u001b[39mcreate_connection(\n\u001b[0;32m    175\u001b[0m         (\u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39m_dns_host, \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mport), \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mtimeout, \u001b[38;5;241m*\u001b[39m\u001b[38;5;241m*\u001b[39mextra_kw\n\u001b[0;32m    176\u001b[0m     )\n\u001b[0;32m    178\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m SocketTimeout:\n\u001b[0;32m    179\u001b[0m     \u001b[38;5;28;01mraise\u001b[39;00m ConnectTimeoutError(\n\u001b[0;32m    180\u001b[0m         \u001b[38;5;28mself\u001b[39m,\n\u001b[0;32m    181\u001b[0m         \u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mConnection to \u001b[39m\u001b[38;5;132;01m%s\u001b[39;00m\u001b[38;5;124m timed out. (connect timeout=\u001b[39m\u001b[38;5;132;01m%s\u001b[39;00m\u001b[38;5;124m)\u001b[39m\u001b[38;5;124m\"\u001b[39m\n\u001b[0;32m    182\u001b[0m         \u001b[38;5;241m%\u001b[39m (\u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mhost, \u001b[38;5;28mself\u001b[39m\u001b[38;5;241m.\u001b[39mtimeout),\n\u001b[0;32m    183\u001b[0m     )\n",
      "File \u001b[1;32m~\\anaconda3\\lib\\site-packages\\urllib3\\util\\connection.py:85\u001b[0m, in \u001b[0;36mcreate_connection\u001b[1;34m(address, timeout, source_address, socket_options)\u001b[0m\n\u001b[0;32m     83\u001b[0m     \u001b[38;5;28;01mif\u001b[39;00m source_address:\n\u001b[0;32m     84\u001b[0m         sock\u001b[38;5;241m.\u001b[39mbind(source_address)\n\u001b[1;32m---> 85\u001b[0m     \u001b[43msock\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mconnect\u001b[49m\u001b[43m(\u001b[49m\u001b[43msa\u001b[49m\u001b[43m)\u001b[49m\n\u001b[0;32m     86\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m sock\n\u001b[0;32m     88\u001b[0m \u001b[38;5;28;01mexcept\u001b[39;00m socket\u001b[38;5;241m.\u001b[39merror \u001b[38;5;28;01mas\u001b[39;00m e:\n",
      "\u001b[1;31mKeyboardInterrupt\u001b[0m: "
     ]
    }
   ],
   "source": [
    "import requests\n",
    "import fitz\n",
    "\n",
    "for index, row in data_pdf.iterrows():\n",
    "    url = row['Policy url']  # Get the URL\n",
    "\n",
    "    try:\n",
    "        with requests.get(url, stream=True) as response:\n",
    "            response.raise_for_status()\n",
    "            pdf_bytes = b''\n",
    "            for chunk in response.iter_content(chunk_size=8192):\n",
    "                pdf_bytes += chunk\n",
    "\n",
    "        pdf_document = fitz.open(stream=pdf_bytes, filetype=\"pdf\")\n",
    "\n",
    "        text = \"\"\n",
    "        for page_num in range(pdf_document.page_count):\n",
    "            page = pdf_document.load_page(page_num)\n",
    "            text += page.get_text()\n",
    "\n",
    "        print(f\"Text from {url}:\")\n",
    "        print(text)\n",
    "\n",
    "    except Exception as e:\n",
    "        print(f\"Failed to extract text from {url}. Error: {str(e)}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "f2eddf02",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "f442dbd3",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "0f6bcd14",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "67f8a9f0",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "b9ac2cf8",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "efcb1fff",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "162996a5",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "04726a5c",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "1b99aef8",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "63d20287",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "21b513ae",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "af792358",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "2e675b17",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "2f231a8c",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "09580a58",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "c9ba7fa0",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "0893fc9f",
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "import requests\n",
    "from bs4 import BeautifulSoup\n",
    "\n",
    "# Function to scrape content from a URL\n",
    "def scrape_content(url):\n",
    "    response = requests.get(url)\n",
    "    soup = BeautifulSoup(response.content, 'html.parser')\n",
    "\n",
    "    # Extract title\n",
    "    title = soup.title.text if soup.title else 'No Title Found'\n",
    "\n",
    "    # Extract date (modify the selector based on the actual structure of the website)\n",
    "    date_element = soup.find('div', class_='date')\n",
    "    date = date_element.text.strip() if date_element else 'No Date Found'\n",
    "\n",
    "    # Extract other content (modify the selector based on the actual structure of the website)\n",
    "    content_element = soup.find('div', class_='content')\n",
    "    content = content_element.text.strip() if content_element else 'No Content Found'\n",
    "\n",
    "    return title, date, content\n",
    "\n",
    "# URLs of the websites you want to scrape\n",
    "urls = [\n",
    "    'https://example.com/page1',\n",
    "    'https://example.com/page2',\n",
    "    # Add more URLs as needed\n",
    "]\n",
    "\n",
    "# Iterate through the URLs and scrape content\n",
    "for url in urls:\n",
    "    title, date, content = scrape_content(url)\n",
    "\n",
    "    # Print or process the extracted data as needed\n",
    "    print(f'URL: {url}')\n",
    "    print(f'Title: {title}')\n",
    "    print(f'Date: {date}')\n",
    "    print(f'Content: {content}')\n",
    "    print('---')\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.12"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
